# Textual diff for go.obo

```diff
--- go-lastrelease.obo	2026-02-20 05:51:13.350471618 +0000
+++ go.obo	2026-02-20 06:00:13.531551901 +0000
@@ -1,5 +1,5 @@
 format-version: 1.2
-data-version: releases/2026-01-23
+data-version: releases/2026-02-20
 subsetdef: chebi_ph7_3 "Rhea list of ChEBI terms representing the major species at pH 7.3."
 subsetdef: gocheck_do_not_annotate "Term not to be used for direct annotation"
 subsetdef: gocheck_obsoletion_candidate "Terms planned for obsoletion"
@@ -44,7 +44,7 @@
 property_value: has_ontology_root_term GO:0003674
 property_value: has_ontology_root_term GO:0005575
 property_value: has_ontology_root_term GO:0008150
-property_value: owl:versionInfo "2026-01-23" xsd:string
+property_value: owl:versionInfo "2026-02-20" xsd:string
 property_value: terms:license http://creativecommons.org/licenses/by/4.0/
 
 [Term]
@@ -490,7 +490,7 @@
 id: GO:0000048
 name: peptidyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2). This reaction is catalyzed by a ribozyme." [PMID:11433365, PMID:9242921]
+def: "Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2). This reaction is catalyzed by a ribozyme." [EC:2.3.2.12, PMID:11433365, PMID:9242921]
 xref: EC:2.3.2.12
 xref: MetaCyc:PEPTIDYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-156912 "Peptide transfer from P-site tRNA to the A-site tRNA"
@@ -509,6 +509,7 @@
 alt_id: GO:0000946
 def: "Binding to a transfer RNA." [GOC:ai]
 synonym: "base pairing with tRNA" NARROW []
+xref: Reactome:R-HSA-2408509 "Sec-tRNA(Sec) binds to EEFSEC:GTP"
 is_a: GO:0003723 ! RNA binding
 
 [Term]
@@ -661,6 +662,7 @@
 synonym: "histidine/arginine/lysine/ornithine porter activity" RELATED []
 synonym: "L-ornithine transporter activity" BROAD []
 xref: Reactome:R-HSA-70634 "ornithine (cytosolic) + citrulline (mitochondrial) => ornithine (mitochondrial) + citrulline (cytosolic)"
+xref: Reactome:R-HSA-9956519 "SLC25A15 variants don't translocate ornithine and citrulline"
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
 relationship: part_of GO:1903352 ! L-ornithine transmembrane transport
 
@@ -967,19 +969,20 @@
 id: GO:0000096
 name: sulfur amino acid metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving amino acids containing sulfur, comprising cysteine, homocysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways involving amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid metabolism" EXACT []
 synonym: "sulphur amino acid metabolic process" EXACT []
 synonym: "sulphur amino acid metabolism" EXACT []
 xref: Reactome:R-HSA-1614635 "Sulfur amino acid metabolism"
 is_a: GO:0006790 ! sulfur compound metabolic process
 is_a: GO:0019752 ! carboxylic acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000097
 name: sulfur amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the formation of amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid anabolism" EXACT []
 synonym: "sulfur amino acid biosynthesis" EXACT []
 synonym: "sulfur amino acid formation" EXACT []
@@ -989,12 +992,13 @@
 is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0044272 ! sulfur compound biosynthetic process
 is_a: GO:0046394 ! carboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000098
 name: sulfur amino acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid breakdown" EXACT []
 synonym: "sulfur amino acid catabolism" EXACT []
 synonym: "sulfur amino acid degradation" EXACT []
@@ -1004,6 +1008,7 @@
 is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0044273 ! sulfur compound catabolic process
 is_a: GO:0046395 ! carboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000099
@@ -1090,10 +1095,14 @@
 synonym: "histidine formation" EXACT []
 synonym: "histidine synthesis" EXACT []
 xref: MetaCyc:HISTSYN-PWY
+xref: MetaCyc:PWY-5029
 is_a: GO:0006547 ! L-histidine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:HISTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5029
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0000107
@@ -1770,7 +1779,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the hydroxylation of sphingolipid long chain bases." [PMID:9556590]
 comment: The reason for obsoletion is that this term is equivalent to sphingolipid C4-monooxygenase activity.
-xref: Reactome:R-HSA-428260 "DEGS2 oxygenates dihydroceramide"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29717" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0102772
@@ -1816,12 +1824,14 @@
 synonym: "3'-5'-exoribonuclease activity" EXACT []
 xref: Reactome:R-HSA-430028 "Exosome Complex hydrolyzes mRNA by 3' to 5' exoribonuclease digestion"
 xref: Reactome:R-HSA-6791222 "21S pre-rRNA is nucleolytically processed at site E (site2a) to yield 18SE pre-rRNA"
-xref: Reactome:R-HSA-6791227 "47S pre-rRNA is nucleolytically processed at A' (01,A1), site A0, and site 02 (site 6)  to yield 45S pre-rRNA"
 xref: Reactome:R-HSA-9682603 "nsp14 acts as a 3'-to-5' exonuclease to remove misincorporated nucleotides from nascent RNA"
 xref: Reactome:R-HSA-9694632 "nsp14 acts as a 3'-to-5' exonuclease to remove misincorporated nucleotides from nascent RNA"
 xref: Reactome:R-HSA-9822437 "DIS3L2 hydrolyzes uridylated mRNA"
 xref: Reactome:R-HSA-9836585 "SUPV3L1:PNPT1 hydrolyzes mitochondrial RNA to yield 4-5 nucleotide oligoribonucleotides"
 xref: Reactome:R-HSA-9836822 "REXO2 hydrolyzes 4-5 nucleotide RNAs to ribonucleotides"
+xref: Reactome:R-HSA-9933585 "RNA exosome in PAXT:PAPOLG:TENT4A,B:ZC3H3:CBCA:polyadenylated m7GpppN-RNA:PABPN1 hydrolyzes RNA"
+xref: Reactome:R-HSA-9934763 "The RNA exosome in RNA exosome:NEXT:ZC3H18:CBCA:m7GpppN-RNA hydrolyzes RNA"
+xref: Reactome:R-HSA-9935824 "TRAMP:RNA exosome hydrolyzes oligoadenylated pre-rRNA 5' external transcribed spacer (5'ETS)"
 is_a: GO:0008408 ! 3'-5' exonuclease activity
 is_a: GO:0016896 ! RNA exonuclease activity, producing 5'-phosphomonoesters
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28062" xsd:anyURI
@@ -2798,6 +2808,8 @@
 xref: EC:1.1.1.170
 xref: Reactome:R-HSA-194642 "4-methyl,4-carboxycholesta-8(9),24-dien-3beta-ol is decarboxylated and oxidized to form 4-methylcholesta-8(9),24-dien-3-one"
 xref: Reactome:R-HSA-194718 "4-carboxycholesta-8(9),24-dien-3beta-ol is decarboxylated and oxidized to form cholesta-8(9),24-dien-3-one (zymosterone)"
+xref: Reactome:R-HSA-9945787 "NSDHL decarboxylates 4a-carboxy-5a-cholest-8-ene-3b-ol to 5a-cholest-8-en-3-one"
+xref: Reactome:R-HSA-9945822 "NSDHL decarboxylates 4a-carboxy-4b-methyl-5a-cholest-8-en-3b-ol to 4a-methyl-5a-cholest-8-en-3b-ol"
 xref: RHEA:34771
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.170
@@ -2818,6 +2830,8 @@
 xref: MetaCyc:RXN-13686
 xref: Reactome:R-HSA-194632 "Zymosterone (cholesta-8(9),24-dien-3-one) is reduced to zymosterol (cholesta-8(9),24-dien-3beta-ol)"
 xref: Reactome:R-HSA-194689 "4-methylcholesta-8(9),24-dien-3-one is reduced to 4-methylcholesta-8(9),24-dien-3beta-ol"
+xref: Reactome:R-HSA-9945784 "HSD17B7 reduces 4a-methyl-5a-cholest-8-en-3-one to 4a-methyl-5a-cholest-8-en-3b-ol"
+xref: Reactome:R-HSA-9945804 "HSD17B7 reduces 5a-cholest-8-en-3-one to ZYMSTNL"
 xref: RHEA:18409
 xref: RHEA:34787
 xref: RHEA:36379
@@ -2841,6 +2855,8 @@
 xref: MetaCyc:RXN-13188
 xref: Reactome:R-HSA-194641 "4,4-dimethylcholesta-8(9),24-dien-3beta-ol is oxidized to 4-methyl,4-carboxycholesta-8(9),24-dien-3beta-ol"
 xref: Reactome:R-HSA-194669 "4-methylcholesta-8(9),24-dien-3beta-ol is oxidized to 4-carboxycholesta-8(9),24-dien-3beta-ol"
+xref: Reactome:R-HSA-9947203 "MSMO1 oxidizes 4,4-diMe-5-cholest-8-en-3-ol to 4a-carboxy-4b-me-5a-cholest-8-en-3b-ol"
+xref: Reactome:R-HSA-9947207 "MSMO1 oxidizes 4a-me-5a-cholest-8-en-3b-ol to 4a-carboxy-5a-cholest-8-ene-3b-ol"
 xref: RHEA:55220
 is_a: GO:0016716 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, another compound as one donor, and incorporation of one atom of oxygen
 property_value: skos:exactMatch EC:1.14.18.9
@@ -2872,7 +2888,7 @@
 id: GO:0000257
 name: nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH3. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides." [EC:3.5.5.1, GOC:kd]
+def: "Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH4+. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides." [RHEA:21724]
 synonym: "acetonitrilase activity" RELATED [EC:3.5.5.1]
 synonym: "benzonitrilase activity" RELATED [EC:3.5.5.1]
 synonym: "nitrile aminohydrolase activity" RELATED [EC:3.5.5.1]
@@ -4113,6 +4129,7 @@
 namespace: molecular_function
 def: "Catalysis of the first transesterification reaction of spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, while it is still joined to the 3' exon." [GOC:krc, ISBN:0879695897, PMID:19239890]
 synonym: "lariat formation, 5'-splice site cleavage" RELATED []
+xref: Reactome:R-HSA-9794542 "Formation of the Spliceosomal C complex containing intron lariat"
 is_a: GO:0140098 ! catalytic activity, acting on RNA
 relationship: part_of GO:0000375 ! RNA splicing, via transesterification reactions
 
@@ -7448,6 +7465,7 @@
 synonym: "RNA polymerase II promoter proximal region sequence-specific DNA binding" RELATED []
 synonym: "RNA polymerase II proximal promoter sequence-specific DNA binding" RELATED []
 synonym: "RNA polymerase II upstream activating sequence (UAS) sequence-specific DNA binding" RELATED []
+xref: Reactome:R-HSA-400204 "PPARA binds RXRA"
 is_a: GO:0000977 ! RNA polymerase II transcription regulatory region sequence-specific DNA binding
 is_a: GO:0000987 ! cis-regulatory region sequence-specific DNA binding
 created_by: krc
@@ -7612,7 +7630,6 @@
 def: "Binding to an RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits." [GOC:txnOH]
 synonym: "RNA polymerase II core binding" EXACT []
 synonym: "RNAP II core binding" EXACT []
-is_a: GO:0001099 ! basal RNA polymerase II transcription machinery binding
 is_a: GO:0043175 ! RNA polymerase core enzyme binding
 created_by: krc
 creation_date: 2010-08-11T03:57:27Z
@@ -7631,7 +7648,7 @@
 name: RNA polymerase III general transcription initiation factor activity
 namespace: molecular_function
 alt_id: GO:0001034
-def: "A general transcription initiation factor activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase III. Factors required for RNA polymerase III transcription initiation include TFIIIA, TFIIIB and TFIIIC. RNA polymerase III transcribes genes encoding short RNAs, including tRNAs, 5S rRNA, U6 snRNA, the short ncRNA component of RNases P, the mitochondrial RNA processing (MRP) RNA, the signal recognition particle SRP RNA, and in higher eukaryotes a number of micro and other small RNAs, though there is some variability across species as to whether a given small noncoding RNA is transcribed by RNA polymerase II or RNA polymerase III." [GOC:txnOH-2018, PMID:12381659, PMID:17977614, PMID:20413673, PMID:27068803, Wikipedia:RNA_polymerase_III]
+def: "A general transcription initiation factor activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase III. Factors required for RNA polymerase III transcription initiation include TFIIIA, TFIIIB and TFIIIC. RNA polymerase III transcribes genes encoding short RNAs, including tRNAs, 5S rRNA, U6 snRNA, the short ncRNA component of RNases P, the mitochondrial RNA processing (MRP) RNA, the signal recognition particle SRP RNA, and in higher eukaryotes a number of micro and other small RNAs, though there is some variability across species as to whether a given small noncoding RNA is transcribed by RNA polymerase II or RNA polymerase III." [GOC:txnOH-2018, PMID:12381659, PMID:17977614, PMID:20413673, PMID:27068803, PMID:40762516, Wikipedia:RNA_polymerase_III]
 synonym: "core RNA polymerase III binding transcription factor activity" NARROW []
 synonym: "RNA polymerase III general initiation factor activity" EXACT []
 synonym: "RNA polymerase III transcription factor activity, sequence-specific DNA binding" NARROW []
@@ -7640,6 +7657,7 @@
 is_a: GO:0140223 ! general transcription initiation factor activity
 relationship: part_of GO:0006383 ! transcription by RNA polymerase III
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15789" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15862" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20253" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22622" xsd:anyURI
 created_by: krc
@@ -7747,14 +7765,17 @@
 
 [Term]
 id: GO:0001010
-name: RNA polymerase II sequence-specific DNA-binding transcription factor recruiting activity
+name: obsolete RNA polymerase II sequence-specific DNA-binding transcription factor recruiting activity
 namespace: molecular_function
-def: "The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions." [GOC:txnOH]
+def: "OBSOLETE. The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions." [GOC:txnOH]
+comment: This term was obsoleted because it was not clearly defined.
 synonym: "sequence-specific DNA binding transcription factor recruiting transcription factor activity" EXACT []
 synonym: "transcription factor activity, sequence-specific DNA binding transcription factor recruiting" EXACT []
 synonym: "transcription factor activity, sequence-specific DNA-binding transcription factor recruiting" RELATED []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
-relationship: has_part GO:0061629 ! RNA polymerase II-specific DNA-binding transcription factor binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31456" xsd:anyURI
+is_obsolete: true
+consider: GO:0003712
+consider: GO:0140463
 created_by: krc
 creation_date: 2010-08-18T06:46:33Z
 
@@ -8014,8 +8035,6 @@
 def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
 comment: The reason for obsoletion is that this term is equivalent to DNA-directed 5'-3' RNA polymerase activity.
 synonym: "DNA-directed RNA polymerase activity involved in transcription from RNA polymerase II promoter" EXACT []
-xref: Reactome:R-HSA-6814549 "Pre-snRNA transcript initiation, Integrator binding, LEC binding"
-xref: Reactome:R-HSA-6814559 "Pre-snRNA is elongated and capped with 7-methylguanosine"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23779" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
 is_obsolete: true
@@ -8030,7 +8049,6 @@
 def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase III specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
 comment: The reason for obsoletion is that this term is equivalent to DNA-directed 5'-3' RNA polymerase activity.
 synonym: "DNA-directed RNA polymerase activity involved in transcription from RNA polymerase III promoter" EXACT []
-xref: Reactome:R-HSA-1964482 "RNA polymerase III transcribes microbial dsDNA to dsRNA"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23779" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
 is_obsolete: true
@@ -8393,7 +8411,7 @@
 namespace: molecular_function
 def: "Binding to a basal RNA polymerase II transcription factor, any of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:txnOH, PMID:16858867]
 synonym: "RNA polymerase II basal transcription factor binding" EXACT []
-is_a: GO:0001099 ! basal RNA polymerase II transcription machinery binding
+is_a: GO:0000993 ! RNA polymerase II complex binding
 is_a: GO:0140296 ! general transcription initiation factor binding
 created_by: krc
 creation_date: 2010-10-28T02:30:02Z
@@ -8405,7 +8423,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIA complex, one of the complexes involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
 synonym: "TFIIA-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:37:19Z
 
@@ -8425,7 +8442,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIID complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIID-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:48:33Z
 
@@ -8436,7 +8452,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIE complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIE-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:49:20Z
 
@@ -8447,7 +8462,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIF complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIF-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:51:20Z
 
@@ -8458,28 +8472,30 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIH complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIH-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:51:41Z
 
 [Term]
 id: GO:0001098
-name: basal transcription machinery binding
+name: obsolete basal transcription machinery binding
 namespace: molecular_function
-def: "Binding to a component of the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
-comment: Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.
-is_a: GO:0005515 ! protein binding
+def: "OBSOLETE. Binding to a component of the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
+comment: This term was obsoleted because it is an unnecessary grouping term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16062" xsd:anyURI
+is_obsolete: true
 created_by: krc
 creation_date: 2010-11-24T12:50:49Z
 
 [Term]
 id: GO:0001099
-name: basal RNA polymerase II transcription machinery binding
+name: obsolete basal RNA polymerase II transcription machinery binding
 namespace: molecular_function
-def: "Binding to a component of the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
-comment: Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.
+def: "OBSOLETE. Binding to a component of the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
+comment: This term was obsoleted because it is equivalent to RNA polymerase II complex binding ; GO:0000993.
 synonym: "basal RNAP II transcription machinery binding" EXACT []
-is_a: GO:0001098 ! basal transcription machinery binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16062" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0000993
 created_by: krc
 creation_date: 2010-11-24T12:54:33Z
 
@@ -8516,7 +8532,7 @@
 namespace: molecular_function
 def: "Binding to a component of the basal transcription machinery which is composed of a bacterial-type RNA polymerase core enzyme and a sigma factor, the minimal set of factors required for formation of the preinitiation complex (PIC) by a bacterial-type RNA polymerase." [GOC:txnOH]
 synonym: "basal bacterial-type RNA polymerase transcription machinery binding" EXACT []
-is_a: GO:0001098 ! basal transcription machinery binding
+is_a: GO:0070063 ! RNA polymerase binding
 created_by: krc
 creation_date: 2010-11-30T04:26:04Z
 
@@ -8742,14 +8758,15 @@
 
 [Term]
 id: GO:0001139
-name: RNA polymerase II complex recruiting activity
+name: obsolete RNA polymerase II complex recruiting activity
 namespace: molecular_function
-def: "Binding to an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
+def: "OBSOLETE. Binding to an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
+comment: This term was obsoleted because it is now represented by transcription coactivator activity ; GO:0003713.
 synonym: "core RNA polymerase II recruiting transcription factor activity" EXACT []
 synonym: "transcription factor activity, core RNA polymerase II recruiting" RELATED []
-is_a: GO:0000993 ! RNA polymerase II complex binding
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
-relationship: part_of GO:0051123 ! RNA polymerase II preinitiation complex assembly
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31456" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0003713
 created_by: krc
 creation_date: 2011-01-20T05:55:01Z
 
@@ -9365,6 +9382,7 @@
 synonym: "transcriptional repressor activity, RNA polymerase II distal enhancer sequence-specific binding" NARROW []
 synonym: "transcriptional repressor activity, RNA polymerase II proximal promoter sequence-specific DNA binding" NARROW []
 synonym: "transcriptional repressor activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding" EXACT []
+xref: Reactome:R-HSA-8936851 "AHRR binds ARNT"
 is_a: GO:0000981 ! DNA-binding transcription factor activity, RNA polymerase II-specific
 is_a: GO:0001217 ! DNA-binding transcription repressor activity
 intersection_of: GO:0140110 ! transcription regulator activity
@@ -9410,6 +9428,7 @@
 synonym: "transcriptional activator activity, RNA polymerase II proximal promoter sequence-specific DNA binding" NARROW []
 synonym: "transcriptional activator activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding" EXACT []
 synonym: "zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription" NARROW []
+xref: Reactome:R-HSA-8937177 "AHR:TCDD binds ARNT"
 xref: Reactome:R-HSA-9856539 "MLXIPL:MLX binds PKLR gene promoter"
 xref: Reactome:R-HSA-9856546 "MLXIPL:MLX binds FASN gene promoter"
 xref: Reactome:R-HSA-9856548 "MLXIPL:MLX binds ACACB gene promoter"
@@ -10046,7 +10065,7 @@
 def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-glucosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-glucosamine 6-sulfate." [GOC:ai, GOC:hjd]
 subset: goslim_chembl
 synonym: "N-acetylglucosamine 6-O-sulphotransferase activity" EXACT []
-xref: Reactome:R-HSA-2046222 "CHST2,5,6 transfer SO4(2-) to GlcNAc residues on keratan-PG to form KSPG"
+xref: Reactome:R-HSA-2046222 "CHST2,3,5,6 transfer sulfate to GlcNAc on keratan chain"
 xref: Reactome:R-HSA-3656269 "Defective CHST6 does not transfer SO4(2-) to GlcNAc residues on keratan-PG"
 xref: Reactome:R-HSA-6786012 "CHST4 transfers SO4(2-) from PAPS to Core 2 mucins"
 is_a: GO:0008146 ! sulfotransferase activity
@@ -10999,6 +11018,7 @@
 subset: goslim_chembl
 subset: goslim_generic
 synonym: "viral receptor activity" EXACT []
+xref: Reactome:R-HSA-9922480 "Dengue virion gets internalized"
 is_a: GO:0140272 ! exogenous protein binding
 relationship: part_of GO:0046718 ! symbiont entry into host cell
 
@@ -11355,6 +11375,7 @@
 synonym: "behavioural fear response" EXACT []
 is_a: GO:0002209 ! behavioral defense response
 is_a: GO:0042596 ! fear response
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 
 [Term]
 id: GO:0001664
@@ -11427,7 +11448,6 @@
 synonym: "ATPase stimulator activity" EXACT []
 xref: Reactome:R-HSA-5251955 "HSP40s activate intrinsic ATPase activity of HSP70s in the nucleoplasm"
 xref: Reactome:R-HSA-5251959 "HSP40s activate intrinsic ATPase activity of HSP70s in the cytosol"
-is_a: GO:0060590 ! ATPase regulator activity
 is_a: GO:0140677 ! molecular function activator activity
 relationship: positively_regulates GO:0140657 ! ATP-dependent activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
@@ -13697,6 +13717,7 @@
 synonym: "UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1->3)-beta-D-galactosyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-xylosyl-proteoglycan 4IV-alpha-N-acetyl-D-glucosaminyltransferase activity" RELATED [EC:2.4.1.223]
 xref: EC:2.4.1.223
 xref: MetaCyc:2.4.1.223-RXN
+xref: Reactome:R-HSA-9953259 "EXTL3 dimer transfers GlcNAc to the GAG linker"
 xref: RHEA:16221
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.223
@@ -15814,7 +15835,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: all-trans-decaprenyl diphosphate + 4-hydroxybenzoate = 3-decaprenyl-4-hydroxybenzoate + diphosphate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific substrate of 4-hydroxybenzoate polyprenyltransferase activity ; GO:0008412.
-xref: Reactome:R-HSA-2162192 "COQ2 ligates all-E-10PrP2 to PHB"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26830" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008412
@@ -29274,7 +29294,7 @@
 name: keratinocyte development
 namespace: biological_process
 def: "The process whose specific outcome is the progression of a keratinocyte over time, from its formation to the mature structure." [GOC:dph]
-xref: Reactome:R-HSA-9725554 "Differentiation of keratinocytes in interfollicular epidermis in mammalian skin"
+xref: Reactome:R-HSA-9725554 "Differentiation of Keratinocytes in Interfollicular Epidermis in Mammalian Skin"
 is_a: GO:0002064 ! epithelial cell development
 relationship: part_of GO:0030216 ! keratinocyte differentiation
 created_by: dph
@@ -30844,6 +30864,7 @@
 namespace: molecular_function
 def: "A nuclear receptor activity regulated by steroid binding and modulating the transcription of specific gene sets transcribed by RNA polymerase II." [GOC:signaling, PMID:14708019]
 synonym: "steroid hormone receptor activity" BROAD []
+xref: Reactome:R-HSA-8963915 "1,25(OH)2D binds VDR"
 is_a: GO:0004879 ! nuclear receptor activity
 relationship: part_of GO:0030518 ! nuclear receptor-mediated steroid hormone signaling pathway
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16717" xsd:anyURI
@@ -30886,7 +30907,7 @@
 synonym: "transcription cofactor activity" EXACT []
 synonym: "transcription coreceptor activity" EXACT []
 synonym: "transcriptional co-regulator" EXACT []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+xref: Reactome:R-HSA-5340251 "NR1H4:DCA,CDCA,LCHA binds RXRA and NCOA1,2"
 is_a: GO:0140110 ! transcription regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15536" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15665" xsd:anyURI
@@ -30897,6 +30918,7 @@
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20962" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26489" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28926" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31471" xsd:anyURI
 created_by: krc
 creation_date: 2010-11-24T03:02:15Z
 
@@ -30931,6 +30953,7 @@
 synonym: "RNA polymerase II transcription co-repressor activity" RELATED []
 synonym: "RNA polymerase II transcription corepressor activity" RELATED []
 synonym: "transcription co-repressor activity" EXACT []
+xref: Reactome:R-HSA-400183 "PPARA:RXRA binds Corepressors of PPARA"
 is_a: GO:0003712 ! transcription coregulator activity
 relationship: part_of GO:0045892 ! negative regulation of DNA-templated transcription
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15665" xsd:anyURI
@@ -31080,7 +31103,7 @@
 name: double-stranded RNA adenosine deaminase activity
 namespace: molecular_function
 alt_id: GO:0003971
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a double-stranded RNA molecule." [GOC:mah, RHEA:10120]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+, in a double-stranded RNA molecule." [GOC:mah, RHEA:10120]
 synonym: "double-stranded RNA specific editase activity" RELATED []
 xref: EC:3.5.4.37
 xref: Reactome:R-HSA-77614 "Deamination at C6 position of adenosine in Editosome (ADAR1)"
@@ -31298,6 +31321,7 @@
 xref: Reactome:R-HSA-8950113 "IL12A binds IL12B"
 xref: Reactome:R-HSA-8950183 "IL12B binds IL23A"
 xref: Reactome:R-HSA-8950456 "IL12B dimerizes"
+xref: Reactome:R-HSA-9760559 "DsbA catalyzes disulfide crosslink formation on STa"
 is_a: GO:0016864 ! intramolecular oxidoreductase activity, transposing S-S bonds
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: skos:exactMatch EC:5.3.4.1
@@ -31875,7 +31899,6 @@
 xref: MetaCyc:2.3.2.13-RXN
 xref: Reactome:R-HSA-140851 "fibrin multimer -> fibrin multimer, crosslinked + NH4+"
 xref: Reactome:R-HSA-6810894 "Envoplakin, periplakin, involucrin, SPR binding mediated by TGM1 crosslinking"
-xref: RESID:AA0124
 xref: RHEA:54816
 is_a: GO:0016755 ! aminoacyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -32173,7 +32196,7 @@
 xref: EC:2.4.1.102
 xref: MetaCyc:2.4.1.102-RXN
 xref: Reactome:R-HSA-914012 "GCNTs transfer GlcNAc from UDP-GlcNAc to Core 1 mucins"
-xref: Reactome:R-HSA-914018 "Addition of GlcNAc to Core 3 forms a Core 4 glycoprotein"
+xref: Reactome:R-HSA-914018 "GCNT3 transfers GlcNAc to Core 3 mucin"
 xref: RHEA:18705
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 is_a: GO:0140103 ! catalytic activity, acting on a glycoprotein
@@ -32315,12 +32338,11 @@
 xref: MetaCyc:2.4.99.4-RXN
 xref: Reactome:R-HSA-1022129 "ST3GAL4 transfers Neu5Ac to terminal Gal of N-glycans"
 xref: Reactome:R-HSA-1912378 "Sialylation of Pre-NOTCH"
-xref: Reactome:R-HSA-2046285 "The keratan chain can be capped by N-acetylneuraminic acid"
+xref: Reactome:R-HSA-2046285 "ST3GAL1-4,6 optionally cap keratan chain"
 xref: Reactome:R-HSA-3656258 "Defective ST3GAL3 does not transfer SA to keratan"
 xref: Reactome:R-HSA-4084984 "ST3GAL1-6 transfer Neu5Ac to terminal Gal (alpha-2,3 link)"
 xref: Reactome:R-HSA-9603987 "ST3GAL3 transfers Neu5Ac to Type 1 chain to form Type 1 MSGG"
 xref: Reactome:R-HSA-9605600 "ST3GAL3,4,6 transfer Neu5Ac to Type 2 chain to form Type 2 MSGG"
-xref: Reactome:R-HSA-981497 "ST3GAL1-4 transfers sialic acid to the T antigen at the alpha 3 position"
 xref: Reactome:R-HSA-9844860 "ST3GAL5 transfers NeuNAc to ceramides"
 xref: Reactome:R-HSA-9845538 "ST3GAL2,3 transfer Neu5Ac to gangliosides"
 xref: RHEA:21616
@@ -32333,7 +32355,7 @@
 id: GO:0003837
 name: beta-ureidopropionase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-carbamoyl-beta-alanine + H2O = beta-alanine + CO2 + NH3." [EC:3.5.1.6]
+def: "Catalysis of the reaction: N-carbamoyl-beta-alanine + H2O = beta-alanine + CO2 + NH4+." [EC:3.5.1.6]
 synonym: "N-carbamoyl-beta-alanine amidohydrolase activity" RELATED [EC:3.5.1.6]
 xref: EC:3.5.1.6
 xref: MetaCyc:BETA-UREIDOPROPIONASE-RXN
@@ -32389,7 +32411,6 @@
 xref: EC:4.3.2.9
 xref: MetaCyc:GAMMA-GLUTAMYLCYCLOTRANSFERASE-RXN
 xref: Reactome:R-HSA-1247922 "GGCT transforms gGluCys to OPRO"
-xref: Reactome:R-HSA-6785928 "CHAC1,2 cleaves GSH to OPRO and CysGly"
 xref: RHEA:20505
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.9
@@ -32455,8 +32476,9 @@
 synonym: "pyrroline-5-carboxylic acid dehydrogenase activity" RELATED [EC:1.2.1.88]
 xref: EC:1.2.1.88
 xref: MetaCyc:RXN-14116
-xref: Reactome:R-HSA-6784399 "ALDH4A1 dimer dehydrogenates 4-OH-L-glutamate semialdehyde to 4-OH-L-glutamate"
+xref: Reactome:R-HSA-6784402 "ALDH4A1 converts 1PYR-3OH-5COOH to 4-OH-L-glutamate"
 xref: Reactome:R-HSA-70679 "ALDH4A1 oxidises L-GluSS to Glu"
+xref: Reactome:R-HSA-9929439 "ALDH4A1 oxidizes 1PYR-3OH-5COOH"
 xref: RHEA:30235
 is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.2.1.88
@@ -32543,7 +32565,6 @@
 synonym: "beta-hydroxysteroid dehydrogenase" BROAD []
 synonym: "corticosteroid 11-reductase" RELATED []
 synonym: "corticosteroid 11beta-dehydrogenase" RELATED []
-xref: Reactome:R-HSA-194023 "HSD11B2,HSD11B1 dimer oxidise CORT to COR"
 xref: Wikipedia:11beta-hydroxysteroid_dehydrogenase
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21915" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28173" xsd:anyURI
@@ -32856,6 +32877,7 @@
 property_value: skos:exactMatch EC:1.1.1.35
 property_value: skos:exactMatch RHEA:22432
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30577" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
 
 [Term]
 id: GO:0003858
@@ -32984,12 +33006,16 @@
 xref: EC:1.2.4.4
 xref: MetaCyc:1.2.4.4-RXN
 xref: RHEA:13457
+xref: RHEA:84639
+xref: RHEA:84643
 is_a: GO:0016624 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, disulfide as acceptor
 property_value: skos:broadMatch EC:1.2.1.25
 property_value: skos:broadMatch MetaCyc:1.2.1.25-RXN
 property_value: skos:exactMatch EC:1.2.4.4
 property_value: skos:exactMatch MetaCyc:1.2.4.4-RXN
 property_value: skos:narrowMatch RHEA:13457
+property_value: skos:narrowMatch RHEA:84639
+property_value: skos:narrowMatch RHEA:84643
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21144" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29951" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30031" xsd:anyURI
@@ -33089,7 +33115,6 @@
 synonym: "gamma-aminobutyric acid transaminase activity" EXACT []
 synonym: "gamma-aminobutyric transaminase activity" EXACT []
 synonym: "glutamate-succinic semialdehyde transaminase activity" EXACT []
-xref: Reactome:R-HSA-916855 "PXLP-K357-ABAT dimer:2Fe-2S transforms GABA to SUCCSA"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28183" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28802" xsd:anyURI
 is_obsolete: true
@@ -33198,7 +33223,7 @@
 synonym: "phosphohexokinase activity" BROAD [EC:2.7.1.11]
 xref: EC:2.7.1.11
 xref: MetaCyc:6PFRUCTPHOS-RXN
-xref: Reactome:R-HSA-70467 "D-fructose 6-phosphate + ATP => D-fructose 1,6-bisphosphate + ADP"
+xref: Reactome:R-HSA-70467 "PFK tetramer phosphorylates Fru(6)P"
 xref: RHEA:16109
 is_a: GO:0008443 ! phosphofructokinase activity
 relationship: part_of GO:0061615 ! glycolytic process through fructose-6-phosphate
@@ -33220,7 +33245,7 @@
 xref: EC:2.7.1.105
 xref: KEGG_REACTION:R02732
 xref: MetaCyc:6-PHOSPHOFRUCTO-2-KINASE-RXN
-xref: Reactome:R-HSA-71802 "D-fructose 6-phosphate + ATP => D-fructose 2,6-bisphosphate + ADP"
+xref: Reactome:R-HSA-71802 "PFKFKB dimer phosphorylates Fru(6)P"
 xref: RHEA:15653
 is_a: GO:0008443 ! phosphofructokinase activity
 property_value: skos:exactMatch EC:2.7.1.105
@@ -33277,7 +33302,7 @@
 id: GO:0003876
 name: AMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: AMP + H2O = IMP + NH3." [EC:3.5.4.6]
+def: "Catalysis of the reaction: AMP + H2O = IMP + NH4+." [RHEA:14777]
 synonym: "5-adenylate deaminase activity" RELATED [EC:3.5.4.6]
 synonym: "5-adenylic acid deaminase activity" RELATED [EC:3.5.4.6]
 synonym: "5-AMP deaminase activity" RELATED [EC:3.5.4.6]
@@ -33357,8 +33382,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00352
 property_value: skos:exactMatch MetaCyc:ATP-CITRATE-PRO-S--LYASE-RXN
 property_value: skos:exactMatch RHEA:21160
-property_value: skos:relatedMatch MetaCyc:PWY-5172
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0003879
@@ -33619,6 +33644,8 @@
 xref: Reactome:R-HSA-69116 "Formation of Okazaki fragments"
 xref: Reactome:R-HSA-73932 "Resynthesis of excised residue by POLB"
 xref: Reactome:R-HSA-9710480 "Decitabine triphosphate incorporates into DNA"
+xref: Reactome:R-HSA-9914364 "POLG in the replisome polymerizes the H strand"
+xref: Reactome:R-HSA-9914498 "POLG in the replisome polymerizes the L strand"
 is_a: GO:0034061 ! DNA polymerase activity
 property_value: skos:exactMatch EC:2.7.7.7
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28442" xsd:anyURI
@@ -33680,10 +33707,13 @@
 xref: Reactome:R-HSA-167121 "Addition of the third nucleotide on the nascent HIV-1 transcript"
 xref: Reactome:R-HSA-167136 "Addition of nucleotides 5 through 9 on the growing HIV-1 transcript"
 xref: Reactome:R-HSA-174425 "The primase component of DNA polymerase:primase synthesizes a 6-10 nucleotide RNA primer on the G strand of the telomere"
+xref: Reactome:R-HSA-1964482 "RNA polymerase III transcribes microbial dsDNA to dsRNA"
 xref: Reactome:R-HSA-203901 "Pol II mediated transcription of microRNA genes"
 xref: Reactome:R-HSA-427366 "Transcription of intergenic spacer of the rRNA gene"
 xref: Reactome:R-HSA-5601926 "RNA polymerase II polymerizes primary piRNA transcript"
 xref: Reactome:R-HSA-6781824 "Active RNA Pol II complex transcribes lesion-containing DNA template"
+xref: Reactome:R-HSA-6814549 "Pre-snRNA transcript initiation, Integrator binding, LEC binding"
+xref: Reactome:R-HSA-6814559 "Pre-snRNA is elongated and capped with 7-methylguanosine"
 xref: Reactome:R-HSA-68913 "The primase component of DNA polymerase:primase synthesizes a 6-10 nucleotide RNA primer at the origin"
 xref: Reactome:R-HSA-74986 "Elongation of pre-rRNA transcript"
 xref: Reactome:R-HSA-75850 "Addition of the third nucleotide on the nascent transcript"
@@ -33693,6 +33723,8 @@
 xref: Reactome:R-HSA-9670149 "TERRA transcription"
 xref: Reactome:R-HSA-9697084 "Defective RpoB in Mtb RNAP transcribes RNA polyanion"
 xref: Reactome:R-HSA-9697085 "RNAP transcribes Mtb RNA polyanion"
+xref: Reactome:R-HSA-9914409 "POLRMT polymerizes RNA primer for mitochondrial L strand replication"
+xref: Reactome:R-HSA-9915448 "POLRMT polymerizes R-loop RNA"
 is_a: GO:0034062 ! 5'-3' RNA polymerase activity
 relationship: part_of GO:0032774 ! RNA biosynthetic process
 property_value: skos:exactMatch EC:2.7.7.6
@@ -33869,6 +33901,7 @@
 synonym: "sealase activity" RELATED [EC:6.5.1.1]
 xref: EC:6.5.1.1
 xref: MetaCyc:DNA-LIGASE-ATP-RXN
+xref: Reactome:R-HSA-9914903 "LIG3 (Ligase III) ligates nascent mitochondrial DNA strands"
 is_a: GO:0003909 ! DNA ligase activity
 property_value: skos:exactMatch EC:6.5.1.1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -33979,6 +34012,7 @@
 synonym: "untwisting enzyme activity" RELATED [EC:5.6.2.1]
 xref: EC:5.6.2.1
 xref: MetaCyc:5.99.1.2-RXN
+xref: Reactome:R-HSA-9914922 "TOP3A (Topoisomerase 3a) decatenates mitochondrial DNA"
 is_a: GO:0003916 ! DNA topoisomerase activity
 property_value: skos:exactMatch EC:5.6.2.1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15575" xsd:anyURI
@@ -34063,7 +34097,7 @@
 id: GO:0003921
 name: GMP synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + XMP + NH4(+) = AMP + diphosphate + GMP + 2H+." [RHEA:18301]
+def: "Catalysis of the reaction: ATP + XMP + NH4+ = AMP + diphosphate + GMP + 2H+." [RHEA:18301]
 xref: MetaCyc:GMP-SYN-NH3-RXN
 xref: RHEA:18301
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
@@ -34195,6 +34229,8 @@
 xref: Reactome:R-HSA-9645598 "RRAGC,D hydrolyzes GTP"
 xref: Reactome:R-HSA-9649736 "RAS intrinsic GTPase activity hydrolyzes GTP to GDP"
 xref: Reactome:R-HSA-983422 "Disassembly of COPII coated vesicle"
+xref: Reactome:R-HSA-9947842 "GBP1 hydrolyzes GTP forming GBP1 oligomers"
+xref: Reactome:R-HSA-9954919 "ABCE1:ATP binds PELO:HBS1L-1:GTP:80S ribosome:non-stop mRNA:peptidyl-tRNA with nascent peptide and HBS1L-1:GDP is released"
 xref: RHEA:19669
 is_a: GO:0017111 ! ribonucleoside triphosphate phosphatase activity
 property_value: skos:exactMatch RHEA:19669
@@ -34223,7 +34259,6 @@
 xref: EC:3.6.5.1
 xref: EC:3.6.5.2
 xref: Reactome:R-HSA-400027 "Gq alpha:G beta:G gamma dissociates to Gq alpha:GTP and G beta:G gamma"
-xref: Reactome:R-HSA-400037 "Gi,Go Heterotrimeric G-protein complex dissociates"
 xref: Reactome:R-HSA-422320 "Heterotrimeric G(s) complex dissociates"
 is_a: GO:0003924 ! GTPase activity
 is_a: GO:0098772 ! molecular function regulator activity
@@ -34429,8 +34464,8 @@
 synonym: "glycosaminoglycan alpha-L-iduronohydrolase activity" RELATED [EC:3.2.1.76]
 xref: EC:3.2.1.76
 xref: MetaCyc:3.2.1.76-RXN
-xref: Reactome:R-HSA-1678716 "IDUA hydrolyses Heparan sulfate chain(1)"
-xref: Reactome:R-HSA-1793186 "IDUA) hydrolyses the unsulfated alpha-L-iduronosidic link in DS"
+xref: Reactome:R-HSA-1678716 "IDUA cleaves iduronate from HS chain"
+xref: Reactome:R-HSA-1793186 "IDUA hydrolyses the unsulfated alpha-L-iduronosidic link in DS"
 xref: Reactome:R-HSA-2090037 "IDUA hydrolyses Heparan sulfate chain(6)"
 xref: Reactome:R-HSA-2206299 "Defective IDUA does not hydrolyse Heparan sulfate chain(6)"
 xref: Reactome:R-HSA-9036037 "Defective IDUA does not hydrolyse Heparan sulfate chain(1)"
@@ -34444,7 +34479,7 @@
 id: GO:0003941
 name: L-serine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine = pyruvate + NH3." [EC:4.3.1.17]
+def: "Catalysis of the reaction: L-serine = pyruvate + NH4+." [RHEA:19169]
 synonym: "L-hydroxyaminoacid dehydratase activity" BROAD []
 synonym: "L-serine ammonia-lyase (pyruvate-forming) activity" RELATED [EC:4.3.1.17]
 synonym: "L-serine deaminase activity" EXACT []
@@ -34454,6 +34489,7 @@
 synonym: "serine deaminase activity" BROAD [EC:4.3.1.17]
 xref: EC:4.3.1.17
 xref: MetaCyc:4.3.1.17-RXN
+xref: Reactome:R-HSA-9929460 "SDS dimers:PXLP convert L-Ser to PYR and NH4+"
 xref: RHEA:19169
 is_a: GO:0016841 ! ammonia-lyase activity
 property_value: skos:exactMatch EC:4.3.1.17
@@ -35091,6 +35127,8 @@
 xref: Reactome:R-HSA-9834719 "Synthesis of negative sense genomic RNA of respiratory syncytial virus"
 xref: Reactome:R-HSA-9834736 "Synthesis of antigenomic RNA of human respiratory syncytial virus"
 xref: Reactome:R-HSA-9837511 "Abortive replication of hRSV A"
+xref: Reactome:R-HSA-9922955 "NS5 synthesizes minus strand genome"
+xref: Reactome:R-HSA-9923068 "NS5 synthesizes plus strand genome"
 is_a: GO:0034062 ! 5'-3' RNA polymerase activity
 relationship: part_of GO:0001172 ! RNA-templated transcription
 property_value: skos:exactMatch EC:2.7.7.48
@@ -35130,7 +35168,7 @@
 alt_id: GO:0052852
 alt_id: GO:0052853
 alt_id: GO:0052854
-def: "Catalysis of the reaction: (S)-2-hydroxy-acid + O2 = 2-oxo acid + hydrogen peroxide." [RHEA:16789]
+def: "Catalysis of the reaction: (S)-2-hydroxy-acid + O2 = 2-oxo acid + H2O2." [RHEA:16789]
 synonym: "glycolate oxidase activity" RELATED []
 synonym: "hydroxy-acid oxidase A activity" NARROW [EC:1.1.3.15]
 synonym: "hydroxy-acid oxidase B activity" NARROW [EC:1.1.3.15]
@@ -35159,7 +35197,6 @@
 name: UDP-N-acetylglucosamine 4-epimerase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine = UDP-N-acetyl-D-galactosamine." [EC:5.1.3.7]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "UDP acetylglucosamine epimerase activity" RELATED [EC:5.1.3.7]
 synonym: "UDP-GlcNAc 4-epimerase activity" RELATED [EC:5.1.3.7]
 synonym: "UDP-N-acetyl-D-glucosamine 4-epimerase activity" RELATED [EC:5.1.3.7]
@@ -35172,9 +35209,9 @@
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
 property_value: skos:exactMatch EC:5.1.3.7
 property_value: skos:exactMatch RHEA:20517
-property_value: skos:relatedMatch MetaCyc:PWY-5512
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0003975
@@ -35327,7 +35364,7 @@
 xref: EC:2.7.7.9
 xref: KEGG_REACTION:R00289
 xref: MetaCyc:GLUC1PURIDYLTRANS-RXN
-xref: Reactome:R-HSA-70286 "UTP + D-glucose 1-phosphate <=> pyrophosphate + UDP-glucose"
+xref: Reactome:R-HSA-70286 "UTP + D-glucose 1-phosphate => pyrophosphate + UDP-glucose"
 xref: RHEA:19889
 is_a: GO:0051748 ! UTP-monosaccharide-1-phosphate uridylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.9
@@ -35369,7 +35406,7 @@
 synonym: "acetyl coenzyme A thiolase activity" RELATED []
 xref: EC:2.3.1.9
 xref: MetaCyc:ACETYL-COA-ACETYLTRANSFER-RXN
-xref: Reactome:R-HSA-70844 "alpha-methyl-acetoacetyl-CoA + CoA => propionyl-CoA + acetyl-CoA"
+xref: Reactome:R-HSA-70844 "alpha-methylacetoacetyl-CoA + CoA => propionyl-CoA + acetyl-CoA"
 xref: Reactome:R-HSA-73916 "2 acetyl-CoA <=> acetoacetyl-CoA + CoA"
 xref: Reactome:R-HSA-74181 "acetoacetyl-CoA + CoA <=> 2 acetyl-CoA"
 xref: Reactome:R-HSA-8848215 "ACAT2 condenses 2 Ac-CoA to form ACA-CoA"
@@ -35529,10 +35566,10 @@
 
 [Term]
 id: GO:0003992
-name: N2-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity
+name: N2-acetyl-L-ornithine:2-oxoglutarate 5-transaminase activity
 namespace: molecular_function
 alt_id: GO:0047318
-def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-acetyl-L-ornithine = N-acetyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.11, RHEA:18049]
+def: "Catalysis of the reaction: N(2)-acetyl-L-ornithine + 2-oxoglutarate = N-acetyl-L-glutamate 5-semialdehyde + L-glutamate." [RHEA:18049]
 synonym: "2-N-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "acetylornithine 5-aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "acetylornithine aminotransferase activity" BROAD []
@@ -35542,6 +35579,7 @@
 synonym: "N(2)-acetylornithine 5-transaminase activity" RELATED [EC:2.6.1.11]
 synonym: "N-acetylornithine aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "N-acetylornithine-delta-transaminase activity" RELATED [EC:2.6.1.11]
+synonym: "N2-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" EXACT []
 synonym: "N2-acetyl-L-ornithine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "N2-acetylornithine 5-aminotransferase activity" EXACT []
 synonym: "N2-acetylornithine 5-transaminase activity" RELATED [EC:2.6.1.11]
@@ -35554,6 +35592,7 @@
 property_value: skos:exactMatch EC:2.6.1.11
 property_value: skos:exactMatch RHEA:18049
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0003993
@@ -35702,7 +35741,7 @@
 id: GO:0004000
 name: adenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3." [RHEA:24408]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+." [RHEA:24408]
 synonym: "adenosine aminohydrolase activity" RELATED [EC:3.5.4.4]
 synonym: "adenosine deaminase reaction" EXACT []
 xref: MetaCyc:ADENODEAMIN-RXN
@@ -35811,9 +35850,9 @@
 
 [Term]
 id: GO:0004015
-name: adenosylmethionine-8-amino-7-oxononanoate transaminase activity
+name: S-adenosyl-L-methionine:8-amino-7-oxononanoate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 8-amino-7-oxononanoate + S-adenosyl-L-methionine(1+) = 7,8-diaminononanoate + S-adenosyl-4-methylthio-2-oxobutanoate." [EC:2.6.1.62, RHEA:16861]
+def: "Catalysis of the reaction: (8S)-8-amino-7-oxononanoate + S-adenosyl-L-methionine = S-adenosyl-4-methylsulfanyl-2-oxobutanoate + (7R,8S)-7,8-diammoniononanoate." [RHEA:16861]
 synonym: "7,8-diamino-pelargonic acid aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "7,8-diaminonanoate transaminase activity" RELATED [EC:2.6.1.62]
 synonym: "7,8-diaminononanoate aminotransferase activity" RELATED [EC:2.6.1.62]
@@ -35824,6 +35863,7 @@
 synonym: "adenosyl methionine-8-amino-7-oxononanoate transaminase activity" EXACT []
 synonym: "adenosylmethionine--8-amino-7-oxononanoate aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "adenosylmethionine-8-amino-7-oxononanoate aminotransferase activity" EXACT []
+synonym: "adenosylmethionine-8-amino-7-oxononanoate transaminase activity" EXACT []
 synonym: "DAPA aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "DAPA transaminase activity" RELATED [EC:2.6.1.62]
 synonym: "diaminopelargonate synthase activity" RELATED [EC:2.6.1.62]
@@ -35836,6 +35876,7 @@
 property_value: skos:exactMatch EC:2.6.1.62
 property_value: skos:exactMatch RHEA:16861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004016
@@ -35959,10 +36000,9 @@
 
 [Term]
 id: GO:0004021
-name: L-alanine:2-oxoglutarate aminotransferase activity
+name: L-alanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-alanine = L-glutamate + pyruvate." [RHEA:19453]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-alanine + 2-oxoglutarate = pyruvate + L-glutamate." [RHEA:19453]
 synonym: "alanine aminotransferase activity" BROAD []
 synonym: "alanine transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "alanine-alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.2]
@@ -35977,6 +36017,7 @@
 synonym: "L-alanine aminotransferase activity" BROAD [EC:2.6.1.2]
 synonym: "L-alanine transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "L-alanine-alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.2]
+synonym: "L-alanine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "pyruvate transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "pyruvate-alanine aminotransferase activity" RELATED [EC:2.6.1.2]
 synonym: "pyruvate-glutamate transaminase activity" RELATED [EC:2.6.1.2]
@@ -35988,13 +36029,14 @@
 xref: Reactome:R-HSA-70523 "PXLP-K314-GPT transaminates L-Ala to form PYR"
 xref: Reactome:R-HSA-70524 "PXLP-K314-GPT transaminates PYR to form L-Ala"
 xref: RHEA:19453
-is_a: GO:0047635 ! alanine-oxo-acid transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.2
 property_value: skos:exactMatch KEGG_REACTION:R00258
 property_value: skos:exactMatch MetaCyc:ALANINE-AMINOTRANSFERASE-RXN
 property_value: skos:exactMatch RHEA:19453
-property_value: skos:relatedMatch MetaCyc:ALANINE-SYN2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004022
@@ -36183,7 +36225,7 @@
 name: aldehyde oxidase activity
 namespace: molecular_function
 alt_id: GO:0050250
-def: "Catalysis of the reaction: an aldehyde + H2O + O2 = a carboxylic acid + hydrogen peroxide." [EC:1.2.3.1]
+def: "Catalysis of the reaction: an aldehyde + O2 + H2O = a carboxylate + H2O2 + H+." [RHEA:16829]
 synonym: "aldehyde:oxygen oxidoreductase activity" RELATED [EC:1.2.3.1]
 synonym: "quinoline oxidase activity" RELATED [EC:1.2.3.1]
 synonym: "retinal oxidase activity" NARROW []
@@ -36218,6 +36260,7 @@
 xref: KEGG_REACTION:R02820
 xref: Reactome:R-HSA-196060 "Reduction of isocaproaldehyde to 4-methylpentan-1-ol"
 xref: Reactome:R-HSA-5652172 "AKR1B1 reduces Glc to D-sorbitol"
+xref: Reactome:R-HSA-9931850 "AKR1B1 reduces galactose to galactitol"
 xref: RHEA:12789
 is_a: GO:0008106 ! alcohol dehydrogenase (NADP+) activity
 property_value: skos:broadMatch EC:1.1.1.21
@@ -36238,26 +36281,6 @@
 synonym: "aldo-keto reductase (NADP+) activity" EXACT []
 synonym: "aldo-keto reductase activity" EXACT []
 synonym: "NADPH-dependent aldo-keto reductase activity" EXACT [GOC:vw]
-xref: Reactome:R-HSA-192033 "4-cholesten-7alpha-ol-3-one is reduced to 5beta-cholestan-7alpha-ol-3-one"
-xref: Reactome:R-HSA-192036 "5Beta-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha, 12alpha-triol"
-xref: Reactome:R-HSA-192067 "4-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholesten-7alpha, 12alpha-diol-3-one"
-xref: Reactome:R-HSA-192160 "5beta-cholestan-7alpha-ol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha-diol"
-xref: Reactome:R-HSA-193746 "4-cholesten-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-7alpha,24(S)-diol-3-one"
-xref: Reactome:R-HSA-193755 "4-cholesten-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,24(S)-triol-3-one"
-xref: Reactome:R-HSA-193758 "5beta-cholestan-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,24(S)-triol"
-xref: Reactome:R-HSA-193781 "5Beta-cholestan-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol"
-xref: Reactome:R-HSA-193800 "5Beta-cholestan-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol"
-xref: Reactome:R-HSA-193821 "4-cholesten-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,27-triol-3-one"
-xref: Reactome:R-HSA-193824 "4-cholesten-7alpha,27-diol-3-one is reduced to 5beta-cholestan-7alpha,27-diol-3-one"
-xref: Reactome:R-HSA-193841 "5beta-cholestan-7alpha,27-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,27-triol"
-xref: Reactome:R-HSA-198845 "CYB5A:heme reduces Asc.- to AscH-"
-xref: Reactome:R-HSA-2855252 "AKRs reduce RBP2:atRAL to RBP2:atROL"
-xref: Reactome:R-HSA-5423637 "AKR dimers reduce AFBDHO to AFBDOH"
-xref: Reactome:R-HSA-5692232 "AKR1A1 oxidises BaPtDHD to BaP-7,8-dione"
-xref: Reactome:R-HSA-9027531 "Dehydrogenase dehydrogenates 13-HDHA to 13-oxo-DHA"
-xref: Reactome:R-HSA-9027562 "Dehydrogenase dehydrogenates 17-HDHA to 17-oxo-DHA"
-xref: Reactome:R-HSA-9027598 "Dehydrogenase dehydrogenates 13(R)-HDPAn-3 to 13-oxo-DPAn-3"
-xref: Reactome:R-HSA-9027600 "Dehydrogenase dehydrogenates 17-HDPAn-3 to 17-oxo-DPAn-3"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27180" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28532" xsd:anyURI
 is_obsolete: true
@@ -36272,6 +36295,8 @@
 synonym: "mutarotase activity" RELATED [EC:5.1.3.3]
 xref: EC:5.1.3.3
 xref: MetaCyc:ALDOSE-1-EPIMERASE-RXN
+xref: Reactome:R-HSA-9931853 "GALM converts beta-D-GAL to alpha-D-GAL"
+xref: Reactome:R-HSA-9931926 "Defective GALM doesn't convert beta-D-GAL to alpha-D-GAL"
 xref: RHEA:10264
 xref: RHEA:28675
 xref: RHEA:63332
@@ -36377,7 +36402,7 @@
 
 [Term]
 id: GO:0004042
-name: L-glutamate N-acetyltransferase activity
+name: L-glutamate N-acetyltransferase activity, acting on acetyl-CoA as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + acetyl-CoA = N-acetyl-L-glutamate + CoA + H+." [RHEA:24292]
 synonym: "acetyl-CoA:L-glutamate N-acetyltransferase activity" EXACT []
@@ -36385,14 +36410,17 @@
 synonym: "AGAS" RELATED [EC:2.3.1.1]
 synonym: "amino acid acetyltransferase activity" RELATED [EC:2.3.1.1]
 synonym: "amino-acid N-acetyltransferase activity" BROAD [EC:2.3.1.1]
+synonym: "L-glutamate N-acetyltransferase activity" BROAD []
 synonym: "N-acetylglutamate synthase activity" BROAD [EC:2.3.1.1]
 xref: KEGG_REACTION:R00259
 xref: MetaCyc:N-ACETYLTRANSFER-RXN
 xref: Reactome:R-HSA-70542 "glutamate + acetyl CoA => N-acetyl glutamate + CoA"
+xref: Reactome:R-HSA-9955697 "NAGS variants don't synthesize N-acetylglutamate"
 xref: RHEA:24292
 is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:broadMatch EC:2.3.1.1
 property_value: skos:exactMatch RHEA:24292
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31438" xsd:anyURI
 
 [Term]
 id: GO:0004043
@@ -36464,6 +36492,8 @@
 synonym: "N-substituted aminoacyl transfer RNA hydrolase activity" RELATED [EC:3.1.1.29]
 xref: EC:3.1.1.29
 xref: MetaCyc:AMINOCYL-TRNA-HYDROLASE-RXN
+xref: Reactome:R-HSA-9937374 "MTRFR:MTRES1:39S ribosomal subunit:mRNA:peptidyl-tRNA:MALSU1:MIEF1:NDUFAB1 dissociates"
+xref: Reactome:R-HSA-9937719 "ICT1 hydrolyzes the peptidyl-tRNA bond of the nascent polypeptide"
 xref: RHEA:54448
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -36658,6 +36688,7 @@
 xref: MetaCyc:ARGINASE-RXN
 xref: Reactome:R-HSA-452036 "arginine + H2O => ornithine + urea [ARG2]"
 xref: Reactome:R-HSA-70569 "arginine + H2O => ornithine + urea [ARG1]"
+xref: Reactome:R-HSA-9956512 "ARG1 variants don't synthesize urea and ornithine"
 xref: RHEA:20569
 is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
 property_value: skos:exactMatch EC:3.5.3.1
@@ -36697,6 +36728,7 @@
 xref: EC:6.3.4.5
 xref: MetaCyc:ARGSUCCINSYN-RXN
 xref: Reactome:R-HSA-70577 "ASS1 tetramer:NMRAL1 dimer:NADPH transforms L-Asp and L-Cit to ARSUA"
+xref: Reactome:R-HSA-9956517 "ASS1 variants don't synthesize arginosuccinate"
 xref: RHEA:10932
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
 property_value: skos:exactMatch EC:6.3.4.5
@@ -36718,6 +36750,7 @@
 xref: EC:4.3.2.1
 xref: MetaCyc:ARGSUCCINLYA-RXN
 xref: Reactome:R-HSA-70573 "argininosuccinate <=> fumarate + arginine"
+xref: Reactome:R-HSA-9956524 "ASL variants don't synthesize fumarate and arginine"
 xref: RHEA:24020
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.1
@@ -36996,7 +37029,6 @@
 name: aspartate 1-decarboxylase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-aspartate = beta-alanine + CO2." [EC:4.1.1.11]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "aspartate alpha-decarboxylase activity" RELATED [EC:4.1.1.11]
 synonym: "aspartic alpha-decarboxylase" BROAD [EC:4.1.1.11]
 synonym: "L-aspartate 1-carboxy-lyase (beta-alanine-forming)" RELATED [EC:4.1.1.11]
@@ -37009,15 +37041,14 @@
 property_value: skos:exactMatch EC:4.1.1.11
 property_value: skos:exactMatch MetaCyc:ASPDECARBOX-RXN
 property_value: skos:exactMatch RHEA:19497
-property_value: skos:relatedMatch MetaCyc:PWY-5155
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004069
-name: L-aspartate:2-oxoglutarate aminotransferase activity
+name: L-aspartate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate." [EC:2.6.1.1]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate." [RHEA:21824]
 synonym: "2-oxoglutarate-glutamate aminotransferase activity" RELATED [EC:2.6.1.1]
 synonym: "AAT" RELATED [EC:2.6.1.1]
 synonym: "aspartate alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.1]
@@ -37043,6 +37074,7 @@
 synonym: "L-aspartate-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.1]
 synonym: "L-aspartate-2-oxoglutarate-transaminase activity" RELATED [EC:2.6.1.1]
 synonym: "L-aspartate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.1]
+synonym: "L-aspartate:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-aspartic aminotransferase activity" BROAD [EC:2.6.1.1]
 synonym: "oxaloacetate transferase activity" RELATED [EC:2.6.1.1]
 synonym: "oxaloacetate-aspartate aminotransferase activity" RELATED [EC:2.6.1.1]
@@ -37052,16 +37084,16 @@
 xref: MetaCyc:ASPAMINOTRANS-RXN
 xref: Reactome:R-HSA-70581 "oxaloacetate + glutamate <=> aspartate + alpha-ketoglutarate [GOT1]"
 xref: Reactome:R-HSA-70592 "PXLP-K259-GOT1 transaminates 2-OG and L-Asp"
-xref: Reactome:R-HSA-70596 "GOT2 dimer deaminates L-Asp"
+xref: Reactome:R-HSA-70596 "GOT2 dimer transaminates L-Asp"
 xref: Reactome:R-HSA-70613 "GOT2 transaminates OA and L-Glu"
 xref: RHEA:21824
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.1
 property_value: skos:exactMatch RHEA:21824
-property_value: skos:relatedMatch MetaCyc:ASPARTATE-DEG1-PWY
-property_value: skos:relatedMatch MetaCyc:ASPARTATESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004070
@@ -37094,7 +37126,7 @@
 id: GO:0004071
 name: aspartate-ammonia ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + L-aspartate + NH3 = AMP + diphosphate + L-asparagine." [EC:6.3.1.1]
+def: "Catalysis of the reaction: ATP + L-aspartate + NH4+ = AMP + diphosphate + L-asparagine." [RHEA:11372]
 synonym: "asparagine synthetase activity" RELATED [EC:6.3.1.1]
 synonym: "L-asparagine synthetase activity" RELATED [EC:6.3.1.1]
 synonym: "L-aspartate:ammonia ligase (AMP-forming)" RELATED [EC:6.3.1.1]
@@ -37220,6 +37252,15 @@
 xref: MetaCyc:6.3.4.11-RXN
 xref: MetaCyc:6.3.4.9-RXN
 xref: MetaCyc:BIOTINLIG-RXN
+xref: Reactome:R-HSA-2993447 "HLCS biotinylates 6x(PCCA:PCCB)"
+xref: Reactome:R-HSA-2993799 "HLCS biotinylates 6xMCCC1:6xMCCC2"
+xref: Reactome:R-HSA-2993802 "HLCS biotinylates PC:Mn2+"
+xref: Reactome:R-HSA-2993814 "HLCS biotinylates ACACA:Mn2+"
+xref: Reactome:R-HSA-3323184 "Defective HLCS does not biotinylate ACACA:Mn2+"
+xref: Reactome:R-HSA-4167511 "HLCS biotinylates ACACB"
+xref: Reactome:R-HSA-9035987 "Defective HLCS does not biotinylate 6xMCCC1:6xMCCC2"
+xref: Reactome:R-HSA-9035988 "Defective HLCS does not biotinylate PC:Mn2+"
+xref: Reactome:R-HSA-9035990 "Defective HLCS does not biotinylate 6x(PCCA:PCCB)"
 xref: RHEA:11204
 xref: RHEA:11756
 xref: RHEA:23668
@@ -37341,12 +37382,13 @@
 
 [Term]
 id: GO:0004084
-name: branched-chain-amino-acid transaminase activity
+name: branched-chain-amino-acid:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a branched-chain amino acid (L-leucine, L-isoleucine and L-valine) + 2-oxoglutarate = L-glutamate + a 2-oxocarboxylate derived from the branched-chain amino acid." [EC:2.6.1.42]
 synonym: "branched-chain amino acid aminotransferase activity" EXACT []
 synonym: "branched-chain amino acid-glutamate transaminase activity" RELATED [EC:2.6.1.42]
 synonym: "branched-chain aminotransferase activity" RELATED [EC:2.6.1.42]
+synonym: "branched-chain-amino-acid transaminase activity" EXACT []
 synonym: "branched-chain-amino-acid:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.42]
 synonym: "glutamate-branched-chain amino acid transaminase activity" RELATED [EC:2.6.1.42]
 synonym: "L-branched chain amino acid aminotransferase activity" RELATED [EC:2.6.1.42]
@@ -37356,9 +37398,10 @@
 xref: Reactome:R-HSA-508189 "a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate <=> leu, ile, or val + alpha-ketoglutarate [BCAT1]"
 xref: Reactome:R-HSA-70723 "leu, ile, or val + alpha-ketoglutarate <=> a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate [BCAT1]"
 xref: Reactome:R-HSA-70724 "leu, ile, or val + alpha-ketoglutarate <=> a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate [BCAT2]"
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.42
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004085
@@ -37406,6 +37449,7 @@
 xref: KEGG_REACTION:R00149
 xref: MetaCyc:RXN-13202
 xref: Reactome:R-HSA-70555 "2 ATP + NH4+ + HCO3- => 2 ADP + orthophosphate + carbamoyl phosphate [mitochondrial]"
+xref: Reactome:R-HSA-9955543 "CPS1 variants don't synthesize carbamoyl phosphate"
 xref: RHEA:18029
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
 property_value: skos:exactMatch EC:6.3.4.16
@@ -37552,7 +37596,6 @@
 xref: EC:2.3.1.21
 xref: MetaCyc:CARNITINE-O-PALMITOYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-200406 "CPT1A transfers PALM to CAR"
-xref: Reactome:R-HSA-200410 "CPT2 converts palmitoyl carnitine to palmitoyl-CoA"
 xref: Reactome:R-HSA-9911362 "CPT1B transfers PALM to CAR"
 xref: RHEA:12661
 is_a: GO:0016406 ! carnitine O-acyltransferase activity
@@ -37567,7 +37610,7 @@
 namespace: molecular_function
 alt_id: GO:0016952
 alt_id: GO:0016953
-def: "Catalysis of the reaction: 2 hydrogen peroxide = O2 + 2 H2O." [EC:1.11.1.6]
+def: "Catalysis of the reaction: 2 H2O2 = O2 + 2 H2O." [EC:1.11.1.6]
 synonym: "bacterial catalase-peroxidase activity" NARROW []
 synonym: "caperase activity" RELATED [EC:1.11.1.6]
 synonym: "CAT" RELATED [EC:1.11.1.6]
@@ -37613,6 +37656,10 @@
 xref: EC:1.10.3.1
 xref: MetaCyc:CATECHOL-OXIDASE-RXN
 xref: MetaCyc:RXN-13061
+xref: Reactome:R-HSA-5662662 "Tyrosinase oxidises tyrosine to dopaquinone"
+xref: Reactome:R-HSA-5662692 "Dopa is oxidized to dopaquinone by TYR"
+xref: Reactome:R-HSA-5663050 "DHI and DHICA polymerize forming eumelanin"
+xref: Reactome:R-HSA-8878581 "TYRP1 oxidises DHICA to IQCA"
 xref: RHEA:21632
 is_a: GO:0004497 ! monooxygenase activity
 is_a: GO:0016682 ! oxidoreductase activity, acting on diphenols and related substances as donors, oxygen as acceptor
@@ -37867,7 +37914,6 @@
 name: creatine kinase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: ATP + creatine = N-phosphocreatine + ADP + 2 H+." [EC:2.7.3.2, RHEA:17157]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "adenosine triphosphate-creatine transphosphorylase activity" RELATED [EC:2.7.3.2]
 synonym: "ATP:creatine N-phosphotransferase activity" RELATED [EC:2.7.3.2]
 synonym: "ATP:creatine phosphotransferase activity" RELATED [EC:2.7.3.2]
@@ -37894,9 +37940,9 @@
 is_a: GO:0016775 ! phosphotransferase activity, nitrogenous group as acceptor
 property_value: skos:exactMatch EC:2.7.3.2
 property_value: skos:exactMatch RHEA:17157
-property_value: skos:relatedMatch MetaCyc:PWY-6158
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004112
@@ -38222,7 +38268,7 @@
 id: GO:0004130
 name: cytochrome-c peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 ferrocytochrome c + hydrogen peroxide = 2 ferricytochrome c + 2 H2O." [EC:1.11.1.5]
+def: "Catalysis of the reaction: 2 ferrocytochrome c + H2O2 = 2 ferricytochrome c + 2 H2O." [EC:1.11.1.5]
 synonym: "apocytochrome c peroxidase activity" RELATED [EC:1.11.1.5]
 synonym: "cytochrome c peroxidase activity" RELATED [EC:1.11.1.5]
 synonym: "cytochrome c-551 peroxidase activity" RELATED [EC:1.11.1.5]
@@ -38244,7 +38290,7 @@
 id: GO:0004131
 name: cytosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cytosine + H2O = uracil + NH3." [EC:3.5.4.1]
+def: "Catalysis of the reaction: cytosine + H2O = uracil + NH4+." [RHEA:20605]
 synonym: "cytosine aminohydrolase activity" RELATED [EC:3.5.4.1]
 synonym: "isocytosine deaminase activity" RELATED [EC:3.5.4.1]
 xref: EC:3.5.4.1
@@ -38260,7 +38306,7 @@
 id: GO:0004132
 name: dCMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dCMP + H2O = dUMP + NH3." [EC:3.5.4.12]
+def: "Catalysis of the reaction: dCMP + H2O = dUMP + NH4+." [RHEA:22924]
 synonym: "dCMP aminohydrolase activity" RELATED [EC:3.5.4.12]
 synonym: "deoxy-CMP-deaminase activity" RELATED [EC:3.5.4.12]
 synonym: "deoxycytidine monophosphate deaminase activity" RELATED [EC:3.5.4.12]
@@ -38654,7 +38700,6 @@
 synonym: "succinyl-CoA:enzyme-N6-(dihydrolipoyl)lysine S-succinyltransferase activity" RELATED [EC:2.3.1.61]
 xref: EC:2.3.1.61
 xref: Reactome:R-HSA-9853512 "DLST transfers succinyl to CoA"
-xref: Reactome:R-HSA-9858590 "DLST transfers glutaryl to CoA"
 xref: RHEA:15213
 is_a: GO:0016751 ! S-succinyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -38718,7 +38763,7 @@
 id: GO:0004153
 name: dihydropterin deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 7,8-dihydropterin + H2O = 7,8-dihydrolumazine + NH3." [GOC:jl, PMID:19567870, PMID:38786926]
+def: "Catalysis of the reaction: 7,8-dihydropterin + H2O = 7,8-dihydrolumazine + NH4+." [GOC:jl, PMID:19567870, PMID:38786926]
 xref: MetaCyc:RXN-15261
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
 is_a: GO:0019239 ! deaminase activity
@@ -39039,6 +39084,7 @@
 xref: EC:5.3.3.12
 xref: KEGG_REACTION:R03673
 xref: MetaCyc:DOPACHROME-DELTA-ISOMERASE-RXN
+xref: Reactome:R-HSA-5662660 "Dopachrome is transformed to DHICA by DCT"
 xref: RHEA:13041
 is_a: GO:0016863 ! intramolecular oxidoreductase activity, transposing C=C bonds
 property_value: skos:exactMatch EC:5.3.3.12
@@ -39080,6 +39126,7 @@
 xref: Reactome:R-HSA-5615556 "Defective POMT2 does not transfer Man from Dol-P-Man to DAG1"
 xref: Reactome:R-HSA-5615604 "Defective POMT1 does not transfer Man from Dol-P-Man to DAG1"
 xref: Reactome:R-HSA-5615637 "POMT1:POMT2 transfers Man from Dol-P-Man to DAG1(30-653)"
+xref: Reactome:R-HSA-9816277 "CDH1 is O-manosylated"
 is_a: GO:0000030 ! mannosyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 relationship: part_of GO:0035269 ! protein O-linked glycosylation via mannose
@@ -39207,7 +39254,7 @@
 xref: Reactome:R-HSA-353125 "26S proteosome degrades ODC holoenzyme complex"
 xref: Reactome:R-HSA-3640874 "Ub-RibC-AXIN is degraded by the proteasome"
 xref: Reactome:R-HSA-3928656 "gamma-secretase cleaves EPHB2"
-xref: Reactome:R-HSA-450466 "AUF1:mRNA complex is destroyed"
+xref: Reactome:R-HSA-450466 "AUF1:mRNA complex is degraded"
 xref: Reactome:R-HSA-4608855 "PRICKLE1 is degraded by the proteasome"
 xref: Reactome:R-HSA-4641256 "Ubiquitinated AXIN is degraded by the proteasome"
 xref: Reactome:R-HSA-4641260 "Ubiquitinated DVL1 is degraded by the proteasome"
@@ -39256,7 +39303,9 @@
 xref: Reactome:R-HSA-9614271 "Autocleavage of ADGRG6"
 xref: Reactome:R-HSA-9755303 "26S proteasome degrades HIFalpha"
 xref: Reactome:R-HSA-9755306 "ub UBXN7 is degraded by the 26S proteasome"
+xref: Reactome:R-HSA-9760521 "Unknown peptidase cleaves pro-STa"
 xref: Reactome:R-HSA-9762096 "Ub,pS335,S338,T NFE2L2 is degraded"
+xref: Reactome:R-HSA-9766223 "Proteasome-dependent degradation of ubiquitinated CDH1"
 xref: Reactome:R-HSA-983150 "Proteasomal cleavage of substrate"
 xref: Reactome:R-HSA-983158 "Trimming of peptides in ER"
 xref: Reactome:R-HSA-9839376 "TGFBR3(784-851) degradation"
@@ -39265,6 +39314,18 @@
 xref: Reactome:R-HSA-9908105 "Maturation of the 20S immunoproteasome core particle"
 xref: Reactome:R-HSA-9908108 "Maturation of the 20S thymoproteasome core particle"
 xref: Reactome:R-HSA-9908780 "Maturation of the preholospermatoproteasome"
+xref: Reactome:R-HSA-9912636 "Proteasomal cleavage of intracellular substrate (PA28-alpha-beta-20S proteasome catalyst)"
+xref: Reactome:R-HSA-9912655 "Proteasomal cleavage of intracellular substrate (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9912747 "Proteasomal clevage of exogenous antigen (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9912751 "Proteasomal cleavage of partially digested antigen (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9929352 "Ubiquitinated CD274 is degraded by the 26S proteasome"
+xref: Reactome:R-HSA-9929486 "SPOP-mediated degradation of CD274 by 26S Proteosome"
+xref: Reactome:R-HSA-9931314 "Proteasomal degradation of polyUb-p-S195-CD274"
+xref: Reactome:R-HSA-9934884 "Proteolysis of K48polyUb,p-S-CRY1,2"
+xref: Reactome:R-HSA-9934887 "Proteolysis of K48polyUb-K107,p-S-CRY1,K48polyUb-K126,p-S-CRY2"
+xref: Reactome:R-HSA-9934893 "Proteolysis of K48polyUb-K,p-S-PER1,2,3"
+xref: Reactome:R-HSA-9949676 "The proteasome degrades the alanine-tailed K48-polyUb nascent peptide bound to LTN1:NEMF:TCF25:VCP hexamer"
+xref: Reactome:R-HSA-9954728 "The proteasome degrades the K48-polyubiquitinated alanine-tailed nascent peptide"
 is_a: GO:0008233 ! peptidase activity
 property_value: skos:narrowMatch EC:3.4.99.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25813" xsd:anyURI
@@ -39384,14 +39445,15 @@
 xref: MetaCyc:CARBOXYPEPTIDASE-A-RXN
 xref: MetaCyc:GLY-X-CARBOXYPEPTIDASE-RXN
 xref: MetaCyc:LYSINEARGININE-CARBOXYPEPTIDASE-RXN
-xref: Reactome:R-HSA-2022378 "ACE2(18-805):Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-9)"
-xref: Reactome:R-HSA-2022379 "ACE2(18-805):Zn2+ hydrolyzes Angiotensin-(1-8) to Angiotensin-(1-7)"
-xref: Reactome:R-HSA-2028294 "Mast Cell Carboxypeptidase hydrolyzes Angiotensin-(1-10) to Yield Angiotensin-(1-9)"
+xref: Reactome:R-HSA-2022378 "ACE2(18-805):Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-33) (Angiotensin-(1-9))"
+xref: Reactome:R-HSA-2022379 "ACE2(18-805):Zn2+ hydrolyzes AGT(25-32) (Angiotensin-(1-8)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2028294 "Mast Cell Carboxypeptidase hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-33) (Angiotensin-(1-9))"
 xref: Reactome:R-HSA-8852809 "CPN, CPB2 cleave C3a, C5a"
 xref: Reactome:R-HSA-8866105 "CCPs deglutamylate tubulin"
 xref: Reactome:R-HSA-8955712 "SVBP:VASH1,VASH2 hydrolyzes the terminal L-Tyr residue from alphaY-beta tubulin dimer"
 xref: Reactome:R-HSA-9023159 "Carboxypeptidase E hydrolyzes Insulin(57-89) to yield C-peptide (Insulin(57-87))"
 xref: Reactome:R-HSA-9023163 "Carboxypeptidase E cleaves Insulin(25-56) to yield Insulin(25-54)"
+xref: Reactome:R-HSA-9944488 "ACE2:Zn2+ cleaves 1-Ala-AGT(25-32) to yield Alamandine (1-Ala-AGT(25-31)"
 is_a: GO:0004180 ! carboxypeptidase activity
 is_a: GO:0008235 ! metalloexopeptidase activity
 property_value: skos:exactMatch EC:3.4.17.-
@@ -39606,9 +39668,9 @@
 xref: MetaCyc:3.4.23.5-RXN
 xref: Reactome:R-HSA-157353 "NEXT1 is cleaved to produce NICD1"
 xref: Reactome:R-HSA-157640 "NEXT2 is cleaved to produce NICD2"
-xref: Reactome:R-HSA-2022403 "Renin:Prorenin Receptor hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
-xref: Reactome:R-HSA-2022412 "Renin hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
-xref: Reactome:R-HSA-2065357 "Prorenin:Prorenin Receptor hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
+xref: Reactome:R-HSA-2022403 "Renin:Prorenin Receptor hydrolyzes Angiotensinogen to AGT(25-34) (Angiotensin-(1-10))"
+xref: Reactome:R-HSA-2022412 "Renin hydrolyzes Angiotensinogen to AGT(25-34) (Angiotensin-(1-10))"
+xref: Reactome:R-HSA-2065357 "Prorenin:Prorenin Receptor hydrolyzes Angiotensinogen to AGT(25-35) (Angiotensin-(1-10))"
 xref: Reactome:R-HSA-2220988 "NEXT1 PEST domain mutants are cleaved to produce NICD1 PEST domain mutants"
 xref: Reactome:R-HSA-373705 "Caspase cleavage of DCC"
 xref: Reactome:R-HSA-9013361 "NEXT3 is cleaved to produce NICD3"
@@ -39807,7 +39869,7 @@
 xref: Reactome:R-HSA-264865 "Caspase-mediated cleavage of Lamin A"
 xref: Reactome:R-HSA-264871 "Caspase-mediated cleavage of Lamin B1"
 xref: Reactome:R-HSA-3465448 "Caspase-8 and FLIP(L) processing at DISC"
-xref: Reactome:R-HSA-350158 "LGMN degrades GC"
+xref: Reactome:R-HSA-350158 "LGMN hydrolyzes GC, releasing CUBN and 25(OH)D"
 xref: Reactome:R-HSA-350318 "Caspase-mediated cleavage of vimentin at TNLD (429)"
 xref: Reactome:R-HSA-350319 "Caspase mediated cleavage of vimentin at IDVD (259)"
 xref: Reactome:R-HSA-350651 "Caspase-mediated cleavage of MASK"
@@ -39829,6 +39891,7 @@
 xref: Reactome:R-HSA-5660663 "Caspase-8 cleaves IL1B precursor"
 xref: Reactome:R-HSA-5681987 "LC3 is cleaved by ATG4"
 xref: Reactome:R-HSA-5682377 "LC3 de-lipidation by ATG4"
+xref: Reactome:R-HSA-6800797 "The PIDDosome activates CASP2"
 xref: Reactome:R-HSA-6814387 "CASP14 cleaves filaggrin"
 xref: Reactome:R-HSA-9012556 "IL37:2x(CASP1(120-197):CASP1(317-404)) cleaves IL37"
 xref: Reactome:R-HSA-9013895 "Caspase-8 processing within TLR3 complex"
@@ -39855,10 +39918,21 @@
 xref: Reactome:R-HSA-9694625 "nsp3 cleaves nsp1-4"
 xref: Reactome:R-HSA-9694732 "3CLp cleaves pp1ab"
 xref: Reactome:R-HSA-9697750 "RIPK1 is cleaved by CASP8:FLIP(L)"
-xref: Reactome:R-HSA-9710101 "CASP4, CASP5 cleave GSDMD"
 xref: Reactome:R-HSA-9729704 "SARS-CoV-2 3CLpro dimer cleaves TAB1"
 xref: Reactome:R-HSA-9729730 "SARS-CoV-2 nsp3 cleaves IRF3"
 xref: Reactome:R-HSA-9729741 "SARS-CoV-2 3CLpro dimer cleaves NLRP12"
+xref: Reactome:R-HSA-9792568 "CTSL1 cleaves proHPSE"
+xref: Reactome:R-HSA-9947849 "CASP5 autocatalysis on LPS micelles"
+xref: Reactome:R-HSA-9947861 "CASP4 autocatalysis on LPS micelles"
+xref: Reactome:R-HSA-9947908 "CASP4 autocatalysis on the bacterial surface"
+xref: Reactome:R-HSA-9947978 "CASP5 cleaves IL18 at D36"
+xref: Reactome:R-HSA-9947987 "CASP4 cleaves IL18 at D36"
+xref: Reactome:R-HSA-9947997 "CASP5 cleaves GSDMD"
+xref: Reactome:R-HSA-9948146 "CASP4 cleaves GSDMD"
+xref: Reactome:R-HSA-9956714 "CASP1 cleaves GBP1"
+xref: Reactome:R-HSA-9958657 "CASP4 cleaves IL1B at D27"
+xref: Reactome:R-HSA-9960522 "CASP5 cleaves IL1B at D27"
+xref: Reactome:R-HSA-9960526 "CASP4 cleaves IL1B at D116"
 is_a: GO:0004175 ! endopeptidase activity
 is_a: GO:0008234 ! cysteine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.22.-
@@ -40283,6 +40357,7 @@
 xref: Reactome:R-HSA-8949649 "PMPCA:PMPCB cleaves the transit peptide of proSMDT1 (proEMRE)"
 xref: Reactome:R-HSA-8949659 "AFG3L2 (m-AAA protease) degrades SMDT1 that is not assembled in MCU"
 xref: Reactome:R-HSA-8949668 "YME1L1 proteolyzes unassembled proSMDT1"
+xref: Reactome:R-HSA-9010034 "ADAM10:Zn2+:TSPANs cleaves APP(18-770)"
 xref: Reactome:R-HSA-9624272 "MMPs cleave HB-EGF"
 xref: Reactome:R-HSA-9822532 "ADAMTS13 cleaves VWF multimer"
 xref: Reactome:R-HSA-9824402 "ADAMTS13 variant does not cleave VWF multimer"
@@ -40765,7 +40840,7 @@
 xref: Reactome:R-HSA-139893 "Granzyme-B activates BID by cleavage"
 xref: Reactome:R-HSA-140599 "factor XIII -> factor XIII cleaved tetramer + 2 factor XIII A activation peptides"
 xref: Reactome:R-HSA-140664 "prothrombin -> activated thrombin (factor IIa) + thrombin activation peptide (prothrombinase catalyst)"
-xref: Reactome:R-HSA-140696 "factor V -> factor Va + factor V activation peptide"
+xref: Reactome:R-HSA-140696 "factor V -> factor Va + factor V activation peptide (thrombin catalyst)"
 xref: Reactome:R-HSA-140700 "prothrombin -> activated thrombin (factor IIa) + thrombin activation peptide (Xa catalyst)"
 xref: Reactome:R-HSA-140736 "factor X -> factor Xa + factor X activation peptide (TF:F7a catalyst)"
 xref: Reactome:R-HSA-140769 "factor VII -> factor VIIa"
@@ -40845,7 +40920,7 @@
 xref: Reactome:R-HSA-187020 "Part of pro-beta-NGF is processed to mature beta-NGF"
 xref: Reactome:R-HSA-1912369 "NOTCH precursor cleaved to form mature NOTCH"
 xref: Reactome:R-HSA-1912372 "Fringe-modified Pre-NOTCH is cleaved by FURIN"
-xref: Reactome:R-HSA-2022411 "Cathepsin G hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022411 "Cathepsin G hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-2129357 "Fibrillin C-terminal processing"
 xref: Reactome:R-HSA-2168923 "Collagen type XVIII endostatin release"
 xref: Reactome:R-HSA-2168960 "Collagen type XVII ectodomain shedding"
@@ -40860,6 +40935,7 @@
 xref: Reactome:R-HSA-2534206 "E-cadherin degradation by PS1:NCSTN (Gamma-secretase)"
 xref: Reactome:R-HSA-2534260 "E-cadherin degradation by caspase-3 and calpain-1"
 xref: Reactome:R-HSA-265301 "Corticotropin cleavage from POMC"
+xref: Reactome:R-HSA-3244626 "DENV protease NS2B3 cleaves STING"
 xref: Reactome:R-HSA-3266557 "Factor I cleaves iC3b"
 xref: Reactome:R-HSA-3785684 "Fibronectin degradation by CTSG"
 xref: Reactome:R-HSA-3788061 "Fibronectin degradation by ADAM8"
@@ -40936,11 +41012,34 @@
 xref: Reactome:R-HSA-9770187 "S2 Cleavage by TMPRSS2 Exposes S2' Initiating Cell-Cell Fusion"
 xref: Reactome:R-HSA-977371 "Factor I inactivates Factor H-boundC3b"
 xref: Reactome:R-HSA-977615 "Factor I inactivates MCP/CR1-bound C4b/C3b"
+xref: Reactome:R-HSA-9816275 "CDH1 is proteolytically cleaved in Golgi"
 xref: Reactome:R-HSA-9829030 "Nascent F signal peptide is cleaved at ER membrane"
 xref: Reactome:R-HSA-9829200 "F0 is cleaved, releasing F1, F2, F(110-136)"
 xref: Reactome:R-HSA-9830805 "Nascent sG localizes to ER lumen, gets glycosylated"
 xref: Reactome:R-HSA-9830882 "Nascent G signal peptide is cleaved at ER membrane"
 xref: Reactome:R-HSA-9839367 "TGFBR3_mem cleavage by MMPs"
+xref: Reactome:R-HSA-9911236 "ELANE cleaves NOTCH2NLA"
+xref: Reactome:R-HSA-9911383 "LepB cleaves prepro-STa"
+xref: Reactome:R-HSA-9912368 "LepB cleaves pre-EltA"
+xref: Reactome:R-HSA-9912371 "LepB cleaves pre-EltB"
+xref: Reactome:R-HSA-9913329 "Autoproteolysis of DAG1 to alpha and beta chains"
+xref: Reactome:R-HSA-9918772 "Dengue polyprotein cis-auto-cleaves"
+xref: Reactome:R-HSA-9918780 "Unknown peptidase cleaves NS1-NS2A"
+xref: Reactome:R-HSA-9918787 "NS2B3 cleaves C-prM-E-NS1-NS2A"
+xref: Reactome:R-HSA-9918795 "Signalase cleaves prM-E-NS1-NS2A"
+xref: Reactome:R-HSA-9918813 "NS2B3 cleaves NS4AB"
+xref: Reactome:R-HSA-9918817 "NS2B3 cleaves polyprotein"
+xref: Reactome:R-HSA-9918823 "NS345:NS2B cleaves NS345"
+xref: Reactome:R-HSA-9918871 "Signalase cleaves prepro-NS4B"
+xref: Reactome:R-HSA-9918876 "NS2B3 cleaves prepro-C at the C-prM junction"
+xref: Reactome:R-HSA-9919050 "Furin cleaves prM:E"
+xref: Reactome:R-HSA-9921595 "NS2B3 cleaves FAM134B"
+xref: Reactome:R-HSA-9921689 "NS3 cleaves GrpEL1"
+xref: Reactome:R-HSA-9926936 "EspC(54-1018) cleaves hemoglobin"
+xref: Reactome:R-HSA-9926963 "Hbp cis-autocleaves"
+xref: Reactome:R-HSA-9927009 "Hbp(53-1100) cleaves hemoglobin"
+xref: Reactome:R-HSA-9932162 "Removal of CDH1 signal peptide"
+xref: Reactome:R-HSA-9947496 "ACOT2 cleaves NS345"
 is_a: GO:0004175 ! endopeptidase activity
 is_a: GO:0008236 ! serine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.21.-
@@ -41522,9 +41621,9 @@
 xref: MetaCyc:ESTRADIOL-17-BETA-DEHYDROGENASE-RXN
 xref: Reactome:R-HSA-5693390 "HSD17B11 dehydrogenates EST17b to E1"
 xref: Reactome:R-HSA-5696822 "AKR1B15 reduces EST17b to E1"
-xref: Reactome:R-HSA-6810594 "HSD17B14 tetramer oxidises estradiol (E2) to estrone (E1)"
+xref: Reactome:R-HSA-6810594 "HSD17B14 tetramer oxidises estradiol (EST17b) to estrone (E1)"
 xref: Reactome:R-HSA-804969 "HSD17B1 hydrogenates E1 to EST17b"
-xref: Reactome:R-HSA-8862137 "HSD17B2 oxidises estradiol (E2) to estrone (E1)"
+xref: Reactome:R-HSA-8862137 "HSD17B2 oxidises estradiol (EST17b) to estrone (E1)"
 xref: RHEA:24612
 xref: RHEA:24616
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
@@ -41665,8 +41764,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 2 farnesyl diphosphate = diphosphate + presqualene diphosphate." [GOC:curators]
 comment: This term was obsoleted because it represents a subreaction of squalene synthase activity ; GO:0051996.
-xref: Reactome:R-HSA-191402 "Reduction of presqualene diphosphate to form squalene"
-xref: Reactome:R-HSA-191405 "Two FPP molecules dimerize to form presqualene diphosphate"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28819" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051996
@@ -41676,7 +41773,7 @@
 name: geranylgeranyl diphosphate synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: (2E,6E)-farnesyl diphosphate + isopentenyl diphosphate = (2E,6E,10E)-geranylgeranyl diphosphate + diphosphate." [RHEA:17653]
-comment: The catalyzed reaction forms (free) geranylgeranyl diphosphate. There is no relationship between this activity and protein farnesyltransferase activity, GO:0004660, where the catalyzed reaction transfers a farnesyl group from farnesyl diphosphate to a target protein. Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+comment: The catalyzed reaction forms (free) geranylgeranyl diphosphate. There is no relationship between this activity and protein farnesyltransferase activity, GO:0004660, where the catalyzed reaction transfers a farnesyl group from farnesyl diphosphate to a target protein.
 synonym: "farnesyltranstransferase activity" EXACT [EC:2.5.1.29]
 synonym: "geranylgeranyl pyrophosphate synthase activity" EXACT [EC:2.5.1.29]
 synonym: "geranylgeranyl pyrophosphate synthetase activity" EXACT [EC:2.5.1.29]
@@ -41690,12 +41787,12 @@
 is_a: GO:0120531 ! prenyl diphosphate synthase activity
 property_value: skos:exactMatch EC:2.5.1.29
 property_value: skos:exactMatch RHEA:17653
-property_value: skos:relatedMatch MetaCyc:PWY-5120
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20661" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28430" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29264" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004312
@@ -41989,6 +42086,8 @@
 synonym: "TPNH-ferredoxin reductase activity" RELATED [EC:1.18.1.2]
 xref: EC:1.18.1.2
 xref: MetaCyc:1.18.1.2-RXN
+xref: Reactome:R-HSA-2395512 "FDXR transfers electrons to FDX1,2 (FDX1L)"
+xref: Reactome:R-HSA-2395517 "NADPH transfers electrons to FDXR"
 xref: RHEA:20125
 is_a: GO:0008937 ! ferredoxin-[NAD(P)H] reductase activity
 property_value: skos:exactMatch EC:1.18.1.2
@@ -42114,7 +42213,7 @@
 xref: EC:3.1.3.46
 xref: MetaCyc:3.1.3.46-RXN
 xref: Reactome:R-HSA-5628905 "TIGAR converts D-fructose-2,6-bisphosphate to D-fructose 6-phosphate"
-xref: Reactome:R-HSA-70262 "Fructose 2,6-bisphosphate is hydrolyzed to form fructose-6-phosphate and orthophosphate"
+xref: Reactome:R-HSA-70262 "PFKFKB dimer dephosphorylates fructose 2,6-bisphosphate"
 xref: RHEA:17289
 is_a: GO:0050308 ! sugar-phosphatase activity
 property_value: skos:exactMatch EC:3.1.3.46
@@ -42144,7 +42243,7 @@
 xref: EC:4.1.2.13
 xref: MetaCyc:F16ALDOLASE-RXN
 xref: Reactome:R-HSA-71495 "Aldolase tetramers convert GA3P and DHAP to F1,6PP"
-xref: Reactome:R-HSA-71496 "D-fructose 1,6-bisphosphate <=> dihydroxyacetone phosphate + D-glyceraldehyde 3-phosphate"
+xref: Reactome:R-HSA-71496 "Aldolase tetramer cleaves F1,6PP"
 xref: RHEA:14729
 is_a: GO:0016832 ! aldehyde-lyase activity
 property_value: skos:exactMatch EC:4.1.2.13
@@ -42436,7 +42535,7 @@
 synonym: "Zwischenferment" RELATED [EC:1.1.1.49]
 xref: EC:1.1.1.49
 xref: MetaCyc:GLU6PDEHYDROG-RXN
-xref: Reactome:R-HSA-70377 "alpha-D-glucose 6-phosphate + NADP+ => D-glucono-1,5-lactone 6-phosphate + NADPH + H+"
+xref: Reactome:R-HSA-70377 "G6PD multimers dehydrogenate G6P"
 xref: RHEA:15841
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.49
@@ -42484,7 +42583,7 @@
 synonym: "phosphosaccharomutase activity" BROAD [EC:5.3.1.9]
 xref: EC:5.3.1.9
 xref: MetaCyc:PGLUCISOM-RXN
-xref: Reactome:R-HSA-70471 "alpha-D-glucose 6-phosphate <=> D-fructose 6-phosphate"
+xref: Reactome:R-HSA-70471 "GPI dimer isomerizes G6P"
 xref: Reactome:R-HSA-70475 "GPI dimer isomerizes Fru(6)P to G6P"
 xref: RHEA:11816
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
@@ -42597,7 +42696,6 @@
 name: L-glutamate dehydrogenase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NAD+ + H2O = 2-oxoglutarate + NH4+ + NADH + H+." [RHEA:15133]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "glutamate dehydrogenase (NAD)" BROAD [EC:1.4.1.2]
 synonym: "glutamate dehydrogenase (NAD+) activity" BROAD []
 synonym: "glutamate oxidoreductase activity" RELATED [EC:1.4.1.2]
@@ -42620,39 +42718,36 @@
 is_a: GO:0004353 ! L-glutamate dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.4.1.2
 property_value: skos:exactMatch RHEA:15133
-property_value: skos:relatedMatch MetaCyc:GLUTAMATE-DEG1-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004353
 name: L-glutamate dehydrogenase [NAD(P)+] activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NAD(P)+ + H2O = 2-oxoglutarate + NH4+ + NAD(P)H + H+." [EC:1.4.1.3]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "glutamate dehydrogenase (NAD+) activity" RELATED []
 synonym: "glutamic dehydrogenase activity" BROAD [EC:1.4.1.3]
 synonym: "L-glutamate:NAD(P)+ oxidoreductase (deaminating)" EXACT [EC:1.4.1.3]
 xref: EC:1.4.1.3
 xref: MetaCyc:GLUTAMATE-DEHYDROGENASE-NADP+-RXN
-xref: Reactome:R-HSA-70589 "alpha-ketoglutarate + NH4+ + NAD(P)H + H+ <=> glutamate + NAD(P)+ [GLUD1]"
-xref: Reactome:R-HSA-70600 "glutamate + NAD(P)+ => alpha-ketoglutarate + NH4+ + NAD(P)H + H+ [GLUD1]"
+xref: Reactome:R-HSA-70589 "alpha-ketoglutarate + NH4+ + NAD(P)H + H+ <=> glutamate + NAD(P)+ (GLUD1,2)"
+xref: Reactome:R-HSA-70600 "glutamate + NAD(P)+ => alpha-ketoglutarate + NH4+ + NAD(P)H + H+ (GLUD1,2)"
 is_a: GO:0050018 ! amino-acid dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.4.1.3
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-DEHYDROGENASE-NADP+-RXN
-property_value: skos:relatedMatch MetaCyc:GLUTAMATE-SYN2-PWY
-property_value: skos:relatedMatch MetaCyc:PWY-5766
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004354
 name: L-glutamate dehydrogenase (NADP+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NADP+ + H2O = 2-oxoglutarate + NH4+ + NADPH + H+." [RHEA:11612]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "dehydrogenase, glutamate (nicotinamide adenine dinucleotide (phosphate))" RELATED [EC:1.4.1.4]
 synonym: "glutamic acid dehydrogenase" BROAD [EC:1.4.1.4]
 synonym: "glutamic dehydrogenase activity" BROAD [EC:1.4.1.4]
@@ -42668,16 +42763,15 @@
 property_value: skos:exactMatch EC:1.4.1.4
 property_value: skos:exactMatch MetaCyc:GLUTDEHYD-RXN
 property_value: skos:exactMatch RHEA:11612
-property_value: skos:relatedMatch MetaCyc:GLUTSYNIII-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004355
 name: glutamate synthase (NADPH) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + NADP+ = 2-oxoglutarate + L-glutamine + H+ + NADPH. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + NADP+ + H2O = NH3 + 2-oxoglutarate + NADPH + H+." [EC:1.4.1.13, RHEA:15501]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + NADP+ = 2-oxoglutarate + L-glutamine + H+ + NADPH. This is a two-step reaction: (a) L-glutamate + NH4+ = L-glutamine + H2O, (b) L-glutamate + NADP+ + H2O = NH4+ + 2-oxoglutarate + NADPH + H+." [RHEA:15501]
 synonym: "glutamate (reduced nicotinamide adenine dinucleotide phosphate) synthase activity" RELATED [EC:1.4.1.13]
 synonym: "glutamate synthetase (NADP) activity" RELATED [EC:1.4.1.13]
 synonym: "glutamine amide-2-oxoglutarate aminotransferase (oxidoreductase, NADP) activity" RELATED [EC:1.4.1.13]
@@ -42699,8 +42793,8 @@
 property_value: skos:exactMatch EC:1.4.1.13
 property_value: skos:exactMatch MetaCyc:GLUTAMATESYN-RXN
 property_value: skos:exactMatch RHEA:15501
-property_value: skos:relatedMatch MetaCyc:GLUTSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004356
@@ -42745,24 +42839,24 @@
 name: L-glutamate N-acetyltransferase activity, acting on acetyl-L-ornithine as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: N2-acetyl-L-ornithine + L-glutamate = N-acetyl-L-glutamate + L-ornithine." [RHEA:15349]
-synonym: "2-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
+synonym: "2-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
 synonym: "acetyl-L-glutamate:acetyl-L-ornithine transacetylase activity" EXACT []
-synonym: "acetylglutamate synthetase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamate-acetylornithine transacetylase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamic synthetase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamic-acetylornithine transacetylase activity" RELATED [EC:2.3.1.35]
+synonym: "acetylglutamate synthetase activity" BROAD [EC:2.3.1.35]
+synonym: "acetylglutamate-acetylornithine transacetylase activity" EXACT [EC:2.3.1.35]
+synonym: "acetylglutamic synthetase activity" BROAD [EC:2.3.1.35]
+synonym: "acetylglutamic-acetylornithine transacetylase activity" EXACT [EC:2.3.1.35]
 synonym: "acetylornithinase activity" BROAD [EC:2.3.1.35]
-synonym: "acetylornithine glutamate acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "alpha-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "glutamate acetyltransferase activity" RELATED [EC:2.3.1.35]
+synonym: "acetylornithine glutamate acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "alpha-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "glutamate acetyltransferase activity" BROAD [EC:2.3.1.35]
 synonym: "glutamate N-acetyltransferase activity" BROAD []
 synonym: "L-glutamate N-acetyltransferase activity" BROAD []
 synonym: "N-acetyl-L-glutamate synthetase activity" BROAD [EC:2.3.1.35]
 synonym: "N-acetylglutamate synthase activity" BROAD [EC:2.3.1.35]
 synonym: "N-acetylglutamate synthetase activity" BROAD [EC:2.3.1.35]
-synonym: "N2-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "ornithine acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "ornithine transacetylase activity" RELATED [EC:2.3.1.35]
+synonym: "N2-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "ornithine acetyltransferase activity" BROAD [EC:2.3.1.35]
+synonym: "ornithine transacetylase activity" BROAD [EC:2.3.1.35]
 xref: EC:2.3.1.35
 xref: KEGG_REACTION:R02282
 xref: MetaCyc:GLUTAMATE-N-ACETYLTRANSFERASE-RXN
@@ -42784,7 +42878,7 @@
 synonym: "L-glutaminase activity" RELATED [EC:3.5.1.2]
 synonym: "L-glutamine amidohydrolase activity" RELATED [EC:3.5.1.2]
 xref: EC:3.5.1.2
-xref: Reactome:R-HSA-70609 "glutamine + H2O => glutamate + NH4+ [GLS]"
+xref: Reactome:R-HSA-70609 "glutamine + H2O => glutamate + NH4+ (GLS)"
 xref: RHEA:15889
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:broadMatch MetaCyc:GLUTAMIN-RXN
@@ -42795,9 +42889,9 @@
 
 [Term]
 id: GO:0004360
-name: glutamine-fructose-6-phosphate transaminase (isomerizing) activity
+name: L-glutamine:D-fructose-6-phosphate transaminase (isomerizing) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-D-fructose 6-phosphate + L-glutamine = D-glucosamine 6-phosphate + L-glutamate." [EC:2.6.1.16, RHEA:13237]
+def: "Catalysis of the reaction: D-fructose 6-phosphate + L-glutamine = D-glucosamine 6-phosphate + L-glutamate." [RHEA:13237]
 synonym: "D-fructose-6-phosphate amidotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "GlcN6P synthase activity" RELATED [EC:2.6.1.16]
 synonym: "glucosamine 6-phosphate synthase activity" RELATED [EC:2.6.1.16]
@@ -42805,6 +42899,7 @@
 synonym: "glucosamine-6-phosphate isomerase (glutamine-forming) activity" EXACT []
 synonym: "glucosamine-6-phosphate synthase activity" RELATED [EC:2.6.1.16]
 synonym: "glucosaminephosphate isomerase" BROAD [EC:2.6.1.16]
+synonym: "glutamine-fructose-6-phosphate transaminase (isomerizing) activity" EXACT []
 synonym: "hexosephosphate aminotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "L-glutamine-D-fructose-6-phosphate amidotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "L-glutamine:D-fructose-6-phosphate isomerase (deaminating)" RELATED [EC:2.6.1.16]
@@ -42814,10 +42909,11 @@
 xref: Reactome:R-HSA-4085027 "Defective GFPT1 does not transfer an amino group from L-Gln to F6P to form GlcN6P"
 xref: Reactome:R-HSA-449715 "GFPT1,2 transfer an amino group from L-Gln to F6P to form GlcN6P"
 xref: RHEA:13237
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.16
 property_value: skos:exactMatch RHEA:13237
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004361
@@ -42941,7 +43037,7 @@
 synonym: "triosephosphate dehydrogenase activity" BROAD [EC:1.2.1.12]
 xref: EC:1.2.1.12
 xref: MetaCyc:GAPOXNPHOSPHN-RXN
-xref: Reactome:R-HSA-70449 "D-glyceraldehyde 3-phosphate + orthophosphate + NAD+ <=> 1,3-bisphospho-D-glycerate + NADH + H+"
+xref: Reactome:R-HSA-70449 "GADPH tetramer dehydrogenates GA3P"
 xref: Reactome:R-HSA-70482 "GAPDH tetramers reduce 1,3BPG to GA3P"
 xref: RHEA:10300
 is_a: GO:0043891 ! glyceraldehyde-3-phosphate dehydrogenase [NAD(P)+] (phosphorylating) activity
@@ -43302,6 +43398,7 @@
 synonym: "guanosine-diphosphatase activity" EXACT []
 xref: EC:3.6.1.42
 xref: MetaCyc:GUANOSINE-DIPHOSPHATASE-RXN
+xref: Reactome:R-HSA-9947887 "GBP1 hydrolyzes GDP"
 xref: RHEA:22156
 is_a: GO:0017110 ! nucleoside diphosphate phosphatase activity
 property_value: skos:exactMatch EC:3.6.1.42
@@ -43388,6 +43485,8 @@
 xref: Reactome:R-HSA-169468 "MCM2-7 mediated fork unwinding"
 xref: Reactome:R-HSA-5686410 "BLM mediates dissolution of double Holliday junction"
 xref: Reactome:R-HSA-5690996 "ERCC2 and ERCC3 DNA helicases form an open bubble structure in damaged DNA"
+xref: Reactome:R-HSA-9922957 "NS3 melts (+)-RNA secondary structures"
+xref: Reactome:R-HSA-9923073 "NS3 unwinds dsRNA secondary structures"
 is_a: GO:0120545 ! nucleic acid conformation isomerase activity
 is_a: GO:0140640 ! catalytic activity, acting on a nucleic acid
 is_a: GO:0140657 ! ATP-dependent activity
@@ -43426,7 +43525,7 @@
 synonym: "heparin 2-sulfotransferase activity" RELATED []
 synonym: "heparin-sulphate 2-sulphotransferase activity" RELATED []
 xref: MetaCyc:RXN-11558
-xref: Reactome:R-HSA-2076508 "HS2ST1 sulfates IdoA at C2 in heparan sulfate"
+xref: Reactome:R-HSA-2076508 "HS2ST1 trimer sulfates IdoA at C2 in heparan sulfate"
 is_a: GO:0034483 ! heparan sulfate sulfotransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24749" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
@@ -43470,7 +43569,7 @@
 id: GO:0004397
 name: histidine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidine = urocanate + NH3." [EC:4.3.1.3]
+def: "Catalysis of the reaction: L-histidine = trans-urocanate + NH4+." [RHEA:21232]
 synonym: "histidase activity" RELATED [GOC:hjd]
 synonym: "histidinase activity" RELATED [GOC:hjd]
 synonym: "histidine alpha-deaminase activity" RELATED [GOC:hjd]
@@ -43518,13 +43617,14 @@
 
 [Term]
 id: GO:0004400
-name: histidinol-phosphate transaminase activity
+name: L-histidinol-phosphate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidinol-phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate." [EC:2.6.1.9]
+def: "Catalysis of the reaction: L-histidinol-phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate." [RHEA:23744]
 synonym: "glutamic-imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "histidine:imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "histidinol phosphate aminotransferase activity" RELATED [EC:2.6.1.9]
 synonym: "histidinol-phosphate aminotransferase activity" EXACT []
+synonym: "histidinol-phosphate transaminase activity" EXACT []
 synonym: "IAP transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "imidazole acetol-phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
@@ -43535,10 +43635,11 @@
 xref: EC:2.6.1.9
 xref: MetaCyc:HISTAMINOTRANS-RXN
 xref: RHEA:23744
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.9
 property_value: skos:exactMatch RHEA:23744
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004401
@@ -43597,6 +43698,7 @@
 xref: Reactome:R-HSA-3697920 "CLOCK acetylates lysine-10 of histone H3, H4"
 xref: Reactome:R-HSA-5144542 "CLOCK acetylates lysine-15 of histone H3, H4"
 xref: Reactome:R-HSA-5250938 "B-WICH:histone acetyltransferase acetylates histone H3 at lysine-9"
+xref: Reactome:R-HSA-9926482 "CREBBP, EP300 acetylate MITF-M"
 xref: RHEA:21992
 is_a: GO:0061733 ! protein-lysine-acetyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
@@ -43774,12 +43876,13 @@
 synonym: "hyaluronoglucosidase activity" RELATED [EC:3.2.1.35]
 xref: EC:3.2.1.35
 xref: MetaCyc:3.2.1.35-RXN
-xref: Reactome:R-HSA-1793209 "HYAL1 hydrolyses Chondroitin chains"
+xref: Reactome:R-HSA-1793209 "HYAL1-4 hydrolyze chondroitin chains"
 xref: Reactome:R-HSA-2160874 "HYAL1 hydrolyses (HA)50"
-xref: Reactome:R-HSA-2160892 "Hyaluronidase 2 (HYAL2) hydrolyses HA into 20kDa fragments"
+xref: Reactome:R-HSA-2160892 "HYAL2 hydrolyses HA into 20kDa fragments"
 xref: Reactome:R-HSA-2318585 "Defective HYAL1 does not hydrolyse Chondroitin chains"
-xref: Reactome:R-HSA-5693356 "CEMIP hydrolyses HA"
+xref: Reactome:R-HSA-5693356 "CEMIP hydrolyzes HA"
 xref: Reactome:R-HSA-9036077 "Defective HYAL1 does not hydrolyse (HA)50"
+xref: Reactome:R-HSA-9941055 "SPAM1 randomly cleaves HA"
 is_a: GO:0015929 ! hexosaminidase activity
 property_value: skos:exactMatch EC:3.2.1.35
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -44317,7 +44420,7 @@
 id: GO:0004447
 name: iodide peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: iodide + H2O2 = iodine + 2 H2O." [RHEA:23336]
+def: "Catalysis of the reactions: (1) 2 iodide + H2O2 + 2 H+ = diiodine + 2 H2O; (2) [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O; (3) [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O; (4) 2 [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-L-thyroxine + [thyroglobulin]-dehydroalanine + 2 H2O; (5) [thyroglobulin]-3-iodo-L-tyrosine + [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-3,3',5-triiodo-L-thyronine + [thyroglobulin]-dehydroalanine + 2 H2O." [EC:1.11.1.8, PMID:8651700]
 synonym: "iodinase activity" RELATED [EC:1.11.1.8]
 synonym: "thyroid peroxidase activity" RELATED [EC:1.11.1.8]
 synonym: "thyroperoxidase activity" RELATED [EC:1.11.1.8]
@@ -44325,12 +44428,21 @@
 xref: EC:1.11.1.8
 xref: MetaCyc:IODIDE-PEROXIDASE-RXN
 xref: RHEA:23336
+xref: RHEA:48956
+xref: RHEA:48960
+xref: RHEA:48964
+xref: RHEA:48968
 is_a: GO:0140905 ! haloperoxidase activity
 property_value: skos:exactMatch EC:1.11.1.8
-property_value: skos:exactMatch RHEA:23336
+property_value: skos:narrowMatch RHEA:23336
+property_value: skos:narrowMatch RHEA:48956
+property_value: skos:narrowMatch RHEA:48960
+property_value: skos:narrowMatch RHEA:48964
+property_value: skos:narrowMatch RHEA:48968
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21027" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23587" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30597" xsd:anyURI
 
 [Term]
 id: GO:0004448
@@ -44690,6 +44802,7 @@
 xref: EC:4.4.1.20
 xref: KEGG_REACTION:R03059
 xref: MetaCyc:LEUKOTRIENE-C4-SYNTHASE-RXN
+xref: Reactome:R-HSA-2162019 "LTA4 is converted to EXA4 by ALOX15"
 xref: Reactome:R-HSA-266050 "LTA4 is converted to LTC4 by LTC4S"
 xref: RHEA:17617
 is_a: GO:0016846 ! carbon-sulfur lyase activity
@@ -44748,7 +44861,7 @@
 namespace: molecular_function
 alt_id: GO:0003996
 def: "Catalysis of the reaction: a long-chain fatty acid + ATP + CoA = a long-chain fatty acyl-CoA + AMP + diphosphate. A long-chain fatty acid has an aliphatic tail containing 13 to 22 carbons." [RHEA:15421]
-comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283). Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
 synonym: "acyl-activating enzyme activity" BROAD []
 synonym: "acyl-CoA ligase activity" RELATED []
 synonym: "acyl-CoA synthetase activity" RELATED [EC:6.2.1.3]
@@ -44789,16 +44902,16 @@
 property_value: skos:exactMatch EC:6.2.1.3
 property_value: skos:exactMatch MetaCyc:RXN-7904
 property_value: skos:exactMatch RHEA:15421
-property_value: skos:relatedMatch MetaCyc:PWY-5143
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18967" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20462" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25799" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26445" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004468
-name: L-lysine N-acetyltransferase activity, acting on acetyl phosphate as donor
+name: L-lysine N6-acetyltransferase activity, acting on acetyl phosphate as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: acetyl phosphate + L-lysine = phosphate + N6-acetyl-L-lysine." [RHEA:14417]
 synonym: "acetyl-phosphate:L-lysine 6-N-acetyltransferase activity" RELATED [EC:2.3.1.32]
@@ -45069,6 +45182,7 @@
 xref: Reactome:R-HSA-9694492 "nsp14 acts as a cap N7 methyltransferase to modify SARS-CoV-2 gRNA complement (minus strand)"
 xref: Reactome:R-HSA-9694737 "nsp14 acts as a cap N7 methyltransferase to modify SARS-CoV-2 gRNA (plus strand)"
 xref: Reactome:R-HSA-9830848 "L protein acts as a cap N7 methyltransferase to modify RSV mRNAs"
+xref: Reactome:R-HSA-9923104 "NS5 acts as a cap N7 methyltransferase to methylate 5 G pppA of DENV genomic +ssRNA"
 is_a: GO:0008170 ! N-methyltransferase activity
 is_a: GO:0008174 ! mRNA methyltransferase activity
 relationship: part_of GO:0006370 ! 7-methylguanosine mRNA capping
@@ -45096,6 +45210,7 @@
 xref: Reactome:R-HSA-9694521 "nsp16 acts as a cap 2'-O-methyltransferase to modify SARS-CoV-2 gRNA complement (minus strand)"
 xref: Reactome:R-HSA-9694721 "nsp16 acts as a cap 2'-O-methyltransferase to modify SARS-CoV-2 gRNA (plus strand)"
 xref: Reactome:R-HSA-9834003 "L protein acts as a cap 2'-O-methyltransferase to modify RSV mRNAs"
+xref: Reactome:R-HSA-9923097 "NS5 acts as a cap 2'-O-methyltransferase to methylate 5 m7GpppA of DENV genomic +ssRNA"
 xref: RHEA:67020
 is_a: GO:0008171 ! O-methyltransferase activity
 is_a: GO:0008173 ! RNA methyltransferase activity
@@ -45118,6 +45233,7 @@
 xref: Reactome:R-HSA-77081 "Formation of the CE:GMP intermediate complex"
 xref: Reactome:R-HSA-77083 "Transfer of GMP from the capping enzyme GT site to 5'-end of mRNA"
 xref: Reactome:R-HSA-9815529 "nsp12 transfers guanylyl onto SARS-CoV-2 plus strand subgenomic RNAs"
+xref: Reactome:R-HSA-9923071 "NS5 transfers guanylyl onto DENV genomic 5'-ppRNA"
 xref: RHEA:54592
 is_a: GO:0008192 ! RNA guanylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.50
@@ -45139,8 +45255,8 @@
 xref: EC:6.4.1.4
 xref: KEGG_REACTION:R04138
 xref: MetaCyc:METHYLCROTONYL-COA-CARBOXYLASE-RXN
-xref: Reactome:R-HSA-508308 "beta-methylglutaconyl-CoA + ADP + orthophosphate <=> beta-methylcrotonyl-CoA + ATP + CO2 (MCCA)"
-xref: Reactome:R-HSA-70773 "beta-methylcrotonyl-CoA + ATP + CO2 <=> beta-methylglutaconyl-CoA + ADP + orthophosphate + H2O [MCCA]"
+xref: Reactome:R-HSA-508308 "beta-methylglutaconyl-CoA + ADP + orthophosphate <=> beta-methylcrotonyl-CoA + ATP + HCO3- (MCCA)"
+xref: Reactome:R-HSA-70773 "beta-methylcrotonyl-CoA + ATP + HCO3- <=> beta-methylglutaconyl-CoA + ADP + orthophosphate + H2O (MCCA)"
 xref: Reactome:R-HSA-9909466 "MCCC mutants don't synthesize beta-methylglutaconyl-CoA"
 xref: RHEA:13589
 is_a: GO:0016421 ! CoA carboxylase activity
@@ -45156,8 +45272,6 @@
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "5,10-methylene-THF dehydrogenase activity" EXACT [GOC:vw]
 synonym: "N5,N10-methylenetetrahydrofolate dehydrogenase activity" RELATED []
-xref: Reactome:R-HSA-200644 "5,10-methyleneTHF polyglutamate + NADP+ <=> 5,10-methenylTHF polyglutamate + NADPH + H+"
-xref: Reactome:R-HSA-200718 "MTHFD1 dimer dehydrogenates 5,10-methenyl-THFPG to 5,10-methylene-THFPG"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30105" xsd:anyURI
 is_obsolete: true
@@ -45191,6 +45305,8 @@
 xref: EC:1.5.1.5
 xref: KEGG_REACTION:R01220
 xref: MetaCyc:METHYLENETHFDEHYDROG-NADP-RXN
+xref: Reactome:R-HSA-200644 "5,10-methyleneTHF polyglutamate + NADP+ <=> 5,10-methenylTHF polyglutamate + NADPH + H+"
+xref: Reactome:R-HSA-200718 "MTHFD1 dimer dehydrogenates 5,10-methenyl-THFPG to 5,10-methylene-THFPG"
 xref: RHEA:22812
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.5
@@ -45272,7 +45388,7 @@
 synonym: "MSDH activity" RELATED [EC:1.2.1.27]
 xref: EC:1.2.1.27
 xref: MetaCyc:1.2.1.27-RXN
-xref: Reactome:R-HSA-70893 "methylmalonate semialdehyde + NAD+ + CoA => propionyl-CoA + CO2 + NADH + H+"
+xref: Reactome:R-HSA-70893 "methylmalonate semialdehyde + NAD+ + CoA + H2O => propionyl-CoA + HCO3- + NADH + H+"
 xref: RHEA:20804
 xref: RHEA:76615
 is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
@@ -45426,9 +45542,6 @@
 xref: Reactome:R-HSA-5602272 "Defective CYP4F22 does not 20-hydroxylate TrXA3"
 xref: Reactome:R-HSA-5602295 "CYP4F22 20-hydroxylates TrXA3"
 xref: Reactome:R-HSA-5605147 "Defective CYP1B1 does not 4-hydroxylate EST17b"
-xref: Reactome:R-HSA-5662662 "Tyrosinase oxidises tyrosine to dopaquinone"
-xref: Reactome:R-HSA-5662692 "Dopa is oxidized to dopaquinone by TYR"
-xref: Reactome:R-HSA-5663050 "DHI and DHICA polymerize forming eumelanin"
 xref: Reactome:R-HSA-76354 "Vinyl chloride is oxidized to 2-Chloroethylene oxide"
 xref: Reactome:R-HSA-76373 "N-hydroxylation of 4-aminobiphenyl"
 xref: Reactome:R-HSA-76386 "CYP1A2 S-demethylates 6MMP"
@@ -45437,7 +45550,7 @@
 xref: Reactome:R-HSA-76426 "N-atom dealkylation of caffeine"
 xref: Reactome:R-HSA-76434 "Dehalogenation of carbon tetrachloride to form a free radical"
 xref: Reactome:R-HSA-76456 "O-atom dealkylation of dextromethorphan"
-xref: Reactome:R-HSA-76466 "CYP4A11 12-hydroxylates DDCX"
+xref: Reactome:R-HSA-76466 "CYP4A11 omega-hydroxylates DDCX"
 xref: Reactome:R-HSA-76472 "Ethylene is oxidized to Ethylene oxide by CYP1A1"
 xref: Reactome:R-HSA-76475 "Dehalogenation of the poly-halogenated hydrocarbon Halothane to form the acylhalide Trifluoroacetlychloride and  hydrogen bromide"
 xref: Reactome:R-HSA-8865107 "MICAL1 produces NADP+, H2O2"
@@ -45971,6 +46084,9 @@
 xref: Reactome:R-HSA-5690990 "5'- incision of DNA by ERCC1:ERCC4 in GG-NER"
 xref: Reactome:R-HSA-5693533 "DCLRE1C (ARTEMIS) processes DNA DSB ends"
 xref: Reactome:R-HSA-72180 "Cleavage of mRNA at the 3'-end"
+xref: Reactome:R-HSA-9970193 "CPSF3 cleaves pre-mRNA at distal 3'UTR PAS"
+xref: Reactome:R-HSA-9970282 "U1 snRNP suppresses CPSF3-mediated pre-mRNA cleavage at intronic PAS"
+xref: Reactome:R-HSA-9970294 "CPSF3 cleaves pre-mRNA at proximal 3'UTR PAS"
 is_a: GO:0004518 ! nuclease activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
 
@@ -46016,6 +46132,7 @@
 xref: Reactome:R-HSA-927836 "SMG6 hydrolyzes mRNA with premature termination codon"
 xref: Reactome:R-HSA-9708327 "ANG cleaves tRNA to yield tRNA halves"
 xref: Reactome:R-HSA-9708812 "ELAC2 cleaves pre-tRNA to yield 3' trailer (type II tRF or tRF-1)"
+xref: Reactome:R-HSA-9948427 "ANKZF1 cleaves the peptidyl-tRNA in VCP hexamer:LTN1:NEMF:60S ribosome subunit:peptidyl-tRNA with K48polyUb-nascent peptide"
 is_a: GO:0004519 ! endonuclease activity
 is_a: GO:0004540 ! RNA nuclease activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
@@ -46084,6 +46201,9 @@
 xref: Reactome:R-HSA-173769 "RNase H-mediated digestion of tRNA, 3'PPT and cPPT RNA primers"
 xref: Reactome:R-HSA-182795 "RNase H-mediated degradation of the template strand"
 xref: Reactome:R-HSA-182859 "RNase H-mediated degradation of the RNA strand of the -sssDNA:RNA duplex"
+xref: Reactome:R-HSA-9913637 "RNASEH1 cleaves R-loop RNA to yield RNA primer for H strand polymerization"
+xref: Reactome:R-HSA-9914858 "RNASEH1 cleaves RNA primer from nascent H strand, leaving RNA dinucleotides"
+xref: Reactome:R-HSA-9915449 "RNASEH1 cleaves RNA primer from nascent L strand, leaving RNA dinucleotides"
 is_a: GO:0016891 ! RNA endonuclease activity producing 5'-phosphomonoesters, hydrolytic mechanism
 property_value: skos:exactMatch EC:3.1.26.4
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -46250,6 +46370,12 @@
 def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' terminus of an RNA molecule." [GOC:mah, ISBN:0198547684]
 synonym: "5'-3' exoribonuclease activity" EXACT []
 xref: Reactome:R-HSA-429845 "5' to 3' exoribonuclease hydrolyzes decapped mRNA"
+xref: Reactome:R-HSA-6791227 "47S pre-rRNA is nucleolytically processed at A' (01,A1), site A0, and site 02 (site 6)  to yield 45S pre-rRNA"
+xref: Reactome:R-HSA-9915442 "EXOG cleaves RNA dinucleotide from nascent mitochondrial DNA"
+xref: Reactome:R-HSA-9935265 "DXO hydrolyzes uncapped RNA"
+xref: Reactome:R-HSA-9935299 "XRN2 5'-3' exonucleolytically hydrolyzes uncapped RNA"
+xref: Reactome:R-HSA-9946917 "XRN1 incompletely hydrolyzes DENV decapped mRNA"
+xref: Reactome:R-HSA-9971054 "XRN2 degrades downstream RNA fragment, promoting transcription termination"
 is_a: GO:0008409 ! 5'-3' exonuclease activity
 is_a: GO:0016896 ! RNA exonuclease activity, producing 5'-phosphomonoesters
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28062" xsd:anyURI
@@ -46267,6 +46393,8 @@
 xref: Reactome:R-HSA-429955 "CCR4-NOT complex deadenylates mRNA"
 xref: Reactome:R-HSA-429992 "PARN deadenylates mRNA"
 xref: Reactome:R-HSA-430021 "PAN2-PAN3 complex partially deadenylates mRNA"
+xref: Reactome:R-HSA-9009950 "PDE12 cleaves 2'-5' oligoadenylates"
+xref: Reactome:R-HSA-9615042 "Viral 2',5'-PDE cleaves 2'-5' oligoadenylates"
 xref: Reactome:R-HSA-9822311 "CCR4-NOT deadenylates mRNA in CCR4-NOT:BTG4:PABPN1L:mRNP"
 xref: Reactome:R-HSA-9822335 "CCR4-NOT(CNOT6L) deadenylates mRNA in CCR4-NOT(CNOT6L):ZFP36L2:mRNA"
 is_a: GO:0000175 ! 3'-5'-RNA exonuclease activity
@@ -46563,18 +46691,14 @@
 xref: MetaCyc:3.2.1.52-RXN
 xref: Reactome:R-HSA-1605595 "bHEXA hydrolyzes GM2A:GM2 to GM2A:GM3"
 xref: Reactome:R-HSA-1605632 "bHEXA,bHEXB hydrolyze PSAP(195-273):Gb4Cer:PE"
-xref: Reactome:R-HSA-1638053 "HEXA cleaves the terminal GalNAc from keratan sulfate"
+xref: Reactome:R-HSA-1638053 "HEXA cleaves the terminal GlcNAc from keratan sulfate"
 xref: Reactome:R-HSA-2105001 "HEXA cleaves the terminal GalNAc from DS"
-xref: Reactome:R-HSA-2162225 "HEXA cleaves the terminal GalNAc from small HA fragments"
-xref: Reactome:R-HSA-3656259 "Defective HEXA does not cleave the terminal GalNAc from DS"
-xref: Reactome:R-HSA-3662344 "Defective HEXB does not cleave the terminal GalNAc from DS"
+xref: Reactome:R-HSA-2162225 "HEXA cleaves the terminal GlcNAc from small HA fragments"
 xref: Reactome:R-HSA-9035976 "Defective HEXA does not cleave the terminall GalNAc from small HA fragments"
-xref: Reactome:R-HSA-9035978 "Defective HEXA does not cleave the terminal GalNAc from keratan sulfate"
 xref: Reactome:R-HSA-9035982 "Defective HEXB does not cleave the terminal GalNAc from HA fragments"
-xref: Reactome:R-HSA-9035983 "Defective HEXB does not cleave the terminal GalNAc from keratan sulfate"
 xref: Reactome:R-HSA-9638075 "HEXB cleaves the terminal GalNAc from DS"
-xref: Reactome:R-HSA-9638076 "HEXB cleaves the terminal GalNAc from small HA fragments"
-xref: Reactome:R-HSA-9638078 "HEXB cleaves the terminal GalNAc from keratan sulfate"
+xref: Reactome:R-HSA-9638076 "HEXB cleaves the terminal GlcNAc from small HA fragments"
+xref: Reactome:R-HSA-9638078 "HEXB cleaves the terminal GlcNAc from keratan sulfate"
 xref: Reactome:R-HSA-9840833 "bHEXA,bHEXB hydrolyze GM2A:GA2 to GM2A:LacCer"
 xref: Reactome:R-HSA-9840884 "bHEXA,bHEXS hydrolyze GM2A:SM2"
 is_a: GO:0015929 ! hexosaminidase activity
@@ -46609,7 +46733,6 @@
 name: beta-galactosidase activity
 namespace: molecular_function
 def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-galactose residues in beta-D-galactosides." [EC:3.2.1.23]
-comment: Note that the inclusion of 'MetaCyc:BGALACT-PWY' is exceptional: normally MetaCyc pathway entries are database references for biological process terms, not molecular function terms. An exception was made in this case because the MetaCyc entry 'BGALACT-PWY' describes only one reaction, that catalyzed by beta-galactosidase.
 synonym: "beta-D-galactanase activity" RELATED []
 synonym: "beta-D-galactoside galactohydrolase activity" RELATED []
 synonym: "beta-D-lactosidase activity" RELATED []
@@ -46621,7 +46744,7 @@
 xref: Reactome:R-HSA-1605624 "Beta-galactosidases hydrolyse mobilized GM1 to mobilized GM2"
 xref: Reactome:R-HSA-1606312 "GLB1 hydrolyzes SapB/C:LacCer"
 xref: Reactome:R-HSA-1630306 "GLB1 hydrolyses a glycosaminoglycan"
-xref: Reactome:R-HSA-1793217 "Keratan sulfate is cleaved from its proteoglycan by an unknown galactosidase"
+xref: Reactome:R-HSA-1793217 "Unknown endo--galactosidase cleaves keratan chain"
 xref: Reactome:R-HSA-2090079 "GLB1 hydrolyses linker chain(2)"
 xref: Reactome:R-HSA-2265534 "Defective GLB1 does not hydrolyse a glycosaminoglycan"
 xref: Reactome:R-HSA-9036061 "Defective GLB1 does not hydrolyse linker chain(2)"
@@ -46629,8 +46752,8 @@
 is_a: GO:0015925 ! galactosidase activity
 property_value: skos:exactMatch EC:3.2.1.23
 property_value: skos:exactMatch MetaCyc:3.2.1.23-RXN
-property_value: skos:relatedMatch MetaCyc:BGALACT-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004566
@@ -46645,7 +46768,7 @@
 xref: EC:3.2.1.31
 xref: MetaCyc:BETA-GLUCURONID-RXN
 xref: Reactome:R-HSA-1678854 "GUSB tetramer hydrolyses CS/HS precursor"
-xref: Reactome:R-HSA-2162226 "GUSB tetramer hydrolyses GlcA-1,3-GlcNAc"
+xref: Reactome:R-HSA-2162226 "GUSB tetramer hydrolyzes GlcA-1,3-GlcNAc"
 xref: Reactome:R-HSA-2162227 "GUSB tetramer hydrolyses (HA)2"
 xref: Reactome:R-HSA-2318373 "Defective GUSB does not hydrolyse (HA)2"
 xref: Reactome:R-HSA-9036068 "Defective GUSB does not hydrolyse GlcA-1,3-GlcNAc"
@@ -46787,6 +46910,7 @@
 xref: Reactome:R-HSA-4793947 "Defective MOGS does not cleave glucose from an N-glycosylated protein"
 xref: Reactome:R-HSA-532678 "Trimming of the first glucose by by mannosyl-oligosaccharide glucosidase"
 xref: Reactome:R-HSA-9694364 "N-glycan glucose trimming of Spike"
+xref: Reactome:R-HSA-9932344 "MOGS removes terminal glucose from Glu3Man9GlcNAc2-CDH1"
 xref: RHEA:55988
 is_a: GO:0090599 ! alpha-glucosidase activity
 property_value: skos:exactMatch EC:3.2.1.106
@@ -46910,6 +47034,11 @@
 xref: MetaCyc:RXN-16761
 xref: Reactome:R-HSA-446209 "Transfer of N-glycan to the protein"
 xref: Reactome:R-HSA-9694793 "Spike protein gets N-glycosylated"
+xref: Reactome:R-HSA-9816276 "CDH1 is N-glycosylated on asparagine residues in endoplasmic reticulum"
+xref: Reactome:R-HSA-9918962 "E is N-glycosylated"
+xref: Reactome:R-HSA-9918988 "pre-M is N-glycosylated"
+xref: Reactome:R-HSA-9919011 "Pre-NS1 folds and is N-glycosylated"
+xref: Reactome:R-HSA-9931286 "CD274 N-linked glycosylation in ER"
 xref: RHEA:22980
 is_a: GO:0004576 ! oligosaccharyl transferase activity
 property_value: skos:exactMatch EC:2.4.99.18
@@ -47021,6 +47150,7 @@
 xref: EC:2.1.3.3
 xref: MetaCyc:ORNCARBAMTRANSFER-RXN
 xref: Reactome:R-HSA-70560 "carbamoyl phosphate + ornithine => citrulline + orthophosphate"
+xref: Reactome:R-HSA-9956527 "OTC variants don't synthesize L-citrulline"
 xref: RHEA:19513
 is_a: GO:0016743 ! carboxyl- or carbamoyltransferase activity
 relationship: part_of GO:0006591 ! ornithine metabolic process
@@ -47048,9 +47178,9 @@
 
 [Term]
 id: GO:0004587
-name: ornithine aminotransferase activity
+name: L-ornithine transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2-oxocarboxylate + L-ornithine = an L-alpha-amino acid + L-glutamate 5-semialdehyde." [EC:2.6.1.13]
+def: "Catalysis of the reaction: a 2-oxocarboxylate + L-ornithine = L-glutamate 5-semialdehyde + an L-alpha-amino acid." [RHEA:13877]
 synonym: "GabT" RELATED [EC:2.6.1.13]
 synonym: "L-ornithine 5-aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "L-ornithine aminotransferase activity" RELATED [EC:2.6.1.13]
@@ -47058,6 +47188,7 @@
 synonym: "L-ornithine:alpha-ketoglutarate delta-aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "OAT" RELATED [EC:2.6.1.13]
 synonym: "ornithine 5-aminotransferase activity" RELATED [EC:2.6.1.13]
+synonym: "ornithine aminotransferase activity" EXACT []
 synonym: "ornithine delta-transaminase activity" RELATED [EC:2.6.1.13]
 synonym: "ornithine ketoacid aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "ornithine transaminase activity" RELATED [EC:2.6.1.13]
@@ -47083,6 +47214,7 @@
 property_value: skos:exactMatch RHEA:13877
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25975" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004588
@@ -47371,14 +47503,9 @@
 xref: KEGG_REACTION:R03532
 xref: Reactome:R-HSA-1222346 "AhpC reduces H2O2"
 xref: Reactome:R-HSA-140359 "PGG2 is reduced to PGH2 by PTGS1"
-xref: Reactome:R-HSA-209815 "Tyrosine is monoiodinated"
-xref: Reactome:R-HSA-209840 "Two DITs combine to form thyroxine"
-xref: Reactome:R-HSA-209925 "DIT and MIT combine to form triiodothyronine"
-xref: Reactome:R-HSA-209973 "Tyrosine is diiodinated"
 xref: Reactome:R-HSA-2309773 "PGG2 is reduced to PGH2 by PTGS2"
 xref: Reactome:R-HSA-2559639 "Collagen type IV sulfilimine cross-linking by peroxidasin"
 xref: Reactome:R-HSA-3341296 "GPX7,8 catalyze peroxidation of P4HB (PDI)"
-xref: Reactome:R-HSA-350901 "Iodide is organified"
 xref: Reactome:R-HSA-5631885 "PRDX1 overoxidizes"
 xref: Reactome:R-HSA-6789031 "Membrane-bound myeloperoxidase (MPO) produces hypochlorous acid (HOCl)"
 xref: Reactome:R-HSA-6789126 "Myeloperoxidase (MPO) produces hypochlorous acid (HOCl)"
@@ -47398,7 +47525,7 @@
 name: glutathione peroxidase activity
 namespace: molecular_function
 alt_id: GO:0016224
-def: "Catalysis of the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O." [EC:1.11.1.9, PMID:36771108]
+def: "Catalysis of the reaction: 2 glutathione + H2O2 = oxidized glutathione + 2 H2O." [EC:1.11.1.9, PMID:36771108]
 synonym: "glutathione:hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.9]
 synonym: "GSH peroxidase activity" RELATED [EC:1.11.1.9]
 synonym: "non-selenium glutathione peroxidase activity" NARROW []
@@ -47693,7 +47820,7 @@
 synonym: "phosphogluconic acid dehydrogenase activity" RELATED [EC:1.1.1.44]
 xref: EC:1.1.1.44
 xref: MetaCyc:RXN-9952
-xref: Reactome:R-HSA-71299 "6-phospho-D-gluconate + NADP+ => D-ribulose 5-phosphate + CO2 + NADPH + H+"
+xref: Reactome:R-HSA-71299 "PGD decarboxylates 6-phospho-D-gluconate"
 xref: RHEA:10116
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.44
@@ -47757,7 +47884,7 @@
 xref: KEGG_REACTION:R01512
 xref: MetaCyc:PHOSGLYPHOS-RXN
 xref: Reactome:R-HSA-70486 "PGK complexes (PGK1,2) phosphorylate 3PG to form 1,3BPG"
-xref: Reactome:R-HSA-71850 "1,3-bisphospho-D-glycerate + ADP <=> 3-phospho-D-glycerate + ATP"
+xref: Reactome:R-HSA-71850 "PGK complexes dephosphorylate 1,3BPG"
 xref: RHEA:14801
 is_a: GO:0016301 ! kinase activity
 is_a: GO:0016774 ! phosphotransferase activity, carboxyl group as acceptor
@@ -47783,8 +47910,8 @@
 xref: KEGG_REACTION:R01516
 xref: MetaCyc:3PGAREARR-RXN
 xref: MetaCyc:RXN-15513
-xref: Reactome:R-HSA-71445 "PGAM dimers (PGAM1,2) isomerise 2PG to 3PG"
-xref: Reactome:R-HSA-71654 "3-Phospho-D-glycerate <=> 2-Phospho-D-glycerate"
+xref: Reactome:R-HSA-71445 "PGAM dimers isomerise 2PG to 3PG"
+xref: Reactome:R-HSA-71654 "PGAM dimers isomerise 3PG to 2PG"
 xref: RHEA:15901
 is_a: GO:0016868 ! intramolecular phosphotransferase activity
 property_value: skos:exactMatch RHEA:15901
@@ -47815,14 +47942,16 @@
 id: GO:0004621
 name: GPI anchor phospholipase D activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycoprotein phosphatidylinositol + H2O = phosphatidate + glycoprotein inositol, by cleavage of the second phosphodiester bond between the phosphate and phospholipid." [PMID:2760042]
+def: "Catalysis of the reaction: glycoprotein phosphatidylinositol + H2O = phosphatidate + glycoprotein inositol, by cleavage of the second phosphodiester bond between the phosphate and phospholipid." [PMID:2760042, RHEA:86263]
 synonym: "glycoprotein phospholipase D activity" BROAD []
 synonym: "glycoprotein-phosphatidylinositol phosphatidohydrolase activity" EXACT []
 synonym: "glycosylphosphatidyl inositol-anchor protein phospholipase D activity" EXACT []
 synonym: "GPI-PLD activity" EXACT []
 xref: Reactome:R-HSA-8940388 "GPLD1 hydrolyses GPI-anchors from proteins"
+xref: RHEA:86263
 is_a: GO:0004630 ! D-type glycerophospholipase activity
 property_value: skos:broadMatch EC:3.1.4.50
+property_value: skos:exactMatch RHEA:86263
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28868" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31078" xsd:anyURI
 
@@ -48104,7 +48233,7 @@
 xref: EC:4.2.1.11
 xref: MetaCyc:2PGADEHYDRAT-RXN
 xref: Reactome:R-HSA-70494 "Enolase dimers (ENO1,2,3) convert PEP to 2PG"
-xref: Reactome:R-HSA-71660 "2-Phospho-D-glycerate <=> Phosphoenolpyruvate + H2O"
+xref: Reactome:R-HSA-71660 "Enolase dimers dehydrates 2PG"
 xref: RHEA:10164
 is_a: GO:0016836 ! hydro-lyase activity
 property_value: skos:exactMatch EC:4.2.1.11
@@ -48364,15 +48493,16 @@
 
 [Term]
 id: GO:0004648
-name: O-phospho-L-serine:2-oxoglutarate aminotransferase activity
+name: O-phospho-L-serine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 alt_id: GO:0004646
-def: "Catalysis of the reaction: O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate + L-glutamate." [EC:2.6.1.52]
+def: "Catalysis of the reaction: O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate + L-glutamate." [RHEA:14329]
 synonym: "3-O-phospho-L-serine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.52]
 synonym: "3-phosphoserine aminotransferase activity" RELATED [EC:2.6.1.52]
 synonym: "3PHP transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "hydroxypyruvic phosphate--glutamic transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "L-phosphoserine aminotransferase activity" RELATED [EC:2.6.1.52]
+synonym: "O-phospho-L-serine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "PdxC" RELATED [EC:2.6.1.52]
 synonym: "phosphohydroxypyruvate transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "phosphohydroxypyruvic--glutamic transaminase activity" RELATED [EC:2.6.1.52]
@@ -48388,6 +48518,7 @@
 property_value: skos:exactMatch EC:2.6.1.52
 property_value: skos:exactMatch RHEA:14329
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004649
@@ -48437,6 +48568,7 @@
 xref: EC:3.1.3.33
 xref: MetaCyc:RXN-21861
 xref: Reactome:R-HSA-77078 "Hydrolysis of the 5'-end of the nascent transcript by the capping enzyme"
+xref: Reactome:R-HSA-9923980 "NS3 cleaves triphosphate moiety of genomic +ssRNA"
 xref: RHEA:67772
 xref: RHEA:67776
 is_a: GO:0016791 ! phosphatase activity
@@ -48594,8 +48726,7 @@
 xref: EC:1.5.5.2
 xref: MetaCyc:RXN-14903
 xref: Reactome:R-HSA-6784224 "PRODH2:FAD dimer dehydrogenates HPRO to 1-pyrroline-3-hydroxy-5-carboxylate"
-xref: Reactome:R-HSA-70670 "PRODH oxidises L-Pro to 1PYR-5COOH"
-xref: Reactome:R-HSA-8955817 "PRODH2 oxidises HPRO to 1PYR-5COOH"
+xref: Reactome:R-HSA-8955817 "PRODH2 oxidises HPRO to 1PYR-3OH-5COOH"
 xref: RHEA:23784
 is_a: GO:0016649 ! oxidoreductase activity, acting on the CH-NH group of donors, quinone or similar compound as acceptor
 property_value: skos:exactMatch EC:1.5.5.2
@@ -48645,6 +48776,7 @@
 xref: MetaCyc:RXN-17573
 xref: Reactome:R-HSA-2530501 "FNTA:FNTB transfers FARN to GNGT1"
 xref: Reactome:R-HSA-9647978 "pro-RAS proteins are farnesylated"
+xref: Reactome:R-HSA-9948016 "FNTA:FNTB transfers farnesyl to GBP1"
 xref: RHEA:13345
 is_a: GO:0008318 ! protein prenyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.58
@@ -48676,6 +48808,8 @@
 synonym: "type I protein geranyl-geranyltransferase activity" RELATED [EC:2.5.1.59]
 xref: EC:2.5.1.59
 xref: MetaCyc:RXN-3701
+xref: Reactome:R-HSA-9947999 "FNTA:PGGT1B geranylgeranylates GBP2"
+xref: Reactome:R-HSA-9954195 "FNTA:PGGT1B geranylgeranylates GBP5"
 is_a: GO:0004661 ! protein geranylgeranyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.59
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25798" xsd:anyURI
@@ -48843,6 +48977,7 @@
 subset: goslim_candida
 synonym: "protamine kinase activity" NARROW []
 xref: Reactome:R-HSA-156832 "INF-gamma induced phosphorylation of L13a"
+xref: Reactome:R-HSA-5693148 "BCKDK phosphorylates BCKDH"
 xref: Reactome:R-HSA-937034 "IRAK1 phosphorylates Pellino"
 xref: Reactome:R-HSA-9604606 "Unknown kinase phosphorylates NICD4"
 xref: Reactome:R-HSA-9673284 "GTP-bound RAC1 contributes to MAPK8 activation"
@@ -48966,9 +49101,9 @@
 xref: Reactome:R-HSA-168140 "Active IKK Complex phosphorylates NF-kappa-B inhibitor"
 xref: Reactome:R-HSA-168184 "Activated TAK1 mediates phosphorylation of the IKK Complex"
 xref: Reactome:R-HSA-170055 "Myt-1 mediated phosphorylation of Cyclin B:Cdc2 complexes"
-xref: Reactome:R-HSA-170076 "CAK-mediated phosphorylation of Cyclin B1:Cdc2 complexes"
-xref: Reactome:R-HSA-170087 "CAK-mediated phosphorylation of Cyclin A:Cdc2 complexes"
-xref: Reactome:R-HSA-170116 "Myt-1 mediated phosphorylation of Cyclin A:Cdc2"
+xref: Reactome:R-HSA-170076 "CAK phosphorylates CDK1 bound to CCNB1"
+xref: Reactome:R-HSA-170087 "CAK phosphorylates CDK1 bound to CCNA"
+xref: Reactome:R-HSA-170116 "PKMYT1 phosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170126 "Phosphorylation of Cyclin B1 in the CRS domain"
 xref: Reactome:R-HSA-170977 "FRS2 is phosphorylated by active TrkA receptor"
 xref: Reactome:R-HSA-174119 "Free APC/C phosphorylated by Plk1"
@@ -48979,6 +49114,7 @@
 xref: Reactome:R-HSA-187949 "CAK-mediated phosphorylation of Cyclin A:Cdk2"
 xref: Reactome:R-HSA-188350 "CAK-mediated phosphorylation of Cyclin E:Cdk2"
 xref: Reactome:R-HSA-193647 "IRAK is activated"
+xref: Reactome:R-HSA-193703 "IKKbeta is activated"
 xref: Reactome:R-HSA-193705 "IKKbeta phosphorylates IkB causing NF-kB to dissociate"
 xref: Reactome:R-HSA-195275 "Phosphorylation of APC component of the destruction complex"
 xref: Reactome:R-HSA-195283 "Phosphorylation of phospho- (Ser45, Thr41) beta-catenin  at Ser37 by GSK-3"
@@ -49117,7 +49253,6 @@
 xref: Reactome:R-HSA-399951 "Phosphorylation of CRMPs by GSK3beta"
 xref: Reactome:R-HSA-399952 "Phosphorylation of LIMK-1 by PAK"
 xref: Reactome:R-HSA-399978 "Protein kinase C, alpha type phosphorylates MARCKS"
-xref: Reactome:R-HSA-400382 "CSNK1E,CSNK1D phosphorylate CRY and PER proteins"
 xref: Reactome:R-HSA-4088134 "PLK1 phosphorylates FOXM1"
 xref: Reactome:R-HSA-419083 "Myosin phosphatase inactivation by ROCK"
 xref: Reactome:R-HSA-419087 "LIM kinase phosphorylation by ROCK"
@@ -49178,8 +49313,10 @@
 xref: Reactome:R-HSA-5610722 "CK1 phosphorylates p-GLI3"
 xref: Reactome:R-HSA-5610730 "GSK3 phosphorylates p-GLI2"
 xref: Reactome:R-HSA-5610732 "GSK3 phosphorylates p-GLI3"
+xref: Reactome:R-HSA-5623667 "PDPK1 phosphorylates PKN1,2,3"
 xref: Reactome:R-HSA-5624473 "Protein kinase A (PKA) and RPS6KA5 (MSK1) phosphorylates p65 (RELA) subunit"
 xref: Reactome:R-HSA-5624492 "PAK phosphorylates p21 RAF1 on S338"
+xref: Reactome:R-HSA-5625784 "PKN1 phosphorylates histone 3 of nucleosomes associate with KLK2 and KLK3 promoters"
 xref: Reactome:R-HSA-5627775 "Autophosphorylation of PAK1,2,3"
 xref: Reactome:R-HSA-5632670 "CSNK1A1 phosphorylates SMO dimer"
 xref: Reactome:R-HSA-5632672 "ADRBK1 phosphorylates SMO dimer"
@@ -49192,6 +49329,7 @@
 xref: Reactome:R-HSA-5668947 "PAK1 phosphorylates myosin phosphatase"
 xref: Reactome:R-HSA-5668984 "PAK1 or PAK2 phosphorylates MYLK"
 xref: Reactome:R-HSA-5669250 "PAK1 phosphorylates FLNA"
+xref: Reactome:R-HSA-5671749 "p-T774-PKN1 phosphorylates CDC25C"
 xref: Reactome:R-HSA-5671763 "p-T774-PKN1 phosphorylates PPP1R14A"
 xref: Reactome:R-HSA-5671919 "Activated CIT phosphorylates MRLCs"
 xref: Reactome:R-HSA-5672008 "Thr-180 of ULK1 is phosphorylated"
@@ -49274,8 +49412,8 @@
 xref: Reactome:R-HSA-6810233 "CDK7 phosphorylates serine-5 and serine-7 of heptad repeats in C-terminal domain of RNA polymerase II at snRNA promoter"
 xref: Reactome:R-HSA-6811454 "MAPKs phosphorylate PP2A"
 xref: Reactome:R-HSA-6814409 "CK2 phosphorylates PDCL"
-xref: Reactome:R-HSA-69604 "Phosphorylation of Cdc25A at Ser-123 by Chk1"
-xref: Reactome:R-HSA-69608 "Phosphorylation of Cdc25A at Ser-123 by Chk2"
+xref: Reactome:R-HSA-69604 "Phosphorylation of CDC25A by CHEK1"
+xref: Reactome:R-HSA-69608 "Phosphorylation of CDC25A by CHEK2"
 xref: Reactome:R-HSA-69685 "CHEK2 phosphorylates TP53"
 xref: Reactome:R-HSA-69891 "Phosphorylation and activation of CHEK2 by ATM"
 xref: Reactome:R-HSA-75010 "Phosphorylation of Cdc25C at Ser 216 by Chk1"
@@ -49323,6 +49461,7 @@
 xref: Reactome:R-HSA-9009208 "Activated ERKs phosphorylate RUNX2"
 xref: Reactome:R-HSA-9012319 "p-TEFb phosphorylates serine 2 in RNA polymerase II CTD"
 xref: Reactome:R-HSA-9013978 "Phosphorylation of IRF-3/IRF7 and their release from the activated TLR3 complex"
+xref: Reactome:R-HSA-9021357 "PRKCI phosphorylates ELF3"
 xref: Reactome:R-HSA-9022314 "HIPK2 phosphorylates MECP2"
 xref: Reactome:R-HSA-9023132 "AURKB phosphorylates MECP2 at S423"
 xref: Reactome:R-HSA-9032751 "Estrogen-independent phosphorylation of ESR1 S118 by MAPK1 and MAPK3"
@@ -49330,6 +49469,7 @@
 xref: Reactome:R-HSA-912470 "ATR phosphorylates Histone H2A.X at unsynapsed regions"
 xref: Reactome:R-HSA-913996 "PKA/PKG phosphorylate Rap1GAP2"
 xref: Reactome:R-HSA-918229 "Phosphorylation and release of IRF3/IRF7"
+xref: Reactome:R-HSA-927889 "SMG1 phosphorylates UPF1 (enhanced by Exon Junction Complex)"
 xref: Reactome:R-HSA-933525 "Phosphorylation and release of IRF7"
 xref: Reactome:R-HSA-934559 "SPRY2 is phosphorylated by phosphorylated MNK1"
 xref: Reactome:R-HSA-936951 "Activation of TAK1 complex bound to activated TLR4 complex"
@@ -49349,6 +49489,7 @@
 xref: Reactome:R-HSA-9624526 "AKT phosphorylates FOXO3 downstream of ESR1 and EGFR"
 xref: Reactome:R-HSA-9626880 "MAPK11 or MAPK14 phosphorylates NCF1 at Ser345"
 xref: Reactome:R-HSA-9627089 "CASP9 is phosphorylated at T412"
+xref: Reactome:R-HSA-9632858 "PRKCZ autophosphorylates downstream of estrogen stimulation"
 xref: Reactome:R-HSA-9632868 "CDKN1B is phosphorylated in response to estrogen"
 xref: Reactome:R-HSA-9633008 "p-T899-EIF2AK4 (GCN2) phosphorylates EIF2AS1"
 xref: Reactome:R-HSA-9633742 "EIF2AK4 (GCN2) dimer autophosphorylates"
@@ -49385,9 +49526,11 @@
 xref: Reactome:R-HSA-975874 "Second phosphorylation of IRAK1 by IRAK4 bound to MyD88:activated TLR complex"
 xref: Reactome:R-HSA-975878 "First phosphorylation of IRAK1 by IRAK4 bound to MyD88:activated TLR"
 xref: Reactome:R-HSA-9762094 "GSK3B phosphorylates p-NFE2L2"
+xref: Reactome:R-HSA-9765960 "ERKs phosphorylate MCRIP1"
 xref: Reactome:R-HSA-9766532 "SQSTM1 oligomer is phosphorylated"
 xref: Reactome:R-HSA-9769115 "MAPK1,(MAPK3) phosphorylates NPAS4"
 xref: Reactome:R-HSA-9773803 "Active IKBKB phosphorylates NF-kappa-B inhibitor"
+xref: Reactome:R-HSA-9796053 "PRKCI phosphorylates NFE2L2"
 xref: Reactome:R-HSA-9796067 "PRKAA2 phosphorylates nuclear NFE2L2"
 xref: Reactome:R-HSA-9796081 "EIF2AK3 phosphorylates NFE2L2"
 xref: Reactome:R-HSA-9815501 "MAPKAPK2 phosphorylates RIPK1 at S320"
@@ -49411,10 +49554,31 @@
 xref: Reactome:R-HSA-9851972 "PLK1 phosphorylates FIRRM at S43"
 xref: Reactome:R-HSA-9853369 "PLK1 phosphorylates FIRMM at S744"
 xref: Reactome:R-HSA-9855910 "USF1 is phosphorylated by p-MAPK14"
+xref: Reactome:R-HSA-9857661 "p-T308,S473-AKT1 (phospho-PKB) phosphorylates GJA1 (Connexin-43)"
+xref: Reactome:R-HSA-9857673 "p-T308,S473-AKT1 phosphorylates ITGA5 (Integrin alpha5) in ITGA5:ITGB1 (Integrin alpha5:Integrin beta1)"
 xref: Reactome:R-HSA-9860292 "Activated IKBKB phosphorylates IRF5"
 xref: Reactome:R-HSA-9860785 "PDPK1 (PDK1) phosphorylates PKN2 on threonine-816"
 xref: Reactome:R-HSA-9860800 "mTORC2 phosphorylates AKT1 on serine-473"
 xref: Reactome:R-HSA-9861642 "NEK1 phosphorylates ME1"
+xref: Reactome:R-HSA-9909369 "CSNK1E,CSNK1D phosphorylate CRY and PER proteins"
+xref: Reactome:R-HSA-9912396 "CDK5 phosphorylates CLOCK on serine-451 and serine-461"
+xref: Reactome:R-HSA-9912401 "CK2alpha phosphorylates BMAL1 (ARNTL) on serine-90"
+xref: Reactome:R-HSA-9920673 "CDK5 phosphorylates PER2 on serine-396"
+xref: Reactome:R-HSA-9922273 "CSNK1D,E phosphorylates CLOCK in p-S-CRY:p-S-PER:CSNK1D,E:p-S90,Ac-538-BMAL:p-2T,3S-CLOCK:target gene"
+xref: Reactome:R-HSA-9926483 "MARK3 phosphorylates MITF-M on S173"
+xref: Reactome:R-HSA-9929360 "GSK3B phosphorylates CD274"
+xref: Reactome:R-HSA-9931292 "AMPK phosphorylates CD274"
+xref: Reactome:R-HSA-9931409 "NEK2 phosphorylates CD274"
+xref: Reactome:R-HSA-9931418 "CK2 phosphorylates CD274"
+xref: Reactome:R-HSA-9933380 "Cytosolic tail of CDH1 is phosphorylated"
+xref: Reactome:R-HSA-9942578 "Phosphorylation of CDC25A by p38 MAPK"
+xref: Reactome:R-HSA-9943662 "CHEK1 phosphorylates NEK11"
+xref: Reactome:R-HSA-9943675 "GSK3B phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943676 "NEK11 phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943678 "CHEK1 phosphorylates CHEK2-phosphorylated CDC25A"
+xref: Reactome:R-HSA-9943687 "PLK3 phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943736 "Casein kinase I phosphorylates CDC25A"
+xref: Reactome:R-HSA-9954237 "PIM1 phosphorylates GBP1"
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch EC:2.7.11.-
 property_value: skos:exactMatch MetaCyc:PROTEIN-KINASE-RXN
@@ -49685,7 +49849,7 @@
 xref: EC:2.7.11.11
 xref: Reactome:R-HSA-163672 "Phosphorylation of ChREBP at Thr(653) by PKA"
 xref: Reactome:R-HSA-163676 "Phosphorylation of pChREBP (Thr 653) at Ser(196) by PKA"
-xref: Reactome:R-HSA-163773 "Phosphorylation of PF2K-Pase by PKA catalytic subunit"
+xref: Reactome:R-HSA-163773 "PKA tetramer:cAMP phosphorylates PF2K-Pase"
 xref: Reactome:R-HSA-177275 "PKA phosphorylates DARPP-32 on Thr34"
 xref: Reactome:R-HSA-177284 "PKA phosphorylates PDE4B"
 xref: Reactome:R-HSA-432232 "Phosphorylation of Aquaporin-2 by Protein Kinase A (PKA)"
@@ -49722,6 +49886,7 @@
 xref: Reactome:R-HSA-1497853 "Sepiapterin reductase (SPR) is phosphorylated by Ca2+/calmodulin-dependent protein kinase II"
 xref: Reactome:R-HSA-418442 "PKG1 phosphorylates IRAG:IP3R1 inhibiting IP3-stimulated Ca2+ release"
 xref: Reactome:R-HSA-418549 "PKG1 phosphorylates BK channels"
+xref: Reactome:R-HSA-9920025 "PKG phosphorylates NS5"
 is_a: GO:0004690 ! cyclic nucleotide-dependent protein kinase activity
 relationship: has_part GO:0030553 ! cGMP binding
 property_value: skos:exactMatch EC:2.7.11.12
@@ -49779,7 +49944,7 @@
 xref: Reactome:R-HSA-2990882 "CDK1 phosphorylates NUP98"
 xref: Reactome:R-HSA-3788705 "CDKN1A (p21) prevents phosphorylation of Cdh1 by Cyclin A:Cdk2"
 xref: Reactome:R-HSA-380278 "CCNB1:p-T160-CDK1 phosphorylates NUMA1"
-xref: Reactome:R-HSA-4086410 "CDK1 phosphorylates BORA"
+xref: Reactome:R-HSA-4086410 "CCNB:CDK1 phosphorylates BORA"
 xref: Reactome:R-HSA-4088024 "CCNA:CDK1/2 complexes and CCNB1:CDK1 complexes phosphorylate FOXM1"
 xref: Reactome:R-HSA-5195402 "CDK1 phosphorylates LPIN"
 xref: Reactome:R-HSA-5244669 "CDK1 phosphorylates lamins and facilitates depolymerization of lamin filaments"
@@ -49790,12 +49955,21 @@
 xref: Reactome:R-HSA-68944 "Orc1 is phosphorylated by cyclin A/CDK2"
 xref: Reactome:R-HSA-69005 "Cdc6 protein is phosphorylated by CDK"
 xref: Reactome:R-HSA-69227 "Cyclin D:CDK4/6 phosphorylates RB1 and prevents RB1 binding to E2F1/2/3:DP1/2 complexes"
-xref: Reactome:R-HSA-69754 "Phosphorylation of proteins involved in G2/M transition by active Cyclin A1:Cdc2 complexes"
-xref: Reactome:R-HSA-69756 "Phosphorylation of proteins involved in G2/M transition by active Cyclin A2:Cdc2 complexes"
 xref: Reactome:R-HSA-9008412 "CDK4 phosphorylates RUNX2"
 xref: Reactome:R-HSA-9009282 "CDK1 phosphorylates RUNX2"
 xref: Reactome:R-HSA-9624800 "CDK1 phosphorylates LBR"
 xref: Reactome:R-HSA-9686521 "CDK2:CCNA phosphorylates TERF2"
+xref: Reactome:R-HSA-9929482 "CDK4:CCND1 phosphorylates SPOP"
+xref: Reactome:R-HSA-9929514 "CDK1 phosphorylates CCNA"
+xref: Reactome:R-HSA-9929532 "CCNA:CDK1 phosphorylates BORA"
+xref: Reactome:R-HSA-9929535 "CCNA:CDK1 phosphorylates WEE1"
+xref: Reactome:R-HSA-9929717 "CCNA:CDK1 phosphorylates FZR1"
+xref: Reactome:R-HSA-9929719 "CCNA:CDK1 phosphorylates TICRR"
+xref: Reactome:R-HSA-9929720 "CCNA:CDK1 phosphorylates OBI1"
+xref: Reactome:R-HSA-9929721 "CCNA:CDK1 phosphorylates SGO1"
+xref: Reactome:R-HSA-9929884 "CCNA:CDK1 phosphorylates HJURP"
+xref: Reactome:R-HSA-9929904 "CCNA:CDK1 phosphorylates MIS18BP1"
+xref: Reactome:R-HSA-9929935 "CCNA:CDK1 phosphorylates CDC25B"
 is_a: GO:0004674 ! protein serine/threonine kinase activity
 is_a: GO:0097472 ! cyclin-dependent protein kinase activity
 property_value: skos:exactMatch EC:2.7.11.22
@@ -49840,33 +50014,9 @@
 synonym: "protein kinase Cepsilon activity" NARROW [EC:2.7.11.13]
 synonym: "STK24" RELATED [EC:2.7.11.13]
 xref: EC:2.7.11.13
-xref: Reactome:R-HSA-1433508 "PKC alpha interacts with and phosphorylates KIT"
-xref: Reactome:R-HSA-193703 "IKKbeta is activated"
-xref: Reactome:R-HSA-198314 "DAG stimulates protein kinase C-delta"
-xref: Reactome:R-HSA-2179413 "Activated PKC-alpha  activate MMP3"
-xref: Reactome:R-HSA-2730863 "Phosphorylation of CARMA1 by PKC-theta"
-xref: Reactome:R-HSA-374664 "Phosphorylation and activation of Ezrin"
-xref: Reactome:R-HSA-429698 "PRKD1,2,3 phosphorylates CERT1-2"
-xref: Reactome:R-HSA-450533 "PKCdelta phosphorylates HuR"
-xref: Reactome:R-HSA-450550 "PKCalpha phosphorylates HuR"
-xref: Reactome:R-HSA-5138432 "DVL2 is phosphorylated by PKC"
 xref: Reactome:R-HSA-5218805 "PKC autophosphorylates"
-xref: Reactome:R-HSA-5218823 "PKC phosphorylates sphingosine kinase 1"
-xref: Reactome:R-HSA-5229194 "Depolymerization of lamin filaments after PKC-mediated phosphorylation"
-xref: Reactome:R-HSA-5623667 "PDPK1 phosphorylates PKN1,2,3"
-xref: Reactome:R-HSA-5625784 "PKN1 phosphorylates histone 3 of nucleosomes associate with KLK2 and KLK3 promoters"
-xref: Reactome:R-HSA-5671749 "p-T774-PKN1 phosphorylates CDC25C"
-xref: Reactome:R-HSA-74615 "PRKCA/Q phosphorylate RGS9-1:GN5B:RGS9BP"
 xref: Reactome:R-HSA-751040 "PKC phosphorylates G alpha (z)"
-xref: Reactome:R-HSA-8932284 "PKC phosphorylates NFE2L2"
-xref: Reactome:R-HSA-8934446 "Activated PKC phosphorylates SHC1"
-xref: Reactome:R-HSA-9010681 "PKC phosphorylates ROBO3.1"
-xref: Reactome:R-HSA-9021357 "PRKCI phosphorylates ELF3"
-xref: Reactome:R-HSA-909552 "Phosphorylation of STAT1 at Ser727"
-xref: Reactome:R-HSA-927889 "SMG1 phosphorylates UPF1 (enhanced by Exon Junction Complex)"
 xref: Reactome:R-HSA-9626817 "PKC phosphorylates NCF1"
-xref: Reactome:R-HSA-9632858 "PRKCZ autophosphorylates downstream of estrogen stimulation"
-xref: Reactome:R-HSA-9796053 "PRKCI phosphorylates NFE2L2"
 is_a: GO:0004674 ! protein serine/threonine kinase activity
 property_value: skos:exactMatch EC:2.7.11.13
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25457" xsd:anyURI
@@ -49881,10 +50031,19 @@
 synonym: "conventional protein kinase C activity" EXACT [PMID:9601053]
 xref: Reactome:R-HSA-114683 "Phosphorylation of Platelet Sec-1"
 xref: Reactome:R-HSA-114684 "Phosphorylation of Syntaxin-4"
+xref: Reactome:R-HSA-1433508 "PKC alpha interacts with and phosphorylates KIT"
+xref: Reactome:R-HSA-2179413 "Activated PKC-alpha  activate MMP3"
 xref: Reactome:R-HSA-416639 "Trafficking of GluR2-containing AMPA receptors to extrasynaptic sites"
 xref: Reactome:R-HSA-421007 "Endocytosis of Ca impermeable AMPA receptors"
+xref: Reactome:R-HSA-450550 "PKCalpha phosphorylates HuR"
+xref: Reactome:R-HSA-5138432 "DVL2 is phosphorylated by PKC"
+xref: Reactome:R-HSA-5218823 "PKC phosphorylates sphingosine kinase 1"
+xref: Reactome:R-HSA-5229194 "Depolymerization of lamin filaments after PKC-mediated phosphorylation"
+xref: Reactome:R-HSA-74615 "PRKCA/Q phosphorylate RGS9-1:GN5B:RGS9BP"
+xref: Reactome:R-HSA-8934446 "Activated PKC phosphorylates SHC1"
 xref: Reactome:R-HSA-8986937 "MECP2 is phosphorylated at T308"
 xref: Reactome:R-HSA-9005561 "Active PKA, CaMK IV do not phosphorylate MECP2 mutants R306C,(R306H) at T308"
+xref: Reactome:R-HSA-9010681 "PKC phosphorylates ROBO3.1"
 is_a: GO:0004697 ! diacylglycerol-dependent serine/threonine kinase activity
 is_a: GO:0009931 ! calcium-dependent protein serine/threonine kinase activity
 property_value: skos:broadMatch EC:2.7.11.13
@@ -49899,7 +50058,14 @@
 synonym: "atypical protein kinase C activity" RELATED []
 synonym: "calcium-independent PKC activity" EXACT []
 synonym: "novel protein kinase C activity" EXACT [PMID:9601053]
+xref: Reactome:R-HSA-198314 "DAG stimulates protein kinase C-delta"
+xref: Reactome:R-HSA-2730863 "Phosphorylation of CARMA1 by PKC-theta"
+xref: Reactome:R-HSA-374664 "Phosphorylation and activation of Ezrin"
+xref: Reactome:R-HSA-429698 "PRKD1,2,3 phosphorylates CERT1-2"
+xref: Reactome:R-HSA-450533 "PKCdelta phosphorylates HuR"
 xref: Reactome:R-HSA-5607740 "PKC-delta phosphorylates CARD9"
+xref: Reactome:R-HSA-8932284 "PKC phosphorylates NFE2L2"
+xref: Reactome:R-HSA-909552 "Phosphorylation of STAT1 at Ser727"
 is_a: GO:0004697 ! diacylglycerol-dependent serine/threonine kinase activity
 property_value: skos:broadMatch EC:2.7.11.13
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25457" xsd:anyURI
@@ -50167,7 +50333,7 @@
 xref: Reactome:R-HSA-1524186 "Phosphorylation of PLCgamma by PDGFR"
 xref: Reactome:R-HSA-1671691 "PRLR-bound STAT5 is phosphorylated"
 xref: Reactome:R-HSA-170070 "Wee1-mediated phosphorylation of Cyclin B1:phospho-Cdc2 complexes"
-xref: Reactome:R-HSA-170156 "Wee1-mediated phosphorylation of Cyclin A:phospho-Cdc2 complexes"
+xref: Reactome:R-HSA-170156 "WEE1 phosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170991 "SRC-1 autophosphorylates"
 xref: Reactome:R-HSA-171011 "Binding and activation of MAP Kinase"
 xref: Reactome:R-HSA-174164 "Phosphorylation of Cyclin A:Cdk2 at Tyr 15"
@@ -50526,6 +50692,8 @@
 xref: Reactome:R-HSA-9860759 "p-T816-PKN2 phosphorylates AKT1 on serine-308"
 xref: Reactome:R-HSA-9861469 "p-Y397-PTK2 (FAK) phosphorylates CHUK (IKKA)"
 xref: Reactome:R-HSA-9865196 "p-Y393-ABL1 phosphorylates tyrosine-407 of YAP1"
+xref: Reactome:R-HSA-9931290 "IL6-activated JAK1 phosphorylates CD274"
+xref: Reactome:R-HSA-9934751 "CDH1 is phosphorylated on tyrosine residues"
 xref: RHEA:10596
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch EC:2.7.10.-
@@ -50678,7 +50846,7 @@
 name: protein-lysine 6-oxidase activity
 namespace: molecular_function
 alt_id: GO:0018056
-def: "Catalysis of the reaction: peptidyl-L-lysyl-peptide + H2O + O2 = peptidyl-allysyl-peptide + NH3 + hydrogen peroxide." [EC:1.4.3.13]
+def: "Catalysis of the reaction: L-lysyl-[protein] + O2 + H2O = (S)-2-amino-6-oxohexanoyl-[protein] + H2O2 + NH4+." [RHEA:24544]
 synonym: "lysyl oxidase activity" RELATED [EC:1.4.3.13]
 synonym: "protein-L-lysine:oxygen 6-oxidoreductase (deaminating)" RELATED [EC:1.4.3.13]
 xref: EC:1.4.3.13
@@ -50711,7 +50879,7 @@
 xref: Reactome:R-HSA-1363276 "Dephosphorylation of p130 (RBL2) by PP2A"
 xref: Reactome:R-HSA-167072 "Hypophosphorylation of RNA Pol II CTD by FCP1P protein"
 xref: Reactome:R-HSA-170153 "Dephosphorylation of nuclear Cyclin B1:phospho-Cdc2 (Thr 14, Tyr15) complexes by Cdc25 phosphatases"
-xref: Reactome:R-HSA-170158 "Dephosphorylation of nuclear Cyclin A:phospho-Cdc2 complexes"
+xref: Reactome:R-HSA-170158 "CDC25A dephosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170161 "Dephosphorylation of cytoplasmic Cyclin B1/B2:phospho-Cdc2 (Thr 14, Tyr 15) complexes by CDC25B"
 xref: Reactome:R-HSA-174110 "Cdc25A mediated dephosphorylation of Cyclin A:phospho-Cdk2"
 xref: Reactome:R-HSA-174124 "Dephosphorylation of phospho-Cdh1"
@@ -50760,8 +50928,8 @@
 xref: MetaCyc:3.1.3.16-RXN
 xref: Reactome:R-HSA-1295632 "PPA2A dephosphorylates SPRY2"
 xref: Reactome:R-HSA-163568 "phosphorylated perilipin + H2O -> perilipin + orthophosphate"
-xref: Reactome:R-HSA-163689 "Dephosphorylation of pChREBP (Ser 196) by PP2A"
-xref: Reactome:R-HSA-163750 "Dephosphorylation of phosphoPFKFB1 by PP2A complex"
+xref: Reactome:R-HSA-163689 "PP2A complex dephosphorylates pChREBP (MLXIPL)"
+xref: Reactome:R-HSA-163750 "PP2A complex dephosphorylates pPFKFB1"
 xref: Reactome:R-HSA-1638821 "PP2A-B56 dephosphorylates centromeric cohesin"
 xref: Reactome:R-HSA-199425 "PHLPP dephosphorylates S473 in AKT"
 xref: Reactome:R-HSA-199959 "ERKs are inactivated by protein phosphatase 2A"
@@ -50784,7 +50952,6 @@
 xref: Reactome:R-HSA-5683405 "PPP5C dephosphorylates TP53BP1"
 xref: Reactome:R-HSA-5687758 "PPP4C:PPP4R2 dephosphorylates RPA2"
 xref: Reactome:R-HSA-5692754 "CDC14A,B dephosphorylate p-3S,T MAPK6"
-xref: Reactome:R-HSA-5693153 "PPM1K dephosphorylates p-BCKDH"
 xref: Reactome:R-HSA-5694421 "PP6 dephosphorylates SEC24"
 xref: Reactome:R-HSA-6792863 "PP2A-PP2R5C dephosphorylates MDM2"
 xref: Reactome:R-HSA-6811504 "AKT1 dephosphorylation by PP2A-B56-beta,gamma"
@@ -50803,6 +50970,7 @@
 xref: Reactome:R-HSA-9865226 "PP2A dephosphorylates serine-127 of YAP1"
 xref: Reactome:R-HSA-9865238 "PP2A dephosphorylates serine-715 of PDE4D5 in p-S715-PDE4D5:integrin alpha5:integrin beta1:fibronectin"
 xref: Reactome:R-HSA-9912527 "H139Hfs13* PPM1K does not dephosphorylate BCKDH"
+xref: Reactome:R-HSA-9970190 "PP1 dephosphorylates RNA Pol II"
 xref: RHEA:20629
 xref: RHEA:47004
 is_a: GO:0004721 ! phosphoprotein phosphatase activity
@@ -51021,7 +51189,7 @@
 id: GO:0004733
 name: pyridoxamine phosphate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH4+ + hydrogen peroxide. This activity can also oxidize pyridoxine 5'-phosphate to pyridoxal 5'-phosphate + hydrogen peroxide." [PMID:11786019]
+def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH4+ + H2O2. This activity can also oxidize pyridoxine 5'-phosphate to pyridoxal 5'-phosphate + H2O2." [EC:1.4.3.5, PMID:11786019]
 synonym: "PMP oxidase activity" RELATED [EC:1.4.3.5]
 synonym: "pyridoxal 5'-phosphate synthase activity" RELATED [EC:1.4.3.5]
 synonym: "pyridoxamine 5'-phosphate oxidase activity" RELATED [EC:1.4.3.5]
@@ -51034,11 +51202,13 @@
 xref: MetaCyc:PMPOXI-RXN
 xref: Reactome:R-HSA-965019 "2xPNPO:2xFMN oxidizes PDXP to PXLP"
 xref: Reactome:R-HSA-965079 "2xPNPO:2xFMN oxidizes PXAP to PXLP"
+xref: RHEA:15149
 xref: RHEA:15817
 is_a: GO:0016641 ! oxidoreductase activity, acting on the CH-NH2 group of donors, oxygen as acceptor
 relationship: part_of GO:0042823 ! pyridoxal phosphate biosynthetic process
 property_value: skos:exactMatch EC:1.4.3.5
-property_value: skos:exactMatch RHEA:15817
+property_value: skos:narrowMatch RHEA:15149
+property_value: skos:narrowMatch RHEA:15817
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22748" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -51071,7 +51241,7 @@
 xref: EC:1.5.1.2
 xref: MetaCyc:PYRROLINECARBREDUCT-RXN
 xref: Reactome:R-HSA-6783939 "PYCR2 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
-xref: Reactome:R-HSA-6783955 "PYCRL decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
+xref: Reactome:R-HSA-6783955 "PYCR3 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
 xref: Reactome:R-HSA-70664 "PYCR1 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
 xref: RHEA:14109
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
@@ -51230,7 +51400,7 @@
 synonym: "phosphoenolpyruvate kinase activity" RELATED [EC:2.7.1.40]
 xref: EC:2.7.1.40
 xref: MetaCyc:PEPDEPHOS-RXN
-xref: Reactome:R-HSA-71670 "PKM dephosphorylates PEP to PYR"
+xref: Reactome:R-HSA-71670 "Pyruvate kinase dephosphorylates PEP to PYR"
 xref: RHEA:18157
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
 relationship: part_of GO:0006096 ! glycolytic process
@@ -51347,7 +51517,6 @@
 name: ribose phosphate diphosphokinase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: D-ribose 5-phosphate + ATP = 5-phospho-alpha-D-ribose 1-diphosphate + AMP + 2 H+." [RHEA:15609]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "5-phosphoribose pyrophosphorylase activity" RELATED [EC:2.7.6.1]
 synonym: "5-phosphoribosyl-1-pyrophosphate synthetase activity" RELATED [EC:2.7.6.1]
 synonym: "5-phosphoribosyl-alpha-1-pyrophosphate synthetase activity" RELATED [EC:2.7.6.1]
@@ -51368,16 +51537,16 @@
 xref: EC:2.7.6.1
 xref: KEGG_REACTION:R01049
 xref: MetaCyc:PRPPSYN-RXN
-xref: Reactome:R-HSA-111215 "D-ribose 5-phosphate + 2'-deoxyadenosine 5'-triphosphate (dATP) => 5-Phospho-alpha-D-ribose 1-diphosphate (PRPP) + 2'-deoxyadenosine 5'-monophosphate"
-xref: Reactome:R-HSA-73580 "D-ribose 5-phosphate + ATP => 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) + adenosine 5'-monophosphate"
+xref: Reactome:R-HSA-111215 "PRPS1 dimer pyrophosphorylates R5P using dATP"
+xref: Reactome:R-HSA-73580 "PRPS dimers pyrophosphorylate R5P using ATP"
 xref: RHEA:15609
 is_a: GO:0016778 ! diphosphotransferase activity
 property_value: skos:exactMatch EC:2.7.6.1
 property_value: skos:exactMatch KEGG_REACTION:R01049
 property_value: skos:exactMatch MetaCyc:PRPPSYN-RXN
 property_value: skos:exactMatch RHEA:15609
-property_value: skos:relatedMatch MetaCyc:PWY0-662
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004750
@@ -51403,7 +51572,7 @@
 xref: EC:5.1.3.1
 xref: KEGG_REACTION:R01529
 xref: MetaCyc:RIBULP3EPIM-RXN
-xref: Reactome:R-HSA-199803 "xylulose 5-phosphate <=> D-ribulose 5-phosphate"
+xref: Reactome:R-HSA-199803 "RPE dimers isomerize XY5P to RU5P"
 xref: Reactome:R-HSA-71303 "RPE dimers isomerise RU5P to XY5P"
 xref: RHEA:13677
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
@@ -51429,10 +51598,10 @@
 xref: EC:5.3.1.6
 xref: KEGG_REACTION:R01056
 xref: MetaCyc:RIB5PISOM-RXN
-xref: Reactome:R-HSA-177784 "ribose 5-phosphate <=> D-ribulose 5-phosphate"
+xref: Reactome:R-HSA-177784 "RPIA isomerizes ribose 5-phosphate to D-ribulose 5-phosphate"
 xref: Reactome:R-HSA-5660013 "Defective RPIA does not isomerize RU5P to R5P"
 xref: Reactome:R-HSA-5660015 "Defective RPIA does not isomerize R5P to RU5P"
-xref: Reactome:R-HSA-71306 "D-ribulose 5-phosphate <=> ribose 5-phosphate"
+xref: Reactome:R-HSA-71306 "RPIA isomerizes D-ribulose 5-phosphate to ribose 5-phosphate"
 xref: RHEA:14657
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.6
@@ -51549,13 +51718,14 @@
 
 [Term]
 id: GO:0004760
-name: L-serine-pyruvate transaminase activity
+name: L-serine:pyruvate transaminase activity
 namespace: molecular_function
 alt_id: GO:0004761
 alt_id: GO:0004762
 alt_id: GO:0004763
-def: "Catalysis of the reaction: L-serine + pyruvate = 3-hydroxypyruvate + L-alanine." [EC:2.6.1.51, RHEA:22852]
+def: "Catalysis of the reaction: L-serine + pyruvate = 3-hydroxypyruvate + L-alanine." [RHEA:22852]
 synonym: "hydroxypyruvate:L-alanine transaminase activity" RELATED [EC:2.6.1.51]
+synonym: "L-serine-pyruvate transaminase activity" EXACT []
 synonym: "L-serine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.51]
 synonym: "serine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.51]
 synonym: "serine-pyruvate aminotransferase activity" EXACT []
@@ -51567,10 +51737,11 @@
 xref: KEGG_REACTION:R00585
 xref: MetaCyc:SERINE--PYRUVATE-AMINOTRANSFERASE-RXN
 xref: RHEA:22852
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.51
 property_value: skos:exactMatch RHEA:22852
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004764
@@ -51982,7 +52153,7 @@
 alt_id: GO:0008382
 alt_id: GO:0008383
 alt_id: GO:0016954
-def: "Catalysis of the reaction: 2 superoxide + 2 H+ = O2 + hydrogen peroxide." [EC:1.15.1.1, GOC:vw, PMID:15064408]
+def: "Catalysis of the reaction: 2 superoxide + 2 H+ = O2 + H2O2." [EC:1.15.1.1, GOC:vw, PMID:15064408]
 synonym: "copper, zinc superoxide dismutase activity" NARROW []
 synonym: "Cu,Zn-SOD" RELATED [EC:1.15.1.1]
 synonym: "Cu-Zn superoxide dismutase activity" NARROW [EC:1.15.1.1]
@@ -52207,6 +52378,7 @@
 synonym: "threonine dehydratase activity" BROAD []
 xref: EC:4.3.1.19
 xref: MetaCyc:THREDEHYD-RXN
+xref: Reactome:R-HSA-9014627 "SDS dimers:PXLP convert L-Thr to 2OBUTA"
 xref: RHEA:22108
 xref: UM-BBD_reactionID:r0433
 is_a: GO:0016841 ! ammonia-lyase activity
@@ -52331,7 +52503,6 @@
 xref: EC:1.21.99.4
 xref: MetaCyc:THYROXINE-DEIODINASE-RXN
 xref: Reactome:R-HSA-209772 "Thyroxine is deiodinated to triiodothyronine"
-xref: Reactome:R-HSA-350869 "Thyroxine is deiodinated to reverse triiodothyronine (RT3)"
 xref: RHEA:19745
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.99.4
@@ -52494,8 +52665,8 @@
 xref: EC:5.3.1.1
 xref: KEGG_REACTION:R01015
 xref: MetaCyc:TRIOSEPISOMERIZATION-RXN
-xref: Reactome:R-HSA-70454 "dihydroxyacetone phosphate <=> D-glyceraldehyde 3-phosphate"
-xref: Reactome:R-HSA-70481 "TPI1 isomerizes GA3P to DHAP"
+xref: Reactome:R-HSA-70454 "TPI1 dimer isomerizes DHAP to GA3P"
+xref: Reactome:R-HSA-70481 "TPI1 dimer isomerizes GA3P to DHAP"
 xref: RHEA:18585
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.1
@@ -53155,14 +53326,15 @@
 
 [Term]
 id: GO:0004838
-name: L-tyrosine-2-oxoglutarate transaminase activity
+name: L-tyrosine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tyrosine + 2-oxoglutarate = 4-hydroxyphenylpyruvate + L-glutamate." [RHEA:15093]
+def: "Catalysis of the reaction: L-tyrosine + 2-oxoglutarate = 3-(4-hydroxyphenyl)pyruvate + L-glutamate." [RHEA:15093]
 synonym: "glutamic phenylpyruvic aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "glutamic-hydroxyphenylpyruvic transaminase activity" RELATED [EC:2.6.1.5]
 synonym: "L-phenylalanine 2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "L-tyrosine aminotransferase activity" BROAD [EC:2.6.1.5]
 synonym: "L-tyrosine transaminase activity" BROAD []
+synonym: "L-tyrosine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "L-tyrosine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "phenylalanine aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "phenylalanine transaminase activity" RELATED [EC:2.6.1.5]
@@ -53180,10 +53352,11 @@
 xref: Reactome:R-HSA-71155 "TAT deaminates tyrosine"
 xref: RHEA:15093
 xref: UM-BBD_reactionID:r0297
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:broadMatch EC:2.6.1.5
 property_value: skos:exactMatch RHEA:15093
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004839
@@ -53219,13 +53392,6 @@
 synonym: "ubiquitin protein-ligase activity" NARROW []
 synonym: "ubiquitin-conjugating enzyme activity" NARROW []
 xref: KEGG_REACTION:R03876
-xref: Reactome:R-HSA-1169394 "ISGylation of IRF3"
-xref: Reactome:R-HSA-1169395 "ISGylation of viral protein NS1"
-xref: Reactome:R-HSA-1169397 "Activation of ISG15 by UBA7 E1 ligase"
-xref: Reactome:R-HSA-1169398 "ISGylation of host protein filamin B"
-xref: Reactome:R-HSA-1169402 "ISGylation of E2 conjugating enzymes"
-xref: Reactome:R-HSA-1169405 "ISGylation of protein phosphatase 1 beta (PP2CB)"
-xref: Reactome:R-HSA-1169406 "ISGylation of host proteins"
 xref: Reactome:R-HSA-1234163 "Cytosolic VBC complex ubiquitinylates hydroxyprolyl-HIF-alpha"
 xref: Reactome:R-HSA-1234172 "Nuclear VBC complex ubiquitinylates HIF-alpha"
 xref: Reactome:R-HSA-1253282 "ERBB4 ubiquitination by WWP1/ITCH"
@@ -53286,7 +53452,6 @@
 xref: Reactome:R-HSA-3781009 "NHLRC1 mediated ubiquitination of EPM2A and PPP1RC3 associated with glycogen-GYG1"
 xref: Reactome:R-HSA-3788724 "Cdh1:APC/C ubiquitinates EHMT1 and EHMT2"
 xref: Reactome:R-HSA-3797226 "Defective NHLRC1 does not ubiquitinate EPM2A (laforin) and PPP1R3C (PTG) (type 2B disease)"
-xref: Reactome:R-HSA-400267 "BTRC:CUL1:SKP1 (SCF-beta-TrCP1) ubiquitinylates PER proteins"
 xref: Reactome:R-HSA-4332236 "CBL neddylates TGFBR2"
 xref: Reactome:R-HSA-446877 "TRAF6 is K63 poly-ubiquitinated"
 xref: Reactome:R-HSA-450358 "Activated TRAF6 synthesizes unanchored polyubiquitin chains upon TLR stimulation"
@@ -53313,7 +53478,7 @@
 xref: Reactome:R-HSA-5691108 "SKP1:FBXL5:CUL1:NEDD8 ubiquitinylates IREB2"
 xref: Reactome:R-HSA-5693108 "TNFAIP3 (A20) ubiquitinates RIPK1 with K48-linked Ub chains"
 xref: Reactome:R-HSA-68712 "The geminin component of geminin:Cdt1 complexes is ubiquitinated, releasing Cdt1"
-xref: Reactome:R-HSA-69598 "Ubiquitination of phosphorylated Cdc25A"
+xref: Reactome:R-HSA-69598 "Ubiquitination of phosphorylated CDC25A"
 xref: Reactome:R-HSA-741386 "RIP2 induces K63-linked ubiquitination of NEMO"
 xref: Reactome:R-HSA-75824 "Ubiquitination of Cyclin D1"
 xref: Reactome:R-HSA-870449 "TRIM33 monoubiquitinates SMAD4"
@@ -53335,6 +53500,7 @@
 xref: Reactome:R-HSA-975118 "TRAF6 ubiquitinqtes IRF7 within the activated TLR7/8 or 9 complex"
 xref: Reactome:R-HSA-975147 "Auto ubiquitination of oligo-TRAF6 bound to p-IRAK2 at endosome membrane"
 xref: Reactome:R-HSA-9758604 "Ubiquitination of IKBKG by TRAF6"
+xref: Reactome:R-HSA-9766227 "MDM2 mediates CDH1 ubiquitination"
 xref: Reactome:R-HSA-9793444 "ITCH polyubiquitinates MLKL at K50"
 xref: Reactome:R-HSA-9793485 "PRKN polyubiquitinates RIPK3"
 xref: Reactome:R-HSA-9793679 "LUBAC ubiquitinates RIPK1 at K627"
@@ -53347,10 +53513,15 @@
 xref: Reactome:R-HSA-983153 "E1 mediated ubiquitin activation"
 xref: Reactome:R-HSA-983156 "Polyubiquitination of substrate"
 xref: Reactome:R-HSA-9833155 "Ubiquitination of STAT2"
-xref: Reactome:R-HSA-9833973 "ISGylation of PKR"
 xref: Reactome:R-HSA-9834070 "PRKN ubiquitinates MOM substrates"
 xref: Reactome:R-HSA-9861563 "CTLH E3 ligase ubiquitinates LDHA"
 xref: Reactome:R-HSA-9861640 "CTLH E3 ligase ubiquitinates PKM-1"
+xref: Reactome:R-HSA-9929350 "Ubiquitination of CD274 by BTRC-SCF E3-ligase complex"
+xref: Reactome:R-HSA-9934753 "CBLL1 ubiquitinates CDH1"
+xref: Reactome:R-HSA-9948301 "ZNF598 K63 polyubiquitinates RPS10 (eS10) and RPS20 (uS10) of the 40S ribosomal subunit"
+xref: Reactome:R-HSA-9954721 "NEDD8-K699-CUL2:ELOB:ELOC:RBX1:KLHDC10 K48 polyubiquitinates alanine-tailed nascent peptide"
+xref: Reactome:R-HSA-9955146 "S.flexneri ipaH9.8 ubiquitinates GBPs"
+xref: Reactome:R-HSA-9970191 "RBBP6 activates CPSF complex"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -53485,7 +53656,7 @@
 id: GO:0004846
 name: urate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: urate + O2 + H2O = 5-hydroxyisourate + hydrogen peroxide." [EC:1.7.3.3]
+def: "Catalysis of the reaction: urate + O2 + H2O = 5-hydroxyisourate + H2O2." [EC:1.7.3.3]
 synonym: "urate:oxygen oxidoreductase activity" RELATED [EC:1.7.3.3]
 synonym: "uric acid oxidase activity" RELATED [EC:1.7.3.3]
 synonym: "uricase activity" RELATED [EC:1.7.3.3]
@@ -53581,7 +53752,7 @@
 id: GO:0004851
 name: uroporphyrin-III C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + uroporphyrin III = 2 S-adenosyl-L-homocysteine + precorrin-2." [EC:2.1.1.107]
+def: "Catalysis of the reaction: uroporphyrinogen III + 2 S-adenosyl-L-methionine = precorrin-2 + 2 S-adenosyl-L-homocysteine + H+." [RHEA:32459]
 synonym: "adenosylmethionine-uroporphyrinogen III methyltransferase activity" RELATED [EC:2.1.1.107]
 synonym: "CobA" RELATED [EC:2.1.1.107]
 synonym: "CysG" RELATED [EC:2.1.1.107]
@@ -53671,7 +53842,7 @@
 id: GO:0004855
 name: xanthine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: xanthine + H2O + O2 = urate + hydrogen peroxide." [EC:1.17.3.2]
+def: "Catalysis of the reaction: xanthine + H2O + O2 = urate + H2O2." [EC:1.17.3.2]
 synonym: "hypoxanthine-xanthine oxidase activity" BROAD [EC:1.17.3.2]
 synonym: "schardinger enzyme" RELATED [EC:1.17.3.2]
 synonym: "Schardinger enzyme activity" RELATED [EC:1.17.3.2]
@@ -53719,6 +53890,8 @@
 def: "A molecular function regulator that reduces a catalytic activity." [GOC:ai, GOC:ebc]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
 synonym: "metalloenzyme inhibitor activity" NARROW []
+xref: Reactome:R-HSA-1474158 "GCHFR binds to GCH1 and negatively regulates its activity"
+xref: Reactome:R-HSA-1614336 "SUMF2 binds SUMF1, inhibiting SUMF1-mediated activation of arylsulfatases"
 is_a: GO:0030234 ! enzyme regulator activity
 is_a: GO:0140678 ! molecular function inhibitor activity
 relationship: negatively_regulates GO:0003824 ! catalytic activity
@@ -53961,6 +54134,10 @@
 synonym: "vitamin A receptor activity" NARROW []
 synonym: "vitamin D receptor activity" NARROW []
 synonym: "vitamin D3 receptor activity" NARROW [GOC:bf, Wikipedia:Calcitriol_receptor]
+xref: Reactome:R-HSA-400143 "Fatty acid ligands activate PPARA"
+xref: Reactome:R-HSA-8936849 "AHR:2xHSP90:AIP:PTGES3 binds TCDD"
+xref: Reactome:R-HSA-8937169 "AHR:TCDD:2xHSP90AB1:AIP:PTGES3 translocates from cytosol to nucleoplasm"
+xref: Reactome:R-HSA-8937191 "AHR:TCDD:2xHSP90AB1:AIP:PTGES3 dissociates"
 is_a: GO:0000981 ! DNA-binding transcription factor activity, RNA polymerase II-specific
 is_a: GO:0038023 ! signaling receptor activity
 is_a: GO:0098531 ! ligand-modulated transcription factor activity
@@ -54411,6 +54588,7 @@
 synonym: "purinoceptor" BROAD []
 synonym: "purinoreceptor" BROAD []
 xref: Reactome:R-HSA-877187 "P2X7 mediates loss of intracellular K+"
+xref: Reactome:R-HSA-9905964 "P2RX7 in ATP:P2RX7:PANX1 transports cations from the extracellular region to the cytosol"
 is_a: GO:0005231 ! excitatory extracellular ligand-gated monoatomic ion channel activity
 is_a: GO:0035381 ! ATP-gated ion channel activity
 is_a: GO:0099094 ! ligand-gated monoatomic cation channel activity
@@ -54704,6 +54882,7 @@
 name: glucagon receptor activity
 namespace: molecular_function
 def: "Combining with glucagon and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:mah, GOC:signaling, PMID:22438981]
+xref: Reactome:R-HSA-163625 "Glucagon binds to Glucagon receptor"
 is_a: GO:0008528 ! G protein-coupled peptide receptor activity
 relationship: part_of GO:0071377 ! cellular response to glucagon stimulus
 
@@ -54969,6 +55148,7 @@
 synonym: "thyroid stimulating hormone receptor activity" EXACT []
 synonym: "thyrotropin receptor" EXACT []
 synonym: "TSH receptor activity" EXACT []
+xref: Reactome:R-HSA-391375 "TSHR binds TSH/Thyrostimulin"
 xref: Wikipedia:Thyrotropin_receptor
 is_a: GO:0004930 ! G protein-coupled receptor activity
 is_a: GO:0016500 ! protein-hormone receptor activity
@@ -55742,11 +55922,12 @@
 
 [Term]
 id: GO:0005078
-name: MAP-kinase scaffold activity
+name: MAP kinase scaffold activity
 namespace: molecular_function
 def: "The binding activity of a molecule that functions as a physical support for the assembly of a multiprotein mitogen-activated protein kinase (MAPK) complex. Binds multiple kinases of the MAPKKK cascade, and also upstream signaling proteins, permitting those molecules to function in a coordinated way. Bringing together multiple enzymes and their substrates enables the signal to be transduced quickly and efficiently." [PMID:12511654, PMID:15213240, PMID:9405336]
 synonym: "MAP-kinase scaffold protein activity" EXACT []
 is_a: GO:0035591 ! signaling adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 relationship: has_part GO:0051019 ! mitogen-activated protein kinase binding
 relationship: part_of GO:0000165 ! MAPK cascade
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14286" xsd:anyURI
@@ -55881,6 +56062,7 @@
 xref: Reactome:R-HSA-379044 "Liganded Gs-activating GPCR acts as a GEF for Gs"
 xref: Reactome:R-HSA-379048 "Liganded Gq/11-activating GPCRs act as GEFs for Gq/11"
 xref: Reactome:R-HSA-380073 "Liganded Gi-activating GPCR acts as a GEF for Gi"
+xref: Reactome:R-HSA-381727 "RAP1A exchanges GDP for GTP"
 xref: Reactome:R-HSA-389348 "Activation of Rac1 by pVav1"
 xref: Reactome:R-HSA-389350 "Activation of Cdc42 by pVav1"
 xref: Reactome:R-HSA-392054 "NCAM1:pFAK:Grb2:Sos-mediated nucleotide exchange of Ras"
@@ -55896,6 +56078,7 @@
 xref: Reactome:R-HSA-3965444 "WNT:FZD complex promotes G-protein nucleotide exchange"
 xref: Reactome:R-HSA-399938 "Activation of Rac1 by FARP2"
 xref: Reactome:R-HSA-399995 "Muscarinic Acetylcholine Receptor M3 activates Gq"
+xref: Reactome:R-HSA-400037 "Gi,Go Heterotrimeric G-protein complex dissociates"
 xref: Reactome:R-HSA-400092 "Alpha-2A,alpha-2C Adrenergic Receptors activate Gi.Go heterotrimeric G proteins"
 xref: Reactome:R-HSA-4093336 "p-TIAM1 exchanges GTP for GDP on RAC1, activating it"
 xref: Reactome:R-HSA-416530 "FFAR1:FFAR1 ligands activate Gq"
@@ -57513,6 +57696,7 @@
 xref: Reactome:R-HSA-5577213 "LTCC multimer transports Ca2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9701055 "TRPA1 tetramer transports Ca2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9701141 "TRPV1 transports Ca2+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-9905934 "CaV3.2 (CACNA1H:CACNA2D1:CACNB1,2,3:CACNG7) transports calcium from the extracellular region to the cytosol"
 is_a: GO:0005262 ! calcium channel activity
 is_a: GO:0022843 ! voltage-gated monoatomic cation channel activity
 
@@ -57654,12 +57838,9 @@
 synonym: "cation channel activity" BROAD []
 synonym: "cation diffusion facilitator activity" EXACT []
 synonym: "non-selective cation channel activity" NARROW []
-xref: Reactome:R-HSA-1168376 "STIM1 oligomerizes"
 xref: Reactome:R-HSA-1296043 "Activation of HCN channels"
-xref: Reactome:R-HSA-169683 "IP3R:I(1,4,5)P3 tetramer transports Ca2+ from ER lumen to cytosol"
 xref: Reactome:R-HSA-2089943 "TRPC1 translocates calcium from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-426223 "Cation influx mediated by TRPC3/6/7"
-xref: Reactome:R-HSA-4420052 "Decreasing cGMP concentration promotes intracellular Ca2+ release in response to WNT"
 xref: Reactome:R-HSA-9706720 "SARS-CoV-1 E transports Ca2+"
 xref: Reactome:R-HSA-9712190 "cAMP:olfactory CNG channel translocates Na+ and Ca2+ from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-9754616 "SARS-CoV-2 E pentamer transports Ca2+"
@@ -58070,6 +58251,7 @@
 alt_id: GO:1901677
 def: "Enables the transfer of phosphate ions from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Secondary active transporters include symporters and antiporters." [GOC:mtg_transport]
 synonym: "inorganic phosphate transmembrane transporter activity" EXACT []
+xref: Reactome:R-HSA-5226964 "ANKH transports PPi from cytosol to extracellular region"
 is_a: GO:0015291 ! secondary active transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27121" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -58096,14 +58278,16 @@
 
 [Term]
 id: GO:0005319
-name: lipid transporter activity
+name: lipid carrier activity
 namespace: molecular_function
-def: "Enables the directed movement of lipids into, out of or within a cell, or between cells." [GOC:ai]
-synonym: "apolipoprotein" RELATED []
+def: "Directly binding to a specific lipid and delivering it either to an acceptor molecule or to a specific location." [GOC:curators]
+synonym: "apolipoprotein" NARROW []
+synonym: "lipid transporter activity" BROAD []
 synonym: "lipophorin" NARROW []
 xref: Reactome:R-HSA-1369028 "ABCAs mediate lipid efflux"
 xref: Reactome:R-HSA-1369052 "ABCAs mediate lipid influx"
 xref: Reactome:R-HSA-174786 "ApoB-48 + 40 triacylglycerol + 60 phospholipid => ApoB-48:TG:PL complex"
+xref: Reactome:R-HSA-5334794 "FABPs bind LCFAs"
 xref: Reactome:R-HSA-5682285 "ABCA12 transports lipids from cytosol to extracellular region"
 xref: Reactome:R-HSA-5682311 "Defective ABCA12 does not transport lipids from cytosol to extracellular region"
 xref: Reactome:R-HSA-5683672 "Defective ABCA3 does not transport PC, PG from ER membrane to lamellar body"
@@ -58112,8 +58296,9 @@
 xref: Reactome:R-HSA-6801250 "TRIAP1:PRELID1, PRELID3A transports PA from the outer to the inner mitochondrial membrane"
 xref: Reactome:R-HSA-8848053 "ABCA5 transports CHOL from lysosomal lumen to cytosol"
 xref: Reactome:R-HSA-8866329 "MTTP lipidates APOB-100, forming a pre-VLDL"
-is_a: GO:0005215 ! transporter activity
-relationship: part_of GO:0006869 ! lipid transport
+xref: Reactome:R-HSA-8873834 "STARD10 transports PC from ER membrane to lamellar body membrane"
+is_a: GO:0140104 ! molecular carrier activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0005320
@@ -58168,10 +58353,11 @@
 def: "Enables the transfer of a long-chain fatty acid from one side of a membrane to the other. A long-chain fatty acid has an aliphatic tail containing 13 to 22 carbons." [ISBN:0198506732]
 comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
 synonym: "long-chain fatty acid transporter activity" RELATED []
+xref: Reactome:R-HSA-2026996 "FABP1 transports fatty acids into the nucleus"
 xref: Reactome:R-HSA-2046087 "Translocation of tetracosahexaenoyl-CoA to peroxisomes"
 xref: Reactome:R-HSA-2046093 "Translocation of tetracosapentaenoyl-CoA to peroxisomes"
 xref: Reactome:R-HSA-382575 "ABCD1-3 dimers transfer LCFAs from cytosol to peroxisomal matrix"
-xref: Reactome:R-HSA-434381 "CD36 (FAT) translocates palmitate from the extracellular region to the cytosol"
+xref: Reactome:R-HSA-434381 "CD36 (FAT) translocates a long chain fatty acid from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-5684043 "Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix"
 xref: RHEA:39283
 is_a: GO:0015245 ! fatty acid transmembrane transporter activity
@@ -58257,7 +58443,7 @@
 synonym: "norepinephrine transmembrane transporter activity" EXACT []
 synonym: "norepinephrine:sodium:chloride symporter activity" EXACT []
 synonym: "sodium/norepinephrine symporter activity" EXACT []
-xref: Reactome:R-HSA-443997 "SLC6A2 cotransports NAd, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-443997 "SLC6A2 cotransports NAd, Na+,Cl- from extracellular region to cytosol"
 xref: Reactome:R-HSA-5659764 "Defective SLC6A2 does not cotransport NAd, Na+ from extracellular region to cytosol"
 xref: Wikipedia:Norepinephrine_transporter
 is_a: GO:0008504 ! monoamine transmembrane transporter activity
@@ -58474,8 +58660,6 @@
 synonym: "glucose permease activity" EXACT []
 synonym: "glucose transmembrane transporter activity" EXACT []
 synonym: "lactose/glucose efflux transporter activity" NARROW []
-xref: Reactome:R-HSA-499981 "Transport of Extracellular Glucose to the Cytosol by GLUT1 and GLUT2"
-xref: Reactome:R-HSA-5653873 "SLC2A1 tetramer transports Glc from cytosol to Golgi lumen"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27326" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0055056
@@ -58578,6 +58762,11 @@
 namespace: molecular_function
 def: "Enables the transfer of myo-inositol from one side of a membrane to the other. Myo-inositol is 1,2,3,4,5/4,6-cyclohexanehexol, a growth factor for animals and microorganisms." [GOC:ai]
 synonym: "vitamin Bh transporter activity" EXACT []
+xref: Reactome:R-HSA-1855155 "I(1,3,4,5,6)P5 transports from the ER lumen to the cytosol"
+xref: Reactome:R-HSA-1855164 "IP6 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855186 "I(1,3,4,5)P4 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855195 "I(1,3,4,5,6)P5 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855217 "I(1,4,5)P3 transports from the ER lumen to the cytosol"
 is_a: GO:0015166 ! polyol transmembrane transporter activity
 relationship: part_of GO:0015798 ! myo-inositol transport
 
@@ -58665,7 +58854,7 @@
 synonym: "copper uptake transmembrane transporter activity" RELATED []
 synonym: "intracellular copper ion transporter" NARROW []
 synonym: "plasma membrane copper transporter" NARROW []
-xref: Reactome:R-HSA-437300 "SLC31A1 transports Cu2+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-437300 "SLC31A1 transports Cu1+ from extracellular region to cytosol"
 is_a: GO:0046915 ! transition metal ion transmembrane transporter activity
 relationship: part_of GO:0035434 ! copper ion transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -58754,6 +58943,7 @@
 xref: Reactome:R-HSA-442422 "SLC39A1-4 transports Zn2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5654125 "Defective SLC39A4 does not transport Zn2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9820566 "ZnT5 transports zinc into the golgi apparatus"
+xref: Reactome:R-HSA-9928035 "ZnuABC transports Zn2+ to cytosol"
 is_a: GO:0046915 ! transition metal ion transmembrane transporter activity
 relationship: part_of GO:0071577 ! zinc ion transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
@@ -59033,7 +59223,7 @@
 namespace: molecular_function
 def: "Enables the transfer of a GDP-fucose from one side of a membrane to the other. GDP-fucose is a substance composed of fucose in glycosidic linkage with guanosine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
 xref: Reactome:R-HSA-5653596 "Defective SLC35C1 does not transport UDP-Fuc from cytosol to Golgi lumen"
-xref: Reactome:R-HSA-742345 "SLC35C1 transport UDP-Fuc from cytosol to Golgi lumen"
+xref: Reactome:R-HSA-742345 "SLC35C1 transport GDP-Fuc from cytosol to Golgi lumen"
 is_a: GO:0036080 ! purine nucleotide-sugar transmembrane transporter activity
 relationship: part_of GO:0015783 ! GDP-fucose transmembrane transport
 
@@ -59157,7 +59347,7 @@
 synonym: "carnitine/acyl carnitine carrier activity" EXACT []
 synonym: "carnitine:acyl carnitine carrier activity" EXACT []
 synonym: "fatty acyl carnitine carrier" EXACT []
-xref: Reactome:R-HSA-200424 "Exchange of palmitoylcarnitine and carnitine across the inner mitochondrial membrane"
+xref: Reactome:R-HSA-200424 "Exchange of acylcarnitine and carnitine across the inner mitochondrial membrane"
 is_a: GO:0015226 ! carnitine transmembrane transporter activity
 is_a: GO:0015227 ! O-acyl-L-carnitine transmembrane transporter activity
 is_a: GO:0015297 ! antiporter activity
@@ -59239,7 +59429,9 @@
 synonym: "t-SNARE activity" NARROW []
 synonym: "v-SNARE activity" NARROW []
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
+is_a: GO:0140522 ! fusogenic activity
 relationship: part_of GO:0061025 ! membrane fusion
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31564" xsd:anyURI
 
 [Term]
 id: GO:0005488
@@ -59711,10 +59903,11 @@
 
 [Term]
 id: GO:0005548
-name: phospholipid transporter activity
+name: obsolete phospholipid transporter activity
 namespace: molecular_function
 alt_id: GO:0008497
-def: "Enables the directed movement of phospholipids into, out of or within a cell, or between cells. Phospholipids are a class of lipids containing phosphoric acid as a mono- or diester." [GOC:ai, ISBN:0198506732]
+def: "OBSOLETE. Enables the directed movement of phospholipids into, out of or within a cell, or between cells. Phospholipids are a class of lipids containing phosphoric acid as a mono- or diester." [GOC:ai, ISBN:0198506732]
+comment: This term was obsoleted because it refers to the substrate rather than to the transport mechanism as a primary classification.
 xref: Reactome:R-HSA-216757 "4xPALM-C-p-2S-ABCA1 tetramer transports PL from transport vesicle membrane to plasma membrane"
 xref: Reactome:R-HSA-382553 "ABCA7:Apo1A-mediated phospholipid efflux"
 xref: Reactome:R-HSA-5678706 "ABCB4 transports PC from plasma membrane to extracellular region"
@@ -59722,8 +59915,11 @@
 xref: Reactome:R-HSA-8857662 "ESYT1:ESYT2:ESYT3 transport GPL from plasma membrane to ER membrane"
 xref: Reactome:R-HSA-8865637 "MFSD2A transports LPC from extracellular region to plasma membrane"
 xref: Reactome:R-HSA-8867876 "OSBPL5,8,10 exchange PS with PI4P from ER membrane to plasma membrane"
-is_a: GO:0005319 ! lipid transporter activity
-relationship: part_of GO:0015914 ! phospholipid transport
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31460" xsd:anyURI
+is_obsolete: true
+consider: GO:0120014
+consider: GO:0170055
 
 [Term]
 id: GO:0005549
@@ -60105,7 +60301,7 @@
 namespace: cellular_component
 def: "A collagen heterotrimer containing type VIII alpha chains; [alpha1(VIII)2]alpha2(VIII) and alpha1(VIII)[alpha2(VIII)]2 trimers have been observed; type VIII collagen triple helices associate to form regular hexagonal nets." [PMID:21421911]
 is_a: GO:0098642 ! network-forming collagen trimer
-relationship: part_of GO:0140155 ! hexagonal collagen network of basement membrane
+relationship: part_of GO:0098646 ! hexagonal collagen network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30148" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31070" xsd:anyURI
 
@@ -60173,7 +60369,7 @@
 def: "A collagen homotrimer of alpha1(X) chains; type X collagen triple helices form hexagonal networks (sheets)." [PMID:21421911]
 comment: Collagen X trimers have been observed to form hexagonal lattices in vitro, but in vivo they have been found in a fibril associated form (PMID:19693541).
 is_a: GO:0098642 ! network-forming collagen trimer
-relationship: part_of GO:0098646 ! interstitial hexagonal collagen network
+relationship: part_of GO:0098646 ! hexagonal collagen network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29415" xsd:anyURI
 
 [Term]
@@ -63880,7 +64076,6 @@
 subset: goslim_prokaryote
 subset: goslim_yeast
 synonym: "carbohydrate metabolism" EXACT []
-xref: Reactome:R-HSA-71387 "Metabolism of carbohydrates"
 xref: Wikipedia:Carbohydrate_metabolism
 is_a: GO:0044238 ! primary metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22880" xsd:anyURI
@@ -64065,8 +64260,11 @@
 synonym: "lactose breakdown" EXACT []
 synonym: "lactose catabolism" EXACT []
 synonym: "lactose degradation" EXACT []
+xref: MetaCyc:BGALACT-PWY
 is_a: GO:0005988 ! lactose metabolic process
 is_a: GO:0046352 ! disaccharide catabolic process
+property_value: skos:narrowMatch MetaCyc:BGALACT-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0005991
@@ -64094,19 +64292,26 @@
 synonym: "trehalose biosynthesis" EXACT []
 synonym: "trehalose formation" EXACT []
 synonym: "trehalose synthesis" EXACT []
+xref: MetaCyc:PWY-2622
 xref: MetaCyc:PWY-2661
+xref: MetaCyc:PWY-5983
+xref: MetaCyc:PWY-5985
 xref: MetaCyc:PWY-881
 xref: MetaCyc:TREHALOSESYN-PWY
 xref: MetaCyc:TRESYN-PWY
 is_a: GO:0005991 ! trehalose metabolic process
 is_a: GO:0046351 ! disaccharide biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-2622
 property_value: skos:narrowMatch MetaCyc:PWY-2661
+property_value: skos:narrowMatch MetaCyc:PWY-5983
+property_value: skos:narrowMatch MetaCyc:PWY-5985
 property_value: skos:narrowMatch MetaCyc:PWY-881
 property_value: skos:narrowMatch MetaCyc:TREHALOSESYN-PWY
 property_value: skos:narrowMatch MetaCyc:TRESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27083" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27111" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0005993
@@ -64336,9 +64541,11 @@
 synonym: "5-phosphoribosyl-1-pyrophosphate biosynthesis" EXACT []
 synonym: "5-phosphoribosyl-1-pyrophosphate biosynthetic process" EXACT []
 synonym: "PRPP biosynthetic process" EXACT []
-xref: Reactome:R-HSA-73843 "5-Phosphoribose 1-diphosphate biosynthesis"
+xref: MetaCyc:PWY0-662
 is_a: GO:0046390 ! ribose phosphate biosynthetic process
 is_a: GO:0046391 ! 5-phosphoribose 1-diphosphate metabolic process
+property_value: skos:narrowMatch MetaCyc:PWY0-662
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006016
@@ -64481,7 +64688,6 @@
 synonym: "glycosaminoglycan breakdown" EXACT []
 synonym: "glycosaminoglycan catabolism" EXACT []
 synonym: "glycosaminoglycan degradation" EXACT []
-xref: Reactome:R-HSA-2024101 "CS/DS degradation"
 is_a: GO:0006026 ! aminoglycan catabolic process
 is_a: GO:0030203 ! glycosaminoglycan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -64505,6 +64711,7 @@
 def: "The chemical reactions and pathways involving proteoglycans, any glycoprotein in which the carbohydrate units are glycosaminoglycans." [PMID:35536939]
 synonym: "proteoglycan metabolism" EXACT []
 synonym: "proteoglycan sulfate transfer" NARROW []
+xref: Reactome:R-HSA-1793185 "Chondroitin sulfate/dermatan sulfate metabolism"
 is_a: GO:0009100 ! glycoprotein metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
 
@@ -64967,10 +65174,12 @@
 name: alcohol metabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways involving alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [ISBN:0198506732]
+subset: gocheck_do_not_annotate
 subset: goslim_drosophila
 subset: goslim_pir
 synonym: "alcohol metabolism" EXACT []
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0006067
@@ -65187,8 +65396,11 @@
 synonym: "acetyl-CoA formation" EXACT []
 synonym: "acetyl-CoA synthesis" EXACT []
 xref: MetaCyc:Acetyl-CoA-Biosynthesis
+xref: MetaCyc:PWY-5172
 is_a: GO:0006084 ! acetyl-CoA metabolic process
 is_a: GO:0071616 ! acyl-CoA biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5172
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006086
@@ -65247,7 +65459,6 @@
 synonym: "pyruvate dehydrogenase bypass" RELATED []
 synonym: "pyruvate metabolism" EXACT []
 xref: MetaCyc:P41-PWY
-xref: Reactome:R-HSA-70268 "Pyruvate metabolism"
 xref: Wikipedia:Pyruvic_acid
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29541" xsd:anyURI
@@ -65817,6 +66028,7 @@
 namespace: biological_process
 alt_id: GO:0055134
 def: "Any cellular metabolic process involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:ai]
+subset: gocheck_do_not_annotate
 subset: goslim_pir
 subset: goslim_plant
 synonym: "cellular nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
@@ -65825,6 +66037,7 @@
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" RELATED [GOC:dph, GOC:tb]
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid metabolism" EXACT []
 is_a: GO:0044238 ! primary metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0006140
@@ -67194,6 +67407,7 @@
 synonym: "mtDNA replication" EXACT []
 synonym: "mtDNA synthesis" RELATED []
 synonym: "replication of mitochondrial DNA" EXACT []
+xref: Reactome:R-HSA-9913635 "Strand-asynchronous mitochondrial DNA replication"
 is_a: GO:0006261 ! DNA-templated DNA replication
 is_a: GO:0032042 ! mitochondrial DNA metabolic process
 intersection_of: GO:0006260 ! DNA replication
@@ -68063,6 +68277,7 @@
 synonym: "regulation of transcription, DNA-templated" EXACT []
 synonym: "transcriptional control" BROAD []
 xref: Reactome:R-HSA-9013695 "NOTCH4 Intracellular Domain Regulates Transcription"
+xref: Reactome:R-HSA-9708530 "Regulation of BACH1 activity"
 xref: Reactome:R-HSA-9762293 "Regulation of CDH11 gene transcription"
 is_a: GO:0010468 ! regulation of gene expression
 is_a: GO:2001141 ! regulation of RNA biosynthetic process
@@ -68602,6 +68817,7 @@
 synonym: "RNA breakdown" EXACT []
 synonym: "RNA catabolism" EXACT []
 synonym: "RNA degradation" EXACT []
+xref: Reactome:R-HSA-9930044 "Nuclear RNA decay"
 is_a: GO:0016070 ! RNA metabolic process
 is_a: GO:0141188 ! nucleic acid catabolic process
 
@@ -69069,6 +69285,7 @@
 synonym: "non-chaperonin molecular chaperone ATPase activity" RELATED []
 synonym: "protein complex assembly, multichaperone pathway" RELATED []
 xref: Reactome:R-HSA-389960 "Formation of tubulin folding intermediates by CCT/TriC"
+xref: Reactome:R-HSA-390450 "Folding of actin by CCT/TriC"
 xref: Reactome:R-HSA-390466 "Chaperonin-mediated protein folding"
 xref: Reactome:R-HSA-532668 "N-glycan trimming in the ER and Calnexin/Calreticulin cycle"
 xref: Wikipedia:Protein_folding
@@ -69130,7 +69347,6 @@
 def: "OBSOLETE. The proteolytic removal of a signal peptide from a protein during or after transport to a specific location in the cell." [GOC:mah, PMID:31214988]
 comment: This term was obsoleted because it reprenset a molecular function, signal peptidase activity ; GO:0009003.
 synonym: "leader peptide processing" EXACT []
-xref: Reactome:R-HSA-159782 "Removal of aminoterminal propeptides from gamma-carboxylated proteins"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30479" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051604
@@ -69152,7 +69368,6 @@
 namespace: biological_process
 def: "The process of introducing a phosphate group on to a protein." [GOC:hb]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid phosphorylation" EXACT [GOC:bf]
 is_a: GO:0016310 ! phosphorylation
 is_a: GO:0036211 ! protein modification process
@@ -69180,7 +69395,6 @@
 namespace: biological_process
 def: "The process of removing one or more phosphoric residues from a protein." [GOC:hb]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid dephosphorylation" EXACT [GOC:bf]
 is_a: GO:0016311 ! dephosphorylation
 is_a: GO:0036211 ! protein modification process
@@ -69343,6 +69557,8 @@
 synonym: "N-glycan biosynthesis" RELATED []
 synonym: "N-glycan metabolism" RELATED []
 synonym: "protein amino acid N-linked glycosylation" EXACT [GOC:bf]
+xref: Reactome:R-HSA-446203 "Asparagine N-linked glycosylation"
+xref: Reactome:R-HSA-480985 "Synthesis of dolichyl-phosphate-glucose"
 is_a: GO:0009101 ! glycoprotein biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29770" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30595" xsd:anyURI
@@ -69359,6 +69575,7 @@
 synonym: "N-linked glycan precursor biosynthesis" EXACT [PMID:8666161]
 synonym: "N-linked glycan precursor biosynthetic process" EXACT [PMID:8666161]
 synonym: "oligosaccharide-PP-dolichol assembly" EXACT []
+xref: Reactome:R-HSA-162699 "Synthesis of dolichyl-phosphate mannose"
 xref: Reactome:R-HSA-446193 "Biosynthesis of the N-glycan precursor (dolichol lipid-linked oligosaccharide, LLO) and transfer to a nascent protein"
 is_a: GO:1901137 ! carbohydrate derivative biosynthetic process
 relationship: part_of GO:0006487 ! protein N-linked glycosylation
@@ -69374,7 +69591,6 @@
 synonym: "dolichyl diphosphate biosynthesis" EXACT []
 synonym: "dolichyl diphosphate formation" EXACT []
 synonym: "dolichyl diphosphate synthesis" EXACT []
-xref: Reactome:R-HSA-446199 "Synthesis of Dolichyl-phosphate"
 is_a: GO:0008654 ! phospholipid biosynthetic process
 relationship: part_of GO:0006488 ! dolichol-linked oligosaccharide biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30376" xsd:anyURI
@@ -69455,7 +69671,6 @@
 comment: For non-covalent interactions with a lipid, consider instead the term 'lipid binding ; GO:0008289' and its children.
 subset: gocheck_obsoletion_candidate
 subset: goslim_drosophila
-subset: goslim_yeast
 synonym: "lipid:protein modification" EXACT []
 synonym: "protein amino acid lipidation" EXACT [GOC:bf]
 xref: RESID:AA0059
@@ -69754,7 +69969,6 @@
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids." [GOC:go_curators]
 synonym: "regulation of amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-350562 "Regulation of ornithine decarboxylase (ODC)"
 is_a: GO:0080090 ! regulation of primary metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006520 ! amino acid metabolic process
@@ -69814,20 +70028,23 @@
 synonym: "arginine synthesis" EXACT []
 xref: MetaCyc:ARGSYN-PWY
 xref: MetaCyc:ARGSYNBSUB-PWY
+xref: MetaCyc:PWY-5154
 xref: MetaCyc:PWY-7400
 is_a: GO:0006525 ! arginine metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:ARGSYN-PWY
 property_value: skos:narrowMatch MetaCyc:ARGSYNBSUB-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5154
 property_value: skos:narrowMatch MetaCyc:PWY-7400
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28686" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006527
 name: L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-arginine." [GOC:curators]
 synonym: "arginine breakdown" EXACT []
 synonym: "arginine catabolism" EXACT []
 synonym: "arginine degradation" EXACT []
@@ -69863,6 +70080,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8187
 property_value: skos:narrowMatch MetaCyc:PWY0-823
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006528
@@ -69870,7 +70088,6 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:go_curators]
 synonym: "asparagine metabolism" EXACT []
-xref: MetaCyc:ASPARAGINESYN-PWY
 is_a: GO:0019752 ! carboxylic acid metabolic process
 
 [Term]
@@ -69891,7 +70108,7 @@
 id: GO:0006530
 name: L-asparagine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-asparagine." [GOC:curators]
 synonym: "asparagine breakdown" EXACT []
 synonym: "asparagine catabolism" EXACT []
 synonym: "asparagine degradation" EXACT []
@@ -69903,6 +70120,7 @@
 property_value: skos:narrowMatch MetaCyc:ASPARAGINE-DEG1-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-4002
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006531
@@ -69932,7 +70150,7 @@
 id: GO:0006533
 name: L-aspartate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-aspartate." [GOC:curators]
 synonym: "aspartate breakdown" EXACT []
 synonym: "aspartate catabolism" EXACT []
 synonym: "aspartate degradation" EXACT []
@@ -69947,30 +70165,34 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8291
 property_value: skos:narrowMatch MetaCyc:PWY-8294
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006534
-name: cysteine metabolic process
+name: obsolete cysteine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:go_curators]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "cysteine metabolism" EXACT []
 xref: Wikipedia:Cysteine_metabolism
-is_a: GO:0000096 ! sulfur amino acid metabolic process
-is_a: GO:1901605 ! alpha-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+consider: GO:0019344
+consider: GO:0019448
 
 [Term]
 id: GO:0006535
-name: L-cysteine biosynthetic process from L-serine
+name: obsolete L-cysteine biosynthetic process from L-serine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of cysteine from L- serine." [MetaCyc:CYSTSYN-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cysteine from L- serine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism from serine" EXACT []
 synonym: "cysteine formation from serine" EXACT []
 synonym: "cysteine synthesis from serine" EXACT []
-xref: MetaCyc:CYSTSYN-PWY
-is_a: GO:0006563 ! L-serine metabolic process
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:CYSTSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0006536
@@ -70009,7 +70231,7 @@
 name: L-glutamate catabolic process
 namespace: biological_process
 alt_id: GO:0019459
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate, the anion of 2-aminopentanedioic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate." [GOC:curators]
 synonym: "glutamate breakdown" EXACT []
 synonym: "glutamate catabolism" EXACT []
 synonym: "glutamate deamidation" RELATED []
@@ -70029,13 +70251,14 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5766
 property_value: skos:narrowMatch MetaCyc:PWY0-1305
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006539
 name: obsolete glutamate catabolic process via 2-oxoglutarate
 namespace: biological_process
 def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate, via the intermediate 2-oxoglutarate." [GOC:go_curators]
-comment: The reason for obsoletion is that this term is equivalent to GO:0019552 glutamate catabolic process via 2-hydroxyglutarate, XREF the same MetaCyc pathway.
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown via 2-oxoglutarate" EXACT []
 synonym: "glutamate catabolic process via 2-ketoglutarate" EXACT []
 synonym: "glutamate catabolic process via alpha-ketoglutarate" EXACT []
@@ -70046,7 +70269,6 @@
 synonym: "glutamate degradation via 2-oxoglutarate" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28390" xsd:anyURI
 is_obsolete: true
-replaced_by: GO:0019552
 
 [Term]
 id: GO:0006540
@@ -70091,9 +70313,7 @@
 synonym: "glutamine biosynthesis" EXACT []
 synonym: "glutamine formation" EXACT []
 synonym: "glutamine synthesis" EXACT []
-xref: MetaCyc:GLNSYN-PWY
 xref: MetaCyc:PWY-6549
-property_value: skos:narrowMatch MetaCyc:GLNSYN-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-6549
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31068" xsd:anyURI
 is_obsolete: true
@@ -70103,7 +70323,7 @@
 id: GO:0006543
 name: L-glutamine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamine, 2-amino-4-carbamoylbutanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-glutamine." [GOC:curators]
 synonym: "glutamine breakdown" EXACT []
 synonym: "glutamine catabolism" EXACT []
 synonym: "glutamine degradation" EXACT []
@@ -70116,6 +70336,7 @@
 property_value: skos:narrowMatch MetaCyc:GLUTAMINEFUM-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28154" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006544
@@ -70136,16 +70357,25 @@
 synonym: "glycine formation" EXACT []
 synonym: "glycine synthesis" EXACT []
 xref: MetaCyc:GLYCINE-SYN2-PWY
+xref: MetaCyc:GLYSYN-ALA-PWY
+xref: MetaCyc:GLYSYN-PWY
+xref: MetaCyc:GLYSYN-THR-PWY
+xref: MetaCyc:PWY-8551
 is_a: GO:0006544 ! glycine metabolic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:GLYCINE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-ALA-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-THR-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-8551
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006546
 name: glycine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glycine, aminoethanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glycine." [GOC:curators]
 synonym: "glycine breakdown" EXACT []
 synonym: "glycine catabolism" EXACT []
 synonym: "glycine degradation" EXACT []
@@ -70156,6 +70386,7 @@
 is_a: GO:1901606 ! alpha-amino acid catabolic process
 property_value: skos:narrowMatch MetaCyc:PWY0-1608
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006547
@@ -70174,7 +70405,7 @@
 id: GO:0006548
 name: L-histidine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-histidine." [GOC:curators]
 synonym: "histidine breakdown" EXACT []
 synonym: "histidine catabolic process" BROAD []
 synonym: "histidine catabolism" EXACT []
@@ -70198,6 +70429,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5030
 property_value: skos:narrowMatch MetaCyc:PWY-5031
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006549
@@ -70212,7 +70444,7 @@
 id: GO:0006550
 name: L-isoleucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-isoleucine." [GOC:curators]
 synonym: "isoleucine breakdown" EXACT []
 synonym: "isoleucine catabolism" EXACT []
 synonym: "isoleucine degradation" EXACT []
@@ -70226,6 +70458,7 @@
 property_value: skos:narrowMatch MetaCyc:ILEUDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-5078
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006551
@@ -70242,7 +70475,7 @@
 id: GO:0006552
 name: L-leucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-leucine, 2-amino-4-methylpentanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-leucine." [GOC:curators]
 synonym: "L-leucine breakdown" EXACT []
 synonym: "L-leucine catabolism" EXACT []
 synonym: "L-leucine degradation" EXACT []
@@ -70262,6 +70495,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-7767
 property_value: skos:narrowMatch MetaCyc:PWY-8185
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006553
@@ -70346,7 +70580,7 @@
 id: GO:0006559
 name: L-phenylalanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-phenylalanine, 2-amino-3-phenylpropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-phenylalanine." [GOC:curators]
 synonym: "L-phenylalanine breakdown" EXACT []
 synonym: "L-phenylalanine catabolism" EXACT []
 synonym: "L-phenylalanine degradation" EXACT []
@@ -70367,6 +70601,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-7158
 property_value: skos:narrowMatch MetaCyc:PWY-8014
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006560
@@ -70396,7 +70631,7 @@
 id: GO:0006562
 name: L-proline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-proline (pyrrolidine-2-carboxylic acid), a chiral, cyclic, nonessential alpha-amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-proline." [GOC:curators]
 synonym: "proline breakdown" EXACT []
 synonym: "proline catabolism" EXACT []
 synonym: "proline degradation" EXACT []
@@ -70409,6 +70644,7 @@
 property_value: skos:narrowMatch MetaCyc:PROUT-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-8186
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006563
@@ -70416,6 +70652,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-serine, the L-enantiomer of serine, i.e. (2S)-2-amino-3-hydroxypropanoic acid." [GOC:ai, GOC:jsg]
 synonym: "L-serine metabolism" EXACT []
+xref: Reactome:R-HSA-977347 "Serine metabolism"
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
 
@@ -70428,19 +70665,21 @@
 synonym: "L-serine biosynthesis" EXACT []
 synonym: "L-serine formation" EXACT []
 synonym: "L-serine synthesis" EXACT []
+xref: MetaCyc:PWY-8011
 xref: MetaCyc:SERSYN-PWY
-xref: Reactome:R-HSA-977347 "Serine biosynthesis"
 is_a: GO:0006563 ! L-serine metabolic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:SERSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-8011
+property_value: skos:narrowMatch MetaCyc:SERSYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006565
 name: L-serine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-serine to pyruvate and free ammonium." [MetaCyc:SERDEG-PWY]
+def: "The chemical reactions and pathways resulting in the breakdown of L-serine." [GOC:curators]
 synonym: "L-serine breakdown" EXACT []
 synonym: "L-serine catabolism" EXACT []
 synonym: "L-serine degradation" EXACT []
@@ -70450,6 +70689,7 @@
 is_a: GO:0170040 ! proteinogenic amino acid catabolic process
 property_value: skos:exactMatch MetaCyc:SERDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006566
@@ -70466,7 +70706,7 @@
 id: GO:0006567
 name: L-threonine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-threonine." [GOC:curators]
 synonym: "threonine breakdown" EXACT []
 synonym: "threonine catabolism" EXACT []
 synonym: "threonine degradation" EXACT []
@@ -70485,6 +70725,7 @@
 property_value: skos:narrowMatch MetaCyc:THRDLCTCAT-PWY
 property_value: skos:narrowMatch MetaCyc:THREONINE-DEG2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006568
@@ -70503,7 +70744,7 @@
 id: GO:0006569
 name: L-tryptophan catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-tryptophan, the chiral amino acid 2-amino-3-(1H-indol-3-yl)propanoic acid." [ISBN:0198547684]
+def: "The chemical reactions and pathways resulting in the breakdown of L-tryptophan." [GOC:curators]
 synonym: "tryptophan breakdown" EXACT []
 synonym: "tryptophan catabolic process" RELATED []
 synonym: "tryptophan catabolic process, using tryptophanase" NARROW []
@@ -70543,6 +70784,7 @@
 property_value: skos:narrowMatch MetaCyc:TRYPDEG-PWY
 property_value: skos:narrowMatch MetaCyc:TRYPTOPHAN-DEGRADATION-1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006570
@@ -70565,16 +70807,25 @@
 synonym: "tyrosine biosynthesis" EXACT []
 synonym: "tyrosine formation" EXACT []
 synonym: "tyrosine synthesis" EXACT []
+xref: MetaCyc:PWY-3461
+xref: MetaCyc:PWY-6120
+xref: MetaCyc:PWY-6134
+xref: MetaCyc:TYRSYN
 is_a: GO:0006570 ! tyrosine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:1902223 ! erythrose 4-phosphate/phosphoenolpyruvate family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-3461
+property_value: skos:narrowMatch MetaCyc:PWY-6120
+property_value: skos:narrowMatch MetaCyc:PWY-6134
+property_value: skos:narrowMatch MetaCyc:TYRSYN
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20583" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006572
 name: L-tyrosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-tyrosine, an aromatic amino acid, 2-amino-3-(4-hydroxyphenyl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-tyrosine." [GOC:curators]
 synonym: "tyrosine breakdown" EXACT []
 synonym: "tyrosine catabolism" EXACT []
 synonym: "tyrosine degradation" EXACT []
@@ -70594,6 +70845,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY3O-4108
 property_value: skos:narrowMatch MetaCyc:TYRFUMCAT-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006573
@@ -70609,7 +70861,7 @@
 name: L-valine catabolic process
 namespace: biological_process
 alt_id: GO:1990292
-def: "The chemical reactions and pathways resulting in the breakdown of L-valine, 2-amino-3-methylbutanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-valine." [GOC:curators]
 synonym: "valine breakdown" EXACT []
 synonym: "valine catabolism" EXACT []
 synonym: "valine degradation" EXACT []
@@ -70625,6 +70877,7 @@
 property_value: skos:narrowMatch MetaCyc:VALDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31221" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006575
@@ -70768,11 +71021,14 @@
 synonym: "serotonin anabolism from tryptophan" EXACT []
 synonym: "serotonin formation from tryptophan" EXACT []
 synonym: "serotonin synthesis from tryptophan" EXACT []
+xref: MetaCyc:PWY-6030
+xref: MetaCyc:PWY-8475
 is_a: GO:0006568 ! L-tryptophan metabolic process
 is_a: GO:0042427 ! serotonin biosynthetic process
-property_value: skos:relatedMatch MetaCyc:PWY-6030
-property_value: skos:relatedMatch MetaCyc:PWY-8475
+property_value: skos:narrowMatch MetaCyc:PWY-6030
+property_value: skos:narrowMatch MetaCyc:PWY-8475
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31021" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006588
@@ -70826,17 +71082,22 @@
 synonym: "ornithine formation" EXACT []
 synonym: "ornithine synthesis" EXACT []
 xref: MetaCyc:ARGININE-SYN4-PWY
+xref: MetaCyc:GLUTORN-PWY
+xref: MetaCyc:PWY-6922
 is_a: GO:0006591 ! ornithine metabolic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:ARGININE-SYN4-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTORN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-6922
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0006593
 name: L-ornithine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine, an amino acid only rarely found in proteins, but which is important in living organisms as an intermediate in the reactions of the urea cycle and in arginine biosynthesis." [GOC:jl, ISBN:0192801023]
+def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine." [GOC:curators]
 synonym: "ornithine breakdown" EXACT []
 synonym: "ornithine catabolism" EXACT []
 synonym: "ornithine degradation" EXACT []
@@ -70844,6 +71105,7 @@
 is_a: GO:0170035 ! L-amino acid catabolic process
 is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006595
@@ -72697,6 +72959,7 @@
 def: "The chemical reactions and pathways involving glutathione, the tripeptide glutamylcysteinylglycine, which acts as a coenzyme for some enzymes and as an antioxidant in the protection of sulfhydryl groups in enzymes and other proteins; it has a specific role in the reduction of hydrogen peroxide (H2O2) and oxidized ascorbate, and it participates in the gamma-glutamyl cycle." [ISBN:0198506732]
 synonym: "glutathione metabolism" EXACT []
 synonym: "oxidized glutathione reduction" NARROW []
+xref: Reactome:R-HSA-174403 "Glutathione synthesis and recycling"
 is_a: GO:0006575 ! modified amino acid metabolic process
 is_a: GO:0006790 ! sulfur compound metabolic process
 
@@ -72710,7 +72973,6 @@
 synonym: "glutathione formation" EXACT []
 synonym: "glutathione synthesis" EXACT []
 xref: MetaCyc:GLUTATHIONESYN-PWY
-xref: Reactome:R-HSA-174403 "Glutathione synthesis and recycling"
 is_a: GO:0006749 ! glutathione metabolic process
 is_a: GO:0019184 ! nonribosomal peptide biosynthetic process
 is_a: GO:0042398 ! modified amino acid biosynthetic process
@@ -72849,7 +73111,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving any of a diverse group of vitamins that are soluble in water." [GOC:jl]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "water-soluble vitamin metabolism" EXACT []
-xref: Reactome:R-HSA-196849 "Metabolism of water-soluble vitamins and cofactors"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -72927,7 +73188,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving of any of a diverse group of vitamins that are soluble in organic solvents and relatively insoluble in water." [GOC:jl, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "fat-soluble vitamin metabolism" EXACT []
-xref: Reactome:R-HSA-6806667 "Metabolism of fat-soluble vitamins"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -73372,7 +73632,6 @@
 subset: goslim_pir
 subset: goslim_yeast
 synonym: "ion transport" BROAD []
-xref: Reactome:R-HSA-425393 "Transport of inorganic cations/anions and amino acids/oligopeptides"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -73573,6 +73832,7 @@
 xref: Reactome:R-HSA-210500 "Glutamate Neurotransmitter Release Cycle"
 xref: Reactome:R-HSA-212676 "Dopamine Neurotransmitter Release Cycle"
 xref: Reactome:R-HSA-264642 "Acetylcholine Neurotransmitter Release Cycle"
+xref: Reactome:R-HSA-442660 "SLC-mediated transport of neurotransmitters"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -73842,7 +74102,7 @@
 def: "The directed movement of amino acids, organic acids containing one or more amino substituents, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:ai]
 subset: goslim_pir
 subset: goslim_yeast
-xref: Reactome:R-HSA-352230 "Amino acid transport across the plasma membrane"
+xref: Reactome:R-HSA-9958863 "SLC-mediated transport of amino acids"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -74407,7 +74667,6 @@
 synonym: "activation of caspase activity" NARROW []
 synonym: "activation of metacaspase activity" NARROW []
 synonym: "caspase activation" EXACT [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-69416 "Dimerization of procaspase-8"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -74679,7 +74938,6 @@
 synonym: "viral-induced cell-cell fusion" EXACT []
 synonym: "viral-induced host cell-cell fusion" EXACT []
 synonym: "viral-induced membrane fusion" NARROW []
-xref: Reactome:R-HSA-9733458 "Induction of Cell-Cell Fusion"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28984" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0060141
@@ -74954,8 +75212,6 @@
 def: "OBSOLETE. A cascade of processes induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage and resulting in the stopping or reduction in rate of the cell cycle." [GOC:go_curators]
 comment: The reason for obsoletion is that this term is about down stream effects, not the signaling pathway itself.
 synonym: "positive regulation of cell cycle arrest by p53-mediated DNA damage response" EXACT [GOC:dph, GOC:mah]
-xref: Reactome:R-HSA-6791312 "TP53 Regulates Transcription of Cell Cycle Genes"
-xref: Reactome:R-HSA-69563 "p53-Dependent G1 DNA Damage Response"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26003" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0030330
@@ -75572,7 +75828,6 @@
 def: "Any process that reduces the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:jid]
 synonym: "lysosome pH reduction" EXACT [GOC:bf, GOC:rph]
 is_a: GO:0007035 ! vacuolar acidification
-is_a: GO:0035751 ! regulation of lysosomal lumen pH
 
 [Term]
 id: GO:0007043
@@ -81201,7 +81456,7 @@
 synonym: "circadian process" EXACT []
 synonym: "circadian response" RELATED []
 synonym: "response to circadian rhythm" RELATED []
-xref: Reactome:R-HSA-400253 "Circadian Clock"
+xref: Reactome:R-HSA-9909396 "Circadian clock"
 xref: Wikipedia:Circadian_rhythm
 is_a: GO:0048511 ! rhythmic process
 
@@ -81612,7 +81867,7 @@
 def: "Enables the transfer of monocarboxylic acids from one side of a membrane to the other. A monocarboxylic acid is an organic acid with one COOH group." [GOC:ai]
 synonym: "monocarboxylate carrier" NARROW []
 synonym: "prostaglandin/thromboxane transporter activity" NARROW []
-xref: Reactome:R-HSA-429749 "SLC5A8 transports monocarboxylates from extracellular region to cytosol"
+xref: Reactome:R-HSA-429749 "SLC5A8 (SMCT) co-transports monocarboxylates and Na+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-433698 "SLC16A3,7,8 cotransport monocarboxylates, H+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5624211 "Defective SLC16A1 does not cotransport monocarboxylates, H+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9645220 "SLC16A1:BSG cotransports monocarboxylates, H+ from extracellular region to cytosol"
@@ -81780,6 +82035,7 @@
 def: "A molecular function regulator that increases a catalytic activity." [GOC:pdt]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
 synonym: "metalloenzyme activator activity" NARROW []
+xref: Reactome:R-HSA-9948070 "POMGNT1 catalyzes FKRP:FKTN:RXYLT1 formation"
 is_a: GO:0030234 ! enzyme regulator activity
 is_a: GO:0140677 ! molecular function activator activity
 relationship: positively_regulates GO:0003824 ! catalytic activity
@@ -82002,6 +82258,7 @@
 name: ornithine decarboxylase inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces the activity of ornithine decarboxylase." [GOC:jl]
+xref: Reactome:R-HSA-350567 "Antizyme OAZ binds to Ornithine decarboxylase"
 is_a: GO:0004857 ! enzyme inhibitor activity
 is_a: GO:0042979 ! ornithine decarboxylase regulator activity
 relationship: negatively_regulates GO:0004586 ! ornithine decarboxylase activity
@@ -82075,6 +82332,7 @@
 xref: Reactome:R-HSA-2473152 "Acetylation of SMC3 subunit of centromeric chromatin associated cohesin by ESCO1 or ESCO2"
 xref: Reactome:R-HSA-6790987 "NAT10 acetylates cytidine-1337 and cytidine-1842 of 18S rRNA yielding 4-acetylcytidine-1377 and 4-acetylcytidine-1842"
 xref: Reactome:R-HSA-9636560 "Eis acetylates DUSP16"
+xref: Reactome:R-HSA-9913373 "AAC(6)-Ib acetylates ciprofloxacin"
 is_a: GO:0016407 ! acetyltransferase activity
 is_a: GO:0016410 ! N-acyltransferase activity
 
@@ -82304,6 +82562,27 @@
 synonym: "NADP-aldehyde reductase activity" EXACT []
 xref: EC:1.1.1.2
 xref: MetaCyc:ALCOHOL-DEHYDROGENASE-NADPORNOP+-RXN
+xref: Reactome:R-HSA-192033 "4-cholesten-7alpha-ol-3-one is reduced to 5beta-cholestan-7alpha-ol-3-one"
+xref: Reactome:R-HSA-192036 "5Beta-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha, 12alpha-triol"
+xref: Reactome:R-HSA-192067 "4-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholesten-7alpha, 12alpha-diol-3-one"
+xref: Reactome:R-HSA-192160 "5beta-cholestan-7alpha-ol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha-diol"
+xref: Reactome:R-HSA-193746 "4-cholesten-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-7alpha,24(S)-diol-3-one"
+xref: Reactome:R-HSA-193755 "4-cholesten-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,24(S)-triol-3-one"
+xref: Reactome:R-HSA-193758 "5beta-cholestan-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,24(S)-triol"
+xref: Reactome:R-HSA-193781 "5Beta-cholestan-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol"
+xref: Reactome:R-HSA-193800 "5Beta-cholestan-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol"
+xref: Reactome:R-HSA-193821 "4-cholesten-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,27-triol-3-one"
+xref: Reactome:R-HSA-193824 "4-cholesten-7alpha,27-diol-3-one is reduced to 5beta-cholestan-7alpha,27-diol-3-one"
+xref: Reactome:R-HSA-193841 "5beta-cholestan-7alpha,27-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,27-triol"
+xref: Reactome:R-HSA-198845 "CYB5A:heme reduces Asc.- to AscH-"
+xref: Reactome:R-HSA-2855252 "AKRs reduce RBP2:atRAL to RBP2:atROL"
+xref: Reactome:R-HSA-5423637 "AKR dimers reduce AFBDHO to AFBDOH"
+xref: Reactome:R-HSA-5692232 "AKR1A1 oxidises BaPtDHD to BaP-7,8-dione"
+xref: Reactome:R-HSA-9027531 "Dehydrogenase dehydrogenates 13-HDHA to 13-oxo-DHA"
+xref: Reactome:R-HSA-9027562 "Dehydrogenase dehydrogenates 17-HDHA to 17-oxo-DHA"
+xref: Reactome:R-HSA-9027598 "Dehydrogenase dehydrogenates 13(R)-HDPAn-3 to 13-oxo-DPAn-3"
+xref: Reactome:R-HSA-9027600 "Dehydrogenase dehydrogenates 17-HDPAn-3 to 17-oxo-DPAn-3"
+xref: Reactome:R-HSA-9937340 "DHRSX reduces dolichal to dolichol"
 xref: RHEA:15937
 is_a: GO:0018455 ! alcohol dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.1.1.2
@@ -82402,20 +82681,22 @@
 
 [Term]
 id: GO:0008110
-name: L-histidine:2-oxoglutarate aminotransferase activity
+name: L-histidine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-histidine = 3-(imidazol-5-yl)pyruvate + L-glutamate." [EC:2.6.1.38, RHEA:16565]
+def: "Catalysis of the reaction: L-histidine + 2-oxoglutarate = 3-(imidazol-5-yl)pyruvate + L-glutamate." [RHEA:16565]
 synonym: "histidine aminotransferase activity" BROAD []
 synonym: "histidine transaminase activity" BROAD [EC:2.6.1.38]
 synonym: "histidine-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.38]
+synonym: "L-histidine:2-oxoglutarate aminotransferase activity" EXACT []
 xref: EC:2.6.1.38
 xref: KEGG_REACTION:R01161
 xref: MetaCyc:HISTTRANSAM-RXN
 xref: RHEA:16565
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.38
 property_value: skos:exactMatch RHEA:16565
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008111
@@ -83273,6 +83554,7 @@
 xref: Reactome:R-HSA-212269 "DNMT1,3A,3B:PRC2 methylates cytosine and histone H3"
 xref: Reactome:R-HSA-379387 "COMT transfers Met to DA to form 3MT"
 xref: Reactome:R-HSA-379464 "COMT transfers Met to DOPAC to form HVA"
+xref: Reactome:R-HSA-5333608 "GSSebGalNac is reduced and methylated to MeSebGalNac"
 xref: Reactome:R-HSA-6800149 "N6AMT1:TRMT112 transfers CH3 group from AdoMet to MMAIII"
 xref: Reactome:R-HSA-71286 "guanidinoacetate + S-adenosylmethionine => creatine + S-adenosylhomocysteine"
 xref: Reactome:R-HSA-9710490 "The GSDME gene promoter is hypermethylated"
@@ -83339,7 +83621,7 @@
 id: GO:0008175
 name: tRNA methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in a tRNA molecule." [GOC:mah]
+def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in a tRNA molecule. The methyl group can be transfered to the nucleobase or to the ribose group of the nucleoside." [GOC:mah]
 xref: Reactome:R-HSA-6782879 "TYW3 methylates yW-86 yielding yW-72 at nucleotide 37 of tRNA(Phe)"
 xref: Reactome:R-HSA-6782881 "LCMT2 methylates yW-72 yielding yW-58 at nucleotide 37 of tRNA(Phe)"
 xref: Reactome:R-HSA-6782890 "LCMT2 methoxycarbonylates yW-58 yielding yW (wybutosine) at nucleotide 37 of tRNA(Phe)"
@@ -83585,6 +83867,7 @@
 name: ferrous iron binding
 namespace: molecular_function
 def: "Binding to a ferrous iron ion, Fe(II)." [GOC:ai]
+xref: Reactome:R-HSA-1362416 "Frataxin binds iron"
 is_a: GO:0005506 ! iron ion binding
 
 [Term]
@@ -83728,7 +84011,6 @@
 namespace: biological_process
 def: "The addition of an alkyl group to a protein amino acid. Alkyl groups are derived from alkanes by removal of one hydrogen atom." [GOC:ma]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid alkylation" EXACT [GOC:bf]
 is_a: GO:0036211 ! protein modification process
 
@@ -83926,6 +84208,7 @@
 xref: Reactome:R-HSA-5684864 "NAPSA, CTSH, PGA3-5 cleave pro-SFTPB"
 xref: Reactome:R-HSA-5685902 "NAPSA, CTSH, PGA3-5 cleave pro-SFTPC"
 xref: Reactome:R-HSA-6803060 "DCD(63-110) is processed to DCD(63-109)"
+xref: Reactome:R-HSA-9935547 "Lysosomal degradation of ubiqutinated CDH1"
 is_a: GO:0016787 ! hydrolase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: skos:exactMatch EC:3.4.-.-
@@ -83939,7 +84222,7 @@
 def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "cysteine protease activity" NARROW []
 synonym: "thiol protease activity" NARROW []
-xref: Reactome:R-HSA-2022381 "Cathepsin Z (Cathepsin X) hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022381 "Cathepsin Z (Cathepsin X) hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-2467775 "Autocleavage of ESPL1 (Separase)"
 xref: Reactome:R-HSA-2467809 "ESPL1 (Separase) cleaves centromeric cohesin"
 xref: Reactome:R-HSA-5660752 "USP9X deubiquitinates Ub-SNCA"
@@ -83961,7 +84244,7 @@
 def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "serine protease activity" NARROW []
 xref: Reactome:R-HSA-1461993 "pro-HD5 is cleaved by trypsin"
-xref: Reactome:R-HSA-2022383 "Chymase hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022383 "Chymase hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-3132753 "Pancreatic proteases degrade TCN1:RCbl"
 xref: Reactome:R-HSA-6801766 "Trypsin cleaves REG3A or REG3G to generate REG3A,G(38-175)"
 xref: Reactome:R-HSA-9839072 "HTRA2 degrades APP (Amyloid-beta precursor protein)"
@@ -83979,10 +84262,10 @@
 xref: Reactome:R-HSA-157629 "NOTCH2-ligand complex is cleaved to produce NEXT2"
 xref: Reactome:R-HSA-157632 "Complex of NOTCH1 with its ligand is cleaved to produce NEXT1"
 xref: Reactome:R-HSA-193679 "alpha-secretase cleaves the p75NTR extracellular domain"
-xref: Reactome:R-HSA-2022368 "MME:Zn2+ (Neprilysin) hydrolyses AGT(34-42)"
-xref: Reactome:R-HSA-2022393 "ANPEP:Zn2+ hydrolyzes Angiotensin-(2-8) to Angiotensin-(3-8)"
-xref: Reactome:R-HSA-2022396 "MME:Zn2+ (Neprilysin) hydrolyses AGT(34-43)"
-xref: Reactome:R-HSA-2022399 "ENPEP:Zn2+ hydrolyzes Angiotensin-(1-8) to Angiotensin-(2-8)"
+xref: Reactome:R-HSA-2022368 "MME:Zn2+ (Neprilysin) hydrolyses AGT(25-33) (Angiotensin-(1-9)) to AGT(25-31) (Angiotensin-(1-7)"
+xref: Reactome:R-HSA-2022393 "ANPEP:Zn2+ hydrolyzes AGT(26-32) (Angiotensin-(2-8)) to AGT(27-32) (Angiotensin-(3-8))"
+xref: Reactome:R-HSA-2022396 "MME:Zn2+ (Neprilysin) hydrolyses AGT(25-34) (Angiotensin-(1-10)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2022399 "ENPEP:Zn2+ hydrolyzes AGT(25-32) (Angiotensin-(1-8)) to AGT(26-32) (Angiotensin-(2-8))"
 xref: Reactome:R-HSA-2220944 "ADAM10/17 cleaves ligand-bound NOTCH1 PEST domain mutants to produce NEXT1 PEST domain mutants"
 xref: Reactome:R-HSA-2220976 "NOTCH1 HD+PEST domain mutants are cleaved by ADAM10/17 irrespective of ligand binding"
 xref: Reactome:R-HSA-2666278 "NOTCH1 t(7;9)(NOTCH1:M1580_K2555) is cleaved to produce NEXT1"
@@ -84079,6 +84362,7 @@
 xref: EC:3.4.19.2
 xref: MetaCyc:ACYLAMINOACYL-PEPTIDASE-RXN
 xref: MetaCyc:PEPTIDYL-GLYCINAMIDASE-RXN
+xref: Reactome:R-HSA-5696365 "ASRGL1 hydrolyses aspartame to L-Asp, L-Phe, MeOH"
 is_a: GO:0008233 ! peptidase activity
 property_value: skos:exactMatch EC:3.4.19.-
 property_value: skos:narrowMatch EC:3.4.19.1
@@ -84181,7 +84465,7 @@
 id: GO:0008251
 name: tRNA-specific adenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a tRNA molecule." [GOC:mah]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+, in a tRNA molecule." [GOC:mah]
 synonym: "tRNA-adenosine deaminase activity" EXACT []
 xref: Reactome:R-HSA-6782311 "ADAT2:ADAT3 (hetADAT) deaminates adenosine-34 in tRNAs"
 xref: Reactome:R-HSA-6782336 "ADAT1 deaminates adenosine-37 in tRNA(Ala)"
@@ -85015,12 +85299,13 @@
 
 [Term]
 id: GO:0008320
-name: protein transmembrane transporter activity
+name: transmembrane protein transporter activity
 namespace: molecular_function
 alt_id: GO:0015266
 alt_id: GO:0015463
 def: "Enables the transfer of a protein from one side of a membrane to the other." [GOC:jl]
 synonym: "protein channel activity" RELATED []
+synonym: "protein transmembrane transporter activity" EXACT []
 xref: Reactome:R-HSA-1268022 "TOMM40 complex translocates proteins from the cytosol to the mitochondrial intermembrane space"
 xref: Reactome:R-HSA-1299475 "TIMM23 PAM translocates proteins from the mitochondrial intermembrane space to the mitochondrial matrix"
 xref: Reactome:R-HSA-184269 "Monoubiquitinated N-myristoyl GAG polyprotein is targeted to the late endosomal vesicle membrane by the ESCRT-I complex"
@@ -85041,6 +85326,7 @@
 xref: Reactome:R-HSA-9636375 "SecA2 transports SapM from cytoplasm to cytosol"
 xref: Reactome:R-HSA-9698930 "HCMV C Nucleocapsid Translocation"
 xref: Reactome:R-HSA-9698933 "HCMV B Nucleocapsid Translocation"
+xref: Reactome:R-HSA-9760705 "GspD2:GspC2:GspS2 transports LT from periplasmic to extracellular space"
 is_a: GO:0022884 ! macromolecule transmembrane transporter activity
 is_a: GO:0140318 ! protein transporter activity
 relationship: part_of GO:0071806 ! protein transmembrane transport
@@ -85528,8 +85814,8 @@
 namespace: molecular_function
 def: "Catalysis of the transfer of an N-acetylglucosaminyl residue from UDP-N-acetyl-glucosamine to a sugar." [ISBN:0198506732]
 synonym: "GlcNAc transferase activity" EXACT []
+xref: Reactome:R-HSA-5617037 "POMGNT1 transfers GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 xref: Reactome:R-HSA-5694487 "A4GNT transfers GlcNAc to core 2 mucins"
-xref: Reactome:R-HSA-8879117 "POMGNT2 transfers GlcNAc to Man-DAG1"
 xref: Reactome:R-HSA-9683648 "Spike trimer glycoside chains are extended"
 xref: Reactome:R-HSA-9694656 "Spike trimer glycoside chains are extended"
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
@@ -85573,7 +85859,7 @@
 name: thioredoxin peroxidase activity
 namespace: molecular_function
 alt_id: GO:0009031
-def: "Catalysis of the reaction: [thioredoxin]-dithiol + hydrogen peroxide = [thioredoxin]-disulfide + H2O." [RHEA:63528]
+def: "Catalysis of the reaction: [thioredoxin]-dithiol + H2O2 = [thioredoxin]-disulfide + H2O." [RHEA:63528]
 synonym: "thiol peroxidase activity" EXACT []
 synonym: "TPx activity" EXACT []
 synonym: "TrxPx activity" EXACT []
@@ -85826,6 +86112,9 @@
 xref: KEGG_REACTION:R07204
 xref: MetaCyc:1.14.13.95-RXN
 xref: MetaCyc:1.14.13.96-RXN
+xref: Reactome:R-HSA-192157 "CYP8B1 12-hydroxylates 4CHOL7aOLONE"
+xref: Reactome:R-HSA-193709 "CYP8B1 12-hydroxylates 4CHOL7a,24(S)DIOL"
+xref: Reactome:R-HSA-193845 "CYP8B1 12-hydroxylates 4CHOL7a,27DONE"
 xref: RHEA:15261
 xref: RHEA:46752
 xref: RHEA:65700
@@ -85857,6 +86146,7 @@
 synonym: "sterol 14alpha-demethylase activity" RELATED []
 xref: EC:1.14.14.154
 xref: MetaCyc:1.14.13.70-RXN
+xref: Reactome:R-HSA-9947148 "CYP51A1 14-demethylates 24,25-dhLAN to 4,4-diMeCholesta-8,14-dien-3-ol"
 xref: RHEA:14917
 xref: RHEA:25286
 xref: RHEA:54028
@@ -86007,6 +86297,7 @@
 synonym: "PHB polyprenyl diphosphate transferase activity" EXACT []
 xref: EC:2.5.1.39
 xref: MetaCyc:RXN-11368
+xref: Reactome:R-HSA-2162192 "COQ2 ligates all-E-10PrP2 to PHB"
 xref: RHEA:44504
 is_a: GO:0004659 ! prenyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.39
@@ -86048,6 +86339,8 @@
 def: "Catalysis of the transfer of a fucosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
 xref: Reactome:R-HSA-9033949 "FUT1 transfers Fuc to Type 2 chains to form H antigen-RBC"
 xref: Reactome:R-HSA-9846332 "FUT1,2 transfer fucose to gangliosides"
+xref: Reactome:R-HSA-9935449 "FUT10 O-fucosylates EMI domain-containing proteins"
+xref: Reactome:R-HSA-9935469 "FUT11 O-fucosylates EMI domain-containing proteins"
 is_a: GO:0016758 ! hexosyltransferase activity
 
 [Term]
@@ -86239,6 +86532,7 @@
 name: phosphatidylethanolamine binding
 namespace: molecular_function
 def: "Binding to a phosphatidylethanolamine, a class of glycerophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of ethanolamine." [ISBN:0198506732]
+xref: Reactome:R-HSA-9842115 "PSAP(195-273) dimer binds PE"
 is_a: GO:0005543 ! phospholipid binding
 
 [Term]
@@ -86519,7 +86813,7 @@
 synonym: "O,N-disulfate O-sulfohydrolase activity" RELATED [EC:3.1.6.14]
 xref: EC:3.1.6.14
 xref: MetaCyc:3.1.6.14-RXN
-xref: Reactome:R-HSA-1638032 "N-acetylglucosamine 6-sulfatase (GNS) hydrolyses 6-sulfate groups of the N-acetyl-D-glucosamine 6-sulfate units of keratan sulfate"
+xref: Reactome:R-HSA-1638032 "GNS:Ca2+ cleaves sulfate from GlcNAc-6-sulfate units of keratan sulfate"
 xref: Reactome:R-HSA-2263495 "Defective GNS does not hydrolyse 6-sulfate from GlcNAc6S"
 is_a: GO:0008484 ! sulfuric ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.6.14
@@ -86566,12 +86860,13 @@
 
 [Term]
 id: GO:0008453
-name: alanine-glyoxylate transaminase activity
+name: L-alanine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + glyoxylate = pyruvate + glycine." [EC:2.6.1.44]
+def: "Catalysis of the reaction: L-alanine + glyoxylate = pyruvate + glycine." [RHEA:24248]
 synonym: "AGT activity" RELATED [EC:2.6.1.44]
 synonym: "alanine--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.44]
 synonym: "alanine-glyoxylate aminotransferase activity" EXACT []
+synonym: "alanine-glyoxylate transaminase activity" EXACT []
 synonym: "alanine-glyoxylic aminotransferase activity" RELATED [EC:2.6.1.44]
 synonym: "L-alanine-glycine transaminase activity" RELATED [EC:2.6.1.44]
 synonym: "L-alanine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.44]
@@ -86580,10 +86875,11 @@
 xref: Reactome:R-HSA-389684 "glyoxylate + alanine => glycine + pyruvate [peroxisome]"
 xref: Reactome:R-HSA-904864 "Mitochondrial AGXT2 tetramer transaminates glyoxylate and alanine to glycine and pyruvate"
 xref: RHEA:24248
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.44
 property_value: skos:exactMatch RHEA:24248
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008454
@@ -86844,7 +87140,7 @@
 xref: MetaCyc:2.8.2.29-RXN
 xref: MetaCyc:2.8.2.30-RXN
 xref: Reactome:R-HSA-2076383 "HS3ST1 sulfates GlcN at C3 in heparan sulfate"
-xref: Reactome:R-HSA-2076611 "HS3STs sulfate GlcN at C3 in heparan sulfate"
+xref: Reactome:R-HSA-2076611 "HS3ST2-6 sulfate GlcN at C3 in heparan sulfate"
 xref: RHEA:15461
 is_a: GO:0034483 ! heparan sulfate sulfotransferase activity
 property_value: skos:exactMatch RHEA:15461
@@ -87080,7 +87376,7 @@
 synonym: "tRNA-guanosine(34) queuine transglycosylase" EXACT [EC:2.4.2.64]
 xref: EC:2.4.2.64
 xref: MetaCyc:QUEUOSINE-TRNA-RIBOSYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-6782443 "QTRT1:QTRTD1 exchange guanine for queuosine at guanosine-34 of tRNA(Tyr)"
+xref: Reactome:R-HSA-6782443 "QTRT1:QTRTD1 exchanges guanine for queuine at guanosine-34 of tRNA(Asn),tRNA(Asp),tRNA(His),tRNA(Tyr) yielding queuosine-34 tRNA(Asn),tRNA(Asp),tRNA(His),tRNA(Tyr)"
 xref: RHEA:16633
 is_a: GO:0016763 ! pentosyltransferase activity
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -87122,6 +87418,8 @@
 synonym: "sphingosine kinase (phosphorylating)" RELATED [EC:2.7.1.91]
 xref: EC:2.7.1.91
 xref: MetaCyc:SPHINGANINE-KINASE-RXN
+xref: Reactome:R-HSA-5218845 "p-SPHK1 phosphorylates sphingosine to sphingosine 1-phosphate"
+xref: Reactome:R-HSA-9625814 "SPHK1 phosphorylates sphingosine in response to E2 stimulation"
 xref: RHEA:51496
 is_a: GO:0001727 ! lipid kinase activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
@@ -87134,7 +87432,6 @@
 name: sulfite oxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: H2O + O2 + sulfite = H2O2 + H+ + sulfate." [RHEA:24600]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "sulfite:oxygen oxidoreductase activity" RELATED [EC:1.8.3.1]
 synonym: "sulphite oxidase activity" EXACT []
 xref: EC:1.8.3.1
@@ -87147,8 +87444,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00533
 property_value: skos:exactMatch MetaCyc:SULFITE-OXIDASE-RXN
 property_value: skos:exactMatch RHEA:24600
-property_value: skos:relatedMatch MetaCyc:PWY-5326
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0008483
@@ -87341,7 +87638,6 @@
 synonym: "UDP-galactose:beta-N-acetylglucosamine beta-1,3-galactosyltransferase activity" EXACT []
 xref: EC:2.4.1.86
 xref: MetaCyc:2.4.1.86-RXN
-xref: Reactome:R-HSA-914010 "Addition of GlcNAc to the Tn antigen forms a Core 3 glycoprotein"
 xref: Reactome:R-HSA-9603989 "B3GALTs transfer Gal to GlcNAc-1,3-Gal-R to form Type 1 chain"
 xref: Reactome:R-HSA-9845516 "B3GALT4 transfers Gal to gangliosides"
 xref: RHEA:16045
@@ -87387,10 +87683,9 @@
 alt_id: GO:0015201
 def: "Enables the transfer of monoamines, organic compounds that contain one amino group that is connected to an aromatic ring by an ethylene group (-CH2-CH2-), from one side of a membrane to the other." [GOC:mah]
 xref: Reactome:R-HSA-372542 "Loading of dopamine into synaptic veiscles"
-xref: Reactome:R-HSA-379393 "SLC6A3 cotransports DA, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-379393 "SLC6A3 cotransports DA, Na+, Cl- from extracellular region to cytosol"
 xref: Reactome:R-HSA-380586 "loading of Serotonin in synaptic vesicles"
 xref: Reactome:R-HSA-380620 "Reuptake of serotonin from the synapse"
-xref: Reactome:R-HSA-444160 "VMAT1/2 can mediate the transport of biogenic amines"
 xref: Reactome:R-HSA-5660706 "Defective SLC6A3 does not cotransport DA, Na+ from extracellular region to cytosol"
 is_a: GO:0022804 ! active transmembrane transporter activity
 relationship: part_of GO:0015844 ! monoamine transport
@@ -87504,7 +87799,6 @@
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
 synonym: "organic cation porter activity" RELATED []
 synonym: "polyspecific organic cation transmembrane transporter activity" NARROW []
-xref: Reactome:R-HSA-549241 "SLC22A4 cotransports ERGT, Na+ from extracellular region to cytosol"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0015291
@@ -87515,9 +87809,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other. Organic anions are atoms or small molecules with a negative charge which contain carbon in covalent linkage." [GOC:ai]
 comment: he reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-2142859 "Growing HA is extruded from the cell by ABCC5"
-xref: Reactome:R-HSA-561041 "OAT1-3 transport organic anions with antiport of dicarboxylic acids"
-xref: Reactome:R-HSA-9794830 "SLC22A8 transports Cipro into renal cell"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -87551,11 +87842,9 @@
 synonym: "folic acid transporter activity" RELATED []
 synonym: "vitamin B9 transporter activity" EXACT []
 synonym: "vitamin M transporter activity" EXACT []
-xref: Reactome:R-HSA-200646 "Cytosolic folate export across the plasma membrane"
 xref: Reactome:R-HSA-200652 "SLC19A1 transports 5-methyl-THF from extracellular region to cytosol"
 xref: Reactome:R-HSA-200680 "SLC25A32 transports THF from cytosol to mitochondrial matrix"
 xref: Reactome:R-HSA-200720 "SLC25A32 transports THF from mitochondrial matrix to cytosol"
-xref: Reactome:R-HSA-200729 "SLC46A1 transports FOLA from extracellular region to cytosol"
 is_a: GO:0046943 ! carboxylic acid transmembrane transporter activity
 is_a: GO:0072349 ! modified amino acid transmembrane transporter activity
 is_a: GO:0090482 ! vitamin transmembrane transporter activity
@@ -87646,11 +87935,15 @@
 
 [Term]
 id: GO:0008525
-name: phosphatidylcholine transporter activity
+name: phosphatidylcholine intramembrane carrier activity
 namespace: molecular_function
-def: "Enables the directed movement of phosphatidylcholine into, out of or within a cell, or between cells. Phosphatidylcholine refers to a class of glycerophospholipids in which the phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:ai, ISBN:0198506732]
+def: "Enables the transport of phosphatidylcholine from a region of a membrane to a different region on the same membrane." [GOC:ai, ISBN:0198506732]
+synonym: "phosphatidylcholine intramembrane transporter activity" BROAD []
 synonym: "phosphatidylcholine transmembrane transporter activity" NARROW [GOC:bf, GOC:vw]
-is_a: GO:0005548 ! phospholipid transporter activity
+synonym: "phosphatidylcholine transporter activity" BROAD []
+is_a: GO:0140303 ! intramembrane lipid carrier activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/10882" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0008526
@@ -87753,6 +88046,7 @@
 xref: MetaCyc:2.4.1.149-RXN
 xref: MetaCyc:2.4.1.163-RXN
 xref: Reactome:R-HSA-2025724 "B3GNT1,2,3,4,7 add GlcNAc to form Keratan-PG"
+xref: Reactome:R-HSA-9931297 "B3GNT3 mediated CD274 glycosylation in Golgi"
 xref: RHEA:14389
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.149
@@ -88129,6 +88423,7 @@
 xref: EC:7.6.2.2
 xref: MetaCyc:3.6.3.44-RXN
 xref: Reactome:R-HSA-9757139 "ABCB1,ABCC2 transport ATVs from cytosol to extracellular region"
+xref: Reactome:R-HSA-9911514 "MacA:MacB:TolC transports azithromycin from the cytosol to the periplasm"
 xref: RHEA:33147
 is_a: GO:0042910 ! xenobiotic transmembrane transporter activity
 is_a: GO:0140359 ! ABC-type transporter activity
@@ -88155,6 +88450,8 @@
 synonym: "ATPase-coupled protein transporter activity" EXACT []
 xref: EC:7.4.2.8
 xref: MetaCyc:3.6.3.50-RXN
+xref: Reactome:R-HSA-9760310 "SecA:SecYEG transports proteins from cytosol to periplasmic space"
+xref: Reactome:R-HSA-9926041 "HlyB:HlyD:TolC transports HlyA from cytosol to periplasm"
 is_a: GO:0015450 ! protein-transporting ATPase activity
 property_value: skos:exactMatch EC:7.4.2.8
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30815" xsd:anyURI
@@ -88629,11 +88926,13 @@
 synonym: "hypusine synthesis from peptidyl-lysine" EXACT []
 synonym: "hypusinylation" EXACT []
 synonym: "protein hypusination" EXACT []
+xref: MetaCyc:PWY-5905
 xref: Reactome:R-HSA-204626 "Hypusine synthesis from eIF5A-lysine"
 is_a: GO:0018205 ! peptidyl-lysine modification
 is_a: GO:0051604 ! protein maturation
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0008613
@@ -88819,7 +89118,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "activation of caspase activity by cytochrome c" NARROW []
 synonym: "caspase activation via cytochrome c" EXACT [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-111459 "Activation of caspases through apoptosome-mediated cleavage"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -88915,7 +89213,8 @@
 id: GO:0008649
 name: rRNA methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an rRNA molecule." [GOC:mah]
+def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an rRNA molecule. The methyl group can be transfered to the nucleobase or to the ribose group of the nucleoside." [GOC:mah]
+xref: Reactome:R-HSA-9911509 "RMTases methylate 16S ribosomal RNA"
 is_a: GO:0008173 ! RNA methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
 is_a: GO:0140102 ! catalytic activity, acting on a rRNA
@@ -88923,9 +89222,10 @@
 
 [Term]
 id: GO:0008650
-name: rRNA (uridine-2'-O-)-methyltransferase activity
+name: rRNA (uridine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a uridine in rRNA + S-adenosyl-L-methionine = a 2'-O-methyluridine in rRNA + S-adenosyl-L-homocysteine + H+." [RHEA:54152]
+synonym: "rRNA (uridine-2'-O-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6793127 "MRM2 (FTSJ2) methylates uridine-1369 of 16S rRNA yielding 2'-O-methyluridine"
 xref: RHEA:54152
 is_a: GO:0016436 ! rRNA (uridine) methyltransferase activity
@@ -90050,7 +90350,7 @@
 id: GO:0008718
 name: D-amino-acid dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a D-amino acid + H2O + acceptor = a 2-oxo acid + NH3 + reduced acceptor." [RHEA:18125]
+def: "Catalysis of the reaction: a D-alpha-amino acid + a quinone + H2O = a 2-oxocarboxylate + a quinol + NH4+." [RHEA:18125]
 synonym: "D-amino-acid:(acceptor) oxidoreductase (deaminating)" EXACT []
 synonym: "D-amino-acid:acceptor oxidoreductase (deaminating)" EXACT []
 xref: EC:1.4.5.1
@@ -90100,7 +90400,7 @@
 id: GO:0008721
 name: D-serine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-serine = pyruvate + NH3." [EC:4.3.1.18]
+def: "Catalysis of the reaction: D-serine = pyruvate + NH4+." [RHEA:13977]
 synonym: "D-hydroxy amino acid dehydratase activity" BROAD [EC:4.3.1.18]
 synonym: "D-hydroxyaminoacid dehydratase activity" RELATED [EC:4.3.1.18]
 synonym: "D-serine ammonia-lyase (pyruvate-forming)" RELATED [EC:4.3.1.18]
@@ -90413,7 +90713,6 @@
 xref: EC:1.1.1.103
 xref: KEGG_REACTION:R01465
 xref: MetaCyc:THREODEHYD-RXN
-xref: Reactome:R-HSA-6798667 "TDH tetramer oxidises L-Thr to 2A-3OB"
 xref: RHEA:13161
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.103
@@ -91231,12 +91530,13 @@
 
 [Term]
 id: GO:0008791
-name: arginine N-succinyltransferase activity
+name: L-arginine N-succinyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: succinyl-CoA + L-arginine = CoA + N2-succinyl-L-arginine." [EC:2.3.1.109]
+def: "Catalysis of the reaction: succinyl-CoA + L-arginine = N2-succinyl-L-arginine + CoA + H+." [RHEA:15185]
 synonym: "AOST activity" RELATED [EC:2.3.1.109]
 synonym: "arginine and ornithine N(2)-succinyltransferase activity" RELATED [EC:2.3.1.109]
 synonym: "arginine and ornithine N2-succinyltransferase activity" RELATED [EC:2.3.1.109]
+synonym: "arginine N-succinyltransferase activity" BROAD []
 synonym: "arginine succinyltransferase activity" RELATED [EC:2.3.1.109]
 synonym: "AST activity" RELATED [EC:2.3.1.109]
 synonym: "AstA" RELATED [EC:2.3.1.109]
@@ -91246,9 +91546,11 @@
 xref: MetaCyc:ARGININE-N-SUCCINYLTRANSFERASE-RXN
 xref: RHEA:15185
 is_a: GO:0016749 ! N-succinyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.109
 property_value: skos:exactMatch RHEA:15185
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0008792
@@ -91270,19 +91572,22 @@
 
 [Term]
 id: GO:0008793
-name: aromatic-amino-acid transaminase activity
+name: aromatic-amino-acid:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an L-alpha aromatic amino acid + 2-oxoglutarate = an aromatic oxo acid + L-glutamate." [RHEA:17533]
 synonym: "ArAT" RELATED [EC:2.6.1.57]
 synonym: "aromatic amino acid aminotransferase activity" EXACT [EC:2.6.1.57]
 synonym: "aromatic amino acid transferase activity" EXACT []
 synonym: "aromatic aminotransferase activity" EXACT []
+synonym: "aromatic-amino-acid transaminase activity" BROAD []
 synonym: "aromatic-amino-acid:2-oxoglutarate aminotransferase activity" EXACT []
+xref: EC:2.6.1.57
 xref: MetaCyc:2.6.1.57-RXN
 xref: RHEA:17533
-is_a: GO:0008483 ! transaminase activity
-property_value: skos:broadMatch EC:2.6.1.57
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: skos:exactMatch EC:2.6.1.57
 property_value: skos:exactMatch RHEA:17533
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008794
@@ -91305,7 +91610,7 @@
 namespace: molecular_function
 alt_id: GO:0008749
 alt_id: GO:0016965
-def: "Catalysis of the reaction: ATP + deamido-NAD+ + NH3 = AMP + diphosphate + NAD+." [EC:6.3.1.5]
+def: "Catalysis of the reaction: deamido-NAD+ + NH4+ + ATP = AMP + diphosphate + NAD+ + H+." [RHEA:21188]
 synonym: "deamido-NAD+:ammonia ligase (AMP-forming)" RELATED [EC:6.3.1.5]
 synonym: "diphosphopyridine nucleotide synthetase activity" RELATED [EC:6.3.1.5]
 synonym: "NAD synthase (AMP-forming)" EXACT []
@@ -91334,7 +91639,7 @@
 id: GO:0008797
 name: aspartate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-aspartate = fumarate + NH3." [EC:4.3.1.1]
+def: "Catalysis of the reaction: L-aspartate = fumarate + NH4+." [RHEA:16601]
 synonym: "aspartase activity" RELATED [EC:4.3.1.1]
 synonym: "fumaric aminase activity" RELATED [EC:4.3.1.1]
 synonym: "L-aspartase activity" RELATED [EC:4.3.1.1]
@@ -91358,7 +91663,6 @@
 xref: EC:3.4.19.5
 xref: MetaCyc:3.4.19.5-RXN
 xref: Reactome:R-HSA-5692495 "BACE1 cleaves APP(18-770) to APP(18-671) and APP(672-770)"
-xref: Reactome:R-HSA-5696365 "ASRGL1 hydrolyses aspartame to L-Asp, L-Phe"
 is_a: GO:0008242 ! omega peptidase activity
 is_a: GO:0070003 ! threonine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.19.5
@@ -91382,6 +91686,11 @@
 synonym: "penicillinase I, II" RELATED [EC:3.5.2.6]
 xref: EC:3.5.2.6
 xref: MetaCyc:BETA-LACTAMASE-RXN
+xref: Reactome:R-HSA-9911513 "BlaSHV-12 hydrolyzes ceftriaxone"
+xref: Reactome:R-HSA-9911515 "KPC-2 hydrolyzes cephalothin"
+xref: Reactome:R-HSA-9911518 "OXA-9 hydrolyzes penicillin G"
+xref: Reactome:R-HSA-9911519 "BlaSHV-12 hydrolyzes cephalothin"
+xref: Reactome:R-HSA-9911520 "TEM-1 hydrolyzes penicillin G"
 xref: RHEA:20401
 is_a: GO:0016812 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides
 property_value: skos:exactMatch EC:3.5.2.6
@@ -91456,7 +91765,7 @@
 id: GO:0008804
 name: carbamate kinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + NH3 + CO2 = ADP + carbamoyl phosphate." [EC:2.7.2.2]
+def: "Catalysis of the reaction: hydrogencarbonate + NH4+ + ATP = carbamoyl phosphate + ADP + H2O + H+." [RHEA:10152]
 synonym: "ATP:carbamate phosphotransferase activity" RELATED [EC:2.7.2.2]
 synonym: "carbamoyl phosphokinase activity" RELATED [EC:2.7.2.2]
 synonym: "carbamyl phosphokinase activity" RELATED [EC:2.7.2.2]
@@ -91913,7 +92222,7 @@
 id: GO:0008829
 name: dCTP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dCTP + H2O = dUTP + NH3." [EC:3.5.4.13]
+def: "Catalysis of the reaction: dCTP + H2O + H+ = dUTP + NH4+." [RHEA:22680]
 synonym: "5-methyl-dCTP deaminase activity" RELATED [EC:3.5.4.13]
 synonym: "dCTP aminohydrolase activity" RELATED [EC:3.5.4.13]
 synonym: "deoxycytidine triphosphate deaminase activity" RELATED [EC:3.5.4.13]
@@ -92097,7 +92406,7 @@
 id: GO:0008838
 name: diaminopropionate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,3-diaminopropionate + H2O = pyruvate + 2 NH3." [EC:4.3.1.15]
+def: "Catalysis of the reaction: 2,3-diaminopropanoate + H2O + H+ = pyruvate + 2 NH4+." [EC:4.3.1.15]
 synonym: "2,3-diaminopropionate ammonia-lyase (adding H2O; pyruvate-forming)" RELATED [EC:4.3.1.15]
 synonym: "2,3-diaminopropionate ammonia-lyase activity" RELATED [EC:4.3.1.15]
 synonym: "alpha,beta-diaminopropionate ammonia-lyase activity" RELATED [EC:4.3.1.15]
@@ -92275,6 +92584,7 @@
 def: "Catalysis of the reaction: enterobactin + 3 H2O = 3 N-23-dihydroxybenzoyl-L-serine + 3 H+." [MetaCyc:RXN0-1661, PMID:4565531]
 synonym: "enterobactin esterase activity" EXACT []
 xref: MetaCyc:RXN0-1661
+xref: Reactome:R-HSA-9911599 "Fes hydrolyses ferrienterobactin to DHBS"
 xref: RHEA:28018
 is_a: GO:0016788 ! hydrolase activity, acting on ester bonds
 property_value: skos:exactMatch RHEA:28018
@@ -92284,7 +92594,7 @@
 id: GO:0008851
 name: ethanolamine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ethanolamine = acetaldehyde + NH3." [EC:4.3.1.7]
+def: "Catalysis of the reaction: ethanolamine = acetaldehyde + NH4+." [RHEA:15313]
 synonym: "ethanolamine ammonia-lyase (acetaldehyde-forming) activity" RELATED [EC:4.3.1.7]
 synonym: "ethanolamine deaminase activity" RELATED [EC:4.3.1.7]
 xref: EC:4.3.1.7
@@ -92920,15 +93230,17 @@
 xref: MetaCyc:AKBLIG-RXN
 xref: RHEA:20736
 is_a: GO:0016453 ! C-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.29
 property_value: skos:exactMatch RHEA:20736
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0008892
 name: guanine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanine + H2O = xanthine + NH3." [EC:3.5.4.3]
+def: "Catalysis of the reaction: guanine + H2O + H+ = xanthine + NH4+." [RHEA:14665]
 synonym: "GAH activity" RELATED [EC:3.5.4.3]
 synonym: "guanase activity" RELATED [EC:3.5.4.3]
 synonym: "guanine aminase activity" RELATED [EC:3.5.4.3]
@@ -93542,16 +93854,15 @@
 name: mannitol-1-phosphate 5-dehydrogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: D-mannitol 1-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+." [EC:1.1.1.17]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:1.1.1.17
 xref: MetaCyc:MANNPDEHYDROG-RXN
 xref: RHEA:19661
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.17
 property_value: skos:exactMatch RHEA:19661
-property_value: skos:relatedMatch MetaCyc:MANNIDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0008927
@@ -94006,6 +94317,7 @@
 synonym: "undecaprenyldiphospho-N-acetyl-D-glucosaminyl-(1->4)-(N-acetyl-D-muramoylpentapeptide):undecaprenyldiphospho-(N-acetyl-D-glucosaminyl-(1->4)-N-acetyl-D-muramoylpentapeptide) disaccharidetransferase activity" RELATED [EC:2.4.99.28]
 xref: EC:2.4.99.28
 xref: MetaCyc:RXN-14712
+xref: Reactome:R-HSA-9911463 "MrcB elongates peptidoglycan"
 xref: RHEA:23708
 is_a: GO:0016758 ! hexosyltransferase activity
 property_value: skos:exactMatch EC:2.4.99.28
@@ -94238,6 +94550,7 @@
 xref: Reactome:R-HSA-1482920 "PG is hydrolyzed to 2-acyl LPG by PLA2[14]"
 xref: Reactome:R-HSA-1482932 "PI is hydrolyzed to 2-acyl LPI by PLA2[13]"
 xref: Reactome:R-HSA-5694485 "ABHD3 hydrolyses LPC(14:0) to 1AGPC"
+xref: Reactome:R-HSA-8869425 "PLA1A hydrolyses PS to 2-acyl LPS"
 xref: RHEA:67424
 is_a: GO:0140322 ! A1-type glycerophospholipase activity
 property_value: skos:exactMatch EC:3.1.1.32
@@ -94407,11 +94720,11 @@
 
 [Term]
 id: GO:0008982
-name: protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+name: protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 namespace: molecular_function
 alt_id: GO:0015455
 alt_id: GO:0015456
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sugar(out) = protein histidine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sugar(out) = protein histidine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, PMID:16339738, PMID:31214989]
 synonym: "enzyme II of the phosphotransferase system" RELATED []
 synonym: "PEP--sugar phosphotransferase enzyme II activity" EXACT []
 synonym: "PEP-dependent phosphotransferase enzyme II" RELATED []
@@ -94419,6 +94732,7 @@
 synonym: "phosphohistidinoprotein-hexose phosphotransferase activity" RELATED []
 synonym: "phosphoprotein factor-hexose phosophotransferase activity" RELATED []
 synonym: "phosphotransfer-driven group translocator" BROAD []
+synonym: "protein-N(PI)-phosphohistidine-sugar phosphotransferase activity" RELATED []
 synonym: "protein-Np-phosphohistidine-sugar phosphotransferase activity" EXACT []
 synonym: "PTS permease activity" RELATED []
 synonym: "PTS transporter" RELATED []
@@ -94426,6 +94740,7 @@
 is_a: GO:0015144 ! carbohydrate transmembrane transporter activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
 is_a: GO:0022804 ! active transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0008983
@@ -94726,13 +95041,14 @@
 
 [Term]
 id: GO:0009001
-name: serine O-acetyltransferase activity
+name: L-serine O-acetyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-serine + acetyl-CoA = O-acetyl-L-serine + CoA." [RHEA:24560]
 synonym: "acetyl-CoA:L-serine O-acetyltransferase activity" RELATED [EC:2.3.1.30]
 synonym: "L-serine acetyltransferase activity" RELATED [EC:2.3.1.30]
 synonym: "SATase activity" RELATED [EC:2.3.1.30]
 synonym: "serine acetyltransferase activity" RELATED [EC:2.3.1.30]
+synonym: "serine O-acetyltransferase activity" BROAD []
 synonym: "serine transacetylase activity" RELATED [EC:2.3.1.30]
 xref: EC:2.3.1.30
 xref: KEGG_REACTION:R00586
@@ -94740,9 +95056,11 @@
 xref: RHEA:24560
 is_a: GO:0016412 ! serine O-acyltransferase activity
 is_a: GO:0016413 ! O-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.30
 property_value: skos:exactMatch RHEA:24560
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0009002
@@ -95008,12 +95326,13 @@
 
 [Term]
 id: GO:0009016
-name: succinyldiaminopimelate transaminase activity
+name: succinyldiaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + N-succinyl-LL-2,6-diaminopimelate = L-2-succinylamino-6-oxopimelate + L-glutamate." [EC:2.6.1.17, RHEA:11960]
+def: "Catalysis of the reaction: N-succinyl-(2S,6S)-2,6-diaminopimelate + 2-oxoglutarate = (S)-2-succinylamino-6-oxoheptanedioate + L-glutamate." [RHEA:11960]
 synonym: "N-succinyl-L-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.17]
 synonym: "N-succinyl-L-diaminopimelic glutamic transaminase activity" RELATED [EC:2.6.1.17]
 synonym: "succinyldiaminopimelate aminotransferase activity" EXACT []
+synonym: "succinyldiaminopimelate transaminase activity" EXACT []
 synonym: "succinyldiaminopimelate transferase activity" RELATED [EC:2.6.1.17]
 xref: EC:2.6.1.17
 xref: KEGG_REACTION:R04475
@@ -95023,6 +95342,7 @@
 property_value: skos:exactMatch EC:2.6.1.17
 property_value: skos:exactMatch RHEA:11960
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0009017
@@ -95287,7 +95607,6 @@
 name: tryptophanase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-tryptophan + H2O = indole + NH4 + pyruvate." [RHEA:19553]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-tryptophan indole-lyase (deaminating) activity" RELATED [EC:4.1.99.1]
 synonym: "L-tryptophan indole-lyase (deaminating; pyruvate forming) activity" RELATED [EC:4.1.99.1]
 synonym: "L-tryptophan indole-lyase activity" RELATED [EC:4.1.99.1]
@@ -95305,8 +95624,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00673
 property_value: skos:exactMatch MetaCyc:TRYPTOPHAN-RXN
 property_value: skos:exactMatch RHEA:19553
-property_value: skos:relatedMatch MetaCyc:TRYPDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0009035
@@ -95371,7 +95690,7 @@
 id: GO:0009039
 name: urease activity
 namespace: molecular_function
-def: "Catalysis of the reaction: urea + H2O = CO2 + 2 NH3." [EC:3.5.1.5, RHEA:20557]
+def: "Catalysis of the reaction: urea + 2 H2O + H+ = hydrogencarbonate + 2 NH4+." [RHEA:20557]
 synonym: "urea amidohydrolase activity" RELATED [EC:3.5.1.5]
 xref: EC:3.5.1.5
 xref: MetaCyc:UREASE-RXN
@@ -95409,23 +95728,25 @@
 
 [Term]
 id: GO:0009042
-name: valine-pyruvate transaminase activity
+name: L-valine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine." [EC:2.6.1.66, RHEA:22912]
+def: "Catalysis of the reaction: L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine." [RHEA:22912]
 synonym: "alanine--valine transaminase activity" RELATED [EC:2.6.1.66]
 synonym: "alanine-oxoisovalerate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "L-valine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "transaminase C activity" NARROW [EC:2.6.1.66]
 synonym: "valine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "valine-pyruvate aminotransferase activity" EXACT []
+synonym: "valine-pyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.66
 xref: KEGG_REACTION:R01215
 xref: MetaCyc:VALINE-PYRUVATE-AMINOTRANSFER-RXN
 xref: RHEA:22912
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.66
 property_value: skos:exactMatch RHEA:22912
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0009044
@@ -95582,7 +95903,6 @@
 synonym: "electron donor activity" NARROW []
 synonym: "electron transporter activity" RELATED []
 xref: Reactome:R-HSA-169260 "Reducing equivalents from beta-oxidation of fatty acids transfer to ETF"
-xref: Reactome:R-HSA-2564826 "4Fe-4S cluster assembles on NUBP2:NUBP1 scaffold"
 is_a: GO:0003674 ! molecular_function
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21334" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26572" xsd:anyURI
@@ -95733,7 +96053,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai]
 comment: The reason for obsoletion is that this is an unnecessary grouping term.
 synonym: "glutamine family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8964539 "Glutamate and glutamine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
 is_obsolete: true
 
@@ -95756,7 +96075,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai]
 comment: The reason for obsoletion is that this is an unnecessary grouping term.
 synonym: "aspartate family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8963693 "Aspartate and asparagine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
 is_obsolete: true
 
@@ -95764,16 +96082,15 @@
 id: GO:0009067
 name: aspartate family amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai, MetaCyc:PWY0-781]
+def: "The chemical reactions and pathways resulting in the formation of amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai]
 synonym: "aspartate family amino acid anabolism" EXACT []
 synonym: "aspartate family amino acid biosynthesis" EXACT []
 synonym: "aspartate family amino acid formation" EXACT []
 synonym: "aspartate family amino acid synthesis" EXACT []
-xref: MetaCyc:PWY0-781
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY0-781
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009068
@@ -95806,14 +96123,9 @@
 synonym: "serine family amino acid biosynthesis" EXACT []
 synonym: "serine family amino acid formation" EXACT []
 synonym: "serine family amino acid synthesis" EXACT []
-xref: MetaCyc:PWY-6292
-xref: MetaCyc:SER-GLYSYN-PWY
-xref: MetaCyc:SULFATE-CYS-PWY
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
-property_value: skos:narrowMatch MetaCyc:PWY-6292
-property_value: skos:narrowMatch MetaCyc:SER-GLYSYN-PWY
-property_value: skos:narrowMatch MetaCyc:SULFATE-CYS-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009071
@@ -95909,7 +96221,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving any amino acid that requires pyruvate for its synthesis, e.g. alanine." [GOC:jl]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "pyruvate family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8964540 "Alanine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -95977,32 +96288,42 @@
 id: GO:0009084
 name: L-glutamine family amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai, MetaCyc:GLUTAMINE-SYN]
+def: "The chemical reactions and pathways resulting in the formation of amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai]
 synonym: "glutamine family amino acid anabolism" EXACT []
 synonym: "glutamine family amino acid biosynthesis" EXACT []
 synonym: "glutamine family amino acid formation" EXACT []
 synonym: "glutamine family amino acid synthesis" EXACT []
-xref: MetaCyc:GLUTAMINE-SYN
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:GLUTAMINE-SYN
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009085
 name: L-lysine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine, 2,6-diaminohexanoic acid." [GOC:go_curators, MetaCyc:LYSINE-SYN]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine, 2,6-diaminohexanoic acid." [GOC:go_curators]
 synonym: "lysine anabolism" EXACT []
 synonym: "lysine biosynthesis" EXACT []
 synonym: "lysine formation" EXACT []
 synonym: "lysine synthesis" EXACT []
-xref: MetaCyc:LYSINE-SYN
+xref: MetaCyc:DAPLYSINESYN-PWY
+xref: MetaCyc:LYSINE-AMINOAD-PWY
+xref: MetaCyc:PWY-2941
+xref: MetaCyc:PWY-2942
+xref: MetaCyc:PWY-3081
+xref: MetaCyc:PWY-5097
 is_a: GO:0006553 ! lysine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:LYSINE-SYN
+property_value: skos:narrowMatch MetaCyc:DAPLYSINESYN-PWY
+property_value: skos:narrowMatch MetaCyc:LYSINE-AMINOAD-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-2941
+property_value: skos:narrowMatch MetaCyc:PWY-2942
+property_value: skos:narrowMatch MetaCyc:PWY-3081
+property_value: skos:narrowMatch MetaCyc:PWY-5097
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28388" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009086
@@ -96022,7 +96343,7 @@
 id: GO:0009087
 name: L-methionine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-methionine (2-amino-4-(methylthio)butanoic acid), a sulfur-containing, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-methionine." [GOC:curators]
 synonym: "methionine breakdown" EXACT []
 synonym: "methionine catabolism" EXACT []
 synonym: "methionine degradation" EXACT []
@@ -96038,23 +96359,23 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5082
 property_value: skos:narrowMatch MetaCyc:PWY-701
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0009088
 name: L-threonine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732, MetaCyc:THRESYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
 synonym: "threonine anabolism" EXACT []
 synonym: "threonine biosynthesis" EXACT []
 synonym: "threonine formation" EXACT []
 synonym: "threonine synthesis" EXACT []
 xref: MetaCyc:HOMOSER-THRESYN-PWY
-xref: MetaCyc:THRESYN-PWY
 is_a: GO:0006566 ! L-threonine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:HOMOSER-THRESYN-PWY
-property_value: skos:narrowMatch MetaCyc:THRESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009089
@@ -96089,17 +96410,17 @@
 
 [Term]
 id: GO:0009091
-name: L-homoserine catabolic process
+name: obsolete L-homoserine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of homoserine, alpha-amino-gamma-hydroxybutyric acid." [ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of homoserine, alpha-amino-gamma-hydroxybutyric acid." [ISBN:0198506732]
+comment: This term was obsoleted because there is no evidence that a specific pathway for L-homoserine degradation exists.
 synonym: "homoserine breakdown" EXACT []
 synonym: "homoserine catabolism" EXACT []
 synonym: "homoserine degradation" EXACT []
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0009092
@@ -96140,9 +96461,16 @@
 synonym: "phenylalanine biosynthetic process" BROAD []
 synonym: "phenylalanine biosynthetic process, prephenate pathway" RELATED []
 synonym: "phenylalanine biosynthetic process, shikimate pathway" RELATED []
+xref: MetaCyc:PHESYN
+xref: MetaCyc:PWY-3462
+xref: MetaCyc:PWY-7432
 is_a: GO:0006558 ! L-phenylalanine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:1902223 ! erythrose 4-phosphate/phosphoenolpyruvate family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PHESYN
+property_value: skos:narrowMatch MetaCyc:PWY-3462
+property_value: skos:narrowMatch MetaCyc:PWY-7432
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009095
@@ -96160,17 +96488,17 @@
 
 [Term]
 id: GO:0009097
-name: isoleucine biosynthetic process
+name: obsolete isoleucine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+comment: The reason for obsoletion is that more specific terms were created.
 synonym: "isoleucine anabolism" EXACT []
 synonym: "isoleucine biosynthesis" EXACT []
 synonym: "isoleucine formation" EXACT []
 synonym: "isoleucine synthesis" EXACT []
-xref: MetaCyc:ILEUSYN-PWY
-is_a: GO:0006549 ! isoleucine metabolic process
-is_a: GO:0009082 ! branched-chain amino acid biosynthetic process
-is_a: GO:1901607 ! alpha-amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30848" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:1901705
 
 [Term]
 id: GO:0009098
@@ -96204,7 +96532,7 @@
 relationship: has_part GO:0003984 ! acetolactate synthase activity
 relationship: has_part GO:0004160 ! dihydroxy-acid dehydratase activity
 relationship: has_part GO:0004455 ! ketol-acid reductoisomerase activity
-relationship: has_part GO:0052655 ! L-valine-2-oxoglutarate transaminase activity
+relationship: has_part GO:0052655 ! L-valine:2-oxoglutarate transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28686" xsd:anyURI
 
 [Term]
@@ -98815,7 +99143,7 @@
 id: GO:0009339
 name: glycolate oxidase complex
 namespace: cellular_component
-def: "An enzyme complex that catalyzes the oxidation of 2-hydroxy acid to form 2-oxo acid and hydrogen peroxide (H2O2). The enzyme is a flavoprotein (FMN)." [PMID:4557653, PMID:8606183]
+def: "An enzyme complex that catalyzes the oxidation of 2-hydroxy acid to form 2-oxo acid and H2O2. The enzyme is a flavoprotein (FMN)." [PMID:4557653, PMID:8606183]
 is_a: GO:1990204 ! oxidoreductase complex
 relationship: part_of GO:0005737 ! cytoplasm
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21246" xsd:anyURI
@@ -99256,10 +99584,18 @@
 id: GO:0009389
 name: dimethyl sulfoxide reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dimethyl sulfoxide + H+ = dimethyl sulfide + H2O." [PMID:8658134]
+def: "Catalysis of the reaction: dimethyl sulfide + a menaquinone + H2O = dimethyl sulfoxide + a menaquinol." [PMID:8658134, RHEA:28494]
 synonym: "dimethyl sulphoxide reductase activity" EXACT []
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+synonym: "respiratory dimethylsulfoxide reductase" EXACT [EC:1.8.5.3]
+xref: EC:1.8.5.3
+xref: MetaCyc:DIMESULFREDUCT-RXN
+xref: RHEA:28494
+is_a: GO:0016672 ! oxidoreductase activity, acting on a sulfur group of donors, quinone or similar compound as acceptor
+property_value: skos:exactMatch EC:1.8.5.3
+property_value: skos:exactMatch MetaCyc:DIMESULFREDUCT-RXN
+property_value: skos:exactMatch RHEA:28494
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28261" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0009390
@@ -99918,7 +100254,7 @@
 id: GO:0009450
 name: gamma-aminobutyric acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of gamma-aminobutyric acid (GABA, 4-aminobutyrate), an amino acid which acts as a neurotransmitter in some organisms." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of gamma-aminobutyric acid (GABA)." [GOC:curators]
 comment: See also the biological process term 'neurotransmitter catabolic process ; GO:0042135'.
 synonym: "4-aminobutanoate catabolic process" EXACT []
 synonym: "4-aminobutanoate catabolism" EXACT []
@@ -99944,6 +100280,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-6536
 property_value: skos:narrowMatch MetaCyc:PWY-6537
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0009451
@@ -103482,8 +103819,11 @@
 synonym: "cytokinin breakdown" EXACT []
 synonym: "cytokinin catabolism" EXACT []
 synonym: "cytokinin degradation" EXACT []
+xref: MetaCyc:PWY-2841
 is_a: GO:0009690 ! cytokinin metabolic process
 is_a: GO:0042447 ! hormone catabolic process
+property_value: skos:exactMatch MetaCyc:PWY-2841
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0009824
@@ -104256,6 +104596,7 @@
 synonym: "up-regulation of metabolic process" EXACT []
 synonym: "up-regulation of organismal metabolic process" NARROW []
 synonym: "upregulation of metabolic process" EXACT []
+xref: Reactome:R-HSA-163765 "ChREBP activates metabolic gene expression"
 is_a: GO:0019222 ! regulation of metabolic process
 is_a: GO:0048522 ! positive regulation of cellular process
 intersection_of: GO:0065007 ! biological regulation
@@ -104318,24 +104659,26 @@
 id: GO:0009897
 name: external side of plasma membrane
 namespace: cellular_component
-def: "The leaflet of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
+def: "The leaflet of the plasma membrane that faces the extracellular side of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, GOC:tb]
 synonym: "external leaflet of plasma membrane" EXACT [GOC:ab]
 synonym: "juxtamembrane" BROAD []
 synonym: "outer surface of cytoplasmic membrane" EXACT []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0005886 ! plasma membrane
 relationship: part_of GO:0009986 ! cell surface
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0009898
 name: cytoplasmic side of plasma membrane
 namespace: cellular_component
-def: "The leaflet the plasma membrane that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
+def: "The leaflet of the plasma membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, GOC:tb]
 synonym: "internal leaflet of plasma membrane" EXACT [GOC:ab]
 synonym: "internal side of plasma membrane" EXACT []
 synonym: "juxtamembrane" BROAD []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005886 ! plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0009899
@@ -104768,10 +105111,13 @@
 
 [Term]
 id: GO:0009940
-name: amino-terminal vacuolar sorting propeptide binding
+name: obsolete amino-terminal vacuolar sorting propeptide binding
 namespace: molecular_function
-def: "Binding to an amino terminal propeptide, which functions as a sorting signal to sort away the soluble vacuolar protein from Golgi to lytic vacuole via clathrin-coated vesicles." [GOC:sm, PMID:10871276]
-is_a: GO:0010209 ! vacuolar sorting signal binding
+def: "OBSOLETE. Binding to an amino terminal propeptide, which functions as a sorting signal to sort away the soluble vacuolar protein from Golgi to lytic vacuole via clathrin-coated vesicles." [GOC:sm, PMID:10871276]
+comment: This term was obsolete because it is out of scope for GO; target regions of interactions are not captured.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31410" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0010209
 
 [Term]
 id: GO:0009941
@@ -105202,7 +105548,7 @@
 synonym: "delta-pH-dependent protein transporter activity" EXACT []
 synonym: "pH-dependent protein transporter activity" EXACT []
 synonym: "twin-arginine targeting transmembrane transporter activity" EXACT []
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 is_a: GO:0015291 ! secondary active transmembrane transporter activity
 
 [Term]
@@ -105260,11 +105606,13 @@
 alt_id: GO:0016439
 def: "Catalysis of the reaction: a uridine in RNA = a pseudouridine in RNA. Conversion of uridine in an RNA molecule to pseudouridine by rotation of the C1'-N-1 glycosidic bond of uridine in RNA to a C1'-C5." [GOC:mah, RHEA:48348]
 comment: Note that this term should not be confused with 'pseudouridylate synthase activity ; GO:0004730', which refers to the formation of free pseudouridine from uracil and ribose-5-phosphate.
-xref: Reactome:R-HSA-6782381 "PUS1 isoform 2 transforms uridine residues to pseudouridine in the anticodon stems of tRNAs"
-xref: Reactome:R-HSA-6786583 "PUS7 transforms uridine to pseudouridine in tRNAs"
-xref: Reactome:R-HSA-6787566 "PUS1 isoform 1 transforms uridine-27, uridine-28 yielding pseudouridine in tRNA(Lys,Ser)"
+xref: Reactome:R-HSA-6782381 "PUS1 isoform 2 isomerizes uridine residues to pseudouridine residues in the anticodon stems of tRNAs"
+xref: Reactome:R-HSA-6786583 "PUS7 isomerizes uridine to pseudouridine in tRNAs"
+xref: Reactome:R-HSA-6787566 "PUS1 isoform 1 isomerizes uridine-27, uridine-28 to pseudouridine residues in tRNA(Lys,Ser)"
 xref: Reactome:R-HSA-6790905 "Box H/ACA snoRNP transforms uridine to pseudouridine in pre-rRNA"
-xref: Reactome:R-HSA-8870289 "PUS3 transforms uridine-39 to pseudouridine-39 in tRNA"
+xref: Reactome:R-HSA-8870289 "PUS3 isomerizes uridine-39 to pseudouridine-39 in tRNA"
+xref: Reactome:R-HSA-9936243 "RPUSD4 in the mitochondrial pseudouridylation module transforms uridine-1397 to pseudouridine-1397 in mitochondrial 16S rRNA"
+xref: Reactome:R-HSA-9937369 "RPUSD3,TRUB2 pseudouridylates MT-CO1,MT-CO3 mRNA"
 xref: RHEA:48348
 is_a: GO:0016866 ! intramolecular transferase activity
 property_value: skos:exactMatch RHEA:48348
@@ -105531,11 +105879,12 @@
 id: GO:0010009
 name: cytoplasmic side of endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "external leaflet of endosome membrane" EXACT [GOC:ab]
 synonym: "external side of endosome membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0010008 ! endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0010011
@@ -107488,10 +107837,12 @@
 
 [Term]
 id: GO:0010209
-name: vacuolar sorting signal binding
+name: vacuolar sorting signal receptor activity
 namespace: molecular_function
 def: "Binding to a vacuolar sorting signal, a specific peptide sequence that acts as a signal to localize the protein within the vacuole." [GOC:mah]
-is_a: GO:0005048 ! signal sequence binding
+synonym: "vacuolar sorting signal binding" RELATED []
+is_a: GO:0038024 ! cargo receptor activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31410" xsd:anyURI
 
 [Term]
 id: GO:0010210
@@ -108218,10 +108569,11 @@
 
 [Term]
 id: GO:0010285
-name: L,L-diaminopimelate aminotransferase activity
+name: L,L-diaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
 alt_id: GO:0043742
-def: "Catalysis of the reaction: 2-oxoglutarate + LL-2,6-diaminopimelate = (S)-2,3,4,5-tetrahydrodipicolinate + L-glutamate + H2O + H+." [EC:2.6.1.83, RHEA:23988]
+def: "Catalysis of the reaction: 2-oxoglutarate + LL-2,6-diaminopimelate = (S)-2,3,4,5-tetrahydrodipicolinate + L-glutamate + H2O + H+." [RHEA:23988]
+synonym: "L,L-diaminopimelate aminotransferase activity" EXACT []
 synonym: "LL-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "LL-DAP aminotransferase activity" RELATED [EC:2.6.1.83]
 synonym: "LL-DAP-AT activity" RELATED [EC:2.6.1.83]
@@ -108235,6 +108587,7 @@
 property_value: skos:exactMatch EC:2.6.1.83
 property_value: skos:exactMatch RHEA:23988
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0010286
@@ -108653,7 +109006,6 @@
 namespace: biological_process
 alt_id: GO:1902534
 def: "The infolding of a membrane." [GOC:tb]
-subset: goslim_yeast
 synonym: "single-organism membrane invagination" RELATED []
 is_a: GO:0061024 ! membrane organization
 created_by: jl
@@ -108669,17 +109021,19 @@
 
 [Term]
 id: GO:0010326
-name: methionine-oxo-acid transaminase activity
+name: L-methionine:oxo-acid transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a 2-oxocarboxylate + L-methionine = 4-methylsulfanyl-2-oxobutanoate + an L-alpha-amino acid." [PMID:17056707, RHEA:31763]
+synonym: "methionine-oxo-acid transaminase activity" EXACT []
 xref: EC:2.6.1.88
 xref: MetaCyc:R15-RXN
 xref: RHEA:31763
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.88
 property_value: skos:exactMatch RHEA:31763
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28199" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0010327
@@ -108938,12 +109292,10 @@
 
 [Term]
 id: GO:0010355
-name: obsolete homogentisate farnesyltransferase activity
+name: homogentisate farnesyltransferase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: homogentisic acid + farnesyl diphosphate = 2-methyl-6-farnesylplastoquinol." [PMID:16989822]
-comment: The reason for obsoletion is that the data from the paper for which the term was requested can be accurately described using homogentisate solanesyltransferase activity.
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29344" xsd:anyURI
-is_obsolete: true
+def: "Catalysis of the reaction: homogentisic acid + farnesyl diphosphate = 2-methyl-6-farnesylplastoquinol." [PMID:16989822]
+is_a: GO:0010354 ! homogentisate prenyltransferase activity
 
 [Term]
 id: GO:0010356
@@ -110307,6 +110659,7 @@
 name: proteasomal ubiquitin-independent protein catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds that is mediated by the proteasome but do not involve ubiquitin." [GOC:tb]
+xref: Reactome:R-HSA-9912633 "Antigen processing: Ub, ATP-independent proteasomal degradation"
 is_a: GO:0010498 ! proteasomal protein catabolic process
 
 [Term]
@@ -110796,23 +111149,27 @@
 name: regulation of macromolecule biosynthetic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency or extent of the chemical reactions and pathways resulting in the formation of a macromolecule, any molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from molecules of low relative molecular mass." [GOC:dph, GOC:tb]
+subset: gocheck_do_not_annotate
 is_a: GO:0009889 ! regulation of biosynthetic process
 is_a: GO:0060255 ! regulation of macromolecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0009059 ! macromolecule biosynthetic process
 relationship: regulates GO:0009059 ! macromolecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31483" xsd:anyURI
 
 [Term]
 id: GO:0010557
 name: positive regulation of macromolecule biosynthetic process
 namespace: biological_process
 def: "Any process that increases the rate, frequency or extent of the chemical reactions and pathways resulting in the formation of a macromolecule, any molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from molecules of low relative molecular mass." [GOC:dph, GOC:tb]
+subset: gocheck_do_not_annotate
 is_a: GO:0009891 ! positive regulation of biosynthetic process
 is_a: GO:0010556 ! regulation of macromolecule biosynthetic process
 is_a: GO:0010604 ! positive regulation of macromolecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0009059 ! macromolecule biosynthetic process
 relationship: positively_regulates GO:0009059 ! macromolecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31483" xsd:anyURI
 
 [Term]
 id: GO:0010558
@@ -112686,7 +113043,6 @@
 synonym: "protein kinase A signal transduction" EXACT [GOC:signaling]
 synonym: "protein kinase A signaling cascade" RELATED [GOC:signaling]
 synonym: "protein kinase A signalling cascade" RELATED [GOC:mah]
-xref: Reactome:R-HSA-163358 "PKA-mediated phosphorylation of key metabolic factors"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26794" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26899" xsd:anyURI
 is_obsolete: true
@@ -114823,10 +115179,13 @@
 
 [Term]
 id: GO:0010944
-name: negative regulation of transcription by competitive promoter binding
+name: obsolete negative regulation of transcription by competitive promoter binding
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA-dependent transcription using a mechanism that involves direct competition for interaction with a promoter binding site." [GOC:tb]
-is_a: GO:0045892 ! negative regulation of DNA-templated transcription
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of DNA-dependent transcription using a mechanism that involves direct competition for interaction with a promoter binding site." [GOC:tb]
+comment: This term was obsoleted because it represents a mechanism and is outside the scope of GO.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19894" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0045892
 
 [Term]
 id: GO:0010945
@@ -114912,6 +115271,7 @@
 namespace: biological_process
 def: "Any process that increases the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins." [GOC:dph, GOC:tb]
 subset: gocheck_obsoletion_candidate
+xref: Reactome:R-HSA-140534 "Caspase activation via Death Receptors in the presence of ligand"
 is_a: GO:0010952 ! positive regulation of peptidase activity
 is_a: GO:0052548 ! regulation of endopeptidase activity
 intersection_of: GO:0065007 ! biological regulation
@@ -114927,6 +115287,7 @@
 def: "Any process that decreases the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins." [GOC:dph, GOC:tb]
 subset: gocheck_obsoletion_candidate
 subset: goslim_chembl
+xref: Reactome:R-HSA-5218900 "CASP8 activity is inhibited"
 is_a: GO:0010466 ! negative regulation of peptidase activity
 is_a: GO:0052548 ! regulation of endopeptidase activity
 intersection_of: GO:0065007 ! biological regulation
@@ -118203,7 +118564,6 @@
 xref: MetaCyc:2.4.1.135-RXN
 xref: Reactome:R-HSA-1889955 "B3GAT dimers transfer GlcA to tetrasaccharide linker"
 xref: Reactome:R-HSA-3560802 "Defective B3GAT3 does not transfer GlcA to tetrasaccharide linker"
-xref: Reactome:R-HSA-9638064 "B3GAT3 dimer transfers GlcA to tetrasaccharide linker"
 xref: RHEA:24168
 is_a: GO:0015020 ! glucuronosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.135
@@ -118304,13 +118664,13 @@
 xref: Reactome:R-HSA-2162099 "abacavir + UDP-glucuronate => abacavir 5'-glucuronide + UDP"
 xref: Reactome:R-HSA-5604954 "Defective UGT1A4 does not transfer GlcA from UDP-GlcA to BIL"
 xref: Reactome:R-HSA-5604975 "Defective UGT1A1 does not transfer GlcA from UDP-GlcA to BIL"
-xref: Reactome:R-HSA-5617143 "B4GAT1:GYLTL1B transfers GlcA from UDP-GlcA to Xyl-GlcA"
+xref: Reactome:R-HSA-5617143 "B4GAT1:LARGE transfers initial GlcA to matriglycan chain"
 xref: Reactome:R-HSA-8941701 "UGT1A10 transfers GlcA from UDP-GlcA to GCTN"
 xref: Reactome:R-HSA-9036102 "Defective UGT1A1 does not transfer GlcA from UDP-GlcA to BMG"
 xref: Reactome:R-HSA-9036104 "Defective UGT1A4 does not transfer GlcA from UDP-GlcA to BMG"
 xref: Reactome:R-HSA-9632038 "UGT1A1 tetramer transfers GlcA from UDP-GlcA to BMG to form BDG"
 xref: Reactome:R-HSA-9632039 "UGT1A1 transfers GlcA from UDP-GlcA to BIL to form BMG"
-xref: Reactome:R-HSA-9638097 "B4GAT1:LARGE transfers GlcA from UDP-GlcA to Xyl-GlcA"
+xref: Reactome:R-HSA-9638097 "LARGE1,2 elongates matriglycan chain with GlcA"
 xref: Reactome:R-HSA-9749977 "UGT1A6 glucuronates ST"
 xref: Reactome:R-HSA-9756134 "UGT1A3 lactonizes 2-OH-ATV to 2-OH-ATVL"
 xref: Reactome:R-HSA-9756156 "UGT1A3 lactonizes ATV to ATVL"
@@ -118453,7 +118813,7 @@
 namespace: molecular_function
 alt_id: GO:0008895
 alt_id: GO:0015037
-def: "Catalysis of the reaction: a protein with reduced sulfide groups = a protein with oxidized disulfide bonds." [PMID:7559385]
+def: "Catalysis of the reaction: a protein with reduced sulfide groups = a protein with oxidized disulfide bonds." [PMID:18331844]
 synonym: "haem lyase disulphide oxidoreductase activity" NARROW []
 synonym: "heme lyase disulfide oxidoreductase activity" NARROW []
 synonym: "peptide disulfide oxidoreductase activity" EXACT []
@@ -118461,7 +118821,6 @@
 synonym: "protein disulfide oxidoreductase activity" EXACT []
 synonym: "protein disulfide-oxidoreductase activity" EXACT []
 synonym: "protein disulphide oxidoreductase activity" EXACT []
-xref: MetaCyc:DISULFOXRED-RXN
 xref: Reactome:R-HSA-1307802 "MIA40:ERV1 (CHCHD4:GFER) oxidizes cysteine residues to cystine disulfide bonds"
 xref: Reactome:R-HSA-264997 "P4HB mediates disulfide bond formation in Proinsulin"
 xref: Reactome:R-HSA-3299753 "CCS transfers Cu to SOD1 and oxidizes cysteine residues in SOD1"
@@ -118469,6 +118828,7 @@
 is_a: GO:0015036 ! disulfide oxidoreductase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20219" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0015036
@@ -119094,6 +119454,8 @@
 xref: Reactome:R-HSA-1362417 "Mitoferrin translocates iron from the mitochondrial intermembrane space to the mitochondrial matrix"
 xref: Reactome:R-HSA-442368 "SLC40A1:HEPH:6Cu2+ transports Fe2+ from cytosol to extracellular region"
 xref: Reactome:R-HSA-5655760 "Defective SLC40A1 does not transport Fe3+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-9911596 "FeoAB transports Fe2+ from periplasmic space to the cytosol"
+xref: Reactome:R-HSA-9911598 "EfeUOB transport Fe2+ from periplasmic space to the cytosol"
 is_a: GO:0005381 ! iron ion transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -119177,16 +119539,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Enables the transfer of organic cations from one side of a membrane to the other. Organic cations are atoms or small molecules with a positive charge that contain carbon in covalent linkage." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-2161500 "abacavir [extracellular] => abacavir [cytosol]"
-xref: Reactome:R-HSA-549129 "OCT1 transports organic cations into hepatic cells"
-xref: Reactome:R-HSA-549279 "OCT2 mediates tubular uptake of organic cations in the kidney"
-xref: Reactome:R-HSA-549304 "OCT3 mediates renal clearance of organic cations"
-xref: Reactome:R-HSA-549322 "OCT1 transports organic cations out of hepatic cells"
-xref: Reactome:R-HSA-561054 "OCT2 mediates tubular secretion of organic cations in the kidney"
-xref: Reactome:R-HSA-561072 "OCT3 mediates renal uptake of organic cations"
-xref: Reactome:R-HSA-9794270 "SLCO1A2 transports Cipro(1+) into the cytosol"
-xref: Reactome:R-HSA-9794523 "SLC22A1 transports Cipro into renal cell"
-xref: Reactome:R-HSA-9795207 "SLC22A1 transports Cipro into hepatic cell"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -119280,7 +119632,7 @@
 namespace: molecular_function
 def: "Enables the transfer of iodide ions from one side of a membrane to the other." [GOC:ai]
 xref: Reactome:R-HSA-209910 "Iodide is taken up by thyroid epithelial cells"
-xref: Reactome:R-HSA-5627802 "SLC26A4 transports I- from cytosol to extracellular region"
+xref: Reactome:R-HSA-5627802 "SLC26A4 exports I- to follicular lumen"
 xref: Reactome:R-HSA-5627870 "SLC26A4 does not transport I- from cytosol to extracellular region"
 is_a: GO:0008509 ! monoatomic anion transmembrane transporter activity
 relationship: part_of GO:0015705 ! iodide transport
@@ -119335,8 +119687,9 @@
 synonym: "sulfate permease activity" RELATED []
 synonym: "sulphate transporter activity" EXACT []
 xref: Reactome:R-HSA-3560789 "Defective SLC26A2 does not cotransport extracellular SO4(2-), H+ to cytosol"
-xref: Reactome:R-HSA-427555 "SLC26A1,2 cotransport SO4(2-), H+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-427555 "SLC26A1,2 cotransport SO4(2-), anion from extracellular region to cytosol"
 xref: Reactome:R-HSA-8875871 "SLC26A11 transports SO4(2-) from extracellular region to cytosol"
+xref: Reactome:R-HSA-9941470 "SLC26A11 imports SO4(2-) and H+ into cytosol"
 is_a: GO:1901682 ! sulfur compound transmembrane transporter activity
 relationship: part_of GO:1902358 ! sulfate transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -119510,6 +119863,7 @@
 name: glucuronate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of glucuronate from one side of a membrane to the other. Glucuronate is the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:ai]
+xref: Reactome:R-HSA-2162229 "SLC17A5 exports GlcA from the lysosome to the cytosol"
 is_a: GO:0015133 ! uronic acid transmembrane transporter activity
 relationship: part_of GO:0015738 ! glucuronate transmembrane transport
 
@@ -119739,6 +120093,8 @@
 name: polysaccharide transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of polysaccharides from one side of a membrane to the other. A polysaccharide is a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
+xref: Reactome:R-HSA-9932121 "Unknown transporter exports CA to extracellular space"
+xref: Reactome:R-HSA-9932466 "Bcs complex exports pEtN-cellulose"
 is_a: GO:0015144 ! carbohydrate transmembrane transporter activity
 is_a: GO:0022884 ! macromolecule transmembrane transporter activity
 relationship: part_of GO:0015774 ! polysaccharide transport
@@ -119842,7 +120198,6 @@
 synonym: "hydroxy/aromatic amino acid permease activity" RELATED []
 xref: Reactome:R-HSA-210439 "glutamate uptake by astrocytes"
 xref: Reactome:R-HSA-212642 "L-Glutamine transport into neurons"
-xref: Reactome:R-HSA-351963 "SLC6A18 transports Gly from extracellular region to cytosol"
 xref: Reactome:R-HSA-351987 "SLC6A6-mediated uptake of taurine and beta-alanine"
 xref: Reactome:R-HSA-352029 "SLC6A12 (BGT-1)-mediated uptake of GABA and betaine"
 xref: Reactome:R-HSA-352052 "SLC6A20 cotransports L-Pro, Na+ from the extracellular region to cytosol"
@@ -119856,7 +120211,6 @@
 xref: Reactome:R-HSA-352174 "SLC38A3-mediated uptake of glutamine, histidine, asparagine, and alanine"
 xref: Reactome:R-HSA-352182 "SLC38A5-mediated uptake of glutamine, histidine, asparagine, and serine"
 xref: Reactome:R-HSA-352191 "SLC7A8-mediated uptake of neutral amino acids"
-xref: Reactome:R-HSA-352232 "SLC7A5:SLC3A2 transports neutral amino acids from extracellular region to cytosol"
 xref: Reactome:R-HSA-352347 "SLC1A4-mediated exchange of extracellular serine for cytosolic alanine, threonine, or cysteine"
 xref: Reactome:R-HSA-352354 "SLC1A4-mediated exchange of extracellular cysteine for cytosolic alanine, serine, or threonine"
 xref: Reactome:R-HSA-352364 "SLC1A4-mediated exchange of extracellular alanine for cytosolic serine, threonine, or cysteine"
@@ -119880,7 +120234,6 @@
 xref: Reactome:R-HSA-5655702 "Defective SLC3A1 (in SLC7A9:SLC3A1) does not exchange L-Arg, CySS-, L-Lys for L-Leu"
 xref: Reactome:R-HSA-5659674 "Variant SLC6A14 cotransports SLC6A14 ligands, Cl-, 2Na+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5659734 "Defective SLC6A19 does not cotransport neutral amino acids, Na+ from extracellular region to cytosol"
-xref: Reactome:R-HSA-5659755 "Defective SLC6A18 does not transport Gly from extracellular region to cytosol"
 xref: Reactome:R-HSA-5660694 "Variant SLC6A20 does not cotransport L-Pro, Na+ from extracellulare region to cytosol"
 xref: Reactome:R-HSA-5660890 "Defective SLC7A9 (in SLC7A9:SLC3A1) does not exchange L-Arg, CySS-, L-Lys for L-Leu"
 xref: Reactome:R-HSA-5660910 "Defective SLC7A7 does not exchange L-Arg for L-Leu, Na+ across the plasma membrane"
@@ -119952,6 +120305,7 @@
 def: "Enables the transfer of an L-amino acid from one side of a membrane to the other. L-amino acids are the L-enantiomers of amino acids." [GOC:ai, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
 synonym: "L-amino acid transporter activity" BROAD []
 xref: Reactome:R-HSA-212614 "Glutamine transport from astrocytes"
+xref: Reactome:R-HSA-352232 "SLC7A5:SLC3A2 transports neutral amino acids from extracellular region to cytosol"
 is_a: GO:0015171 ! amino acid transmembrane transporter activity
 is_a: GO:0046943 ! carboxylic acid transmembrane transporter activity
 relationship: part_of GO:1902475 ! L-alpha-amino acid transmembrane transport
@@ -120123,7 +120477,6 @@
 synonym: "L-serine transporter activity" BROAD []
 synonym: "serine transporter activity" BROAD []
 synonym: "threonine/serine:sodium symporter activity" RELATED []
-xref: Reactome:R-HSA-8932980 "SERINC3,5,(1,2,4) transport L-Ser from cytosol to plasma membrane"
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
 relationship: part_of GO:0015825 ! L-serine transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -120306,6 +120659,7 @@
 name: pyrimidine nucleotide transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of a pyrimidine nucleotide, any compound consisting of a pyrimidine nucleoside esterified with (ortho)phosphate, from one side of a membrane to the other." [GOC:ai]
+xref: Reactome:R-HSA-9940804 "SLC35A1,4 import CDP-ribitol"
 is_a: GO:0015215 ! nucleotide transmembrane transporter activity
 relationship: part_of GO:0006864 ! pyrimidine nucleotide transport
 
@@ -120468,6 +120822,8 @@
 xref: Reactome:R-HSA-917870 "SLC46A1 transports hemes from extracellular region to cytosol"
 xref: Reactome:R-HSA-917892 "FLVCR1-1 transports heme from cytosol to extracellular region"
 xref: Reactome:R-HSA-9661408 "FLVCR1-2 transports heme from mitochondrial matrix to cytosol"
+xref: Reactome:R-HSA-9927224 "ChuA imports heme to periplasm"
+xref: Reactome:R-HSA-9927930 "DppBCDF transports heme to cytosol"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0015886 ! heme transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19472" xsd:anyURI
@@ -120558,7 +120914,6 @@
 def: "Enables the transfer of aminophospholipids from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP. Aminophospholipids contain phosphoric acid as a mono- or diester and an amino (NH2) group." [GOC:pg]
 synonym: "aminophospholipid transmembrane transporter activity" RELATED []
 synonym: "aminophospholipid transporter activity" RELATED []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140327 ! flippase activity
 relationship: part_of GO:0140331 ! aminophospholipid translocation
 property_value: skos:broadMatch RHEA:66132
@@ -120755,6 +121110,9 @@
 synonym: "substrate-specific channel activity" RELATED []
 xref: Reactome:R-HSA-3779381 "H2O2 diffuses from the mitochondrial matrix to the cytosol"
 xref: Reactome:R-HSA-8953430 "PXMP2 trimer transports glycolate from cytosol to peroxisomal matrix"
+xref: Reactome:R-HSA-9857649 "p-S373-GJA1:p-S-ITGA5:ITGB1 (p-S373-Connexin-43:p-S-Integrin alpha5:Integrin beta1) transports ATP from the cytosol to the extracellular region"
+xref: Reactome:R-HSA-9857682 "p-S373-GJA1:p-S-ITGA5:ITGB1 (p-S373-Connexin-43:p-S-Integrin alpha5:Integrin beta1) transports prostaglandin E2 (PGE2) from the cytosol to the extracellular region"
+xref: Reactome:R-HSA-9905999 "Pannexin1 (PANX1) channel in a complex with P2RX7 transports ATP from the cytosol to the extracellular region"
 xref: TC:1
 is_a: GO:0022803 ! passive transmembrane transporter activity
 
@@ -120897,6 +121255,11 @@
 def: "Enables the transfer of substances, sized less than 1000 Da, from one side of a membrane to the other. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820, TC:1.B.1.-.-]
 synonym: "outer membrane exporter porin" NARROW []
 synonym: "porin" EXACT []
+xref: Reactome:R-HSA-9760573 "TolC complex exports STa"
+xref: Reactome:R-HSA-9911510 "MacA:MacB:TolC transports azithromycin from the periplasm to the extracellular region"
+xref: Reactome:R-HSA-9911516 "AcrA:AcrB:TolC transports antibiotics from the periplasm to the extracellular region"
+xref: Reactome:R-HSA-9914083 "MdtE:MdtF:TolC exports beta-lactams to extracellular space"
+xref: Reactome:R-HSA-9926021 "HlyB:HlyD:TolC transports HlyA from periplasm to extracellular space"
 xref: Wikipedia:Porin_(protein)
 is_a: GO:0022829 ! wide pore channel activity
 
@@ -120934,6 +121297,18 @@
 synonym: "porter activity" EXACT []
 synonym: "porters" EXACT []
 synonym: "secondary carrier-type facilitators" EXACT []
+xref: Reactome:R-HSA-549241 "SLC22A4 cotransports ERGT, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-561059 "OAT2 and OAT4 mediate transport of sulphate conjugates"
+xref: Reactome:R-HSA-879562 "SLCO2B1 has a narrow substrate specificity"
+xref: Reactome:R-HSA-879584 "SLCO3A1 isoform 1 has a broad substrate specificity"
+xref: Reactome:R-HSA-879594 "SLCO4C1 mediates the transport of digoxin"
+xref: Reactome:R-HSA-9661397 "SLCO1B1 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9661446 "BMG, BDG translocates from ER lumen to cytosol"
+xref: Reactome:R-HSA-9661723 "SLCO2B1-3 transports BIL  from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9661799 "SLCO1B3 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9749583 "SLC22A7 transports ASA-,ST from extracellular region to cytosol of hepatocytes"
+xref: Reactome:R-HSA-9749607 "SLCO2B1-1 transports ASA- from extracellular region to cytosol of GI cells"
+xref: Reactome:R-HSA-9757010 "SLCO1B1,1B3,2B1-3 transport ATV from extracelluar region to cytosol"
 xref: TC:2.A
 is_a: GO:0022804 ! active transmembrane transporter activity
 
@@ -121444,6 +121819,11 @@
 synonym: "siderophore transporter activity" RELATED []
 synonym: "siderophore-iron transporter activity" RELATED []
 xref: Reactome:R-HSA-1222597 "Loaded mycobactin gets imported"
+xref: Reactome:R-HSA-9911605 "FyuA transports Ybt:Fe/Cu from extracellular space to the periplasm"
+xref: Reactome:R-HSA-9928256 "YbtPQ transports Ybt:Fe/Cu to the cytosol"
+xref: Reactome:R-HSA-9930902 "IutA imports FeAer to the periplasm"
+xref: Reactome:R-HSA-9930911 "FhuBCD imports FeAer to the cytosol"
+xref: Reactome:R-HSA-9931043 "FecA:TonB:ExbB:ExbD imports Fe3+-citrates to periplasm"
 xref: TC:2.A.1.16.1
 xref: TC:2.A.1.16.2
 xref: TC:2.A.1.16.3
@@ -121504,17 +121884,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other, in a sodium independent manner." [GOC:go_curators]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-561059 "OAT2 and OAT4 mediate transport of sulphate conjugates"
-xref: Reactome:R-HSA-879562 "SLCO2B1 has a narrow substrate specificity"
-xref: Reactome:R-HSA-879584 "SLCO3A1 isoform 1 has abroad substrate specificity"
-xref: Reactome:R-HSA-879594 "SLCO4C1 mediates the transport of digoxin"
-xref: Reactome:R-HSA-9661397 "SLCO1B1 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9661446 "BMG, BDG translocates from ER lumen to cytosol"
-xref: Reactome:R-HSA-9661723 "SLCO2B1-3 transports BIL  from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9661799 "SLCO1B3 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9749583 "SLC22A7 transports ASA-,ST from extracellular region to cytosol of hepatocytes"
-xref: Reactome:R-HSA-9749607 "SLCO2B1-1 transports ASA- from extracellular region to cytosol of GI cells"
-xref: Reactome:R-HSA-9757010 "SLCO1B1,1B3,2B1-3 transport ATV from extracelluar region to cytosol"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0015291
@@ -122852,7 +123221,7 @@
 synonym: "P-P-bond-hydrolysis-driven protein transmembrane transporter activity" RELATED []
 synonym: "protein translocase activity" EXACT []
 xref: Reactome:R-HSA-1222523 "SodB gets secreted"
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 is_a: GO:0042626 ! ATPase-coupled transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21271" xsd:anyURI
 
@@ -124154,6 +124523,7 @@
 name: iron chelate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of an iron chelate from one side of a membrane to the other. An iron chelate is a heterocyclic compound having a metal ion attached by coordinate bonds to at least two nonmetal ions." [PMID:17660286]
+xref: Reactome:R-HSA-9931048 "FecBCDE imports Fe3+-citrates to cytosol"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0033214 ! siderophore-iron import into cell
 
@@ -124317,6 +124687,8 @@
 name: ferric-enterobactin transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of ferric-enterobactin from one side of a membrane to the other." [GOC:ai]
+xref: Reactome:R-HSA-9911602 "FepA:TonB:ExbB:ExbD transports ferrienterobactin from the extracellular region to the periplasmic space"
+xref: Reactome:R-HSA-9911603 "FepC:FepD:FepG transports ferrienterobactin from the periplasmic space to the cytosol"
 is_a: GO:0015343 ! siderophore-iron transmembrane transporter activity
 relationship: part_of GO:0015685 ! ferric-enterobactin import into cell
 
@@ -124724,6 +125096,12 @@
 name: alcohol transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of an alcohol from one side of a membrane to the other. An alcohol is any carbon compound that contains a hydroxyl group." [ISBN:0198506732]
+xref: Reactome:R-HSA-192010 "5beta-cholestan-3alpha, 7alpha, 12alpha-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193537 "5beta-cholestan-3alpha, 7alpha-diol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193715 "5beta-cholestan-3alpha,7alpha,24(S)-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193774 "5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193808 "5beta-cholestan-3alpha,7alpha,27-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193832 "5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol is translocated from the cytosol to the mitochondrial matrix"
 is_a: GO:0022857 ! transmembrane transporter activity
 
 [Term]
@@ -125066,6 +125444,7 @@
 name: organic anion transport
 namespace: biological_process
 def: "The directed movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. Organic anions are atoms or small molecules with a negative charge which contain carbon in covalent linkage." [GOC:ai, GOC:krc]
+xref: Reactome:R-HSA-9955298 "SLC-mediated transport of organic anions"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -125154,6 +125533,7 @@
 synonym: "bile acid transport" NARROW []
 synonym: "bile salt transport" NARROW []
 xref: Reactome:R-HSA-159418 "Recycling of bile acids and salts"
+xref: Reactome:R-HSA-9958517 "SLC-mediated bile acid transport"
 is_a: GO:0006869 ! lipid transport
 is_a: GO:0015718 ! monocarboxylic acid transport
 is_a: GO:0015850 ! organic hydroxy compound transport
@@ -126105,6 +126485,7 @@
 def: "The directed movement of amines, including polyamines, organic compounds containing one or more amino groups, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:ai, ISBN:0198506732]
 subset: goslim_pir
 synonym: "amine/polyamine transport" RELATED []
+xref: Reactome:R-HSA-549127 "SLC-mediated transport of organic cations"
 is_a: GO:0071705 ! nitrogen compound transport
 
 [Term]
@@ -127919,8 +128300,7 @@
 id: GO:0016040
 name: glutamate synthase (NADH) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + NAD+ = 2-oxoglutarate + L-glutamine + H+ + NADH." [EC:1.4.1.14, RHEA:13753]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + NAD+ = 2-oxoglutarate + L-glutamine + H+ + NADH." [RHEA:13753]
 synonym: "glutamate (reduced nicotinamide adenine dinucleotide) synthase" RELATED [EC:1.4.1.14]
 synonym: "GOGAT activity" BROAD []
 synonym: "L-glutamate synthase (NADH)" RELATED [EC:1.4.1.14]
@@ -127938,15 +128318,14 @@
 property_value: skos:exactMatch EC:1.4.1.14
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-SYNTHASE-NADH-RXN
 property_value: skos:exactMatch RHEA:13753
-property_value: skos:relatedMatch MetaCyc:GLUGLNSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0016041
 name: glutamate synthase (ferredoxin) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + 2 oxidized ferredoxin + H2O = NH3 + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+." [EC:1.4.7.1]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+. This is a two-step reaction: (a) L-glutamate + NH4+ = L-glutamine + H2O, (b) L-glutamate + 2 oxidized ferredoxin + H2O = NH4+ + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+." [EC:1.4.7.1]
 synonym: "ferredoxin-dependent glutamate synthase activity" RELATED [EC:1.4.7.1]
 synonym: "ferredoxin-glutamate synthase activity" RELATED [EC:1.4.7.1]
 synonym: "glutamate synthase (ferredoxin-dependent)" RELATED [EC:1.4.7.1]
@@ -127959,8 +128338,8 @@
 property_value: skos:exactMatch EC:1.4.7.1
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-SYNTHASE-FERREDOXIN-RXN
 property_value: skos:exactMatch RHEA:12128
-property_value: skos:relatedMatch MetaCyc:PWY-4341
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0016042
@@ -129188,7 +129567,7 @@
 xref: EC:4.2.1.49
 xref: KEGG_REACTION:R02914
 xref: MetaCyc:UROCANATE-HYDRATASE-RXN
-xref: Reactome:R-HSA-70903 "urocanate + H2O => 4-imidazolone-5-propionate"
+xref: Reactome:R-HSA-70903 "urocanate + H2O => 4-imidazolone-5-propanoate"
 xref: RHEA:13101
 is_a: GO:0016836 ! hydro-lyase activity
 property_value: skos:exactMatch EC:4.2.1.49
@@ -129278,6 +129657,7 @@
 synonym: "phytate 3-phosphatase activity" EXACT []
 xref: EC:3.1.3.8
 xref: MetaCyc:RXN0-1001
+xref: Reactome:R-HSA-1855225 "IP6 is dephosphorylated to I(1,2,4,5,6)P5 by MINPP1 in the ER lumen"
 xref: RHEA:16989
 is_a: GO:0004446 ! inositol hexakisphosphate phosphatase activity
 property_value: skos:broadMatch EC:3.1.3.62
@@ -129487,7 +129867,7 @@
 id: GO:0016174
 name: NAD(P)H oxidase H2O2-forming activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + hydrogen peroxide." [EC:1.6.3.1, PMID:10401672, PMID:10601291, PMID:11822874]
+def: "Catalysis of the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + H2O2." [EC:1.6.3.1, PMID:10401672, PMID:10601291, PMID:11822874]
 synonym: "dual oxidase activity" RELATED [EC:1.6.3.1]
 synonym: "NAD(P)H oxidase activity" BROAD []
 synonym: "NAD(P)H:oxygen oxidoreductase activity" RELATED [EC:1.6.3.1]
@@ -129499,7 +129879,7 @@
 synonym: "thyroid oxidase 2 activity" NARROW [EC:1.6.3.1]
 synonym: "thyroid oxidase activity" NARROW [EC:1.6.3.1]
 xref: EC:1.6.3.1
-xref: Reactome:R-HSA-5693681 "DUOX1,2 reduce O2 to H2O2"
+xref: Reactome:R-HSA-5693681 "DUOX1 reduces O2 to H2O2"
 xref: Reactome:R-HSA-9698758 "FLT3 ITD- and NOX4-dependent H2O2 production"
 xref: RHEA:11260
 xref: RHEA:11264
@@ -129815,6 +130195,8 @@
 namespace: molecular_function
 def: "Binding to AMP, adenosine monophosphate." [GOC:go_curators]
 xref: Reactome:R-HSA-380930 "Phosphorylated AMPK binds AMP"
+xref: Reactome:R-HSA-453342 "PYGM dimer, b form + 2 AMP => PYGM b dimer:AMP complex"
+xref: Reactome:R-HSA-453356 "PYGB dimer, b form + 2 AMP => PYGB b dimer:AMP complex"
 is_a: GO:0032559 ! adenyl ribonucleotide binding
 is_a: GO:0043168 ! anion binding
 is_a: GO:0043169 ! cation binding
@@ -129856,13 +130238,13 @@
 id: GO:0016211
 name: ammonia ligase activity
 namespace: molecular_function
-def: "Catalysis of the ligation of ammonia (NH3) to another substance via a carbon-nitrogen bond with concomitant breakage of a diphosphate linkage, usually in a nucleoside triphosphate." [GOC:jl]
+def: "Catalysis of the ligation of ammonia (NH4+) to another substance via a carbon-nitrogen bond with concomitant breakage of a diphosphate linkage, usually in a nucleoside triphosphate." [GOC:jl]
 is_a: GO:0016880 ! acid-ammonia (or amide) ligase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28243" xsd:anyURI
 
 [Term]
 id: GO:0016212
-name: kynurenine-oxoglutarate transaminase activity
+name: L-kynurenine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2-oxoglutarate + L-kynurenine = H2O + kynurenate + L-glutamate. The product 4-(2-aminophenyl)-2,4-dioxobutanoate is converted into kynurenate by a spontaneous reaction. Also acts on 3-hydroxykynurenine to form xanthurenate." [EC:2.6.1.7]
 synonym: "kynurenine 2-oxoglutarate transaminase activity" RELATED [EC:2.6.1.7]
@@ -129870,14 +130252,15 @@
 synonym: "kynurenine transaminase (cyclizing)" RELATED [EC:2.6.1.7]
 synonym: "kynurenine--oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.7]
 synonym: "kynurenine-oxoglutarate aminotransferase activity" EXACT []
+synonym: "kynurenine-oxoglutarate transaminase activity" EXACT []
 synonym: "L-kynurenine aminotransferase activity" RELATED [EC:2.6.1.7]
 synonym: "L-kynurenine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.7]
 xref: EC:2.6.1.7
 xref: MetaCyc:RXN-21858
 xref: MetaCyc:RXN-21859
 xref: Reactome:R-HSA-893583 "kynurenine + 2-oxoglutarate => 4-(2-aminophenyl)-2,4-dioxobutanoic acid + glutamate"
-xref: Reactome:R-HSA-893596 "PXLP-KYAT1 dimer transaminates L-KYN to AP-DOBu"
-xref: Reactome:R-HSA-901097 "kynurenine + pyruvate => 4-(2-aminophenyl)-2,4-dioxobutanoic acid + alanine [CCBL2]"
+xref: Reactome:R-HSA-893596 "PXLP-KYAT1 dimer transaminates L-KYN to KYNA"
+xref: Reactome:R-HSA-901097 "PXLP-KYAT3 (CCBL2) dimer transaminates L-KYN to KYNA"
 xref: RHEA:65560
 xref: RHEA:65884
 is_a: GO:0008483 ! transaminase activity
@@ -129887,6 +130270,7 @@
 property_value: skos:narrowMatch RHEA:65560
 property_value: skos:narrowMatch RHEA:65884
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25811" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0016213
@@ -129955,11 +130339,12 @@
 
 [Term]
 id: GO:0016217
-name: N-ethylammeline chlorohydrolase activity
+name: obsolete N-ethylammeline chlorohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: deethylsimazine + H2O = N-ethylammeline + chloride + H+." [MetaCyc:R465-RXN]
-xref: MetaCyc:R465-RXN
-is_a: GO:0019120 ! hydrolase activity, acting on halide bonds, in C-halide compounds
+def: "OBSOLETE. Catalysis of the reaction: deethylsimazine + H2O = N-ethylammeline + chloride + H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0016218
@@ -129987,7 +130372,7 @@
 id: GO:0016223
 name: beta-alanine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + 2-oxopropanoate = pyruvate + beta-alanine." [RHEA:14077]
+def: "Catalysis of the reaction: L-alanine + 3-oxopropanoate = beta-alanine + pyruvate." [RHEA:14077]
 synonym: "beta-alanine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.18]
 synonym: "beta-alanine-alpha-alanine transaminase activity" RELATED [EC:2.6.1.18]
 synonym: "beta-alanine-pyruvate aminotransferase activity" EXACT []
@@ -129997,7 +130382,7 @@
 xref: MetaCyc:2.6.1.18-RXN
 xref: Reactome:R-HSA-909776 "beta-alanine + pyruvate => 3-oxopropanoate + alanine"
 xref: RHEA:14077
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.18
 property_value: skos:exactMatch RHEA:14077
 
@@ -130064,6 +130449,7 @@
 def: "Catalysis of the synthesis of the HKK-1 carbohydrate epitope; adds a sulfate group to a precursor, GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R), forming sulfo-3GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R)." [PMID:9478973]
 synonym: "HNK-1 sulphotransferase activity" EXACT []
 xref: Reactome:R-HSA-6786048 "CHST10 transfers SO4(2-) from PAPS to GlcA-LacN on NCAM1"
+xref: Reactome:R-HSA-9940718 "CHST10 sulfates matriglycan chain"
 is_a: GO:0008146 ! sulfotransferase activity
 
 [Term]
@@ -131514,7 +131900,6 @@
 xref: Reactome:R-HSA-159443 "benzoate + Coenzyme A + ATP => benzoyl-CoA + AMP + pyrophosphate"
 xref: Reactome:R-HSA-159567 "ACSM2B-like proteins transform ST to ST-CoA"
 xref: Reactome:R-HSA-177157 "phenylacetate + Coenzyme A + ATP => phenylacetyl-CoA + AMP + pyrophosphate"
-xref: Reactome:R-HSA-6798345 "PXLP-GCAT dimer ligates CoASH to 2A-3OB to form Gly and Ac-CoA"
 xref: Reactome:R-HSA-9749971 "ACSM2B-like proteins transform 2,5-DHBA to 2,5-DHB-CoA"
 is_a: GO:0016877 ! ligase activity, forming carbon-sulfur bonds
 
@@ -131523,6 +131908,7 @@
 name: carnitine O-acyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an acyl group to an oxygen atom on the carnitine molecule." [GOC:ai]
+xref: Reactome:R-HSA-200410 "CPT2 converts acylcarnitine to acyl-CoA"
 is_a: GO:0008374 ! O-acyltransferase activity
 
 [Term]
@@ -131713,12 +132099,13 @@
 
 [Term]
 id: GO:0016428
-name: tRNA (cytidine-5-)-methyltransferase activity
+name: tRNA (cytidine-N5)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = a 5-methylcytidine in tRNA + S-adenosyl-L-homocysteine + H+. This modification can occur on seversal residues, including cytidine(34), cytidine(40), cytidine(48), and cytidine(49)." [RHEA:61468]
+def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = a 5-methylcytidine in tRNA + S-adenosyl-L-homocysteine + H+. This modification can occur on several residues, including cytidine(34), cytidine(40), cytidine(48), and cytidine(49)." [RHEA:61468]
 synonym: "S-adenosyl-L-methionine:tRNA (cytosine-5-)-methyltransferase activity" RELATED []
 synonym: "transfer ribonucleate cytosine 5-methyltransferase activity" RELATED []
 synonym: "transfer RNA cytosine 5-methyltransferase activity" RELATED []
+synonym: "tRNA (cytidine-5-)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine-5-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6782388 "NSUN2 methylates cytidine-34, cytidine-48 of unspliced tRNA(Leu)(CAA)"
 xref: Reactome:R-HSA-6782419 "TRDMT1 (DNMT2) methylates cytidine-38 of tRNA(Asp)"
@@ -131730,6 +132117,7 @@
 property_value: skos:exactMatch RHEA:61468
 property_value: skos:relatedMatch EC:2.1.1.202
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24771" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -131754,20 +132142,15 @@
 
 [Term]
 id: GO:0016430
-name: tRNA (adenine-N6)-methyltransferase activity
+name: obsolete tRNA (adenine-N6)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N6-methyladenine." [EC:2.1.1.55]
+def: "OBSOLETE. Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N6-methyladenine." [EC:2.1.1.55]
+comment: This term was obsoleted because there is no evidence that this reaction exists, see PMID:33658722.
 synonym: "S-adenosyl-L-methionine:tRNA (adenine-6-N-)-methyltransferase activity" RELATED [EC:2.1.1.55]
 synonym: "S-adenosyl-L-methionine:tRNA (adenine-N6-)-methyltransferase activity" RELATED [EC:2.1.1.55]
 synonym: "tRNA (adenine-N6-)-methyltransferase activity" EXACT []
-xref: EC:2.1.1.55
-xref: MetaCyc:TRNA-ADENINE-N6--METHYLTRANSFERASE-RXN
-xref: RHEA:16785
-is_a: GO:0008170 ! N-methyltransferase activity
-is_a: GO:0016426 ! tRNA (adenine) methyltransferase activity
-property_value: skos:exactMatch EC:2.1.1.55
-property_value: skos:exactMatch RHEA:16785
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0016432
@@ -132216,6 +132599,7 @@
 synonym: "protein maturation by peptide bond cleavage" EXACT [GOC:bf]
 synonym: "protein maturation by peptide bond hydrolysis" EXACT [GOC:mah]
 synonym: "protein maturation by proteolysis" RELATED [GOC:mah]
+xref: Reactome:R-HSA-159782 "Removal of aminoterminal propeptides from gamma-carboxylated proteins"
 xref: Reactome:R-HSA-9033500 "TYSND1 cleaves peroxisomal proteins"
 is_a: GO:0006508 ! proteolysis
 is_a: GO:0051604 ! protein maturation
@@ -132280,15 +132664,11 @@
 synonym: "redox activity" EXACT []
 xref: EC:1.-.-.-
 xref: Reactome:R-HSA-1614362 "SUMF1 mediates the oxidation of cysteine to formylglycine, producing active arylsulfatases"
-xref: Reactome:R-HSA-209921 "Monoiodinated tyrosine can be deiodinated"
-xref: Reactome:R-HSA-209960 "Diiodinated tyrosine can be deiodinated"
 xref: Reactome:R-HSA-2161612 "PGH2 is reduced to PGF2a by FAM213B"
 xref: Reactome:R-HSA-265296 "Arachidonate is oxidised to 5S-HpETE by ALOX5"
 xref: Reactome:R-HSA-266051 "5S-HpETE is dehydrated to LTA4 by ALOX5"
 xref: Reactome:R-HSA-390425 "FAR1 reduces PalmCoA to HXOL"
 xref: Reactome:R-HSA-390438 "FAR2 reduces PalmCoA to HXOL"
-xref: Reactome:R-HSA-5662660 "Dopachrome is transformed to DHICA by DCT"
-xref: Reactome:R-HSA-8878581 "TYRP1 oxidises DHICA to IQCA"
 xref: Reactome:R-HSA-8936442 "MARC1,MARC2 reduce N-hydroxylated compounds"
 xref: Reactome:R-HSA-9020249 "Hydroperoxy reductase reduces 4(S)-Hp-17(S)-HDHA to RvD6"
 xref: Reactome:R-HSA-9020260 "Hydroperoxy reductase reduces 7(S)-Hp-17(S)-HDHA to RvD5"
@@ -132839,6 +133219,7 @@
 comment: The term 'RNA editing' (GO:0016547) was merged into 'RNA modification' (GO:0009451) on the basis of statements in the preface of Modification and Editing of RNA (ISBN:1555811337) that there is no clear distinction between modification and editing. Parallel changes were made for substrate (e.g. tRNA, rRNA, etc.) specific child terms of 'RNA editing'.
 synonym: "mRNA editing" NARROW [GOC:dph, GOC:hjd, GOC:tb]
 xref: Reactome:R-HSA-75072 "mRNA Editing"
+xref: Reactome:R-HSA-9937008 "Mitochondrial mRNA modification"
 is_a: GO:0009451 ! RNA modification
 is_a: GO:0016071 ! mRNA metabolic process
 
@@ -133572,7 +133953,6 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [EC:1.3.-.-]
 synonym: "oxidoreductase activity, acting on the CH-CH group of donors, other acceptors" NARROW []
 xref: EC:1.3.-.-
-xref: Reactome:R-HSA-2995334 "COX15 transforms heme O to heme A"
 xref: Reactome:R-HSA-9661710 "An unknown oxidase oxidises D-UBGN to UBN"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.3.-.-
@@ -133584,7 +133964,6 @@
 namespace: molecular_function
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces NAD or NADP." [EC:1.3.1.-]
 xref: EC:1.3.1.-
-xref: Reactome:R-HSA-4419979 "SRD5A3 reduces pPNOL to DCHOL"
 xref: Reactome:R-HSA-4755572 "Defective SRD5A3 does not reduce pPNOL to DCHOL"
 xref: Reactome:R-HSA-9661726 "An unknown reductase reduces D-UBGN to STBN"
 xref: Reactome:R-HSA-9661745 "An unknown BILR reduces BIL to D-UBGN"
@@ -133837,6 +134216,7 @@
 namespace: molecular_function
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces quinone or similar compound." [GOC:jl]
 xref: EC:1.5.5.-
+xref: Reactome:R-HSA-70670 "PRODH oxidises L-Pro to 1PYR-5COOH"
 is_a: GO:0016645 ! oxidoreductase activity, acting on the CH-NH group of donors
 property_value: skos:exactMatch EC:1.5.5.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134067,6 +134447,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces disulfide." [GOC:jl]
 synonym: "oxidoreductase activity, acting on sulphur group of donors, disulphide as acceptor" EXACT []
 xref: EC:1.8.4.-
+xref: Reactome:R-HSA-9035484 "MPST and TXN2 enable the synthesis of disulfanide from CysS248-MPST"
 is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
 property_value: skos:exactMatch EC:1.8.4.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134112,7 +134493,6 @@
 synonym: "oxidoreductase activity, acting on haem group of donors" EXACT []
 synonym: "oxidoreductase activity, acting on heme group of donors, other acceptors" NARROW []
 xref: EC:1.9.-.-
-xref: Reactome:R-HSA-5340226 "CYGB dioxygenates NO"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.9.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134213,7 +134593,7 @@
 id: GO:0016688
 name: L-ascorbate peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-ascorbate + hydrogen peroxide = dehydroascorbate + 2 H2O." [EC:1.11.1.11]
+def: "Catalysis of the reaction: L-ascorbate + H2O2 = dehydroascorbate + 2 H2O." [EC:1.11.1.11]
 synonym: "ascorbate peroxidase activity" RELATED [EC:1.11.1.11]
 synonym: "ascorbic acid peroxidase activity" RELATED [EC:1.11.1.11]
 synonym: "L-ascorbate:hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.11]
@@ -134231,7 +134611,7 @@
 id: GO:0016689
 name: manganese peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 Mn2+ + 2 H+ + hydrogen peroxide = 2 Mn3+ + 2 H2O." [EC:1.11.1.13]
+def: "Catalysis of the reaction: 2 Mn2+ + 2 H+ + H2O2 = 2 Mn3+ + 2 H2O." [EC:1.11.1.13]
 synonym: "Mn(II):hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.13]
 synonym: "Mn-dependent (NADH-oxidizing) peroxidase activity" RELATED [EC:1.11.1.13]
 synonym: "Mn-dependent peroxidase activity" RELATED [EC:1.11.1.13]
@@ -134461,6 +134841,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from each of two donors, and molecular oxygen is reduced or incorporated into a donor." [EC:1.14.-.-]
 synonym: "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, miscellaneous" NARROW []
 xref: EC:1.14.-.-
+xref: Reactome:R-HSA-5333612 "MeSebGalNac is oxidatively cleaved to MeSeO2H and 2-acetamidoglucal"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.14.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134598,7 +134979,7 @@
 xref: EC:1.14.14.-
 xref: EC:1.14.14.1
 xref: MetaCyc:UNSPECIFIC-MONOOXYGENASE-RXN
-xref: Reactome:R-HSA-194678 "CYP51A1 demethylates LNSOL"
+xref: Reactome:R-HSA-194678 "CYP51A1 demethylates LAN"
 xref: Reactome:R-HSA-211966 "CYP2D6 4-hydroxylates debrisoquine"
 xref: RHEA:17149
 is_a: GO:0004497 ! monooxygenase activity
@@ -134929,6 +135310,7 @@
 xref: EC:2.-.-.-
 xref: Reactome:R-HSA-1483089 "PE is converted to PS by PTDSS2"
 xref: Reactome:R-HSA-1483186 "PC is converted to PS by PTDSS1"
+xref: Reactome:R-HSA-5333607 "GSSeH combines with bGalNAc derivative to form GSSebGalNac"
 xref: Reactome:R-HSA-5668414 "TRAF2 ubiquitinates cIAP1,2 in cIAP1,2:TRAF1:TRAF2:TRAF3:NIK"
 xref: Reactome:R-HSA-8868783 "TSR3 transfers aminocarboxypropyl group from S-adenosylmethionine to N(1)-methylpseudouridine-1248 of 18SE rRNA yielding N(1)-methyl-N(3)-aminocarboxypropylpseudouridine-1248"
 is_a: GO:0003824 ! catalytic activity
@@ -135168,6 +135550,7 @@
 synonym: "uridine diphosphoglucose-cellulose glucosyltransferase activity" RELATED [EC:2.4.1.12]
 xref: EC:2.4.1.12
 xref: MetaCyc:CELLULOSE-SYNTHASE-UDP-FORMING-RXN
+xref: Reactome:R-HSA-9932210 "Bcs complex synthesizes cellulose from UDP-Glc"
 xref: RHEA:19929
 is_a: GO:0016759 ! cellulose synthase activity
 is_a: GO:0035251 ! UDP-glucosyltransferase activity
@@ -135309,6 +135692,7 @@
 comment: Note that this term encompasses all kinase activities, as well as activities that transfer other phosphorus-containing groups such as diphosphate or nucleotides.
 subset: goslim_chembl
 xref: EC:2.7.-.-
+xref: Reactome:R-HSA-9932322 "BcsG transfers pEtN from PE onto cellulose"
 is_a: GO:0016740 ! transferase activity
 property_value: skos:exactMatch EC:2.7.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135321,6 +135705,7 @@
 subset: goslim_drosophila
 xref: EC:2.7.1.-
 xref: Reactome:R-HSA-2161193 "abacavir + AMP => abacavir monophosphate + adenosine"
+xref: Reactome:R-HSA-9941039 "FAM20B phosphorylates Xyl moiety"
 is_a: GO:0016772 ! transferase activity, transferring phosphorus-containing groups
 property_value: skos:exactMatch EC:2.7.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135389,6 +135774,8 @@
 xref: EC:2.7.8.-
 xref: Reactome:R-HSA-162742 "(ethanolamineP) mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI -> (ethanolamineP) mannose (a1-2) (ethanolamineP) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI"
 xref: Reactome:R-HSA-163069 "mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI + phosphatidylethanolamine -> (ethanolamineP) mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI (acyl-GPI) + diacylglycerol"
+xref: Reactome:R-HSA-9940735 "FKTN transfers RboP to GalNAc-GlcNAc-ManP-DAG1"
+xref: Reactome:R-HSA-9940756 "FKRP dimer transfers RboP to matriglycan chain"
 is_a: GO:0016772 ! transferase activity, transferring phosphorus-containing groups
 property_value: skos:exactMatch EC:2.7.8.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135423,6 +135810,10 @@
 xref: EC:2.8.1.-
 xref: Reactome:R-HSA-1614618 "Persulfide sulfur is transferred onto sulfite"
 xref: Reactome:R-HSA-6782264 "CTU1:CTU2:URM1 thiolates uridine-34 in tRNAs"
+xref: Reactome:R-HSA-9012721 "MPST transfers sulfur atom from 3MPYR to HSO3- to form S2O3(2-) and PYR"
+xref: Reactome:R-HSA-9013471 "MPST transfers sulfur from 3MPYR to HCN to form HSCN"
+xref: Reactome:R-HSA-9013533 "MPST transfers sulfur from sulfanegen to HCN to form HSCN"
+xref: Reactome:R-HSA-9034756 "MPST transfers sulfur atom from 3MPYR to form CysS248-MPST"
 xref: Reactome:R-HSA-947538 "Transfer of sulfur from MOCS3-S-S onto MOCS2A"
 is_a: GO:0016782 ! transferase activity, transferring sulphur-containing groups
 property_value: skos:exactMatch EC:2.8.1.-
@@ -135439,10 +135830,6 @@
 synonym: "mercaptopyruvate sulfurtransferase activity" EXACT []
 xref: EC:2.8.1.2
 xref: MetaCyc:MERCAPYSTRANS-RXN
-xref: Reactome:R-HSA-9012721 "MPST transfers sulfur atom from 3MPYR to HSO3- to form S2O3(2-) and PYR"
-xref: Reactome:R-HSA-9013471 "MPST transfers sulfur from 3MPYR to HCN to form HSCN"
-xref: Reactome:R-HSA-9013533 "MPST transfers sulfur from sulfanegen to HCN to form HSCN"
-xref: Reactome:R-HSA-9034756 "MPST transfers sulfur atom from 3MPYR to form CysS248-MPST"
 xref: RHEA:21740
 is_a: GO:0016783 ! sulfurtransferase activity
 property_value: skos:exactMatch EC:2.8.1.2
@@ -135486,6 +135873,7 @@
 xref: Reactome:R-HSA-6788295 "HDHD1:Mg2+ dephosphorylates PURIDP"
 xref: Reactome:R-HSA-8938314 "ENPPs hydrolyse CoA-SH to PPANT, PAP"
 xref: Reactome:R-HSA-8952137 "Phospholipid phosphatase 6 hydrolyses Presqualene diphosphate to presqualene monophosphate"
+xref: Reactome:R-HSA-9942295 "QNG1 hydrolyzes queuosine 5'-monophosphate yielding queuine and ribose 5'-monophosphate"
 is_a: GO:0003824 ! catalytic activity
 property_value: skos:exactMatch EC:3.-.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135535,6 +135923,7 @@
 xref: Reactome:R-HSA-4419986 "Unknown pPPP phosphatase dephosphorylates pPPP to pPNOL"
 xref: Reactome:R-HSA-9636457 "SapM dephosphorylates PI3P"
 xref: Reactome:R-HSA-9837419 "PANK4 hydrolyzes PPANT to pantetheine"
+xref: Reactome:R-HSA-9940993 "PXYLP1 dephosphorylates Xyl moiety"
 is_a: GO:0042578 ! phosphoric ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.3.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25755" xsd:anyURI
@@ -135608,6 +135997,7 @@
 xref: EC:3.2.-.-
 xref: Reactome:R-HSA-1793176 "DS is cleaved from its proteoglycan"
 xref: Reactome:R-HSA-2065233 "CS is cleaved from its proteoglycan"
+xref: Reactome:R-HSA-5333609 "MeSebGalNac is hydrolysed to MeSeH and bGalNac"
 is_a: GO:0016787 ! hydrolase activity
 property_value: skos:exactMatch EC:3.2.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135763,7 +136153,7 @@
 namespace: molecular_function
 def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a cyclic amide." [EC:3.5.2.-]
 xref: EC:3.5.2.-
-xref: Reactome:R-HSA-70906 "4-imidazolone-5-propionate + H2O => N-formiminoglutamate + 2H+"
+xref: Reactome:R-HSA-70906 "4-imidazolone-5-propanoate + H2O => N-formiminoglutamate"
 is_a: GO:0016810 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
 property_value: skos:exactMatch EC:3.5.2.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135976,6 +136366,7 @@
 xref: Reactome:R-HSA-6787757 "PXLP-K333-GADL1 decarboxylates acidic AAs"
 xref: Reactome:R-HSA-6814165 "PXLP-K333-GADL1 decarboxylates CSA to HTAU"
 xref: Reactome:R-HSA-71223 "2-amino-3-carboxymuconate semialdehyde => 2-aminomuconate semialdehyde + CO2"
+xref: Reactome:R-HSA-9856737 "Unknown enzyme(s) process 4-HMA to 4-Hbz"
 is_a: GO:0016830 ! carbon-carbon lyase activity
 property_value: skos:exactMatch EC:4.1.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136035,7 +136426,6 @@
 def: "Catalysis of the cleavage of a carbon-oxygen bond by elimination of water." [EC:4.2.1.-]
 xref: EC:4.2.1.-
 xref: Reactome:R-HSA-2161659 "PGE2 is dehydrated to PGA2"
-xref: Reactome:R-HSA-9014627 "SDS dimers:PXLP dehydrate L-Thr to 2AA"
 is_a: GO:0016835 ! carbon-oxygen lyase activity
 property_value: skos:exactMatch EC:4.2.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136075,7 +136465,7 @@
 id: GO:0016840
 name: carbon-nitrogen lyase activity
 namespace: molecular_function
-def: "Catalysis of the release of ammonia or one of its derivatives, with the formation of a double bond or ring. Enzymes with this activity may catalyze the actual elimination of the ammonia, amine or amide, e.g. CH-CH(-NH-R) = C=CH- + NH2-R. Others, however, catalyze elimination of another component, e.g. water, which is followed by spontaneous reactions that lead to breakage of the C-N bond, e.g. L-serine ammonia-lyase (EC:4.3.1.17), so that the overall reaction is C(-OH)-CH(-NH2) = CH2-CO- + NH3, i.e. an elimination with rearrangement. The sub-subclasses of EC:4.3 are the ammonia-lyases (EC:4.3.1), lyases acting on amides, amidines, etc. (EC:4.3.2), the amine-lyases (EC:4.3.3), and other carbon-nitrogen lyases (EC:4.3.99)." [EC:4.3.-.-]
+def: "Catalysis of the release of ammonia or one of its derivatives, with the formation of a double bond or ring. Enzymes with this activity may catalyze the actual elimination of the ammonia, amine or amide, e.g. CH-CH(-NH-R) = C=CH- + NH2-R. Others, however, catalyze elimination of another component, e.g. water, which is followed by spontaneous reactions that lead to breakage of the C-N bond, e.g. L-serine ammonia-lyase (EC:4.3.1.17), so that the overall reaction is C(-OH)-CH(-NH2) = CH2-CO- + NH4+, i.e. an elimination with rearrangement. The sub-subclasses of EC:4.3 are the ammonia-lyases (EC:4.3.1), lyases acting on amides, amidines, etc. (EC:4.3.2), the amine-lyases (EC:4.3.3), and other carbon-nitrogen lyases (EC:4.3.99)." [EC:4.3.-.-]
 synonym: "other carbon-nitrogen lyase activity" NARROW []
 xref: EC:4.3.-.-
 is_a: GO:0016829 ! lyase activity
@@ -136152,6 +136542,8 @@
 synonym: "carbon-sulphur lyase activity" EXACT []
 xref: EC:4.4.-.-
 xref: Reactome:R-HSA-1614567 "Excess homocysteine yields homolanthionine and H2S"
+xref: Reactome:R-HSA-1614591 "Excess cysteine yields lanthionine and H2S"
+xref: Reactome:R-HSA-1614614 "Cysteine is degraded to pyruvate and H2S"
 is_a: GO:0016829 ! lyase activity
 property_value: skos:exactMatch EC:4.4.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136272,7 +136664,7 @@
 xref: EC:5.-.-.-
 xref: Reactome:R-HSA-2161666 "PGA2 is isomerised to PGC2"
 xref: Reactome:R-HSA-2161735 "PGC2 is isomerised to PGB2"
-xref: Reactome:R-HSA-6787623 "TSTA3 dimer epimerises GDP-DHDMan to GDP-KDGal"
+xref: Reactome:R-HSA-6787623 "GFUS dimer epimerises GDP-DHDMan to GDP-KDGal"
 is_a: GO:0003824 ! catalytic activity
 property_value: skos:exactMatch EC:5.-.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136382,6 +136774,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor are the same molecule, one or more carbon-carbon double bonds in the molecule are rearranged, and no oxidized product appears." [EC:5.3.3.-, GOC:mah]
 synonym: "intramolecular isomerase activity, transposing C=C bonds" EXACT []
 xref: EC:5.3.3.-
+xref: Reactome:R-HSA-2161563 "PGJ2 is isomerised to delta12-PGJ2"
 is_a: GO:0016860 ! intramolecular oxidoreductase activity
 property_value: skos:exactMatch EC:5.3.3.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136576,7 +136969,7 @@
 id: GO:0016880
 name: acid-ammonia (or amide) ligase activity
 namespace: molecular_function
-def: "Catalysis of the ligation of an acid to ammonia (NH3) or an amide via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
+def: "Catalysis of the ligation of an acid to ammonia (NH4+) or an amide via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
 synonym: "amide synthase activity" EXACT []
 xref: EC:6.3.1.-
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
@@ -136681,6 +137074,9 @@
 xref: Reactome:R-HSA-9668415 "VPS4 mediates disassembly of ESCRTIII subunits to promote sealing of holes in the nuclear envelope"
 xref: Reactome:R-HSA-9706399 "RHOBTB3 hydrolyzes ATP"
 xref: Reactome:R-HSA-9749350 "ATP-dependent release of CDT1 from the OCCM complex"
+xref: Reactome:R-HSA-9948300 "RQT complex:K63polyUb-80S ribosome dissociates yielding K63polyUb-40S subunit and 60S subunit:peptidyl-tRNA with nascent peptide"
+xref: Reactome:R-HSA-9949632 "VCP hexamer extracts K48 polyubiquitinated alanine-tailed nascent peptide from VCP hexamer:UFD1:NPLOC4:LTN1:NEMF:TCF25:60S ribosome subunit:K48polyUb-nascent peptide"
+xref: Reactome:R-HSA-9955731 "ABCE1:PELO:80S Ribosome:non-stop mRNA:peptidyl-tRNA with elongating peptide dissociates yielding ABCE1:40S ribosomal subunit, PELO, and 60S ribosomal subunit:peptidyl-tRNA with nascent peptide"
 xref: RHEA:13065
 is_a: GO:0017111 ! ribonucleoside triphosphate phosphatase activity
 relationship: part_of GO:0140657 ! ATP-dependent activity
@@ -136808,6 +137204,7 @@
 synonym: "exoribonuclease activity, producing 5'-phosphomonoesters" EXACT []
 xref: EC:3.1.13.-
 xref: Reactome:R-HSA-429860 "DCP1-DCP2 complex decaps mRNA"
+xref: Reactome:R-HSA-9946784 "DENV mRNA is decapped"
 is_a: GO:0004532 ! RNA exonuclease activity
 property_value: skos:exactMatch EC:3.1.13.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
@@ -137613,7 +138010,7 @@
 id: GO:0016990
 name: arginine deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-arginine + H2O = L-citrulline + NH3." [EC:3.5.3.6]
+def: "Catalysis of the reaction: L-arginine + H2O = L-citrulline + NH4+." [RHEA:19597]
 synonym: "arginine dihydrolase activity" RELATED [EC:3.5.3.6]
 synonym: "citrulline iminase activity" RELATED [EC:3.5.3.6]
 synonym: "L-arginine deiminase activity" RELATED [EC:3.5.3.6]
@@ -138095,6 +138492,7 @@
 xref: Reactome:R-HSA-1606602 "ASAH1 hydrolyzes ceramide"
 xref: Reactome:R-HSA-428205 "ACER2 hydrolyzes ceramide (Golgi)"
 xref: Reactome:R-HSA-428231 "ACER1 hydrolyzes ceramide (endoplasmic reticulum)"
+xref: Reactome:R-HSA-428262 "ACER3 hydrolyzes phytoceramide"
 xref: RHEA:20856
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:exactMatch EC:3.5.1.23
@@ -138179,8 +138577,6 @@
 def: "OBSOLETE. Catalysis of the reaction: sphingosine + ATP = sphingosine 1-phosphate + ADP." [RHEA:35847]
 comment: This term was obsoleted because it represents a specific substrate of sphingosine kinase activity ; GO:0008481.
 synonym: "sphingosine kinase activity" BROAD []
-xref: Reactome:R-HSA-5218845 "p-SPHK1 phosphorylates sphingosine to sphingosine 1-phosphate"
-xref: Reactome:R-HSA-9625814 "SPHK1 phosphorylates sphingosine in response to E2 stimulation"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30243" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008481
@@ -138255,11 +138651,10 @@
 synonym: "nuclear pore activity" RELATED []
 synonym: "nucleocytoplasmic transporter activity" EXACT []
 xref: Reactome:R-HSA-1176059 "Translocation of Influenza A virus nonstructural protein 1 (NS1A) into the nucleus"
-xref: Reactome:R-HSA-170796 "NPC transports GCK1:GKRP from cytosol to nucleoplasm"
+xref: Reactome:R-HSA-170796 "NPC transports GCK1:GCKR from cytosol to nucleoplasm"
 xref: Reactome:R-HSA-192627 "Viral mRNA Export"
 xref: Reactome:R-HSA-192925 "Export of Spliced Viral mRNA"
 xref: Reactome:R-HSA-5252041 "NPC transports Hikeshi:HSP70s:ATP from cytosol to nucleoplasm"
-xref: Reactome:R-HSA-5578744 "Importin-8 imports AGO2:miRNA into the nucleus"
 xref: Reactome:R-HSA-5661474 "Defective NPC does not transport GCK1:GKRP from cytosol to nucleoplasm"
 xref: Reactome:R-HSA-6783483 "tRNA:XPOT:RAN:GTP translocates from the nucleus to the cytosol"
 is_a: GO:0005198 ! structural molecule activity
@@ -138281,7 +138676,7 @@
 xref: EC:3.1.1.31
 xref: KEGG_REACTION:R02035
 xref: MetaCyc:6PGLUCONOLACT-RXN
-xref: Reactome:R-HSA-71296 "D-glucono-1,5-lactone 6-phosphate + H2O => 6-phospho-D-gluconate"
+xref: Reactome:R-HSA-71296 "PGLS hydrolyzes D-glucono-1,5-lactone 6-phosphate"
 xref: RHEA:12556
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.1.31
@@ -138873,6 +139268,7 @@
 xref: Reactome:R-HSA-5651782 "FEN1 bound to POLB cleaves displaced DNA strand (flap)"
 xref: Reactome:R-HSA-5687664 "FEN1 cleaves displaced ssDNA flaps during MMEJ"
 xref: Reactome:R-HSA-69152 "Removal of remaining Flap"
+xref: Reactome:R-HSA-9914870 "MGME1 cleaves 5' DNA flap from nascent mitochondrial DNA"
 is_a: GO:0016888 ! DNA endonuclease activity, producing 5'-phosphomonoesters
 is_a: GO:0048256 ! flap endonuclease activity
 
@@ -139095,8 +139491,7 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phospholipids from one membrane bilayer leaflet to the other, by an ATP-independent mechanism." [GOC:cjm, PMID:20043909, PMID:20302864]
 comment: Nomenclature note. Scramblases are ATP-independent, non-selective, translocases inducing non-specific transbilayer movements across the membrane. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases).
-is_a: GO:0005548 ! phospholipid transporter activity
-is_a: GO:0140303 ! intramembrane lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 relationship: part_of GO:0017121 ! plasma membrane phospholipid scrambling
 
 [Term]
@@ -139715,7 +140110,7 @@
 xref: EC:2.3.1.17
 xref: KEGG_REACTION:R00487
 xref: MetaCyc:ASPARTATE-N-ACETYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-8954468 "NAT8L transfers acetyl group from Ac-CoA to L-Asp, forming NAA"
+xref: Reactome:R-HSA-8954468 "NAT8L transfers acetyl group from Ac-CoA to L-Asp, forming NAASP"
 xref: RHEA:14165
 is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.17
@@ -141677,7 +142072,6 @@
 namespace: biological_process
 def: "The alteration of an amino acid residue in a peptide." [GOC:mah]
 subset: gocheck_do_not_annotate
-subset: goslim_yeast
 is_a: GO:0036211 ! protein modification process
 
 [Term]
@@ -142392,15 +142786,6 @@
 def: "OBSOLETE. Catalysis of the reaction: ATP + biotin + protein = AMP + diphosphate + biotin-protein." [GOC:mah]
 comment: The reason for obsoletion is this term is an unnecessary grouping term.
 synonym: "biotin-apoprotein ligase activity" EXACT []
-xref: Reactome:R-HSA-2993447 "HLCS biotinylates 6x(PCCA:PCCB)"
-xref: Reactome:R-HSA-2993799 "HLCS biotinylates 6xMCCC1:6xMCCC2"
-xref: Reactome:R-HSA-2993802 "HLCS biotinylates PC:Mn2+"
-xref: Reactome:R-HSA-2993814 "HLCS biotinylates ACACA:Mn2+"
-xref: Reactome:R-HSA-3323184 "Defective HLCS does not biotinylate ACACA:Mn2+"
-xref: Reactome:R-HSA-4167511 "HLCS biotinylates ACACB"
-xref: Reactome:R-HSA-9035987 "Defective HLCS does not biotinylate 6xMCCC1:6xMCCC2"
-xref: Reactome:R-HSA-9035988 "Defective HLCS does not biotinylate PC:Mn2+"
-xref: Reactome:R-HSA-9035990 "Defective HLCS does not biotinylate 6x(PCCA:PCCB)"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28107" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004077
@@ -142479,9 +142864,6 @@
 def: "OBSOLETE. The glycosylation of protein via the N4 atom of peptidyl-asparagine forming N4-glycosyl-L-asparagine; the most common form is N-acetylglucosaminyl asparagine; N-acetylgalactosaminyl asparagine and N4 glucosyl asparagine also occur. This modification typically occurs in extracellular peptides with an N-X-(ST) motif. Partial modification has been observed to occur with cysteine, rather than serine or threonine, in the third position; secondary structure features are important, and proline in the second or fourth positions inhibits modification." [GOC:jsg]
 comment: The reason for obsoletion is that this term is equivalent to protein N-linked glycosylation.
 synonym: "protein amino acid N-linked glycosylation via asparagine" EXACT [GOC:bf]
-xref: Reactome:R-HSA-162699 "Synthesis of dolichyl-phosphate mannose"
-xref: Reactome:R-HSA-446203 "Asparagine N-linked glycosylation"
-xref: Reactome:R-HSA-480985 "Synthesis of dolichyl-phosphate-glucose"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30880" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0006487
@@ -144403,7 +144785,7 @@
 id: GO:0018488
 name: aryl-aldehyde oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an aromatic aldehyde + O2 + H2O = an aromatic acid + hydrogen peroxide." [EC:1.2.3.9]
+def: "Catalysis of the reaction: an aromatic aldehyde + O2 + H2O = an aromatic carboxylate + H2O2 + H+." [RHEA:18569]
 synonym: "aryl-aldehyde:oxygen oxidoreductase activity" RELATED [EC:1.2.3.9]
 xref: EC:1.2.3.9
 xref: MetaCyc:ARYL-ALDEHYDE-OXIDASE-RXN
@@ -144953,7 +145335,7 @@
 id: GO:0018527
 name: cyclohexylamine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclohexylamine + O2 + H2O = cyclohexanone + NH3 + hydrogen peroxide." [PMID:18451, RHEA:18433]
+def: "Catalysis of the reaction: cyclohexylamine + O2 + H2O = cyclohexanone + H2O2 + NH4+." [PMID:18451, RHEA:18433]
 synonym: "cyclohexylamine:oxygen oxidoreductase (deaminating)" RELATED [EC:1.4.3.12]
 xref: EC:1.4.3.12
 xref: MetaCyc:CYCLOHEXYLAMINE-OXIDASE-RXN
@@ -145001,7 +145383,7 @@
 xref: MetaCyc:R-6-HYDROXYNICOTINE-OXIDASE-RXN
 xref: RHEA:10012
 xref: UM-BBD_reactionID:r0477
-is_a: GO:0019116 ! hydroxy-nicotine oxidase activity
+is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
 property_value: skos:exactMatch EC:1.5.3.6
 property_value: skos:exactMatch RHEA:10012
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
@@ -145021,7 +145403,7 @@
 xref: MetaCyc:S-6-HYDROXYNICOTINE-OXIDASE-RXN
 xref: RHEA:11880
 xref: UM-BBD_reactionID:r0478
-is_a: GO:0019116 ! hydroxy-nicotine oxidase activity
+is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
 property_value: skos:exactMatch EC:1.5.3.5
 property_value: skos:exactMatch RHEA:11880
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -145190,7 +145572,6 @@
 def: "OBSOLETE. Catalysis of the removal of one or more nitrite (NO2-) groups from nitroglycerin or a derivative." [GOC:curators]
 comment: This term was obsoleted because there is no evidence that this specific reaction exists.
 synonym: "NG reductase activity" EXACT []
-xref: Reactome:R-HSA-9620103 "ALDH2 transforms GTN to NO"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28261" xsd:anyURI
 is_obsolete: true
 
@@ -145207,7 +145588,7 @@
 id: GO:0018549
 name: methanethiol oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: methanethiol + O2 + H2O = formaldehyde + hydrogen sulfide + hydrogen peroxide." [EC:1.8.3.4]
+def: "Catalysis of the reaction: methanethiol + O2 + H2O = hydrogen sulfide + formaldehyde + H2O2 + H+." [RHEA:11812]
 synonym: "(MM)-oxidase activity" RELATED [EC:1.8.3.4]
 synonym: "methanethiol:oxygen oxidoreductase activity" RELATED [EC:1.8.3.4]
 synonym: "methyl mercaptan oxidase activity" RELATED [EC:1.8.3.4]
@@ -145965,7 +146346,7 @@
 id: GO:0018618
 name: anthranilate 1,2-dioxygenase (deaminating, decarboxylating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: anthranilate + NADPH + H+ + O2 = catechol + CO2 + NADP+ + NH3." [EC:1.14.12.1]
+def: "Catalysis of the reaction: anthranilate + NAD(P)H + O2 + 3 H+ = catechol + NH4+ + CO2 + NAD(P)+." [EC:1.14.12.1]
 synonym: "AntA" RELATED []
 synonym: "AntB" RELATED []
 synonym: "AntC" RELATED []
@@ -147278,30 +147659,30 @@
 
 [Term]
 id: GO:0018711
-name: benzoyl acetate-CoA thiolase activity
+name: obsolete benzoyl acetate-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: benzoyl acetyl-CoA + CoA = acetyl-CoA + benzoyl-CoA." [UM-BBD_reactionID:r0243]
-xref: MetaCyc:RXN-2006
-xref: UM-BBD_reactionID:r0243
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: benzoyl acetyl-CoA + CoA = acetyl-CoA + benzoyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018712
-name: 3-hydroxybutyryl-CoA thiolase activity
+name: obsolete 3-hydroxybutyryl-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-hydroxy-5-oxohexanoyl-CoA + CoASH = 3-hydroxybutyryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0010]
-xref: MetaCyc:R7-RXN
-xref: UM-BBD_reactionID:r0010
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: 3-hydroxy-5-oxohexanoyl-CoA + CoASH = 3-hydroxybutyryl-CoA + acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018713
-name: 3-ketopimelyl-CoA thiolase activity
+name: obsolete 3-ketopimelyl-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-ketopimeloyl-CoA + CoA = glutaryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0197]
-xref: MetaCyc:RXN-8032
-xref: UM-BBD_reactionID:r0197
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: 3-ketopimeloyl-CoA + CoA = glutaryl-CoA + acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018715
@@ -147341,12 +147722,18 @@
 
 [Term]
 id: GO:0018719
-name: 6-aminohexanoate transaminase activity
+name: 6-aminohexanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-aminohexanoate + alpha-ketoglutarate = glutamate + 6-oxohexanoate." [UM-BBD_reactionID:r0449]
+def: "Catalysis of the reaction: 6-aminohexanoate + 2-oxoglutarate = 6-oxohexanoate + L-glutamate." [RHEA:58200]
+synonym: "6-aminohexanoate aminotransferase activity" EXACT []
+xref: EC:2.6.1.116
 xref: MetaCyc:R562-RXN
+xref: RHEA:58200
 xref: UM-BBD_reactionID:r0449
 is_a: GO:0008483 ! transaminase activity
+property_value: skos:exactMatch EC:2.6.1.116
+property_value: skos:exactMatch RHEA:58200
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0018720
@@ -147681,7 +148068,7 @@
 id: GO:0018750
 name: biuret amidohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: biuret + H2O = urea + CO2 + NH3." [EC:3.5.1.84]
+def: "Catalysis of the reaction: biuret + H2O = urea-1-carboxylate + NH4+." [RHEA:17525]
 xref: EC:3.5.1.84
 xref: RHEA:17525
 xref: UM-BBD_reactionID:r0846
@@ -147701,12 +148088,12 @@
 
 [Term]
 id: GO:0018752
-name: epsilon-caprolactam lactamase activity
+name: obsolete epsilon-caprolactam lactamase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: epsilon-caprolactam + H2O = H+ + 6-aminohexanoate." [UM-BBD_reactionID:r0448]
-xref: MetaCyc:R561-RXN
-xref: UM-BBD_reactionID:r0448
-is_a: GO:0016812 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides
+def: "OBSOLETE. Catalysis of the reaction: epsilon-caprolactam + H2O = H+ + 6-aminohexanoate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018753
@@ -147725,40 +148112,48 @@
 
 [Term]
 id: GO:0018754
-name: ammelide aminohydrolase activity
+name: obsolete ammelide aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ammelide + H2O = cyanuric acid + NH3." [PMID:1991731]
+def: "OBSOLETE. Catalysis of the reaction: ammelide + H2O = cyanuric acid + NH4+." [PMID:1991731]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8017
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018755
-name: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine aminohydrolase activity
+name: obsolete 2-chloro-4-hydroxy-6-amino-1,3,5-triazine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + OH- = 2,4-dihydroxy-6-amino-1,3,5-triazine + Cl-." [UM-BBD_reactionID:r1414]
-xref: UM-BBD_reactionID:r1414
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+def: "OBSOLETE. Catalysis of the reaction: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + OH- = 2,4-dihydroxy-6-amino-1,3,5-triazine + Cl-." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018756
 name: ammeline aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ammeline + H2O = ammelide + NH3." [PMID:1991731]
+def: "Catalysis of the reaction: ammeline + H2O = ammelide + NH4+. Also converts melamine to ammeline." [EC:3.5.4.45, PMID:1991731]
+xref: EC:3.5.4.45
 xref: MetaCyc:RXN-8016
 xref: RHEA:26201
 is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: skos:exactMatch EC:3.5.4.45
 property_value: skos:exactMatch RHEA:26201
+property_value: skos:relatedMatch RHEA:26197
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0018757
-name: deisopropylhydroxyatrazine aminohydrolase activity
+name: obsolete deisopropylhydroxyatrazine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: deisopropylhydroxyatrazine + H2O = NH3 + 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine." [UM-BBD_reactionID:r0121]
+def: "OBSOLETE. Catalysis of the reaction: deisopropylhydroxyatrazine + H2O = NH4+ + 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine." [UM-BBD_reactionID:r0121]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: KEGG_REACTION:R05574
 xref: UM-BBD_reactionID:r0121
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018758
@@ -147826,7 +148221,7 @@
 id: GO:0018762
 name: aliphatic nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: R-CN + H2O = R-COOH + NH3." [EC:3.5.5.7]
+def: "Catalysis of the reaction: an aliphatic nitrile + 2 H2O = a carboxylate + NH4+." [RHEA:46188]
 synonym: "aliphatic nitrile aminohydrolase activity" RELATED [EC:3.5.5.7]
 xref: EC:3.5.5.7
 xref: MetaCyc:3.5.5.7-RXN
@@ -147886,13 +148281,12 @@
 
 [Term]
 id: GO:0018766
-name: dihydrophloroglucinol hydrolase activity
+name: obsolete dihydrophloroglucinol hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dihydrophloroglucinol + OH- = 3-hydroxy-5-oxohexanoate." [UM-BBD_reactionID:r0008]
-xref: KEGG_REACTION:R07831
-xref: MetaCyc:R6-RXN
-xref: UM-BBD_reactionID:r0008
-is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
+def: "OBSOLETE. Catalysis of the reaction: dihydrophloroglucinol + OH- = 3-hydroxy-5-oxohexanoate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018767
@@ -147954,12 +148348,12 @@
 
 [Term]
 id: GO:0018772
-name: trioxoheptanoate hydrolase activity
+name: obsolete trioxoheptanoate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4,6-trioxoheptanoate + H2O = acetylpyruvate + acetate." [MetaCyc:R306-RXN, UM-BBD_reactionID:r0094]
-xref: MetaCyc:R306-RXN
-xref: UM-BBD_reactionID:r0094
-is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
+def: "OBSOLETE. Catalysis of the reaction: 2,4,6-trioxoheptanoate + H2O = acetylpyruvate + acetate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018773
@@ -148423,12 +148817,12 @@
 
 [Term]
 id: GO:0018811
-name: cyclohex-1-ene-1-carboxyl-CoA hydratase activity
+name: obsolete cyclohex-1-ene-1-carboxyl-CoA hydratase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclohex-1-ene-1-carboxyl-CoA + H2O = 2-hydroxycyclohexane-1-carboxyl-CoA." [MetaCyc:R266-RXN, UM-BBD_reactionID:r0191]
-xref: MetaCyc:R266-RXN
-xref: UM-BBD_reactionID:r0191
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: cyclohex-1-ene-1-carboxyl-CoA + H2O = 2-hydroxycyclohexane-1-carboxyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018812
@@ -148496,12 +148890,12 @@
 
 [Term]
 id: GO:0018815
-name: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon hydratase-aldolase activity
+name: obsolete 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon hydratase-aldolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon + H2O = 2-oxobut-3-enanoate + 2,5,6-trihydroxy-3-methylpyridine." [MetaCyc:RXN-645, UM-BBD_reactionID:r0051]
-xref: MetaCyc:RXN-645
-xref: UM-BBD_reactionID:r0051
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon + H2O = 2-oxobut-3-enanoate + 2,5,6-trihydroxy-3-methylpyridine." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018816
@@ -148657,7 +149051,7 @@
 id: GO:0018826
 name: methionine gamma-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine = methanethiol + NH3 + 2-oxobutanoate." [EC:4.4.1.11, RHEA:23800]
+def: "Catalysis of the reaction: L-methionine + H2O = methanethiol + 2-oxobutanoate + NH4+." [RHEA:23800]
 synonym: "L-methioninase activity" RELATED [EC:4.4.1.11]
 synonym: "L-methionine gamma-lyase activity" RELATED [EC:4.4.1.11]
 synonym: "L-methionine methanethiol-lyase (deaminating)" RELATED [EC:4.4.1.11]
@@ -148988,14 +149382,12 @@
 
 [Term]
 id: GO:0018856
-name: benzoyl acetate-CoA ligase activity
+name: obsolete benzoyl acetate-CoA ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-oxo-3-phenylpropionate + CoA + ATP = AMP + diphosphate + benzoyl acetyl-CoA." [UM-BBD_reactionID:r0242]
-xref: KEGG_REACTION:R05452
-xref: SABIO-RK:5058
-xref: UM-BBD_reactionID:r0242
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+def: "OBSOLETE. Catalysis of the reaction: 3-oxo-3-phenylpropionate + CoA + ATP = AMP + diphosphate + benzoyl acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018857
@@ -150952,10 +151344,6 @@
 synonym: "viral host defence evasion" NARROW []
 synonym: "viral host defense evasion" NARROW []
 synonym: "viral regulation of antiviral response" EXACT []
-xref: Reactome:R-HSA-164938 "Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters"
-xref: Reactome:R-HSA-164944 "Nef and signal transduction"
-xref: Reactome:R-HSA-168305 "Inhibition of Interferon Synthesis"
-xref: Reactome:R-HSA-169131 "Inhibition of PKR"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23043" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25016" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29236" xsd:anyURI
@@ -151016,8 +151404,6 @@
 synonym: "regulation of cellular process in host by virus" EXACT []
 synonym: "regulation of host cellular process by virus" EXACT []
 synonym: "viral host cell process manipulation" EXACT []
-xref: Reactome:R-HSA-168276 "NS1 Mediated Effects on Host Pathways"
-xref: Reactome:R-HSA-9833110 "RSV-host interactions"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26621" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0044068
@@ -151626,6 +152012,7 @@
 def: "Catalysis of the transfer of a myristoyl (CH3-[CH2]12-CO-) group to an acceptor molecule." [GOC:ai]
 xref: Reactome:R-HSA-141367 "Myristoylation of tBID by NMT1"
 xref: Reactome:R-HSA-162914 "Myristoylation of Nef"
+xref: Reactome:R-HSA-9919009 "NMT1 interacts with E"
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 
 [Term]
@@ -151657,10 +152044,12 @@
 
 [Term]
 id: GO:0019112
-name: phenanthrol glycosyltransferase activity
+name: obsolete phenanthrol glycosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: phenanthrol + glucose = phenanthryl-beta-D-glucopyranoside + H2O." [GOC:ai]
-is_a: GO:0016758 ! hexosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: phenanthrol + glucose = phenanthryl-beta-D-glucopyranoside + H2O." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019113
@@ -151696,11 +152085,14 @@
 
 [Term]
 id: GO:0019116
-name: hydroxy-nicotine oxidase activity
+name: obsolete hydroxy-nicotine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyrid-3-yl)-4-(methylamino)butan-1-one + hydrogen peroxide." [GOC:jl, PMID:16095622]
-is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyrid-3-yl)-4-(methylamino)butan-1-one + hydrogen peroxide." [GOC:jl, PMID:16095622]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+is_obsolete: true
+consider: GO:0018530
+consider: GO:0018531
 
 [Term]
 id: GO:0019117
@@ -151954,7 +152346,6 @@
 namespace: molecular_function
 alt_id: GO:0046420
 def: "Catalysis of the reaction: N6-dimethylallyladenine + acceptor + H2O = adenine + 3-methylbut-2-enal + reduced electron acceptor." [EC:1.5.99.12]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "6-N-dimethylallyladenine:acceptor oxidoreductase activity" RELATED [EC:1.5.99.12]
 synonym: "cytokinin oxidase activity" RELATED []
 synonym: "N6-dimethylallyladenine:(acceptor) oxidoreductase activity" RELATED [EC:1.5.99.12]
@@ -151966,8 +152357,8 @@
 property_value: skos:exactMatch EC:1.5.99.12
 property_value: skos:exactMatch MetaCyc:1.5.99.12-RXN
 property_value: skos:exactMatch RHEA:13625
-property_value: skos:relatedMatch MetaCyc:PWY-2841
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019140
@@ -152118,7 +152509,7 @@
 id: GO:0019148
 name: D-cysteine desulfhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-cysteine + H2O = sulfide + NH3 + pyruvate." [EC:4.4.1.15]
+def: "Catalysis of the reaction: D-cysteine + H2O = hydrogen sulfide + pyruvate + NH4+ + H+." [RHEA:11268]
 synonym: "D-cysteine lyase activity" RELATED [EC:4.4.1.15]
 synonym: "D-cysteine sulfide-lyase (deaminating)" RELATED [EC:4.4.1.15]
 synonym: "D-cysteine sulfide-lyase (deaminating; pyruvate-forming)" RELATED [EC:4.4.1.15]
@@ -152134,7 +152525,7 @@
 id: GO:0019149
 name: 3-chloro-D-alanine dehydrochlorinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-chloro-D-alanine + H2O = pyruvate + chloride + NH3." [EC:4.5.1.2]
+def: "Catalysis of the reaction: 3-chloro-D-alanine + H2O = chloride + pyruvate + NH4+ + H+." [RHEA:18873]
 synonym: "3-chloro-D-alanine chloride-lyase (deaminating)" RELATED [EC:4.5.1.2]
 synonym: "3-chloro-D-alanine chloride-lyase (deaminating; pyruvate-forming)" RELATED [EC:4.5.1.2]
 synonym: "beta-chloro-D-alanine dehydrochlorinase activity" RELATED [EC:4.5.1.2]
@@ -152283,6 +152674,7 @@
 synonym: "mannokinase (phosphorylating)" RELATED [EC:2.7.1.7]
 xref: EC:2.7.1.7
 xref: MetaCyc:MANNKIN-RXN
+xref: Reactome:R-HSA-8931653 "POMK 6-phosphorylates mannose in GalNAc-GlcNAc-Man-DAG1"
 xref: Reactome:R-HSA-9909159 "HK1 phosphorylates Mannose to Man6P"
 xref: RHEA:11028
 is_a: GO:0004396 ! hexokinase activity
@@ -152294,7 +152686,7 @@
 id: GO:0019159
 name: nicotinamide-nucleotide amidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide + H2O = beta-nicotinate D-ribonucleotide + NH3." [EC:3.5.1.42]
+def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide + H2O = nicotinate beta-D-ribonucleotide + NH4+." [RHEA:12400]
 synonym: "nicotinamide mononucleotide amidohydrolase activity" RELATED [EC:3.5.1.42]
 synonym: "nicotinamide mononucleotide deamidase activity" RELATED [EC:3.5.1.42]
 synonym: "nicotinamide-D-ribonucleotide amidohydrolase activity" RELATED [EC:3.5.1.42]
@@ -152330,12 +152722,13 @@
 
 [Term]
 id: GO:0019161
-name: diamine transaminase activity
+name: diamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde + L-glutamate." [EC:2.6.1.29]
+def: "Catalysis of the reaction: an alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde + L-glutamate." [RHEA:18217]
 synonym: "amine transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "amine-ketoacid transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "diamine aminotransferase activity" EXACT []
+synonym: "diamine transaminase activity" BROAD []
 synonym: "diamine-ketoglutaric transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "diamine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.29]
 xref: EC:2.6.1.29
@@ -152345,14 +152738,16 @@
 property_value: skos:exactMatch EC:2.6.1.29
 property_value: skos:exactMatch RHEA:18217
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019162
-name: pyridoxamine-oxaloacetate transaminase activity
+name: pyridoxamine:oxaloacetate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: oxaloacetate + pyridoxamine = L-aspartate + pyridoxal." [EC:2.6.1.31, RHEA:10844]
+def: "Catalysis of the reaction: oxaloacetate + pyridoxamine = L-aspartate + pyridoxal." [RHEA:10844]
 synonym: "pyridoxamine--oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.31]
 synonym: "pyridoxamine-oxaloacetate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-oxaloacetate transaminase activity" EXACT []
 synonym: "pyridoxamine:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.31]
 xref: EC:2.6.1.31
 xref: KEGG_REACTION:R01713
@@ -152362,17 +152757,19 @@
 property_value: skos:exactMatch EC:2.6.1.31
 property_value: skos:exactMatch RHEA:10844
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019163
-name: pyridoxamine-phosphate transaminase activity
+name: pyridoxamine-phosphate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + 2-oxoglutarate = pyridoxal 5'-phosphate + D-glutamate." [EC:2.6.1.54]
+def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + 2-oxoglutarate = pyridoxal 5'-phosphate + D-glutamate." [RHEA:15877]
 synonym: "pyridoxamine 5'-phosphate transaminase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine 5'-phosphate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine phosphate aminotransferase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine-5'-phosphate:2-oxoglutarate aminotransferase (D-glutamate-forming)" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine-phosphate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-phosphate transaminase activity" BROAD []
 xref: EC:2.6.1.54
 xref: MetaCyc:PYRDAMPTRANS-RXN
 xref: RHEA:15877
@@ -152380,6 +152777,7 @@
 property_value: skos:exactMatch EC:2.6.1.54
 property_value: skos:exactMatch RHEA:15877
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019164
@@ -152560,10 +152958,11 @@
 
 [Term]
 id: GO:0019179
-name: dTDP-4-amino-4,6-dideoxy-D-glucose transaminase activity
+name: dTDP-4-amino-4,6-dideoxy-D-glucose:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-D-glucose + L-glutamate." [EC:2.6.1.33]
+def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-D-glucose + L-glutamate." [RHEA:19085]
 synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose aminotransferase activity" EXACT []
+synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose transaminase activity" EXACT []
 synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.33]
 synonym: "TDP-4-keto-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.33]
 synonym: "TDP-4-oxo-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.33]
@@ -152578,14 +152977,16 @@
 property_value: skos:exactMatch EC:2.6.1.33
 property_value: skos:exactMatch RHEA:19085
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019180
-name: dTDP-4-amino-4,6-dideoxygalactose transaminase activity
+name: dTDP-4-amino-4,6-dideoxygalactose:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + dTDP-4-amino-4,6-dideoxy-D-galactose = L-glutamate + dTDP-4-dehydro-6-deoxy-D-galactose." [EC:2.6.1.59, RHEA:10368]
+def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-alpha-D-galactose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + L-glutamate." [RHEA:10368]
 synonym: "dTDP-4,6-dideoxy-D-galactose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.59]
 synonym: "dTDP-4-amino-4,6-dideoxygalactose aminotransferase activity" RELATED [EC:2.6.1.59]
+synonym: "dTDP-4-amino-4,6-dideoxygalactose transaminase activity" EXACT []
 synonym: "dTDP-fucosamine aminotransferase activity" EXACT []
 synonym: "thymidine diphosphate 4-keto-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.59]
 synonym: "thymidine diphosphoaminodideoxygalactose aminotransferase activity" RELATED [EC:2.6.1.59]
@@ -152597,6 +152998,7 @@
 property_value: skos:exactMatch EC:2.6.1.59
 property_value: skos:exactMatch RHEA:10368
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019181
@@ -152721,7 +153123,6 @@
 name: carbohydrate kinase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a carbohydrate substrate molecule." [GOC:jl]
-xref: Reactome:R-HSA-8931653 "POMK 6-phosphorylates Mannose in GalNAc-GlcNAc-Man-DAG1"
 is_a: GO:0016301 ! kinase activity
 
 [Term]
@@ -152807,6 +153208,7 @@
 name: kinase inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces the activity of a kinase." [GOC:mah]
+xref: Reactome:R-HSA-170824 "glucokinase (GCK1) + glucokinase regulatory protein (GKRP) <=> GCK1:GKRP complex"
 is_a: GO:0004857 ! enzyme inhibitor activity
 is_a: GO:0019207 ! kinase regulator activity
 relationship: negatively_regulates GO:0016301 ! kinase activity
@@ -152816,6 +153218,11 @@
 name: phosphatase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a phosphatase." [GOC:ai]
+xref: Reactome:R-HSA-6809238 "MTMR7 binds MTMR9"
+xref: Reactome:R-HSA-6809254 "MTMR8 binds MTMR9"
+xref: Reactome:R-HSA-6809309 "MTMR6 binds MTMR9"
+xref: Reactome:R-HSA-6809680 "MTMR12 binds MTM1"
+xref: Reactome:R-HSA-6809793 "MTMR2 binds SBF2"
 is_a: GO:0008047 ! enzyme activator activity
 is_a: GO:0019208 ! phosphatase regulator activity
 relationship: positively_regulates GO:0016791 ! phosphatase activity
@@ -152902,12 +153309,14 @@
 name: regulation of nucleobase-containing compound metabolic process
 namespace: biological_process
 def: "Any cellular process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:go_curators]
+subset: gocheck_do_not_annotate
 synonym: "regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" RELATED [GOC:dph, GOC:tb]
 synonym: "regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolism" EXACT []
 is_a: GO:0080090 ! regulation of primary metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006139 ! nucleobase-containing compound metabolic process
 relationship: regulates GO:0006139 ! nucleobase-containing compound metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0019220
@@ -153068,10 +153477,9 @@
 id: GO:0019239
 name: deaminase activity
 namespace: molecular_function
-def: "Catalysis of the removal of an amino group from a substrate, producing a substituted or nonsubstituted ammonia (NH3/NH2R)." [GOC:jl]
+def: "Catalysis of the removal of an amino group from a substrate, producing a substituted or nonsubstituted ammonia (NH4+/NH2R)." [GOC:jl]
 subset: goslim_pir
 xref: Reactome:R-HSA-74241 "ADA catalyzes the deamination of (deoxy)adenosine"
-xref: Reactome:R-HSA-9014641 "HRSP12 deaminates 2AA to 2OBUTA"
 xref: Reactome:R-HSA-9734745 "Defective ADA does not deaminate (deoxy)adenosine"
 xref: Reactome:R-HSA-9754964 "ADA deamidates RBV"
 is_a: GO:0016810 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
@@ -153094,7 +153502,7 @@
 id: GO:0019241
 name: L-citrulline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-citrulline, N5-carbamoyl-L-ornithine, an alpha amino acid not found in proteins." [ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-citrulline." [GOC:curators]
 synonym: "citrulline breakdown" EXACT []
 synonym: "citrulline catabolism" EXACT []
 synonym: "citrulline degradation" EXACT []
@@ -153103,6 +153511,7 @@
 is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 is_a: GO:1901606 ! alpha-amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019242
@@ -153361,6 +153770,7 @@
 is_a: GO:0006054 ! N-acetylneuraminate metabolic process
 is_a: GO:0046348 ! amino sugar catabolic process
 is_a: GO:0046395 ! carboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
 
 [Term]
 id: GO:0019263
@@ -153382,10 +153792,8 @@
 synonym: "glycine anabolism from serine" EXACT []
 synonym: "glycine formation from serine" EXACT []
 synonym: "glycine synthesis from serine" EXACT []
-xref: MetaCyc:GLYSYN-PWY
 is_a: GO:0006545 ! glycine biosynthetic process
 is_a: GO:0006563 ! L-serine metabolic process
-property_value: skos:exactMatch MetaCyc:GLYSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
 
 [Term]
@@ -153396,9 +153804,7 @@
 synonym: "glycine anabolism, by transamination of glyoxylate" EXACT []
 synonym: "glycine formation, by transamination of glyoxylate" EXACT []
 synonym: "glycine synthesis, by transamination of glyoxylate" EXACT []
-xref: MetaCyc:GLYSYN-ALA-PWY
 is_a: GO:0006545 ! glycine biosynthetic process
-property_value: skos:exactMatch MetaCyc:GLYSYN-ALA-PWY
 
 [Term]
 id: GO:0019266
@@ -153408,7 +153814,6 @@
 synonym: "asparagine anabolism from oxaloacetate" EXACT []
 synonym: "asparagine formation from oxaloacetate" EXACT []
 synonym: "asparagine synthesis from oxaloacetate" EXACT []
-xref: MetaCyc:ASPARAGINE-BIOSYNTHESIS
 is_a: GO:0006107 ! oxaloacetate metabolic process
 is_a: GO:0043649 ! dicarboxylic acid catabolic process
 is_a: GO:0070981 ! L-asparagine biosynthetic process
@@ -153416,16 +153821,16 @@
 
 [Term]
 id: GO:0019267
-name: L-asparagine biosynthetic process from L-cysteine
+name: obsolete L-asparagine biosynthetic process from L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-asparagine from other compounds, including L-cysteine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-asparagine from other compounds, including L-cysteine." [GOC:go_curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "asparagine anabolism from cysteine" EXACT []
 synonym: "asparagine formation from cysteine" EXACT []
 synonym: "asparagine synthesis from cysteine" EXACT []
-xref: MetaCyc:ASPSYNII-PWY
-is_a: GO:0046439 ! L-cysteine metabolic process
-is_a: GO:0070981 ! L-asparagine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0070981
 
 [Term]
 id: GO:0019268
@@ -153477,14 +153882,12 @@
 id: GO:0019272
 name: L-alanine biosynthetic process from pyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of alanine from other compounds, including pyruvate." [GOC:go_curators, MetaCyc:ALANINE-SYN2-PWY]
+def: "The chemical reactions and pathways resulting in the formation of alanine from other compounds, including pyruvate." [GOC:go_curators]
 synonym: "L-alanine anabolism from pyruvate" EXACT []
 synonym: "L-alanine formation from pyruvate" EXACT []
 synonym: "L-alanine synthesis from pyruvate" EXACT []
-xref: MetaCyc:ALANINE-SYN2-PWY
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0042852 ! L-alanine biosynthetic process
-property_value: skos:exactMatch MetaCyc:ALANINE-SYN2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 
 [Term]
@@ -153517,15 +153920,18 @@
 synonym: "UDP-N-acetylgalactosamine biosynthesis" EXACT []
 synonym: "UDP-N-acetylgalactosamine formation" EXACT []
 synonym: "UDP-N-acetylgalactosamine synthesis" EXACT []
+xref: MetaCyc:PWY-5512
 xref: MetaCyc:PWY-5514
 xref: MetaCyc:PWY-8013
 is_a: GO:0009226 ! nucleotide-sugar biosynthetic process
 is_a: GO:0019276 ! UDP-N-acetylgalactosamine metabolic process
 is_a: GO:0046349 ! amino sugar biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5512
 property_value: skos:narrowMatch MetaCyc:PWY-5514
 property_value: skos:narrowMatch MetaCyc:PWY-8013
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019278
@@ -153541,58 +153947,60 @@
 
 [Term]
 id: GO:0019279
-name: L-methionine biosynthetic process from L-homoserine via cystathionine
+name: obsolete L-methionine biosynthetic process from L-homoserine via cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediate cystathionine." [MetaCyc:HOMOSER-METSYN-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediate cystathionine." [MetaCyc:HOMOSER-METSYN-PWY]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine anabolism from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine formation from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from L-homoserine via cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:HOMOSER-METSYN-PWY
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0071265 ! L-methionine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019280
-name: L-methionine biosynthetic process from L-homoserine via O-acetyl-L-homoserine
+name: obsolete L-methionine biosynthetic process from L-homoserine via O-acetyl-L-homoserine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of methionine from L-homoserine, via the intermediate O-acetyl-L-homoserine using sulfur from hydrogen sulfide assimilation." [MetaCyc:HSERMETANA-PWY]
-xref: MetaCyc:HSERMETANA-PWY
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0071265 ! L-methionine biosynthetic process
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of methionine from L-homoserine, via the intermediate O-acetyl-L-homoserine using sulfur from hydrogen sulfide assimilation." [MetaCyc:HSERMETANA-PWY]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30025" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019281
-name: L-methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine
+name: obsolete L-methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine from other compounds, including homoserine, via the intermediates O-succinyl-L-homoserine and cystathionine." [MetaCyc:MET-SAM-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from other compounds, including homoserine, via the intermediates O-succinyl-L-homoserine and cystathionine." [GOC:curators]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine anabolism from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine formation from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:MET-SAM-PWY
-is_a: GO:0019279 ! L-methionine biosynthetic process from L-homoserine via cystathionine
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019283
-name: L-methionine biosynthetic process from L-homoserine via O-phospho-L-homoserine and cystathionine
+name: obsolete L-methionine biosynthetic process from L-homoserine via O-phospho-L-homoserine and cystathionine
 namespace: biological_process
-def: "he chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediates O-phospho-L-homoserine and cystathionine." [GOC:go_curators, MetaCyc:PWY-702]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediates O-phospho-L-homoserine and cystathionine." [GOC:go_curators, MetaCyc:PWY-702]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine formation from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:PWY-702
-is_a: GO:0006575 ! modified amino acid metabolic process
-is_a: GO:0006796 ! phosphate-containing compound metabolic process
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0019637 ! organophosphate metabolic process
-is_a: GO:0071266 ! 'de novo' L-methionine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019284
@@ -153601,9 +154009,16 @@
 def: "The chemical reactions and pathways resulting in the formation of L-methionine from S-adenosylmethionine." [GOC:go_curators, GOC:vw, PMID:31950558]
 synonym: "L-methionine formation from S-adenosylmethionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from S-adenosylmethionine" EXACT [GOC:mah]
+xref: MetaCyc:PWY-7270
+xref: MetaCyc:PWY-7527
+xref: MetaCyc:PWY-7528
 is_a: GO:0071267 ! L-methionine salvage
 relationship: part_of GO:0033353 ! S-adenosylmethionine cycle
+property_value: skos:narrowMatch MetaCyc:PWY-7270
+property_value: skos:narrowMatch MetaCyc:PWY-7527
+property_value: skos:narrowMatch MetaCyc:PWY-7528
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29900" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0019285
@@ -153725,7 +154140,6 @@
 synonym: "L-tyrosine formation from chorismate via 4-hydroxyphenylpyruvate" EXACT []
 synonym: "L-tyrosine synthesis from chorismate via 4-hydroxyphenylpyruvate" EXACT []
 synonym: "tyrosine biosynthetic process from chorismate via 4-hydroxyphenylpyruvate" BROAD []
-xref: MetaCyc:TYRSYN
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 is_a: GO:0046417 ! chorismate metabolic process
 
@@ -153739,7 +154153,6 @@
 synonym: "tyrosine anabolism, by oxidation of phenylalanine" EXACT []
 synonym: "tyrosine formation, by oxidation of phenylalanine" EXACT []
 synonym: "tyrosine synthesis, by oxidation of phenylalanine" EXACT []
-xref: MetaCyc:PWY-6134
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20583" xsd:anyURI
 
@@ -154317,16 +154730,16 @@
 
 [Term]
 id: GO:0019343
-name: L-cysteine biosynthetic process via L-cystathionine
+name: obsolete L-cysteine biosynthetic process via L-cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate L-cystathionine." [GOC:go_curators, MetaCyc:HOMOCYSDEGR-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate L-cystathionine." [GOC:go_curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism via cystathionine" EXACT []
 synonym: "cysteine formation via cystathionine" EXACT []
 synonym: "cysteine synthesis via cystathionine" EXACT []
-xref: MetaCyc:HOMOCYSDEGR-PWY
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:HOMOCYSDEGR-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0019344
@@ -154337,27 +154750,44 @@
 synonym: "cysteine biosynthesis" EXACT []
 synonym: "cysteine formation" EXACT []
 synonym: "cysteine synthesis" EXACT []
+xref: MetaCyc:CYSTSYN-PWY
+xref: MetaCyc:HOMOCYSDEGR-PWY
+xref: MetaCyc:PWY-6308
+xref: MetaCyc:PWY-7289
+xref: MetaCyc:PWY-7870
+xref: MetaCyc:PWY-8009
+xref: MetaCyc:PWY-8010
+xref: MetaCyc:PWY-I9
 is_a: GO:0000097 ! sulfur amino acid biosynthetic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0046439 ! L-cysteine metabolic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:CYSTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:HOMOCYSDEGR-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-6308
+property_value: skos:narrowMatch MetaCyc:PWY-7289
+property_value: skos:narrowMatch MetaCyc:PWY-7870
+property_value: skos:narrowMatch MetaCyc:PWY-8009
+property_value: skos:narrowMatch MetaCyc:PWY-8010
+property_value: skos:narrowMatch MetaCyc:PWY-I9
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0019345
-name: L-cysteine biosynthetic process via S-sulfo-L-cysteine
+name: obsolete L-cysteine biosynthetic process via S-sulfo-L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate S-sulfo-L-cysteine." [MetaCyc:PWY-7289]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate S-sulfo-L-cysteine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism via S-sulfo-L-cysteine" EXACT []
 synonym: "cysteine biosynthesis via S-sulpho-L-cysteine" EXACT []
 synonym: "cysteine biosynthetic process via S-sulpho-L-cysteine" EXACT []
 synonym: "cysteine formation via S-sulfo-L-cysteine" EXACT []
 synonym: "cysteine synthesis via S-sulfo-L-cysteine" EXACT []
-xref: MetaCyc:PWY-7289
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY-7289
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0019346
@@ -155078,7 +155508,7 @@
 id: GO:0019409
 name: aerobic respiration, using ammonia as electron donor
 namespace: biological_process
-def: "The metabolic process in which ammonia (NH3) is oxidized to nitrite (NO2) in the presence of oxygen; enzymatic reactions convert ammonia to hydrazine, and hydrazine to nitrite." [MetaCyc:AMMOXID-PWY]
+def: "The metabolic process in which ammonia (NH4+) is oxidized to nitrite (NO2) in the presence of oxygen; enzymatic reactions convert ammonia to hydrazine, and hydrazine to nitrite." [MetaCyc:AMMOXID-PWY]
 synonym: "aerobic ammonia oxidation to nitrite via hydrazine" EXACT [GOC:mah]
 synonym: "nitrification" RELATED []
 xref: MetaCyc:AMMOXID-PWY
@@ -155500,21 +155930,21 @@
 
 [Term]
 id: GO:0019447
-name: D-cysteine catabolic process
+name: obsolete D-cysteine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-cysteine, (S)-2-amino-3-mercaptopropanoic acid, which occurs naturally in firefly luciferin." [PMID:11527960]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-cysteine, (S)-2-amino-3-mercaptopropanoic acid, which occurs naturally in firefly luciferin." [PMID:11527960]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-cysteine.
 synonym: "D-cysteine breakdown" EXACT []
 synonym: "D-cysteine catabolism" EXACT []
 synonym: "D-cysteine degradation" EXACT []
-is_a: GO:0000098 ! sulfur amino acid catabolic process
-is_a: GO:0006534 ! cysteine metabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019448
 name: L-cysteine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-cysteine, the L-enantiomer of 2-amino-3-mercaptopropanoic acid, i.e. (2R)-2-amino-3-mercaptopropanoic acid." [GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-cysteine." [GOC:curators]
 synonym: "L-cysteine breakdown" EXACT []
 synonym: "L-cysteine catabolism" EXACT []
 synonym: "L-cysteine degradation" EXACT []
@@ -155529,6 +155959,7 @@
 property_value: skos:narrowMatch MetaCyc:LCYSDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-5329
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019449
@@ -155556,31 +155987,27 @@
 
 [Term]
 id: GO:0019451
-name: L-cysteine catabolic process to pyruvate, using cysteine dioxygenase
+name: obsolete L-cysteine catabolic process to pyruvate, using cysteine dioxygenase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown into pyruvate of L-cystine, catalyzed by the enzyme cysteine dioxygenase (EC:1.13.11.20)." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown into pyruvate of L-cystine, catalyzed by the enzyme cysteine dioxygenase (EC:1.13.11.20)." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown to pyruvate, using cysteine dioxygenase" EXACT []
 synonym: "L-cysteine degradation to pyruvate, using cysteine dioxygenase" EXACT []
-is_a: GO:0006090 ! pyruvate metabolic process
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0017172 ! cysteine dioxygenase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019452
-name: taurine biosynthetic process from L-cysteine
+name: obsolete taurine biosynthetic process from L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of taurine (2-aminoethanesulfonic acid) from L-cysteine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of taurine (2-aminoethanesulfonic acid) from L-cysteine." [GOC:go_curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "L-cysteine breakdown to taurine" EXACT []
 synonym: "L-cysteine degradation to taurine" EXACT []
-xref: MetaCyc:PWY-5331
-xref: MetaCyc:PWY-7850
-xref: MetaCyc:PWY-8359
-is_a: GO:0042412 ! taurine biosynthetic process
-is_a: GO:0046439 ! L-cysteine metabolic process
-property_value: skos:narrowMatch MetaCyc:PWY-5331
-property_value: skos:narrowMatch MetaCyc:PWY-7850
-property_value: skos:narrowMatch MetaCyc:PWY-8359
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0042412
 
 [Term]
 id: GO:0019453
@@ -155596,34 +156023,40 @@
 
 [Term]
 id: GO:0019454
-name: L-cysteine catabolic process via cystine, using glutathione-cystine transhydrogenase
+name: obsolete L-cysteine catabolic process via cystine, using glutathione-cystine transhydrogenase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme glutathione-cystine transhydrogenase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme glutathione-cystine transhydrogenase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using glutathione-cystine transhydrogenase" EXACT []
 synonym: "L-cysteine degradation via cystine, using glutathione-cystine transhydrogenase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0047141 ! glutathione-cystine transhydrogenase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019455
-name: L-cysteine catabolic process via cystine, using cystine reductase
+name: obsolete L-cysteine catabolic process via cystine, using cystine reductase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cystine reductase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cystine reductase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using cystine reductase" EXACT []
 synonym: "L-cysteine degradation via cystine, using cystine reductase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0050456 ! L-cystine reductase (NADH) activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019456
-name: L-cysteine catabolic process via cystine, using cysteine transaminase
+name: obsolete L-cysteine catabolic process via cystine, using cysteine transaminase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cysteine transaminase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cysteine transaminase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using cysteine transaminase" EXACT []
 synonym: "L-cysteine degradation via cystine, using cysteine transaminase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0047801 ! L-cysteine transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22538" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019457
@@ -155722,26 +156155,30 @@
 
 [Term]
 id: GO:0019466
-name: L-ornithine catabolic process via proline
+name: obsolete L-ornithine catabolic process via proline
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine, via the intermediate proline." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-ornithine, via the intermediate proline." [GOC:go_curators]
+comment: This term was obsoleted because it represents a variant pathway of L-ornithine degradation and is out of scope for GO.
 synonym: "ornithine breakdown via proline" EXACT []
 synonym: "ornithine degradation via proline" EXACT []
-xref: MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
-is_a: GO:0006593 ! L-ornithine catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0006593
 
 [Term]
 id: GO:0019467
-name: L-ornithine catabolic process, by decarboxylation
+name: obsolete L-ornithine catabolic process, by decarboxylation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine by decarboxylation." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-ornithine by decarboxylation." [GOC:go_curators]
+comment: This term was obsoleted because it represents a variant pathway of L-ornithine degradation and is out of scope for GO.
 synonym: "ornithine breakdown, by decarboxylation" EXACT []
 synonym: "ornithine degradation, by decarboxylation" EXACT []
 xref: MetaCyc:ORNDEG-PWY
-is_a: GO:0006593 ! L-ornithine catabolic process
-relationship: has_part GO:0016831 ! carboxy-lyase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0006593
 
 [Term]
 id: GO:0019468
@@ -155771,7 +156208,7 @@
 id: GO:0019470
 name: trans-4-hydroxy-L-proline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of trans-4-hydroxy-L-proline, C5H9NO3, a derivative of the amino acid proline." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of trans-4-hydroxy-L-proline." [GOC:curators]
 synonym: "4-hydroxyproline breakdown" EXACT []
 synonym: "4-hydroxyproline catabolism" EXACT []
 synonym: "4-hydroxyproline degradation" EXACT []
@@ -155784,6 +156221,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5159
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019471
@@ -155852,20 +156290,21 @@
 
 [Term]
 id: GO:0019476
-name: D-lysine catabolic process
+name: obsolete D-lysine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-lysine, the D-enantiomer of lysine; i.e. (2R)-2,6-diaminohexanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-lysine, the D-enantiomer of lysine; i.e. (2R)-2,6-diaminohexanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-lysine.
 synonym: "D-lysine breakdown" EXACT []
 synonym: "D-lysine catabolism" EXACT []
 synonym: "D-lysine degradation" EXACT []
-is_a: GO:0006553 ! lysine metabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019477
 name: L-lysine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine, the L-enantiomer of (S)-2,6-diaminohexanoic acid." [GOC:go_curators, GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-lysine." [GOC:curators]
 synonym: "L-lysine breakdown" EXACT []
 synonym: "L-lysine catabolism" EXACT []
 synonym: "L-lysine degradation" EXACT []
@@ -155874,6 +156313,9 @@
 xref: MetaCyc:PWY-5280
 xref: MetaCyc:PWY-5283
 xref: MetaCyc:PWY-5298
+xref: MetaCyc:PWY-5311
+xref: MetaCyc:PWY-5314
+xref: MetaCyc:PWY-5324
 xref: MetaCyc:PWY-6328
 xref: MetaCyc:PWY0-461
 xref: MetaCyc:PWY66-425
@@ -155885,10 +156327,15 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5280
 property_value: skos:narrowMatch MetaCyc:PWY-5283
 property_value: skos:narrowMatch MetaCyc:PWY-5298
+property_value: skos:narrowMatch MetaCyc:PWY-5311
+property_value: skos:narrowMatch MetaCyc:PWY-5314
+property_value: skos:narrowMatch MetaCyc:PWY-5324
 property_value: skos:narrowMatch MetaCyc:PWY-6328
 property_value: skos:narrowMatch MetaCyc:PWY0-461
 property_value: skos:narrowMatch MetaCyc:PWY66-425
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019478
@@ -155952,15 +156399,26 @@
 synonym: "beta-alanine biosynthesis" EXACT []
 synonym: "beta-alanine formation" EXACT []
 synonym: "beta-alanine synthesis" EXACT []
+xref: MetaCyc:PWY-3941
+xref: MetaCyc:PWY-3981
+xref: MetaCyc:PWY-3982
+xref: MetaCyc:PWY-5155
+xref: MetaCyc:PWY-5760
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0019482 ! beta-alanine metabolic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-3941
+property_value: skos:narrowMatch MetaCyc:PWY-3981
+property_value: skos:narrowMatch MetaCyc:PWY-3982
+property_value: skos:narrowMatch MetaCyc:PWY-5155
+property_value: skos:narrowMatch MetaCyc:PWY-5760
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0019484
 name: beta-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine (3-aminopropanoic acid), an achiral amino acid and an isomer of alanine. It occurs free (e.g. in brain) and in combination (e.g. in pantothenate) but it is not a constituent of proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine." [GOC:curators]
 synonym: "beta-alanine breakdown" EXACT []
 synonym: "beta-alanine catabolism" EXACT []
 synonym: "beta-alanine degradation" EXACT []
@@ -155974,25 +156432,31 @@
 property_value: skos:narrowMatch MetaCyc:PWY-1781
 property_value: skos:narrowMatch MetaCyc:PWY-8120
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019485
-name: beta-alanine catabolic process to L-alanine
+name: obsolete beta-alanine catabolic process to L-alanine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including L-alanine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including L-alanine." [GOC:go_curators]
+comment: This term was obsoleted because it represents a variant pathway of beta-alanine degradation and is out of scope for GO.
 synonym: "beta-alanine breakdown to L-alanine" EXACT []
 synonym: "beta-alanine degradation to L-alanine" EXACT []
-is_a: GO:0019484 ! beta-alanine catabolic process
-is_a: GO:0042851 ! L-alanine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019484
 
 [Term]
 id: GO:0019486
-name: beta-alanine catabolic process to mevalonate semialdehyde, by transamination
+name: obsolete beta-alanine catabolic process to mevalonate semialdehyde, by transamination
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including mevalonate semialdehyde, by transamination." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including mevalonate semialdehyde, by transamination." [GOC:go_curators]
+comment: This term was obsoleted because it represents a variant pathway of beta-alanine degradation and is out of scope for GO.
 synonym: "beta-alanine breakdown to mevalonate semialdehyde, by transamination" EXACT []
 synonym: "beta-alanine degradation to mevalonate semialdehyde, by transamination" EXACT []
-is_a: GO:0019484 ! beta-alanine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019484
 
 [Term]
 id: GO:0019487
@@ -156185,7 +156649,7 @@
 id: GO:0019504
 name: L-proline betaine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of stachydrine, N-methylproline methylbetaine, the betaine derivative of L-proline." [PMID:10689197, PMID:1804402, PMID:24520058]
+def: "The chemical reactions and pathways resulting in the breakdown of L-proline betaine (stachydrine)." [PMID:10689197, PMID:1804402, PMID:24520058]
 synonym: "stachydrine breakdown" EXACT []
 synonym: "stachydrine catabolic process" EXACT []
 synonym: "stachydrine catabolism" EXACT []
@@ -156200,6 +156664,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8109
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23577" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019505
@@ -156255,19 +156720,19 @@
 
 [Term]
 id: GO:0019509
-name: L-methionine salvage from methylthioadenosine
+name: obsolete L-methionine salvage from methylthioadenosine
 namespace: biological_process
-def: "The generation of L-methionine (2-amino-4-(methylthio)butanoic acid) from methylthioadenosine." [GOC:jl, MetaCyc:PWY-4361, PMID:19946895]
+def: "OBSOLETE. The generation of L-methionine (2-amino-4-(methylthio)butanoic acid) from methylthioadenosine." [GOC:jl, PMID:19946895]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "methionine recycling" NARROW []
 synonym: "methionine regeneration" NARROW []
 synonym: "methionine salvage from methylthioadenosine" EXACT [GOC:mah]
 synonym: "methionine salvage pathway" EXACT []
-xref: MetaCyc:PWY-4361
-xref: MetaCyc:PWY-6753
-xref: MetaCyc:PWY-6754
-xref: MetaCyc:PWY-6756
 xref: Reactome:R-HSA-1237112 "Methionine salvage pathway"
-is_a: GO:0071267 ! L-methionine salvage
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033353
 
 [Term]
 id: GO:0019510
@@ -156754,29 +157219,28 @@
 
 [Term]
 id: GO:0019552
-name: L-glutamate fermentation via 2-hydroxyglutarate
+name: obsolete L-glutamate fermentation via 2-hydroxyglutarate
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate 2-hydroxyglutarate, yielding acetyl-CoA and butanoate. Acetyl-CoA can be further converted into acetate, releasing one ATP molecule." [PMID:17166837]
+def: "OBSOLETE. The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate 2-hydroxyglutarate, yielding acetyl-CoA and butanoate. Acetyl-CoA can be further converted into acetate, releasing one ATP molecule." [PMID:17166837]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-glutamate catabolic process via 2-hydroxyglutarate" EXACT []
-xref: MetaCyc:P162-PWY
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019665 ! amino acid fermentation
-property_value: skos:narrowMatch MetaCyc:P162-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29511" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0019553
-name: L-glutamate fermentation via L-citramalate
+name: obsolete L-glutamate fermentation via L-citramalate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate L-citramalate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate L-citramalate." [GOC:go_curators]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown via L-citramalate" EXACT []
 synonym: "glutamate degradation via L-citramalate" EXACT []
 synonym: "L-glutamate catabolic process via L-citramalate" EXACT []
-xref: MetaCyc:GLUDEG-II-PWY
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019665 ! amino acid fermentation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0019554
@@ -157012,23 +157476,18 @@
 
 [Term]
 id: GO:0019574
-name: sucrose catabolic process via 3'-ketosucrose
+name: obsolete sucrose catabolic process via 3'-ketosucrose
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sucrose, which proceeds via the conversion of sucrose to 3'-ketosucrose. 3'-ketosucrose is hydrolyzed to 3-ketoglucose and fructose, and the 3-ketoglucose is then be converted to glucose." [GOC:bf, GOC:dgf, MetaCyc:SUCROSEUTIL2-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of sucrose, which proceeds via the conversion of sucrose to 3'-ketosucrose. 3'-ketosucrose is hydrolyzed to 3-ketoglucose and fructose, and the 3-ketoglucose is then be converted to glucose." [GOC:bf, GOC:dgf, MetaCyc:SUCROSEUTIL2-PWY]
+comment: This term was obsoleted because it represents a specific pathway out of scope for GO.
 synonym: "sucrose breakdown, using glucoside 3-dehydrogenase" EXACT []
 synonym: "sucrose catabolic process to D-glucose" EXACT [GOC:bf, MetaCyc:SUCROSEUTIL2-PWY]
 synonym: "sucrose catabolic process, using glucoside 3-dehydrogenase" EXACT [GOC:bf, MetaCyc:SUCROSEUTIL2-PWY]
 synonym: "sucrose catabolism, using glucoside 3-dehydrogenase" EXACT []
 synonym: "sucrose degradation, using glucoside 3-dehydrogenase" EXACT []
-xref: MetaCyc:SUCROSEUTIL2-PWY
-is_a: GO:0005987 ! sucrose catabolic process
-is_a: GO:0006006 ! glucose metabolic process
-is_a: GO:0006796 ! phosphate-containing compound metabolic process
-is_a: GO:0019637 ! organophosphate metabolic process
-is_a: GO:1901135 ! carbohydrate derivative metabolic process
-relationship: has_part GO:0008865 ! fructokinase activity
-relationship: has_part GO:0033757 ! glucoside 3-dehydrogenase activity
-relationship: has_part GO:0048258 ! 3-ketoglucose-reductase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0005987
 
 [Term]
 id: GO:0019575
@@ -157128,6 +157587,7 @@
 alt_id: GO:0019699
 def: "The chemical reactions and pathways involving glucuronate, any salt or ester of glucuronic acid, the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:go_curators, ISBN:0198506732]
 synonym: "glucuronate metabolism" EXACT []
+xref: Reactome:R-HSA-156588 "Glucuronidation"
 xref: Wikipedia:Glucuronic_acid
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 
@@ -157183,11 +157643,14 @@
 synonym: "mannitol breakdown" EXACT []
 synonym: "mannitol catabolism" EXACT []
 synonym: "mannitol degradation" EXACT []
+xref: MetaCyc:MANNIDEG-PWY
 xref: MetaCyc:PWY-3861
 is_a: GO:0019407 ! hexitol catabolic process
 is_a: GO:0019594 ! mannitol metabolic process
+property_value: skos:narrowMatch MetaCyc:MANNIDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019593
@@ -158022,7 +158485,7 @@
 id: GO:0019665
 name: amino acid fermentation
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of amino acids to ammonia, CO2, H2, acetate and short chain fatty acids, under anoxic conditions." [GOC:curators, GOC:jl]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of amino acids to ammonia, CO2, H2, acetate and short chain fatty acids, under anoxic conditions." [PMID:38821505]
 synonym: "anaerobic amino acid catabolic process" EXACT []
 is_a: GO:0006113 ! fermentation
 is_a: GO:0009063 ! amino acid catabolic process
@@ -158042,7 +158505,7 @@
 id: GO:0019667
 name: L-alanine fermentation
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-alanine, yielding acetylCoA. AcetylCoA can be further converted to ATP." [GOC:jl]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-alanine, yielding acetylCoA. AcetylCoA can be further converted to ATP." [GOC:jl, PMID:38821505]
 synonym: "anaerobic L-alanine catabolic process" EXACT []
 xref: MetaCyc:PROPFERM-PWY
 xref: MetaCyc:PWY-8188
@@ -158093,13 +158556,11 @@
 name: obsolete glutamate catabolic process via mesaconate and citramalate
 namespace: biological_process
 def: "OBSOLETE. The anaerobic chemical reactions and pathways resulting in the breakdown of glutamate via the intermediates mesaconate and S-citramalate, yielding energy in the form of ATP." [GOC:rynl]
-comment: The reason for obsoletion is that this term is an unnecessary grouping term.
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate fermentation via mesaconate and citramalate" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28390" xsd:anyURI
 is_obsolete: true
-consider: GO:0019553
 consider: GO:0033508
-consider: GO:0033509
 
 [Term]
 id: GO:0019672
@@ -158135,7 +158596,6 @@
 synonym: "oxidized NAD metabolism" EXACT []
 synonym: "oxidized nicotinamide adenine dinucleotide metabolic process" EXACT []
 synonym: "oxidized nicotinamide adenine dinucleotide metabolism" EXACT []
-xref: Reactome:R-HSA-196807 "Nicotinate metabolism"
 is_a: GO:0006163 ! purine nucleotide metabolic process
 is_a: GO:0046496 ! nicotinamide nucleotide metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21577" xsd:anyURI
@@ -158523,6 +158983,8 @@
 name: protein-cysteine S-acyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an acyl group to a sulfur atom on the cysteine of a protein molecule." [GOC:ai, RHEA:63372]
+xref: Reactome:R-HSA-158832 "The acetyl group from acetyl-CoA is transferred to NAT2"
+xref: Reactome:R-HSA-174959 "The acetyl group from acetyl-CoA is transferred to NAT1"
 xref: RHEA:63372
 is_a: GO:0016417 ! S-acyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -159197,6 +159659,7 @@
 namespace: molecular_function
 def: "Catalysis of the activation of the small ubiquitin-related modifier ISG15, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
 synonym: "UBE1L" NARROW []
+xref: Reactome:R-HSA-1169397 "Activation of ISG15 by UBA7 E1 enzyme"
 is_a: GO:0008641 ! ubiquitin-like modifier activating enzyme activity
 
 [Term]
@@ -159450,6 +159913,8 @@
 xref: Reactome:R-HSA-6804468 "PIAS1 SUMOylates SP3 with SUMO2"
 xref: Reactome:R-HSA-6804485 "PIAS1 SUMOylates L3MBTL2 with SUMO2"
 xref: Reactome:R-HSA-8956365 "ZBED1 (DREF) SUMOylates CHD3 with SUMO1"
+xref: Reactome:R-HSA-9920126 "UBC9 SUMOylates NS5"
+xref: Reactome:R-HSA-9926476 "UBE2I sumoylates MITF-M"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -159668,7 +160133,7 @@
 synonym: "N-acyl-L-aspartate amidohydrolase activity" RELATED [EC:3.5.1.15]
 xref: EC:3.5.1.15
 xref: MetaCyc:ASPARTOACYLASE-RXN
-xref: Reactome:R-HSA-5691507 "ASPA deacetylates NAA to acetate and L-aspartate"
+xref: Reactome:R-HSA-5691507 "ASPA deacetylates NAASP to acetate and L-aspartate"
 xref: RHEA:10872
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:exactMatch EC:3.5.1.15
@@ -160137,6 +160602,7 @@
 name: calcium channel inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents, or reduces the activity of a calcium channel." [GOC:mah]
+xref: Reactome:R-HSA-400046 "L-type Calcium Channels close in pancreatic beta cells"
 is_a: GO:0005246 ! calcium channel regulator activity
 is_a: GO:0008200 ! ion channel inhibitor activity
 relationship: negatively_regulates GO:0005262 ! calcium channel activity
@@ -160264,6 +160730,8 @@
 name: potassium channel inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents, or reduces the activity of a potassium channel." [GOC:mah]
+xref: Reactome:R-HSA-381644 "ER calcium channels open in response to activated PKA"
+xref: Reactome:R-HSA-381713 "Potassium voltage-gated channels close in reponse to PKA"
 is_a: GO:0008200 ! ion channel inhibitor activity
 is_a: GO:0015459 ! potassium channel regulator activity
 relationship: negatively_regulates GO:0005267 ! potassium channel activity
@@ -160621,12 +161089,9 @@
 alt_id: GO:0030349
 alt_id: GO:0050430
 alt_id: GO:0051535
-def: "Binding to a syntaxin, a SNAP receptor involved in the docking of synaptic vesicles at the presynaptic zone of a synapse." [ISBN:0198506732]
-synonym: "syntaxin-13 binding" NARROW []
-synonym: "syntaxin-2 binding" NARROW []
-synonym: "syntaxin-5 binding" NARROW []
-synonym: "syntaxin-6 binding" NARROW []
+def: "Binding to a syntaxin, a SNAP receptor involved in the docking of vesicles." [ISBN:0198506732]
 is_a: GO:0000149 ! SNARE binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31404" xsd:anyURI
 
 [Term]
 id: GO:0019907
@@ -161629,6 +162094,7 @@
 namespace: molecular_function
 def: "Binding to a heme, a compound composed of iron complexed in a porphyrin (tetrapyrrole) ring." [GOC:ai]
 synonym: "haem binding" EXACT []
+xref: Reactome:R-HSA-9661419 "ALB binds extracellular heme"
 is_a: GO:0046906 ! tetrapyrrole binding
 
 [Term]
@@ -167248,8 +167714,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + glucose(out) = protein histidine + glucose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "glucose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022856
@@ -167257,8 +167724,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sorbitol(out) = protein histidine + sorbitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "sorbitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015576 ! sorbitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022857
@@ -167278,12 +167746,26 @@
 synonym: "uptake permease activity" RELATED []
 synonym: "uptake transmembrane transporter activity" RELATED []
 xref: Reactome:R-HSA-1236947 "Egress of internalized antigen to the cytosol via sec61"
+xref: Reactome:R-HSA-2142859 "Growing HA is extruded from the cell by ABCC5"
+xref: Reactome:R-HSA-2161500 "abacavir [extracellular] => abacavir [cytosol]"
 xref: Reactome:R-HSA-429036 "SLC2A9 transports Fru, Glc, urate"
 xref: Reactome:R-HSA-434650 "MATEs mediate extrusion of xenobiotics"
+xref: Reactome:R-HSA-549129 "OCT1 transports organic cations into hepatic cells"
+xref: Reactome:R-HSA-549279 "OCT2 mediates tubular uptake of organic cations in the kidney"
+xref: Reactome:R-HSA-549304 "OCT3 mediates renal clearance of organic cations"
+xref: Reactome:R-HSA-549322 "OCT1 transports organic cations out of hepatic cells"
+xref: Reactome:R-HSA-561041 "OAT1,2,3 transport organic anions with antiport of dicarboxylic acids"
+xref: Reactome:R-HSA-561054 "OCT2 mediates tubular secretion of organic cations in the kidney"
+xref: Reactome:R-HSA-561072 "OCT3 mediates renal uptake of organic cations"
 xref: Reactome:R-HSA-5638209 "Defective SLC2A9 does not transport Fru, Glc, urate"
 xref: Reactome:R-HSA-5671707 "Fe3+ dissociates from SLC22A17:LCN2:2,5DHBA"
 xref: Reactome:R-HSA-6784434 "An unknown carrier transports cytosolic glyoxylate to the peroxisome"
 xref: Reactome:R-HSA-6784436 "An unknown carrier transports mitochondrial glyoxylate to the cytosol"
+xref: Reactome:R-HSA-8954513 "NAASP translocates from mitochondrial matrix to cytosol"
+xref: Reactome:R-HSA-9794270 "SLCO1A2 transports Cipro(1+) into the cytosol"
+xref: Reactome:R-HSA-9794523 "SLC22A1 transports Cipro into renal cell"
+xref: Reactome:R-HSA-9794830 "SLC22A8 transports Cipro into renal cell"
+xref: Reactome:R-HSA-9795207 "SLC22A1 transports Cipro into hepatic cell"
 is_a: GO:0005215 ! transporter activity
 relationship: part_of GO:0055085 ! transmembrane transport
 
@@ -167337,17 +167819,23 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + lactose(out) = protein histidine + lactose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "lactose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015155 ! lactose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022870
 name: protein-N(PI)-phosphohistidine-mannose phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannose(out) = protein histidine + mannose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane according to the reaction: D-mannose(out) + N(pros)-phospho-L-histidyl-[protein] = D-mannose 6-phosphate(in) + L-histidyl-[protein]. This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49232]
 synonym: "mannose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.191
+xref: RHEA:49232
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015578 ! mannose transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.191
+property_value: skos:exactMatch RHEA:49232
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022871
@@ -167358,13 +167846,14 @@
 xref: EC:2.7.1.206
 xref: MetaCyc:RXN-15094
 xref: RHEA:49296
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
 property_value: skos:exactMatch EC:2.7.1.206
-property_value: skos:narrowMatch RHEA:49296
+property_value: skos:exactMatch RHEA:49296
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17289" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022872
@@ -167372,8 +167861,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannitol(out) = protein histidine + mannitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "mannitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015575 ! mannitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022873
@@ -167382,16 +167872,22 @@
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + maltose(out) = protein histidine + maltose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "maltose PTS transporter activity" EXACT []
 is_a: GO:0005363 ! maltose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022874
 name: protein-N(PI)-phosphohistidine-cellobiose phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + cellobiose(out) = protein histidine + cellobiose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane according to the reaction: D-cellobiose(out) + N(pros)-phospho-L-histidyl-[protein] = 6-phospho-beta-D-glucosyl-(1->4)-D-glucose(in) + L-histidyl-[protein]. This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49292]
 synonym: "cellobiose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.205
+xref: RHEA:49292
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0019191 ! cellobiose transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.205
+property_value: skos:exactMatch RHEA:49292
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022875
@@ -167399,8 +167895,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactitol(out) = protein histidine + galactitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "galactitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015577 ! galactitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022876
@@ -167408,8 +167905,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactosamine(out) = protein histidine + galactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "galactosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0019196 ! galactosamine transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022877
@@ -167418,7 +167916,8 @@
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + fructose(out) = protein histidine + fructose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "fructose PTS transporter activity" EXACT []
 is_a: GO:0005353 ! fructose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022878
@@ -167426,12 +167925,15 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sucrose(out) = protein histidine + sucrose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "sucrose PTS transporter activity" EXACT []
+xref: EC:2.7.1.211
 xref: MetaCyc:SUCROSEPHOSPHO-RXN
 xref: RHEA:49236
 is_a: GO:0008515 ! sucrose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: skos:exactMatch EC:2.7.1.211
 property_value: skos:exactMatch RHEA:49236
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022879
@@ -167439,18 +167941,24 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + trehalose(out) = protein histidine + trehalose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "trehalose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015574 ! trehalose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022880
 name: protein-N(PI)-phosphohistidine-N-acetylglucosamine phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylglucosamine(out) = protein histidine + N-acetylglucosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane by the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49240]
 synonym: "N-acetylglucosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.193
+xref: RHEA:49240
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015572 ! N-acetylglucosamine transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.193
+property_value: skos:exactMatch RHEA:49240
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27517" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022881
@@ -167458,18 +167966,19 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylgalactosamine(out) = protein histidine + N-acetylgalactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "N-acetylgalactosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015571 ! N-acetylgalactosamine transmembrane transporter activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022882
 name: protein-N(PI)-phosphohistidine-beta-glucoside phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, PMID:19251853]
 synonym: "beta-glucoside PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
-is_a: GO:0015573 ! beta-glucoside transmembrane transporter activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022883
@@ -168279,6 +168788,7 @@
 synonym: "divalent metal ion transport" NARROW []
 synonym: "heavy metal ion transport" NARROW []
 synonym: "metal ion export" NARROW []
+xref: Reactome:R-HSA-425410 "Metal ion SLC transporters"
 is_a: GO:0006812 ! monoatomic cation transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23033" xsd:anyURI
 created_by: mah
@@ -169627,6 +170137,7 @@
 synonym: "uridine diphosphoacetylglucosamine-alpha-mannoside beta1->6-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.155]
 xref: EC:2.4.1.155
 xref: MetaCyc:2.4.1.155-RXN
+xref: Reactome:R-HSA-8932427 "MGAT5B transfers GlcNAc from UDP-GlcNAc to GlcNAc-Man-DAG1"
 xref: Reactome:R-HSA-9696980 "Spike trimer glycoside chains get additional branches"
 xref: Reactome:R-HSA-975916 "Addition of GlcNAc to position 5 by MGAT5"
 xref: RHEA:16921
@@ -169870,6 +170381,7 @@
 subset: goslim_generic
 subset: goslim_pombe
 subset: goslim_prokaryote
+subset: goslim_yeast
 synonym: "cellular protein breakdown" EXACT []
 synonym: "cellular protein catabolic process" EXACT []
 synonym: "cellular protein catabolism" EXACT []
@@ -169923,6 +170435,7 @@
 synonym: "proteoglycan breakdown" EXACT []
 synonym: "proteoglycan catabolism" EXACT []
 synonym: "proteoglycan degradation" EXACT []
+xref: Reactome:R-HSA-2024101 "CS/DS degradation"
 is_a: GO:0006029 ! proteoglycan metabolic process
 is_a: GO:0006516 ! glycoprotein catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -170346,6 +170859,7 @@
 synonym: "heparan sulphate proteoglycan metabolic process" EXACT []
 synonym: "heparan sulphate proteoglycan metabolism" EXACT []
 synonym: "heparin proteoglycan metabolic process" RELATED []
+xref: Reactome:R-HSA-1638091 "Heparan sulfate/heparin (HS-GAG) metabolism"
 is_a: GO:0006029 ! proteoglycan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29294" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -170380,7 +170894,6 @@
 synonym: "chondroitin sulfate metabolism" EXACT []
 synonym: "chondroitin sulphate metabolic process" EXACT []
 synonym: "chondroitin sulphate metabolism" EXACT []
-xref: Reactome:R-HSA-1793185 "Chondroitin sulfate/dermatan sulfate metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0050654
@@ -170412,7 +170925,6 @@
 synonym: "chondroitin sulfate synthesis" EXACT []
 synonym: "chondroitin sulphate biosynthesis" EXACT []
 synonym: "chondroitin sulphate biosynthetic process" EXACT []
-xref: Reactome:R-HSA-2022870 "Chondroitin sulfate biosynthesis"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
@@ -170453,7 +170965,6 @@
 synonym: "dermatan sulfate synthesis" EXACT []
 synonym: "dermatan sulphate biosynthesis" EXACT []
 synonym: "dermatan sulphate biosynthetic process" EXACT []
-xref: Reactome:R-HSA-2022923 "Dermatan sulfate biosynthesis"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
@@ -170531,7 +171042,6 @@
 synonym: "hyaluronan biosynthesis" EXACT []
 synonym: "hyaluronan formation" EXACT []
 synonym: "hyaluronan synthesis" EXACT []
-xref: Reactome:R-HSA-2142850 "Hyaluronan biosynthesis and export"
 is_a: GO:0006024 ! glycosaminoglycan biosynthetic process
 is_a: GO:0030212 ! hyaluronan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -170545,7 +171055,7 @@
 synonym: "hyaluronan catabolism" EXACT []
 synonym: "hyaluronan degradation" EXACT []
 xref: MetaCyc:PWY-7645
-xref: Reactome:R-HSA-2160916 "Hyaluronan uptake and degradation"
+xref: Reactome:R-HSA-2160916 "Hyaluronan degradation"
 is_a: GO:0006027 ! glycosaminoglycan catabolic process
 is_a: GO:0030212 ! hyaluronan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
@@ -170581,6 +171091,7 @@
 synonym: "T lymphocyte differentiation" EXACT []
 synonym: "T-cell differentiation" EXACT []
 synonym: "T-lymphocyte differentiation" EXACT []
+xref: Reactome:R-HSA-9945266 "Differentiation of T cells"
 is_a: GO:0030098 ! lymphocyte differentiation
 is_a: GO:0042110 ! T cell activation
 
@@ -171291,7 +171802,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: testosterone + NAD(P)+ = androst-4-ene-3,17-dione + NAD(P)H + H+." [EC:1.1.1.51]
 comment: This term was obsoleted because it is an unnecessary grouing class.
-xref: Reactome:R-HSA-193064 "HSD17B3-like proteins reducde ANDST to TEST"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28298" xsd:anyURI
 is_obsolete: true
 consider: GO:0047035
@@ -171751,6 +172261,8 @@
 synonym: "p53 signaling pathway" RELATED [GOC:mah, GOC:vk]
 synonym: "p53-mediated DNA damage response" BROAD []
 synonym: "TP53 signaling pathway" RELATED [GOC:mah, GOC:vk]
+xref: Reactome:R-HSA-6791312 "TP53 Regulates Transcription of Cell Cycle Genes"
+xref: Reactome:R-HSA-69563 "p53-Dependent G1 DNA Damage Response"
 is_a: GO:0042770 ! signal transduction in response to DNA damage
 is_a: GO:0072331 ! signal transduction by p53 class mediator
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26003" xsd:anyURI
@@ -172576,26 +173088,28 @@
 id: GO:0030418
 name: nicotianamine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+def: "The chemical reactions and pathways resulting in the formation of nicotianamine." [GOC:curators, PMID:10069850]
 synonym: "nicotianamine anabolism" EXACT []
 synonym: "nicotianamine biosynthesis" EXACT []
 synonym: "nicotianamine formation" EXACT []
 synonym: "nicotianamine synthesis" EXACT []
+xref: MetaCyc:PWY-5957
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0042401 ! biogenic amine biosynthetic process
 is_a: GO:0072351 ! tricarboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0030419
-name: nicotianamine catabolic process
+name: obsolete nicotianamine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+comment: This term was obsoleted because there is no evidence that a specific pathway for nicotianamine degradation exists.
 synonym: "nicotianamine breakdown" EXACT []
 synonym: "nicotianamine catabolism" EXACT []
 synonym: "nicotianamine degradation" EXACT []
-is_a: GO:0009063 ! amino acid catabolic process
-is_a: GO:0042402 ! biogenic amine catabolic process
-is_a: GO:0072352 ! tricarboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0030420
@@ -172698,7 +173212,7 @@
 synonym: "L-kynurenine hydrolase activity" RELATED [EC:3.7.1.3]
 xref: EC:3.7.1.3
 xref: MetaCyc:KYNURENINASE-RXN
-xref: Reactome:R-HSA-71217 "3-hydroxykynurenine + H2O => 3-hydroxyanthranilate + alanine"
+xref: Reactome:R-HSA-71217 "3-hydroxykynurenine + H2O => 3-hydroxyanthranilate + alanine + H+"
 xref: RHEA:16813
 xref: RHEA:25143
 is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
@@ -173307,7 +173821,6 @@
 def: "OBSOLETE. Enables the transfer of inorganic diphosphate across a membrane." [PMID:11326272]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
 synonym: "inorganic pyrophosphate transporter activity" EXACT []
-xref: Reactome:R-HSA-5226964 "ANKH transports PPi from cytosol to extracellular region"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -174632,7 +175145,6 @@
 synonym: "D-alanine formation" EXACT []
 synonym: "D-alanine synthesis" EXACT []
 is_a: GO:0006523 ! alanine biosynthetic process
-is_a: GO:0046436 ! D-alanine metabolic process
 is_a: GO:0046437 ! D-amino acid biosynthetic process
 
 [Term]
@@ -175056,6 +175568,12 @@
 synonym: "protein recruiting activity" RELATED []
 synonym: "protein-containing complex scaffold activity" BROAD []
 synonym: "protein-protein adaptor" NARROW []
+xref: Reactome:R-HSA-3780997 "PPP1R3C binds to glycogen:GYG2:GYS2"
+xref: Reactome:R-HSA-3781001 "EPM2A dimer binds PPP1R3C:phosphoglycogen-GYG1 complex"
+xref: Reactome:R-HSA-3781021 "EPM2A dimer binds PPP1R3C:phosphoglycogen-GYG2 complex"
+xref: Reactome:R-HSA-3781023 "PPP1R3C binds to glycogen:GYG1:GYS1"
+xref: Reactome:R-HSA-8871366 "p-Y389,400-RUFY1 binds RAB4A:GTP, RAB5:GTP, RAB14:GTP"
+xref: Reactome:R-HSA-8871376 "p-Y389,400-RUFY1 binds PI3P"
 is_a: GO:0060090 ! molecular adaptor activity
 relationship: has_part GO:0005515 ! protein binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18655" xsd:anyURI
@@ -175777,7 +176295,7 @@
 id: GO:0030738
 name: tyramine N-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tyramine = N-methyltyramine + S-adenosyl-L-homocysteine + H+." [EC:2.1.1.27, RHEA:14865]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + tyramine = N-methyltyramine + S-adenosyl-L-homocysteine + H+." [RHEA:14865]
 synonym: "DIB O-methyltransferase (3,5-diiodo-4-hydroxy-benzoic acid)" RELATED [EC:2.1.1.27]
 synonym: "S-adenosyl-L-methionine:tyramine N-methyltransferase activity" RELATED [EC:2.1.1.27]
 synonym: "S-adenosyl-methionine:tyramine N-methyltransferase activity" RELATED [EC:2.1.1.27]
@@ -175858,9 +176376,10 @@
 
 [Term]
 id: GO:0030743
-name: 23S rRNA (adenosine(1067)-2'-O)-methyltransferase activity
+name: 23S rRNA (adenosine(1067)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: adenosine(1067) in 23S rRNA + S-adenosyl-L-methionine = 2'-O-methyladenosine(1067) in 23S rRNA + H+ + S-adenosyl-L-homocysteine." [EC:2.1.1.230]
+synonym: "23S rRNA (adenosine(1067)-2'-O)-methyltransferase activity" EXACT []
 synonym: "ribosomal ribonucleate adenosine 2'-methyltransferase activity" RELATED [EC:2.1.1.230]
 synonym: "RNA-pentose methylase activity" RELATED []
 synonym: "rRNA adenosine 2'-methylase activity" RELATED [EC:2.1.1.230]
@@ -176329,7 +176848,7 @@
 id: GO:0030772
 name: tryptophan 2-C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + L-tryptophan = S-adenosyl-L-homocysteine + L-2-methyltryptophan + H+." [EC:2.1.1.106, RHEA:17321]
+def: "Catalysis of the reaction: L-tryptophan + S-adenosyl-L-methionine = 2-methyl-L-tryptophan + S-adenosyl-L-homocysteine + H+." [RHEA:17321]
 synonym: "S-adenosyl-L-methionine:L-tryptophan 2-C-methyltransferase activity" RELATED [EC:2.1.1.106]
 synonym: "S-adenosylmethionine:tryptophan 2-methyltransferase activity" RELATED [EC:2.1.1.106]
 synonym: "tryptophan 2-methyltransferase activity" RELATED [EC:2.1.1.106]
@@ -180003,12 +180522,14 @@
 name: organelle membrane
 namespace: cellular_component
 def: "A membrane that is one of the two lipid bilayers of an organelle envelope or the outermost membrane of single membrane bound organelle." [GOC:dos, GOC:mah]
+subset: gocheck_do_not_annotate
 synonym: "intracellular membrane" RELATED [NIF_Subcellular:sao830981606]
 xref: NIF_Subcellular:sao830981606
 is_a: GO:0016020 ! membrane
 intersection_of: GO:0016020 ! membrane
 intersection_of: part_of GO:0043227 ! membrane-bounded organelle
 relationship: part_of GO:0043227 ! membrane-bounded organelle
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31446" xsd:anyURI
 
 [Term]
 id: GO:0031091
@@ -181066,6 +181587,8 @@
 name: phosphatidylcholine binding
 namespace: molecular_function
 def: "Binding to a phosphatidylcholine, a glycophospholipid in which a phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:mah, ISBN:0198506732]
+xref: Reactome:R-HSA-8873794 "PCTP binds PC"
+xref: Reactome:R-HSA-8873923 "STARD10 binds LPCAT1 and PC"
 is_a: GO:0005543 ! phospholipid binding
 is_a: GO:0043169 ! cation binding
 is_a: GO:0050997 ! quaternary ammonium group binding
@@ -181376,24 +181899,26 @@
 id: GO:0031240
 name: external side of cell outer membrane
 namespace: cellular_component
-def: "The side of the outer membrane that is opposite to the side that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
+def: "The leaflet of the outer membrane that is opposite to the side that faces the periplasm of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mlg, GOC:mtg_sensu]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of cell outer membrane" EXACT [GOC:ab]
 synonym: "external side of outer membrane" RELATED []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009279 ! cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0031241
 name: periplasmic side of cell outer membrane
 namespace: cellular_component
-def: "The side (leaflet) of the outer membrane that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
+def: "The leaflet of a outer cell membrane that faces the periplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mlg, GOC:mtg_sensu]
 comment: In GO, 'internal side' still refers to part of the membrane and does not refer to components beyond (inside of) the membrane.
 synonym: "internal leaflet of cell outer membrane" EXACT [GOC:ab]
 synonym: "internal side of cell outer membrane" EXACT []
 synonym: "internal side of outer membrane" EXACT []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009279 ! cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0031242
@@ -182004,7 +182529,7 @@
 synonym: "T-cell co-stimulation" EXACT []
 synonym: "T-cell costimulation" EXACT []
 synonym: "T-lymphocyte costimulation" EXACT []
-xref: Reactome:R-HSA-388841 "Costimulation by the CD28 family"
+xref: Reactome:R-HSA-388841 "Regulation of T cell activation by CD28 family"
 is_a: GO:0031294 ! lymphocyte costimulation
 is_a: GO:0050870 ! positive regulation of T cell activation
 
@@ -182055,9 +182580,10 @@
 
 [Term]
 id: GO:0031299
-name: taurine-pyruvate aminotransferase activity
+name: taurine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyruvate + taurine = L-alanine + sulfoacetaldehyde." [PMID:11082195, RHEA:10420]
+def: "Catalysis of the reaction: taurine + pyruvate = sulfoacetaldehyde + L-alanine." [PMID:11082195, RHEA:10420]
+synonym: "taurine-pyruvate aminotransferase activity" EXACT []
 synonym: "taurine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.77]
 synonym: "Tpa" RELATED [EC:2.6.1.77]
 xref: EC:2.6.1.77
@@ -182068,6 +182594,7 @@
 property_value: skos:exactMatch EC:2.6.1.77
 property_value: skos:exactMatch RHEA:10420
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0031300
@@ -182370,7 +182897,6 @@
 synonym: "up regulation of cellular metabolic process" EXACT []
 synonym: "up-regulation of cellular metabolic process" EXACT []
 synonym: "upregulation of cellular metabolic process" EXACT []
-xref: Reactome:R-HSA-163765 "ChREBP activates metabolic gene expression"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27189" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
@@ -183251,7 +183777,6 @@
 name: regulation of protein modification process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the covalent alteration of one or more amino acid residues within a protein." [GOC:mah, GOC:tb]
-subset: goslim_yeast
 is_a: GO:0051246 ! regulation of protein metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0036211 ! protein modification process
@@ -184842,6 +185367,7 @@
 xref: Reactome:R-HSA-1234177 "Cytosolic PHD2,3 hydroxylates proline residues on HIF1A"
 xref: Reactome:R-HSA-1234179 "Cytosolic PHD2,3 hydroxylates proline residues on EPAS1 (HIF2A)"
 xref: Reactome:R-HSA-1234181 "Nuclear PHD1,3 hydroxylates proline residues on HIF1A"
+xref: Reactome:R-HSA-9918779 "Proline hydroxylases hydroxylate Polyprotein"
 xref: RHEA:63484
 is_a: GO:0031543 ! peptidyl-proline dioxygenase activity
 property_value: skos:exactMatch RHEA:63484
@@ -189384,7 +189910,9 @@
 synonym: "up regulation of insulin secretion" EXACT []
 synonym: "up-regulation of insulin secretion" EXACT []
 synonym: "upregulation of insulin secretion" EXACT []
+xref: Reactome:R-HSA-381676 "Glucagon-like Peptide-1 (GLP1) regulates insulin secretion"
 xref: Reactome:R-HSA-399997 "Acetylcholine regulates insulin secretion"
+xref: Reactome:R-HSA-400451 "Free fatty acids regulate insulin secretion"
 is_a: GO:0050714 ! positive regulation of protein secretion
 is_a: GO:0050796 ! regulation of insulin secretion
 is_a: GO:0090277 ! positive regulation of peptide hormone secretion
@@ -189644,6 +190172,7 @@
 name: bile acid binding
 namespace: molecular_function
 def: "Binding to a bile acid, a steroid carboxylic acids occurring in bile." [GOC:rph]
+xref: Reactome:R-HSA-8873850 "STARD5 binds DCA, LCA"
 is_a: GO:0033293 ! monocarboxylic acid binding
 
 [Term]
@@ -192054,6 +192583,7 @@
 def: "Binding to phosphatidylinositol-3-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' position." [GOC:bf, PMID:10209156, PMID:11395417, PMID:11557775]
 synonym: "phosphatidylinositol 3-phosphate binding" EXACT []
 synonym: "PtdIns-3-P binding" EXACT []
+xref: Reactome:R-HSA-8870489 "PLEKHA4,(5,6) bind PI3P"
 is_a: GO:1901981 ! phosphatidylinositol phosphate binding
 
 [Term]
@@ -194344,7 +194874,7 @@
 id: GO:0032473
 name: cytoplasmic side of mitochondrial outer membrane
 namespace: cellular_component
-def: "The external (cytoplasmic) face of the mitochondrial outer membrane." [GOC:mah]
+def: "The leaflet of the mitochondrial outer membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mah]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "cytosolic side of mitochondrial outer membrane" EXACT []
 synonym: "external leaflet of mitochondrial outer membrane" EXACT [GOC:ab]
@@ -194352,6 +194882,7 @@
 synonym: "external side of mitochondrial outer membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005741 ! mitochondrial outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0032474
@@ -198701,6 +199232,7 @@
 name: lead ion binding
 namespace: molecular_function
 def: "Binding to lead (Pb) ions." [GOC:mah]
+xref: Reactome:R-HSA-190141 "ALAD binds to Pb2+"
 is_a: GO:0046872 ! metal ion binding
 
 [Term]
@@ -199397,64 +199929,34 @@
 
 [Term]
 id: GO:0032847
-name: regulation of cellular pH reduction
+name: obsolete regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "regulation of cell pH reduction" EXACT []
-synonym: "regulation of cellular acidification" EXACT []
-synonym: "regulation of intracellular acidification" RELATED []
-synonym: "regulation of intracellular pH reduction" EXACT []
-synonym: "regulation of reduction of cellular pH" EXACT []
-synonym: "regulation of reduction of pH in cell" EXACT []
-is_a: GO:0050789 ! regulation of biological process
-is_a: GO:0051453 ! regulation of intracellular pH
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0051452 ! intracellular pH reduction
-relationship: regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that modulates the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0051452
+consider: GO:0051454
 
 [Term]
 id: GO:0032848
-name: negative regulation of cellular pH reduction
+name: obsolete negative regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "down regulation of cellular pH reduction" EXACT []
-synonym: "down-regulation of cellular pH reduction" EXACT []
-synonym: "downregulation of cellular pH reduction" EXACT []
-synonym: "inhibition of cellular pH reduction" NARROW []
-synonym: "negative regulation of cell pH reduction" EXACT []
-synonym: "negative regulation of cellular acidification" EXACT []
-synonym: "negative regulation of intracellular acidification" RELATED []
-synonym: "negative regulation of intracellular pH reduction" EXACT []
-synonym: "negative regulation of reduction of cellular pH" EXACT []
-synonym: "negative regulation of reduction of pH in cell" EXACT []
-is_a: GO:0032847 ! regulation of cellular pH reduction
-is_a: GO:0048519 ! negative regulation of biological process
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: negatively_regulates GO:0051452 ! intracellular pH reduction
-relationship: negatively_regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0051454
 
 [Term]
 id: GO:0032849
-name: positive regulation of cellular pH reduction
+name: obsolete positive regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "activation of cellular pH reduction" NARROW []
-synonym: "positive regulation of cell pH reduction" EXACT []
-synonym: "positive regulation of cellular acidification" EXACT []
-synonym: "positive regulation of intracellular acidification" RELATED []
-synonym: "positive regulation of intracellular pH reduction" EXACT []
-synonym: "positive regulation of reduction of cellular pH" EXACT []
-synonym: "positive regulation of reduction of pH in cell" EXACT []
-synonym: "stimulation of cellular pH reduction" NARROW []
-synonym: "up regulation of cellular pH reduction" EXACT []
-synonym: "up-regulation of cellular pH reduction" EXACT []
-synonym: "upregulation of cellular pH reduction" EXACT []
-is_a: GO:0032847 ! regulation of cellular pH reduction
-is_a: GO:0048518 ! positive regulation of biological process
-is_a: GO:0051452 ! intracellular pH reduction
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0051452 ! intracellular pH reduction
-relationship: positively_regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that activates or increases the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0051452
 
 [Term]
 id: GO:0032865
@@ -200892,7 +201394,8 @@
 xref: Reactome:R-HSA-1307803 "TIMM22 inserts proteins into inner membrane"
 xref: Reactome:R-HSA-9902096 "COX18 inserts nascent MT-CO2 in COX20:TMEM77"
 xref: Reactome:R-HSA-9906955 "MT-ND4 is translated"
-is_a: GO:0140597 ! protein carrier chaperone
+xref: Reactome:R-HSA-9926981 "Bam complex inserts Hbp into outer membrane"
+is_a: GO:0140597 ! protein carrier activity
 relationship: part_of GO:0090150 ! establishment of protein localization to membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16976" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20414" xsd:anyURI
@@ -202130,13 +202633,14 @@
 
 [Term]
 id: GO:0033094
-name: putrescine--2-oxoglutarate transaminase activity
+name: putrescine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H2O. The enzymatic part of the reaction produces 4-aminobutanal that spontaneously cyclizes to form 1-pyrroline." [EC:2.6.1.82, GOC:mlg, RHEA:12268]
+def: "Catalysis of the reaction: putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H2O. The enzymatic part of the reaction produces 4-aminobutanal that spontaneously cyclizes to form 1-pyrroline." [EC:2.6.1.82, RHEA:12268]
 synonym: "butane-1,4-diamine:2-oxoglutarate aminotransferase activity" RELATED []
 synonym: "PAT activity" RELATED []
 synonym: "putrescine aminotransferase activity" BROAD [EC:2.6.1.82]
 synonym: "putrescine transaminase activity" BROAD []
+synonym: "putrescine--2-oxoglutarate transaminase activity" EXACT []
 synonym: "putrescine-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.82]
 synonym: "putrescine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.82]
 synonym: "putrescine:alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.82]
@@ -202149,6 +202653,7 @@
 property_value: skos:exactMatch RHEA:12268
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25992" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033095
@@ -203164,7 +203669,7 @@
 id: GO:0033196
 name: tryparedoxin peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: tryparedoxin + hydrogen peroxide = tryparedoxin disulfide + H2O." [GOC:mah, PMID:32388269]
+def: "Catalysis of the reaction: tryparedoxin + H2O2 = tryparedoxin disulfide + H2O." [GOC:mah, PMID:32388269]
 synonym: "TXNPx activity" EXACT []
 xref: MetaCyc:1.11.1.15-RXN
 is_a: GO:0004601 ! peroxidase activity
@@ -204810,12 +205315,13 @@
 id: GO:0033353
 name: S-adenosylmethionine cycle
 namespace: biological_process
-def: "A cyclic series of interconversions involving S-adenosylmethionine, S-adenosyl-L-homocysteine, L-cysteine, and L-methionine. Couples utilization of the methyl group of SAM with recycling of the homocysteinyl group and regeneration of methionine." [GOC:mah, MetaCyc:PWY-5041]
+def: "A cyclic series of interconversions involving S-adenosylmethionine (SAM), S-adenosyl-L-homocysteine, L-homocysteine, and L-methionine. Couples utilization of the methyl group of SAM with recycling of the homocysteinyl group and regeneration of L-methionine." [MetaCyc:PWY-5041, PMID:31950558, PMID:39394448]
 synonym: "activated methyl cycle" EXACT []
 synonym: "SAM cycle" EXACT []
 xref: MetaCyc:PWY-5041
 is_a: GO:0046500 ! S-adenosylmethionine metabolic process
 relationship: has_part GO:0008898 ! S-adenosylmethionine-homocysteine S-methyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
 
 [Term]
 id: GO:0033354
@@ -204829,7 +205335,7 @@
 id: GO:0033355
 name: ascorbate glutathione cycle
 namespace: biological_process
-def: "A cyclic series of interconversions involving L-ascorbate and glutathione that scavenges hydrogen peroxide and reduces it to water, with concomitant oxidation of NADPH." [GOC:mah, MetaCyc:PWY-2261]
+def: "A cyclic series of interconversions involving L-ascorbate and glutathione that scavenges H2O2 and reduces it to water, with concomitant oxidation of NADPH." [GOC:mah, MetaCyc:PWY-2261]
 synonym: "hydrogen peroxide detoxification" RELATED []
 xref: MetaCyc:PWY-2261
 is_a: GO:0006749 ! glutathione metabolic process
@@ -204877,13 +205383,12 @@
 id: GO:0033359
 name: L-lysine biosynthetic process via diaminopimelate and N-succinyl-2-amino-6-ketopimelate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and N-succinyl-2-amino-6-ketopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah, MetaCyc:DAPLYSINESYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and N-succinyl-2-amino-6-ketopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah]
 synonym: "lysine anabolism via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine biosynthetic process via diaminopimelic acid and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine formation via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine synthesis via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
-xref: MetaCyc:DAPLYSINESYN-PWY
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
@@ -204892,13 +205397,12 @@
 id: GO:0033360
 name: L-lysine biosynthetic process via diaminopimelate and L-2-acetamido-6-oxoheptanedioate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and L-2-acetamido-6-oxoheptanedioate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah, MetaCyc:PWY-2941]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and L-2-acetamido-6-oxoheptanedioate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah]
 synonym: "lysine anabolism via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine biosynthetic process via diaminopimelic acid and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine formation via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine synthesis via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
-xref: MetaCyc:PWY-2941
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -204906,12 +205410,11 @@
 id: GO:0033361
 name: L-lysine biosynthetic process via diaminopimelate, dehydrogenase pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in a single enzymatic step." [GOC:mah, GOC:pr, MetaCyc:PWY-2942]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in a single enzymatic step." [GOC:mah, GOC:pr]
 synonym: "lysine anabolism via diaminopimelate, dehydrogenase pathway" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid, dehydrogenase pathway" EXACT []
 synonym: "lysine formation via diaminopimelate, dehydrogenase pathway" EXACT []
 synonym: "lysine synthesis via diaminopimelate, dehydrogenase pathway" EXACT []
-xref: MetaCyc:PWY-2942
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 relationship: has_part GO:0047850 ! diaminopimelate dehydrogenase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
@@ -204920,15 +205423,14 @@
 id: GO:0033362
 name: L-lysine biosynthetic process via diaminopimelate, diaminopimelate-aminotransferase pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in two enzymatic steps." [GOC:mah, GOC:pr, MetaCyc:PWY-5097]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in two enzymatic steps." [GOC:mah, GOC:pr]
 synonym: "lysine anabolism via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine biosynthesis via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine formation via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine synthesis via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
-xref: MetaCyc:PWY-5097
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 relationship: has_part GO:0008837 ! diaminopimelate epimerase activity
-relationship: has_part GO:0010285 ! L,L-diaminopimelate aminotransferase activity
+relationship: has_part GO:0010285 ! L,L-diaminopimelate:2-oxoglutarate transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
 [Term]
@@ -205199,14 +205701,17 @@
 synonym: "geranylgeranyl diphosphate biosynthesis" EXACT []
 synonym: "geranylgeranyl diphosphate formation" EXACT []
 synonym: "geranylgeranyl diphosphate synthesis" EXACT []
+xref: MetaCyc:PWY-5120
 xref: MetaCyc:PWY-5121
 xref: MetaCyc:PWY-5910
 is_a: GO:0008654 ! phospholipid biosynthetic process
 is_a: GO:0016114 ! terpenoid biosynthetic process
 is_a: GO:0033385 ! geranylgeranyl diphosphate metabolic process
+property_value: skos:narrowMatch MetaCyc:PWY-5120
 property_value: skos:narrowMatch MetaCyc:PWY-5121
 property_value: skos:narrowMatch MetaCyc:PWY-5910
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0033387
@@ -205303,7 +205808,6 @@
 synonym: "beta-alanine biosynthesis via 1,3 diaminopropane" EXACT []
 synonym: "beta-alanine formation via 1,3 diaminopropane" EXACT []
 synonym: "beta-alanine synthesis via 1,3 diaminopropane" EXACT []
-xref: MetaCyc:PWY-3981
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -205315,7 +205819,6 @@
 synonym: "beta-alanine biosynthesis via 3-hydroxypropionate" EXACT []
 synonym: "beta-alanine formation via 3-hydroxypropionate" EXACT []
 synonym: "beta-alanine synthesis via 3-hydroxypropionate" EXACT []
-xref: MetaCyc:PWY-3941
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -205326,7 +205829,6 @@
 synonym: "beta-alanine anabolism via 3-ureidopropionate" EXACT []
 synonym: "beta-alanine formation via 3-ureidopropionate" EXACT []
 synonym: "beta-alanine synthesis via 3-ureidopropionate" EXACT []
-xref: MetaCyc:PWY-3982
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -206180,7 +206682,11 @@
 synonym: "UDP-D-galacturonate formation" EXACT []
 synonym: "UDP-D-galacturonate synthesis" EXACT []
 xref: MetaCyc:PWY-4
+xref: MetaCyc:PWY-4861
 is_a: GO:0009226 ! nucleotide-sugar biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-4
+property_value: skos:narrowMatch MetaCyc:PWY-4861
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0033481
@@ -206290,7 +206796,6 @@
 synonym: "cholesterol formation via desmosterol" EXACT []
 synonym: "cholesterol synthesis via desmosterol" EXACT []
 xref: MetaCyc:PWY66-4
-xref: Reactome:R-HSA-6807047 "Cholesterol biosynthesis via desmosterol"
 is_a: GO:0006695 ! cholesterol biosynthetic process
 
 [Term]
@@ -206303,7 +206808,6 @@
 synonym: "cholesterol formation via lathosterol" EXACT []
 synonym: "cholesterol synthesis via lathosterol" EXACT []
 xref: MetaCyc:PWY66-341
-xref: Reactome:R-HSA-6807062 "Cholesterol biosynthesis via lathosterol"
 is_a: GO:0006695 ! cholesterol biosynthetic process
 
 [Term]
@@ -206505,31 +207009,44 @@
 
 [Term]
 id: GO:0033508
-name: L-glutamate catabolic process to pyruvate
+name: L-glutamate fermentation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate into pyruvate." [MetaCyc:PWY-5087]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, yielding energy in the form of ATP." [PMID:38821505]
 synonym: "glutamate breakdown to butyrate" EXACT []
 synonym: "glutamate degradation to butyrate" EXACT []
+synonym: "L-glutamate catabolic process to pyruvate" NARROW []
+synonym: "L-glutamate fermentation to propionate" NARROW []
+synonym: "L-glutamate fermentation to pyruvate" NARROW []
+synonym: "L-glutamate fermentation via 2-hydroxyglutarate" NARROW []
+synonym: "L-glutamate fermentation via L-citramalate" NARROW []
+xref: MetaCyc:GLUDEG-II-PWY
+xref: MetaCyc:P162-PWY
 xref: MetaCyc:PWY-5087
+xref: MetaCyc:PWY-5088
+xref: MetaCyc:PWY-8190
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0006538 ! L-glutamate catabolic process
 is_a: GO:0019665 ! amino acid fermentation
+property_value: skos:narrowMatch MetaCyc:GLUDEG-II-PWY
+property_value: skos:narrowMatch MetaCyc:P162-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5087
+property_value: skos:narrowMatch MetaCyc:PWY-5088
+property_value: skos:narrowMatch MetaCyc:PWY-8190
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31218" xsd:anyURI
 
 [Term]
 id: GO:0033509
-name: L-glutamate fermentation to propionate
+name: obsolete L-glutamate fermentation to propionate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including propionate." [GOC:mah, MetaCyc:PWY-5088]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including propionate." [GOC:mah]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown to propionate" EXACT []
 synonym: "glutamate degradation to propionate" EXACT []
 synonym: "L-glutamate catabolic process to propionate" EXACT []
-xref: MetaCyc:PWY-5088
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019541 ! propionate metabolic process
-is_a: GO:0019665 ! amino acid fermentation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0033510
@@ -206555,34 +207072,40 @@
 
 [Term]
 id: GO:0033512
-name: L-lysine catabolic process to acetyl-CoA via L-saccharopine
+name: obsolete L-lysine catabolic process to acetyl-CoA via L-saccharopine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-saccharopine." [GOC:mah, MetaCyc:LYSINE-DEG1-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-saccharopine." [GOC:mah, MetaCyc:LYSINE-DEG1-PWY]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via saccharopine" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via saccharopine" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033513
-name: L-lysine catabolic process to acetyl-CoA via 5-aminopentanamide
+name: obsolete L-lysine catabolic process to acetyl-CoA via 5-aminopentanamide
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate 5-aminopentanamide." [GOC:mah, MetaCyc:PWY-5280]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate 5-aminopentanamide." [GOC:mah, MetaCyc:PWY-5280]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via 5-aminopentanamide" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via 5-aminopentanamide" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033514
-name: L-lysine catabolic process to acetyl-CoA via L-pipecolate
+name: obsolete L-lysine catabolic process to acetyl-CoA via L-pipecolate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-pipecolate." [GOC:mah, MetaCyc:PWY66-425]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-pipecolate." [GOC:mah, MetaCyc:PWY66-425]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via L-pipecolate" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via L-pipecolate" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033515
@@ -206607,7 +207130,7 @@
 synonym: "L-methionine synthesis from homoserine via O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from homoserine via O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 is_obsolete: true
-replaced_by: GO:0019283
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0033517
@@ -207169,7 +207692,7 @@
 name: lactoferrin transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of lactoferrin from one side of a membrane to the other." [GOC:mlg]
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 relationship: part_of GO:0033571 ! lactoferrin transport
 
 [Term]
@@ -207323,7 +207846,6 @@
 synonym: "L-tyrosine formation from chorismate via L-arogenate" EXACT []
 synonym: "L-tyrosine synthesis from chorismate via L-arogenate" EXACT []
 synonym: "tyrosine biosynthetic process from chorismate via L-arogenate" BROAD []
-xref: MetaCyc:PWY-3461
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 is_a: GO:0009095 ! aromatic amino acid family biosynthetic process, prephenate pathway
 is_a: GO:0046417 ! chorismate metabolic process
@@ -207338,7 +207860,6 @@
 synonym: "L-phenylalanine biosynthesis from chorismate via phenylpyruvate" EXACT []
 synonym: "L-phenylalanine formation from chorismate via phenylpyruvate" EXACT []
 synonym: "L-phenylalanine synthesis from chorismate via phenylpyruvate" EXACT []
-xref: MetaCyc:PHESYN
 is_a: GO:0009094 ! L-phenylalanine biosynthetic process
 is_a: GO:0046417 ! chorismate metabolic process
 
@@ -207351,7 +207872,6 @@
 synonym: "L-phenylalanine biosynthesis from chorismate via L-arogenate" EXACT []
 synonym: "L-phenylalanine formation from chorismate via L-arogenate" EXACT []
 synonym: "L-phenylalanine synthesis from chorismate via L-arogenate" EXACT []
-xref: MetaCyc:PWY-3462
 is_a: GO:0009094 ! L-phenylalanine biosynthetic process
 is_a: GO:0009095 ! aromatic amino acid family biosynthetic process, prephenate pathway
 is_a: GO:0046417 ! chorismate metabolic process
@@ -208758,7 +209278,7 @@
 id: GO:0033713
 name: choline:oxygen 1-oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: choline + O2 = betaine aldehyde + hydrogen peroxide." [EC:1.1.3.17, RHEA:13505]
+def: "Catalysis of the reaction: choline + O2 = betaine aldehyde + H2O2." [EC:1.1.3.17, RHEA:13505]
 synonym: "choline oxidase activity" BROAD [EC:1.1.3.17]
 xref: EC:1.1.3.17
 xref: MetaCyc:CHOLINE-OXIDASE-RXN
@@ -208803,7 +209323,7 @@
 id: GO:0033716
 name: nucleoside oxidase (hydrogen peroxide-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reactions: adenosine + 2 O2 = 9-riburonosyladenine + 2 hydrogen peroxide." [EC:1.1.3.39, RHEA:15489]
+def: "Catalysis of the reaction: adenosine + 2 O2 + H2O = 9-riburonosyladenine + 2 H2O2 + H+." [RHEA:15489]
 comment: Two other reactions might be associated with this activity: (1a) adenosine + O2 = 5'-dehydroadenosine + hydrogen peroxide, and (1b) 5'-dehydroadenosine + O2 = 9-riburonosyladenine + hydrogen peroxide.
 synonym: "nucleoside:oxygen 5'-oxidoreductase (hydrogen peroxide-forming) activity" RELATED [EC:1.1.3.39]
 xref: EC:1.1.3.39
@@ -209782,9 +210302,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 7alpha-hydroxycholest-4-en-3-one + H+ + NADPH + O2 = 7alpha,12alpha-dihydroxycholest-4-en-3-one + H2O + NADP+." [PMID:10051404, PMID:1400444]
 comment: This term was obsoleted because it represents a specific susbstrate of sterol 12-alpha-hydroxylase activity ; GO:0008397.
-xref: Reactome:R-HSA-192157 "CYP8B1 12-hydroxylates 4CHOL7aOLONE"
-xref: Reactome:R-HSA-193709 "CYP8B1 12-hydroxylates 4CHOL7a,24(S)DIOL"
-xref: Reactome:R-HSA-193845 "CYP8B1 12-hydroxylates 4CHOL7a,27DONE"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24843" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29331" xsd:anyURI
 is_obsolete: true
@@ -210125,6 +210642,7 @@
 synonym: "type III iodothyronine deiodinase activity" NARROW [EC:1.21.99.3]
 xref: EC:1.21.99.3
 xref: MetaCyc:1.97.1.11-RXN
+xref: Reactome:R-HSA-350869 "Thyroxine is deiodinated to reverse triiodothyronine (RT3)"
 xref: RHEA:18897
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.99.3
@@ -210171,7 +210689,7 @@
 id: GO:0033801
 name: vitexin 2''-O-rhamnoside 7-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + vitexin 2''-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2''-O-beta-L-rhamnoside." [EC:2.1.1.153]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + vitexin 2''-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2''-O-beta-L-rhamnoside." [RHEA:11432]
 synonym: "S-adenosyl-L-methionine:vitexin-2''-O-beta-L-rhamnoside 7-O-methyltransferase activity" RELATED [EC:2.1.1.153]
 xref: EC:2.1.1.153
 xref: MetaCyc:RXN-4981
@@ -211009,14 +211527,15 @@
 
 [Term]
 id: GO:0033852
-name: thyroid-hormone transaminase activity
+name: thyroid-hormone:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3,5,3'-triiodo-L-thyronine = 3,5,3'-triiodothyropyruvate + L-glutamate." [EC:2.6.1.26, RHEA:19133]
+def: "Catalysis of the reaction: 3,3',5-triiodo-L-thyronine + 2-oxoglutarate = 3,5,3'-triiodothyropyruvate + L-glutamate." [RHEA:19133]
 synonym: "3,5-dinitrotyrosine aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "3,5-dinitrotyrosine transaminase activity" RELATED [EC:2.6.1.26]
 synonym: "L-3,5,3'-triiodothyronine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "thyroid hormone aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "thyroid-hormone aminotransferase activity" RELATED [EC:2.6.1.26]
+synonym: "thyroid-hormone transaminase activity" BROAD []
 xref: EC:2.6.1.26
 xref: KEGG_REACTION:R03952
 xref: MetaCyc:THYROID-HORMONE-AMINOTRANSFERASE-RXN
@@ -211025,12 +211544,14 @@
 property_value: skos:exactMatch EC:2.6.1.26
 property_value: skos:exactMatch RHEA:19133
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033853
-name: aspartate-prephenate aminotransferase activity
+name: L-aspartate:prephenate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-arogenate + oxaloacetate = prephenate + L-aspartate." [RHEA:20445]
+synonym: "aspartate-prephenate aminotransferase activity" EXACT []
 synonym: "L-arogenate:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.78]
 synonym: "L-aspartate:prephenate aminotransferase activity" RELATED [EC:2.6.1.78]
 synonym: "PAT" RELATED [EC:2.6.1.78]
@@ -211039,16 +211560,18 @@
 xref: EC:2.6.1.78
 xref: MetaCyc:PREPHENATE-ASP-TRANSAMINE-RXN
 xref: RHEA:20445
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.78
 property_value: skos:exactMatch RHEA:20445
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033854
-name: glutamate-prephenate aminotransferase activity
+name: L-glutamate:prephenate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-arogenate = L-glutamate + prephenate." [EC:2.6.1.79, RHEA:22880]
+def: "Catalysis of the reaction: 2-oxoglutarate + L-arogenate = L-glutamate + prephenate." [RHEA:22880]
+synonym: "glutamate-prephenate aminotransferase activity" EXACT []
 synonym: "L-arogenate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.79]
 synonym: "L-glutamate:prephenate aminotransferase activity" RELATED [EC:2.6.1.79]
 synonym: "PAT" RELATED [EC:2.6.1.79]
@@ -211057,20 +211580,22 @@
 xref: KEGG_REACTION:R07276
 xref: MetaCyc:PREPHENATE-TRANSAMINE-RXN
 xref: RHEA:22880
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.79
 property_value: skos:exactMatch RHEA:22880
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033855
-name: nicotianamine aminotransferase activity
+name: nicotianamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + nicotianamine = 3''-deamino-3''-oxonicotianamine + L-glutamate." [EC:2.6.1.80, RHEA:22104]
+def: "Catalysis of the reaction: nicotianamine + 2-oxoglutarate = 3''-deamino-3''-oxonicotianamine + L-glutamate." [RHEA:22104]
 synonym: "NAAT" RELATED [EC:2.6.1.80]
 synonym: "NAAT-I" RELATED [EC:2.6.1.80]
 synonym: "NAAT-II" RELATED [EC:2.6.1.80]
 synonym: "NAAT-III" RELATED [EC:2.6.1.80]
+synonym: "nicotianamine aminotransferase activity" EXACT []
 synonym: "nicotianamine transaminase activity" RELATED [EC:2.6.1.80]
 synonym: "nicotianamine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.80]
 xref: EC:2.6.1.80
@@ -211081,6 +211606,7 @@
 property_value: skos:exactMatch EC:2.6.1.80
 property_value: skos:exactMatch RHEA:22104
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033856
@@ -212034,6 +212560,8 @@
 xref: Reactome:R-HSA-548890 "Removal of the third glucose by glucosidase II and release from the chaperone"
 xref: Reactome:R-HSA-9683663 "N-glycan trimming of Spike"
 xref: Reactome:R-HSA-9694364 "N-glycan glucose trimming of Spike"
+xref: Reactome:R-HSA-9932352 "Glucosidase II removes glucose residue from Glu2Man9GlucNAc2-CDH1"
+xref: Reactome:R-HSA-9932913 "Glucosidase II removes glucose residue from Glu1Man9GlucNAc2-CDH1"
 is_a: GO:0090599 ! alpha-glucosidase activity
 property_value: skos:exactMatch EC:3.2.1.84
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -214092,7 +214620,6 @@
 synonym: "negative regulation by symbiont of plant hypersensitive response" NARROW [GOC:pamgo_curators]
 synonym: "suppression by symbiont of host defense-related programmed cell death" EXACT []
 synonym: "suppression of HR" NARROW [GOC:pamgo_curators]
-xref: Reactome:R-HSA-9636569 "Suppression of autophagy"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29278" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0052041
@@ -216533,16 +217060,15 @@
 
 [Term]
 id: GO:0034269
-name: discadenine catabolic process
+name: obsolete discadenine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of discadenine, (2S)-2-amino-4-{6-[(3-methylbut-2-en-1-yl)amino]-3H-purin-3-yl}butanoic acid." [GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of discadenine, (2S)-2-amino-4-{6-[(3-methylbut-2-en-1-yl)amino]-3H-purin-3-yl}butanoic acid." [GOC:mah]
+comment: This term was obsoleted because there is no evidence that a specific pathway for discadenine degradation exists.
 synonym: "discadenine breakdown" EXACT []
 synonym: "discadenine catabolism" EXACT []
 synonym: "discadenine degradation" EXACT []
-is_a: GO:0009823 ! cytokinin catabolic process
-is_a: GO:0072523 ! purine-containing compound catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034270
@@ -217473,7 +217999,6 @@
 synonym: "nicotinamide riboside salvage pathway" EXACT []
 synonym: "NR salvage pathway" EXACT []
 xref: MetaCyc:PWY3O-4106
-xref: Reactome:R-HSA-197264 "Nicotinamide salvaging"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29050" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0034355
@@ -217786,6 +218311,7 @@
 synonym: "gamma-aminobutyric acid-alpha-ketoglutaric acid aminotransferase activity" RELATED [EC:2.6.1.19]
 xref: EC:2.6.1.19
 xref: MetaCyc:GABATRANSAM-RXN
+xref: Reactome:R-HSA-916855 "PXLP-K357-ABAT dimer:2Fe-2S transforms GABA to SUCCSA"
 xref: RHEA:23352
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch EC:2.6.1.19
@@ -218806,6 +219332,7 @@
 synonym: "chondroitin 2-O-sulfotransferase activity" EXACT []
 synonym: "chondroitin 2-O-sulphotransferase activity" EXACT []
 xref: MetaCyc:RXN-11560
+xref: Reactome:R-HSA-9941305 "UST transfers sulfate to (2-)GlcA-GalNAc-6-sulfate"
 is_a: GO:0034481 ! chondroitin sulfotransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
 
@@ -219379,31 +219906,37 @@
 
 [Term]
 id: GO:0034547
-name: N-cyclopropylmelamine deaminase activity
+name: obsolete N-cyclopropylmelamine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyromazine + H2O = N-cyclopropylammeline + NH3." [UM-BBD_reactionID:r0825]
+def: "OBSOLETE. Catalysis of the reaction: cyromazine + H2O = N-cyclopropylammeline + NH3." [UM-BBD_reactionID:r0825]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8018
 xref: UM-BBD_reactionID:r0825
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034548
-name: N-cyclopropylammeline deaminase activity
+name: obsolete N-cyclopropylammeline deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-cyclopropylammeline + H2O = N-cyclopropylammelide + NH3." [UM-BBD_reactionID:r0826]
+def: "OBSOLETE. Catalysis of the reaction: N-cyclopropylammeline + H2O = N-cyclopropylammelide + NH3." [UM-BBD_reactionID:r0826]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8019
 xref: UM-BBD_reactionID:r0826
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034549
-name: N-cyclopropylammelide alkylamino hydrolase activity
+name: obsolete N-cyclopropylammelide alkylamino hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-cyclopropylammelide + H2O = cyclopropylamine + cyanuric acid." [UM-BBD_reactionID:r0827]
+def: "OBSOLETE. Catalysis of the reaction: N-cyclopropylammelide + H2O = cyclopropylamine + cyanuric acid." [UM-BBD_reactionID:r0827]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: KEGG_REACTION:R06972
 xref: MetaCyc:RXN-8020
 xref: UM-BBD_reactionID:r0827
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034550
@@ -219472,13 +220005,13 @@
 
 [Term]
 id: GO:0034557
-name: 2-hydroxylaminobenzoate reductase activity
+name: obsolete 2-hydroxylaminobenzoate reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-hydroxylaminobenzoate + NAD(P)H = anthranilate + NAD(P)+ + H2O." [MetaCyc:RXN-8848]
+def: "OBSOLETE. Catalysis of the reaction: 2-hydroxylaminobenzoate + NAD(P)H = anthranilate + NAD(P)+ + H2O." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "o-hydroxylaminobenzoate nitroreductase activity" RELATED [UM-BBD_reactionID:r0850]
-xref: MetaCyc:RXN-8848
-xref: UM-BBD_reactionID:r0850
-is_a: GO:0016638 ! oxidoreductase activity, acting on the CH-NH2 group of donors
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034558
@@ -219985,8 +220518,6 @@
 def: "OBSOLETE. Catalysis of the exonucleolytic cleavage of oligoribonucleotides to yield ribonucleoside 5'-phosphates." [GOC:mah]
 comment: This term was obsoleted because it corresponds to an assay.
 synonym: "RNA oligonucleotidase activity" EXACT []
-xref: Reactome:R-HSA-9009950 "PDE12 cleaves 2'-5' oligoadenylates"
-xref: Reactome:R-HSA-9615042 "Viral 2',5'-PDE cleaves 2'-5' oligoadenylates"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
 is_obsolete: true
 
@@ -220038,6 +220569,7 @@
 synonym: "H4biopterin binding" EXACT []
 synonym: "sapropterin binding" EXACT []
 xref: Reactome:R-HSA-1497784 "The cofactor BH4 is required for electron transfer in the eNOS catalytic cycle"
+xref: Reactome:R-HSA-1497796 "BH2 binding can lead to eNOS uncoupling"
 is_a: GO:0005488 ! binding
 
 [Term]
@@ -220411,18 +220943,21 @@
 name: nucleobase-containing compound biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:mah]
+subset: gocheck_do_not_annotate
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid anabolism" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid biosynthesis" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid formation" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid synthesis" NARROW []
 is_a: GO:0006139 ! nucleobase-containing compound metabolic process
 is_a: GO:0009058 ! biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0034655
 name: nucleobase-containing compound catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:mah]
+subset: gocheck_do_not_annotate
 subset: goslim_chembl
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid breakdown" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid catabolic process" RELATED [GOC:dph, GOC:tb]
@@ -220430,6 +220965,7 @@
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid degradation" EXACT []
 is_a: GO:0006139 ! nucleobase-containing compound metabolic process
 is_a: GO:0009056 ! catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0034656
@@ -222091,13 +222627,12 @@
 
 [Term]
 id: GO:0034823
-name: citronellyl-CoA ligase activity
+name: obsolete citronellyl-CoA ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: citronellate + CoASH + ATP = citronellyl-CoA + AMP + PPi." [UM-BBD_reactionID:r1157]
-xref: KEGG_REACTION:R08088
-xref: UM-BBD_reactionID:r1157
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+def: "OBSOLETE. Catalysis of the reaction: citronellate + CoASH + ATP = citronellyl-CoA + AMP + PPi." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034824
@@ -222916,12 +223451,12 @@
 
 [Term]
 id: GO:0034912
-name: phthalate 3,4-cis-dihydrodiol dehydrogenase activity
+name: obsolete phthalate 3,4-cis-dihydrodiol dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: phthalate 3,4-cis-dihydrodiol + NAD+ = 3,4-dihydroxyphthalate + NADH + H+." [UM-BBD_reactionID:r1445]
-xref: MetaCyc:RXN5F9-56
-xref: UM-BBD_reactionID:r1445
-is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: phthalate 3,4-cis-dihydrodiol + NAD+ = 3,4-dihydroxyphthalate + NADH + H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034914
@@ -226015,9 +226550,11 @@
 id: GO:0035248
 name: alpha-1,4-N-acetylgalactosaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an N-acetyl-beta-D-galactosaminyl derivative + UDP-N-acetyl-alpha-D-galactosamine = an N-acetyl-alpha-D-galactosaminyl-(1->4)-N-acetyl-beta-D-galactosaminyl derivative + UDP + H+." [PMID:15130086]
+def: "Catalysis of the reaction: an N-acetyl-beta-D-galactosaminyl derivative + UDP-N-acetyl-alpha-D-galactosamine = an N-acetyl-alpha-D-galactosaminyl-(1->4)-N-acetyl-beta-D-galactosaminyl derivative + UDP + H+." [PMID:15130086, RHEA:86251]
 synonym: "alpha-1,4-GalNAc transferase activity" EXACT []
+xref: RHEA:86251
 is_a: GO:0008376 ! acetylgalactosaminyltransferase activity
+property_value: skos:exactMatch RHEA:86251
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31147" xsd:anyURI
 
 [Term]
@@ -226051,9 +226588,8 @@
 name: UDP-xylosyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a xylosyl group from UDP-xylose to an acceptor molecule." [PMID:30127001]
-xref: Reactome:R-HSA-5617138 "B4GAT1:GYLTL1B transfers Xyl from UDP-Xyl to GlcA-Xyl-GlcA"
 xref: Reactome:R-HSA-6785668 "Defective LARGE does not transfer Xyl from UDP-Xyl to GlcA"
-xref: Reactome:R-HSA-9638090 "B4GAT1:LARGE transfers Xyl from UDP-Xyl to GlcA-Xyl-GlcA"
+xref: Reactome:R-HSA-9638090 "LARGE1,2 elongates matriglycan chain with Xyl"
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
 is_a: GO:0042285 ! xylosyltransferase activity
 
@@ -227284,6 +227820,7 @@
 namespace: molecular_function
 def: "Binding to a selenocysteine insertion sequence (SECIS), a regulatory sequence within mRNA which directs incorporation of a selenocysteine at a stop codon (UGA) during translation." [GOC:imk, PMID:10760958]
 synonym: "SECIS binding" EXACT []
+xref: Reactome:R-HSA-2408529 "Sec-tRNA(Sec):EEFSEC:GTP binds to 80S Ribosome"
 is_a: GO:0003729 ! mRNA binding
 created_by: bf
 creation_date: 2010-03-11T10:42:28Z
@@ -227410,12 +227947,14 @@
 
 [Term]
 id: GO:0035380
-name: very long-chain-3-hydroxyacyl-CoA dehydrogenase activity
+name: very long-chain (3S)-3-hydroxyacyl-CoA dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a very long-chain fatty acid residue. A very long-chain fatty acid has an aliphatic tail containing more than 22 carbons." [GOC:pde]
+def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a very long-chain fatty acid residue. A very long-chain fatty acid has an aliphatic tail containing more than 22 carbons." [RHEA:78927]
 comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
+synonym: "very long-chain-3-hydroxyacyl-CoA dehydrogenase activity" BROAD []
 synonym: "very-long-chain-3-hydroxyacyl-CoA dehydrogenase activity" EXACT []
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+xref: RHEA:78927
+is_a: GO:0003857 ! (3S)-3-hydroxyacyl-CoA dehydrogenase (NAD+) activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26445" xsd:anyURI
 created_by: bf
 creation_date: 2010-03-18T11:23:05Z
@@ -228027,6 +228566,7 @@
 def: "The process in which a phosphate is transported across a membrane." [GOC:vw]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "phosphate ion membrane transport" EXACT []
+xref: Reactome:R-HSA-427652 "Sodium-coupled phosphate cotransporters"
 is_a: GO:0006817 ! phosphate ion transport
 is_a: GO:0055085 ! transmembrane transport
 created_by: bf
@@ -228806,9 +229346,11 @@
 synonym: "carnosine biosynthesis" EXACT []
 synonym: "carnosine formation" EXACT []
 synonym: "carnosine synthesis" EXACT []
+xref: MetaCyc:PWY66-420
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0035498 ! carnosine metabolic process
 is_a: GO:0046394 ! carboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: bf
 creation_date: 2010-04-29T01:20:56Z
 
@@ -229690,9 +230232,13 @@
 synonym: "histone H4K20me demethylase activity" EXACT []
 xref: Reactome:R-HSA-2172678 "PHF8 demethylates histone H4K20me1"
 xref: RHEA:67804
+xref: RHEA:85907
+xref: RHEA:85911
 is_a: GO:0016706 ! 2-oxoglutarate-dependent dioxygenase activity
 is_a: GO:0141058 ! histone H4 demethylase activity
 property_value: skos:narrowMatch RHEA:67804
+property_value: skos:narrowMatch RHEA:85907
+property_value: skos:narrowMatch RHEA:85911
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21089" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29636" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30578" xsd:anyURI
@@ -230859,6 +231405,7 @@
 def: "The process in which an oligopeptide is transported across a membrane. Oligopeptides are molecules that contain a small number (2 to 20) of amino-acid residues connected by peptide linkages." [GOC:vw, ISBN:0198506732]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "oligopeptide membrane transport" EXACT []
+xref: Reactome:R-HSA-9959399 "SLC-mediated transport of oligopeptides"
 is_a: GO:0006857 ! oligopeptide transport
 is_a: GO:0055085 ! transmembrane transport
 created_by: bf
@@ -231421,6 +231968,7 @@
 def: "A process in which a sodium ion is transported from one side of a membrane to the other by means of some agent such as a transporter or pore." [GOC:vw]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "sodium ion membrane transport" EXACT []
+xref: Reactome:R-HSA-425986 "Sodium/Proton exchangers"
 is_a: GO:0006814 ! sodium ion transport
 is_a: GO:0098655 ! monoatomic cation transmembrane transport
 created_by: bf
@@ -231696,11 +232244,14 @@
 
 [Term]
 id: GO:0035751
-name: regulation of lysosomal lumen pH
+name: obsolete regulation of lysosomal lumen pH
 namespace: biological_process
-def: "Any process that modulates the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:rph]
-is_a: GO:0051453 ! regulation of intracellular pH
-relationship: part_of GO:0007040 ! lysosome organization
+def: "OBSOLETE. Any process that modulates the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:rph]
+comment: The reason for obsoletion is that this term was an unnecessary grouping term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0007042
+consider: GO:0035752
 created_by: bf
 creation_date: 2011-03-23T10:23:41Z
 
@@ -231710,7 +232261,6 @@
 namespace: biological_process
 def: "Any process that increases the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:bf, GOC:rph]
 synonym: "lysosome pH elevation" EXACT [GOC:bf, GOC:rph]
-is_a: GO:0035751 ! regulation of lysosomal lumen pH
 is_a: GO:0051454 ! intracellular pH elevation
 created_by: bf
 creation_date: 2011-03-23T10:24:24Z
@@ -233509,18 +234059,21 @@
 namespace: biological_process
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
 synonym: "response to social isolation" EXACT [PMID:20203532]
-is_a: GO:0006950 ! response to stress
+is_a: GO:0033555 ! multicellular organismal response to stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 created_by: bf
 creation_date: 2011-06-22T09:39:57Z
 
 [Term]
 id: GO:0035901
-name: cellular response to isolation stress
+name: obsolete cellular response to isolation stress
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
+def: "OBSOLETE. Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
+comment: This term was obsoleted because it represents the molecular activities involved in response to isolation stress ; GO:0035900.
 synonym: "cellular response to social isolation" EXACT [PMID:20203532]
-is_a: GO:0033554 ! cellular response to stress
-is_a: GO:0035900 ! response to isolation stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0035900
 created_by: bf
 creation_date: 2011-06-22T09:43:34Z
 
@@ -235421,6 +235974,7 @@
 synonym: "peptidyl-succinyllysine desuccinylase activity" EXACT []
 synonym: "succinyl lysine desuccinylase activity" BROAD [GOC:bf]
 synonym: "succinyllysine desuccinylase activity" BROAD []
+xref: Reactome:R-HSA-9955518 "SIRT5 desuccinylates CPS1"
 xref: RHEA:47668
 is_a: GO:0141218 ! NAD-dependent protein lysine deacylase activity
 relationship: part_of GO:0036049 ! peptidyl-lysine desuccinylation
@@ -235711,7 +236265,7 @@
 id: GO:0036081
 name: extracellular ammonia-gated monoatomic ion channel activity
 namespace: molecular_function
-def: "Enables the transmembrane transfer of an ion by a channel that opens when extracellular ammonia (NH3) has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
+def: "Enables the transmembrane transfer of an ion by a channel that opens when extracellular ammonia (NH4+) has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
 synonym: "extracellular ammonia-gated ion channel activity" BROAD []
 synonym: "ionotropic ammonia receptor activity" RELATED [PMID:19135896]
 is_a: GO:0005230 ! extracellular ligand-gated monoatomic ion channel activity
@@ -235793,12 +236347,14 @@
 id: GO:0036088
 name: D-serine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-serine, the D-enantiomer of serine, i.e. (2S)-2-amino-3-hydroxypropanoic acid." [GOC:imk]
+def: "The chemical reactions and pathways resulting in the breakdown of D-serine." [GOC:curators]
 synonym: "D-serine breakdown" EXACT [GOC:bf]
 synonym: "D-serine catabolism" EXACT [GOC:bf]
 synonym: "D-serine degradation" EXACT [GOC:bf]
+xref: MetaCyc:PWY0-1535
 is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:0070178 ! D-serine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 created_by: bf
 creation_date: 2012-01-06T10:58:51Z
 
@@ -236782,7 +237338,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid -> 2-methoxy-6-decaprenylphenol + CO2." [GOC:mw, PMID:620805, PMID:7028108]
 comment: The reason for obsoletion is that this term is equivalent to 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity.
-xref: Reactome:R-HSA-2162195 "COQ4 decarboxylates MHDB"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0120539
@@ -239302,7 +239857,7 @@
 id: GO:0036381
 name: pyridoxal 5'-phosphate synthase (glutamine hydrolysing) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine = pyridoxal 5'-phosphate + L-glutamate + 3 H2O + phosphate. The reaction occurs in two steps: L-glutamine + H2O = L-glutamate + NH3, and subsequently D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + NH3 = pyridoxal 5'-phosphate + 4 H2O + phosphate." [EC:4.3.3.6, GOC:rs]
+def: "Catalysis of the reaction: D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine = pyridoxal 5'-phosphate + L-glutamate + 3 H2O + phosphate. The reaction occurs in two steps: L-glutamine + H2O = L-glutamate + NH4+, and subsequently D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + NH4+ = pyridoxal 5'-phosphate + 4 H2O + phosphate." [EC:4.3.3.6, GOC:rs]
 synonym: "PdxST activity" RELATED [EC:4.3.3.6]
 synonym: "pyridoxal 5'-phosphate synthase (glutamine hydrolyzing) activity" RELATED [EC:4.3.3.6]
 xref: EC:4.3.3.6
@@ -239471,8 +240026,9 @@
 id: GO:0036393
 name: thiocyanate peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: thiocyanate (SCN-) + hydrogen peroxide (H2O2) = hypothiocyanite (OSCN-) + 2 H2O. Catalyzes the hydrogen peroxide oxidation of thiocyanate." [GOC:pm, PMID:12626341]
+def: "Catalysis of the reaction: thiocyanate + H2O2 + H+ = hypothiocyanous acid + H2O." [GOC:pm, PMID:12626341, RHEA:69416]
 synonym: "lactoperoxidase activity" BROAD [PMID:12626341, Wikipedia:Lactoperoxidase]
+xref: RHEA:69416
 is_a: GO:0004601 ! peroxidase activity
 relationship: part_of GO:0018969 ! thiocyanate metabolic process
 created_by: bf
@@ -239911,12 +240467,12 @@
 
 [Term]
 id: GO:0036426
-name: ditrans, polycis-undecaprenyl-phosphate mannosyltransferase activity
+name: obsolete ditrans, polycis-undecaprenyl-phosphate mannosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ditrans,octacis-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate+ GDP." [GOC:curators]
-xref: KEGG_REACTION:R07257
-xref: MetaCyc:2.4.1.54-RXN
-is_a: GO:0047267 ! undecaprenyl-phosphate mannosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: ditrans,octacis-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate+ GDP." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: bf
 creation_date: 2013-09-16T13:21:01Z
 
@@ -240262,6 +240818,7 @@
 def: "Catalysis of the transfer of a iron-sulfur cluster from one compound (donor) to another (acceptor)." [GOC:bhm]
 synonym: "Fe-S transferase activity" EXACT [GOC:bf]
 xref: Reactome:R-HSA-2564828 "CIA Targeting Complex transfers 4Fe-4S cluster to apoproteins"
+xref: Reactome:R-HSA-8878815 "Formation of 4Fe-4S cluster on ISCA1:ISCA2"
 is_a: GO:0016782 ! transferase activity, transferring sulphur-containing groups
 created_by: bf
 creation_date: 2013-11-14T11:17:37Z
@@ -241585,11 +242142,13 @@
 synonym: "receptor activity involved in receptor-mediated endocytosis" NARROW [GOC:bf]
 synonym: "transport receptor activity" BROAD [GOC:signaling]
 xref: Reactome:R-HSA-203716 "eNOS:Caveolin-1 complex binds to Nostrin"
+xref: Reactome:R-HSA-2160915 "Receptor-mediated uptake of HA"
 xref: Reactome:R-HSA-3000103 "CUBN:AMN binds CBLIF:RCbl"
 xref: Reactome:R-HSA-3000112 "CD320-mediated TCN2:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-3000122 "CD320 binds extracellular TCN2:RCbl"
 xref: Reactome:R-HSA-3000137 "CUBN:AMN-mediated CBLIF:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-350168 "LRP2-mediated uptake of extracellular CUBN:GC:25(OH)D"
+xref: Reactome:R-HSA-350186 "CUBN binds GC:25(OH)D"
 xref: Reactome:R-HSA-9759202 "LRP2-mediated TCN2:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-9759209 "LRP2 binds extracellular TCN2:RCbl"
 is_a: GO:0003674 ! molecular_function
@@ -247211,6 +247770,7 @@
 synonym: "translocation of peptides or proteins into other organism involved in symbiotic interaction" BROAD []
 synonym: "transport of peptides or proteins into host" EXACT []
 synonym: "transport of peptides or proteins into other organism during symbiotic interaction" BROAD []
+xref: Reactome:R-HSA-9760173 "Secretion of toxins"
 is_a: GO:0044417 ! translocation of molecules into host
 
 [Term]
@@ -247449,7 +248009,6 @@
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces an ATP hydrolysis activity." [GOC:jl]
 synonym: "adenosinetriphosphatase inhibitor" EXACT []
-is_a: GO:0060590 ! ATPase regulator activity
 is_a: GO:0140678 ! molecular function inhibitor activity
 relationship: negatively_regulates GO:0140657 ! ATP-dependent activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
@@ -248690,7 +249249,7 @@
 id: GO:0042170
 name: plastid membrane
 namespace: cellular_component
-def: "Either of the lipid bilayers that surround a plastid and form the plastid envelope." [GOC:mah]
+def: "Any lipid bilayer that surrounds a plastid or is part of the plastid envelope. Primary plastids (in plants, green algae, red algae, and glaucophytes) are surrounded by two membranes; complex plastids derived from secondary endosymbiosis are surrounded by three or four membranes." [GOC:mah, PMID:20124341]
 is_a: GO:0031090 ! organelle membrane
 intersection_of: GO:0016020 ! membrane
 intersection_of: part_of GO:0009536 ! plastid
@@ -249305,15 +249864,15 @@
 
 [Term]
 id: GO:0042217
-name: 1-aminocyclopropane-1-carboxylate catabolic process
+name: obsolete 1-aminocyclopropane-1-carboxylate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:go_curators]
+comment: This term was obsoleted because there is no evidence that a specific pathway for 1-aminocyclopropane-1-carboxylate degradation exists.
 synonym: "1-aminocyclopropane-1-carboxylate breakdown" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate catabolism" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate degradation" EXACT []
-is_a: GO:0018871 ! 1-aminocyclopropane-1-carboxylate metabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
-is_a: GO:1901606 ! alpha-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0042218
@@ -249895,6 +250454,7 @@
 def: "Catalysis of the transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y = Y-ISG15 + X, where both X-ISG15 and Y-ISG15 are covalent linkages." [GOC:mah, PMID:12826404]
 synonym: "ISG15 conjugating enzyme activity" NARROW []
 xref: Reactome:R-HSA-5653754 "UBE2L6:TRIM25 ISGylates monoUb:K164-PCNA"
+xref: Reactome:R-HSA-9927247 "ISGylation of DDX58 (RIG-I)"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -250618,7 +251178,7 @@
 synonym: "tocopherol metabolic process" EXACT []
 synonym: "tocopherol metabolism" EXACT []
 synonym: "vitamin E metabolism" EXACT []
-xref: Reactome:R-HSA-8877627 "Vitamin E"
+xref: Reactome:R-HSA-8877627 "Vitamin E transport"
 is_a: GO:0008152 ! metabolic process
 
 [Term]
@@ -250647,6 +251207,7 @@
 synonym: "fat-soluble vitamin biosynthesis" EXACT []
 synonym: "fat-soluble vitamin formation" EXACT []
 synonym: "fat-soluble vitamin synthesis" EXACT []
+xref: Reactome:R-HSA-6806667 "Metabolism of fat-soluble vitamins"
 is_a: GO:0009110 ! vitamin biosynthetic process
 
 [Term]
@@ -250668,6 +251229,7 @@
 synonym: "water-soluble vitamin biosynthesis" EXACT []
 synonym: "water-soluble vitamin formation" EXACT []
 synonym: "water-soluble vitamin synthesis" EXACT []
+xref: Reactome:R-HSA-196849 "Metabolism of water-soluble vitamins and cofactors"
 is_a: GO:0009110 ! vitamin biosynthetic process
 
 [Term]
@@ -251204,8 +251766,15 @@
 synonym: "taurine biosynthesis" EXACT []
 synonym: "taurine formation" EXACT []
 synonym: "taurine synthesis" EXACT []
+xref: MetaCyc:PWY-5331
+xref: MetaCyc:PWY-7850
+xref: MetaCyc:PWY-8359
 is_a: GO:0019530 ! taurine metabolic process
 is_a: GO:0046305 ! alkanesulfonate biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5331
+property_value: skos:narrowMatch MetaCyc:PWY-7850
+property_value: skos:narrowMatch MetaCyc:PWY-8359
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
 
 [Term]
 id: GO:0042413
@@ -251385,6 +251954,7 @@
 xref: MetaCyc:CHOLINE-BETAINE-ANA-PWY
 xref: MetaCyc:P542-PWY
 xref: MetaCyc:PWY-3721
+xref: MetaCyc:PWY-7167
 xref: MetaCyc:PWY-7494
 xref: Reactome:R-HSA-6798163 "Choline catabolism"
 is_a: GO:0019695 ! choline metabolic process
@@ -251392,9 +251962,11 @@
 property_value: skos:narrowMatch MetaCyc:CHOLINE-BETAINE-ANA-PWY
 property_value: skos:narrowMatch MetaCyc:P542-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3721
+property_value: skos:narrowMatch MetaCyc:PWY-7167
 property_value: skos:narrowMatch MetaCyc:PWY-7494
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30537" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0042427
@@ -253160,10 +253732,12 @@
 synonym: "P-P-bond-hydrolysis-driven transmembrane transporter activity" BROAD []
 synonym: "P-P-bond-hydrolysis-driven transporter" BROAD []
 xref: EC:7.2.2.-
+xref: Reactome:R-HSA-1454916 "The ABCC family mediates organic anion transport"
 xref: Reactome:R-HSA-1467457 "ABCA8,B1,B5 transport xenobiotics from cytosol to extracellular region"
 xref: Reactome:R-HSA-2161506 "abacavir [cytosol] + ATP + H2O => abacavir[extracellular] + ADP + phosphate"
 xref: Reactome:R-HSA-2161538 "abacavir [cytosol] + ATP + H2O => abacavir[extracellular] + ADP + phosphate"
 xref: Reactome:R-HSA-390393 "Peroxisomal uptake of very long-chain fatty acyl CoA"
+xref: Reactome:R-HSA-5690340 "Defective ABCC6 does not transport organic anion from cytosol to extracellular region"
 xref: Reactome:R-HSA-9033499 "PEX1:PEX6:PEX26:ZFAND6 dissociates Ub:PEX5L and PEX7 from PEX14:PEX13:PEX2:PEX10:PEX12 and translocates PEX5L and PEX7 from the peroxisomal membrane to the cytosol"
 xref: Reactome:R-HSA-9033505 "PEX1:PEX6:PEX26:ZFAND6:Ub:PEX5S,L:PEX14:PEX13:PEX2:PEX10:PEX12 dissociates yielding cytosolic Ub:PEX5S,L and membrane PEX14:PEX13:PEX2:PEX10:PEX12"
 xref: Reactome:R-HSA-9659680 "ABCB1 transports xenobiotics out of the cell"
@@ -253998,6 +254572,7 @@
 subset: gocheck_do_not_annotate
 subset: goslim_candida
 subset: goslim_pir
+xref: Reactome:R-HSA-9931953 "Biofilm formation"
 is_a: GO:0098630 ! aggregation of unicellular organisms
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17727" xsd:anyURI
 
@@ -255044,13 +255619,14 @@
 synonym: "histone methyltransferase activity (H3-K4 specific)" EXACT []
 synonym: "histone-H3K4 methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-1214188 "PRDM9 trimethylates histone H3"
-xref: Reactome:R-HSA-3364026 "SET1 complex trimethylates H3K4 at the MYC gene"
+xref: Reactome:R-HSA-3364026 "MLL2 complex trimethylates H3K4 at the MYC gene"
 xref: Reactome:R-HSA-8865498 "KMT2A trimethylates nucleosomes at the SPI1 gene locus producing H3K4Me3 mark"
 xref: Reactome:R-HSA-8936481 "Core MLL complex methylates H3K4Me2-Nucleosome at the ITGA2B gene promoter"
 xref: Reactome:R-HSA-8936621 "Core MLL complex methylates H3K4Me2-Nucleosome at the GP1BA gene promoter"
 xref: Reactome:R-HSA-8937016 "Core MLL complex methylates H3K4Me2-Nucleosome at the THBS1 gene promoter"
 xref: Reactome:R-HSA-8937050 "Core MLL complex methylates H3K4Me2-Nucleosome at the MIR27A gene promoter"
-xref: Reactome:R-HSA-9843121 "KMT2D,(KMT2C) complex monomethylates nucleosomes at PPARG:RXRA-bound enhancers"
+xref: Reactome:R-HSA-9843121 "MLL4-ASCOM, (MLL3-ASCOM) complex monomethylates nucleosomes at PPARG:RXRA-bound enhancers"
+xref: Reactome:R-HSA-9909628 "Trimethylation of CD274 gene chromatin by MLL1/3 complex"
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24866" xsd:anyURI
@@ -255570,6 +256146,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
 synonym: "L-alanine metabolism" EXACT []
+xref: Reactome:R-HSA-8964540 "Alanine metabolism"
 is_a: GO:0006522 ! alanine metabolic process
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
@@ -255578,23 +256155,28 @@
 id: GO:0042852
 name: L-alanine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah, MetaCyc:PWY0-1061]
+def: "The chemical reactions and pathways resulting in the formation of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
 synonym: "L-alanine anabolism" EXACT []
 synonym: "L-alanine biosynthesis" EXACT []
 synonym: "L-alanine formation" EXACT []
 synonym: "L-alanine synthesis" EXACT []
-xref: MetaCyc:PWY0-1061
+xref: MetaCyc:ALANINE-SYN2-PWY
+xref: MetaCyc:ALANINE-VALINESYN-PWY
+xref: MetaCyc:PWY0-1021
 is_a: GO:0006523 ! alanine biosynthetic process
 is_a: GO:0009079 ! pyruvate family amino acid biosynthetic process
 is_a: GO:0042851 ! L-alanine metabolic process
-property_value: skos:exactMatch MetaCyc:PWY0-1061
+property_value: skos:narrowMatch MetaCyc:ALANINE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:ALANINE-VALINESYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY0-1021
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0042853
 name: L-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-alanine." [GOC:curators]
 synonym: "L-alanine breakdown" EXACT []
 synonym: "L-alanine catabolism" EXACT []
 synonym: "L-alanine degradation" EXACT []
@@ -255608,6 +256190,7 @@
 property_value: skos:narrowMatch MetaCyc:ALADEG-PWY
 property_value: skos:narrowMatch MetaCyc:ALANINE-DEG3-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0042854
@@ -255789,6 +256372,7 @@
 synonym: "pyruvate biosynthesis" EXACT []
 synonym: "pyruvate formation" EXACT []
 synonym: "pyruvate synthesis" EXACT []
+xref: Reactome:R-HSA-70268 "Pyruvate metabolism"
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0072330 ! monocarboxylic acid biosynthetic process
 
@@ -256169,6 +256753,15 @@
 synonym: "xenobiotic transporter activity" RELATED []
 xref: Reactome:R-HSA-5625574 "Defective SLC22A18 does not exchange extracellular organic cations for cytosolic H+"
 xref: Reactome:R-HSA-597628 "SLC22A18 exchanges extracellular organic cations for cytosolic H+"
+xref: Reactome:R-HSA-9911512 "AcrA:AcrB:TolC transports antibiotics from the cytosol to the periplasm"
+xref: Reactome:R-HSA-9911517 "MdfA transports tetracyclines from the cytosol to the periplasmic space"
+xref: Reactome:R-HSA-9913106 "TetB exports tetracyclines"
+xref: Reactome:R-HSA-9913128 "TetA exports tetracyclines"
+xref: Reactome:R-HSA-9913437 "Pef exports fluoroquinolones"
+xref: Reactome:R-HSA-9913672 "OqxA exports fluoroquinolones"
+xref: Reactome:R-HSA-9914089 "MdtE:MdtF:TolC exports beta-lactams to the periplasm"
+xref: Reactome:R-HSA-9931163 "Gdx dimer exports cationic aromatics to periplasm"
+xref: Reactome:R-HSA-9931565 "EmrE dimer exports xenobiotics to periplasm"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0042908 ! xenobiotic transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19460" xsd:anyURI
@@ -256331,6 +256924,8 @@
 def: "Enables the directed movement of the siderochrome enterochelin, a cyclic trimer of 2, 3 dihydroxybenzoylserine from one side of a membrane to the other." [GOC:jl]
 synonym: "enterobactin transporter activity" RELATED []
 synonym: "enterochelin transporter activity" EXACT []
+xref: Reactome:R-HSA-9911595 "EntS transports enterobactin from cytosol to periplasmic space"
+xref: Reactome:R-HSA-9911600 "Efflux pump complexes transport enterobactin from periplasmic space to extracellular region"
 is_a: GO:0015343 ! siderophore-iron transmembrane transporter activity
 relationship: part_of GO:0042930 ! enterobactin transport
 
@@ -256791,6 +257386,7 @@
 namespace: molecular_function
 def: "Binds to and increases ornithine decarboxylase activity." [GOC:jl]
 synonym: "L-ornithine carboxy-lyase activator activity" NARROW []
+xref: Reactome:R-HSA-9955574 "ODC dimerizes"
 is_a: GO:0008047 ! enzyme activator activity
 is_a: GO:0042979 ! ornithine decarboxylase regulator activity
 relationship: positively_regulates GO:0004586 ! ornithine decarboxylase activity
@@ -257545,6 +258141,7 @@
 synonym: "dolichyl monophosphate formation" EXACT []
 synonym: "dolichyl monophosphate synthesis" EXACT []
 xref: MetaCyc:PWY-6129
+xref: Reactome:R-HSA-446199 "Synthesis of dolichyl-phosphate"
 is_a: GO:0008654 ! phospholipid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30675" xsd:anyURI
 
@@ -258348,7 +258945,6 @@
 name: ubiquitin binding
 namespace: molecular_function
 def: "Binding to ubiquitin, a protein that when covalently bound to other cellular proteins marks them for proteolytic degradation." [GOC:ecd]
-xref: Reactome:R-HSA-1169404 "Transfer of ISG15 from E1 to E2 (UBCH8)"
 xref: Reactome:R-HSA-205008 "Polyubiquitinated NRIF binds to p62 (Sequestosome)"
 xref: Reactome:R-HSA-983152 "Transfer of ubiquitin from E1 to E2"
 is_a: GO:0032182 ! ubiquitin-like protein binding
@@ -258413,7 +259009,7 @@
 synonym: "glycerol-1-phosphatase activity" BROAD [EC:3.1.3.21]
 synonym: "glycerol-3-phosphatase activity" BROAD []
 xref: MetaCyc:RXN-14965
-xref: Reactome:R-HSA-8955794 "PGP:Mg2+ dimer hydrolyses 3PG to glycerol"
+xref: Reactome:R-HSA-8955794 "PGP:Mg2+ dimer hydrolyses G3P to glycerol"
 xref: RHEA:66372
 is_a: GO:0016791 ! phosphatase activity
 property_value: skos:broadMatch EC:3.1.3.21
@@ -258468,6 +259064,7 @@
 synonym: "DNA helicase IV activity" NARROW []
 xref: EC:5.6.2.3
 xref: Reactome:R-HSA-6782131 "ERCC2-facilitated RNA Pol II backtracking in TC-NER"
+xref: Reactome:R-HSA-9914438 "TWNK (PEO1) unwinds double-stranded mitochondrial DNA at the replication fork"
 is_a: GO:0003678 ! DNA helicase activity
 property_value: skos:exactMatch EC:5.6.2.3
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23533" xsd:anyURI
@@ -258573,7 +259170,6 @@
 synonym: "inhibition of caspase activity" NARROW []
 synonym: "negative regulation of caspase activation" EXACT []
 synonym: "negative regulation of caspase activity" BROAD []
-xref: Reactome:R-HSA-5218900 "CASP8 activity is inhibited"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -259247,8 +259843,6 @@
 synonym: "anion-transporting ATPase activity" EXACT []
 synonym: "ATP-dependent anion transmembrane transporter activity" EXACT []
 synonym: "ATPase-coupled anion transmembrane transporter activity" EXACT []
-xref: Reactome:R-HSA-1454916 "The ABCC family mediates organic anion transport"
-xref: Reactome:R-HSA-5690340 "Defective ABCC6 does not transport organic anion from cytosol to extracellular region"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0042626
@@ -259511,7 +260105,7 @@
 name: sodium-independent organic anion transport
 namespace: biological_process
 def: "The directed, sodium-independent, movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
-xref: Reactome:R-HSA-879518 "Transport of organic anions"
+xref: Reactome:R-HSA-879518 "Organic anion transport by SLCO transporters"
 is_a: GO:0015711 ! organic anion transport
 
 [Term]
@@ -259559,9 +260153,9 @@
 subset: goslim_pir
 synonym: "laminin complex" BROAD []
 is_a: GO:0032991 ! protein-containing complex
-relationship: part_of GO:0140144 ! non-collagenous component of basement membrane
 relationship: part_of GO:0140349 ! laminin network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30807" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31490" xsd:anyURI
 
 [Term]
 id: GO:0043257
@@ -259895,7 +260489,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "regulation of caspase activation" NARROW []
 synonym: "regulation of caspase activity" BROAD []
-xref: Reactome:R-HSA-205025 "NADE modulates death signalling"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -262186,7 +262779,7 @@
 xref: Reactome:R-HSA-9856872 "Malate-aspartate shuttle"
 xref: Wikipedia:Malate-aspartate_shuttle
 is_a: GO:0019674 ! NAD+ metabolic process
-relationship: has_part GO:0004069 ! L-aspartate:2-oxoglutarate aminotransferase activity
+relationship: has_part GO:0004069 ! L-aspartate:2-oxoglutarate transaminase activity
 relationship: has_part GO:0030060 ! L-malate dehydrogenase (NAD+) activity
 relationship: has_part GO:1990542 ! mitochondrial transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28982" xsd:anyURI
@@ -262249,6 +262842,8 @@
 synonym: "protein membrane adaptor" EXACT []
 synonym: "protein membrane adaptor activity" EXACT []
 synonym: "protein membrane anchor" RELATED []
+xref: Reactome:R-HSA-5333678 "CPNEs bind PL"
+xref: Reactome:R-HSA-6809764 "SBF1 binds MTMR2"
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19090" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24964" xsd:anyURI
@@ -262800,7 +263395,6 @@
 namespace: biological_process
 def: "The addition of an acyl group, any group or radical of the form RCO- where R is an organic group, to a protein amino acid." [GOC:jl]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid acylation" EXACT [GOC:bf]
 is_a: GO:0036211 ! protein modification process
 
@@ -264229,10 +264823,6 @@
 synonym: "Cu2+-exporting ATPase activity" RELATED [EC:7.2.2.9]
 xref: EC:7.2.2.9
 xref: MetaCyc:TRANS-RXN-178
-xref: Reactome:R-HSA-3697838 "ATP7A transfers Cu from ATOX1 to SOD3"
-xref: Reactome:R-HSA-6803545 "ATP7A transports cytosolic Cu2+ to phagosomal lumen"
-xref: Reactome:R-HSA-936802 "ATP7A transports cytosolic Cu2+ to extracellular region"
-xref: Reactome:R-HSA-936895 "ATP7B transports cytosolic Cu2+ to Golgi lumen"
 xref: RHEA:10376
 is_a: GO:0005375 ! copper ion transmembrane transporter activity
 is_a: GO:0015662 ! P-type ion transporter activity
@@ -264717,10 +265307,12 @@
 
 [Term]
 id: GO:0043730
-name: 5-ureido-4-imidazole carboxylate hydrolase activity
+name: obsolete 5-ureido-4-imidazole carboxylate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-ureido-4-imidazole carboxylate + H2O = 5-amino-4-imidazole carboxylate + NH3 + CO2." [GOC:jl]
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+def: "OBSOLETE. Catalysis of the reaction: 5-ureido-4-imidazole carboxylate + H2O = 5-amino-4-imidazole carboxylate + NH3 + CO2." [GOC:jl]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0043731
@@ -265011,12 +265603,16 @@
 xref: EC:2.3.1.168
 xref: MetaCyc:2.3.1.168-RXN
 xref: RHEA:18865
+xref: RHEA:84647
+xref: RHEA:84651
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 property_value: skos:broadMatch EC:1.2.1.25
 property_value: skos:broadMatch MetaCyc:1.2.1.25-RXN
 property_value: skos:exactMatch EC:2.3.1.168
 property_value: skos:exactMatch MetaCyc:2.3.1.168-RXN
 property_value: skos:narrowMatch RHEA:18865
+property_value: skos:narrowMatch RHEA:84647
+property_value: skos:narrowMatch RHEA:84651
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17835" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29951" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30031" xsd:anyURI
@@ -265111,12 +265707,16 @@
 
 [Term]
 id: GO:0043760
-name: acetyldiaminopimelate aminotransferase activity
+name: obsolete acetyldiaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-acetyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-acetyl-2-L-amino-6-oxoheptanedioate + L-glutamate." [PMID:1906065]
+def: "OBSOLETE. Catalysis of the reaction: N-acetyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-acetyl-2-L-amino-6-oxoheptanedioate + L-glutamate." [PMID:1906065]
+comment: This term was obsoleted because there is no evidence that this reaction exists. This was an alternative succinyldiaminopimelate aminotransferase substrate tested in PMID:1906065.
+synonym: "acetyldiaminopimelate aminotransferase activity" BROAD []
 synonym: "N-acetyl-diaminopimelate aminotransferase activity" EXACT []
 synonym: "N-acetyl-L,L-diaminopimelate aminotransferase activity" EXACT []
-is_a: GO:0008483 ! transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+consider: GO:0009016
 
 [Term]
 id: GO:0043761
@@ -265558,7 +266158,7 @@
 id: GO:0043799
 name: glycine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reactions: (1) glycine + H2O + O2 = glyoxylate + NH3 + hydrogen peroxide; (2) D-alanine + H2O + O2 = pyruvate + NH3 + hydrogen peroxide; (3) sarcosine + H2O + O2 = glyoxylate + methylamine + hydrogen peroxide; (4) N-ethylglycine + H2O + O2 = glyoxylate + ethylamine + hydrogen peroxide." [EC:1.4.3.19]
+def: "Catalysis of the reactions: (1) glycine + H2O + O2 = glyoxylate + NH4+ + H2O2; (2) D-alanine + H2O + O2 = pyruvate + NH4+ + H2O2; (3) sarcosine + H2O + O2 = glyoxylate + methylamine + H2O2; (4) N-ethylglycine + H2O + O2 = glyoxylate + ethylamine + H2O2." [EC:1.4.3.19]
 synonym: "glycine:oxygen oxidoreductase (deaminating)" EXACT []
 xref: EC:1.4.3.19
 xref: MetaCyc:1.4.3.19-RXN
@@ -265948,9 +266548,9 @@
 
 [Term]
 id: GO:0043825
-name: succinylornithine transaminase activity
+name: succinylornithine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-succinyl-L-ornithine = N-succinyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.81, RHEA:16953]
+def: "Catalysis of the reaction: N(2)-succinyl-L-ornithine + 2-oxoglutarate = N-succinyl-L-glutamate 5-semialdehyde + L-glutamate." [RHEA:16953]
 synonym: "2-N-succinyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" RELATED [EC:2.6.1.81]
 synonym: "AstC" RELATED []
 synonym: "N(2)-succinylornithine 5-aminotransferase activity" RELATED [EC:2.6.1.81]
@@ -265960,6 +266560,7 @@
 synonym: "succinyl ornithine transaminase activity" EXACT []
 synonym: "succinyl-ornithine transaminase activity" RELATED [EC:2.6.1.81]
 synonym: "succinylornithine aminotransferase activity" BROAD [EC:2.6.1.81]
+synonym: "succinylornithine transaminase activity" EXACT []
 xref: EC:2.6.1.81
 xref: KEGG_REACTION:R04217
 xref: MetaCyc:SUCCORNTRANSAM-RXN
@@ -265968,6 +266569,7 @@
 property_value: skos:exactMatch EC:2.6.1.81
 property_value: skos:exactMatch RHEA:16953
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0043826
@@ -266018,7 +266620,7 @@
 id: GO:0043829
 name: tRNA-specific adenosine-37 deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine-37 + H2O = inosine-37 + NH3, in a tRNA-Ala molecule." [PMID:8915538, PMID:9707437, RHEA:50968]
+def: "Catalysis of the reaction: adenosine-37 + H2O = inosine-37 + NH4+, in a tRNA-Ala molecule." [PMID:8915538, PMID:9707437, RHEA:50968]
 synonym: "TAD1" RELATED []
 synonym: "tRNA(Ala)-A37 deaminase activity" EXACT []
 synonym: "tRNA-specific adenosine deaminase 1" EXACT []
@@ -266314,7 +266916,7 @@
 id: GO:0043852
 name: monomethylamine methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: monomethylamine + a monomethylamine corrinoid protein = a methylated monomethylamine corrinoid protein + NH3." [PMID:9195968]
+def: "Catalysis of the reaction: Co(I)-[methylamine-specific corrinoid protein] + methylamine + H+ = methyl-Co(III)-[methylamine-specific corrinoid protein] + NH4+." [PMID:9195968, RHEA:26059]
 comment: This function is the first step in the pathway of methanogenesis from monomethylamine.
 synonym: "MMAMT" RELATED []
 synonym: "monomethylamine:corrinoid methyltransferase activity" EXACT []
@@ -266644,7 +267246,7 @@
 id: GO:0043877
 name: galactosamine-6-phosphate isomerase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-galactosamine 6-phosphate + H2O = D-tagatose 6-phosphate + NH3." [PMID:10931310]
+def: "Catalysis of the reaction: D-galactosamine 6-phosphate + H2O = D-tagatose 6-phosphate + NH4+." [PMID:10931310]
 comment: This function is part of the pathway of N-acetyl-galactosamine and galactosamine utilization.
 synonym: "AgaI" RELATED []
 synonym: "galactosamine-6-phosphate deaminase activity" EXACT []
@@ -267139,11 +267741,15 @@
 
 [Term]
 id: GO:0043911
-name: D-lysine transaminase activity
+name: D-lysine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-lysine + 2-oxoglutarate = L-glutamate + 6-amino-2-oxohexanoate." [GOC:jl, PMID:17259313]
+def: "Catalysis of the reaction: D-lysine + 2-oxoglutarate = 6-amino-2-oxohexanoate + L-glutamate." [PMID:17259313, RHEA:37587]
 synonym: "D-lysine aminotransferase activity" EXACT []
-is_a: GO:0008483 ! transaminase activity
+synonym: "D-lysine transaminase activity" BROAD []
+xref: RHEA:37587
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: skos:exactMatch RHEA:37587
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0043912
@@ -267168,21 +267774,27 @@
 
 [Term]
 id: GO:0043914
-name: NADPH:sulfur oxidoreductase activity
+name: NAD(P)H sulfur oxidoreductase (CoA-dependent) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NADPH + H+ + sulfur = hydrogen sulfide + NADP+." [GOC:jl, PMID:17449625]
+def: "Catalysis of the reaction: hydrogen sulfide + NAD(P)+ = sulfur + NAD(P)H." [EC:1.8.1.18, PMID:17449625]
 synonym: "CoA-dependent NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "coenzyme A-dependent NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H elemental sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H sulphur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H:sulfur oxidoreductase activity" EXACT []
+synonym: "NADPH:sulfur oxidoreductase activity" NARROW []
 synonym: "NADPH:sulphur oxidoreductase activity" EXACT []
 synonym: "NSR" EXACT []
 xref: EC:1.8.1.18
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+xref: RHEA:36595
+xref: RHEA:36599
+is_a: GO:0016668 ! oxidoreductase activity, acting on a sulfur group of donors, NAD(P) as acceptor
 property_value: skos:exactMatch EC:1.8.1.18
+property_value: skos:narrowMatch RHEA:36595
+property_value: skos:narrowMatch RHEA:36599
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0043915
@@ -267728,10 +268340,12 @@
 
 [Term]
 id: GO:0043960
-name: L-erythro-3-methylmalyl-CoA dehydratase activity
+name: obsolete L-erythro-3-methylmalyl-CoA dehydratase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-erythro-3-methylmalyl-CoA = mesaconyl-CoA + H2O." [GOC:jl]
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: L-erythro-3-methylmalyl-CoA = mesaconyl-CoA + H2O." [GOC:jl]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0043961
@@ -268249,6 +268863,8 @@
 synonym: "regulation by symbiont of host system process" NARROW []
 xref: Reactome:R-HSA-9635644 "Inhibition of membrane repair"
 xref: Reactome:R-HSA-9636667 "Manipulation of host energy metabolism"
+xref: Reactome:R-HSA-9833110 "RSV-host interactions"
+xref: Reactome:R-HSA-9918481 "Dengue Virus-Host Interactions"
 is_a: GO:0035821 ! modulation of process of another organism
 is_a: GO:0051701 ! biological process involved in interaction with host
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18719" xsd:anyURI
@@ -270092,6 +270708,7 @@
 name: protein folding chaperone
 namespace: molecular_function
 def: "Binding to a protein or a protein-containing complex to assist the protein folding process." [GOC:mtg_cambridge_2009]
+comment: Note that a protein folding chaperone binds an unfolded protein to fold it. Do not confuse with unfolded protein holdase activity, which binds an unfolded protein and keeps it unfolded to deliver it to its final destination.
 subset: goslim_chembl
 subset: goslim_drosophila
 subset: goslim_generic
@@ -270105,6 +270722,7 @@
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0051082 ! unfolded protein binding
 relationship: part_of GO:0006457 ! protein folding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30552" xsd:anyURI
 created_by: jl
 creation_date: 2009-09-25T11:33:48Z
 
@@ -270632,7 +271250,6 @@
 comment: The reason for obsoletion is that this term is now redundant with GO:0008152, metabolic process
 synonym: "cellular metabolism" EXACT []
 synonym: "intermediary metabolism" RELATED [GOC:mah]
-xref: Reactome:R-HSA-1428517 "Aerobic respiration and respiratory electron transport"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008152
@@ -270648,6 +271265,7 @@
 subset: goslim_plant_ribbon
 subset: goslim_prokaryote_ribbon
 synonym: "primary metabolism" EXACT []
+xref: Reactome:R-HSA-71291 "Metabolism of amino acids and derivatives"
 xref: Wikipedia:Primary_metabolite
 is_a: GO:0008152 ! metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26424" xsd:anyURI
@@ -270953,11 +271571,13 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:pde, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 subset: goslim_chembl
 subset: goslim_flybase_ribbon
 subset: goslim_metagenomics
 synonym: "small molecule metabolism" EXACT []
 is_a: GO:0008152 ! metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:05:20Z
 
@@ -270967,9 +271587,11 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 synonym: "small molecule catabolism" EXACT []
 is_a: GO:0009056 ! catabolic process
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:06:10Z
 
@@ -270979,11 +271601,13 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:pde, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 subset: goslim_prokaryote
 subset: goslim_prokaryote_ribbon
 synonym: "small molecule biosynthesis" EXACT []
 is_a: GO:0009058 ! biosynthetic process
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:06:49Z
 
@@ -272284,6 +272908,7 @@
 synonym: "symbiotic interaction between organisms" RELATED []
 synonym: "symbiotic interaction between species" RELATED []
 synonym: "symbiotic process" RELATED []
+xref: Reactome:R-HSA-9839923 "Dengue Virus Infection"
 is_a: GO:0044419 ! biological process involved in interspecies interaction between organisms
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14807" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20191" xsd:anyURI
@@ -272309,6 +272934,7 @@
 synonym: "adhesion to other organism during symbiotic interaction" RELATED [GOC:dph]
 synonym: "adhesion to other organism involved in symbiotic interaction" EXACT []
 synonym: "host adhesion" NARROW []
+xref: Reactome:R-HSA-9638630 "Attachment of bacteria to epithelial cells"
 is_a: GO:0051701 ! biological process involved in interaction with host
 
 [Term]
@@ -273876,7 +274502,7 @@
 id: GO:0044540
 name: L-cystine L-cysteine-lyase (deaminating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-cystine + H2O = pyruvate + NH3 + thiocysteine. Thiocysteine is also known as cysteine persulfide." [RHEA:24927]
+def: "Catalysis of the reaction: L-cystine + H2O = S-sulfanyl-L-cysteine + pyruvate + NH4+." [RHEA:24927]
 xref: EC:4.4.1.35
 xref: KEGG_REACTION:R02408
 xref: RHEA:24927
@@ -277099,6 +277725,7 @@
 synonym: "iron acquisition" BROAD []
 synonym: "iron acquisition by symbiont from host" EXACT []
 synonym: "iron acquisition by symbiont from host heme" NARROW []
+xref: Reactome:R-HSA-9638482 "Metal ion assimilation from the host"
 is_a: GO:0044002 ! acquisition of nutrients from host
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17896" xsd:anyURI
 created_by: jl
@@ -278125,6 +278752,7 @@
 def: "The process in which a relatively unspecialized T cell acquires the specialized features of a T-helper 1 (Th1) cell. A Th1 cell is a CD4-positive, alpha-beta T cell that has the phenotype T-bet-positive and produces interferon-gamma." [CL:0000545, GOC:ebc]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "T-helper 1 cell development" RELATED [GOC:add]
+xref: Reactome:R-HSA-9942503 "Differentiation of naive CD+ T cells to T helper 1 cells (Th1 cells)"
 is_a: GO:0002287 ! alpha-beta T cell activation involved in immune response
 is_a: GO:0002292 ! T cell differentiation involved in immune response
 is_a: GO:0042093 ! T-helper cell differentiation
@@ -278627,6 +279255,7 @@
 name: bioactive lipid receptor activity
 namespace: molecular_function
 def: "Combining with a bioactive lipid and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex. A bioactive lipid is a lipid for which changes in lipid levels result in functional consequences in a variety of cellular processes." [GOC:bf, GOC:mah, PMID:12215548, PMID:18216770]
+xref: Reactome:R-HSA-400434 "FFAR1 binds fatty acids"
 is_a: GO:0004930 ! G protein-coupled receptor activity
 
 [Term]
@@ -278692,7 +279321,7 @@
 synonym: "keratan sulphotransferase activity" EXACT []
 xref: EC:2.8.2.21
 xref: MetaCyc:KERATAN-SULFOTRANSFERASE-RXN
-xref: Reactome:R-HSA-2046175 "Further sulfation on galactose residues produces KSPG"
+xref: Reactome:R-HSA-2046175 "CHST1 transfers sulfate to Gal on keratan chain"
 is_a: GO:0050698 ! proteoglycan sulfotransferase activity
 property_value: skos:exactMatch EC:2.8.2.21
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
@@ -280551,15 +281180,16 @@
 
 [Term]
 id: GO:0045303
-name: diaminobutyrate-2-oxoglutarate transaminase activity
+name: L-2,4-diaminobutyrate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-2,4-diaminobutyrate = L-aspartate 4-semialdehyde + L-glutamate." [EC:2.6.1.76, RHEA:11160]
+def: "Catalysis of the reaction: L-2,4-diaminobutanoate + 2-oxoglutarate = L-aspartate 4-semialdehyde + L-glutamate." [RHEA:11160]
 synonym: "2,4-diaminobutyrate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "DAB aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "DABA aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminibutyric acid aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminobutyrate transaminase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminobutyrate--2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.76]
+synonym: "diaminobutyrate-2-oxoglutarate transaminase activity" EXACT []
 synonym: "EctB" RELATED [EC:2.6.1.76]
 synonym: "L-2,4-diaminobutanoate:2-oxoglutarate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "L-2,4-diaminobutyrate:2-ketoglutarate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
@@ -280572,6 +281202,7 @@
 property_value: skos:exactMatch EC:2.6.1.76
 property_value: skos:exactMatch RHEA:11160
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0045304
@@ -281632,7 +282263,7 @@
 id: GO:0045480
 name: galactose oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-galactose + O2 = D-galacto-hexodialdose + hydrogen peroxide." [EC:1.1.3.9]
+def: "Catalysis of the reaction: D-galactose + O2 = D-galacto-hexodialdose + H2O2." [RHEA:24160]
 synonym: "beta-galactose oxidase activity" RELATED [EC:1.1.3.9]
 synonym: "D-galactose oxidase activity" RELATED [EC:1.1.3.9]
 synonym: "D-galactose:oxygen 6-oxidoreductase activity" RELATED [EC:1.1.3.9]
@@ -281696,9 +282327,10 @@
 
 [Term]
 id: GO:0045484
-name: L-lysine 6-transaminase activity
+name: L-lysine:2-oxoglutarate 6-transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-lysine = L-glutamate + allysine." [EC:2.6.1.36, RHEA:21200]
+def: "Catalysis of the reaction: 2-oxoglutarate + L-lysine = L-glutamate + allysine." [RHEA:21200]
+synonym: "L-lysine 6-transaminase activity" EXACT []
 synonym: "L-lysine aminotransferase activity" RELATED [EC:2.6.1.36]
 synonym: "L-lysine transaminase activity" RELATED [EC:2.6.1.36]
 synonym: "L-lysine-alpha-ketoglutarate 6-aminotransferase activity" RELATED [EC:2.6.1.36]
@@ -281712,10 +282344,11 @@
 xref: KEGG_REACTION:R00457
 xref: MetaCyc:L-LYSINE-AMINOTRANSFERASE-RXN
 xref: RHEA:21200
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.36
 property_value: skos:exactMatch RHEA:21200
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0045485
@@ -287883,6 +288516,7 @@
 name: negative regulation of nucleobase-containing compound metabolic process
 namespace: biological_process
 def: "Any cellular process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:go_curators]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
 synonym: "down-regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
 synonym: "downregulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
@@ -287894,6 +288528,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0006139 ! nucleobase-containing compound metabolic process
 relationship: negatively_regulates GO:0006139 ! nucleobase-containing compound metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0045935
@@ -290842,23 +291477,27 @@
 name: alcohol catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [GOC:ai]
+subset: gocheck_do_not_annotate
 synonym: "alcohol breakdown" EXACT []
 synonym: "alcohol catabolism" EXACT []
 synonym: "alcohol degradation" EXACT []
 is_a: GO:0006066 ! alcohol metabolic process
 is_a: GO:0044282 ! small molecule catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0046165
 name: alcohol biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [GOC:ai]
+subset: gocheck_do_not_annotate
 synonym: "alcohol anabolism" EXACT []
 synonym: "alcohol biosynthesis" EXACT []
 synonym: "alcohol formation" EXACT []
 synonym: "alcohol synthesis" EXACT []
 is_a: GO:0006066 ! alcohol metabolic process
 is_a: GO:0044283 ! small molecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0046166
@@ -294152,12 +294791,15 @@
 
 [Term]
 id: GO:0046436
-name: D-alanine metabolic process
+name: obsolete D-alanine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-alanine, the D-enantiomer of the amino acid alanine, i.e. (2R)-2-aminopropanoic acid." [GOC:ai, GOC:jsg]
+def: "OBSOLETE. The chemical reactions and pathways involving D-alanine, the D-enantiomer of the amino acid alanine, i.e. (2R)-2-aminopropanoic acid." [GOC:ai, GOC:jsg]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-alanine metabolism" EXACT []
-is_a: GO:0006522 ! alanine metabolic process
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
+consider: GO:0030632
+consider: GO:0055130
 
 [Term]
 id: GO:0046437
@@ -294188,7 +294830,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-cysteine, the L-enantiomer of 2-amino-3-mercaptopropanoic acid, i.e. (2R)-2-amino-3-mercaptopropanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
 synonym: "L-cysteine metabolism" EXACT []
-is_a: GO:0006534 ! cysteine metabolic process
+is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
 
@@ -295153,8 +295795,8 @@
 xref: EC:5.3.1.23
 xref: KEGG_REACTION:R04420
 xref: MetaCyc:5.3.1.23-RXN
-xref: Reactome:R-HSA-1237096 "Methylthio-ribose-P = Methylthio-ribulose-P"
-xref: Reactome:R-HSA-1299507 "Methylthio-ribulose-P = Methylthio-ribose-P"
+xref: Reactome:R-HSA-1237096 "Methylthio-ribose-P => Methylthio-ribulose-P"
+xref: Reactome:R-HSA-1299507 "Methylthio-ribulose-P => Methylthio-ribose-P"
 xref: RHEA:19989
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.23
@@ -296437,21 +297079,23 @@
 synonym: "sphingolipid flippase activity" EXACT []
 synonym: "sphingolipid floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "sphingolipid-translocating ATPase activity" RELATED []
-is_a: GO:0046624 ! sphingolipid transporter activity
+is_a: GO:0046624 ! sphingolipid intramembrane carrier activity
 is_a: GO:0140328 ! floppase activity
 relationship: part_of GO:0099039 ! sphingolipid translocation
 property_value: skos:broadMatch RHEA:66132
 
 [Term]
 id: GO:0046624
-name: sphingolipid transporter activity
+name: sphingolipid intramembrane carrier activity
 namespace: molecular_function
 def: "Enables the directed movement of sphingolipids into, out of or within a cell, or between cells. Sphingolipids are a class of lipids containing the long-chain amine diol sphingosine or a closely related base (a sphingoid)." [GOC:ai, ISBN:0198506732]
 subset: gocheck_do_not_annotate
+synonym: "sphingolipid intramembrane transporter activity" BROAD []
 xref: Reactome:R-HSA-9695890 "SPNS2,MFSD2B transport S1P from cytosol to extracellular region"
 xref: Reactome:R-HSA-9843721 "ABCC1,ABCG2 transport C18-S1P to extracellular region"
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31121" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0046625
@@ -297133,7 +297777,9 @@
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an antibiotic stimulus. An antibiotic is a chemical substance produced by a microorganism which has the capacity to inhibit the growth of or to kill other microorganisms." [GOC:ai, GOC:ef]
 subset: goslim_chembl
 synonym: "antibiotic susceptibility/resistance" RELATED []
+xref: Reactome:R-HSA-9638771 "Action of antimicrobials"
 xref: Reactome:R-HSA-9639775 "Antimicrobial action and antimicrobial resistance in Mtb"
+xref: Reactome:R-HSA-9913143 "Antimicrobial resistance"
 is_a: GO:0042221 ! response to chemical
 
 [Term]
@@ -298390,12 +299036,12 @@
 
 [Term]
 id: GO:0046820
-name: 4-amino-4-deoxychorismate synthase activity
+name: aminodeoxychorismate synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamine + chorismate = 4-amino-4-deoxychorismate + L-glutamate. It is composed of two enzymatic activities (which may be present on one or two polypeptides); the first is a glutaminase which yields ammonia from glutamine, releasing glutamate. The ammonia is used by the second activity which catalyzes the amination of chorismate to form 4-amino-4-deoxychorismate." [EC:2.6.1.85, RHEA:11672]
 comment: Note that the name 'para-aminobenzoic acid synthase' was initially given to the 'aminodeoxychorismate synthase' activity before the additional 4-amino-4-deoxychorismate lyase activity was discovered. It is the lyase activity that actually produces para-aminobenzoic acid from 4-amino-4-deoxychorismate.
+synonym: "4-amino-4-deoxychorismate synthase activity" EXACT []
 synonym: "ADC synthase activity" RELATED [EC:2.6.1.85]
-synonym: "aminodeoxychorismate synthase activity" RELATED []
 synonym: "chorismate:L-glutamine amido-ligase activity" RELATED [EC:2.6.1.85]
 synonym: "p-aminobenzoate synthetase" RELATED []
 synonym: "PabB activity" RELATED [EC:2.6.1.85]
@@ -298409,6 +299055,7 @@
 property_value: skos:exactMatch EC:2.6.1.85
 property_value: skos:exactMatch RHEA:11672
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0046821
@@ -299488,7 +300135,7 @@
 id: GO:0046912
 name: acyltransferase activity, acyl groups converted into alkyl on transfer
 namespace: molecular_function
-def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor), with the acyl group being converted into alkyl on transfer." [GOC:jl]
+def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor), with the acyl group being converted into alkyl on transfer." [EC:2.3.3.-, GOC:jl]
 synonym: "acyltransferase, acyl groups converted into alkyl on transfer" EXACT []
 synonym: "transferase activity, transferring acyl groups, acyl groups converted into alkyl on transfer" EXACT []
 xref: EC:2.3.3.-
@@ -299564,7 +300211,7 @@
 synonym: "alpha(1,3)-fucosyltransferase activity" EXACT []
 synonym: "alpha-(1,3)-fucosyltransferase activity" EXACT []
 synonym: "alpha-1,3-fucosyltransferase activity" EXACT []
-xref: Reactome:R-HSA-9603984 "FUT4,5,9,(10,11) transfer Fuc to Type 2 chains to form LeX"
+xref: Reactome:R-HSA-9603984 "FUT4,5,9 transfer Fuc to Type 2 chains to form LeX"
 xref: Reactome:R-HSA-9605609 "FUT3 transfers Fuc to Type 1 MSGG to form sLeA"
 xref: Reactome:R-HSA-9605644 "FUT3 transfers Fuc to Type 1 DSGG to form dsLeA"
 xref: Reactome:R-HSA-9605682 "FUT3,5,6,7 transfers Fuc to Type 2 MSGG to form sLeX"
@@ -299595,7 +300242,6 @@
 xref: Reactome:R-HSA-5173192 "POFUT2 transfers fucose to TSR domain-containing proteins"
 is_a: GO:0008417 ! fucosyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
-relationship: part_of GO:0036066 ! protein O-linked glycosylation via fucose
 property_value: skos:exactMatch EC:2.4.1.221
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -299969,7 +300615,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving ketone bodies, any one of the three substances: acetoacetate, D-3-hydroxybutyrate (beta-hydroxybutyrate) or acetone, as carried out by individual cells. Although 3-hydroxybutyrate is not a ketone, it is classed as a ketone body because it exists in an equilibrium with acetoacetate. Ketone bodies may accumulate in excessive amounts in the body in starvation, diabetes mellitus or in other defects of carbohydrate metabolism." [ISBN:0198506732]
 comment: The reason for obsoletion is that this term is redundant to GO:1902224, ketone body metabolic process
 synonym: "cellular ketone body metabolism" EXACT []
-xref: Reactome:R-HSA-74182 "Ketone body metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:1902224
@@ -300138,7 +300783,7 @@
 synonym: "3'-phosphoadenosine 5'-phosphosulphate transporter activity" EXACT []
 synonym: "adenosine 3'-phosphate 5'-phosphosulfate transmembrane transporter activity" EXACT []
 synonym: "PAPS transporter activity" EXACT []
-xref: Reactome:R-HSA-741449 "SLC35B2,3 transport cytosolic PAPS to Golgi lumen"
+xref: Reactome:R-HSA-741449 "SLC35B2,3 exchange cytosolic PAPS with Golgi PAP"
 is_a: GO:0000295 ! adenine nucleotide transmembrane transporter activity
 is_a: GO:0005346 ! purine ribonucleotide transmembrane transporter activity
 is_a: GO:1901682 ! sulfur compound transmembrane transporter activity
@@ -300287,6 +300932,8 @@
 synonym: "histone methyltransferase activity (H3-K27 specific)" EXACT []
 synonym: "histone-H3K27 methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-3240295 "PRC2-EZH2 trimethylates nucleosomes associated with CDKN2A promoter"
+xref: Reactome:R-HSA-9764932 "PRC2 (EZH2) core complex methylates nucleosomes at CDH1 gene promoter"
+xref: Reactome:R-HSA-9909631 "Trimethylation of CD274 gene promoter by EZH2"
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24337" xsd:anyURI
@@ -301348,6 +301995,7 @@
 xref: EC:1.1.1.64 {status="skos:exactMatch"}
 xref: KEGG_REACTION:R01838
 xref: MetaCyc:1.1.1.64-RXN
+xref: Reactome:R-HSA-193064 "HSD17B3-like proteins reducde ANDST to TEST"
 xref: RHEA:14981
 xref: Wikipedia:Testosterone_17beta-dehydrogenase_(NADP+)
 is_a: GO:0072582 ! 17-beta-hydroxysteroid dehydrogenase (NADP+) activity
@@ -302698,6 +303346,7 @@
 synonym: "P2C reductase activity" RELATED [EC:1.5.1.21]
 xref: EC:1.5.1.21
 xref: MetaCyc:RXN-8166
+xref: Reactome:R-HSA-5693347 "CRYM reduces P2C to PPCA"
 xref: RHEA:12524
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.21
@@ -302730,7 +303379,6 @@
 synonym: "thiomorpholine-3-carboxylate:NAD(P)+ 5,6-oxidoreductase activity" RELATED [EC:1.5.1.25]
 xref: EC:1.5.1.25
 xref: MetaCyc:1.5.1.25-RXN
-xref: Reactome:R-HSA-5693347 "CRYM reduces P2C to PPCA"
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.25
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -303156,7 +303804,7 @@
 id: GO:0047151
 name: tRNA (uracil(54)-C5)-methyltransferase activity, 5,10-methylenetetrahydrofolate-dependent
 namespace: molecular_function
-def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + tRNA containing uridine at position 54 + FADH + H+ = tetrahydrofolate + tRNA containing ribothymidine at position 54 + FAD+." [EC:2.1.1.74, MetaCyc:2.1.1.74-RXN]
+def: "Catalysis of the reaction: uridine54 in tRNA + (6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + NAD(P)H + H+ = 5-methyluridine54 in tRNA + (6S)-5,6,7,8-tetrahydrofolate + NAD(P)+." [EC:2.1.1.74]
 synonym: "5,10-methylenetetrahydrofolate:tRNA (uracil-5-)-methyltransferase activity" RELATED [EC:2.1.1.74]
 synonym: "5,10-methylenetetrahydrofolate:tRNA-UPsiC (uracil-5-)-methyl-transferase activity" RELATED [EC:2.1.1.74]
 synonym: "folate-dependent ribothymidyl synthase activity" RELATED [EC:2.1.1.74]
@@ -303167,9 +303815,11 @@
 xref: KEGG_REACTION:R03704
 xref: MetaCyc:2.1.1.74-RXN
 xref: RHEA:16873
+xref: RHEA:62372
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
 property_value: skos:exactMatch EC:2.1.1.74
-property_value: skos:exactMatch RHEA:16873
+property_value: skos:narrowMatch RHEA:16873
+property_value: skos:narrowMatch RHEA:62372
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26118" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -303976,7 +304626,7 @@
 id: GO:0047198
 name: L-cysteine-S-conjugate N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-substituted L-cysteine + acetyl-CoA = S-substituted N-acetyl-L-cysteine + CoA + H+." [EC:2.3.1.80, RHEA:19213]
+def: "Catalysis of the reaction: S-substituted L-cysteine + acetyl-CoA = S-substituted N-acetyl-L-cysteine + CoA + H+." [RHEA:19213]
 synonym: "acetyl-CoA:S-substituted L-cysteine N-acetyltransferase activity" RELATED [EC:2.3.1.80]
 synonym: "cysteine-S-conjugate N-acetyltransferase activity" EXACT []
 xref: EC:2.3.1.80
@@ -304008,7 +304658,7 @@
 id: GO:0047200
 name: tetrahydrodipicolinate N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + acetyl-CoA + H2O = L-2-acetamido-6-oxopimelate + CoA." [EC:2.3.1.89, RHEA:13085]
+def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + acetyl-CoA + H2O = L-2-acetamido-6-oxopimelate + CoA." [RHEA:13085]
 synonym: "acetyl-CoA:(S)-2,3,4,5-tetrahydrodipicolinate-2,6-dicarboxylate N2-acetyltransferase activity" RELATED [EC:2.3.1.89]
 synonym: "acetyl-CoA:(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate 2-N-acetyltransferase activity" RELATED [EC:2.3.1.89]
 synonym: "acetyl-CoA:L-2,3,4,5-tetrahydrodipicolinate N2-acetyltransferase activity" RELATED [EC:2.3.1.89]
@@ -304411,7 +305061,6 @@
 synonym: "uridine diphosphoacetylglucosamine-mucin beta(1->3)-acetylglucosaminyltransferase (elongating)" RELATED [EC:2.4.1.146]
 xref: EC:2.4.1.146
 xref: MetaCyc:2.4.1.146-RXN
-xref: Reactome:R-HSA-5617037 "POMGNT1 transfers GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 xref: Reactome:R-HSA-5617096 "Defective POMGNT1 does not transfer GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 is_a: GO:0140103 ! catalytic activity, acting on a glycoprotein
@@ -304431,6 +305080,7 @@
 synonym: "uridine diphosphoacetylglucosamine-mucin beta(1->3)-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.147]
 xref: EC:2.4.1.147
 xref: MetaCyc:2.4.1.147-RXN
+xref: Reactome:R-HSA-914010 "B3GNT6 transfers GlcNAc to Tn antigen"
 xref: RHEA:46880
 xref: RHEA:46884
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
@@ -304671,10 +305321,10 @@
 synonym: "uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II activity" RELATED [EC:2.4.1.175]
 xref: EC:2.4.1.175
 xref: MetaCyc:RXN-16325
-xref: Reactome:R-HSA-1971482 "The addition of GalNAc to the terminal glucuronate residue forms chondroitin"
-xref: Reactome:R-HSA-1971487 "CHPF,CHSY3 transfer GalNAc to chondroitin"
+xref: Reactome:R-HSA-1971482 "CSGALNACTs transfer GalNAc to CS linker glycan"
+xref: Reactome:R-HSA-1971487 "Chondroitin polymerase complex transfers GalNAc to CS chain"
 xref: Reactome:R-HSA-3595176 "Defective CHSY1 does not transfer GalNAc to chondroitin"
-xref: Reactome:R-HSA-9632033 "CHSY1 transfers GalNAc to chondroitin"
+xref: Reactome:R-HSA-9632033 "CSGALNACTs transfer GalNAc to CS chain"
 is_a: GO:0008376 ! acetylgalactosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.175
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -305550,6 +306200,7 @@
 synonym: "monosialoganglioside sialyltransferase" EXACT [EC:2.4.3.2]
 xref: EC:2.4.3.2
 xref: MetaCyc:RXN-18256
+xref: Reactome:R-HSA-981497 "ST3GAL1-4 transfers sialic acid to the T antigen at the alpha 3 position"
 xref: RHEA:52380
 is_a: GO:0008373 ! sialyltransferase activity
 property_value: skos:exactMatch EC:2.4.3.2
@@ -305689,39 +306340,41 @@
 id: GO:0047296
 name: homospermidine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 putrescine = NH3 + sym-homospermidine." [EC:2.5.1.44]
+def: "Catalysis of the reaction: 2 putrescine = sym-homospermidine + NH4+." [RHEA:18645]
 synonym: "putrescine:putrescine 4-aminobutyltransferase (ammonia-forming)" RELATED [EC:2.5.1.44]
-xref: EC:2.5.1.44
 xref: MetaCyc:2.5.1.44-RXN
 xref: RHEA:18645
 is_a: GO:0016765 ! transferase activity, transferring alkyl or aryl (other than methyl) groups
-property_value: skos:exactMatch EC:2.5.1.44
+property_value: skos:broadMatch EC:2.5.1.44
 property_value: skos:exactMatch RHEA:18645
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0047297
-name: asparagine-oxo-acid transaminase activity
+name: L-asparagine:oxo-acid transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2-oxo acid + L-asparagine = an amino acid + 2-oxosuccinamate." [RHEA:19813]
+def: "Catalysis of the reaction: L-asparagine + a 2-oxocarboxylate = 2-oxosuccinamate + an L-alpha-amino acid." [RHEA:19813]
 synonym: "asparagine--oxo-acid aminotransferase activity" RELATED [EC:2.6.1.14]
 synonym: "asparagine-keto acid aminotransferase activity" RELATED [EC:2.6.1.14]
 synonym: "asparagine-oxo-acid aminotransferase activity" EXACT []
+synonym: "asparagine-oxo-acid transaminase activity" EXACT []
 synonym: "L-asparagine:2-oxo-acid aminotransferase activity" RELATED [EC:2.6.1.14]
 xref: EC:2.6.1.14
 xref: MetaCyc:2.6.1.14-RXN
 xref: RHEA:19813
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.14
 property_value: skos:exactMatch RHEA:19813
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047298
-name: (S)-3-amino-2-methylpropionate transaminase activity
+name: (S)-3-amino-2-methylpropionate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate." [EC:2.6.1.22, RHEA:13993]
+def: "Catalysis of the reaction: (S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate." [RHEA:13993]
 synonym: "(S)-3-amino-2-methylpropanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.22]
 synonym: "(S)-3-amino-2-methylpropionate aminotransferase activity" EXACT []
+synonym: "(S)-3-amino-2-methylpropionate transaminase activity" EXACT []
 synonym: "beta-aminobutyric transaminase activity" RELATED [EC:2.6.1.22]
 synonym: "beta-aminoisobutyrate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.22]
 synonym: "L-3-aminoisobutyrate aminotransferase activity" RELATED [EC:2.6.1.22]
@@ -305736,32 +306389,36 @@
 property_value: skos:exactMatch EC:2.6.1.22
 property_value: skos:exactMatch RHEA:13993
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047299
-name: tryptophan-phenylpyruvate transaminase activity
+name: L-tryptophan:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-tryptophan = 3-(indol-3-yl)pyruvate + L-phenylalanine." [EC:2.6.1.28, RHEA:13741]
+def: "Catalysis of the reaction: 3-phenylpyruvate + L-tryptophan = indole-3-pyruvate + L-phenylalanine." [RHEA:13741]
 synonym: "L-tryptophan-alpha-ketoisocaproate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "L-tryptophan:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "tryptophan--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "tryptophan-phenylpyruvate aminotransferase activity" EXACT []
+synonym: "tryptophan-phenylpyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.28
 xref: KEGG_REACTION:R01376
 xref: MetaCyc:2.6.1.28-RXN
 xref: RHEA:13741
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.28
 property_value: skos:exactMatch RHEA:13741
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047300
-name: pyridoxamine-pyruvate transaminase activity
+name: pyridoxamine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine + pyruvate = L-alanine + pyridoxal." [EC:2.6.1.30, RHEA:12841]
+def: "Catalysis of the reaction: pyridoxamine + pyruvate = L-alanine + pyridoxal." [RHEA:12841]
 synonym: "pyridoxamine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.30]
 synonym: "pyridoxamine-pyruvate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-pyruvate transaminase activity" EXACT []
 synonym: "pyridoxamine-pyruvic transaminase" BROAD [EC:2.6.1.30]
 synonym: "pyridoxamine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.30]
 synonym: "pyridoxamineu-pyruvic transaminase activity" RELATED [EC:2.6.1.30]
@@ -305773,32 +306430,36 @@
 property_value: skos:exactMatch EC:2.6.1.30
 property_value: skos:exactMatch RHEA:12841
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047301
-name: valine-3-methyl-2-oxovalerate transaminase activity
+name: L-valine:3-methyl-2-oxovalerate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + L-valine = 3-methyl-2-oxobutanoate + L-isoleucine." [EC:2.6.1.32, RHEA:11468]
+def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + L-valine = 3-methyl-2-oxobutanoate + L-isoleucine." [RHEA:11468]
 synonym: "L-valine:(S)-3-methyl-2-oxopentanoate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--3-methyl-2-oxovalerate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--isoleucine aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--isoleucine transaminase activity" RELATED [EC:2.6.1.32]
 synonym: "valine-2-keto-methylvalerate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine-3-methyl-2-oxovalerate aminotransferase activity" EXACT []
+synonym: "valine-3-methyl-2-oxovalerate transaminase activity" EXACT []
 xref: EC:2.6.1.32
 xref: KEGG_REACTION:R02200
 xref: MetaCyc:2.6.1.32-RXN
 xref: RHEA:11468
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.32
 property_value: skos:exactMatch RHEA:11468
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047302
-name: UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose transaminase activity
+name: UDP-N-acetylbacillosamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + UDP-2-acetamido-4-amino-2,4,6-trideoxy-D-glucose = L-glutamate + UDP-2-acetamido-4-dehydro-2,6-dideoxy-beta-D-glucose." [PMID:16286454, RHEA:31663]
+def: "Catalysis of the reaction: UDP-N-acetylbacillosamine + 2-oxoglutarate = UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-hex-4-ulose + L-glutamate." [PMID:16286454, RHEA:31663]
+synonym: "UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose transaminase activity" BROAD []
 synonym: "UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.34]
 synonym: "UDP-4-amino-2-acetamido-2,4,6-trideoxyglucose aminotransferase activity" EXACT []
 synonym: "UDP-4-amino-2-acetamido-2,4,6-trideoxyglucose transaminase activity" EXACT []
@@ -305811,30 +306472,33 @@
 property_value: skos:exactMatch EC:2.6.1.34
 property_value: skos:exactMatch RHEA:31663
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047303
-name: glycine-oxaloacetate transaminase activity
+name: glycine:oxaloacetate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + oxaloacetate = L-aspartate + glyoxylate." [EC:2.6.1.35, RHEA:17141]
+def: "Catalysis of the reaction: glycine + oxaloacetate = L-aspartate + glyoxylate." [RHEA:17141]
 synonym: "glycine--oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 synonym: "glycine-oxalacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 synonym: "glycine-oxaloacetate aminotransferase activity" EXACT []
+synonym: "glycine-oxaloacetate transaminase activity" EXACT []
 synonym: "glycine:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 xref: EC:2.6.1.35
 xref: KEGG_REACTION:R00373
 xref: MetaCyc:2.6.1.35-RXN
 xref: RHEA:17141
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.35
 property_value: skos:exactMatch RHEA:17141
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047304
-name: 2-aminoethylphosphonate-pyruvate transaminase activity
+name: 2-aminoethylphosphonate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (2-aminoethyl)phosphonate + pyruvate = L-alanine + phosphonoacetaldehyde." [EC:2.6.1.37, RHEA:17021]
+def: "Catalysis of the reaction: (2-aminoethyl)phosphonate + pyruvate = L-alanine + phosphonoacetaldehyde." [RHEA:17021]
 synonym: "(2-aminoethyl)phosphonate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "(2-aminoethyl)phosphonate transaminase activity" RELATED [EC:2.6.1.37]
 synonym: "(2-aminoethyl)phosphonate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.37]
@@ -305843,6 +306507,7 @@
 synonym: "2-aminoethylphosphonate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "2-aminoethylphosphonate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "2-aminoethylphosphonate-pyruvate aminotransferase activity" EXACT []
+synonym: "2-aminoethylphosphonate-pyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.37
 xref: KEGG_REACTION:R04152
 xref: MetaCyc:2.6.1.37-RXN
@@ -305851,20 +306516,21 @@
 property_value: skos:exactMatch EC:2.6.1.37
 property_value: skos:exactMatch RHEA:17021
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047305
-name: (R)-3-amino-2-methylpropionate-pyruvate transaminase activity
+name: (R)-3-amino-2-methylpropionate:pyruvate transaminase activity
 namespace: molecular_function
 alt_id: GO:0047314
-def: "Catalysis of the reaction: (2R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate + L-alanine." [EC:2.6.1.40, RHEA:18393]
-comment: Note that this function was EC:2.6.1.61.
+def: "Catalysis of the reaction: (R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate + L-alanine." [RHEA:18393]
 synonym: "(R)-3-amino-2-methylpropanoate aminotransferase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropanoate transaminase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropanoate:pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "(R)-3-amino-2-methylpropionate transaminase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropionate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "(R)-3-amino-2-methylpropionate-pyruvate aminotransferase activity" EXACT []
+synonym: "(R)-3-amino-2-methylpropionate-pyruvate transaminase activity" EXACT []
 synonym: "beta-aminoisobutyrate--pyruvate transaminase activity" RELATED [EC:2.6.1.40]
 synonym: "beta-aminoisobutyrate-pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "D-3-aminoisobutyrate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
@@ -305881,33 +306547,37 @@
 property_value: skos:exactMatch EC:2.6.1.40
 property_value: skos:exactMatch RHEA:18393
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047306
-name: D-methionine-pyruvate transaminase activity
+name: D-methionine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine." [EC:2.6.1.41, RHEA:23836]
+def: "Catalysis of the reaction: D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine." [RHEA:23836]
 synonym: "D-methionine aminotransferase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine transaminase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine-pyruvate aminotransferase activity" EXACT []
+synonym: "D-methionine-pyruvate transaminase activity" EXACT []
 synonym: "D-methionine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.41]
 xref: EC:2.6.1.41
 xref: KEGG_REACTION:R03001
 xref: MetaCyc:2.6.1.41-RXN
 xref: RHEA:23836
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.41
 property_value: skos:exactMatch RHEA:23836
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047307
-name: diaminobutyrate-pyruvate transaminase activity
+name: L-2,4-diaminobutyrate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-2,4-diaminobutyrate + pyruvate = L-alanine + L-aspartate 4-semialdehyde." [EC:2.6.1.46, RHEA:12380]
+def: "Catalysis of the reaction: L-2,4-diaminobutyrate + pyruvate = L-alanine + L-aspartate 4-semialdehyde." [RHEA:12380]
 synonym: "diaminobutyrate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.46]
 synonym: "diaminobutyrate-pyruvate aminotransferase activity" EXACT []
+synonym: "diaminobutyrate-pyruvate transaminase activity" EXACT []
 synonym: "L-2,4-diaminobutanoate:pyruvate aminotransferase activity" RELATED [EC:2.6.1.46]
 synonym: "L-diaminobutyric acid transaminase activity" RELATED [EC:2.6.1.46]
 xref: EC:2.6.1.46
@@ -305918,12 +306588,13 @@
 property_value: skos:exactMatch EC:2.6.1.46
 property_value: skos:exactMatch RHEA:12380
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047308
-name: alanine-oxomalonate transaminase activity
+name: L-alanine:oxomalonate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + oxomalonate = aminomalonate + pyruvate." [EC:2.6.1.47, RHEA:18809]
+def: "Catalysis of the reaction: L-alanine + oxomalonate = aminomalonate + pyruvate." [RHEA:18809]
 synonym: "alanine--oxomalonate aminotransferase activity" RELATED [EC:2.6.1.47]
 synonym: "alanine-ketomalonate (mesoxalate) transaminase activity" RELATED [EC:2.6.1.47]
 synonym: "alanine-oxomalonate aminotransferase activity" EXACT []
@@ -305933,19 +306604,21 @@
 xref: KEGG_REACTION:R02970
 xref: MetaCyc:2.6.1.47-RXN
 xref: RHEA:18809
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.47
 property_value: skos:exactMatch RHEA:18809
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047309
-name: dihydroxyphenylalanine transaminase activity
+name: dihydroxyphenylalanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-dopa = 3,4-dihydroxyphenylpyruvate + L-glutamate." [EC:2.6.1.49, RHEA:15273]
+def: "Catalysis of the reaction: L-dopa + 2-oxoglutarate = 3-(3,4-dihydroxyphenyl)pyruvate + L-glutamate." [RHEA:15273]
 synonym: "3,4-dihydroxy-L-phenylalanine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.49]
 synonym: "aspartate-DOPP transaminase (ADT)" RELATED [EC:2.6.1.49]
 synonym: "dihydroxyphenylalanine aminotransferase activity" EXACT []
+synonym: "dihydroxyphenylalanine transaminase activity" EXACT []
 synonym: "DOPA aminotransferase activity" RELATED [EC:2.6.1.49]
 synonym: "dopa transaminase activity" RELATED [EC:2.6.1.49]
 synonym: "glutamate-DOPP transaminase (GDT)" RELATED [EC:2.6.1.49]
@@ -305955,19 +306628,20 @@
 xref: KEGG_REACTION:R02077
 xref: MetaCyc:2.6.1.49-RXN
 xref: RHEA:15273
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.49
 property_value: skos:exactMatch RHEA:15273
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0047310
-name: glutamine-scyllo-inositol transaminase activity
+name: L-glutamine:scyllo-inositol transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4,6/3,5-pentahydroxycyclohexanone + L-glutamine = 1-amino-1-deoxy-scyllo-inositol + 2-oxoglutaramate." [EC:2.6.1.50, RHEA:22920]
+def: "Catalysis of the reaction: scyllo-inosose + L-glutamine = 1-amino-1-deoxy-scyllo-inositol + 2-oxoglutaramate." [RHEA:22920]
 synonym: "glutamine scyllo-inosose aminotransferase activity" RELATED [EC:2.6.1.50]
 synonym: "glutamine--scyllo-inosose aminotransferase activity" RELATED [EC:2.6.1.50]
 synonym: "glutamine--scyllo-inosose transaminase activity" RELATED [EC:2.6.1.50]
+synonym: "glutamine-scyllo-inositol transaminase activity" EXACT []
 synonym: "glutamine-scyllo-inosose aminotransferase activity" EXACT []
 synonym: "glutamine-scyllo-inosose transaminase activity" EXACT []
 synonym: "L-glutamine-keto-scyllo-inositol aminotransferase activity" RELATED [EC:2.6.1.50]
@@ -305977,17 +306651,19 @@
 xref: KEGG_REACTION:R02781
 xref: MetaCyc:2.6.1.50-RXN
 xref: RHEA:22920
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.50
 property_value: skos:exactMatch RHEA:22920
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047311
-name: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase activity
+name: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine." [EC:2.6.1.56, RHEA:15497]
+def: "Catalysis of the reaction: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine." [RHEA:15497]
 synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol aminotransferase activity" EXACT []
+synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase activity" EXACT []
 synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol:pyruvate aminotransferase activity" RELATED [EC:2.6.1.56]
 synonym: "guanidinoaminodideoxy-scyllo-inositol-pyruvate aminotransferase activity" RELATED [EC:2.6.1.56]
 synonym: "L-alanine-N-amidino-3-(or 5-)keto-scyllo-inosamine transaminase activity" RELATED [EC:2.6.1.56]
@@ -305999,53 +306675,58 @@
 property_value: skos:exactMatch EC:2.6.1.56
 property_value: skos:exactMatch RHEA:15497
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047312
-name: L-phenylalanine-pyruvate transaminase activity
+name: L-phenylalanine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyruvate + L-phenylalanine = phenylpyruvate + L-alanine." [RHEA:13053]
+def: "Catalysis of the reaction: L-phenylalanine + pyruvate = 3-phenylpyruvate + L-alanine." [RHEA:13053]
 synonym: "histidine aminotransferase activity" RELATED [GOC:kad]
 synonym: "histidine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.58, GOC:kad]
 synonym: "L-histidine:pyruvate aminotransferase activity" RELATED [GOC:kad]
 synonym: "L-phenylalanine(L-histidine):pyruvate aminotransferase activity" BROAD [EC:2.6.1.58]
+synonym: "L-phenylalanine-pyruvate transaminase activity" EXACT []
 synonym: "L-phenylalanine:pyruvate aminotransferase activity" EXACT []
-synonym: "L-phenylalanine:pyruvate transaminase activity" EXACT [GOC:kad]
 synonym: "phenylalanine (histidine) aminotransferase activity" BROAD [EC:2.6.1.58]
 synonym: "phenylalanine(histidine) aminotransferase activity" BROAD []
 synonym: "phenylalanine(histidine) transaminase activity" BROAD [EC:2.6.1.58]
 synonym: "phenylalanine(histidine):pyruvate aminotransferase activity" BROAD [EC:2.6.1.58]
 xref: KEGG_REACTION:R00692
 xref: MetaCyc:2.6.1.58-RXN
-xref: Reactome:R-HSA-893593 "phenylalanine + pyruvate => 3-(indol-3-yl)pyruvate + alanine"
+xref: Reactome:R-HSA-893593 "phenylalanine + pyruvate => keto-phenylpyruvate + alanine"
 xref: RHEA:13053
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:broadMatch EC:2.6.1.58
 property_value: skos:exactMatch RHEA:13053
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047313
-name: aromatic-amino-acid-glyoxylate transaminase activity
+name: aromatic-amino-acid:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glyoxylate + an aromatic amino acid = L-glycine + an aromatic oxo acid." [EC:2.6.1.60, MetaCyc:2.6.1.60-RXN]
+def: "Catalysis of the reaction: an aromatic L-alpha-amino acid + glyoxylate = an aromatic oxo-acid + glycine." [RHEA:10900]
 synonym: "aromatic-amino-acid--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.60]
 synonym: "aromatic-amino-acid-glyoxylate aminotransferase activity" EXACT []
+synonym: "aromatic-amino-acid-glyoxylate transaminase activity" EXACT []
 synonym: "aromatic-amino-acid:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.60]
 xref: EC:2.6.1.60
 xref: MetaCyc:2.6.1.60-RXN
 xref: RHEA:10900
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.60
 property_value: skos:exactMatch RHEA:10900
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047315
-name: kynurenine-glyoxylate transaminase activity
+name: L-kynurenine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate + glycine." [EC:2.6.1.63, RHEA:19249]
+def: "Catalysis of the reaction: L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate + glycine." [RHEA:19249]
 synonym: "kynurenine--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.63]
 synonym: "kynurenine-glyoxylate aminotransferase activity" EXACT []
+synonym: "kynurenine-glyoxylate transaminase activity" EXACT []
 synonym: "L-kynurenine:glyoxylate aminotransferase (cyclizing)" RELATED [EC:2.6.1.63]
 xref: EC:2.6.1.63
 xref: KEGG_REACTION:R01957
@@ -306055,34 +306736,38 @@
 property_value: skos:exactMatch EC:2.6.1.63
 property_value: skos:exactMatch RHEA:19249
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047316
-name: glutamine-phenylpyruvate transaminase activity
+name: L-glutamine:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-glutamine = 2-oxoglutaramate + L-phenylalanine." [EC:2.6.1.64, RHEA:17593]
+def: "Catalysis of the reaction: 3-phenylpyruvate + L-glutamine = 2-oxoglutaramate + L-phenylalanine." [RHEA:17593]
 synonym: "glutamine transaminase K activity" RELATED [EC:2.6.1.64]
 synonym: "glutamine--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.64]
 synonym: "glutamine-phenylpyruvate aminotransferase activity" EXACT []
+synonym: "glutamine-phenylpyruvate transaminase activity" EXACT []
 synonym: "L-glutamine:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.64]
 xref: EC:2.6.1.64
 xref: KEGG_REACTION:R01375
 xref: MetaCyc:2.6.1.64-RXN
 xref: RHEA:17593
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.64
 property_value: skos:exactMatch RHEA:17593
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047317
-name: N6-acetyl-beta-lysine transaminase activity
+name: N6-acetyl-beta-lysine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3-amino-6-acetamidohexanoate = L-glutamate + 3-oxo-6-acetamidohexanoate." [EC:2.6.1.65, MetaCyc:2.6.1.65-RXN]
+def: "Catalysis of the reaction: 2-oxoglutarate + 3-amino-6-acetamidohexanoate = L-glutamate + 3-oxo-6-acetamidohexanoate." [RHEA:16889]
 synonym: "6-acetamido-3-aminohexanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "epsilon-acetyl-beta-lysine aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "N(6)-acetyl-beta-lysine aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "N6-acetyl-beta-lysine aminotransferase activity" EXACT []
+synonym: "N6-acetyl-beta-lysine transaminase activity" EXACT []
 xref: EC:2.6.1.65
 xref: MetaCyc:2.6.1.65-RXN
 xref: RHEA:16889
@@ -306090,29 +306775,33 @@
 property_value: skos:exactMatch EC:2.6.1.65
 property_value: skos:exactMatch RHEA:16889
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047319
-name: aspartate-phenylpyruvate transaminase activity
+name: L-aspartate:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-aspartate = L-phenylalanine + oxaloacetate." [RHEA:14097]
+def: "Catalysis of the reaction: L-aspartate + 3-phenylpyruvate = oxaloacetate + L-phenylalanine." [RHEA:14097]
 synonym: "aspartate--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.70]
+synonym: "aspartate-phenylpyruvate transaminase activity" EXACT []
 synonym: "aspartate:phenylpyruvate aminotransferase activity" EXACT []
 synonym: "L-aspartate:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.70]
 xref: EC:2.6.1.70
 xref: KEGG_REACTION:R00695
 xref: MetaCyc:2.6.1.70-RXN
 xref: RHEA:14097
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.70
 property_value: skos:exactMatch RHEA:14097
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047320
-name: D-4-hydroxyphenylglycine transaminase activity
+name: D-4-hydroxyphenylglycine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + D-4-hydroxyphenylglycine = 4-hydroxyphenylglyoxylate + L-glutamate." [EC:2.6.1.72, RHEA:15589]
+def: "Catalysis of the reaction: 2-oxoglutarate + D-4-hydroxyphenylglycine = 4-hydroxyphenylglyoxylate + L-glutamate." [RHEA:15589]
 synonym: "D-4-hydroxyphenylglycine aminotransferase activity" EXACT []
+synonym: "D-4-hydroxyphenylglycine transaminase activity" EXACT []
 synonym: "D-4-hydroxyphenylglycine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.72]
 synonym: "D-hydroxyphenylglycine aminotransferase activity" RELATED [EC:2.6.1.72]
 xref: EC:2.6.1.72
@@ -306123,6 +306812,7 @@
 property_value: skos:exactMatch EC:2.6.1.72
 property_value: skos:exactMatch RHEA:15589
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047321
@@ -306190,7 +306880,6 @@
 synonym: "STK2" RELATED [EC:2.7.11.4]
 xref: EC:2.7.11.4
 xref: MetaCyc:2.7.11.4-RXN
-xref: Reactome:R-HSA-5693148 "BCKDK phosphorylates BCKDH"
 xref: Reactome:R-HSA-9912480 "BCKDK loss-of-function mutations do not phosphorylate BCKDH"
 xref: RHEA:17301
 is_a: GO:0004674 ! protein serine/threonine kinase activity
@@ -306299,7 +306988,7 @@
 id: GO:0047329
 name: phosphoramidate-hexose phosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: hexose + phosphoramidate = hexose 1-phosphate + NH3." [EC:2.7.1.62, MetaCyc:2.7.1.62-RXN]
+def: "Catalysis of the reaction: phosphoramidate + a D-hexose + H+ = alpha-D-hexose 1-phosphate + NH4+." [RHEA:10972]
 synonym: "phosphoramidate-hexose transphosphorylase activity" RELATED [EC:2.7.1.62]
 synonym: "phosphoramidate:hexose 1-phosphotransferase activity" RELATED [EC:2.7.1.62]
 synonym: "phosphoramidic-hexose transphosphorylase activity" RELATED [EC:2.7.1.62]
@@ -306687,6 +307376,7 @@
 xref: EC:2.7.7.40
 xref: KEGG_REACTION:R02921
 xref: MetaCyc:2.7.7.40-RXN
+xref: Reactome:R-HSA-9940844 "CRPPA dimer transfers cytidylyl to Rbt5P"
 xref: RHEA:12456
 is_a: GO:0070567 ! cytidylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.40
@@ -307335,6 +308025,7 @@
 synonym: "branched-chain oxo-acid dehydrogenase phosphatase activity" EXACT []
 xref: EC:3.1.3.52
 xref: MetaCyc:3.1.3.52-RXN
+xref: Reactome:R-HSA-5693153 "PPM1K dephosphorylates p-BCKDH"
 xref: RHEA:77247
 is_a: GO:0004721 ! phosphoprotein phosphatase activity
 property_value: skos:exactMatch EC:3.1.3.52
@@ -307767,7 +308458,7 @@
 id: GO:0047411
 name: 2-(acetamidomethylene)succinate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + 2-(acetamidomethylene)succinate = CO2 + NH3 + succinate semialdehyde + acetate." [EC:3.5.1.29, MetaCyc:3.5.1.29-RXN]
+def: "Catalysis of the reaction: 2-(acetamidomethylene)succinate + 2 H2O + H+ = succinate semialdehyde + acetate + NH4+ + CO2." [RHEA:10432]
 synonym: "2-(acetamidomethylene)succinate amidohydrolase (deaminating, decarboxylating)" RELATED [EC:3.5.1.29]
 synonym: "alpha-(N-acetylaminomethylene)succinic acid hydrolase activity" RELATED [EC:3.5.1.29]
 xref: EC:3.5.1.29
@@ -307867,7 +308558,7 @@
 id: GO:0047417
 name: N-carbamoyl-D-amino acid hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + N-carbamoyl-D-amino acid = CO2 + NH3 + D-amino acid." [EC:3.5.1.77, MetaCyc:3.5.1.77-RXN]
+def: "Catalysis of the reaction: an N-carbamoyl-D-amino acid + H2O + 2 H+ = a D-alpha-amino acid + NH4+ + CO2." [RHEA:11000]
 synonym: "N-carbamoyl-D-amino acid amidohydrolase activity" RELATED [EC:3.5.1.77]
 xref: EC:3.5.1.77
 xref: MetaCyc:3.5.1.77-RXN
@@ -307974,7 +308665,7 @@
 id: GO:0047424
 name: methylenediurea deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + methylenediurea = CO2 + 2 NH3 + N-hydroxymethylurea." [EC:3.5.3.21, MetaCyc:3.5.3.21-RXN, RHEA:15929]
+def: "Catalysis of the reaction: methylenediurea + 2 H2O + 2 H+ = N-(hydroxymethyl)urea + 2 NH4+ + CO2." [RHEA:15929]
 synonym: "methylenediurea aminohydrolase activity" RELATED [EC:3.5.3.21]
 synonym: "methylenediurease activity" RELATED [EC:3.5.3.21]
 xref: EC:3.5.3.21
@@ -308007,7 +308698,7 @@
 id: GO:0047426
 name: ricinine nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + ricinine = NH3 + 3-carboxy-4-methoxy-N-methyl-2-pyridone." [EC:3.5.5.2]
+def: "Catalysis of the reaction: 	ricinine + 2 H2O = 4-methoxy-1-methyl-2-oxo-1,2-dihydropyridine-3-carboxylate + NH4+." [RHEA:22704]
 synonym: "ricinine aminohydrolase activity" RELATED [EC:3.5.5.2]
 xref: EC:3.5.5.2
 xref: MetaCyc:3.5.5.2-RXN
@@ -308037,7 +308728,7 @@
 id: GO:0047428
 name: arylacetonitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + 4-chlorophenylacetonitrile = 4-chlorophenylacetate + NH3." [EC:3.5.5.5]
+def: "Catalysis of the reaction: 4-chlorophenylacetonitrile + 2 H2O = 4-chlorophenylacetate + NH4+." [RHEA:20657]
 synonym: "arylacetonitrile aminohydrolase activity" RELATED [EC:3.5.5.5]
 xref: EC:3.5.5.5
 xref: MetaCyc:3.5.5.5-RXN
@@ -308379,7 +309070,7 @@
 id: GO:0047447
 name: erythro-3-hydroxyaspartate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: erythro-3-hydroxy-L-aspartate = NH3 + oxaloacetic acid." [EC:4.3.1.20, MetaCyc:4.3.1.20-RXN]
+def: "Catalysis of the reaction: 3-hydroxy-L-aspartate = oxaloacetate + NH4+." [RHEA:19757]
 synonym: "3-hydroxyaspartate dehydratase activity" BROAD [EC:4.3.1.20]
 synonym: "erythro-3-hydroxy-L(s)-aspartate hydro-lyase (deaminating) activity" RELATED [EC:4.3.1.20]
 synonym: "erythro-3-hydroxy-Ls-aspartate ammonia-lyase (oxaloacetate-forming)" RELATED [EC:4.3.1.20]
@@ -308844,11 +309535,13 @@
 id: GO:0047473
 name: D-alanine [D-alanyl carrier protein] ligase activity
 namespace: molecular_function
-def: "Catalysis of the ATP-dependent activation of D-alanine and its transfer as a thiol ester to the phosphopantheinyl prosthetic group of a D-alanyl carrier protein, according to the reaction: ATP + D-alanine + a [D-alaninyl carrier protein] = a D-alanyl-[D-alanyl carrier protein] + AMP + diphosphate." [GOC:pg]
+def: "Catalysis of the ATP-dependent activation of D-alanine and its transfer as a thiol ester to the phosphopantheinyl prosthetic group of a D-alanyl carrier protein, according to the reaction: holo-[D-alanyl-carrier protein] + D-alanine + ATP = D-alanyl-[D-alanyl-carrier protein] + AMP + diphosphate." [RHEA:55132]
 synonym: "D-alanine-activating enzyme activity" RELATED []
 synonym: "D-alanine-D-alanyl carrier protein ligase activity" RELATED []
 synonym: "D-alanine-poly(phosphoribitol) ligase activity" BROAD []
 synonym: "D-alanine:poly(phosphoribitol) ligase (AMP-forming)" BROAD []
+xref: EC:6.2.1.54
+xref: RHEA:55132
 is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14237" xsd:anyURI
 
@@ -309774,13 +310467,17 @@
 id: GO:0047530
 name: 2,4-diaminopentanoate dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4-diaminopentanoate + H2O + NAD(P)+ = 2-amino-4-oxopentanoate + NH3 + NAD(P)H + H+." [EC:1.4.1.12, MetaCyc:24-DIAMINOPENTANOATE-DEHYDROGENASE-RXN]
+def: "Catalysis of the reaction: (2R,4S)-2,4-diaminopentanoate + NADP+ + H2O = (2R)-2-amino-4-oxopentanoate + NH4+ + NADPH + H+." [EC:1.4.1.12]
 synonym: "2,4-diaminopentanoate:NAD(P)+ oxidoreductase (deaminating)" RELATED [EC:1.4.1.12]
 synonym: "2,4-diaminopentanoic acid C4 dehydrogenase activity" RELATED [EC:1.4.1.12]
 xref: EC:1.4.1.12
 xref: MetaCyc:24-DIAMINOPENTANOATE-DEHYDROGENASE-RXN
+xref: RHEA:24192
+xref: RHEA:24196
 is_a: GO:0016639 ! oxidoreductase activity, acting on the CH-NH2 group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.4.1.12
+property_value: skos:narrowMatch RHEA:24192
+property_value: skos:narrowMatch RHEA:24196
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -309859,10 +310556,11 @@
 
 [Term]
 id: GO:0047536
-name: 2-aminoadipate transaminase activity
+name: L-2-aminoadipate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-2-aminoadipate = 2-oxoadipate + L-glutamate." [EC:2.6.1.39, RHEA:12601]
+def: "Catalysis of the reaction: L-2-aminoadipate + 2-oxoglutarate = 2-oxoadipate + L-glutamate." [RHEA:12601]
 synonym: "2-aminoadipate aminotransferase activity" EXACT []
+synonym: "2-aminoadipate transaminase activity" EXACT []
 synonym: "2-aminoadipic aminotransferase activity" RELATED [EC:2.6.1.39]
 synonym: "alpha-aminoadipate aminotransferase activity" RELATED [EC:2.6.1.39]
 synonym: "glutamate-alpha-ketoadipate transaminase activity" RELATED [EC:2.6.1.39]
@@ -309878,13 +310576,15 @@
 property_value: skos:exactMatch EC:2.6.1.39
 property_value: skos:exactMatch RHEA:12601
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047537
-name: 2-aminohexanoate transaminase activity
+name: L-2-aminohexanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate." [EC:2.6.1.67, MetaCyc:2-AMINOHEXANOATE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate." [RHEA:23600]
 synonym: "2-aminohexanoate aminotransferase activity" EXACT []
+synonym: "2-aminohexanoate transaminase activity" EXACT []
 synonym: "L-2-aminohexanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.67]
 synonym: "leucine L-norleucine: 2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.67]
 synonym: "norleucine (leucine) aminotransferase activity" RELATED [EC:2.6.1.67]
@@ -309897,6 +310597,7 @@
 property_value: skos:exactMatch EC:2.6.1.67
 property_value: skos:exactMatch RHEA:23600
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047538
@@ -310625,19 +311326,21 @@
 
 [Term]
 id: GO:0047578
-name: 4-hydroxyglutamate transaminase activity
+name: (4S)-4-hydroxy-L-glutamate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-hydroxy-L-glutamate + 2-oxoglutarate = 4-hydroxy-2-oxoglutarate + L-glutamate." [EC:2.6.1.23, MetaCyc:4-HYDROXYGLUTAMATE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: (4S)-4-hydroxy-L-glutamate + 2-oxoglutarate = (4R)-4-hydroxy-2-oxoglutarate + L-glutamate." [RHEA:31235]
 synonym: "4-hydroxy-L-glutamate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.23]
 synonym: "4-hydroxyglutamate aminotransferase activity" EXACT []
+synonym: "4-hydroxyglutamate transaminase activity" EXACT []
 xref: EC:2.6.1.23
 xref: MetaCyc:4-HYDROXYGLUTAMATE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-6784393 "PXLP-K279-GOT2 dimer transaminates 4-OH-L-glutamate to 4-hydroxy-2-oxoglutarate (HOG)"
-xref: RHEA:10480
+xref: RHEA:31235
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch EC:2.6.1.23
-property_value: skos:exactMatch RHEA:10480
+property_value: skos:exactMatch RHEA:31235
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047579
@@ -310792,7 +311495,7 @@
 id: GO:0047588
 name: 5-aminopentanamidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-aminopentanamide + H2O = 5-aminopentanoate + NH3." [EC:3.5.1.30, MetaCyc:5-AMINOPENTANAMIDASE-RXN]
+def: "Catalysis of the reaction: 5-aminopentanamide + H2O = 5-aminopentanoate + NH4+." [RHEA:15677]
 synonym: "5-aminonorvaleramidase activity" RELATED [EC:3.5.1.30]
 synonym: "5-aminopentanamide amidohydrolase activity" RELATED [EC:3.5.1.30]
 synonym: "5-aminovaleramidase activity" RELATED [EC:3.5.1.30]
@@ -310806,11 +311509,12 @@
 
 [Term]
 id: GO:0047589
-name: 5-aminovalerate transaminase activity
+name: 5-aminovalerate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 5-aminopentanoate = 5-oxopentanoate + L-glutamate." [EC:2.6.1.48, RHEA:10212]
+def: "Catalysis of the reaction: 5-aminopentanoate + 2-oxoglutarate = 5-oxopentanoate + L-glutamate." [RHEA:10212]
 synonym: "5-aminopentanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.48]
 synonym: "5-aminovalerate aminotransferase activity" EXACT []
+synonym: "5-aminovalerate transaminase activity" EXACT []
 synonym: "delta-aminovalerate aminotransferase activity" RELATED [EC:2.6.1.48]
 synonym: "delta-aminovalerate transaminase activity" RELATED [EC:2.6.1.48]
 xref: EC:2.6.1.48
@@ -310821,6 +311525,7 @@
 property_value: skos:exactMatch EC:2.6.1.48
 property_value: skos:exactMatch RHEA:10212
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047590
@@ -311393,7 +312098,7 @@
 id: GO:0047623
 name: adenosine-phosphate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an adenosine-phosphate + H20 = an inosine phosphate + NH3. Catalyzes the deamination of AMP, ADP or ATP." [EC:3.5.4.17]
+def: "Catalysis of the reaction: an adenosine-phosphate + H20 = an inosine phosphate + NH4+. Catalyzes the deamination of AMP, ADP or ATP." [EC:3.5.4.17]
 comment: Consider instead annotating to one of the more specific terms: AMP deaminase activity ; GO:0003876, ADP deaminase activity ; GO:0047629, or ATP deaminase activity ; GO:0047692.
 synonym: "adenine nucleotide deaminase activity" RELATED [EC:3.5.4.17]
 synonym: "adenosine (phosphate) deaminase activity" RELATED [EC:3.5.4.17]
@@ -311485,7 +312190,7 @@
 id: GO:0047629
 name: ADP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ADP + H2O = IDP + NH3." [EC:3.5.4.7, MetaCyc:ADP-DEAMINASE-RXN]
+def: "Catalysis of the reaction: ADP + H2O + H+ = IDP + NH4+." [RHEA:12741]
 synonym: "adenosine diphosphate deaminase activity" RELATED [EC:3.5.4.7]
 synonym: "adenosinepyrophosphate deaminase activity" RELATED [EC:3.5.4.7]
 synonym: "ADP aminohydrolase activity" RELATED [EC:3.5.4.7]
@@ -311539,7 +312244,7 @@
 id: GO:0047632
 name: agmatine deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: agmatine + H2O = N-carbamoylputrescine + NH3." [EC:3.5.3.12, MetaCyc:AGMATINE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: agmatine + H2O = N-carbamoylputrescine + NH4+." [RHEA:18037]
 synonym: "agmatine amidinohydrolase" BROAD [EC:3.5.3.12]
 synonym: "agmatine iminohydrolase activity" RELATED [EC:3.5.3.12]
 xref: EC:3.5.3.12
@@ -311588,23 +312293,25 @@
 
 [Term]
 id: GO:0047635
-name: alanine-oxo-acid transaminase activity
+name: L-alanine:oxo-acid transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + a 2-oxo acid = pyruvate + an L-amino acid." [EC:2.6.1.12, MetaCyc:ALANINE--OXO-ACID-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-alanine + a 2-oxocarboxylate = an L-alpha-amino acid + pyruvate." [RHEA:19953]
 synonym: "alanine--oxo-acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-keto acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-oxo acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-oxo-acid aminotransferase activity" EXACT []
+synonym: "GO Molecular Function help_outline	\nGO:0047635" EXACT []
 synonym: "L-alanine-alpha-keto acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "L-alanine:2-oxo-acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "leucine-alanine transaminase activity" RELATED [EC:2.6.1.12]
 xref: EC:2.6.1.12
 xref: MetaCyc:ALANINE--OXO-ACID-AMINOTRANSFERASE-RXN
 xref: RHEA:19953
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.12
 property_value: skos:exactMatch RHEA:19953
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047636
@@ -311883,7 +312590,7 @@
 id: GO:0047652
 name: allantoate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: allantoate + H2O + H+ = CO2 + NH3 + ureidoglycine." [EC:3.5.3.9, MetaCyc:ALLANTOATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: allantoate + H2O + 2 H+ = (S)-2-ureidoglycine + NH4+ + CO2." [RHEA:27485]
 synonym: "allantoate amidinohydrolase (decarboxylating)" RELATED [EC:3.5.3.9]
 synonym: "allantoate amidohydrolase activity" RELATED [EC:3.5.3.9]
 xref: EC:3.5.3.9
@@ -312081,7 +312788,7 @@
 id: GO:0047664
 name: aminoimidazolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction:4-aminoimidazole + H20 = imidazol-4-one + NH3." [RHEA:22348]
+def: "Catalysis of the reaction: 4-aminoimidazole + H2O + H+ = imidazol-4-one + NH4+." [RHEA:22348]
 synonym: "4-aminoimidazole aminohydrolase activity" RELATED [EC:3.5.4.8]
 synonym: "4-aminoimidazole hydrolase activity" RELATED [EC:3.5.4.8]
 xref: EC:3.5.4.8
@@ -312094,9 +312801,9 @@
 
 [Term]
 id: GO:0047665
-name: aminolevulinate transaminase activity
+name: aminolevulinate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine." [EC:2.6.1.43, RHEA:12480]
+def: "Catalysis of the reaction: 5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine." [RHEA:12480]
 synonym: "4,5-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "4,5-dioxovaleric acid aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "4,5-dioxovaleric acid transaminase activity" RELATED [EC:2.6.1.43]
@@ -312106,7 +312813,7 @@
 synonym: "alanine-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "alanine-gamma,delta-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "alanine:4,5-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
-synonym: "aminolevulinate aminotransferase activity" EXACT []
+synonym: "aminolevulinate aminotransferase activity" BROAD []
 synonym: "aminolevulinic acid transaminase activity" RELATED [EC:2.6.1.43]
 synonym: "dioxovalerate transaminase activity" RELATED [EC:2.6.1.43]
 synonym: "DOVA transaminase activity" RELATED [EC:2.6.1.43]
@@ -312123,6 +312830,7 @@
 property_value: skos:exactMatch EC:2.6.1.43
 property_value: skos:exactMatch RHEA:12480
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047666
@@ -312562,15 +313270,14 @@
 id: GO:0047692
 name: ATP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + H2O = ITP + NH3." [EC:3.5.4.18, MetaCyc:ATP-DEAMINASE-RXN]
+def: "Catalysis of the reaction: ATP + H2O + H+ = ITP + NH4+." [RHEA:13037]
 synonym: "adenosine triphosphate deaminase activity" RELATED [EC:3.5.4.18]
 synonym: "ATP aminohydrolase activity" RELATED [EC:3.5.4.18]
 xref: EC:3.5.4.18
 xref: MetaCyc:ATP-DEAMINASE-RXN
-xref: RHEA:13037
 is_a: GO:0047623 ! adenosine-phosphate deaminase activity
+property_value: skos:broadMatch RHEA:13037
 property_value: skos:exactMatch EC:3.5.4.18
-property_value: skos:exactMatch RHEA:13037
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -312679,7 +313386,7 @@
 id: GO:0047698
 name: beta-alanyl-CoA ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-alanyl-CoA = acryloyl-CoA + NH3." [EC:4.3.1.6, MetaCyc:BETA-ALANYL-COA-AMMONIA-LYASE-RXN]
+def: "Catalysis of the reaction: beta-alanyl-CoA = acryloyl-CoA + NH4+." [RHEA:12416]
 synonym: "b-alanyl-CoA ammonia-lyase activity" EXACT []
 synonym: "beta-alanyl coenzyme A ammonia-lyase activity" RELATED [EC:4.3.1.6]
 synonym: "beta-alanyl-CoA ammonia-lyase (acryloyl-CoA-forming)" RELATED [EC:4.3.1.6]
@@ -312900,7 +313607,7 @@
 id: GO:0047711
 name: blasticidin-S deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: blasticidin S + H2O = deaminohydroxyblasticidin S + NH3." [EC:3.5.4.23, MetaCyc:BLASTICIDIN-S-DEAMINASE-RXN]
+def: "Catalysis of the reaction: blasticidin S + H2O + H+ = deaminohydroxyblasticidin S + NH4+." [RHEA:10148]
 synonym: "blasticidin-S aminohydrolase activity" RELATED [EC:3.5.4.23]
 xref: EC:3.5.4.23
 xref: MetaCyc:BLASTICIDIN-S-DEAMINASE-RXN
@@ -313153,16 +313860,17 @@
 id: GO:0047726
 name: iron-cytochrome-c reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ferrocytochrome c + Fe3+ = ferricytochrome c + Fe2+." [EC:1.9.98.1, MetaCyc:IRON--CYTOCHROME-C-REDUCTASE-RXN]
-synonym: "ferrocytochrome-c:Fe3+ oxidoreductase activity" RELATED [EC:1.9.98.1]
-synonym: "iron-cytochrome c reductase activity" RELATED [EC:1.9.98.1]
-xref: EC:1.9.98.1
+def: "Catalysis of the reaction: Fe3+ + Fe(II)-[cytochrome c] = Fe2+ + Fe(III)-[cytochrome c]." [EC:1.16.2.1, RHEA:15617]
+synonym: "ferrocytochrome-c:Fe3+ oxidoreductase activity" RELATED [EC:1.16.2.1]
+synonym: "iron-cytochrome c reductase activity" RELATED [EC:1.16.2.1]
+xref: EC:1.16.2.1
 xref: MetaCyc:IRON--CYTOCHROME-C-REDUCTASE-RXN
 xref: RHEA:15617
-is_a: GO:0016675 ! oxidoreductase activity, acting on a heme group of donors
-property_value: skos:exactMatch EC:1.9.98.1
+is_a: GO:0016722 ! oxidoreductase activity, acting on metal ions
+property_value: skos:exactMatch EC:1.16.2.1
 property_value: skos:exactMatch RHEA:15617
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31488" xsd:anyURI
 
 [Term]
 id: GO:0047727
@@ -313393,10 +314101,11 @@
 
 [Term]
 id: GO:0047740
-name: cephalosporin-C transaminase activity
+name: cephalosporin-C:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (7R)-7-(5-carboxylato-5-oxopentanamido)deacetylcephalosporanate + D-glutamate = 2-oxoglutarate + cephalosporin C." [EC:2.6.1.74, RHEA:14553]
+def: "Catalysis of the reaction: (7R)-7-(5-carboxylato-5-oxopentanamido)deacetylcephalosporanate + D-glutamate = 2-oxoglutarate + cephalosporin C." [RHEA:14553]
 synonym: "cephalosporin C aminotransferase activity" RELATED [EC:2.6.1.74]
+synonym: "cephalosporin-C transaminase activity" EXACT []
 synonym: "cephalosporin-C:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.74]
 synonym: "L-alanine:cephalosporin-C aminotransferase activity" RELATED [EC:2.6.1.74]
 xref: EC:2.6.1.74
@@ -313407,6 +314116,7 @@
 property_value: skos:exactMatch EC:2.6.1.74
 property_value: skos:exactMatch RHEA:14553
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047741
@@ -313681,7 +314391,7 @@
 synonym: "chondroitin 4-sulphotransferase activity" EXACT []
 xref: EC:2.8.2.5
 xref: MetaCyc:CHONDROITIN-4-SULFOTRANSFERASE-RXN
-xref: Reactome:R-HSA-1971483 "Chondroitin can be sulfated on position 4 of GalNAc by CHST9, 11, 12 and 13"
+xref: Reactome:R-HSA-1971483 "CHST11,12,13,9 transfer sulfate to (4-)GalNAc"
 xref: RHEA:16101
 is_a: GO:0034481 ! chondroitin sulfotransferase activity
 property_value: skos:exactMatch EC:2.8.2.5
@@ -313700,7 +314410,7 @@
 synonym: "urunosyl C-5 epimerase activity" RELATED [EC:5.1.3.19]
 xref: EC:5.1.3.19
 xref: MetaCyc:CHONDROITIN-GLUCURONATE-5-EPIMERASE-RXN
-xref: Reactome:R-HSA-2022052 "Dermatan-sulfate epimerase (DSE) converts chondroitin sulfate (CS) to dermatan sulfate (DS)"
+xref: Reactome:R-HSA-2022052 "DSE converts GlcA to IdoA"
 xref: RHEA:21084
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
 property_value: skos:exactMatch EC:5.1.3.19
@@ -314062,7 +314772,7 @@
 id: GO:0047781
 name: citrullinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + citrulline = NH3 + CO2 + L-ornithine." [EC:3.5.1.20, MetaCyc:CITRULLINASE-RXN]
+def: "Catalysis of the reaction: L-citrulline + H2O + 2 H+ = L-ornithine + NH4+ + CO2." [RHEA:11940]
 synonym: "citrulline hydrolase activity" RELATED [EC:3.5.1.20]
 synonym: "citrulline ureidase activity" RELATED [EC:3.5.1.20]
 synonym: "L-citrulline 5-N-carbamoyldihydrolase activity" RELATED [EC:3.5.1.20]
@@ -314248,7 +314958,7 @@
 id: GO:0047790
 name: creatinine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: creatinine + H2O = N-methylhydantoin + NH3." [EC:3.5.4.21, MetaCyc:CREATININE-DEAMINASE-RXN]
+def: "Catalysis of the reaction: creatinine + H2O + H+ = N-methylhydantoin + NH4+." [RHEA:12681]
 synonym: "creatinine desiminase activity" RELATED [EC:3.5.4.21]
 synonym: "creatinine hydrolase" BROAD [EC:3.5.4.21]
 synonym: "creatinine iminohydrolase activity" RELATED [EC:3.5.4.21]
@@ -314434,28 +315144,31 @@
 
 [Term]
 id: GO:0047801
-name: L-cysteine transaminase activity
+name: L-cysteine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-cysteine + 2-oxoglutarate = mercaptopyruvate + L-glutamate." [RHEA:17441]
+def: "Catalysis of the reaction: L-cysteine + 2-oxoglutarate = 2-oxo-3-sulfanylpropanoate + L-glutamate." [RHEA:17441]
 synonym: "cysteine aminotransferase activity" BROAD []
 synonym: "cysteine transaminase activity" EXACT [EC:2.6.1.3]
+synonym: "L-cysteine transaminase activity" BROAD []
 synonym: "L-cysteine:2-oxoglutarate aminotransferase activity" EXACT []
 xref: EC:2.6.1.3
 xref: MetaCyc:CYSTEINE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-9012597 "GOT2 dimer transfers amino group from L-Cys to 2OG to form 3MPYR and Glu"
 xref: RHEA:17441
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.3
 property_value: skos:exactMatch RHEA:17441
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047802
-name: cysteine-conjugate transaminase activity
+name: cysteine-conjugate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + S-(4-bromophenyl)-L-cysteine = (4-bromophenylsulfanyl)pyruvate + L-glutamate." [EC:2.6.1.75, RHEA:13485]
+def: "Catalysis of the reaction: 2-oxoglutarate + S-(4-bromophenyl)-L-cysteine = (4-bromophenylsulfanyl)pyruvate + L-glutamate." [RHEA:13485]
 synonym: "cysteine conjugate aminotransferase activity" RELATED [EC:2.6.1.75]
 synonym: "cysteine-conjugate alpha-ketoglutarate transaminase (CAT-1)" RELATED [EC:2.6.1.75]
+synonym: "cysteine-conjugate transaminase activity" EXACT []
 synonym: "S-(4-bromophenyl)-L-cysteine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.75]
 xref: EC:2.6.1.75
 xref: KEGG_REACTION:R04338
@@ -314465,6 +315178,7 @@
 property_value: skos:exactMatch EC:2.6.1.75
 property_value: skos:exactMatch RHEA:13485
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047803
@@ -314607,11 +315321,12 @@
 
 [Term]
 id: GO:0047810
-name: D-alanine-2-oxoglutarate aminotransferase activity
+name: D-alanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-alanine + 2-oxoglutarate = pyruvate + D-glutamate." [EC:2.6.1.21]
+def: "Catalysis of the reaction: D-alanine + 2-oxoglutarate = pyruvate + D-glutamate." [RHEA:15869]
 synonym: "D-alanine aminotransferase activity" EXACT []
 synonym: "D-alanine transaminase activity" BROAD [EC:2.6.1.21]
+synonym: "D-alanine-2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "D-alanine-D-glutamate transaminase activity" RELATED [EC:2.6.1.21]
 synonym: "D-amino acid aminotransferase activity" BROAD [EC:2.6.1.21]
 synonym: "D-amino acid transaminase activity" BROAD [EC:2.6.1.21]
@@ -314622,10 +315337,11 @@
 xref: EC:2.6.1.21
 xref: MetaCyc:D-ALANINE-AMINOTRANSFERASE-RXN
 xref: RHEA:15869
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.21
 property_value: skos:exactMatch RHEA:15869
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047811
@@ -314653,10 +315369,11 @@
 xref: EC:2.3.1.36
 xref: MetaCyc:D-AMINO-ACID-N-ACETYLTRANSFERASE-RXN
 xref: RHEA:20704
-is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.36
 property_value: skos:exactMatch RHEA:20704
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047813
@@ -315057,9 +315774,11 @@
 xref: MetaCyc:D-TRYPTOPHAN-N-MALONYLTRANSFERASE-RXN
 xref: RHEA:23320
 is_a: GO:0050735 ! N-malonyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.112
 property_value: skos:exactMatch RHEA:23320
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047837
@@ -315453,25 +316172,20 @@
 name: diiodophenylpyruvate reductase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 3-(3,5-diiodo-4-hydroxyphenyl)lactate + NAD+ = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + H+ + NADH." [RHEA:20293]
-synonym: "2-oxo acid reductase activity" RELATED [EC:1.1.1.96]
-synonym: "3-(3,5-diiodo-4-hydroxyphenyl)lactate:NAD+ oxidoreductase activity" RELATED [EC:1.1.1.96]
-synonym: "aromatic alpha-keto acid" RELATED [EC:1.1.1.96]
-synonym: "KAR" RELATED [EC:1.1.1.96]
-xref: EC:1.1.1.96
 xref: KEGG_REACTION:R03431
 xref: MetaCyc:DIIODOPHENYLPYRUVATE-REDUCTASE-RXN
 xref: RHEA:20293
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
-property_value: skos:exactMatch EC:1.1.1.96
 property_value: skos:exactMatch RHEA:20293
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30138" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31488" xsd:anyURI
 
 [Term]
 id: GO:0047861
-name: diiodotyrosine transaminase activity
+name: 3,5-diiodo-L-tyrosine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3,5-diiodo-L-tyrosine = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + L-glutamate." [EC:2.6.1.24, RHEA:19781]
+def: "Catalysis of the reaction: 2-oxoglutarate + 3,5-diiodo-L-tyrosine = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + L-glutamate." [RHEA:19781]
 synonym: "3,5-diiodo-L-tyrosine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.24]
 synonym: "diiodotyrosine aminotransferase activity" EXACT []
 synonym: "halogenated tyrosine aminotransferase activity" RELATED [EC:2.6.1.24]
@@ -315484,6 +316198,7 @@
 property_value: skos:exactMatch EC:2.6.1.24
 property_value: skos:exactMatch RHEA:19781
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047862
@@ -315871,7 +316586,7 @@
 id: GO:0047883
 name: ethanolamine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ethanolamine + H2O + O2 = glycolaldehyde + NH3 + H2O2." [RHEA:18581]
+def: "Catalysis of the reaction: ethanolamine + O2 + H2O = glycolaldehyde + H2O2 + NH4+." [RHEA:18581]
 synonym: "ethanolamine:oxygen oxidoreductase (deaminating)" RELATED [EC:1.4.3.8]
 xref: EC:1.4.3.8
 xref: MetaCyc:ETHANOLAMINE-OXIDASE-RXN
@@ -316672,7 +317387,7 @@
 name: glucosaminate ammonia-lyase activity
 namespace: molecular_function
 alt_id: GO:0050443
-def: "Catalysis of the reaction: D-glucosaminate = 2-dehydro-3-deoxy-D-gluconate + NH3." [EC:4.3.1.9]
+def: "Catalysis of the reaction: 2-amino-2-deoxy-D-gluconate = 2-dehydro-3-deoxy-D-gluconate + NH4+." [RHEA:12488]
 comment: Note that this function was EC:4.3.1.21.
 synonym: "2-amino-2-deoxy-D-gluconate ammonia-lyase activity" RELATED [EC:4.3.1.9]
 synonym: "2-amino-2-deoxy-D-gluconate hydro-lyase (deaminating) activity" RELATED [EC:4.3.1.9]
@@ -316933,9 +317648,9 @@
 
 [Term]
 id: GO:0047945
-name: L-glutamine:pyruvate aminotransferase activity
+name: L-glutamine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine." [EC:2.6.1.15, RHEA:10400]
+def: "Catalysis of the reaction: L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine." [RHEA:10400]
 synonym: "gamma-glutaminyltransferase activity" RELATED [EC:2.6.1.15]
 synonym: "glutaminase II activity" RELATED [EC:2.6.1.15]
 synonym: "glutamine transaminase activity" BROAD [EC:2.6.1.15]
@@ -316948,26 +317663,29 @@
 synonym: "glutamine-oxo acid aminotransferase activity" BROAD [EC:2.6.1.15]
 synonym: "glutamine-pyruvate transaminase activity" BROAD [EC:2.6.1.15]
 synonym: "L-glutamine transaminase L" RELATED [EC:2.6.1.15]
+synonym: "L-glutamine:pyruvate aminotransferase activity" EXACT []
 xref: EC:2.6.1.15
 xref: KEGG_REACTION:R00576
 xref: MetaCyc:GLUTAMINE--PYRUVATE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-893616 "glutamine + pyruvate => 2-oxoglutaramate + alanine"
 xref: RHEA:10400
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.15
 property_value: skos:exactMatch RHEA:10400
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047946
-name: glutamine N-acyltransferase activity
+name: L-glutamine N-acyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine." [EC:2.3.1.68, MetaCyc:GLUTAMINE-N-ACYLTRANSFERASE-RXN]
+def: "Catalysis of the reaction: acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine." [RHEA:18469]
 synonym: "acyl-CoA:L-glutamine N-acyltransferase activity" RELATED [EC:2.3.1.68]
+synonym: "glutamine N-acyltransferase activity" BROAD []
 xref: EC:2.3.1.68
 xref: MetaCyc:GLUTAMINE-N-ACYLTRANSFERASE-RXN
 xref: RHEA:18469
-is_a: GO:0016410 ! N-acyltransferase activity
+is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.68
 property_value: skos:exactMatch RHEA:18469
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -317020,16 +317738,17 @@
 id: GO:0047950
 name: glutathione oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 glutathione + O2 = glutathione disulfide + H2O2." [EC:1.8.3.3, RHEA:24112]
+def: "Catalysis of the reaction: 2 glutathione + O2 = glutathione disulfide + H2O2." [RHEA:24112]
 synonym: "glutathione:oxygen oxidoreductase activity" RELATED [EC:1.8.3.3]
 xref: EC:1.8.3.3
 xref: KEGG_REACTION:R00120
 xref: MetaCyc:GLUTATHIONE-OXIDASE-RXN
 xref: RHEA:24112
-is_a: GO:0016670 ! oxidoreductase activity, acting on a sulfur group of donors, oxygen as acceptor
+is_a: GO:0016972 ! thiol oxidase activity
 property_value: skos:exactMatch EC:1.8.3.3
 property_value: skos:exactMatch RHEA:24112
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0047951
@@ -317160,29 +317879,31 @@
 
 [Term]
 id: GO:0047958
-name: glycine:2-oxoglutarate aminotransferase activity
+name: glycine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + 2-oxoglutarate = glyoxylate + L-glutamate." [EC:2.6.1.4, MetaCyc:GLYCINE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: glycine + 2-oxoglutarate = glyoxylate + L-glutamate." [RHEA:14089]
 synonym: "glutamate-glyoxylate transaminase activity" EXACT []
 synonym: "glutamic-glyoxylic transaminase activity" RELATED [EC:2.6.1.4]
 synonym: "glycine aminotransferase activity" BROAD []
 synonym: "glycine transaminase activity" BROAD [EC:2.6.1.4]
+synonym: "glycine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "glyoxylate-glutamate aminotransferase activity" RELATED [EC:2.6.1.4]
 synonym: "glyoxylate-glutamic transaminase activity" RELATED [EC:2.6.1.4]
 synonym: "L-glutamate:glyoxylate aminotransferase activity" EXACT []
 xref: EC:2.6.1.4
 xref: MetaCyc:GLYCINE-AMINOTRANSFERASE-RXN
 xref: RHEA:14089
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.4
 property_value: skos:exactMatch RHEA:14089
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047959
 name: glycine dehydrogenase (cytochrome) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + H2O + 2 ferricytochrome c = glyoxylate + NH3 + 2 ferrocytochrome c." [EC:1.4.2.1, MetaCyc:GLYCINE-DEHYDROGENASE-CYTOCHROME-RXN]
+def: "Catalysis of the reaction: 2 Fe(III)-[cytochrome c] + glycine + H2O = 2 Fe(II)-[cytochrome c] + glyoxylate + NH4+ + 2 H+." [RHEA:16909]
 synonym: "glycine-cytochrome c reductase activity" RELATED [EC:1.4.2.1]
 synonym: "glycine:ferricytochrome-c oxidoreductase (deaminating)" RELATED [EC:1.4.2.1]
 synonym: "reductase, glycine-cytochrome c" RELATED [EC:1.4.2.1]
@@ -317214,7 +317935,7 @@
 id: GO:0047961
 name: glycine N-acyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acyl-CoA + glycine = CoA + N-acylglycine." [EC:2.3.1.13, MetaCyc:GLYCINE-N-ACYLTRANSFERASE-RXN]
+def: "Catalysis of the reaction: acyl-CoA + glycine = CoA + N-acylglycine." [RHEA:19869]
 synonym: "acyl-CoA:glycine N-acyltransferase activity" RELATED [EC:2.3.1.13]
 synonym: "glycine acyltransferase activity" RELATED [EC:2.3.1.13]
 synonym: "glycine-N-acylase activity" RELATED [EC:2.3.1.13]
@@ -317222,10 +317943,11 @@
 xref: MetaCyc:GLYCINE-N-ACYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-2534040 "Unknown NAT N-acylates Gly in GNAT1"
 xref: RHEA:19869
-is_a: GO:0016410 ! N-acyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.13
 property_value: skos:exactMatch RHEA:19869
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047962
@@ -317443,7 +318165,7 @@
 id: GO:0047974
 name: guanosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanosine + H2O = xanthosine + NH3." [EC:3.5.4.15, MetaCyc:GUANOSINE-DEAMINASE-RXN]
+def: "Catalysis of the reaction: guanosine + H2O + H+ = xanthosine + NH4+." [RHEA:12861]
 synonym: "guanosine aminase activity" RELATED [EC:3.5.4.15]
 synonym: "guanosine aminohydrolase activity" RELATED [EC:3.5.4.15]
 xref: EC:3.5.4.15
@@ -317564,7 +318286,7 @@
 id: GO:0047981
 name: L-histidine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidine + acetyl-CoA = N(alpha)-acetyl-L-histidine + CoA + H+." [EC:2.3.1.33, RHEA:24596]
+def: "Catalysis of the reaction: L-histidine + acetyl-CoA = N(alpha)-acetyl-L-histidine + CoA + H+." [RHEA:24596]
 synonym: "acetyl-CoA:L-histidine N-acetyltransferase activity" RELATED [EC:2.3.1.33]
 synonym: "acetylhistidine synthetase activity" RELATED [EC:2.3.1.33]
 synonym: "histidine acetyltransferase activity" RELATED [EC:2.3.1.33]
@@ -317582,7 +318304,7 @@
 id: GO:0047982
 name: homocysteine desulfhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-homocysteine + H2O = sulfide + NH3 + 2-oxobutanoate." [EC:4.4.1.2, MetaCyc:HOMOCYSTEINE-DESULFHYDRASE-RXN]
+def: "Catalysis of the reaction: L-homocysteine + H2O = 2-oxobutanoate + hydrogen sulfide + NH4+ + H+." [RHEA:14501]
 synonym: "homocysteine desulfurase activity" RELATED [EC:4.4.1.2]
 synonym: "L-homocysteine hydrogen-sulfide-lyase (deaminating)" RELATED [EC:4.4.1.2]
 synonym: "L-homocysteine hydrogen-sulfide-lyase (deaminating; 2-oxobutanoate-forming)" RELATED [EC:4.4.1.2]
@@ -318058,8 +318780,8 @@
 synonym: "receptor agonist activity" RELATED [GOC:molecular_function_refactoring]
 synonym: "signaling molecule" BROAD []
 synonym: "signaling receptor ligand activity" EXACT []
-xref: Reactome:R-HSA-163625 "Glucagon binds to Glucagon receptor"
 xref: Reactome:R-HSA-381612 "GLP1R binds GLP1"
+xref: Reactome:R-HSA-8848663 "ADIPOQ trimer binds ADIPOR dimers"
 is_a: GO:0030546 ! signaling receptor activator activity
 relationship: has_part GO:0005102 ! signaling receptor binding
 relationship: part_of GO:0007165 ! signal transduction
@@ -320059,11 +320781,12 @@
 
 [Term]
 id: GO:0048258
-name: 3-ketoglucose-reductase activity
+name: obsolete 3-ketoglucose-reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NADP+ + 3-dehydro-alpha-D-glucose = NADPH + alpha-D-glucose." [MetaCyc:KETOGLUCOSE-REDUCTASE-RXN]
-xref: MetaCyc:KETOGLUCOSE-REDUCTASE-RXN
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: NADP+ + 3-dehydro-alpha-D-glucose = NADPH + alpha-D-glucose." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0048259
@@ -320154,7 +320877,8 @@
 def: "Any process that results in a change in the behavior of an organism as a result of a pain stimulus. Pain stimuli cause activation of nociceptors, peripheral receptors for pain, include receptors which are sensitive to painful mechanical stimuli, extreme heat or cold, and chemical stimuli." [GOC:jid]
 synonym: "behavioural response to pain" EXACT []
 is_a: GO:0007610 ! behavior
-relationship: part_of GO:0048265 ! response to pain
+is_a: GO:0048265 ! response to pain
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 
 [Term]
 id: GO:0048268
@@ -326576,7 +327300,7 @@
 id: GO:0050001
 name: D-glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + D-glutamate." [EC:3.5.1.35, MetaCyc:D-GLUTAMINASE-RXN]
+def: "Catalysis of the reaction: D-glutamine + H2O = D-glutamate + NH4+." [RHEA:22840]
 synonym: "D-glutamine amidohydrolase activity" RELATED [EC:3.5.1.35]
 xref: EC:3.5.1.35
 xref: MetaCyc:D-GLUTAMINASE-RXN
@@ -326979,7 +327703,7 @@
 id: GO:0050025
 name: L-glutamate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutamate + O2 + H2O = 2-oxoglutarate + NH3 + H2O2." [RHEA:20728]
+def: "Catalysis of the reaction: L-glutamate + O2 + H2O = 2-oxoglutarate + H2O2 + NH4+." [RHEA:20728]
 synonym: "glutamate (acceptor) dehydrogenase activity" RELATED [EC:1.4.3.11]
 synonym: "glutamate oxidase activity" RELATED [EC:1.4.3.11]
 synonym: "glutamic acid oxidase activity" RELATED [EC:1.4.3.11]
@@ -327049,7 +327773,6 @@
 name: L-lysine oxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: H2O + L-lysine + O2 = 6-amino-2-oxohexanoate + H2O2 + NH4+." [RHEA:14437]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-lysine alpha-oxidase activity" RELATED [EC:1.4.3.14]
 synonym: "L-lysine:oxygen 2-oxidoreductase (deaminating)" RELATED [EC:1.4.3.14]
 synonym: "L-lysyl-alpha-oxidase activity" RELATED [EC:1.4.3.14]
@@ -327062,8 +327785,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00447
 property_value: skos:exactMatch MetaCyc:L-LYSINE-OXIDASE-RXN
 property_value: skos:exactMatch RHEA:14437
-property_value: skos:relatedMatch MetaCyc:PWY-5311
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050030
@@ -327409,7 +328132,7 @@
 id: GO:0050050
 name: L-leucine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-leucine + acetyl-CoA = N-acetyl-L-leucine + CoA + H+." [EC:2.3.1.66, RHEA:20089]
+def: "Catalysis of the reaction: L-leucine + acetyl-CoA = N-acetyl-L-leucine + CoA + H+." [RHEA:20089]
 synonym: "acetyl-CoA:L-leucine N-acetyltransferase activity" RELATED [EC:2.3.1.66]
 synonym: "leucine acetyltransferase activity" RELATED [EC:2.3.1.66]
 xref: EC:2.3.1.66
@@ -327664,25 +328387,26 @@
 
 [Term]
 id: GO:0050065
-name: lysine-pyruvate 6-transaminase activity
+name: L-lysine:pyruvate 6-transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-lysine + pyruvate = L-alanine + L-allysine." [EC:2.6.1.71, RHEA:19393]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-lysine + pyruvate = (S)-2-amino-6-oxohexanoate + L-alanine." [RHEA:19393]
 synonym: "L-lysine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.71]
 synonym: "Lys-AT" RELATED [EC:2.6.1.71]
 synonym: "lysine--pyruvate 6-aminotransferase activity" EXACT []
+synonym: "lysine-pyruvate 6-transaminase activity" EXACT []
 synonym: "lysine-pyruvate aminotransferase activity" RELATED [EC:2.6.1.71]
 xref: EC:2.6.1.71
 xref: KEGG_REACTION:R00453
 xref: MetaCyc:LYSINE--PYRUVATE-6-AMINOTRANSFERASE-RXN
 xref: RHEA:19393
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.71
 property_value: skos:exactMatch KEGG_REACTION:R00453
 property_value: skos:exactMatch MetaCyc:LYSINE--PYRUVATE-6-AMINOTRANSFERASE-RXN
 property_value: skos:exactMatch RHEA:19393
-property_value: skos:relatedMatch MetaCyc:PWY-5324
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050066
@@ -328143,20 +328867,22 @@
 
 [Term]
 id: GO:0050094
-name: methionine-glyoxylate transaminase activity
+name: L-methionine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine + glyoxylate = 4-methylthio-2-oxobutanoate + glycine." [EC:2.6.1.73, RHEA:22884]
+def: "Catalysis of the reaction: glyoxylate + L-methionine = 4-methylsulfanyl-2-oxobutanoate + glycine." [RHEA:22884]
 synonym: "L-methionine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.73]
 synonym: "methionine-glyoxylate aminotransferase activity" RELATED [EC:2.6.1.73]
+synonym: "methionine-glyoxylate transaminase activity" EXACT []
 synonym: "MGAT activity" RELATED [EC:2.6.1.73]
 xref: EC:2.6.1.73
 xref: KEGG_REACTION:R00652
 xref: MetaCyc:METHIONINE--GLYOXYLATE-TRANSAMINASE-RXN
 xref: RHEA:22884
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0010326 ! L-methionine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.73
 property_value: skos:exactMatch RHEA:22884
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050095
@@ -328825,9 +329551,11 @@
 xref: MetaCyc:N6-HYDROXYLYSINE-O-ACETYLTRANSFERASE-RXN
 xref: RHEA:22388
 is_a: GO:0016413 ! O-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.102
 property_value: skos:exactMatch RHEA:22388
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0050134
@@ -328970,7 +329698,7 @@
 id: GO:0050142
 name: nitrogenase (flavodoxin) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6 reduced flavodoxin + 6 H+ + N2 + n ATP = 6 oxidized flavodoxin + 2 NH3 + n ADP + n phosphate." [EC:1.19.6.1, MetaCyc:NITROGENASE-FLAVODOXIN-RXN]
+def: "Catalysis of the reaction: N2 + 4 reduced [flavodoxin] + 16 ATP + 16 H2O = 4 oxidized [flavodoxin] + H2 + 2 NH4+ + 16 ADP + 16 phosphate + 18 H+." [RHEA:15645]
 synonym: "reduced flavodoxin:dinitrogen oxidoreductase (ATP-hydrolysing)" RELATED [EC:1.19.6.1]
 xref: EC:1.19.6.1
 xref: MetaCyc:NITROGENASE-FLAVODOXIN-RXN
@@ -329439,12 +330167,13 @@
 
 [Term]
 id: GO:0050170
-name: peptidyl-glutaminase activity
+name: protein-glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: alpha-N-peptidyl-L-glutamine + H2O = alpha-N-peptidyl-L-glutamate + NH3." [EC:3.5.1.43, MetaCyc:PEPTIDYL-GLUTAMINASE-RXN]
+def: "Catalysis of the reaction: [protein]-C-terminal-L-glutamine + H2O = [protein]-C-terminal-L-glutamate + NH4+." [RHEA:10032]
 synonym: "peptideglutaminase activity" RELATED [EC:3.5.1.43]
 synonym: "peptidoglutaminase activity" RELATED [EC:3.5.1.43]
 synonym: "peptidoglutaminase I activity" NARROW [EC:3.5.1.43]
+synonym: "peptidyl-glutaminase activity" EXACT []
 synonym: "peptidyl-L-glutamine amidohydrolase activity" RELATED [EC:3.5.1.43]
 xref: EC:3.5.1.43
 xref: MetaCyc:PEPTIDYL-GLUTAMINASE-RXN
@@ -329456,9 +330185,10 @@
 
 [Term]
 id: GO:0050171
-name: phenol beta-glucosyltransferase activity
+name: obsolete phenol beta-glucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside." [EC:2.4.1.35, MetaCyc:PHENOL-BETA-GLUCOSYLTRANSFERASE-RXN]
+def: "OBSOLETE. Catalysis of the reaction: UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside." [EC:2.4.1.35]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 synonym: "phenol b-glucosyltransferase activity" EXACT []
 synonym: "phenol-beta-D-glucosyltransferase activity" RELATED [EC:2.4.1.35]
 synonym: "UDP glucosyltransferase activity" RELATED [EC:2.4.1.35]
@@ -329469,9 +330199,9 @@
 synonym: "uridine diphosphoglucosyltransferase activity" RELATED [EC:2.4.1.35]
 xref: EC:2.4.1.35
 xref: MetaCyc:PHENOL-BETA-GLUCOSYLTRANSFERASE-RXN
-is_a: GO:0035251 ! UDP-glucosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.35
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0050172
@@ -329544,7 +330274,7 @@
 id: GO:0050176
 name: L-phenylalanine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-phenylalanine + acetyl-CoA = N-acetyl-L-phenylalanine + CoA + H+." [EC:2.3.1.53, RHEA:17801]
+def: "Catalysis of the reaction: L-phenylalanine + acetyl-CoA = N-acetyl-L-phenylalanine + CoA + H+." [RHEA:17801]
 synonym: "acetyl-CoA-L-phenylalanine alpha-N-acetyltransferase activity" RELATED [EC:2.3.1.53]
 synonym: "acetyl-CoA:L-phenylalanine N-acetyltransferase activity" RELATED [EC:2.3.1.53]
 synonym: "phenylalanine N-acetyltransferase activity" EXACT []
@@ -330009,7 +330739,7 @@
 xref: EC:2.7.1.52
 xref: KEGG_REACTION:R03161
 xref: MetaCyc:FUCOKINASE-RXN
-xref: Reactome:R-HSA-6787540 "FUK phosphorylates beta-Fuc to Fuc1P"
+xref: Reactome:R-HSA-6787540 "FCSK phosphorylates beta-Fuc to Fuc1P"
 xref: RHEA:13241
 is_a: GO:0016301 ! kinase activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
@@ -330197,22 +330927,20 @@
 id: GO:0050212
 name: progesterone 11-alpha-monooxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: AH2 + O2 + progesterone = 11alpha-hydroxyprogesterone + A + H2O." [RHEA:18205]
-synonym: "progesterone 11-alpha-hydroxylase activity" RELATED [EC:1.14.99.14]
+def: "Catalysis of the reaction: AH2 + O2 + progesterone = 11alpha-hydroxyprogesterone + A + H2O." [PMID:39766209, RHEA:86055]
+synonym: "progesterone 11-alpha-hydroxylase activity" RELATED [EC:1.14.14.197]
 synonym: "progesterone 11a-monooxygenase activity" EXACT []
-synonym: "progesterone 11alpha-hydroxylase activity" RELATED [EC:1.14.99.14]
-synonym: "progesterone 11alpha-monooxygenase activity" RELATED [EC:1.14.99.14]
-synonym: "progesterone,hydrogen-donor:oxygen oxidoreductase (11alpha-hydroxylating)" RELATED [EC:1.14.99.14]
-xref: EC:1.14.99.14
+synonym: "progesterone 11alpha-hydroxylase activity" RELATED [EC:1.14.14.197]
+synonym: "progesterone 11alpha-monooxygenase activity" RELATED [EC:1.14.14.197]
+synonym: "progesterone,hydrogen-donor:oxygen oxidoreductase (11alpha-hydroxylating)" RELATED [EC:1.14.14.197]
 xref: KEGG_REACTION:R02214
 xref: MetaCyc:PROGESTERONE-11-ALPHA-MONOOXYGENASE-RXN
-xref: PMID:39766209
-xref: RHEA:18205
+xref: RHEA:86055
 is_a: GO:0050214 ! progesterone monooxygenase activity
-property_value: skos:exactMatch EC:1.14.99.14
-property_value: skos:exactMatch RHEA:18205
+property_value: skos:exactMatch RHEA:86055
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29311" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31423" xsd:anyURI
 
 [Term]
 id: GO:0050213
@@ -330661,7 +331389,7 @@
 id: GO:0050239
 name: pyrithiamine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + H2O = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + NH3." [EC:3.5.4.20, MetaCyc:PYRITHIAMIN-DEAMINASE-RXN]
+def: "Catalysis of the reaction: 1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(2-hydroxyethyl)-2-methylpyridinium + H2O + H+ = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(2-hydroxyethyl)-2-methylpyridinium + NH4+." [RHEA:14537]
 synonym: "1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium-bromide aminohydrolase activity" RELATED [EC:3.5.4.20]
 xref: EC:3.5.4.20
 xref: MetaCyc:PYRITHIAMIN-DEAMINASE-RXN
@@ -331347,7 +332075,7 @@
 id: GO:0050279
 name: sepiapterin deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: sepiapterin + H2O = xanthopterin-B2 + NH3." [EC:3.5.4.24, MetaCyc:SEPIAPTERIN-DEAMINASE-RXN]
+def: "Catalysis of the reaction: L-sepiapterin + H2O + H+ = (S)-xanthopterin-B2 + NH4+." [RHEA:14025]
 synonym: "sepiapterin aminohydrolase activity" RELATED [EC:3.5.4.24]
 xref: EC:3.5.4.24
 xref: MetaCyc:SEPIAPTERIN-DEAMINASE-RXN
@@ -331375,19 +332103,21 @@
 
 [Term]
 id: GO:0050281
-name: L-serine-glyoxylate transaminase activity
+name: L-serine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine + glyoxylate = 3-hydroxypyruvate + glycine." [EC:2.6.1.45]
+def: "Catalysis of the reaction: L-serine + glyoxylate = 3-hydroxypyruvate + glycine." [RHEA:19125]
+synonym: "L-serine-glyoxylate transaminase activity" EXACT []
 synonym: "L-serine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.45]
 synonym: "serine--glyoxylate aminotransferase activity" EXACT []
 synonym: "SGAT activity" RELATED [EC:2.6.1.45]
 xref: EC:2.6.1.45
 xref: MetaCyc:SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN
 xref: RHEA:19125
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.45
 property_value: skos:exactMatch RHEA:19125
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050282
@@ -331410,7 +332140,7 @@
 id: GO:0050283
 name: serine-sulfate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine O-sulfate + H2O = pyruvate + NH3 + sulfate." [EC:4.3.1.10, MetaCyc:SERINE-SULFATE-AMMONIA-LYASE-RXN]
+def: "Catalysis of the reaction: -serine O-sulfate + H2O = sulfate + pyruvate + NH4+ + H+." [RHEA:15605]
 synonym: "(L-SOS)lyase activity" RELATED [EC:4.3.1.10]
 synonym: "L-serine-O-sulfate ammonia-lyase (pyruvate-forming)" RELATED [EC:4.3.1.10]
 synonym: "serine-sulphate ammonia-lyase activity" EXACT []
@@ -331758,7 +332488,6 @@
 name: L-lysine 6-dehydrogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-lysine + NAD+ = L-1-piperideine-6-carboxylate + NH4+ + NADH + 2 H+." [RHEA:12408]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-lysine epsilon-dehydrogenase activity" RELATED [EC:1.4.1.18]
 synonym: "L-lysine:NAD+ 6-oxidoreductase (deaminating)" RELATED [EC:1.4.1.18]
 synonym: "LysDH activity" RELATED [EC:1.4.1.18]
@@ -331769,11 +332498,11 @@
 property_value: skos:exactMatch EC:1.4.1.18
 property_value: skos:exactMatch MetaCyc:LYSINE-6-DEHYDROGENASE-RXN
 property_value: skos:exactMatch RHEA:12408
-property_value: skos:relatedMatch MetaCyc:PWY-5314
 property_value: skos:relatedMatch RHEA:64792
 property_value: skos:relatedMatch RHEA:64800
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050304
@@ -332084,13 +332813,14 @@
 
 [Term]
 id: GO:0050322
-name: taurine-2-oxoglutarate transaminase activity
+name: taurine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: taurine + 2-oxoglutarate = sulfoacetaldehyde + L-glutamate." [PMID:16535664, RHEA:16353]
 synonym: "taurine aminotransferase activity" EXACT []
 synonym: "taurine transaminase activity" EXACT []
 synonym: "taurine--alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.55]
 synonym: "taurine--glutamate transaminase activity" RELATED [EC:2.6.1.55]
+synonym: "taurine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "taurine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.55]
 xref: EC:2.6.1.55
 xref: MetaCyc:RXN-2301
@@ -332099,6 +332829,7 @@
 property_value: skos:exactMatch EC:2.6.1.55
 property_value: skos:exactMatch RHEA:16353
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050323
@@ -332723,7 +333454,7 @@
 id: GO:0050360
 name: tryptophan 2'-dioxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + O2 = 3-indoleglycolaldehyde + CO2 + NH3." [EC:1.13.99.3, MetaCyc:TRYPTOPHAN-2-DIOXYGENASE-RXN]
+def: "Catalysis of the reaction: L-tryptophan + O2 + H+ = 3-indoleglycolaldehyde + NH4+ + CO2." [RHEA:22620]
 synonym: "indole-3-alkane alpha-hydroxylase activity" RELATED [EC:1.13.99.3]
 synonym: "indolyl-3-alkan alpha-hydroxylase activity" RELATED [EC:1.13.99.3]
 synonym: "L-tryptophan:oxygen 2'-oxidoreductase (side-chain-cleaving)" RELATED [EC:1.13.99.3]
@@ -332760,23 +333491,25 @@
 
 [Term]
 id: GO:0050362
-name: L-tryptophan:2-oxoglutarate aminotransferase activity
+name: L-tryptophan:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + 2-oxoglutarate = indolepyruvate + L-glutamate." [EC:2.6.1.27, MetaCyc:TRYPTOPHAN-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-tryptophan + 2-oxoglutarate = indole-3-pyruvate + L-glutamate." [RHEA:14093]
 synonym: "5-hydroxytryptophan-ketoglutaric transaminase activity" RELATED [EC:2.6.1.27]
 synonym: "hydroxytryptophan aminotransferase activity" RELATED [EC:2.6.1.27]
 synonym: "L-phenylalanine-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.27]
 synonym: "L-tryptophan aminotransferase activity" BROAD [EC:2.6.1.27]
 synonym: "L-tryptophan transaminase activity" BROAD [EC:2.6.1.27]
+synonym: "L-tryptophan:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "tryptophan aminotransferase activity" EXACT []
 synonym: "tryptophan transaminase activity" BROAD [EC:2.6.1.27]
 xref: EC:2.6.1.27
 xref: MetaCyc:TRYPTOPHAN-AMINOTRANSFERASE-RXN
 xref: RHEA:14093
-is_a: GO:0070529 ! L-tryptophan aminotransferase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch EC:2.6.1.27
 property_value: skos:exactMatch RHEA:14093
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050363
@@ -333036,8 +333769,7 @@
 id: GO:0050378
 name: UDP-glucuronate 4-epimerase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-alpha-D-galacturonate." [EC:5.1.3.6, RHEA:11404]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-alpha-D-galacturonate." [RHEA:11404]
 synonym: "UDP glucuronic epimerase activity" RELATED [EC:5.1.3.6]
 synonym: "UDP-D-galacturonic acid 4-epimerase activity" RELATED [EC:5.1.3.6]
 synonym: "UDP-galacturonate 4-epimerase activity" RELATED [EC:5.1.3.6]
@@ -333054,8 +333786,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R01385
 property_value: skos:exactMatch MetaCyc:UDP-GLUCURONATE-4-EPIMERASE-RXN
 property_value: skos:exactMatch RHEA:11404
-property_value: skos:relatedMatch MetaCyc:PWY-4861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050379
@@ -333646,7 +334378,7 @@
 id: GO:0050414
 name: formimidoylaspartate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-formimidoyl-L-aspartate + H2O = N-formyl-L-aspartate + NH3." [EC:3.5.3.5, MetaCyc:FORMIMINOASPARTATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: N-formimidoyl-L-aspartate + H2O = N-formyl-L-aspartate + NH4+." [RHEA:13661]
 synonym: "formiminoaspartate deiminase activity" EXACT []
 synonym: "N-formimidoyl-L-aspartate iminohydrolase activity" RELATED [EC:3.5.3.5]
 xref: EC:3.5.3.5
@@ -333680,7 +334412,7 @@
 id: GO:0050416
 name: formimidoylglutamate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H2O = N-formyl-L-glutamate + NH3." [EC:3.5.3.13, MetaCyc:FORMIMINOGLUTAMATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H2O = N-formyl-L-glutamate + NH4+." [RHEA:22832]
 synonym: "formiminoglutamate deiminase activity" EXACT []
 synonym: "formiminoglutamic iminohydrolase activity" RELATED [EC:3.5.3.13]
 synonym: "N-formimidoyl-L-glutamate iminohydrolase activity" RELATED [EC:3.5.3.13]
@@ -333696,7 +334428,7 @@
 id: GO:0050417
 name: glutamin-(asparagin-)ase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + L-glutamate; and H2O + L-asparagine = NH3 + L-aspartate." [EC:3.5.1.38]
+def: "Catalysis of the reactions: H2O + L-glutamine = NH4 + L-glutamate; and H2O + L-asparagine = NH4 + L-aspartate." [EC:3.5.1.38]
 synonym: "glutaminase-(asparagin-)ase activity" EXACT []
 synonym: "glutaminase-asparaginase activity" RELATED [EC:3.5.1.38]
 synonym: "L-ASNase/L-GLNase activity" RELATED [EC:3.5.1.38]
@@ -333714,7 +334446,7 @@
 id: GO:0050418
 name: hydroxylamine reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NH3 + H2O + acceptor = hydroxylamine + reduced acceptor." [EC:1.7.99.1]
+def: "Catalysis of the reaction: an acceptor + NH4+ + H2O = hydroxylamine + a reduced acceptor + H+." [RHEA:22052]
 synonym: "ammonia:(acceptor) oxidoreductase activity" RELATED [EC:1.7.99.1]
 synonym: "ammonia:acceptor oxidoreductase activity" RELATED [EC:1.7.99.1]
 synonym: "hydroxylamine (acceptor) reductase activity" RELATED [EC:1.7.99.1]
@@ -333869,6 +334601,7 @@
 synonym: "phosphoadenosine phosphosulfate metabolic process" EXACT []
 synonym: "phosphoadenosine phosphosulfate metabolism" EXACT []
 xref: Reactome:R-HSA-156584 "Cytosolic sulfonation of small molecules"
+xref: Reactome:R-HSA-174362 "Transport and metabolism of PAPS"
 is_a: GO:0006790 ! sulfur compound metabolic process
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0034035 ! purine ribonucleoside bisphosphate metabolic process
@@ -335002,7 +335735,7 @@
 synonym: "spHAS" RELATED [EC:2.4.1.212]
 xref: EC:2.4.1.212
 xref: MetaCyc:2.4.1.212-RXN
-xref: Reactome:R-HSA-2160851 "HAS1,2,3 mediate the polymerization of HA"
+xref: Reactome:R-HSA-2160851 "HAS1,2,3 mediate the polymerization and export of HA"
 xref: RHEA:12528
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
 is_a: GO:0016758 ! hexosyltransferase activity
@@ -335120,12 +335853,9 @@
 synonym: "UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1->4)-N-acetyl-alpha-D-glucosaminyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.224]
 xref: EC:2.4.1.224
 xref: MetaCyc:2.4.1.224-RXN
-xref: Reactome:R-HSA-2022851 "EXT1:EXT2 transfer GlcNAc to the heparan chain"
-xref: Reactome:R-HSA-2022919 "EXT1:EXT2 transfers GlcNAc to the terminal GlcA residue"
+xref: Reactome:R-HSA-2022851 "EXT1:EXT2 transfers GlcNAc to the heparan chain"
 xref: Reactome:R-HSA-3656254 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcNAc to the heparan chain"
 xref: Reactome:R-HSA-3656261 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcNAc to the heparan chain"
-xref: Reactome:R-HSA-9036283 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcNAc to the terminal GlcA residue"
-xref: Reactome:R-HSA-9036290 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcNAc to heparan"
 xref: RHEA:16213
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.224
@@ -335144,7 +335874,7 @@
 synonym: "UDP-alpha-D-glucuronate:N-acetyl-alpha-D-glucosaminyl-(1->4)-beta-D-glucuronosyl-proteoglycan 4-beta-glucuronosyltransferase activity" RELATED [EC:2.4.1.225]
 xref: EC:2.4.1.225
 xref: MetaCyc:2.4.1.225-RXN
-xref: Reactome:R-HSA-2022856 "EXT1:EXT2 transfers GlcNAc to heparan"
+xref: Reactome:R-HSA-2022856 "EXT1:EXT2 transfers GlcA to heparan"
 xref: Reactome:R-HSA-2076392 "EXT1:EXT2 transfers GlcA to heparan"
 xref: Reactome:R-HSA-3656257 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcA to heparan"
 xref: Reactome:R-HSA-3656267 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcA to heparan"
@@ -335167,9 +335897,8 @@
 synonym: "N-acetylgalactosaminyl-proteoglycan 3-b-glucuronosyltransferase activity" EXACT []
 xref: EC:2.4.1.226
 xref: MetaCyc:RXN-16324
-xref: Reactome:R-HSA-1971491 "CHPF,CHPF2,CHSY3 transfer GlcA to chondroitin"
+xref: Reactome:R-HSA-1971491 "Chondroitin polymerase complex transfers GlcA to CS chain"
 xref: Reactome:R-HSA-3595178 "Defective CHSY1 does not transfer GlcA to chondroitin"
-xref: Reactome:R-HSA-9632034 "CHSY1 transfers GlcA to chondroitin"
 is_a: GO:0015020 ! glucuronosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.226
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -335233,7 +335962,7 @@
 id: GO:0050514
 name: homospermidine synthase (spermidine-specific) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: spermidine + putrescine = sym-homospermidine + propane-1,3-diamine." [EC:2.5.1.45]
+def: "Catalysis of the reaction: spermidine + putrescine = sym-homospermidine + propane-1,3-diamine." [RHEA:11236]
 synonym: "spermidine:putrescine 4-aminobutyltransferase (propane-1,3-diamine-forming)" RELATED [EC:2.5.1.45]
 xref: EC:2.5.1.45
 xref: MetaCyc:2.5.1.45-RXN
@@ -336129,7 +336858,7 @@
 id: GO:0050568
 name: protein-glutamine glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: protein L-glutamine + H2O = protein L-glutamate + NH3." [EC:3.5.1.44, MetaCyc:CHEBDEAMID-RXN]
+def: "Catalysis of the reaction: L-glutaminyl-[protein] + H2O = L-glutamyl-[protein] + NH4+." [RHEA:16441]
 synonym: "destabilase activity" RELATED [EC:3.5.1.44]
 synonym: "glutaminyl-peptide glutaminase activity" RELATED [EC:3.5.1.44]
 synonym: "glutaminylpeptide glutaminase activity" RELATED [EC:3.5.1.44]
@@ -336188,6 +336917,7 @@
 xref: EC:1.1.1.263
 xref: KEGG_REACTION:R05682
 xref: MetaCyc:1.1.1.263-RXN
+xref: Reactome:R-HSA-9773976 "AKR1E2 reduces 1,5-aF to 1,5-aG"
 xref: RHEA:20665
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.263
@@ -336269,7 +336999,7 @@
 synonym: "GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)" RELATED [EC:1.1.1.271]
 xref: EC:1.1.1.271
 xref: MetaCyc:1.1.1.271-RXN
-xref: Reactome:R-HSA-6787642 "TSTA3 dimer reduces GDP-KDGal to GDP-Fuc"
+xref: Reactome:R-HSA-6787642 "GFUS dimer reduces GDP-KDGal to GDP-Fuc"
 xref: RHEA:18885
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 relationship: part_of GO:0042350 ! GDP-L-fucose biosynthetic process
@@ -336860,6 +337590,7 @@
 xref: MetaCyc:RXN66-14
 xref: Reactome:R-HSA-194674 "4,4-dimethylcholesta-8(9),14,24-trien-3beta-ol is reduced to 4,4-dimethylcholesta-8(9),24-dien-3beta-ol [LBR]"
 xref: Reactome:R-HSA-194698 "4,4-dimethylcholesta-8(9),14,24-trien-3beta-ol is reduced to 4,4-dimethylcholesta-8(9),24-dien-3beta-ol [TM7SF2]"
+xref: Reactome:R-HSA-9946140 "TM7SF2 reduces 4,4-diMeCholesta-8,14-dien-3-ol to 4,4-diMe5-cholest-8-en-3-ol"
 xref: RHEA:18561
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.3.1.70
@@ -337459,6 +338190,7 @@
 synonym: "chondroitin sulphate proteoglycan biosynthesis" EXACT []
 synonym: "chondroitin sulphate proteoglycan biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6567
+xref: Reactome:R-HSA-2022870 "CS-GAG biosynthesis"
 is_a: GO:0030166 ! proteoglycan biosynthetic process
 is_a: GO:0050654 ! chondroitin sulfate proteoglycan metabolic process
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
@@ -337480,6 +338212,7 @@
 synonym: "dermatan sulphate proteoglycan biosynthesis" EXACT []
 synonym: "dermatan sulphate proteoglycan biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6568
+xref: Reactome:R-HSA-2022923 "DS-GAG biosynthesis"
 is_a: GO:0030166 ! proteoglycan biosynthetic process
 is_a: GO:0050655 ! dermatan sulfate proteoglycan metabolic process
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
@@ -337592,7 +338325,8 @@
 xref: KEGG_REACTION:R10869
 xref: MetaCyc:RXN-7953
 xref: MetaCyc:RXN-7954
-xref: Reactome:R-HSA-2018659 "Chondroitin 4-sulfate (C4S) can be further sulfated on position 6 by CHST15"
+xref: Reactome:R-HSA-2018659 "CHST15 transfers sulfate to (6-)GalNAc-4-sulfate"
+xref: Reactome:R-HSA-9941312 "CHST15 transfers sulfate to (6-)GalNAc-4-sulfate"
 xref: RHEA:54300
 xref: RHEA:54304
 is_a: GO:0008146 ! sulfotransferase activity
@@ -337931,6 +338665,9 @@
 synonym: "downregulation of antiviral response" EXACT []
 synonym: "inhibition of antiviral response" NARROW []
 synonym: "negative regulation of antiviral response" EXACT [GOC:dph]
+xref: Reactome:R-HSA-164938 "Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters"
+xref: Reactome:R-HSA-164944 "Nef and signal transduction"
+xref: Reactome:R-HSA-168276 "NS1 Mediated Effects on Host Pathways"
 is_a: GO:0002832 ! negative regulation of response to biotic stimulus
 is_a: GO:0031348 ! negative regulation of defense response
 is_a: GO:0032102 ! negative regulation of response to external stimulus
@@ -338383,6 +339120,7 @@
 def: "Binding to a low-density lipoprotein receptor." [GOC:ai]
 synonym: "LDL receptor binding" EXACT []
 synonym: "low-density lipoprotein receptor binding" EXACT [GOC:dph]
+xref: Reactome:R-HSA-2423785 "CR:atREs binds apoE and HSPG"
 is_a: GO:0070325 ! lipoprotein particle receptor binding
 
 [Term]
@@ -342337,7 +343075,6 @@
 def: "OBSOLETE. The process of assisting in the correct noncovalent assembly of posttranslational proteins and does not depend on additional protein cofactors. This function occurs over one or more cycles of nucleotide-dependent binding and release." [GOC:rb]
 comment: This term represent a GO-CAM model
 synonym: "chaperone cofactor-independent protein folding" EXACT [GOC:rb]
-xref: Reactome:R-HSA-390450 "Folding of actin by CCT/TriC"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21869" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0006457
@@ -342383,7 +343120,6 @@
 synonym: "regulation of sequence-specific DNA binding transcription factor activity" EXACT []
 synonym: "regulation of thyroid hormone receptor activity" NARROW []
 synonym: "regulation of transcription factor activity" BROAD [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-9708530 "Regulation of BACH1 activity"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30856" xsd:anyURI
 is_obsolete: true
 
@@ -342762,8 +343498,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid = (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate. 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid is also known as 12S-HpETE, and (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate as hepoxilin A3." [PMID:15123652]
 comment: The reason for obsoletion is that this term is equivalent to hydroperoxy icosatetraenoate isomerase activity.
-xref: Reactome:R-HSA-2161794 "Arachidonate is converted to HXA3/B3 by ALOX12"
-xref: Reactome:R-HSA-8942208 "ALOXE3 isomerises 12R-HpETE to HXA3"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29695" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0106255
@@ -342775,7 +343509,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving hepoxilins, a class of bioactive icosanoids with roles in the regulation of cell physiology." [PMID:15123652]
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "hepoxilin metabolism" EXACT []
-xref: Reactome:R-HSA-2142696 "Synthesis of Hepoxilins (HX) and Trioxilins (TrX)"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -348667,6 +349400,7 @@
 subset: goslim_pombe
 subset: goslim_prokaryote
 subset: goslim_yeast
+xref: Reactome:R-HSA-9918432 "Maturation of DENV proteins"
 is_a: GO:0019538 ! protein metabolic process
 relationship: part_of GO:0010467 ! gene expression
 
@@ -349797,6 +350531,7 @@
 synonym: "growth of symbiont in host vacuole" NARROW []
 synonym: "interaction with host" EXACT []
 xref: Reactome:R-HSA-9635486 "Infection with Mycobacterium tuberculosis"
+xref: Reactome:R-HSA-9640148 "Infection with Enterobacteria"
 xref: Reactome:R-HSA-9658195 "Leishmania infection"
 is_a: GO:0044403 ! biological process involved in symbiotic interaction
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20191" xsd:anyURI
@@ -350069,7 +350804,6 @@
 def: "OBSOLETE. Catalysis of the reaction: C-terminal protein amino acid methyl ester + H2O = protein amino acid + methanol." [PMID:10318862, PMID:8650216]
 comment: This seems to be outside the scope of GO: it represent a specific protein and a 'domain.
 synonym: "protein phosphatase methylesterase activity" NARROW []
-xref: Reactome:R-HSA-8856951 "PP2A demethylation by PPME1"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27558" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051723
@@ -350086,6 +350820,7 @@
 xref: EC:3.1.1.89
 xref: MetaCyc:RXN-12322
 xref: Reactome:R-HSA-5367022 "DPH7 hydrolyzes a methyl group on Me-diphthine EEF2"
+xref: Reactome:R-HSA-8856951 "PP2A demethylation by PPME1"
 xref: RHEA:48548
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -350111,6 +350846,7 @@
 synonym: "oxidized nicotinamide adenine dinucleotide transmembrane transporter activity" EXACT []
 synonym: "reduced NAD transporter activity" EXACT []
 synonym: "reduced nicotinamide adenine dinucleotide transmembrane transporter activity" EXACT []
+xref: Reactome:R-HSA-9938195 "SLC25A51 mediates mitochondrial uptake of NAD+"
 is_a: GO:0000295 ! adenine nucleotide transmembrane transporter activity
 relationship: part_of GO:0035352 ! NAD transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21577" xsd:anyURI
@@ -353031,10 +353767,9 @@
 id: GO:0051975
 name: L-lysine biosynthetic process via alpha-aminoadipate and saccharopine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and saccharopine. This pathway is used by yeast and fungi to synthesize the essential amino acid L-lysine, and pathway intermediates are often incorporated into secondary metabolic processes. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This is then reduced to form alpha-aminoadipate 6-semialdehyde, which is metabolized to saccharopine and finally L-lysine." [MetaCyc:LYSINE-AMINOAD-PWY]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and saccharopine. This pathway is used by yeast and fungi to synthesize the essential amino acid L-lysine, and pathway intermediates are often incorporated into secondary metabolic processes. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This is then reduced to form alpha-aminoadipate 6-semialdehyde, which is metabolized to saccharopine and finally L-lysine." [GOC:curators]
 synonym: "lysine biosynthesis via aminoadipic acid and saccharopine" EXACT []
 synonym: "lysine biosynthetic process via aminoadipic acid and saccharopine" EXACT []
-xref: MetaCyc:LYSINE-AMINOAD-PWY
 is_a: GO:0019878 ! L-lysine biosynthetic process via aminoadipic acid
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -353042,10 +353777,9 @@
 id: GO:0051976
 name: L-lysine biosynthetic process via alpha-aminoadipate and N2-acetyl-alpha-aminoadipate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and N2-acetyl-alpha-aminoadipate. This pathway of prokaryotic lysine biosynthesis via alpha-aminoadipate was discovered in the hyper-thermophilic Gram-negative eubacterium Thermus thermophilus. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This undergoes acetylation, to form N2-acetyl-alpha-aminoadipate, and is then phosphorylated to give N2-acetyl-alpha-aminoadipyl-delta-phosphate. This is converted to N2-acetyl-alpha-aminoadipate semialdehyde, which is then converted to N2-acetyl-L-lysine. A final deacetylation reaction produces L-lysine." [MetaCyc:PWY-3081]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and N2-acetyl-alpha-aminoadipate. This pathway of prokaryotic lysine biosynthesis via alpha-aminoadipate was discovered in the hyper-thermophilic Gram-negative eubacterium Thermus thermophilus. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This undergoes acetylation, to form N2-acetyl-alpha-aminoadipate, and is then phosphorylated to give N2-acetyl-alpha-aminoadipyl-delta-phosphate. This is converted to N2-acetyl-alpha-aminoadipate semialdehyde, which is then converted to N2-acetyl-L-lysine. A final deacetylation reaction produces L-lysine." [GOC:curators]
 synonym: "lysine biosynthesis via aminoadipic acid and N2-acetyl-alpha-aminoadipate" EXACT []
 synonym: "lysine biosynthetic process via aminoadipic acid and N2-acetyl-alpha-aminoadipate" EXACT []
-xref: MetaCyc:PWY-3081
 is_a: GO:0019878 ! L-lysine biosynthetic process via aminoadipic acid
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -353063,7 +353797,6 @@
 namespace: molecular_function
 def: "Enables the directed movement of lysophospholipids from one side of a membrane to the other. A lysophospholipid is a phospholipid that lacks one of its fatty acyl chains; it is an intermediate formed during digestion of dietary and biliary phospholipids." [GOC:ai]
 synonym: "lysophospholipid transporter activity" BROAD []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0015370 ! solute:sodium symporter activity
 is_a: GO:0015605 ! organophosphate ester transmembrane transporter activity
 is_a: GO:0170055 ! lipid transmembrane transporter activity
@@ -353268,11 +354001,13 @@
 
 [Term]
 id: GO:0051994
-name: P-methyltransferase activity
+name: obsolete P-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group to the phosphorus atom of an acceptor molecule." [GOC:ai]
-is_a: GO:0008168 ! methyltransferase activity
+def: "OBSOLETE. Catalysis of the transfer of a methyl group to the phosphorus atom of an acceptor molecule." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25591" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2010-07-16T04:39:34Z
 
@@ -353298,6 +354033,8 @@
 synonym: "squalene synthetase activity" RELATED [EC:2.5.1.21]
 xref: EC:2.5.1.21
 xref: MetaCyc:RXN66-281
+xref: Reactome:R-HSA-191402 "Reduction of presqualene diphosphate to form squalene"
+xref: Reactome:R-HSA-191405 "Two FAPP molecules dimerize to form presqualene diphosphate"
 xref: RHEA:32295
 xref: RHEA:32299
 is_a: GO:0004659 ! prenyltransferase activity
@@ -353863,6 +354600,7 @@
 synonym: "negative regulation by symbiont of host programmed cell death" EXACT []
 synonym: "suppression by symbiont of host PCD" EXACT []
 synonym: "suppression by symbiont of host programmed cell death" EXACT []
+xref: Reactome:R-HSA-9636569 "Suppression of autophagy"
 is_a: GO:0052040 ! symbiont-mediated perturbation of host programmed cell death
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23652" xsd:anyURI
 
@@ -355005,8 +355743,8 @@
 synonym: "modulation by symbiont of host apoptotic process" EXACT []
 synonym: "modulation by virus of host apoptosis" NARROW []
 synonym: "regulation by virus of host apoptosis" NARROW []
+xref: Reactome:R-HSA-9920951 "Dengue virus modulates apoptosis"
 xref: VZ:1518 "Apoptosis modulation"
-is_a: GO:0051709 ! regulation of killing of cells of another organism
 is_a: GO:0052040 ! symbiont-mediated perturbation of host programmed cell death
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25986" xsd:anyURI
 
@@ -358266,6 +359004,7 @@
 synonym: "modulation by organism of immune response of other organism involved in symbiotic interaction" BROAD []
 synonym: "modulation by symbiont of host immune response" EXACT []
 synonym: "perturbation of host immune response" EXACT []
+xref: Reactome:R-HSA-9920588 "Dengue virus activates/modulates innate and adaptive immune responses"
 is_a: GO:0052031 ! symbiont-mediated perturbation of host defense response
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25569" xsd:anyURI
 
@@ -358685,7 +359424,7 @@
 id: GO:0052595
 name: aliphatic amine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an aliphatic amine + H2O + O2 = an aldehyde + NH3 + hydrogen peroxide + H+." [MetaCyc:AMINEOXID-RXN]
+def: "Catalysis of the reaction: an aliphatic amine + H2O + O2 = an aldehyde + NH4+ + H2O2 + H+." [MetaCyc:AMINEOXID-RXN]
 synonym: "aliphatic-amine oxidase activity" EXACT []
 synonym: "aliphatic-amine:oxygen oxidoreductase(deaminating) activity" EXACT []
 xref: MetaCyc:AMINEOXID-RXN
@@ -359549,44 +360288,51 @@
 
 [Term]
 id: GO:0052654
-name: L-leucine-2-oxoglutarate transaminase activity
+name: L-leucine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-leucine = 4-methyl-2-oxopentanoate + L-glutamatic acid." [RHEA:18321]
+def: "Catalysis of the reaction:L-leucine + 2-oxoglutarate = 4-methyl-2-oxopentanoate + L-glutamate." [RHEA:18321]
 subset: goslim_chembl
 synonym: "L-leucine aminotransferase activity" EXACT []
+synonym: "L-leucine-2-oxoglutarate transaminase activity" EXACT []
+xref: EC:2.6.1.6
 xref: KEGG_REACTION:R01090
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERLEU-RXN
 xref: RHEA:18321
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
+property_value: skos:exactMatch EC:2.6.1.6
 property_value: skos:exactMatch RHEA:18321
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28176" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0052655
-name: L-valine-2-oxoglutarate transaminase activity
+name: L-valine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-valine = 3-methyl-2-oxobutanoic acid + L-glutamatic acid." [RHEA:24813]
+def: "Catalysis of the reaction: L-valine + 2-oxoglutarate = 3-methyl-2-oxobutanoate + L-glutamate." [RHEA:24813]
 subset: goslim_chembl
 synonym: "L-valine aminotransferase activity" EXACT []
+synonym: "L-valine-2-oxoglutarate transaminase activity" EXACT []
 xref: KEGG_REACTION:R01214
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERVAL-RXN
 xref: RHEA:24813
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch RHEA:24813
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0052656
-name: L-isoleucine-2-oxoglutarate transaminase activity
+name: L-isoleucine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-isoleucine = (S)-3-methyl-2-oxopentanoic acid + L-glutamic acid." [RHEA:24801]
+def: "Catalysis of the reaction: L-isoleucine + 2-oxoglutarate = (S)-3-methyl-2-oxopentanoate + L-glutamate." [RHEA:24801]
 subset: goslim_chembl
 synonym: "L-isoleucine aminotransferase activity" EXACT []
+synonym: "L-isoleucine-2-oxoglutarate transaminase activity" EXACT []
 xref: KEGG_REACTION:R02199
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERILEU-RXN
 xref: RHEA:24801
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch RHEA:24801
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 
@@ -360068,23 +360814,23 @@
 
 [Term]
 id: GO:0052687
-name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (ADP-forming) activity
+name: obsolete (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (ADP-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP + phosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06396]
+def: "OBSOLETE. Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP + phosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06396]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "3-isopropenyl-6-oxoheptanoyl-CoA synthetase activity" BROAD [UM-BBD_reactionID:r0737]
-xref: KEGG_REACTION:R06396
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052688
-name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (AMP-forming) activity
+name: obsolete (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (AMP-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + AMP + diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06515]
+def: "OBSOLETE. Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + AMP + diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06515]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "3-isopropenyl-6-oxoheptanoyl-CoA synthetase activity" BROAD [UM-BBD_reactionID:r0737]
-xref: KEGG_REACTION:R06515
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052689
@@ -360193,7 +360939,6 @@
 synonym: "cellular glucuronide biosynthetic process" EXACT []
 synonym: "cellular glucuronoside biosynthesis" EXACT []
 synonym: "cellular glucuronoside biosynthetic process" EXACT []
-xref: Reactome:R-HSA-156588 "Glucuronidation"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28506" xsd:anyURI
 is_obsolete: true
 creation_date: 2011-07-27T05:28:18Z
@@ -360530,10 +361275,12 @@
 
 [Term]
 id: GO:0052715
-name: mannosyl-diinositol phosphorylceramide phospholipase activity
+name: obsolete mannosyl-diinositol phosphorylceramide phospholipase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: mannosyl-diinositol phosphorylceramide + H2O = C26-phytoceramide + mannosyldiphosphorylinositol." [GOC:ai]
-is_a: GO:0052713 ! inositol phosphorylceramide phospholipase activity
+def: "OBSOLETE. Catalysis of the reaction: mannosyl-diinositol phosphorylceramide + H2O = C26-phytoceramide + mannosyldiphosphorylinositol." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2011-08-05T08:02:57Z
 
@@ -360560,7 +361307,7 @@
 id: GO:0052717
 name: tRNA-specific adenosine-34 deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine-34 + H2O = inosine-34 + NH3, in a tRNA-Ala molecule." [PMID:17875641, RHEA:43168]
+def: "Catalysis of the reaction: adenosine34 in tRNA + H2O + H+ = inosine34 in tRNA + NH4+." [PMID:17875641, RHEA:43168]
 synonym: "tRNA(Ala)-A34 deaminase activity" EXACT []
 xref: EC:3.5.4.33
 xref: RHEA:43168
@@ -360873,12 +361620,13 @@
 
 [Term]
 id: GO:0052735
-name: tRNA (cytidine-3-)-methyltransferase activity
+name: tRNA (cytidine-N3)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = an N3-methylcytidine in tRNA + H+ + S-adenosyl-L-homocysteine." [RHEA:60908]
 synonym: "S-adenosyl-L-methionine:tRNA (cytosine-3-)-methyltransferase activity" EXACT []
 synonym: "S-adenosyl-L-methionine:tRNA cytosine-3-methyltransferase activity" EXACT []
 synonym: "transfer ribonucleate cytosine 3-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine-3-)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine 3)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine-3-)-methyltransferase activity" EXACT []
 synonym: "tRNA cytosine 3-methyltransferase activity" EXACT []
@@ -360887,6 +361635,7 @@
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
 property_value: skos:exactMatch RHEA:60908
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25722" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -360933,7 +361682,6 @@
 def: "OBSOLETE. Catalysis of the reaction: phosphatidylserine + H2O = 2-acyl-sn-glycero-3-phosphoserine + fatty acid." [KEGG_REACTION:R04034]
 comment: The reason for obsoletion is that this molecular function is equivalent to phospholipase A1 activity.
 xref: KEGG_REACTION:R04034
-xref: Reactome:R-HSA-8869425 "PLA1A hydrolyses PS to 2-acyl LPS"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28816" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008970
@@ -361075,7 +361823,7 @@
 id: GO:0052750
 name: versatile peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: reactive black 5 + hydrogen peroxide = oxidized reactive black 5 + 2 H2O." [RHEA:22396]
+def: "Catalysis of the reaction: reactive black 5 + H2O2 = oxidized reactive black 5 + 2 H2O." [RHEA:22396]
 synonym: "reactive-black-5:hydrogen-peroxide oxidoreductase activity" BROAD [EC:1.11.1.16]
 xref: EC:1.11.1.16
 xref: KEGG_REACTION:R07612
@@ -361100,15 +361848,12 @@
 
 [Term]
 id: GO:0052752
-name: reduced coenzyme F420:heterodisulfide oxidoreductase activity
+name: obsolete reduced coenzyme F420:heterodisulfide oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: reduced coenzyme F420 + CoB-S-S-CoM = coenzyme F420 + CoM-SH + CoB-SH." [GOC:mengo_curators, PMID:9914308]
-synonym: "coenzyme F420-dependent heterodisulfide oxidoreductase activity" EXACT [PMID:9914308]
-synonym: "F420-dependent heterodisulfide oxidoreductase activity" EXACT [PMID:9914308]
-synonym: "F420H2:heterodisulfide oxidoreductase activity" EXACT []
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
-created_by: ai
-creation_date: 2011-09-21T04:21:16Z
+def: "OBSOLETE. Catalysis of the reaction: reduced coenzyme F420 + CoB-S-S-CoM = coenzyme F420 + CoM-SH + CoB-SH." [GOC:mengo_curators, PMID:9914308]
+comment: The reason for obsoletion is that this reaction MF has no EC or RHEA xref and no annotations.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052753
@@ -361123,17 +361868,17 @@
 
 [Term]
 id: GO:0052754
-name: GTP:coenzyme F420 guanyltransferase activity
+name: obsolete GTP:coenzyme F420 guanyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: GTP + factor gamma-F420-2 + H+ = coenzyme F390-G + diphosphate." [GOC:mengo_curators, MetaCyc:RXN-18273]
+def: "OBSOLETE. Catalysis of the reaction: GTP + factor gamma-F420-2 + H+ = coenzyme F390-G + diphosphate." [GOC:mengo_curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "coenzyme F390-G synthetase activity" EXACT []
 synonym: "GTP:coenzyme F420 guanylyltransferase activity" EXACT []
-xref: MetaCyc:RXN-18273
-is_a: GO:0070568 ! guanylyltransferase activity
-property_value: skos:exactMatch MetaCyc:RXN-18273
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28324" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28503" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28776" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2011-09-22T03:25:30Z
 
@@ -362011,7 +362756,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = myo-inositol 1,3,4,5,6-pentakisphosphate + phosphate." [GOC:curators]
 comment: The reason for obsoletion is that this term is equivalent to inositol hexakisphosphate 3-phosphatase activity.
-xref: Reactome:R-HSA-1855225 "IP6 is dephosphorylated to I(1,2,4,5,6)P5 by MINPP1 in the ER lumen"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30385" xsd:anyURI
 is_obsolete: true
@@ -362354,6 +363098,7 @@
 xref: KEGG_REACTION:R09541
 xref: MetaCyc:RXN0-6555
 xref: Reactome:R-HSA-917811 "STEAP3,STEAP4 reduce Fe3+ to Fe2+"
+xref: Reactome:R-HSA-9927945 "YqjH reduces FeEnt"
 xref: RHEA:28795
 is_a: GO:0000293 ! ferric-chelate reductase activity
 is_a: GO:0016723 ! oxidoreductase activity, acting on metal ions, NAD or NADP as acceptor
@@ -362391,6 +363136,7 @@
 synonym: "NADHX epimerase activity" NARROW []
 synonym: "NADPHX epimerase activity" NARROW []
 xref: EC:5.1.99.6
+xref: Reactome:R-HSA-6806966 "APOA1BP dimer epimerises R-NAD(P)HX to S-NAD(P)HX"
 xref: RHEA:32215
 xref: RHEA:32227
 is_a: GO:0016854 ! racemase and epimerase activity
@@ -362405,7 +363151,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: (R)-NADPHX = (S)-NADPHX." [PMID:21994945]
 comment: This term was obsoleted because it represents the same activity as NADHX epimerase activity ; GO:0052856.
-xref: Reactome:R-HSA-6806966 "APOA1BP dimer epimerises R-NAD(P)HX to S-NAD(P)HX"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0052856
@@ -362565,9 +363310,11 @@
 id: GO:0052868
 name: protein-lysine lysyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: protein-lysine + protein-lysine = protein N6-(lysyl)-L-lysine + protein. This reaction is the addition of lysine group from one protein to a lysine residue in a second protein, producing N6-(lysyl)-L-lysine." [PMID:20729861]
+def: "Catalysis of the reaction: D-beta-lysine + L-lysyl-[protein] + ATP = N6-((3R)-3,6-diaminohexanoyl)-L-lysyl-[protein] + AMP + diphosphate + H+n. This reaction is the addition of lysine group from one protein to a lysine residue in a second protein, producing N6-(lysyl)-L-lysine." [PMID:20729861, RHEA:83435]
+xref: RHEA:83435
 is_a: GO:0016755 ! aminoacyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31478" xsd:anyURI
 created_by: ai
 creation_date: 2012-01-11T03:05:27Z
 
@@ -363079,23 +363826,29 @@
 synonym: "N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity" NARROW [EC:1.5.3.13]
 xref: EC:1.5.3.13
 xref: KEGG_REACTION:R03899
-xref: MetaCyc:RXN-12090
+xref: MetaCyc:RXN-10460
+xref: MetaCyc:RXN-12091
+xref: MetaCyc:RXN-9940
+xref: Reactome:R-HSA-141348 "PAO:FAD oxidises NASPM to PTCN"
+xref: Reactome:R-HSA-141351 "PAOX:FAD oxidises NASPN to SPM"
 xref: RHEA:25800
 xref: RHEA:25812
 xref: RHEA:25868
 is_a: GO:0046592 ! polyamine oxidase activity
 property_value: skos:exactMatch EC:1.5.3.13
 property_value: skos:exactMatch KEGG_REACTION:R03899
-property_value: skos:exactMatch MetaCyc:RXN-12090
+property_value: skos:narrowMatch MetaCyc:RXN-10460
+property_value: skos:narrowMatch MetaCyc:RXN-12091
+property_value: skos:narrowMatch MetaCyc:RXN-9940
 property_value: skos:narrowMatch RHEA:25800
 property_value: skos:narrowMatch RHEA:25812
 property_value: skos:narrowMatch RHEA:25868
-property_value: skos:relatedMatch MetaCyc:RXN-9940
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24701" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25837" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31136" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0052904
@@ -363103,8 +363856,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: H2O + N(1)-acetylspermidine + O2 = 3-acetamidopropanal + H2O2 + putrescine." [GOC:curators]
 comment: This term was obsoleted because it represents a specific substrate of N(1)-acetylpolyamine oxidase (3-acetamidopropanal-forming) activity ; GO:0052903.
-xref: Reactome:R-HSA-141348 "PAO:FAD oxidises NASPM to PTCN"
-xref: Reactome:R-HSA-141351 "PAOX:FAD oxidises NASPN to SPM"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24701" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 is_obsolete: true
@@ -363993,17 +364744,21 @@
 
 [Term]
 id: GO:0055042
-name: 5-valerolactone hydrolase activity
+name: obsolete 5-valerolactone hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-valerolactone + H2O = 5-hydroxyvalerate." [GOC:jid, GOC:mlg]
-is_a: GO:0046573 ! lactonohydrolase activity
+def: "OBSOLETE. Catalysis of the reaction: 5-valerolactone + H2O = 5-hydroxyvalerate." [GOC:jid, GOC:mlg]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0055043
-name: 5-oxovalerate dehydrogenase activity
+name: obsolete 5-oxovalerate dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-oxovalerate + NADP+ + H2O = glutarate + NADPH + H+." [GOC:jid, GOC:mlg]
-is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 5-oxovalerate + NADP+ + H2O = glutarate + NADPH + H+." [GOC:jid, GOC:mlg]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0055044
@@ -364118,9 +364873,11 @@
 def: "Enables the transfer of the D-enantiomer of the hexose monosaccharide glucose from one side of a membrane to the other." [GOC:jid, GOC:jsg, GOC:mah]
 xref: Reactome:R-HSA-429094 "SLC2A6,8,10,12 transport Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-450095 "GLUT2 (SLC2A2) transports Glc from cytosol to extracellular region"
+xref: Reactome:R-HSA-499981 "Transport of Extracellular Glucose to the Cytosol by GLUT1 and GLUT2"
 xref: Reactome:R-HSA-5339524 "GLUT1 (SLC2A1) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-5632804 "SLC2A1 tetramer does not transport Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-5632871 "Defective SLC2A10 does not transport Glc from extracellular region to cytosol"
+xref: Reactome:R-HSA-5653873 "SLC2A1 tetramer transports Glc from cytosol to Golgi lumen"
 xref: Reactome:R-HSA-8981553 "GLUT14 (SLC2A14) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-8981564 "GLUT3 (SLC2A3) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-8981570 "GLUT4 (SLC2A4) tetramer transports Glc from extracellular region to cytosol"
@@ -364781,15 +365538,18 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of L-proline, an L-enantiomer of a chiral, cyclic, nonessential alpha-amino acid found in peptide linkage in proteins." [GOC:ecd]
 synonym: "pyrrolidine-2-carboxylic acid biosynthetic process" BROAD []
+xref: MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
 xref: MetaCyc:PROSYN-PWY
 xref: MetaCyc:PWY-3341
 xref: MetaCyc:PWY-4281 {status="skos:narrowMatch"}
 xref: MetaCyc:PWY-4981
 is_a: GO:0006560 ! L-proline metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
 property_value: skos:narrowMatch MetaCyc:PROSYN-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3341
 property_value: skos:narrowMatch MetaCyc:PWY-4981
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: jid
 creation_date: 2009-10-22T01:59:56Z
 
@@ -364797,10 +365557,13 @@
 id: GO:0055130
 name: D-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-alanine, the D-enantiomer of the amino acid alanine." [GOC:ecd]
+def: "The chemical reactions and pathways resulting in the breakdown of D-alanine." [GOC:curators]
 synonym: "(2R)-2-aminopropanoic acid catabolic process" EXACT []
+xref: MetaCyc:PWY-8448
+is_a: GO:0006522 ! alanine metabolic process
 is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:0046436 ! D-alanine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 created_by: jid
 creation_date: 2009-10-22T02:02:42Z
 
@@ -365681,6 +366444,7 @@
 synonym: "binding, bridging" EXACT []
 synonym: "protein complex scaffold activity" BROAD []
 synonym: "protein-containing complex scaffold activity" BROAD []
+xref: Reactome:R-HSA-2564826 "4Fe-4S cluster assembles on NUBP2:NUBP1 scaffold"
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0005488 ! binding
 
@@ -366191,10 +366955,11 @@
 id: GO:0060138
 name: fetal process involved in parturition
 namespace: biological_process
-def: "A reproductive process occurring in the embryo that results in birth." [GOC:dph]
+def: "A reproductive process occurring in the fetus that results in birth." [GOC:dph]
 is_a: GO:0048609 ! multicellular organismal reproductive process
 relationship: part_of GO:0007567 ! parturition
 relationship: part_of GO:0009792 ! embryo development ending in birth or egg hatching
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31526" xsd:anyURI
 
 [Term]
 id: GO:0060140
@@ -366213,6 +366978,7 @@
 def: "A process in which a symbiont initiates, promotes, or enhances the formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual host cells. Syncytia are produced by viruses that are able to fuse directly at the cell surface without requiring endocytosis." [GOC:dph]
 synonym: "positive regulation of syncytium formation by virus" EXACT []
 synonym: "syncytium formation induced by viral infection" EXACT [VZ:5957]
+xref: Reactome:R-HSA-9733458 "Induction of Cell-Cell Fusion"
 xref: VZ:5957
 is_a: GO:0044068 ! symbiont-mediated perturbation of host cellular process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28791" xsd:anyURI
@@ -367080,6 +367846,7 @@
 name: lipase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a lipase, an enzyme that catalyzes of the hydrolysis of a lipid." [GOC:BHF, GOC:dph, GOC:tb]
+xref: Reactome:R-HSA-163549 "phosphorylated HSL dimer + FABP4 -> phosphorylated HSL dimer:FABP4 complex"
 is_a: GO:0008047 ! enzyme activator activity
 relationship: positively_regulates GO:0016298 ! lipase activity
 
@@ -370825,9 +371592,10 @@
 namespace: molecular_function
 def: "Binds to and modulates the activity of an ATP hydrolysis activity." [GOC:dph, GOC:tb]
 synonym: "ATP hydrolysis regulator activity" EXACT []
-is_a: GO:0098772 ! molecular function regulator activity
-relationship: regulates GO:0140657 ! ATP-dependent activity
+is_a: GO:0060589 ! nucleoside-triphosphatase regulator activity
+relationship: regulates GO:0016887 ! ATP hydrolysis activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31398" xsd:anyURI
 created_by: dph
 creation_date: 2009-05-08T02:49:24Z
 
@@ -381106,12 +381874,14 @@
 
 [Term]
 id: GO:0061502
-name: early endosome to recycling endosome transport
+name: obsolete early endosome to recycling endosome transport
 namespace: biological_process
-def: "The directed movement of substances, in membrane-bounded vesicles, from the early sorting endosomes to the recycling endosomes." [GOC:dph, GOC:kmv, PMID:21474295]
-is_a: GO:0098927 ! vesicle-mediated transport between endosomal compartments
-relationship: occurs_in GO:0005737 ! cytoplasm
+def: "OBSOLETE. The directed movement of substances, in membrane-bounded vesicles, from the early sorting endosomes to the recycling endosomes." [GOC:dph, GOC:kmv, PMID:21474295]
+comment: This term is described by another GO term
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26386" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0032456
 created_by: dph
 creation_date: 2013-01-04T07:24:56Z
 
@@ -382277,7 +383047,6 @@
 synonym: "N-terminal propionyltransferase activity" EXACT [PMID:23043182]
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 is_a: GO:0140096 ! catalytic activity, acting on a protein
-relationship: part_of GO:0061606 ! N-terminal protein amino acid propionylation
 created_by: dph
 creation_date: 2014-02-11T14:24:35Z
 
@@ -382287,6 +383056,7 @@
 namespace: molecular_function
 def: "Combining with a nuclear import signal (NIS) on a cargo to be transported, to mediate transport of the cargo through the nuclear pore, from the cytoplasm to the nuclear lumen. The cargo can be either a RNA or a protein." [GOC:dph, GOC:pg, GOC:vw, PMID:28713609, Wikipedia:Nuclear_transport]
 synonym: "importin activity" RELATED []
+xref: Reactome:R-HSA-2065550 "SREBP1A,1C,2 binds Importin beta-1"
 is_a: GO:0140142 ! nucleocytoplasmic carrier activity
 intersection_of: GO:0140142 ! nucleocytoplasmic carrier activity
 intersection_of: part_of GO:0051170 ! import into nucleus
@@ -382714,6 +383484,15 @@
 xref: Reactome:R-HSA-9762091 "NFE2L2 is ubiquitinated by BTRC:SKP1:CUL1:RBX1"
 xref: Reactome:R-HSA-9766645 "CUL3:RBX1 ubiquitinates KEAP1"
 xref: Reactome:R-HSA-990526 "Recruitment of ITCH and K48 ubiquitination of MAVS"
+xref: Reactome:R-HSA-9929490 "CD274 ubiquitination by SPOP:CUL3:RBX1 complex"
+xref: Reactome:R-HSA-9931298 "Ubiquitination of CD274 by ERAD complex"
+xref: Reactome:R-HSA-9931430 "MIB2-mediated K63-ubiquitination of CD274"
+xref: Reactome:R-HSA-9932510 "BTRC:CUL1:SKP1:RBX1 (SCF-beta-TrCP1) ubiquitinylates phosphorylated PER1,2,3"
+xref: Reactome:R-HSA-9932512 "FBXL3:CUL1:SKP1:RBX1 ubiquitinylates phosphorylated CRY1,2"
+xref: Reactome:R-HSA-9945766 "NS3 is ubiquitinated"
+xref: Reactome:R-HSA-9946390 "NS1 dimer is ubiquitinated"
+xref: Reactome:R-HSA-9948362 "LTN1 K48 polyubiquitinates the nascent peptide in NEMF:60S ribosomal subunit:peptidyl-tRNA with nascent peptide"
+xref: Reactome:R-HSA-9954737 "RCHY1 K48 polyubiquitinates alanine-tailed nascent peptide"
 is_a: GO:0004842 ! ubiquitin-protein transferase activity
 is_a: GO:0061659 ! ubiquitin-like protein ligase activity
 property_value: skos:narrowMatch EC:2.3.2.26
@@ -382949,6 +383728,7 @@
 namespace: molecular_function
 def: "Isoenergetic transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y = Y-ISG15 + X, where both the X-ISG15 and Y-ISG15 linkages are thioester bonds between the C-terminal amino acid of ISG15 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
 synonym: "E2" BROAD [GOC:dph]
+xref: Reactome:R-HSA-1169404 "Transfer of ISG15 from E1 to E2 (UBCH8)"
 is_a: GO:0042296 ! ISG15 transferase activity
 is_a: GO:0061650 ! ubiquitin-like protein conjugating enzyme activity
 created_by: dph
@@ -383052,6 +383832,13 @@
 namespace: molecular_function
 def: "Catalysis of the transfer of a ISG15 to a substrate protein via the reaction X-ISG15 + S = X + S-ISG15, where X is either an E2 or E3 enzyme, the X-ISG15 linkage is a thioester bond, and the S-ISG15 linkage is an isopeptide bond between the C-terminal amino acid of ISG15 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
 synonym: "E3" BROAD [GOC:dph]
+xref: Reactome:R-HSA-1169394 "ISGylation of IRF3"
+xref: Reactome:R-HSA-1169395 "ISGylation of viral protein NS1"
+xref: Reactome:R-HSA-1169398 "ISGylation of host protein filamin B"
+xref: Reactome:R-HSA-1169402 "ISGylation of E2 conjugating enzymes"
+xref: Reactome:R-HSA-1169405 "ISGylation of protein phosphatase 1 beta (PP2CB)"
+xref: Reactome:R-HSA-1169406 "ISGylation of host proteins"
+xref: Reactome:R-HSA-9833973 "ISGylation of PKR"
 is_a: GO:0042296 ! ISG15 transferase activity
 is_a: GO:0061659 ! ubiquitin-like protein ligase activity
 created_by: dph
@@ -383204,6 +383991,7 @@
 name: importin-alpha family protein binding
 namespace: molecular_function
 def: "Binding to a member of the importin-alpha family." [PMID:15350979, PMID:17170104, PMID:23734157]
+xref: Reactome:R-HSA-2065539 "SREBP1A,1C,2:Importin beta-1 dissociates"
 is_a: GO:0005515 ! protein binding
 created_by: dph
 creation_date: 2015-01-10T18:25:22Z
@@ -383315,7 +384103,7 @@
 is_a: GO:0030163 ! protein catabolic process
 relationship: has_part GO:0005102 ! signaling receptor binding
 relationship: has_part GO:0051087 ! protein-folding chaperone binding
-relationship: has_part GO:0140597 ! protein carrier chaperone
+relationship: has_part GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20850" xsd:anyURI
 created_by: dph
 creation_date: 2015-03-04T14:35:26Z
@@ -383472,6 +384260,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: N(6)-glutaryl-L-lysyl-[protein] + NAD+ + H2O = 2''-O-glutaryl-ADP-D-ribose + nicotinamide + L-lysyl-[protein]." [PMID:24703693, RHEA:47664]
 comment: This reaction is the removal of a glutaryl group from a glutarylated lysine residue of a protein or peptide.
+xref: Reactome:R-HSA-9955504 "SIRT5 deglutarylates CPS1"
 xref: RHEA:47664
 is_a: GO:0141218 ! NAD-dependent protein lysine deacylase activity
 property_value: skos:exactMatch RHEA:47664
@@ -383632,6 +384421,7 @@
 def: "Catalysis of the reaction: L-threonine + hydrogencarbonate + ATP = L-threonylcarbamoyladenylate + diphosphate + H2O." [RHEA:36407]
 xref: EC:2.7.7.87
 xref: MetaCyc:RXN-14569
+xref: Reactome:R-HSA-9611218 "YRDC synthesizes threonylcarbamoyladenylate"
 xref: RHEA:36407
 is_a: GO:0016779 ! nucleotidyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.87
@@ -383649,7 +384439,8 @@
 synonym: "t6A synthase activity" EXACT []
 xref: EC:2.3.1.234
 xref: MetaCyc:RXN-14570
-xref: Reactome:R-HSA-6784494 "EKC complex threonylcarbamoylates A37 of tRNAs"
+xref: Reactome:R-HSA-6784494 "KEOPS (EKC) complex threonylcarbamoylates A37 of tRNAs"
+xref: Reactome:R-HSA-9611197 "OSGEPL1 threonylcarbamoylates adenosine-37 in tRNA"
 xref: RHEA:37059
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -383687,6 +384478,7 @@
 namespace: molecular_function
 def: "Combining selectively with extracellular folic acid and delivering it into the cell via endocytosis." [GOC:BHF, GOC:hal]
 synonym: "folate receptor activity" EXACT []
+xref: Reactome:R-HSA-8940134 "FOLR2 binds FOLA"
 is_a: GO:0038024 ! cargo receptor activity
 relationship: has_part GO:0005542 ! folic acid binding
 relationship: part_of GO:0015884 ! folic acid transport
@@ -383947,6 +384739,7 @@
 xref: Reactome:R-HSA-9701531 "p-Y705,S727 STAT3 is acetylated"
 xref: Reactome:R-HSA-9756494 "CREBBP acetylates SARS-CoV-2 N at K375"
 xref: Reactome:R-HSA-9825747 "CREBBP acetylates HINT1 dimer"
+xref: Reactome:R-HSA-9920716 "CLOCK acetylates BMAL1 on lysine-538"
 xref: RHEA:45948
 is_a: GO:0034212 ! protein N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.48
@@ -385957,6 +386750,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: glutathione = 5-oxoproline + L-cysteinylglycine." [PMID:23070364, PMID:27913623]
 xref: EC:4.3.2.7
+xref: Reactome:R-HSA-6785928 "CHAC1,2 cleaves GSH to OPRO and CysGly"
 xref: RHEA:47724
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.7
@@ -386206,6 +387000,12 @@
 name: mRNA (adenine-N1-)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + adenine in mRNA = S-adenosyl-L-homocysteine + N(1)-methyladenine in mRNA." [PMID:29072297]
+xref: Reactome:R-HSA-9937508 "TRMT61B methylates adenosine-1472 of MT-CO1 mRNA"
+xref: Reactome:R-HSA-9937512 "TRMT61B methylates adenosine-371 of MT-ND1 mRNA"
+xref: Reactome:R-HSA-9937518 "TRMT61B methylates adenosine-333, adenosine-458, and adenosine-707 of MT-CO3 mRNA"
+xref: Reactome:R-HSA-9937521 "TRMT61B methylates adenosine-185 of MT-CO2 mRNA"
+xref: Reactome:R-HSA-9937529 "TRMT61B methylates adenosine-616 and adenosine-617 of MT-CYB mRNA"
+xref: Reactome:R-HSA-9937535 "TRMT61B methylates adenosine-252 of MT-ND4L mRNA"
 is_a: GO:0008174 ! mRNA methyltransferase activity
 created_by: dph
 creation_date: 2017-11-08T13:26:35Z
@@ -386713,12 +387513,14 @@
 name: regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "regulation of small molecule metabolism" EXACT []
 xref: Reactome:R-HSA-9861718 "Regulation of pyruvate metabolism"
 is_a: GO:0019222 ! regulation of metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044281 ! small molecule metabolic process
 relationship: regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:11:54Z
 
@@ -386727,12 +387529,14 @@
 name: positive regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "positive regulation of small molecule metabolism" EXACT []
 is_a: GO:0009893 ! positive regulation of metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0044281 ! small molecule metabolic process
 relationship: positively_regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:25:10Z
 
@@ -386741,12 +387545,14 @@
 name: negative regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "negative regulation of small molecule metabolism" EXACT []
 is_a: GO:0009892 ! negative regulation of metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0044281 ! small molecule metabolic process
 relationship: negatively_regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:27:23Z
 
@@ -387024,10 +387830,12 @@
 
 [Term]
 id: GO:0062045
-name: L-lysine alpha-aminotransferase activity
+name: L-lysine:pyruvate alpha-transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-lysine + pyruvate= epsilon-amino-alpha-ketocaproic acid (KAC) + alanine." [PMID:27758894, PMID:28330936]
-is_a: GO:0008483 ! transaminase activity
+synonym: "L-lysine alpha-aminotransferase activity" BROAD []
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dph
 creation_date: 2018-05-11T12:13:57Z
 
@@ -388044,7 +388852,7 @@
 id: GO:0062146
 name: 4-chloro-allylglycine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-chloro-L-lysine + AH2 + O2 = A + formaldehyde + H2O + L-2-amino-4-chloropent-4-enoate + NH4(+)." [PMID:30867596, RHEA:59888]
+def: "Catalysis of the reaction: 4-chloro-L-lysine + AH2 + O2 = A + formaldehyde + H2O + L-2-amino-4-chloropent-4-enoate + NH4+." [PMID:30867596, RHEA:59888]
 xref: RHEA:59888
 is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
 property_value: skos:exactMatch RHEA:59888
@@ -389313,11 +390121,12 @@
 
 [Term]
 id: GO:0070007
-name: glutamic-type endopeptidase activity
+name: obsolete glutamic-type endopeptidase activity
 namespace: molecular_function
-def: "Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE, https://www.ebi.ac.uk/merops/about/glossary.shtml#ENDOPEPTIDASE]
-is_a: GO:0004175 ! endopeptidase activity
-is_a: GO:0070002 ! glutamic-type peptidase activity
+def: "OBSOLETE. Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE, https://www.ebi.ac.uk/merops/about/glossary.shtml#ENDOPEPTIDASE]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0070008
@@ -389591,9 +390400,10 @@
 
 [Term]
 id: GO:0070039
-name: rRNA (guanosine-2'-O-)-methyltransferase activity
+name: rRNA (guanosine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylguanosine." [GOC:imk, GOC:mah]
+synonym: "rRNA (guanosine-2'-O-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6793096 "MRM3 (RNMTL1) methylates guanosine-1370 of 16S rRNA yielding 2'-O-methylguanosine-1370"
 xref: Reactome:R-HSA-6793122 "MRM1 methylates guanosine-1145 of 16S rRNA yielding 2'-O-methylguanosine-1145"
 is_a: GO:0016435 ! rRNA (guanine) methyltransferase activity
@@ -390463,6 +391273,7 @@
 def: "The process resulting in the release of a polypeptide chain from the ribosome in a mitochondrion, usually in response to a termination codon (note that mitochondria use variants of the universal genetic code that differ between different taxa)." [GOC:mah, http://mitogenome.org/index.php/Genetic_Code_of_mitochondria]
 synonym: "mitochondrial translation termination" EXACT [GOC:mah]
 xref: Reactome:R-HSA-5419276 "Mitochondrial translation termination"
+xref: Reactome:R-HSA-9937383 "Mitochondrial ribosome-associated quality control"
 is_a: GO:0006415 ! translational termination
 intersection_of: GO:0006415 ! translational termination
 intersection_of: occurs_in GO:0005739 ! mitochondrion
@@ -390984,11 +391795,15 @@
 
 [Term]
 id: GO:0070178
-name: D-serine metabolic process
+name: obsolete D-serine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-serine, the D-enantiomer of serine, i.e. (2R)-2-amino-3-hydroxypropanoic acid." [CHEBI:16523, GOC:jsg, GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways involving D-serine, the D-enantiomer of serine, i.e. (2R)-2-amino-3-hydroxypropanoic acid." [CHEBI:16523, GOC:jsg, GOC:mah]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-serine metabolism" EXACT []
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
+consider: GO:0036088
+consider: GO:0070179
 
 [Term]
 id: GO:0070179
@@ -391000,7 +391815,6 @@
 synonym: "D-serine formation" EXACT []
 synonym: "D-serine synthesis" EXACT []
 is_a: GO:0046437 ! D-amino acid biosynthetic process
-is_a: GO:0070178 ! D-serine metabolic process
 
 [Term]
 id: GO:0070180
@@ -391494,12 +392308,14 @@
 
 [Term]
 id: GO:0070226
-name: sulfur:ferric ion oxidoreductase activity
+name: obsolete sulfur:ferric ion oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a perthiol + 4 Fe3+ + 3 H2O = sulfite + a thiol + 4 Fe2+ + 8 H+." [MetaCyc:SULFFEOXIDO-RXN]
+def: "OBSOLETE. Catalysis of the reaction: S-sulfanylglutathione + 4 Fe3+ + 3 H2O = sulfite + 4 Fe2+ + glutathione + 6 H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "hydrogen sulfide:ferric ion oxidoreductase" EXACT [MetaCyc:SULFFEOXIDO-RXN]
-xref: MetaCyc:SULFFEOXIDO-RXN
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0070227
@@ -392079,6 +392895,8 @@
 def: "A gasdermin-dependent inflammatory response that is associated with the generation of pyrogenic mediators such as IL-1beta and IL-18. Gasdermins are activated by caspase-1 or caspase-4/11, or by certain granzymes. In some, but not all cells, it can lead to pyroptotic programmed cell death." [PMID:18846107, PMID:21760595, PMID:33883744]
 synonym: "pyroptosis" BROAD []
 xref: Reactome:R-HSA-5620971 "Pyroptosis"
+xref: Reactome:R-HSA-9960519 "CASP4-mediated substrate cleavage"
+xref: Reactome:R-HSA-9960525 "CASP5-mediated substrate cleavage"
 is_a: GO:0006954 ! inflammatory response
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27753" xsd:anyURI
 
@@ -392132,7 +392950,7 @@
 id: GO:0070275
 name: aerobic ammonia oxidation to nitrite via pyruvic oxime
 namespace: biological_process
-def: "The metabolic process in which ammonia (NH3) is oxidized to nitrite (NO2) in the presence of oxygen. Hydroxylamine is produced enzymatically, and, in the presence of pyruvate, forms pyruvic oxime in a spontaneous, non-enzymatic reaction; pyruvic oxime is then converted to nitrite." [MetaCyc:PWY-2242]
+def: "The metabolic process in which ammonia (NH4+) is oxidized to nitrite (NO2) in the presence of oxygen. Hydroxylamine is produced enzymatically, and, in the presence of pyruvate, forms pyruvic oxime in a spontaneous, non-enzymatic reaction; pyruvic oxime is then converted to nitrite." [MetaCyc:PWY-2242]
 xref: MetaCyc:PWY-2242
 is_a: GO:0019329 ! ammonia oxidation
 
@@ -394756,6 +395574,7 @@
 name: 11-beta-hydroxysteroid dehydrogenase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an 11-beta-hydroxysteroid + NAD+ = an 11-oxosteroid + NADH + H+." [PMID:15761036, RHEA:53116]
+xref: Reactome:R-HSA-194023 "HSD11B2,HSD11B1 dimer oxidise CORT to COR"
 xref: RHEA:53116
 xref: Wikipedia:11beta-hydroxysteroid_dehydrogenase
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
@@ -394817,10 +395636,14 @@
 
 [Term]
 id: GO:0070529
-name: L-tryptophan aminotransferase activity
+name: obsolete L-tryptophan transaminase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of an amino group from L-tryptophan to an acceptor, usually a 2-oxo acid." [GOC:mah]
-is_a: GO:0008483 ! transaminase activity
+def: "OBSOLETE. Catalysis of the transfer of an amino group from L-tryptophan to an acceptor, usually a 2-oxo acid." [GOC:mah]
+comment: This term was obsoleted because it is an unnecessary grouping term.
+synonym: "L-tryptophan aminotransferase activity" EXACT []
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0047299
 
 [Term]
 id: GO:0070530
@@ -394972,10 +395795,14 @@
 
 [Term]
 id: GO:0070548
-name: L-glutamine aminotransferase activity
+name: obsolete L-glutamine transaminase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of an amino group from L-glutamine to an acceptor, usually a 2-oxo acid." [GOC:mah]
-is_a: GO:0008483 ! transaminase activity
+def: "OBSOLETE. Catalysis of the transfer of an amino group from L-glutamine to an acceptor, usually a 2-oxo acid." [GOC:mah]
+comment: This term was obsoleted because it is too general. Consider other glutamine transaminase activity terms.
+synonym: "L-glutamine aminotransferase activity" EXACT []
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+consider: GO:0047945
 
 [Term]
 id: GO:0070549
@@ -395173,6 +396000,7 @@
 name: adenylyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an adenylyl group to an acceptor." [GOC:mah]
+xref: Reactome:R-HSA-9935837 "The TRAMP:RNA exosome oligoadenylates pre-rRNA 5' external transcribed spacer (5'ETS) fragment"
 is_a: GO:0016779 ! nucleotidyltransferase activity
 created_by: mah
 creation_date: 2009-04-15T11:04:22Z
@@ -395258,9 +396086,9 @@
 def: "Catalysis of the hydrolysis of a dipeptide by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "metallo-exo-dipeptidase activity" EXACT [GOC:rb]
 synonym: "metalloexodipeptidase activity" EXACT [GOC:mah]
-xref: Reactome:R-HSA-2022398 "ACE:Zn2+ hydrolyzes Angiotensin-(1-9) to Angiotensin-(1-7)"
-xref: Reactome:R-HSA-2022405 "ACE:Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
-xref: Reactome:R-HSA-2065355 "Secreted ACE:Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022398 "ACE:Zn2+ hydrolyzes AGT(25-33) (Angiotensin-(1-9)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2022405 "ACE:Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
+xref: Reactome:R-HSA-2065355 "Secreted ACE:Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 is_a: GO:0008235 ! metalloexopeptidase activity
 is_a: GO:0016805 ! dipeptidase activity
 created_by: mah
@@ -395465,6 +396293,7 @@
 synonym: "calcium ion membrane transport" EXACT []
 synonym: "transmembrane calcium transport" EXACT [GOC:mah]
 xref: Reactome:R-HSA-3295583 "TRP channels"
+xref: Reactome:R-HSA-425561 "Sodium/Calcium exchangers"
 is_a: GO:0006816 ! calcium ion transport
 is_a: GO:0098655 ! monoatomic cation transmembrane transport
 created_by: mah
@@ -396609,9 +397438,10 @@
 
 [Term]
 id: GO:0070677
-name: rRNA (cytosine-2'-O-)-methyltransferase activity
+name: rRNA (cytosine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylcytosine." [GOC:mah, PMID:19400805]
+synonym: "rRNA (cytosine-2'-O-)-methyltransferase activity" EXACT []
 is_a: GO:0016434 ! rRNA (cytosine) methyltransferase activity
 is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
 property_value: skos:broadMatch RHEA:58956
@@ -397678,7 +398508,7 @@
 id: GO:0070773
 name: protein-N-terminal glutamine amidohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-terminal L-glutaminyl-[protein] + H2O = N-terminal L-glutamyl-[protein] + NH3. This reaction is the deamidation of an N-terminal glutamine residue of a protein." [PMID:19560421, RHEA:50680]
+def: "Catalysis of the reaction: N-terminal L-glutaminyl-[protein] + H2O = N-terminal L-glutamyl-[protein] + NH4+. This reaction is the deamidation of an N-terminal glutamine residue of a protein." [PMID:19560421, RHEA:50680]
 synonym: "NtQ-amidase activity" EXACT [PMID:19560421]
 xref: EC:3.5.1.122
 xref: RHEA:50680
@@ -397697,7 +398527,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: a phytoceramide + H2O = a fatty acid + phytosphingosine." [GOC:pde, PMID:11356846]
 comment: The reason for obsoletion is that this term represents a specific substrate of GO:0017040 N-acylsphingosine amidohydrolase activity.
-xref: Reactome:R-HSA-428262 "ACER3 hydrolyzes phytoceramide"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28294" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0017040
@@ -400224,8 +401053,13 @@
 synonym: "L-asparagine biosynthesis" EXACT [GOC:mah]
 synonym: "L-asparagine formation" EXACT [GOC:mah]
 synonym: "L-asparagine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:ASPARAGINE-BIOSYNTHESIS
+xref: MetaCyc:ASPARAGINESYN-PWY
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 is_a: GO:0070982 ! L-asparagine metabolic process
+property_value: skos:narrowMatch MetaCyc:ASPARAGINE-BIOSYNTHESIS
+property_value: skos:narrowMatch MetaCyc:ASPARAGINESYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-10-27T12:59:42Z
 
@@ -403372,6 +404206,7 @@
 is_a: GO:0006555 ! methionine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31068" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:28:15Z
 
@@ -403379,12 +404214,26 @@
 id: GO:0071266
 name: 'de novo' L-methionine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine, the L-enantiomer of (2S)-2-amino-4-(methylsulfanyl)butanoic acid, from simpler components." [GOC:ecd]
+def: "The chemical reactions and pathways resulting in the formation of L-methionine, the L-enantiomer of (2S)-2-amino-4-(methylsulfanyl)butanoic acid, from simpler components such as L-aspartate or L-homoserine. This process occurs in plants, bacteria, archae and fungi." [PMID:12818659, PMID:24939187]
 synonym: "'de novo' L-methionine anabolism" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine biosynthesis" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine formation" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:HOMOSER-METSYN-PWY
+xref: MetaCyc:HSERMETANA-PWY
+xref: MetaCyc:PWY-5345
+xref: MetaCyc:PWY-5347
+xref: MetaCyc:PWY-702
+xref: MetaCyc:PWY-7977
 is_a: GO:0071265 ! L-methionine biosynthetic process
+property_value: skos:narrowMatch MetaCyc:HOMOSER-METSYN-PWY
+property_value: skos:narrowMatch MetaCyc:HSERMETANA-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5345
+property_value: skos:narrowMatch MetaCyc:PWY-5347
+property_value: skos:narrowMatch MetaCyc:PWY-702
+property_value: skos:narrowMatch MetaCyc:PWY-7977
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:34:41Z
 
@@ -403393,9 +404242,18 @@
 name: L-methionine salvage
 namespace: biological_process
 def: "Any process that generates L-methionine from derivatives of it, without de novo synthesis." [GOC:ecd]
+xref: MetaCyc:ADENOSYLHOMOCYSCAT-PWY
+xref: MetaCyc:PWY-4361
+xref: MetaCyc:PWY-7174
+xref: MetaCyc:PWY-8132
 is_a: GO:0043102 ! amino acid salvage
 is_a: GO:0071265 ! L-methionine biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ADENOSYLHOMOCYSCAT-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-4361
+property_value: skos:narrowMatch MetaCyc:PWY-7174
+property_value: skos:narrowMatch MetaCyc:PWY-8132
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29900" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:36:49Z
 
@@ -403419,16 +404277,18 @@
 id: GO:0071269
 name: L-homocysteine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-homocysteine, the L-enantiomer of 2-amino-4-sulfanylbutanoic acid." [GOC:ecd, GOC:mah]
+def: "The chemical reactions and pathways resulting in the formation of L-homocysteine." [GOC:curators]
 synonym: "L-homocysteine anabolism" EXACT [GOC:mah]
 synonym: "L-homocysteine biosynthesis" EXACT [GOC:mah]
 synonym: "L-homocysteine formation" EXACT [GOC:mah]
 synonym: "L-homocysteine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:PWY-5344
 is_a: GO:0000097 ! sulfur amino acid biosynthetic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0050667 ! homocysteine metabolic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T05:36:37Z
 
@@ -406389,7 +407249,7 @@
 id: GO:0071522
 name: ureidoglycine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ureidoglycine + H2O = S-ureidoglycolate + NH3." [MetaCyc:URUR-RXN, PMID:19935661, PMID:20038185]
+def: "Catalysis of the reaction: (S)-2-ureidoglycine + H2O = (S)-ureidoglycolate + NH4+." [PMID:19935661, PMID:20038185, RHEA:25241]
 xref: EC:3.5.3.26
 xref: MetaCyc:URUR-RXN
 xref: RHEA:25241
@@ -406414,15 +407274,17 @@
 id: GO:0071524
 name: pyrrolysine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrrolysine, N6-{[(2R,3R)-3-methyl-3,4-dihydro-2H-pyrrol-2-yl]carbonyl}-L-lysine." [GOC:dh, PMID:17204561]
+def: "The chemical reactions and pathways resulting in the formation of pyrrolysine." [GOC:curators, PMID:17204561]
 synonym: "monomethylamine methyltransferase cofactor lysine adduct biosynthetic process" EXACT [CHEBI:21860]
 synonym: "pyrrolysine anabolism" EXACT [GOC:mah]
 synonym: "pyrrolysine biosynthesis" EXACT [GOC:mah]
 synonym: "pyrrolysine formation" EXACT [GOC:mah]
 synonym: "pyrrolysine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:PWY-6994
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0042398 ! modified amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: mah
 creation_date: 2010-01-07T02:40:24Z
 
@@ -411418,6 +412280,7 @@
 def: "Binding to the oxidized form, FAD, of flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [GOC:mah]
 synonym: "oxidized flavin adenine dinucleotide binding" EXACT [GOC:mah]
 synonym: "oxidized flavine-adenine dinucleotide binding" EXACT [GOC:mah]
+xref: Reactome:R-HSA-9854672 "FAD attaches to SDHA"
 is_a: GO:0050660 ! flavin adenine dinucleotide binding
 created_by: mah
 creation_date: 2010-10-07T12:30:28Z
@@ -415491,7 +416354,7 @@
 def: "The directed movement of proteins from the plasma membrane across the periplasmic space to the outer membrane or cell wall." [GOC:mah, PMID:20378773]
 synonym: "chaperone-mediated protein transport across periplasmic space" NARROW []
 is_a: GO:0015031 ! protein transport
-relationship: has_part GO:0140597 ! protein carrier chaperone
+relationship: has_part GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 created_by: mah
 creation_date: 2010-10-28T04:24:59Z
@@ -415735,6 +416598,7 @@
 name: rescue of stalled cytosolic ribosome
 namespace: biological_process
 def: "A process of cytosolic translational elongation that takes place when a cytosolic ribosome has stalled during translation, and results in freeing the ribosome from the stalled translation complex." [GOC:jh2, GOC:mah, PMID:18557701, PMID:19170872, PMID:20117091, PMID:20185543]
+xref: Reactome:R-HSA-9948299 "Ribosome-associated quality control"
 is_a: GO:0002182 ! cytoplasmic translational elongation
 is_a: GO:0032790 ! ribosome disassembly
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22227" xsd:anyURI
@@ -417044,6 +417908,7 @@
 synonym: "ammonia transport" BROAD []
 synonym: "ammonium membrane transport" EXACT []
 synonym: "ammonium transport" BROAD []
+xref: Reactome:R-HSA-444411 "Rhesus glycoproteins mediate ammonium transport"
 is_a: GO:0055085 ! transmembrane transport
 is_a: GO:0071705 ! nitrogen compound transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22228" xsd:anyURI
@@ -418059,7 +418924,6 @@
 namespace: molecular_function
 def: "Binds to and modulates the activity of ATP-hydrolyzing DNA topoisomerase. DNA topoisomerase (ATP-hydrolyzing) regulator activity catalyzes a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:mah]
 is_a: GO:0030234 ! enzyme regulator activity
-is_a: GO:0060590 ! ATPase regulator activity
 relationship: regulates GO:0003918 ! DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity
 created_by: mah
 creation_date: 2011-02-09T05:31:50Z
@@ -423306,12 +424170,14 @@
 id: GO:0080002
 name: UDP-glucose:4-aminobenzoate acylglucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-aminobenzoate + UDP-alpha-D-glucose = 1-O-(4-aminobenzoyl)-beta-D-glucose + UDP." [PMID:18385129]
+def: "Catalysis of the reaction: 4-aminobenzoate + UDP-alpha-D-glucose = 1-O-(4-aminobenzoyl)-beta-D-glucose + UDP." [PMID:18385129, RHEA:85851]
 synonym: "UDP-glucose:p-aminobenzoate acylglucosyltransferase activity" RELATED []
 synonym: "UDP-glucose:p-aminobenzoate glucosyltransferase activity" RELATED []
 synonym: "UDP-glucose:pABA acylglucosyltransferase activity" RELATED []
 xref: MetaCyc:RXN-6142
+xref: RHEA:85851
 is_a: GO:0035251 ! UDP-glucosyltransferase activity
+property_value: skos:exactMatch RHEA:85851
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28273" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28775" xsd:anyURI
 
@@ -423923,10 +424789,11 @@
 id: GO:0080061
 name: indole-3-acetonitrile nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: indole-3-acetonitrile + 2 H2O = indole-3-acetic acid + NH3." [RHEA:45776]
+def: "Catalysis of the reaction: (indol-3-yl)acetonitrile + 2 H2O = (indol-3-yl)acetate + NH4+." [RHEA:45776]
 xref: MetaCyc:RXN-1404
 xref: RHEA:45776
 is_a: GO:0000257 ! nitrilase activity
+property_value: skos:broadMatch EC:3.5.5.1
 property_value: skos:exactMatch RHEA:45776
 created_by: dhl
 creation_date: 2009-04-13T03:30:42Z
@@ -424195,10 +425062,12 @@
 name: regulation of primary metabolic process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism involving those compounds formed as a part of the normal anabolic and catabolic processes. These processes take place in most, if not all, cells of the organism." [PMID:19211694]
+subset: gocheck_do_not_annotate
 is_a: GO:0019222 ! regulation of metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044238 ! primary metabolic process
 relationship: regulates GO:0044238 ! primary metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-22T04:30:52Z
 
@@ -424271,40 +425140,48 @@
 
 [Term]
 id: GO:0080097
-name: L-tryptophan:pyruvate aminotransferase activity
+name: L-tryptophan:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + pyruvate = 3-(indol-3-yl)pyruvate + L-alanine." [RHEA:27586]
+def: "Catalysis of the reaction: L-tryptophan + pyruvate = indole-3-pyruvate + L-alanine." [RHEA:27586]
+synonym: "L-tryptophan:pyruvate aminotransferase activity" EXACT []
 xref: EC:2.6.1.99
 xref: MetaCyc:RXN-10139
 xref: RHEA:27586
-is_a: GO:0070529 ! L-tryptophan aminotransferase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.99
 property_value: skos:exactMatch RHEA:27586
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:45:23Z
 
 [Term]
 id: GO:0080098
-name: L-tyrosine-pyruvate transaminase activity
+name: L-tyrosine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tyrosine + pyruvate = (4-hydroxyphenyl)pyruvate + L-alanine." [MetaCyc:RXN3O-4157]
+def: "Catalysis of the reaction: L-tyrosine + pyruvate = 3-(4-hydroxyphenyl)pyruvate + L-alanine." [MetaCyc:RXN3O-4157]
 synonym: "L-tyrosine aminotransferase activity" BROAD []
+synonym: "L-tyrosine-pyruvate transaminase activity" EXACT []
 synonym: "L-tyrosine:pyruvate aminotransferase activity" EXACT []
 xref: MetaCyc:RXN3O-4157
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:broadMatch EC:2.6.1.58
+property_value: skos:exactMatch MetaCyc:RXN3O-4157
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:47:53Z
 
 [Term]
 id: GO:0080099
-name: L-methionine:2-oxoglutarate aminotransferase activity
+name: L-methionine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine + 2-oxoglutarate = 4-methylthio-2-oxobutyrate + L-glutamate." [PMID:18394996]
-comment: This reaction falls within the larger set of reactions associated with EC:2.6.1.5 (See BRENDA: http://www.brenda-enzymes.org/php/result_flat.php4?ecno=2.6.1.5).
+def: "Catalysis of the reaction: L-methionine + 2-oxoglutarate = 4-methylsulfanyl-2-oxobutanoate + L-glutamate." [PMID:18394996, RHEA:25796]
+synonym: "L-methionine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-methionine:alpha-ketoglutarate aminotransferase activity" EXACT []
-is_a: GO:0010326 ! methionine-oxo-acid transaminase activity
+xref: MetaCyc:RXN-14147
+xref: RHEA:25796
+is_a: GO:0010326 ! L-methionine:oxo-acid transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:51:20Z
 
@@ -424654,20 +425531,21 @@
 
 [Term]
 id: GO:0080130
-name: L-phenylalanine-2-oxoglutarate transaminase activity
+name: L-phenylalanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-phenylalanine + 2-oxoglutarate = 3-phenylpyruvate + L-glutamate." [PMID:18394996, RHEA:25152]
+synonym: "L-phenylalanine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "L-phenylalanine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-phenylalanine:alpha-ketoglutarate aminotransferase activity" EXACT []
 xref: KEGG_REACTION:R00694
 xref: MetaCyc:RXN-10814
 xref: RHEA:25152
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:broadMatch EC:2.6.1.5
-property_value: skos:broadMatch EC:2.6.1.57
 property_value: skos:exactMatch RHEA:25152
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31139" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-29T03:42:47Z
 
@@ -424847,8 +425725,6 @@
 def: "Catalysis of the reaction: L-cysteine + H2O = hydrogen sulfide + pyruvate + NH4+ + H+." [PMID:19955263, RHEA:24931]
 xref: EC:4.4.1.28
 xref: MetaCyc:LCYSDESULF-RXN
-xref: Reactome:R-HSA-1614591 "Excess cysteine yields lanthionine and H2S"
-xref: Reactome:R-HSA-1614614 "Cysteine is degraded to serine and H2S"
 xref: RHEA:24931
 is_a: GO:0016846 ! carbon-sulfur lyase activity
 property_value: skos:broadMatch EC:4.4.1.1
@@ -425125,7 +426001,7 @@
 id: GO:0080170
 name: hydrogen peroxide transmembrane transport
 namespace: biological_process
-def: "The process in which hydrogen peroxide is transported across a membrane." [GOC:tb]
+def: "The process in which H2O2 is transported across a membrane." [GOC:tb]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "hydrogen peroxide membrane transport" EXACT []
 is_a: GO:0055085 ! transmembrane transport
@@ -431720,6 +432596,7 @@
 name: regulation of ornithine metabolic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency, or extent of the chemical reactions and pathways involving ornithine, an amino acid only rarely found in proteins, but which is important in living organisms as an intermediate in the reactions of the urea cycle and in arginine biosynthesis." [GOC:dph, GOC:jp, GOC:tb]
+xref: Reactome:R-HSA-350562 "Regulation of ornithine decarboxylase (ODC)"
 is_a: GO:0006521 ! regulation of amino acid metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -432280,7 +433157,7 @@
 name: nicotinate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of nicotinate from one side of a membrane to the other." [GOC:tb]
-xref: Reactome:R-HSA-8869603 "SLC22A13 transports NCA from extracellular region to cytosol"
+xref: Reactome:R-HSA-8869603 "SLC22A13 exchanges extracellular NCA for cytosolic urate"
 is_a: GO:0008028 ! monocarboxylic acid transmembrane transporter activity
 relationship: part_of GO:2001142 ! nicotinate transport
 created_by: tb
@@ -432474,7 +433351,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: ATP + palmitic acid + CoA = AMP + diphosphate + palmitoyl-CoA." [GOC:al, PMID:18071249]
 synonym: "palmitoyl-CoA synthetase activity" EXACT []
-xref: Reactome:R-HSA-434382 "ACSL3,4 ligates coenzyme A (CoA-SH) to palmitate yielding palmitoyl-coenzyme A in the pancreatic beta cell"
+xref: Reactome:R-HSA-434382 "ACSL3,4 ligates coenzyme A (CoA-SH) to a long chain fatty acid yielding fatty acyl-coenzyme A"
 is_a: GO:0004467 ! long-chain fatty acid-CoA ligase activity
 created_by: tb
 creation_date: 2012-07-25T03:31:31Z
@@ -432941,10 +433818,11 @@
 
 [Term]
 id: GO:0090486
-name: small RNA 2'-O-methyltransferase activity
+name: small RNA 2'-O-ribose methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a nucleoside residue in a small RNA molecule. Reaction: S-adenosyl-L-methionine + small RNA = S-adenosyl-L-homocysteine + small RNA containing a 3'-terminal 2'-O-methylnucleotide." [GOC:tb, GOC:vw, RHEA:37887]
 synonym: "small RNA 2'-O-methyltransferase" EXACT []
+synonym: "small RNA 2'-O-methyltransferase activity" EXACT []
 xref: EC:2.1.1.386
 xref: RHEA:37887
 is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
@@ -433644,7 +434522,7 @@
 synonym: "phosphatidylcholine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "phosphatidylcholine-translocating ATPase activity" BROAD []
 xref: RHEA:38583
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+is_a: GO:0008525 ! phosphatidylcholine intramembrane carrier activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch RHEA:38583
@@ -433678,7 +434556,6 @@
 synonym: "phosphatidylserine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "phosphatidylserine-translocating ATPase activity" BROAD []
 xref: RHEA:38567
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch RHEA:38567
@@ -433744,7 +434621,8 @@
 name: protein-N(PI)-phosphohistidine-N,N'-diacetylchitobiose phosphotransferase system transporter activity
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N,N'-diacetylchitobiose(out) = protein histidine + N,N'-diacetylchitobiose phosphate(in)." [GOC:am, PMID:10913119]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 created_by: tb
 creation_date: 2014-04-08T14:55:02Z
 
@@ -433995,10 +434873,13 @@
 
 [Term]
 id: GO:0090590
-name: protein-N(PI)-phosphohistidine-D-glucosamine phosphotransferase system transporter activity
+name: obsolete protein-N(PI)-phosphohistidine-D-glucosamine phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + D-glucosamine(out) = protein histidine + glucosamine-6-phosphate(in)." [PMID:8246840]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+def: "OBSOLETE. Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + D-glucosamine(out) = protein histidine + glucosamine-6-phosphate(in)." [PMID:8246840]
+comment: This term was obsoleted because it represents a specific substrate of protein-N(PI)-phosphohistidine-mannose phosphotransferase system transporter activity ; GO:0022870 (see PMID:8246840).
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0022870
 created_by: tb
 creation_date: 2014-06-04T16:33:01Z
 
@@ -434007,7 +434888,8 @@
 name: protein-N(PI)-phosphohistidine-N-acetyl-mannosamine phosphotransferase system transporter activity
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetyl-mannosamine(out) = protein histidine +N-acetyl- mannosamine-6-phosphate(in)." [PMID:9864311]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 created_by: tb
 creation_date: 2014-06-04T16:33:01Z
 
@@ -434048,7 +434930,7 @@
 id: GO:0090595
 name: L-lysine N6-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acetyl-CoA + L-lysine = CoA + H+ + N(6)-acetyl-L-lysine." [RHEA:69476]
+def: "Catalysis of the reaction: acetyl-CoA + L-lysine = CoA + H+ + N6-acetyl-L-lysine." [RHEA:69476]
 synonym: "acetyl-CoA:L-lysine N6-acetyltransferase" EXACT []
 xref: EC:2.3.1.306
 xref: MetaCyc:LYSACET-RXN
@@ -434208,7 +435090,7 @@
 id: GO:0090612
 name: cAMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclic adenosine monophosphate + H2O = cyclic inosine monophosphate + NH3." [PMID:24074367, RHEA:22908]
+def: "Catalysis of the reaction: 3',5'-cyclic AMP + H2O + H+ = 3',5'-cyclic IMP + NH4+." [PMID:24074367, RHEA:22908]
 synonym: "cyclic adenosine monophosphate deaminase activity" EXACT []
 xref: EC:3.5.4.46
 xref: RHEA:22908
@@ -434225,7 +435107,7 @@
 id: GO:0090613
 name: 5'-deoxyadenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5'deoxyadenosine + H2O = 5'deoxyinosine + NH3." [PMID:23968233, RHEA:42892]
+def: "Catalysis of the reaction: 5'-deoxyadenosine + H2O + H+ = 5'-deoxyinosine + NH4+." [PMID:23968233, RHEA:42892]
 xref: EC:3.5.4.41
 xref: RHEA:42892
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
@@ -434240,7 +435122,7 @@
 id: GO:0090614
 name: 5'-methylthioadenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5'methyl thioadenosine + H2O = 5'methyl thioinosine + NH3." [PMID:23968233, RHEA:25025]
+def: "Catalysis of the reaction: S-methyl-5'-thioadenosine + H2O + H+ = S-methyl-5'-thioinosine + NH4+." [PMID:23968233, RHEA:25025]
 xref: EC:3.5.4.31
 xref: RHEA:25025
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
@@ -436163,10 +437045,21 @@
 synonym: "L-glutamate biosynthesis" EXACT []
 synonym: "L-glutamate formation" EXACT []
 synonym: "L-glutamate synthesis" EXACT []
+xref: MetaCyc:GLUGLNSYN-PWY
+xref: MetaCyc:GLUTAMATE-SYN2-PWY
+xref: MetaCyc:GLUTSYN-PWY
+xref: MetaCyc:GLUTSYNIII-PWY
+xref: MetaCyc:PWY-4341
 is_a: GO:0006536 ! glutamate metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
 is_a: GO:0043650 ! dicarboxylic acid biosynthetic process
 property_value: skos:broadMatch MetaCyc:PWY-5505
+property_value: skos:narrowMatch MetaCyc:GLUGLNSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTAMATE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTSYNIII-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-4341
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pr
 creation_date: 2011-05-24T04:30:38Z
 
@@ -437217,7 +438110,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "caspase activity" BROAD []
 synonym: "metacaspase activity" NARROW [GOC:mtg_apoptosis]
-xref: Reactome:R-HSA-6800797 "The PIDDosome activates CASP2"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004197
@@ -437279,7 +438171,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Binding to an organic cyclic compound, any molecular entity that contains carbon arranged in a cyclic molecular structure." [GOC:sjw, PMID:7583672]
 comment: The reason for obsoletion is that this term was an unnecessary grouping term.
-xref: Reactome:R-HSA-1497796 "BH2 binding can lead to eNOS uncoupling"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29143" xsd:anyURI
 is_obsolete: true
 created_by: pr
@@ -437647,6 +438538,7 @@
 comment: This term can be used to annotate gene products involved in apoptotic events happening downstream of the cross-talk point between the extrinsic and intrinsic apoptotic pathways. The cross-talk starts when caspase-8 cleaves Bid and truncated Bid interacts with mitochondria. From this point on it is not possible to distinguish between extrinsic and intrinsic pathways.
 synonym: "apoptotic signalling pathway" EXACT [GOC:mah]
 synonym: "induction of apoptosis by extracellular signals" EXACT []
+xref: Reactome:R-HSA-111459 "Activation of caspases through apoptosome-mediated cleavage"
 xref: Reactome:R-HSA-193681 "Ceramide signalling"
 xref: Reactome:R-HSA-204998 "Cell death signalling via NRAGE, NRIF and NADE"
 xref: Reactome:R-HSA-75157 "FasL/ CD95L signaling"
@@ -438450,7 +439342,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: leukotriene A4 = eoxin A4." [GOC:mw, PMID:18184802, PMID:18647347]
 comment: The reason for obsoletion is that this term is equivalent to leukotriene-C4 synthase activity.
-xref: Reactome:R-HSA-2162019 "LTA4 is converted to EXA4 by ALOX15"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29695" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004464
@@ -438877,7 +439768,6 @@
 comment: Components of the DISC (death-inducing signaling complex) may be annotated to this term. These include ligand-bound receptors such as FAS/CD95 (though care should be taken because FAS can also act as a non-apoptotic signal transducer), and signaling molecules such as FADD (FAS-associated protein with a death domain), cIAPs (cellular inhibitor of apoptosis proteins), c-FLIPs and caspases 8 and 10. Note that GO:0097296 should only be used to annotate gene products directly involved in the initiation or start up of an inactive cysteine-type endopeptidase activity involved in the apoptotic signaling pathway (also known as an initiator caspase). If the evidence provided is not sufficient to account for direct activation of this enzymatic activity, consider using the more generic terms GO:2001269 positive regulation of cysteine-type endopeptidase activity involved in apoptotic signaling pathway or GO:0043280 positive regulation of cysteine-type endopeptidase activity involved in apoptotic process.\nThe reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "activation of cysteine-type endopeptidase activity involved in apoptotic signalling pathway" EXACT []
 synonym: "activation of initiator caspase activity" NARROW []
-xref: Reactome:R-HSA-140534 "Caspase activation via Death Receptors in the presence of ligand"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 created_by: pr
@@ -439561,6 +440451,7 @@
 synonym: "protein O-GlcNAc transferase activity" EXACT []
 synonym: "UDP-N-acetyl-D-glucosamine:protein-O-beta-N-acetyl-D-glucosaminyl transferase" EXACT []
 xref: EC:2.4.1.255
+xref: Reactome:R-HSA-8879117 "POMGNT2 transfers GlcNAc to Man-DAG1"
 xref: Reactome:R-HSA-9687828 "O-GlcNAcylation of RIPK3 (TLR4 signaling)"
 xref: RHEA:48904
 xref: RHEA:48908
@@ -443245,7 +444136,7 @@
 id: GO:0097697
 name: tRNA (5-carboxymethoxyuridine(34)-5-O)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-carboxymethoxyuridine34 in tRNA + S-adenosyl-L-methionine = 5-methoxycarbonylmethoxyuridine34 in tRNA + S-adenosyl-L-homocysteine." [PMID:26681692, RHEA:54080]
+def: "Catalysis of the reaction: 5-carboxymethoxyuridine34 in tRNA + S-adenosyl-L-methionine = 5-methoxycarbonylmethoxyuridine34 in tRNA + S-adenosyl-L-homocysteine. The methylation occurs on the modified base." [PMID:26681692, RHEA:54080]
 synonym: "tRNA 5-carboxymethoxyuridine methyltransferase activity" RELATED []
 xref: RHEA:54080
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -444697,10 +445588,11 @@
 id: GO:0098538
 name: lumenal side of transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the transport vesicle membrane that faces the lumen." [GOC:ab]
+def: "The leaflet of a transport vesicle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "internal side of transport vesicle membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0030658 ! transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:43:55Z
 
@@ -444708,10 +445600,11 @@
 id: GO:0098539
 name: cytoplasmic side of transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the transport vesicle membrane that faces the cytoplasm." [GOC:ab]
+def: "The leaflet of the transport vesicle membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "external side of transport vesicle membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0030658 ! transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:48:07Z
 
@@ -444719,10 +445612,11 @@
 id: GO:0098540
 name: lumenal side of trans-Golgi network transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the lumen." [GOC:ab]
+def: "The leaflet of a trans-Golgi network transport vesicle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "internal side of trans-Golgi network transport vesicle membrane" RELATED []
 is_a: GO:0098538 ! lumenal side of transport vesicle membrane
 relationship: part_of GO:0012510 ! trans-Golgi network transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:55:02Z
 
@@ -444730,10 +445624,11 @@
 id: GO:0098541
 name: cytoplasmic side of trans-Golgi network transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the cytoplasm." [GOC:ab]
+def: "The leaflet of the trans-Golgi network transport vesicle membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "external side of trans-Golgi network transport vesicle membrane" EXACT []
 is_a: GO:0098539 ! cytoplasmic side of transport vesicle membrane
 relationship: part_of GO:0012510 ! trans-Golgi network transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:56:30Z
 
@@ -444793,9 +445688,10 @@
 id: GO:0098547
 name: lumenal side of Golgi membrane
 namespace: cellular_component
-def: "The side of the Golgi membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the Golgi membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0000139 ! Golgi membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T10:44:23Z
 
@@ -444803,9 +445699,10 @@
 id: GO:0098548
 name: cytoplasmic side of Golgi membrane
 namespace: cellular_component
-def: "The side (leaflet) of the Golgi membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the Golgi membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0000139 ! Golgi membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T10:49:33Z
 
@@ -444820,27 +445717,29 @@
 id: GO:0098550
 name: lumenal side of early endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the early endosome membrane that faces the lumen." [GOC:lr]
+def: "The leaflet of the early endosome membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "internal leaflet of early endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of early endosome membrane" EXACT []
 is_a: GO:0098565 ! lumenal side of endosome membrane
 relationship: part_of GO:0031901 ! early endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098551
 name: lumenal side of late endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the late endosome membrane that faces the lumen." [GOC:lr]
+def: "The leaflet of the late endosomal membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "internal leaflet of late endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of late endosome membrane" EXACT []
 is_a: GO:0098565 ! lumenal side of endosome membrane
 relationship: part_of GO:0031902 ! late endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098552
 name: side of membrane
 namespace: cellular_component
-def: "A cellular component consisting of one leaflet of a membrane bilayer and any proteins embedded or anchored in it or attached to its surface." [GOC:dos]
+def: "A cellular component consisting of one leaflet of a membrane bilayer and any protein embedded or anchored in it or attached to its surface." [GOC:dos]
 is_a: GO:0110165 ! cellular anatomical structure
 relationship: has_part GO:0097478 ! leaflet of membrane bilayer
 relationship: part_of GO:0016020 ! membrane
@@ -444851,9 +445750,10 @@
 id: GO:0098553
 name: lumenal side of endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plasma membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005789 ! endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:08:37Z
 
@@ -444861,9 +445761,10 @@
 id: GO:0098554
 name: cytoplasmic side of endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005789 ! endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:09:47Z
 
@@ -444871,9 +445772,10 @@
 id: GO:0098555
 name: lumenal side of rough endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the rough endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098553 ! lumenal side of endoplasmic reticulum membrane
 relationship: part_of GO:0030867 ! rough endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:13:24Z
 
@@ -444881,9 +445783,10 @@
 id: GO:0098556
 name: cytoplasmic side of rough endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the rough endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098554 ! cytoplasmic side of endoplasmic reticulum membrane
 relationship: part_of GO:0030867 ! rough endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:15:44Z
 
@@ -444891,9 +445794,10 @@
 id: GO:0098557
 name: cytoplasmic side of smooth endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the smooth endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098554 ! cytoplasmic side of endoplasmic reticulum membrane
 relationship: part_of GO:0030868 ! smooth endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:17:58Z
 
@@ -444901,9 +445805,10 @@
 id: GO:0098558
 name: lumenal side of smooth endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the smooth endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098553 ! lumenal side of endoplasmic reticulum membrane
 relationship: part_of GO:0030868 ! smooth endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:19:28Z
 
@@ -444911,23 +445816,25 @@
 id: GO:0098559
 name: cytoplasmic side of early endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the early endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the early endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of early endosome membrane" EXACT [GOC:ab]
 synonym: "external side of early endosome membrane" EXACT []
 is_a: GO:0010009 ! cytoplasmic side of endosome membrane
 relationship: part_of GO:0031901 ! early endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098560
 name: cytoplasmic side of late endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the late endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the late endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of late endosome membrane" EXACT [GOC:ab]
 synonym: "external side of late endosome membrane" EXACT []
 is_a: GO:0010009 ! cytoplasmic side of endosome membrane
 relationship: part_of GO:0031902 ! late endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098561
@@ -444941,8 +445848,9 @@
 id: GO:0098562
 name: cytoplasmic side of membrane
 namespace: cellular_component
-def: "The side of a membrane that faces the cytoplasm." [GOC:dos]
+def: "The leaflet of a membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-12T11:21:30Z
 
@@ -444969,11 +445877,12 @@
 id: GO:0098565
 name: lumenal side of endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the endosome membrane that faces the lumen." [GOC:dos]
+def: "The leaflet of the endosome membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 synonym: "internal leaflet of endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of endosome membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0010008 ! endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098566
@@ -444987,8 +445896,9 @@
 id: GO:0098567
 name: periplasmic side of plasma membrane
 namespace: cellular_component
-def: "The side (leaflet) of a plasma membrane that faces the periplasm, and all proteins embedded in it or attached to its surface." [GOC:dos]
+def: "The leaflet of the plasma membrane that faces the periplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0009897 ! external side of plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-19T09:31:30Z
 
@@ -444996,39 +445906,44 @@
 id: GO:0098568
 name: external side of mycolate outer membrane
 namespace: cellular_component
-def: "The side (leaflet) of the mycolate outer membrane that faces the environment and any proteins embedded in it or loosely bound to its surface." [GOC:dos, PMID:18316738, PMID:18567661]
+def: "The leaflet of the mycolate outer membrane that faces the periplasm of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, PMID:18316738, PMID:18567661]
 is_a: GO:0031240 ! external side of cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098569
 name: internal side of mycolate outer membrane
 namespace: cellular_component
-def: "The side of the mycolate outer membrane that faces the cell wall peptidoglycan. It is rich in long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network." [GOC:dos, PMID:18316738, PMID:18567661]
+def: "The leaflet of the mycolate outer membrane that faces the cell wall peptidoglycan, including any protein embedded in, attached to, or peripherally associated with it. It is rich in long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network." [GOC:dos, PMID:18316738, PMID:18567661]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098570
 name: stromal side of plastid inner membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid inner membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of the plastid inner membrane that faces the stroma, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009528 ! plastid inner membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098571
 name: lumenal side of plastid thylakoid membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid thylakoid membrane that faces the lumen, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of a plastid thylakoid membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0055035 ! plastid thylakoid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098572
 name: stromal side of plastid thylakoid membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid thylakoid membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of the plastid thylakoid membrane that faces the stroma, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0055035 ! plastid thylakoid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098573
@@ -445045,28 +445960,31 @@
 id: GO:0098574
 name: cytoplasmic side of lysosomal membrane
 namespace: cellular_component
-def: "The side (leaflet) of the lysosomal membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the lysosomal membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 synonym: "external leaflet of lysosomal membrane" EXACT [GOC:dos]
 synonym: "external side of lysosomal membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005765 ! lysosomal membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098575
 name: lumenal side of lysosomal membrane
 namespace: cellular_component
-def: "The side (leaflet) of the lysosomal membrane that faces the lumen." [GOC:dos]
+def: "The leaflet of the lysosomal membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 synonym: "internal leaflet of lysosomal membrane" EXACT [GOC:ab]
 synonym: "internal side of lysosomal membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005765 ! lysosomal membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098576
 name: lumenal side of membrane
 namespace: cellular_component
-def: "Any side (leaflet) of a membrane that faces the lumen of an organelle." [GOC:dos]
+def: "The leaflet of an organelle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098577
@@ -445191,9 +446109,10 @@
 id: GO:0098591
 name: external side of apical plasma membrane
 namespace: cellular_component
-def: "The leaflet the apical region of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:ab, GOC:dos]
+def: "The leaflet the apical region of the plasma membrane that faces the extracellular side of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0009897 ! external side of plasma membrane
 relationship: part_of GO:0016324 ! apical plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2014-03-06T12:04:33Z
 
@@ -445201,9 +446120,10 @@
 id: GO:0098592
 name: cytoplasmic side of apical plasma membrane
 namespace: cellular_component
-def: "The side (leaflet) of the apical region of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the apical region of the plasma membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0016324 ! apical plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2014-03-06T12:07:32Z
 
@@ -445352,12 +446272,13 @@
 
 [Term]
 id: GO:0098607
-name: methylselenocysteine deselenhydrase activity
+name: obsolete methylselenocysteine deselenhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: Se-Methyl-L-selenocysteine + H2O = pyruvic acid + NH3 + Methaneselenol." [PMID:17451884, PMID:20383543]
+def: "OBSOLETE. Catalysis of the reaction: Se-Methyl-L-selenocysteine + H2O = pyruvic acid + NH3 + Methaneselenol." [PMID:17451884, PMID:20383543]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: Reactome:R-HSA-2408539 "MeSec is hydrolysed to MeSeH by PXLP-K212-CTH"
-is_a: GO:0016846 ! carbon-sulfur lyase activity
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30815" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 created_by: dos
 creation_date: 2014-04-16T11:24:12Z
 
@@ -445694,6 +446615,7 @@
 is_a: GO:0005581 ! collagen trimer
 intersection_of: GO:0005581 ! collagen trimer
 intersection_of: part_of GO:0098645 ! complex of network-forming collagens
+relationship: part_of GO:0031012 ! extracellular matrix
 relationship: part_of GO:0098645 ! complex of network-forming collagens
 
 [Term]
@@ -445717,7 +446639,7 @@
 synonym: "Supramolecular collagen assembly" EXACT [PMID:21421911]
 is_a: GO:0099080 ! supramolecular complex
 relationship: has_part GO:0005581 ! collagen trimer
-relationship: part_of GO:0031012 ! extracellular matrix
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31491" xsd:anyURI
 
 [Term]
 id: GO:0098645
@@ -445730,14 +446652,18 @@
 
 [Term]
 id: GO:0098646
-name: interstitial hexagonal collagen network
+name: hexagonal collagen network
 namespace: cellular_component
-def: "A hexagonal shaped collagen network formed by collagen type X trimer, mostly found in the cartilage hyaline matrix, which located in the interstitial area." [PMID:21421911, PMID:34948124]
+def: "A hexagonal-shaped collagen network formed by collagen type X trimer or collagen type VIII trimers. Collagen type X trimers are mostly found in the cartilage hyaline matrix, which is located in the interstitial area. Note that collagen type VIII has been found in the basement membrane, but the localization is not definitive." [PMID:21421911, PMID:3047147, PMID:31387942, PMID:34948124]
+synonym: "basement membrane hexagonal collagen network" NARROW []
 synonym: "collagen hexagonal network" EXACT [PMID:21421911]
+synonym: "hexagonal collagen network of basement membrane" NARROW []
+synonym: "interstitial hexagonal collagen network" NARROW []
 is_a: GO:0098645 ! complex of network-forming collagens
 relationship: part_of GO:0140152 ! collagenous component of interstitial matrix
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29415" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30146" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31494" xsd:anyURI
 
 [Term]
 id: GO:0098647
@@ -446759,6 +447685,7 @@
 name: 1-phosphatidylinositol 4-kinase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of 1-phosphatidylinositol 4-kinase." [PMID:21288895]
+xref: Reactome:R-HSA-1676152 "PI4KB binds to ARF1/3:GTP at the Golgi membrane"
 is_a: GO:0019209 ! kinase activator activity
 relationship: positively_regulates GO:0004430 ! 1-phosphatidylinositol 4-kinase activity
 
@@ -450289,6 +451216,7 @@
 name: potassium channel activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a potassium channel, resulting in its opening." [GOC:dos]
+xref: Reactome:R-HSA-400063 "Low conductance potassium channels in pancreatic beta cells open in response to epinephrine"
 is_a: GO:0015459 ! potassium channel regulator activity
 is_a: GO:0099103 ! channel activator activity
 relationship: positively_regulates GO:0005267 ! potassium channel activity
@@ -452296,6 +453224,10 @@
 name: ligand-gated calcium channel activity
 namespace: molecular_function
 def: "Enables the transmembrane transfer of a calcium ions by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:dos]
+xref: Reactome:R-HSA-1168376 "STIM1 oligomerizes"
+xref: Reactome:R-HSA-169680 "IP3 binds to the IP3 receptor, opening the endoplasmic reticulum Ca2+ channel"
+xref: Reactome:R-HSA-169683 "IP3R:I(1,4,5)P3 tetramer transports Ca2+ from ER lumen to cytosol"
+xref: Reactome:R-HSA-4420052 "Decreasing cGMP concentration promotes intracellular Ca2+ release in response to WNT"
 is_a: GO:0005262 ! calcium channel activity
 is_a: GO:0099094 ! ligand-gated monoatomic cation channel activity
 relationship: part_of GO:0019722 ! calcium-mediated signaling
@@ -452412,9 +453344,10 @@
 id: GO:0099617
 name: matrix side of mitochondrial inner membrane
 namespace: cellular_component
-def: "The side (leaflet) of the mitochondrial inner membrane that faces the matrix." [GOC:dos]
+def: "The leaflet of a mitochondrial inner membrane that faces the matrix, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005743 ! mitochondrial inner membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0099618
@@ -452449,11 +453382,12 @@
 
 [Term]
 id: GO:0099620
-name: UDP-4-amino-4-deoxy-L-arabinose aminotransferase activity
+name: UDP-4-amino-4-deoxy-L-arabinose:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-4-amino-4-deoxy-beta-L-arabinose + 2-oxoglutarate = UDP-beta-L-threo-pentopyranos-4-ulose + L-glutamate." [PMID:12429098, PMID:12704196, RHEA:24710]
 synonym: "UDP-(beta-L-threo-pentapyranosyl-4''-ulose diphosphate) aminotransferase" EXACT []
 synonym: "UDP-4-amino-4-deoxy-beta-L-arabinose:2-oxoglutarate aminotransferase" EXACT []
+synonym: "UDP-4-amino-4-deoxy-L-arabinose aminotransferase" BROAD []
 synonym: "UDP-4-amino-4-deoxy-L-arabinose---oxoglutarate aminotransferase" EXACT []
 synonym: "UDP-Ara4O aminotransferase" EXACT []
 synonym: "UDP-L-Ara4N transaminase" EXACT []
@@ -452463,6 +453397,7 @@
 property_value: skos:exactMatch EC:2.6.1.87
 property_value: skos:exactMatch RHEA:24710
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0099621
@@ -453523,6 +454458,7 @@
 comment: There are two main classes of deubiquitinating enzymes: cysteine proteases (i.e., thiol dependent) and metalloproteases.
 synonym: "ubiquitinyl hydrolase activity" EXACT []
 xref: Reactome:R-HSA-9674127 "USP30 deubiquitinates ATM dimer:Ub-p-PEX5"
+xref: Reactome:R-HSA-9929353 "Deubiquitination of CD274"
 is_a: GO:0019783 ! ubiquitin-like protein peptidase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17398" xsd:anyURI
 created_by: mec
@@ -453989,19 +454925,22 @@
 
 [Term]
 id: GO:0102037
-name: 4-nitrotoluene monooxygenase activity
+name: obsolete 4-nitrotoluene monooxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + 4-nitrotoluene + NADH + O2 = 4-nitrobenzyl alcohol + NAD+ + H2O." [GOC:pz]
-xref: MetaCyc:R361-RXN
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + 4-nitrotoluene + NADH + O2 = 4-nitrobenzyl alcohol + NAD+ + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102038
-name: 4-nitrobenzyl alcohol oxidase activity
+name: obsolete 4-nitrobenzyl alcohol oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-nitrobenzyl alcohol + O2 = 4-nitrobenzaldehyde + hydrogen peroxide." [GOC:pz]
+def: "OBSOLETE. Catalysis of the reaction: 4-nitrobenzyl alcohol + O2 = 4-nitrobenzaldehyde + hydrogen peroxide." [GOC:pz]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:R362-RXN
-is_a: GO:0016899 ! oxidoreductase activity, acting on the CH-OH group of donors, oxygen as acceptor
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102039
@@ -454070,11 +455009,12 @@
 
 [Term]
 id: GO:0102044
-name: 3-chlorobenzoate-4,5-oxygenase activity
+name: obsolete 3-chlorobenzoate-4,5-oxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-chlorobenzoate + O2 + a reduced electron acceptor = 3-chlorobenzoate-cis-4,5-diol + an oxidized electron acceptor." [GOC:pz]
-xref: MetaCyc:RXN-10421
-is_a: GO:0016708 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of two atoms of oxygen into one donor
+def: "OBSOLETE. Catalysis of the reaction: 3-chlorobenzoate + O2 + a reduced electron acceptor = 3-chlorobenzoate-cis-4,5-diol + an oxidized electron acceptor." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102045
@@ -454725,11 +455665,12 @@
 
 [Term]
 id: GO:0102115
-name: peptidoglycan asparagine synthase activity
+name: obsolete peptidoglycan asparagine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparatyl)-L-lysyl-D-alanyl-D-alanine + ammonium + ATP = H+ + ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparaginyl)-L-lysyl-D-alanyl-D-alanine + AMP + diphosphoric acid." [EC:6.3.1.-, GOC:pz]
-xref: MetaCyc:RXN-11338
-is_a: GO:0016880 ! acid-ammonia (or amide) ligase activity
+def: "OBSOLETE. Catalysis of the reaction: ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparatyl)-L-lysyl-D-alanyl-D-alanine + ammonium + ATP = H+ + ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparaginyl)-L-lysyl-D-alanyl-D-alanine + AMP + diphosphoric acid." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102116
@@ -454827,11 +455768,12 @@
 
 [Term]
 id: GO:0102126
-name: coniferyl aldehyde 5-hydroxylase activity
+name: obsolete coniferyl aldehyde 5-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + coniferyl aldehyde + NADPH + O2 = 5-hydroxy-coniferaldehyde + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-1142
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + coniferyl aldehyde + NADPH + O2 = 5-hydroxy-coniferaldehyde + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102127
@@ -454900,19 +455842,21 @@
 
 [Term]
 id: GO:0102134
-name: (22S)-22-hydroxy-campesterol C-23 hydroxylase activity
+name: obsolete (22S)-22-hydroxy-campesterol C-23 hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (22S)-22-hydroxycampesterol + NADPH + O2 = (22R,23R)-22,23-dihydroxycampesterol + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-11529
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (22S)-22-hydroxycampesterol + NADPH + O2 = (22R,23R)-22,23-dihydroxycampesterol + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102135
-name: (22S)-22-hydroxy-campest-4-en-3-one C-23 hydroxylase activity
+name: obsolete (22S)-22-hydroxy-campest-4-en-3-one C-23 hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (22S)-22-hydroxycampest-4-en-3-one + NADPH + O2 = (22R,23R)-22,23-dihydroxy-campest-4-en-3-one + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-11530
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (22S)-22-hydroxycampest-4-en-3-one + NADPH + O2 = (22R,23R)-22,23-dihydroxy-campest-4-en-3-one + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102136
@@ -455065,7 +456009,7 @@
 id: GO:0102149
 name: farnesylcysteine lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-[(2E,6E)-farnesyl]-L-cysteine + O2 + H2O = (2-trans,6-trans)-farnesal + L-cysteine + hydrogen peroxide." [EC:1.8.3.6, GOC:pz]
+def: "Catalysis of the reaction: S-[(2E,6E)-farnesyl]-L-cysteine + O2 + H2O = (2-trans,6-trans)-farnesal + L-cysteine + H2O2." [EC:1.8.3.6, GOC:pz]
 xref: EC:1.8.3.6
 xref: MetaCyc:RXN-11623
 xref: RHEA:30231
@@ -455923,11 +456867,12 @@
 
 [Term]
 id: GO:0102235
-name: 1-penten-3-one reductase activity
+name: obsolete 1-penten-3-one reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1-penten-3-one + NADPH + H+ = 1-pentan-3-one + NADP." [GOC:pz]
-xref: MetaCyc:RXN-12295
-is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 1-penten-3-one + NADPH + H+ = 1-pentan-3-one + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102236
@@ -456060,11 +457005,12 @@
 
 [Term]
 id: GO:0102249
-name: phosphatidylcholine:diacylglycerol cholinephosphotransferase activity
+name: obsolete phosphatidylcholine:diacylglycerol cholinephosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a phosphatidylcholine + a 1,2-diacyl-sn-glycerol = a 1,2-diacyl-sn-glycerol + a phosphatidylcholine." [GOC:pz, MetaCyc:RXN-12386]
-xref: MetaCyc:RXN-12386
-is_a: GO:0016780 ! phosphotransferase activity, for other substituted phosphate groups
+def: "OBSOLETE. Catalysis of the reaction: a phosphatidylcholine + a 1,2-diacyl-sn-glycerol = a 1,2-diacyl-sn-glycerol + a phosphatidylcholine." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102250
@@ -456723,7 +457669,7 @@
 id: GO:0102317
 name: 4-methylaminobutyrate oxidase (demethylating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-(methylamino)butyric acid + O2 + H2O = gamma-aminobutyric acid + formaldehyde + hydrogen peroxide." [EC:1.5.3.19, GOC:pz]
+def: "Catalysis of the reaction: 4-(methylamino)butanoate + O2 + H2O = 4-aminobutanoate + formaldehyde + H2O2." [RHEA:33907]
 xref: EC:1.5.3.19
 xref: MetaCyc:RXN-13067
 xref: RHEA:33907
@@ -457134,7 +458080,9 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: mithramycin DK + NADPH + H+ = iso-mithramycin + NADP+. Note that  iso-mithramycin (iso-MTM) is a C2-epimer of mithramycin.  This intermediate appears to be biologically important because it is non-toxic, then exported out of the cell and finally spontaneously epimerized into the toxic mithramycin. This could represent a self-resistance mechanism against MTM toxicity." [PMID:31702856]
 xref: MetaCyc:RXN-13385
+xref: RHEA:85451
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+property_value: skos:exactMatch RHEA:85451
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30329" xsd:anyURI
 
@@ -457249,11 +458197,12 @@
 
 [Term]
 id: GO:0102369
-name: 11alpha-30-dihydroxy beta-amyrin dehydrogenase activity
+name: obsolete 11alpha-30-dihydroxy beta-amyrin dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 11alpha,30-dihydroxy-beta-amyrin + NADPH + O2 + H+ = 30-hydroxy-11-oxo-beta-amyrin + NADP + 2 H2O." [GOC:pz]
-xref: MetaCyc:RXN-13491
-is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
+def: "OBSOLETE. Catalysis of the reaction: 11alpha,30-dihydroxy-beta-amyrin + NADPH + O2 + H+ = 30-hydroxy-11-oxo-beta-amyrin + NADP + 2 H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102370
@@ -458174,13 +459123,14 @@
 
 [Term]
 id: GO:0102459
-name: 8-oxo-dADP diphosphate phosphatase activity
+name: obsolete 8-oxo-dADP diphosphate phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 8-oxo-dADP + H2O = 8-oxo-dAMP + hydrogenphosphate + H+." [GOC:pz]
+def: "OBSOLETE. Catalysis of the reaction: 8-oxo-dADP + H2O = 8-oxo-dAMP + hydrogenphosphate + H+." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "8-oxo-deoxyadenine diphosphate phosphatase activity" EXACT []
-xref: MetaCyc:RXN-14005
-is_a: GO:0017110 ! nucleoside diphosphate phosphatase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23401" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102460
@@ -458228,11 +459178,12 @@
 
 [Term]
 id: GO:0102465
-name: zeaxanthin 2,2'-beta-hydroxylase activity
+name: obsolete zeaxanthin 2,2'-beta-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: zeaxanthin + 2 NADH + 2 H+ + 2 O2 = nostoxanthin + 2 NAD + 2 H2O." [GOC:pz]
-xref: MetaCyc:RXN-14018
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: zeaxanthin + 2 NADH + 2 H+ + 2 O2 = nostoxanthin + 2 NAD + 2 H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102466
@@ -459982,11 +460933,12 @@
 
 [Term]
 id: GO:0102641
-name: (R)-lactaldehyde dehydrogenase activity
+name: obsolete (R)-lactaldehyde dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (R)-propane-1,2-diol + NADP = (R)-lactaldehyde + NADPH + H+." [GOC:pz]
-xref: MetaCyc:RXN-15743
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: (R)-propane-1,2-diol + NADP = (R)-lactaldehyde + NADPH + H+." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102643
@@ -461203,11 +462155,12 @@
 
 [Term]
 id: GO:0102762
-name: eriodictyol 4'-O-methyltransferase activity
+name: obsolete eriodictyol 4'-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: eriodictyol + S-adenosyl-L-methionine = H+ + hesperetin(1-) + S-adenosyl-L-homocysteine." [GOC:pz, MetaCyc:RXN-7754]
-xref: MetaCyc:RXN-7754
-is_a: GO:0008168 ! methyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: eriodictyol + S-adenosyl-L-methionine = H+ + hesperetin(1-) + S-adenosyl-L-homocysteine." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102763
@@ -461324,6 +462277,7 @@
 synonym: "sphingolipid long-chain base 4-hydroxylase activity" RELATED []
 xref: EC:1.14.18.5
 xref: MetaCyc:RXN-14250
+xref: Reactome:R-HSA-428260 "DEGS2 oxygenates dihydroceramide"
 xref: RHEA:55808
 is_a: GO:0016716 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, another compound as one donor, and incorporation of one atom of oxygen
 property_value: skos:exactMatch EC:1.14.18.5
@@ -461382,7 +462336,7 @@
 id: GO:0102778
 name: delta9-tetrahydrocannabinolate synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cannabigerolate + O2 = delta(9)-tetrahydrocannabinolic acid + hydrogen peroxide." [GOC:pz, RHEA:34135]
+def: "Catalysis of the reaction: cannabigerolate + O2 = delta(9)-tetrahydrocannabinolic acid +  H2O2." [RHEA:34135]
 xref: EC:1.21.3.7
 xref: MetaCyc:RXN-7854
 xref: RHEA:34135
@@ -461395,7 +462349,7 @@
 id: GO:0102779
 name: cannabidiolate synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cannabigerolate + O2 = cannabidiolate + hydrogen peroxide." [GOC:pz, RHEA:34411]
+def: "Catalysis of the reaction: cannabigerolate + O2 = cannabidiolate + H2O2." [RHEA:34411]
 xref: EC:1.21.3.8
 xref: MetaCyc:RXN-7855
 xref: RHEA:34411
@@ -461792,12 +462746,13 @@
 
 [Term]
 id: GO:0102818
-name: lycopene cleavage oxygenase activity
+name: obsolete lycopene cleavage oxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: lycopene + 2 O2 = 2 sulcatone + bixin aldehyde." [GOC:pz, MetaCyc:RXN-8236]
-xref: MetaCyc:RXN-8236
-is_a: GO:0016703 ! oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of one atom of oxygen (internal monooxygenases or internal mixed function oxidases)
+def: "OBSOLETE. Catalysis of the reaction: lycopene + 2 O2 = 2 sulcatone + bixin aldehyde." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28942" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102819
@@ -462992,9 +463947,10 @@
 
 [Term]
 id: GO:0102933
-name: GDP-4-dehydro-6-deoxy-D-mannose-4-aminotransferase activity
+name: GDP-perosamine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: GDP-4-amino-4,6-dideoxy-alpha-D-mannose + 2-oxoglutarate = GDP-4-dehydro-6-deoxy-alpha-D-mannose + L-glutamate." [GOC:pz, RHEA:36779]
+def: "Catalysis of the reaction: GDP-alpha-D-perosamine + 2-oxoglutarate = GDP-4-dehydro-alpha-D-rhamnose + L-glutamate." [RHEA:36779]
+synonym: "GDP-4-dehydro-6-deoxy-D-mannose-4-aminotransferase activity" EXACT []
 xref: EC:2.6.1.102
 xref: MetaCyc:RXN-8953
 xref: RHEA:36779
@@ -463002,6 +463958,7 @@
 property_value: skos:exactMatch EC:2.6.1.102
 property_value: skos:exactMatch RHEA:36779
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0102934
@@ -463036,11 +463993,12 @@
 
 [Term]
 id: GO:0102937
-name: 16-alpha-hydroxygypsogenate-UDP-glucosyltransferase activity
+name: obsolete 16-alpha-hydroxygypsogenate-UDP-glucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-alpha-D-glucose + 16-alpha-hydroxygypsogenate = 16-alpha-hydroxygypsogenate-28-beta-D-glucoside + UDP." [GOC:pz]
-xref: MetaCyc:RXN-9014
-is_a: GO:0016758 ! hexosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: UDP-alpha-D-glucose + 16-alpha-hydroxygypsogenate = 16-alpha-hydroxygypsogenate-28-beta-D-glucoside + UDP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102938
@@ -463129,11 +464087,12 @@
 
 [Term]
 id: GO:0102947
-name: (+)-delta-cadinene-8-hydroxylase activity
+name: obsolete (+)-delta-cadinene-8-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (+)-delta-cadinene + NADPH + O2 = 8-hydroxy-(+)-delta-cadinene + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-9045
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (+)-delta-cadinene + NADPH + O2 = 8-hydroxy-(+)-delta-cadinene + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102948
@@ -463181,11 +464140,12 @@
 
 [Term]
 id: GO:0102953
-name: hypoglycin A gamma-glutamyl transpeptidase activity
+name: obsolete hypoglycin A gamma-glutamyl transpeptidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glutathionate + hypoglycin A = L-cysteinylglycine + hypoglycin B." [GOC:pz]
-xref: MetaCyc:RXN-9157
-is_a: GO:0016755 ! aminoacyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: glutathionate + hypoglycin A = L-cysteinylglycine + hypoglycin B." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102954
@@ -463526,7 +464486,6 @@
 name: trehalose synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an NDP-alpha-D-glucose + D-glucose = a ribonucleoside 5'-diphosphate + alpha,alpha-trehalose + H+." [RHEA:47416]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:2.4.1.245
 xref: MetaCyc:RXN-9603
 xref: RHEA:47416
@@ -463534,10 +464493,8 @@
 property_value: skos:exactMatch EC:2.4.1.245
 property_value: skos:exactMatch MetaCyc:RXN-9603
 property_value: skos:exactMatch RHEA:47416
-property_value: skos:relatedMatch MetaCyc:PWY-2622
-property_value: skos:relatedMatch MetaCyc:PWY-5983
-property_value: skos:relatedMatch MetaCyc:PWY-5985
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0102987
@@ -463753,11 +464710,12 @@
 
 [Term]
 id: GO:0103004
-name: 9,10-epoxystearate hydroxylase activity
+name: obsolete 9,10-epoxystearate hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + 9,10-epoxystearate + O2 + NADPH = 9,10-epoxy-18-hydroxystearate + H2O + NADP." [GOC:pz]
-xref: MetaCyc:RXN-9805
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + 9,10-epoxystearate + O2 + NADPH = 9,10-epoxy-18-hydroxystearate + H2O + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103005
@@ -463770,11 +464728,12 @@
 
 [Term]
 id: GO:0103006
-name: 9,10-dihydroxystearate hydroxylase activity
+name: obsolete 9,10-dihydroxystearate hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (9R,10S)-dihydroxystearate + O2 + NADPH = 9,10,18-trihydroxystearate + H2O + NADP." [GOC:pz]
-xref: MetaCyc:RXN-9807
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (9R,10S)-dihydroxystearate + O2 + NADPH = 9,10,18-trihydroxystearate + H2O + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103007
@@ -463978,11 +464937,12 @@
 
 [Term]
 id: GO:0103033
-name: beta-galactosidase activity (lactose isomerization)
+name: obsolete beta-galactosidase activity (lactose isomerization)
 namespace: molecular_function
-def: "Catalysis of the reaction: alpha-lactose = beta-(1->6)-galactobiose." [EC:5.4.1.-, GOC:pz]
-xref: MetaCyc:RXN0-5363
-is_a: GO:0016867 ! intramolecular acyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: alpha-lactose = beta-(1->6)-galactobiose." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103035
@@ -464268,14 +465228,14 @@
 
 [Term]
 id: GO:0103068
-name: leukotriene C4 gamma-glutamyl transferase activity
+name: obsolete leukotriene C4 gamma-glutamyl transferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: leukotriene C4 + a standard alpha amino acid = leukotriene D4 + an (gamma-L-glutamyl)-L-amino acid." [GOC:pz, PMID:1676842, PMID:9139674]
+def: "OBSOLETE. Catalysis of the reaction: leukotriene C4 + a standard alpha amino acid = leukotriene D4 + an (gamma-L-glutamyl)-L-amino acid." [GOC:pz, PMID:1676842, PMID:9139674]
 xref: EC:2.3.2.2
-xref: MetaCyc:RXN66-336
-is_a: GO:0016755 ! aminoacyltransferase activity
 property_value: skos:exactMatch EC:2.3.2.2
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31373" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103069
@@ -464621,19 +465581,18 @@
 
 [Term]
 id: GO:0103111
-name: protein-N(pi)-phosphohistidine--N-acetyl-D-glucosamine phosphotransferase activity
+name: obsolete protein-N(pi)-phosphohistidine--N-acetyl-D-glucosamine phosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in)." [RHEA:49240]
+def: "OBSOLETE. Catalysis of the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in)." [RHEA:49240]
+comment: This term was obsoleted because it represents the same reaction as [protein-N(PI)-phosphohistidine-N-acetylglucosamine phosphotransferase system transporter activity ; GO:0022880.
 synonym: "D-glucosamine PTS permease activity" EXACT []
 synonym: "N-acetyl-D-glucosamine PTS permease activity" EXACT [EC:2.7.1.193]
-xref: EC:2.7.1.193
-xref: RHEA:49240
-is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
-property_value: skos:exactMatch EC:2.7.1.193
-property_value: skos:exactMatch RHEA:49240
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27410" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27694" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0022880
 
 [Term]
 id: GO:0103113
@@ -465234,15 +466193,13 @@
 
 [Term]
 id: GO:0106045
-name: guanine deglycation, methylglyoxal removal
+name: obsolete guanine deglycation, methylglyoxal removal
 namespace: biological_process
-def: "The removal of methylglyoxal from a glycated guanine, to form lactate and a deglycated guanine." [PMID:28596309]
-is_a: GO:0009438 ! methylglyoxal metabolic process
-is_a: GO:0051595 ! response to methylglyoxal
-is_a: GO:0106044 ! guanine deglycation
-is_a: GO:0140041 ! cellular detoxification of methylglyoxal
-created_by: hjd
-creation_date: 2017-08-01T18:17:28Z
+def: "OBSOLETE. The removal of methylglyoxal from a glycated guanine, to form lactate and a deglycated guanine." [PMID:28596309]
+comment: The reason for obsoletion is that this term represents a molecular function rather than a biological process.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31367" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0036524
 
 [Term]
 id: GO:0106046
@@ -465289,14 +466246,17 @@
 
 [Term]
 id: GO:0106050
-name: tRNA 2'-O-methyltransferase activity
+name: obsolete tRNA 2'-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing a 2'-O-nucleotide." [PMID:17242307]
+def: "OBSOLETE. Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing a 2'-O-nucleotide." [PMID:17242307]
+comment: This term was obsoleted because it is an unnecessray grouping term.
 xref: Reactome:R-HSA-6788668 "TRMT13 2'-O-methylates adenosine-4 in tRNA"
 xref: Reactome:R-HSA-6788684 "TRMT13 2'-O-methylates cytidine-4 in tRNA"
 xref: Reactome:R-HSA-9024159 "FTSJ1 2'-O-methylates cytidine-32 in tRNA(Phe)"
-is_a: GO:0008175 ! tRNA methyltransferase activity
-is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31519" xsd:anyURI
+is_obsolete: true
+consider: GO:0106339
+consider: GO:0106340
 created_by: hjd
 creation_date: 2017-08-11T20:30:31Z
 
@@ -465368,10 +466328,11 @@
 
 [Term]
 id: GO:0106059
-name: tRNA (cytidine(56)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(56)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + cytidine56 in tRNA= S-adenosyl-L-homocysteine + 2'-O-methylcytidine56 in tRNA." [RHEA:42968]
 synonym: "tRNA (cytidine 56-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(56)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.206
 xref: MetaCyc:RXN-11869
 xref: RHEA:42968
@@ -466459,13 +467420,16 @@
 
 [Term]
 id: GO:0106162
-name: mRNA N-acetyltransferase activity
+name: mRNA cytidine N-acetyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a cytidine in mRNA + acetyl-CoA + ATP + H2O = ADP + an N(4)-acetylcytidine in mRNA + CoA + H+ + phosphate." [GOC:sp, PMID:30449621, RHEA:58480]
+synonym: "mRNA N-acetyltransferase activity" BROAD []
 xref: RHEA:58480
 is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140640 ! catalytic activity, acting on a nucleic acid
 property_value: skos:exactMatch RHEA:58480
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: terms:contributor "https://github.com/geneontology/go-ontology/issues/16755" xsd:anyURI
 created_by: hjd
 creation_date: 2018-12-18T19:19:06Z
 
@@ -466681,9 +467645,10 @@
 id: GO:0106186
 name: cytoplasmic side of plasma membrane, cell tip
 namespace: cellular_component
-def: "The leaflet the plasma membrane at the cell tip that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:vw, PMID:28292899]
+def: "The leaflet the plasma membrane at the cell tip that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:vw, PMID:28292899]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0031520 ! plasma membrane of cell tip
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: hjd
 creation_date: 2019-05-01T17:48:02Z
 
@@ -467053,6 +468018,7 @@
 name: peptide glutaryltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: glutaryl-CoA + L-lysyl-[protein] = CoA + H+ + N6-glutaryl-L-lysyl-[protein]." [GOC:sp, PMID:31542297]
+xref: Reactome:R-HSA-9858590 "DLST transfers glutaryl to CoA"
 xref: RHEA:18009
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:18009
@@ -467357,6 +468323,8 @@
 namespace: molecular_function
 def: "A hydroperoxy icosatetraenoate = a hydroxy epoxy icosatrienoate." [PMID:12881489, RHEA:55560]
 xref: EC:5.4.4.7
+xref: Reactome:R-HSA-2161794 "Arachidonate is converted to HXA3/B3 by ALOX12"
+xref: Reactome:R-HSA-8942208 "ALOXE3 isomerises 12R-HpETE to HXA3"
 xref: RHEA:55560
 is_a: GO:0050486 ! intramolecular hydroxytransferase activity
 property_value: skos:exactMatch EC:5.4.4.7
@@ -467964,6 +468932,7 @@
 xref: Reactome:R-HSA-9841265 "p-S473-AKT1 phosphorylates ESR1 on serine-167"
 xref: Reactome:R-HSA-9860792 "Phospho-AKT1 phosphorylates NOS3 (eNOS) on serine-1177"
 xref: Reactome:R-HSA-9860808 "p-T816-PKN2 phosphorylates NOS3 (eNOS) on serine-1179"
+xref: Reactome:R-HSA-9948087 "CAMK2 complex phosphorylates VIM:NS4A"
 xref: RHEA:17989
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch RHEA:17989
@@ -468182,7 +469151,7 @@
 id: GO:0106329
 name: L-phenylalaine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-phenylalanine + O2 = 3-phenylpyruvate + H2O2 + NH4(+)." [RHEA:61240]
+def: "Catalysis of the reaction: H2O + L-phenylalanine + O2 = 3-phenylpyruvate + H2O2 + NH4+." [RHEA:61240]
 xref: RHEA:61240
 is_a: GO:0001716 ! L-amino-acid oxidase activity
 property_value: skos:exactMatch RHEA:61240
@@ -468256,7 +469225,7 @@
 id: GO:0106335
 name: tRNA (5-carboxymethyluridine(34)-5-O)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-(carboxymethyl)uridine34 in tRNA + S-adenosyl-L-methionine = 5-(2-methoxy-2-oxoethyl)uridine34 in tRNA + S-adenosyl-L-homocysteine." [PMID:20123966, RHEA:43208]
+def: "Catalysis of the reaction: 5-(carboxymethyl)uridine34 in tRNA + S-adenosyl-L-methionine = 5-(2-methoxy-2-oxoethyl)uridine34 in tRNA + S-adenosyl-L-homocysteine. The methylation occurs on the modified base." [PMID:20123966, RHEA:43208]
 xref: EC:2.1.1.229
 xref: RHEA:43208
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -468280,10 +469249,11 @@
 
 [Term]
 id: GO:0106339
-name: tRNA (cytidine(32)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(32)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(32) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(32) in tRNA + S-adenosyl-L-homocysteine + H+." [PMID:25404562, RHEA:42932]
 synonym: "tRNA (cytidine 32-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(32)-2'-O)-methyltransferase activity" EXACT []
 xref: MetaCyc:RXN-11866
 xref: RHEA:42932
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
@@ -468302,12 +469272,15 @@
 
 [Term]
 id: GO:0106340
-name: tRNA (guanosine(34)-2'-O)-methyltransferase activity
+name: tRNA (guanosine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanosine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylguanosine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105]
+def: "Catalysis of the reaction: guanosine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylguanosine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105, RHEA:85171]
+synonym: "tRNA (guanosine(34)-2'-O)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-9024161 "FTSJ1 2'-O-methylates guanosine-34 in tRNA(Phe)"
+xref: RHEA:85171
 is_a: GO:0016423 ! tRNA (guanine) methyltransferase activity
 property_value: skos:broadMatch EC:2.1.1.205
+property_value: skos:exactMatch RHEA:85171
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25257" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26085" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26086" xsd:anyURI
@@ -468389,9 +469362,10 @@
 
 [Term]
 id: GO:0106347
-name: U2 snRNA 2'-O-methyladenosine m6 methyltransferase activity
+name: U2 snRNA (2'-O-methyladenosine-N6)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+." [PMID:31913360, RHEA:62672]
+def: "Catalysis of the reaction: a 2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+; methylates the 6th position of adenine residues with a pre-deposited 2'-O-methylation." [PMID:31913360, RHEA:62672]
+synonym: "U2 snRNA (2'-O-methyladenosine) m6 methyltransferase activity" EXACT []
 xref: RHEA:62672
 is_a: GO:0106346 ! snRNA methyltransferase activity
 property_value: skos:exactMatch RHEA:62672
@@ -468401,9 +469375,11 @@
 
 [Term]
 id: GO:0106348
-name: U2 snRNA adenosine m6 methyltransferase activity
+name: U2 snRNA (adenine-N6)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a adenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-adenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+." [PMID:32637152]
+synonym: "U2 snRNA adenosine m6 methyltransferase activity" EXACT []
+synonym: "U2 snRNA adenosine N6 methyltransferase activity" EXACT []
 is_a: GO:0106346 ! snRNA methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27392" xsd:anyURI
 created_by: hjd
@@ -469851,11 +470827,14 @@
 
 [Term]
 id: GO:0110035
-name: rDNA spacer replication fork barrier binding, bending
+name: obsolete rDNA spacer replication fork barrier binding, bending
 namespace: molecular_function
-def: "The activity of binding selectively, and in a sequence-specific manner, a replication fork barrier found in rDNA spacers, and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:al, GOC:vw, PMID:27035982]
-is_a: GO:0043110 ! rDNA spacer replication fork barrier binding
-is_a: GO:0044374 ! sequence-specific DNA binding, bending
+def: "OBSOLETE. The activity of binding selectively, and in a sequence-specific manner, a replication fork barrier found in rDNA spacers, and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:al, GOC:vw, PMID:27035982]
+comment: This term was obsoleted because it is precomposed.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+consider: GO:0008301
+consider: GO:0043110
 created_by: kmv
 creation_date: 2017-07-17T21:09:50Z
 
@@ -471502,13 +472481,19 @@
 namespace: molecular_function
 def: "Removes a lipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle. This results in intermembrane transfer of lipids." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane lipid transfer activity" EXACT []
-synonym: "lipid carrier activity" EXACT []
+synonym: "lipid carrier activity" BROAD []
+xref: Reactome:R-HSA-8856630 "CIDEA:CIDEC binds lipid droplets"
+xref: Reactome:R-HSA-8858270 "HILPDA binds lipid droplets"
+xref: Reactome:R-HSA-8862380 "HSD17B13 binds lipid droplets"
+xref: Reactome:R-HSA-8874705 "TTPA transports alpha-TOH from lysosomal membrane to plasma membrane"
 xref: Reactome:R-HSA-9856502 "mito-STARD7 transports Q10 to outer mitochondrial membrane"
 xref: Reactome:R-HSA-9856510 "STARD7 transports Q10 to plasma membrane"
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0005215 ! transporter activity
+is_a: GO:0005319 ! lipid carrier activity
 relationship: part_of GO:0120009 ! intermembrane lipid transfer
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17648" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 created_by: krc
 creation_date: 2017-03-17T03:17:29Z
 
@@ -471519,9 +472504,9 @@
 def: "Removes a phospholipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane phospholipid transfer activity" NARROW []
 synonym: "phospholipid carrier activity" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0120013 ! lipid transfer activity
 relationship: has_part GO:0005543 ! phospholipid binding
+relationship: part_of GO:0015914 ! phospholipid transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
 created_by: krc
 creation_date: 2017-03-17T03:34:17Z
@@ -471550,7 +472535,6 @@
 def: "Removes a sphingolipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane sphingolipid transfer activity" NARROW []
 synonym: "sphingolipid carrier activity" NARROW []
-is_a: GO:0046624 ! sphingolipid transporter activity
 is_a: GO:0120013 ! lipid transfer activity
 relationship: has_part GO:0046625 ! sphingolipid binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471566,6 +472550,7 @@
 synonym: "ceramide carrier activity" EXACT []
 synonym: "ceramide transporter activity" BROAD []
 synonym: "intermembrane ceramide transfer activity" NARROW []
+xref: Reactome:R-HSA-9956909 "CERT1-2 transfers CERA from the ER membrane to the Golgi membrane"
 is_a: GO:0120016 ! sphingolipid transfer activity
 relationship: has_part GO:0097001 ! ceramide binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471582,7 +472567,7 @@
 xref: Reactome:R-HSA-1483087 "PI is exchanged with PC by PITPNB"
 xref: Reactome:R-HSA-1483211 "PC:PITPNB is transported from the Golgi membrane to the ER membrane"
 xref: Reactome:R-HSA-1483219 "PC is exchanged with PI by PITPNB"
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+xref: Reactome:R-HSA-9956308 "STARD7 transports PC from ER membrane to mitochondrial outer membrane"
 is_a: GO:0120014 ! phospholipid transfer activity
 relationship: has_part GO:0031210 ! phosphatidylcholine binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471923,9 +472908,10 @@
 
 [Term]
 id: GO:0120048
-name: U6 snRNA (adenine-(43)-N(6))-methyltransferase activity
+name: U6 snRNA (adenine(43)-N6)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: adenosine in U6 snRNA + S-adenosyl-L-methionine = H+ + N(6)-methyladenosine in U6 snRNA + S-adenosyl-L-homocysteine." [PMID:28525753, PMID:32266935, RHEA:52808]
+synonym: "U6 snRNA m(6)A methyltransferase activity" EXACT []
 xref: EC:2.1.1.346
 xref: MetaCyc:RXN-18779
 xref: RHEA:52808
@@ -471953,6 +472939,7 @@
 name: ribitol beta-1,4-xylosyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-D-xylose + D-ribitol 5-phosphate-R = UDP + beta1,4-xylosyl-D-ribitol 5-phosphate-R." [PMID:27733679]
+xref: Reactome:R-HSA-9940805 "RXYLT1 transfers initial Xyl to matriglycan chain"
 is_a: GO:0035252 ! UDP-xylosyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/13641" xsd:anyURI
 created_by: krc
@@ -474020,7 +475007,7 @@
 id: GO:0120216
 name: matrilin complex
 namespace: cellular_component
-def: "A cartilage extracellular matrix complex that mediates interactions between major components of the extracellular matrix such as collagens and proteoglycans and contributes to their fibrillar network. Exists as an obligate homotrimer." [PMID:10367731, PMID:15075323, PMID:15094109, PMID:9699631]
+def: "A cartilage extracellular matrix complex that mediates interactions between major components of the extracellular matrix such as collagens and proteoglycans and contributes to their fibrillar network. Exists as an obligate homotrimer." [PMID:10367731, PMID:15075323, PMID:29310791, PMID:9699631]
 synonym: "matrilin family complex" RELATED [GOC:bhm]
 synonym: "matrilin-1 complex" NARROW [GOC:bhm]
 synonym: "matrilin-2 complex" NARROW [GOC:bhm]
@@ -474366,10 +475353,11 @@
 id: GO:0120242
 name: 2-iminobutanoate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-iminobutanoate + H2O = 2-oxobutanoate + NH4(+)." [RHEA:39975]
+def: "Catalysis of the reaction: 2-iminobutanoate + H2O = 2-oxobutanoate + NH4+." [RHEA:39975]
 synonym: "2-iminobutanoate/2-iminopropanoate deaminase" BROAD [EC:3.5.99.10]
 xref: KEGG_REACTION:R11098
 xref: MetaCyc:RXN-15123
+xref: Reactome:R-HSA-9014641 "HRSP12 deaminates 2-iminobutanoate to 2OBUTA"
 xref: RHEA:39975
 is_a: GO:0120241 ! 2-iminobutanoate/2-iminopropanoate deaminase activity
 property_value: skos:exactMatch RHEA:39975
@@ -474382,7 +475370,7 @@
 id: GO:0120243
 name: 2-iminopropanoate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-iminopropanoate + H2O = NH4(+) + pyruvate." [RHEA:40671]
+def: "Catalysis of the reaction: 2-iminopropanoate + H2O = NH4+ + pyruvate." [RHEA:40671]
 synonym: "2-iminobutanoate/2-iminopropanoate deaminase" BROAD [EC:3.5.99.10]
 xref: KEGG_REACTION:R11099
 xref: MetaCyc:RXN-15127
@@ -475114,7 +476102,7 @@
 id: GO:0120294
 name: peptide serotonyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutaminyl-[protein] + serotonin = 5-serotonyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:14697203]
+def: "Catalysis of the reaction: L-glutaminyl-[protein] + serotonin = 5-serotonyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:14697203]
 xref: RHEA:66552
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66552
@@ -475127,7 +476115,7 @@
 id: GO:0120295
 name: histone serotonyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutaminyl-[histone] + serotonin = 5-serotonyl-L-glutamyl-[histone] + NH4(+)." [GOC:sp, PMID:30867594]
+def: "Catalysis of the reaction: L-glutaminyl-[histone] + serotonin = 5-serotonyl-L-glutamyl-[histone] + NH4+." [GOC:sp, PMID:30867594]
 is_a: GO:0120294 ! peptide serotonyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
 property_value: skos:broadMatch RHEA:66552
@@ -475140,7 +476128,7 @@
 id: GO:0120296
 name: peptide dopaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dopamine + L-glutaminyl-[protein] = 5-dopaminyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:22858378, PMID:32273471]
+def: "Catalysis of the reaction: dopamine + L-glutaminyl-[protein] = 5-dopaminyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:22858378, PMID:32273471]
 xref: RHEA:66556
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66556
@@ -475153,7 +476141,7 @@
 id: GO:0120297
 name: histone dopaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dopamine + L-glutaminyl-[histone] = 5-dopaminyl-L-glutamyl-[histone] + NH4(+)." [GOC:sp, PMID:32273471]
+def: "Catalysis of the reaction: dopamine + L-glutaminyl-[histone] = 5-dopaminyl-L-glutamyl-[histone] + NH4+." [GOC:sp, PMID:32273471]
 is_a: GO:0120296 ! peptide dopaminyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
 property_value: skos:broadMatch RHEA:66556
@@ -475166,7 +476154,7 @@
 id: GO:0120298
 name: peptide noradrenalinyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (R)-noradrenaline + L-glutaminyl-[protein] = 5-(R)-noradrenalinyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:22858378]
+def: "Catalysis of the reaction: (R)-noradrenaline + L-glutaminyl-[protein] = 5-(R)-noradrenalinyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:22858378]
 xref: RHEA:66560
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66560
@@ -476107,7 +477095,6 @@
 name: choline trimethylamine lyase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: choline = acetaldehyde + trimethylamine." [RHEA:35095]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:4.3.99.4
 xref: KEGG_REACTION:R10285
 xref: MetaCyc:RXN-13946
@@ -476117,8 +477104,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R10285
 property_value: skos:exactMatch MetaCyc:RXN-13946
 property_value: skos:exactMatch RHEA:35095
-property_value: skos:relatedMatch MetaCyc:PWY-7167
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28808" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 created_by: sjm
 creation_date: 2024-08-30T13:13:55Z
 
@@ -476228,6 +477215,7 @@
 synonym: "Glycosaminoglycan biosynthesis, linkage tetrasaccharide" EXACT []
 synonym: "glycosaminoglycan-protein linkage region biosynthesis" EXACT [MetaCyc:PWY-6557]
 xref: MetaCyc:PWY-6557
+xref: Reactome:R-HSA-1971475 "Glycosaminoglycan-protein linkage region biosynthesis"
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
 relationship: part_of GO:0015012 ! heparan sulfate proteoglycan biosynthetic process
 relationship: part_of GO:0030210 ! heparin proteoglycan biosynthetic process
@@ -476316,6 +477304,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: a 4-hydroxy-3-methoxy-5-(all-trans-polyprenyl)benzoate + H+ = a 2-methoxy-6-(all-trans-polyprenyl)phenol + CO2." [PMID:38295803, PMID:38425362, RHEA:81179]
 xref: EC:4.1.1.130
+xref: Reactome:R-HSA-2162195 "COQ4 decarboxylates MHDB"
 xref: RHEA:81179
 is_a: GO:0016831 ! carboxy-lyase activity
 relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
@@ -476423,6 +477412,7 @@
 def: "Catalysis of the reaction: Fe(II)-heme o + 2 acceptor + H2O = Fe(II)-heme a + 2 acceptor-H2. The conversion of heme o to heme a occcurs by two successive hydroxylations of the methyl group at C8 using water as the oxygen source. The first hydroxylation forms heme i, the second hydroxylation results in an unstable dihydroxymethyl group, which spontaneously dehydrates, resulting in the formyl group of heme A." [PMID:30397130, RHEA:63388]
 xref: EC:1.17.99.9
 xref: MetaCyc:RXN-18399
+xref: Reactome:R-HSA-2995334 "COX15 transforms heme O to heme A"
 xref: RHEA:63388
 is_a: GO:0016725 ! oxidoreductase activity, acting on CH or CH2 groups
 property_value: skos:exactMatch EC:1.17.99.9
@@ -476527,15 +477517,17 @@
 
 [Term]
 id: GO:0120554
-name: 2-aminobutanoate transaminase activity
+name: (2S)-2-aminobutanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: (2S)-2-aminobutanoate + 2-oxoglutarate = 2-oxobutanoate + L-glutamate." [PMID:27827456, RHEA:70223]
+synonym: "2-aminobutanoate transaminase activity" EXACT []
 xref: MetaCyc:RXN-20408
 xref: RHEA:70223
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch MetaCyc:RXN-20408
 property_value: skos:exactMatch RHEA:70223
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30260" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: sjm
 creation_date: 2025-05-15T14:38:07Z
 
@@ -477576,7 +478568,7 @@
 id: GO:0140070
 name: hydrogen peroxide channel activity
 namespace: molecular_function
-def: "Enables the energy-independent facilitated diffusion of hydrogen peroxide through a transmembrane aqueous pore or channel." [PMID:23541115, PMID:27256569]
+def: "Enables the energy-independent facilitated diffusion of H2O2 through a transmembrane aqueous pore or channel." [PMID:23541115, PMID:27256569]
 synonym: "hydrogen peroxide transmembrane transporter activity" RELATED []
 is_a: GO:0015267 ! channel activity
 relationship: part_of GO:0080170 ! hydrogen peroxide transmembrane transport
@@ -477757,11 +478749,12 @@
 id: GO:0140085
 name: L-amino-acid N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acetyl-CoA + a L-amino acid = CoA + an N-acetyl-L-amino-acid. In some cases acetyl phosphate can be used as a donor." [GOC:pg, RHEA:83863]
+def: "Catalysis of the reaction: acetyl-CoA + a L-amino acid = CoA + an N-acetyl-L-amino-acid. In some cases other acetyl containing molecules can be used as donor." [GOC:pg, RHEA:83863]
 xref: RHEA:83863
-is_a: GO:0008080 ! N-acetyltransferase activity
-property_value: skos:exactMatch RHEA:83863
+is_a: GO:0140379 ! amino acid acyltransferase activity
+property_value: skos:narrowMatch RHEA:83863
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29604" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 created_by: pg
 creation_date: 2025-01-24T10:40:36Z
 
@@ -477985,10 +478978,23 @@
 subset: goslim_prokaryote
 subset: goslim_prokaryote_ribbon
 xref: Reactome:R-HSA-1449687 "Corticosteroids bind to CBG in blood"
+xref: Reactome:R-HSA-196126 "STAR mediates CHOL translocation from cytosol to mitochondrial intermembrane space"
+xref: Reactome:R-HSA-209738 "VD3 binds GC"
+xref: Reactome:R-HSA-209944 "25(OH)D binds GC"
+xref: Reactome:R-HSA-2395764 "atREs binds to nascent CM"
+xref: Reactome:R-HSA-2399913 "atRAL binds to RBP2"
+xref: Reactome:R-HSA-2404142 "atROL binds RBP1"
+xref: Reactome:R-HSA-2855259 "atROL binds RBP1 (parenchymal cell)"
 xref: Reactome:R-HSA-3149494 "MMACHC:cob(II)alamin binds MMADHC"
 xref: Reactome:R-HSA-3149563 "MMADHC targets transport of cytosolic cob(II)alamin to mitochondria"
 xref: Reactome:R-HSA-3159259 "MMAA:MUT binds AdoCbl"
 xref: Reactome:R-HSA-3204318 "cob(II)alamin is transferred from MMACHC:MMADHC:cob(II)alamin to MTRR:MTR"
+xref: Reactome:R-HSA-5246478 "APOM binds retinoids"
+xref: Reactome:R-HSA-8870499 "PLEKHA3,8 bind PI4P, ARF1"
+xref: Reactome:R-HSA-8932980 "SERINC3,5,(1,2,4) transport L-Ser from cytosol to plasma membrane"
+xref: Reactome:R-HSA-8963851 "VD3 dissociates from GC"
+xref: Reactome:R-HSA-9661432 "BIL binds ALB"
+xref: Reactome:R-HSA-9663511 "BIL binds GSTA1, FABP1"
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0005488 ! binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14221" xsd:anyURI
@@ -478599,12 +479605,14 @@
 
 [Term]
 id: GO:0140155
-name: hexagonal collagen network of basement membrane
+name: obsolete hexagonal collagen network of basement membrane
 namespace: cellular_component
-def: "Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane." [PMID:21421911, PMID:2376131, PMID:3047147, PMID:31387942]
-is_a: GO:0098645 ! complex of network-forming collagens
-relationship: part_of GO:0140143 ! collagenous component of basement membrane
+def: "OBSOLETE. Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane." [PMID:21421911, PMID:2376131, PMID:3047147, PMID:31387942]
+comment: This term was obsoleted because it is an uncessary grouping term. There is not sufficient evidence to show that collagen VIII is only located in the basement membrane in all organs and species.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29402" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31492" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0098646
 created_by: pg
 creation_date: 2025-03-26T06:55:36Z
 
@@ -478875,10 +479883,11 @@
 id: GO:0140178
 name: lumenal side of cis-Golgi network membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi network directly contacts the Golgi lumen and may be involved in glycosylation, cargo sorting, or interactions with resident Golgi proteins." [PMID:1747103, PMID:29802621, PMID:34080016]
+def: "The leaflet of the cis-Golgi network membrane directly contacts the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where glycosylation, cargo sorting, or interactions with resident Golgi proteins occur." [PMID:1747103, PMID:29802621, PMID:34080016]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0033106 ! cis-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30389" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-05T08:47:14Z
 
@@ -478886,10 +479895,11 @@
 id: GO:0140179
 name: cytoplasmic side of cis-Golgi network membrane
 namespace: cellular_component
-def: "The leaflet of the membrane bilayer of the cis-Golgi network faces the cytoplasm and is where interactions with cytosolic proteins occur, including those for vesicle tethering, fusion, and Golgi-associated trafficking activities." [PMID:1747103, PMID:23913272, PMID:34080016, PMID:34597626]
+def: "The leaflet of the cis-Golgi network membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins, including those for vesicle tethering, fusion, and Golgi-associated trafficking activities." [PMID:1747103, PMID:23913272, PMID:34080016, PMID:34597626]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0033106 ! cis-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30388" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-05T08:50:47Z
 
@@ -478898,8 +479908,9 @@
 name: centriole scaffold activity
 namespace: molecular_function
 def: "The binding activity of a protein that contributes to the stable formation of a centriole by forming persistent structures, or templates, upon which other centriolar proteins assemble." [PMID:29784964, PMID:32956907, PMID:34702818]
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30443" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31499" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-11T13:59:35Z
 
@@ -479250,6 +480261,8 @@
 name: folic acid:proton symporter activity
 namespace: molecular_function
 def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: folic acid(out) + H+(out) = folic acid(in) + H+(in). The main folic acid symporter is the Proton-Coupled Folate Transporter (PCFT/SLC46A1), which has similar affinity for transport of reduced folates (5-methyl THF, 5-formyl THF) and folic acid." [PMID:24745983]
+xref: Reactome:R-HSA-200646 "Cytosolic folate export across the plasma membrane"
+xref: Reactome:R-HSA-200729 "SLC46A1 transports FOLA from extracellular region to cytosol"
 xref: TC:2.A.1.50.1
 is_a: GO:0008517 ! folic acid transmembrane transporter activity
 is_a: GO:0015295 ! solute:proton symporter activity
@@ -480414,6 +481427,7 @@
 synonym: "small molecular sensor activity" EXACT []
 synonym: "small molecule sensing activity" EXACT []
 synonym: "small molecule sensor activity" NARROW []
+xref: Reactome:R-HSA-9854368 "ROS,RNS oxidize ACO2:4Fe-4S"
 is_a: GO:0098772 ! molecular function regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28772" xsd:anyURI
 created_by: pg
@@ -480451,15 +481465,18 @@
 
 [Term]
 id: GO:0140303
-name: intramembrane lipid transporter activity
+name: intramembrane lipid carrier activity
 namespace: molecular_function
 def: "Enables the transport of a lipid from a region of a membrane to a different region on the same membrane." [PMID:16828084]
 synonym: "flippase activity" RELATED []
+synonym: "intramembrane lipid transporter activity" BROAD []
 synonym: "translocase activity" BROAD []
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0005215 ! transporter activity
+is_a: GO:0005319 ! lipid carrier activity
 relationship: part_of GO:0034204 ! lipid translocation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17648" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19477" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 created_by: pg
 creation_date: 2019-03-01T20:14:09Z
 
@@ -480527,13 +481544,16 @@
 
 [Term]
 id: GO:0140309
-name: unfolded protein carrier activity
+name: unfolded protein holdase activity
 namespace: molecular_function
-def: "A protein carrier activity that binds to a protein in an unfolded state and escorts it between two different cellular components. The unfolded protein carrier prevents aggregation of the target protein." [PMID:39488384]
-synonym: "holdase" BROAD []
+def: "A protein carrier activity that binds to a protein in an unfolded state and escorts it to an acceptor molecule or to a specific location. The unfolded protein carrier prevents aggregation of the target protein while it's being delivers to its final destination." [PMID:39488384]
+comment: Note that an holdase binds an unfolded protein and keeps it unfolded, unlike a protein folding chaperone, which binds an unfolded protein to fold it.
+synonym: "holdase" EXACT []
 synonym: "holdase-carrier chaperone" EXACT []
-is_a: GO:0140597 ! protein carrier chaperone
+synonym: "unfolded protein carrier activity" EXACT []
+is_a: GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30552" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31601" xsd:anyURI
 created_by: pg
 creation_date: 2025-11-03T13:14:32Z
 
@@ -480556,6 +481576,7 @@
 name: protein sequestering activity
 namespace: molecular_function
 def: "Binding to a protein to prevent it from interacting with other partners or to inhibit its localization to the area of the cell or complex where it is active." [PMID:1493333]
+xref: Reactome:R-HSA-1678694 "Heparanase 2 (HPSE2) binds heparan sulfate proteoglycans"
 is_a: GO:0140313 ! molecular sequestering activity
 intersection_of: GO:0140313 ! molecular sequestering activity
 intersection_of: has_part GO:0005515 ! protein binding
@@ -480593,6 +481614,7 @@
 subset: goslim_prokaryote_ribbon
 xref: Reactome:R-HSA-203553 "eNOS binds NOSIP"
 xref: Reactome:R-HSA-203680 "eNOS:NOSIP translocation from caveolae to intracellular compartments"
+xref: Reactome:R-HSA-2404134 "RBP4:atROL binds TTR"
 xref: Reactome:R-HSA-3245898 "TCN1 binds correnoids in the circulation"
 is_a: GO:0003674 ! molecular_function
 created_by: pg
@@ -480726,7 +481748,11 @@
 name: lysophospholipase C activity
 namespace: molecular_function
 def: "Hydrolysis of a lysoglycerophospholipid at the first phosphodiester bond between the phosphate and glycerol." [PMID:32917725]
+xref: RHEA:86375
+xref: RHEA:86379
 is_a: GO:0004629 ! C-type glycerophospholipase activity
+property_value: skos:narrowMatch RHEA:86375
+property_value: skos:narrowMatch RHEA:86379
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30949" xsd:anyURI
 created_by: pg
 creation_date: 2025-11-10T12:35:51Z
@@ -480747,12 +481773,13 @@
 
 [Term]
 id: GO:0140326
-name: ATPase-coupled intramembrane lipid transporter activity
+name: ATPase-coupled intramembrane lipid carrier activity
 namespace: molecular_function
 alt_id: GO:0004012
 alt_id: GO:0008557
 def: "Catalysis of the movement of lipids from one membrane leaflet to the other, driven by ATP hydrolysis. This includes flippases and floppases." [PMID:16828084]
 synonym: "aminophospholipid-transporting ATPase" RELATED []
+synonym: "ATPase-coupled intramembrane lipid transporter activity" BROAD []
 synonym: "ATPase-coupled phospholipid transporter activity" RELATED []
 synonym: "ATPase-dependent phospholipid transporter activity" RELATED []
 synonym: "phospholipid flippase activity" NARROW []
@@ -480763,7 +481790,7 @@
 xref: MetaCyc:3.6.3.1-RXN
 xref: Reactome:R-HSA-939763 "P-type ATPases type IV transport external-facing APLs to internal side of the plasma membrane"
 xref: Reactome:R-HSA-947591 "P-type ATPases type IV transport internal-facing APLs to external side of the plasma membrane"
-is_a: GO:0140303 ! intramembrane lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 is_a: GO:0140657 ! ATP-dependent activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch EC:7.6.2.1
@@ -480781,7 +481808,7 @@
 def: "Catalysis of the movement of lipids from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:20043909, PMID:25284293, Wikipedia:Flippase]
 comment: Nomenclature note. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases). Scramblases are ATP-independent, non-selective, inducing non-specific transbilayer movements across the membrane. The direction of the translocation should be taken into account for annotation (from the exoplasmic to the cytosolic leaflet of a membrane).
 synonym: "flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0140326 ! ATPase-coupled intramembrane lipid transporter activity
+is_a: GO:0140326 ! ATPase-coupled intramembrane lipid carrier activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
 creation_date: 2019-04-29T12:51:21Z
@@ -480793,7 +481820,7 @@
 def: "Catalysis of the movement of a lipid from the cytosolic to the exoplasmic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:20043909, PMID:25284293, Wikipedia:Flippase]
 comment: Nomenclature note. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases). Scramblases are ATP-independent, non-selective, inducing non-specific transbilayer movements across the membrane. The direction of the translocation should be taken into account for annotation (from the cytosolic to the exoplasmic leaflet of a membrane).
 synonym: "floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
-is_a: GO:0140326 ! ATPase-coupled intramembrane lipid transporter activity
+is_a: GO:0140326 ! ATPase-coupled intramembrane lipid carrier activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
 creation_date: 2019-04-29T12:58:53Z
@@ -480849,7 +481876,6 @@
 namespace: molecular_function
 def: "Catalysis of the movement of a glycerophospholipid from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:26212235]
 synonym: "glycerophospholipid flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140327 ! flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -480874,7 +481900,9 @@
 synonym: "glycoprotein phospholipase C activity" EXACT []
 synonym: "glycosylphosphatidyl inositol-anchor protein phospholipase C activity" EXACT []
 synonym: "GPI-PLC activity" EXACT []
+xref: RHEA:86259
 is_a: GO:0004629 ! C-type glycerophospholipase activity
+property_value: skos:narrowMatch RHEA:86259
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28868" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31078" xsd:anyURI
 created_by: pg
@@ -480945,7 +481973,6 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phosphatidylethanolamine from the cytosolic to the exoplasmic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:10029989]
 synonym: "phosphatidylethanolamine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -480987,7 +482014,7 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phosphatidylcholine from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:11870854]
 synonym: "phosphatidylcholine flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+is_a: GO:0008525 ! phosphatidylcholine intramembrane carrier activity
 is_a: GO:0140333 ! glycerophospholipid flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -481039,7 +482066,9 @@
 def: "A supramolecular network that forms mainly through linkages of the short laminin arms between the N-terminal regions of one alpha, one beta, and one gamma chain. The laminin network associates with the collagen network to create the stable, sheet-like structure of the basement membrane ECM." [PMID:1577869, PMID:21421915, PMID:23263632, PMID:28187500]
 synonym: "laminin complex" BROAD []
 is_a: GO:0099080 ! supramolecular complex
+relationship: part_of GO:0140144 ! non-collagenous component of basement membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31053" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31490" xsd:anyURI
 created_by: pg
 creation_date: 2025-12-01T12:38:57Z
 
@@ -481063,7 +482092,7 @@
 synonym: "glucosylceramide flippase activity" NARROW []
 synonym: "glycoceramide flippase activity" EXACT []
 synonym: "glycosylceramide flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0046624 ! sphingolipid transporter activity
+is_a: GO:0046624 ! sphingolipid intramembrane carrier activity
 is_a: GO:0140327 ! flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -481224,7 +482253,7 @@
 namespace: cellular_component
 def: "A ribonucleoprotein granule located in the cytoplasm and the nucleus. GW-bodies minimally contain the Argonaute2 (Ago2) and TNRC6B proteins, together with specific target RNAs." [PMID:16418578, PMID:26930655, PMID:29576456]
 synonym: "GW-body" EXACT []
-is_a: GO:0036464 ! cytoplasmic ribonucleoprotein granule
+is_a: GO:0035770 ! ribonucleoprotein granule
 created_by: pg
 creation_date: 2019-07-25T19:26:48Z
 
@@ -481297,8 +482326,10 @@
 name: ornithine lipid biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of ornithine lipids, any of a class of phosphorus-free bacterial membrane lipids with an L-ornithine-derived head group." [PMID:22724388]
+xref: MetaCyc:PWY-6818
 is_a: GO:0008652 ! amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31271" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: pg
 creation_date: 2026-01-20T12:30:41Z
 
@@ -481368,6 +482399,29 @@
 creation_date: 2026-01-21T10:01:05Z
 
 [Term]
+id: GO:0140378
+name: protein complex scaffold activity
+namespace: molecular_function
+def: "A structural molecule activity of a protein-containing complex component that serves to hold the complex together. Protein complex scaffolds are integral members of complexes." [PMID:41298522]
+synonym: "protein-complex scaffolding activity" EXACT []
+synonym: "protein-containing complex scaffold activity" EXACT []
+is_a: GO:0005198 ! structural molecule activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31444" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T08:05:02Z
+
+[Term]
+id: GO:0140379
+name: amino acid acyltransferase activity
+namespace: molecular_function
+def: "Catalysis of the transfer of an acyl group to an acceptor group on a free amino acid, usually using acetyl-CoA as donor." [GOC:curators]
+synonym: "amino acid N-acetyltransferase activity" NARROW []
+is_a: GO:0016746 ! acyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T11:14:33Z
+
+[Term]
 id: GO:0140380
 name: psilocybin biosynthetic process
 namespace: biological_process
@@ -481443,6 +482497,64 @@
 creation_date: 2019-11-20T13:25:55Z
 
 [Term]
+id: GO:0140385
+name: amino acid transaminase activity
+namespace: molecular_function
+def: "Transfer of an amino group from an amino acid to an acceptor, usually a 2-oxo acid." [GOC:curators]
+is_a: GO:0008483 ! transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T13:30:09Z
+
+[Term]
+id: GO:0140386
+name: tRNA (cytidine(4)/adenine(4)-2'-O-ribose)-methyltransferase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: cytidine(4) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(4) in tRNA + S-adenosyl-L-homocysteine + H+. Acts on  cytidine(4) on tRNAPro and tRNAGly(GCC), and on adenosine(4) in tRNAHis." [EC:2.1.1.225, PMID:17242307]
+xref: EC:2.1.1.225
+xref: RHEA:32767
+xref: RHEA:43192
+xref: RHEA:43196
+is_a: GO:0016426 ! tRNA (adenine) methyltransferase activity
+is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
+property_value: skos:exactMatch EC:2.1.1.225
+property_value: skos:narrowMatch RHEA:32767
+property_value: skos:narrowMatch RHEA:43192
+property_value: skos:narrowMatch RHEA:43196
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31521" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-06T12:44:45Z
+
+[Term]
+id: GO:0140387
+name: negative regulation of L-glutamate biosynthetic process
+namespace: biological_process
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of L-glutamate biosynthetic process." [PMID:16899818, PMID:29649217]
+synonym: "negative regulation of glutamate biosynthetic process" BROAD []
+synonym: "negative regulation of L-glutamate biosynthesis" BROAD []
+is_a: GO:2000212 ! negative regulation of glutamate metabolic process
+is_a: GO:2000283 ! negative regulation of amino acid biosynthetic process
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: negatively_regulates GO:0097054 ! L-glutamate biosynthetic process
+relationship: negatively_regulates GO:0097054 ! L-glutamate biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31582" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-17T08:55:47Z
+
+[Term]
+id: GO:0140388
+name: protein translocation chaperone activity
+namespace: molecular_function
+def: "Binding to the exposed unfolded portion of a polypeptide as it emerges from a membrane translocation channel/pore to drive its unidirectional transport across a membrane. Iterative cycles of ATP-dependent binding/rebinding by the chaperone prevents back sliding and drive vectorial transport." [PMID:11733493]
+synonym: "import motor chaperone activity" RELATED []
+synonym: "translocation ratchet activity" RELATED []
+is_a: GO:0140597 ! protein carrier activity
+is_a: GO:0140657 ! ATP-dependent activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31580" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-18T12:58:22Z
+
+[Term]
 id: GO:0140393
 name: norsolorinic acid ketoreductase activity
 namespace: molecular_function
@@ -483074,6 +484186,7 @@
 name: defense response to symbiont
 namespace: biological_process
 def: "Reactions triggered in response to the presence of a symbiont that act to protect or prevent damage to the host." [GOC:pg]
+xref: Reactome:R-HSA-9953170 "GBP-mediated host defense"
 is_a: GO:0098542 ! defense response to other organism
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20261" xsd:anyURI
 created_by: pg
@@ -483228,13 +484341,15 @@
 name: membrane protein dislocase activity
 namespace: molecular_function
 def: "The activity of removing a protein from a membrane, by binding to a transmembrane helical fragment of a tail-anchored protein and releasing it from the the hydrophobic region of one or both lipid bilayers. The reaction is driven by ATP hydrolysis." [PMID:24821790, PMID:28712723, PMID:32973005]
-synonym: "transmembrane helix dislocase\ntransmembrane protein dislocase activity" RELATED []
+synonym: "transmembrane helix dislocase" RELATED []
+synonym: "transmembrane protein dislocase activity" RELATED []
 xref: RHEA:66168
-is_a: GO:0140318 ! protein transporter activity
+is_a: GO:0140597 ! protein carrier activity
 is_a: GO:0140657 ! ATP-dependent activity
 property_value: skos:narrowMatch RHEA:66168
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20237" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22209" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30369" xsd:anyURI
 created_by: pg
 creation_date: 2020-12-15T07:13:21Z
 
@@ -483366,6 +484481,10 @@
 comment: Note that this enzyme transports Cu(+) or Ag(+), and cannot transport the divalent ions, contrary to EC:7.2.2.9, which mainly transports the divalent copper ion (source: EC:7.2.2.8).
 xref: EC:7.2.2.8
 xref: MetaCyc:RXN-14455
+xref: Reactome:R-HSA-3697838 "ATP7A transfers Cu from ATOX1 to SOD3"
+xref: Reactome:R-HSA-6803545 "ATP7A transports cytosolic Cu2+ to phagosomal lumen"
+xref: Reactome:R-HSA-936802 "ATP7A transports cytosolic Cu1+ to extracellular region"
+xref: Reactome:R-HSA-936895 "ATP7B transports cytosolic Cu1+ to Golgi lumen"
 xref: RHEA:25792
 is_a: GO:0005375 ! copper ion transmembrane transporter activity
 is_a: GO:0015662 ! P-type ion transporter activity
@@ -483549,16 +484668,16 @@
 
 [Term]
 id: GO:0140597
-name: protein carrier chaperone
+name: protein carrier activity
 namespace: molecular_function
-def: "Binding to and carrying a protein between two different cellular components by moving along with the target protein." [PMID:7628437]
-synonym: "protein carrier activity" EXACT []
+def: "Binding to and carrying a protein to an acceptor molecule or to a specific location by moving along with the target protein." [PMID:7628437]
+synonym: "protein carrier chaperone" EXACT []
 synonym: "protein chaperone" BROAD []
 is_a: GO:0140104 ! molecular carrier activity
-relationship: has_part GO:0005515 ! protein binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22127" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22356" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31601" xsd:anyURI
 created_by: pg
 creation_date: 2021-02-10T06:16:16Z
 
@@ -483567,7 +484686,7 @@
 name: lipoprotein carrier activity
 namespace: molecular_function
 def: "Binding to and carrying a lipoprotein between two different cellular locations by moving along with the target lipoprotein." [PMID:7628437]
-is_a: GO:0140597 ! protein carrier chaperone
+is_a: GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 created_by: pg
 creation_date: 2021-02-10T06:16:26Z
@@ -483732,6 +484851,8 @@
 synonym: "monoiodotyrosine deiodinase activity" RELATED []
 synonym: "tyrosine iodinase activity" RELATED []
 xref: EC:1.21.1.1
+xref: Reactome:R-HSA-209921 "Monoiodinated tyrosine can be deiodinated"
+xref: Reactome:R-HSA-209960 "Diiodinated tyrosine can be deiodinated"
 xref: RHEA:32479
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.1.1
@@ -484551,6 +485672,7 @@
 namespace: molecular_function
 def: "A molecular function regulator that activates or increases the activity of its target via non-covalent binding that does not result in covalent modification to the target." [GOC:curators]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
+xref: Reactome:R-HSA-8866941 "MID1IP1 binds THRSP"
 is_a: GO:0098772 ! molecular function regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21782" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25165" xsd:anyURI
@@ -484996,7 +486118,7 @@
 synonym: "histone carrier activity" EXACT []
 synonym: "nucleosome remodeling activity" RELATED []
 xref: Reactome:R-HSA-9822185 "HIRA and NPM2 assemble H3.3-containing nucleosomes on paternal DNA"
-is_a: GO:0140597 ! protein carrier chaperone
+is_a: GO:0140597 ! protein carrier activity
 relationship: has_part GO:0042393 ! histone binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22126" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22129" xsd:anyURI
@@ -485352,8 +486474,9 @@
 id: GO:0140740
 name: ADP-riboxanase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-arginyl-[protein] + NAD+ = ADP-riboxanated L-argininyl-[protein] + H+ + NH4(+) + nicotinamide." [PMID:34671164, RHEA:69500]
+def: "Catalysis of the reaction: L-arginyl-[protein] + NAD+ = ADP-riboxanated L-argininyl-[protein] + H+ + NH4+ + nicotinamide." [PMID:34671164, RHEA:69500]
 comment: Note that this activity has two steps: a transfer of an ADP-ribose group, followed by the elimination of an ammonia group.
+xref: Reactome:R-HSA-9956624 "OspC3 ADP-riboxanates CASP4"
 xref: RHEA:69500
 xref: RHEA:75883
 xref: RHEA:75887
@@ -485588,14 +486711,16 @@
 id: GO:0140759
 name: histone H3K56 methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 56) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 56). This reaction is the addition of a methyl group to the lysine residue at position 56 of the histone H3 protein." [PMID:23451023, PMID:28743002]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 56) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 56). This reaction is the addition of a methyl group to the lysine residue at position 56 of the histone H3 protein." [PMID:23451023, PMID:28743002, RHEA:85519]
 comment: Comment: Note that the residue position corresponds to the canonical human H3 histone (UniProtKB:P84243); this residue is conserved across all eukaryotes. Residue 1 is the first residue following removal of the initiating Methionine (Met). Note that each histone is encoded by multiple genes, and sequences may vary across different genes within an organism.
 synonym: "histone H3K56 methylase activity" EXACT []
 synonym: "histone lysine N-methyltransferase activity (H3-K56 specific)" EXACT []
 synonym: "histone methyltransferase activity (H3-K56 specific)" EXACT []
 synonym: "histone-H3K56 methyltransferase activity" EXACT []
+xref: RHEA:85519
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
+property_value: skos:exactMatch RHEA:85519
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22757" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24337" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
@@ -485615,8 +486740,12 @@
 synonym: "histone H3-tri/dimethyl-lysine-56 demethylase activity" EXACT []
 synonym: "histone H3K56me2 demethylase activity" RELATED []
 synonym: "histone H3K56me3 demethylase activity" RELATED []
+xref: RHEA:85927
+xref: RHEA:85931
 is_a: GO:0016706 ! 2-oxoglutarate-dependent dioxygenase activity
 is_a: GO:0141052 ! histone H3 demethylase activity
+property_value: skos:narrowMatch RHEA:85927
+property_value: skos:narrowMatch RHEA:85931
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22757" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29636" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30578" xsd:anyURI
@@ -485724,6 +486853,7 @@
 synonym: "protein substrate chaperone activity" RELATED []
 synonym: "protein-substrate adaptor activity" BROAD []
 xref: Reactome:R-HSA-202137 "AKT1 binds eNOS complex via HSP90"
+xref: Reactome:R-HSA-5653886 "B4GALT1 binds LALBA"
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22972" xsd:anyURI
 created_by: pg
@@ -486533,6 +487663,7 @@
 name: lactoperoxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2 a phenolic donor + H2O2 = 2 a phenolic radical donor + 2 H2O." [PMID:19339248, RHEA:56136]
+synonym: "heme peroxidase activity" RELATED []
 synonym: "peroxidase activity" BROAD []
 xref: EC:1.11.1.7
 xref: MetaCyc:PEROXID-RXN
@@ -487690,6 +488821,11 @@
 name: halogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: R-CH + a halogen + oxygen donor = R-C-halogen +H2O." [PMID:28466644, PMID:34368824]
+xref: Reactome:R-HSA-209815 "Tyrosine is monoiodinated"
+xref: Reactome:R-HSA-209840 "Two DITs combine to form thyroxine"
+xref: Reactome:R-HSA-209925 "DIT and MIT combine to form triiodothyronine"
+xref: Reactome:R-HSA-209973 "Tyrosine is diiodinated"
+xref: Reactome:R-HSA-350901 "Iodide is organified"
 is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23587" xsd:anyURI
 created_by: pg
@@ -490001,6 +491137,7 @@
 namespace: molecular_function
 def: "Binding to and responding, e.g. by conformational change, to changes in the cellular level of glucose." [PMID:17470517]
 synonym: "glucose sensing activity" EXACT []
+xref: Reactome:R-HSA-170810 "nucleoplasmic GCK1:GCKR complex => glucokinase (GCK1) + glucokinase regulatory protein (GCKR)"
 is_a: GO:0140299 ! molecular sensor activity
 relationship: has_part GO:0005536 ! D-glucose binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25267" xsd:anyURI
@@ -490117,10 +491254,11 @@
 
 [Term]
 id: GO:0141098
-name: tRNA (cytidine(34)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105, RHEA:43084]
 synonym: "tRNA (cytidine 34-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(34)-2'-O)-methyltransferase activity" EXACT []
 xref: MetaCyc:RXN-11860
 xref: RHEA:43084
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
@@ -490134,9 +491272,10 @@
 
 [Term]
 id: GO:0141100
-name: tRNA (guanine(18)-2'-O)-methyltransferase activity
+name: tRNA (guanine(18)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine 18 in tRNA= S-adenosyl-L-homocysteine + 2'-O-methylguanosine 18 in tRNA + H+." [PMID:25404562]
+synonym: "tRNA (guanine(18)-2'-O-)-methyltransferase activity" EXACT []
 synonym: "tRNA (guanosine(18)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.34
 xref: MetaCyc:2.1.1.34-RXN
@@ -490151,10 +491290,11 @@
 
 [Term]
 id: GO:0141101
-name: tRNA(Ser) (uridine(44)-2'-O-)-methyltransferase activity
+name: tRNA(Ser) (uridine(44)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: Catalysis of the reaction: S-adenosyl-L-methionine + uridine44 in tRNASer = 2'-O-methyluridine44 in tRNASer + H+ + S-adenosyl-L-homocysteine." [PMID:18025252, RHEA:43100]
 synonym: "tRNA(Ser) (uracil(44)-2'-O-)-methyltransferase activity" EXACT []
+synonym: "tRNA(Ser) (uridine(44)-2'-O-)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.211
 xref: MetaCyc:RXN-12368
 xref: Reactome:R-HSA-6788707 "TRMT44 2'-O-methylates uridine-44 in tRNA(Ser)"
@@ -490169,9 +491309,10 @@
 
 [Term]
 id: GO:0141102
-name: tRNA (5-carboxymethylaminomethyluridine(34)-2'-O)-methyltransferase activity
+name: tRNA (5-carboxymethylaminomethyluridine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 5-carboxymethylaminomethyluridine34 in tRNA(Leu) + S-adenosyl-L-methionine = 5-carboxymethylaminomethyl-2'-O-methyluridine34 in tRNA(Leu) + H+ + S-adenosyl-L-homocysteine." [PMID:20855540, RHEA:43088]
+synonym: "tRNA (5-carboxymethylaminomethyluridine(34)-2'-O)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytidine(34)/5-carboxymethylaminomethyluridine(34)-2'-O)- methyltransferase activity" EXACT []
 xref: RHEA:43088
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -490360,6 +491501,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: Fe(II)-heme b-[protein] + nitric oxide + O2 = Fe(III)-heme b-[protein] + nitrate." [PMID:20511233]
 comment: Note that this activity is similar to nitric oxide dioxygenase activity ; GO:0008941, but GO:000894 uses NADPH as the electron donor.
+xref: Reactome:R-HSA-5340226 "CYGB dioxygenates NO"
 xref: RHEA:78091
 is_a: GO:0016675 ! oxidoreductase activity, acting on a heme group of donors
 property_value: skos:exactMatch RHEA:78091
@@ -490836,6 +491978,7 @@
 synonym: "cAMP/PKA signaling cassette" EXACT []
 synonym: "PKA signaling" BROAD []
 synonym: "protein kinase A signaling" BROAD []
+xref: Reactome:R-HSA-163358 "PKA-mediated phosphorylation of key metabolic factors"
 is_a: GO:0141124 ! intracellular signaling cassette
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26899" xsd:anyURI
 created_by: pg
@@ -492485,9 +493628,10 @@
 id: GO:0150103
 name: reactive gliosis
 namespace: biological_process
-def: "A neuroinflammatory response, occurring over several days, during which glial cells undergo nonspecific reactive changes in response to damage to the central nervous system (CNS); typically involves the proliferation or hypertrophy of different types of glial cells." [GOC:aruk, GOC:bc, PMID:24462092]
+def: "A coordinated response of central nervous system (CNS) glial cells-including astrocytes, microglia, and NG2 glia-to injury, infection, or disease, characterized by changes in gene expression, morphology, proliferation, and extracellular matrix production. Reactive gliosis can range from transient glial activation to the formation of a persistent glial scar border." [GOC:aruk, GOC:bc, PMID:20012068, PMID:24267886, PMID:24462092, PMID:27049947]
 synonym: "gliosis" BROAD []
 is_a: GO:0150076 ! neuroinflammatory response
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30886" xsd:anyURI
 created_by: bc
 creation_date: 2019-05-21T12:37:31Z
 
@@ -493982,8 +495126,9 @@
 def: "The binding activity of a molecule that brings together an ubiquitin ligase and an ubiquitin ligase-substrate adaptor, permitting those molecules to function in a coordinated way." [PMID:27664236]
 synonym: "core protein activity for the CUL-RING ubiquitin ligase complex" NARROW []
 synonym: "core protein activity for the Cullin-RING ubiquitin ligase complex" NARROW []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24591" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31499" xsd:anyURI
 created_by: rynl
 creation_date: 2022-12-21T22:41:32Z
 
@@ -494013,6 +495158,8 @@
 name: non-canonical inflammasome complex assembly
 namespace: biological_process
 def: "The aggregation, arrangement and bonding together of a set of components to form a non-canonical inflammasome complex." [PMID:27291964]
+xref: Reactome:R-HSA-9948001 "CASP4 inflammasome assembly"
+xref: Reactome:R-HSA-9948011 "CASP5 inflammasome assembly"
 is_a: GO:0065003 ! protein-containing complex assembly
 created_by: rynl
 creation_date: 2023-01-25T22:38:01Z
@@ -495446,6 +496593,7 @@
 def: "Catalysis of the reaction: a di-trans,poly-cis-polyprenol + NAD+ = a di-trans,poly-cis-polyprenal + NADH + H+." [PMID:38821050, RHEA:80719]
 synonym: "polyprenol dehydrogenase activity" BROAD []
 xref: MetaCyc:RXN-24567
+xref: Reactome:R-HSA-9937298 "DHRSX oxidizes polyprenol to polyprenal"
 xref: RHEA:80719
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:broadMatch EC:1.1.1.441
@@ -495478,6 +496626,7 @@
 def: "Catalysis of the reaction: ditrans,polycis-polyprenal+ H+ + NADPH = ditrans,polycis-dolichal+ NADP." [PMID:38821050, RHEA:80727]
 xref: EC:1.3.1.94
 xref: MetaCyc:RXN-9971
+xref: Reactome:R-HSA-9937295 "SRD5A3 reduces polyprenal to dolichal"
 xref: RHEA:80727
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.3.1.94
@@ -495564,10 +496713,11 @@
 
 [Term]
 id: GO:0160206
-name: tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(32)/uridine(32)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(32)/uridine(32) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(32)/2'-O-methyluridine(32) in tRNA + H+ + S-adenosyl-L-homocysteine." [EC:2.1.1.200]
 synonym: "TrMet(Xm32) activity" EXACT []
+synonym: "tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.200
 xref: RHEA:42936
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -496050,6 +497200,7 @@
 name: NADPH-iron-sulfur [2Fe-2S] protein oxidoreductase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2 oxidized [2Fe-2S]-[protein] + NADPH = 2 reduced [2Fe-2S]-[protein] + NADP+ + H+." [RHEA:67716]
+xref: Reactome:R-HSA-2564824 "NADPH reduces NDOR1:CIAPIN1"
 xref: RHEA:67716
 is_a: GO:0016731 ! oxidoreductase activity, acting on iron-sulfur proteins as donors, NAD or NADP as acceptor
 property_value: skos:exactMatch RHEA:67716
@@ -496358,9 +497509,11 @@
 id: GO:0160272
 name: RNA 2',3'-cyclic phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 3'-end 2',3'-cyclophospho-ribonucleotide-RNA + 2 H2O = a 3'-end ribonucleotide-RNA + phosphate + H+." [PMID:32732418]
+def: "Catalysis of the reaction: a 3'-end 2',3'-cyclophospho-ribonucleotide-RNA + 2 H2O = a 3'-end ribonucleotide-RNA + phosphate + H+." [PMID:32732418, RHEA:85011]
+xref: RHEA:85011
 is_a: GO:0016791 ! phosphatase activity
 is_a: GO:0140098 ! catalytic activity, acting on RNA
+property_value: skos:exactMatch RHEA:85011
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28001" xsd:anyURI
 created_by: rynl
 creation_date: 2025-05-28T05:44:27Z
@@ -496369,11 +497522,13 @@
 id: GO:0160273
 name: RNA 2'-phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + a 3'-end 2'-phospho-ribonucleotide-RNA = a 3'-end ribonucleotide-RNA + phosphate." [PMID:36180430]
+def: "Catalysis of the reaction: H2O + a 3'-end 2'-phospho-ribonucleotide-RNA = a 3'-end ribonucleotide-RNA + phosphate." [PMID:36180430, RHEA:85191]
 synonym: "Ribonucleic acid 2'-phosphatase activity" EXACT []
 synonym: "RNA 2'-Pase activity" EXACT []
+xref: RHEA:85191
 is_a: GO:0016791 ! phosphatase activity
 is_a: GO:0140098 ! catalytic activity, acting on RNA
+property_value: skos:exactMatch RHEA:85191
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28002" xsd:anyURI
 created_by: rynl
 creation_date: 2025-05-28T16:38:49Z
@@ -496420,12 +497575,13 @@
 id: GO:0160277
 name: lumenal side of cis-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi cisternae membrane that faces the Golgi lumen and is involved in glycosylation and lumen-facing cargo recognition." [PMID:23913272, PMID:34597626]
+def: "The leaflet of the cis-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where glycosylation and lumen-facing cargo recognition occur." [PMID:23913272, PMID:34597626]
 synonym: "lumenal face of cis-Golgi cisternae membrane" EXACT []
 synonym: "lumenal leaflet of cis-Golgi cisternae membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990674 ! Golgi cis cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30405" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-21T00:29:16Z
 
@@ -496433,12 +497589,13 @@
 id: GO:0160278
 name: cytoplasmic side of cis-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi cisternae membrane that faces the cytoplasm and is the site of interactions with cytosolic proteins, including those involved in vesicle budding, membrane tethering, and lipid or protein trafficking." [PMID:10922460, PMID:34597626, PMID:39331042]
+def: "The leaflet of the cis-Golgi cisternae membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins, including those involved in vesicle budding, membrane tethering, and lipid or protein trafficking." [PMID:10922460, PMID:34597626, PMID:39331042]
 synonym: "cytoplasmic face of cis-Golgi cisternae membrane" EXACT []
 synonym: "cytoplasmic leaflet of cis-Golgi cisternae membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990674 ! Golgi cis cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30406" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T17:19:30Z
 
@@ -496446,12 +497603,13 @@
 id: GO:0160279
 name: cytoplasmic side of medial-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the medial-Golgi cisternae that faces the cytoplasm participates in interactions with cytosolic proteins involved in membrane trafficking, vesicle docking, and cargo sorting." [PMID:28777890, PMID:34597626, PMID:39658747]
+def: "The leaflet of the medial-Golgi cisternae membrane that faces the cytoplasm,including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins involved in membrane trafficking, vesicle docking, and cargo sorting." [PMID:28777890, PMID:34597626, PMID:39658747]
 synonym: "cytoplasmic face of Golgi medial cisterna membrane" EXACT []
 synonym: "cytoplasmic leaflet of Golgi medial cisterna membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990675 ! Golgi medial cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30407" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T21:12:40Z
 
@@ -496459,12 +497617,13 @@
 id: GO:0160280
 name: lumenal side of medial-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the medial-Golgi cisternae that directly contacts the Golgi lumen, hosts glycosyltransferases and enzymes to modify N-linked oligosaccharides by trimming mannose and adding N-acetylglucosamine." [PMID:28777890, PMID:34597626, PMID:36658747, PMID:39331042]
+def: "The leaflet of the medial-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where protein glycosylation occur." [PMID:28777890, PMID:34597626, PMID:36658747, PMID:39331042]
 synonym: "lumenal face of medial-Golgi cisterna membrane" EXACT []
 synonym: "lumenal leaflet of Golgi medial cisterna membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990675 ! Golgi medial cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30418" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T21:36:28Z
 
@@ -496472,13 +497631,14 @@
 id: GO:0160281
 name: cytoplasmic side of trans-Golgi network membrane
 namespace: cellular_component
-def: "The leaflet of the membrane bilayer of the trans-Golgi network faces the cytoplasm and is crucial for lipid and protein sorting and trafficking, as it interacts with cytoplasmic proteins involved in vesicle formation and targeting." [PMID:10922460, PMID:23913272, PMID:34597626]
+def: "The leaflet of the trans-Golgi network membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interaction where cytoplasmic proteins interact with proteins involved in vesicle formation and targeting." [PMID:10922460, PMID:23913272, PMID:34597626]
 synonym: "cytoplasmic face of trans-Golgi network membrane" EXACT []
 synonym: "cytoplasmic leaflet of trans-Golgi network membrane" EXACT []
 synonym: "cytoplasmic side of trans-Golgi network" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0032588 ! trans-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30427" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T23:41:33Z
 
@@ -496486,13 +497646,14 @@
 id: GO:0160282
 name: lumenal side of trans-Golgi network membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi network membrane that faces the Golgi lumen." [PMID:23913272, PMID:34597626, PMID:38307322]
+def: "The leaflet of the trans-Golgi network membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it." [PMID:23913272, PMID:34597626, PMID:38307322]
 synonym: "lumenal face of trans-Golgi network membrane" EXACT []
 synonym: "lumenal leaflet of trans-Golgi network membrane" EXACT []
 synonym: "lumenal side of trans-Golgi network" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0032588 ! trans-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30428" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-23T00:11:57Z
 
@@ -496500,9 +497661,11 @@
 id: GO:0160283
 name: queuine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of queuine from one side of a membrane to the other according to the reaction: queuine(out) = queuine(in)." [PMID:40526720]
+def: "Enables the transfer of queuine from one side of a membrane to the other according to the reaction: queuine(out) = queuine(in)." [PMID:40526720, RHEA:85679]
+xref: RHEA:85679
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0160284 ! queuine import across plasma membrane
+property_value: skos:exactMatch RHEA:85679
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30470" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-24T17:27:38Z
@@ -496522,9 +497685,11 @@
 id: GO:0160286
 name: queuosine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of queuosine from one side of a membrane to the other according to the reaction: queuosine(out) = queuosine(in)." [PMID:40526720]
+def: "Enables the transfer of queuosine from one side of a membrane to the other according to the reaction: queuosine(out) = queuosine(in)." [PMID:40526720, RHEA:85683]
+xref: RHEA:85683
 is_a: GO:0005337 ! nucleoside transmembrane transporter activity
 relationship: part_of GO:0160287 ! queuosine import across plasma membrane
+property_value: skos:exactMatch RHEA:85683
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30470" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T00:30:53Z
@@ -496544,13 +497709,14 @@
 id: GO:0160288
 name: cytoplasmic side of trans-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi cisternae that faces the cytoplasm. It is the site of protein and lipid interaction, vesicle formation, and cargo sorting towards post-Golgi destinations such as endosomes, lysosomes, and the plasma membrane." [PMID:23913272, PMID:34597626, PMID:37566051]
+def: "The leaflet of the trans-Golgi cisternae membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site where vesicle formation, and cargo sorting towards post-Golgi destinations such as endosomes, lysosomes and the plasma membrane occur." [PMID:23913272, PMID:34597626, PMID:37566051]
 synonym: "cytoplasmic face of Golgi trans cisterna membrane" EXACT []
 synonym: "cytoplasmic leaflet of Golgi trans cisterna membrane" EXACT []
 synonym: "cytoplasmic side of Golgi trans cisternae membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990676 ! Golgi trans cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30509" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T05:51:59Z
 
@@ -496558,13 +497724,14 @@
 id: GO:0160289
 name: lumenal side of trans-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi cisternae membrane that faces the Golgi lumen. It is involved in the final steps of glycan processing, proteolytic modification, and cargo maturation prior to sorting at the trans-Golgi network." [PMID:34597626, PMID:38307322, PMID:6121819]
+def: "The leaflet of the trans-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where the final steps of glycan processing, proteolytic modification, and cargo maturation prior to sorting at the trans-Golgi network occur." [PMID:34597626, PMID:38307322, PMID:6121819]
 synonym: "lumenal face of trans-Golgi cisternae membrane" EXACT []
 synonym: "lumenal leaflet of trans-Golgi cisternae membrane" EXACT []
 synonym: "lumenal side of Golgi trans cisternae membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990676 ! Golgi trans cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30510" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T06:11:48Z
 
@@ -496584,10 +497751,12 @@
 id: GO:0160291
 name: phosphatidylinositol-4-phosphate-cholesterol exchange activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(out) + cholesterol(in) = 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(in) + cholesterol(out). This reaction results in the exchange of cholesterol for phosphatidylinositol-4-phosphate (PI(4)P) between membranes." [PMID:24209621, PMID:29596003]
+def: "Catalysis of the reaction: a 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(out) + cholesterol(in) = 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(in) + cholesterol(out). This reaction results in the exchange of cholesterol for phosphatidylinositol-4-phosphate (PI(4)P) between membranes." [PMID:24209621, PMID:29596003, RHEA:84179]
+xref: RHEA:84179
 is_a: GO:0008526 ! phosphatidylinositol transfer activity
 is_a: GO:0120020 ! cholesterol transfer activity
 is_a: GO:7770011 ! lipid exchange activity
+property_value: skos:exactMatch RHEA:84179
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30432" xsd:anyURI
 created_by: rynl
@@ -496663,9 +497832,13 @@
 id: GO:0160298
 name: pyridinol guanylyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + GTP + H+ = guanylylpyridinol + diphosphate." [PMID:24249552]
+def: "Catalysis of the reaction: 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + GTP + H+ = guanylylpyridinol + diphosphate." [PMID:24249552, RHEA:85479]
+xref: EC:2.7.7.109
 xref: KEGG_REACTION:R13518
+xref: RHEA:85479
 is_a: GO:0070568 ! guanylyltransferase activity
+property_value: skos:exactMatch EC:2.7.7.109
+property_value: skos:exactMatch RHEA:85479
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30718" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T00:46:39Z
@@ -496685,9 +497858,13 @@
 id: GO:0160301
 name: guanylylpyridinol adenylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanylylpyridinol + ATP + H+ = guanylylpyridinol-AMP + diphosphate." [PMID:25882909]
+def: "Catalysis of the reaction: guanylylpyridinol + ATP + H+ = guanylylpyridinol-AMP + diphosphate." [PMID:25882909, RHEA:85487]
+xref: EC:2.7.7.110
 xref: KEGG_REACTION:R13520
+xref: RHEA:85487
 is_a: GO:0016779 ! nucleotidyltransferase activity
+property_value: skos:exactMatch EC:2.7.7.110
+property_value: skos:exactMatch RHEA:85487
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30720" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T22:49:37Z
@@ -496696,9 +497873,11 @@
 id: GO:0160302
 name: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol 3-C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol + S-adenosyl-L-methionine = 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + S-adenosyl-L-homocysteine + H+." [PMID:27391308]
+def: "Catalysis of the reaction: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol + S-adenosyl-L-methionine = 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + S-adenosyl-L-homocysteine + H+." [PMID:27391308, RHEA:85483]
 xref: KEGG_REACTION:R13499
+xref: RHEA:85483
 is_a: GO:0008168 ! methyltransferase activity
+property_value: skos:exactMatch RHEA:85483
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30721" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T23:15:13Z
@@ -496721,9 +497900,11 @@
 id: GO:0160304
 name: 4-vinylphenol methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-vinylphenol + S-adenosyl-L-methionine = 4-vinylanisole + S-adenosyl-L-homocysteine + H+." [PMID:40562929]
+def: "Catalysis of the reaction: 4-vinylphenol + S-adenosyl-L-methionine = 4-vinylanisole + S-adenosyl-L-homocysteine + H+." [PMID:40562929, RHEA:86075]
+xref: RHEA:86075
 is_a: GO:0008171 ! O-methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
+property_value: skos:exactMatch RHEA:86075
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30708" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-23T00:16:21Z
@@ -496890,17 +498071,13 @@
 
 [Term]
 id: GO:0160320
-name: protein-tyrosine iodotransferase activity
+name: obsolete protein-tyrosine iodotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O and [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O." [PMID:36537574, PMID:8651700]
-xref: RHEA:48956
-xref: RHEA:48960
-is_a: GO:0140905 ! haloperoxidase activity
-property_value: skos:narrowMatch RHEA:48956
-property_value: skos:narrowMatch RHEA:48960
+def: "OBSOLETE. Catalysis of the reaction: [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O and [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O." [PMID:36537574, PMID:8651700]
+comment: The reason for obsoletion is that the reactions covered by this term (RHEA:48956 and RHEA:48960) are now subsumed by the updated GO:0004447 (iodide peroxidase activity), which covers all 5 thyroid peroxidase reactions.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30596" xsd:anyURI
-created_by: rynl
-creation_date: 2026-01-21T23:56:40Z
+is_obsolete: true
+replaced_by: GO:0004447
 
 [Term]
 id: GO:0170001
@@ -496970,6 +498147,8 @@
 name: endoplasmic reticulum-lipid droplet tether activity
 namespace: molecular_function
 def: "The binding activity of a molecule that brings together a lipid droplet with an endoplasmic reticulum membrane, via membrane lipid binding, to establish membrane contact sites and mediate exchange and communication." [PMID:35389430]
+xref: Reactome:R-HSA-8857686 "FITM2 binds TAGs"
+xref: Reactome:R-HSA-9942190 "FITM1 binds TAGs"
 is_a: GO:0043495 ! protein-membrane adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25428" xsd:anyURI
 created_by: ew
@@ -497135,6 +498314,8 @@
 def: "The chemical reactions and pathways involving an L-amino acid." [GOC:edw]
 synonym: "L-alpha-amino acid metabolism" EXACT []
 synonym: "L-amino acid metabolism" EXACT []
+xref: Reactome:R-HSA-8963693 "Aspartate and asparagine metabolism"
+xref: Reactome:R-HSA-8964539 "Glutamate and glutamine metabolism"
 is_a: GO:1901605 ! alpha-amino acid metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23268" xsd:anyURI
 created_by: ew
@@ -497278,7 +498459,6 @@
 def: "Enables the transfer of isopentenyl pyrophosphate from one side of a membrane to the other." [GOC:ew, PMID:37813972]
 synonym: "IPP transmembrane transport activity" EXACT []
 synonym: "isopentenyl diphosphate transmembrane transport activity" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0015605 ! organophosphate ester transmembrane transporter activity
 is_a: GO:0170055 ! lipid transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26285" xsd:anyURI
@@ -497393,7 +498573,6 @@
 namespace: molecular_function
 def: "Enables the transfer of a lipid from one side of a membrane to the other." [GOC:curators]
 subset: gocheck_do_not_annotate
-is_a: GO:0005319 ! lipid transporter activity
 is_a: GO:0022857 ! transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27252" xsd:anyURI
 created_by: ew
@@ -497479,6 +498658,7 @@
 name: transmembrane transport from lysosomal lumen to cytosol
 namespace: biological_process
 def: "The directed movement of a solute from the lysosomal lumen across the lysosomal membrane and into the cytosol." [PMID:26631267]
+synonym: "export from lysosome lumen to cytosol" EXACT []
 is_a: GO:0007041 ! lysosomal transport
 is_a: GO:0034486 ! vacuolar transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20070" xsd:anyURI
@@ -497517,9 +498697,7 @@
 def: "The chemical reactions and pathways resulting in the formation of arginine (2-amino-5-guanidinopentanoic acid) via the intermediate compound N-acetyl-L-citrulline." [PMID:16585758]
 synonym: "arginine biosynthesis via N-acetyl intermediates" EXACT []
 synonym: "arginine biosynthesis, N-acetylated pathway" EXACT []
-xref: MetaCyc:PWY-5154
 is_a: GO:0006526 ! L-arginine biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY-5154
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29602" xsd:anyURI
 created_by: ew
 creation_date: 2025-03-14T17:04:09Z
@@ -497583,6 +498761,17 @@
 creation_date: 2025-08-12T21:56:57Z
 
 [Term]
+id: GO:0170074
+name: RADAR pathway
+namespace: biological_process
+def: "A quality control pathway that degrades peroxisomal matrix protein receptors when the recycling machinery is blocked. When recycling to the cytosol fails, cargo-free receptors accumulate at the peroxisomal membrane, where they are polyubiquitinated and subsequently degraded by the ubiquitin-proteasome system (UPS)." [PMID:16390998, PMID:17011644, PMID:41076631]
+synonym: "Receptor Accumulation and Degradation in the Absence of Recycling" EXACT []
+is_a: GO:0043161 ! proteasome-mediated ubiquitin-dependent protein catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31570" xsd:anyURI
+created_by: ew
+creation_date: 2026-02-17T22:02:27Z
+
+[Term]
 id: GO:0180000
 name: histone methyltransferase inhibitor activity
 namespace: molecular_function
@@ -497729,6 +498918,7 @@
 def: "Any process involved in transcription termination-coupled 3' processing of RNA polymerase II mRNA transcripts by the 3' end cleavage and addition of a poly(A) tail." [PMID:31499460]
 synonym: "cotranscriptional 3' processing of RNA polymerase II mRNA transcripts" EXACT []
 synonym: "mRNA polyadenylation" RELATED []
+xref: Reactome:R-HSA-9770562 "mRNA Polyadenylation"
 is_a: GO:0031124 ! mRNA 3'-end processing
 is_a: GO:0180012 ! co-transcriptional RNA 3'-end processing, cleavage and polyadenylation pathway
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24921" xsd:anyURI
@@ -497853,7 +499043,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + U6 snRNA = S-adenosyl-L-homocysteine + U6 containing a 3'-terminal 2'-O-methylnucleotide + H+." [PMID:37403782, RHEA:63088]
 xref: RHEA:63088
-is_a: GO:0090486 ! small RNA 2'-O-methyltransferase activity
+is_a: GO:0090486 ! small RNA 2'-O-ribose methyltransferase activity
 property_value: skos:exactMatch RHEA:63088
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25706" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -507294,12 +508484,14 @@
 name: regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 is_a: GO:0009894 ! regulation of catabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0046164 ! alcohol catabolic process
 relationship: regulates GO:0046164 ! alcohol catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28908" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:18Z
 
@@ -507308,6 +508500,7 @@
 name: negative regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of  alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "down-regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "downregulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
@@ -507318,6 +508511,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0046164 ! alcohol catabolic process
 relationship: negatively_regulates GO:0046164 ! alcohol catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:28Z
 
@@ -507326,6 +508520,7 @@
 name: positive regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 synonym: "activation of cellular alcohol catabolic process" NARROW [GOC:TermGenie]
 synonym: "up regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "up-regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
@@ -507337,6 +508532,7 @@
 intersection_of: positively_regulates GO:0046164 ! alcohol catabolic process
 relationship: positively_regulates GO:0046164 ! alcohol catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28908" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:46Z
 
@@ -507432,6 +508628,7 @@
 synonym: "negative regulation of defence response to bacterium" EXACT [GOC:TermGenie]
 synonym: "negative regulation of defense response to bacteria" EXACT [GOC:TermGenie]
 synonym: "negative regulation of defense response to bacterium, incompatible interaction" NARROW []
+xref: Reactome:R-HSA-9968551 "Regulation of GBP-mediated host defense"
 is_a: GO:0002832 ! negative regulation of response to biotic stimulus
 is_a: GO:0031348 ! negative regulation of defense response
 is_a: GO:0032102 ! negative regulation of response to external stimulus
@@ -515476,14 +516673,15 @@
 
 [Term]
 id: GO:1900829
-name: D-tyrosine catabolic process
+name: obsolete D-tyrosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-tyrosine." [GOC:TermGenie, PMID:10766779]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-tyrosine." [GOC:TermGenie, PMID:10766779]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-tyrosine.
 synonym: "D-tyrosine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-tyrosine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-tyrosine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009074 ! aromatic amino acid family catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: se
 creation_date: 2012-06-06T09:27:24Z
 
@@ -515516,14 +516714,15 @@
 
 [Term]
 id: GO:1900832
-name: D-leucine catabolic process
+name: obsolete D-leucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-leucine." [GOC:TermGenie, PMID:10918062]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-leucine." [GOC:TermGenie, PMID:10918062]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-leucine.
 synonym: "D-leucine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-leucine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-leucine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009083 ! branched-chain amino acid catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: se
 creation_date: 2012-06-06T09:37:40Z
 
@@ -519793,16 +520992,15 @@
 
 [Term]
 id: GO:1901053
-name: sarcosine catabolic process
+name: obsolete sarcosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sarcosine." [GOC:TermGenie, GOC:yaf, PMID:17951379]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of sarcosine." [GOC:TermGenie, GOC:yaf, PMID:17951379]
+comment: This term was obsoleted because there is no evidence that a specific pathway for sarcosine degradation exists.
 synonym: "sarcosine breakdown" EXACT [GOC:TermGenie]
 synonym: "sarcosine catabolism" EXACT [GOC:TermGenie]
 synonym: "sarcosine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
-is_a: GO:1901052 ! sarcosine metabolic process
-is_a: GO:1901606 ! alpha-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: yaf
 creation_date: 2012-06-28T04:52:29Z
 
@@ -527843,16 +529041,13 @@
 
 [Term]
 id: GO:1901494
-name: regulation of cysteine metabolic process
+name: obsolete regulation of cysteine metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cysteine metabolic process." [GOC:TermGenie]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of cysteine metabolic process." [GOC:TermGenie]
+comment: This term was obsoleted because it was an unnecessray grouping term.
 synonym: "regulation of cysteine metabolism" EXACT [GOC:TermGenie]
-is_a: GO:0006521 ! regulation of amino acid metabolic process
-is_a: GO:0042762 ! regulation of sulfur metabolic process
-is_a: GO:0062012 ! regulation of small molecule metabolic process
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0006534 ! cysteine metabolic process
-relationship: regulates GO:0006534 ! cysteine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
 created_by: vw
 creation_date: 2012-10-16T14:28:42Z
 
@@ -528870,7 +530065,6 @@
 namespace: biological_process
 def: "OBSOLETE. The chemical reactions and pathways involving organonitrogen compound." [GOC:pr, GOC:TermGenie]
 comment: The reason for obsoletion is that this term was an unnecessary grouping term.
-xref: Reactome:R-HSA-71291 "Metabolism of amino acids and derivatives"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26424" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27194" xsd:anyURI
 is_obsolete: true
@@ -531282,8 +532476,12 @@
 synonym: "L-glutamine biosynthesis" EXACT [GOC:TermGenie]
 synonym: "L-glutamine formation" EXACT [GOC:TermGenie]
 synonym: "L-glutamine synthesis" EXACT [GOC:TermGenie]
+xref: MetaCyc:GLNSYN-PWY
 is_a: GO:0006541 ! glutamine metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
+property_value: skos:broadMatch MetaCyc:PWY-5505
+property_value: skos:narrowMatch MetaCyc:GLNSYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pde
 creation_date: 2012-12-13T23:32:52Z
 
@@ -531296,8 +532494,21 @@
 synonym: "L-isoleucine biosynthesis" EXACT [GOC:TermGenie]
 synonym: "L-isoleucine formation" EXACT [GOC:TermGenie]
 synonym: "L-isoleucine synthesis" EXACT [GOC:TermGenie]
+xref: MetaCyc:ILEUSYN-PWY
+xref: MetaCyc:PWY-5101
+xref: MetaCyc:PWY-5103
+xref: MetaCyc:PWY-5104
+xref: MetaCyc:PWY-5108
+is_a: GO:0006549 ! isoleucine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
-is_a: GO:0009097 ! isoleucine biosynthetic process
+is_a: GO:0009082 ! branched-chain amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ILEUSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5101
+property_value: skos:narrowMatch MetaCyc:PWY-5103
+property_value: skos:narrowMatch MetaCyc:PWY-5104
+property_value: skos:narrowMatch MetaCyc:PWY-5108
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30848" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pde
 creation_date: 2012-12-13T23:32:56Z
 
@@ -533560,6 +534771,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving nicotinate." [GOC:TermGenie, GOC:yaf, UniPathway:UPA00830]
 synonym: "nicotinate metabolism" EXACT [GOC:TermGenie]
+xref: Reactome:R-HSA-196807 "Nicotinate metabolism"
 is_a: GO:0009820 ! alkaloid metabolic process
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 is_a: GO:0072524 ! pyridine-containing compound metabolic process
@@ -537728,15 +538940,15 @@
 
 [Term]
 id: GO:1902079
-name: D-valine catabolic process
+name: obsolete D-valine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-valine." [GOC:TermGenie, PMID:23085840]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-valine." [GOC:TermGenie, PMID:23085840]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-valine.
 synonym: "D-valine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-valine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-valine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009083 ! branched-chain amino acid catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:1902114 ! D-valine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: al
 creation_date: 2013-04-23T21:34:29Z
 
@@ -538446,12 +539658,13 @@
 
 [Term]
 id: GO:1902114
-name: D-valine metabolic process
+name: obsolete D-valine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-valine." [GOC:TermGenie, PMID:23085840]
+def: "OBSOLETE. The chemical reactions and pathways involving D-valine." [GOC:TermGenie, PMID:23085840]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-valine metabolism" EXACT [GOC:TermGenie]
-is_a: GO:0006573 ! valine metabolic process
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: al
 creation_date: 2013-05-13T23:27:23Z
 
@@ -540295,6 +541508,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving ketone body." [GOC:pr, GOC:TermGenie]
 synonym: "ketone body metabolism" EXACT [GOC:TermGenie]
+xref: Reactome:R-HSA-74182 "Ketone body metabolism"
 is_a: GO:0006091 ! generation of precursor metabolites and energy
 is_a: GO:0044281 ! small molecule metabolic process
 is_a: GO:1901568 ! fatty acid derivative metabolic process
@@ -544318,6 +545532,7 @@
 name: chloride transmembrane transport
 namespace: biological_process
 def: "The process in which chloride is transported across a membrane." [GOC:TermGenie, GOC:vw]
+xref: Reactome:R-HSA-426117 "Cation-coupled Chloride cotransporters"
 is_a: GO:0006821 ! chloride transport
 is_a: GO:0098656 ! monoatomic anion transmembrane transport
 created_by: bf
@@ -548947,8 +550162,7 @@
 synonym: "sulfurated eukaryotic molybdenum cofactor(2-) formation" EXACT [GOC:TermGenie]
 synonym: "sulfurated eukaryotic molybdenum cofactor(2-) synthesis" EXACT [GOC:TermGenie]
 xref: MetaCyc:PWY-5963
-is_a: GO:0006793 ! phosphorus metabolic process
-is_a: GO:0009058 ! biosynthetic process
+is_a: GO:0006777 ! Mo-molybdopterin cofactor biosynthetic process
 relationship: has_part GO:0008265 ! molybdenum cofactor sulfurtransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28388" xsd:anyURI
 created_by: dph
@@ -549009,6 +550223,7 @@
 synonym: "Mo-molybdopterin-cytosine-dinucleotide cofactor biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6476
 is_a: GO:0006777 ! Mo-molybdopterin cofactor biosynthetic process
+is_a: GO:0009165 ! nucleotide biosynthetic process
 relationship: has_part GO:0061602 ! molybdenum cofactor cytidylyltransferase activity
 created_by: dph
 creation_date: 2014-03-06T14:00:43Z
@@ -552035,6 +553250,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0046165 ! alcohol biosynthetic process
 relationship: regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:03Z
 
@@ -552043,6 +553259,7 @@
 name: negative regulation of alcohol biosynthetic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of alcohol biosynthetic process." [GO_REF:0000058, GOC:mengo_curators, GOC:TermGenie, PMID:23332010]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of alcohol anabolism" EXACT [GOC:TermGenie]
 synonym: "down regulation of alcohol biosynthesis" EXACT [GOC:TermGenie]
 synonym: "down regulation of alcohol biosynthetic process" EXACT [GOC:TermGenie]
@@ -552074,6 +553291,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0046165 ! alcohol biosynthetic process
 relationship: negatively_regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:13Z
 
@@ -552082,6 +553300,7 @@
 name: positive regulation of alcohol biosynthetic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of alcohol biosynthetic process." [GO_REF:0000058, GOC:mengo_curators, GOC:TermGenie, PMID:23332010]
+subset: gocheck_do_not_annotate
 synonym: "activation of alcohol anabolism" NARROW [GOC:TermGenie]
 synonym: "activation of alcohol biosynthesis" NARROW [GOC:TermGenie]
 synonym: "activation of alcohol biosynthetic process" NARROW [GOC:TermGenie]
@@ -552113,6 +553332,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0046165 ! alcohol biosynthetic process
 relationship: positively_regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:19Z
 
@@ -552160,6 +553380,7 @@
 namespace: molecular_function
 def: "Binding to phosphatidylinositol bisphosphate." [GO_REF:0000067, GOC:bhm, GOC:TermGenie, PMID:18690034]
 comment: An example of this is KCNJ2 in human (P63252) in PMID:18690034 (inferred from direct assay)
+xref: Reactome:R-HSA-8870332 "PTPN13:PLEKHA1,2 bind PI(3,4)P2"
 is_a: GO:0043168 ! anion binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 created_by: bhm
@@ -552349,7 +553570,6 @@
 synonym: "metalloendoproteinase activity involved in amyloid precursor protein degradation" NARROW [GOC:TermGenie]
 synonym: "metalloendoproteinase activity involved in APP catabolic process" NARROW [GOC:TermGenie]
 synonym: "metalloendoproteinase activity involved in APP catabolism" NARROW [GOC:TermGenie]
-xref: Reactome:R-HSA-9010034 "ADAM10:Zn2+:TSPANs cleaves APP(18-770)"
 is_a: GO:0004222 ! metalloendopeptidase activity
 intersection_of: GO:0004222 ! metalloendopeptidase activity
 intersection_of: part_of GO:0042987 ! amyloid precursor protein catabolic process
@@ -552593,7 +553813,6 @@
 synonym: "upregulation of TPK I" RELATED [GOC:TermGenie]
 synonym: "upregulation of TPK II" RELATED [GOC:TermGenie]
 synonym: "upregulation of TTK" RELATED [GOC:TermGenie]
-xref: Reactome:R-HSA-9032845 "Activated NTRK2 signals through CDK5"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22455" xsd:anyURI
 is_obsolete: true
 created_by: sjp
@@ -552700,31 +553919,25 @@
 
 [Term]
 id: GO:1902954
-name: regulation of early endosome to recycling endosome transport
+name: obsolete regulation of early endosome to recycling endosome transport
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
-is_a: GO:0032386 ! regulation of intracellular transport
-is_a: GO:0060627 ! regulation of vesicle-mediated transport
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0061502 ! early endosome to recycling endosome transport
-relationship: regulates GO:0061502 ! early endosome to recycling endosome transport
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
+comment: This term is described by another GO term
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:2001135
 created_by: sjp
 creation_date: 2014-05-02T09:06:23Z
 
 [Term]
 id: GO:1902955
-name: positive regulation of early endosome to recycling endosome transport
+name: obsolete positive regulation of early endosome to recycling endosome transport
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
-synonym: "activation of early endosome to recycling endosome transport" NARROW [GOC:TermGenie]
-synonym: "up regulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-synonym: "up-regulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-synonym: "upregulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-is_a: GO:0032388 ! positive regulation of intracellular transport
-is_a: GO:1902954 ! regulation of early endosome to recycling endosome transport
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0061502 ! early endosome to recycling endosome transport
-relationship: positively_regulates GO:0061502 ! early endosome to recycling endosome transport
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
+comment: This term is described by another GO term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:2001137
 created_by: sjp
 creation_date: 2014-05-02T09:06:29Z
 
@@ -558015,13 +559228,16 @@
 
 [Term]
 id: GO:1903212
-name: protein localization to mating-type region heterochromatin
+name: obsolete protein localization to mating-type region heterochromatin
 namespace: biological_process
-def: "A process in which a protein is transported to, or maintained in, a location within a mating-type region heterochromatin." [GO_REF:0000087, GOC:TermGenie, PMID:18761674]
+def: "OBSOLETE. A process in which a protein is transported to, or maintained in, a location within a mating-type region heterochromatin." [GO_REF:0000087, GOC:TermGenie, PMID:18761674]
+comment: This term was obsoleted because it represents a precomposed term.
 synonym: "protein localisation in mating-type region heterochromatin" EXACT [GOC:TermGenie]
 synonym: "protein localisation to mating-type region heterochromatin" EXACT [GOC:TermGenie]
 synonym: "protein localization in mating-type region heterochromatin" EXACT [GOC:TermGenie]
-is_a: GO:0097355 ! protein localization to heterochromatin
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071168
 created_by: mah
 creation_date: 2014-07-24T09:38:12Z
 
@@ -564778,15 +565994,14 @@
 
 [Term]
 id: GO:1903559
-name: 3-cyano-L-alanine catabolic process
+name: obsolete 3-cyano-L-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of 3-cyano-L-alanine." [GO_REF:0000068, GOC:kmv, GOC:TermGenie, PMID:24100226, PMID:24843024]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 3-cyano-L-alanine." [GO_REF:0000068, GOC:kmv, GOC:TermGenie, PMID:24100226, PMID:24843024]
+comment: This term was obsoleted because there is no evidence that a specific pathway for 3-cyano-L-alanine degradation exists.
 synonym: "3-cyano-L-alanine breakdown" EXACT [GOC:TermGenie]
 synonym: "3-cyano-L-alanine catabolism" EXACT [GOC:TermGenie]
 synonym: "3-cyano-L-alanine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0050899 ! nitrile catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+is_obsolete: true
 created_by: kmv
 creation_date: 2014-10-21T17:13:10Z
 
@@ -589048,6 +590263,8 @@
 name: all-trans-retinol binding
 namespace: molecular_function
 def: "Binding to all-trans-retinol." [GO_REF:0000067, GOC:kmv, GOC:TermGenie, PMID:19828452]
+xref: Reactome:R-HSA-2404135 "RBP4 binds atROL"
+xref: Reactome:R-HSA-975633 "atROL binds to RBP2"
 is_a: GO:0019841 ! retinol binding
 created_by: kmv
 creation_date: 2015-10-27T21:14:47Z
@@ -604651,47 +605868,45 @@
 
 [Term]
 id: GO:1905526
-name: regulation of Golgi lumen acidification
+name: obsolete regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
-is_a: GO:0032847 ! regulation of cellular pH reduction
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0061795 ! Golgi lumen acidification
-relationship: regulates GO:0061795 ! Golgi lumen acidification
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant. Direction is unspecified.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:25Z
 
 [Term]
 id: GO:1905527
-name: negative regulation of Golgi lumen acidification
+name: obsolete negative regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+def: "OBSOLETE. Any process that stops, prevents or reduces the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant.
 synonym: "down regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "down-regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "downregulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "inhibition of Golgi lumen acidification" NARROW [GOC:TermGenie]
-is_a: GO:0032848 ! negative regulation of cellular pH reduction
-is_a: GO:1905526 ! regulation of Golgi lumen acidification
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: negatively_regulates GO:0061795 ! Golgi lumen acidification
-relationship: negatively_regulates GO:0061795 ! Golgi lumen acidification
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:33Z
 
 [Term]
 id: GO:1905528
-name: positive regulation of Golgi lumen acidification
+name: obsolete positive regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant.
 synonym: "activation of Golgi lumen acidification" NARROW [GOC:TermGenie]
 synonym: "up regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "up-regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "upregulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
-is_a: GO:0032849 ! positive regulation of cellular pH reduction
-is_a: GO:1905526 ! regulation of Golgi lumen acidification
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0061795 ! Golgi lumen acidification
-relationship: positively_regulates GO:0061795 ! Golgi lumen acidification
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:41Z
 
@@ -614860,6 +616075,8 @@
 namespace: molecular_function
 def: "Catalysis of the removal of the cap from an unmethylated 5'-end capped RNA resulting in the release of the entire cap structure (GpppN) and a 5' monophosphorylated RNA." [GOC:dgf, PMID:20802481]
 synonym: "G(5')pppN pyrophosphatase activity" RELATED []
+xref: Reactome:R-HSA-9935300 "DXO decaps NAD-capped RNA (NAD-RNA)"
+xref: Reactome:R-HSA-9935319 "DXO decaps incompletely capped RNA (GpppN-RNA)"
 is_a: GO:0016891 ! RNA endonuclease activity producing 5'-phosphomonoesters, hydrolytic mechanism
 created_by: tb
 creation_date: 2013-08-26T17:51:00Z
@@ -621301,9 +622518,10 @@
 id: GO:1990780
 name: cytoplasmic side of dendritic spine plasma membrane
 namespace: cellular_component
-def: "The leaflet of the plasma membrane that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface surrounding a dendritic spine." [PMID:9275233]
+def: "The leaflet of the plasma membrane in the dendritic spine region that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [PMID:9275233]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0032591 ! dendritic spine membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: sl
 creation_date: 2015-06-23T18:35:39Z
 
@@ -621719,8 +622937,11 @@
 synonym: "RNA adenylyltransferase activity" EXACT []
 xref: EC:2.7.7.19
 xref: MetaCyc:POLYNUCLEOTIDE-ADENYLYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-72185 "mRNA polyadenylation"
 xref: Reactome:R-HSA-9833031 "Polyadenylation of respiratory syncytial virus subgenomic positive-sense mRNAs"
+xref: Reactome:R-HSA-9936998 "MTPAP polyadenylates mitochondrial mRNA"
+xref: Reactome:R-HSA-9970189 "PABPN1-stimulated PAPOL synthesizes polyA tail at distal 3'UTR PAS"
+xref: Reactome:R-HSA-9970318 "PABPN1-stimulated PAPOL synthesizes polyA tail at proximal 3'UTR PAS"
+xref: Reactome:R-HSA-9970320 "PABPN1-stimulated PAPOL synthesizes polyA tail at intronic PAS"
 xref: RHEA:11332
 is_a: GO:0070566 ! adenylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.19
@@ -622461,6 +623682,7 @@
 def: "Catalysis of the reaction: a cytidine in 18S rRNA + acetyl-CoA + ATP + H2O = an N(4)-acetylcytidine in 18S rRNA + ADP + phosphate + CoA + H+." [PMID:25402480, RHEA:51424]
 xref: RHEA:51424
 is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140102 ! catalytic activity, acting on a rRNA
 property_value: skos:exactMatch RHEA:51424
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29506" xsd:anyURI
 created_by: al
@@ -622746,7 +623968,8 @@
 name: response to psychosocial stress
 namespace: biological_process
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of exposure to aversive or demanding psychological and social conditions that tax or exceed the behavioral resources of the organism." [PMID:22922217, PMID:26458179]
-is_a: GO:0006950 ! response to stress
+is_a: GO:0033555 ! multicellular organismal response to stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 created_by: sl
 creation_date: 2015-12-07T20:38:35Z
 
@@ -623091,12 +624314,15 @@
 
 [Term]
 id: GO:1990943
-name: mating type region replication fork barrier binding
+name: obsolete mating type region replication fork barrier binding
 namespace: molecular_function
-def: "Binding to the replication fork barrier found in the mating type region of fission yeast." [PMID:18723894]
+def: "OBSOLETE. Binding to the replication fork barrier found in the mating type region of fission yeast." [PMID:18723894]
+comment: This term was obsoleted because it represents a precomposed term.
 synonym: "RTS1 barrier binding" EXACT []
 synonym: "RTS1 element binding" EXACT []
-is_a: GO:0031634 ! replication fork barrier binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0031634
 created_by: mah
 creation_date: 2016-04-06T16:08:43Z
 
@@ -624154,6 +625380,7 @@
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of cell-cell adhesion mediated by cadherin." [GOC:obol]
 xref: Reactome:R-HSA-9764260 "Regulation of Expression and Function of Type II Classical Cadherins"
+xref: Reactome:R-HSA-9764274 "Regulation of Expression and Function of Type I Classical Cadherins"
 is_a: GO:0022407 ! regulation of cell-cell adhesion
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044331 ! cell-cell adhesion mediated by cadherin
@@ -640593,6 +641820,7 @@
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of apoptotic signaling pathway." [GOC:mtg_apoptosis]
 synonym: "regulation of apoptotic signalling pathway" EXACT [GOC:mah]
+xref: Reactome:R-HSA-205025 "NADE modulates death signalling"
 is_a: GO:0009966 ! regulation of signal transduction
 is_a: GO:0042981 ! regulation of apoptotic process
 intersection_of: GO:0065007 ! biological regulation
@@ -641552,15 +642780,15 @@
 
 [Term]
 id: GO:2001297
-name: N(omega)-methyl-L-arginine catabolic process
+name: obsolete N(omega)-methyl-L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of N(omega)-methyl-L-arginine." [GOC:rs, PMID:10510241]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of N(omega)-methyl-L-arginine." [GOC:rs, PMID:10510241]
+comment: The reason for obsoletion is that there is no evidence that this function/process/component exists.
 synonym: "N(omega)-methyl-L-arginine breakdown" EXACT [GOC:obol]
 synonym: "N(omega)-methyl-L-arginine catabolism" EXACT [GOC:obol]
 synonym: "N(omega)-methyl-L-arginine degradation" EXACT [GOC:obol]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: pr
 creation_date: 2012-02-14T03:38:21Z
 
@@ -641578,15 +642806,15 @@
 
 [Term]
 id: GO:2001299
-name: N(omega),N(omega)-dimethyl-L-arginine catabolic process
+name: obsolete N(omega),N(omega)-dimethyl-L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of N(omega),N(omega)-dimethyl-L-arginine, a methyl-L-arginine having two methyl groups both attached to the primary amino moiety of the guanidino group." [GOC:rs, PMID:10510241]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of N(omega),N(omega)-dimethyl-L-arginine, a methyl-L-arginine having two methyl groups both attached to the primary amino moiety of the guanidino group." [GOC:rs, PMID:10510241]
+comment: The reason for obsoletion is that there is no evidence that this function/process/component exists.
 synonym: "N(omega),N(omega)-dimethyl-L-arginine breakdown" EXACT [GOC:obol]
 synonym: "N(omega),N(omega)-dimethyl-L-arginine catabolism" EXACT [GOC:obol]
 synonym: "N(omega),N(omega)-dimethyl-L-arginine degradation" EXACT [GOC:obol]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: pr
 creation_date: 2012-02-14T03:40:03Z
 
@@ -641911,8 +643139,10 @@
 id: GO:7770002
 name: sodium:ammonium:chloride symporter activity
 namespace: molecular_function
-def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + NH4+(out) + Cl-(out) = Na+(in) + NH4+(in) + Cl-(in)." [PMID:12657561]
+def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + NH4+(out) + Cl-(out) = Na+(in) + NH4+(in) + Cl-(in)." [PMID:12657561, RHEA:85511]
+xref: RHEA:85511
 is_a: GO:0015378 ! sodium:chloride symporter activity
+property_value: skos:exactMatch RHEA:85511
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30468" xsd:anyURI
 created_by: dragon-ai-agent
 creation_date: 2025-06-19T23:39:51Z
@@ -641993,9 +643223,11 @@
 id: GO:7770009
 name: L-tryptophan conjugated cholate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cholate + L-tryptophan = L-tryptophocholate + H2O." [PMID:40446798]
+def: "Catalysis of the reaction: cholate + L-tryptophan = L-tryptophocholate + H2O." [PMID:40446798, RHEA:85423]
 xref: CHEBI:234456
+xref: RHEA:85423
 is_a: GO:7770003 ! amino acid conjugated cholate hydrolase activity
+property_value: skos:exactMatch RHEA:85423
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30487" xsd:anyURI
 created_by: dragon-ai-agent
 creation_date: 2025-06-25T00:18:58Z
@@ -642313,6 +643545,196 @@
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30887" xsd:anyURI
 created_by: dragon-ai-agent
 
+[Term]
+id: GO:7770038
+name: melanosomal lumen acidification
+namespace: biological_process
+def: "Any process that reduces the pH of the melanosomal lumen, measured by the concentration of the hydrogen ion." [PMID:23205854, PMID:35469906]
+synonym: "melanosome pH reduction" EXACT []
+is_a: GO:0051452 ! intracellular pH reduction
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28838" xsd:anyURI
+created_by: dragon-ai-agent
+
+[Term]
+id: GO:7770039
+name: melanosomal lumen pH elevation
+namespace: biological_process
+def: "Any process that increases the pH of the melanosomal lumen, measured by the concentration of the hydrogen ion." [PMID:11310796, PMID:27140606, PMID:32966160, PMID:35469906]
+synonym: "melanosome pH elevation" EXACT []
+is_a: GO:0051454 ! intracellular pH elevation
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28838" xsd:anyURI
+created_by: dragon-ai-agent
+
+[Term]
+id: GO:7770040
+name: epiplastidial membrane
+namespace: cellular_component
+def: "The outermost lipid bilayer surrounding complex plastids in organisms that acquired their plastids through secondary endosymbiosis. This membrane is derived from the host cell's endomembrane system and is continuous with the rough endoplasmic reticulum and the outer nuclear envelope. In some species, the cytoplasmic face may be studded with ribosomes." [PMID:15937721, PMID:27179349, PMID:39383259]
+synonym: "cERM" EXACT []
+synonym: "chloroplast ER membrane" EXACT []
+synonym: "epiplastid rough ER" EXACT []
+synonym: "EpM" EXACT []
+synonym: "EPrER" EXACT []
+is_a: GO:0042170 ! plastid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31421" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:11:34Z
+
+[Term]
+id: GO:7770041
+name: periplastidial membrane
+namespace: cellular_component
+def: "The lipid bilayer that forms the second outermost membrane of complex plastids in organisms that acquired their plastids through secondary endosymbiosis. The periplastidial membrane is evolutionarily derived from the plasma membrane of the engulfed algal endosymbiont and together with the outer envelope membrane of the plastid defines the periplastidial compartment." [PMID:15937721, PMID:27179349, PMID:39383259]
+synonym: "PPM" EXACT []
+is_a: GO:0042170 ! plastid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31421" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:11:34Z
+
+[Term]
+id: GO:7770042
+name: bindosome complex
+namespace: cellular_component
+def: "A membrane-associated protein complex found in archaea that consists of sugar binding proteins assembled via a type IV pilin-like assembly system. The complex is located at the cell surface and functions in sugar uptake by facilitating the interaction between solute binding proteins and ABC transporters. The complex forms high molecular mass structures (400-600 kDa) that are integrated within or closely associated with the S-layer." [PMID:21234771, PMID:38877064]
+is_a: GO:0098796 ! membrane protein complex
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31422" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:57:42Z
+
+[Term]
+id: GO:7770043
+name: lipid chaperone activity
+namespace: molecular_function
+def: "Binding to a lipid and delivering it to an acceptor protein, including enzymes, nuclear receptors, and other lipid-binding proteins. The lipid may be presented while bound to the chaperone for enzymatic modification or signaling, or fully transferred to the acceptor protein." [PMID:16216074, PMID:18511927, PMID:25847235, PMID:26658141]
+is_a: GO:0005319 ! lipid carrier activity
+relationship: has_part GO:0008289 ! lipid binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31403" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-30T01:54:24Z
+
+[Term]
+id: GO:7770044
+name: phycoerythrobilin synthase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: biliverdin IXalpha + 2 reduced ferredoxin + 4 H+ = (3Z)-phycoerythrobilin + 2 oxidized ferredoxin." [PMID:18662988, RHEA:25359]
+xref: EC:1.3.7.6
+xref: RHEA:25359
+is_a: GO:0016636 ! oxidoreductase activity, acting on the CH-CH group of donors, iron-sulfur protein as acceptor
+property_value: skos:exactMatch EC:1.3.7.6
+property_value: skos:exactMatch RHEA:25359
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31443" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-30T04:39:58Z
+
+[Term]
+id: GO:7770045
+name: negative regulation of GCN2-mediated signaling
+namespace: biological_process
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of GCN2-mediated signaling." [PMID:23899355]
+is_a: GO:0080135 ! regulation of cellular response to stress
+is_a: GO:1902532 ! negative regulation of intracellular signal transduction
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: negatively_regulates GO:0140469 ! GCN2-mediated signaling
+relationship: negatively_regulates GO:0140469 ! GCN2-mediated signaling
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31453" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-04T18:20:09Z
+
+[Term]
+id: GO:7770046
+name: obsolete N6-methyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6-methyladenosine = ADP + H+ + N6-methyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770047
+name: obsolete N6,N6-dimethyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6,N6-dimethyladenosine = ADP + H+ + N6,N6-dimethyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770048
+name: obsolete N6-isopentenyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6-isopentenyladenosine = ADP + H+ + N6-isopentenyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770049
+name: obsolete N6,N6-dimethyl-AMP deaminase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: H2O + H+ + N6,N6-dimethyl-AMP = IMP + dimethylamine." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770050
+name: obsolete N6-isopentenyl-AMP deaminase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: H2O + H+ + N6-isopentenyl-AMP = IMP + dimethylallylamine." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770051
+name: mitochondrial-lysosomal membrane contact site
+namespace: cellular_component
+def: "An organelle membrane contact site between the mitochondrial outer membrane and the lysosomal membrane." [PMID:29364868, PMID:30898429, PMID:32703809, PMID:33372681, PMID:33753743, PMID:35249745]
+synonym: "lysosome-mitochondrion membrane contact site" EXACT []
+synonym: "mitochondria-lysosome contact site" EXACT []
+synonym: "mitochondria-lysosome MCS" EXACT []
+synonym: "mitochondria-lysosome membrane contact site" EXACT []
+is_a: GO:1990816 ! vacuole-mitochondrion membrane contact site
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31553" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-11T18:32:29Z
+
+[Term]
+id: GO:7770052
+name: endoplasmic reticulum-lysosome membrane contact site
+namespace: cellular_component
+def: "An organelle membrane contact site between the endoplasmic reticulum membrane and the lysosomal membrane." [PMID:31537798, PMID:39370902]
+synonym: "ER-lysosome contact site" EXACT []
+synonym: "ER-lysosome MCS" EXACT []
+synonym: "lysosome-ER membrane contact site" EXACT []
+is_a: GO:0098853 ! endoplasmic reticulum-vacuole membrane contact site
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31541" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-12T00:09:12Z
+
+[Term]
+id: GO:7770053
+name: nucleotide-binding leucine-rich repeat receptor complex
+namespace: cellular_component
+def: "A protein complex that contains one or more nucleotide-binding leucine-rich repeat (NLR) proteins and functions as an intracellular innate immune receptor. Upon activation by pathogen effectors or danger signals, the complex oligomerizes with adaptor and effector proteins to trigger downstream immune responses including programmed cell death." [PMID:26355215, PMID:30948526, PMID:30948527]
+synonym: "NLR complex" EXACT []
+synonym: "NLR receptor complex" EXACT []
+synonym: "NOD-like receptor complex" RELATED []
+is_a: GO:0043235 ! receptor complex
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31604" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-19T17:46:49Z
+
 [Typedef]
 id: ends_during
 name: ends during
```
