# Textual diff for go.obo

```diff
--- go-lastrelease.obo	2026-02-25 05:54:03.008179998 +0000
+++ go.obo	2026-02-25 06:03:09.632118743 +0000
@@ -1,5 +1,5 @@
 format-version: 1.2
-data-version: releases/2026-01-23
+data-version: releases/2026-02-25
 subsetdef: chebi_ph7_3 "Rhea list of ChEBI terms representing the major species at pH 7.3."
 subsetdef: gocheck_do_not_annotate "Term not to be used for direct annotation"
 subsetdef: gocheck_obsoletion_candidate "Terms planned for obsoletion"
@@ -44,7 +44,7 @@
 property_value: has_ontology_root_term GO:0003674
 property_value: has_ontology_root_term GO:0005575
 property_value: has_ontology_root_term GO:0008150
-property_value: owl:versionInfo "2026-01-23" xsd:string
+property_value: owl:versionInfo "2026-02-25" xsd:string
 property_value: terms:license http://creativecommons.org/licenses/by/4.0/
 
 [Term]
@@ -69,7 +69,7 @@
 id: GO:0000003
 name: obsolete reproduction
 namespace: biological_process
-def: "OBSOLETE. The production of new individuals that contain some portion of genetic material inherited from one or more parent organisms." [GOC:go_curators, GOC:isa_complete, GOC:jl, ISBN:0198506732]
+def: "OBSOLETE. The production of new individuals that contain some portion of genetic material inherited from one or more parent organisms." [GOC:curators, GOC:isa_complete, GOC:jl, ISBN:0198506732]
 comment: The reason for obsoletion is that this term is equivalent to reproductive process.
 synonym: "reproductive physiological process" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27054" xsd:anyURI
@@ -213,7 +213,7 @@
 id: GO:0000018
 name: regulation of DNA recombination
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of DNA recombination, a DNA metabolic process in which a new genotype is formed by reassortment of genes resulting in gene combinations different from those that were present in the parents." [GOC:go_curators, ISBN:0198506732]
+def: "Any process that modulates the frequency, rate or extent of DNA recombination, a DNA metabolic process in which a new genotype is formed by reassortment of genes resulting in gene combinations different from those that were present in the parents." [GOC:curators, ISBN:0198506732]
 is_a: GO:0051052 ! regulation of DNA metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006310 ! DNA recombination
@@ -223,7 +223,7 @@
 id: GO:0000019
 name: regulation of mitotic recombination
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of DNA recombination during mitosis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of DNA recombination during mitosis." [GOC:curators]
 synonym: "regulation of recombination within rDNA repeats" NARROW []
 is_a: GO:0000018 ! regulation of DNA recombination
 intersection_of: GO:0065007 ! biological regulation
@@ -234,7 +234,7 @@
 id: GO:0000020
 name: obsolete negative regulation of recombination within rDNA repeats
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:curators, ISBN:0198506732]
 comment: This term was made obsolete because it describes a substrate-specific process.
 synonym: "negative regulation of recombination within rDNA repeats" EXACT []
 is_obsolete: true
@@ -490,7 +490,7 @@
 id: GO:0000048
 name: peptidyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2). This reaction is catalyzed by a ribozyme." [PMID:11433365, PMID:9242921]
+def: "Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2). This reaction is catalyzed by a ribozyme." [EC:2.3.2.12, PMID:11433365, PMID:9242921]
 xref: EC:2.3.2.12
 xref: MetaCyc:PEPTIDYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-156912 "Peptide transfer from P-site tRNA to the A-site tRNA"
@@ -509,6 +509,7 @@
 alt_id: GO:0000946
 def: "Binding to a transfer RNA." [GOC:ai]
 synonym: "base pairing with tRNA" NARROW []
+xref: Reactome:R-HSA-2408509 "Sec-tRNA(Sec) binds to EEFSEC:GTP"
 is_a: GO:0003723 ! RNA binding
 
 [Term]
@@ -661,6 +662,7 @@
 synonym: "histidine/arginine/lysine/ornithine porter activity" RELATED []
 synonym: "L-ornithine transporter activity" BROAD []
 xref: Reactome:R-HSA-70634 "ornithine (cytosolic) + citrulline (mitochondrial) => ornithine (mitochondrial) + citrulline (cytosolic)"
+xref: Reactome:R-HSA-9956519 "SLC25A15 variants don't translocate ornithine and citrulline"
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
 relationship: part_of GO:1903352 ! L-ornithine transmembrane transport
 
@@ -784,7 +786,7 @@
 id: GO:0000079
 name: regulation of cyclin-dependent protein serine/threonine kinase activity
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cyclin-dependent protein serine/threonine kinase activity." [GOC:go_curators, GOC:pr]
+def: "Any process that modulates the frequency, rate or extent of cyclin-dependent protein serine/threonine kinase activity." [GOC:curators, GOC:pr]
 subset: gocheck_obsoletion_candidate
 synonym: "regulation of CDK activity" EXACT []
 synonym: "regulation of cyclin-dependent protein kinase activity" BROAD []
@@ -939,7 +941,7 @@
 id: GO:0000094
 name: obsolete septin assembly and septum formation
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was not defined and the string name implied two separate processes.
 synonym: "septin assembly and septum formation" EXACT []
 is_obsolete: true
@@ -967,19 +969,20 @@
 id: GO:0000096
 name: sulfur amino acid metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving amino acids containing sulfur, comprising cysteine, homocysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways involving amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid metabolism" EXACT []
 synonym: "sulphur amino acid metabolic process" EXACT []
 synonym: "sulphur amino acid metabolism" EXACT []
 xref: Reactome:R-HSA-1614635 "Sulfur amino acid metabolism"
 is_a: GO:0006790 ! sulfur compound metabolic process
 is_a: GO:0019752 ! carboxylic acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000097
 name: sulfur amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the formation of amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid anabolism" EXACT []
 synonym: "sulfur amino acid biosynthesis" EXACT []
 synonym: "sulfur amino acid formation" EXACT []
@@ -989,12 +992,13 @@
 is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0044272 ! sulfur compound biosynthetic process
 is_a: GO:0046394 ! carboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000098
 name: sulfur amino acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of amino acids containing sulfur." [GOC:curators]
 synonym: "sulfur amino acid breakdown" EXACT []
 synonym: "sulfur amino acid catabolism" EXACT []
 synonym: "sulfur amino acid degradation" EXACT []
@@ -1004,6 +1008,7 @@
 is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0044273 ! sulfur compound catabolic process
 is_a: GO:0046395 ! carboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31500" xsd:anyURI
 
 [Term]
 id: GO:0000099
@@ -1083,17 +1088,21 @@
 id: GO:0000105
 name: L-histidine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:curators]
 synonym: "histidine anabolism" EXACT []
 synonym: "histidine biosynthesis" EXACT []
 synonym: "histidine biosynthetic process" BROAD []
 synonym: "histidine formation" EXACT []
 synonym: "histidine synthesis" EXACT []
 xref: MetaCyc:HISTSYN-PWY
+xref: MetaCyc:PWY-5029
 is_a: GO:0006547 ! L-histidine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:HISTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5029
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0000107
@@ -1269,7 +1278,7 @@
 namespace: biological_process
 alt_id: GO:0010553
 alt_id: GO:0045816
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase II." [GOC:go_curators, GOC:txnOH]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase II." [GOC:curators, GOC:txnOH]
 synonym: "down regulation of global transcription from RNA polymerase II promoter" RELATED []
 synonym: "down regulation of transcription from RNA polymerase II promoter" EXACT []
 synonym: "down-regulation of global transcription from RNA polymerase II promoter" RELATED []
@@ -1770,7 +1779,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the hydroxylation of sphingolipid long chain bases." [PMID:9556590]
 comment: The reason for obsoletion is that this term is equivalent to sphingolipid C4-monooxygenase activity.
-xref: Reactome:R-HSA-428260 "DEGS2 oxygenates dihydroceramide"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29717" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0102772
@@ -1816,12 +1824,14 @@
 synonym: "3'-5'-exoribonuclease activity" EXACT []
 xref: Reactome:R-HSA-430028 "Exosome Complex hydrolyzes mRNA by 3' to 5' exoribonuclease digestion"
 xref: Reactome:R-HSA-6791222 "21S pre-rRNA is nucleolytically processed at site E (site2a) to yield 18SE pre-rRNA"
-xref: Reactome:R-HSA-6791227 "47S pre-rRNA is nucleolytically processed at A' (01,A1), site A0, and site 02 (site 6)  to yield 45S pre-rRNA"
 xref: Reactome:R-HSA-9682603 "nsp14 acts as a 3'-to-5' exonuclease to remove misincorporated nucleotides from nascent RNA"
 xref: Reactome:R-HSA-9694632 "nsp14 acts as a 3'-to-5' exonuclease to remove misincorporated nucleotides from nascent RNA"
 xref: Reactome:R-HSA-9822437 "DIS3L2 hydrolyzes uridylated mRNA"
 xref: Reactome:R-HSA-9836585 "SUPV3L1:PNPT1 hydrolyzes mitochondrial RNA to yield 4-5 nucleotide oligoribonucleotides"
 xref: Reactome:R-HSA-9836822 "REXO2 hydrolyzes 4-5 nucleotide RNAs to ribonucleotides"
+xref: Reactome:R-HSA-9933585 "RNA exosome in PAXT:PAPOLG:TENT4A,B:ZC3H3:CBCA:polyadenylated m7GpppN-RNA:PABPN1 hydrolyzes RNA"
+xref: Reactome:R-HSA-9934763 "The RNA exosome in RNA exosome:NEXT:ZC3H18:CBCA:m7GpppN-RNA hydrolyzes RNA"
+xref: Reactome:R-HSA-9935824 "TRAMP:RNA exosome hydrolyzes oligoadenylated pre-rRNA 5' external transcribed spacer (5'ETS)"
 is_a: GO:0008408 ! 3'-5' exonuclease activity
 is_a: GO:0016896 ! RNA exonuclease activity, producing 5'-phosphomonoesters
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28062" xsd:anyURI
@@ -2269,7 +2279,7 @@
 name: meiotic spindle organization
 namespace: biological_process
 alt_id: GO:0043147
-def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the microtubule spindle during a meiotic cell cycle." [GOC:go_curators, GOC:mah]
+def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the microtubule spindle during a meiotic cell cycle." [GOC:curators, GOC:mah]
 synonym: "meiotic spindle organisation" EXACT []
 synonym: "meiotic spindle organization and biogenesis" RELATED [GOC:mah]
 synonym: "meiotic spindle stabilization" RELATED []
@@ -2348,7 +2358,7 @@
 id: GO:0000219
 name: obsolete vacuolar hydrogen-transporting ATPase
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "V-ATPase" EXACT []
 synonym: "vacuolar hydrogen-transporting ATPase" EXACT []
@@ -2798,6 +2808,8 @@
 xref: EC:1.1.1.170
 xref: Reactome:R-HSA-194642 "4-methyl,4-carboxycholesta-8(9),24-dien-3beta-ol is decarboxylated and oxidized to form 4-methylcholesta-8(9),24-dien-3-one"
 xref: Reactome:R-HSA-194718 "4-carboxycholesta-8(9),24-dien-3beta-ol is decarboxylated and oxidized to form cholesta-8(9),24-dien-3-one (zymosterone)"
+xref: Reactome:R-HSA-9945787 "NSDHL decarboxylates 4a-carboxy-5a-cholest-8-ene-3b-ol to 5a-cholest-8-en-3-one"
+xref: Reactome:R-HSA-9945822 "NSDHL decarboxylates 4a-carboxy-4b-methyl-5a-cholest-8-en-3b-ol to 4a-methyl-5a-cholest-8-en-3b-ol"
 xref: RHEA:34771
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.170
@@ -2818,6 +2830,8 @@
 xref: MetaCyc:RXN-13686
 xref: Reactome:R-HSA-194632 "Zymosterone (cholesta-8(9),24-dien-3-one) is reduced to zymosterol (cholesta-8(9),24-dien-3beta-ol)"
 xref: Reactome:R-HSA-194689 "4-methylcholesta-8(9),24-dien-3-one is reduced to 4-methylcholesta-8(9),24-dien-3beta-ol"
+xref: Reactome:R-HSA-9945784 "HSD17B7 reduces 4a-methyl-5a-cholest-8-en-3-one to 4a-methyl-5a-cholest-8-en-3b-ol"
+xref: Reactome:R-HSA-9945804 "HSD17B7 reduces 5a-cholest-8-en-3-one to ZYMSTNL"
 xref: RHEA:18409
 xref: RHEA:34787
 xref: RHEA:36379
@@ -2841,6 +2855,8 @@
 xref: MetaCyc:RXN-13188
 xref: Reactome:R-HSA-194641 "4,4-dimethylcholesta-8(9),24-dien-3beta-ol is oxidized to 4-methyl,4-carboxycholesta-8(9),24-dien-3beta-ol"
 xref: Reactome:R-HSA-194669 "4-methylcholesta-8(9),24-dien-3beta-ol is oxidized to 4-carboxycholesta-8(9),24-dien-3beta-ol"
+xref: Reactome:R-HSA-9947203 "MSMO1 oxidizes 4,4-diMe-5-cholest-8-en-3-ol to 4a-carboxy-4b-me-5a-cholest-8-en-3b-ol"
+xref: Reactome:R-HSA-9947207 "MSMO1 oxidizes 4a-me-5a-cholest-8-en-3b-ol to 4a-carboxy-5a-cholest-8-ene-3b-ol"
 xref: RHEA:55220
 is_a: GO:0016716 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, another compound as one donor, and incorporation of one atom of oxygen
 property_value: skos:exactMatch EC:1.14.18.9
@@ -2872,7 +2888,7 @@
 id: GO:0000257
 name: nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH3. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides." [EC:3.5.5.1, GOC:kd]
+def: "Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH4+. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides." [RHEA:21724]
 synonym: "acetonitrilase activity" RELATED [EC:3.5.5.1]
 synonym: "benzonitrilase activity" RELATED [EC:3.5.5.1]
 synonym: "nitrile aminohydrolase activity" RELATED [EC:3.5.5.1]
@@ -3031,7 +3047,7 @@
 id: GO:0000271
 name: polysaccharide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a polysaccharide, a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of a polysaccharide, a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [GOC:curators]
 synonym: "glycan biosynthesis" EXACT []
 synonym: "glycan biosynthetic process" EXACT []
 synonym: "polysaccharide anabolism" EXACT []
@@ -4113,6 +4129,7 @@
 namespace: molecular_function
 def: "Catalysis of the first transesterification reaction of spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, while it is still joined to the 3' exon." [GOC:krc, ISBN:0879695897, PMID:19239890]
 synonym: "lariat formation, 5'-splice site cleavage" RELATED []
+xref: Reactome:R-HSA-9794542 "Formation of the Spliceosomal C complex containing intron lariat"
 is_a: GO:0140098 ! catalytic activity, acting on RNA
 relationship: part_of GO:0000375 ! RNA splicing, via transesterification reactions
 
@@ -4337,7 +4354,7 @@
 id: GO:0000409
 name: regulation of transcription by galactose
 namespace: biological_process
-def: "Any process involving galactose that modulates the frequency, rate or extent or transcription." [GOC:go_curators]
+def: "Any process involving galactose that modulates the frequency, rate or extent or transcription." [GOC:curators]
 is_a: GO:0006355 ! regulation of DNA-templated transcription
 
 [Term]
@@ -4356,7 +4373,7 @@
 id: GO:0000411
 name: positive regulation of transcription by galactose
 namespace: biological_process
-def: "Any process involving galactose that activates or increases the rate of transcription." [GOC:go_curators]
+def: "Any process involving galactose that activates or increases the rate of transcription." [GOC:curators]
 synonym: "activation of transcription by galactose" NARROW []
 synonym: "stimulation of transcription by galactose" NARROW []
 synonym: "up regulation of transcription by galactose" EXACT []
@@ -5245,7 +5262,7 @@
 id: GO:0000578
 name: embryonic axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of a pattern along a line or a point in an embryo." [GOC:dph, GOC:go_curators, GOC:sdb_2009, GOC:tb]
+def: "The establishment, maintenance and elaboration of a pattern along a line or a point in an embryo." [GOC:curators, GOC:dph, GOC:sdb_2009, GOC:tb]
 synonym: "embryonic axis determination" RELATED []
 is_a: GO:0009798 ! axis specification
 is_a: GO:0009880 ! embryonic pattern specification
@@ -5288,7 +5305,7 @@
 synonym: "endodeoxyribonuclease III" RELATED []
 synonym: "endonuclease III" RELATED []
 synonym: "endonuclease VIII activity" RELATED []
-synonym: "oxidized pyrimidine base lesion DNA N-glycosylase activity" EXACT [GOC:go_curators]
+synonym: "oxidized pyrimidine base lesion DNA N-glycosylase activity" EXACT [GOC:curators]
 synonym: "pyrimidine-specific oxidized base lesion DNA N-glycosylase activity" RELATED []
 xref: Reactome:R-HSA-110224 "Cleavage of thymine glycol by NTHL1 glycosylase"
 xref: Reactome:R-HSA-110226 "Cleavage of cytosine glycol by NTHL1 glycosylase"
@@ -5835,10 +5852,13 @@
 namespace: biological_process
 alt_id: GO:0000218
 alt_id: GO:0030462
-def: "During conjugation with cellular fusion, the process resulting in creating a single cell from complementary mating types. The localized remodeling and dissolution of external protective structures allow the fusion of the plasma membranes and cytoplasmic mixing. An example of this process is found in Saccharomyces cerevisiae." [GOC:elh]
+def: "A reproductive process in a single-celled organism in which the cytoplasm of two mating cells fuse, resulting in the formation of a single cell containing the combined cellular contents." [GOC:vw]
+comment: Cytogamy follows cell recognition and mating projection formation (when present) and precedes or accompanies nuclear fusion (karyogamy).
 synonym: "zygote formation" RELATED []
 is_a: GO:0022413 ! reproductive process in single-celled organism
+is_a: GO:0140253 ! cell-cell fusion
 relationship: part_of GO:0000747 ! conjugation with cellular fusion
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
 
 [Term]
 id: GO:0000756
@@ -5961,14 +5981,12 @@
 
 [Term]
 id: GO:0000768
-name: syncytium formation by plasma membrane fusion
+name: syncytium formation by cell-cell fusion
 namespace: biological_process
 def: "The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual cells." [GOC:mtg_muscle, GOC:tb]
-synonym: "cell fusion" BROAD []
-is_a: GO:0006949 ! syncytium formation
+synonym: "syncytium formation by plasma membrane fusion" EXACT []
 is_a: GO:0140253 ! cell-cell fusion
-intersection_of: GO:0006949 ! syncytium formation
-intersection_of: has_part GO:0045026 ! plasma membrane fusion
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
 
 [Term]
 id: GO:0000769
@@ -5976,10 +5994,8 @@
 namespace: biological_process
 def: "The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by one or more rounds of nuclear division without cytokinesis." [GOC:mah, GOC:tb]
 synonym: "syncytium formation by mitosis without cell division" RELATED [GOC:mah]
-is_a: GO:0006949 ! syncytium formation
-intersection_of: GO:0006949 ! syncytium formation
-intersection_of: has_part GO:0033301 ! cell cycle comprising mitosis without cytokinesis
-relationship: has_part GO:0033301 ! cell cycle comprising mitosis without cytokinesis
+is_a: GO:0033301 ! cell cycle comprising mitosis without cytokinesis
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
 
 [Term]
 id: GO:0000770
@@ -6428,7 +6444,7 @@
 id: GO:0000820
 name: obsolete regulation of glutamine family amino acid metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:curators]
 comment: The reason for obsoletion is that this is an unnecessary grouping term.
 synonym: "regulation of glutamine family amino acid metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
@@ -6438,7 +6454,7 @@
 id: GO:0000821
 name: obsolete regulation of arginine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "regulation of arginine metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30570" xsd:anyURI
@@ -6448,7 +6464,7 @@
 id: GO:0000822
 name: inositol hexakisphosphate binding
 namespace: molecular_function
-def: "Binding to inositol hexakisphosphate." [GOC:go_curators]
+def: "Binding to inositol hexakisphosphate." [GOC:curators]
 synonym: "InsP6 binding" EXACT []
 synonym: "IP6 binding" EXACT []
 is_a: GO:0043168 ! anion binding
@@ -6679,7 +6695,7 @@
 alt_id: GO:0007148
 alt_id: GO:0045790
 alt_id: GO:0045791
-def: "The developmental process in which the size or shape of a cell is generated and organized." [GOC:clt, GOC:dph, GOC:go_curators, GOC:tb]
+def: "The developmental process in which the size or shape of a cell is generated and organized." [GOC:clt, GOC:curators, GOC:dph, GOC:tb]
 subset: goslim_chembl
 subset: goslim_drosophila
 subset: goslim_yeast
@@ -6691,7 +6707,7 @@
 name: obsolete regulation of cell shape during vegetative growth phase
 namespace: biological_process
 alt_id: GO:0090061
-def: "OBSOLETE. Any process that modulates the surface configuration of a cell during the vegetative growth phase. The vegetative growth phase is the growth phase during which single celled organisms reproduce by budding or other asexual methods." [GOC:clt, GOC:go_curators, GOC:vw]
+def: "OBSOLETE. Any process that modulates the surface configuration of a cell during the vegetative growth phase. The vegetative growth phase is the growth phase during which single celled organisms reproduce by budding or other asexual methods." [GOC:clt, GOC:curators, GOC:vw]
 comment: The reason for obsoletion is that biological phases should be captured as extensions.
 synonym: "cell morphogenesis during vegetative growth phase" RELATED [GOC:vw]
 synonym: "cellular morphogenesis during vegetative growth" RELATED [GOC:dph, GOC:tb]
@@ -6702,7 +6718,7 @@
 id: GO:0000904
 name: obsolete cell morphogenesis involved in differentiation
 namespace: biological_process
-def: "OBSOLETE. The change in form (cell shape and size) that occurs when relatively unspecialized cells, e.g. embryonic or regenerative cells, acquire specialized structural and/or functional features that characterize the cells, tissues, or organs of the mature organism or some other relatively stable phase of the organism's life history." [GOC:go_curators]
+def: "OBSOLETE. The change in form (cell shape and size) that occurs when relatively unspecialized cells, e.g. embryonic or regenerative cells, acquire specialized structural and/or functional features that characterize the cells, tissues, or organs of the mature organism or some other relatively stable phase of the organism's life history." [GOC:curators]
 comment: This term is equivalent to cell morphogenesis.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24299" xsd:anyURI
 is_obsolete: true
@@ -7448,6 +7464,7 @@
 synonym: "RNA polymerase II promoter proximal region sequence-specific DNA binding" RELATED []
 synonym: "RNA polymerase II proximal promoter sequence-specific DNA binding" RELATED []
 synonym: "RNA polymerase II upstream activating sequence (UAS) sequence-specific DNA binding" RELATED []
+xref: Reactome:R-HSA-400204 "PPARA binds RXRA"
 is_a: GO:0000977 ! RNA polymerase II transcription regulatory region sequence-specific DNA binding
 is_a: GO:0000987 ! cis-regulatory region sequence-specific DNA binding
 created_by: krc
@@ -7612,7 +7629,6 @@
 def: "Binding to an RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits." [GOC:txnOH]
 synonym: "RNA polymerase II core binding" EXACT []
 synonym: "RNAP II core binding" EXACT []
-is_a: GO:0001099 ! basal RNA polymerase II transcription machinery binding
 is_a: GO:0043175 ! RNA polymerase core enzyme binding
 created_by: krc
 creation_date: 2010-08-11T03:57:27Z
@@ -7631,7 +7647,7 @@
 name: RNA polymerase III general transcription initiation factor activity
 namespace: molecular_function
 alt_id: GO:0001034
-def: "A general transcription initiation factor activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase III. Factors required for RNA polymerase III transcription initiation include TFIIIA, TFIIIB and TFIIIC. RNA polymerase III transcribes genes encoding short RNAs, including tRNAs, 5S rRNA, U6 snRNA, the short ncRNA component of RNases P, the mitochondrial RNA processing (MRP) RNA, the signal recognition particle SRP RNA, and in higher eukaryotes a number of micro and other small RNAs, though there is some variability across species as to whether a given small noncoding RNA is transcribed by RNA polymerase II or RNA polymerase III." [GOC:txnOH-2018, PMID:12381659, PMID:17977614, PMID:20413673, PMID:27068803, Wikipedia:RNA_polymerase_III]
+def: "A general transcription initiation factor activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase III. Factors required for RNA polymerase III transcription initiation include TFIIIA, TFIIIB and TFIIIC. RNA polymerase III transcribes genes encoding short RNAs, including tRNAs, 5S rRNA, U6 snRNA, the short ncRNA component of RNases P, the mitochondrial RNA processing (MRP) RNA, the signal recognition particle SRP RNA, and in higher eukaryotes a number of micro and other small RNAs, though there is some variability across species as to whether a given small noncoding RNA is transcribed by RNA polymerase II or RNA polymerase III." [GOC:txnOH-2018, PMID:12381659, PMID:17977614, PMID:20413673, PMID:27068803, PMID:40762516, Wikipedia:RNA_polymerase_III]
 synonym: "core RNA polymerase III binding transcription factor activity" NARROW []
 synonym: "RNA polymerase III general initiation factor activity" EXACT []
 synonym: "RNA polymerase III transcription factor activity, sequence-specific DNA binding" NARROW []
@@ -7640,6 +7656,7 @@
 is_a: GO:0140223 ! general transcription initiation factor activity
 relationship: part_of GO:0006383 ! transcription by RNA polymerase III
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15789" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15862" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20253" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22622" xsd:anyURI
 created_by: krc
@@ -7747,14 +7764,17 @@
 
 [Term]
 id: GO:0001010
-name: RNA polymerase II sequence-specific DNA-binding transcription factor recruiting activity
+name: obsolete RNA polymerase II sequence-specific DNA-binding transcription factor recruiting activity
 namespace: molecular_function
-def: "The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions." [GOC:txnOH]
+def: "OBSOLETE. The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions." [GOC:txnOH]
+comment: This term was obsoleted because it was not clearly defined.
 synonym: "sequence-specific DNA binding transcription factor recruiting transcription factor activity" EXACT []
 synonym: "transcription factor activity, sequence-specific DNA binding transcription factor recruiting" EXACT []
 synonym: "transcription factor activity, sequence-specific DNA-binding transcription factor recruiting" RELATED []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
-relationship: has_part GO:0061629 ! RNA polymerase II-specific DNA-binding transcription factor binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31456" xsd:anyURI
+is_obsolete: true
+consider: GO:0003712
+consider: GO:0140463
 created_by: krc
 creation_date: 2010-08-18T06:46:33Z
 
@@ -8014,8 +8034,6 @@
 def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
 comment: The reason for obsoletion is that this term is equivalent to DNA-directed 5'-3' RNA polymerase activity.
 synonym: "DNA-directed RNA polymerase activity involved in transcription from RNA polymerase II promoter" EXACT []
-xref: Reactome:R-HSA-6814549 "Pre-snRNA transcript initiation, Integrator binding, LEC binding"
-xref: Reactome:R-HSA-6814559 "Pre-snRNA is elongated and capped with 7-methylguanosine"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23779" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
 is_obsolete: true
@@ -8030,7 +8048,6 @@
 def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase III specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
 comment: The reason for obsoletion is that this term is equivalent to DNA-directed 5'-3' RNA polymerase activity.
 synonym: "DNA-directed RNA polymerase activity involved in transcription from RNA polymerase III promoter" EXACT []
-xref: Reactome:R-HSA-1964482 "RNA polymerase III transcribes microbial dsDNA to dsRNA"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23779" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
 is_obsolete: true
@@ -8393,7 +8410,7 @@
 namespace: molecular_function
 def: "Binding to a basal RNA polymerase II transcription factor, any of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:txnOH, PMID:16858867]
 synonym: "RNA polymerase II basal transcription factor binding" EXACT []
-is_a: GO:0001099 ! basal RNA polymerase II transcription machinery binding
+is_a: GO:0000993 ! RNA polymerase II complex binding
 is_a: GO:0140296 ! general transcription initiation factor binding
 created_by: krc
 creation_date: 2010-10-28T02:30:02Z
@@ -8405,7 +8422,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIA complex, one of the complexes involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
 synonym: "TFIIA-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:37:19Z
 
@@ -8425,7 +8441,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIID complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIID-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:48:33Z
 
@@ -8436,7 +8451,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIE complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIE-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:49:20Z
 
@@ -8447,7 +8461,6 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIF complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIF-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:51:20Z
 
@@ -8458,28 +8471,30 @@
 def: "Binding to a general RNA polymerase II transcription factor belonging to the TFIIH complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II." [GOC:krc, PMID:16858867]
 synonym: "TFIIH-class transcription factor binding" EXACT []
 is_a: GO:0001091 ! RNA polymerase II general transcription initiation factor binding
-is_a: GO:0044877 ! protein-containing complex binding
 created_by: krc
 creation_date: 2010-10-28T02:51:41Z
 
 [Term]
 id: GO:0001098
-name: basal transcription machinery binding
+name: obsolete basal transcription machinery binding
 namespace: molecular_function
-def: "Binding to a component of the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
-comment: Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.
-is_a: GO:0005515 ! protein binding
+def: "OBSOLETE. Binding to a component of the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
+comment: This term was obsoleted because it is an unnecessary grouping term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16062" xsd:anyURI
+is_obsolete: true
 created_by: krc
 creation_date: 2010-11-24T12:50:49Z
 
 [Term]
 id: GO:0001099
-name: basal RNA polymerase II transcription machinery binding
+name: obsolete basal RNA polymerase II transcription machinery binding
 namespace: molecular_function
-def: "Binding to a component of the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
-comment: Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.
+def: "OBSOLETE. Binding to a component of the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
+comment: This term was obsoleted because it is equivalent to RNA polymerase II complex binding ; GO:0000993.
 synonym: "basal RNAP II transcription machinery binding" EXACT []
-is_a: GO:0001098 ! basal transcription machinery binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16062" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0000993
 created_by: krc
 creation_date: 2010-11-24T12:54:33Z
 
@@ -8502,7 +8517,7 @@
 id: GO:0001101
 name: response to acid chemical
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:go_curators, GOC:rn]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:curators, GOC:rn]
 comment: This term should be used to describe a response to a specific acid as a chemical. E.g., if an organism were responding to glutamate, then the response would be glutamate-specific; the organism is actually responding to the chemical structure of the anion portion of the dissociated acid. Note that this term is in the subset of terms that should not be used for direct gene product annotation. Instead, select a child term or, if no appropriate child term exists, please request a new term. Direct annotations to this term may be amended during annotation QC. If annotating experiments where an acid is playing a role as a proton donor, please annotate to GO:0010447 'response to acidic pH' instead.
 subset: gocheck_do_not_annotate
 synonym: "response to acid" BROAD []
@@ -8516,7 +8531,7 @@
 namespace: molecular_function
 def: "Binding to a component of the basal transcription machinery which is composed of a bacterial-type RNA polymerase core enzyme and a sigma factor, the minimal set of factors required for formation of the preinitiation complex (PIC) by a bacterial-type RNA polymerase." [GOC:txnOH]
 synonym: "basal bacterial-type RNA polymerase transcription machinery binding" EXACT []
-is_a: GO:0001098 ! basal transcription machinery binding
+is_a: GO:0070063 ! RNA polymerase binding
 created_by: krc
 creation_date: 2010-11-30T04:26:04Z
 
@@ -8742,14 +8757,15 @@
 
 [Term]
 id: GO:0001139
-name: RNA polymerase II complex recruiting activity
+name: obsolete RNA polymerase II complex recruiting activity
 namespace: molecular_function
-def: "Binding to an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
+def: "OBSOLETE. Binding to an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
+comment: This term was obsoleted because it is now represented by transcription coactivator activity ; GO:0003713.
 synonym: "core RNA polymerase II recruiting transcription factor activity" EXACT []
 synonym: "transcription factor activity, core RNA polymerase II recruiting" RELATED []
-is_a: GO:0000993 ! RNA polymerase II complex binding
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
-relationship: part_of GO:0051123 ! RNA polymerase II preinitiation complex assembly
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31456" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0003713
 created_by: krc
 creation_date: 2011-01-20T05:55:01Z
 
@@ -9365,6 +9381,7 @@
 synonym: "transcriptional repressor activity, RNA polymerase II distal enhancer sequence-specific binding" NARROW []
 synonym: "transcriptional repressor activity, RNA polymerase II proximal promoter sequence-specific DNA binding" NARROW []
 synonym: "transcriptional repressor activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding" EXACT []
+xref: Reactome:R-HSA-8936851 "AHRR binds ARNT"
 is_a: GO:0000981 ! DNA-binding transcription factor activity, RNA polymerase II-specific
 is_a: GO:0001217 ! DNA-binding transcription repressor activity
 intersection_of: GO:0140110 ! transcription regulator activity
@@ -9410,6 +9427,7 @@
 synonym: "transcriptional activator activity, RNA polymerase II proximal promoter sequence-specific DNA binding" NARROW []
 synonym: "transcriptional activator activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding" EXACT []
 synonym: "zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription" NARROW []
+xref: Reactome:R-HSA-8937177 "AHR:TCDD binds ARNT"
 xref: Reactome:R-HSA-9856539 "MLXIPL:MLX binds PKLR gene promoter"
 xref: Reactome:R-HSA-9856546 "MLXIPL:MLX binds FASN gene promoter"
 xref: Reactome:R-HSA-9856548 "MLXIPL:MLX binds ACACB gene promoter"
@@ -9923,7 +9941,7 @@
 id: GO:0001506
 name: obsolete neurotransmitter biosynthetic process and storage
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of neurotransmitters and the storage of the synthesized molecules." [GOC:go_curators, ISBN:0123668387]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of neurotransmitters and the storage of the synthesized molecules." [GOC:curators, ISBN:0123668387]
 comment: This term was made obsolete because it is an amalgamation of its two children.
 synonym: "neurotransmitter anabolism and storage" EXACT []
 synonym: "neurotransmitter biosynthetic process and storage" EXACT []
@@ -9948,7 +9966,7 @@
 id: GO:0001508
 name: action potential
 namespace: biological_process
-def: "A process in which membrane potential cycles through a depolarizing spike, triggered in response to depolarization above some threshold, followed by repolarization. This cycle is driven by the flow of ions through various voltage gated channels with different thresholds and ion specificities." [GOC:dph, GOC:go_curators, GOC:tb, ISBN:978-0-07-139011-8]
+def: "A process in which membrane potential cycles through a depolarizing spike, triggered in response to depolarization above some threshold, followed by repolarization. This cycle is driven by the flow of ions through various voltage gated channels with different thresholds and ion specificities." [GOC:curators, GOC:dph, GOC:tb, ISBN:978-0-07-139011-8]
 comment: Action potentials typically propagate across excitable membranes. This class covers both action potentials that propagate and those that fail to do so.
 is_a: GO:0042391 ! regulation of membrane potential
 
@@ -9975,8 +9993,10 @@
 name: RNA methylation
 namespace: biological_process
 def: "Posttranscriptional addition of a methyl group to either a nucleotide or 2'-O ribose in a polyribonucleotide. Usually uses S-adenosylmethionine as a cofactor." [GOC:hjd, PMID:21823225]
+subset: gocheck_obsoletion_candidate
 is_a: GO:0009451 ! RNA modification
 is_a: GO:0043414 ! macromolecule methylation
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27423" xsd:anyURI
 
 [Term]
 id: GO:0001511
@@ -10046,7 +10066,7 @@
 def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-glucosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-glucosamine 6-sulfate." [GOC:ai, GOC:hjd]
 subset: goslim_chembl
 synonym: "N-acetylglucosamine 6-O-sulphotransferase activity" EXACT []
-xref: Reactome:R-HSA-2046222 "CHST2,5,6 transfer SO4(2-) to GlcNAc residues on keratan-PG to form KSPG"
+xref: Reactome:R-HSA-2046222 "CHST2,3,5,6 transfer sulfate to GlcNAc on keratan chain"
 xref: Reactome:R-HSA-3656269 "Defective CHST6 does not transfer SO4(2-) to GlcNAc residues on keratan-PG"
 xref: Reactome:R-HSA-6786012 "CHST4 transfers SO4(2-) from PAPS to Core 2 mucins"
 is_a: GO:0008146 ! sulfotransferase activity
@@ -10438,7 +10458,7 @@
 id: GO:0001558
 name: regulation of cell growth
 namespace: biological_process
-def: "Any process that modulates the frequency, rate, extent or direction of cell growth." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate, extent or direction of cell growth." [GOC:curators]
 is_a: GO:0040008 ! regulation of growth
 is_a: GO:0051128 ! regulation of cellular component organization
 intersection_of: GO:0065007 ! biological regulation
@@ -10512,7 +10532,7 @@
 id: GO:0001564
 name: obsolete resistance to pathogenic protozoa
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "resistance to pathogenic protozoa" EXACT []
 is_obsolete: true
@@ -10683,7 +10703,7 @@
 id: GO:0001580
 name: detection of chemical stimulus involved in sensory perception of bitter taste
 namespace: biological_process
-def: "The series of events required for a bitter taste stimulus to be received and converted to a molecular signal." [GOC:go_curators]
+def: "The series of events required for a bitter taste stimulus to be received and converted to a molecular signal." [GOC:curators]
 synonym: "bitter taste detection" EXACT []
 synonym: "perception of bitter taste, detection of chemical stimulus" EXACT []
 synonym: "perception of bitter taste, sensory transduction of chemical stimulus" EXACT []
@@ -10700,7 +10720,7 @@
 id: GO:0001581
 name: detection of chemical stimulus involved in sensory perception of sour taste
 namespace: biological_process
-def: "The series of events required for a sour taste stimulus to be received and converted to a molecular signal." [GOC:go_curators]
+def: "The series of events required for a sour taste stimulus to be received and converted to a molecular signal." [GOC:curators]
 synonym: "perception of sour taste, detection of chemical stimulus" EXACT []
 synonym: "perception of sour taste, sensory transduction of chemical stimulus" EXACT []
 synonym: "sensory detection of chemical stimulus during perception of sour taste" EXACT []
@@ -10717,7 +10737,7 @@
 id: GO:0001582
 name: detection of chemical stimulus involved in sensory perception of sweet taste
 namespace: biological_process
-def: "The series of events required for a sweet taste stimulus to be received and converted to a molecular signal." [GOC:go_curators]
+def: "The series of events required for a sweet taste stimulus to be received and converted to a molecular signal." [GOC:curators]
 synonym: "perception of sweet taste, detection of chemical stimulus" EXACT []
 synonym: "perception of sweet taste, sensory transduction of chemical stimulus" EXACT []
 synonym: "sensory detection of chemical stimulus during perception of sweet taste" EXACT []
@@ -10734,7 +10754,7 @@
 id: GO:0001583
 name: obsolete detection of chemical stimulus involved in sensory perception of salty taste
 namespace: biological_process
-def: "OBSOLETE. The series of events required for a salty taste stimulus to be received and converted to a molecular signal." [GOC:go_curators]
+def: "OBSOLETE. The series of events required for a salty taste stimulus to be received and converted to a molecular signal." [GOC:curators]
 comment: This term was obsoleted because it is a pre-composed term that should be captured as a GO-CAM model.
 synonym: "perception of salty taste, detection of chemical stimulus" EXACT []
 synonym: "perception of salty taste, sensory transduction of chemical stimulus" EXACT []
@@ -10999,6 +11019,7 @@
 subset: goslim_chembl
 subset: goslim_generic
 synonym: "viral receptor activity" EXACT []
+xref: Reactome:R-HSA-9922480 "Dengue virion gets internalized"
 is_a: GO:0140272 ! exogenous protein binding
 relationship: part_of GO:0046718 ! symbiont entry into host cell
 
@@ -11288,7 +11309,7 @@
 id: GO:0001655
 name: urogenital system development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the urogenital system over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the urogenital system over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048731 ! system development
 relationship: has_part GO:0072001 ! renal system development
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22178" xsd:anyURI
@@ -11304,7 +11325,7 @@
 id: GO:0001657
 name: ureteric bud development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the ureteric bud over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the ureteric bud over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0072164 ! mesonephric tubule development
 
 [Term]
@@ -11355,6 +11376,7 @@
 synonym: "behavioural fear response" EXACT []
 is_a: GO:0002209 ! behavioral defense response
 is_a: GO:0042596 ! fear response
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 
 [Term]
 id: GO:0001664
@@ -11427,7 +11449,6 @@
 synonym: "ATPase stimulator activity" EXACT []
 xref: Reactome:R-HSA-5251955 "HSP40s activate intrinsic ATPase activity of HSP70s in the nucleoplasm"
 xref: Reactome:R-HSA-5251959 "HSP40s activate intrinsic ATPase activity of HSP70s in the cytosol"
-is_a: GO:0060590 ! ATPase regulator activity
 is_a: GO:0140677 ! molecular function activator activity
 relationship: positively_regulates GO:0140657 ! ATP-dependent activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
@@ -11486,7 +11507,7 @@
 id: GO:0001678
 name: intracellular glucose homeostasis
 namespace: biological_process
-def: "A homeostatic process involved in the maintenance of a steady state level of glucose within a cell." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "A homeostatic process involved in the maintenance of a steady state level of glucose within a cell." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "cell glucose homeostasis" EXACT [GOC:dph, GOC:tb]
 synonym: "cellular glucose homeostasis" EXACT []
 is_a: GO:0042593 ! glucose homeostasis
@@ -11496,7 +11517,7 @@
 id: GO:0001680
 name: tRNA 3'-terminal CCA addition
 namespace: biological_process
-def: "Post-transcriptional addition of the terminal 3' CCA sequence to a tRNA which does not encode this sequence within the primary transcript. CCA addition proceeds by the sequential addition of CTP, CTP, and then ATP to the 3' end of the tRNA, yielding a diphosphate with each nucleotide addition." [GOC:go_curators, PMID:2247609]
+def: "Post-transcriptional addition of the terminal 3' CCA sequence to a tRNA which does not encode this sequence within the primary transcript. CCA addition proceeds by the sequential addition of CTP, CTP, and then ATP to the 3' end of the tRNA, yielding a diphosphate with each nucleotide addition." [GOC:curators, PMID:2247609]
 is_a: GO:0042780 ! tRNA 3'-end processing
 relationship: has_part GO:0004810 ! CCA tRNA nucleotidyltransferase activity
 relationship: has_part GO:0052927 ! CC tRNA cytidylyltransferase activity
@@ -11682,14 +11703,14 @@
 id: GO:0001700
 name: embryonic development via the syncytial blastoderm
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the embryo over time, from zygote formation through syncytial blastoderm to the hatching of the first instar larva. An example of this process is found in Drosophila melanogaster." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of the embryo over time, from zygote formation through syncytial blastoderm to the hatching of the first instar larva. An example of this process is found in Drosophila melanogaster." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0009792 ! embryo development ending in birth or egg hatching
 
 [Term]
 id: GO:0001701
 name: in utero embryonic development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the embryo in the uterus over time, from formation of the zygote in the oviduct, to birth. An example of this process is found in Mus musculus." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of the embryo in the uterus over time, from formation of the zygote in the oviduct, to birth. An example of this process is found in Mus musculus." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0043009 ! chordate embryonic development
 
 [Term]
@@ -11698,7 +11719,7 @@
 namespace: biological_process
 alt_id: GO:0010003
 alt_id: GO:0048276
-def: "A gastrulation process in which the initial invagination becomes the anus and the mouth forms second." [GOC:go_curators, GOC:mtg_sensu]
+def: "A gastrulation process in which the initial invagination becomes the anus and the mouth forms second." [GOC:curators, GOC:mtg_sensu]
 synonym: "deuterostomic gastrulation" EXACT [GOC:dph]
 is_a: GO:0007369 ! gastrulation
 
@@ -11706,7 +11727,7 @@
 id: GO:0001703
 name: gastrulation with mouth forming first
 namespace: biological_process
-def: "A gastrulation process in which the initial invagination becomes the mouth and the anus forms second." [GOC:go_curators, GOC:mtg_sensu]
+def: "A gastrulation process in which the initial invagination becomes the mouth and the anus forms second." [GOC:curators, GOC:mtg_sensu]
 synonym: "protostomic gastrulation" NARROW [GOC:dph]
 is_a: GO:0007369 ! gastrulation
 
@@ -11714,7 +11735,7 @@
 id: GO:0001704
 name: formation of primary germ layer
 namespace: biological_process
-def: "The formation of the ectoderm, mesoderm and endoderm during gastrulation." [GOC:go_curators]
+def: "The formation of the ectoderm, mesoderm and endoderm during gastrulation." [GOC:curators]
 is_a: GO:0048646 ! anatomical structure formation involved in morphogenesis
 relationship: part_of GO:0007369 ! gastrulation
 
@@ -11722,7 +11743,7 @@
 id: GO:0001705
 name: ectoderm formation
 namespace: biological_process
-def: "The formation of ectoderm during gastrulation." [GOC:go_curators]
+def: "The formation of ectoderm during gastrulation." [GOC:curators]
 is_a: GO:0001704 ! formation of primary germ layer
 relationship: part_of GO:0007398 ! ectoderm development
 
@@ -11730,7 +11751,7 @@
 id: GO:0001706
 name: endoderm formation
 namespace: biological_process
-def: "The formation of the endoderm during gastrulation." [GOC:go_curators]
+def: "The formation of the endoderm during gastrulation." [GOC:curators]
 synonym: "endoblast formation" NARROW [GOC:dph, GOC:sdb_2009, GOC:tb]
 xref: Reactome:R-HSA-9823730 "Formation of definitive endoderm"
 is_a: GO:0001704 ! formation of primary germ layer
@@ -11740,7 +11761,7 @@
 id: GO:0001707
 name: mesoderm formation
 namespace: biological_process
-def: "The process that gives rise to the mesoderm. This process pertains to the initial formation of the structure from unspecified parts." [GOC:go_curators]
+def: "The process that gives rise to the mesoderm. This process pertains to the initial formation of the structure from unspecified parts." [GOC:curators]
 is_a: GO:0001704 ! formation of primary germ layer
 relationship: part_of GO:0048332 ! mesoderm morphogenesis
 
@@ -11748,7 +11769,7 @@
 id: GO:0001708
 name: cell fate specification
 namespace: biological_process
-def: "The cellular developmental process involved in cell fate commitment in which the cell is designated to follow a developmental path, unless they receive extrinsic cues that direct an alternative fate." [GOC:go_curators]
+def: "The cellular developmental process involved in cell fate commitment in which the cell is designated to follow a developmental path, unless they receive extrinsic cues that direct an alternative fate." [GOC:curators]
 is_a: GO:0048869 ! cellular developmental process
 relationship: part_of GO:0045165 ! cell fate commitment
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24390" xsd:anyURI
@@ -11767,7 +11788,7 @@
 id: GO:0001710
 name: mesodermal cell fate commitment
 namespace: biological_process
-def: "The cell differentiation process that results in commitment of a cell to become part of the mesoderm." [GOC:go_curators, ISBN:0878932437]
+def: "The cell differentiation process that results in commitment of a cell to become part of the mesoderm." [GOC:curators, ISBN:0878932437]
 synonym: "mesoderm cell fate commitment" EXACT []
 is_a: GO:0060795 ! cell fate commitment involved in formation of primary germ layer
 relationship: part_of GO:0048333 ! mesodermal cell differentiation
@@ -11776,7 +11797,7 @@
 id: GO:0001711
 name: endodermal cell fate commitment
 namespace: biological_process
-def: "The cell differentiation process that results in commitment of a cell to become part of the endoderm." [GOC:go_curators, ISBN:0878932437]
+def: "The cell differentiation process that results in commitment of a cell to become part of the endoderm." [GOC:curators, ISBN:0878932437]
 synonym: "endoderm cell fate commitment" EXACT []
 is_a: GO:0060795 ! cell fate commitment involved in formation of primary germ layer
 relationship: part_of GO:0035987 ! endodermal cell differentiation
@@ -11785,7 +11806,7 @@
 id: GO:0001712
 name: ectodermal cell fate commitment
 namespace: biological_process
-def: "The cell differentiation process that results in commitment of a cell to become part of the ectoderm." [GOC:go_curators, ISBN:0878932437]
+def: "The cell differentiation process that results in commitment of a cell to become part of the ectoderm." [GOC:curators, ISBN:0878932437]
 synonym: "ectoderm cell fate commitment" EXACT []
 is_a: GO:0060795 ! cell fate commitment involved in formation of primary germ layer
 relationship: part_of GO:0001705 ! ectoderm formation
@@ -11795,7 +11816,7 @@
 id: GO:0001713
 name: ectodermal cell fate determination
 namespace: biological_process
-def: "The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an ectoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators, ISBN:0878932437]
+def: "The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an ectoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators, ISBN:0878932437]
 synonym: "ectoderm cell fate determination" EXACT []
 is_a: GO:0001709 ! cell fate determination
 relationship: part_of GO:0001712 ! ectodermal cell fate commitment
@@ -11804,7 +11825,7 @@
 id: GO:0001714
 name: endodermal cell fate specification
 namespace: biological_process
-def: "The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an endoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:go_curators]
+def: "The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an endoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:curators]
 synonym: "endoderm cell fate specification" EXACT []
 is_a: GO:0001708 ! cell fate specification
 relationship: part_of GO:0001711 ! endodermal cell fate commitment
@@ -11813,7 +11834,7 @@
 id: GO:0001715
 name: ectodermal cell fate specification
 namespace: biological_process
-def: "The cell fate determination process that results in a cell becoming becomes capable of differentiating autonomously into an ectoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:go_curators]
+def: "The cell fate determination process that results in a cell becoming becomes capable of differentiating autonomously into an ectoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:curators]
 synonym: "ectoderm cell fate specification" EXACT []
 is_a: GO:0001708 ! cell fate specification
 relationship: part_of GO:0001712 ! ectodermal cell fate commitment
@@ -12094,7 +12115,7 @@
 id: GO:0001742
 name: oenocyte differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires the specialized features of an oenocyte. Oenocytes are large secretory cells found in clusters underlying the epidermis of larval abdominal segments." [GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires the specialized features of an oenocyte. Oenocytes are large secretory cells found in clusters underlying the epidermis of larval abdominal segments." [GOC:curators]
 synonym: "oenocyte cell differentiation" EXACT []
 is_a: GO:0030154 ! cell differentiation
 
@@ -12158,7 +12179,7 @@
 id: GO:0001751
 name: compound eye photoreceptor cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires the specialized features of an eye photoreceptor cell." [GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires the specialized features of an eye photoreceptor cell." [GOC:curators]
 is_a: GO:0001754 ! eye photoreceptor cell differentiation
 relationship: part_of GO:0001745 ! compound eye morphogenesis
 
@@ -12175,7 +12196,7 @@
 id: GO:0001753
 name: obsolete adult eye photoreceptor development (sensu Drosophila)
 namespace: biological_process
-def: "OBSOLETE. Development of a photoreceptor, a receptor that responds to light, in the adult Drosophila eye." [GOC:go_curators]
+def: "OBSOLETE. Development of a photoreceptor, a receptor that responds to light, in the adult Drosophila eye." [GOC:curators]
 comment: This term was made obsolete because eye photoreceptors have completed their development by the time the adult phase begins.
 synonym: "adult eye photoreceptor development (sensu Drosophila)" EXACT []
 is_obsolete: true
@@ -12185,7 +12206,7 @@
 id: GO:0001754
 name: eye photoreceptor cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires the specialized features of a photoreceptor cell, as found in the eye, the primary visual organ of most organisms." [GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires the specialized features of a photoreceptor cell, as found in the eye, the primary visual organ of most organisms." [GOC:curators]
 is_a: GO:0046530 ! photoreceptor cell differentiation
 relationship: part_of GO:0048592 ! eye morphogenesis
 
@@ -12304,7 +12325,7 @@
 id: GO:0001764
 name: neuron migration
 namespace: biological_process
-def: "The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature." [CL:0000540, GOC:go_curators]
+def: "The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature." [CL:0000540, GOC:curators]
 synonym: "neuron chemotaxis" EXACT []
 synonym: "neuron guidance" RELATED []
 synonym: "neuronal migration" EXACT []
@@ -13697,6 +13718,7 @@
 synonym: "UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1->3)-beta-D-galactosyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-xylosyl-proteoglycan 4IV-alpha-N-acetyl-D-glucosaminyltransferase activity" RELATED [EC:2.4.1.223]
 xref: EC:2.4.1.223
 xref: MetaCyc:2.4.1.223-RXN
+xref: Reactome:R-HSA-9953259 "EXTL3 dimer transfers GlcNAc to the GAG linker"
 xref: RHEA:16221
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.223
@@ -15626,7 +15648,7 @@
 name: purine nucleobase binding
 namespace: molecular_function
 def: "Binding to a purine nucleobase, an organic nitrogenous base with a purine skeleton." [GOC:hjd]
-synonym: "purine base binding" EXACT [GOC:go_curators]
+synonym: "purine base binding" EXACT [GOC:curators]
 synonym: "purine binding" RELATED []
 is_a: GO:0002054 ! nucleobase binding
 
@@ -15636,7 +15658,7 @@
 namespace: molecular_function
 def: "Binding to a pyrimidine nucleobase, an organic nitrogenous base with a pyrimidine skeleton." [GOC:hjd]
 synonym: "1,3-diazine binding" NARROW []
-synonym: "pyrimidine base binding" EXACT [GOC:go_curators]
+synonym: "pyrimidine base binding" EXACT [GOC:curators]
 synonym: "pyrimidine binding" RELATED []
 is_a: GO:0002054 ! nucleobase binding
 
@@ -15814,7 +15836,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: all-trans-decaprenyl diphosphate + 4-hydroxybenzoate = 3-decaprenyl-4-hydroxybenzoate + diphosphate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific substrate of 4-hydroxybenzoate polyprenyltransferase activity ; GO:0008412.
-xref: Reactome:R-HSA-2162192 "COQ2 ligates all-E-10PrP2 to PHB"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26830" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008412
@@ -29274,7 +29295,7 @@
 name: keratinocyte development
 namespace: biological_process
 def: "The process whose specific outcome is the progression of a keratinocyte over time, from its formation to the mature structure." [GOC:dph]
-xref: Reactome:R-HSA-9725554 "Differentiation of keratinocytes in interfollicular epidermis in mammalian skin"
+xref: Reactome:R-HSA-9725554 "Differentiation of Keratinocytes in Interfollicular Epidermis in Mammalian Skin"
 is_a: GO:0002064 ! epithelial cell development
 relationship: part_of GO:0030216 ! keratinocyte differentiation
 created_by: dph
@@ -30480,7 +30501,7 @@
 id: GO:0003673
 name: obsolete Gene_Ontology
 namespace: biological_process
-def: "OBSOLETE. A controlled vocabulary that can be applied to all organisms even as knowledge of gene and protein roles in cells is accumulating and changing. GO provides three structured networks of defined terms to describe gene product attributes." [GOC:go_curators]
+def: "OBSOLETE. A controlled vocabulary that can be applied to all organisms even as knowledge of gene and protein roles in cells is accumulating and changing. GO provides three structured networks of defined terms to describe gene product attributes." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "Gene_Ontology" EXACT []
 is_obsolete: true
@@ -30686,7 +30707,7 @@
 namespace: molecular_function
 def: "Binding to double-stranded telomere-associated DNA." [GOC:jl, ISBN:0321000382]
 is_a: GO:0003690 ! double-stranded DNA binding
-is_a: GO:0042162 ! telomeric DNA binding
+is_a: GO:0042162 ! telomeric repeat DNA binding
 
 [Term]
 id: GO:0003692
@@ -30844,6 +30865,7 @@
 namespace: molecular_function
 def: "A nuclear receptor activity regulated by steroid binding and modulating the transcription of specific gene sets transcribed by RNA polymerase II." [GOC:signaling, PMID:14708019]
 synonym: "steroid hormone receptor activity" BROAD []
+xref: Reactome:R-HSA-8963915 "1,25(OH)2D binds VDR"
 is_a: GO:0004879 ! nuclear receptor activity
 relationship: part_of GO:0030518 ! nuclear receptor-mediated steroid hormone signaling pathway
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16717" xsd:anyURI
@@ -30886,7 +30908,7 @@
 synonym: "transcription cofactor activity" EXACT []
 synonym: "transcription coreceptor activity" EXACT []
 synonym: "transcriptional co-regulator" EXACT []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+xref: Reactome:R-HSA-5340251 "NR1H4:DCA,CDCA,LCHA binds RXRA and NCOA1,2"
 is_a: GO:0140110 ! transcription regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15536" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15665" xsd:anyURI
@@ -30897,6 +30919,7 @@
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20962" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26489" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28926" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31471" xsd:anyURI
 created_by: krc
 creation_date: 2010-11-24T03:02:15Z
 
@@ -30931,6 +30954,7 @@
 synonym: "RNA polymerase II transcription co-repressor activity" RELATED []
 synonym: "RNA polymerase II transcription corepressor activity" RELATED []
 synonym: "transcription co-repressor activity" EXACT []
+xref: Reactome:R-HSA-400183 "PPARA:RXRA binds Corepressors of PPARA"
 is_a: GO:0003712 ! transcription coregulator activity
 relationship: part_of GO:0045892 ! negative regulation of DNA-templated transcription
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15665" xsd:anyURI
@@ -31080,7 +31104,7 @@
 name: double-stranded RNA adenosine deaminase activity
 namespace: molecular_function
 alt_id: GO:0003971
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a double-stranded RNA molecule." [GOC:mah, RHEA:10120]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+, in a double-stranded RNA molecule." [GOC:mah, RHEA:10120]
 synonym: "double-stranded RNA specific editase activity" RELATED []
 xref: EC:3.5.4.37
 xref: Reactome:R-HSA-77614 "Deamination at C6 position of adenosine in Editosome (ADAR1)"
@@ -31298,6 +31322,7 @@
 xref: Reactome:R-HSA-8950113 "IL12A binds IL12B"
 xref: Reactome:R-HSA-8950183 "IL12B binds IL23A"
 xref: Reactome:R-HSA-8950456 "IL12B dimerizes"
+xref: Reactome:R-HSA-9760559 "DsbA catalyzes disulfide crosslink formation on STa"
 is_a: GO:0016864 ! intramolecular oxidoreductase activity, transposing S-S bonds
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: skos:exactMatch EC:5.3.4.1
@@ -31598,7 +31623,7 @@
 id: GO:0003793
 name: obsolete defense/immunity protein activity
 namespace: molecular_function
-def: "OBSOLETE. Any activity that plays a role in the defense/immune response of an organism against infection and disease." [GOC:go_curators]
+def: "OBSOLETE. Any activity that plays a role in the defense/immune response of an organism against infection and disease." [GOC:curators]
 comment: This term was made obsolete because it refers to involvement in a biological process.
 synonym: "defence/immunity protein activity" EXACT []
 synonym: "defense/immunity protein activity" EXACT []
@@ -31619,7 +31644,7 @@
 id: GO:0003795
 name: obsolete antimicrobial peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, microbial cells." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, microbial cells." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "antimicrobial peptide activity" EXACT []
 is_obsolete: true
@@ -31653,7 +31678,7 @@
 id: GO:0003797
 name: obsolete antibacterial peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "antibacterial peptide activity" EXACT []
 is_obsolete: true
@@ -31663,7 +31688,7 @@
 id: GO:0003798
 name: obsolete male-specific antibacterial peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells, but which is only expressed in males." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells, but which is only expressed in males." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "male-specific antibacterial peptide activity" EXACT []
 is_obsolete: true
@@ -31673,7 +31698,7 @@
 id: GO:0003799
 name: obsolete antifungal peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, fungal cells." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, fungal cells." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "antifungal peptide activity" EXACT []
 is_obsolete: true
@@ -31875,7 +31900,6 @@
 xref: MetaCyc:2.3.2.13-RXN
 xref: Reactome:R-HSA-140851 "fibrin multimer -> fibrin multimer, crosslinked + NH4+"
 xref: Reactome:R-HSA-6810894 "Envoplakin, periplakin, involucrin, SPR binding mediated by TGM1 crosslinking"
-xref: RESID:AA0124
 xref: RHEA:54816
 is_a: GO:0016755 ! aminoacyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -32173,7 +32197,7 @@
 xref: EC:2.4.1.102
 xref: MetaCyc:2.4.1.102-RXN
 xref: Reactome:R-HSA-914012 "GCNTs transfer GlcNAc from UDP-GlcNAc to Core 1 mucins"
-xref: Reactome:R-HSA-914018 "Addition of GlcNAc to Core 3 forms a Core 4 glycoprotein"
+xref: Reactome:R-HSA-914018 "GCNT3 transfers GlcNAc to Core 3 mucin"
 xref: RHEA:18705
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 is_a: GO:0140103 ! catalytic activity, acting on a glycoprotein
@@ -32315,12 +32339,11 @@
 xref: MetaCyc:2.4.99.4-RXN
 xref: Reactome:R-HSA-1022129 "ST3GAL4 transfers Neu5Ac to terminal Gal of N-glycans"
 xref: Reactome:R-HSA-1912378 "Sialylation of Pre-NOTCH"
-xref: Reactome:R-HSA-2046285 "The keratan chain can be capped by N-acetylneuraminic acid"
+xref: Reactome:R-HSA-2046285 "ST3GAL1-4,6 optionally cap keratan chain"
 xref: Reactome:R-HSA-3656258 "Defective ST3GAL3 does not transfer SA to keratan"
 xref: Reactome:R-HSA-4084984 "ST3GAL1-6 transfer Neu5Ac to terminal Gal (alpha-2,3 link)"
 xref: Reactome:R-HSA-9603987 "ST3GAL3 transfers Neu5Ac to Type 1 chain to form Type 1 MSGG"
 xref: Reactome:R-HSA-9605600 "ST3GAL3,4,6 transfer Neu5Ac to Type 2 chain to form Type 2 MSGG"
-xref: Reactome:R-HSA-981497 "ST3GAL1-4 transfers sialic acid to the T antigen at the alpha 3 position"
 xref: Reactome:R-HSA-9844860 "ST3GAL5 transfers NeuNAc to ceramides"
 xref: Reactome:R-HSA-9845538 "ST3GAL2,3 transfer Neu5Ac to gangliosides"
 xref: RHEA:21616
@@ -32333,7 +32356,7 @@
 id: GO:0003837
 name: beta-ureidopropionase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-carbamoyl-beta-alanine + H2O = beta-alanine + CO2 + NH3." [EC:3.5.1.6]
+def: "Catalysis of the reaction: N-carbamoyl-beta-alanine + H2O = beta-alanine + CO2 + NH4+." [EC:3.5.1.6]
 synonym: "N-carbamoyl-beta-alanine amidohydrolase activity" RELATED [EC:3.5.1.6]
 xref: EC:3.5.1.6
 xref: MetaCyc:BETA-UREIDOPROPIONASE-RXN
@@ -32389,7 +32412,6 @@
 xref: EC:4.3.2.9
 xref: MetaCyc:GAMMA-GLUTAMYLCYCLOTRANSFERASE-RXN
 xref: Reactome:R-HSA-1247922 "GGCT transforms gGluCys to OPRO"
-xref: Reactome:R-HSA-6785928 "CHAC1,2 cleaves GSH to OPRO and CysGly"
 xref: RHEA:20505
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.9
@@ -32455,8 +32477,9 @@
 synonym: "pyrroline-5-carboxylic acid dehydrogenase activity" RELATED [EC:1.2.1.88]
 xref: EC:1.2.1.88
 xref: MetaCyc:RXN-14116
-xref: Reactome:R-HSA-6784399 "ALDH4A1 dimer dehydrogenates 4-OH-L-glutamate semialdehyde to 4-OH-L-glutamate"
+xref: Reactome:R-HSA-6784402 "ALDH4A1 converts 1PYR-3OH-5COOH to 4-OH-L-glutamate"
 xref: Reactome:R-HSA-70679 "ALDH4A1 oxidises L-GluSS to Glu"
+xref: Reactome:R-HSA-9929439 "ALDH4A1 oxidizes 1PYR-3OH-5COOH"
 xref: RHEA:30235
 is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.2.1.88
@@ -32543,7 +32566,6 @@
 synonym: "beta-hydroxysteroid dehydrogenase" BROAD []
 synonym: "corticosteroid 11-reductase" RELATED []
 synonym: "corticosteroid 11beta-dehydrogenase" RELATED []
-xref: Reactome:R-HSA-194023 "HSD11B2,HSD11B1 dimer oxidise CORT to COR"
 xref: Wikipedia:11beta-hydroxysteroid_dehydrogenase
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21915" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28173" xsd:anyURI
@@ -32856,6 +32878,7 @@
 property_value: skos:exactMatch EC:1.1.1.35
 property_value: skos:exactMatch RHEA:22432
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30577" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
 
 [Term]
 id: GO:0003858
@@ -32984,12 +33007,16 @@
 xref: EC:1.2.4.4
 xref: MetaCyc:1.2.4.4-RXN
 xref: RHEA:13457
+xref: RHEA:84639
+xref: RHEA:84643
 is_a: GO:0016624 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, disulfide as acceptor
 property_value: skos:broadMatch EC:1.2.1.25
 property_value: skos:broadMatch MetaCyc:1.2.1.25-RXN
 property_value: skos:exactMatch EC:1.2.4.4
 property_value: skos:exactMatch MetaCyc:1.2.4.4-RXN
 property_value: skos:narrowMatch RHEA:13457
+property_value: skos:narrowMatch RHEA:84639
+property_value: skos:narrowMatch RHEA:84643
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21144" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29951" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30031" xsd:anyURI
@@ -33089,7 +33116,6 @@
 synonym: "gamma-aminobutyric acid transaminase activity" EXACT []
 synonym: "gamma-aminobutyric transaminase activity" EXACT []
 synonym: "glutamate-succinic semialdehyde transaminase activity" EXACT []
-xref: Reactome:R-HSA-916855 "PXLP-K357-ABAT dimer:2Fe-2S transforms GABA to SUCCSA"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28183" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28802" xsd:anyURI
 is_obsolete: true
@@ -33198,7 +33224,7 @@
 synonym: "phosphohexokinase activity" BROAD [EC:2.7.1.11]
 xref: EC:2.7.1.11
 xref: MetaCyc:6PFRUCTPHOS-RXN
-xref: Reactome:R-HSA-70467 "D-fructose 6-phosphate + ATP => D-fructose 1,6-bisphosphate + ADP"
+xref: Reactome:R-HSA-70467 "PFK tetramer phosphorylates Fru(6)P"
 xref: RHEA:16109
 is_a: GO:0008443 ! phosphofructokinase activity
 relationship: part_of GO:0061615 ! glycolytic process through fructose-6-phosphate
@@ -33220,7 +33246,7 @@
 xref: EC:2.7.1.105
 xref: KEGG_REACTION:R02732
 xref: MetaCyc:6-PHOSPHOFRUCTO-2-KINASE-RXN
-xref: Reactome:R-HSA-71802 "D-fructose 6-phosphate + ATP => D-fructose 2,6-bisphosphate + ADP"
+xref: Reactome:R-HSA-71802 "PFKFKB dimer phosphorylates Fru(6)P"
 xref: RHEA:15653
 is_a: GO:0008443 ! phosphofructokinase activity
 property_value: skos:exactMatch EC:2.7.1.105
@@ -33277,7 +33303,7 @@
 id: GO:0003876
 name: AMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: AMP + H2O = IMP + NH3." [EC:3.5.4.6]
+def: "Catalysis of the reaction: AMP + H2O = IMP + NH4+." [RHEA:14777]
 synonym: "5-adenylate deaminase activity" RELATED [EC:3.5.4.6]
 synonym: "5-adenylic acid deaminase activity" RELATED [EC:3.5.4.6]
 synonym: "5-AMP deaminase activity" RELATED [EC:3.5.4.6]
@@ -33357,8 +33383,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00352
 property_value: skos:exactMatch MetaCyc:ATP-CITRATE-PRO-S--LYASE-RXN
 property_value: skos:exactMatch RHEA:21160
-property_value: skos:relatedMatch MetaCyc:PWY-5172
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0003879
@@ -33619,6 +33645,8 @@
 xref: Reactome:R-HSA-69116 "Formation of Okazaki fragments"
 xref: Reactome:R-HSA-73932 "Resynthesis of excised residue by POLB"
 xref: Reactome:R-HSA-9710480 "Decitabine triphosphate incorporates into DNA"
+xref: Reactome:R-HSA-9914364 "POLG in the replisome polymerizes the H strand"
+xref: Reactome:R-HSA-9914498 "POLG in the replisome polymerizes the L strand"
 is_a: GO:0034061 ! DNA polymerase activity
 property_value: skos:exactMatch EC:2.7.7.7
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28442" xsd:anyURI
@@ -33680,10 +33708,13 @@
 xref: Reactome:R-HSA-167121 "Addition of the third nucleotide on the nascent HIV-1 transcript"
 xref: Reactome:R-HSA-167136 "Addition of nucleotides 5 through 9 on the growing HIV-1 transcript"
 xref: Reactome:R-HSA-174425 "The primase component of DNA polymerase:primase synthesizes a 6-10 nucleotide RNA primer on the G strand of the telomere"
+xref: Reactome:R-HSA-1964482 "RNA polymerase III transcribes microbial dsDNA to dsRNA"
 xref: Reactome:R-HSA-203901 "Pol II mediated transcription of microRNA genes"
 xref: Reactome:R-HSA-427366 "Transcription of intergenic spacer of the rRNA gene"
 xref: Reactome:R-HSA-5601926 "RNA polymerase II polymerizes primary piRNA transcript"
 xref: Reactome:R-HSA-6781824 "Active RNA Pol II complex transcribes lesion-containing DNA template"
+xref: Reactome:R-HSA-6814549 "Pre-snRNA transcript initiation, Integrator binding, LEC binding"
+xref: Reactome:R-HSA-6814559 "Pre-snRNA is elongated and capped with 7-methylguanosine"
 xref: Reactome:R-HSA-68913 "The primase component of DNA polymerase:primase synthesizes a 6-10 nucleotide RNA primer at the origin"
 xref: Reactome:R-HSA-74986 "Elongation of pre-rRNA transcript"
 xref: Reactome:R-HSA-75850 "Addition of the third nucleotide on the nascent transcript"
@@ -33693,6 +33724,8 @@
 xref: Reactome:R-HSA-9670149 "TERRA transcription"
 xref: Reactome:R-HSA-9697084 "Defective RpoB in Mtb RNAP transcribes RNA polyanion"
 xref: Reactome:R-HSA-9697085 "RNAP transcribes Mtb RNA polyanion"
+xref: Reactome:R-HSA-9914409 "POLRMT polymerizes RNA primer for mitochondrial L strand replication"
+xref: Reactome:R-HSA-9915448 "POLRMT polymerizes R-loop RNA"
 is_a: GO:0034062 ! 5'-3' RNA polymerase activity
 relationship: part_of GO:0032774 ! RNA biosynthetic process
 property_value: skos:exactMatch EC:2.7.7.6
@@ -33869,6 +33902,7 @@
 synonym: "sealase activity" RELATED [EC:6.5.1.1]
 xref: EC:6.5.1.1
 xref: MetaCyc:DNA-LIGASE-ATP-RXN
+xref: Reactome:R-HSA-9914903 "LIG3 (Ligase III) ligates nascent mitochondrial DNA strands"
 is_a: GO:0003909 ! DNA ligase activity
 property_value: skos:exactMatch EC:6.5.1.1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -33979,6 +34013,7 @@
 synonym: "untwisting enzyme activity" RELATED [EC:5.6.2.1]
 xref: EC:5.6.2.1
 xref: MetaCyc:5.99.1.2-RXN
+xref: Reactome:R-HSA-9914922 "TOP3A (Topoisomerase 3a) decatenates mitochondrial DNA"
 is_a: GO:0003916 ! DNA topoisomerase activity
 property_value: skos:exactMatch EC:5.6.2.1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15575" xsd:anyURI
@@ -34063,7 +34098,7 @@
 id: GO:0003921
 name: GMP synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + XMP + NH4(+) = AMP + diphosphate + GMP + 2H+." [RHEA:18301]
+def: "Catalysis of the reaction: ATP + XMP + NH4+ = AMP + diphosphate + GMP + 2H+." [RHEA:18301]
 xref: MetaCyc:GMP-SYN-NH3-RXN
 xref: RHEA:18301
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
@@ -34195,6 +34230,8 @@
 xref: Reactome:R-HSA-9645598 "RRAGC,D hydrolyzes GTP"
 xref: Reactome:R-HSA-9649736 "RAS intrinsic GTPase activity hydrolyzes GTP to GDP"
 xref: Reactome:R-HSA-983422 "Disassembly of COPII coated vesicle"
+xref: Reactome:R-HSA-9947842 "GBP1 hydrolyzes GTP forming GBP1 oligomers"
+xref: Reactome:R-HSA-9954919 "ABCE1:ATP binds PELO:HBS1L-1:GTP:80S ribosome:non-stop mRNA:peptidyl-tRNA with nascent peptide and HBS1L-1:GDP is released"
 xref: RHEA:19669
 is_a: GO:0017111 ! ribonucleoside triphosphate phosphatase activity
 property_value: skos:exactMatch RHEA:19669
@@ -34223,7 +34260,6 @@
 xref: EC:3.6.5.1
 xref: EC:3.6.5.2
 xref: Reactome:R-HSA-400027 "Gq alpha:G beta:G gamma dissociates to Gq alpha:GTP and G beta:G gamma"
-xref: Reactome:R-HSA-400037 "Gi,Go Heterotrimeric G-protein complex dissociates"
 xref: Reactome:R-HSA-422320 "Heterotrimeric G(s) complex dissociates"
 is_a: GO:0003924 ! GTPase activity
 is_a: GO:0098772 ! molecular function regulator activity
@@ -34429,8 +34465,8 @@
 synonym: "glycosaminoglycan alpha-L-iduronohydrolase activity" RELATED [EC:3.2.1.76]
 xref: EC:3.2.1.76
 xref: MetaCyc:3.2.1.76-RXN
-xref: Reactome:R-HSA-1678716 "IDUA hydrolyses Heparan sulfate chain(1)"
-xref: Reactome:R-HSA-1793186 "IDUA) hydrolyses the unsulfated alpha-L-iduronosidic link in DS"
+xref: Reactome:R-HSA-1678716 "IDUA cleaves iduronate from HS chain"
+xref: Reactome:R-HSA-1793186 "IDUA hydrolyses the unsulfated alpha-L-iduronosidic link in DS"
 xref: Reactome:R-HSA-2090037 "IDUA hydrolyses Heparan sulfate chain(6)"
 xref: Reactome:R-HSA-2206299 "Defective IDUA does not hydrolyse Heparan sulfate chain(6)"
 xref: Reactome:R-HSA-9036037 "Defective IDUA does not hydrolyse Heparan sulfate chain(1)"
@@ -34444,7 +34480,7 @@
 id: GO:0003941
 name: L-serine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine = pyruvate + NH3." [EC:4.3.1.17]
+def: "Catalysis of the reaction: L-serine = pyruvate + NH4+." [RHEA:19169]
 synonym: "L-hydroxyaminoacid dehydratase activity" BROAD []
 synonym: "L-serine ammonia-lyase (pyruvate-forming) activity" RELATED [EC:4.3.1.17]
 synonym: "L-serine deaminase activity" EXACT []
@@ -34454,6 +34490,7 @@
 synonym: "serine deaminase activity" BROAD [EC:4.3.1.17]
 xref: EC:4.3.1.17
 xref: MetaCyc:4.3.1.17-RXN
+xref: Reactome:R-HSA-9929460 "SDS dimers:PXLP convert L-Ser to PYR and NH4+"
 xref: RHEA:19169
 is_a: GO:0016841 ! ammonia-lyase activity
 property_value: skos:exactMatch EC:4.3.1.17
@@ -35091,6 +35128,8 @@
 xref: Reactome:R-HSA-9834719 "Synthesis of negative sense genomic RNA of respiratory syncytial virus"
 xref: Reactome:R-HSA-9834736 "Synthesis of antigenomic RNA of human respiratory syncytial virus"
 xref: Reactome:R-HSA-9837511 "Abortive replication of hRSV A"
+xref: Reactome:R-HSA-9922955 "NS5 synthesizes minus strand genome"
+xref: Reactome:R-HSA-9923068 "NS5 synthesizes plus strand genome"
 is_a: GO:0034062 ! 5'-3' RNA polymerase activity
 relationship: part_of GO:0001172 ! RNA-templated transcription
 property_value: skos:exactMatch EC:2.7.7.48
@@ -35130,7 +35169,7 @@
 alt_id: GO:0052852
 alt_id: GO:0052853
 alt_id: GO:0052854
-def: "Catalysis of the reaction: (S)-2-hydroxy-acid + O2 = 2-oxo acid + hydrogen peroxide." [RHEA:16789]
+def: "Catalysis of the reaction: (S)-2-hydroxy-acid + O2 = 2-oxo acid + H2O2." [RHEA:16789]
 synonym: "glycolate oxidase activity" RELATED []
 synonym: "hydroxy-acid oxidase A activity" NARROW [EC:1.1.3.15]
 synonym: "hydroxy-acid oxidase B activity" NARROW [EC:1.1.3.15]
@@ -35159,7 +35198,6 @@
 name: UDP-N-acetylglucosamine 4-epimerase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine = UDP-N-acetyl-D-galactosamine." [EC:5.1.3.7]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "UDP acetylglucosamine epimerase activity" RELATED [EC:5.1.3.7]
 synonym: "UDP-GlcNAc 4-epimerase activity" RELATED [EC:5.1.3.7]
 synonym: "UDP-N-acetyl-D-glucosamine 4-epimerase activity" RELATED [EC:5.1.3.7]
@@ -35172,9 +35210,9 @@
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
 property_value: skos:exactMatch EC:5.1.3.7
 property_value: skos:exactMatch RHEA:20517
-property_value: skos:relatedMatch MetaCyc:PWY-5512
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0003975
@@ -35327,7 +35365,7 @@
 xref: EC:2.7.7.9
 xref: KEGG_REACTION:R00289
 xref: MetaCyc:GLUC1PURIDYLTRANS-RXN
-xref: Reactome:R-HSA-70286 "UTP + D-glucose 1-phosphate <=> pyrophosphate + UDP-glucose"
+xref: Reactome:R-HSA-70286 "UTP + D-glucose 1-phosphate => pyrophosphate + UDP-glucose"
 xref: RHEA:19889
 is_a: GO:0051748 ! UTP-monosaccharide-1-phosphate uridylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.9
@@ -35369,7 +35407,7 @@
 synonym: "acetyl coenzyme A thiolase activity" RELATED []
 xref: EC:2.3.1.9
 xref: MetaCyc:ACETYL-COA-ACETYLTRANSFER-RXN
-xref: Reactome:R-HSA-70844 "alpha-methyl-acetoacetyl-CoA + CoA => propionyl-CoA + acetyl-CoA"
+xref: Reactome:R-HSA-70844 "alpha-methylacetoacetyl-CoA + CoA => propionyl-CoA + acetyl-CoA"
 xref: Reactome:R-HSA-73916 "2 acetyl-CoA <=> acetoacetyl-CoA + CoA"
 xref: Reactome:R-HSA-74181 "acetoacetyl-CoA + CoA <=> 2 acetyl-CoA"
 xref: Reactome:R-HSA-8848215 "ACAT2 condenses 2 Ac-CoA to form ACA-CoA"
@@ -35432,7 +35470,6 @@
 xref: Reactome:R-HSA-8875071 "ACSS3 ligates CoA to CH3COO-"
 xref: RHEA:23176
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.1
 property_value: skos:exactMatch RHEA:23176
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -35529,10 +35566,10 @@
 
 [Term]
 id: GO:0003992
-name: N2-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity
+name: N2-acetyl-L-ornithine:2-oxoglutarate 5-transaminase activity
 namespace: molecular_function
 alt_id: GO:0047318
-def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-acetyl-L-ornithine = N-acetyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.11, RHEA:18049]
+def: "Catalysis of the reaction: N(2)-acetyl-L-ornithine + 2-oxoglutarate = N-acetyl-L-glutamate 5-semialdehyde + L-glutamate." [RHEA:18049]
 synonym: "2-N-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "acetylornithine 5-aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "acetylornithine aminotransferase activity" BROAD []
@@ -35542,6 +35579,7 @@
 synonym: "N(2)-acetylornithine 5-transaminase activity" RELATED [EC:2.6.1.11]
 synonym: "N-acetylornithine aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "N-acetylornithine-delta-transaminase activity" RELATED [EC:2.6.1.11]
+synonym: "N2-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" EXACT []
 synonym: "N2-acetyl-L-ornithine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.11]
 synonym: "N2-acetylornithine 5-aminotransferase activity" EXACT []
 synonym: "N2-acetylornithine 5-transaminase activity" RELATED [EC:2.6.1.11]
@@ -35554,6 +35592,7 @@
 property_value: skos:exactMatch EC:2.6.1.11
 property_value: skos:exactMatch RHEA:18049
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0003993
@@ -35702,7 +35741,7 @@
 id: GO:0004000
 name: adenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3." [RHEA:24408]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+." [RHEA:24408]
 synonym: "adenosine aminohydrolase activity" RELATED [EC:3.5.4.4]
 synonym: "adenosine deaminase reaction" EXACT []
 xref: MetaCyc:ADENODEAMIN-RXN
@@ -35811,9 +35850,9 @@
 
 [Term]
 id: GO:0004015
-name: adenosylmethionine-8-amino-7-oxononanoate transaminase activity
+name: S-adenosyl-L-methionine:8-amino-7-oxononanoate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 8-amino-7-oxononanoate + S-adenosyl-L-methionine(1+) = 7,8-diaminononanoate + S-adenosyl-4-methylthio-2-oxobutanoate." [EC:2.6.1.62, RHEA:16861]
+def: "Catalysis of the reaction: (8S)-8-amino-7-oxononanoate + S-adenosyl-L-methionine = S-adenosyl-4-methylsulfanyl-2-oxobutanoate + (7R,8S)-7,8-diammoniononanoate." [RHEA:16861]
 synonym: "7,8-diamino-pelargonic acid aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "7,8-diaminonanoate transaminase activity" RELATED [EC:2.6.1.62]
 synonym: "7,8-diaminononanoate aminotransferase activity" RELATED [EC:2.6.1.62]
@@ -35824,6 +35863,7 @@
 synonym: "adenosyl methionine-8-amino-7-oxononanoate transaminase activity" EXACT []
 synonym: "adenosylmethionine--8-amino-7-oxononanoate aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "adenosylmethionine-8-amino-7-oxononanoate aminotransferase activity" EXACT []
+synonym: "adenosylmethionine-8-amino-7-oxononanoate transaminase activity" EXACT []
 synonym: "DAPA aminotransferase activity" RELATED [EC:2.6.1.62]
 synonym: "DAPA transaminase activity" RELATED [EC:2.6.1.62]
 synonym: "diaminopelargonate synthase activity" RELATED [EC:2.6.1.62]
@@ -35836,6 +35876,7 @@
 property_value: skos:exactMatch EC:2.6.1.62
 property_value: skos:exactMatch RHEA:16861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004016
@@ -35959,10 +36000,9 @@
 
 [Term]
 id: GO:0004021
-name: L-alanine:2-oxoglutarate aminotransferase activity
+name: L-alanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-alanine = L-glutamate + pyruvate." [RHEA:19453]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-alanine + 2-oxoglutarate = pyruvate + L-glutamate." [RHEA:19453]
 synonym: "alanine aminotransferase activity" BROAD []
 synonym: "alanine transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "alanine-alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.2]
@@ -35977,6 +36017,7 @@
 synonym: "L-alanine aminotransferase activity" BROAD [EC:2.6.1.2]
 synonym: "L-alanine transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "L-alanine-alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.2]
+synonym: "L-alanine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "pyruvate transaminase activity" BROAD [EC:2.6.1.2]
 synonym: "pyruvate-alanine aminotransferase activity" RELATED [EC:2.6.1.2]
 synonym: "pyruvate-glutamate transaminase activity" RELATED [EC:2.6.1.2]
@@ -35988,13 +36029,14 @@
 xref: Reactome:R-HSA-70523 "PXLP-K314-GPT transaminates L-Ala to form PYR"
 xref: Reactome:R-HSA-70524 "PXLP-K314-GPT transaminates PYR to form L-Ala"
 xref: RHEA:19453
-is_a: GO:0047635 ! alanine-oxo-acid transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.2
 property_value: skos:exactMatch KEGG_REACTION:R00258
 property_value: skos:exactMatch MetaCyc:ALANINE-AMINOTRANSFERASE-RXN
 property_value: skos:exactMatch RHEA:19453
-property_value: skos:relatedMatch MetaCyc:ALANINE-SYN2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004022
@@ -36183,7 +36225,7 @@
 name: aldehyde oxidase activity
 namespace: molecular_function
 alt_id: GO:0050250
-def: "Catalysis of the reaction: an aldehyde + H2O + O2 = a carboxylic acid + hydrogen peroxide." [EC:1.2.3.1]
+def: "Catalysis of the reaction: an aldehyde + O2 + H2O = a carboxylate + H2O2 + H+." [RHEA:16829]
 synonym: "aldehyde:oxygen oxidoreductase activity" RELATED [EC:1.2.3.1]
 synonym: "quinoline oxidase activity" RELATED [EC:1.2.3.1]
 synonym: "retinal oxidase activity" NARROW []
@@ -36218,6 +36260,7 @@
 xref: KEGG_REACTION:R02820
 xref: Reactome:R-HSA-196060 "Reduction of isocaproaldehyde to 4-methylpentan-1-ol"
 xref: Reactome:R-HSA-5652172 "AKR1B1 reduces Glc to D-sorbitol"
+xref: Reactome:R-HSA-9931850 "AKR1B1 reduces galactose to galactitol"
 xref: RHEA:12789
 is_a: GO:0008106 ! alcohol dehydrogenase (NADP+) activity
 property_value: skos:broadMatch EC:1.1.1.21
@@ -36238,26 +36281,6 @@
 synonym: "aldo-keto reductase (NADP+) activity" EXACT []
 synonym: "aldo-keto reductase activity" EXACT []
 synonym: "NADPH-dependent aldo-keto reductase activity" EXACT [GOC:vw]
-xref: Reactome:R-HSA-192033 "4-cholesten-7alpha-ol-3-one is reduced to 5beta-cholestan-7alpha-ol-3-one"
-xref: Reactome:R-HSA-192036 "5Beta-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha, 12alpha-triol"
-xref: Reactome:R-HSA-192067 "4-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholesten-7alpha, 12alpha-diol-3-one"
-xref: Reactome:R-HSA-192160 "5beta-cholestan-7alpha-ol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha-diol"
-xref: Reactome:R-HSA-193746 "4-cholesten-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-7alpha,24(S)-diol-3-one"
-xref: Reactome:R-HSA-193755 "4-cholesten-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,24(S)-triol-3-one"
-xref: Reactome:R-HSA-193758 "5beta-cholestan-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,24(S)-triol"
-xref: Reactome:R-HSA-193781 "5Beta-cholestan-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol"
-xref: Reactome:R-HSA-193800 "5Beta-cholestan-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol"
-xref: Reactome:R-HSA-193821 "4-cholesten-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,27-triol-3-one"
-xref: Reactome:R-HSA-193824 "4-cholesten-7alpha,27-diol-3-one is reduced to 5beta-cholestan-7alpha,27-diol-3-one"
-xref: Reactome:R-HSA-193841 "5beta-cholestan-7alpha,27-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,27-triol"
-xref: Reactome:R-HSA-198845 "CYB5A:heme reduces Asc.- to AscH-"
-xref: Reactome:R-HSA-2855252 "AKRs reduce RBP2:atRAL to RBP2:atROL"
-xref: Reactome:R-HSA-5423637 "AKR dimers reduce AFBDHO to AFBDOH"
-xref: Reactome:R-HSA-5692232 "AKR1A1 oxidises BaPtDHD to BaP-7,8-dione"
-xref: Reactome:R-HSA-9027531 "Dehydrogenase dehydrogenates 13-HDHA to 13-oxo-DHA"
-xref: Reactome:R-HSA-9027562 "Dehydrogenase dehydrogenates 17-HDHA to 17-oxo-DHA"
-xref: Reactome:R-HSA-9027598 "Dehydrogenase dehydrogenates 13(R)-HDPAn-3 to 13-oxo-DPAn-3"
-xref: Reactome:R-HSA-9027600 "Dehydrogenase dehydrogenates 17-HDPAn-3 to 17-oxo-DPAn-3"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27180" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28532" xsd:anyURI
 is_obsolete: true
@@ -36272,6 +36295,8 @@
 synonym: "mutarotase activity" RELATED [EC:5.1.3.3]
 xref: EC:5.1.3.3
 xref: MetaCyc:ALDOSE-1-EPIMERASE-RXN
+xref: Reactome:R-HSA-9931853 "GALM converts beta-D-GAL to alpha-D-GAL"
+xref: Reactome:R-HSA-9931926 "Defective GALM doesn't convert beta-D-GAL to alpha-D-GAL"
 xref: RHEA:10264
 xref: RHEA:28675
 xref: RHEA:63332
@@ -36377,7 +36402,7 @@
 
 [Term]
 id: GO:0004042
-name: L-glutamate N-acetyltransferase activity
+name: L-glutamate N-acetyltransferase activity, acting on acetyl-CoA as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + acetyl-CoA = N-acetyl-L-glutamate + CoA + H+." [RHEA:24292]
 synonym: "acetyl-CoA:L-glutamate N-acetyltransferase activity" EXACT []
@@ -36385,14 +36410,17 @@
 synonym: "AGAS" RELATED [EC:2.3.1.1]
 synonym: "amino acid acetyltransferase activity" RELATED [EC:2.3.1.1]
 synonym: "amino-acid N-acetyltransferase activity" BROAD [EC:2.3.1.1]
+synonym: "L-glutamate N-acetyltransferase activity" BROAD []
 synonym: "N-acetylglutamate synthase activity" BROAD [EC:2.3.1.1]
 xref: KEGG_REACTION:R00259
 xref: MetaCyc:N-ACETYLTRANSFER-RXN
 xref: Reactome:R-HSA-70542 "glutamate + acetyl CoA => N-acetyl glutamate + CoA"
+xref: Reactome:R-HSA-9955697 "NAGS variants don't synthesize N-acetylglutamate"
 xref: RHEA:24292
 is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:broadMatch EC:2.3.1.1
 property_value: skos:exactMatch RHEA:24292
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31438" xsd:anyURI
 
 [Term]
 id: GO:0004043
@@ -36464,6 +36492,8 @@
 synonym: "N-substituted aminoacyl transfer RNA hydrolase activity" RELATED [EC:3.1.1.29]
 xref: EC:3.1.1.29
 xref: MetaCyc:AMINOCYL-TRNA-HYDROLASE-RXN
+xref: Reactome:R-HSA-9937374 "MTRFR:MTRES1:39S ribosomal subunit:mRNA:peptidyl-tRNA:MALSU1:MIEF1:NDUFAB1 dissociates"
+xref: Reactome:R-HSA-9937719 "ICT1 hydrolyzes the peptidyl-tRNA bond of the nascent polypeptide"
 xref: RHEA:54448
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -36658,6 +36688,7 @@
 xref: MetaCyc:ARGINASE-RXN
 xref: Reactome:R-HSA-452036 "arginine + H2O => ornithine + urea [ARG2]"
 xref: Reactome:R-HSA-70569 "arginine + H2O => ornithine + urea [ARG1]"
+xref: Reactome:R-HSA-9956512 "ARG1 variants don't synthesize urea and ornithine"
 xref: RHEA:20569
 is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
 property_value: skos:exactMatch EC:3.5.3.1
@@ -36697,6 +36728,7 @@
 xref: EC:6.3.4.5
 xref: MetaCyc:ARGSUCCINSYN-RXN
 xref: Reactome:R-HSA-70577 "ASS1 tetramer:NMRAL1 dimer:NADPH transforms L-Asp and L-Cit to ARSUA"
+xref: Reactome:R-HSA-9956517 "ASS1 variants don't synthesize arginosuccinate"
 xref: RHEA:10932
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
 property_value: skos:exactMatch EC:6.3.4.5
@@ -36718,6 +36750,7 @@
 xref: EC:4.3.2.1
 xref: MetaCyc:ARGSUCCINLYA-RXN
 xref: Reactome:R-HSA-70573 "argininosuccinate <=> fumarate + arginine"
+xref: Reactome:R-HSA-9956524 "ASL variants don't synthesize fumarate and arginine"
 xref: RHEA:24020
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.1
@@ -36996,7 +37029,6 @@
 name: aspartate 1-decarboxylase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-aspartate = beta-alanine + CO2." [EC:4.1.1.11]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "aspartate alpha-decarboxylase activity" RELATED [EC:4.1.1.11]
 synonym: "aspartic alpha-decarboxylase" BROAD [EC:4.1.1.11]
 synonym: "L-aspartate 1-carboxy-lyase (beta-alanine-forming)" RELATED [EC:4.1.1.11]
@@ -37009,15 +37041,14 @@
 property_value: skos:exactMatch EC:4.1.1.11
 property_value: skos:exactMatch MetaCyc:ASPDECARBOX-RXN
 property_value: skos:exactMatch RHEA:19497
-property_value: skos:relatedMatch MetaCyc:PWY-5155
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004069
-name: L-aspartate:2-oxoglutarate aminotransferase activity
+name: L-aspartate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate." [EC:2.6.1.1]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate." [RHEA:21824]
 synonym: "2-oxoglutarate-glutamate aminotransferase activity" RELATED [EC:2.6.1.1]
 synonym: "AAT" RELATED [EC:2.6.1.1]
 synonym: "aspartate alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.1]
@@ -37043,6 +37074,7 @@
 synonym: "L-aspartate-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.1]
 synonym: "L-aspartate-2-oxoglutarate-transaminase activity" RELATED [EC:2.6.1.1]
 synonym: "L-aspartate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.1]
+synonym: "L-aspartate:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-aspartic aminotransferase activity" BROAD [EC:2.6.1.1]
 synonym: "oxaloacetate transferase activity" RELATED [EC:2.6.1.1]
 synonym: "oxaloacetate-aspartate aminotransferase activity" RELATED [EC:2.6.1.1]
@@ -37052,16 +37084,16 @@
 xref: MetaCyc:ASPAMINOTRANS-RXN
 xref: Reactome:R-HSA-70581 "oxaloacetate + glutamate <=> aspartate + alpha-ketoglutarate [GOT1]"
 xref: Reactome:R-HSA-70592 "PXLP-K259-GOT1 transaminates 2-OG and L-Asp"
-xref: Reactome:R-HSA-70596 "GOT2 dimer deaminates L-Asp"
+xref: Reactome:R-HSA-70596 "GOT2 dimer transaminates L-Asp"
 xref: Reactome:R-HSA-70613 "GOT2 transaminates OA and L-Glu"
 xref: RHEA:21824
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.1
 property_value: skos:exactMatch RHEA:21824
-property_value: skos:relatedMatch MetaCyc:ASPARTATE-DEG1-PWY
-property_value: skos:relatedMatch MetaCyc:ASPARTATESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004070
@@ -37094,7 +37126,7 @@
 id: GO:0004071
 name: aspartate-ammonia ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + L-aspartate + NH3 = AMP + diphosphate + L-asparagine." [EC:6.3.1.1]
+def: "Catalysis of the reaction: ATP + L-aspartate + NH4+ = AMP + diphosphate + L-asparagine." [RHEA:11372]
 synonym: "asparagine synthetase activity" RELATED [EC:6.3.1.1]
 synonym: "L-asparagine synthetase activity" RELATED [EC:6.3.1.1]
 synonym: "L-aspartate:ammonia ligase (AMP-forming)" RELATED [EC:6.3.1.1]
@@ -37220,6 +37252,15 @@
 xref: MetaCyc:6.3.4.11-RXN
 xref: MetaCyc:6.3.4.9-RXN
 xref: MetaCyc:BIOTINLIG-RXN
+xref: Reactome:R-HSA-2993447 "HLCS biotinylates 6x(PCCA:PCCB)"
+xref: Reactome:R-HSA-2993799 "HLCS biotinylates 6xMCCC1:6xMCCC2"
+xref: Reactome:R-HSA-2993802 "HLCS biotinylates PC:Mn2+"
+xref: Reactome:R-HSA-2993814 "HLCS biotinylates ACACA:Mn2+"
+xref: Reactome:R-HSA-3323184 "Defective HLCS does not biotinylate ACACA:Mn2+"
+xref: Reactome:R-HSA-4167511 "HLCS biotinylates ACACB"
+xref: Reactome:R-HSA-9035987 "Defective HLCS does not biotinylate 6xMCCC1:6xMCCC2"
+xref: Reactome:R-HSA-9035988 "Defective HLCS does not biotinylate PC:Mn2+"
+xref: Reactome:R-HSA-9035990 "Defective HLCS does not biotinylate 6x(PCCA:PCCB)"
 xref: RHEA:11204
 xref: RHEA:11756
 xref: RHEA:23668
@@ -37341,12 +37382,13 @@
 
 [Term]
 id: GO:0004084
-name: branched-chain-amino-acid transaminase activity
+name: branched-chain-amino-acid:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a branched-chain amino acid (L-leucine, L-isoleucine and L-valine) + 2-oxoglutarate = L-glutamate + a 2-oxocarboxylate derived from the branched-chain amino acid." [EC:2.6.1.42]
 synonym: "branched-chain amino acid aminotransferase activity" EXACT []
 synonym: "branched-chain amino acid-glutamate transaminase activity" RELATED [EC:2.6.1.42]
 synonym: "branched-chain aminotransferase activity" RELATED [EC:2.6.1.42]
+synonym: "branched-chain-amino-acid transaminase activity" EXACT []
 synonym: "branched-chain-amino-acid:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.42]
 synonym: "glutamate-branched-chain amino acid transaminase activity" RELATED [EC:2.6.1.42]
 synonym: "L-branched chain amino acid aminotransferase activity" RELATED [EC:2.6.1.42]
@@ -37356,9 +37398,10 @@
 xref: Reactome:R-HSA-508189 "a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate <=> leu, ile, or val + alpha-ketoglutarate [BCAT1]"
 xref: Reactome:R-HSA-70723 "leu, ile, or val + alpha-ketoglutarate <=> a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate [BCAT1]"
 xref: Reactome:R-HSA-70724 "leu, ile, or val + alpha-ketoglutarate <=> a-ketoisocaproate, a-keto-b-methylvalerate, or a-ketoisovalerate + glutamate [BCAT2]"
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.42
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004085
@@ -37406,6 +37449,7 @@
 xref: KEGG_REACTION:R00149
 xref: MetaCyc:RXN-13202
 xref: Reactome:R-HSA-70555 "2 ATP + NH4+ + HCO3- => 2 ADP + orthophosphate + carbamoyl phosphate [mitochondrial]"
+xref: Reactome:R-HSA-9955543 "CPS1 variants don't synthesize carbamoyl phosphate"
 xref: RHEA:18029
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
 property_value: skos:exactMatch EC:6.3.4.16
@@ -37552,7 +37596,6 @@
 xref: EC:2.3.1.21
 xref: MetaCyc:CARNITINE-O-PALMITOYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-200406 "CPT1A transfers PALM to CAR"
-xref: Reactome:R-HSA-200410 "CPT2 converts palmitoyl carnitine to palmitoyl-CoA"
 xref: Reactome:R-HSA-9911362 "CPT1B transfers PALM to CAR"
 xref: RHEA:12661
 is_a: GO:0016406 ! carnitine O-acyltransferase activity
@@ -37567,7 +37610,7 @@
 namespace: molecular_function
 alt_id: GO:0016952
 alt_id: GO:0016953
-def: "Catalysis of the reaction: 2 hydrogen peroxide = O2 + 2 H2O." [EC:1.11.1.6]
+def: "Catalysis of the reaction: 2 H2O2 = O2 + 2 H2O." [EC:1.11.1.6]
 synonym: "bacterial catalase-peroxidase activity" NARROW []
 synonym: "caperase activity" RELATED [EC:1.11.1.6]
 synonym: "CAT" RELATED [EC:1.11.1.6]
@@ -37613,6 +37656,10 @@
 xref: EC:1.10.3.1
 xref: MetaCyc:CATECHOL-OXIDASE-RXN
 xref: MetaCyc:RXN-13061
+xref: Reactome:R-HSA-5662662 "Tyrosinase oxidises tyrosine to dopaquinone"
+xref: Reactome:R-HSA-5662692 "Dopa is oxidized to dopaquinone by TYR"
+xref: Reactome:R-HSA-5663050 "DHI and DHICA polymerize forming eumelanin"
+xref: Reactome:R-HSA-8878581 "TYRP1 oxidises DHICA to IQCA"
 xref: RHEA:21632
 is_a: GO:0004497 ! monooxygenase activity
 is_a: GO:0016682 ! oxidoreductase activity, acting on diphenols and related substances as donors, oxygen as acceptor
@@ -37867,7 +37914,6 @@
 name: creatine kinase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: ATP + creatine = N-phosphocreatine + ADP + 2 H+." [EC:2.7.3.2, RHEA:17157]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "adenosine triphosphate-creatine transphosphorylase activity" RELATED [EC:2.7.3.2]
 synonym: "ATP:creatine N-phosphotransferase activity" RELATED [EC:2.7.3.2]
 synonym: "ATP:creatine phosphotransferase activity" RELATED [EC:2.7.3.2]
@@ -37894,9 +37940,9 @@
 is_a: GO:0016775 ! phosphotransferase activity, nitrogenous group as acceptor
 property_value: skos:exactMatch EC:2.7.3.2
 property_value: skos:exactMatch RHEA:17157
-property_value: skos:relatedMatch MetaCyc:PWY-6158
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004112
@@ -38222,7 +38268,7 @@
 id: GO:0004130
 name: cytochrome-c peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 ferrocytochrome c + hydrogen peroxide = 2 ferricytochrome c + 2 H2O." [EC:1.11.1.5]
+def: "Catalysis of the reaction: 2 ferrocytochrome c + H2O2 = 2 ferricytochrome c + 2 H2O." [EC:1.11.1.5]
 synonym: "apocytochrome c peroxidase activity" RELATED [EC:1.11.1.5]
 synonym: "cytochrome c peroxidase activity" RELATED [EC:1.11.1.5]
 synonym: "cytochrome c-551 peroxidase activity" RELATED [EC:1.11.1.5]
@@ -38244,7 +38290,7 @@
 id: GO:0004131
 name: cytosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cytosine + H2O = uracil + NH3." [EC:3.5.4.1]
+def: "Catalysis of the reaction: cytosine + H2O = uracil + NH4+." [RHEA:20605]
 synonym: "cytosine aminohydrolase activity" RELATED [EC:3.5.4.1]
 synonym: "isocytosine deaminase activity" RELATED [EC:3.5.4.1]
 xref: EC:3.5.4.1
@@ -38260,7 +38306,7 @@
 id: GO:0004132
 name: dCMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dCMP + H2O = dUMP + NH3." [EC:3.5.4.12]
+def: "Catalysis of the reaction: dCMP + H2O = dUMP + NH4+." [RHEA:22924]
 synonym: "dCMP aminohydrolase activity" RELATED [EC:3.5.4.12]
 synonym: "deoxy-CMP-deaminase activity" RELATED [EC:3.5.4.12]
 synonym: "deoxycytidine monophosphate deaminase activity" RELATED [EC:3.5.4.12]
@@ -38654,7 +38700,6 @@
 synonym: "succinyl-CoA:enzyme-N6-(dihydrolipoyl)lysine S-succinyltransferase activity" RELATED [EC:2.3.1.61]
 xref: EC:2.3.1.61
 xref: Reactome:R-HSA-9853512 "DLST transfers succinyl to CoA"
-xref: Reactome:R-HSA-9858590 "DLST transfers glutaryl to CoA"
 xref: RHEA:15213
 is_a: GO:0016751 ! S-succinyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -38718,7 +38763,7 @@
 id: GO:0004153
 name: dihydropterin deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 7,8-dihydropterin + H2O = 7,8-dihydrolumazine + NH3." [GOC:jl, PMID:19567870, PMID:38786926]
+def: "Catalysis of the reaction: 7,8-dihydropterin + H2O = 7,8-dihydrolumazine + NH4+." [GOC:jl, PMID:19567870, PMID:38786926]
 xref: MetaCyc:RXN-15261
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
 is_a: GO:0019239 ! deaminase activity
@@ -39039,6 +39084,7 @@
 xref: EC:5.3.3.12
 xref: KEGG_REACTION:R03673
 xref: MetaCyc:DOPACHROME-DELTA-ISOMERASE-RXN
+xref: Reactome:R-HSA-5662660 "Dopachrome is transformed to DHICA by DCT"
 xref: RHEA:13041
 is_a: GO:0016863 ! intramolecular oxidoreductase activity, transposing C=C bonds
 property_value: skos:exactMatch EC:5.3.3.12
@@ -39080,6 +39126,7 @@
 xref: Reactome:R-HSA-5615556 "Defective POMT2 does not transfer Man from Dol-P-Man to DAG1"
 xref: Reactome:R-HSA-5615604 "Defective POMT1 does not transfer Man from Dol-P-Man to DAG1"
 xref: Reactome:R-HSA-5615637 "POMT1:POMT2 transfers Man from Dol-P-Man to DAG1(30-653)"
+xref: Reactome:R-HSA-9816277 "CDH1 is O-manosylated"
 is_a: GO:0000030 ! mannosyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 relationship: part_of GO:0035269 ! protein O-linked glycosylation via mannose
@@ -39207,7 +39254,7 @@
 xref: Reactome:R-HSA-353125 "26S proteosome degrades ODC holoenzyme complex"
 xref: Reactome:R-HSA-3640874 "Ub-RibC-AXIN is degraded by the proteasome"
 xref: Reactome:R-HSA-3928656 "gamma-secretase cleaves EPHB2"
-xref: Reactome:R-HSA-450466 "AUF1:mRNA complex is destroyed"
+xref: Reactome:R-HSA-450466 "AUF1:mRNA complex is degraded"
 xref: Reactome:R-HSA-4608855 "PRICKLE1 is degraded by the proteasome"
 xref: Reactome:R-HSA-4641256 "Ubiquitinated AXIN is degraded by the proteasome"
 xref: Reactome:R-HSA-4641260 "Ubiquitinated DVL1 is degraded by the proteasome"
@@ -39256,7 +39303,9 @@
 xref: Reactome:R-HSA-9614271 "Autocleavage of ADGRG6"
 xref: Reactome:R-HSA-9755303 "26S proteasome degrades HIFalpha"
 xref: Reactome:R-HSA-9755306 "ub UBXN7 is degraded by the 26S proteasome"
+xref: Reactome:R-HSA-9760521 "Unknown peptidase cleaves pro-STa"
 xref: Reactome:R-HSA-9762096 "Ub,pS335,S338,T NFE2L2 is degraded"
+xref: Reactome:R-HSA-9766223 "Proteasome-dependent degradation of ubiquitinated CDH1"
 xref: Reactome:R-HSA-983150 "Proteasomal cleavage of substrate"
 xref: Reactome:R-HSA-983158 "Trimming of peptides in ER"
 xref: Reactome:R-HSA-9839376 "TGFBR3(784-851) degradation"
@@ -39265,6 +39314,18 @@
 xref: Reactome:R-HSA-9908105 "Maturation of the 20S immunoproteasome core particle"
 xref: Reactome:R-HSA-9908108 "Maturation of the 20S thymoproteasome core particle"
 xref: Reactome:R-HSA-9908780 "Maturation of the preholospermatoproteasome"
+xref: Reactome:R-HSA-9912636 "Proteasomal cleavage of intracellular substrate (PA28-alpha-beta-20S proteasome catalyst)"
+xref: Reactome:R-HSA-9912655 "Proteasomal cleavage of intracellular substrate (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9912747 "Proteasomal clevage of exogenous antigen (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9912751 "Proteasomal cleavage of partially digested antigen (immunoproteasome catalyst)"
+xref: Reactome:R-HSA-9929352 "Ubiquitinated CD274 is degraded by the 26S proteasome"
+xref: Reactome:R-HSA-9929486 "SPOP-mediated degradation of CD274 by 26S Proteosome"
+xref: Reactome:R-HSA-9931314 "Proteasomal degradation of polyUb-p-S195-CD274"
+xref: Reactome:R-HSA-9934884 "Proteolysis of K48polyUb,p-S-CRY1,2"
+xref: Reactome:R-HSA-9934887 "Proteolysis of K48polyUb-K107,p-S-CRY1,K48polyUb-K126,p-S-CRY2"
+xref: Reactome:R-HSA-9934893 "Proteolysis of K48polyUb-K,p-S-PER1,2,3"
+xref: Reactome:R-HSA-9949676 "The proteasome degrades the alanine-tailed K48-polyUb nascent peptide bound to LTN1:NEMF:TCF25:VCP hexamer"
+xref: Reactome:R-HSA-9954728 "The proteasome degrades the K48-polyubiquitinated alanine-tailed nascent peptide"
 is_a: GO:0008233 ! peptidase activity
 property_value: skos:narrowMatch EC:3.4.99.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25813" xsd:anyURI
@@ -39384,14 +39445,15 @@
 xref: MetaCyc:CARBOXYPEPTIDASE-A-RXN
 xref: MetaCyc:GLY-X-CARBOXYPEPTIDASE-RXN
 xref: MetaCyc:LYSINEARGININE-CARBOXYPEPTIDASE-RXN
-xref: Reactome:R-HSA-2022378 "ACE2(18-805):Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-9)"
-xref: Reactome:R-HSA-2022379 "ACE2(18-805):Zn2+ hydrolyzes Angiotensin-(1-8) to Angiotensin-(1-7)"
-xref: Reactome:R-HSA-2028294 "Mast Cell Carboxypeptidase hydrolyzes Angiotensin-(1-10) to Yield Angiotensin-(1-9)"
+xref: Reactome:R-HSA-2022378 "ACE2(18-805):Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-33) (Angiotensin-(1-9))"
+xref: Reactome:R-HSA-2022379 "ACE2(18-805):Zn2+ hydrolyzes AGT(25-32) (Angiotensin-(1-8)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2028294 "Mast Cell Carboxypeptidase hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-33) (Angiotensin-(1-9))"
 xref: Reactome:R-HSA-8852809 "CPN, CPB2 cleave C3a, C5a"
 xref: Reactome:R-HSA-8866105 "CCPs deglutamylate tubulin"
 xref: Reactome:R-HSA-8955712 "SVBP:VASH1,VASH2 hydrolyzes the terminal L-Tyr residue from alphaY-beta tubulin dimer"
 xref: Reactome:R-HSA-9023159 "Carboxypeptidase E hydrolyzes Insulin(57-89) to yield C-peptide (Insulin(57-87))"
 xref: Reactome:R-HSA-9023163 "Carboxypeptidase E cleaves Insulin(25-56) to yield Insulin(25-54)"
+xref: Reactome:R-HSA-9944488 "ACE2:Zn2+ cleaves 1-Ala-AGT(25-32) to yield Alamandine (1-Ala-AGT(25-31)"
 is_a: GO:0004180 ! carboxypeptidase activity
 is_a: GO:0008235 ! metalloexopeptidase activity
 property_value: skos:exactMatch EC:3.4.17.-
@@ -39606,9 +39668,9 @@
 xref: MetaCyc:3.4.23.5-RXN
 xref: Reactome:R-HSA-157353 "NEXT1 is cleaved to produce NICD1"
 xref: Reactome:R-HSA-157640 "NEXT2 is cleaved to produce NICD2"
-xref: Reactome:R-HSA-2022403 "Renin:Prorenin Receptor hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
-xref: Reactome:R-HSA-2022412 "Renin hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
-xref: Reactome:R-HSA-2065357 "Prorenin:Prorenin Receptor hydrolyzes Angiotensinogen to Angiotensin-(1-10)"
+xref: Reactome:R-HSA-2022403 "Renin:Prorenin Receptor hydrolyzes Angiotensinogen to AGT(25-34) (Angiotensin-(1-10))"
+xref: Reactome:R-HSA-2022412 "Renin hydrolyzes Angiotensinogen to AGT(25-34) (Angiotensin-(1-10))"
+xref: Reactome:R-HSA-2065357 "Prorenin:Prorenin Receptor hydrolyzes Angiotensinogen to AGT(25-35) (Angiotensin-(1-10))"
 xref: Reactome:R-HSA-2220988 "NEXT1 PEST domain mutants are cleaved to produce NICD1 PEST domain mutants"
 xref: Reactome:R-HSA-373705 "Caspase cleavage of DCC"
 xref: Reactome:R-HSA-9013361 "NEXT3 is cleaved to produce NICD3"
@@ -39807,7 +39869,7 @@
 xref: Reactome:R-HSA-264865 "Caspase-mediated cleavage of Lamin A"
 xref: Reactome:R-HSA-264871 "Caspase-mediated cleavage of Lamin B1"
 xref: Reactome:R-HSA-3465448 "Caspase-8 and FLIP(L) processing at DISC"
-xref: Reactome:R-HSA-350158 "LGMN degrades GC"
+xref: Reactome:R-HSA-350158 "LGMN hydrolyzes GC, releasing CUBN and 25(OH)D"
 xref: Reactome:R-HSA-350318 "Caspase-mediated cleavage of vimentin at TNLD (429)"
 xref: Reactome:R-HSA-350319 "Caspase mediated cleavage of vimentin at IDVD (259)"
 xref: Reactome:R-HSA-350651 "Caspase-mediated cleavage of MASK"
@@ -39829,6 +39891,7 @@
 xref: Reactome:R-HSA-5660663 "Caspase-8 cleaves IL1B precursor"
 xref: Reactome:R-HSA-5681987 "LC3 is cleaved by ATG4"
 xref: Reactome:R-HSA-5682377 "LC3 de-lipidation by ATG4"
+xref: Reactome:R-HSA-6800797 "The PIDDosome activates CASP2"
 xref: Reactome:R-HSA-6814387 "CASP14 cleaves filaggrin"
 xref: Reactome:R-HSA-9012556 "IL37:2x(CASP1(120-197):CASP1(317-404)) cleaves IL37"
 xref: Reactome:R-HSA-9013895 "Caspase-8 processing within TLR3 complex"
@@ -39855,10 +39918,21 @@
 xref: Reactome:R-HSA-9694625 "nsp3 cleaves nsp1-4"
 xref: Reactome:R-HSA-9694732 "3CLp cleaves pp1ab"
 xref: Reactome:R-HSA-9697750 "RIPK1 is cleaved by CASP8:FLIP(L)"
-xref: Reactome:R-HSA-9710101 "CASP4, CASP5 cleave GSDMD"
 xref: Reactome:R-HSA-9729704 "SARS-CoV-2 3CLpro dimer cleaves TAB1"
 xref: Reactome:R-HSA-9729730 "SARS-CoV-2 nsp3 cleaves IRF3"
 xref: Reactome:R-HSA-9729741 "SARS-CoV-2 3CLpro dimer cleaves NLRP12"
+xref: Reactome:R-HSA-9792568 "CTSL1 cleaves proHPSE"
+xref: Reactome:R-HSA-9947849 "CASP5 autocatalysis on LPS micelles"
+xref: Reactome:R-HSA-9947861 "CASP4 autocatalysis on LPS micelles"
+xref: Reactome:R-HSA-9947908 "CASP4 autocatalysis on the bacterial surface"
+xref: Reactome:R-HSA-9947978 "CASP5 cleaves IL18 at D36"
+xref: Reactome:R-HSA-9947987 "CASP4 cleaves IL18 at D36"
+xref: Reactome:R-HSA-9947997 "CASP5 cleaves GSDMD"
+xref: Reactome:R-HSA-9948146 "CASP4 cleaves GSDMD"
+xref: Reactome:R-HSA-9956714 "CASP1 cleaves GBP1"
+xref: Reactome:R-HSA-9958657 "CASP4 cleaves IL1B at D27"
+xref: Reactome:R-HSA-9960522 "CASP5 cleaves IL1B at D27"
+xref: Reactome:R-HSA-9960526 "CASP4 cleaves IL1B at D116"
 is_a: GO:0004175 ! endopeptidase activity
 is_a: GO:0008234 ! cysteine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.22.-
@@ -40113,7 +40187,7 @@
 id: GO:0004219
 name: obsolete pyroglutamyl-peptidase I activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: pyroglutamyl-peptide + H2O = pyroglutamate + peptide." [GOC:curator]
+def: "OBSOLETE. Catalysis of the reaction: pyroglutamyl-peptide + H2O = pyroglutamate + peptide." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product.
 synonym: "pyroglutamyl-peptidase I activity" EXACT []
 is_obsolete: true
@@ -40283,6 +40357,7 @@
 xref: Reactome:R-HSA-8949649 "PMPCA:PMPCB cleaves the transit peptide of proSMDT1 (proEMRE)"
 xref: Reactome:R-HSA-8949659 "AFG3L2 (m-AAA protease) degrades SMDT1 that is not assembled in MCU"
 xref: Reactome:R-HSA-8949668 "YME1L1 proteolyzes unassembled proSMDT1"
+xref: Reactome:R-HSA-9010034 "ADAM10:Zn2+:TSPANs cleaves APP(18-770)"
 xref: Reactome:R-HSA-9624272 "MMPs cleave HB-EGF"
 xref: Reactome:R-HSA-9822532 "ADAMTS13 cleaves VWF multimer"
 xref: Reactome:R-HSA-9824402 "ADAMTS13 variant does not cleave VWF multimer"
@@ -40604,7 +40679,7 @@
 id: GO:0004245
 name: obsolete neprilysin activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the preferential cleavage at the amino group of hydrophobic residues in insulin, casein, hemoglobin, and a number of other proteins and polypeptides." [GOC:curator]
+def: "OBSOLETE. Catalysis of the preferential cleavage at the amino group of hydrophobic residues in insulin, casein, hemoglobin, and a number of other proteins and polypeptides." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product.
 synonym: "acute lymphoblastic leukemia antigen" RELATED []
 synonym: "CALLA" RELATED []
@@ -40765,7 +40840,7 @@
 xref: Reactome:R-HSA-139893 "Granzyme-B activates BID by cleavage"
 xref: Reactome:R-HSA-140599 "factor XIII -> factor XIII cleaved tetramer + 2 factor XIII A activation peptides"
 xref: Reactome:R-HSA-140664 "prothrombin -> activated thrombin (factor IIa) + thrombin activation peptide (prothrombinase catalyst)"
-xref: Reactome:R-HSA-140696 "factor V -> factor Va + factor V activation peptide"
+xref: Reactome:R-HSA-140696 "factor V -> factor Va + factor V activation peptide (thrombin catalyst)"
 xref: Reactome:R-HSA-140700 "prothrombin -> activated thrombin (factor IIa) + thrombin activation peptide (Xa catalyst)"
 xref: Reactome:R-HSA-140736 "factor X -> factor Xa + factor X activation peptide (TF:F7a catalyst)"
 xref: Reactome:R-HSA-140769 "factor VII -> factor VIIa"
@@ -40845,7 +40920,7 @@
 xref: Reactome:R-HSA-187020 "Part of pro-beta-NGF is processed to mature beta-NGF"
 xref: Reactome:R-HSA-1912369 "NOTCH precursor cleaved to form mature NOTCH"
 xref: Reactome:R-HSA-1912372 "Fringe-modified Pre-NOTCH is cleaved by FURIN"
-xref: Reactome:R-HSA-2022411 "Cathepsin G hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022411 "Cathepsin G hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-2129357 "Fibrillin C-terminal processing"
 xref: Reactome:R-HSA-2168923 "Collagen type XVIII endostatin release"
 xref: Reactome:R-HSA-2168960 "Collagen type XVII ectodomain shedding"
@@ -40860,6 +40935,7 @@
 xref: Reactome:R-HSA-2534206 "E-cadherin degradation by PS1:NCSTN (Gamma-secretase)"
 xref: Reactome:R-HSA-2534260 "E-cadherin degradation by caspase-3 and calpain-1"
 xref: Reactome:R-HSA-265301 "Corticotropin cleavage from POMC"
+xref: Reactome:R-HSA-3244626 "DENV protease NS2B3 cleaves STING"
 xref: Reactome:R-HSA-3266557 "Factor I cleaves iC3b"
 xref: Reactome:R-HSA-3785684 "Fibronectin degradation by CTSG"
 xref: Reactome:R-HSA-3788061 "Fibronectin degradation by ADAM8"
@@ -40936,11 +41012,34 @@
 xref: Reactome:R-HSA-9770187 "S2 Cleavage by TMPRSS2 Exposes S2' Initiating Cell-Cell Fusion"
 xref: Reactome:R-HSA-977371 "Factor I inactivates Factor H-boundC3b"
 xref: Reactome:R-HSA-977615 "Factor I inactivates MCP/CR1-bound C4b/C3b"
+xref: Reactome:R-HSA-9816275 "CDH1 is proteolytically cleaved in Golgi"
 xref: Reactome:R-HSA-9829030 "Nascent F signal peptide is cleaved at ER membrane"
 xref: Reactome:R-HSA-9829200 "F0 is cleaved, releasing F1, F2, F(110-136)"
 xref: Reactome:R-HSA-9830805 "Nascent sG localizes to ER lumen, gets glycosylated"
 xref: Reactome:R-HSA-9830882 "Nascent G signal peptide is cleaved at ER membrane"
 xref: Reactome:R-HSA-9839367 "TGFBR3_mem cleavage by MMPs"
+xref: Reactome:R-HSA-9911236 "ELANE cleaves NOTCH2NLA"
+xref: Reactome:R-HSA-9911383 "LepB cleaves prepro-STa"
+xref: Reactome:R-HSA-9912368 "LepB cleaves pre-EltA"
+xref: Reactome:R-HSA-9912371 "LepB cleaves pre-EltB"
+xref: Reactome:R-HSA-9913329 "Autoproteolysis of DAG1 to alpha and beta chains"
+xref: Reactome:R-HSA-9918772 "Dengue polyprotein cis-auto-cleaves"
+xref: Reactome:R-HSA-9918780 "Unknown peptidase cleaves NS1-NS2A"
+xref: Reactome:R-HSA-9918787 "NS2B3 cleaves C-prM-E-NS1-NS2A"
+xref: Reactome:R-HSA-9918795 "Signalase cleaves prM-E-NS1-NS2A"
+xref: Reactome:R-HSA-9918813 "NS2B3 cleaves NS4AB"
+xref: Reactome:R-HSA-9918817 "NS2B3 cleaves polyprotein"
+xref: Reactome:R-HSA-9918823 "NS345:NS2B cleaves NS345"
+xref: Reactome:R-HSA-9918871 "Signalase cleaves prepro-NS4B"
+xref: Reactome:R-HSA-9918876 "NS2B3 cleaves prepro-C at the C-prM junction"
+xref: Reactome:R-HSA-9919050 "Furin cleaves prM:E"
+xref: Reactome:R-HSA-9921595 "NS2B3 cleaves FAM134B"
+xref: Reactome:R-HSA-9921689 "NS3 cleaves GrpEL1"
+xref: Reactome:R-HSA-9926936 "EspC(54-1018) cleaves hemoglobin"
+xref: Reactome:R-HSA-9926963 "Hbp cis-autocleaves"
+xref: Reactome:R-HSA-9927009 "Hbp(53-1100) cleaves hemoglobin"
+xref: Reactome:R-HSA-9932162 "Removal of CDH1 signal peptide"
+xref: Reactome:R-HSA-9947496 "ACOT2 cleaves NS345"
 is_a: GO:0004175 ! endopeptidase activity
 is_a: GO:0008236 ! serine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.21.-
@@ -41522,9 +41621,9 @@
 xref: MetaCyc:ESTRADIOL-17-BETA-DEHYDROGENASE-RXN
 xref: Reactome:R-HSA-5693390 "HSD17B11 dehydrogenates EST17b to E1"
 xref: Reactome:R-HSA-5696822 "AKR1B15 reduces EST17b to E1"
-xref: Reactome:R-HSA-6810594 "HSD17B14 tetramer oxidises estradiol (E2) to estrone (E1)"
+xref: Reactome:R-HSA-6810594 "HSD17B14 tetramer oxidises estradiol (EST17b) to estrone (E1)"
 xref: Reactome:R-HSA-804969 "HSD17B1 hydrogenates E1 to EST17b"
-xref: Reactome:R-HSA-8862137 "HSD17B2 oxidises estradiol (E2) to estrone (E1)"
+xref: Reactome:R-HSA-8862137 "HSD17B2 oxidises estradiol (EST17b) to estrone (E1)"
 xref: RHEA:24612
 xref: RHEA:24616
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
@@ -41665,8 +41764,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 2 farnesyl diphosphate = diphosphate + presqualene diphosphate." [GOC:curators]
 comment: This term was obsoleted because it represents a subreaction of squalene synthase activity ; GO:0051996.
-xref: Reactome:R-HSA-191402 "Reduction of presqualene diphosphate to form squalene"
-xref: Reactome:R-HSA-191405 "Two FPP molecules dimerize to form presqualene diphosphate"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28819" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051996
@@ -41676,7 +41773,7 @@
 name: geranylgeranyl diphosphate synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: (2E,6E)-farnesyl diphosphate + isopentenyl diphosphate = (2E,6E,10E)-geranylgeranyl diphosphate + diphosphate." [RHEA:17653]
-comment: The catalyzed reaction forms (free) geranylgeranyl diphosphate. There is no relationship between this activity and protein farnesyltransferase activity, GO:0004660, where the catalyzed reaction transfers a farnesyl group from farnesyl diphosphate to a target protein. Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+comment: The catalyzed reaction forms (free) geranylgeranyl diphosphate. There is no relationship between this activity and protein farnesyltransferase activity, GO:0004660, where the catalyzed reaction transfers a farnesyl group from farnesyl diphosphate to a target protein.
 synonym: "farnesyltranstransferase activity" EXACT [EC:2.5.1.29]
 synonym: "geranylgeranyl pyrophosphate synthase activity" EXACT [EC:2.5.1.29]
 synonym: "geranylgeranyl pyrophosphate synthetase activity" EXACT [EC:2.5.1.29]
@@ -41690,12 +41787,12 @@
 is_a: GO:0120531 ! prenyl diphosphate synthase activity
 property_value: skos:exactMatch EC:2.5.1.29
 property_value: skos:exactMatch RHEA:17653
-property_value: skos:relatedMatch MetaCyc:PWY-5120
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20661" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28430" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29264" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004312
@@ -41989,6 +42086,8 @@
 synonym: "TPNH-ferredoxin reductase activity" RELATED [EC:1.18.1.2]
 xref: EC:1.18.1.2
 xref: MetaCyc:1.18.1.2-RXN
+xref: Reactome:R-HSA-2395512 "FDXR transfers electrons to FDX1,2 (FDX1L)"
+xref: Reactome:R-HSA-2395517 "NADPH transfers electrons to FDXR"
 xref: RHEA:20125
 is_a: GO:0008937 ! ferredoxin-[NAD(P)H] reductase activity
 property_value: skos:exactMatch EC:1.18.1.2
@@ -41998,10 +42097,11 @@
 
 [Term]
 id: GO:0004325
-name: ferrochelatase activity
+name: protoporphyrin ferrochelatase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: heme B (protoheme) + H+ = Fe(2+) + protoporphyrin IX." [RHEA:22584]
 synonym: "ferro-protoporphyrin chelatase activity" EXACT []
+synonym: "ferrochelatase activity" BROAD []
 synonym: "heme synthase activity" BROAD []
 synonym: "heme synthetase activity" BROAD [EC:4.98.1.1]
 synonym: "iron chelatase activity" BROAD [EC:4.98.1.1]
@@ -42018,6 +42118,7 @@
 property_value: skos:exactMatch RHEA:22584
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23371" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31620" xsd:anyURI
 
 [Term]
 id: GO:0004326
@@ -42114,7 +42215,7 @@
 xref: EC:3.1.3.46
 xref: MetaCyc:3.1.3.46-RXN
 xref: Reactome:R-HSA-5628905 "TIGAR converts D-fructose-2,6-bisphosphate to D-fructose 6-phosphate"
-xref: Reactome:R-HSA-70262 "Fructose 2,6-bisphosphate is hydrolyzed to form fructose-6-phosphate and orthophosphate"
+xref: Reactome:R-HSA-70262 "PFKFKB dimer dephosphorylates fructose 2,6-bisphosphate"
 xref: RHEA:17289
 is_a: GO:0050308 ! sugar-phosphatase activity
 property_value: skos:exactMatch EC:3.1.3.46
@@ -42144,7 +42245,7 @@
 xref: EC:4.1.2.13
 xref: MetaCyc:F16ALDOLASE-RXN
 xref: Reactome:R-HSA-71495 "Aldolase tetramers convert GA3P and DHAP to F1,6PP"
-xref: Reactome:R-HSA-71496 "D-fructose 1,6-bisphosphate <=> dihydroxyacetone phosphate + D-glyceraldehyde 3-phosphate"
+xref: Reactome:R-HSA-71496 "Aldolase tetramer cleaves F1,6PP"
 xref: RHEA:14729
 is_a: GO:0016832 ! aldehyde-lyase activity
 property_value: skos:exactMatch EC:4.1.2.13
@@ -42436,7 +42537,7 @@
 synonym: "Zwischenferment" RELATED [EC:1.1.1.49]
 xref: EC:1.1.1.49
 xref: MetaCyc:GLU6PDEHYDROG-RXN
-xref: Reactome:R-HSA-70377 "alpha-D-glucose 6-phosphate + NADP+ => D-glucono-1,5-lactone 6-phosphate + NADPH + H+"
+xref: Reactome:R-HSA-70377 "G6PD multimers dehydrogenate G6P"
 xref: RHEA:15841
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.49
@@ -42484,7 +42585,7 @@
 synonym: "phosphosaccharomutase activity" BROAD [EC:5.3.1.9]
 xref: EC:5.3.1.9
 xref: MetaCyc:PGLUCISOM-RXN
-xref: Reactome:R-HSA-70471 "alpha-D-glucose 6-phosphate <=> D-fructose 6-phosphate"
+xref: Reactome:R-HSA-70471 "GPI dimer isomerizes G6P"
 xref: Reactome:R-HSA-70475 "GPI dimer isomerizes Fru(6)P to G6P"
 xref: RHEA:11816
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
@@ -42597,7 +42698,6 @@
 name: L-glutamate dehydrogenase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NAD+ + H2O = 2-oxoglutarate + NH4+ + NADH + H+." [RHEA:15133]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "glutamate dehydrogenase (NAD)" BROAD [EC:1.4.1.2]
 synonym: "glutamate dehydrogenase (NAD+) activity" BROAD []
 synonym: "glutamate oxidoreductase activity" RELATED [EC:1.4.1.2]
@@ -42620,39 +42720,36 @@
 is_a: GO:0004353 ! L-glutamate dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.4.1.2
 property_value: skos:exactMatch RHEA:15133
-property_value: skos:relatedMatch MetaCyc:GLUTAMATE-DEG1-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004353
 name: L-glutamate dehydrogenase [NAD(P)+] activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NAD(P)+ + H2O = 2-oxoglutarate + NH4+ + NAD(P)H + H+." [EC:1.4.1.3]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "glutamate dehydrogenase (NAD+) activity" RELATED []
 synonym: "glutamic dehydrogenase activity" BROAD [EC:1.4.1.3]
 synonym: "L-glutamate:NAD(P)+ oxidoreductase (deaminating)" EXACT [EC:1.4.1.3]
 xref: EC:1.4.1.3
 xref: MetaCyc:GLUTAMATE-DEHYDROGENASE-NADP+-RXN
-xref: Reactome:R-HSA-70589 "alpha-ketoglutarate + NH4+ + NAD(P)H + H+ <=> glutamate + NAD(P)+ [GLUD1]"
-xref: Reactome:R-HSA-70600 "glutamate + NAD(P)+ => alpha-ketoglutarate + NH4+ + NAD(P)H + H+ [GLUD1]"
+xref: Reactome:R-HSA-70589 "alpha-ketoglutarate + NH4+ + NAD(P)H + H+ <=> glutamate + NAD(P)+ (GLUD1,2)"
+xref: Reactome:R-HSA-70600 "glutamate + NAD(P)+ => alpha-ketoglutarate + NH4+ + NAD(P)H + H+ (GLUD1,2)"
 is_a: GO:0050018 ! amino-acid dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.4.1.3
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-DEHYDROGENASE-NADP+-RXN
-property_value: skos:relatedMatch MetaCyc:GLUTAMATE-SYN2-PWY
-property_value: skos:relatedMatch MetaCyc:PWY-5766
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004354
 name: L-glutamate dehydrogenase (NADP+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamate + NADP+ + H2O = 2-oxoglutarate + NH4+ + NADPH + H+." [RHEA:11612]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "dehydrogenase, glutamate (nicotinamide adenine dinucleotide (phosphate))" RELATED [EC:1.4.1.4]
 synonym: "glutamic acid dehydrogenase" BROAD [EC:1.4.1.4]
 synonym: "glutamic dehydrogenase activity" BROAD [EC:1.4.1.4]
@@ -42668,16 +42765,15 @@
 property_value: skos:exactMatch EC:1.4.1.4
 property_value: skos:exactMatch MetaCyc:GLUTDEHYD-RXN
 property_value: skos:exactMatch RHEA:11612
-property_value: skos:relatedMatch MetaCyc:GLUTSYNIII-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004355
 name: glutamate synthase (NADPH) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + NADP+ = 2-oxoglutarate + L-glutamine + H+ + NADPH. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + NADP+ + H2O = NH3 + 2-oxoglutarate + NADPH + H+." [EC:1.4.1.13, RHEA:15501]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + NADP+ = 2-oxoglutarate + L-glutamine + H+ + NADPH. This is a two-step reaction: (a) L-glutamate + NH4+ = L-glutamine + H2O, (b) L-glutamate + NADP+ + H2O = NH4+ + 2-oxoglutarate + NADPH + H+." [RHEA:15501]
 synonym: "glutamate (reduced nicotinamide adenine dinucleotide phosphate) synthase activity" RELATED [EC:1.4.1.13]
 synonym: "glutamate synthetase (NADP) activity" RELATED [EC:1.4.1.13]
 synonym: "glutamine amide-2-oxoglutarate aminotransferase (oxidoreductase, NADP) activity" RELATED [EC:1.4.1.13]
@@ -42699,8 +42795,8 @@
 property_value: skos:exactMatch EC:1.4.1.13
 property_value: skos:exactMatch MetaCyc:GLUTAMATESYN-RXN
 property_value: skos:exactMatch RHEA:15501
-property_value: skos:relatedMatch MetaCyc:GLUTSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004356
@@ -42745,24 +42841,24 @@
 name: L-glutamate N-acetyltransferase activity, acting on acetyl-L-ornithine as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: N2-acetyl-L-ornithine + L-glutamate = N-acetyl-L-glutamate + L-ornithine." [RHEA:15349]
-synonym: "2-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
+synonym: "2-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
 synonym: "acetyl-L-glutamate:acetyl-L-ornithine transacetylase activity" EXACT []
-synonym: "acetylglutamate synthetase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamate-acetylornithine transacetylase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamic synthetase activity" RELATED [EC:2.3.1.35]
-synonym: "acetylglutamic-acetylornithine transacetylase activity" RELATED [EC:2.3.1.35]
+synonym: "acetylglutamate synthetase activity" BROAD [EC:2.3.1.35]
+synonym: "acetylglutamate-acetylornithine transacetylase activity" EXACT [EC:2.3.1.35]
+synonym: "acetylglutamic synthetase activity" BROAD [EC:2.3.1.35]
+synonym: "acetylglutamic-acetylornithine transacetylase activity" EXACT [EC:2.3.1.35]
 synonym: "acetylornithinase activity" BROAD [EC:2.3.1.35]
-synonym: "acetylornithine glutamate acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "alpha-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "glutamate acetyltransferase activity" RELATED [EC:2.3.1.35]
+synonym: "acetylornithine glutamate acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "alpha-N-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "glutamate acetyltransferase activity" BROAD [EC:2.3.1.35]
 synonym: "glutamate N-acetyltransferase activity" BROAD []
 synonym: "L-glutamate N-acetyltransferase activity" BROAD []
 synonym: "N-acetyl-L-glutamate synthetase activity" BROAD [EC:2.3.1.35]
 synonym: "N-acetylglutamate synthase activity" BROAD [EC:2.3.1.35]
 synonym: "N-acetylglutamate synthetase activity" BROAD [EC:2.3.1.35]
-synonym: "N2-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "ornithine acetyltransferase activity" RELATED [EC:2.3.1.35]
-synonym: "ornithine transacetylase activity" RELATED [EC:2.3.1.35]
+synonym: "N2-acetyl-L-ornithine:L-glutamate N-acetyltransferase activity" EXACT [EC:2.3.1.35]
+synonym: "ornithine acetyltransferase activity" BROAD [EC:2.3.1.35]
+synonym: "ornithine transacetylase activity" BROAD [EC:2.3.1.35]
 xref: EC:2.3.1.35
 xref: KEGG_REACTION:R02282
 xref: MetaCyc:GLUTAMATE-N-ACETYLTRANSFERASE-RXN
@@ -42784,7 +42880,7 @@
 synonym: "L-glutaminase activity" RELATED [EC:3.5.1.2]
 synonym: "L-glutamine amidohydrolase activity" RELATED [EC:3.5.1.2]
 xref: EC:3.5.1.2
-xref: Reactome:R-HSA-70609 "glutamine + H2O => glutamate + NH4+ [GLS]"
+xref: Reactome:R-HSA-70609 "glutamine + H2O => glutamate + NH4+ (GLS)"
 xref: RHEA:15889
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:broadMatch MetaCyc:GLUTAMIN-RXN
@@ -42795,9 +42891,9 @@
 
 [Term]
 id: GO:0004360
-name: glutamine-fructose-6-phosphate transaminase (isomerizing) activity
+name: L-glutamine:D-fructose-6-phosphate transaminase (isomerizing) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-D-fructose 6-phosphate + L-glutamine = D-glucosamine 6-phosphate + L-glutamate." [EC:2.6.1.16, RHEA:13237]
+def: "Catalysis of the reaction: D-fructose 6-phosphate + L-glutamine = D-glucosamine 6-phosphate + L-glutamate." [RHEA:13237]
 synonym: "D-fructose-6-phosphate amidotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "GlcN6P synthase activity" RELATED [EC:2.6.1.16]
 synonym: "glucosamine 6-phosphate synthase activity" RELATED [EC:2.6.1.16]
@@ -42805,6 +42901,7 @@
 synonym: "glucosamine-6-phosphate isomerase (glutamine-forming) activity" EXACT []
 synonym: "glucosamine-6-phosphate synthase activity" RELATED [EC:2.6.1.16]
 synonym: "glucosaminephosphate isomerase" BROAD [EC:2.6.1.16]
+synonym: "glutamine-fructose-6-phosphate transaminase (isomerizing) activity" EXACT []
 synonym: "hexosephosphate aminotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "L-glutamine-D-fructose-6-phosphate amidotransferase activity" RELATED [EC:2.6.1.16]
 synonym: "L-glutamine:D-fructose-6-phosphate isomerase (deaminating)" RELATED [EC:2.6.1.16]
@@ -42814,10 +42911,11 @@
 xref: Reactome:R-HSA-4085027 "Defective GFPT1 does not transfer an amino group from L-Gln to F6P to form GlcN6P"
 xref: Reactome:R-HSA-449715 "GFPT1,2 transfer an amino group from L-Gln to F6P to form GlcN6P"
 xref: RHEA:13237
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.16
 property_value: skos:exactMatch RHEA:13237
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004361
@@ -42941,7 +43039,7 @@
 synonym: "triosephosphate dehydrogenase activity" BROAD [EC:1.2.1.12]
 xref: EC:1.2.1.12
 xref: MetaCyc:GAPOXNPHOSPHN-RXN
-xref: Reactome:R-HSA-70449 "D-glyceraldehyde 3-phosphate + orthophosphate + NAD+ <=> 1,3-bisphospho-D-glycerate + NADH + H+"
+xref: Reactome:R-HSA-70449 "GADPH tetramer dehydrogenates GA3P"
 xref: Reactome:R-HSA-70482 "GAPDH tetramers reduce 1,3BPG to GA3P"
 xref: RHEA:10300
 is_a: GO:0043891 ! glyceraldehyde-3-phosphate dehydrogenase [NAD(P)+] (phosphorylating) activity
@@ -43302,6 +43400,7 @@
 synonym: "guanosine-diphosphatase activity" EXACT []
 xref: EC:3.6.1.42
 xref: MetaCyc:GUANOSINE-DIPHOSPHATASE-RXN
+xref: Reactome:R-HSA-9947887 "GBP1 hydrolyzes GDP"
 xref: RHEA:22156
 is_a: GO:0017110 ! nucleoside diphosphate phosphatase activity
 property_value: skos:exactMatch EC:3.6.1.42
@@ -43388,6 +43487,8 @@
 xref: Reactome:R-HSA-169468 "MCM2-7 mediated fork unwinding"
 xref: Reactome:R-HSA-5686410 "BLM mediates dissolution of double Holliday junction"
 xref: Reactome:R-HSA-5690996 "ERCC2 and ERCC3 DNA helicases form an open bubble structure in damaged DNA"
+xref: Reactome:R-HSA-9922957 "NS3 melts (+)-RNA secondary structures"
+xref: Reactome:R-HSA-9923073 "NS3 unwinds dsRNA secondary structures"
 is_a: GO:0120545 ! nucleic acid conformation isomerase activity
 is_a: GO:0140640 ! catalytic activity, acting on a nucleic acid
 is_a: GO:0140657 ! ATP-dependent activity
@@ -43426,7 +43527,7 @@
 synonym: "heparin 2-sulfotransferase activity" RELATED []
 synonym: "heparin-sulphate 2-sulphotransferase activity" RELATED []
 xref: MetaCyc:RXN-11558
-xref: Reactome:R-HSA-2076508 "HS2ST1 sulfates IdoA at C2 in heparan sulfate"
+xref: Reactome:R-HSA-2076508 "HS2ST1 trimer sulfates IdoA at C2 in heparan sulfate"
 is_a: GO:0034483 ! heparan sulfate sulfotransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24749" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
@@ -43470,7 +43571,7 @@
 id: GO:0004397
 name: histidine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidine = urocanate + NH3." [EC:4.3.1.3]
+def: "Catalysis of the reaction: L-histidine = trans-urocanate + NH4+." [RHEA:21232]
 synonym: "histidase activity" RELATED [GOC:hjd]
 synonym: "histidinase activity" RELATED [GOC:hjd]
 synonym: "histidine alpha-deaminase activity" RELATED [GOC:hjd]
@@ -43518,13 +43619,14 @@
 
 [Term]
 id: GO:0004400
-name: histidinol-phosphate transaminase activity
+name: L-histidinol-phosphate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidinol-phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate." [EC:2.6.1.9]
+def: "Catalysis of the reaction: L-histidinol-phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate." [RHEA:23744]
 synonym: "glutamic-imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "histidine:imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "histidinol phosphate aminotransferase activity" RELATED [EC:2.6.1.9]
 synonym: "histidinol-phosphate aminotransferase activity" EXACT []
+synonym: "histidinol-phosphate transaminase activity" EXACT []
 synonym: "IAP transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "imidazole acetol-phosphate transaminase activity" RELATED [EC:2.6.1.9]
 synonym: "imidazoleacetol phosphate transaminase activity" RELATED [EC:2.6.1.9]
@@ -43535,10 +43637,11 @@
 xref: EC:2.6.1.9
 xref: MetaCyc:HISTAMINOTRANS-RXN
 xref: RHEA:23744
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.9
 property_value: skos:exactMatch RHEA:23744
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004401
@@ -43597,6 +43700,7 @@
 xref: Reactome:R-HSA-3697920 "CLOCK acetylates lysine-10 of histone H3, H4"
 xref: Reactome:R-HSA-5144542 "CLOCK acetylates lysine-15 of histone H3, H4"
 xref: Reactome:R-HSA-5250938 "B-WICH:histone acetyltransferase acetylates histone H3 at lysine-9"
+xref: Reactome:R-HSA-9926482 "CREBBP, EP300 acetylate MITF-M"
 xref: RHEA:21992
 is_a: GO:0061733 ! protein-lysine-acetyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
@@ -43774,12 +43878,13 @@
 synonym: "hyaluronoglucosidase activity" RELATED [EC:3.2.1.35]
 xref: EC:3.2.1.35
 xref: MetaCyc:3.2.1.35-RXN
-xref: Reactome:R-HSA-1793209 "HYAL1 hydrolyses Chondroitin chains"
+xref: Reactome:R-HSA-1793209 "HYAL1-4 hydrolyze chondroitin chains"
 xref: Reactome:R-HSA-2160874 "HYAL1 hydrolyses (HA)50"
-xref: Reactome:R-HSA-2160892 "Hyaluronidase 2 (HYAL2) hydrolyses HA into 20kDa fragments"
+xref: Reactome:R-HSA-2160892 "HYAL2 hydrolyses HA into 20kDa fragments"
 xref: Reactome:R-HSA-2318585 "Defective HYAL1 does not hydrolyse Chondroitin chains"
-xref: Reactome:R-HSA-5693356 "CEMIP hydrolyses HA"
+xref: Reactome:R-HSA-5693356 "CEMIP hydrolyzes HA"
 xref: Reactome:R-HSA-9036077 "Defective HYAL1 does not hydrolyse (HA)50"
+xref: Reactome:R-HSA-9941055 "SPAM1 randomly cleaves HA"
 is_a: GO:0015929 ! hexosaminidase activity
 property_value: skos:exactMatch EC:3.2.1.35
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -44247,7 +44352,7 @@
 id: GO:0004442
 name: obsolete inositol-1,4,-bisphosphate 3-phosphatase
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 1,3-bisphosphate + H2O = D-myo-inositol 1-monophosphate + phosphate." [EC:3.1.3.65, GOC:go_curators]
+def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 1,3-bisphosphate + H2O = D-myo-inositol 1-monophosphate + phosphate." [EC:3.1.3.65, GOC:curators]
 synonym: "inositol-1,4,-bisphosphate 3-phosphatase" EXACT []
 is_obsolete: true
 consider: GO:0004438
@@ -44256,7 +44361,7 @@
 id: GO:0004443
 name: obsolete inositol-1,4,-bisphosphate 4-phosphatase
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 3,4-bisphosphate + H2O = D-myo-inositol 3-monophosphate + phosphate." [EC:3.1.3.66, GOC:go_curators]
+def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 3,4-bisphosphate + H2O = D-myo-inositol 3-monophosphate + phosphate." [EC:3.1.3.66, GOC:curators]
 synonym: "inositol-1,4,-bisphosphate 4-phosphatase" EXACT []
 is_obsolete: true
 consider: GO:0016316
@@ -44317,7 +44422,7 @@
 id: GO:0004447
 name: iodide peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: iodide + H2O2 = iodine + 2 H2O." [RHEA:23336]
+def: "Catalysis of the reactions: (1) 2 iodide + H2O2 + 2 H+ = diiodine + 2 H2O; (2) [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O; (3) [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O; (4) 2 [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-L-thyroxine + [thyroglobulin]-dehydroalanine + 2 H2O; (5) [thyroglobulin]-3-iodo-L-tyrosine + [thyroglobulin]-3,5-diiodo-L-tyrosine + H2O2 = [thyroglobulin]-3,3',5-triiodo-L-thyronine + [thyroglobulin]-dehydroalanine + 2 H2O." [EC:1.11.1.8, PMID:8651700]
 synonym: "iodinase activity" RELATED [EC:1.11.1.8]
 synonym: "thyroid peroxidase activity" RELATED [EC:1.11.1.8]
 synonym: "thyroperoxidase activity" RELATED [EC:1.11.1.8]
@@ -44325,12 +44430,21 @@
 xref: EC:1.11.1.8
 xref: MetaCyc:IODIDE-PEROXIDASE-RXN
 xref: RHEA:23336
+xref: RHEA:48956
+xref: RHEA:48960
+xref: RHEA:48964
+xref: RHEA:48968
 is_a: GO:0140905 ! haloperoxidase activity
 property_value: skos:exactMatch EC:1.11.1.8
-property_value: skos:exactMatch RHEA:23336
+property_value: skos:narrowMatch RHEA:23336
+property_value: skos:narrowMatch RHEA:48956
+property_value: skos:narrowMatch RHEA:48960
+property_value: skos:narrowMatch RHEA:48964
+property_value: skos:narrowMatch RHEA:48968
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21027" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23587" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30597" xsd:anyURI
 
 [Term]
 id: GO:0004448
@@ -44690,6 +44804,7 @@
 xref: EC:4.4.1.20
 xref: KEGG_REACTION:R03059
 xref: MetaCyc:LEUKOTRIENE-C4-SYNTHASE-RXN
+xref: Reactome:R-HSA-2162019 "LTA4 is converted to EXA4 by ALOX15"
 xref: Reactome:R-HSA-266050 "LTA4 is converted to LTC4 by LTC4S"
 xref: RHEA:17617
 is_a: GO:0016846 ! carbon-sulfur lyase activity
@@ -44748,7 +44863,7 @@
 namespace: molecular_function
 alt_id: GO:0003996
 def: "Catalysis of the reaction: a long-chain fatty acid + ATP + CoA = a long-chain fatty acyl-CoA + AMP + diphosphate. A long-chain fatty acid has an aliphatic tail containing 13 to 22 carbons." [RHEA:15421]
-comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283). Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
 synonym: "acyl-activating enzyme activity" BROAD []
 synonym: "acyl-CoA ligase activity" RELATED []
 synonym: "acyl-CoA synthetase activity" RELATED [EC:6.2.1.3]
@@ -44789,16 +44904,16 @@
 property_value: skos:exactMatch EC:6.2.1.3
 property_value: skos:exactMatch MetaCyc:RXN-7904
 property_value: skos:exactMatch RHEA:15421
-property_value: skos:relatedMatch MetaCyc:PWY-5143
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18967" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20462" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25799" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26445" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004468
-name: L-lysine N-acetyltransferase activity, acting on acetyl phosphate as donor
+name: L-lysine N6-acetyltransferase activity, acting on acetyl phosphate as donor
 namespace: molecular_function
 def: "Catalysis of the reaction: acetyl phosphate + L-lysine = phosphate + N6-acetyl-L-lysine." [RHEA:14417]
 synonym: "acetyl-phosphate:L-lysine 6-N-acetyltransferase activity" RELATED [EC:2.3.1.32]
@@ -45008,8 +45123,8 @@
 alt_id: GO:0070128
 def: "Catalysis of the reaction: 10-formyltetrahydrofolate + L-methionyl-tRNA + H2O = tetrahydrofolate + N-formylmethionyl-tRNA." [EC:2.1.2.9]
 synonym: "10-formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase activity" RELATED [EC:2.1.2.9]
-synonym: "conversion of met-tRNAf to fmet-tRNA" RELATED [GO:curators]
-synonym: "conversion of mitochondrial met-tRNAf to fmet-tRNA" RELATED [GO:curators]
+synonym: "conversion of met-tRNAf to fmet-tRNA" RELATED [GOC:curators]
+synonym: "conversion of mitochondrial met-tRNAf to fmet-tRNA" RELATED [GOC:curators]
 synonym: "formylmethionyl-transfer ribonucleic synthetase activity" RELATED [EC:2.1.2.9]
 synonym: "methionyl ribonucleic formyltransferase activity" RELATED [EC:2.1.2.9]
 synonym: "methionyl-transfer ribonucleate methyltransferase activity" RELATED [EC:2.1.2.9]
@@ -45017,9 +45132,9 @@
 synonym: "methionyl-transfer RNA transformylase activity" RELATED [EC:2.1.2.9]
 synonym: "methionyl-tRNA Met formyltransferase activity" RELATED [EC:2.1.2.9]
 synonym: "methionyl-tRNA transformylase activity" RELATED [EC:2.1.2.9]
-synonym: "mitochondrial N-terminal peptidyl-methionine N-formylation" RELATED [GO:curators]
+synonym: "mitochondrial N-terminal peptidyl-methionine N-formylation" RELATED [GOC:curators]
 synonym: "N(10)-formyltetrahydrofolic-methionyl-transfer ribonucleic transformylase activity" RELATED [EC:2.1.2.9]
-synonym: "N-terminal peptidyl-methionine N-formylation" RELATED [GO:curators]
+synonym: "N-terminal peptidyl-methionine N-formylation" RELATED [GOC:curators]
 synonym: "N10-formyltetrahydrofolic-methionyl-transfer ribonucleic transformylase activity" RELATED [EC:2.1.2.9]
 xref: EC:2.1.2.9
 xref: MetaCyc:METHIONYL-TRNA-FORMYLTRANSFERASE-RXN
@@ -45069,6 +45184,7 @@
 xref: Reactome:R-HSA-9694492 "nsp14 acts as a cap N7 methyltransferase to modify SARS-CoV-2 gRNA complement (minus strand)"
 xref: Reactome:R-HSA-9694737 "nsp14 acts as a cap N7 methyltransferase to modify SARS-CoV-2 gRNA (plus strand)"
 xref: Reactome:R-HSA-9830848 "L protein acts as a cap N7 methyltransferase to modify RSV mRNAs"
+xref: Reactome:R-HSA-9923104 "NS5 acts as a cap N7 methyltransferase to methylate 5 G pppA of DENV genomic +ssRNA"
 is_a: GO:0008170 ! N-methyltransferase activity
 is_a: GO:0008174 ! mRNA methyltransferase activity
 relationship: part_of GO:0006370 ! 7-methylguanosine mRNA capping
@@ -45096,6 +45212,7 @@
 xref: Reactome:R-HSA-9694521 "nsp16 acts as a cap 2'-O-methyltransferase to modify SARS-CoV-2 gRNA complement (minus strand)"
 xref: Reactome:R-HSA-9694721 "nsp16 acts as a cap 2'-O-methyltransferase to modify SARS-CoV-2 gRNA (plus strand)"
 xref: Reactome:R-HSA-9834003 "L protein acts as a cap 2'-O-methyltransferase to modify RSV mRNAs"
+xref: Reactome:R-HSA-9923097 "NS5 acts as a cap 2'-O-methyltransferase to methylate 5 m7GpppA of DENV genomic +ssRNA"
 xref: RHEA:67020
 is_a: GO:0008171 ! O-methyltransferase activity
 is_a: GO:0008173 ! RNA methyltransferase activity
@@ -45118,6 +45235,7 @@
 xref: Reactome:R-HSA-77081 "Formation of the CE:GMP intermediate complex"
 xref: Reactome:R-HSA-77083 "Transfer of GMP from the capping enzyme GT site to 5'-end of mRNA"
 xref: Reactome:R-HSA-9815529 "nsp12 transfers guanylyl onto SARS-CoV-2 plus strand subgenomic RNAs"
+xref: Reactome:R-HSA-9923071 "NS5 transfers guanylyl onto DENV genomic 5'-ppRNA"
 xref: RHEA:54592
 is_a: GO:0008192 ! RNA guanylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.50
@@ -45139,8 +45257,8 @@
 xref: EC:6.4.1.4
 xref: KEGG_REACTION:R04138
 xref: MetaCyc:METHYLCROTONYL-COA-CARBOXYLASE-RXN
-xref: Reactome:R-HSA-508308 "beta-methylglutaconyl-CoA + ADP + orthophosphate <=> beta-methylcrotonyl-CoA + ATP + CO2 (MCCA)"
-xref: Reactome:R-HSA-70773 "beta-methylcrotonyl-CoA + ATP + CO2 <=> beta-methylglutaconyl-CoA + ADP + orthophosphate + H2O [MCCA]"
+xref: Reactome:R-HSA-508308 "beta-methylglutaconyl-CoA + ADP + orthophosphate <=> beta-methylcrotonyl-CoA + ATP + HCO3- (MCCA)"
+xref: Reactome:R-HSA-70773 "beta-methylcrotonyl-CoA + ATP + HCO3- <=> beta-methylglutaconyl-CoA + ADP + orthophosphate + H2O (MCCA)"
 xref: Reactome:R-HSA-9909466 "MCCC mutants don't synthesize beta-methylglutaconyl-CoA"
 xref: RHEA:13589
 is_a: GO:0016421 ! CoA carboxylase activity
@@ -45156,8 +45274,6 @@
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "5,10-methylene-THF dehydrogenase activity" EXACT [GOC:vw]
 synonym: "N5,N10-methylenetetrahydrofolate dehydrogenase activity" RELATED []
-xref: Reactome:R-HSA-200644 "5,10-methyleneTHF polyglutamate + NADP+ <=> 5,10-methenylTHF polyglutamate + NADPH + H+"
-xref: Reactome:R-HSA-200718 "MTHFD1 dimer dehydrogenates 5,10-methenyl-THFPG to 5,10-methylene-THFPG"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30105" xsd:anyURI
 is_obsolete: true
@@ -45191,6 +45307,8 @@
 xref: EC:1.5.1.5
 xref: KEGG_REACTION:R01220
 xref: MetaCyc:METHYLENETHFDEHYDROG-NADP-RXN
+xref: Reactome:R-HSA-200644 "5,10-methyleneTHF polyglutamate + NADP+ <=> 5,10-methenylTHF polyglutamate + NADPH + H+"
+xref: Reactome:R-HSA-200718 "MTHFD1 dimer dehydrogenates 5,10-methenyl-THFPG to 5,10-methylene-THFPG"
 xref: RHEA:22812
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.5
@@ -45272,7 +45390,7 @@
 synonym: "MSDH activity" RELATED [EC:1.2.1.27]
 xref: EC:1.2.1.27
 xref: MetaCyc:1.2.1.27-RXN
-xref: Reactome:R-HSA-70893 "methylmalonate semialdehyde + NAD+ + CoA => propionyl-CoA + CO2 + NADH + H+"
+xref: Reactome:R-HSA-70893 "methylmalonate semialdehyde + NAD+ + CoA + H2O => propionyl-CoA + HCO3- + NADH + H+"
 xref: RHEA:20804
 xref: RHEA:76615
 is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
@@ -45426,9 +45544,6 @@
 xref: Reactome:R-HSA-5602272 "Defective CYP4F22 does not 20-hydroxylate TrXA3"
 xref: Reactome:R-HSA-5602295 "CYP4F22 20-hydroxylates TrXA3"
 xref: Reactome:R-HSA-5605147 "Defective CYP1B1 does not 4-hydroxylate EST17b"
-xref: Reactome:R-HSA-5662662 "Tyrosinase oxidises tyrosine to dopaquinone"
-xref: Reactome:R-HSA-5662692 "Dopa is oxidized to dopaquinone by TYR"
-xref: Reactome:R-HSA-5663050 "DHI and DHICA polymerize forming eumelanin"
 xref: Reactome:R-HSA-76354 "Vinyl chloride is oxidized to 2-Chloroethylene oxide"
 xref: Reactome:R-HSA-76373 "N-hydroxylation of 4-aminobiphenyl"
 xref: Reactome:R-HSA-76386 "CYP1A2 S-demethylates 6MMP"
@@ -45437,7 +45552,7 @@
 xref: Reactome:R-HSA-76426 "N-atom dealkylation of caffeine"
 xref: Reactome:R-HSA-76434 "Dehalogenation of carbon tetrachloride to form a free radical"
 xref: Reactome:R-HSA-76456 "O-atom dealkylation of dextromethorphan"
-xref: Reactome:R-HSA-76466 "CYP4A11 12-hydroxylates DDCX"
+xref: Reactome:R-HSA-76466 "CYP4A11 omega-hydroxylates DDCX"
 xref: Reactome:R-HSA-76472 "Ethylene is oxidized to Ethylene oxide by CYP1A1"
 xref: Reactome:R-HSA-76475 "Dehalogenation of the poly-halogenated hydrocarbon Halothane to form the acylhalide Trifluoroacetlychloride and  hydrogen bromide"
 xref: Reactome:R-HSA-8865107 "MICAL1 produces NADP+, H2O2"
@@ -45971,6 +46086,9 @@
 xref: Reactome:R-HSA-5690990 "5'- incision of DNA by ERCC1:ERCC4 in GG-NER"
 xref: Reactome:R-HSA-5693533 "DCLRE1C (ARTEMIS) processes DNA DSB ends"
 xref: Reactome:R-HSA-72180 "Cleavage of mRNA at the 3'-end"
+xref: Reactome:R-HSA-9970193 "CPSF3 cleaves pre-mRNA at distal 3'UTR PAS"
+xref: Reactome:R-HSA-9970282 "U1 snRNP suppresses CPSF3-mediated pre-mRNA cleavage at intronic PAS"
+xref: Reactome:R-HSA-9970294 "CPSF3 cleaves pre-mRNA at proximal 3'UTR PAS"
 is_a: GO:0004518 ! nuclease activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
 
@@ -46016,6 +46134,7 @@
 xref: Reactome:R-HSA-927836 "SMG6 hydrolyzes mRNA with premature termination codon"
 xref: Reactome:R-HSA-9708327 "ANG cleaves tRNA to yield tRNA halves"
 xref: Reactome:R-HSA-9708812 "ELAC2 cleaves pre-tRNA to yield 3' trailer (type II tRF or tRF-1)"
+xref: Reactome:R-HSA-9948427 "ANKZF1 cleaves the peptidyl-tRNA in VCP hexamer:LTN1:NEMF:60S ribosome subunit:peptidyl-tRNA with K48polyUb-nascent peptide"
 is_a: GO:0004519 ! endonuclease activity
 is_a: GO:0004540 ! RNA nuclease activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
@@ -46084,6 +46203,9 @@
 xref: Reactome:R-HSA-173769 "RNase H-mediated digestion of tRNA, 3'PPT and cPPT RNA primers"
 xref: Reactome:R-HSA-182795 "RNase H-mediated degradation of the template strand"
 xref: Reactome:R-HSA-182859 "RNase H-mediated degradation of the RNA strand of the -sssDNA:RNA duplex"
+xref: Reactome:R-HSA-9913637 "RNASEH1 cleaves R-loop RNA to yield RNA primer for H strand polymerization"
+xref: Reactome:R-HSA-9914858 "RNASEH1 cleaves RNA primer from nascent H strand, leaving RNA dinucleotides"
+xref: Reactome:R-HSA-9915449 "RNASEH1 cleaves RNA primer from nascent L strand, leaving RNA dinucleotides"
 is_a: GO:0016891 ! RNA endonuclease activity producing 5'-phosphomonoesters, hydrolytic mechanism
 property_value: skos:exactMatch EC:3.1.26.4
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -46250,6 +46372,12 @@
 def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' terminus of an RNA molecule." [GOC:mah, ISBN:0198547684]
 synonym: "5'-3' exoribonuclease activity" EXACT []
 xref: Reactome:R-HSA-429845 "5' to 3' exoribonuclease hydrolyzes decapped mRNA"
+xref: Reactome:R-HSA-6791227 "47S pre-rRNA is nucleolytically processed at A' (01,A1), site A0, and site 02 (site 6)  to yield 45S pre-rRNA"
+xref: Reactome:R-HSA-9915442 "EXOG cleaves RNA dinucleotide from nascent mitochondrial DNA"
+xref: Reactome:R-HSA-9935265 "DXO hydrolyzes uncapped RNA"
+xref: Reactome:R-HSA-9935299 "XRN2 5'-3' exonucleolytically hydrolyzes uncapped RNA"
+xref: Reactome:R-HSA-9946917 "XRN1 incompletely hydrolyzes DENV decapped mRNA"
+xref: Reactome:R-HSA-9971054 "XRN2 degrades downstream RNA fragment, promoting transcription termination"
 is_a: GO:0008409 ! 5'-3' exonuclease activity
 is_a: GO:0016896 ! RNA exonuclease activity, producing 5'-phosphomonoesters
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28062" xsd:anyURI
@@ -46267,6 +46395,8 @@
 xref: Reactome:R-HSA-429955 "CCR4-NOT complex deadenylates mRNA"
 xref: Reactome:R-HSA-429992 "PARN deadenylates mRNA"
 xref: Reactome:R-HSA-430021 "PAN2-PAN3 complex partially deadenylates mRNA"
+xref: Reactome:R-HSA-9009950 "PDE12 cleaves 2'-5' oligoadenylates"
+xref: Reactome:R-HSA-9615042 "Viral 2',5'-PDE cleaves 2'-5' oligoadenylates"
 xref: Reactome:R-HSA-9822311 "CCR4-NOT deadenylates mRNA in CCR4-NOT:BTG4:PABPN1L:mRNP"
 xref: Reactome:R-HSA-9822335 "CCR4-NOT(CNOT6L) deadenylates mRNA in CCR4-NOT(CNOT6L):ZFP36L2:mRNA"
 is_a: GO:0000175 ! 3'-5'-RNA exonuclease activity
@@ -46563,18 +46693,14 @@
 xref: MetaCyc:3.2.1.52-RXN
 xref: Reactome:R-HSA-1605595 "bHEXA hydrolyzes GM2A:GM2 to GM2A:GM3"
 xref: Reactome:R-HSA-1605632 "bHEXA,bHEXB hydrolyze PSAP(195-273):Gb4Cer:PE"
-xref: Reactome:R-HSA-1638053 "HEXA cleaves the terminal GalNAc from keratan sulfate"
+xref: Reactome:R-HSA-1638053 "HEXA cleaves the terminal GlcNAc from keratan sulfate"
 xref: Reactome:R-HSA-2105001 "HEXA cleaves the terminal GalNAc from DS"
-xref: Reactome:R-HSA-2162225 "HEXA cleaves the terminal GalNAc from small HA fragments"
-xref: Reactome:R-HSA-3656259 "Defective HEXA does not cleave the terminal GalNAc from DS"
-xref: Reactome:R-HSA-3662344 "Defective HEXB does not cleave the terminal GalNAc from DS"
+xref: Reactome:R-HSA-2162225 "HEXA cleaves the terminal GlcNAc from small HA fragments"
 xref: Reactome:R-HSA-9035976 "Defective HEXA does not cleave the terminall GalNAc from small HA fragments"
-xref: Reactome:R-HSA-9035978 "Defective HEXA does not cleave the terminal GalNAc from keratan sulfate"
 xref: Reactome:R-HSA-9035982 "Defective HEXB does not cleave the terminal GalNAc from HA fragments"
-xref: Reactome:R-HSA-9035983 "Defective HEXB does not cleave the terminal GalNAc from keratan sulfate"
 xref: Reactome:R-HSA-9638075 "HEXB cleaves the terminal GalNAc from DS"
-xref: Reactome:R-HSA-9638076 "HEXB cleaves the terminal GalNAc from small HA fragments"
-xref: Reactome:R-HSA-9638078 "HEXB cleaves the terminal GalNAc from keratan sulfate"
+xref: Reactome:R-HSA-9638076 "HEXB cleaves the terminal GlcNAc from small HA fragments"
+xref: Reactome:R-HSA-9638078 "HEXB cleaves the terminal GlcNAc from keratan sulfate"
 xref: Reactome:R-HSA-9840833 "bHEXA,bHEXB hydrolyze GM2A:GA2 to GM2A:LacCer"
 xref: Reactome:R-HSA-9840884 "bHEXA,bHEXS hydrolyze GM2A:SM2"
 is_a: GO:0015929 ! hexosaminidase activity
@@ -46609,7 +46735,6 @@
 name: beta-galactosidase activity
 namespace: molecular_function
 def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-galactose residues in beta-D-galactosides." [EC:3.2.1.23]
-comment: Note that the inclusion of 'MetaCyc:BGALACT-PWY' is exceptional: normally MetaCyc pathway entries are database references for biological process terms, not molecular function terms. An exception was made in this case because the MetaCyc entry 'BGALACT-PWY' describes only one reaction, that catalyzed by beta-galactosidase.
 synonym: "beta-D-galactanase activity" RELATED []
 synonym: "beta-D-galactoside galactohydrolase activity" RELATED []
 synonym: "beta-D-lactosidase activity" RELATED []
@@ -46621,7 +46746,7 @@
 xref: Reactome:R-HSA-1605624 "Beta-galactosidases hydrolyse mobilized GM1 to mobilized GM2"
 xref: Reactome:R-HSA-1606312 "GLB1 hydrolyzes SapB/C:LacCer"
 xref: Reactome:R-HSA-1630306 "GLB1 hydrolyses a glycosaminoglycan"
-xref: Reactome:R-HSA-1793217 "Keratan sulfate is cleaved from its proteoglycan by an unknown galactosidase"
+xref: Reactome:R-HSA-1793217 "Unknown endo--galactosidase cleaves keratan chain"
 xref: Reactome:R-HSA-2090079 "GLB1 hydrolyses linker chain(2)"
 xref: Reactome:R-HSA-2265534 "Defective GLB1 does not hydrolyse a glycosaminoglycan"
 xref: Reactome:R-HSA-9036061 "Defective GLB1 does not hydrolyse linker chain(2)"
@@ -46629,8 +46754,8 @@
 is_a: GO:0015925 ! galactosidase activity
 property_value: skos:exactMatch EC:3.2.1.23
 property_value: skos:exactMatch MetaCyc:3.2.1.23-RXN
-property_value: skos:relatedMatch MetaCyc:BGALACT-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004566
@@ -46645,7 +46770,7 @@
 xref: EC:3.2.1.31
 xref: MetaCyc:BETA-GLUCURONID-RXN
 xref: Reactome:R-HSA-1678854 "GUSB tetramer hydrolyses CS/HS precursor"
-xref: Reactome:R-HSA-2162226 "GUSB tetramer hydrolyses GlcA-1,3-GlcNAc"
+xref: Reactome:R-HSA-2162226 "GUSB tetramer hydrolyzes GlcA-1,3-GlcNAc"
 xref: Reactome:R-HSA-2162227 "GUSB tetramer hydrolyses (HA)2"
 xref: Reactome:R-HSA-2318373 "Defective GUSB does not hydrolyse (HA)2"
 xref: Reactome:R-HSA-9036068 "Defective GUSB does not hydrolyse GlcA-1,3-GlcNAc"
@@ -46787,6 +46912,7 @@
 xref: Reactome:R-HSA-4793947 "Defective MOGS does not cleave glucose from an N-glycosylated protein"
 xref: Reactome:R-HSA-532678 "Trimming of the first glucose by by mannosyl-oligosaccharide glucosidase"
 xref: Reactome:R-HSA-9694364 "N-glycan glucose trimming of Spike"
+xref: Reactome:R-HSA-9932344 "MOGS removes terminal glucose from Glu3Man9GlcNAc2-CDH1"
 xref: RHEA:55988
 is_a: GO:0090599 ! alpha-glucosidase activity
 property_value: skos:exactMatch EC:3.2.1.106
@@ -46910,6 +47036,11 @@
 xref: MetaCyc:RXN-16761
 xref: Reactome:R-HSA-446209 "Transfer of N-glycan to the protein"
 xref: Reactome:R-HSA-9694793 "Spike protein gets N-glycosylated"
+xref: Reactome:R-HSA-9816276 "CDH1 is N-glycosylated on asparagine residues in endoplasmic reticulum"
+xref: Reactome:R-HSA-9918962 "E is N-glycosylated"
+xref: Reactome:R-HSA-9918988 "pre-M is N-glycosylated"
+xref: Reactome:R-HSA-9919011 "Pre-NS1 folds and is N-glycosylated"
+xref: Reactome:R-HSA-9931286 "CD274 N-linked glycosylation in ER"
 xref: RHEA:22980
 is_a: GO:0004576 ! oligosaccharyl transferase activity
 property_value: skos:exactMatch EC:2.4.99.18
@@ -47021,6 +47152,7 @@
 xref: EC:2.1.3.3
 xref: MetaCyc:ORNCARBAMTRANSFER-RXN
 xref: Reactome:R-HSA-70560 "carbamoyl phosphate + ornithine => citrulline + orthophosphate"
+xref: Reactome:R-HSA-9956527 "OTC variants don't synthesize L-citrulline"
 xref: RHEA:19513
 is_a: GO:0016743 ! carboxyl- or carbamoyltransferase activity
 relationship: part_of GO:0006591 ! ornithine metabolic process
@@ -47048,9 +47180,9 @@
 
 [Term]
 id: GO:0004587
-name: ornithine aminotransferase activity
+name: L-ornithine transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2-oxocarboxylate + L-ornithine = an L-alpha-amino acid + L-glutamate 5-semialdehyde." [EC:2.6.1.13]
+def: "Catalysis of the reaction: a 2-oxocarboxylate + L-ornithine = L-glutamate 5-semialdehyde + an L-alpha-amino acid." [RHEA:13877]
 synonym: "GabT" RELATED [EC:2.6.1.13]
 synonym: "L-ornithine 5-aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "L-ornithine aminotransferase activity" RELATED [EC:2.6.1.13]
@@ -47058,6 +47190,7 @@
 synonym: "L-ornithine:alpha-ketoglutarate delta-aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "OAT" RELATED [EC:2.6.1.13]
 synonym: "ornithine 5-aminotransferase activity" RELATED [EC:2.6.1.13]
+synonym: "ornithine aminotransferase activity" EXACT []
 synonym: "ornithine delta-transaminase activity" RELATED [EC:2.6.1.13]
 synonym: "ornithine ketoacid aminotransferase activity" RELATED [EC:2.6.1.13]
 synonym: "ornithine transaminase activity" RELATED [EC:2.6.1.13]
@@ -47083,6 +47216,7 @@
 property_value: skos:exactMatch RHEA:13877
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25975" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004588
@@ -47371,14 +47505,9 @@
 xref: KEGG_REACTION:R03532
 xref: Reactome:R-HSA-1222346 "AhpC reduces H2O2"
 xref: Reactome:R-HSA-140359 "PGG2 is reduced to PGH2 by PTGS1"
-xref: Reactome:R-HSA-209815 "Tyrosine is monoiodinated"
-xref: Reactome:R-HSA-209840 "Two DITs combine to form thyroxine"
-xref: Reactome:R-HSA-209925 "DIT and MIT combine to form triiodothyronine"
-xref: Reactome:R-HSA-209973 "Tyrosine is diiodinated"
 xref: Reactome:R-HSA-2309773 "PGG2 is reduced to PGH2 by PTGS2"
 xref: Reactome:R-HSA-2559639 "Collagen type IV sulfilimine cross-linking by peroxidasin"
 xref: Reactome:R-HSA-3341296 "GPX7,8 catalyze peroxidation of P4HB (PDI)"
-xref: Reactome:R-HSA-350901 "Iodide is organified"
 xref: Reactome:R-HSA-5631885 "PRDX1 overoxidizes"
 xref: Reactome:R-HSA-6789031 "Membrane-bound myeloperoxidase (MPO) produces hypochlorous acid (HOCl)"
 xref: Reactome:R-HSA-6789126 "Myeloperoxidase (MPO) produces hypochlorous acid (HOCl)"
@@ -47398,7 +47527,7 @@
 name: glutathione peroxidase activity
 namespace: molecular_function
 alt_id: GO:0016224
-def: "Catalysis of the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O." [EC:1.11.1.9, PMID:36771108]
+def: "Catalysis of the reaction: 2 glutathione + H2O2 = oxidized glutathione + 2 H2O." [EC:1.11.1.9, PMID:36771108]
 synonym: "glutathione:hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.9]
 synonym: "GSH peroxidase activity" RELATED [EC:1.11.1.9]
 synonym: "non-selenium glutathione peroxidase activity" NARROW []
@@ -47693,7 +47822,7 @@
 synonym: "phosphogluconic acid dehydrogenase activity" RELATED [EC:1.1.1.44]
 xref: EC:1.1.1.44
 xref: MetaCyc:RXN-9952
-xref: Reactome:R-HSA-71299 "6-phospho-D-gluconate + NADP+ => D-ribulose 5-phosphate + CO2 + NADPH + H+"
+xref: Reactome:R-HSA-71299 "PGD decarboxylates 6-phospho-D-gluconate"
 xref: RHEA:10116
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.44
@@ -47757,7 +47886,7 @@
 xref: KEGG_REACTION:R01512
 xref: MetaCyc:PHOSGLYPHOS-RXN
 xref: Reactome:R-HSA-70486 "PGK complexes (PGK1,2) phosphorylate 3PG to form 1,3BPG"
-xref: Reactome:R-HSA-71850 "1,3-bisphospho-D-glycerate + ADP <=> 3-phospho-D-glycerate + ATP"
+xref: Reactome:R-HSA-71850 "PGK complexes dephosphorylate 1,3BPG"
 xref: RHEA:14801
 is_a: GO:0016301 ! kinase activity
 is_a: GO:0016774 ! phosphotransferase activity, carboxyl group as acceptor
@@ -47783,8 +47912,8 @@
 xref: KEGG_REACTION:R01516
 xref: MetaCyc:3PGAREARR-RXN
 xref: MetaCyc:RXN-15513
-xref: Reactome:R-HSA-71445 "PGAM dimers (PGAM1,2) isomerise 2PG to 3PG"
-xref: Reactome:R-HSA-71654 "3-Phospho-D-glycerate <=> 2-Phospho-D-glycerate"
+xref: Reactome:R-HSA-71445 "PGAM dimers isomerise 2PG to 3PG"
+xref: Reactome:R-HSA-71654 "PGAM dimers isomerise 3PG to 2PG"
 xref: RHEA:15901
 is_a: GO:0016868 ! intramolecular phosphotransferase activity
 property_value: skos:exactMatch RHEA:15901
@@ -47815,14 +47944,16 @@
 id: GO:0004621
 name: GPI anchor phospholipase D activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycoprotein phosphatidylinositol + H2O = phosphatidate + glycoprotein inositol, by cleavage of the second phosphodiester bond between the phosphate and phospholipid." [PMID:2760042]
+def: "Catalysis of the reaction: glycoprotein phosphatidylinositol + H2O = phosphatidate + glycoprotein inositol, by cleavage of the second phosphodiester bond between the phosphate and phospholipid." [PMID:2760042, RHEA:86263]
 synonym: "glycoprotein phospholipase D activity" BROAD []
 synonym: "glycoprotein-phosphatidylinositol phosphatidohydrolase activity" EXACT []
 synonym: "glycosylphosphatidyl inositol-anchor protein phospholipase D activity" EXACT []
 synonym: "GPI-PLD activity" EXACT []
 xref: Reactome:R-HSA-8940388 "GPLD1 hydrolyses GPI-anchors from proteins"
+xref: RHEA:86263
 is_a: GO:0004630 ! D-type glycerophospholipase activity
 property_value: skos:broadMatch EC:3.1.4.50
+property_value: skos:exactMatch RHEA:86263
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28868" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31078" xsd:anyURI
 
@@ -48104,7 +48235,7 @@
 xref: EC:4.2.1.11
 xref: MetaCyc:2PGADEHYDRAT-RXN
 xref: Reactome:R-HSA-70494 "Enolase dimers (ENO1,2,3) convert PEP to 2PG"
-xref: Reactome:R-HSA-71660 "2-Phospho-D-glycerate <=> Phosphoenolpyruvate + H2O"
+xref: Reactome:R-HSA-71660 "Enolase dimers dehydrates 2PG"
 xref: RHEA:10164
 is_a: GO:0016836 ! hydro-lyase activity
 property_value: skos:exactMatch EC:4.2.1.11
@@ -48364,15 +48495,16 @@
 
 [Term]
 id: GO:0004648
-name: O-phospho-L-serine:2-oxoglutarate aminotransferase activity
+name: O-phospho-L-serine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 alt_id: GO:0004646
-def: "Catalysis of the reaction: O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate + L-glutamate." [EC:2.6.1.52]
+def: "Catalysis of the reaction: O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate + L-glutamate." [RHEA:14329]
 synonym: "3-O-phospho-L-serine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.52]
 synonym: "3-phosphoserine aminotransferase activity" RELATED [EC:2.6.1.52]
 synonym: "3PHP transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "hydroxypyruvic phosphate--glutamic transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "L-phosphoserine aminotransferase activity" RELATED [EC:2.6.1.52]
+synonym: "O-phospho-L-serine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "PdxC" RELATED [EC:2.6.1.52]
 synonym: "phosphohydroxypyruvate transaminase activity" RELATED [EC:2.6.1.52]
 synonym: "phosphohydroxypyruvic--glutamic transaminase activity" RELATED [EC:2.6.1.52]
@@ -48388,6 +48520,7 @@
 property_value: skos:exactMatch EC:2.6.1.52
 property_value: skos:exactMatch RHEA:14329
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004649
@@ -48437,6 +48570,7 @@
 xref: EC:3.1.3.33
 xref: MetaCyc:RXN-21861
 xref: Reactome:R-HSA-77078 "Hydrolysis of the 5'-end of the nascent transcript by the capping enzyme"
+xref: Reactome:R-HSA-9923980 "NS3 cleaves triphosphate moiety of genomic +ssRNA"
 xref: RHEA:67772
 xref: RHEA:67776
 is_a: GO:0016791 ! phosphatase activity
@@ -48594,8 +48728,7 @@
 xref: EC:1.5.5.2
 xref: MetaCyc:RXN-14903
 xref: Reactome:R-HSA-6784224 "PRODH2:FAD dimer dehydrogenates HPRO to 1-pyrroline-3-hydroxy-5-carboxylate"
-xref: Reactome:R-HSA-70670 "PRODH oxidises L-Pro to 1PYR-5COOH"
-xref: Reactome:R-HSA-8955817 "PRODH2 oxidises HPRO to 1PYR-5COOH"
+xref: Reactome:R-HSA-8955817 "PRODH2 oxidises HPRO to 1PYR-3OH-5COOH"
 xref: RHEA:23784
 is_a: GO:0016649 ! oxidoreductase activity, acting on the CH-NH group of donors, quinone or similar compound as acceptor
 property_value: skos:exactMatch EC:1.5.5.2
@@ -48645,6 +48778,7 @@
 xref: MetaCyc:RXN-17573
 xref: Reactome:R-HSA-2530501 "FNTA:FNTB transfers FARN to GNGT1"
 xref: Reactome:R-HSA-9647978 "pro-RAS proteins are farnesylated"
+xref: Reactome:R-HSA-9948016 "FNTA:FNTB transfers farnesyl to GBP1"
 xref: RHEA:13345
 is_a: GO:0008318 ! protein prenyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.58
@@ -48676,6 +48810,8 @@
 synonym: "type I protein geranyl-geranyltransferase activity" RELATED [EC:2.5.1.59]
 xref: EC:2.5.1.59
 xref: MetaCyc:RXN-3701
+xref: Reactome:R-HSA-9947999 "FNTA:PGGT1B geranylgeranylates GBP2"
+xref: Reactome:R-HSA-9954195 "FNTA:PGGT1B geranylgeranylates GBP5"
 is_a: GO:0004661 ! protein geranylgeranyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.59
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25798" xsd:anyURI
@@ -48843,6 +48979,7 @@
 subset: goslim_candida
 synonym: "protamine kinase activity" NARROW []
 xref: Reactome:R-HSA-156832 "INF-gamma induced phosphorylation of L13a"
+xref: Reactome:R-HSA-5693148 "BCKDK phosphorylates BCKDH"
 xref: Reactome:R-HSA-937034 "IRAK1 phosphorylates Pellino"
 xref: Reactome:R-HSA-9604606 "Unknown kinase phosphorylates NICD4"
 xref: Reactome:R-HSA-9673284 "GTP-bound RAC1 contributes to MAPK8 activation"
@@ -48966,9 +49103,9 @@
 xref: Reactome:R-HSA-168140 "Active IKK Complex phosphorylates NF-kappa-B inhibitor"
 xref: Reactome:R-HSA-168184 "Activated TAK1 mediates phosphorylation of the IKK Complex"
 xref: Reactome:R-HSA-170055 "Myt-1 mediated phosphorylation of Cyclin B:Cdc2 complexes"
-xref: Reactome:R-HSA-170076 "CAK-mediated phosphorylation of Cyclin B1:Cdc2 complexes"
-xref: Reactome:R-HSA-170087 "CAK-mediated phosphorylation of Cyclin A:Cdc2 complexes"
-xref: Reactome:R-HSA-170116 "Myt-1 mediated phosphorylation of Cyclin A:Cdc2"
+xref: Reactome:R-HSA-170076 "CAK phosphorylates CDK1 bound to CCNB1"
+xref: Reactome:R-HSA-170087 "CAK phosphorylates CDK1 bound to CCNA"
+xref: Reactome:R-HSA-170116 "PKMYT1 phosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170126 "Phosphorylation of Cyclin B1 in the CRS domain"
 xref: Reactome:R-HSA-170977 "FRS2 is phosphorylated by active TrkA receptor"
 xref: Reactome:R-HSA-174119 "Free APC/C phosphorylated by Plk1"
@@ -48979,6 +49116,7 @@
 xref: Reactome:R-HSA-187949 "CAK-mediated phosphorylation of Cyclin A:Cdk2"
 xref: Reactome:R-HSA-188350 "CAK-mediated phosphorylation of Cyclin E:Cdk2"
 xref: Reactome:R-HSA-193647 "IRAK is activated"
+xref: Reactome:R-HSA-193703 "IKKbeta is activated"
 xref: Reactome:R-HSA-193705 "IKKbeta phosphorylates IkB causing NF-kB to dissociate"
 xref: Reactome:R-HSA-195275 "Phosphorylation of APC component of the destruction complex"
 xref: Reactome:R-HSA-195283 "Phosphorylation of phospho- (Ser45, Thr41) beta-catenin  at Ser37 by GSK-3"
@@ -49117,7 +49255,6 @@
 xref: Reactome:R-HSA-399951 "Phosphorylation of CRMPs by GSK3beta"
 xref: Reactome:R-HSA-399952 "Phosphorylation of LIMK-1 by PAK"
 xref: Reactome:R-HSA-399978 "Protein kinase C, alpha type phosphorylates MARCKS"
-xref: Reactome:R-HSA-400382 "CSNK1E,CSNK1D phosphorylate CRY and PER proteins"
 xref: Reactome:R-HSA-4088134 "PLK1 phosphorylates FOXM1"
 xref: Reactome:R-HSA-419083 "Myosin phosphatase inactivation by ROCK"
 xref: Reactome:R-HSA-419087 "LIM kinase phosphorylation by ROCK"
@@ -49178,8 +49315,10 @@
 xref: Reactome:R-HSA-5610722 "CK1 phosphorylates p-GLI3"
 xref: Reactome:R-HSA-5610730 "GSK3 phosphorylates p-GLI2"
 xref: Reactome:R-HSA-5610732 "GSK3 phosphorylates p-GLI3"
+xref: Reactome:R-HSA-5623667 "PDPK1 phosphorylates PKN1,2,3"
 xref: Reactome:R-HSA-5624473 "Protein kinase A (PKA) and RPS6KA5 (MSK1) phosphorylates p65 (RELA) subunit"
 xref: Reactome:R-HSA-5624492 "PAK phosphorylates p21 RAF1 on S338"
+xref: Reactome:R-HSA-5625784 "PKN1 phosphorylates histone 3 of nucleosomes associate with KLK2 and KLK3 promoters"
 xref: Reactome:R-HSA-5627775 "Autophosphorylation of PAK1,2,3"
 xref: Reactome:R-HSA-5632670 "CSNK1A1 phosphorylates SMO dimer"
 xref: Reactome:R-HSA-5632672 "ADRBK1 phosphorylates SMO dimer"
@@ -49192,6 +49331,7 @@
 xref: Reactome:R-HSA-5668947 "PAK1 phosphorylates myosin phosphatase"
 xref: Reactome:R-HSA-5668984 "PAK1 or PAK2 phosphorylates MYLK"
 xref: Reactome:R-HSA-5669250 "PAK1 phosphorylates FLNA"
+xref: Reactome:R-HSA-5671749 "p-T774-PKN1 phosphorylates CDC25C"
 xref: Reactome:R-HSA-5671763 "p-T774-PKN1 phosphorylates PPP1R14A"
 xref: Reactome:R-HSA-5671919 "Activated CIT phosphorylates MRLCs"
 xref: Reactome:R-HSA-5672008 "Thr-180 of ULK1 is phosphorylated"
@@ -49274,8 +49414,8 @@
 xref: Reactome:R-HSA-6810233 "CDK7 phosphorylates serine-5 and serine-7 of heptad repeats in C-terminal domain of RNA polymerase II at snRNA promoter"
 xref: Reactome:R-HSA-6811454 "MAPKs phosphorylate PP2A"
 xref: Reactome:R-HSA-6814409 "CK2 phosphorylates PDCL"
-xref: Reactome:R-HSA-69604 "Phosphorylation of Cdc25A at Ser-123 by Chk1"
-xref: Reactome:R-HSA-69608 "Phosphorylation of Cdc25A at Ser-123 by Chk2"
+xref: Reactome:R-HSA-69604 "Phosphorylation of CDC25A by CHEK1"
+xref: Reactome:R-HSA-69608 "Phosphorylation of CDC25A by CHEK2"
 xref: Reactome:R-HSA-69685 "CHEK2 phosphorylates TP53"
 xref: Reactome:R-HSA-69891 "Phosphorylation and activation of CHEK2 by ATM"
 xref: Reactome:R-HSA-75010 "Phosphorylation of Cdc25C at Ser 216 by Chk1"
@@ -49323,6 +49463,7 @@
 xref: Reactome:R-HSA-9009208 "Activated ERKs phosphorylate RUNX2"
 xref: Reactome:R-HSA-9012319 "p-TEFb phosphorylates serine 2 in RNA polymerase II CTD"
 xref: Reactome:R-HSA-9013978 "Phosphorylation of IRF-3/IRF7 and their release from the activated TLR3 complex"
+xref: Reactome:R-HSA-9021357 "PRKCI phosphorylates ELF3"
 xref: Reactome:R-HSA-9022314 "HIPK2 phosphorylates MECP2"
 xref: Reactome:R-HSA-9023132 "AURKB phosphorylates MECP2 at S423"
 xref: Reactome:R-HSA-9032751 "Estrogen-independent phosphorylation of ESR1 S118 by MAPK1 and MAPK3"
@@ -49330,6 +49471,7 @@
 xref: Reactome:R-HSA-912470 "ATR phosphorylates Histone H2A.X at unsynapsed regions"
 xref: Reactome:R-HSA-913996 "PKA/PKG phosphorylate Rap1GAP2"
 xref: Reactome:R-HSA-918229 "Phosphorylation and release of IRF3/IRF7"
+xref: Reactome:R-HSA-927889 "SMG1 phosphorylates UPF1 (enhanced by Exon Junction Complex)"
 xref: Reactome:R-HSA-933525 "Phosphorylation and release of IRF7"
 xref: Reactome:R-HSA-934559 "SPRY2 is phosphorylated by phosphorylated MNK1"
 xref: Reactome:R-HSA-936951 "Activation of TAK1 complex bound to activated TLR4 complex"
@@ -49349,6 +49491,7 @@
 xref: Reactome:R-HSA-9624526 "AKT phosphorylates FOXO3 downstream of ESR1 and EGFR"
 xref: Reactome:R-HSA-9626880 "MAPK11 or MAPK14 phosphorylates NCF1 at Ser345"
 xref: Reactome:R-HSA-9627089 "CASP9 is phosphorylated at T412"
+xref: Reactome:R-HSA-9632858 "PRKCZ autophosphorylates downstream of estrogen stimulation"
 xref: Reactome:R-HSA-9632868 "CDKN1B is phosphorylated in response to estrogen"
 xref: Reactome:R-HSA-9633008 "p-T899-EIF2AK4 (GCN2) phosphorylates EIF2AS1"
 xref: Reactome:R-HSA-9633742 "EIF2AK4 (GCN2) dimer autophosphorylates"
@@ -49385,9 +49528,11 @@
 xref: Reactome:R-HSA-975874 "Second phosphorylation of IRAK1 by IRAK4 bound to MyD88:activated TLR complex"
 xref: Reactome:R-HSA-975878 "First phosphorylation of IRAK1 by IRAK4 bound to MyD88:activated TLR"
 xref: Reactome:R-HSA-9762094 "GSK3B phosphorylates p-NFE2L2"
+xref: Reactome:R-HSA-9765960 "ERKs phosphorylate MCRIP1"
 xref: Reactome:R-HSA-9766532 "SQSTM1 oligomer is phosphorylated"
 xref: Reactome:R-HSA-9769115 "MAPK1,(MAPK3) phosphorylates NPAS4"
 xref: Reactome:R-HSA-9773803 "Active IKBKB phosphorylates NF-kappa-B inhibitor"
+xref: Reactome:R-HSA-9796053 "PRKCI phosphorylates NFE2L2"
 xref: Reactome:R-HSA-9796067 "PRKAA2 phosphorylates nuclear NFE2L2"
 xref: Reactome:R-HSA-9796081 "EIF2AK3 phosphorylates NFE2L2"
 xref: Reactome:R-HSA-9815501 "MAPKAPK2 phosphorylates RIPK1 at S320"
@@ -49411,10 +49556,31 @@
 xref: Reactome:R-HSA-9851972 "PLK1 phosphorylates FIRRM at S43"
 xref: Reactome:R-HSA-9853369 "PLK1 phosphorylates FIRMM at S744"
 xref: Reactome:R-HSA-9855910 "USF1 is phosphorylated by p-MAPK14"
+xref: Reactome:R-HSA-9857661 "p-T308,S473-AKT1 (phospho-PKB) phosphorylates GJA1 (Connexin-43)"
+xref: Reactome:R-HSA-9857673 "p-T308,S473-AKT1 phosphorylates ITGA5 (Integrin alpha5) in ITGA5:ITGB1 (Integrin alpha5:Integrin beta1)"
 xref: Reactome:R-HSA-9860292 "Activated IKBKB phosphorylates IRF5"
 xref: Reactome:R-HSA-9860785 "PDPK1 (PDK1) phosphorylates PKN2 on threonine-816"
 xref: Reactome:R-HSA-9860800 "mTORC2 phosphorylates AKT1 on serine-473"
 xref: Reactome:R-HSA-9861642 "NEK1 phosphorylates ME1"
+xref: Reactome:R-HSA-9909369 "CSNK1E,CSNK1D phosphorylate CRY and PER proteins"
+xref: Reactome:R-HSA-9912396 "CDK5 phosphorylates CLOCK on serine-451 and serine-461"
+xref: Reactome:R-HSA-9912401 "CK2alpha phosphorylates BMAL1 (ARNTL) on serine-90"
+xref: Reactome:R-HSA-9920673 "CDK5 phosphorylates PER2 on serine-396"
+xref: Reactome:R-HSA-9922273 "CSNK1D,E phosphorylates CLOCK in p-S-CRY:p-S-PER:CSNK1D,E:p-S90,Ac-538-BMAL:p-2T,3S-CLOCK:target gene"
+xref: Reactome:R-HSA-9926483 "MARK3 phosphorylates MITF-M on S173"
+xref: Reactome:R-HSA-9929360 "GSK3B phosphorylates CD274"
+xref: Reactome:R-HSA-9931292 "AMPK phosphorylates CD274"
+xref: Reactome:R-HSA-9931409 "NEK2 phosphorylates CD274"
+xref: Reactome:R-HSA-9931418 "CK2 phosphorylates CD274"
+xref: Reactome:R-HSA-9933380 "Cytosolic tail of CDH1 is phosphorylated"
+xref: Reactome:R-HSA-9942578 "Phosphorylation of CDC25A by p38 MAPK"
+xref: Reactome:R-HSA-9943662 "CHEK1 phosphorylates NEK11"
+xref: Reactome:R-HSA-9943675 "GSK3B phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943676 "NEK11 phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943678 "CHEK1 phosphorylates CHEK2-phosphorylated CDC25A"
+xref: Reactome:R-HSA-9943687 "PLK3 phosphorylates CDC25A"
+xref: Reactome:R-HSA-9943736 "Casein kinase I phosphorylates CDC25A"
+xref: Reactome:R-HSA-9954237 "PIM1 phosphorylates GBP1"
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch EC:2.7.11.-
 property_value: skos:exactMatch MetaCyc:PROTEIN-KINASE-RXN
@@ -49685,7 +49851,7 @@
 xref: EC:2.7.11.11
 xref: Reactome:R-HSA-163672 "Phosphorylation of ChREBP at Thr(653) by PKA"
 xref: Reactome:R-HSA-163676 "Phosphorylation of pChREBP (Thr 653) at Ser(196) by PKA"
-xref: Reactome:R-HSA-163773 "Phosphorylation of PF2K-Pase by PKA catalytic subunit"
+xref: Reactome:R-HSA-163773 "PKA tetramer:cAMP phosphorylates PF2K-Pase"
 xref: Reactome:R-HSA-177275 "PKA phosphorylates DARPP-32 on Thr34"
 xref: Reactome:R-HSA-177284 "PKA phosphorylates PDE4B"
 xref: Reactome:R-HSA-432232 "Phosphorylation of Aquaporin-2 by Protein Kinase A (PKA)"
@@ -49722,6 +49888,7 @@
 xref: Reactome:R-HSA-1497853 "Sepiapterin reductase (SPR) is phosphorylated by Ca2+/calmodulin-dependent protein kinase II"
 xref: Reactome:R-HSA-418442 "PKG1 phosphorylates IRAG:IP3R1 inhibiting IP3-stimulated Ca2+ release"
 xref: Reactome:R-HSA-418549 "PKG1 phosphorylates BK channels"
+xref: Reactome:R-HSA-9920025 "PKG phosphorylates NS5"
 is_a: GO:0004690 ! cyclic nucleotide-dependent protein kinase activity
 relationship: has_part GO:0030553 ! cGMP binding
 property_value: skos:exactMatch EC:2.7.11.12
@@ -49779,7 +49946,7 @@
 xref: Reactome:R-HSA-2990882 "CDK1 phosphorylates NUP98"
 xref: Reactome:R-HSA-3788705 "CDKN1A (p21) prevents phosphorylation of Cdh1 by Cyclin A:Cdk2"
 xref: Reactome:R-HSA-380278 "CCNB1:p-T160-CDK1 phosphorylates NUMA1"
-xref: Reactome:R-HSA-4086410 "CDK1 phosphorylates BORA"
+xref: Reactome:R-HSA-4086410 "CCNB:CDK1 phosphorylates BORA"
 xref: Reactome:R-HSA-4088024 "CCNA:CDK1/2 complexes and CCNB1:CDK1 complexes phosphorylate FOXM1"
 xref: Reactome:R-HSA-5195402 "CDK1 phosphorylates LPIN"
 xref: Reactome:R-HSA-5244669 "CDK1 phosphorylates lamins and facilitates depolymerization of lamin filaments"
@@ -49790,12 +49957,21 @@
 xref: Reactome:R-HSA-68944 "Orc1 is phosphorylated by cyclin A/CDK2"
 xref: Reactome:R-HSA-69005 "Cdc6 protein is phosphorylated by CDK"
 xref: Reactome:R-HSA-69227 "Cyclin D:CDK4/6 phosphorylates RB1 and prevents RB1 binding to E2F1/2/3:DP1/2 complexes"
-xref: Reactome:R-HSA-69754 "Phosphorylation of proteins involved in G2/M transition by active Cyclin A1:Cdc2 complexes"
-xref: Reactome:R-HSA-69756 "Phosphorylation of proteins involved in G2/M transition by active Cyclin A2:Cdc2 complexes"
 xref: Reactome:R-HSA-9008412 "CDK4 phosphorylates RUNX2"
 xref: Reactome:R-HSA-9009282 "CDK1 phosphorylates RUNX2"
 xref: Reactome:R-HSA-9624800 "CDK1 phosphorylates LBR"
 xref: Reactome:R-HSA-9686521 "CDK2:CCNA phosphorylates TERF2"
+xref: Reactome:R-HSA-9929482 "CDK4:CCND1 phosphorylates SPOP"
+xref: Reactome:R-HSA-9929514 "CDK1 phosphorylates CCNA"
+xref: Reactome:R-HSA-9929532 "CCNA:CDK1 phosphorylates BORA"
+xref: Reactome:R-HSA-9929535 "CCNA:CDK1 phosphorylates WEE1"
+xref: Reactome:R-HSA-9929717 "CCNA:CDK1 phosphorylates FZR1"
+xref: Reactome:R-HSA-9929719 "CCNA:CDK1 phosphorylates TICRR"
+xref: Reactome:R-HSA-9929720 "CCNA:CDK1 phosphorylates OBI1"
+xref: Reactome:R-HSA-9929721 "CCNA:CDK1 phosphorylates SGO1"
+xref: Reactome:R-HSA-9929884 "CCNA:CDK1 phosphorylates HJURP"
+xref: Reactome:R-HSA-9929904 "CCNA:CDK1 phosphorylates MIS18BP1"
+xref: Reactome:R-HSA-9929935 "CCNA:CDK1 phosphorylates CDC25B"
 is_a: GO:0004674 ! protein serine/threonine kinase activity
 is_a: GO:0097472 ! cyclin-dependent protein kinase activity
 property_value: skos:exactMatch EC:2.7.11.22
@@ -49840,33 +50016,9 @@
 synonym: "protein kinase Cepsilon activity" NARROW [EC:2.7.11.13]
 synonym: "STK24" RELATED [EC:2.7.11.13]
 xref: EC:2.7.11.13
-xref: Reactome:R-HSA-1433508 "PKC alpha interacts with and phosphorylates KIT"
-xref: Reactome:R-HSA-193703 "IKKbeta is activated"
-xref: Reactome:R-HSA-198314 "DAG stimulates protein kinase C-delta"
-xref: Reactome:R-HSA-2179413 "Activated PKC-alpha  activate MMP3"
-xref: Reactome:R-HSA-2730863 "Phosphorylation of CARMA1 by PKC-theta"
-xref: Reactome:R-HSA-374664 "Phosphorylation and activation of Ezrin"
-xref: Reactome:R-HSA-429698 "PRKD1,2,3 phosphorylates CERT1-2"
-xref: Reactome:R-HSA-450533 "PKCdelta phosphorylates HuR"
-xref: Reactome:R-HSA-450550 "PKCalpha phosphorylates HuR"
-xref: Reactome:R-HSA-5138432 "DVL2 is phosphorylated by PKC"
 xref: Reactome:R-HSA-5218805 "PKC autophosphorylates"
-xref: Reactome:R-HSA-5218823 "PKC phosphorylates sphingosine kinase 1"
-xref: Reactome:R-HSA-5229194 "Depolymerization of lamin filaments after PKC-mediated phosphorylation"
-xref: Reactome:R-HSA-5623667 "PDPK1 phosphorylates PKN1,2,3"
-xref: Reactome:R-HSA-5625784 "PKN1 phosphorylates histone 3 of nucleosomes associate with KLK2 and KLK3 promoters"
-xref: Reactome:R-HSA-5671749 "p-T774-PKN1 phosphorylates CDC25C"
-xref: Reactome:R-HSA-74615 "PRKCA/Q phosphorylate RGS9-1:GN5B:RGS9BP"
 xref: Reactome:R-HSA-751040 "PKC phosphorylates G alpha (z)"
-xref: Reactome:R-HSA-8932284 "PKC phosphorylates NFE2L2"
-xref: Reactome:R-HSA-8934446 "Activated PKC phosphorylates SHC1"
-xref: Reactome:R-HSA-9010681 "PKC phosphorylates ROBO3.1"
-xref: Reactome:R-HSA-9021357 "PRKCI phosphorylates ELF3"
-xref: Reactome:R-HSA-909552 "Phosphorylation of STAT1 at Ser727"
-xref: Reactome:R-HSA-927889 "SMG1 phosphorylates UPF1 (enhanced by Exon Junction Complex)"
 xref: Reactome:R-HSA-9626817 "PKC phosphorylates NCF1"
-xref: Reactome:R-HSA-9632858 "PRKCZ autophosphorylates downstream of estrogen stimulation"
-xref: Reactome:R-HSA-9796053 "PRKCI phosphorylates NFE2L2"
 is_a: GO:0004674 ! protein serine/threonine kinase activity
 property_value: skos:exactMatch EC:2.7.11.13
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25457" xsd:anyURI
@@ -49881,10 +50033,19 @@
 synonym: "conventional protein kinase C activity" EXACT [PMID:9601053]
 xref: Reactome:R-HSA-114683 "Phosphorylation of Platelet Sec-1"
 xref: Reactome:R-HSA-114684 "Phosphorylation of Syntaxin-4"
+xref: Reactome:R-HSA-1433508 "PKC alpha interacts with and phosphorylates KIT"
+xref: Reactome:R-HSA-2179413 "Activated PKC-alpha  activate MMP3"
 xref: Reactome:R-HSA-416639 "Trafficking of GluR2-containing AMPA receptors to extrasynaptic sites"
 xref: Reactome:R-HSA-421007 "Endocytosis of Ca impermeable AMPA receptors"
+xref: Reactome:R-HSA-450550 "PKCalpha phosphorylates HuR"
+xref: Reactome:R-HSA-5138432 "DVL2 is phosphorylated by PKC"
+xref: Reactome:R-HSA-5218823 "PKC phosphorylates sphingosine kinase 1"
+xref: Reactome:R-HSA-5229194 "Depolymerization of lamin filaments after PKC-mediated phosphorylation"
+xref: Reactome:R-HSA-74615 "PRKCA/Q phosphorylate RGS9-1:GN5B:RGS9BP"
+xref: Reactome:R-HSA-8934446 "Activated PKC phosphorylates SHC1"
 xref: Reactome:R-HSA-8986937 "MECP2 is phosphorylated at T308"
 xref: Reactome:R-HSA-9005561 "Active PKA, CaMK IV do not phosphorylate MECP2 mutants R306C,(R306H) at T308"
+xref: Reactome:R-HSA-9010681 "PKC phosphorylates ROBO3.1"
 is_a: GO:0004697 ! diacylglycerol-dependent serine/threonine kinase activity
 is_a: GO:0009931 ! calcium-dependent protein serine/threonine kinase activity
 property_value: skos:broadMatch EC:2.7.11.13
@@ -49899,7 +50060,14 @@
 synonym: "atypical protein kinase C activity" RELATED []
 synonym: "calcium-independent PKC activity" EXACT []
 synonym: "novel protein kinase C activity" EXACT [PMID:9601053]
+xref: Reactome:R-HSA-198314 "DAG stimulates protein kinase C-delta"
+xref: Reactome:R-HSA-2730863 "Phosphorylation of CARMA1 by PKC-theta"
+xref: Reactome:R-HSA-374664 "Phosphorylation and activation of Ezrin"
+xref: Reactome:R-HSA-429698 "PRKD1,2,3 phosphorylates CERT1-2"
+xref: Reactome:R-HSA-450533 "PKCdelta phosphorylates HuR"
 xref: Reactome:R-HSA-5607740 "PKC-delta phosphorylates CARD9"
+xref: Reactome:R-HSA-8932284 "PKC phosphorylates NFE2L2"
+xref: Reactome:R-HSA-909552 "Phosphorylation of STAT1 at Ser727"
 is_a: GO:0004697 ! diacylglycerol-dependent serine/threonine kinase activity
 property_value: skos:broadMatch EC:2.7.11.13
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25457" xsd:anyURI
@@ -50167,7 +50335,7 @@
 xref: Reactome:R-HSA-1524186 "Phosphorylation of PLCgamma by PDGFR"
 xref: Reactome:R-HSA-1671691 "PRLR-bound STAT5 is phosphorylated"
 xref: Reactome:R-HSA-170070 "Wee1-mediated phosphorylation of Cyclin B1:phospho-Cdc2 complexes"
-xref: Reactome:R-HSA-170156 "Wee1-mediated phosphorylation of Cyclin A:phospho-Cdc2 complexes"
+xref: Reactome:R-HSA-170156 "WEE1 phosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170991 "SRC-1 autophosphorylates"
 xref: Reactome:R-HSA-171011 "Binding and activation of MAP Kinase"
 xref: Reactome:R-HSA-174164 "Phosphorylation of Cyclin A:Cdk2 at Tyr 15"
@@ -50526,6 +50694,8 @@
 xref: Reactome:R-HSA-9860759 "p-T816-PKN2 phosphorylates AKT1 on serine-308"
 xref: Reactome:R-HSA-9861469 "p-Y397-PTK2 (FAK) phosphorylates CHUK (IKKA)"
 xref: Reactome:R-HSA-9865196 "p-Y393-ABL1 phosphorylates tyrosine-407 of YAP1"
+xref: Reactome:R-HSA-9931290 "IL6-activated JAK1 phosphorylates CD274"
+xref: Reactome:R-HSA-9934751 "CDH1 is phosphorylated on tyrosine residues"
 xref: RHEA:10596
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch EC:2.7.10.-
@@ -50678,7 +50848,7 @@
 name: protein-lysine 6-oxidase activity
 namespace: molecular_function
 alt_id: GO:0018056
-def: "Catalysis of the reaction: peptidyl-L-lysyl-peptide + H2O + O2 = peptidyl-allysyl-peptide + NH3 + hydrogen peroxide." [EC:1.4.3.13]
+def: "Catalysis of the reaction: L-lysyl-[protein] + O2 + H2O = (S)-2-amino-6-oxohexanoyl-[protein] + H2O2 + NH4+." [RHEA:24544]
 synonym: "lysyl oxidase activity" RELATED [EC:1.4.3.13]
 synonym: "protein-L-lysine:oxygen 6-oxidoreductase (deaminating)" RELATED [EC:1.4.3.13]
 xref: EC:1.4.3.13
@@ -50711,7 +50881,7 @@
 xref: Reactome:R-HSA-1363276 "Dephosphorylation of p130 (RBL2) by PP2A"
 xref: Reactome:R-HSA-167072 "Hypophosphorylation of RNA Pol II CTD by FCP1P protein"
 xref: Reactome:R-HSA-170153 "Dephosphorylation of nuclear Cyclin B1:phospho-Cdc2 (Thr 14, Tyr15) complexes by Cdc25 phosphatases"
-xref: Reactome:R-HSA-170158 "Dephosphorylation of nuclear Cyclin A:phospho-Cdc2 complexes"
+xref: Reactome:R-HSA-170158 "CDC25A dephosphorylates CCNA:CDK1"
 xref: Reactome:R-HSA-170161 "Dephosphorylation of cytoplasmic Cyclin B1/B2:phospho-Cdc2 (Thr 14, Tyr 15) complexes by CDC25B"
 xref: Reactome:R-HSA-174110 "Cdc25A mediated dephosphorylation of Cyclin A:phospho-Cdk2"
 xref: Reactome:R-HSA-174124 "Dephosphorylation of phospho-Cdh1"
@@ -50760,8 +50930,8 @@
 xref: MetaCyc:3.1.3.16-RXN
 xref: Reactome:R-HSA-1295632 "PPA2A dephosphorylates SPRY2"
 xref: Reactome:R-HSA-163568 "phosphorylated perilipin + H2O -> perilipin + orthophosphate"
-xref: Reactome:R-HSA-163689 "Dephosphorylation of pChREBP (Ser 196) by PP2A"
-xref: Reactome:R-HSA-163750 "Dephosphorylation of phosphoPFKFB1 by PP2A complex"
+xref: Reactome:R-HSA-163689 "PP2A complex dephosphorylates pChREBP (MLXIPL)"
+xref: Reactome:R-HSA-163750 "PP2A complex dephosphorylates pPFKFB1"
 xref: Reactome:R-HSA-1638821 "PP2A-B56 dephosphorylates centromeric cohesin"
 xref: Reactome:R-HSA-199425 "PHLPP dephosphorylates S473 in AKT"
 xref: Reactome:R-HSA-199959 "ERKs are inactivated by protein phosphatase 2A"
@@ -50784,7 +50954,6 @@
 xref: Reactome:R-HSA-5683405 "PPP5C dephosphorylates TP53BP1"
 xref: Reactome:R-HSA-5687758 "PPP4C:PPP4R2 dephosphorylates RPA2"
 xref: Reactome:R-HSA-5692754 "CDC14A,B dephosphorylate p-3S,T MAPK6"
-xref: Reactome:R-HSA-5693153 "PPM1K dephosphorylates p-BCKDH"
 xref: Reactome:R-HSA-5694421 "PP6 dephosphorylates SEC24"
 xref: Reactome:R-HSA-6792863 "PP2A-PP2R5C dephosphorylates MDM2"
 xref: Reactome:R-HSA-6811504 "AKT1 dephosphorylation by PP2A-B56-beta,gamma"
@@ -50803,6 +50972,7 @@
 xref: Reactome:R-HSA-9865226 "PP2A dephosphorylates serine-127 of YAP1"
 xref: Reactome:R-HSA-9865238 "PP2A dephosphorylates serine-715 of PDE4D5 in p-S715-PDE4D5:integrin alpha5:integrin beta1:fibronectin"
 xref: Reactome:R-HSA-9912527 "H139Hfs13* PPM1K does not dephosphorylate BCKDH"
+xref: Reactome:R-HSA-9970190 "PP1 dephosphorylates RNA Pol II"
 xref: RHEA:20629
 xref: RHEA:47004
 is_a: GO:0004721 ! phosphoprotein phosphatase activity
@@ -51021,7 +51191,7 @@
 id: GO:0004733
 name: pyridoxamine phosphate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH4+ + hydrogen peroxide. This activity can also oxidize pyridoxine 5'-phosphate to pyridoxal 5'-phosphate + hydrogen peroxide." [PMID:11786019]
+def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH4+ + H2O2. This activity can also oxidize pyridoxine 5'-phosphate to pyridoxal 5'-phosphate + H2O2." [EC:1.4.3.5, PMID:11786019]
 synonym: "PMP oxidase activity" RELATED [EC:1.4.3.5]
 synonym: "pyridoxal 5'-phosphate synthase activity" RELATED [EC:1.4.3.5]
 synonym: "pyridoxamine 5'-phosphate oxidase activity" RELATED [EC:1.4.3.5]
@@ -51034,11 +51204,13 @@
 xref: MetaCyc:PMPOXI-RXN
 xref: Reactome:R-HSA-965019 "2xPNPO:2xFMN oxidizes PDXP to PXLP"
 xref: Reactome:R-HSA-965079 "2xPNPO:2xFMN oxidizes PXAP to PXLP"
+xref: RHEA:15149
 xref: RHEA:15817
 is_a: GO:0016641 ! oxidoreductase activity, acting on the CH-NH2 group of donors, oxygen as acceptor
 relationship: part_of GO:0042823 ! pyridoxal phosphate biosynthetic process
 property_value: skos:exactMatch EC:1.4.3.5
-property_value: skos:exactMatch RHEA:15817
+property_value: skos:narrowMatch RHEA:15149
+property_value: skos:narrowMatch RHEA:15817
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22748" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -51071,7 +51243,7 @@
 xref: EC:1.5.1.2
 xref: MetaCyc:PYRROLINECARBREDUCT-RXN
 xref: Reactome:R-HSA-6783939 "PYCR2 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
-xref: Reactome:R-HSA-6783955 "PYCRL decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
+xref: Reactome:R-HSA-6783955 "PYCR3 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
 xref: Reactome:R-HSA-70664 "PYCR1 decamer reduces (S)-1-pyrroline-5-carboxylate to L-Pro"
 xref: RHEA:14109
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
@@ -51230,7 +51402,7 @@
 synonym: "phosphoenolpyruvate kinase activity" RELATED [EC:2.7.1.40]
 xref: EC:2.7.1.40
 xref: MetaCyc:PEPDEPHOS-RXN
-xref: Reactome:R-HSA-71670 "PKM dephosphorylates PEP to PYR"
+xref: Reactome:R-HSA-71670 "Pyruvate kinase dephosphorylates PEP to PYR"
 xref: RHEA:18157
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
 relationship: part_of GO:0006096 ! glycolytic process
@@ -51347,7 +51519,6 @@
 name: ribose phosphate diphosphokinase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: D-ribose 5-phosphate + ATP = 5-phospho-alpha-D-ribose 1-diphosphate + AMP + 2 H+." [RHEA:15609]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "5-phosphoribose pyrophosphorylase activity" RELATED [EC:2.7.6.1]
 synonym: "5-phosphoribosyl-1-pyrophosphate synthetase activity" RELATED [EC:2.7.6.1]
 synonym: "5-phosphoribosyl-alpha-1-pyrophosphate synthetase activity" RELATED [EC:2.7.6.1]
@@ -51368,16 +51539,16 @@
 xref: EC:2.7.6.1
 xref: KEGG_REACTION:R01049
 xref: MetaCyc:PRPPSYN-RXN
-xref: Reactome:R-HSA-111215 "D-ribose 5-phosphate + 2'-deoxyadenosine 5'-triphosphate (dATP) => 5-Phospho-alpha-D-ribose 1-diphosphate (PRPP) + 2'-deoxyadenosine 5'-monophosphate"
-xref: Reactome:R-HSA-73580 "D-ribose 5-phosphate + ATP => 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) + adenosine 5'-monophosphate"
+xref: Reactome:R-HSA-111215 "PRPS1 dimer pyrophosphorylates R5P using dATP"
+xref: Reactome:R-HSA-73580 "PRPS dimers pyrophosphorylate R5P using ATP"
 xref: RHEA:15609
 is_a: GO:0016778 ! diphosphotransferase activity
 property_value: skos:exactMatch EC:2.7.6.1
 property_value: skos:exactMatch KEGG_REACTION:R01049
 property_value: skos:exactMatch MetaCyc:PRPPSYN-RXN
 property_value: skos:exactMatch RHEA:15609
-property_value: skos:relatedMatch MetaCyc:PWY0-662
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0004750
@@ -51403,7 +51574,7 @@
 xref: EC:5.1.3.1
 xref: KEGG_REACTION:R01529
 xref: MetaCyc:RIBULP3EPIM-RXN
-xref: Reactome:R-HSA-199803 "xylulose 5-phosphate <=> D-ribulose 5-phosphate"
+xref: Reactome:R-HSA-199803 "RPE dimers isomerize XY5P to RU5P"
 xref: Reactome:R-HSA-71303 "RPE dimers isomerise RU5P to XY5P"
 xref: RHEA:13677
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
@@ -51429,10 +51600,10 @@
 xref: EC:5.3.1.6
 xref: KEGG_REACTION:R01056
 xref: MetaCyc:RIB5PISOM-RXN
-xref: Reactome:R-HSA-177784 "ribose 5-phosphate <=> D-ribulose 5-phosphate"
+xref: Reactome:R-HSA-177784 "RPIA isomerizes ribose 5-phosphate to D-ribulose 5-phosphate"
 xref: Reactome:R-HSA-5660013 "Defective RPIA does not isomerize RU5P to R5P"
 xref: Reactome:R-HSA-5660015 "Defective RPIA does not isomerize R5P to RU5P"
-xref: Reactome:R-HSA-71306 "D-ribulose 5-phosphate <=> ribose 5-phosphate"
+xref: Reactome:R-HSA-71306 "RPIA isomerizes D-ribulose 5-phosphate to ribose 5-phosphate"
 xref: RHEA:14657
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.6
@@ -51549,13 +51720,14 @@
 
 [Term]
 id: GO:0004760
-name: L-serine-pyruvate transaminase activity
+name: L-serine:pyruvate transaminase activity
 namespace: molecular_function
 alt_id: GO:0004761
 alt_id: GO:0004762
 alt_id: GO:0004763
-def: "Catalysis of the reaction: L-serine + pyruvate = 3-hydroxypyruvate + L-alanine." [EC:2.6.1.51, RHEA:22852]
+def: "Catalysis of the reaction: L-serine + pyruvate = 3-hydroxypyruvate + L-alanine." [RHEA:22852]
 synonym: "hydroxypyruvate:L-alanine transaminase activity" RELATED [EC:2.6.1.51]
+synonym: "L-serine-pyruvate transaminase activity" EXACT []
 synonym: "L-serine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.51]
 synonym: "serine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.51]
 synonym: "serine-pyruvate aminotransferase activity" EXACT []
@@ -51567,10 +51739,11 @@
 xref: KEGG_REACTION:R00585
 xref: MetaCyc:SERINE--PYRUVATE-AMINOTRANSFERASE-RXN
 xref: RHEA:22852
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.51
 property_value: skos:exactMatch RHEA:22852
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004764
@@ -51793,7 +51966,7 @@
 name: succinate-CoA ligase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: succinate + CoA + nucleotide triphosphate = nucleotide diphosphate + phosphate + succinyl-CoA." [GOC:ai]
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 
 [Term]
 id: GO:0004775
@@ -51982,7 +52155,7 @@
 alt_id: GO:0008382
 alt_id: GO:0008383
 alt_id: GO:0016954
-def: "Catalysis of the reaction: 2 superoxide + 2 H+ = O2 + hydrogen peroxide." [EC:1.15.1.1, GOC:vw, PMID:15064408]
+def: "Catalysis of the reaction: 2 superoxide + 2 H+ = O2 + H2O2." [EC:1.15.1.1, GOC:vw, PMID:15064408]
 synonym: "copper, zinc superoxide dismutase activity" NARROW []
 synonym: "Cu,Zn-SOD" RELATED [EC:1.15.1.1]
 synonym: "Cu-Zn superoxide dismutase activity" NARROW [EC:1.15.1.1]
@@ -52032,7 +52205,7 @@
 id: GO:0004786
 name: obsolete Mn, Fe superoxide dismutase
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the function is covered by an existing term.
 synonym: "Mn, Fe superoxide dismutase" EXACT []
 is_obsolete: true
@@ -52207,6 +52380,7 @@
 synonym: "threonine dehydratase activity" BROAD []
 xref: EC:4.3.1.19
 xref: MetaCyc:THREDEHYD-RXN
+xref: Reactome:R-HSA-9014627 "SDS dimers:PXLP convert L-Thr to 2OBUTA"
 xref: RHEA:22108
 xref: UM-BBD_reactionID:r0433
 is_a: GO:0016841 ! ammonia-lyase activity
@@ -52331,7 +52505,6 @@
 xref: EC:1.21.99.4
 xref: MetaCyc:THYROXINE-DEIODINASE-RXN
 xref: Reactome:R-HSA-209772 "Thyroxine is deiodinated to triiodothyronine"
-xref: Reactome:R-HSA-350869 "Thyroxine is deiodinated to reverse triiodothyronine (RT3)"
 xref: RHEA:19745
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.99.4
@@ -52494,8 +52667,8 @@
 xref: EC:5.3.1.1
 xref: KEGG_REACTION:R01015
 xref: MetaCyc:TRIOSEPISOMERIZATION-RXN
-xref: Reactome:R-HSA-70454 "dihydroxyacetone phosphate <=> D-glyceraldehyde 3-phosphate"
-xref: Reactome:R-HSA-70481 "TPI1 isomerizes GA3P to DHAP"
+xref: Reactome:R-HSA-70454 "TPI1 dimer isomerizes DHAP to GA3P"
+xref: Reactome:R-HSA-70481 "TPI1 dimer isomerizes GA3P to DHAP"
 xref: RHEA:18585
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.1
@@ -53155,14 +53328,15 @@
 
 [Term]
 id: GO:0004838
-name: L-tyrosine-2-oxoglutarate transaminase activity
+name: L-tyrosine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tyrosine + 2-oxoglutarate = 4-hydroxyphenylpyruvate + L-glutamate." [RHEA:15093]
+def: "Catalysis of the reaction: L-tyrosine + 2-oxoglutarate = 3-(4-hydroxyphenyl)pyruvate + L-glutamate." [RHEA:15093]
 synonym: "glutamic phenylpyruvic aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "glutamic-hydroxyphenylpyruvic transaminase activity" RELATED [EC:2.6.1.5]
 synonym: "L-phenylalanine 2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "L-tyrosine aminotransferase activity" BROAD [EC:2.6.1.5]
 synonym: "L-tyrosine transaminase activity" BROAD []
+synonym: "L-tyrosine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "L-tyrosine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "phenylalanine aminotransferase activity" RELATED [EC:2.6.1.5]
 synonym: "phenylalanine transaminase activity" RELATED [EC:2.6.1.5]
@@ -53180,10 +53354,11 @@
 xref: Reactome:R-HSA-71155 "TAT deaminates tyrosine"
 xref: RHEA:15093
 xref: UM-BBD_reactionID:r0297
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:broadMatch EC:2.6.1.5
 property_value: skos:exactMatch RHEA:15093
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0004839
@@ -53219,13 +53394,6 @@
 synonym: "ubiquitin protein-ligase activity" NARROW []
 synonym: "ubiquitin-conjugating enzyme activity" NARROW []
 xref: KEGG_REACTION:R03876
-xref: Reactome:R-HSA-1169394 "ISGylation of IRF3"
-xref: Reactome:R-HSA-1169395 "ISGylation of viral protein NS1"
-xref: Reactome:R-HSA-1169397 "Activation of ISG15 by UBA7 E1 ligase"
-xref: Reactome:R-HSA-1169398 "ISGylation of host protein filamin B"
-xref: Reactome:R-HSA-1169402 "ISGylation of E2 conjugating enzymes"
-xref: Reactome:R-HSA-1169405 "ISGylation of protein phosphatase 1 beta (PP2CB)"
-xref: Reactome:R-HSA-1169406 "ISGylation of host proteins"
 xref: Reactome:R-HSA-1234163 "Cytosolic VBC complex ubiquitinylates hydroxyprolyl-HIF-alpha"
 xref: Reactome:R-HSA-1234172 "Nuclear VBC complex ubiquitinylates HIF-alpha"
 xref: Reactome:R-HSA-1253282 "ERBB4 ubiquitination by WWP1/ITCH"
@@ -53286,7 +53454,6 @@
 xref: Reactome:R-HSA-3781009 "NHLRC1 mediated ubiquitination of EPM2A and PPP1RC3 associated with glycogen-GYG1"
 xref: Reactome:R-HSA-3788724 "Cdh1:APC/C ubiquitinates EHMT1 and EHMT2"
 xref: Reactome:R-HSA-3797226 "Defective NHLRC1 does not ubiquitinate EPM2A (laforin) and PPP1R3C (PTG) (type 2B disease)"
-xref: Reactome:R-HSA-400267 "BTRC:CUL1:SKP1 (SCF-beta-TrCP1) ubiquitinylates PER proteins"
 xref: Reactome:R-HSA-4332236 "CBL neddylates TGFBR2"
 xref: Reactome:R-HSA-446877 "TRAF6 is K63 poly-ubiquitinated"
 xref: Reactome:R-HSA-450358 "Activated TRAF6 synthesizes unanchored polyubiquitin chains upon TLR stimulation"
@@ -53313,7 +53480,7 @@
 xref: Reactome:R-HSA-5691108 "SKP1:FBXL5:CUL1:NEDD8 ubiquitinylates IREB2"
 xref: Reactome:R-HSA-5693108 "TNFAIP3 (A20) ubiquitinates RIPK1 with K48-linked Ub chains"
 xref: Reactome:R-HSA-68712 "The geminin component of geminin:Cdt1 complexes is ubiquitinated, releasing Cdt1"
-xref: Reactome:R-HSA-69598 "Ubiquitination of phosphorylated Cdc25A"
+xref: Reactome:R-HSA-69598 "Ubiquitination of phosphorylated CDC25A"
 xref: Reactome:R-HSA-741386 "RIP2 induces K63-linked ubiquitination of NEMO"
 xref: Reactome:R-HSA-75824 "Ubiquitination of Cyclin D1"
 xref: Reactome:R-HSA-870449 "TRIM33 monoubiquitinates SMAD4"
@@ -53335,6 +53502,7 @@
 xref: Reactome:R-HSA-975118 "TRAF6 ubiquitinqtes IRF7 within the activated TLR7/8 or 9 complex"
 xref: Reactome:R-HSA-975147 "Auto ubiquitination of oligo-TRAF6 bound to p-IRAK2 at endosome membrane"
 xref: Reactome:R-HSA-9758604 "Ubiquitination of IKBKG by TRAF6"
+xref: Reactome:R-HSA-9766227 "MDM2 mediates CDH1 ubiquitination"
 xref: Reactome:R-HSA-9793444 "ITCH polyubiquitinates MLKL at K50"
 xref: Reactome:R-HSA-9793485 "PRKN polyubiquitinates RIPK3"
 xref: Reactome:R-HSA-9793679 "LUBAC ubiquitinates RIPK1 at K627"
@@ -53347,10 +53515,15 @@
 xref: Reactome:R-HSA-983153 "E1 mediated ubiquitin activation"
 xref: Reactome:R-HSA-983156 "Polyubiquitination of substrate"
 xref: Reactome:R-HSA-9833155 "Ubiquitination of STAT2"
-xref: Reactome:R-HSA-9833973 "ISGylation of PKR"
 xref: Reactome:R-HSA-9834070 "PRKN ubiquitinates MOM substrates"
 xref: Reactome:R-HSA-9861563 "CTLH E3 ligase ubiquitinates LDHA"
 xref: Reactome:R-HSA-9861640 "CTLH E3 ligase ubiquitinates PKM-1"
+xref: Reactome:R-HSA-9929350 "Ubiquitination of CD274 by BTRC-SCF E3-ligase complex"
+xref: Reactome:R-HSA-9934753 "CBLL1 ubiquitinates CDH1"
+xref: Reactome:R-HSA-9948301 "ZNF598 K63 polyubiquitinates RPS10 (eS10) and RPS20 (uS10) of the 40S ribosomal subunit"
+xref: Reactome:R-HSA-9954721 "NEDD8-K699-CUL2:ELOB:ELOC:RBX1:KLHDC10 K48 polyubiquitinates alanine-tailed nascent peptide"
+xref: Reactome:R-HSA-9955146 "S.flexneri ipaH9.8 ubiquitinates GBPs"
+xref: Reactome:R-HSA-9970191 "RBBP6 activates CPSF complex"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -53485,7 +53658,7 @@
 id: GO:0004846
 name: urate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: urate + O2 + H2O = 5-hydroxyisourate + hydrogen peroxide." [EC:1.7.3.3]
+def: "Catalysis of the reaction: urate + O2 + H2O = 5-hydroxyisourate + H2O2." [EC:1.7.3.3]
 synonym: "urate:oxygen oxidoreductase activity" RELATED [EC:1.7.3.3]
 synonym: "uric acid oxidase activity" RELATED [EC:1.7.3.3]
 synonym: "uricase activity" RELATED [EC:1.7.3.3]
@@ -53581,7 +53754,7 @@
 id: GO:0004851
 name: uroporphyrin-III C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + uroporphyrin III = 2 S-adenosyl-L-homocysteine + precorrin-2." [EC:2.1.1.107]
+def: "Catalysis of the reaction: uroporphyrinogen III + 2 S-adenosyl-L-methionine = precorrin-2 + 2 S-adenosyl-L-homocysteine + H+." [RHEA:32459]
 synonym: "adenosylmethionine-uroporphyrinogen III methyltransferase activity" RELATED [EC:2.1.1.107]
 synonym: "CobA" RELATED [EC:2.1.1.107]
 synonym: "CysG" RELATED [EC:2.1.1.107]
@@ -53671,7 +53844,7 @@
 id: GO:0004855
 name: xanthine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: xanthine + H2O + O2 = urate + hydrogen peroxide." [EC:1.17.3.2]
+def: "Catalysis of the reaction: xanthine + H2O + O2 = urate + H2O2." [EC:1.17.3.2]
 synonym: "hypoxanthine-xanthine oxidase activity" BROAD [EC:1.17.3.2]
 synonym: "schardinger enzyme" RELATED [EC:1.17.3.2]
 synonym: "Schardinger enzyme activity" RELATED [EC:1.17.3.2]
@@ -53719,6 +53892,8 @@
 def: "A molecular function regulator that reduces a catalytic activity." [GOC:ai, GOC:ebc]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
 synonym: "metalloenzyme inhibitor activity" NARROW []
+xref: Reactome:R-HSA-1474158 "GCHFR binds to GCH1 and negatively regulates its activity"
+xref: Reactome:R-HSA-1614336 "SUMF2 binds SUMF1, inhibiting SUMF1-mediated activation of arylsulfatases"
 is_a: GO:0030234 ! enzyme regulator activity
 is_a: GO:0140678 ! molecular function inhibitor activity
 relationship: negatively_regulates GO:0003824 ! catalytic activity
@@ -53854,7 +54029,7 @@
 alt_id: GO:0005062
 alt_id: GO:0009369
 alt_id: GO:0009370
-def: "OBSOLETE. Conveys a signal across a cell to trigger a change in cell function or state. A signal is a physical entity or change in state that is used to transfer information in order to trigger a response." [GOC:go_curators]
+def: "OBSOLETE. Conveys a signal across a cell to trigger a change in cell function or state. A signal is a physical entity or change in state that is used to transfer information in order to trigger a response." [GOC:curators]
 comment: This term was obsoleted because it was not clearly defined and is not a useful grouping term.
 synonym: "hematopoietin/interferon-class (D200-domain) cytokine receptor signal transducer activity" NARROW []
 synonym: "quorum sensing response regulator activity" NARROW []
@@ -53961,6 +54136,10 @@
 synonym: "vitamin A receptor activity" NARROW []
 synonym: "vitamin D receptor activity" NARROW []
 synonym: "vitamin D3 receptor activity" NARROW [GOC:bf, Wikipedia:Calcitriol_receptor]
+xref: Reactome:R-HSA-400143 "Fatty acid ligands activate PPARA"
+xref: Reactome:R-HSA-8936849 "AHR:2xHSP90:AIP:PTGES3 binds TCDD"
+xref: Reactome:R-HSA-8937169 "AHR:TCDD:2xHSP90AB1:AIP:PTGES3 translocates from cytosol to nucleoplasm"
+xref: Reactome:R-HSA-8937191 "AHR:TCDD:2xHSP90AB1:AIP:PTGES3 dissociates"
 is_a: GO:0000981 ! DNA-binding transcription factor activity, RNA polymerase II-specific
 is_a: GO:0038023 ! signaling receptor activity
 is_a: GO:0098531 ! ligand-modulated transcription factor activity
@@ -53986,7 +54165,7 @@
 namespace: molecular_function
 alt_id: GO:0004926
 alt_id: GO:0099600
-def: "Combining with an extracellular or intracellular signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity or state as part of signal transduction." [GOC:go_curators, Wikipedia:Transmembrane_receptor]
+def: "Combining with an extracellular or intracellular signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity or state as part of signal transduction." [GOC:curators, Wikipedia:Transmembrane_receptor]
 comment: This term includes intracellular membrane receptors, e.g. IP3 triggered release of Ca2+ from intracellular stores.
 synonym: "transmembrane receptor activity" BROAD [GOC:bf, GOC:signaling]
 synonym: "transmembrane signalling receptor activity" EXACT [GOC:mah]
@@ -54348,7 +54527,7 @@
 id: GO:0004928
 name: obsolete frizzled receptor activity
 namespace: molecular_function
-def: "OBSOLETE. Combining with a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:go_curators]
+def: "OBSOLETE. Combining with a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product rather than an activity.
 synonym: "frizzled receptor activity" EXACT []
 is_obsolete: true
@@ -54358,7 +54537,7 @@
 id: GO:0004929
 name: obsolete frizzled-2 receptor activity
 namespace: molecular_function
-def: "OBSOLETE. Combining with to a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:go_curators]
+def: "OBSOLETE. Combining with to a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product rather than an activity.
 synonym: "frizzled-2 receptor activity" EXACT []
 is_obsolete: true
@@ -54411,6 +54590,7 @@
 synonym: "purinoceptor" BROAD []
 synonym: "purinoreceptor" BROAD []
 xref: Reactome:R-HSA-877187 "P2X7 mediates loss of intracellular K+"
+xref: Reactome:R-HSA-9905964 "P2RX7 in ATP:P2RX7:PANX1 transports cations from the extracellular region to the cytosol"
 is_a: GO:0005231 ! excitatory extracellular ligand-gated monoatomic ion channel activity
 is_a: GO:0035381 ! ATP-gated ion channel activity
 is_a: GO:0099094 ! ligand-gated monoatomic cation channel activity
@@ -54704,6 +54884,7 @@
 name: glucagon receptor activity
 namespace: molecular_function
 def: "Combining with glucagon and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:mah, GOC:signaling, PMID:22438981]
+xref: Reactome:R-HSA-163625 "Glucagon binds to Glucagon receptor"
 is_a: GO:0008528 ! G protein-coupled peptide receptor activity
 relationship: part_of GO:0071377 ! cellular response to glucagon stimulus
 
@@ -54969,6 +55150,7 @@
 synonym: "thyroid stimulating hormone receptor activity" EXACT []
 synonym: "thyrotropin receptor" EXACT []
 synonym: "TSH receptor activity" EXACT []
+xref: Reactome:R-HSA-391375 "TSHR binds TSH/Thyrostimulin"
 xref: Wikipedia:Thyrotropin_receptor
 is_a: GO:0004930 ! G protein-coupled receptor activity
 is_a: GO:0016500 ! protein-hormone receptor activity
@@ -55145,7 +55327,7 @@
 id: GO:0005012
 name: obsolete Neu/ErbB-2 receptor activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a class of receptor rather than an activity.
 synonym: "Neu/ErbB-2 receptor activity" EXACT []
 is_obsolete: true
@@ -55542,7 +55724,7 @@
 id: GO:0005050
 name: obsolete peroxisome receptor
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes a cellular component.
 synonym: "peroxisome receptor" EXACT []
 is_obsolete: true
@@ -55579,7 +55761,7 @@
 id: GO:0005054
 name: obsolete peroxisome integral membrane receptor
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes a cellular component.
 synonym: "Pas20p" RELATED []
 synonym: "peroxisome integral membrane receptor" EXACT []
@@ -55627,7 +55809,7 @@
 id: GO:0005065
 name: obsolete heterotrimeric G-protein
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a cellular component and not a molecular function.
 synonym: "heterotrimeric G-protein" EXACT []
 is_obsolete: true
@@ -55637,7 +55819,7 @@
 id: GO:0005066
 name: obsolete transmembrane receptor protein tyrosine kinase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "transmembrane receptor protein tyrosine kinase signaling protein activity" EXACT []
 synonym: "transmembrane receptor protein tyrosine kinase signalling protein activity" EXACT []
@@ -55670,7 +55852,7 @@
 id: GO:0005071
 name: obsolete transmembrane receptor protein serine/threonine kinase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "transmembrane receptor protein serine/threonine kinase signaling protein activity" EXACT []
 synonym: "transmembrane receptor protein serine/threonine kinase signalling protein activity" EXACT []
@@ -55721,7 +55903,7 @@
 id: GO:0005076
 name: obsolete receptor signaling protein serine/threonine kinase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "receptor signaling protein serine/threonine kinase signaling protein activity" EXACT []
 synonym: "receptor signalling protein serine/threonine kinase signalling protein activity" EXACT []
@@ -55742,11 +55924,12 @@
 
 [Term]
 id: GO:0005078
-name: MAP-kinase scaffold activity
+name: MAP kinase scaffold activity
 namespace: molecular_function
 def: "The binding activity of a molecule that functions as a physical support for the assembly of a multiprotein mitogen-activated protein kinase (MAPK) complex. Binds multiple kinases of the MAPKKK cascade, and also upstream signaling proteins, permitting those molecules to function in a coordinated way. Bringing together multiple enzymes and their substrates enables the signal to be transduced quickly and efficiently." [PMID:12511654, PMID:15213240, PMID:9405336]
 synonym: "MAP-kinase scaffold protein activity" EXACT []
 is_a: GO:0035591 ! signaling adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 relationship: has_part GO:0051019 ! mitogen-activated protein kinase binding
 relationship: part_of GO:0000165 ! MAPK cascade
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14286" xsd:anyURI
@@ -55785,7 +55968,7 @@
 id: GO:0005081
 name: obsolete receptor signaling protein serine/threonine phosphatase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "receptor signaling protein serine/threonine phosphatase signaling protein activity" EXACT []
 synonym: "receptor signalling protein serine/threonine phosphatase signalling protein activity" EXACT []
@@ -55796,7 +55979,7 @@
 id: GO:0005082
 name: obsolete receptor signaling protein tyrosine phosphatase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "receptor signaling protein tyrosine phosphatase signaling protein activity" EXACT []
 synonym: "receptor signalling protein tyrosine phosphatase signalling protein activity" EXACT []
@@ -55881,6 +56064,7 @@
 xref: Reactome:R-HSA-379044 "Liganded Gs-activating GPCR acts as a GEF for Gs"
 xref: Reactome:R-HSA-379048 "Liganded Gq/11-activating GPCRs act as GEFs for Gq/11"
 xref: Reactome:R-HSA-380073 "Liganded Gi-activating GPCR acts as a GEF for Gi"
+xref: Reactome:R-HSA-381727 "RAP1A exchanges GDP for GTP"
 xref: Reactome:R-HSA-389348 "Activation of Rac1 by pVav1"
 xref: Reactome:R-HSA-389350 "Activation of Cdc42 by pVav1"
 xref: Reactome:R-HSA-392054 "NCAM1:pFAK:Grb2:Sos-mediated nucleotide exchange of Ras"
@@ -55896,6 +56080,7 @@
 xref: Reactome:R-HSA-3965444 "WNT:FZD complex promotes G-protein nucleotide exchange"
 xref: Reactome:R-HSA-399938 "Activation of Rac1 by FARP2"
 xref: Reactome:R-HSA-399995 "Muscarinic Acetylcholine Receptor M3 activates Gq"
+xref: Reactome:R-HSA-400037 "Gi,Go Heterotrimeric G-protein complex dissociates"
 xref: Reactome:R-HSA-400092 "Alpha-2A,alpha-2C Adrenergic Receptors activate Gi.Go heterotrimeric G proteins"
 xref: Reactome:R-HSA-4093336 "p-TIAM1 exchanges GTP for GDP on RAC1, activating it"
 xref: Reactome:R-HSA-416530 "FFAR1:FFAR1 ligands activate Gq"
@@ -56606,7 +56791,7 @@
 id: GO:0005149
 name: interleukin-1 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-1 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-1 receptor." [GOC:curators]
 synonym: "IL-1" NARROW []
 synonym: "interleukin-1 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -56645,7 +56830,7 @@
 id: GO:0005153
 name: interleukin-8 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-8 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-8 receptor." [GOC:curators]
 subset: goslim_chembl
 synonym: "IL-8" NARROW []
 synonym: "interleukin-8 receptor ligand" NARROW []
@@ -56991,7 +57176,7 @@
 id: GO:0005187
 name: obsolete storage protein
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes a cellular location rather than a function.
 synonym: "storage protein" EXACT []
 is_obsolete: true
@@ -57129,7 +57314,7 @@
 id: GO:0005202
 name: obsolete collagen
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not represent a molecular function.
 synonym: "collagen" EXACT []
 is_obsolete: true
@@ -57139,7 +57324,7 @@
 id: GO:0005203
 name: obsolete proteoglycan
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes the composition (i.e. proteoglycan) of a gene product, not a molecular function.
 synonym: "proteoglycan" EXACT []
 is_obsolete: true
@@ -57148,7 +57333,7 @@
 id: GO:0005204
 name: obsolete chondroitin sulfate proteoglycan
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes the composition (i.e. proteoglycan) of a gene product, not a molecular function.
 synonym: "chondroitin sulfate proteoglycan" EXACT []
 synonym: "chondroitin sulphate proteoglycan" EXACT []
@@ -57158,7 +57343,7 @@
 id: GO:0005205
 name: obsolete chondroitin sulfate/dermatan sulfate proteoglycan
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes the composition (i.e. proteoglycan) of a gene product, not a molecular function.
 synonym: "chondroitin sulfate/dermatan sulfate proteoglycan" EXACT []
 synonym: "chondroitin sulphate/dermatan sulphate proteoglycan" EXACT []
@@ -57440,7 +57625,7 @@
 id: GO:0005241
 name: obsolete inward rectifier channel
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "inward rectifier channel" EXACT []
 is_obsolete: true
@@ -57513,6 +57698,7 @@
 xref: Reactome:R-HSA-5577213 "LTCC multimer transports Ca2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9701055 "TRPA1 tetramer transports Ca2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9701141 "TRPV1 transports Ca2+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-9905934 "CaV3.2 (CACNA1H:CACNA2D1:CACNB1,2,3:CACNG7) transports calcium from the extracellular region to the cytosol"
 is_a: GO:0005262 ! calcium channel activity
 is_a: GO:0022843 ! voltage-gated monoatomic cation channel activity
 
@@ -57654,12 +57840,9 @@
 synonym: "cation channel activity" BROAD []
 synonym: "cation diffusion facilitator activity" EXACT []
 synonym: "non-selective cation channel activity" NARROW []
-xref: Reactome:R-HSA-1168376 "STIM1 oligomerizes"
 xref: Reactome:R-HSA-1296043 "Activation of HCN channels"
-xref: Reactome:R-HSA-169683 "IP3R:I(1,4,5)P3 tetramer transports Ca2+ from ER lumen to cytosol"
 xref: Reactome:R-HSA-2089943 "TRPC1 translocates calcium from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-426223 "Cation influx mediated by TRPC3/6/7"
-xref: Reactome:R-HSA-4420052 "Decreasing cGMP concentration promotes intracellular Ca2+ release in response to WNT"
 xref: Reactome:R-HSA-9706720 "SARS-CoV-1 E transports Ca2+"
 xref: Reactome:R-HSA-9712190 "cAMP:olfactory CNG channel translocates Na+ and Ca2+ from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-9754616 "SARS-CoV-2 E pentamer transports Ca2+"
@@ -58070,6 +58253,7 @@
 alt_id: GO:1901677
 def: "Enables the transfer of phosphate ions from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Secondary active transporters include symporters and antiporters." [GOC:mtg_transport]
 synonym: "inorganic phosphate transmembrane transporter activity" EXACT []
+xref: Reactome:R-HSA-5226964 "ANKH transports PPi from cytosol to extracellular region"
 is_a: GO:0015291 ! secondary active transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27121" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -58088,7 +58272,7 @@
 id: GO:0005318
 name: obsolete phosphate:hydrogen symporter
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 synonym: "phosphate:hydrogen symporter" EXACT []
 synonym: "phosphate:proton symporter activity" EXACT []
 is_obsolete: true
@@ -58096,14 +58280,16 @@
 
 [Term]
 id: GO:0005319
-name: lipid transporter activity
+name: lipid carrier activity
 namespace: molecular_function
-def: "Enables the directed movement of lipids into, out of or within a cell, or between cells." [GOC:ai]
-synonym: "apolipoprotein" RELATED []
+def: "Directly binding to a specific lipid and delivering it either to an acceptor molecule or to a specific location." [GOC:curators]
+synonym: "apolipoprotein" NARROW []
+synonym: "lipid transporter activity" BROAD []
 synonym: "lipophorin" NARROW []
 xref: Reactome:R-HSA-1369028 "ABCAs mediate lipid efflux"
 xref: Reactome:R-HSA-1369052 "ABCAs mediate lipid influx"
 xref: Reactome:R-HSA-174786 "ApoB-48 + 40 triacylglycerol + 60 phospholipid => ApoB-48:TG:PL complex"
+xref: Reactome:R-HSA-5334794 "FABPs bind LCFAs"
 xref: Reactome:R-HSA-5682285 "ABCA12 transports lipids from cytosol to extracellular region"
 xref: Reactome:R-HSA-5682311 "Defective ABCA12 does not transport lipids from cytosol to extracellular region"
 xref: Reactome:R-HSA-5683672 "Defective ABCA3 does not transport PC, PG from ER membrane to lamellar body"
@@ -58112,8 +58298,9 @@
 xref: Reactome:R-HSA-6801250 "TRIAP1:PRELID1, PRELID3A transports PA from the outer to the inner mitochondrial membrane"
 xref: Reactome:R-HSA-8848053 "ABCA5 transports CHOL from lysosomal lumen to cytosol"
 xref: Reactome:R-HSA-8866329 "MTTP lipidates APOB-100, forming a pre-VLDL"
-is_a: GO:0005215 ! transporter activity
-relationship: part_of GO:0006869 ! lipid transport
+xref: Reactome:R-HSA-8873834 "STARD10 transports PC from ER membrane to lamellar body membrane"
+is_a: GO:0140104 ! molecular carrier activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0005320
@@ -58168,10 +58355,11 @@
 def: "Enables the transfer of a long-chain fatty acid from one side of a membrane to the other. A long-chain fatty acid has an aliphatic tail containing 13 to 22 carbons." [ISBN:0198506732]
 comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
 synonym: "long-chain fatty acid transporter activity" RELATED []
+xref: Reactome:R-HSA-2026996 "FABP1 transports fatty acids into the nucleus"
 xref: Reactome:R-HSA-2046087 "Translocation of tetracosahexaenoyl-CoA to peroxisomes"
 xref: Reactome:R-HSA-2046093 "Translocation of tetracosapentaenoyl-CoA to peroxisomes"
 xref: Reactome:R-HSA-382575 "ABCD1-3 dimers transfer LCFAs from cytosol to peroxisomal matrix"
-xref: Reactome:R-HSA-434381 "CD36 (FAT) translocates palmitate from the extracellular region to the cytosol"
+xref: Reactome:R-HSA-434381 "CD36 (FAT) translocates a long chain fatty acid from the extracellular region to the cytosol"
 xref: Reactome:R-HSA-5684043 "Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix"
 xref: RHEA:39283
 is_a: GO:0015245 ! fatty acid transmembrane transporter activity
@@ -58257,7 +58445,7 @@
 synonym: "norepinephrine transmembrane transporter activity" EXACT []
 synonym: "norepinephrine:sodium:chloride symporter activity" EXACT []
 synonym: "sodium/norepinephrine symporter activity" EXACT []
-xref: Reactome:R-HSA-443997 "SLC6A2 cotransports NAd, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-443997 "SLC6A2 cotransports NAd, Na+,Cl- from extracellular region to cytosol"
 xref: Reactome:R-HSA-5659764 "Defective SLC6A2 does not cotransport NAd, Na+ from extracellular region to cytosol"
 xref: Wikipedia:Norepinephrine_transporter
 is_a: GO:0008504 ! monoamine transmembrane transporter activity
@@ -58371,7 +58559,7 @@
 name: purine nucleobase transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, from one side of a membrane to the other." [ISBN:0198506732]
-synonym: "purine base transmembrane transporter activity" EXACT [GOC:go_curators]
+synonym: "purine base transmembrane transporter activity" EXACT [GOC:curators]
 synonym: "purine transmembrane transporter activity" RELATED []
 is_a: GO:0015205 ! nucleobase transmembrane transporter activity
 relationship: part_of GO:1904823 ! purine nucleobase transmembrane transport
@@ -58400,7 +58588,7 @@
 name: pyrimidine nucleobase transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of pyrimidine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, from one side of a membrane to the other." [GOC:ai]
-synonym: "pyrimidine base transmembrane transporter activity" EXACT [GOC:go_curators]
+synonym: "pyrimidine base transmembrane transporter activity" EXACT [GOC:curators]
 synonym: "pyrimidine transmembrane transporter activity" RELATED []
 is_a: GO:0015205 ! nucleobase transmembrane transporter activity
 relationship: part_of GO:0015855 ! pyrimidine nucleobase transport
@@ -58474,8 +58662,6 @@
 synonym: "glucose permease activity" EXACT []
 synonym: "glucose transmembrane transporter activity" EXACT []
 synonym: "lactose/glucose efflux transporter activity" NARROW []
-xref: Reactome:R-HSA-499981 "Transport of Extracellular Glucose to the Cytosol by GLUT1 and GLUT2"
-xref: Reactome:R-HSA-5653873 "SLC2A1 tetramer transports Glc from cytosol to Golgi lumen"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27326" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0055056
@@ -58578,6 +58764,11 @@
 namespace: molecular_function
 def: "Enables the transfer of myo-inositol from one side of a membrane to the other. Myo-inositol is 1,2,3,4,5/4,6-cyclohexanehexol, a growth factor for animals and microorganisms." [GOC:ai]
 synonym: "vitamin Bh transporter activity" EXACT []
+xref: Reactome:R-HSA-1855155 "I(1,3,4,5,6)P5 transports from the ER lumen to the cytosol"
+xref: Reactome:R-HSA-1855164 "IP6 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855186 "I(1,3,4,5)P4 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855195 "I(1,3,4,5,6)P5 transports from the cytosol to the ER lumen"
+xref: Reactome:R-HSA-1855217 "I(1,4,5)P3 transports from the ER lumen to the cytosol"
 is_a: GO:0015166 ! polyol transmembrane transporter activity
 relationship: part_of GO:0015798 ! myo-inositol transport
 
@@ -58665,7 +58856,7 @@
 synonym: "copper uptake transmembrane transporter activity" RELATED []
 synonym: "intracellular copper ion transporter" NARROW []
 synonym: "plasma membrane copper transporter" NARROW []
-xref: Reactome:R-HSA-437300 "SLC31A1 transports Cu2+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-437300 "SLC31A1 transports Cu1+ from extracellular region to cytosol"
 is_a: GO:0046915 ! transition metal ion transmembrane transporter activity
 relationship: part_of GO:0035434 ! copper ion transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -58754,6 +58945,7 @@
 xref: Reactome:R-HSA-442422 "SLC39A1-4 transports Zn2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5654125 "Defective SLC39A4 does not transport Zn2+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9820566 "ZnT5 transports zinc into the golgi apparatus"
+xref: Reactome:R-HSA-9928035 "ZnuABC transports Zn2+ to cytosol"
 is_a: GO:0046915 ! transition metal ion transmembrane transporter activity
 relationship: part_of GO:0071577 ! zinc ion transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
@@ -59033,7 +59225,7 @@
 namespace: molecular_function
 def: "Enables the transfer of a GDP-fucose from one side of a membrane to the other. GDP-fucose is a substance composed of fucose in glycosidic linkage with guanosine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
 xref: Reactome:R-HSA-5653596 "Defective SLC35C1 does not transport UDP-Fuc from cytosol to Golgi lumen"
-xref: Reactome:R-HSA-742345 "SLC35C1 transport UDP-Fuc from cytosol to Golgi lumen"
+xref: Reactome:R-HSA-742345 "SLC35C1 transport GDP-Fuc from cytosol to Golgi lumen"
 is_a: GO:0036080 ! purine nucleotide-sugar transmembrane transporter activity
 relationship: part_of GO:0015783 ! GDP-fucose transmembrane transport
 
@@ -59111,7 +59303,7 @@
 namespace: molecular_function
 alt_id: GO:0005453
 alt_id: GO:0005454
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not provide a useful functional classification.
 synonym: "small-molecule carrier or transporter" EXACT []
 is_obsolete: true
@@ -59157,7 +59349,7 @@
 synonym: "carnitine/acyl carnitine carrier activity" EXACT []
 synonym: "carnitine:acyl carnitine carrier activity" EXACT []
 synonym: "fatty acyl carnitine carrier" EXACT []
-xref: Reactome:R-HSA-200424 "Exchange of palmitoylcarnitine and carnitine across the inner mitochondrial membrane"
+xref: Reactome:R-HSA-200424 "Exchange of acylcarnitine and carnitine across the inner mitochondrial membrane"
 is_a: GO:0015226 ! carnitine transmembrane transporter activity
 is_a: GO:0015227 ! O-acyl-L-carnitine transmembrane transporter activity
 is_a: GO:0015297 ! antiporter activity
@@ -59175,7 +59367,7 @@
 id: GO:0005479
 name: obsolete vacuolar assembly
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a biological process and not a molecular function.
 synonym: "vacuolar assembly" EXACT []
 is_obsolete: true
@@ -59185,7 +59377,7 @@
 id: GO:0005480
 name: obsolete vesicle transport
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a biological process and not a molecular function.
 synonym: "vesicle transport" EXACT []
 is_obsolete: true
@@ -59195,7 +59387,7 @@
 id: GO:0005481
 name: obsolete vesicle fusion
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a biological process and not a molecular function.
 synonym: "vesicle fusion" EXACT []
 is_obsolete: true
@@ -59208,7 +59400,7 @@
 id: GO:0005482
 name: obsolete vesicle targeting
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a biological process and not a molecular function.
 synonym: "vesicle targeting" EXACT []
 is_obsolete: true
@@ -59240,6 +59432,8 @@
 synonym: "v-SNARE activity" NARROW []
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
 relationship: part_of GO:0061025 ! membrane fusion
+relationship: part_of GO:0140522 ! fusogenic activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31564" xsd:anyURI
 
 [Term]
 id: GO:0005488
@@ -59442,7 +59636,7 @@
 namespace: molecular_function
 alt_id: GO:0001948
 alt_id: GO:0045308
-def: "Binding to a protein." [GOC:go_curators]
+def: "Binding to a protein." [GOC:curators]
 subset: goslim_candida
 subset: goslim_chembl
 subset: goslim_euk_cellular_processes_ribbon
@@ -59486,7 +59680,7 @@
 id: GO:0005519
 name: cytoskeletal regulatory protein binding
 namespace: molecular_function
-def: "Binding to a protein involved in modulating the reorganization of the cytoskeleton." [GOC:go_curators, PMID:15163540]
+def: "Binding to a protein involved in modulating the reorganization of the cytoskeleton." [GOC:curators, PMID:15163540]
 is_a: GO:0008092 ! cytoskeletal protein binding
 
 [Term]
@@ -59711,10 +59905,11 @@
 
 [Term]
 id: GO:0005548
-name: phospholipid transporter activity
+name: obsolete phospholipid transporter activity
 namespace: molecular_function
 alt_id: GO:0008497
-def: "Enables the directed movement of phospholipids into, out of or within a cell, or between cells. Phospholipids are a class of lipids containing phosphoric acid as a mono- or diester." [GOC:ai, ISBN:0198506732]
+def: "OBSOLETE. Enables the directed movement of phospholipids into, out of or within a cell, or between cells. Phospholipids are a class of lipids containing phosphoric acid as a mono- or diester." [GOC:ai, ISBN:0198506732]
+comment: This term was obsoleted because it refers to the substrate rather than to the transport mechanism as a primary classification.
 xref: Reactome:R-HSA-216757 "4xPALM-C-p-2S-ABCA1 tetramer transports PL from transport vesicle membrane to plasma membrane"
 xref: Reactome:R-HSA-382553 "ABCA7:Apo1A-mediated phospholipid efflux"
 xref: Reactome:R-HSA-5678706 "ABCB4 transports PC from plasma membrane to extracellular region"
@@ -59722,8 +59917,11 @@
 xref: Reactome:R-HSA-8857662 "ESYT1:ESYT2:ESYT3 transport GPL from plasma membrane to ER membrane"
 xref: Reactome:R-HSA-8865637 "MFSD2A transports LPC from extracellular region to plasma membrane"
 xref: Reactome:R-HSA-8867876 "OSBPL5,8,10 exchange PS with PI4P from ER membrane to plasma membrane"
-is_a: GO:0005319 ! lipid transporter activity
-relationship: part_of GO:0015914 ! phospholipid transport
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31460" xsd:anyURI
+is_obsolete: true
+consider: GO:0120014
+consider: GO:0170055
 
 [Term]
 id: GO:0005549
@@ -59744,7 +59942,7 @@
 id: GO:0005551
 name: obsolete ubiquitin
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product.
 is_obsolete: true
 consider: GO:0016567
@@ -59754,7 +59952,7 @@
 id: GO:0005552
 name: obsolete polyubiquitin
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product and not a molecular function.
 synonym: "polyubiquitin" EXACT []
 is_obsolete: true
@@ -59961,7 +60159,7 @@
 id: GO:0005576
 name: extracellular region
 namespace: cellular_component
-def: "The space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers the host cell environment outside an intracellular parasite." [GOC:go_curators]
+def: "The space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers the host cell environment outside an intracellular parasite." [GOC:curators]
 comment: Note that this term is intended to annotate gene products that are not attached to the cell surface. For gene products from multicellular organisms which are secreted from a cell but retained within the organism (i.e. released into the interstitial fluid or blood), consider the cellular component term 'extracellular space ; GO:0005615'.
 subset: goslim_agr
 subset: goslim_candida
@@ -60105,7 +60303,7 @@
 namespace: cellular_component
 def: "A collagen heterotrimer containing type VIII alpha chains; [alpha1(VIII)2]alpha2(VIII) and alpha1(VIII)[alpha2(VIII)]2 trimers have been observed; type VIII collagen triple helices associate to form regular hexagonal nets." [PMID:21421911]
 is_a: GO:0098642 ! network-forming collagen trimer
-relationship: part_of GO:0140155 ! hexagonal collagen network of basement membrane
+relationship: part_of GO:0098646 ! hexagonal collagen network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30148" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31070" xsd:anyURI
 
@@ -60173,7 +60371,7 @@
 def: "A collagen homotrimer of alpha1(X) chains; type X collagen triple helices form hexagonal networks (sheets)." [PMID:21421911]
 comment: Collagen X trimers have been observed to form hexagonal lattices in vitro, but in vivo they have been found in a fibril associated form (PMID:19693541).
 is_a: GO:0098642 ! network-forming collagen trimer
-relationship: part_of GO:0098646 ! interstitial hexagonal collagen network
+relationship: part_of GO:0098646 ! hexagonal collagen network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29415" xsd:anyURI
 
 [Term]
@@ -60410,7 +60608,7 @@
 id: GO:0005623
 name: obsolete cell
 namespace: cellular_component
-def: "OBSOLETE. The basic structural and functional unit of all organisms. Includes the plasma membrane and any external encapsulating structures such as the cell wall and cell envelope." [GOC:go_curators]
+def: "OBSOLETE. The basic structural and functional unit of all organisms. Includes the plasma membrane and any external encapsulating structures such as the cell wall and cell envelope." [GOC:curators]
 comment: This term was obsoleted because it was redundant with the root class of the cell ontology, CL:0000000.
 xref: NIF_Subcellular:sao1813327414
 xref: Wikipedia:Cell_(biology)
@@ -60506,7 +60704,7 @@
 id: GO:0005634
 name: nucleus
 namespace: cellular_component
-def: "A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent." [GOC:go_curators]
+def: "A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent." [GOC:curators]
 subset: goslim_agr
 subset: goslim_candida
 subset: goslim_chembl
@@ -60568,7 +60766,7 @@
 id: GO:0005639
 name: obsolete integral component of nuclear inner membrane
 namespace: cellular_component
-def: "OBSOLETE. The component of the nuclear inner membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:go_curators]
+def: "OBSOLETE. The component of the nuclear inner membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:curators]
 comment: This term was obsoleted because it represents protein topology, not a cellular component.
 synonym: "integral to nuclear inner membrane" NARROW []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23424" xsd:anyURI
@@ -61691,7 +61889,7 @@
 id: GO:0005755
 name: obsolete hydrogen-transporting ATP synthase, coupling factor CF(0)
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "hydrogen-transporting ATP synthase, coupling factor CF(0)" EXACT []
 is_obsolete: true
@@ -62208,7 +62406,7 @@
 id: GO:0005799
 name: obsolete coatomer
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because its position and synonym implied that it confused a coated vesicle with the coat itself.
 synonym: "coatomer" EXACT []
 synonym: "COPI vesicle" RELATED []
@@ -62220,7 +62418,7 @@
 id: GO:0005800
 name: obsolete COPII vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology suggested that a new term was required.
 synonym: "COPII vesicle" EXACT []
 is_obsolete: true
@@ -62275,7 +62473,7 @@
 id: GO:0005804
 name: obsolete secretory vesicle membrane
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because its parent was made obsolete.
 synonym: "secretory vesicle membrane" EXACT []
 is_obsolete: true
@@ -62286,7 +62484,7 @@
 id: GO:0005805
 name: obsolete ER-Golgi transport vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology suggested that a new term was required.
 synonym: "ER-Golgi transport vesicle" EXACT []
 is_obsolete: true
@@ -62296,7 +62494,7 @@
 id: GO:0005806
 name: obsolete Golgi-ER transport vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology suggested that a new term was required.
 synonym: "Golgi-ER transport vesicle" EXACT []
 is_obsolete: true
@@ -62306,7 +62504,7 @@
 id: GO:0005808
 name: obsolete Golgi-plasma membrane transport vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; a personal communication from John Armstrong indicated that a replacement term was not required, as TGN-Golgi transport is likely to be mediated by tubules rather than vesicles.
 synonym: "Golgi-plasma membrane transport vesicle" EXACT []
 is_obsolete: true
@@ -62316,7 +62514,7 @@
 id: GO:0005809
 name: obsolete Golgi-vacuole transport vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology suggested that a new term was required.
 synonym: "Golgi-vacuole transport vesicle" EXACT []
 is_obsolete: true
@@ -62326,7 +62524,7 @@
 id: GO:0005810
 name: obsolete endocytotic transport vesicle
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology suggested that a new term was required.
 synonym: "endocytotic transport vesicle" EXACT []
 is_obsolete: true
@@ -63037,7 +63235,7 @@
 id: GO:0005888
 name: obsolete proteoglycan integral to plasma membrane
 namespace: cellular_component
-def: "OBSOLETE. Penetrating at least one phospholipid bilayer of a plasma membrane and consisting of proteoglycan. Also refers to the state of being buried in the bilayer with no exposure outside the bilayer." [GOC:go_curators]
+def: "OBSOLETE. Penetrating at least one phospholipid bilayer of a plasma membrane and consisting of proteoglycan. Also refers to the state of being buried in the bilayer with no exposure outside the bilayer." [GOC:curators]
 comment: This term was made obsolete because it describes the composition (i.e. proteoglycan) and not the location of a gene product.
 synonym: "proteoglycan integral to plasma membrane" EXACT []
 is_obsolete: true
@@ -63202,7 +63400,7 @@
 id: GO:0005906
 name: obsolete clathrin adaptor
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was positioned incorrectly, so the replacement term has a definition different from what this term appeared to mean.
 synonym: "adaptin" RELATED []
 synonym: "clathrin adaptor" EXACT []
@@ -63213,7 +63411,7 @@
 id: GO:0005907
 name: obsolete HA1 clathrin adaptor
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology was erroneous.
 synonym: "AP1" RELATED []
 synonym: "HA1 clathrin adaptor" EXACT []
@@ -63224,7 +63422,7 @@
 id: GO:0005908
 name: obsolete HA2 clathrin adaptor
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete as part of a general reorganization of vesicle and vesicle coat terms; its position in the ontology was erroneous.
 synonym: "AP2" RELATED []
 synonym: "HA2 clathrin adaptor" EXACT []
@@ -63880,7 +64078,6 @@
 subset: goslim_prokaryote
 subset: goslim_yeast
 synonym: "carbohydrate metabolism" EXACT []
-xref: Reactome:R-HSA-71387 "Metabolism of carbohydrates"
 xref: Wikipedia:Carbohydrate_metabolism
 is_a: GO:0044238 ! primary metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22880" xsd:anyURI
@@ -63933,7 +64130,7 @@
 id: GO:0005979
 name: regulation of glycogen biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:curators]
 synonym: "regulation of glycogen anabolism" EXACT []
 synonym: "regulation of glycogen biosynthesis" EXACT []
 synonym: "regulation of glycogen formation" EXACT []
@@ -63965,7 +64162,7 @@
 id: GO:0005981
 name: regulation of glycogen catabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:curators]
 synonym: "regulation of glycogen breakdown" EXACT []
 synonym: "regulation of glycogen catabolism" EXACT []
 synonym: "regulation of glycogen degradation" EXACT []
@@ -64008,7 +64205,7 @@
 id: GO:0005985
 name: sucrose metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:go_curators]
+def: "The chemical reactions and pathways involving sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:curators]
 synonym: "sucrose metabolism" EXACT []
 is_a: GO:0005984 ! disaccharide metabolic process
 
@@ -64016,7 +64213,7 @@
 id: GO:0005986
 name: sucrose biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:curators]
 synonym: "sucrose anabolism" EXACT []
 synonym: "sucrose biosynthesis" EXACT []
 synonym: "sucrose formation" EXACT []
@@ -64029,7 +64226,7 @@
 id: GO:0005987
 name: sucrose catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of sucrose, the disaccharide fructofuranosyl-glucopyranoside." [GOC:curators]
 synonym: "sucrose breakdown" EXACT []
 synonym: "sucrose catabolism" EXACT []
 synonym: "sucrose degradation" EXACT []
@@ -64040,7 +64237,7 @@
 id: GO:0005988
 name: lactose metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving lactose, the disaccharide galactopyranosyl-glucose." [GOC:go_curators]
+def: "The chemical reactions and pathways involving lactose, the disaccharide galactopyranosyl-glucose." [GOC:curators]
 synonym: "lactose metabolism" EXACT []
 is_a: GO:0005984 ! disaccharide metabolic process
 
@@ -64048,7 +64245,7 @@
 id: GO:0005989
 name: lactose biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lactose, the disaccharide galactopyranosyl-glucose." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of lactose, the disaccharide galactopyranosyl-glucose." [GOC:curators]
 synonym: "lactose anabolism" EXACT []
 synonym: "lactose biosynthesis" EXACT []
 synonym: "lactose formation" EXACT []
@@ -64061,12 +64258,15 @@
 id: GO:0005990
 name: lactose catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of lactose, the disaccharide galactopyranosyl-glucose." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of lactose, the disaccharide galactopyranosyl-glucose." [GOC:curators]
 synonym: "lactose breakdown" EXACT []
 synonym: "lactose catabolism" EXACT []
 synonym: "lactose degradation" EXACT []
+xref: MetaCyc:BGALACT-PWY
 is_a: GO:0005988 ! lactose metabolic process
 is_a: GO:0046352 ! disaccharide catabolic process
+property_value: skos:narrowMatch MetaCyc:BGALACT-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0005991
@@ -64094,19 +64294,26 @@
 synonym: "trehalose biosynthesis" EXACT []
 synonym: "trehalose formation" EXACT []
 synonym: "trehalose synthesis" EXACT []
+xref: MetaCyc:PWY-2622
 xref: MetaCyc:PWY-2661
+xref: MetaCyc:PWY-5983
+xref: MetaCyc:PWY-5985
 xref: MetaCyc:PWY-881
 xref: MetaCyc:TREHALOSESYN-PWY
 xref: MetaCyc:TRESYN-PWY
 is_a: GO:0005991 ! trehalose metabolic process
 is_a: GO:0046351 ! disaccharide biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-2622
 property_value: skos:narrowMatch MetaCyc:PWY-2661
+property_value: skos:narrowMatch MetaCyc:PWY-5983
+property_value: skos:narrowMatch MetaCyc:PWY-5985
 property_value: skos:narrowMatch MetaCyc:PWY-881
 property_value: skos:narrowMatch MetaCyc:TREHALOSESYN-PWY
 property_value: skos:narrowMatch MetaCyc:TRESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27083" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27111" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0005993
@@ -64336,9 +64543,11 @@
 synonym: "5-phosphoribosyl-1-pyrophosphate biosynthesis" EXACT []
 synonym: "5-phosphoribosyl-1-pyrophosphate biosynthetic process" EXACT []
 synonym: "PRPP biosynthetic process" EXACT []
-xref: Reactome:R-HSA-73843 "5-Phosphoribose 1-diphosphate biosynthesis"
+xref: MetaCyc:PWY0-662
 is_a: GO:0046390 ! ribose phosphate biosynthetic process
 is_a: GO:0046391 ! 5-phosphoribose 1-diphosphate metabolic process
+property_value: skos:narrowMatch MetaCyc:PWY0-662
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006016
@@ -64481,7 +64690,6 @@
 synonym: "glycosaminoglycan breakdown" EXACT []
 synonym: "glycosaminoglycan catabolism" EXACT []
 synonym: "glycosaminoglycan degradation" EXACT []
-xref: Reactome:R-HSA-2024101 "CS/DS degradation"
 is_a: GO:0006026 ! aminoglycan catabolic process
 is_a: GO:0030203 ! glycosaminoglycan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -64505,6 +64713,7 @@
 def: "The chemical reactions and pathways involving proteoglycans, any glycoprotein in which the carbohydrate units are glycosaminoglycans." [PMID:35536939]
 synonym: "proteoglycan metabolism" EXACT []
 synonym: "proteoglycan sulfate transfer" NARROW []
+xref: Reactome:R-HSA-1793185 "Chondroitin sulfate/dermatan sulfate metabolism"
 is_a: GO:0009100 ! glycoprotein metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
 
@@ -64936,7 +65145,7 @@
 id: GO:0006064
 name: glucuronate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glucuronate, any salt or ester of glucuronic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glucuronate, any salt or ester of glucuronic acid." [GOC:curators]
 synonym: "glucuronate breakdown" EXACT []
 synonym: "glucuronate catabolism" EXACT []
 synonym: "glucuronate degradation" EXACT []
@@ -64967,10 +65176,12 @@
 name: alcohol metabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways involving alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [ISBN:0198506732]
+subset: gocheck_do_not_annotate
 subset: goslim_drosophila
 subset: goslim_pir
 synonym: "alcohol metabolism" EXACT []
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0006067
@@ -65005,7 +65216,7 @@
 id: GO:0006070
 name: octanol metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving octanol, the 8-carbon alcohol with the formula C8H17OH." [GOC:go_curators]
+def: "The chemical reactions and pathways involving octanol, the 8-carbon alcohol with the formula C8H17OH." [GOC:curators]
 synonym: "octanol metabolism" EXACT []
 is_a: GO:0034308 ! primary alcohol metabolic process
 is_a: GO:1903173 ! fatty alcohol metabolic process
@@ -65024,7 +65235,7 @@
 id: GO:0006072
 name: glycerol-3-phosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving glycerol-3-phosphate, a phosphoric monoester of glycerol." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving glycerol-3-phosphate, a phosphoric monoester of glycerol." [GOC:curators, ISBN:0198506732]
 synonym: "glycerol-3-phosphate metabolism" EXACT []
 is_a: GO:0052646 ! alditol phosphate metabolic process
 
@@ -65142,7 +65353,7 @@
 id: GO:0006081
 name: aldehyde metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving aldehydes, any organic compound with the formula R-CH=O, as carried out by individual cells." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving aldehydes, any organic compound with the formula R-CH=O, as carried out by individual cells." [GOC:curators, ISBN:0198506732]
 subset: goslim_pir
 synonym: "aldehyde metabolism" EXACT []
 synonym: "alkanal metabolic process" EXACT []
@@ -65163,7 +65374,7 @@
 id: GO:0006083
 name: acetate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving acetate, the anion of acetic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving acetate, the anion of acetic acid." [GOC:curators]
 synonym: "acetate metabolism" EXACT []
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 
@@ -65181,14 +65392,17 @@
 id: GO:0006085
 name: acetyl-CoA biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of acetyl-CoA, a derivative of coenzyme A in which the sulfhydryl group is acetylated." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of acetyl-CoA, a derivative of coenzyme A in which the sulfhydryl group is acetylated." [GOC:curators]
 synonym: "acetyl-CoA anabolism" EXACT []
 synonym: "acetyl-CoA biosynthesis" EXACT []
 synonym: "acetyl-CoA formation" EXACT []
 synonym: "acetyl-CoA synthesis" EXACT []
 xref: MetaCyc:Acetyl-CoA-Biosynthesis
+xref: MetaCyc:PWY-5172
 is_a: GO:0006084 ! acetyl-CoA metabolic process
 is_a: GO:0071616 ! acyl-CoA biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5172
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006086
@@ -65241,13 +65455,12 @@
 name: pyruvate metabolic process
 namespace: biological_process
 alt_id: GO:0006087
-def: "The chemical reactions and pathways involving pyruvate, 2-oxopropanoate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving pyruvate, 2-oxopropanoate." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_drosophila
 synonym: "pyruvate dehydrogenase bypass" RELATED []
 synonym: "pyruvate metabolism" EXACT []
 xref: MetaCyc:P41-PWY
-xref: Reactome:R-HSA-70268 "Pyruvate metabolism"
 xref: Wikipedia:Pyruvic_acid
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29541" xsd:anyURI
@@ -65488,7 +65701,7 @@
 id: GO:0006109
 name: regulation of carbohydrate metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving carbohydrates." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving carbohydrates." [GOC:curators]
 synonym: "regulation of carbohydrate metabolism" EXACT []
 is_a: GO:0080090 ! regulation of primary metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -65500,7 +65713,7 @@
 name: regulation of glycolytic process
 namespace: biological_process
 alt_id: GO:0090525
-def: "Any process that modulates the frequency, rate or extent of glycolysis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of glycolysis." [GOC:curators]
 synonym: "regulation of glycolysis involved in cellular glucose homeostasis" EXACT []
 xref: Reactome:R-HSA-170822 "Regulation of Glucokinase by Glucokinase Regulatory Protein"
 is_a: GO:0033121 ! regulation of purine nucleotide catabolic process
@@ -65516,7 +65729,7 @@
 name: regulation of gluconeogenesis
 namespace: biological_process
 alt_id: GO:0090526
-def: "Any process that modulates the frequency, rate or extent of gluconeogenesis, the formation of glucose from noncarbohydrate precursors, such as pyruvate, amino acids and glycerol." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of gluconeogenesis, the formation of glucose from noncarbohydrate precursors, such as pyruvate, amino acids and glycerol." [GOC:curators]
 synonym: "regulation of gluconeogenesis involved in cellular glucose homeostasis" RELATED []
 synonym: "regulation of glucose biosynthesis" BROAD []
 synonym: "regulation of glucose biosynthetic process" BROAD []
@@ -65596,7 +65809,7 @@
 id: GO:0006117
 name: acetaldehyde metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving acetaldehyde, a colorless, flammable liquid intermediate in the metabolism of alcohol." [GOC:go_curators]
+def: "The chemical reactions and pathways involving acetaldehyde, a colorless, flammable liquid intermediate in the metabolism of alcohol." [GOC:curators]
 synonym: "acetaldehyde metabolism" EXACT []
 synonym: "ethanal metabolic process" EXACT []
 synonym: "ethanal metabolism" EXACT []
@@ -65696,7 +65909,7 @@
 id: GO:0006126
 name: obsolete other pathways of electron transport
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was an unnecessary grouping term.
 synonym: "other pathways of electron transport" EXACT []
 is_obsolete: true
@@ -65817,6 +66030,7 @@
 namespace: biological_process
 alt_id: GO:0055134
 def: "Any cellular metabolic process involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:ai]
+subset: gocheck_do_not_annotate
 subset: goslim_pir
 subset: goslim_plant
 synonym: "cellular nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
@@ -65825,12 +66039,13 @@
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" RELATED [GOC:dph, GOC:tb]
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid metabolism" EXACT []
 is_a: GO:0044238 ! primary metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0006140
 name: regulation of nucleotide metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:curators]
 synonym: "regulation of nucleotide metabolism" EXACT []
 is_a: GO:0019219 ! regulation of nucleobase-containing compound metabolic process
 is_a: GO:0051174 ! regulation of phosphorus metabolic process
@@ -65843,8 +66058,8 @@
 id: GO:0006141
 name: regulation of purine nucleobase metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving purines." [GOC:go_curators]
-synonym: "regulation of purine base metabolic process" EXACT [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving purines." [GOC:curators]
+synonym: "regulation of purine base metabolic process" EXACT [GOC:curators]
 synonym: "regulation of purine base metabolism" EXACT []
 is_a: GO:0019219 ! regulation of nucleobase-containing compound metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
@@ -65856,9 +66071,9 @@
 id: GO:0006142
 name: obsolete regulation of pyrimidine nucleobase metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
-synonym: "regulation of pyrimidine base metabolic process" EXACT [GOC:go_curators]
+synonym: "regulation of pyrimidine base metabolic process" EXACT [GOC:curators]
 synonym: "regulation of pyrimidine base metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30570" xsd:anyURI
 is_obsolete: true
@@ -65867,7 +66082,7 @@
 id: GO:0006143
 name: obsolete purine metabolic process
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more appropriate terms were created.
 synonym: "purine metabolic process" EXACT []
 is_obsolete: true
@@ -65879,8 +66094,8 @@
 id: GO:0006144
 name: purine nucleobase metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, which include adenine and guanine." [GOC:go_curators]
-synonym: "purine base metabolic process" EXACT [GOC:go_curators]
+def: "The chemical reactions and pathways involving purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, which include adenine and guanine." [GOC:curators]
+synonym: "purine base metabolic process" EXACT [GOC:curators]
 synonym: "purine base metabolism" EXACT []
 synonym: "purine metabolic process" NARROW []
 synonym: "purine metabolism" NARROW []
@@ -65891,9 +66106,9 @@
 id: GO:0006145
 name: purine nucleobase catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, which include adenine and guanine." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, which include adenine and guanine." [GOC:curators]
 synonym: "purine base breakdown" EXACT []
-synonym: "purine base catabolic process" EXACT [GOC:go_curators]
+synonym: "purine base catabolic process" EXACT [GOC:curators]
 synonym: "purine base catabolism" EXACT []
 synonym: "purine base degradation" EXACT []
 is_a: GO:0006144 ! purine nucleobase metabolic process
@@ -65915,7 +66130,7 @@
 id: GO:0006147
 name: guanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:curators]
 synonym: "guanine breakdown" EXACT []
 synonym: "guanine catabolism" EXACT []
 synonym: "guanine degradation" EXACT []
@@ -65926,7 +66141,7 @@
 id: GO:0006148
 name: inosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:curators]
 synonym: "inosine breakdown" EXACT []
 synonym: "inosine catabolism" EXACT []
 synonym: "inosine degradation" EXACT []
@@ -65937,7 +66152,7 @@
 id: GO:0006149
 name: deoxyinosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of deoxyinosine, hypoxanthine deoxyriboside." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of deoxyinosine, hypoxanthine deoxyriboside." [GOC:curators]
 synonym: "deoxyinosine breakdown" EXACT []
 synonym: "deoxyinosine catabolism" EXACT []
 synonym: "deoxyinosine degradation" EXACT []
@@ -65948,7 +66163,7 @@
 id: GO:0006152
 name: purine nucleoside catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside, one of a family of organic molecules consisting of a purine base covalently bonded to a sugar ribose (a ribonucleoside) or deoxyribose (a deoxyribonucleoside)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside, one of a family of organic molecules consisting of a purine base covalently bonded to a sugar ribose (a ribonucleoside) or deoxyribose (a deoxyribonucleoside)." [GOC:curators]
 synonym: "purine nucleoside breakdown" EXACT []
 synonym: "purine nucleoside catabolism" EXACT []
 synonym: "purine nucleoside degradation" EXACT []
@@ -65971,7 +66186,7 @@
 name: adenosine catabolic process
 namespace: biological_process
 alt_id: GO:0006156
-def: "The chemical reactions and pathways resulting in the breakdown of adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:curators]
 synonym: "adenosine breakdown" EXACT []
 synonym: "adenosine catabolism" EXACT []
 synonym: "adenosine degradation" EXACT []
@@ -65997,7 +66212,7 @@
 name: deoxyadenosine catabolic process
 namespace: biological_process
 alt_id: GO:0006159
-def: "The chemical reactions and pathways resulting in the breakdown of deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyadenosine breakdown" EXACT []
 synonym: "deoxyadenosine catabolism" EXACT []
 synonym: "deoxyadenosine degradation" EXACT []
@@ -66039,7 +66254,7 @@
 id: GO:0006163
 name: purine nucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine metabolic process" NARROW []
 synonym: "purine metabolism" NARROW []
 synonym: "purine nucleotide metabolism" EXACT []
@@ -66050,7 +66265,7 @@
 id: GO:0006164
 name: purine nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleotide anabolism" EXACT []
 synonym: "purine nucleotide biosynthesis" EXACT []
 synonym: "purine nucleotide formation" EXACT []
@@ -66082,7 +66297,7 @@
 id: GO:0006167
 name: AMP biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of AMP, adenosine monophosphate." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of AMP, adenosine monophosphate." [GOC:curators, ISBN:0198506732]
 synonym: "AMP anabolism" EXACT []
 synonym: "AMP biosynthesis" EXACT []
 synonym: "AMP formation" EXACT []
@@ -66402,7 +66617,7 @@
 id: GO:0006195
 name: purine nucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a purine nucleotide, a compound consisting of nucleoside (a purine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleotide breakdown" EXACT []
 synonym: "purine nucleotide catabolism" EXACT []
 synonym: "purine nucleotide degradation" EXACT []
@@ -66525,7 +66740,7 @@
 id: GO:0006205
 name: obsolete pyrimidine metabolic process
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more appropriate terms were created.
 synonym: "pyrimidine metabolic process" EXACT []
 is_obsolete: true
@@ -66537,8 +66752,8 @@
 id: GO:0006206
 name: pyrimidine nucleobase metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:go_curators]
-synonym: "pyrimidine base metabolic process" EXACT [GOC:go_curators]
+def: "The chemical reactions and pathways involving pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:curators]
+synonym: "pyrimidine base metabolic process" EXACT [GOC:curators]
 synonym: "pyrimidine base metabolism" EXACT []
 synonym: "pyrimidine metabolic process" RELATED []
 synonym: "pyrimidine metabolism" RELATED []
@@ -66552,7 +66767,7 @@
 def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases, beginning with the synthesis of a pyrimidine ring from simpler precursors." [GOC:mah, ISBN:0716720094]
 synonym: "'de novo' pyrimidine base anabolism" EXACT []
 synonym: "'de novo' pyrimidine base biosynthesis" EXACT []
-synonym: "'de novo' pyrimidine base biosynthetic process" EXACT [GOC:go_curators]
+synonym: "'de novo' pyrimidine base biosynthetic process" EXACT [GOC:curators]
 synonym: "'de novo' pyrimidine base formation" EXACT []
 synonym: "'de novo' pyrimidine base synthesis" EXACT []
 is_a: GO:0019856 ! pyrimidine nucleobase biosynthetic process
@@ -66561,9 +66776,9 @@
 id: GO:0006208
 name: pyrimidine nucleobase catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:curators]
 synonym: "pyrimidine base breakdown" EXACT []
-synonym: "pyrimidine base catabolic process" EXACT [GOC:go_curators]
+synonym: "pyrimidine base catabolic process" EXACT [GOC:curators]
 synonym: "pyrimidine base catabolism" EXACT []
 synonym: "pyrimidine base degradation" EXACT []
 is_a: GO:0006206 ! pyrimidine nucleobase metabolic process
@@ -66574,7 +66789,7 @@
 id: GO:0006209
 name: cytosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of cytosine, 4-amino-2-hydroxypyrimidine, a pyrimidine derivative that is one of the five main bases found in nucleic acids; it occurs widely in cytidine derivatives." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of cytosine, 4-amino-2-hydroxypyrimidine, a pyrimidine derivative that is one of the five main bases found in nucleic acids; it occurs widely in cytidine derivatives." [GOC:curators]
 synonym: "cytosine breakdown" EXACT []
 synonym: "cytosine catabolism" EXACT []
 synonym: "cytosine degradation" EXACT []
@@ -66585,7 +66800,7 @@
 id: GO:0006210
 name: thymine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:curators]
 synonym: "thymine breakdown" EXACT []
 synonym: "thymine catabolism" EXACT []
 synonym: "thymine degradation" EXACT []
@@ -66596,7 +66811,7 @@
 id: GO:0006211
 name: obsolete 5-methylcytosine catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 5-methylcytosine, a methylated base of DNA." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 5-methylcytosine, a methylated base of DNA." [GOC:curators]
 comment: This term was obsoleted because it it redundant with chromosomal 5-methylcytosine DNA demethylation pathway ; GO:0141166 or its children.
 synonym: "5-methylcytosine breakdown" EXACT []
 synonym: "5-methylcytosine catabolism" EXACT []
@@ -66609,7 +66824,7 @@
 id: GO:0006212
 name: uracil catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:curators]
 synonym: "uracil breakdown" EXACT []
 synonym: "uracil catabolism" EXACT []
 synonym: "uracil degradation" EXACT []
@@ -66632,7 +66847,7 @@
 name: thymidine catabolic process
 namespace: biological_process
 alt_id: GO:0006215
-def: "The chemical reactions and pathways resulting in the breakdown of thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:curators]
 synonym: "deoxyribosylthymine catabolic process" EXACT []
 synonym: "deoxyribosylthymine catabolism" EXACT []
 synonym: "thymidine breakdown" EXACT []
@@ -66656,7 +66871,7 @@
 id: GO:0006217
 name: deoxycytidine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxycytidine breakdown" EXACT []
 synonym: "deoxycytidine catabolism" EXACT []
 synonym: "deoxycytidine degradation" EXACT []
@@ -66667,7 +66882,7 @@
 id: GO:0006218
 name: uridine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:curators]
 synonym: "uridine breakdown" EXACT []
 synonym: "uridine catabolism" EXACT []
 synonym: "uridine degradation" EXACT []
@@ -66678,7 +66893,7 @@
 id: GO:0006219
 name: deoxyuridine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyuridine breakdown" EXACT []
 synonym: "deoxyuridine catabolism" EXACT []
 synonym: "deoxyuridine degradation" EXACT []
@@ -66689,7 +66904,7 @@
 id: GO:0006220
 name: pyrimidine nucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine metabolic process" BROAD []
 synonym: "pyrimidine metabolism" BROAD []
 synonym: "pyrimidine nucleotide metabolism" EXACT []
@@ -66700,7 +66915,7 @@
 id: GO:0006221
 name: pyrimidine nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleotide anabolism" EXACT []
 synonym: "pyrimidine nucleotide biosynthesis" EXACT []
 synonym: "pyrimidine nucleotide formation" EXACT []
@@ -66961,7 +67176,7 @@
 id: GO:0006244
 name: pyrimidine nucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine nucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose or ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleotide breakdown" EXACT []
 synonym: "pyrimidine nucleotide catabolism" EXACT []
 synonym: "pyrimidine nucleotide degradation" EXACT []
@@ -67194,6 +67409,7 @@
 synonym: "mtDNA replication" EXACT []
 synonym: "mtDNA synthesis" RELATED []
 synonym: "replication of mitochondrial DNA" EXACT []
+xref: Reactome:R-HSA-9913635 "Strand-asynchronous mitochondrial DNA replication"
 is_a: GO:0006261 ! DNA-templated DNA replication
 is_a: GO:0032042 ! mitochondrial DNA metabolic process
 intersection_of: GO:0006260 ! DNA replication
@@ -67317,7 +67533,7 @@
 id: GO:0006275
 name: regulation of DNA replication
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of DNA replication." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of DNA replication." [GOC:curators]
 is_a: GO:0051052 ! regulation of DNA metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006260 ! DNA replication
@@ -67399,7 +67615,7 @@
 id: GO:0006282
 name: regulation of DNA repair
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of DNA repair." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of DNA repair." [GOC:curators]
 subset: goslim_metagenomics
 is_a: GO:0051052 ! regulation of DNA metabolic process
 is_a: GO:0080135 ! regulation of cellular response to stress
@@ -67657,7 +67873,7 @@
 id: GO:0006308
 name: DNA catabolic process
 namespace: biological_process
-def: "The cellular DNA metabolic process resulting in the breakdown of DNA, deoxyribonucleic acid, one of the two main types of nucleic acid, consisting of a long unbranched macromolecule formed from one or two strands of linked deoxyribonucleotides, the 3'-phosphate group of each constituent deoxyribonucleotide being joined in 3',5'-phosphodiester linkage to the 5'-hydroxyl group of the deoxyribose moiety of the next one." [GOC:go_curators, ISBN:0198506732]
+def: "The cellular DNA metabolic process resulting in the breakdown of DNA, deoxyribonucleic acid, one of the two main types of nucleic acid, consisting of a long unbranched macromolecule formed from one or two strands of linked deoxyribonucleotides, the 3'-phosphate group of each constituent deoxyribonucleotide being joined in 3',5'-phosphodiester linkage to the 5'-hydroxyl group of the deoxyribose moiety of the next one." [GOC:curators, ISBN:0198506732]
 synonym: "DNA breakdown" EXACT []
 synonym: "DNA catabolism" EXACT []
 synonym: "DNA degradation" EXACT []
@@ -67796,7 +68012,7 @@
 id: GO:0006326
 name: obsolete bent DNA binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "bent DNA binding" EXACT []
 is_obsolete: true
@@ -67806,7 +68022,7 @@
 id: GO:0006327
 name: obsolete random coil binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "random coil binding" EXACT []
 is_obsolete: true
@@ -67815,7 +68031,7 @@
 id: GO:0006328
 name: obsolete AT binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "AT binding" EXACT []
 is_obsolete: true
@@ -67825,7 +68041,7 @@
 id: GO:0006329
 name: obsolete satellite DNA binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "satellite DNA binding" EXACT []
 is_obsolete: true
@@ -67837,7 +68053,7 @@
 namespace: biological_process
 alt_id: GO:0006331
 alt_id: GO:0006332
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "single-stranded DNA binding" EXACT []
 is_obsolete: true
@@ -67895,7 +68111,7 @@
 id: GO:0006339
 name: obsolete positive regulation of transcription of homeotic gene (trithorax group)
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "positive regulation of transcription of homeotic gene (trithorax group)" EXACT []
 is_obsolete: true
@@ -67907,7 +68123,7 @@
 id: GO:0006340
 name: obsolete negative regulation of transcription of homeotic gene (Polycomb group)
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "negative regulation of transcription of homeotic gene (Polycomb group)" EXACT []
 is_obsolete: true
@@ -68052,7 +68268,7 @@
 alt_id: GO:0032583
 alt_id: GO:0045449
 alt_id: GO:0061019
-def: "Any process that modulates the frequency, rate or extent of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any process that modulates the frequency, rate or extent of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 subset: goslim_drosophila
 subset: goslim_generic
 subset: goslim_prokaryote
@@ -68063,6 +68279,7 @@
 synonym: "regulation of transcription, DNA-templated" EXACT []
 synonym: "transcriptional control" BROAD []
 xref: Reactome:R-HSA-9013695 "NOTCH4 Intracellular Domain Regulates Transcription"
+xref: Reactome:R-HSA-9708530 "Regulation of BACH1 activity"
 xref: Reactome:R-HSA-9762293 "Regulation of CDH11 gene transcription"
 is_a: GO:0010468 ! regulation of gene expression
 is_a: GO:2001141 ! regulation of RNA biosynthetic process
@@ -68075,7 +68292,7 @@
 id: GO:0006356
 name: regulation of transcription by RNA polymerase I
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:curators]
 synonym: "regulation of transcription from Pol I promoter" EXACT []
 synonym: "regulation of transcription from RNA polymerase I promoter" EXACT []
 is_a: GO:0006355 ! regulation of DNA-templated transcription
@@ -68089,7 +68306,7 @@
 namespace: biological_process
 alt_id: GO:0006358
 alt_id: GO:0010551
-def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA polymerase II." [GOC:go_curators, GOC:txnOH]
+def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA polymerase II." [GOC:curators, GOC:txnOH]
 synonym: "global transcription regulation from Pol II promoter" RELATED []
 synonym: "regulation of gene-specific transcription from RNA polymerase II promoter" RELATED []
 synonym: "regulation of global transcription from Pol II promoter" RELATED []
@@ -68106,7 +68323,7 @@
 id: GO:0006359
 name: regulation of transcription by RNA polymerase III
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA ploymerase III." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of transcription mediated by RNA ploymerase III." [GOC:curators]
 synonym: "regulation of transcription from Pol III promoter" EXACT []
 synonym: "regulation of transcription from RNA polymerase III promoter" EXACT []
 is_a: GO:0006355 ! regulation of DNA-templated transcription
@@ -68363,7 +68580,7 @@
 id: GO:0006380
 name: obsolete poly-A binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "poly-A binding" EXACT []
 is_obsolete: true
@@ -68602,6 +68819,7 @@
 synonym: "RNA breakdown" EXACT []
 synonym: "RNA catabolism" EXACT []
 synonym: "RNA degradation" EXACT []
+xref: Reactome:R-HSA-9930044 "Nuclear RNA decay"
 is_a: GO:0016070 ! RNA metabolic process
 is_a: GO:0141188 ! nucleic acid catabolic process
 
@@ -68719,7 +68937,7 @@
 alt_id: GO:0006416
 alt_id: GO:0006453
 alt_id: GO:0043037
-def: "The cellular metabolic process in which a protein is formed, using the sequence of a mature mRNA or circRNA molecule to specify the sequence of amino acids in a polypeptide chain. Translation is mediated by the ribosome, and begins with the formation of a ternary complex between aminoacylated initiator methionine tRNA, GTP, and initiation factor 2, which subsequently associates with the small subunit of the ribosome and an mRNA or circRNA. Translation ends with the release of a polypeptide chain from the ribosome." [GOC:go_curators]
+def: "The cellular metabolic process in which a protein is formed, using the sequence of a mature mRNA or circRNA molecule to specify the sequence of amino acids in a polypeptide chain. Translation is mediated by the ribosome, and begins with the formation of a ternary complex between aminoacylated initiator methionine tRNA, GTP, and initiation factor 2, which subsequently associates with the small subunit of the ribosome and an mRNA or circRNA. Translation ends with the release of a polypeptide chain from the ribosome." [GOC:curators]
 subset: goslim_candida
 subset: goslim_chembl
 subset: goslim_drosophila
@@ -68979,7 +69197,7 @@
 id: GO:0006446
 name: regulation of translational initiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of translational initiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of translational initiation." [GOC:curators]
 is_a: GO:0006417 ! regulation of translation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006413 ! translational initiation
@@ -68996,7 +69214,7 @@
 id: GO:0006448
 name: regulation of translational elongation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate, extent or accuracy of translational elongation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate, extent or accuracy of translational elongation." [GOC:curators]
 is_a: GO:0006417 ! regulation of translation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006414 ! translational elongation
@@ -69006,7 +69224,7 @@
 id: GO:0006449
 name: regulation of translational termination
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of translational termination." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of translational termination." [GOC:curators]
 is_a: GO:0006417 ! regulation of translation
 is_a: GO:0043244 ! regulation of protein-containing complex disassembly
 intersection_of: GO:0065007 ! biological regulation
@@ -69046,7 +69264,7 @@
 alt_id: GO:0007022
 alt_id: GO:0007024
 alt_id: GO:0007025
-def: "The process of assisting in the covalent and noncovalent assembly of single chain polypeptides or multisubunit complexes into the correct tertiary structure." [GOC:go_curators, GOC:rb]
+def: "The process of assisting in the covalent and noncovalent assembly of single chain polypeptides or multisubunit complexes into the correct tertiary structure." [GOC:curators, GOC:rb]
 subset: goslim_candida
 subset: goslim_chembl
 subset: goslim_drosophila
@@ -69069,6 +69287,7 @@
 synonym: "non-chaperonin molecular chaperone ATPase activity" RELATED []
 synonym: "protein complex assembly, multichaperone pathway" RELATED []
 xref: Reactome:R-HSA-389960 "Formation of tubulin folding intermediates by CCT/TriC"
+xref: Reactome:R-HSA-390450 "Folding of actin by CCT/TriC"
 xref: Reactome:R-HSA-390466 "Chaperonin-mediated protein folding"
 xref: Reactome:R-HSA-532668 "N-glycan trimming in the ER and Calnexin/Calreticulin cycle"
 xref: Wikipedia:Protein_folding
@@ -69099,7 +69318,7 @@
 id: GO:0006460
 name: obsolete peptidyl-prolyl isomerase B reaction
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "peptidyl-prolyl isomerase B reaction" EXACT []
 is_obsolete: true
@@ -69130,7 +69349,6 @@
 def: "OBSOLETE. The proteolytic removal of a signal peptide from a protein during or after transport to a specific location in the cell." [GOC:mah, PMID:31214988]
 comment: This term was obsoleted because it reprenset a molecular function, signal peptidase activity ; GO:0009003.
 synonym: "leader peptide processing" EXACT []
-xref: Reactome:R-HSA-159782 "Removal of aminoterminal propeptides from gamma-carboxylated proteins"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30479" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051604
@@ -69152,7 +69370,6 @@
 namespace: biological_process
 def: "The process of introducing a phosphate group on to a protein." [GOC:hb]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid phosphorylation" EXACT [GOC:bf]
 is_a: GO:0016310 ! phosphorylation
 is_a: GO:0036211 ! protein modification process
@@ -69161,7 +69378,7 @@
 id: GO:0006469
 name: negative regulation of protein kinase activity
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of protein kinase activity." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of protein kinase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "down regulation of protein kinase activity" EXACT []
 synonym: "down-regulation of protein kinase activity" EXACT []
@@ -69180,7 +69397,6 @@
 namespace: biological_process
 def: "The process of removing one or more phosphoric residues from a protein." [GOC:hb]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid dephosphorylation" EXACT [GOC:bf]
 is_a: GO:0016311 ! dephosphorylation
 is_a: GO:0036211 ! protein modification process
@@ -69343,6 +69559,8 @@
 synonym: "N-glycan biosynthesis" RELATED []
 synonym: "N-glycan metabolism" RELATED []
 synonym: "protein amino acid N-linked glycosylation" EXACT [GOC:bf]
+xref: Reactome:R-HSA-446203 "Asparagine N-linked glycosylation"
+xref: Reactome:R-HSA-480985 "Synthesis of dolichyl-phosphate-glucose"
 is_a: GO:0009101 ! glycoprotein biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29770" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30595" xsd:anyURI
@@ -69359,6 +69577,7 @@
 synonym: "N-linked glycan precursor biosynthesis" EXACT [PMID:8666161]
 synonym: "N-linked glycan precursor biosynthetic process" EXACT [PMID:8666161]
 synonym: "oligosaccharide-PP-dolichol assembly" EXACT []
+xref: Reactome:R-HSA-162699 "Synthesis of dolichyl-phosphate mannose"
 xref: Reactome:R-HSA-446193 "Biosynthesis of the N-glycan precursor (dolichol lipid-linked oligosaccharide, LLO) and transfer to a nascent protein"
 is_a: GO:1901137 ! carbohydrate derivative biosynthetic process
 relationship: part_of GO:0006487 ! protein N-linked glycosylation
@@ -69374,7 +69593,6 @@
 synonym: "dolichyl diphosphate biosynthesis" EXACT []
 synonym: "dolichyl diphosphate formation" EXACT []
 synonym: "dolichyl diphosphate synthesis" EXACT []
-xref: Reactome:R-HSA-446199 "Synthesis of Dolichyl-phosphate"
 is_a: GO:0008654 ! phospholipid biosynthetic process
 relationship: part_of GO:0006488 ! dolichol-linked oligosaccharide biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30376" xsd:anyURI
@@ -69431,7 +69649,7 @@
 id: GO:0006495
 name: obsolete terminal O-glycosylation
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators, GOC:jsg]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators, GOC:jsg]
 comment: This term was made obsolete because there is no evidence for the existence of this process.
 synonym: "terminal O-glycosylation" EXACT []
 is_obsolete: true
@@ -69455,7 +69673,6 @@
 comment: For non-covalent interactions with a lipid, consider instead the term 'lipid binding ; GO:0008289' and its children.
 subset: gocheck_obsoletion_candidate
 subset: goslim_drosophila
-subset: goslim_yeast
 synonym: "lipid:protein modification" EXACT []
 synonym: "protein amino acid lipidation" EXACT [GOC:bf]
 xref: RESID:AA0059
@@ -69571,7 +69788,7 @@
 name: GPI anchor biosynthetic process
 namespace: biological_process
 alt_id: GO:0015998
-def: "The chemical reactions and pathways resulting in the formation of a glycosylphosphatidylinositol (GPI) anchor that attaches some membrane proteins to the lipid bilayer of the cell membrane. The phosphatidylinositol group is linked via the C-6 hydroxyl residue of inositol to a carbohydrate chain which is itself linked to the protein via an ethanolamine phosphate group, its amino group forming an amide linkage with the C-terminal carboxyl of the protein. Some GPI anchors have variants on this canonical linkage." [GOC:go_curators, ISBN:0198547684]
+def: "The chemical reactions and pathways resulting in the formation of a glycosylphosphatidylinositol (GPI) anchor that attaches some membrane proteins to the lipid bilayer of the cell membrane. The phosphatidylinositol group is linked via the C-6 hydroxyl residue of inositol to a carbohydrate chain which is itself linked to the protein via an ethanolamine phosphate group, its amino group forming an amide linkage with the C-terminal carboxyl of the protein. Some GPI anchors have variants on this canonical linkage." [GOC:curators, ISBN:0198547684]
 synonym: "glycosylphosphatidylinositol biosynthesis" EXACT []
 synonym: "glycosylphosphatidylinositol biosynthetic process" EXACT []
 synonym: "GPI anchor anabolism" EXACT []
@@ -69641,7 +69858,7 @@
 namespace: biological_process
 alt_id: GO:0042787
 alt_id: GO:0043432
-def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of a ubiquitin group, or multiple ubiquitin groups, to the protein." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of a ubiquitin group, or multiple ubiquitin groups, to the protein." [GOC:curators]
 synonym: "myofibrillar protein ubiquitination during ubiquitin-dependent protein breakdown" NARROW []
 synonym: "myofibrillar protein ubiquitination during ubiquitin-dependent protein catabolic process" NARROW []
 synonym: "myofibrillar protein ubiquitination during ubiquitin-dependent protein catabolism" NARROW []
@@ -69699,7 +69916,7 @@
 id: GO:0006516
 name: glycoprotein catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a glycoprotein, a protein that contains covalently bound glycose (i.e. monosaccharide) residues; the glycose occurs most commonly as oligosaccharide or fairly small polysaccharide but occasionally as monosaccharide." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a glycoprotein, a protein that contains covalently bound glycose (i.e. monosaccharide) residues; the glycose occurs most commonly as oligosaccharide or fairly small polysaccharide but occasionally as monosaccharide." [GOC:curators, ISBN:0198506732]
 synonym: "glycoprotein breakdown" EXACT []
 synonym: "glycoprotein catabolism" EXACT []
 synonym: "glycoprotein degradation" EXACT []
@@ -69721,7 +69938,7 @@
 id: GO:0006518
 name: peptide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving peptides, compounds of two or more amino acids where the alpha carboxyl group of one is bound to the alpha amino group of another." [GOC:go_curators]
+def: "The chemical reactions and pathways involving peptides, compounds of two or more amino acids where the alpha carboxyl group of one is bound to the alpha amino group of another." [GOC:curators]
 subset: goslim_pir
 synonym: "peptide metabolism" EXACT []
 xref: Reactome:R-HSA-2980736 "Peptide hormone metabolism"
@@ -69752,9 +69969,8 @@
 id: GO:0006521
 name: regulation of amino acid metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids." [GOC:curators]
 synonym: "regulation of amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-350562 "Regulation of ornithine decarboxylase (ODC)"
 is_a: GO:0080090 ! regulation of primary metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006520 ! amino acid metabolic process
@@ -69765,7 +69981,7 @@
 id: GO:0006522
 name: alanine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving alanine, 2-aminopropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving alanine, 2-aminopropanoic acid." [GOC:curators]
 synonym: "alanine metabolism" EXACT []
 is_a: GO:1901605 ! alpha-amino acid metabolic process
 
@@ -69773,7 +69989,7 @@
 id: GO:0006523
 name: alanine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of alanine, 2-aminopropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of alanine, 2-aminopropanoic acid." [GOC:curators]
 synonym: "alanine anabolism" EXACT []
 synonym: "alanine biosynthesis" EXACT []
 synonym: "alanine formation" EXACT []
@@ -69785,7 +70001,7 @@
 id: GO:0006524
 name: obsolete alanine catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of alanine, 2-aminopropanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of alanine, 2-aminopropanoic acid." [GOC:curators]
 comment: The reason for obsoletion is that a more specific term has been created.
 synonym: "alanine breakdown" EXACT []
 synonym: "alanine catabolism" EXACT []
@@ -69798,7 +70014,7 @@
 id: GO:0006525
 name: arginine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:curators]
 synonym: "arginine metabolism" EXACT []
 xref: Wikipedia:Arginine
 is_a: GO:1901605 ! alpha-amino acid metabolic process
@@ -69814,20 +70030,23 @@
 synonym: "arginine synthesis" EXACT []
 xref: MetaCyc:ARGSYN-PWY
 xref: MetaCyc:ARGSYNBSUB-PWY
+xref: MetaCyc:PWY-5154
 xref: MetaCyc:PWY-7400
 is_a: GO:0006525 ! arginine metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:ARGSYN-PWY
 property_value: skos:narrowMatch MetaCyc:ARGSYNBSUB-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5154
 property_value: skos:narrowMatch MetaCyc:PWY-7400
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28686" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006527
 name: L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-arginine, 2-amino-5-(carbamimidamido)pentanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-arginine." [GOC:curators]
 synonym: "arginine breakdown" EXACT []
 synonym: "arginine catabolism" EXACT []
 synonym: "arginine degradation" EXACT []
@@ -69863,21 +70082,21 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8187
 property_value: skos:narrowMatch MetaCyc:PWY0-823
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006528
 name: asparagine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:curators]
 synonym: "asparagine metabolism" EXACT []
-xref: MetaCyc:ASPARAGINESYN-PWY
 is_a: GO:0019752 ! carboxylic acid metabolic process
 
 [Term]
 id: GO:0006529
 name: obsolete asparagine biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:curators]
 comment: The reason for obsoletion is that more specific terms were created.
 synonym: "asparagine anabolism" EXACT []
 synonym: "asparagine biosynthesis" EXACT []
@@ -69891,7 +70110,7 @@
 id: GO:0006530
 name: L-asparagine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-asparagine, 2-amino-3-carbamoylpropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-asparagine." [GOC:curators]
 synonym: "asparagine breakdown" EXACT []
 synonym: "asparagine catabolism" EXACT []
 synonym: "asparagine degradation" EXACT []
@@ -69903,12 +70122,13 @@
 property_value: skos:narrowMatch MetaCyc:ASPARAGINE-DEG1-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-4002
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006531
 name: aspartate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:curators, ISBN:0198506732]
 synonym: "aspartate metabolism" EXACT []
 is_a: GO:0043648 ! dicarboxylic acid metabolic process
 is_a: GO:1901605 ! alpha-amino acid metabolic process
@@ -69917,7 +70137,7 @@
 id: GO:0006532
 name: L-aspartate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:go_curators, ISBN:0198506732, MetaCyc:ASPARTATESYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:curators, ISBN:0198506732, MetaCyc:ASPARTATESYN-PWY]
 synonym: "aspartate anabolism" EXACT []
 synonym: "aspartate biosynthesis" EXACT []
 synonym: "aspartate formation" EXACT []
@@ -69932,7 +70152,7 @@
 id: GO:0006533
 name: L-aspartate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-aspartate, the anion derived from aspartic acid, 2-aminobutanedioic acid." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-aspartate." [GOC:curators]
 synonym: "aspartate breakdown" EXACT []
 synonym: "aspartate catabolism" EXACT []
 synonym: "aspartate degradation" EXACT []
@@ -69947,36 +70167,40 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8291
 property_value: skos:narrowMatch MetaCyc:PWY-8294
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006534
-name: cysteine metabolic process
+name: obsolete cysteine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:curators]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "cysteine metabolism" EXACT []
 xref: Wikipedia:Cysteine_metabolism
-is_a: GO:0000096 ! sulfur amino acid metabolic process
-is_a: GO:1901605 ! alpha-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+consider: GO:0019344
+consider: GO:0019448
 
 [Term]
 id: GO:0006535
-name: L-cysteine biosynthetic process from L-serine
+name: obsolete L-cysteine biosynthetic process from L-serine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of cysteine from L- serine." [MetaCyc:CYSTSYN-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cysteine from L- serine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism from serine" EXACT []
 synonym: "cysteine formation from serine" EXACT []
 synonym: "cysteine synthesis from serine" EXACT []
-xref: MetaCyc:CYSTSYN-PWY
-is_a: GO:0006563 ! L-serine metabolic process
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:CYSTSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0006536
 name: glutamate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving glutamate, the anion of 2-aminopentanedioic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving glutamate, the anion of 2-aminopentanedioic acid." [GOC:curators]
 synonym: "glutamate metabolism" EXACT []
 synonym: "glutamic acid metabolic process" EXACT []
 synonym: "glutamic acid metabolism" EXACT []
@@ -69988,7 +70212,7 @@
 id: GO:0006537
 name: obsolete glutamate biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, the anion of 2-aminopentanedioic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, the anion of 2-aminopentanedioic acid." [GOC:curators]
 comment: This is a duplicate term.
 synonym: "glutamate anabolism" EXACT []
 synonym: "glutamate biosynthesis" EXACT []
@@ -70009,7 +70233,7 @@
 name: L-glutamate catabolic process
 namespace: biological_process
 alt_id: GO:0019459
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate, the anion of 2-aminopentanedioic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate." [GOC:curators]
 synonym: "glutamate breakdown" EXACT []
 synonym: "glutamate catabolism" EXACT []
 synonym: "glutamate deamidation" RELATED []
@@ -70029,13 +70253,14 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5766
 property_value: skos:narrowMatch MetaCyc:PWY0-1305
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006539
 name: obsolete glutamate catabolic process via 2-oxoglutarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate, via the intermediate 2-oxoglutarate." [GOC:go_curators]
-comment: The reason for obsoletion is that this term is equivalent to GO:0019552 glutamate catabolic process via 2-hydroxyglutarate, XREF the same MetaCyc pathway.
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate, via the intermediate 2-oxoglutarate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown via 2-oxoglutarate" EXACT []
 synonym: "glutamate catabolic process via 2-ketoglutarate" EXACT []
 synonym: "glutamate catabolic process via alpha-ketoglutarate" EXACT []
@@ -70046,7 +70271,6 @@
 synonym: "glutamate degradation via 2-oxoglutarate" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28390" xsd:anyURI
 is_obsolete: true
-replaced_by: GO:0019552
 
 [Term]
 id: GO:0006540
@@ -70091,9 +70315,7 @@
 synonym: "glutamine biosynthesis" EXACT []
 synonym: "glutamine formation" EXACT []
 synonym: "glutamine synthesis" EXACT []
-xref: MetaCyc:GLNSYN-PWY
 xref: MetaCyc:PWY-6549
-property_value: skos:narrowMatch MetaCyc:GLNSYN-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-6549
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31068" xsd:anyURI
 is_obsolete: true
@@ -70103,7 +70325,7 @@
 id: GO:0006543
 name: L-glutamine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamine, 2-amino-4-carbamoylbutanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-glutamine." [GOC:curators]
 synonym: "glutamine breakdown" EXACT []
 synonym: "glutamine catabolism" EXACT []
 synonym: "glutamine degradation" EXACT []
@@ -70116,12 +70338,13 @@
 property_value: skos:narrowMatch MetaCyc:GLUTAMINEFUM-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28154" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006544
 name: glycine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving glycine, aminoethanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving glycine, aminoethanoic acid." [GOC:curators]
 synonym: "glycine metabolism" EXACT []
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
 is_a: GO:1901605 ! alpha-amino acid metabolic process
@@ -70130,22 +70353,31 @@
 id: GO:0006545
 name: glycine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of glycine, aminoethanoic acid." [GOC:go_curators, MetaCyc:GLYCINE-SYN2-PWY]
+def: "The chemical reactions and pathways resulting in the formation of glycine, aminoethanoic acid." [GOC:curators, MetaCyc:GLYCINE-SYN2-PWY]
 synonym: "glycine anabolism" EXACT []
 synonym: "glycine biosynthesis" EXACT []
 synonym: "glycine formation" EXACT []
 synonym: "glycine synthesis" EXACT []
 xref: MetaCyc:GLYCINE-SYN2-PWY
+xref: MetaCyc:GLYSYN-ALA-PWY
+xref: MetaCyc:GLYSYN-PWY
+xref: MetaCyc:GLYSYN-THR-PWY
+xref: MetaCyc:PWY-8551
 is_a: GO:0006544 ! glycine metabolic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:GLYCINE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-ALA-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLYSYN-THR-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-8551
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006546
 name: glycine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glycine, aminoethanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glycine." [GOC:curators]
 synonym: "glycine breakdown" EXACT []
 synonym: "glycine catabolism" EXACT []
 synonym: "glycine degradation" EXACT []
@@ -70156,12 +70388,13 @@
 is_a: GO:1901606 ! alpha-amino acid catabolic process
 property_value: skos:narrowMatch MetaCyc:PWY0-1608
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006547
 name: L-histidine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:curators]
 synonym: "histidine metabolic process" BROAD []
 synonym: "histidine metabolism" EXACT []
 xref: Wikipedia:Histidine
@@ -70174,7 +70407,7 @@
 id: GO:0006548
 name: L-histidine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-histidine." [GOC:curators]
 synonym: "histidine breakdown" EXACT []
 synonym: "histidine catabolic process" BROAD []
 synonym: "histidine catabolism" EXACT []
@@ -70198,6 +70431,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5030
 property_value: skos:narrowMatch MetaCyc:PWY-5031
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006549
@@ -70212,7 +70446,7 @@
 id: GO:0006550
 name: L-isoleucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-isoleucine." [GOC:curators]
 synonym: "isoleucine breakdown" EXACT []
 synonym: "isoleucine catabolism" EXACT []
 synonym: "isoleucine degradation" EXACT []
@@ -70226,6 +70460,7 @@
 property_value: skos:narrowMatch MetaCyc:ILEUDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-5078
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006551
@@ -70242,7 +70477,7 @@
 id: GO:0006552
 name: L-leucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-leucine, 2-amino-4-methylpentanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-leucine." [GOC:curators]
 synonym: "L-leucine breakdown" EXACT []
 synonym: "L-leucine catabolism" EXACT []
 synonym: "L-leucine degradation" EXACT []
@@ -70262,12 +70497,13 @@
 property_value: skos:narrowMatch MetaCyc:PWY-7767
 property_value: skos:narrowMatch MetaCyc:PWY-8185
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006553
 name: lysine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving lysine, 2,6-diaminohexanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving lysine, 2,6-diaminohexanoic acid." [GOC:curators]
 synonym: "lysine metabolism" EXACT []
 xref: Wikipedia:Lysine
 is_a: GO:1901605 ! alpha-amino acid metabolic process
@@ -70276,7 +70512,7 @@
 id: GO:0006554
 name: obsolete lysine catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of lysine, 2,6-diaminohexanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of lysine, 2,6-diaminohexanoic acid." [GOC:curators]
 comment: The reason for obsoletion is that more specific terms were created.
 synonym: "lysine breakdown" EXACT []
 synonym: "lysine catabolism" EXACT []
@@ -70302,7 +70538,7 @@
 id: GO:0006556
 name: S-adenosylmethionine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of S-adenosylmethionine, S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of S-adenosylmethionine, S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism." [GOC:curators, ISBN:0198506732]
 synonym: "S-adenosyl methionine biosynthesis" EXACT []
 synonym: "S-adenosyl methionine biosynthetic process" EXACT []
 synonym: "S-adenosylmethionine anabolism" EXACT []
@@ -70346,7 +70582,7 @@
 id: GO:0006559
 name: L-phenylalanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-phenylalanine, 2-amino-3-phenylpropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-phenylalanine." [GOC:curators]
 synonym: "L-phenylalanine breakdown" EXACT []
 synonym: "L-phenylalanine catabolism" EXACT []
 synonym: "L-phenylalanine degradation" EXACT []
@@ -70367,6 +70603,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-7158
 property_value: skos:narrowMatch MetaCyc:PWY-8014
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006560
@@ -70396,7 +70633,7 @@
 id: GO:0006562
 name: L-proline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-proline (pyrrolidine-2-carboxylic acid), a chiral, cyclic, nonessential alpha-amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-proline." [GOC:curators]
 synonym: "proline breakdown" EXACT []
 synonym: "proline catabolism" EXACT []
 synonym: "proline degradation" EXACT []
@@ -70409,6 +70646,7 @@
 property_value: skos:narrowMatch MetaCyc:PROUT-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-8186
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006563
@@ -70416,6 +70654,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-serine, the L-enantiomer of serine, i.e. (2S)-2-amino-3-hydroxypropanoic acid." [GOC:ai, GOC:jsg]
 synonym: "L-serine metabolism" EXACT []
+xref: Reactome:R-HSA-977347 "Serine metabolism"
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
 
@@ -70428,19 +70667,21 @@
 synonym: "L-serine biosynthesis" EXACT []
 synonym: "L-serine formation" EXACT []
 synonym: "L-serine synthesis" EXACT []
+xref: MetaCyc:PWY-8011
 xref: MetaCyc:SERSYN-PWY
-xref: Reactome:R-HSA-977347 "Serine biosynthesis"
 is_a: GO:0006563 ! L-serine metabolic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:SERSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-8011
+property_value: skos:narrowMatch MetaCyc:SERSYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006565
 name: L-serine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-serine to pyruvate and free ammonium." [MetaCyc:SERDEG-PWY]
+def: "The chemical reactions and pathways resulting in the breakdown of L-serine." [GOC:curators]
 synonym: "L-serine breakdown" EXACT []
 synonym: "L-serine catabolism" EXACT []
 synonym: "L-serine degradation" EXACT []
@@ -70450,6 +70691,7 @@
 is_a: GO:0170040 ! proteinogenic amino acid catabolic process
 property_value: skos:exactMatch MetaCyc:SERDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006566
@@ -70466,7 +70708,7 @@
 id: GO:0006567
 name: L-threonine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-threonine." [GOC:curators]
 synonym: "threonine breakdown" EXACT []
 synonym: "threonine catabolism" EXACT []
 synonym: "threonine degradation" EXACT []
@@ -70485,6 +70727,7 @@
 property_value: skos:narrowMatch MetaCyc:THRDLCTCAT-PWY
 property_value: skos:narrowMatch MetaCyc:THREONINE-DEG2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006568
@@ -70503,7 +70746,7 @@
 id: GO:0006569
 name: L-tryptophan catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-tryptophan, the chiral amino acid 2-amino-3-(1H-indol-3-yl)propanoic acid." [ISBN:0198547684]
+def: "The chemical reactions and pathways resulting in the breakdown of L-tryptophan." [GOC:curators]
 synonym: "tryptophan breakdown" EXACT []
 synonym: "tryptophan catabolic process" RELATED []
 synonym: "tryptophan catabolic process, using tryptophanase" NARROW []
@@ -70543,12 +70786,13 @@
 property_value: skos:narrowMatch MetaCyc:TRYPDEG-PWY
 property_value: skos:narrowMatch MetaCyc:TRYPTOPHAN-DEGRADATION-1
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006570
 name: tyrosine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving tyrosine, an aromatic amino acid, 2-amino-3-(4-hydroxyphenyl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways involving tyrosine, an aromatic amino acid, 2-amino-3-(4-hydroxyphenyl)propanoic acid." [GOC:curators]
 synonym: "tyrosine metabolism" EXACT []
 xref: UM-BBD_pathwayID:tyr
 xref: Wikipedia:Tyrosine
@@ -70565,16 +70809,25 @@
 synonym: "tyrosine biosynthesis" EXACT []
 synonym: "tyrosine formation" EXACT []
 synonym: "tyrosine synthesis" EXACT []
+xref: MetaCyc:PWY-3461
+xref: MetaCyc:PWY-6120
+xref: MetaCyc:PWY-6134
+xref: MetaCyc:TYRSYN
 is_a: GO:0006570 ! tyrosine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:1902223 ! erythrose 4-phosphate/phosphoenolpyruvate family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-3461
+property_value: skos:narrowMatch MetaCyc:PWY-6120
+property_value: skos:narrowMatch MetaCyc:PWY-6134
+property_value: skos:narrowMatch MetaCyc:TYRSYN
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20583" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0006572
 name: L-tyrosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-tyrosine, an aromatic amino acid, 2-amino-3-(4-hydroxyphenyl)propanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-tyrosine." [GOC:curators]
 synonym: "tyrosine breakdown" EXACT []
 synonym: "tyrosine catabolism" EXACT []
 synonym: "tyrosine degradation" EXACT []
@@ -70594,6 +70847,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY3O-4108
 property_value: skos:narrowMatch MetaCyc:TYRFUMCAT-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006573
@@ -70609,7 +70863,7 @@
 name: L-valine catabolic process
 namespace: biological_process
 alt_id: GO:1990292
-def: "The chemical reactions and pathways resulting in the breakdown of L-valine, 2-amino-3-methylbutanoic acid." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of L-valine." [GOC:curators]
 synonym: "valine breakdown" EXACT []
 synonym: "valine catabolism" EXACT []
 synonym: "valine degradation" EXACT []
@@ -70625,6 +70879,7 @@
 property_value: skos:narrowMatch MetaCyc:VALDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30047" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31221" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006575
@@ -70711,7 +70966,7 @@
 id: GO:0006582
 name: melanin metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving melanins, pigments largely of animal origin. High molecular weight polymers of indole quinone, they are irregular polymeric structures and are divided into three groups: allomelanins in the plant kingdom and eumelanins and phaeomelanins in the animal kingdom." [GOC:go_curators]
+def: "The chemical reactions and pathways involving melanins, pigments largely of animal origin. High molecular weight polymers of indole quinone, they are irregular polymeric structures and are divided into three groups: allomelanins in the plant kingdom and eumelanins and phaeomelanins in the animal kingdom." [GOC:curators]
 synonym: "melanin metabolism" EXACT []
 is_a: GO:0018958 ! phenol-containing compound metabolic process
 is_a: GO:0019748 ! secondary metabolic process
@@ -70721,7 +70976,7 @@
 id: GO:0006583
 name: melanin biosynthetic process from tyrosine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of melanin from other compounds, including tyrosine." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of melanin from other compounds, including tyrosine." [GOC:curators]
 synonym: "melanin anabolism from tyrosine" EXACT []
 synonym: "melanin formation from tyrosine" EXACT []
 synonym: "melanin synthesis from tyrosine" EXACT []
@@ -70768,11 +71023,14 @@
 synonym: "serotonin anabolism from tryptophan" EXACT []
 synonym: "serotonin formation from tryptophan" EXACT []
 synonym: "serotonin synthesis from tryptophan" EXACT []
+xref: MetaCyc:PWY-6030
+xref: MetaCyc:PWY-8475
 is_a: GO:0006568 ! L-tryptophan metabolic process
 is_a: GO:0042427 ! serotonin biosynthetic process
-property_value: skos:relatedMatch MetaCyc:PWY-6030
-property_value: skos:relatedMatch MetaCyc:PWY-8475
+property_value: skos:narrowMatch MetaCyc:PWY-6030
+property_value: skos:narrowMatch MetaCyc:PWY-8475
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31021" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0006588
@@ -70826,17 +71084,22 @@
 synonym: "ornithine formation" EXACT []
 synonym: "ornithine synthesis" EXACT []
 xref: MetaCyc:ARGININE-SYN4-PWY
+xref: MetaCyc:GLUTORN-PWY
+xref: MetaCyc:PWY-6922
 is_a: GO:0006591 ! ornithine metabolic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:ARGININE-SYN4-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTORN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-6922
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0006593
 name: L-ornithine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine, an amino acid only rarely found in proteins, but which is important in living organisms as an intermediate in the reactions of the urea cycle and in arginine biosynthesis." [GOC:jl, ISBN:0192801023]
+def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine." [GOC:curators]
 synonym: "ornithine breakdown" EXACT []
 synonym: "ornithine catabolism" EXACT []
 synonym: "ornithine degradation" EXACT []
@@ -70844,6 +71107,7 @@
 is_a: GO:0170035 ! L-amino acid catabolic process
 is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0006595
@@ -71282,7 +71546,7 @@
 id: GO:0006630
 name: obsolete lipid binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "lipid binding" EXACT []
 is_obsolete: true
@@ -71494,7 +71758,7 @@
 id: GO:0006649
 name: phospholipid transfer to membrane
 namespace: biological_process
-def: "The transfer of a phospholipid from its site of synthesis to the plasma membrane." [GOC:go_curators]
+def: "The transfer of a phospholipid from its site of synthesis to the plasma membrane." [GOC:curators]
 is_a: GO:0015914 ! phospholipid transport
 relationship: part_of GO:0061024 ! membrane organization
 
@@ -71993,7 +72257,7 @@
 id: GO:0006694
 name: steroid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of steroids, compounds with a 1,2,cyclopentanoperhydrophenanthrene nucleus; includes de novo formation and steroid interconversion by modification." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of steroids, compounds with a 1,2,cyclopentanoperhydrophenanthrene nucleus; includes de novo formation and steroid interconversion by modification." [GOC:curators]
 synonym: "steroid anabolism" EXACT []
 synonym: "steroid biosynthesis" EXACT []
 synonym: "steroid formation" EXACT []
@@ -72060,7 +72324,7 @@
 id: GO:0006699
 name: bile acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of bile acids, any of a group of steroid carboxylic acids occurring in bile." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of bile acids, any of a group of steroid carboxylic acids occurring in bile." [GOC:curators]
 synonym: "bile acid anabolism" EXACT []
 synonym: "bile acid biosynthesis" EXACT []
 synonym: "bile acid formation" EXACT []
@@ -72158,7 +72422,7 @@
 id: GO:0006706
 name: steroid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of steroids, compounds with a 1,2,cyclopentanoperhydrophenanthrene nucleus." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of steroids, compounds with a 1,2,cyclopentanoperhydrophenanthrene nucleus." [GOC:curators]
 synonym: "steroid breakdown" EXACT []
 synonym: "steroid catabolism" EXACT []
 synonym: "steroid degradation" EXACT []
@@ -72278,7 +72542,7 @@
 id: GO:0006716
 name: juvenile hormone metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:go_curators, ISBN:0198547684]
+def: "The chemical reactions and pathways involving juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:curators, ISBN:0198547684]
 synonym: "juvenile hormone metabolism" EXACT []
 is_a: GO:0006714 ! sesquiterpenoid metabolic process
 is_a: GO:0042445 ! hormone metabolic process
@@ -72287,7 +72551,7 @@
 id: GO:0006717
 name: obsolete juvenile hormone binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "juvenile hormone binding" EXACT []
 is_obsolete: true
@@ -72297,7 +72561,7 @@
 id: GO:0006718
 name: juvenile hormone biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:go_curators, ISBN:0198547684]
+def: "The chemical reactions and pathways resulting in the formation of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:curators, ISBN:0198547684]
 synonym: "juvenile hormone anabolism" EXACT []
 synonym: "juvenile hormone biosynthesis" EXACT []
 synonym: "juvenile hormone formation" EXACT []
@@ -72310,7 +72574,7 @@
 id: GO:0006719
 name: juvenile hormone catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:go_curators, ISBN:0198547684]
+def: "The chemical reactions and pathways resulting in the breakdown of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:curators, ISBN:0198547684]
 synonym: "juvenile hormone breakdown" EXACT []
 synonym: "juvenile hormone catabolism" EXACT []
 synonym: "juvenile hormone degradation" EXACT []
@@ -72684,7 +72948,7 @@
 id: GO:0006748
 name: obsolete lipoamide metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving lipoamide, the functional form of lipoic acid in which the carboxyl group is attached to protein by an amide linkage to a lysine amino group." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving lipoamide, the functional form of lipoic acid in which the carboxyl group is attached to protein by an amide linkage to a lysine amino group." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "lipoamide metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -72697,6 +72961,7 @@
 def: "The chemical reactions and pathways involving glutathione, the tripeptide glutamylcysteinylglycine, which acts as a coenzyme for some enzymes and as an antioxidant in the protection of sulfhydryl groups in enzymes and other proteins; it has a specific role in the reduction of hydrogen peroxide (H2O2) and oxidized ascorbate, and it participates in the gamma-glutamyl cycle." [ISBN:0198506732]
 synonym: "glutathione metabolism" EXACT []
 synonym: "oxidized glutathione reduction" NARROW []
+xref: Reactome:R-HSA-174403 "Glutathione synthesis and recycling"
 is_a: GO:0006575 ! modified amino acid metabolic process
 is_a: GO:0006790 ! sulfur compound metabolic process
 
@@ -72710,7 +72975,6 @@
 synonym: "glutathione formation" EXACT []
 synonym: "glutathione synthesis" EXACT []
 xref: MetaCyc:GLUTATHIONESYN-PWY
-xref: Reactome:R-HSA-174403 "Glutathione synthesis and recycling"
 is_a: GO:0006749 ! glutathione metabolic process
 is_a: GO:0019184 ! nonribosomal peptide biosynthetic process
 is_a: GO:0042398 ! modified amino acid biosynthetic process
@@ -72744,7 +73008,7 @@
 namespace: biological_process
 alt_id: GO:0006758
 alt_id: GO:0006759
-def: "The chemical reactions and pathways resulting in the formation of ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator." [GOC:curators, ISBN:0198506732]
 synonym: "ATP anabolism" EXACT []
 synonym: "ATP biosynthesis" EXACT []
 synonym: "ATP formation" EXACT []
@@ -72849,7 +73113,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving any of a diverse group of vitamins that are soluble in water." [GOC:jl]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "water-soluble vitamin metabolism" EXACT []
-xref: Reactome:R-HSA-196849 "Metabolism of water-soluble vitamins and cofactors"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -72927,7 +73190,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving of any of a diverse group of vitamins that are soluble in organic solvents and relatively insoluble in water." [GOC:jl, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "fat-soluble vitamin metabolism" EXACT []
-xref: Reactome:R-HSA-6806667 "Metabolism of fat-soluble vitamins"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -73012,7 +73274,7 @@
 id: GO:0006782
 name: protoporphyrinogen IX biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of protoporphyrinogen IX." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of protoporphyrinogen IX." [GOC:curators]
 synonym: "protoporphyrinogen IX anabolism" EXACT []
 synonym: "protoporphyrinogen IX biosynthesis" EXACT []
 synonym: "protoporphyrinogen IX formation" EXACT []
@@ -73138,7 +73400,7 @@
 id: GO:0006792
 name: regulation of sulfur utilization
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of sulfur utilization." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of sulfur utilization." [GOC:curators]
 synonym: "regulation of sulphur utilization" EXACT []
 is_a: GO:0032107 ! regulation of response to nutrient levels
 intersection_of: GO:0065007 ! biological regulation
@@ -73165,7 +73427,7 @@
 id: GO:0006795
 name: regulation of phosphorus utilization
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of phosphorus utilization." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of phosphorus utilization." [GOC:curators]
 is_a: GO:0051174 ! regulation of phosphorus metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006794 ! phosphorus utilization
@@ -73184,7 +73446,7 @@
 id: GO:0006797
 name: polyphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a polyphosphate, the anion or salt of polyphosphoric acid." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a polyphosphate, the anion or salt of polyphosphoric acid." [GOC:curators, ISBN:0198506732]
 synonym: "polyphosphate metabolism" EXACT []
 is_a: GO:0006793 ! phosphorus metabolic process
 is_a: GO:0043436 ! oxoacid metabolic process
@@ -73193,7 +73455,7 @@
 id: GO:0006798
 name: polyphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a polyphosphate, the anion or salt of polyphosphoric acid." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a polyphosphate, the anion or salt of polyphosphoric acid." [GOC:curators, ISBN:0198506732]
 synonym: "polyphosphate breakdown" EXACT []
 synonym: "polyphosphate catabolism" EXACT []
 synonym: "polyphosphate degradation" EXACT []
@@ -73299,7 +73561,7 @@
 id: GO:0006806
 name: obsolete insecticide resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "insecticide resistance" EXACT []
 is_obsolete: true
@@ -73319,7 +73581,7 @@
 id: GO:0006808
 name: regulation of nitrogen utilization
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of nitrogen utilization." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of nitrogen utilization." [GOC:curators]
 subset: goslim_metagenomics
 is_a: GO:0032107 ! regulation of response to nutrient levels
 intersection_of: GO:0065007 ! biological regulation
@@ -73372,7 +73634,6 @@
 subset: goslim_pir
 subset: goslim_yeast
 synonym: "ion transport" BROAD []
-xref: Reactome:R-HSA-425393 "Transport of inorganic cations/anions and amino acids/oligopeptides"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -73426,7 +73687,7 @@
 id: GO:0006815
 name: obsolete sodium/potassium transport
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it is a composite term that represents two individual processes.
 synonym: "sodium/potassium transport" EXACT []
 is_obsolete: true
@@ -73538,7 +73799,7 @@
 id: GO:0006832
 name: obsolete small molecule transport
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was an unnecessary grouping term.
 synonym: "small molecule transport" EXACT []
 is_obsolete: true
@@ -73573,6 +73834,7 @@
 xref: Reactome:R-HSA-210500 "Glutamate Neurotransmitter Release Cycle"
 xref: Reactome:R-HSA-212676 "Dopamine Neurotransmitter Release Cycle"
 xref: Reactome:R-HSA-264642 "Acetylcholine Neurotransmitter Release Cycle"
+xref: Reactome:R-HSA-442660 "SLC-mediated transport of neurotransmitters"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -73587,7 +73849,7 @@
 id: GO:0006838
 name: obsolete allantoin/allantoate transport
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it is a composite term that represents two individual processes.
 synonym: "allantoin/allantoate transport" EXACT []
 is_obsolete: true
@@ -73782,7 +74044,7 @@
 id: GO:0006858
 name: extracellular transport
 namespace: biological_process
-def: "The transport of substances that occurs outside cells." [GOC:go_curators]
+def: "The transport of substances that occurs outside cells." [GOC:curators]
 is_a: GO:0006810 ! transport
 intersection_of: GO:0006810 ! transport
 intersection_of: occurs_in GO:0005576 ! extracellular region
@@ -73822,7 +74084,7 @@
 alt_id: GO:0015852
 def: "The directed movement of purine bases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [ISBN:0198506732]
 synonym: "purine base transmembrane transport" EXACT [GOC:mah]
-synonym: "purine base transport" EXACT [GOC:go_curators]
+synonym: "purine base transport" EXACT [GOC:curators]
 synonym: "purine transmembrane transport" RELATED [GOC:mah]
 synonym: "purine transport" RELATED []
 is_a: GO:0015851 ! nucleobase transport
@@ -73842,7 +74104,7 @@
 def: "The directed movement of amino acids, organic acids containing one or more amino substituents, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:ai]
 subset: goslim_pir
 subset: goslim_yeast
-xref: Reactome:R-HSA-352230 "Amino acid transport across the plasma membrane"
+xref: Reactome:R-HSA-9958863 "SLC-mediated transport of amino acids"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -74012,7 +74274,7 @@
 id: GO:0006884
 name: cell volume homeostasis
 namespace: biological_process
-def: "Any process involved in maintaining the steady state of a cell's volume. The cell's volume refers to the three-dimensional space occupied by a cell." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process involved in maintaining the steady state of a cell's volume. The cell's volume refers to the three-dimensional space occupied by a cell." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "regulation of cell volume" EXACT [GOC:dph, GOC:tb]
 is_a: GO:0008361 ! regulation of cell size
 is_a: GO:0019725 ! cellular homeostasis
@@ -74021,7 +74283,7 @@
 id: GO:0006885
 name: regulation of pH
 namespace: biological_process
-def: "Any process involved in the maintenance of an internal equilibrium of hydrogen ions, thereby modulating the internal pH, within an organism or cell." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process involved in the maintenance of an internal equilibrium of hydrogen ions, thereby modulating the internal pH, within an organism or cell." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "hydrogen ion homeostasis" EXACT [GOC:dph, GOC:tb]
 is_a: GO:0055080 ! monoatomic cation homeostasis
 is_a: GO:0065007 ! biological regulation
@@ -74082,7 +74344,7 @@
 id: GO:0006889
 name: obsolete regulation of calcium in ER
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the concentration of calcium in the endoplasmic reticulum." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the concentration of calcium in the endoplasmic reticulum." [GOC:curators]
 comment: This term was made obsolete because the term name is ambiguous and as a result, the term was incorrectly placed in the ontology (it was a descendant of 'protein transport').
 synonym: "regulation of calcium in ER" EXACT []
 is_obsolete: true
@@ -74250,7 +74512,7 @@
 id: GO:0006905
 name: obsolete vesicle transport
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous.
 synonym: "vesicle transport" EXACT []
 is_obsolete: true
@@ -74331,7 +74593,7 @@
 name: nucleocytoplasmic transport
 namespace: biological_process
 alt_id: GO:0000063
-def: "The directed movement of molecules between the nucleus and the cytoplasm." [GOC:go_curators]
+def: "The directed movement of molecules between the nucleus and the cytoplasm." [GOC:curators]
 comment: Note that transport through the nuclear pore complex is not transmembrane because the nuclear membrane is a double membrane, and is not traversed.
 subset: goslim_chembl
 subset: goslim_drosophila
@@ -74365,7 +74627,7 @@
 namespace: biological_process
 alt_id: GO:0006917
 alt_id: GO:0008632
-def: "A programmed cell death process which begins when a cell receives an internal (e.g. DNA damage) or external signal (e.g. an extracellular death ligand), and proceeds through a series of biochemical events (signaling pathway phase) which trigger an execution phase. The execution phase is the last step of an apoptotic process, and is typically characterized by rounding-up of the cell, retraction of pseudopodes, reduction of cellular volume (pyknosis), chromatin condensation, nuclear fragmentation (karyorrhexis), plasma membrane blebbing and fragmentation of the cell into apoptotic bodies. When the execution phase is completed, the cell has died." [GOC:cjm, GOC:dhl, GOC:ecd, GOC:go_curators, GOC:mtg_apoptosis, GOC:tb, ISBN:0198506732, PMID:18846107, PMID:21494263]
+def: "A programmed cell death process which begins when a cell receives an internal (e.g. DNA damage) or external signal (e.g. an extracellular death ligand), and proceeds through a series of biochemical events (signaling pathway phase) which trigger an execution phase. The execution phase is the last step of an apoptotic process, and is typically characterized by rounding-up of the cell, retraction of pseudopodes, reduction of cellular volume (pyknosis), chromatin condensation, nuclear fragmentation (karyorrhexis), plasma membrane blebbing and fragmentation of the cell into apoptotic bodies. When the execution phase is completed, the cell has died." [GOC:cjm, GOC:curators, GOC:dhl, GOC:ecd, GOC:mtg_apoptosis, GOC:tb, ISBN:0198506732, PMID:18846107, PMID:21494263]
 synonym: "activation of apoptosis" NARROW []
 synonym: "apoptosis" NARROW []
 synonym: "apoptosis activator activity" RELATED []
@@ -74392,7 +74654,7 @@
 id: GO:0006918
 name: obsolete induction of apoptosis by p53
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more accurate terms were created.
 synonym: "induction of apoptosis by p53" EXACT []
 is_obsolete: true
@@ -74407,7 +74669,6 @@
 synonym: "activation of caspase activity" NARROW []
 synonym: "activation of metacaspase activity" NARROW []
 synonym: "caspase activation" EXACT [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-69416 "Dimerization of procaspase-8"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -74415,7 +74676,7 @@
 id: GO:0006920
 name: obsolete commitment to apoptosis
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more accurate terms were created.
 synonym: "commitment to apoptosis" EXACT []
 is_obsolete: true
@@ -74605,7 +74866,7 @@
 id: GO:0006937
 name: regulation of muscle contraction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of muscle contraction." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of muscle contraction." [GOC:curators]
 is_a: GO:0090257 ! regulation of muscle system process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006936 ! muscle contraction
@@ -74623,7 +74884,7 @@
 id: GO:0006940
 name: regulation of smooth muscle contraction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of smooth muscle contraction." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of smooth muscle contraction." [GOC:curators]
 is_a: GO:0006937 ! regulation of muscle contraction
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006939 ! smooth muscle contraction
@@ -74641,7 +74902,7 @@
 id: GO:0006942
 name: regulation of striated muscle contraction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of striated muscle contraction." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of striated muscle contraction." [GOC:curators]
 is_a: GO:0006937 ! regulation of muscle contraction
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006941 ! striated muscle contraction
@@ -74679,19 +74940,19 @@
 synonym: "viral-induced cell-cell fusion" EXACT []
 synonym: "viral-induced host cell-cell fusion" EXACT []
 synonym: "viral-induced membrane fusion" NARROW []
-xref: Reactome:R-HSA-9733458 "Induction of Cell-Cell Fusion"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28984" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0060141
 
 [Term]
 id: GO:0006949
-name: syncytium formation
+name: obsolete syncytium formation
 namespace: biological_process
-def: "The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane. Syncytia are normally derived from single cells that fuse or fail to complete cell division." [ISBN:0198506732]
-subset: goslim_pir
-is_a: GO:0009987 ! cellular process
-is_a: GO:0048646 ! anatomical structure formation involved in morphogenesis
+def: "OBSOLETE. The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane. Syncytia are normally derived from single cells that fuse or fail to complete cell division." [ISBN:0198506732]
+comment: The reason for obsoletion is that this term was an unnecessary grouping term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
+is_obsolete: true
+consider: GO:0000768
 
 [Term]
 id: GO:0006950
@@ -74715,7 +74976,7 @@
 namespace: biological_process
 alt_id: GO:0002217
 alt_id: GO:0042829
-def: "Reactions, triggered in response to the presence of a foreign body or the occurrence of an injury, which result in restriction of damage to the organism attacked or prevention/recovery from the infection caused by the attack." [GOC:go_curators]
+def: "Reactions, triggered in response to the presence of a foreign body or the occurrence of an injury, which result in restriction of damage to the organism attacked or prevention/recovery from the infection caused by the attack." [GOC:curators]
 synonym: "antimicrobial peptide activity" RELATED []
 synonym: "defence response" EXACT []
 synonym: "defense/immunity protein activity" RELATED []
@@ -74794,7 +75055,7 @@
 id: GO:0006962
 name: male-specific antibacterial humoral response
 namespace: biological_process
-def: "An immune response against bacteria, specific to males and mediated through a body fluid." [GOC:go_curators]
+def: "An immune response against bacteria, specific to males and mediated through a body fluid." [GOC:curators]
 is_a: GO:0019731 ! antibacterial humoral response
 is_a: GO:0050831 ! male-specific defense response to bacterium
 
@@ -74938,7 +75199,7 @@
 name: DNA damage response
 namespace: biological_process
 alt_id: GO:0034984
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to its DNA from environmental insults or errors during metabolism." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to its DNA from environmental insults or errors during metabolism." [GOC:curators]
 subset: goslim_yeast
 synonym: "cellular DNA damage response" EXACT []
 synonym: "cellular response to DNA damage stimulus" EXACT []
@@ -74951,11 +75212,9 @@
 id: GO:0006977
 name: obsolete DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest
 namespace: biological_process
-def: "OBSOLETE. A cascade of processes induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage and resulting in the stopping or reduction in rate of the cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cascade of processes induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage and resulting in the stopping or reduction in rate of the cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that this term is about down stream effects, not the signaling pathway itself.
 synonym: "positive regulation of cell cycle arrest by p53-mediated DNA damage response" EXACT [GOC:dph, GOC:mah]
-xref: Reactome:R-HSA-6791312 "TP53 Regulates Transcription of Cell Cycle Genes"
-xref: Reactome:R-HSA-69563 "p53-Dependent G1 DNA Damage Response"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26003" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0030330
@@ -75176,7 +75435,7 @@
 id: GO:0007002
 name: obsolete centromere binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "centromere binding" EXACT []
 is_obsolete: true
@@ -75186,7 +75445,7 @@
 id: GO:0007003
 name: obsolete telomere binding
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function and not a biological process.
 synonym: "telomere binding" EXACT []
 is_obsolete: true
@@ -75291,7 +75550,7 @@
 id: GO:0007011
 name: obsolete regulation of cytoskeleton
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:curators]
 comment: This term was made obsolete because cytoskeleton is neither a process nor a trait, so the term made no sense.
 synonym: "regulation of cytoskeleton" EXACT []
 is_obsolete: true
@@ -75364,7 +75623,7 @@
 id: GO:0007020
 name: microtubule nucleation
 namespace: biological_process
-def: "The process in which tubulin alpha-beta heterodimers begin aggregation to form an oligomeric tubulin structure (a microtubule seed). Microtubule nucleation is the initiating step in the formation of a microtubule in the absence of any existing microtubules ('de novo' microtubule formation)." [GOC:go_curators, ISBN:0815316194, PMID:12517712]
+def: "The process in which tubulin alpha-beta heterodimers begin aggregation to form an oligomeric tubulin structure (a microtubule seed). Microtubule nucleation is the initiating step in the formation of a microtubule in the absence of any existing microtubules ('de novo' microtubule formation)." [GOC:curators, ISBN:0815316194, PMID:12517712]
 xref: Wikipedia:Microtubule_nucleation
 is_a: GO:0000226 ! microtubule cytoskeleton organization
 relationship: part_of GO:0046785 ! microtubule polymerization
@@ -75572,7 +75831,6 @@
 def: "Any process that reduces the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:jid]
 synonym: "lysosome pH reduction" EXACT [GOC:bf, GOC:rph]
 is_a: GO:0007035 ! vacuolar acidification
-is_a: GO:0035751 ! regulation of lysosomal lumen pH
 
 [Term]
 id: GO:0007043
@@ -75598,7 +75856,7 @@
 id: GO:0007048
 name: obsolete oncogenesis
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a pathological process.
 synonym: "oncogenesis" EXACT []
 is_obsolete: true
@@ -75607,7 +75865,7 @@
 id: GO:0007049
 name: cell cycle
 namespace: biological_process
-def: "The progression of biochemical and morphological phases and events that occur in a cell during successive cell replication or nuclear replication events. Canonically, the cell cycle comprises the replication and segregation of genetic material followed by the division of the cell, but in endocycles or syncytial cells nuclear replication or nuclear division may not be followed by cell division." [GOC:go_curators, GOC:mtg_cell_cycle]
+def: "The progression of biochemical and morphological phases and events that occur in a cell during successive cell replication or nuclear replication events. Canonically, the cell cycle comprises the replication and segregation of genetic material followed by the division of the cell, but in endocycles or syncytial cells nuclear replication or nuclear division may not be followed by cell division." [GOC:curators, GOC:mtg_cell_cycle]
 subset: gocheck_do_not_annotate
 subset: goslim_agr
 subset: goslim_candida
@@ -75626,7 +75884,7 @@
 name: spindle organization
 namespace: biological_process
 alt_id: GO:0043146
-def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the spindle, the array of microtubules and associated molecules that forms between opposite poles of a eukaryotic cell during DNA segregation and serves to move the duplicated chromosomes apart." [GOC:go_curators, GOC:mah]
+def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the spindle, the array of microtubules and associated molecules that forms between opposite poles of a eukaryotic cell during DNA segregation and serves to move the duplicated chromosomes apart." [GOC:curators, GOC:mah]
 synonym: "spindle organisation" EXACT []
 synonym: "spindle organization and biogenesis" RELATED [GOC:mah]
 synonym: "spindle stabilization" RELATED []
@@ -75640,7 +75898,7 @@
 alt_id: GO:0000071
 alt_id: GO:0030472
 alt_id: GO:0043148
-def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the microtubule spindle during a mitotic cell cycle." [GOC:go_curators, GOC:mah]
+def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the microtubule spindle during a mitotic cell cycle." [GOC:curators, GOC:mah]
 comment: In fission yeast most mitotic spindle organization occurs in the nucleus.
 synonym: "mitotic spindle organisation" EXACT []
 synonym: "mitotic spindle organisation in nucleus" NARROW []
@@ -75762,7 +76020,7 @@
 id: GO:0007063
 name: regulation of sister chromatid cohesion
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of sister chromatid cohesion." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of sister chromatid cohesion." [GOC:curators]
 is_a: GO:0010564 ! regulation of cell cycle process
 is_a: GO:0033044 ! regulation of chromosome organization
 intersection_of: GO:0065007 ! biological regulation
@@ -75794,7 +76052,7 @@
 id: GO:0007068
 name: obsolete negative regulation of transcription during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "down regulation of transcription during mitosis" EXACT [GOC:mah]
 synonym: "down-regulation of transcription during mitosis" EXACT [GOC:mah]
@@ -75811,7 +76069,7 @@
 id: GO:0007069
 name: obsolete negative regulation of transcription from RNA polymerase I promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "down regulation of transcription from RNA polymerase I promoter during mitosis" EXACT [GOC:mah]
 synonym: "down-regulation of transcription from RNA polymerase I promoter during mitosis" EXACT [GOC:mah]
@@ -75829,7 +76087,7 @@
 id: GO:0007070
 name: obsolete negative regulation of transcription from RNA polymerase II promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "down regulation of transcription from RNA polymerase II promoter during mitosis" EXACT [GOC:mah]
 synonym: "down-regulation of transcription from RNA polymerase II promoter during mitosis" EXACT [GOC:mah]
@@ -75847,7 +76105,7 @@
 id: GO:0007071
 name: obsolete negative regulation of transcription from RNA polymerase III promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase III promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription from an RNA polymerase III promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "down regulation of transcription from RNA polymerase III promoter during mitosis" EXACT [GOC:mah]
 synonym: "down-regulation of transcription from RNA polymerase III promoter during mitosis" EXACT [GOC:mah]
@@ -76049,7 +76307,7 @@
 id: GO:0007088
 name: regulation of mitotic nuclear division
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of mitosis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of mitosis." [GOC:curators]
 synonym: "regulation of mitosis" EXACT []
 is_a: GO:0007346 ! regulation of mitotic cell cycle
 is_a: GO:0010564 ! regulation of cell cycle process
@@ -76070,7 +76328,7 @@
 id: GO:0007090
 name: obsolete regulation of S phase of mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of the progression through the S phase of the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of the progression through the S phase of the mitotic cell cycle." [GOC:curators]
 comment: This term was made obsolete as part of the cell cycle overhaul. This is because it was decided that cell cycle phases are intervals and so cannot be regulated, rather it is the transitions between the phases that are regulated.
 synonym: "regulation of S phase of mitotic cell cycle" EXACT []
 synonym: "regulation of S-phase of mitotic cell cycle" EXACT []
@@ -76280,7 +76538,7 @@
 id: GO:0007106
 name: obsolete cytokinesis, protein recruitment
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was not defined.
 synonym: "cytokinesis, protein recruitment" EXACT []
 is_obsolete: true
@@ -76370,7 +76628,7 @@
 id: GO:0007115
 name: obsolete bud site selection/establishment of cell polarity (sensu Saccharomyces)
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents two processes.
 synonym: "bud site selection/establishment of cell polarity (sensu Saccharomyces)" EXACT []
 is_obsolete: true
@@ -76393,7 +76651,7 @@
 id: GO:0007117
 name: budding cell bud growth
 namespace: biological_process
-def: "The process in which the bud portion of a cell that reproduces by budding irreversibly increases in size over time by accretion and biosynthetic production of matter similar to that already present." [GOC:go_curators]
+def: "The process in which the bud portion of a cell that reproduces by budding irreversibly increases in size over time by accretion and biosynthetic production of matter similar to that already present." [GOC:curators]
 synonym: "bud growth" BROAD []
 is_a: GO:0040007 ! growth
 relationship: part_of GO:0007114 ! cell budding
@@ -76402,7 +76660,7 @@
 id: GO:0007118
 name: budding cell apical bud growth
 namespace: biological_process
-def: "Growth at the tip of a bud, in a cell that reproduces by budding." [GOC:go_curators]
+def: "Growth at the tip of a bud, in a cell that reproduces by budding." [GOC:curators]
 synonym: "apical bud growth" BROAD []
 is_a: GO:0007117 ! budding cell bud growth
 
@@ -76410,7 +76668,7 @@
 id: GO:0007119
 name: budding cell isotropic bud growth
 namespace: biological_process
-def: "Unlocalized bud growth such that the entire surface of the bud expands evenly, in a cell that reproduces by budding." [GOC:go_curators]
+def: "Unlocalized bud growth such that the entire surface of the bud expands evenly, in a cell that reproduces by budding." [GOC:curators]
 synonym: "isotropic bud growth" BROAD []
 is_a: GO:0007117 ! budding cell bud growth
 
@@ -76775,7 +77033,7 @@
 id: GO:0007158
 name: neuron cell-cell adhesion
 namespace: biological_process
-def: "The attachment of a neuron to another cell via adhesion molecules." [GOC:go_curators]
+def: "The attachment of a neuron to another cell via adhesion molecules." [GOC:curators]
 synonym: "neuron adhesion" EXACT []
 synonym: "neuronal cell adhesion" EXACT []
 is_a: GO:0098609 ! cell-cell adhesion
@@ -76784,7 +77042,7 @@
 id: GO:0007159
 name: leukocyte cell-cell adhesion
 namespace: biological_process
-def: "The attachment of a leukocyte to another cell via adhesion molecules." [GOC:go_curators]
+def: "The attachment of a leukocyte to another cell via adhesion molecules." [GOC:curators]
 synonym: "leukocyte adhesion" EXACT []
 synonym: "leukocyte cell adhesion" EXACT []
 is_a: GO:0098609 ! cell-cell adhesion
@@ -76808,7 +77066,7 @@
 id: GO:0007162
 name: negative regulation of cell adhesion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell adhesion." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell adhesion." [GOC:curators]
 synonym: "cell adhesion receptor inhibitor activity" RELATED []
 synonym: "down regulation of cell adhesion" EXACT []
 synonym: "down-regulation of cell adhesion" EXACT []
@@ -76852,7 +77110,7 @@
 alt_id: GO:0023016
 alt_id: GO:0023033
 alt_id: GO:0023045
-def: "The cellular process in which a signal is conveyed to trigger a change in the activity or state of a cell. Signal transduction begins with reception of a signal (e.g. a ligand binding to a receptor or receptor activation by a stimulus such as light), or for signal transduction in the absence of ligand, signal-withdrawal or the activity of a constitutively active receptor. Signal transduction ends with regulation of a downstream cellular process, e.g. regulation of transcription or regulation of a metabolic process. Signal transduction covers signaling from receptors located on the surface of the cell and signaling via molecules located within the cell. For signaling between cells, signal transduction is restricted to events at and within the receiving cell." [GOC:go_curators, GOC:mtg_signaling_feb11]
+def: "The cellular process in which a signal is conveyed to trigger a change in the activity or state of a cell. Signal transduction begins with reception of a signal (e.g. a ligand binding to a receptor or receptor activation by a stimulus such as light), or for signal transduction in the absence of ligand, signal-withdrawal or the activity of a constitutively active receptor. Signal transduction ends with regulation of a downstream cellular process, e.g. regulation of transcription or regulation of a metabolic process. Signal transduction covers signaling from receptors located on the surface of the cell and signaling via molecules located within the cell. For signaling between cells, signal transduction is restricted to events at and within the receiving cell." [GOC:curators, GOC:mtg_signaling_feb11]
 comment: Note that signal transduction is defined broadly to include a ligand interacting with a receptor, downstream signaling steps and a response being triggered. A change in form of the signal in every step is not necessary. Note that in many cases the end of this process is regulation of the initiation of transcription. Note that specific transcription factors may be annotated to this term, but core/general transcription machinery such as RNA polymerase should not.
 subset: goslim_candida
 subset: goslim_chembl
@@ -76981,7 +77239,7 @@
 name: negative regulation of epidermal growth factor-activated receptor activity
 namespace: biological_process
 alt_id: GO:0007177
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of EGF-activated receptor activity." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of EGF-activated receptor activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "down regulation of epidermal growth factor receptor activity" EXACT []
 synonym: "down-regulation of epidermal growth factor receptor activity" EXACT []
@@ -77003,7 +77261,7 @@
 id: GO:0007176
 name: obsolete regulation of epidermal growth factor-activated receptor activity
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of EGF-activated receptor activity." [GOC:dph, GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of EGF-activated receptor activity." [GOC:curators, GOC:dph]
 comment: This term was obsoleted because it represents a molecular function. Consider molecular function regulator activity ; GO:0098772 or a child.
 synonym: "regulation of EGF receptor activity" EXACT []
 synonym: "regulation of EGFR activity" EXACT []
@@ -77247,7 +77505,7 @@
 id: GO:0007194
 name: negative regulation of adenylate cyclase activity
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of adenylate cyclase activity." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of adenylate cyclase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "adenylate cyclase inhibitor" RELATED []
 synonym: "down regulation of adenylate cyclase activity" EXACT []
@@ -77555,7 +77813,7 @@
 name: Notch signaling pathway
 namespace: biological_process
 alt_id: GO:0030179
-def: "The series of molecular signals initiated by an extracellular ligand binding to the receptor Notch on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:go_curators, GOC:signaling]
+def: "The series of molecular signals initiated by an extracellular ligand binding to the receptor Notch on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:curators, GOC:signaling]
 synonym: "N signaling pathway" EXACT []
 synonym: "N signalling pathway" EXACT []
 synonym: "Notch receptor signaling pathway" EXACT []
@@ -77847,7 +78105,7 @@
 id: GO:0007251
 name: obsolete activation of the inhibitor of kappa kinase
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because its meaning was unclear.
 synonym: "activation of the inhibitor of kappa kinase" EXACT []
 is_obsolete: true
@@ -78147,7 +78405,7 @@
 id: GO:0007278
 name: pole cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a pole cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a pole cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 is_a: GO:0001709 ! cell fate determination
 is_a: GO:0003006 ! developmental process involved in reproduction
 relationship: part_of GO:0007277 ! pole cell development
@@ -78176,7 +78434,7 @@
 id: GO:0007281
 name: germ cell development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of an immature germ cell over time, from its formation to the mature structure (gamete). A germ cell is any reproductive cell in a multicellular organism." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of an immature germ cell over time, from its formation to the mature structure (gamete). A germ cell is any reproductive cell in a multicellular organism." [GOC:curators]
 synonym: "gametogenesis" NARROW []
 synonym: "germ-cell development" EXACT []
 synonym: "primordial germ cell development" NARROW []
@@ -78228,7 +78486,7 @@
 id: GO:0007286
 name: spermatid development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of a spermatid over time, from its formation to the mature structure." [GOC:dph, GOC:go_curators]
+def: "The process whose specific outcome is the progression of a spermatid over time, from its formation to the mature structure." [GOC:curators, GOC:dph]
 synonym: "spermatid cell development" EXACT []
 synonym: "spermiogenesis" EXACT []
 xref: Wikipedia:Spermiogenesis
@@ -78629,7 +78887,7 @@
 id: GO:0007329
 name: obsolete positive regulation of transcription from RNA polymerase II promoter by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that activates or increases the rate of transcription from an RNA polymerase II promoter." [GOC:go_curators]
+def: "OBSOLETE. Any process involving pheromones that activates or increases the rate of transcription from an RNA polymerase II promoter." [GOC:curators]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 synonym: "activation of transcription from RNA polymerase II promoter by pheromones" NARROW []
 synonym: "positive regulation of transcription from Pol II promoter by pheromones" EXACT []
@@ -78705,7 +78963,6 @@
 def: "The binding and fusion of a sperm, with the plasma membrane of the oocyte as part of the process of single fertilization. In sperm with flagella, binding occurs at the posterior (post-acrosomal) region of the sperm head." [GOC:dph, GOC:jl, PMID:17644969]
 synonym: "sperm-oocyte fusion" NARROW []
 is_a: GO:0022412 ! cellular process involved in reproduction in multicellular organism
-is_a: GO:0045026 ! plasma membrane fusion
 relationship: part_of GO:0007338 ! single fertilization
 
 [Term]
@@ -78740,7 +78997,7 @@
 id: GO:0007346
 name: regulation of mitotic cell cycle
 namespace: biological_process
-def: "Any process that modulates the rate or extent of progress through the mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the rate or extent of progress through the mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "mitotic cell cycle modulation" EXACT []
 synonym: "mitotic cell cycle regulation" EXACT []
 synonym: "mitotic cell cycle regulator" RELATED []
@@ -78758,7 +79015,7 @@
 id: GO:0007347
 name: regulation of preblastoderm mitotic cell cycle
 namespace: biological_process
-def: "A cell cycle process that modulates the rate or extent of the progression through the preblastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "A cell cycle process that modulates the rate or extent of the progression through the preblastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "modulation of preblastoderm mitotic cell cycle progression" EXACT []
 synonym: "preblastoderm mitotic cell cycle modulation" EXACT []
 synonym: "preblastoderm mitotic cell cycle regulation" EXACT []
@@ -78774,7 +79031,7 @@
 id: GO:0007348
 name: regulation of syncytial blastoderm mitotic cell cycle
 namespace: biological_process
-def: "A cell cycle process that modulates the rate or extent of the progression through the syncytial blastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "A cell cycle process that modulates the rate or extent of the progression through the syncytial blastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "modulation of syncytial blastoderm cell cycle progression" EXACT []
 synonym: "regulation of progression through syncytial blastoderm mitotic cell cycle" EXACT [GOC:dph, GOC:tb]
 synonym: "regulation of syncytial blastoderm cell cycle" EXACT []
@@ -78793,7 +79050,7 @@
 name: cellularization
 namespace: biological_process
 alt_id: GO:0009796
-def: "The separation of a multi-nucleate cell or syncytium into individual cells. An example of this is found in Drosophila melanogaster embryo development." [GOC:go_curators, GOC:mtg_sensu, ISBN:0716731363]
+def: "The separation of a multi-nucleate cell or syncytium into individual cells. An example of this is found in Drosophila melanogaster embryo development." [GOC:curators, GOC:mtg_sensu, ISBN:0716731363]
 is_a: GO:0048646 ! anatomical structure formation involved in morphogenesis
 relationship: part_of GO:0007275 ! multicellular organism development
 
@@ -79135,7 +79392,7 @@
 id: GO:0007389
 name: pattern specification process
 namespace: biological_process
-def: "Any developmental process that results in the creation of defined areas or spaces within an organism to which cells respond and eventually are instructed to differentiate." [GOC:go_curators, GOC:isa_complete, ISBN:0521436125]
+def: "Any developmental process that results in the creation of defined areas or spaces within an organism to which cells respond and eventually are instructed to differentiate." [GOC:curators, GOC:isa_complete, ISBN:0521436125]
 subset: goslim_drosophila
 synonym: "pattern biosynthesis" RELATED []
 synonym: "pattern formation" RELATED []
@@ -79171,7 +79428,7 @@
 id: GO:0007393
 name: dorsal closure, leading edge cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell within the dorsal ectoderm becomes capable of differentiating autonomously into a leading edge cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:bf, GOC:go_curators, PMID:12147138]
+def: "The cell fate determination process in which a cell within the dorsal ectoderm becomes capable of differentiating autonomously into a leading edge cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:bf, GOC:curators, PMID:12147138]
 is_a: GO:0035028 ! leading edge cell fate determination
 intersection_of: GO:0035028 ! leading edge cell fate determination
 intersection_of: part_of GO:0007391 ! dorsal closure
@@ -79238,7 +79495,7 @@
 alt_id: GO:0007408
 alt_id: GO:0043347
 alt_id: GO:0043348
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a neuroblast cell regardless of its environment; upon determination, the cell fate cannot be reversed. An example of this process is found in Mus musculus." [GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a neuroblast cell regardless of its environment; upon determination, the cell fate cannot be reversed. An example of this process is found in Mus musculus." [GOC:curators]
 synonym: "neuroblast cell fate determination" EXACT []
 synonym: "neuroblast identity determination" EXACT []
 is_a: GO:0001709 ! cell fate determination
@@ -79259,7 +79516,7 @@
 id: GO:0007402
 name: ganglion mother cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a ganglion mother cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a ganglion mother cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 is_a: GO:0001709 ! cell fate determination
 
 [Term]
@@ -79267,7 +79524,7 @@
 name: glial cell fate determination
 namespace: biological_process
 alt_id: GO:0043361
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a glial cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators, GOC:mtg_sensu]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a glial cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0001709 ! cell fate determination
 relationship: part_of GO:0021781 ! glial cell fate commitment
 
@@ -79304,7 +79561,7 @@
 namespace: biological_process
 alt_id: GO:0043351
 alt_id: GO:0043352
-def: "A change in the morphology or behavior of a neuroblast resulting from exposure to an activating factor such as a cellular or soluble ligand." [GOC:go_curators, GOC:mtg_sensu]
+def: "A change in the morphology or behavior of a neuroblast resulting from exposure to an activating factor such as a cellular or soluble ligand." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0001775 ! cell activation
 relationship: part_of GO:0048699 ! generation of neurons
 
@@ -79397,7 +79654,7 @@
 id: GO:0007418
 name: ventral midline development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the ventral midline over time, from its formation to the mature structure. In protostomes (such as insects, snails and worms) as well as deuterostomes (vertebrates), the midline is an embryonic region that functions in patterning of the adjacent nervous tissue. The ventral midline in insects is a cell population extending along the ventral surface of the embryo and is the region from which cells detach to form the ventrally located nerve cords. In vertebrates, the midline is originally located dorsally. During development, it folds inwards and becomes the ventral part of the dorsally located neural tube and is then called the ventral midline, or floor plate." [GOC:bf, GOC:go_curators, PMID:12075342]
+def: "The process whose specific outcome is the progression of the ventral midline over time, from its formation to the mature structure. In protostomes (such as insects, snails and worms) as well as deuterostomes (vertebrates), the midline is an embryonic region that functions in patterning of the adjacent nervous tissue. The ventral midline in insects is a cell population extending along the ventral surface of the embryo and is the region from which cells detach to form the ventrally located nerve cords. In vertebrates, the midline is originally located dorsally. During development, it folds inwards and becomes the ventral part of the dorsally located neural tube and is then called the ventral midline, or floor plate." [GOC:bf, GOC:curators, PMID:12075342]
 is_a: GO:0048856 ! anatomical structure development
 relationship: part_of GO:0007417 ! central nervous system development
 
@@ -79405,7 +79662,7 @@
 id: GO:0007419
 name: ventral cord development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the ventral cord over time, from its formation to the mature structure. The ventral cord is one of the distinguishing traits of the central nervous system of all arthropods (such as insects, crustaceans and arachnids) as well as many other invertebrates, such as the annelid worms." [GOC:bf, GOC:go_curators, PMID:30826502]
+def: "The process whose specific outcome is the progression of the ventral cord over time, from its formation to the mature structure. The ventral cord is one of the distinguishing traits of the central nervous system of all arthropods (such as insects, crustaceans and arachnids) as well as many other invertebrates, such as the annelid worms." [GOC:bf, GOC:curators, PMID:30826502]
 is_a: GO:0048513 ! animal organ development
 relationship: part_of GO:0007417 ! central nervous system development
 
@@ -79430,7 +79687,7 @@
 id: GO:0007422
 name: peripheral nervous system development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the peripheral nervous system over time, from its formation to the mature structure. The peripheral nervous system is one of the two major divisions of the nervous system. Nerves in the PNS connect the central nervous system (CNS) with sensory organs, other organs, muscles, blood vessels and glands." [GOC:go_curators, UBERON:0000010]
+def: "The process whose specific outcome is the progression of the peripheral nervous system over time, from its formation to the mature structure. The peripheral nervous system is one of the two major divisions of the nervous system. Nerves in the PNS connect the central nervous system (CNS) with sensory organs, other organs, muscles, blood vessels and glands." [GOC:curators, UBERON:0000010]
 is_a: GO:0048731 ! system development
 relationship: part_of GO:0007399 ! nervous system development
 
@@ -79438,7 +79695,7 @@
 id: GO:0007423
 name: sensory organ development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of sensory organs over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of sensory organs over time, from its formation to the mature structure." [GOC:curators]
 subset: goslim_drosophila
 synonym: "sense organ development" EXACT [GOC:dph]
 is_a: GO:0048513 ! animal organ development
@@ -79561,7 +79818,7 @@
 id: GO:0007437
 name: adult salivary gland morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the adult salivary gland are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the adult salivary gland are generated and organized." [GOC:curators]
 is_a: GO:0007435 ! salivary gland morphogenesis
 
 [Term]
@@ -79593,7 +79850,7 @@
 id: GO:0007441
 name: anterior midgut (ectodermal) morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the anterior midgut (ectodermal) are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the anterior midgut (ectodermal) are generated and organized." [GOC:curators]
 is_a: GO:0002009 ! morphogenesis of an epithelium
 relationship: part_of GO:0048567 ! ectodermal digestive tract morphogenesis
 
@@ -79768,7 +80025,7 @@
 id: GO:0007468
 name: obsolete regulation of rhodopsin gene expression
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of rhodopsin gene expression. This includes transcriptional, translational, or posttranslational regulation." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of rhodopsin gene expression. This includes transcriptional, translational, or posttranslational regulation." [GOC:curators, GOC:dph, GOC:tb]
 comment: This term was obsoleted because it refers to a specific gene product. The gene product should be captured as an annotation extension using 'has input'.
 synonym: "regulation of rhodopsin gene activity" EXACT [GOC:dph, GOC:tb]
 is_obsolete: true
@@ -79984,7 +80241,7 @@
 id: GO:0007493
 name: endodermal cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into an endoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators, ISBN:0878932437]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into an endoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators, ISBN:0878932437]
 comment: Note that this term was 'endoderm determination'. Changed string to make it more consistent with parent term 'cell fate determination'.
 synonym: "endoderm cell fate determination" EXACT []
 is_a: GO:0001709 ! cell fate determination
@@ -80018,7 +80275,7 @@
 id: GO:0007497
 name: posterior midgut development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the posterior midgut over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the posterior midgut over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048856 ! anatomical structure development
 relationship: part_of GO:0007494 ! midgut development
 
@@ -80043,7 +80300,7 @@
 id: GO:0007500
 name: mesodermal cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a mesoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators, ISBN:0878932437]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a mesoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators, ISBN:0878932437]
 comment: Note that this term was 'mesoderm determination'. Changed string to make more consistent with parent term 'cell fate determination'.
 synonym: "mesoderm cell fate determination" EXACT []
 is_a: GO:0001709 ! cell fate determination
@@ -80053,7 +80310,7 @@
 id: GO:0007501
 name: mesodermal cell fate specification
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a mesoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a mesoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:curators]
 comment: Note that this term was 'fate specification in mesoderm'. String of term was changed to correspond to format of sibling terms 'endoderm cell fate specification' and 'ectoderm cell fate specification'.
 synonym: "mesoderm cell fate specification" EXACT []
 is_a: GO:0001708 ! cell fate specification
@@ -80135,7 +80392,7 @@
 id: GO:0007510
 name: cardioblast cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a cardioblast cell regardless of its environment; upon determination, the cell fate cannot be reversed. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a cardioblast cell regardless of its environment; upon determination, the cell fate cannot be reversed. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 is_a: GO:0060913 ! cardiac cell fate determination
 relationship: part_of GO:0042684 ! cardioblast cell fate commitment
 
@@ -80179,7 +80436,7 @@
 id: GO:0007518
 name: myoblast fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a myoblast regardless of its environment; upon determination, the cell fate cannot be reversed. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a myoblast regardless of its environment; upon determination, the cell fate cannot be reversed. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:curators]
 is_a: GO:0001709 ! cell fate determination
 relationship: part_of GO:0048625 ! myoblast fate commitment
 
@@ -80198,14 +80455,14 @@
 name: myoblast fusion
 namespace: biological_process
 def: "A process in which non-proliferating myoblasts fuse to existing fibers or to myotubes to form new fibers. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:mtg_muscle]
-is_a: GO:0000768 ! syncytium formation by plasma membrane fusion
+is_a: GO:0000768 ! syncytium formation by cell-cell fusion
 relationship: part_of GO:0014902 ! myotube differentiation
 
 [Term]
 id: GO:0007521
 name: muscle cell fate determination
 namespace: biological_process
-def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a muscle cell regardless of its environment; upon determination, the cell fate cannot be reversed." [CL:0000187, GOC:go_curators]
+def: "The cell fate determination process in which a cell becomes capable of differentiating autonomously into a muscle cell regardless of its environment; upon determination, the cell fate cannot be reversed." [CL:0000187, GOC:curators]
 is_a: GO:0001709 ! cell fate determination
 relationship: part_of GO:0042693 ! muscle cell fate commitment
 
@@ -80213,7 +80470,7 @@
 id: GO:0007522
 name: visceral muscle development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the visceral muscle over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the visceral muscle over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0007517 ! muscle organ development
 
 [Term]
@@ -80291,7 +80548,7 @@
 id: GO:0007532
 name: regulation of mating-type specific transcription, DNA-templated
 namespace: biological_process
-def: "Any mating-type specific process that modulates the frequency, rate or extent of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any mating-type specific process that modulates the frequency, rate or extent of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 synonym: "mating-type specific transcriptional control" EXACT []
 synonym: "regulation of mating-type specific transcription, DNA-dependent" EXACT [GOC:txnOH]
 is_a: GO:0006355 ! regulation of DNA-templated transcription
@@ -80503,7 +80760,7 @@
 id: GO:0007554
 name: regulation of ecdysteroid biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:curators]
 synonym: "regulation of ecdysteroid anabolism" EXACT []
 synonym: "regulation of ecdysteroid biosynthesis" EXACT []
 synonym: "regulation of ecdysteroid formation" EXACT []
@@ -80519,7 +80776,7 @@
 id: GO:0007555
 name: regulation of ecdysteroid secretion
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the regulated release of ecdysteroid from a cell." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the regulated release of ecdysteroid from a cell." [GOC:curators]
 is_a: GO:2000831 ! regulation of steroid hormone secretion
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0045457 ! ecdysteroid secretion
@@ -80529,7 +80786,7 @@
 id: GO:0007556
 name: obsolete regulation of juvenile hormone metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "regulation of juvenile hormone metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30570" xsd:anyURI
@@ -80539,7 +80796,7 @@
 id: GO:0007557
 name: regulation of juvenile hormone biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:curators]
 synonym: "regulation of juvenile hormone anabolism" EXACT []
 synonym: "regulation of juvenile hormone biosynthesis" EXACT []
 synonym: "regulation of juvenile hormone formation" EXACT []
@@ -80555,7 +80812,7 @@
 id: GO:0007558
 name: regulation of juvenile hormone secretion
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of juvenile hormone secretion." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of juvenile hormone secretion." [GOC:curators]
 is_a: GO:0032368 ! regulation of lipid transport
 is_a: GO:0044060 ! regulation of endocrine process
 is_a: GO:0046883 ! regulation of hormone secretion
@@ -80607,7 +80864,7 @@
 id: GO:0007563
 name: regulation of eclosion
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the emergence of an insect from a pupa-case or of a larva from an egg." [GOC:go_curators, ISBN:0198600461]
+def: "Any process that modulates the frequency, rate or extent of the emergence of an insect from a pupa-case or of a larva from an egg." [GOC:curators, ISBN:0198600461]
 is_a: GO:2000026 ! regulation of multicellular organismal development
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0007562 ! eclosion
@@ -80617,7 +80874,7 @@
 id: GO:0007564
 name: regulation of chitin-based cuticle tanning
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of chitin-based cuticular tanning." [GOC:go_curators, GOC:jid, GOC:mtg_sensu]
+def: "Any process that modulates the frequency, rate or extent of chitin-based cuticular tanning." [GOC:curators, GOC:jid, GOC:mtg_sensu]
 synonym: "regulation of cuticle hardening" NARROW []
 synonym: "regulation of cuticle tanning" EXACT []
 is_a: GO:2000026 ! regulation of multicellular organismal development
@@ -80683,7 +80940,7 @@
 id: GO:0007570
 name: obsolete age dependent accumulation of genetic damage
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not describe a biological process.
 synonym: "age dependent accumulation of genetic damage" EXACT []
 is_obsolete: true
@@ -80705,7 +80962,7 @@
 id: GO:0007572
 name: obsolete age dependent decreased translational activity
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not describe a biological process.
 synonym: "age dependent decreased translational activity" EXACT []
 is_obsolete: true
@@ -80714,7 +80971,7 @@
 id: GO:0007573
 name: obsolete age dependent increased protein content
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not describe a biological process.
 synonym: "age dependent increased protein content" EXACT []
 is_obsolete: true
@@ -80802,7 +81059,7 @@
 id: GO:0007583
 name: obsolete killer activity
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it is ambiguous; 'killer activity' could refer to several different processes.
 synonym: "killer activity" EXACT []
 is_obsolete: true
@@ -80812,7 +81069,7 @@
 id: GO:0007584
 name: response to nutrient
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a nutrient stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a nutrient stimulus." [GOC:curators]
 synonym: "nutritional response pathway" NARROW []
 synonym: "response to nutrients" EXACT []
 is_a: GO:0031667 ! response to nutrient levels
@@ -80862,7 +81119,7 @@
 id: GO:0007590
 name: obsolete fat body metabolic process (sensu Insecta)
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving the fat body. A fat body is a fat-containing cellular structure which serves as an energy reserve. As in, but not restricted to, the true insects (Insecta, ncbi_taxonomy_id:50557)." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways involving the fat body. A fat body is a fat-containing cellular structure which serves as an energy reserve. As in, but not restricted to, the true insects (Insecta, ncbi_taxonomy_id:50557)." [GOC:curators, ISBN:0198506732]
 comment: This term was made obsolete because it represents metabolism within the fat body not of the fat body.
 synonym: "fat body metabolic process (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -80992,7 +81249,7 @@
 id: GO:0007602
 name: phototransduction
 namespace: biological_process
-def: "The sequence of reactions within a cell required to convert absorbed photons into a molecular signal." [GOC:go_curators]
+def: "The sequence of reactions within a cell required to convert absorbed photons into a molecular signal." [GOC:curators]
 synonym: "opsin" RELATED []
 synonym: "phototransduction, visible light, light adaptation" NARROW []
 synonym: "phototrophin mediated phototransduction" NARROW []
@@ -81004,7 +81261,7 @@
 id: GO:0007603
 name: phototransduction, visible light
 namespace: biological_process
-def: "The sequence of reactions within a cell required to convert absorbed photons from visible light into a molecular signal. A visible light stimulus is electromagnetic radiation that can be perceived visually by an organism; for organisms lacking a visual system, this can be defined as light with a wavelength within the range 380 to 780 nm." [GOC:go_curators, ISBN:0198506732]
+def: "The sequence of reactions within a cell required to convert absorbed photons from visible light into a molecular signal. A visible light stimulus is electromagnetic radiation that can be perceived visually by an organism; for organisms lacking a visual system, this can be defined as light with a wavelength within the range 380 to 780 nm." [GOC:curators, ISBN:0198506732]
 synonym: "visual cascade" EXACT [PMID:1962207, PMID:9822721]
 synonym: "visual transduction" EXACT [PMID:10611962]
 xref: Reactome:R-HSA-2187338 "Visual phototransduction"
@@ -81015,7 +81272,7 @@
 id: GO:0007604
 name: phototransduction, UV
 namespace: biological_process
-def: "The sequence of reactions within a cell required to convert absorbed photons from UV light into a molecular signal; ultraviolet radiation is electromagnetic radiation with a wavelength in the range of 10 to 400 nanometers." [GOC:go_curators, ISBN:0198506732]
+def: "The sequence of reactions within a cell required to convert absorbed photons from UV light into a molecular signal; ultraviolet radiation is electromagnetic radiation with a wavelength in the range of 10 to 400 nanometers." [GOC:curators, ISBN:0198506732]
 synonym: "phototransduction, ultraviolet light" EXACT []
 synonym: "phototransduction, ultraviolet radiation" EXACT []
 synonym: "phototransduction, UV light" EXACT []
@@ -81194,14 +81451,14 @@
 name: circadian rhythm
 namespace: biological_process
 alt_id: GO:0050895
-def: "Any biological process in an organism that recurs with a regularity of approximately 24 hours." [GOC:bf, GOC:go_curators]
+def: "Any biological process in an organism that recurs with a regularity of approximately 24 hours." [GOC:bf, GOC:curators]
 subset: goslim_drosophila
 subset: goslim_plant
 subset: goslim_plant_ribbon
 synonym: "circadian process" EXACT []
 synonym: "circadian response" RELATED []
 synonym: "response to circadian rhythm" RELATED []
-xref: Reactome:R-HSA-400253 "Circadian Clock"
+xref: Reactome:R-HSA-9909396 "Circadian clock"
 xref: Wikipedia:Circadian_rhythm
 is_a: GO:0048511 ! rhythmic process
 
@@ -81329,7 +81586,7 @@
 id: GO:0007635
 name: chemosensory behavior
 namespace: biological_process
-def: "Behavior that is dependent upon the sensation of chemicals." [GOC:go_curators]
+def: "Behavior that is dependent upon the sensation of chemicals." [GOC:curators]
 synonym: "behavioral response to chemical stimulus" EXACT []
 synonym: "behavioural response to chemical stimulus" EXACT []
 synonym: "chemosensory behaviour" EXACT []
@@ -81361,7 +81618,7 @@
 id: GO:0007638
 name: mechanosensory behavior
 namespace: biological_process
-def: "Behavior that is dependent upon the sensation of a mechanical stimulus." [GOC:go_curators]
+def: "Behavior that is dependent upon the sensation of a mechanical stimulus." [GOC:curators]
 synonym: "behavioral response to mechanical stimulus" EXACT []
 synonym: "behavioural response to mechanical stimulus" EXACT []
 synonym: "mechanosensory behaviour" EXACT []
@@ -81482,7 +81739,7 @@
 id: GO:0008016
 name: regulation of heart contraction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of heart contraction. Heart contraction is the process in which the heart decreases in volume in a characteristic way to propel blood through the body." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of heart contraction. Heart contraction is the process in which the heart decreases in volume in a characteristic way to propel blood through the body." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "regulation of cardiac contraction" EXACT []
 is_a: GO:1903522 ! regulation of blood circulation
 intersection_of: GO:0065007 ! biological regulation
@@ -81513,7 +81770,7 @@
 id: GO:0008019
 name: obsolete macrophage receptor activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because its name is ambiguous, and it was never defined.
 synonym: "macrophage receptor activity" EXACT []
 is_obsolete: true
@@ -81612,7 +81869,7 @@
 def: "Enables the transfer of monocarboxylic acids from one side of a membrane to the other. A monocarboxylic acid is an organic acid with one COOH group." [GOC:ai]
 synonym: "monocarboxylate carrier" NARROW []
 synonym: "prostaglandin/thromboxane transporter activity" NARROW []
-xref: Reactome:R-HSA-429749 "SLC5A8 transports monocarboxylates from extracellular region to cytosol"
+xref: Reactome:R-HSA-429749 "SLC5A8 (SMCT) co-transports monocarboxylates and Na+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-433698 "SLC16A3,7,8 cotransport monocarboxylates, H+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5624211 "Defective SLC16A1 does not cotransport monocarboxylates, H+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-9645220 "SLC16A1:BSG cotransports monocarboxylates, H+ from extracellular region to cytosol"
@@ -81686,7 +81943,7 @@
 id: GO:0008037
 name: cell recognition
 namespace: biological_process
-def: "The process in which a cell in an organism interprets its surroundings." [GOC:go_curators]
+def: "The process in which a cell in an organism interprets its surroundings." [GOC:curators]
 subset: goslim_pir
 synonym: "recognition of surroundings by cell" EXACT []
 is_a: GO:0009987 ! cellular process
@@ -81695,7 +81952,7 @@
 id: GO:0008038
 name: neuron recognition
 namespace: biological_process
-def: "The process in which a neuronal cell in a multicellular organism interprets its surroundings." [GOC:go_curators]
+def: "The process in which a neuronal cell in a multicellular organism interprets its surroundings." [GOC:curators]
 synonym: "neuronal cell recognition" EXACT []
 is_a: GO:0008037 ! cell recognition
 relationship: part_of GO:0048666 ! neuron development
@@ -81713,7 +81970,7 @@
 id: GO:0008041
 name: obsolete storage protein of fat body (sensu Insecta)
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it describes a cellular location rather than a function.
 synonym: "storage protein of fat body (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -81780,6 +82037,7 @@
 def: "A molecular function regulator that increases a catalytic activity." [GOC:pdt]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
 synonym: "metalloenzyme activator activity" NARROW []
+xref: Reactome:R-HSA-9948070 "POMGNT1 catalyzes FKRP:FKTN:RXYLT1 formation"
 is_a: GO:0030234 ! enzyme regulator activity
 is_a: GO:0140677 ! molecular function activator activity
 relationship: positively_regulates GO:0003824 ! catalytic activity
@@ -81904,7 +82162,7 @@
 id: GO:0008063
 name: Toll signaling pathway
 namespace: biological_process
-def: "The series of molecular signals initiated by an extracellular ligand binding to the receptor Toll on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:go_curators, PMID:11135568, PMID:19126860]
+def: "The series of molecular signals initiated by an extracellular ligand binding to the receptor Toll on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:curators, PMID:11135568, PMID:19126860]
 synonym: "Tl signaling pathway" EXACT []
 synonym: "Tl signalling pathway" EXACT []
 synonym: "Toll signalling pathway" EXACT []
@@ -82002,6 +82260,7 @@
 name: ornithine decarboxylase inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces the activity of ornithine decarboxylase." [GOC:jl]
+xref: Reactome:R-HSA-350567 "Antizyme OAZ binds to Ornithine decarboxylase"
 is_a: GO:0004857 ! enzyme inhibitor activity
 is_a: GO:0042979 ! ornithine decarboxylase regulator activity
 relationship: negatively_regulates GO:0004586 ! ornithine decarboxylase activity
@@ -82075,6 +82334,7 @@
 xref: Reactome:R-HSA-2473152 "Acetylation of SMC3 subunit of centromeric chromatin associated cohesin by ESCO1 or ESCO2"
 xref: Reactome:R-HSA-6790987 "NAT10 acetylates cytidine-1337 and cytidine-1842 of 18S rRNA yielding 4-acetylcytidine-1377 and 4-acetylcytidine-1842"
 xref: Reactome:R-HSA-9636560 "Eis acetylates DUSP16"
+xref: Reactome:R-HSA-9913373 "AAC(6)-Ib acetylates ciprofloxacin"
 is_a: GO:0016407 ! acetyltransferase activity
 is_a: GO:0016410 ! N-acyltransferase activity
 
@@ -82304,6 +82564,27 @@
 synonym: "NADP-aldehyde reductase activity" EXACT []
 xref: EC:1.1.1.2
 xref: MetaCyc:ALCOHOL-DEHYDROGENASE-NADPORNOP+-RXN
+xref: Reactome:R-HSA-192033 "4-cholesten-7alpha-ol-3-one is reduced to 5beta-cholestan-7alpha-ol-3-one"
+xref: Reactome:R-HSA-192036 "5Beta-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha, 12alpha-triol"
+xref: Reactome:R-HSA-192067 "4-cholesten-7alpha, 12alpha-diol-3-one is reduced to 5beta-cholesten-7alpha, 12alpha-diol-3-one"
+xref: Reactome:R-HSA-192160 "5beta-cholestan-7alpha-ol-3-one is reduced to 5beta-cholestan-3alpha, 7alpha-diol"
+xref: Reactome:R-HSA-193746 "4-cholesten-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-7alpha,24(S)-diol-3-one"
+xref: Reactome:R-HSA-193755 "4-cholesten-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,24(S)-triol-3-one"
+xref: Reactome:R-HSA-193758 "5beta-cholestan-7alpha,24(S)-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,24(S)-triol"
+xref: Reactome:R-HSA-193781 "5Beta-cholestan-7alpha,12alpha,24(S)-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol"
+xref: Reactome:R-HSA-193800 "5Beta-cholestan-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol"
+xref: Reactome:R-HSA-193821 "4-cholesten-7alpha,12alpha,27-triol-3-one is reduced to 5beta-cholestan-7alpha,12alpha,27-triol-3-one"
+xref: Reactome:R-HSA-193824 "4-cholesten-7alpha,27-diol-3-one is reduced to 5beta-cholestan-7alpha,27-diol-3-one"
+xref: Reactome:R-HSA-193841 "5beta-cholestan-7alpha,27-diol-3-one is reduced to 5beta-cholestan-3alpha,7alpha,27-triol"
+xref: Reactome:R-HSA-198845 "CYB5A:heme reduces Asc.- to AscH-"
+xref: Reactome:R-HSA-2855252 "AKRs reduce RBP2:atRAL to RBP2:atROL"
+xref: Reactome:R-HSA-5423637 "AKR dimers reduce AFBDHO to AFBDOH"
+xref: Reactome:R-HSA-5692232 "AKR1A1 oxidises BaPtDHD to BaP-7,8-dione"
+xref: Reactome:R-HSA-9027531 "Dehydrogenase dehydrogenates 13-HDHA to 13-oxo-DHA"
+xref: Reactome:R-HSA-9027562 "Dehydrogenase dehydrogenates 17-HDHA to 17-oxo-DHA"
+xref: Reactome:R-HSA-9027598 "Dehydrogenase dehydrogenates 13(R)-HDPAn-3 to 13-oxo-DPAn-3"
+xref: Reactome:R-HSA-9027600 "Dehydrogenase dehydrogenates 17-HDPAn-3 to 17-oxo-DPAn-3"
+xref: Reactome:R-HSA-9937340 "DHRSX reduces dolichal to dolichol"
 xref: RHEA:15937
 is_a: GO:0018455 ! alcohol dehydrogenase [NAD(P)+] activity
 property_value: skos:exactMatch EC:1.1.1.2
@@ -82402,20 +82683,22 @@
 
 [Term]
 id: GO:0008110
-name: L-histidine:2-oxoglutarate aminotransferase activity
+name: L-histidine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-histidine = 3-(imidazol-5-yl)pyruvate + L-glutamate." [EC:2.6.1.38, RHEA:16565]
+def: "Catalysis of the reaction: L-histidine + 2-oxoglutarate = 3-(imidazol-5-yl)pyruvate + L-glutamate." [RHEA:16565]
 synonym: "histidine aminotransferase activity" BROAD []
 synonym: "histidine transaminase activity" BROAD [EC:2.6.1.38]
 synonym: "histidine-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.38]
+synonym: "L-histidine:2-oxoglutarate aminotransferase activity" EXACT []
 xref: EC:2.6.1.38
 xref: KEGG_REACTION:R01161
 xref: MetaCyc:HISTTRANSAM-RXN
 xref: RHEA:16565
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.38
 property_value: skos:exactMatch RHEA:16565
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008111
@@ -83073,7 +83356,7 @@
 namespace: biological_process
 alt_id: GO:0044236
 alt_id: GO:0044710
-def: "A cellular process consisting of the biochemical pathways by which a living organism transforms chemical substances. This includes including anabolism (biosynthetic process) and catabolism (catabolic process). Metabolic processes includes the transformation of small molecules, as well  macromolecular processes such as DNA repair and replication, protein synthesis and degradation." [GOC:go_curators, ISBN:0198547684]
+def: "A cellular process consisting of the biochemical pathways by which a living organism transforms chemical substances. This includes including anabolism (biosynthetic process) and catabolism (catabolic process). Metabolic processes includes the transformation of small molecules, as well  macromolecular processes such as DNA repair and replication, protein synthesis and degradation." [GOC:curators, ISBN:0198547684]
 comment: Note that metabolic processes do not include single functions or processes such as protein-protein interactions, protein-nucleic acids, nor receptor-ligand interactions.
 subset: gocheck_do_not_annotate
 subset: goslim_chembl
@@ -83144,7 +83427,7 @@
 id: GO:0008156
 name: negative regulation of DNA replication
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA replication." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA replication." [GOC:curators]
 synonym: "DNA replication inhibitor" RELATED []
 synonym: "down regulation of DNA replication" EXACT []
 synonym: "down-regulation of DNA replication" EXACT []
@@ -83167,7 +83450,7 @@
 id: GO:0008158
 name: hedgehog receptor activity
 namespace: molecular_function
-def: "Combining with a member of the hedgehog protein family and transmitting the signal across the membrane to initiate a change in cell activity." [GOC:bf, GOC:go_curators, PMID:9278137]
+def: "Combining with a member of the hedgehog protein family and transmitting the signal across the membrane to initiate a change in cell activity." [GOC:bf, GOC:curators, PMID:9278137]
 synonym: "patched activity" EXACT []
 is_a: GO:0004888 ! transmembrane signaling receptor activity
 relationship: has_part GO:0097108 ! hedgehog family protein binding
@@ -83194,7 +83477,7 @@
 id: GO:0008161
 name: obsolete carbamate resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "carbamate resistance" EXACT []
 is_obsolete: true
@@ -83204,7 +83487,7 @@
 id: GO:0008162
 name: obsolete cyclodiene resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "cyclodiene resistance" EXACT []
 is_obsolete: true
@@ -83214,7 +83497,7 @@
 id: GO:0008163
 name: obsolete DDT resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "DDT resistance" EXACT []
 is_obsolete: true
@@ -83224,7 +83507,7 @@
 id: GO:0008164
 name: obsolete organophosphorus resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "organophosphorus resistance" EXACT []
 is_obsolete: true
@@ -83234,7 +83517,7 @@
 id: GO:0008165
 name: obsolete pyrethroid resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "pyrethroid resistance" EXACT []
 is_obsolete: true
@@ -83244,7 +83527,7 @@
 id: GO:0008166
 name: obsolete viral replication
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it covers multiple processes and is now covered by more specific terms.
 synonym: "viral replication" EXACT []
 is_obsolete: true
@@ -83254,7 +83537,7 @@
 id: GO:0008167
 name: obsolete sigma virus replication
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 synonym: "sigma virus replication" EXACT []
 is_obsolete: true
 replaced_by: GO:0019079
@@ -83273,6 +83556,7 @@
 xref: Reactome:R-HSA-212269 "DNMT1,3A,3B:PRC2 methylates cytosine and histone H3"
 xref: Reactome:R-HSA-379387 "COMT transfers Met to DA to form 3MT"
 xref: Reactome:R-HSA-379464 "COMT transfers Met to DOPAC to form HVA"
+xref: Reactome:R-HSA-5333608 "GSSebGalNac is reduced and methylated to MeSebGalNac"
 xref: Reactome:R-HSA-6800149 "N6AMT1:TRMT112 transfers CH3 group from AdoMet to MMAIII"
 xref: Reactome:R-HSA-71286 "guanidinoacetate + S-adenosylmethionine => creatine + S-adenosylhomocysteine"
 xref: Reactome:R-HSA-9710490 "The GSDME gene promoter is hypermethylated"
@@ -83339,7 +83623,7 @@
 id: GO:0008175
 name: tRNA methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in a tRNA molecule." [GOC:mah]
+def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in a tRNA molecule. The methyl group can be transfered to the nucleobase or to the ribose group of the nucleoside." [GOC:mah]
 xref: Reactome:R-HSA-6782879 "TYW3 methylates yW-86 yielding yW-72 at nucleotide 37 of tRNA(Phe)"
 xref: Reactome:R-HSA-6782881 "LCMT2 methylates yW-72 yielding yW-58 at nucleotide 37 of tRNA(Phe)"
 xref: Reactome:R-HSA-6782890 "LCMT2 methoxycarbonylates yW-58 yielding yW (wybutosine) at nucleotide 37 of tRNA(Phe)"
@@ -83422,7 +83706,7 @@
 id: GO:0008181
 name: obsolete tumor suppressor
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not represent a true biological function, rather a pathology that occurs when a particular gene product is inactivated in some way.
 synonym: "tumor suppressor" EXACT []
 is_obsolete: true
@@ -83585,6 +83869,7 @@
 name: ferrous iron binding
 namespace: molecular_function
 def: "Binding to a ferrous iron ion, Fe(II)." [GOC:ai]
+xref: Reactome:R-HSA-1362416 "Frataxin binds iron"
 is_a: GO:0005506 ! iron ion binding
 
 [Term]
@@ -83656,7 +83941,7 @@
 id: GO:0008206
 name: bile acid metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving bile acids, a group of steroid carboxylic acids occurring in bile, where they are present as the sodium salts of their amides with glycine or taurine." [GOC:go_curators]
+def: "The chemical reactions and pathways involving bile acids, a group of steroid carboxylic acids occurring in bile, where they are present as the sodium salts of their amides with glycine or taurine." [GOC:curators]
 synonym: "bile acid metabolism" EXACT []
 xref: Reactome:R-HSA-194068 "Bile acid and bile salt metabolism"
 is_a: GO:0008202 ! steroid metabolic process
@@ -83728,7 +84013,6 @@
 namespace: biological_process
 def: "The addition of an alkyl group to a protein amino acid. Alkyl groups are derived from alkanes by removal of one hydrogen atom." [GOC:ma]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid alkylation" EXACT [GOC:bf]
 is_a: GO:0036211 ! protein modification process
 
@@ -83817,7 +84101,7 @@
 id: GO:0008224
 name: obsolete Gram-positive antibacterial peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-positive bacterial cells." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-positive bacterial cells." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "Gram-positive antibacterial peptide activity" EXACT []
 is_obsolete: true
@@ -83827,7 +84111,7 @@
 id: GO:0008225
 name: obsolete Gram-negative antibacterial peptide activity
 namespace: molecular_function
-def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-negative bacterial cells." [GOC:go_curators]
+def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-negative bacterial cells." [GOC:curators]
 comment: This term was made obsolete because it describes involvement in a process and not a function.
 synonym: "Gram-negative antibacterial peptide activity" EXACT []
 is_obsolete: true
@@ -83926,6 +84210,7 @@
 xref: Reactome:R-HSA-5684864 "NAPSA, CTSH, PGA3-5 cleave pro-SFTPB"
 xref: Reactome:R-HSA-5685902 "NAPSA, CTSH, PGA3-5 cleave pro-SFTPC"
 xref: Reactome:R-HSA-6803060 "DCD(63-110) is processed to DCD(63-109)"
+xref: Reactome:R-HSA-9935547 "Lysosomal degradation of ubiqutinated CDH1"
 is_a: GO:0016787 ! hydrolase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: skos:exactMatch EC:3.4.-.-
@@ -83939,7 +84224,7 @@
 def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "cysteine protease activity" NARROW []
 synonym: "thiol protease activity" NARROW []
-xref: Reactome:R-HSA-2022381 "Cathepsin Z (Cathepsin X) hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022381 "Cathepsin Z (Cathepsin X) hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-2467775 "Autocleavage of ESPL1 (Separase)"
 xref: Reactome:R-HSA-2467809 "ESPL1 (Separase) cleaves centromeric cohesin"
 xref: Reactome:R-HSA-5660752 "USP9X deubiquitinates Ub-SNCA"
@@ -83961,7 +84246,7 @@
 def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "serine protease activity" NARROW []
 xref: Reactome:R-HSA-1461993 "pro-HD5 is cleaved by trypsin"
-xref: Reactome:R-HSA-2022383 "Chymase hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022383 "Chymase hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 xref: Reactome:R-HSA-3132753 "Pancreatic proteases degrade TCN1:RCbl"
 xref: Reactome:R-HSA-6801766 "Trypsin cleaves REG3A or REG3G to generate REG3A,G(38-175)"
 xref: Reactome:R-HSA-9839072 "HTRA2 degrades APP (Amyloid-beta precursor protein)"
@@ -83979,10 +84264,10 @@
 xref: Reactome:R-HSA-157629 "NOTCH2-ligand complex is cleaved to produce NEXT2"
 xref: Reactome:R-HSA-157632 "Complex of NOTCH1 with its ligand is cleaved to produce NEXT1"
 xref: Reactome:R-HSA-193679 "alpha-secretase cleaves the p75NTR extracellular domain"
-xref: Reactome:R-HSA-2022368 "MME:Zn2+ (Neprilysin) hydrolyses AGT(34-42)"
-xref: Reactome:R-HSA-2022393 "ANPEP:Zn2+ hydrolyzes Angiotensin-(2-8) to Angiotensin-(3-8)"
-xref: Reactome:R-HSA-2022396 "MME:Zn2+ (Neprilysin) hydrolyses AGT(34-43)"
-xref: Reactome:R-HSA-2022399 "ENPEP:Zn2+ hydrolyzes Angiotensin-(1-8) to Angiotensin-(2-8)"
+xref: Reactome:R-HSA-2022368 "MME:Zn2+ (Neprilysin) hydrolyses AGT(25-33) (Angiotensin-(1-9)) to AGT(25-31) (Angiotensin-(1-7)"
+xref: Reactome:R-HSA-2022393 "ANPEP:Zn2+ hydrolyzes AGT(26-32) (Angiotensin-(2-8)) to AGT(27-32) (Angiotensin-(3-8))"
+xref: Reactome:R-HSA-2022396 "MME:Zn2+ (Neprilysin) hydrolyses AGT(25-34) (Angiotensin-(1-10)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2022399 "ENPEP:Zn2+ hydrolyzes AGT(25-32) (Angiotensin-(1-8)) to AGT(26-32) (Angiotensin-(2-8))"
 xref: Reactome:R-HSA-2220944 "ADAM10/17 cleaves ligand-bound NOTCH1 PEST domain mutants to produce NEXT1 PEST domain mutants"
 xref: Reactome:R-HSA-2220976 "NOTCH1 HD+PEST domain mutants are cleaved by ADAM10/17 irrespective of ligand binding"
 xref: Reactome:R-HSA-2666278 "NOTCH1 t(7;9)(NOTCH1:M1580_K2555) is cleaved to produce NEXT1"
@@ -84079,6 +84364,7 @@
 xref: EC:3.4.19.2
 xref: MetaCyc:ACYLAMINOACYL-PEPTIDASE-RXN
 xref: MetaCyc:PEPTIDYL-GLYCINAMIDASE-RXN
+xref: Reactome:R-HSA-5696365 "ASRGL1 hydrolyses aspartame to L-Asp, L-Phe, MeOH"
 is_a: GO:0008233 ! peptidase activity
 property_value: skos:exactMatch EC:3.4.19.-
 property_value: skos:narrowMatch EC:3.4.19.1
@@ -84123,7 +84409,7 @@
 id: GO:0008245
 name: obsolete lysosomal membrane hydrogen-transporting ATPase
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "lysosomal membrane hydrogen-transporting ATPase" EXACT []
 synonym: "V-ATPase" RELATED []
@@ -84181,7 +84467,7 @@
 id: GO:0008251
 name: tRNA-specific adenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a tRNA molecule." [GOC:mah]
+def: "Catalysis of the reaction: adenosine + H2O = inosine + NH4+, in a tRNA molecule." [GOC:mah]
 synonym: "tRNA-adenosine deaminase activity" EXACT []
 xref: Reactome:R-HSA-6782311 "ADAT2:ADAT3 (hetADAT) deaminates adenosine-34 in tRNAs"
 xref: Reactome:R-HSA-6782336 "ADAT1 deaminates adenosine-37 in tRNA(Ala)"
@@ -84410,7 +84696,7 @@
 id: GO:0008268
 name: obsolete receptor signaling protein tyrosine kinase signaling protein activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the meaning of the term is ambiguous and it incorporates process information.
 synonym: "receptor signaling protein tyrosine kinase signaling protein activity" EXACT []
 synonym: "receptor signalling protein tyrosine kinase signalling protein activity" EXACT []
@@ -84506,7 +84792,7 @@
 id: GO:0008277
 name: regulation of G protein-coupled receptor signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of G protein-coupled receptor signaling pathway." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of G protein-coupled receptor signaling pathway." [GOC:curators]
 synonym: "regulation of G protein coupled receptor protein signaling pathway" EXACT []
 synonym: "regulation of G protein coupled receptor protein signalling pathway" EXACT []
 synonym: "regulation of G-protein coupled receptor protein signaling pathway" EXACT []
@@ -84588,7 +84874,7 @@
 id: GO:0008284
 name: positive regulation of cell population proliferation
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of cell proliferation." [GOC:go_curators]
+def: "Any process that activates or increases the rate or extent of cell proliferation." [GOC:curators]
 synonym: "activation of cell proliferation" NARROW []
 synonym: "positive regulation of cell proliferation" RELATED []
 synonym: "stimulation of cell proliferation" NARROW []
@@ -84605,7 +84891,7 @@
 id: GO:0008285
 name: negative regulation of cell population proliferation
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of cell proliferation." [GOC:go_curators]
+def: "Any process that stops, prevents or reduces the rate or extent of cell proliferation." [GOC:curators]
 synonym: "down regulation of cell proliferation" EXACT []
 synonym: "down-regulation of cell proliferation" EXACT []
 synonym: "downregulation of cell proliferation" EXACT []
@@ -84671,7 +84957,7 @@
 id: GO:0008290
 name: F-actin capping protein complex
 namespace: cellular_component
-def: "A heterodimer consisting of alpha and beta subunits that binds to and caps the barbed ends of actin filaments, thereby regulating the polymerization of actin monomers but not severing actin filaments." [GOC:go_curators, ISBN:0198599560]
+def: "A heterodimer consisting of alpha and beta subunits that binds to and caps the barbed ends of actin filaments, thereby regulating the polymerization of actin monomers but not severing actin filaments." [GOC:curators, ISBN:0198599560]
 subset: goslim_pir
 is_a: GO:0032991 ! protein-containing complex
 relationship: part_of GO:0015629 ! actin cytoskeleton
@@ -84700,7 +84986,7 @@
 id: GO:0008293
 name: torso signaling pathway
 namespace: biological_process
-def: "The series of molecular signals initiated by an extracellular ligand binding to torso (a receptor tyrosine kinase) on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:go_curators, PMID:8343949]
+def: "The series of molecular signals initiated by an extracellular ligand binding to torso (a receptor tyrosine kinase) on the surface of a target cell, and ending with the regulation of a downstream cellular process, e.g. transcription." [GOC:curators, PMID:8343949]
 synonym: "torso signalling pathway" EXACT []
 is_a: GO:0007169 ! cell surface receptor protein tyrosine kinase signaling pathway
 
@@ -84720,7 +85006,7 @@
 id: GO:0008295
 name: spermidine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of spermidine, N-(3-aminopropyl)-1,4-diaminobutane." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of spermidine, N-(3-aminopropyl)-1,4-diaminobutane." [GOC:curators, ISBN:0198506732]
 synonym: "spermidine anabolism" EXACT []
 synonym: "spermidine biosynthesis" EXACT []
 synonym: "spermidine formation" EXACT []
@@ -84842,7 +85128,7 @@
 id: GO:0008304
 name: obsolete eukaryotic translation initiation factor 4 complex
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "eIF-4" EXACT []
 synonym: "eukaryotic translation initiation factor 4 complex" EXACT []
@@ -85015,12 +85301,13 @@
 
 [Term]
 id: GO:0008320
-name: protein transmembrane transporter activity
+name: transmembrane protein transporter activity
 namespace: molecular_function
 alt_id: GO:0015266
 alt_id: GO:0015463
 def: "Enables the transfer of a protein from one side of a membrane to the other." [GOC:jl]
 synonym: "protein channel activity" RELATED []
+synonym: "protein transmembrane transporter activity" EXACT []
 xref: Reactome:R-HSA-1268022 "TOMM40 complex translocates proteins from the cytosol to the mitochondrial intermembrane space"
 xref: Reactome:R-HSA-1299475 "TIMM23 PAM translocates proteins from the mitochondrial intermembrane space to the mitochondrial matrix"
 xref: Reactome:R-HSA-184269 "Monoubiquitinated N-myristoyl GAG polyprotein is targeted to the late endosomal vesicle membrane by the ESCRT-I complex"
@@ -85041,6 +85328,7 @@
 xref: Reactome:R-HSA-9636375 "SecA2 transports SapM from cytoplasm to cytosol"
 xref: Reactome:R-HSA-9698930 "HCMV C Nucleocapsid Translocation"
 xref: Reactome:R-HSA-9698933 "HCMV B Nucleocapsid Translocation"
+xref: Reactome:R-HSA-9760705 "GspD2:GspC2:GspS2 transports LT from periplasmic to extracellular space"
 is_a: GO:0022884 ! macromolecule transmembrane transporter activity
 is_a: GO:0140318 ! protein transporter activity
 relationship: part_of GO:0071806 ! protein transmembrane transport
@@ -85187,7 +85475,7 @@
 id: GO:0008341
 name: obsolete response to cocaine (sensu Insecta)
 namespace: biological_process
-def: "OBSOLETE. A change in state or activity of a mature individual as the result of a cocaine stimulus. As in, but not restricted to, the true insects (Insecta, ncbi_taxonomy_id:50557)." [GOC:go_curators, GOC:jid]
+def: "OBSOLETE. A change in state or activity of a mature individual as the result of a cocaine stimulus. As in, but not restricted to, the true insects (Insecta, ncbi_taxonomy_id:50557)." [GOC:curators, GOC:jid]
 comment: This term was made obsolete because there is no clear difference between the sensu Insecta term and the generic term.
 synonym: "response to cocaine (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -85237,7 +85525,7 @@
 id: GO:0008346
 name: larval walking behavior
 namespace: biological_process
-def: "The behavior of a larval organism relating to the progression of that organism along the ground by the process of lifting and setting down each leg." [GOC:go_curators, GOC:pr]
+def: "The behavior of a larval organism relating to the progression of that organism along the ground by the process of lifting and setting down each leg." [GOC:curators, GOC:pr]
 synonym: "larval walking behaviour" EXACT []
 is_a: GO:0008345 ! larval locomotory behavior
 is_a: GO:0090659 ! walking behavior
@@ -85258,7 +85546,7 @@
 namespace: biological_process
 alt_id: GO:0042154
 alt_id: GO:0042155
-def: "Any process that stops, prevents, or reduces the frequency, rate, or extent of an antimicrobial humoral response." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate, or extent of an antimicrobial humoral response." [GOC:curators]
 synonym: "attenuation of antimicrobial humoral response" EXACT []
 synonym: "down regulation of antimicrobial humoral response" EXACT []
 synonym: "down-regulation of antimicrobial humoral response" EXACT []
@@ -85393,7 +85681,7 @@
 namespace: biological_process
 alt_id: GO:0045788
 alt_id: GO:0045789
-def: "Any process that modulates the surface configuration of a cell." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the surface configuration of a cell." [GOC:curators, GOC:dph, GOC:tb]
 is_a: GO:0022604 ! regulation of cell morphogenesis
 is_a: GO:0065008 ! regulation of biological quality
 
@@ -85401,7 +85689,7 @@
 id: GO:0008361
 name: regulation of cell size
 namespace: biological_process
-def: "Any process that modulates the size of a cell." [GOC:go_curators]
+def: "Any process that modulates the size of a cell." [GOC:curators]
 synonym: "cell size control" EXACT []
 is_a: GO:0032535 ! regulation of cellular component size
 
@@ -85469,7 +85757,7 @@
 id: GO:0008369
 name: obsolete molecular function
 namespace: molecular_function
-def: "OBSOLETE. These are terms that have been removed from the active function ontology." [GOC:go_curators]
+def: "OBSOLETE. These are terms that have been removed from the active function ontology." [GOC:curators]
 comment: This term was made obsolete because it is unnecessary in the OBO flat file format.
 is_obsolete: true
 
@@ -85477,7 +85765,7 @@
 id: GO:0008370
 name: obsolete cellular component
 namespace: cellular_component
-def: "OBSOLETE. These are terms that have been removed from the active component ontology." [GOC:go_curators]
+def: "OBSOLETE. These are terms that have been removed from the active component ontology." [GOC:curators]
 comment: This term was made obsolete because it is unnecessary in the OBO flat file format.
 is_obsolete: true
 
@@ -85485,7 +85773,7 @@
 id: GO:0008371
 name: obsolete biological process
 namespace: biological_process
-def: "OBSOLETE. These are terms that have been removed from the active process ontology." [GOC:go_curators]
+def: "OBSOLETE. These are terms that have been removed from the active process ontology." [GOC:curators]
 comment: This term was made obsolete because it is unnecessary in the OBO flat file format.
 is_obsolete: true
 
@@ -85528,8 +85816,8 @@
 namespace: molecular_function
 def: "Catalysis of the transfer of an N-acetylglucosaminyl residue from UDP-N-acetyl-glucosamine to a sugar." [ISBN:0198506732]
 synonym: "GlcNAc transferase activity" EXACT []
+xref: Reactome:R-HSA-5617037 "POMGNT1 transfers GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 xref: Reactome:R-HSA-5694487 "A4GNT transfers GlcNAc to core 2 mucins"
-xref: Reactome:R-HSA-8879117 "POMGNT2 transfers GlcNAc to Man-DAG1"
 xref: Reactome:R-HSA-9683648 "Spike trimer glycoside chains are extended"
 xref: Reactome:R-HSA-9694656 "Spike trimer glycoside chains are extended"
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
@@ -85573,7 +85861,7 @@
 name: thioredoxin peroxidase activity
 namespace: molecular_function
 alt_id: GO:0009031
-def: "Catalysis of the reaction: [thioredoxin]-dithiol + hydrogen peroxide = [thioredoxin]-disulfide + H2O." [RHEA:63528]
+def: "Catalysis of the reaction: [thioredoxin]-dithiol + H2O2 = [thioredoxin]-disulfide + H2O." [RHEA:63528]
 synonym: "thiol peroxidase activity" EXACT []
 synonym: "TPx activity" EXACT []
 synonym: "TrxPx activity" EXACT []
@@ -85826,6 +86114,9 @@
 xref: KEGG_REACTION:R07204
 xref: MetaCyc:1.14.13.95-RXN
 xref: MetaCyc:1.14.13.96-RXN
+xref: Reactome:R-HSA-192157 "CYP8B1 12-hydroxylates 4CHOL7aOLONE"
+xref: Reactome:R-HSA-193709 "CYP8B1 12-hydroxylates 4CHOL7a,24(S)DIOL"
+xref: Reactome:R-HSA-193845 "CYP8B1 12-hydroxylates 4CHOL7a,27DONE"
 xref: RHEA:15261
 xref: RHEA:46752
 xref: RHEA:65700
@@ -85857,6 +86148,7 @@
 synonym: "sterol 14alpha-demethylase activity" RELATED []
 xref: EC:1.14.14.154
 xref: MetaCyc:1.14.13.70-RXN
+xref: Reactome:R-HSA-9947148 "CYP51A1 14-demethylates 24,25-dhLAN to 4,4-diMeCholesta-8,14-dien-3-ol"
 xref: RHEA:14917
 xref: RHEA:25286
 xref: RHEA:54028
@@ -85949,7 +86241,7 @@
 id: GO:0008407
 name: chaeta morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the chaeta are generated and organized. A chaeta is a sensory multicellular cuticular outgrowth of a specifically differentiated cell." [FBbt:00005177, GOC:bf, GOC:cjm, GOC:dos, GOC:go_curators]
+def: "The process in which the anatomical structures of the chaeta are generated and organized. A chaeta is a sensory multicellular cuticular outgrowth of a specifically differentiated cell." [FBbt:00005177, GOC:bf, GOC:cjm, GOC:curators, GOC:dos]
 synonym: "bristle morphogenesis" NARROW [GOC:bf, GOC:dos]
 is_a: GO:0009887 ! animal organ morphogenesis
 relationship: part_of GO:0022416 ! chaeta development
@@ -86007,6 +86299,7 @@
 synonym: "PHB polyprenyl diphosphate transferase activity" EXACT []
 xref: EC:2.5.1.39
 xref: MetaCyc:RXN-11368
+xref: Reactome:R-HSA-2162192 "COQ2 ligates all-E-10PrP2 to PHB"
 xref: RHEA:44504
 is_a: GO:0004659 ! prenyltransferase activity
 property_value: skos:exactMatch EC:2.5.1.39
@@ -86048,6 +86341,8 @@
 def: "Catalysis of the transfer of a fucosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
 xref: Reactome:R-HSA-9033949 "FUT1 transfers Fuc to Type 2 chains to form H antigen-RBC"
 xref: Reactome:R-HSA-9846332 "FUT1,2 transfer fucose to gangliosides"
+xref: Reactome:R-HSA-9935449 "FUT10 O-fucosylates EMI domain-containing proteins"
+xref: Reactome:R-HSA-9935469 "FUT11 O-fucosylates EMI domain-containing proteins"
 is_a: GO:0016758 ! hexosyltransferase activity
 
 [Term]
@@ -86239,6 +86534,7 @@
 name: phosphatidylethanolamine binding
 namespace: molecular_function
 def: "Binding to a phosphatidylethanolamine, a class of glycerophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of ethanolamine." [ISBN:0198506732]
+xref: Reactome:R-HSA-9842115 "PSAP(195-273) dimer binds PE"
 is_a: GO:0005543 ! phospholipid binding
 
 [Term]
@@ -86519,7 +86815,7 @@
 synonym: "O,N-disulfate O-sulfohydrolase activity" RELATED [EC:3.1.6.14]
 xref: EC:3.1.6.14
 xref: MetaCyc:3.1.6.14-RXN
-xref: Reactome:R-HSA-1638032 "N-acetylglucosamine 6-sulfatase (GNS) hydrolyses 6-sulfate groups of the N-acetyl-D-glucosamine 6-sulfate units of keratan sulfate"
+xref: Reactome:R-HSA-1638032 "GNS:Ca2+ cleaves sulfate from GlcNAc-6-sulfate units of keratan sulfate"
 xref: Reactome:R-HSA-2263495 "Defective GNS does not hydrolyse 6-sulfate from GlcNAc6S"
 is_a: GO:0008484 ! sulfuric ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.6.14
@@ -86566,12 +86862,13 @@
 
 [Term]
 id: GO:0008453
-name: alanine-glyoxylate transaminase activity
+name: L-alanine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + glyoxylate = pyruvate + glycine." [EC:2.6.1.44]
+def: "Catalysis of the reaction: L-alanine + glyoxylate = pyruvate + glycine." [RHEA:24248]
 synonym: "AGT activity" RELATED [EC:2.6.1.44]
 synonym: "alanine--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.44]
 synonym: "alanine-glyoxylate aminotransferase activity" EXACT []
+synonym: "alanine-glyoxylate transaminase activity" EXACT []
 synonym: "alanine-glyoxylic aminotransferase activity" RELATED [EC:2.6.1.44]
 synonym: "L-alanine-glycine transaminase activity" RELATED [EC:2.6.1.44]
 synonym: "L-alanine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.44]
@@ -86580,10 +86877,11 @@
 xref: Reactome:R-HSA-389684 "glyoxylate + alanine => glycine + pyruvate [peroxisome]"
 xref: Reactome:R-HSA-904864 "Mitochondrial AGXT2 tetramer transaminates glyoxylate and alanine to glycine and pyruvate"
 xref: RHEA:24248
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.44
 property_value: skos:exactMatch RHEA:24248
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008454
@@ -86844,7 +87142,7 @@
 xref: MetaCyc:2.8.2.29-RXN
 xref: MetaCyc:2.8.2.30-RXN
 xref: Reactome:R-HSA-2076383 "HS3ST1 sulfates GlcN at C3 in heparan sulfate"
-xref: Reactome:R-HSA-2076611 "HS3STs sulfate GlcN at C3 in heparan sulfate"
+xref: Reactome:R-HSA-2076611 "HS3ST2-6 sulfate GlcN at C3 in heparan sulfate"
 xref: RHEA:15461
 is_a: GO:0034483 ! heparan sulfate sulfotransferase activity
 property_value: skos:exactMatch RHEA:15461
@@ -87080,7 +87378,7 @@
 synonym: "tRNA-guanosine(34) queuine transglycosylase" EXACT [EC:2.4.2.64]
 xref: EC:2.4.2.64
 xref: MetaCyc:QUEUOSINE-TRNA-RIBOSYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-6782443 "QTRT1:QTRTD1 exchange guanine for queuosine at guanosine-34 of tRNA(Tyr)"
+xref: Reactome:R-HSA-6782443 "QTRT1:QTRTD1 exchanges guanine for queuine at guanosine-34 of tRNA(Asn),tRNA(Asp),tRNA(His),tRNA(Tyr) yielding queuosine-34 tRNA(Asn),tRNA(Asp),tRNA(His),tRNA(Tyr)"
 xref: RHEA:16633
 is_a: GO:0016763 ! pentosyltransferase activity
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -87122,6 +87420,8 @@
 synonym: "sphingosine kinase (phosphorylating)" RELATED [EC:2.7.1.91]
 xref: EC:2.7.1.91
 xref: MetaCyc:SPHINGANINE-KINASE-RXN
+xref: Reactome:R-HSA-5218845 "p-SPHK1 phosphorylates sphingosine to sphingosine 1-phosphate"
+xref: Reactome:R-HSA-9625814 "SPHK1 phosphorylates sphingosine in response to E2 stimulation"
 xref: RHEA:51496
 is_a: GO:0001727 ! lipid kinase activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
@@ -87134,7 +87434,6 @@
 name: sulfite oxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: H2O + O2 + sulfite = H2O2 + H+ + sulfate." [RHEA:24600]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "sulfite:oxygen oxidoreductase activity" RELATED [EC:1.8.3.1]
 synonym: "sulphite oxidase activity" EXACT []
 xref: EC:1.8.3.1
@@ -87147,8 +87446,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00533
 property_value: skos:exactMatch MetaCyc:SULFITE-OXIDASE-RXN
 property_value: skos:exactMatch RHEA:24600
-property_value: skos:relatedMatch MetaCyc:PWY-5326
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0008483
@@ -87341,7 +87640,6 @@
 synonym: "UDP-galactose:beta-N-acetylglucosamine beta-1,3-galactosyltransferase activity" EXACT []
 xref: EC:2.4.1.86
 xref: MetaCyc:2.4.1.86-RXN
-xref: Reactome:R-HSA-914010 "Addition of GlcNAc to the Tn antigen forms a Core 3 glycoprotein"
 xref: Reactome:R-HSA-9603989 "B3GALTs transfer Gal to GlcNAc-1,3-Gal-R to form Type 1 chain"
 xref: Reactome:R-HSA-9845516 "B3GALT4 transfers Gal to gangliosides"
 xref: RHEA:16045
@@ -87387,10 +87685,9 @@
 alt_id: GO:0015201
 def: "Enables the transfer of monoamines, organic compounds that contain one amino group that is connected to an aromatic ring by an ethylene group (-CH2-CH2-), from one side of a membrane to the other." [GOC:mah]
 xref: Reactome:R-HSA-372542 "Loading of dopamine into synaptic veiscles"
-xref: Reactome:R-HSA-379393 "SLC6A3 cotransports DA, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-379393 "SLC6A3 cotransports DA, Na+, Cl- from extracellular region to cytosol"
 xref: Reactome:R-HSA-380586 "loading of Serotonin in synaptic vesicles"
 xref: Reactome:R-HSA-380620 "Reuptake of serotonin from the synapse"
-xref: Reactome:R-HSA-444160 "VMAT1/2 can mediate the transport of biogenic amines"
 xref: Reactome:R-HSA-5660706 "Defective SLC6A3 does not cotransport DA, Na+ from extracellular region to cytosol"
 is_a: GO:0022804 ! active transmembrane transporter activity
 relationship: part_of GO:0015844 ! monoamine transport
@@ -87504,7 +87801,6 @@
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
 synonym: "organic cation porter activity" RELATED []
 synonym: "polyspecific organic cation transmembrane transporter activity" NARROW []
-xref: Reactome:R-HSA-549241 "SLC22A4 cotransports ERGT, Na+ from extracellular region to cytosol"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0015291
@@ -87515,9 +87811,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other. Organic anions are atoms or small molecules with a negative charge which contain carbon in covalent linkage." [GOC:ai]
 comment: he reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-2142859 "Growing HA is extruded from the cell by ABCC5"
-xref: Reactome:R-HSA-561041 "OAT1-3 transport organic anions with antiport of dicarboxylic acids"
-xref: Reactome:R-HSA-9794830 "SLC22A8 transports Cipro into renal cell"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -87551,11 +87844,9 @@
 synonym: "folic acid transporter activity" RELATED []
 synonym: "vitamin B9 transporter activity" EXACT []
 synonym: "vitamin M transporter activity" EXACT []
-xref: Reactome:R-HSA-200646 "Cytosolic folate export across the plasma membrane"
 xref: Reactome:R-HSA-200652 "SLC19A1 transports 5-methyl-THF from extracellular region to cytosol"
 xref: Reactome:R-HSA-200680 "SLC25A32 transports THF from cytosol to mitochondrial matrix"
 xref: Reactome:R-HSA-200720 "SLC25A32 transports THF from mitochondrial matrix to cytosol"
-xref: Reactome:R-HSA-200729 "SLC46A1 transports FOLA from extracellular region to cytosol"
 is_a: GO:0046943 ! carboxylic acid transmembrane transporter activity
 is_a: GO:0072349 ! modified amino acid transmembrane transporter activity
 is_a: GO:0090482 ! vitamin transmembrane transporter activity
@@ -87646,11 +87937,15 @@
 
 [Term]
 id: GO:0008525
-name: phosphatidylcholine transporter activity
+name: phosphatidylcholine intramembrane carrier activity
 namespace: molecular_function
-def: "Enables the directed movement of phosphatidylcholine into, out of or within a cell, or between cells. Phosphatidylcholine refers to a class of glycerophospholipids in which the phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:ai, ISBN:0198506732]
+def: "Enables the transport of phosphatidylcholine from a region of a membrane to a different region on the same membrane." [GOC:ai, ISBN:0198506732]
+synonym: "phosphatidylcholine intramembrane transporter activity" BROAD []
 synonym: "phosphatidylcholine transmembrane transporter activity" NARROW [GOC:bf, GOC:vw]
-is_a: GO:0005548 ! phospholipid transporter activity
+synonym: "phosphatidylcholine transporter activity" BROAD []
+is_a: GO:0140303 ! intramembrane lipid carrier activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/10882" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0008526
@@ -87753,6 +88048,7 @@
 xref: MetaCyc:2.4.1.149-RXN
 xref: MetaCyc:2.4.1.163-RXN
 xref: Reactome:R-HSA-2025724 "B3GNT1,2,3,4,7 add GlcNAc to form Keratan-PG"
+xref: Reactome:R-HSA-9931297 "B3GNT3 mediated CD274 glycosylation in Golgi"
 xref: RHEA:14389
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.149
@@ -87889,7 +88185,7 @@
 id: GO:0008544
 name: epidermis development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the epidermis over time, from its formation to the mature structure. The epidermis is the outer epithelial layer of an animal, it may be a single layer that produces an extracellular material (e.g. the cuticle of arthropods) or a complex stratified squamous epithelium, as in the case of many vertebrate species." [GOC:go_curators, UBERON:0001003]
+def: "The process whose specific outcome is the progression of the epidermis over time, from its formation to the mature structure. The epidermis is the outer epithelial layer of an animal, it may be a single layer that produces an extracellular material (e.g. the cuticle of arthropods) or a complex stratified squamous epithelium, as in the case of many vertebrate species." [GOC:curators, UBERON:0001003]
 synonym: "hypodermis development" RELATED [GOC:kmv, GOC:rk]
 is_a: GO:0009888 ! tissue development
 
@@ -88129,6 +88425,7 @@
 xref: EC:7.6.2.2
 xref: MetaCyc:3.6.3.44-RXN
 xref: Reactome:R-HSA-9757139 "ABCB1,ABCC2 transport ATVs from cytosol to extracellular region"
+xref: Reactome:R-HSA-9911514 "MacA:MacB:TolC transports azithromycin from the cytosol to the periplasm"
 xref: RHEA:33147
 is_a: GO:0042910 ! xenobiotic transmembrane transporter activity
 is_a: GO:0140359 ! ABC-type transporter activity
@@ -88155,6 +88452,8 @@
 synonym: "ATPase-coupled protein transporter activity" EXACT []
 xref: EC:7.4.2.8
 xref: MetaCyc:3.6.3.50-RXN
+xref: Reactome:R-HSA-9760310 "SecA:SecYEG transports proteins from cytosol to periplasmic space"
+xref: Reactome:R-HSA-9926041 "HlyB:HlyD:TolC transports HlyA from cytosol to periplasm"
 is_a: GO:0015450 ! protein-transporting ATPase activity
 property_value: skos:exactMatch EC:7.4.2.8
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30815" xsd:anyURI
@@ -88356,7 +88655,7 @@
 id: GO:0008582
 name: regulation of synaptic assembly at neuromuscular junction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of synaptic assembly at neuromuscular junctions." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of synaptic assembly at neuromuscular junctions." [GOC:curators]
 synonym: "regulation of synaptic growth at neuromuscular junction" RELATED []
 is_a: GO:0048638 ! regulation of developmental growth
 is_a: GO:0051963 ! regulation of synapse assembly
@@ -88420,7 +88719,7 @@
 namespace: biological_process
 alt_id: GO:0007226
 alt_id: GO:0043109
-def: "Any process that modulates the frequency, rate or extent of smoothened signaling." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of smoothened signaling." [GOC:curators]
 synonym: "regulation of hedgehog signaling pathway" EXACT [GOC:bf, GOC:ecd]
 synonym: "regulation of hh signaling pathway" EXACT [GOC:bf, GOC:ecd]
 synonym: "regulation of smoothened activity" RELATED []
@@ -88436,7 +88735,7 @@
 id: GO:0008591
 name: regulation of Wnt signaling pathway, calcium modulating pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "regulation of frizzled-2 signaling pathway" EXACT []
 synonym: "regulation of frizzled-2 signalling pathway" EXACT []
 synonym: "regulation of Wnt receptor signaling pathway, calcium modulating pathway" EXACT []
@@ -88450,7 +88749,7 @@
 id: GO:0008592
 name: regulation of Toll signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the Tl signaling pathway." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the Tl signaling pathway." [GOC:curators]
 synonym: "regulation of Tl signaling pathway" EXACT []
 synonym: "regulation of Tl signalling pathway" EXACT []
 synonym: "regulation of Toll signalling pathway" EXACT []
@@ -88463,7 +88762,7 @@
 id: GO:0008593
 name: regulation of Notch signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the Notch signaling pathway." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the Notch signaling pathway." [GOC:curators]
 synonym: "regulation of N signaling pathway" EXACT []
 synonym: "regulation of N signalling pathway" EXACT []
 synonym: "regulation of Notch signalling pathway" EXACT []
@@ -88587,7 +88886,7 @@
 id: GO:0008610
 name: lipid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent." [GOC:curators]
 synonym: "lipid anabolism" EXACT []
 synonym: "lipid biosynthesis" EXACT []
 synonym: "lipid formation" EXACT []
@@ -88629,11 +88928,13 @@
 synonym: "hypusine synthesis from peptidyl-lysine" EXACT []
 synonym: "hypusinylation" EXACT []
 synonym: "protein hypusination" EXACT []
+xref: MetaCyc:PWY-5905
 xref: Reactome:R-HSA-204626 "Hypusine synthesis from eIF5A-lysine"
 is_a: GO:0018205 ! peptidyl-lysine modification
 is_a: GO:0051604 ! protein maturation
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0008613
@@ -88772,7 +89073,7 @@
 id: GO:0008630
 name: intrinsic apoptotic signaling pathway in response to DNA damage
 namespace: biological_process
-def: "The series of molecular signals in which an intracellular signal is conveyed to trigger the apoptotic death of a cell. The pathway is induced by the detection of DNA damage, and ends when the execution phase of apoptosis is triggered." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "The series of molecular signals in which an intracellular signal is conveyed to trigger the apoptotic death of a cell. The pathway is induced by the detection of DNA damage, and ends when the execution phase of apoptosis is triggered." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "DNA damage response, signal transduction resulting in induction of apoptosis" RELATED []
 is_a: GO:0006974 ! DNA damage response
 is_a: GO:0097193 ! intrinsic apoptotic signaling pathway
@@ -88800,7 +89101,7 @@
 id: GO:0008634
 name: obsolete negative regulation of survival gene product expression
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:dph, GOC:go_curators, GOC:mtg_apoptosis, GOC:tb]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:curators, GOC:dph, GOC:mtg_apoptosis, GOC:tb]
 comment: This term was made obsolete because the meaning of the term is ambiguous: the same gene product may or may not have a role in apoptosis depending on the cell type, tissue type, condition, etc.
 synonym: "down regulation of survival gene product activity" BROAD []
 synonym: "down-regulation of survival gene product activity" BROAD []
@@ -88819,7 +89120,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "activation of caspase activity by cytochrome c" NARROW []
 synonym: "caspase activation via cytochrome c" EXACT [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-111459 "Activation of caspases through apoptosome-mediated cleavage"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -88915,7 +89215,8 @@
 id: GO:0008649
 name: rRNA methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an rRNA molecule." [GOC:mah]
+def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an rRNA molecule. The methyl group can be transfered to the nucleobase or to the ribose group of the nucleoside." [GOC:mah]
+xref: Reactome:R-HSA-9911509 "RMTases methylate 16S ribosomal RNA"
 is_a: GO:0008173 ! RNA methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
 is_a: GO:0140102 ! catalytic activity, acting on a rRNA
@@ -88923,9 +89224,10 @@
 
 [Term]
 id: GO:0008650
-name: rRNA (uridine-2'-O-)-methyltransferase activity
+name: rRNA (uridine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a uridine in rRNA + S-adenosyl-L-methionine = a 2'-O-methyluridine in rRNA + S-adenosyl-L-homocysteine + H+." [RHEA:54152]
+synonym: "rRNA (uridine-2'-O-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6793127 "MRM2 (FTSJ2) methylates uridine-1369 of 16S rRNA yielding 2'-O-methyluridine"
 xref: RHEA:54152
 is_a: GO:0016436 ! rRNA (uridine) methyltransferase activity
@@ -90050,7 +90352,7 @@
 id: GO:0008718
 name: D-amino-acid dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a D-amino acid + H2O + acceptor = a 2-oxo acid + NH3 + reduced acceptor." [RHEA:18125]
+def: "Catalysis of the reaction: a D-alpha-amino acid + a quinone + H2O = a 2-oxocarboxylate + a quinol + NH4+." [RHEA:18125]
 synonym: "D-amino-acid:(acceptor) oxidoreductase (deaminating)" EXACT []
 synonym: "D-amino-acid:acceptor oxidoreductase (deaminating)" EXACT []
 xref: EC:1.4.5.1
@@ -90100,7 +90402,7 @@
 id: GO:0008721
 name: D-serine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-serine = pyruvate + NH3." [EC:4.3.1.18]
+def: "Catalysis of the reaction: D-serine = pyruvate + NH4+." [RHEA:13977]
 synonym: "D-hydroxy amino acid dehydratase activity" BROAD [EC:4.3.1.18]
 synonym: "D-hydroxyaminoacid dehydratase activity" RELATED [EC:4.3.1.18]
 synonym: "D-serine ammonia-lyase (pyruvate-forming)" RELATED [EC:4.3.1.18]
@@ -90413,7 +90715,6 @@
 xref: EC:1.1.1.103
 xref: KEGG_REACTION:R01465
 xref: MetaCyc:THREODEHYD-RXN
-xref: Reactome:R-HSA-6798667 "TDH tetramer oxidises L-Thr to 2A-3OB"
 xref: RHEA:13161
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.103
@@ -90613,7 +90914,6 @@
 xref: MetaCyc:O-SUCCINYLBENZOATE-COA-LIG-RXN
 xref: RHEA:17009
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.26
 property_value: skos:exactMatch RHEA:17009
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -91231,12 +91531,13 @@
 
 [Term]
 id: GO:0008791
-name: arginine N-succinyltransferase activity
+name: L-arginine N-succinyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: succinyl-CoA + L-arginine = CoA + N2-succinyl-L-arginine." [EC:2.3.1.109]
+def: "Catalysis of the reaction: succinyl-CoA + L-arginine = N2-succinyl-L-arginine + CoA + H+." [RHEA:15185]
 synonym: "AOST activity" RELATED [EC:2.3.1.109]
 synonym: "arginine and ornithine N(2)-succinyltransferase activity" RELATED [EC:2.3.1.109]
 synonym: "arginine and ornithine N2-succinyltransferase activity" RELATED [EC:2.3.1.109]
+synonym: "arginine N-succinyltransferase activity" BROAD []
 synonym: "arginine succinyltransferase activity" RELATED [EC:2.3.1.109]
 synonym: "AST activity" RELATED [EC:2.3.1.109]
 synonym: "AstA" RELATED [EC:2.3.1.109]
@@ -91246,9 +91547,11 @@
 xref: MetaCyc:ARGININE-N-SUCCINYLTRANSFERASE-RXN
 xref: RHEA:15185
 is_a: GO:0016749 ! N-succinyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.109
 property_value: skos:exactMatch RHEA:15185
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0008792
@@ -91270,19 +91573,22 @@
 
 [Term]
 id: GO:0008793
-name: aromatic-amino-acid transaminase activity
+name: aromatic-amino-acid:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an L-alpha aromatic amino acid + 2-oxoglutarate = an aromatic oxo acid + L-glutamate." [RHEA:17533]
 synonym: "ArAT" RELATED [EC:2.6.1.57]
 synonym: "aromatic amino acid aminotransferase activity" EXACT [EC:2.6.1.57]
 synonym: "aromatic amino acid transferase activity" EXACT []
 synonym: "aromatic aminotransferase activity" EXACT []
+synonym: "aromatic-amino-acid transaminase activity" BROAD []
 synonym: "aromatic-amino-acid:2-oxoglutarate aminotransferase activity" EXACT []
+xref: EC:2.6.1.57
 xref: MetaCyc:2.6.1.57-RXN
 xref: RHEA:17533
-is_a: GO:0008483 ! transaminase activity
-property_value: skos:broadMatch EC:2.6.1.57
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: skos:exactMatch EC:2.6.1.57
 property_value: skos:exactMatch RHEA:17533
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0008794
@@ -91305,7 +91611,7 @@
 namespace: molecular_function
 alt_id: GO:0008749
 alt_id: GO:0016965
-def: "Catalysis of the reaction: ATP + deamido-NAD+ + NH3 = AMP + diphosphate + NAD+." [EC:6.3.1.5]
+def: "Catalysis of the reaction: deamido-NAD+ + NH4+ + ATP = AMP + diphosphate + NAD+ + H+." [RHEA:21188]
 synonym: "deamido-NAD+:ammonia ligase (AMP-forming)" RELATED [EC:6.3.1.5]
 synonym: "diphosphopyridine nucleotide synthetase activity" RELATED [EC:6.3.1.5]
 synonym: "NAD synthase (AMP-forming)" EXACT []
@@ -91334,7 +91640,7 @@
 id: GO:0008797
 name: aspartate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-aspartate = fumarate + NH3." [EC:4.3.1.1]
+def: "Catalysis of the reaction: L-aspartate = fumarate + NH4+." [RHEA:16601]
 synonym: "aspartase activity" RELATED [EC:4.3.1.1]
 synonym: "fumaric aminase activity" RELATED [EC:4.3.1.1]
 synonym: "L-aspartase activity" RELATED [EC:4.3.1.1]
@@ -91358,7 +91664,6 @@
 xref: EC:3.4.19.5
 xref: MetaCyc:3.4.19.5-RXN
 xref: Reactome:R-HSA-5692495 "BACE1 cleaves APP(18-770) to APP(18-671) and APP(672-770)"
-xref: Reactome:R-HSA-5696365 "ASRGL1 hydrolyses aspartame to L-Asp, L-Phe"
 is_a: GO:0008242 ! omega peptidase activity
 is_a: GO:0070003 ! threonine-type peptidase activity
 property_value: skos:exactMatch EC:3.4.19.5
@@ -91382,6 +91687,11 @@
 synonym: "penicillinase I, II" RELATED [EC:3.5.2.6]
 xref: EC:3.5.2.6
 xref: MetaCyc:BETA-LACTAMASE-RXN
+xref: Reactome:R-HSA-9911513 "BlaSHV-12 hydrolyzes ceftriaxone"
+xref: Reactome:R-HSA-9911515 "KPC-2 hydrolyzes cephalothin"
+xref: Reactome:R-HSA-9911518 "OXA-9 hydrolyzes penicillin G"
+xref: Reactome:R-HSA-9911519 "BlaSHV-12 hydrolyzes cephalothin"
+xref: Reactome:R-HSA-9911520 "TEM-1 hydrolyzes penicillin G"
 xref: RHEA:20401
 is_a: GO:0016812 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides
 property_value: skos:exactMatch EC:3.5.2.6
@@ -91456,7 +91766,7 @@
 id: GO:0008804
 name: carbamate kinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + NH3 + CO2 = ADP + carbamoyl phosphate." [EC:2.7.2.2]
+def: "Catalysis of the reaction: hydrogencarbonate + NH4+ + ATP = carbamoyl phosphate + ADP + H2O + H+." [RHEA:10152]
 synonym: "ATP:carbamate phosphotransferase activity" RELATED [EC:2.7.2.2]
 synonym: "carbamoyl phosphokinase activity" RELATED [EC:2.7.2.2]
 synonym: "carbamyl phosphokinase activity" RELATED [EC:2.7.2.2]
@@ -91913,7 +92223,7 @@
 id: GO:0008829
 name: dCTP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dCTP + H2O = dUTP + NH3." [EC:3.5.4.13]
+def: "Catalysis of the reaction: dCTP + H2O + H+ = dUTP + NH4+." [RHEA:22680]
 synonym: "5-methyl-dCTP deaminase activity" RELATED [EC:3.5.4.13]
 synonym: "dCTP aminohydrolase activity" RELATED [EC:3.5.4.13]
 synonym: "deoxycytidine triphosphate deaminase activity" RELATED [EC:3.5.4.13]
@@ -92097,7 +92407,7 @@
 id: GO:0008838
 name: diaminopropionate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,3-diaminopropionate + H2O = pyruvate + 2 NH3." [EC:4.3.1.15]
+def: "Catalysis of the reaction: 2,3-diaminopropanoate + H2O + H+ = pyruvate + 2 NH4+." [EC:4.3.1.15]
 synonym: "2,3-diaminopropionate ammonia-lyase (adding H2O; pyruvate-forming)" RELATED [EC:4.3.1.15]
 synonym: "2,3-diaminopropionate ammonia-lyase activity" RELATED [EC:4.3.1.15]
 synonym: "alpha,beta-diaminopropionate ammonia-lyase activity" RELATED [EC:4.3.1.15]
@@ -92275,6 +92585,7 @@
 def: "Catalysis of the reaction: enterobactin + 3 H2O = 3 N-23-dihydroxybenzoyl-L-serine + 3 H+." [MetaCyc:RXN0-1661, PMID:4565531]
 synonym: "enterobactin esterase activity" EXACT []
 xref: MetaCyc:RXN0-1661
+xref: Reactome:R-HSA-9911599 "Fes hydrolyses ferrienterobactin to DHBS"
 xref: RHEA:28018
 is_a: GO:0016788 ! hydrolase activity, acting on ester bonds
 property_value: skos:exactMatch RHEA:28018
@@ -92284,7 +92595,7 @@
 id: GO:0008851
 name: ethanolamine ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ethanolamine = acetaldehyde + NH3." [EC:4.3.1.7]
+def: "Catalysis of the reaction: ethanolamine = acetaldehyde + NH4+." [RHEA:15313]
 synonym: "ethanolamine ammonia-lyase (acetaldehyde-forming) activity" RELATED [EC:4.3.1.7]
 synonym: "ethanolamine deaminase activity" RELATED [EC:4.3.1.7]
 xref: EC:4.3.1.7
@@ -92920,15 +93231,17 @@
 xref: MetaCyc:AKBLIG-RXN
 xref: RHEA:20736
 is_a: GO:0016453 ! C-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.29
 property_value: skos:exactMatch RHEA:20736
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0008892
 name: guanine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanine + H2O = xanthine + NH3." [EC:3.5.4.3]
+def: "Catalysis of the reaction: guanine + H2O + H+ = xanthine + NH4+." [RHEA:14665]
 synonym: "GAH activity" RELATED [EC:3.5.4.3]
 synonym: "guanase activity" RELATED [EC:3.5.4.3]
 synonym: "guanine aminase activity" RELATED [EC:3.5.4.3]
@@ -93542,16 +93855,15 @@
 name: mannitol-1-phosphate 5-dehydrogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: D-mannitol 1-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+." [EC:1.1.1.17]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:1.1.1.17
 xref: MetaCyc:MANNPDEHYDROG-RXN
 xref: RHEA:19661
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.17
 property_value: skos:exactMatch RHEA:19661
-property_value: skos:relatedMatch MetaCyc:MANNIDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0008927
@@ -94006,6 +94318,7 @@
 synonym: "undecaprenyldiphospho-N-acetyl-D-glucosaminyl-(1->4)-(N-acetyl-D-muramoylpentapeptide):undecaprenyldiphospho-(N-acetyl-D-glucosaminyl-(1->4)-N-acetyl-D-muramoylpentapeptide) disaccharidetransferase activity" RELATED [EC:2.4.99.28]
 xref: EC:2.4.99.28
 xref: MetaCyc:RXN-14712
+xref: Reactome:R-HSA-9911463 "MrcB elongates peptidoglycan"
 xref: RHEA:23708
 is_a: GO:0016758 ! hexosyltransferase activity
 property_value: skos:exactMatch EC:2.4.99.28
@@ -94238,6 +94551,7 @@
 xref: Reactome:R-HSA-1482920 "PG is hydrolyzed to 2-acyl LPG by PLA2[14]"
 xref: Reactome:R-HSA-1482932 "PI is hydrolyzed to 2-acyl LPI by PLA2[13]"
 xref: Reactome:R-HSA-5694485 "ABHD3 hydrolyses LPC(14:0) to 1AGPC"
+xref: Reactome:R-HSA-8869425 "PLA1A hydrolyses PS to 2-acyl LPS"
 xref: RHEA:67424
 is_a: GO:0140322 ! A1-type glycerophospholipase activity
 property_value: skos:exactMatch EC:3.1.1.32
@@ -94407,11 +94721,11 @@
 
 [Term]
 id: GO:0008982
-name: protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+name: protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 namespace: molecular_function
 alt_id: GO:0015455
 alt_id: GO:0015456
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sugar(out) = protein histidine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sugar(out) = protein histidine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, PMID:16339738, PMID:31214989]
 synonym: "enzyme II of the phosphotransferase system" RELATED []
 synonym: "PEP--sugar phosphotransferase enzyme II activity" EXACT []
 synonym: "PEP-dependent phosphotransferase enzyme II" RELATED []
@@ -94419,6 +94733,7 @@
 synonym: "phosphohistidinoprotein-hexose phosphotransferase activity" RELATED []
 synonym: "phosphoprotein factor-hexose phosophotransferase activity" RELATED []
 synonym: "phosphotransfer-driven group translocator" BROAD []
+synonym: "protein-N(PI)-phosphohistidine-sugar phosphotransferase activity" RELATED []
 synonym: "protein-Np-phosphohistidine-sugar phosphotransferase activity" EXACT []
 synonym: "PTS permease activity" RELATED []
 synonym: "PTS transporter" RELATED []
@@ -94426,6 +94741,7 @@
 is_a: GO:0015144 ! carbohydrate transmembrane transporter activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
 is_a: GO:0022804 ! active transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0008983
@@ -94726,13 +95042,14 @@
 
 [Term]
 id: GO:0009001
-name: serine O-acetyltransferase activity
+name: L-serine O-acetyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-serine + acetyl-CoA = O-acetyl-L-serine + CoA." [RHEA:24560]
 synonym: "acetyl-CoA:L-serine O-acetyltransferase activity" RELATED [EC:2.3.1.30]
 synonym: "L-serine acetyltransferase activity" RELATED [EC:2.3.1.30]
 synonym: "SATase activity" RELATED [EC:2.3.1.30]
 synonym: "serine acetyltransferase activity" RELATED [EC:2.3.1.30]
+synonym: "serine O-acetyltransferase activity" BROAD []
 synonym: "serine transacetylase activity" RELATED [EC:2.3.1.30]
 xref: EC:2.3.1.30
 xref: KEGG_REACTION:R00586
@@ -94740,9 +95057,11 @@
 xref: RHEA:24560
 is_a: GO:0016412 ! serine O-acyltransferase activity
 is_a: GO:0016413 ! O-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.30
 property_value: skos:exactMatch RHEA:24560
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0009002
@@ -95008,12 +95327,13 @@
 
 [Term]
 id: GO:0009016
-name: succinyldiaminopimelate transaminase activity
+name: succinyldiaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + N-succinyl-LL-2,6-diaminopimelate = L-2-succinylamino-6-oxopimelate + L-glutamate." [EC:2.6.1.17, RHEA:11960]
+def: "Catalysis of the reaction: N-succinyl-(2S,6S)-2,6-diaminopimelate + 2-oxoglutarate = (S)-2-succinylamino-6-oxoheptanedioate + L-glutamate." [RHEA:11960]
 synonym: "N-succinyl-L-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.17]
 synonym: "N-succinyl-L-diaminopimelic glutamic transaminase activity" RELATED [EC:2.6.1.17]
 synonym: "succinyldiaminopimelate aminotransferase activity" EXACT []
+synonym: "succinyldiaminopimelate transaminase activity" EXACT []
 synonym: "succinyldiaminopimelate transferase activity" RELATED [EC:2.6.1.17]
 xref: EC:2.6.1.17
 xref: KEGG_REACTION:R04475
@@ -95023,6 +95343,7 @@
 property_value: skos:exactMatch EC:2.6.1.17
 property_value: skos:exactMatch RHEA:11960
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0009017
@@ -95108,7 +95429,7 @@
 id: GO:0009023
 name: obsolete tRNA sulfurtransferase
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: L-cysteine + activated-tRNA = L-serine + tRNA containing a thionucleotide." [EC:2.8.1.4, GOC:go_curators]
+def: "OBSOLETE. Catalysis of the reaction: L-cysteine + activated-tRNA = L-serine + tRNA containing a thionucleotide." [EC:2.8.1.4, GOC:curators]
 synonym: "tRNA sulfurtransferase" EXACT []
 synonym: "tRNA sulphurtransferase activity" EXACT []
 is_obsolete: true
@@ -95287,7 +95608,6 @@
 name: tryptophanase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-tryptophan + H2O = indole + NH4 + pyruvate." [RHEA:19553]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-tryptophan indole-lyase (deaminating) activity" RELATED [EC:4.1.99.1]
 synonym: "L-tryptophan indole-lyase (deaminating; pyruvate forming) activity" RELATED [EC:4.1.99.1]
 synonym: "L-tryptophan indole-lyase activity" RELATED [EC:4.1.99.1]
@@ -95305,8 +95625,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00673
 property_value: skos:exactMatch MetaCyc:TRYPTOPHAN-RXN
 property_value: skos:exactMatch RHEA:19553
-property_value: skos:relatedMatch MetaCyc:TRYPDEG-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0009035
@@ -95371,7 +95691,7 @@
 id: GO:0009039
 name: urease activity
 namespace: molecular_function
-def: "Catalysis of the reaction: urea + H2O = CO2 + 2 NH3." [EC:3.5.1.5, RHEA:20557]
+def: "Catalysis of the reaction: urea + 2 H2O + H+ = hydrogencarbonate + 2 NH4+." [RHEA:20557]
 synonym: "urea amidohydrolase activity" RELATED [EC:3.5.1.5]
 xref: EC:3.5.1.5
 xref: MetaCyc:UREASE-RXN
@@ -95398,7 +95718,7 @@
 id: GO:0009041
 name: obsolete UMP/dUMP kinase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: ATP + (d)UMP = ADP + (d)UDP." [GOC:go_curators]
+def: "OBSOLETE. Catalysis of the reaction: ATP + (d)UMP = ADP + (d)UDP." [GOC:curators]
 comment: The reason for obsoletion is that this term was an unnecessary grouping term.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24402" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25991" xsd:anyURI
@@ -95409,23 +95729,25 @@
 
 [Term]
 id: GO:0009042
-name: valine-pyruvate transaminase activity
+name: L-valine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine." [EC:2.6.1.66, RHEA:22912]
+def: "Catalysis of the reaction: L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine." [RHEA:22912]
 synonym: "alanine--valine transaminase activity" RELATED [EC:2.6.1.66]
 synonym: "alanine-oxoisovalerate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "L-valine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "transaminase C activity" NARROW [EC:2.6.1.66]
 synonym: "valine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.66]
 synonym: "valine-pyruvate aminotransferase activity" EXACT []
+synonym: "valine-pyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.66
 xref: KEGG_REACTION:R01215
 xref: MetaCyc:VALINE-PYRUVATE-AMINOTRANSFER-RXN
 xref: RHEA:22912
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.66
 property_value: skos:exactMatch RHEA:22912
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0009044
@@ -95515,7 +95837,7 @@
 id: GO:0009050
 name: glycopeptide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glycopeptides, a compound in which carbohydrate is covalently attached to an oligopeptide composed of residues of L and/or D-amino acids. The term usually denotes a product of proteolytic degradation of a glycoprotein but includes glycated peptide." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of glycopeptides, a compound in which carbohydrate is covalently attached to an oligopeptide composed of residues of L and/or D-amino acids. The term usually denotes a product of proteolytic degradation of a glycoprotein but includes glycated peptide." [GOC:curators, ISBN:0198506732]
 synonym: "glycopeptide breakdown" EXACT []
 synonym: "glycopeptide catabolism" EXACT []
 synonym: "glycopeptide degradation" EXACT []
@@ -95582,7 +95904,6 @@
 synonym: "electron donor activity" NARROW []
 synonym: "electron transporter activity" RELATED []
 xref: Reactome:R-HSA-169260 "Reducing equivalents from beta-oxidation of fatty acids transfer to ETF"
-xref: Reactome:R-HSA-2564826 "4Fe-4S cluster assembles on NUBP2:NUBP1 scaffold"
 is_a: GO:0003674 ! molecular_function
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21334" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26572" xsd:anyURI
@@ -95702,7 +96023,7 @@
 id: GO:0009062
 name: fatty acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a fatty acid, any of the aliphatic monocarboxylic acids that can be liberated by hydrolysis from naturally occurring fats and oils. Fatty acids are predominantly straight-chain acids of 4 to 24 carbon atoms, which may be saturated or unsaturated; branched fatty acids and hydroxy fatty acids also occur, and very long chain acids of over 30 carbons are found in waxes." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of a fatty acid, any of the aliphatic monocarboxylic acids that can be liberated by hydrolysis from naturally occurring fats and oils. Fatty acids are predominantly straight-chain acids of 4 to 24 carbon atoms, which may be saturated or unsaturated; branched fatty acids and hydroxy fatty acids also occur, and very long chain acids of over 30 carbons are found in waxes." [GOC:curators]
 synonym: "fatty acid breakdown" EXACT []
 synonym: "fatty acid catabolism" EXACT []
 synonym: "fatty acid degradation" EXACT []
@@ -95733,7 +96054,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai]
 comment: The reason for obsoletion is that this is an unnecessary grouping term.
 synonym: "glutamine family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8964539 "Glutamate and glutamine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
 is_obsolete: true
 
@@ -95756,7 +96076,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai]
 comment: The reason for obsoletion is that this is an unnecessary grouping term.
 synonym: "aspartate family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8963693 "Aspartate and asparagine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
 is_obsolete: true
 
@@ -95764,16 +96083,15 @@
 id: GO:0009067
 name: aspartate family amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai, MetaCyc:PWY0-781]
+def: "The chemical reactions and pathways resulting in the formation of amino acids of the aspartate family, comprising asparagine, aspartate, lysine, methionine and threonine." [GOC:ai]
 synonym: "aspartate family amino acid anabolism" EXACT []
 synonym: "aspartate family amino acid biosynthesis" EXACT []
 synonym: "aspartate family amino acid formation" EXACT []
 synonym: "aspartate family amino acid synthesis" EXACT []
-xref: MetaCyc:PWY0-781
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY0-781
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009068
@@ -95806,14 +96124,9 @@
 synonym: "serine family amino acid biosynthesis" EXACT []
 synonym: "serine family amino acid formation" EXACT []
 synonym: "serine family amino acid synthesis" EXACT []
-xref: MetaCyc:PWY-6292
-xref: MetaCyc:SER-GLYSYN-PWY
-xref: MetaCyc:SULFATE-CYS-PWY
 is_a: GO:1901607 ! alpha-amino acid biosynthetic process
-property_value: skos:narrowMatch MetaCyc:PWY-6292
-property_value: skos:narrowMatch MetaCyc:SER-GLYSYN-PWY
-property_value: skos:narrowMatch MetaCyc:SULFATE-CYS-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009071
@@ -95831,7 +96144,7 @@
 id: GO:0009072
 name: aromatic amino acid metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:curators]
 synonym: "aromatic amino acid family metabolic process" EXACT []
 synonym: "aromatic amino acid family metabolism" EXACT []
 xref: Reactome:R-HSA-8963691 "Phenylalanine and tyrosine metabolism"
@@ -95843,7 +96156,7 @@
 name: aromatic amino acid family biosynthetic process
 namespace: biological_process
 alt_id: GO:0016089
-def: "The chemical reactions and pathways resulting in the formation of aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:curators]
 synonym: "aromatic amino acid family anabolism" EXACT []
 synonym: "aromatic amino acid family biosynthesis" EXACT []
 synonym: "aromatic amino acid family biosynthetic process, shikimate pathway" RELATED []
@@ -95857,7 +96170,7 @@
 id: GO:0009074
 name: aromatic amino acid family catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of aromatic amino acid family, amino acids with aromatic ring (phenylalanine, tyrosine, tryptophan)." [GOC:curators]
 synonym: "aromatic amino acid family breakdown" EXACT []
 synonym: "aromatic amino acid family catabolism" EXACT []
 synonym: "aromatic amino acid family degradation" EXACT []
@@ -95909,7 +96222,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving any amino acid that requires pyruvate for its synthesis, e.g. alanine." [GOC:jl]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "pyruvate family amino acid metabolism" EXACT []
-xref: Reactome:R-HSA-8964540 "Alanine metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -95977,32 +96289,42 @@
 id: GO:0009084
 name: L-glutamine family amino acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai, MetaCyc:GLUTAMINE-SYN]
+def: "The chemical reactions and pathways resulting in the formation of amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline." [GOC:ai]
 synonym: "glutamine family amino acid anabolism" EXACT []
 synonym: "glutamine family amino acid biosynthesis" EXACT []
 synonym: "glutamine family amino acid formation" EXACT []
 synonym: "glutamine family amino acid synthesis" EXACT []
-xref: MetaCyc:GLUTAMINE-SYN
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:GLUTAMINE-SYN
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009085
 name: L-lysine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine, 2,6-diaminohexanoic acid." [GOC:go_curators, MetaCyc:LYSINE-SYN]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine, 2,6-diaminohexanoic acid." [GOC:curators]
 synonym: "lysine anabolism" EXACT []
 synonym: "lysine biosynthesis" EXACT []
 synonym: "lysine formation" EXACT []
 synonym: "lysine synthesis" EXACT []
-xref: MetaCyc:LYSINE-SYN
+xref: MetaCyc:DAPLYSINESYN-PWY
+xref: MetaCyc:LYSINE-AMINOAD-PWY
+xref: MetaCyc:PWY-2941
+xref: MetaCyc:PWY-2942
+xref: MetaCyc:PWY-3081
+xref: MetaCyc:PWY-5097
 is_a: GO:0006553 ! lysine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
-property_value: skos:exactMatch MetaCyc:LYSINE-SYN
+property_value: skos:narrowMatch MetaCyc:DAPLYSINESYN-PWY
+property_value: skos:narrowMatch MetaCyc:LYSINE-AMINOAD-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-2941
+property_value: skos:narrowMatch MetaCyc:PWY-2942
+property_value: skos:narrowMatch MetaCyc:PWY-3081
+property_value: skos:narrowMatch MetaCyc:PWY-5097
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28388" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009086
@@ -96022,7 +96344,7 @@
 id: GO:0009087
 name: L-methionine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-methionine (2-amino-4-(methylthio)butanoic acid), a sulfur-containing, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-methionine." [GOC:curators]
 synonym: "methionine breakdown" EXACT []
 synonym: "methionine catabolism" EXACT []
 synonym: "methionine degradation" EXACT []
@@ -96038,29 +96360,29 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5082
 property_value: skos:narrowMatch MetaCyc:PWY-701
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0009088
 name: L-threonine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732, MetaCyc:THRESYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of L-threonine (2-amino-3-hydroxybutyric acid), a polar, uncharged, essential amino acid found in peptide linkage in proteins." [GOC:jl, ISBN:0198506732]
 synonym: "threonine anabolism" EXACT []
 synonym: "threonine biosynthesis" EXACT []
 synonym: "threonine formation" EXACT []
 synonym: "threonine synthesis" EXACT []
 xref: MetaCyc:HOMOSER-THRESYN-PWY
-xref: MetaCyc:THRESYN-PWY
 is_a: GO:0006566 ! L-threonine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 property_value: skos:narrowMatch MetaCyc:HOMOSER-THRESYN-PWY
-property_value: skos:narrowMatch MetaCyc:THRESYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009089
 name: L-lysine biosynthetic process via diaminopimelate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate." [GOC:curators]
 synonym: "diaminopimelate pathway" EXACT []
 synonym: "diaminopimelic acid pathway" EXACT []
 synonym: "lysine anabolism via diaminopimelate" EXACT []
@@ -96075,7 +96397,7 @@
 id: GO:0009090
 name: L-homoserine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of homoserine, alpha-amino-gamma-hydroxybutyric acid." [GOC:go_curators, ISBN:0198506732, MetaCyc:HOMOSERSYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of homoserine, alpha-amino-gamma-hydroxybutyric acid." [GOC:curators, ISBN:0198506732, MetaCyc:HOMOSERSYN-PWY]
 synonym: "homoserine anabolism" EXACT []
 synonym: "homoserine biosynthesis" EXACT []
 synonym: "homoserine formation" EXACT []
@@ -96089,17 +96411,17 @@
 
 [Term]
 id: GO:0009091
-name: L-homoserine catabolic process
+name: obsolete L-homoserine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of homoserine, alpha-amino-gamma-hydroxybutyric acid." [ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of homoserine, alpha-amino-gamma-hydroxybutyric acid." [ISBN:0198506732]
+comment: This term was obsoleted because there is no evidence that a specific pathway for L-homoserine degradation exists.
 synonym: "homoserine breakdown" EXACT []
 synonym: "homoserine catabolism" EXACT []
 synonym: "homoserine degradation" EXACT []
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0009092
@@ -96115,7 +96437,7 @@
 id: GO:0009093
 name: obsolete cysteine catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:curators]
 comment: The reason for obsoletion is that a more specific term has been created.
 synonym: "cysteine breakdown" EXACT []
 synonym: "cysteine catabolism" EXACT []
@@ -96140,9 +96462,16 @@
 synonym: "phenylalanine biosynthetic process" BROAD []
 synonym: "phenylalanine biosynthetic process, prephenate pathway" RELATED []
 synonym: "phenylalanine biosynthetic process, shikimate pathway" RELATED []
+xref: MetaCyc:PHESYN
+xref: MetaCyc:PWY-3462
+xref: MetaCyc:PWY-7432
 is_a: GO:0006558 ! L-phenylalanine metabolic process
 is_a: GO:0009073 ! aromatic amino acid family biosynthetic process
 is_a: GO:1902223 ! erythrose 4-phosphate/phosphoenolpyruvate family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PHESYN
+property_value: skos:narrowMatch MetaCyc:PWY-3462
+property_value: skos:narrowMatch MetaCyc:PWY-7432
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0009095
@@ -96160,17 +96489,17 @@
 
 [Term]
 id: GO:0009097
-name: isoleucine biosynthetic process
+name: obsolete isoleucine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of isoleucine, (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai]
+comment: The reason for obsoletion is that more specific terms were created.
 synonym: "isoleucine anabolism" EXACT []
 synonym: "isoleucine biosynthesis" EXACT []
 synonym: "isoleucine formation" EXACT []
 synonym: "isoleucine synthesis" EXACT []
-xref: MetaCyc:ILEUSYN-PWY
-is_a: GO:0006549 ! isoleucine metabolic process
-is_a: GO:0009082 ! branched-chain amino acid biosynthetic process
-is_a: GO:1901607 ! alpha-amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30848" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:1901705
 
 [Term]
 id: GO:0009098
@@ -96204,14 +96533,14 @@
 relationship: has_part GO:0003984 ! acetolactate synthase activity
 relationship: has_part GO:0004160 ! dihydroxy-acid dehydratase activity
 relationship: has_part GO:0004455 ! ketol-acid reductoisomerase activity
-relationship: has_part GO:0052655 ! L-valine-2-oxoglutarate transaminase activity
+relationship: has_part GO:0052655 ! L-valine:2-oxoglutarate transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28686" xsd:anyURI
 
 [Term]
 id: GO:0009100
 name: glycoprotein metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving glycoproteins, a protein that contains covalently bound glycose (i.e. monosaccharide) residues; the glycose occurs most commonly as oligosaccharide or fairly small polysaccharide but occasionally as monosaccharide." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving glycoproteins, a protein that contains covalently bound glycose (i.e. monosaccharide) residues; the glycose occurs most commonly as oligosaccharide or fairly small polysaccharide but occasionally as monosaccharide." [GOC:curators, ISBN:0198506732]
 subset: goslim_drosophila
 synonym: "glycoprotein metabolism" EXACT []
 is_a: GO:0019538 ! protein metabolic process
@@ -96345,7 +96674,7 @@
 id: GO:0009110
 name: vitamin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a vitamin, one of a number of unrelated organic substances that occur in many foods in small amounts and that are necessary in trace amounts for the normal metabolic functioning of the body." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a vitamin, one of a number of unrelated organic substances that occur in many foods in small amounts and that are necessary in trace amounts for the normal metabolic functioning of the body." [GOC:curators, ISBN:0198506732]
 synonym: "vitamin anabolism" EXACT []
 synonym: "vitamin biosynthesis" EXACT []
 synonym: "vitamin formation" EXACT []
@@ -96380,7 +96709,7 @@
 def: "The chemical reactions and pathways resulting in the formation of purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, which include adenine and guanine." [ISBN:0198506732]
 synonym: "purine base anabolism" EXACT []
 synonym: "purine base biosynthesis" EXACT []
-synonym: "purine base biosynthetic process" EXACT [GOC:go_curators]
+synonym: "purine base biosynthetic process" EXACT [GOC:curators]
 synonym: "purine base formation" EXACT []
 synonym: "purine base synthesis" EXACT []
 xref: MetaCyc:PWY-841
@@ -96406,7 +96735,7 @@
 name: xanthine catabolic process
 namespace: biological_process
 alt_id: GO:0006151
-def: "The chemical reactions and pathways resulting in the breakdown of xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:curators]
 synonym: "xanthine breakdown" EXACT []
 synonym: "xanthine catabolism" EXACT []
 synonym: "xanthine degradation" EXACT []
@@ -96437,7 +96766,7 @@
 id: GO:0009118
 name: regulation of nucleoside metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:curators]
 synonym: "regulation of nucleoside metabolism" EXACT []
 is_a: GO:0019219 ! regulation of nucleobase-containing compound metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
@@ -96465,7 +96794,7 @@
 id: GO:0009123
 name: nucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside monophosphate metabolism" EXACT []
 is_a: GO:0006753 ! nucleoside phosphate metabolic process
 
@@ -96473,7 +96802,7 @@
 id: GO:0009124
 name: nucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside monophosphate anabolism" EXACT []
 synonym: "nucleoside monophosphate biosynthesis" EXACT []
 synonym: "nucleoside monophosphate formation" EXACT []
@@ -96485,7 +96814,7 @@
 id: GO:0009125
 name: nucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside monophosphate breakdown" EXACT []
 synonym: "nucleoside monophosphate catabolism" EXACT []
 synonym: "nucleoside monophosphate degradation" EXACT []
@@ -96496,7 +96825,7 @@
 id: GO:0009126
 name: purine nucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009123 ! nucleoside monophosphate metabolic process
 
@@ -96504,7 +96833,7 @@
 id: GO:0009127
 name: purine nucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside monophosphate anabolism" EXACT []
 synonym: "purine nucleoside monophosphate biosynthesis" EXACT []
 synonym: "purine nucleoside monophosphate formation" EXACT []
@@ -96516,7 +96845,7 @@
 id: GO:0009128
 name: purine nucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside monophosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside monophosphate breakdown" EXACT []
 synonym: "purine nucleoside monophosphate catabolism" EXACT []
 synonym: "purine nucleoside monophosphate degradation" EXACT []
@@ -96527,7 +96856,7 @@
 id: GO:0009129
 name: pyrimidine nucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009123 ! nucleoside monophosphate metabolic process
 
@@ -96535,7 +96864,7 @@
 id: GO:0009130
 name: pyrimidine nucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside monophosphate anabolism" EXACT []
 synonym: "pyrimidine nucleoside monophosphate biosynthesis" EXACT []
 synonym: "pyrimidine nucleoside monophosphate formation" EXACT []
@@ -96547,7 +96876,7 @@
 id: GO:0009131
 name: pyrimidine nucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside monophosphate breakdown" EXACT []
 synonym: "pyrimidine nucleoside monophosphate catabolism" EXACT []
 synonym: "pyrimidine nucleoside monophosphate degradation" EXACT []
@@ -96558,7 +96887,7 @@
 id: GO:0009132
 name: nucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside diphosphate metabolism" EXACT []
 is_a: GO:0006753 ! nucleoside phosphate metabolic process
 
@@ -96566,7 +96895,7 @@
 id: GO:0009133
 name: nucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside diphosphate anabolism" EXACT []
 synonym: "nucleoside diphosphate biosynthesis" EXACT []
 synonym: "nucleoside diphosphate formation" EXACT []
@@ -96578,7 +96907,7 @@
 id: GO:0009134
 name: nucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside diphosphate breakdown" EXACT []
 synonym: "nucleoside diphosphate catabolism" EXACT []
 synonym: "nucleoside diphosphate degradation" EXACT []
@@ -96589,7 +96918,7 @@
 id: GO:0009135
 name: purine nucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009132 ! nucleoside diphosphate metabolic process
 
@@ -96597,7 +96926,7 @@
 id: GO:0009136
 name: purine nucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside diphosphate anabolism" EXACT []
 synonym: "purine nucleoside diphosphate biosynthesis" EXACT []
 synonym: "purine nucleoside diphosphate formation" EXACT []
@@ -96609,7 +96938,7 @@
 id: GO:0009137
 name: purine nucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside diphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside diphosphate breakdown" EXACT []
 synonym: "purine nucleoside diphosphate catabolism" EXACT []
 synonym: "purine nucleoside diphosphate degradation" EXACT []
@@ -96620,7 +96949,7 @@
 id: GO:0009138
 name: pyrimidine nucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009132 ! nucleoside diphosphate metabolic process
 
@@ -96628,7 +96957,7 @@
 id: GO:0009139
 name: pyrimidine nucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside diphosphate anabolism" EXACT []
 synonym: "pyrimidine nucleoside diphosphate biosynthesis" EXACT []
 synonym: "pyrimidine nucleoside diphosphate formation" EXACT []
@@ -96640,7 +96969,7 @@
 id: GO:0009140
 name: pyrimidine nucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside diphosphate breakdown" EXACT []
 synonym: "pyrimidine nucleoside diphosphate catabolism" EXACT []
 synonym: "pyrimidine nucleoside diphosphate degradation" EXACT []
@@ -96651,7 +96980,7 @@
 id: GO:0009141
 name: nucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside triphosphate metabolism" EXACT []
 is_a: GO:0006753 ! nucleoside phosphate metabolic process
 
@@ -96659,7 +96988,7 @@
 id: GO:0009142
 name: nucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside triphosphate anabolism" EXACT []
 synonym: "nucleoside triphosphate biosynthesis" EXACT []
 synonym: "nucleoside triphosphate formation" EXACT []
@@ -96671,7 +97000,7 @@
 id: GO:0009143
 name: nucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a nucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose or ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "nucleoside triphosphate breakdown" EXACT []
 synonym: "nucleoside triphosphate catabolism" EXACT []
 synonym: "nucleoside triphosphate degradation" EXACT []
@@ -96682,7 +97011,7 @@
 id: GO:0009144
 name: purine nucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009141 ! nucleoside triphosphate metabolic process
 
@@ -96690,7 +97019,7 @@
 id: GO:0009145
 name: purine nucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside triphosphate anabolism" EXACT []
 synonym: "purine nucleoside triphosphate biosynthesis" EXACT []
 synonym: "purine nucleoside triphosphate formation" EXACT []
@@ -96702,7 +97031,7 @@
 id: GO:0009146
 name: purine nucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine nucleoside triphosphate, a compound consisting of a purine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine nucleoside triphosphate breakdown" EXACT []
 synonym: "purine nucleoside triphosphate catabolism" EXACT []
 synonym: "purine nucleoside triphosphate degradation" EXACT []
@@ -96713,7 +97042,7 @@
 id: GO:0009147
 name: pyrimidine nucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009141 ! nucleoside triphosphate metabolic process
 
@@ -96721,7 +97050,7 @@
 id: GO:0009148
 name: pyrimidine nucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside triphosphate anabolism" EXACT []
 synonym: "pyrimidine nucleoside triphosphate biosynthesis" EXACT []
 synonym: "pyrimidine nucleoside triphosphate formation" EXACT []
@@ -96733,7 +97062,7 @@
 id: GO:0009149
 name: pyrimidine nucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine nucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose or deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine nucleoside triphosphate breakdown" EXACT []
 synonym: "pyrimidine nucleoside triphosphate catabolism" EXACT []
 synonym: "pyrimidine nucleoside triphosphate degradation" EXACT []
@@ -96744,7 +97073,7 @@
 id: GO:0009150
 name: purine ribonucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleotide metabolism" EXACT []
 is_a: GO:0006163 ! purine nucleotide metabolic process
 is_a: GO:0009259 ! ribonucleotide metabolic process
@@ -96753,7 +97082,7 @@
 id: GO:0009151
 name: purine deoxyribonucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleotide metabolism" EXACT []
 is_a: GO:0006163 ! purine nucleotide metabolic process
 is_a: GO:0009394 ! 2'-deoxyribonucleotide metabolic process
@@ -96762,7 +97091,7 @@
 id: GO:0009152
 name: purine ribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleotide anabolism" EXACT []
 synonym: "purine ribonucleotide biosynthesis" EXACT []
 synonym: "purine ribonucleotide formation" EXACT []
@@ -96775,7 +97104,7 @@
 id: GO:0009153
 name: purine deoxyribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleotide anabolism" EXACT []
 synonym: "purine deoxyribonucleotide biosynthesis" EXACT []
 synonym: "purine deoxyribonucleotide formation" EXACT []
@@ -96788,7 +97117,7 @@
 id: GO:0009154
 name: purine ribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a purine ribonucleotide, a compound consisting of ribonucleoside (a purine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleotide breakdown" EXACT []
 synonym: "purine ribonucleotide catabolism" EXACT []
 synonym: "purine ribonucleotide degradation" EXACT []
@@ -96800,7 +97129,7 @@
 id: GO:0009155
 name: purine deoxyribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a purine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleotide breakdown" EXACT []
 synonym: "purine deoxyribonucleotide catabolism" EXACT []
 synonym: "purine deoxyribonucleotide degradation" EXACT []
@@ -96813,7 +97142,7 @@
 id: GO:0009156
 name: ribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside monophosphate anabolism" EXACT []
 synonym: "ribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "ribonucleoside monophosphate formation" EXACT []
@@ -96824,7 +97153,7 @@
 id: GO:0009157
 name: deoxyribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside monophosphate anabolism" EXACT []
 synonym: "deoxyribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "deoxyribonucleoside monophosphate formation" EXACT []
@@ -96835,7 +97164,7 @@
 id: GO:0009158
 name: ribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside monophosphate breakdown" EXACT []
 synonym: "ribonucleoside monophosphate catabolism" EXACT []
 synonym: "ribonucleoside monophosphate degradation" EXACT []
@@ -96845,7 +97174,7 @@
 id: GO:0009159
 name: deoxyribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside monophosphate breakdown" EXACT []
 synonym: "deoxyribonucleoside monophosphate catabolism" EXACT []
 synonym: "deoxyribonucleoside monophosphate degradation" EXACT []
@@ -96855,7 +97184,7 @@
 id: GO:0009161
 name: obsolete ribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways involving a ribonucleoside monophosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "ribonucleoside monophosphate metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -96865,7 +97194,7 @@
 id: GO:0009162
 name: obsolete deoxyribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways involving a deoxyribonucleoside monophosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "deoxyribonucleoside monophosphate metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -96900,7 +97229,7 @@
 id: GO:0009165
 name: nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nucleotides, any nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the glycose moiety; may be mono-, di- or triphosphate; this definition includes cyclic-nucleotides (nucleoside cyclic phosphates)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of nucleotides, any nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the glycose moiety; may be mono-, di- or triphosphate; this definition includes cyclic-nucleotides (nucleoside cyclic phosphates)." [GOC:curators]
 synonym: "nucleotide anabolism" EXACT []
 synonym: "nucleotide biosynthesis" EXACT []
 synonym: "nucleotide formation" EXACT []
@@ -96912,7 +97241,7 @@
 id: GO:0009166
 name: nucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of nucleotides, any nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the glycose moiety; may be mono-, di- or triphosphate; this definition includes cyclic-nucleotides (nucleoside cyclic phosphates)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of nucleotides, any nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the glycose moiety; may be mono-, di- or triphosphate; this definition includes cyclic-nucleotides (nucleoside cyclic phosphates)." [GOC:curators]
 synonym: "nucleotide breakdown" EXACT []
 synonym: "nucleotide catabolism" EXACT []
 synonym: "nucleotide degradation" EXACT []
@@ -96923,7 +97252,7 @@
 id: GO:0009167
 name: purine ribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009126 ! purine nucleoside monophosphate metabolic process
 
@@ -96931,7 +97260,7 @@
 id: GO:0009168
 name: purine ribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside monophosphate anabolism" EXACT []
 synonym: "purine ribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "purine ribonucleoside monophosphate formation" EXACT []
@@ -96945,7 +97274,7 @@
 id: GO:0009169
 name: purine ribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside monophosphate, a compound consisting of a purine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside monophosphate breakdown" EXACT []
 synonym: "purine ribonucleoside monophosphate catabolism" EXACT []
 synonym: "purine ribonucleoside monophosphate degradation" EXACT []
@@ -96957,7 +97286,7 @@
 id: GO:0009170
 name: purine deoxyribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009126 ! purine nucleoside monophosphate metabolic process
 
@@ -96965,7 +97294,7 @@
 id: GO:0009171
 name: purine deoxyribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside monophosphate anabolism" EXACT []
 synonym: "purine deoxyribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "purine deoxyribonucleoside monophosphate formation" EXACT []
@@ -96978,7 +97307,7 @@
 id: GO:0009172
 name: purine deoxyribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside monophosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside monophosphate breakdown" EXACT []
 synonym: "purine deoxyribonucleoside monophosphate catabolism" EXACT []
 synonym: "purine deoxyribonucleoside monophosphate degradation" EXACT []
@@ -96990,7 +97319,7 @@
 id: GO:0009173
 name: pyrimidine ribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009129 ! pyrimidine nucleoside monophosphate metabolic process
 
@@ -96998,7 +97327,7 @@
 id: GO:0009174
 name: pyrimidine ribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside monophosphate anabolism" EXACT []
 synonym: "pyrimidine ribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "pyrimidine ribonucleoside monophosphate formation" EXACT []
@@ -97011,7 +97340,7 @@
 id: GO:0009175
 name: pyrimidine ribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside monophosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside monophosphate breakdown" EXACT []
 synonym: "pyrimidine ribonucleoside monophosphate catabolism" EXACT []
 synonym: "pyrimidine ribonucleoside monophosphate degradation" EXACT []
@@ -97023,7 +97352,7 @@
 id: GO:0009176
 name: pyrimidine deoxyribonucleoside monophosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside monophosphate metabolism" EXACT []
 is_a: GO:0009129 ! pyrimidine nucleoside monophosphate metabolic process
 
@@ -97031,7 +97360,7 @@
 id: GO:0009177
 name: pyrimidine deoxyribonucleoside monophosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside monophosphate anabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside monophosphate biosynthesis" EXACT []
 synonym: "pyrimidine deoxyribonucleoside monophosphate formation" EXACT []
@@ -97044,7 +97373,7 @@
 id: GO:0009178
 name: pyrimidine deoxyribonucleoside monophosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxynucleoside monophosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with phosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside monophosphate breakdown" EXACT []
 synonym: "pyrimidine deoxyribonucleoside monophosphate catabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside monophosphate degradation" EXACT []
@@ -97056,7 +97385,7 @@
 id: GO:0009179
 name: purine ribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009135 ! purine nucleoside diphosphate metabolic process
 is_a: GO:0009185 ! ribonucleoside diphosphate metabolic process
@@ -97065,7 +97394,7 @@
 id: GO:0009180
 name: purine ribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside diphosphate anabolism" EXACT []
 synonym: "purine ribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "purine ribonucleoside diphosphate formation" EXACT []
@@ -97078,7 +97407,7 @@
 id: GO:0009181
 name: purine ribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside diphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside diphosphate breakdown" EXACT []
 synonym: "purine ribonucleoside diphosphate catabolism" EXACT []
 synonym: "purine ribonucleoside diphosphate degradation" EXACT []
@@ -97090,7 +97419,7 @@
 id: GO:0009182
 name: purine deoxyribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009135 ! purine nucleoside diphosphate metabolic process
 
@@ -97098,7 +97427,7 @@
 id: GO:0009183
 name: purine deoxyribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside diphosphate anabolism" EXACT []
 synonym: "purine deoxyribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "purine deoxyribonucleoside diphosphate formation" EXACT []
@@ -97111,7 +97440,7 @@
 id: GO:0009184
 name: purine deoxyribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside diphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside diphosphate breakdown" EXACT []
 synonym: "purine deoxyribonucleoside diphosphate catabolism" EXACT []
 synonym: "purine deoxyribonucleoside diphosphate degradation" EXACT []
@@ -97123,7 +97452,7 @@
 id: GO:0009185
 name: ribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009132 ! nucleoside diphosphate metabolic process
 
@@ -97131,7 +97460,7 @@
 id: GO:0009186
 name: obsolete deoxyribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways involving a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "deoxyribonucleoside diphosphate metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -97141,7 +97470,7 @@
 id: GO:0009187
 name: cyclic nucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:curators, ISBN:0198506732]
 synonym: "cyclic nucleotide metabolism" EXACT []
 is_a: GO:0009117 ! nucleotide metabolic process
 
@@ -97149,7 +97478,7 @@
 id: GO:0009188
 name: ribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside diphosphate anabolism" EXACT []
 synonym: "ribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "ribonucleoside diphosphate formation" EXACT []
@@ -97161,7 +97490,7 @@
 id: GO:0009189
 name: deoxyribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside diphosphate anabolism" EXACT []
 synonym: "deoxyribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "deoxyribonucleoside diphosphate formation" EXACT []
@@ -97172,7 +97501,7 @@
 id: GO:0009190
 name: cyclic nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:curators, ISBN:0198506732]
 synonym: "cyclic nucleotide anabolism" EXACT []
 synonym: "cyclic nucleotide biosynthesis" EXACT []
 synonym: "cyclic nucleotide formation" EXACT []
@@ -97184,7 +97513,7 @@
 id: GO:0009191
 name: ribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside diphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside diphosphate breakdown" EXACT []
 synonym: "ribonucleoside diphosphate catabolism" EXACT []
 synonym: "ribonucleoside diphosphate degradation" EXACT []
@@ -97195,7 +97524,7 @@
 id: GO:0009192
 name: deoxyribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside diphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside diphosphate breakdown" EXACT []
 synonym: "deoxyribonucleoside diphosphate catabolism" EXACT []
 synonym: "deoxyribonucleoside diphosphate degradation" EXACT []
@@ -97205,7 +97534,7 @@
 id: GO:0009193
 name: pyrimidine ribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009138 ! pyrimidine nucleoside diphosphate metabolic process
 is_a: GO:0009185 ! ribonucleoside diphosphate metabolic process
@@ -97214,7 +97543,7 @@
 id: GO:0009194
 name: pyrimidine ribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside diphosphate anabolism" EXACT []
 synonym: "pyrimidine ribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "pyrimidine ribonucleoside diphosphate formation" EXACT []
@@ -97227,7 +97556,7 @@
 id: GO:0009195
 name: pyrimidine ribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside diphosphate breakdown" EXACT []
 synonym: "pyrimidine ribonucleoside diphosphate catabolism" EXACT []
 synonym: "pyrimidine ribonucleoside diphosphate degradation" EXACT []
@@ -97239,7 +97568,7 @@
 id: GO:0009196
 name: pyrimidine deoxyribonucleoside diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine deoxynucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine deoxynucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside diphosphate metabolism" EXACT []
 is_a: GO:0009138 ! pyrimidine nucleoside diphosphate metabolic process
 
@@ -97247,7 +97576,7 @@
 id: GO:0009197
 name: pyrimidine deoxyribonucleoside diphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxyribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxyribonucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside diphosphate anabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside diphosphate biosynthesis" EXACT []
 synonym: "pyrimidine deoxyribonucleoside diphosphate formation" EXACT []
@@ -97260,7 +97589,7 @@
 id: GO:0009198
 name: pyrimidine deoxyribonucleoside diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxynucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxynucleoside diphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with diphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside diphosphate breakdown" EXACT []
 synonym: "pyrimidine deoxyribonucleoside diphosphate catabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside diphosphate degradation" EXACT []
@@ -97272,7 +97601,7 @@
 id: GO:0009199
 name: obsolete ribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways involving a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "ribonucleoside triphosphate metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -97282,7 +97611,7 @@
 id: GO:0009200
 name: deoxyribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009141 ! nucleoside triphosphate metabolic process
 
@@ -97290,7 +97619,7 @@
 id: GO:0009201
 name: ribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside triphosphate anabolism" EXACT []
 synonym: "ribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "ribonucleoside triphosphate formation" EXACT []
@@ -97301,7 +97630,7 @@
 id: GO:0009202
 name: deoxyribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside triphosphate anabolism" EXACT []
 synonym: "deoxyribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "deoxyribonucleoside triphosphate formation" EXACT []
@@ -97313,7 +97642,7 @@
 id: GO:0009203
 name: ribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleoside triphosphate, a compound consisting of a nucleobase linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleoside triphosphate breakdown" EXACT []
 synonym: "ribonucleoside triphosphate catabolism" EXACT []
 synonym: "ribonucleoside triphosphate degradation" EXACT []
@@ -97323,7 +97652,7 @@
 id: GO:0009204
 name: deoxyribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleoside triphosphate, a compound consisting of a nucleobase linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleoside triphosphate breakdown" EXACT []
 synonym: "deoxyribonucleoside triphosphate catabolism" EXACT []
 synonym: "deoxyribonucleoside triphosphate degradation" EXACT []
@@ -97334,7 +97663,7 @@
 id: GO:0009205
 name: purine ribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009144 ! purine nucleoside triphosphate metabolic process
 
@@ -97342,7 +97671,7 @@
 id: GO:0009206
 name: purine ribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside triphosphate anabolism" EXACT []
 synonym: "purine ribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "purine ribonucleoside triphosphate formation" EXACT []
@@ -97355,7 +97684,7 @@
 id: GO:0009207
 name: purine ribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine ribonucleoside triphosphate breakdown" EXACT []
 synonym: "purine ribonucleoside triphosphate catabolism" EXACT []
 synonym: "purine ribonucleoside triphosphate degradation" EXACT []
@@ -97367,7 +97696,7 @@
 id: GO:0009208
 name: pyrimidine ribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009147 ! pyrimidine nucleoside triphosphate metabolic process
 
@@ -97375,7 +97704,7 @@
 id: GO:0009209
 name: pyrimidine ribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside triphosphate anabolism" EXACT []
 synonym: "pyrimidine ribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "pyrimidine ribonucleoside triphosphate formation" EXACT []
@@ -97388,7 +97717,7 @@
 id: GO:0009210
 name: pyrimidine ribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine ribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a ribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleoside triphosphate breakdown" EXACT []
 synonym: "pyrimidine ribonucleoside triphosphate catabolism" EXACT []
 synonym: "pyrimidine ribonucleoside triphosphate degradation" EXACT []
@@ -97400,7 +97729,7 @@
 id: GO:0009211
 name: pyrimidine deoxyribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009147 ! pyrimidine nucleoside triphosphate metabolic process
 
@@ -97408,7 +97737,7 @@
 id: GO:0009212
 name: pyrimidine deoxyribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside triphosphate anabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "pyrimidine deoxyribonucleoside triphosphate formation" EXACT []
@@ -97420,7 +97749,7 @@
 id: GO:0009213
 name: pyrimidine deoxyribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of pyrimidine deoxyribonucleoside triphosphate, a compound consisting of a pyrimidine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleoside triphosphate breakdown" EXACT []
 synonym: "pyrimidine deoxyribonucleoside triphosphate catabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleoside triphosphate degradation" EXACT []
@@ -97431,7 +97760,7 @@
 id: GO:0009214
 name: cyclic nucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:curators, ISBN:0198506732]
 synonym: "cyclic nucleotide breakdown" EXACT []
 synonym: "cyclic nucleotide catabolism" EXACT []
 synonym: "cyclic nucleotide degradation" EXACT []
@@ -97442,7 +97771,7 @@
 id: GO:0009215
 name: purine deoxyribonucleoside triphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside triphosphate metabolism" EXACT []
 is_a: GO:0009144 ! purine nucleoside triphosphate metabolic process
 is_a: GO:0009200 ! deoxyribonucleoside triphosphate metabolic process
@@ -97451,7 +97780,7 @@
 id: GO:0009216
 name: purine deoxyribonucleoside triphosphate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside triphosphate anabolism" EXACT []
 synonym: "purine deoxyribonucleoside triphosphate biosynthesis" EXACT []
 synonym: "purine deoxyribonucleoside triphosphate formation" EXACT []
@@ -97464,7 +97793,7 @@
 id: GO:0009217
 name: purine deoxyribonucleoside triphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of purine deoxyribonucleoside triphosphate, a compound consisting of a purine base linked to a deoxyribose sugar esterified with triphosphate on the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "purine deoxyribonucleoside triphosphate breakdown" EXACT []
 synonym: "purine deoxyribonucleoside triphosphate catabolism" EXACT []
 synonym: "purine deoxyribonucleoside triphosphate degradation" EXACT []
@@ -97476,7 +97805,7 @@
 id: GO:0009218
 name: pyrimidine ribonucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleotide metabolism" EXACT []
 is_a: GO:0006220 ! pyrimidine nucleotide metabolic process
 is_a: GO:0009259 ! ribonucleotide metabolic process
@@ -97485,7 +97814,7 @@
 id: GO:0009219
 name: pyrimidine deoxyribonucleotide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving a pyrimidine deoxynucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a pyrimidine deoxynucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleotide metabolism" EXACT []
 is_a: GO:0006220 ! pyrimidine nucleotide metabolic process
 is_a: GO:0009394 ! 2'-deoxyribonucleotide metabolic process
@@ -97494,7 +97823,7 @@
 id: GO:0009220
 name: pyrimidine ribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleotide anabolism" EXACT []
 synonym: "pyrimidine ribonucleotide biosynthesis" EXACT []
 synonym: "pyrimidine ribonucleotide formation" EXACT []
@@ -97508,7 +97837,7 @@
 id: GO:0009221
 name: pyrimidine deoxyribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a pyrimidine deoxyribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a pyrimidine deoxyribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleotide anabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleotide biosynthesis" EXACT []
 synonym: "pyrimidine deoxyribonucleotide formation" EXACT []
@@ -97522,7 +97851,7 @@
 id: GO:0009222
 name: pyrimidine ribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine ribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine ribonucleotide breakdown" EXACT []
 synonym: "pyrimidine ribonucleotide catabolism" EXACT []
 synonym: "pyrimidine ribonucleotide degradation" EXACT []
@@ -97534,7 +97863,7 @@
 id: GO:0009223
 name: pyrimidine deoxyribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine deoxyribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a pyrimidine deoxyribonucleotide, a compound consisting of nucleoside (a pyrimidine base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "pyrimidine deoxyribonucleotide breakdown" EXACT []
 synonym: "pyrimidine deoxyribonucleotide catabolism" EXACT []
 synonym: "pyrimidine deoxyribonucleotide degradation" EXACT []
@@ -97717,7 +98046,7 @@
 id: GO:0009235
 name: cobalamin metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom." [GOC:go_curators]
+def: "The chemical reactions and pathways involving cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom." [GOC:curators]
 synonym: "cobalamin metabolism" EXACT []
 synonym: "vitamin B12 metabolic process" EXACT []
 synonym: "vitamin B12 metabolism" EXACT []
@@ -97768,7 +98097,7 @@
 namespace: biological_process
 alt_id: GO:0031191
 alt_id: GO:0031192
-def: "The chemical reactions and pathways resulting in the formation of enterobactin, a catechol-derived siderochrome of Enterobacteria; enterobactin (N',N',N''-(2,6,10-trioxo-1,5,9-triacyclodecane-3,7,11-triyl)tris(2,3-dihydroxy)benzamide) is a self-triester of 2,3-dihydroxy-N-benzoyl-L-serine and a product of the shikimate pathway." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of enterobactin, a catechol-derived siderochrome of Enterobacteria; enterobactin (N',N',N''-(2,6,10-trioxo-1,5,9-triacyclodecane-3,7,11-triyl)tris(2,3-dihydroxy)benzamide) is a self-triester of 2,3-dihydroxy-N-benzoyl-L-serine and a product of the shikimate pathway." [GOC:curators]
 synonym: "enterobactin anabolism" EXACT []
 synonym: "enterobactin biosynthesis" EXACT []
 synonym: "enterobactin biosynthetic process, peptide formation" NARROW []
@@ -97873,7 +98202,7 @@
 id: GO:0009247
 name: glycolipid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of glycolipid, a class of 1,2-di-O-acylglycerols joined at oxygen 3 by a glycosidic linkage to a carbohydrate part (usually a mono-, di- or tri-saccharide)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of glycolipid, a class of 1,2-di-O-acylglycerols joined at oxygen 3 by a glycosidic linkage to a carbohydrate part (usually a mono-, di- or tri-saccharide)." [GOC:curators]
 synonym: "glycolipid anabolism" EXACT []
 synonym: "glycolipid biosynthesis" EXACT []
 synonym: "glycolipid formation" EXACT []
@@ -97914,7 +98243,7 @@
 id: GO:0009250
 name: glucan biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of glucans, polysaccharides consisting only of glucose residues." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of glucans, polysaccharides consisting only of glucose residues." [GOC:curators]
 synonym: "glucan anabolism" EXACT []
 synonym: "glucan biosynthesis" EXACT []
 synonym: "glucan formation" EXACT []
@@ -97926,7 +98255,7 @@
 id: GO:0009251
 name: glucan catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glucans, polysaccharides consisting only of glucose residues." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glucans, polysaccharides consisting only of glucose residues." [GOC:curators]
 synonym: "glucan breakdown" EXACT []
 synonym: "glucan catabolism" EXACT []
 synonym: "glucan degradation" EXACT []
@@ -98037,7 +98366,7 @@
 name: ribonucleotide metabolic process
 namespace: biological_process
 alt_id: GO:0009121
-def: "The chemical reactions and pathways involving a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleotide metabolism" EXACT []
 is_a: GO:0009117 ! nucleotide metabolic process
 is_a: GO:0019693 ! ribose phosphate metabolic process
@@ -98046,7 +98375,7 @@
 id: GO:0009260
 name: ribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleotide anabolism" EXACT []
 synonym: "ribonucleotide biosynthesis" EXACT []
 synonym: "ribonucleotide formation" EXACT []
@@ -98059,7 +98388,7 @@
 id: GO:0009261
 name: ribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a ribonucleotide, a compound consisting of ribonucleoside (a base linked to a ribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "ribonucleotide breakdown" EXACT []
 synonym: "ribonucleotide catabolism" EXACT []
 synonym: "ribonucleotide degradation" EXACT []
@@ -98073,7 +98402,7 @@
 namespace: biological_process
 alt_id: GO:0009122
 alt_id: GO:0009393
-def: "The chemical reactions and pathways involving a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleotide metabolism" EXACT []
 is_a: GO:0009117 ! nucleotide metabolic process
 is_a: GO:1901135 ! carbohydrate derivative metabolic process
@@ -98082,7 +98411,7 @@
 id: GO:0009263
 name: deoxyribonucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleotide anabolism" EXACT []
 synonym: "deoxyribonucleotide biosynthesis" EXACT []
 synonym: "deoxyribonucleotide formation" EXACT []
@@ -98095,7 +98424,7 @@
 id: GO:0009264
 name: deoxyribonucleotide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of a deoxyribonucleotide, a compound consisting of deoxyribonucleoside (a base linked to a deoxyribose sugar) esterified with a phosphate group at either the 3' or 5'-hydroxyl group of the sugar." [GOC:curators, ISBN:0198506732]
 synonym: "deoxyribonucleotide breakdown" EXACT []
 synonym: "deoxyribonucleotide catabolism" EXACT []
 synonym: "deoxyribonucleotide degradation" EXACT []
@@ -98167,7 +98496,7 @@
 id: GO:0009272
 name: fungal-type cell wall biogenesis
 namespace: biological_process
-def: "A cellular process that results in the biosynthesis of constituent macromolecules, assembly, and arrangement of constituent parts of a fungal-type cell wall. The fungal-type cell wall contains beta-glucan and may contain chitin." [GOC:go_curators, GOC:mtg_sensu]
+def: "A cellular process that results in the biosynthesis of constituent macromolecules, assembly, and arrangement of constituent parts of a fungal-type cell wall. The fungal-type cell wall contains beta-glucan and may contain chitin." [GOC:curators, GOC:mtg_sensu]
 synonym: "chitin- and beta-glucan-containing cell wall biogenesis" NARROW [GOC:mah]
 synonym: "fungal-type cell wall anabolism" RELATED [GOC:mah]
 synonym: "fungal-type cell wall biosynthetic process" RELATED [GOC:mah]
@@ -98180,7 +98509,7 @@
 id: GO:0009273
 name: peptidoglycan-based cell wall biogenesis
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of the peptidoglycan-based cell wall. An example of this process is found in Escherichia coli." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of the peptidoglycan-based cell wall. An example of this process is found in Escherichia coli." [GOC:curators]
 synonym: "cell envelope biosynthesis" BROAD []
 synonym: "cell envelope biosynthetic process" BROAD []
 synonym: "cell wall anabolism" BROAD []
@@ -98586,7 +98915,7 @@
 id: GO:0009315
 name: obsolete drug resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "drug resistance" EXACT []
 is_obsolete: true
@@ -98815,7 +99144,7 @@
 id: GO:0009339
 name: glycolate oxidase complex
 namespace: cellular_component
-def: "An enzyme complex that catalyzes the oxidation of 2-hydroxy acid to form 2-oxo acid and hydrogen peroxide (H2O2). The enzyme is a flavoprotein (FMN)." [PMID:4557653, PMID:8606183]
+def: "An enzyme complex that catalyzes the oxidation of 2-hydroxy acid to form 2-oxo acid and H2O2. The enzyme is a flavoprotein (FMN)." [PMID:4557653, PMID:8606183]
 is_a: GO:1990204 ! oxidoreductase complex
 relationship: part_of GO:0005737 ! cytoplasm
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21246" xsd:anyURI
@@ -99059,7 +99388,7 @@
 id: GO:0009371
 name: obsolete positive regulation of transcription by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that activates or increases the rate of transcription." [GOC:go_curators]
+def: "OBSOLETE. Any process involving pheromones that activates or increases the rate of transcription." [GOC:curators]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 synonym: "activation of transcription by pheromones" NARROW []
 synonym: "stimulation of transcription by pheromones" NARROW []
@@ -99092,7 +99421,7 @@
 id: GO:0009373
 name: obsolete regulation of transcription by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that modulates the frequency, rate or extent of transcription." [GOC:go_curators]
+def: "OBSOLETE. Any process involving pheromones that modulates the frequency, rate or extent of transcription." [GOC:curators]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12739" xsd:anyURI
 is_obsolete: true
@@ -99256,10 +99585,18 @@
 id: GO:0009389
 name: dimethyl sulfoxide reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dimethyl sulfoxide + H+ = dimethyl sulfide + H2O." [PMID:8658134]
+def: "Catalysis of the reaction: dimethyl sulfide + a menaquinone + H2O = dimethyl sulfoxide + a menaquinol." [PMID:8658134, RHEA:28494]
 synonym: "dimethyl sulphoxide reductase activity" EXACT []
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+synonym: "respiratory dimethylsulfoxide reductase" EXACT [EC:1.8.5.3]
+xref: EC:1.8.5.3
+xref: MetaCyc:DIMESULFREDUCT-RXN
+xref: RHEA:28494
+is_a: GO:0016672 ! oxidoreductase activity, acting on a sulfur group of donors, quinone or similar compound as acceptor
+property_value: skos:exactMatch EC:1.8.5.3
+property_value: skos:exactMatch MetaCyc:DIMESULFREDUCT-RXN
+property_value: skos:exactMatch RHEA:28494
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28261" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0009390
@@ -99401,7 +99738,7 @@
 id: GO:0009403
 name: toxin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of toxin, a poisonous compound (typically a protein) that is produced by cells or organisms and that can cause disease when introduced into the body or tissues of an organism." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of toxin, a poisonous compound (typically a protein) that is produced by cells or organisms and that can cause disease when introduced into the body or tissues of an organism." [GOC:curators]
 subset: goslim_metagenomics
 synonym: "toxin anabolism" EXACT []
 synonym: "toxin biosynthesis" EXACT []
@@ -99423,7 +99760,7 @@
 id: GO:0009405
 name: obsolete pathogenesis
 namespace: biological_process
-def: "OBSOLETE. The set of specific processes that generate the ability of an organism to induce an abnormal, generally detrimental state in another organism." [GOC:go_curators]
+def: "OBSOLETE. The set of specific processes that generate the ability of an organism to induce an abnormal, generally detrimental state in another organism." [GOC:curators]
 comment: This term was obsoleted because it does not describe a single, normal biological process; rather it is the effect of an interaction between two organisms, under specific conditions.
 synonym: "virulence" RELATED []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/11057" xsd:anyURI
@@ -99436,7 +99773,7 @@
 id: GO:0009406
 name: obsolete virulence
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a phenotype rather than a biological process.
 synonym: "virulence" EXACT []
 is_obsolete: true
@@ -99447,7 +99784,7 @@
 id: GO:0009407
 name: toxin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of toxin, a poisonous compound (typically a protein) that is produced by cells or organisms and that can cause disease when introduced into the body or tissues of an organism." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of toxin, a poisonous compound (typically a protein) that is produced by cells or organisms and that can cause disease when introduced into the body or tissues of an organism." [GOC:curators]
 synonym: "toxin breakdown" EXACT []
 synonym: "toxin catabolism" EXACT []
 synonym: "toxin degradation" EXACT []
@@ -99546,7 +99883,7 @@
 id: GO:0009416
 name: response to light stimulus
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a light stimulus, electromagnetic radiation of wavelengths classified as infrared, visible or ultraviolet light." [GOC:go_curators, ISBN:0582227089]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a light stimulus, electromagnetic radiation of wavelengths classified as infrared, visible or ultraviolet light." [GOC:curators, ISBN:0582227089]
 subset: goslim_plant
 is_a: GO:0009314 ! response to radiation
 
@@ -99858,7 +100195,7 @@
 id: GO:0009446
 name: putrescine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of putrescine, 1,4-diaminobutane; putrescine can be synthesized from arginine or ornithine and is the metabolic precursor of spermidine and spermine." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of putrescine, 1,4-diaminobutane; putrescine can be synthesized from arginine or ornithine and is the metabolic precursor of spermidine and spermine." [GOC:curators, ISBN:0198506732]
 synonym: "putrescine anabolism" EXACT []
 synonym: "putrescine biosynthesis" EXACT []
 synonym: "putrescine formation" EXACT []
@@ -99918,7 +100255,7 @@
 id: GO:0009450
 name: gamma-aminobutyric acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of gamma-aminobutyric acid (GABA, 4-aminobutyrate), an amino acid which acts as a neurotransmitter in some organisms." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of gamma-aminobutyric acid (GABA)." [GOC:curators]
 comment: See also the biological process term 'neurotransmitter catabolic process ; GO:0042135'.
 synonym: "4-aminobutanoate catabolic process" EXACT []
 synonym: "4-aminobutanoate catabolism" EXACT []
@@ -99944,13 +100281,14 @@
 property_value: skos:narrowMatch MetaCyc:PWY-6536
 property_value: skos:narrowMatch MetaCyc:PWY-6537
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0009451
 name: RNA modification
 namespace: biological_process
 alt_id: GO:0016547
-def: "The covalent alteration of one or more nucleotides within an RNA molecule to produce an RNA molecule with a sequence that differs from that coded genetically." [GOC:go_curators, ISBN:1555811337]
+def: "The covalent alteration of one or more nucleotides within an RNA molecule to produce an RNA molecule with a sequence that differs from that coded genetically." [GOC:curators, ISBN:1555811337]
 comment: The term 'RNA editing' (GO:0016547) was merged into 'RNA modification' (GO:0009451) on the basis of statements in the preface of Modification and Editing of RNA (ISBN:1555811337) that there is no clear distinction between modification and editing.
 subset: goslim_yeast
 synonym: "RNA editing" NARROW [GOC:hjd]
@@ -101024,6 +101362,8 @@
 name: fertilization
 namespace: biological_process
 def: "The union of gametes of opposite sexes during the process of sexual reproduction to form a zygote. It involves the fusion of the gametic nuclei (karyogamy) and cytoplasm (plasmogamy)." [GOC:tb, ISBN:0198506732]
+synonym: "cell-cell fusion" RELATED []
+synonym: "https://github.com/geneontology/go-ontology/issues/31270" RELATED []
 synonym: "syngamy" EXACT []
 xref: Wikipedia:Fertilisation
 is_a: GO:0022414 ! reproductive process
@@ -101152,7 +101492,7 @@
 id: GO:0009580
 name: obsolete thylakoid (sensu Bacteria)
 namespace: cellular_component
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific children exist.
 synonym: "thylakoid (sensu Bacteria)" EXACT []
 is_obsolete: true
@@ -101181,7 +101521,7 @@
 id: GO:0009583
 name: detection of light stimulus
 namespace: biological_process
-def: "The series of events in which a light stimulus (in the form of photons) is received and converted into a molecular signal." [GOC:go_curators]
+def: "The series of events in which a light stimulus (in the form of photons) is received and converted into a molecular signal." [GOC:curators]
 synonym: "detection of light" EXACT []
 synonym: "perception of light" RELATED []
 is_a: GO:0009416 ! response to light stimulus
@@ -101192,7 +101532,7 @@
 id: GO:0009584
 name: detection of visible light
 namespace: biological_process
-def: "The series of events in which a visible light stimulus is received by a cell and converted into a molecular signal. A visible light stimulus is electromagnetic radiation that can be perceived visually by an organism; for organisms lacking a visual system, this can be defined as light with a wavelength within the range 380 to 780 nm." [GOC:go_curators, ISBN:0198506732]
+def: "The series of events in which a visible light stimulus is received by a cell and converted into a molecular signal. A visible light stimulus is electromagnetic radiation that can be perceived visually by an organism; for organisms lacking a visual system, this can be defined as light with a wavelength within the range 380 to 780 nm." [GOC:curators, ISBN:0198506732]
 synonym: "perception of visible light" RELATED []
 is_a: GO:0009583 ! detection of light stimulus
 
@@ -101210,7 +101550,7 @@
 id: GO:0009587
 name: obsolete phototrophin mediated phototransduction
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 synonym: "phototrophin mediated phototransduction" EXACT []
 is_obsolete: true
 replaced_by: GO:0007602
@@ -101229,7 +101569,7 @@
 id: GO:0009589
 name: detection of UV
 namespace: biological_process
-def: "The series of events in which an ultraviolet radiation (UV light) stimulus is received and converted into a molecular signal. Ultraviolet radiation is electromagnetic radiation with a wavelength in the range of 10 to 380 nanometers." [GOC:dos, GOC:go_curators, GOC:hb, ISBN:0198506732]
+def: "The series of events in which an ultraviolet radiation (UV light) stimulus is received and converted into a molecular signal. Ultraviolet radiation is electromagnetic radiation with a wavelength in the range of 10 to 380 nanometers." [GOC:curators, GOC:dos, GOC:hb, ISBN:0198506732]
 synonym: "detection of ultraviolet light stimulus" EXACT []
 synonym: "detection of ultraviolet radiation stimulus" EXACT []
 synonym: "detection of UV light stimulus" EXACT []
@@ -101419,7 +101759,7 @@
 name: response to wounding
 namespace: biological_process
 alt_id: GO:0002245
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism." [GOC:curators]
 synonym: "physiological response to wounding" EXACT []
 is_a: GO:0006950 ! response to stress
 
@@ -101437,7 +101777,7 @@
 id: GO:0009614
 name: obsolete disease resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "disease resistance" EXACT []
 is_obsolete: true
@@ -101487,7 +101827,7 @@
 id: GO:0009619
 name: obsolete resistance to pathogenic bacteria
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "resistance to pathogenic bacteria" EXACT []
 is_obsolete: true
@@ -101506,7 +101846,7 @@
 id: GO:0009622
 name: obsolete resistance to pathogenic fungi
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'resistance' implies a phenotype rather than a biological process.
 synonym: "resistance to pathogenic fungi" EXACT []
 is_obsolete: true
@@ -101593,7 +101933,7 @@
 id: GO:0009632
 name: obsolete freezing tolerance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'tolerance' implies a phenotype rather than a biological process.
 synonym: "freezing tolerance" EXACT []
 is_obsolete: true
@@ -101603,7 +101943,7 @@
 id: GO:0009633
 name: obsolete drought tolerance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'tolerance' implies a phenotype rather than a biological process.
 synonym: "drought tolerance" EXACT []
 is_obsolete: true
@@ -101614,7 +101954,7 @@
 id: GO:0009634
 name: obsolete heavy metal sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "heavy metal sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -101688,7 +102028,7 @@
 id: GO:0009642
 name: response to light intensity
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a light intensity stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a light intensity stimulus." [GOC:curators]
 is_a: GO:0009416 ! response to light stimulus
 
 [Term]
@@ -101704,21 +102044,21 @@
 id: GO:0009644
 name: response to high light intensity
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a high light intensity stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a high light intensity stimulus." [GOC:curators]
 is_a: GO:0009642 ! response to light intensity
 
 [Term]
 id: GO:0009645
 name: response to low light intensity stimulus
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a low light intensity stimulus. Low light intensity is defined as a level of electromagnetic radiation at or below 0.1 micromols/m2." [GOC:go_curators, GOC:mtg_far_red]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a low light intensity stimulus. Low light intensity is defined as a level of electromagnetic radiation at or below 0.1 micromols/m2." [GOC:curators, GOC:mtg_far_red]
 is_a: GO:0009642 ! response to light intensity
 
 [Term]
 id: GO:0009646
 name: response to absence of light
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an absence of light stimuli." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an absence of light stimuli." [GOC:curators]
 synonym: "response to darkness" RELATED []
 is_a: GO:0009642 ! response to light intensity
 
@@ -101785,7 +102125,7 @@
 id: GO:0009653
 name: anatomical structure morphogenesis
 namespace: biological_process
-def: "The process in which anatomical structures are generated and organized. Morphogenesis pertains to the creation of form." [GOC:go_curators, ISBN:0521436125]
+def: "The process in which anatomical structures are generated and organized. Morphogenesis pertains to the creation of form." [GOC:curators, ISBN:0521436125]
 synonym: "anatomical structure organization" EXACT []
 synonym: "embryogenesis and morphogenesis" BROAD []
 synonym: "morphogenesis" EXACT []
@@ -102461,7 +102801,7 @@
 id: GO:0009713
 name: catechol-containing compound biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of catechol-containing compounds. Catechol is a compound containing a pyrocatechol nucleus or substituent." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of catechol-containing compounds. Catechol is a compound containing a pyrocatechol nucleus or substituent." [GOC:curators]
 synonym: "catechol anabolism" RELATED []
 synonym: "catechol biosynthesis" RELATED []
 synonym: "catechol biosynthetic process" RELATED []
@@ -102484,7 +102824,7 @@
 id: GO:0009715
 name: chalcone biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of chalcone, phenyl steryl ketone or its hydroxylated derivatives." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of chalcone, phenyl steryl ketone or its hydroxylated derivatives." [GOC:curators]
 synonym: "chalcone anabolism" EXACT []
 synonym: "chalcone biosynthesis" EXACT []
 synonym: "chalcone formation" EXACT []
@@ -103174,7 +103514,7 @@
 name: embryo development
 namespace: biological_process
 alt_id: GO:0009795
-def: "The process whose specific outcome is the progression of an embryo from its formation until the end of its embryonic life stage. The end of the embryonic stage is organism-specific. For example, for mammals, the process would begin with zygote formation and end with birth. For insects, the process would begin at zygote formation and end with larval hatching. For plant zygotic embryos, this would be from zygote formation to the end of seed dormancy. For plant vegetative embryos, this would be from the initial determination of the cell or group of cells to form an embryo until the point when the embryo becomes independent of the parent plant." [GOC:go_curators, GOC:isa_complete, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of an embryo from its formation until the end of its embryonic life stage. The end of the embryonic stage is organism-specific. For example, for mammals, the process would begin with zygote formation and end with birth. For insects, the process would begin at zygote formation and end with larval hatching. For plant zygotic embryos, this would be from zygote formation to the end of seed dormancy. For plant vegetative embryos, this would be from the initial determination of the cell or group of cells to form an embryo until the point when the embryo becomes independent of the parent plant." [GOC:curators, GOC:isa_complete, GOC:mtg_sensu]
 subset: gocheck_do_not_annotate
 subset: goslim_chembl
 subset: goslim_plant
@@ -103189,7 +103529,7 @@
 id: GO:0009791
 name: post-embryonic development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the organism over time, from the completion of embryonic development to the mature structure. See embryonic development." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the organism over time, from the completion of embryonic development to the mature structure. See embryonic development." [GOC:curators]
 subset: goslim_plant
 is_a: GO:0032501 ! multicellular organismal process
 relationship: part_of GO:0007275 ! multicellular organism development
@@ -103198,7 +103538,7 @@
 id: GO:0009792
 name: embryo development ending in birth or egg hatching
 namespace: biological_process
-def: "The process whose specific outcome is the progression of an embryo over time, from zygote formation until the end of the embryonic life stage. The end of the embryonic life stage is organism-specific and may be somewhat arbitrary; for mammals it is usually considered to be birth, for insects the hatching of the first instar larva from the eggshell." [GOC:go_curators, GOC:isa_complete, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of an embryo over time, from zygote formation until the end of the embryonic life stage. The end of the embryonic life stage is organism-specific and may be somewhat arbitrary; for mammals it is usually considered to be birth, for insects the hatching of the first instar larva from the eggshell." [GOC:curators, GOC:isa_complete, GOC:mtg_sensu]
 synonym: "embryogenesis" BROAD []
 is_a: GO:0009790 ! embryo development
 
@@ -103206,7 +103546,7 @@
 id: GO:0009793
 name: embryo development ending in seed dormancy
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the embryo over time, from zygote formation to the end of seed dormancy. An example of this process is found in Arabidopsis thaliana." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of the embryo over time, from zygote formation to the end of seed dormancy. An example of this process is found in Arabidopsis thaliana." [GOC:curators, GOC:mtg_sensu]
 synonym: "embryogenesis" BROAD []
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0009790 ! embryo development
@@ -103216,7 +103556,7 @@
 id: GO:0009794
 name: regulation of mitotic cell cycle, embryonic
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of replication and segregation of genetic material in the embryo." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of replication and segregation of genetic material in the embryo." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "embryonic mitotic cell cycle modulation" EXACT []
 synonym: "embryonic mitotic cell cycle regulation" EXACT []
 synonym: "embryonic mitotic cell cycle regulator" RELATED []
@@ -103233,7 +103573,7 @@
 id: GO:0009798
 name: axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of a pattern along a line or around a point." [GOC:dph, GOC:go_curators, GOC:isa_complete]
+def: "The establishment, maintenance and elaboration of a pattern along a line or around a point." [GOC:curators, GOC:dph, GOC:isa_complete]
 synonym: "axis determination" RELATED []
 is_a: GO:0007389 ! pattern specification process
 
@@ -103241,7 +103581,7 @@
 id: GO:0009799
 name: specification of symmetry
 namespace: biological_process
-def: "The establishment of an organism's body plan or part of an organism such that a similar arrangement in form and relationship of parts around a common axis, or around each side of a plane is created." [GOC:go_curators]
+def: "The establishment of an organism's body plan or part of an organism such that a similar arrangement in form and relationship of parts around a common axis, or around each side of a plane is created." [GOC:curators]
 synonym: "determination of symmetry" EXACT [GOC:dph]
 is_a: GO:0007389 ! pattern specification process
 
@@ -103482,8 +103822,11 @@
 synonym: "cytokinin breakdown" EXACT []
 synonym: "cytokinin catabolism" EXACT []
 synonym: "cytokinin degradation" EXACT []
+xref: MetaCyc:PWY-2841
 is_a: GO:0009690 ! cytokinin metabolic process
 is_a: GO:0042447 ! hormone catabolic process
+property_value: skos:exactMatch MetaCyc:PWY-2841
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0009824
@@ -103589,7 +103932,7 @@
 id: GO:0009832
 name: plant-type cell wall biogenesis
 namespace: biological_process
-def: "A cellular process that results in the biosynthesis of constituent macromolecules, assembly, and arrangement of constituent parts of a cellulose and pectin-containing cell wall. An example of this is found in Arabidopsis thaliana." [GOC:go_curators, GOC:lr, GOC:mtg_sensu]
+def: "A cellular process that results in the biosynthesis of constituent macromolecules, assembly, and arrangement of constituent parts of a cellulose and pectin-containing cell wall. An example of this is found in Arabidopsis thaliana." [GOC:curators, GOC:lr, GOC:mtg_sensu]
 synonym: "cell wall anabolism" BROAD []
 synonym: "cell wall assembly" BROAD []
 synonym: "cell wall biosynthetic process" BROAD []
@@ -103767,7 +104110,7 @@
 id: GO:0009849
 name: tryptophan-independent indoleacetic acid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of indoleacetic acid, independent of tryptophan." [GOC:go_curators, GOC:lm, GOC:lr, PMID:10375566]
+def: "The chemical reactions and pathways resulting in the formation of indoleacetic acid, independent of tryptophan." [GOC:curators, GOC:lm, GOC:lr, PMID:10375566]
 synonym: "indoleacetic acid biosynthesis, tryptophan-independent" EXACT []
 synonym: "indoleacetic acid biosynthetic process, tryptophan-independent" EXACT []
 synonym: "tryptophan-independent IAA biosynthetic process" EXACT []
@@ -103831,7 +104174,7 @@
 id: GO:0009855
 name: determination of bilateral symmetry
 namespace: biological_process
-def: "The establishment of an organism's body plan or part of an organism with respect to a single longitudinal plane. The pattern can either be symmetric, such that the halves are mirror images, or asymmetric where the pattern deviates from this symmetry." [GOC:go_curators]
+def: "The establishment of an organism's body plan or part of an organism with respect to a single longitudinal plane. The pattern can either be symmetric, such that the halves are mirror images, or asymmetric where the pattern deviates from this symmetry." [GOC:curators]
 synonym: "determination of bilateral asymmetry" EXACT [GOC:dph]
 is_a: GO:0009799 ! specification of symmetry
 
@@ -104067,7 +104410,7 @@
 id: GO:0009879
 name: determination of radial symmetry
 namespace: biological_process
-def: "The establishment of an organism's body plan or a part of an organism such that it is symmetric around a central axis." [GOC:go_curators]
+def: "The establishment of an organism's body plan or a part of an organism such that it is symmetric around a central axis." [GOC:curators]
 synonym: "determination of radial asymmetry" EXACT [GOC:dph]
 is_a: GO:0009799 ! specification of symmetry
 relationship: part_of GO:0009956 ! radial pattern formation
@@ -104076,7 +104419,7 @@
 id: GO:0009880
 name: embryonic pattern specification
 namespace: biological_process
-def: "The process that results in the patterns of cell differentiation that will arise in an embryo." [GOC:go_curators, ISBN:0521436125]
+def: "The process that results in the patterns of cell differentiation that will arise in an embryo." [GOC:curators, ISBN:0521436125]
 synonym: "embryonic pattern biosynthesis" BROAD []
 synonym: "embryonic pattern formation" BROAD []
 synonym: "ventral/lateral system" RELATED []
@@ -104087,7 +104430,7 @@
 id: GO:0009881
 name: photoreceptor activity
 namespace: molecular_function
-def: "The function of absorbing and responding to incidental electromagnetic radiation, particularly visible light. The response may involve a change in conformation." [GOC:ai, GOC:go_curators]
+def: "The function of absorbing and responding to incidental electromagnetic radiation, particularly visible light. The response may involve a change in conformation." [GOC:ai, GOC:curators]
 synonym: "blue-sensitive opsin" NARROW []
 synonym: "green-sensitive opsin" NARROW []
 synonym: "long-wave-sensitive opsin" NARROW []
@@ -104135,7 +104478,7 @@
 id: GO:0009886
 name: post-embryonic animal morphogenesis
 namespace: biological_process
-def: "The process, occurring after animal embryonic development, by which anatomical structures are generated and organized." [GOC:go_curators]
+def: "The process, occurring after animal embryonic development, by which anatomical structures are generated and organized." [GOC:curators]
 synonym: "post-embryonic morphogenesis of an anatomical structure" EXACT []
 is_a: GO:0009653 ! anatomical structure morphogenesis
 relationship: part_of GO:0009791 ! post-embryonic development
@@ -104144,7 +104487,7 @@
 id: GO:0009887
 name: animal organ morphogenesis
 namespace: biological_process
-def: "Morphogenesis of an animal organ. An organ is defined as a tissue or set of tissues that work together to perform a specific function or functions. Morphogenesis is the process in which anatomical structures are generated and organized. Organs are commonly observed as visibly distinct structures, but may also exist as loosely associated clusters of cells that work together to perform a specific function or functions." [GOC:dgh, GOC:go_curators, ISBN:0471245208, ISBN:0721662544]
+def: "Morphogenesis of an animal organ. An organ is defined as a tissue or set of tissues that work together to perform a specific function or functions. Morphogenesis is the process in which anatomical structures are generated and organized. Organs are commonly observed as visibly distinct structures, but may also exist as loosely associated clusters of cells that work together to perform a specific function or functions." [GOC:curators, GOC:dgh, ISBN:0471245208, ISBN:0721662544]
 synonym: "histogenesis and organogenesis" BROAD []
 is_a: GO:0009653 ! anatomical structure morphogenesis
 relationship: part_of GO:0048513 ! animal organ development
@@ -104163,7 +104506,7 @@
 id: GO:0009889
 name: regulation of biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of substances." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of substances." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "regulation of anabolism" EXACT []
 synonym: "regulation of biosynthesis" EXACT []
@@ -104178,7 +104521,7 @@
 id: GO:0009890
 name: negative regulation of biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the rate of the chemical reactions and pathways resulting in the formation of substances." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the rate of the chemical reactions and pathways resulting in the formation of substances." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "down regulation of biosynthetic process" EXACT []
 synonym: "down-regulation of biosynthetic process" EXACT []
@@ -104199,7 +104542,7 @@
 id: GO:0009891
 name: positive regulation of biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of substances." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of substances." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "activation of biosynthetic process" NARROW []
 synonym: "positive regulation of anabolism" EXACT []
@@ -104222,7 +104565,7 @@
 name: negative regulation of metabolic process
 namespace: biological_process
 alt_id: GO:0044252
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "down regulation of metabolic process" EXACT []
 synonym: "down-regulation of metabolic process" EXACT []
@@ -104244,7 +104587,7 @@
 name: positive regulation of metabolic process
 namespace: biological_process
 alt_id: GO:0044253
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "activation of metabolic process" NARROW []
 synonym: "positive regulation of metabolism" EXACT []
@@ -104256,6 +104599,7 @@
 synonym: "up-regulation of metabolic process" EXACT []
 synonym: "up-regulation of organismal metabolic process" NARROW []
 synonym: "upregulation of metabolic process" EXACT []
+xref: Reactome:R-HSA-163765 "ChREBP activates metabolic gene expression"
 is_a: GO:0019222 ! regulation of metabolic process
 is_a: GO:0048522 ! positive regulation of cellular process
 intersection_of: GO:0065007 ! biological regulation
@@ -104267,7 +104611,7 @@
 id: GO:0009894
 name: regulation of catabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate, or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate, or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "regulation of breakdown" EXACT []
 synonym: "regulation of catabolism" EXACT []
@@ -104281,7 +104625,7 @@
 id: GO:0009895
 name: negative regulation of catabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:curators]
 synonym: "down regulation of catabolic process" EXACT []
 synonym: "down-regulation of catabolic process" EXACT []
 synonym: "downregulation of catabolic process" EXACT []
@@ -104299,7 +104643,7 @@
 id: GO:0009896
 name: positive regulation of catabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of substances." [GOC:curators]
 synonym: "activation of catabolic process" NARROW []
 synonym: "positive regulation of breakdown" EXACT []
 synonym: "positive regulation of catabolism" EXACT []
@@ -104318,24 +104662,26 @@
 id: GO:0009897
 name: external side of plasma membrane
 namespace: cellular_component
-def: "The leaflet of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
+def: "The leaflet of the plasma membrane that faces the extracellular side of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, GOC:tb]
 synonym: "external leaflet of plasma membrane" EXACT [GOC:ab]
 synonym: "juxtamembrane" BROAD []
 synonym: "outer surface of cytoplasmic membrane" EXACT []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0005886 ! plasma membrane
 relationship: part_of GO:0009986 ! cell surface
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0009898
 name: cytoplasmic side of plasma membrane
 namespace: cellular_component
-def: "The leaflet the plasma membrane that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
+def: "The leaflet of the plasma membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, GOC:tb]
 synonym: "internal leaflet of plasma membrane" EXACT [GOC:ab]
 synonym: "internal side of plasma membrane" EXACT []
 synonym: "juxtamembrane" BROAD []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005886 ! plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0009899
@@ -104419,7 +104765,7 @@
 id: GO:0009906
 name: response to photoperiod, blue light
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of the detection of a blue light photoperiod stimulus. Blue light is electromagnetic radiation with a wavelength of between 440 and 500nm." [GOC:go_curators, GOC:mtg_far_red]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of the detection of a blue light photoperiod stimulus. Blue light is electromagnetic radiation with a wavelength of between 440 and 500nm." [GOC:curators, GOC:mtg_far_red]
 is_a: GO:0009637 ! response to blue light
 is_a: GO:0009648 ! photoperiodism
 
@@ -104427,7 +104773,7 @@
 id: GO:0009907
 name: response to photoperiod, red light
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a red light photoperiod stimulus. Red light is electromagnetic radiation of wavelength of 580-700nm." [GOC:go_curators, GOC:mtg_far_red]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a red light photoperiod stimulus. Red light is electromagnetic radiation of wavelength of 580-700nm." [GOC:curators, GOC:mtg_far_red]
 is_a: GO:0009648 ! photoperiodism
 is_a: GO:0010114 ! response to red light
 
@@ -104445,7 +104791,7 @@
 id: GO:0009909
 name: regulation of flower development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of flower development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of flower development." [GOC:curators]
 is_a: GO:0048580 ! regulation of post-embryonic development
 is_a: GO:0048831 ! regulation of shoot system development
 is_a: GO:2000241 ! regulation of reproductive process
@@ -104457,7 +104803,7 @@
 id: GO:0009910
 name: negative regulation of flower development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of flower development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of flower development." [GOC:curators]
 synonym: "down regulation of flower development" EXACT []
 synonym: "down-regulation of flower development" EXACT []
 synonym: "downregulation of flower development" EXACT []
@@ -104473,7 +104819,7 @@
 id: GO:0009911
 name: positive regulation of flower development
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of flower development." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of flower development." [GOC:curators]
 synonym: "activation of flower development" NARROW []
 synonym: "stimulation of flower development" NARROW []
 synonym: "up regulation of flower development" EXACT []
@@ -104500,7 +104846,7 @@
 name: epidermal cell differentiation
 namespace: biological_process
 alt_id: GO:0043355
-def: "The process in which a relatively unspecialized cell acquires specialized features of an epidermal cell, any of the cells making up the epidermis." [GOC:dph, GOC:go_curators, GOC:mtg_sensu, GOC:sdb_2009, GOC:tb]
+def: "The process in which a relatively unspecialized cell acquires specialized features of an epidermal cell, any of the cells making up the epidermis." [GOC:curators, GOC:dph, GOC:mtg_sensu, GOC:sdb_2009, GOC:tb]
 synonym: "hypodermal cell differentiation" RELATED [GOC:kmv, GOC:rk]
 is_a: GO:0030855 ! epithelial cell differentiation
 relationship: part_of GO:0008544 ! epidermis development
@@ -104636,7 +104982,7 @@
 id: GO:0009925
 name: basal plasma membrane
 namespace: cellular_component
-def: "The region of the plasma membrane located at the basal end of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:go_curators]
+def: "The region of the plasma membrane located at the basal end of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:curators]
 is_a: GO:0098590 ! plasma membrane region
 relationship: part_of GO:0045178 ! basal part of cell
 
@@ -104726,7 +105072,7 @@
 id: GO:0009937
 name: regulation of gibberellic acid mediated signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of gibberellic acid mediated signaling." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of gibberellic acid mediated signaling." [GOC:curators]
 synonym: "regulation of gibberellic acid mediated signalling" EXACT []
 is_a: GO:0009966 ! regulation of signal transduction
 intersection_of: GO:0065007 ! biological regulation
@@ -104768,10 +105114,13 @@
 
 [Term]
 id: GO:0009940
-name: amino-terminal vacuolar sorting propeptide binding
+name: obsolete amino-terminal vacuolar sorting propeptide binding
 namespace: molecular_function
-def: "Binding to an amino terminal propeptide, which functions as a sorting signal to sort away the soluble vacuolar protein from Golgi to lytic vacuole via clathrin-coated vesicles." [GOC:sm, PMID:10871276]
-is_a: GO:0010209 ! vacuolar sorting signal binding
+def: "OBSOLETE. Binding to an amino terminal propeptide, which functions as a sorting signal to sort away the soluble vacuolar protein from Golgi to lytic vacuole via clathrin-coated vesicles." [GOC:sm, PMID:10871276]
+comment: This term was obsolete because it is out of scope for GO; target regions of interactions are not captured.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31410" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0010209
 
 [Term]
 id: GO:0009941
@@ -104813,7 +105162,7 @@
 id: GO:0009945
 name: radial axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of an axis that initiates at a point and radiates outward from the point." [GOC:dph, GOC:go_curators, GOC:isa_complete]
+def: "The establishment, maintenance and elaboration of an axis that initiates at a point and radiates outward from the point." [GOC:curators, GOC:dph, GOC:isa_complete]
 is_a: GO:0009798 ! axis specification
 relationship: part_of GO:0009956 ! radial pattern formation
 
@@ -104821,7 +105170,7 @@
 id: GO:0009946
 name: proximal/distal axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of the proximal/distal axis. The proximal/distal axis is defined by a line that runs from main body (proximal end) of an organism outward (distal end)." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "The establishment, maintenance and elaboration of the proximal/distal axis. The proximal/distal axis is defined by a line that runs from main body (proximal end) of an organism outward (distal end)." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "proximal/distal axis determination" RELATED [GOC:dph]
 synonym: "proximodistal axis specification" EXACT []
 is_a: GO:0009798 ! axis specification
@@ -104841,7 +105190,7 @@
 id: GO:0009948
 name: anterior/posterior axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of the anterior/posterior axis. The anterior-posterior axis is defined by a line that runs from the head or mouth of an organism to the tail or opposite end of the organism." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "The establishment, maintenance and elaboration of the anterior/posterior axis. The anterior-posterior axis is defined by a line that runs from the head or mouth of an organism to the tail or opposite end of the organism." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "anterior/posterior axis determination" RELATED [GOC:dph]
 is_a: GO:0009798 ! axis specification
 relationship: part_of GO:0009952 ! anterior/posterior pattern specification
@@ -104850,7 +105199,7 @@
 id: GO:0009949
 name: polarity specification of anterior/posterior axis
 namespace: biological_process
-def: "Any process resulting in the establishment of polarity along the anterior/posterior axis." [GOC:go_curators]
+def: "Any process resulting in the establishment of polarity along the anterior/posterior axis." [GOC:curators]
 is_a: GO:0065001 ! specification of axis polarity
 relationship: part_of GO:0009948 ! anterior/posterior axis specification
 
@@ -104858,7 +105207,7 @@
 id: GO:0009950
 name: dorsal/ventral axis specification
 namespace: biological_process
-def: "The establishment, maintenance and elaboration of the dorsal/ventral axis. The dorsal/ventral axis is defined by a line that runs orthogonal to both the anterior/posterior and left/right axes. The dorsal end is defined by the upper or back side of an organism. The ventral end is defined by the lower or front side of an organism." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "The establishment, maintenance and elaboration of the dorsal/ventral axis. The dorsal/ventral axis is defined by a line that runs orthogonal to both the anterior/posterior and left/right axes. The dorsal end is defined by the upper or back side of an organism. The ventral end is defined by the lower or front side of an organism." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "dorsal-ventral axis specification" EXACT [GOC:mah]
 synonym: "dorsal/ventral axis determination" RELATED [GOC:dph]
 synonym: "dorsoventral axis specification" EXACT [GOC:mah]
@@ -104869,7 +105218,7 @@
 id: GO:0009951
 name: polarity specification of dorsal/ventral axis
 namespace: biological_process
-def: "Any process resulting in the establishment of polarity along the dorsal/ventral axis." [GOC:go_curators]
+def: "Any process resulting in the establishment of polarity along the dorsal/ventral axis." [GOC:curators]
 synonym: "polarity specification of dorsal-ventral axis" EXACT [GOC:mah]
 synonym: "polarity specification of dorsoventral axis" EXACT [GOC:mah]
 is_a: GO:0065001 ! specification of axis polarity
@@ -104879,7 +105228,7 @@
 id: GO:0009952
 name: anterior/posterior pattern specification
 namespace: biological_process
-def: "The regionalization process in which specific areas of cell differentiation are determined along the anterior-posterior axis. The anterior-posterior axis is defined by a line that runs from the head or mouth of an organism to the tail or opposite end of the organism." [GOC:dph, GOC:go_curators, GOC:isa_complete, GOC:tb]
+def: "The regionalization process in which specific areas of cell differentiation are determined along the anterior-posterior axis. The anterior-posterior axis is defined by a line that runs from the head or mouth of an organism to the tail or opposite end of the organism." [GOC:curators, GOC:dph, GOC:isa_complete, GOC:tb]
 synonym: "anterior/posterior pattern formation" RELATED []
 is_a: GO:0003002 ! regionalization
 
@@ -104887,7 +105236,7 @@
 id: GO:0009953
 name: dorsal/ventral pattern formation
 namespace: biological_process
-def: "The regionalization process in which the areas along the dorsal/ventral axis are established that will lead to differences in cell differentiation. The dorsal/ventral axis is defined by a line that runs orthogonal to both the anterior/posterior and left/right axes. The dorsal end is defined by the upper or back side of an organism. The ventral end is defined by the lower or front side of an organism." [GOC:dph, GOC:go_curators, GOC:isa_complete, GOC:tb]
+def: "The regionalization process in which the areas along the dorsal/ventral axis are established that will lead to differences in cell differentiation. The dorsal/ventral axis is defined by a line that runs orthogonal to both the anterior/posterior and left/right axes. The dorsal end is defined by the upper or back side of an organism. The ventral end is defined by the lower or front side of an organism." [GOC:curators, GOC:dph, GOC:isa_complete, GOC:tb]
 synonym: "dorsal-ventral pattern formation" EXACT [GOC:mah]
 synonym: "dorsal/ventral pattern specification" NARROW []
 synonym: "dorsoventral pattern formation" EXACT [GOC:mah]
@@ -104897,7 +105246,7 @@
 id: GO:0009954
 name: proximal/distal pattern formation
 namespace: biological_process
-def: "The regionalization process in which specific areas of cell differentiation are determined along a proximal/distal axis. The proximal/distal axis is defined by a line that runs from main body (proximal end) of an organism outward (distal end)." [GOC:dph, GOC:go_curators, GOC:isa_complete]
+def: "The regionalization process in which specific areas of cell differentiation are determined along a proximal/distal axis. The proximal/distal axis is defined by a line that runs from main body (proximal end) of an organism outward (distal end)." [GOC:curators, GOC:dph, GOC:isa_complete]
 synonym: "proximal/distal pattern specification" NARROW []
 is_a: GO:0003002 ! regionalization
 
@@ -104913,7 +105262,7 @@
 id: GO:0009956
 name: radial pattern formation
 namespace: biological_process
-def: "The regionalization process that results in defined areas around a point in which specific types of cell differentiation will occur." [GOC:dph, GOC:go_curators, GOC:isa_complete]
+def: "The regionalization process that results in defined areas around a point in which specific types of cell differentiation will occur." [GOC:curators, GOC:dph, GOC:isa_complete]
 synonym: "radial pattern specification" NARROW []
 is_a: GO:0003002 ! regionalization
 
@@ -105019,7 +105368,7 @@
 id: GO:0009965
 name: leaf morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the leaf are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the leaf are generated and organized." [GOC:curators]
 is_a: GO:1905392 ! plant organ morphogenesis
 relationship: part_of GO:0010016 ! shoot system morphogenesis
 relationship: part_of GO:0048366 ! leaf development
@@ -105202,7 +105551,7 @@
 synonym: "delta-pH-dependent protein transporter activity" EXACT []
 synonym: "pH-dependent protein transporter activity" EXACT []
 synonym: "twin-arginine targeting transmembrane transporter activity" EXACT []
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 is_a: GO:0015291 ! secondary active transmembrane transporter activity
 
 [Term]
@@ -105260,11 +105609,13 @@
 alt_id: GO:0016439
 def: "Catalysis of the reaction: a uridine in RNA = a pseudouridine in RNA. Conversion of uridine in an RNA molecule to pseudouridine by rotation of the C1'-N-1 glycosidic bond of uridine in RNA to a C1'-C5." [GOC:mah, RHEA:48348]
 comment: Note that this term should not be confused with 'pseudouridylate synthase activity ; GO:0004730', which refers to the formation of free pseudouridine from uracil and ribose-5-phosphate.
-xref: Reactome:R-HSA-6782381 "PUS1 isoform 2 transforms uridine residues to pseudouridine in the anticodon stems of tRNAs"
-xref: Reactome:R-HSA-6786583 "PUS7 transforms uridine to pseudouridine in tRNAs"
-xref: Reactome:R-HSA-6787566 "PUS1 isoform 1 transforms uridine-27, uridine-28 yielding pseudouridine in tRNA(Lys,Ser)"
+xref: Reactome:R-HSA-6782381 "PUS1 isoform 2 isomerizes uridine residues to pseudouridine residues in the anticodon stems of tRNAs"
+xref: Reactome:R-HSA-6786583 "PUS7 isomerizes uridine to pseudouridine in tRNAs"
+xref: Reactome:R-HSA-6787566 "PUS1 isoform 1 isomerizes uridine-27, uridine-28 to pseudouridine residues in tRNA(Lys,Ser)"
 xref: Reactome:R-HSA-6790905 "Box H/ACA snoRNP transforms uridine to pseudouridine in pre-rRNA"
-xref: Reactome:R-HSA-8870289 "PUS3 transforms uridine-39 to pseudouridine-39 in tRNA"
+xref: Reactome:R-HSA-8870289 "PUS3 isomerizes uridine-39 to pseudouridine-39 in tRNA"
+xref: Reactome:R-HSA-9936243 "RPUSD4 in the mitochondrial pseudouridylation module transforms uridine-1397 to pseudouridine-1397 in mitochondrial 16S rRNA"
+xref: Reactome:R-HSA-9937369 "RPUSD3,TRUB2 pseudouridylates MT-CO1,MT-CO3 mRNA"
 xref: RHEA:48348
 is_a: GO:0016866 ! intramolecular transferase activity
 property_value: skos:exactMatch RHEA:48348
@@ -105320,7 +105671,7 @@
 alt_id: GO:0008151
 alt_id: GO:0044763
 alt_id: GO:0050875
-def: "Any process that is carried out at the cellular level, but not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level." [GOC:go_curators, GOC:isa_complete]
+def: "Any process that is carried out at the cellular level, but not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level." [GOC:curators, GOC:isa_complete]
 comment: This term should not be used for direct annotation. It should be possible to make a more specific annotation to one of the children of this term.
 subset: gocheck_do_not_annotate
 subset: goslim_plant
@@ -105358,7 +105709,7 @@
 id: GO:0009991
 name: obsolete response to extracellular stimulus
 namespace: biological_process
-def: "OBSOLETE. Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an extracellular stimulus." [GOC:go_curators]
+def: "OBSOLETE. Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an extracellular stimulus." [GOC:curators]
 comment: The reason for obsoletion is that this term is too vague.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27020" xsd:anyURI
 is_obsolete: true
@@ -105378,7 +105729,7 @@
 id: GO:0009994
 name: oocyte differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized immature germ cell acquires the specialized features of a mature female gamete." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process in which a relatively unspecialized immature germ cell acquires the specialized features of a mature female gamete." [GOC:curators, GOC:mtg_sensu]
 synonym: "oocyte cell differentiation" EXACT []
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0030154 ! cell differentiation
@@ -105388,14 +105739,14 @@
 id: GO:0009995
 name: soluble molecule recognition
 namespace: biological_process
-def: "The recognition of soluble molecules in the environment." [GOC:go_curators]
+def: "The recognition of soluble molecules in the environment." [GOC:curators]
 is_a: GO:0009593 ! detection of chemical stimulus
 
 [Term]
 id: GO:0009996
 name: negative regulation of cell fate specification
 namespace: biological_process
-def: "Any process that restricts, stops or prevents a cell from adopting a specific cell fate." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from adopting a specific cell fate." [GOC:curators]
 synonym: "down regulation of cell fate specification" EXACT []
 synonym: "down-regulation of cell fate specification" EXACT []
 synonym: "downregulation of cell fate specification" EXACT []
@@ -105412,7 +105763,7 @@
 name: negative regulation of cardioblast cell fate specification
 namespace: biological_process
 alt_id: GO:0042687
-def: "Any process that restricts, stops or prevents a cell from specifying into a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from specifying into a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 synonym: "down regulation of cardioblast cell fate specification" EXACT []
 synonym: "down-regulation of cardioblast cell fate specification" EXACT []
 synonym: "downregulation of cardioblast cell fate specification" EXACT []
@@ -105429,7 +105780,7 @@
 id: GO:0009998
 name: negative regulation of retinal cone cell fate specification
 namespace: biological_process
-def: "Any process that restricts, stops or prevents a cell from specifying into a retinal cone cell." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from specifying into a retinal cone cell." [GOC:curators]
 synonym: "down regulation of retinal cone cell fate specification" EXACT []
 synonym: "down-regulation of retinal cone cell fate specification" EXACT []
 synonym: "downregulation of retinal cone cell fate specification" EXACT []
@@ -105448,7 +105799,7 @@
 id: GO:0009999
 name: negative regulation of auditory receptor cell fate specification
 namespace: biological_process
-def: "Any process that restricts, stops or prevents a cell from specifying into an auditory hair cell." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from specifying into an auditory hair cell." [GOC:curators]
 synonym: "down regulation of auditory receptor cell fate specification" EXACT []
 synonym: "down-regulation of auditory receptor cell fate specification" EXACT []
 synonym: "downregulation of auditory receptor cell fate specification" EXACT []
@@ -105469,7 +105820,7 @@
 namespace: biological_process
 alt_id: GO:0007404
 alt_id: GO:0043360
-def: "The process in which a relatively unspecialized cell acquires the specialized features of a glial cell." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process in which a relatively unspecialized cell acquires the specialized features of a glial cell." [GOC:curators, GOC:mtg_sensu]
 synonym: "glia cell differentiation" EXACT []
 synonym: "neuroglia differentiation" EXACT []
 is_a: GO:0030154 ! cell differentiation
@@ -105479,7 +105830,7 @@
 id: GO:0010002
 name: cardioblast differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized mesodermal cell acquires the specialized structural and/or functional features of a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "The process in which a relatively unspecialized mesodermal cell acquires the specialized structural and/or functional features of a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 synonym: "cardiac precursor cell differentiation" EXACT [GOC:mtg_heart]
 synonym: "cardioblast cell differentiation" EXACT []
 synonym: "cardiomyocyte generation" RELATED []
@@ -105490,7 +105841,7 @@
 id: GO:0010004
 name: gastrulation involving germ band extension
 namespace: biological_process
-def: "A complex and coordinated series of cellular movements, including germ band extension, that occurs at the end of cleavage during embryonic development. An example of this process is found in Drosophila melanogaster." [GOC:go_curators, GOC:mtg_sensu]
+def: "A complex and coordinated series of cellular movements, including germ band extension, that occurs at the end of cleavage during embryonic development. An example of this process is found in Drosophila melanogaster." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0001703 ! gastrulation with mouth forming first
 
 [Term]
@@ -105531,11 +105882,12 @@
 id: GO:0010009
 name: cytoplasmic side of endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "external leaflet of endosome membrane" EXACT [GOC:ab]
 synonym: "external side of endosome membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0010008 ! endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0010011
@@ -107488,10 +107840,12 @@
 
 [Term]
 id: GO:0010209
-name: vacuolar sorting signal binding
+name: vacuolar sorting signal receptor activity
 namespace: molecular_function
 def: "Binding to a vacuolar sorting signal, a specific peptide sequence that acts as a signal to localize the protein within the vacuole." [GOC:mah]
-is_a: GO:0005048 ! signal sequence binding
+synonym: "vacuolar sorting signal binding" RELATED []
+is_a: GO:0038024 ! cargo receptor activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31410" xsd:anyURI
 
 [Term]
 id: GO:0010210
@@ -107529,7 +107883,7 @@
 id: GO:0010214
 name: seed coat development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the seed coat over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the seed coat over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0048856 ! anatomical structure development
 relationship: part_of GO:0048316 ! seed development
@@ -108218,10 +108572,11 @@
 
 [Term]
 id: GO:0010285
-name: L,L-diaminopimelate aminotransferase activity
+name: L,L-diaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
 alt_id: GO:0043742
-def: "Catalysis of the reaction: 2-oxoglutarate + LL-2,6-diaminopimelate = (S)-2,3,4,5-tetrahydrodipicolinate + L-glutamate + H2O + H+." [EC:2.6.1.83, RHEA:23988]
+def: "Catalysis of the reaction: 2-oxoglutarate + LL-2,6-diaminopimelate = (S)-2,3,4,5-tetrahydrodipicolinate + L-glutamate + H2O + H+." [RHEA:23988]
+synonym: "L,L-diaminopimelate aminotransferase activity" EXACT []
 synonym: "LL-2,6-diaminoheptanedioate:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "LL-DAP aminotransferase activity" RELATED [EC:2.6.1.83]
 synonym: "LL-DAP-AT activity" RELATED [EC:2.6.1.83]
@@ -108235,6 +108590,7 @@
 property_value: skos:exactMatch EC:2.6.1.83
 property_value: skos:exactMatch RHEA:23988
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0010286
@@ -108653,7 +109009,6 @@
 namespace: biological_process
 alt_id: GO:1902534
 def: "The infolding of a membrane." [GOC:tb]
-subset: goslim_yeast
 synonym: "single-organism membrane invagination" RELATED []
 is_a: GO:0061024 ! membrane organization
 created_by: jl
@@ -108669,17 +109024,19 @@
 
 [Term]
 id: GO:0010326
-name: methionine-oxo-acid transaminase activity
+name: L-methionine:oxo-acid transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a 2-oxocarboxylate + L-methionine = 4-methylsulfanyl-2-oxobutanoate + an L-alpha-amino acid." [PMID:17056707, RHEA:31763]
+synonym: "methionine-oxo-acid transaminase activity" EXACT []
 xref: EC:2.6.1.88
 xref: MetaCyc:R15-RXN
 xref: RHEA:31763
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.88
 property_value: skos:exactMatch RHEA:31763
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28199" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0010327
@@ -108938,12 +109295,10 @@
 
 [Term]
 id: GO:0010355
-name: obsolete homogentisate farnesyltransferase activity
+name: homogentisate farnesyltransferase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: homogentisic acid + farnesyl diphosphate = 2-methyl-6-farnesylplastoquinol." [PMID:16989822]
-comment: The reason for obsoletion is that the data from the paper for which the term was requested can be accurately described using homogentisate solanesyltransferase activity.
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29344" xsd:anyURI
-is_obsolete: true
+def: "Catalysis of the reaction: homogentisic acid + farnesyl diphosphate = 2-methyl-6-farnesylplastoquinol." [PMID:16989822]
+is_a: GO:0010354 ! homogentisate prenyltransferase activity
 
 [Term]
 id: GO:0010356
@@ -109802,7 +110157,7 @@
 id: GO:0010446
 name: response to alkaline pH
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH > 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:go_curators, GOC:tb, Wikipedia:PH]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH > 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:curators, GOC:tb, Wikipedia:PH]
 synonym: "response to alkalinity" BROAD []
 synonym: "response to basic pH" EXACT []
 is_a: GO:0009268 ! response to pH
@@ -109811,7 +110166,7 @@
 id: GO:0010447
 name: response to acidic pH
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH < 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:go_curators, GOC:tb, Wikipedia:PH]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH < 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:curators, GOC:tb, Wikipedia:PH]
 comment: This term should be used to annotate instances where a cell or organism is responding to a chemical that is playing the role of an acid (e.g. proton donor) and therefore lowering the pH. If instead you wish to describe a response to a specific acid as a chemical, such as the anion portion of glutamate, please annotate to the appropriate child of GO:0001101 'response to acid chemical'.
 synonym: "response to acidity" BROAD []
 is_a: GO:0009268 ! response to pH
@@ -110307,6 +110662,7 @@
 name: proteasomal ubiquitin-independent protein catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds that is mediated by the proteasome but do not involve ubiquitin." [GOC:tb]
+xref: Reactome:R-HSA-9912633 "Antigen processing: Ub, ATP-independent proteasomal degradation"
 is_a: GO:0010498 ! proteasomal protein catabolic process
 
 [Term]
@@ -110796,23 +111152,27 @@
 name: regulation of macromolecule biosynthetic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency or extent of the chemical reactions and pathways resulting in the formation of a macromolecule, any molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from molecules of low relative molecular mass." [GOC:dph, GOC:tb]
+subset: gocheck_do_not_annotate
 is_a: GO:0009889 ! regulation of biosynthetic process
 is_a: GO:0060255 ! regulation of macromolecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0009059 ! macromolecule biosynthetic process
 relationship: regulates GO:0009059 ! macromolecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31483" xsd:anyURI
 
 [Term]
 id: GO:0010557
 name: positive regulation of macromolecule biosynthetic process
 namespace: biological_process
 def: "Any process that increases the rate, frequency or extent of the chemical reactions and pathways resulting in the formation of a macromolecule, any molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from molecules of low relative molecular mass." [GOC:dph, GOC:tb]
+subset: gocheck_do_not_annotate
 is_a: GO:0009891 ! positive regulation of biosynthetic process
 is_a: GO:0010556 ! regulation of macromolecule biosynthetic process
 is_a: GO:0010604 ! positive regulation of macromolecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0009059 ! macromolecule biosynthetic process
 relationship: positively_regulates GO:0009059 ! macromolecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31483" xsd:anyURI
 
 [Term]
 id: GO:0010558
@@ -112686,7 +113046,6 @@
 synonym: "protein kinase A signal transduction" EXACT [GOC:signaling]
 synonym: "protein kinase A signaling cascade" RELATED [GOC:signaling]
 synonym: "protein kinase A signalling cascade" RELATED [GOC:mah]
-xref: Reactome:R-HSA-163358 "PKA-mediated phosphorylation of key metabolic factors"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26794" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26899" xsd:anyURI
 is_obsolete: true
@@ -114823,10 +115182,13 @@
 
 [Term]
 id: GO:0010944
-name: negative regulation of transcription by competitive promoter binding
+name: obsolete negative regulation of transcription by competitive promoter binding
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA-dependent transcription using a mechanism that involves direct competition for interaction with a promoter binding site." [GOC:tb]
-is_a: GO:0045892 ! negative regulation of DNA-templated transcription
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of DNA-dependent transcription using a mechanism that involves direct competition for interaction with a promoter binding site." [GOC:tb]
+comment: This term was obsoleted because it represents a mechanism and is outside the scope of GO.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19894" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0045892
 
 [Term]
 id: GO:0010945
@@ -114912,6 +115274,7 @@
 namespace: biological_process
 def: "Any process that increases the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins." [GOC:dph, GOC:tb]
 subset: gocheck_obsoletion_candidate
+xref: Reactome:R-HSA-140534 "Caspase activation via Death Receptors in the presence of ligand"
 is_a: GO:0010952 ! positive regulation of peptidase activity
 is_a: GO:0052548 ! regulation of endopeptidase activity
 intersection_of: GO:0065007 ! biological regulation
@@ -114927,6 +115290,7 @@
 def: "Any process that decreases the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins." [GOC:dph, GOC:tb]
 subset: gocheck_obsoletion_candidate
 subset: goslim_chembl
+xref: Reactome:R-HSA-5218900 "CASP8 activity is inhibited"
 is_a: GO:0010466 ! negative regulation of peptidase activity
 is_a: GO:0052548 ! regulation of endopeptidase activity
 intersection_of: GO:0065007 ! biological regulation
@@ -118203,7 +118567,6 @@
 xref: MetaCyc:2.4.1.135-RXN
 xref: Reactome:R-HSA-1889955 "B3GAT dimers transfer GlcA to tetrasaccharide linker"
 xref: Reactome:R-HSA-3560802 "Defective B3GAT3 does not transfer GlcA to tetrasaccharide linker"
-xref: Reactome:R-HSA-9638064 "B3GAT3 dimer transfers GlcA to tetrasaccharide linker"
 xref: RHEA:24168
 is_a: GO:0015020 ! glucuronosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.135
@@ -118304,13 +118667,13 @@
 xref: Reactome:R-HSA-2162099 "abacavir + UDP-glucuronate => abacavir 5'-glucuronide + UDP"
 xref: Reactome:R-HSA-5604954 "Defective UGT1A4 does not transfer GlcA from UDP-GlcA to BIL"
 xref: Reactome:R-HSA-5604975 "Defective UGT1A1 does not transfer GlcA from UDP-GlcA to BIL"
-xref: Reactome:R-HSA-5617143 "B4GAT1:GYLTL1B transfers GlcA from UDP-GlcA to Xyl-GlcA"
+xref: Reactome:R-HSA-5617143 "B4GAT1:LARGE transfers initial GlcA to matriglycan chain"
 xref: Reactome:R-HSA-8941701 "UGT1A10 transfers GlcA from UDP-GlcA to GCTN"
 xref: Reactome:R-HSA-9036102 "Defective UGT1A1 does not transfer GlcA from UDP-GlcA to BMG"
 xref: Reactome:R-HSA-9036104 "Defective UGT1A4 does not transfer GlcA from UDP-GlcA to BMG"
 xref: Reactome:R-HSA-9632038 "UGT1A1 tetramer transfers GlcA from UDP-GlcA to BMG to form BDG"
 xref: Reactome:R-HSA-9632039 "UGT1A1 transfers GlcA from UDP-GlcA to BIL to form BMG"
-xref: Reactome:R-HSA-9638097 "B4GAT1:LARGE transfers GlcA from UDP-GlcA to Xyl-GlcA"
+xref: Reactome:R-HSA-9638097 "LARGE1,2 elongates matriglycan chain with GlcA"
 xref: Reactome:R-HSA-9749977 "UGT1A6 glucuronates ST"
 xref: Reactome:R-HSA-9756134 "UGT1A3 lactonizes 2-OH-ATV to 2-OH-ATVL"
 xref: Reactome:R-HSA-9756156 "UGT1A3 lactonizes ATV to ATVL"
@@ -118380,7 +118743,7 @@
 namespace: molecular_function
 alt_id: GO:0015027
 alt_id: GO:0015028
-def: "Combining with an extracellular or intracellular messenger, and in cooperation with a nearby primary receptor, initiating a change in cell activity." [GOC:go_curators]
+def: "Combining with an extracellular or intracellular messenger, and in cooperation with a nearby primary receptor, initiating a change in cell activity." [GOC:curators]
 synonym: "coreceptor, insoluble ligand activity" RELATED [GOC:mah]
 synonym: "coreceptor, soluble ligand activity" RELATED [GOC:mah]
 is_a: GO:0038023 ! signaling receptor activity
@@ -118453,7 +118816,7 @@
 namespace: molecular_function
 alt_id: GO:0008895
 alt_id: GO:0015037
-def: "Catalysis of the reaction: a protein with reduced sulfide groups = a protein with oxidized disulfide bonds." [PMID:7559385]
+def: "Catalysis of the reaction: a protein with reduced sulfide groups = a protein with oxidized disulfide bonds." [PMID:18331844]
 synonym: "haem lyase disulphide oxidoreductase activity" NARROW []
 synonym: "heme lyase disulfide oxidoreductase activity" NARROW []
 synonym: "peptide disulfide oxidoreductase activity" EXACT []
@@ -118461,7 +118824,6 @@
 synonym: "protein disulfide oxidoreductase activity" EXACT []
 synonym: "protein disulfide-oxidoreductase activity" EXACT []
 synonym: "protein disulphide oxidoreductase activity" EXACT []
-xref: MetaCyc:DISULFOXRED-RXN
 xref: Reactome:R-HSA-1307802 "MIA40:ERV1 (CHCHD4:GFER) oxidizes cysteine residues to cystine disulfide bonds"
 xref: Reactome:R-HSA-264997 "P4HB mediates disulfide bond formation in Proinsulin"
 xref: Reactome:R-HSA-3299753 "CCS transfers Cu to SOD1 and oxidizes cysteine residues in SOD1"
@@ -118469,6 +118831,7 @@
 is_a: GO:0015036 ! disulfide oxidoreductase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20219" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0015036
@@ -119094,6 +119457,8 @@
 xref: Reactome:R-HSA-1362417 "Mitoferrin translocates iron from the mitochondrial intermembrane space to the mitochondrial matrix"
 xref: Reactome:R-HSA-442368 "SLC40A1:HEPH:6Cu2+ transports Fe2+ from cytosol to extracellular region"
 xref: Reactome:R-HSA-5655760 "Defective SLC40A1 does not transport Fe3+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-9911596 "FeoAB transports Fe2+ from periplasmic space to the cytosol"
+xref: Reactome:R-HSA-9911598 "EfeUOB transport Fe2+ from periplasmic space to the cytosol"
 is_a: GO:0005381 ! iron ion transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -119177,16 +119542,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Enables the transfer of organic cations from one side of a membrane to the other. Organic cations are atoms or small molecules with a positive charge that contain carbon in covalent linkage." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-2161500 "abacavir [extracellular] => abacavir [cytosol]"
-xref: Reactome:R-HSA-549129 "OCT1 transports organic cations into hepatic cells"
-xref: Reactome:R-HSA-549279 "OCT2 mediates tubular uptake of organic cations in the kidney"
-xref: Reactome:R-HSA-549304 "OCT3 mediates renal clearance of organic cations"
-xref: Reactome:R-HSA-549322 "OCT1 transports organic cations out of hepatic cells"
-xref: Reactome:R-HSA-561054 "OCT2 mediates tubular secretion of organic cations in the kidney"
-xref: Reactome:R-HSA-561072 "OCT3 mediates renal uptake of organic cations"
-xref: Reactome:R-HSA-9794270 "SLCO1A2 transports Cipro(1+) into the cytosol"
-xref: Reactome:R-HSA-9794523 "SLC22A1 transports Cipro into renal cell"
-xref: Reactome:R-HSA-9795207 "SLC22A1 transports Cipro into hepatic cell"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -119280,7 +119635,7 @@
 namespace: molecular_function
 def: "Enables the transfer of iodide ions from one side of a membrane to the other." [GOC:ai]
 xref: Reactome:R-HSA-209910 "Iodide is taken up by thyroid epithelial cells"
-xref: Reactome:R-HSA-5627802 "SLC26A4 transports I- from cytosol to extracellular region"
+xref: Reactome:R-HSA-5627802 "SLC26A4 exports I- to follicular lumen"
 xref: Reactome:R-HSA-5627870 "SLC26A4 does not transport I- from cytosol to extracellular region"
 is_a: GO:0008509 ! monoatomic anion transmembrane transporter activity
 relationship: part_of GO:0015705 ! iodide transport
@@ -119335,8 +119690,9 @@
 synonym: "sulfate permease activity" RELATED []
 synonym: "sulphate transporter activity" EXACT []
 xref: Reactome:R-HSA-3560789 "Defective SLC26A2 does not cotransport extracellular SO4(2-), H+ to cytosol"
-xref: Reactome:R-HSA-427555 "SLC26A1,2 cotransport SO4(2-), H+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-427555 "SLC26A1,2 cotransport SO4(2-), anion from extracellular region to cytosol"
 xref: Reactome:R-HSA-8875871 "SLC26A11 transports SO4(2-) from extracellular region to cytosol"
+xref: Reactome:R-HSA-9941470 "SLC26A11 imports SO4(2-) and H+ into cytosol"
 is_a: GO:1901682 ! sulfur compound transmembrane transporter activity
 relationship: part_of GO:1902358 ! sulfate transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -119510,6 +119866,7 @@
 name: glucuronate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of glucuronate from one side of a membrane to the other. Glucuronate is the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:ai]
+xref: Reactome:R-HSA-2162229 "SLC17A5 exports GlcA from the lysosome to the cytosol"
 is_a: GO:0015133 ! uronic acid transmembrane transporter activity
 relationship: part_of GO:0015738 ! glucuronate transmembrane transport
 
@@ -119739,6 +120096,8 @@
 name: polysaccharide transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of polysaccharides from one side of a membrane to the other. A polysaccharide is a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
+xref: Reactome:R-HSA-9932121 "Unknown transporter exports CA to extracellular space"
+xref: Reactome:R-HSA-9932466 "Bcs complex exports pEtN-cellulose"
 is_a: GO:0015144 ! carbohydrate transmembrane transporter activity
 is_a: GO:0022884 ! macromolecule transmembrane transporter activity
 relationship: part_of GO:0015774 ! polysaccharide transport
@@ -119842,7 +120201,6 @@
 synonym: "hydroxy/aromatic amino acid permease activity" RELATED []
 xref: Reactome:R-HSA-210439 "glutamate uptake by astrocytes"
 xref: Reactome:R-HSA-212642 "L-Glutamine transport into neurons"
-xref: Reactome:R-HSA-351963 "SLC6A18 transports Gly from extracellular region to cytosol"
 xref: Reactome:R-HSA-351987 "SLC6A6-mediated uptake of taurine and beta-alanine"
 xref: Reactome:R-HSA-352029 "SLC6A12 (BGT-1)-mediated uptake of GABA and betaine"
 xref: Reactome:R-HSA-352052 "SLC6A20 cotransports L-Pro, Na+ from the extracellular region to cytosol"
@@ -119856,7 +120214,6 @@
 xref: Reactome:R-HSA-352174 "SLC38A3-mediated uptake of glutamine, histidine, asparagine, and alanine"
 xref: Reactome:R-HSA-352182 "SLC38A5-mediated uptake of glutamine, histidine, asparagine, and serine"
 xref: Reactome:R-HSA-352191 "SLC7A8-mediated uptake of neutral amino acids"
-xref: Reactome:R-HSA-352232 "SLC7A5:SLC3A2 transports neutral amino acids from extracellular region to cytosol"
 xref: Reactome:R-HSA-352347 "SLC1A4-mediated exchange of extracellular serine for cytosolic alanine, threonine, or cysteine"
 xref: Reactome:R-HSA-352354 "SLC1A4-mediated exchange of extracellular cysteine for cytosolic alanine, serine, or threonine"
 xref: Reactome:R-HSA-352364 "SLC1A4-mediated exchange of extracellular alanine for cytosolic serine, threonine, or cysteine"
@@ -119880,7 +120237,6 @@
 xref: Reactome:R-HSA-5655702 "Defective SLC3A1 (in SLC7A9:SLC3A1) does not exchange L-Arg, CySS-, L-Lys for L-Leu"
 xref: Reactome:R-HSA-5659674 "Variant SLC6A14 cotransports SLC6A14 ligands, Cl-, 2Na+ from extracellular region to cytosol"
 xref: Reactome:R-HSA-5659734 "Defective SLC6A19 does not cotransport neutral amino acids, Na+ from extracellular region to cytosol"
-xref: Reactome:R-HSA-5659755 "Defective SLC6A18 does not transport Gly from extracellular region to cytosol"
 xref: Reactome:R-HSA-5660694 "Variant SLC6A20 does not cotransport L-Pro, Na+ from extracellulare region to cytosol"
 xref: Reactome:R-HSA-5660890 "Defective SLC7A9 (in SLC7A9:SLC3A1) does not exchange L-Arg, CySS-, L-Lys for L-Leu"
 xref: Reactome:R-HSA-5660910 "Defective SLC7A7 does not exchange L-Arg for L-Leu, Na+ across the plasma membrane"
@@ -119952,6 +120308,7 @@
 def: "Enables the transfer of an L-amino acid from one side of a membrane to the other. L-amino acids are the L-enantiomers of amino acids." [GOC:ai, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
 synonym: "L-amino acid transporter activity" BROAD []
 xref: Reactome:R-HSA-212614 "Glutamine transport from astrocytes"
+xref: Reactome:R-HSA-352232 "SLC7A5:SLC3A2 transports neutral amino acids from extracellular region to cytosol"
 is_a: GO:0015171 ! amino acid transmembrane transporter activity
 is_a: GO:0046943 ! carboxylic acid transmembrane transporter activity
 relationship: part_of GO:1902475 ! L-alpha-amino acid transmembrane transport
@@ -119960,7 +120317,7 @@
 id: GO:0015180
 name: L-alanine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of L-alanine from one side of a membrane to the other. L-alanine is the L-enantiomer of 2-aminopropanoic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of L-alanine from one side of a membrane to the other. L-alanine is the L-enantiomer of 2-aminopropanoic acid." [GOC:curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "L-alanine transporter activity" BROAD []
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
 is_a: GO:0022858 ! alanine transmembrane transporter activity
@@ -119970,7 +120327,7 @@
 id: GO:0015182
 name: L-asparagine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of L-asparagine from one side of a membrane to the other. L-asparagine is the L-enantiomer of alpha-aminosuccinamic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of L-asparagine from one side of a membrane to the other. L-asparagine is the L-enantiomer of alpha-aminosuccinamic acid." [GOC:curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "asparagine/glutamine permease activity" NARROW []
 synonym: "L-asparagine transporter activity" BROAD []
 is_a: GO:0015175 ! neutral L-amino acid transmembrane transporter activity
@@ -119981,7 +120338,7 @@
 id: GO:0015183
 name: L-aspartate transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of L-aspartate from one side of a membrane to the other. L-aspartate is the anion derived from aspartic acid." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of L-aspartate from one side of a membrane to the other. L-aspartate is the anion derived from aspartic acid." [GOC:curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "glutamate/aspartate porter activity" NARROW []
 synonym: "glutamate/aspartate:sodium symporter activity" NARROW []
 synonym: "L-aspartate transporter activity" BROAD []
@@ -119995,7 +120352,7 @@
 id: GO:0015184
 name: L-cystine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of L-cystine from one side of a membrane to the other." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of L-cystine from one side of a membrane to the other." [GOC:curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "cystine/diaminopimelate porter activity" NARROW []
 synonym: "L-cystine transporter activity" BROAD []
 xref: Reactome:R-HSA-5340130 "CTNS cotransports CySS-, H+ from lysosomal lumen to cytosol"
@@ -120008,7 +120365,7 @@
 id: GO:0015185
 name: gamma-aminobutyric acid transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of gamma-aminobutyric acid from one side of a membrane to the other. Gamma-aminobutyric acid is 4-aminobutyrate (GABA)." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of gamma-aminobutyric acid from one side of a membrane to the other. Gamma-aminobutyric acid is 4-aminobutyrate (GABA)." [GOC:curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "4-aminobutanoate transporter activity" EXACT []
 synonym: "4-aminobutyrate transporter activity" EXACT []
 synonym: "betaine/GABA:sodium symporter activity" NARROW []
@@ -120123,7 +120480,6 @@
 synonym: "L-serine transporter activity" BROAD []
 synonym: "serine transporter activity" BROAD []
 synonym: "threonine/serine:sodium symporter activity" RELATED []
-xref: Reactome:R-HSA-8932980 "SERINC3,5,(1,2,4) transport L-Ser from cytosol to plasma membrane"
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
 relationship: part_of GO:0015825 ! L-serine transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27521" xsd:anyURI
@@ -120210,7 +120566,7 @@
 id: GO:0015207
 name: adenine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of adenine, 6-aminopurine, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of adenine, 6-aminopurine, from one side of a membrane to the other." [GOC:curators]
 xref: Reactome:R-HSA-163215 "SLC25A5,6 dimers exchange ATP for ADP across the mitochondrial inner membrane"
 xref: Reactome:R-HSA-5672027 "ARL2:GTP:ARL2BP:SLC25A4 dimer exchanges ATP for ADP across the mitochondrial inner membrane"
 is_a: GO:0005345 ! purine nucleobase transmembrane transporter activity
@@ -120221,7 +120577,7 @@
 id: GO:0015208
 name: guanine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of guanine, 2-amino-6-hydroxypurine, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of guanine, 2-amino-6-hydroxypurine, from one side of a membrane to the other." [GOC:curators]
 is_a: GO:0005345 ! purine nucleobase transmembrane transporter activity
 relationship: part_of GO:1903716 ! guanine transmembrane transport
 
@@ -120229,7 +120585,7 @@
 id: GO:0015209
 name: cytosine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of cytosine, 4-amino-2-hydroxypyrimidine from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of cytosine, 4-amino-2-hydroxypyrimidine from one side of a membrane to the other." [GOC:curators]
 is_a: GO:0005350 ! pyrimidine nucleobase transmembrane transporter activity
 relationship: part_of GO:0015856 ! cytosine transport
 
@@ -120237,7 +120593,7 @@
 id: GO:0015210
 name: uracil transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of uracil, 2,4-dioxopyrimidine, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of uracil, 2,4-dioxopyrimidine, from one side of a membrane to the other." [GOC:curators]
 synonym: "uracil/uridine permease activity" RELATED []
 is_a: GO:0005350 ! pyrimidine nucleobase transmembrane transporter activity
 relationship: part_of GO:1903791 ! uracil transmembrane transport
@@ -120254,7 +120610,7 @@
 id: GO:0015212
 name: cytidine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of cytidine, cytosine riboside, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of cytidine, cytosine riboside, from one side of a membrane to the other." [GOC:curators]
 is_a: GO:0015214 ! pyrimidine nucleoside transmembrane transporter activity
 relationship: part_of GO:0015861 ! cytidine transport
 
@@ -120262,7 +120618,7 @@
 id: GO:0015213
 name: uridine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of uridine, uracil riboside, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of uridine, uracil riboside, from one side of a membrane to the other." [GOC:curators]
 synonym: "uracil/uridine permease activity" NARROW []
 is_a: GO:0015214 ! pyrimidine nucleoside transmembrane transporter activity
 relationship: part_of GO:0015862 ! uridine transmembrane transport
@@ -120306,6 +120662,7 @@
 name: pyrimidine nucleotide transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of a pyrimidine nucleotide, any compound consisting of a pyrimidine nucleoside esterified with (ortho)phosphate, from one side of a membrane to the other." [GOC:ai]
+xref: Reactome:R-HSA-9940804 "SLC35A1,4 import CDP-ribitol"
 is_a: GO:0015215 ! nucleotide transmembrane transporter activity
 relationship: part_of GO:0006864 ! pyrimidine nucleotide transport
 
@@ -120468,6 +120825,8 @@
 xref: Reactome:R-HSA-917870 "SLC46A1 transports hemes from extracellular region to cytosol"
 xref: Reactome:R-HSA-917892 "FLVCR1-1 transports heme from cytosol to extracellular region"
 xref: Reactome:R-HSA-9661408 "FLVCR1-2 transports heme from mitochondrial matrix to cytosol"
+xref: Reactome:R-HSA-9927224 "ChuA imports heme to periplasm"
+xref: Reactome:R-HSA-9927930 "DppBCDF transports heme to cytosol"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0015886 ! heme transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19472" xsd:anyURI
@@ -120558,7 +120917,6 @@
 def: "Enables the transfer of aminophospholipids from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP. Aminophospholipids contain phosphoric acid as a mono- or diester and an amino (NH2) group." [GOC:pg]
 synonym: "aminophospholipid transmembrane transporter activity" RELATED []
 synonym: "aminophospholipid transporter activity" RELATED []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140327 ! flippase activity
 relationship: part_of GO:0140331 ! aminophospholipid translocation
 property_value: skos:broadMatch RHEA:66132
@@ -120755,6 +121113,9 @@
 synonym: "substrate-specific channel activity" RELATED []
 xref: Reactome:R-HSA-3779381 "H2O2 diffuses from the mitochondrial matrix to the cytosol"
 xref: Reactome:R-HSA-8953430 "PXMP2 trimer transports glycolate from cytosol to peroxisomal matrix"
+xref: Reactome:R-HSA-9857649 "p-S373-GJA1:p-S-ITGA5:ITGB1 (p-S373-Connexin-43:p-S-Integrin alpha5:Integrin beta1) transports ATP from the cytosol to the extracellular region"
+xref: Reactome:R-HSA-9857682 "p-S373-GJA1:p-S-ITGA5:ITGB1 (p-S373-Connexin-43:p-S-Integrin alpha5:Integrin beta1) transports prostaglandin E2 (PGE2) from the cytosol to the extracellular region"
+xref: Reactome:R-HSA-9905999 "Pannexin1 (PANX1) channel in a complex with P2RX7 transports ATP from the cytosol to the extracellular region"
 xref: TC:1
 is_a: GO:0022803 ! passive transmembrane transporter activity
 
@@ -120897,6 +121258,11 @@
 def: "Enables the transfer of substances, sized less than 1000 Da, from one side of a membrane to the other. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820, TC:1.B.1.-.-]
 synonym: "outer membrane exporter porin" NARROW []
 synonym: "porin" EXACT []
+xref: Reactome:R-HSA-9760573 "TolC complex exports STa"
+xref: Reactome:R-HSA-9911510 "MacA:MacB:TolC transports azithromycin from the periplasm to the extracellular region"
+xref: Reactome:R-HSA-9911516 "AcrA:AcrB:TolC transports antibiotics from the periplasm to the extracellular region"
+xref: Reactome:R-HSA-9914083 "MdtE:MdtF:TolC exports beta-lactams to extracellular space"
+xref: Reactome:R-HSA-9926021 "HlyB:HlyD:TolC transports HlyA from periplasm to extracellular space"
 xref: Wikipedia:Porin_(protein)
 is_a: GO:0022829 ! wide pore channel activity
 
@@ -120934,6 +121300,18 @@
 synonym: "porter activity" EXACT []
 synonym: "porters" EXACT []
 synonym: "secondary carrier-type facilitators" EXACT []
+xref: Reactome:R-HSA-549241 "SLC22A4 cotransports ERGT, Na+ from extracellular region to cytosol"
+xref: Reactome:R-HSA-561059 "OAT2 and OAT4 mediate transport of sulphate conjugates"
+xref: Reactome:R-HSA-879562 "SLCO2B1 has a narrow substrate specificity"
+xref: Reactome:R-HSA-879584 "SLCO3A1 isoform 1 has a broad substrate specificity"
+xref: Reactome:R-HSA-879594 "SLCO4C1 mediates the transport of digoxin"
+xref: Reactome:R-HSA-9661397 "SLCO1B1 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9661446 "BMG, BDG translocates from ER lumen to cytosol"
+xref: Reactome:R-HSA-9661723 "SLCO2B1-3 transports BIL  from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9661799 "SLCO1B3 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
+xref: Reactome:R-HSA-9749583 "SLC22A7 transports ASA-,ST from extracellular region to cytosol of hepatocytes"
+xref: Reactome:R-HSA-9749607 "SLCO2B1-1 transports ASA- from extracellular region to cytosol of GI cells"
+xref: Reactome:R-HSA-9757010 "SLCO1B1,1B3,2B1-3 transport ATV from extracelluar region to cytosol"
 xref: TC:2.A
 is_a: GO:0022804 ! active transmembrane transporter activity
 
@@ -121444,6 +121822,11 @@
 synonym: "siderophore transporter activity" RELATED []
 synonym: "siderophore-iron transporter activity" RELATED []
 xref: Reactome:R-HSA-1222597 "Loaded mycobactin gets imported"
+xref: Reactome:R-HSA-9911605 "FyuA transports Ybt:Fe/Cu from extracellular space to the periplasm"
+xref: Reactome:R-HSA-9928256 "YbtPQ transports Ybt:Fe/Cu to the cytosol"
+xref: Reactome:R-HSA-9930902 "IutA imports FeAer to the periplasm"
+xref: Reactome:R-HSA-9930911 "FhuBCD imports FeAer to the cytosol"
+xref: Reactome:R-HSA-9931043 "FecA:TonB:ExbB:ExbD imports Fe3+-citrates to periplasm"
 xref: TC:2.A.1.16.1
 xref: TC:2.A.1.16.2
 xref: TC:2.A.1.16.3
@@ -121502,19 +121885,8 @@
 id: GO:0015347
 name: obsolete sodium-independent organic anion transmembrane transporter activity
 namespace: molecular_function
-def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other, in a sodium independent manner." [GOC:go_curators]
+def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other, in a sodium independent manner." [GOC:curators]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
-xref: Reactome:R-HSA-561059 "OAT2 and OAT4 mediate transport of sulphate conjugates"
-xref: Reactome:R-HSA-879562 "SLCO2B1 has a narrow substrate specificity"
-xref: Reactome:R-HSA-879584 "SLCO3A1 isoform 1 has abroad substrate specificity"
-xref: Reactome:R-HSA-879594 "SLCO4C1 mediates the transport of digoxin"
-xref: Reactome:R-HSA-9661397 "SLCO1B1 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9661446 "BMG, BDG translocates from ER lumen to cytosol"
-xref: Reactome:R-HSA-9661723 "SLCO2B1-3 transports BIL  from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9661799 "SLCO1B3 transports BIL from extracellular region (blood) to cytosol (hepatocyte)"
-xref: Reactome:R-HSA-9749583 "SLC22A7 transports ASA-,ST from extracellular region to cytosol of hepatocytes"
-xref: Reactome:R-HSA-9749607 "SLCO2B1-1 transports ASA- from extracellular region to cytosol of GI cells"
-xref: Reactome:R-HSA-9757010 "SLCO1B1,1B3,2B1-3 transport ATV from extracelluar region to cytosol"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0015291
@@ -122852,7 +123224,7 @@
 synonym: "P-P-bond-hydrolysis-driven protein transmembrane transporter activity" RELATED []
 synonym: "protein translocase activity" EXACT []
 xref: Reactome:R-HSA-1222523 "SodB gets secreted"
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 is_a: GO:0042626 ! ATPase-coupled transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21271" xsd:anyURI
 
@@ -124154,6 +124526,7 @@
 name: iron chelate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of an iron chelate from one side of a membrane to the other. An iron chelate is a heterocyclic compound having a metal ion attached by coordinate bonds to at least two nonmetal ions." [PMID:17660286]
+xref: Reactome:R-HSA-9931048 "FecBCDE imports Fe3+-citrates to cytosol"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0033214 ! siderophore-iron import into cell
 
@@ -124317,6 +124690,8 @@
 name: ferric-enterobactin transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of ferric-enterobactin from one side of a membrane to the other." [GOC:ai]
+xref: Reactome:R-HSA-9911602 "FepA:TonB:ExbB:ExbD transports ferrienterobactin from the extracellular region to the periplasmic space"
+xref: Reactome:R-HSA-9911603 "FepC:FepD:FepG transports ferrienterobactin from the periplasmic space to the cytosol"
 is_a: GO:0015343 ! siderophore-iron transmembrane transporter activity
 relationship: part_of GO:0015685 ! ferric-enterobactin import into cell
 
@@ -124377,7 +124752,7 @@
 id: GO:0015626
 name: L-diaminopimelate transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of L-diaminopimelate from one side of a membrane to the other. L-diaminopimelate is the L-enantiomer anion of 2,6-diaminoheptanedioic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
+def: "Enables the transfer of L-diaminopimelate from one side of a membrane to the other. L-diaminopimelate is the L-enantiomer anion of 2,6-diaminoheptanedioic acid." [GOC:curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
 synonym: "cystine/diaminopimelate porter activity" NARROW []
 synonym: "L-diaminopimelate transporter activity" BROAD []
 is_a: GO:0015179 ! L-amino acid transmembrane transporter activity
@@ -124724,6 +125099,12 @@
 name: alcohol transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of an alcohol from one side of a membrane to the other. An alcohol is any carbon compound that contains a hydroxyl group." [ISBN:0198506732]
+xref: Reactome:R-HSA-192010 "5beta-cholestan-3alpha, 7alpha, 12alpha-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193537 "5beta-cholestan-3alpha, 7alpha-diol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193715 "5beta-cholestan-3alpha,7alpha,24(S)-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193774 "5beta-cholestan-3alpha,7alpha,12alpha,24(S)-tetrol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193808 "5beta-cholestan-3alpha,7alpha,27-triol is translocated from the cytosol to the mitochondrial matrix"
+xref: Reactome:R-HSA-193832 "5beta-cholestan-3alpha,7alpha,12alpha,27-tetrol is translocated from the cytosol to the mitochondrial matrix"
 is_a: GO:0022857 ! transmembrane transporter activity
 
 [Term]
@@ -125066,6 +125447,7 @@
 name: organic anion transport
 namespace: biological_process
 def: "The directed movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. Organic anions are atoms or small molecules with a negative charge which contain carbon in covalent linkage." [GOC:ai, GOC:krc]
+xref: Reactome:R-HSA-9955298 "SLC-mediated transport of organic anions"
 is_a: GO:0006810 ! transport
 
 [Term]
@@ -125154,6 +125536,7 @@
 synonym: "bile acid transport" NARROW []
 synonym: "bile salt transport" NARROW []
 xref: Reactome:R-HSA-159418 "Recycling of bile acids and salts"
+xref: Reactome:R-HSA-9958517 "SLC-mediated bile acid transport"
 is_a: GO:0006869 ! lipid transport
 is_a: GO:0015718 ! monocarboxylic acid transport
 is_a: GO:0015850 ! organic hydroxy compound transport
@@ -125845,7 +126228,7 @@
 id: GO:0015810
 name: aspartate transmembrane transport
 namespace: biological_process
-def: "The process in which aspartate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:go_curators, ISBN:0198506732]
+def: "The process in which aspartate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:curators, ISBN:0198506732]
 synonym: "aspartate transport" BROAD []
 synonym: "mitochondrial aspartate/glutamate transport" RELATED []
 is_a: GO:0003333 ! amino acid transmembrane transport
@@ -125859,7 +126242,7 @@
 id: GO:0015811
 name: L-cystine transport
 namespace: biological_process
-def: "The directed movement of L-cystine (also known as dicysteine) into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators, ISBN:0198506732]
+def: "The directed movement of L-cystine (also known as dicysteine) into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators, ISBN:0198506732]
 is_a: GO:0000101 ! sulfur amino acid transport
 is_a: GO:0015804 ! neutral amino acid transport
 is_a: GO:0015807 ! L-amino acid transport
@@ -125869,7 +126252,7 @@
 id: GO:0015812
 name: gamma-aminobutyric acid transport
 namespace: biological_process
-def: "The directed movement of gamma-aminobutyric acid (GABA, 4-aminobutyrate), an amino acid which acts as a neurotransmitter in some organisms, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators, ISBN:0198506732]
+def: "The directed movement of gamma-aminobutyric acid (GABA, 4-aminobutyrate), an amino acid which acts as a neurotransmitter in some organisms, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators, ISBN:0198506732]
 comment: See also the biological process term 'neurotransmitter transport ; GO:0006836'.
 synonym: "4-aminobutanoate transport" EXACT []
 synonym: "4-aminobutyrate transport" EXACT []
@@ -126047,7 +126430,7 @@
 id: GO:0015830
 name: diaminopimelate transport
 namespace: biological_process
-def: "The directed movement of diaminopimelate, the anion of 2,6-diaminoheptanedioic acid, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators, ISBN:0198506732]
+def: "The directed movement of diaminopimelate, the anion of 2,6-diaminoheptanedioic acid, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators, ISBN:0198506732]
 is_a: GO:0046942 ! carboxylic acid transport
 is_a: GO:0071705 ! nitrogen compound transport
 is_a: GO:1901571 ! fatty acid derivative transport
@@ -126056,7 +126439,7 @@
 id: GO:0015832
 name: obsolete holin
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it represents a class of gene products.
 synonym: "holin" EXACT []
 is_obsolete: true
@@ -126105,6 +126488,7 @@
 def: "The directed movement of amines, including polyamines, organic compounds containing one or more amino groups, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:ai, ISBN:0198506732]
 subset: goslim_pir
 synonym: "amine/polyamine transport" RELATED []
+xref: Reactome:R-HSA-549127 "SLC-mediated transport of organic cations"
 is_a: GO:0071705 ! nitrogen compound transport
 
 [Term]
@@ -126215,7 +126599,7 @@
 id: GO:0015853
 name: adenine transport
 namespace: biological_process
-def: "The directed movement of adenine, 6-aminopurine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators, ISBN:0198506732]
+def: "The directed movement of adenine, 6-aminopurine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators, ISBN:0198506732]
 synonym: "adenine transmembrane transport" EXACT [GOC:mah]
 is_a: GO:0006863 ! purine nucleobase transport
 
@@ -126223,7 +126607,7 @@
 id: GO:0015854
 name: guanine transport
 namespace: biological_process
-def: "The directed movement of guanine, 2-amino-6-hydroxypurine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed movement of guanine, 2-amino-6-hydroxypurine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
 is_a: GO:0006863 ! purine nucleobase transport
 
 [Term]
@@ -126232,7 +126616,7 @@
 namespace: biological_process
 def: "The directed movement of pyrimidine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:ai]
 synonym: "pyrimidine base transmembrane transport" EXACT [GOC:mah]
-synonym: "pyrimidine base transport" EXACT [GOC:go_curators]
+synonym: "pyrimidine base transport" EXACT [GOC:curators]
 synonym: "pyrimidine transmembrane transport" RELATED [GOC:mah]
 synonym: "pyrimidine transport" RELATED []
 is_a: GO:0015851 ! nucleobase transport
@@ -126241,7 +126625,7 @@
 id: GO:0015856
 name: cytosine transport
 namespace: biological_process
-def: "The directed movement of cytosine, 4-amino-2-hydroxypyrimidine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed movement of cytosine, 4-amino-2-hydroxypyrimidine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
 synonym: "cytosine transmembrane transport" EXACT [GOC:mah]
 is_a: GO:0015855 ! pyrimidine nucleobase transport
 
@@ -126249,7 +126633,7 @@
 id: GO:0015857
 name: uracil transport
 namespace: biological_process
-def: "The directed movement of uracil, 2,4-dioxopyrimidine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed movement of uracil, 2,4-dioxopyrimidine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
 is_a: GO:0015855 ! pyrimidine nucleobase transport
 
 [Term]
@@ -126284,14 +126668,14 @@
 id: GO:0015861
 name: cytidine transport
 namespace: biological_process
-def: "The directed movement of cytidine, cytosine riboside, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed movement of cytidine, cytosine riboside, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
 is_a: GO:0015864 ! pyrimidine nucleoside transport
 
 [Term]
 id: GO:0015862
 name: uridine transmembrane transport
 namespace: biological_process
-def: "The directed movement of uridine, uracil riboside, across a lipid bilayer, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed movement of uridine, uracil riboside, across a lipid bilayer, by means of some agent such as a transporter or pore." [GOC:curators]
 is_a: GO:0015864 ! pyrimidine nucleoside transport
 is_a: GO:0072531 ! pyrimidine-containing compound transmembrane transport
 is_a: GO:1901642 ! nucleoside transmembrane transport
@@ -127084,7 +127468,7 @@
 id: GO:0015945
 name: methanol metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving methanol, CH3-OH, a colorless, flammable, mobile, poisonous liquid, widely used as a solvent." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving methanol, CH3-OH, a colorless, flammable, mobile, poisonous liquid, widely used as a solvent." [GOC:curators, ISBN:0198506732]
 synonym: "methanol metabolism" EXACT []
 is_a: GO:0034308 ! primary alcohol metabolic process
 
@@ -127703,7 +128087,7 @@
 id: GO:0016015
 name: morphogen activity
 namespace: molecular_function
-def: "Acts as a trigger for a pattern specification process when present at a specific concentration within a gradient." [GOC:go_curators]
+def: "Acts as a trigger for a pattern specification process when present at a specific concentration within a gradient." [GOC:curators]
 is_a: GO:0048018 ! receptor ligand activity
 
 [Term]
@@ -127781,7 +128165,7 @@
 id: GO:0016025
 name: obsolete proteasome endopeptidase regulator
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not represent a molecular function.
 synonym: "proteasome endopeptidase regulator" EXACT []
 is_obsolete: true
@@ -127791,7 +128175,7 @@
 id: GO:0016026
 name: obsolete proteasome endopeptidase core
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not represent a molecular function.
 synonym: "proteasome endopeptidase core" EXACT []
 is_obsolete: true
@@ -127896,7 +128280,7 @@
 id: GO:0016037
 name: light absorption
 namespace: biological_process
-def: "The reception of a photon by a cell." [GOC:go_curators]
+def: "The reception of a photon by a cell." [GOC:curators]
 synonym: "absorption of light" EXACT []
 is_a: GO:0009987 ! cellular process
 relationship: part_of GO:0009583 ! detection of light stimulus
@@ -127905,22 +128289,21 @@
 id: GO:0016038
 name: absorption of visible light
 namespace: biological_process
-def: "The reception of a (visible light) photon by a cell, visible light being defined as having a wavelength within the range 380-780 nm." [GOC:go_curators, ISBN:0198506732]
+def: "The reception of a (visible light) photon by a cell, visible light being defined as having a wavelength within the range 380-780 nm." [GOC:curators, ISBN:0198506732]
 is_a: GO:0016037 ! light absorption
 
 [Term]
 id: GO:0016039
 name: absorption of UV light
 namespace: biological_process
-def: "The reception of a (UV light) photon by a cell, UV light being defined as having a wavelength within the range 13.6-400 nm." [GOC:go_curators, ISBN:0198506732]
+def: "The reception of a (UV light) photon by a cell, UV light being defined as having a wavelength within the range 13.6-400 nm." [GOC:curators, ISBN:0198506732]
 is_a: GO:0016037 ! light absorption
 
 [Term]
 id: GO:0016040
 name: glutamate synthase (NADH) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + NAD+ = 2-oxoglutarate + L-glutamine + H+ + NADH." [EC:1.4.1.14, RHEA:13753]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + NAD+ = 2-oxoglutarate + L-glutamine + H+ + NADH." [RHEA:13753]
 synonym: "glutamate (reduced nicotinamide adenine dinucleotide) synthase" RELATED [EC:1.4.1.14]
 synonym: "GOGAT activity" BROAD []
 synonym: "L-glutamate synthase (NADH)" RELATED [EC:1.4.1.14]
@@ -127938,15 +128321,14 @@
 property_value: skos:exactMatch EC:1.4.1.14
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-SYNTHASE-NADH-RXN
 property_value: skos:exactMatch RHEA:13753
-property_value: skos:relatedMatch MetaCyc:GLUGLNSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0016041
 name: glutamate synthase (ferredoxin) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + 2 oxidized ferredoxin + H2O = NH3 + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+." [EC:1.4.7.1]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: 2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+. This is a two-step reaction: (a) L-glutamate + NH4+ = L-glutamine + H2O, (b) L-glutamate + 2 oxidized ferredoxin + H2O = NH4+ + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+." [EC:1.4.7.1]
 synonym: "ferredoxin-dependent glutamate synthase activity" RELATED [EC:1.4.7.1]
 synonym: "ferredoxin-glutamate synthase activity" RELATED [EC:1.4.7.1]
 synonym: "glutamate synthase (ferredoxin-dependent)" RELATED [EC:1.4.7.1]
@@ -127959,8 +128341,8 @@
 property_value: skos:exactMatch EC:1.4.7.1
 property_value: skos:exactMatch MetaCyc:GLUTAMATE-SYNTHASE-FERREDOXIN-RXN
 property_value: skos:exactMatch RHEA:12128
-property_value: skos:relatedMatch MetaCyc:PWY-4341
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0016042
@@ -127968,7 +128350,7 @@
 namespace: biological_process
 alt_id: GO:0006724
 alt_id: GO:0044240
-def: "The chemical reactions and pathways resulting in the breakdown of lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent." [GOC:curators]
 synonym: "lipid breakdown" EXACT []
 synonym: "lipid catabolism" EXACT []
 synonym: "lipid degradation" EXACT []
@@ -128220,7 +128602,7 @@
 id: GO:0016061
 name: obsolete regulation of light-activated channel activity
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of light-activated channel activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of light-activated channel activity." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22455" xsd:anyURI
 is_obsolete: true
@@ -128506,7 +128888,7 @@
 id: GO:0016091
 name: prenol biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of prenols, isoprenoids of general formula (H-CH2-C(CH3)=CH-CH2-)n-OH, any primary monohydroxy alcohol whose carbon skeleton consists of two or more isoprenoid residues linked head to tail." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of prenols, isoprenoids of general formula (H-CH2-C(CH3)=CH-CH2-)n-OH, any primary monohydroxy alcohol whose carbon skeleton consists of two or more isoprenoid residues linked head to tail." [GOC:curators]
 synonym: "prenol anabolism" EXACT []
 synonym: "prenol biosynthesis" EXACT []
 synonym: "prenol formation" EXACT []
@@ -128519,7 +128901,7 @@
 id: GO:0016092
 name: prenol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of prenols, isoprenoids of general formula (H-CH2-C(CH3)=CH-CH2-)n-OH, any primary monohydroxy alcohol whose carbon skeleton consists of two or more isoprenoid residues linked head to tail." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of prenols, isoprenoids of general formula (H-CH2-C(CH3)=CH-CH2-)n-OH, any primary monohydroxy alcohol whose carbon skeleton consists of two or more isoprenoid residues linked head to tail." [GOC:curators]
 synonym: "prenol breakdown" EXACT []
 synonym: "prenol catabolism" EXACT []
 synonym: "prenol degradation" EXACT []
@@ -128540,7 +128922,7 @@
 id: GO:0016094
 name: polyprenol biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of polyprenols, prenols with more than 4 isoprenoid residues, which may be all-trans, or a mixture of cis and trans." [GOC:go_curators, PMID:11108713]
+def: "The chemical reactions and pathways resulting in the formation of polyprenols, prenols with more than 4 isoprenoid residues, which may be all-trans, or a mixture of cis and trans." [GOC:curators, PMID:11108713]
 synonym: "polyprenol anabolism" EXACT []
 synonym: "polyprenol biosynthesis" EXACT []
 synonym: "polyprenol formation" EXACT []
@@ -128553,7 +128935,7 @@
 id: GO:0016095
 name: polyprenol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of polyprenols, prenols with more than 4 isoprenoid residues, which may be all-trans, or a mixture of cis and trans." [GOC:go_curators, Wikipedia:Polyprenol]
+def: "The chemical reactions and pathways resulting in the breakdown of polyprenols, prenols with more than 4 isoprenoid residues, which may be all-trans, or a mixture of cis and trans." [GOC:curators, Wikipedia:Polyprenol]
 synonym: "polyprenol breakdown" EXACT []
 synonym: "polyprenol catabolism" EXACT []
 synonym: "polyprenol degradation" EXACT []
@@ -128573,7 +128955,7 @@
 id: GO:0016099
 name: monoterpenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of monoterpenoid compounds, terpenoids having a C10 skeleton." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of monoterpenoid compounds, terpenoids having a C10 skeleton." [GOC:curators]
 synonym: "monoterpenoid anabolism" EXACT []
 synonym: "monoterpenoid biosynthesis" EXACT []
 synonym: "monoterpenoid formation" EXACT []
@@ -128586,7 +128968,7 @@
 id: GO:0016100
 name: monoterpenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of monoterpenoid compounds, terpenoids having a C10 skeleton." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of monoterpenoid compounds, terpenoids having a C10 skeleton." [GOC:curators]
 synonym: "monoterpenoid breakdown" EXACT []
 synonym: "monoterpenoid catabolism" EXACT []
 synonym: "monoterpenoid degradation" EXACT []
@@ -128634,7 +129016,7 @@
 id: GO:0016104
 name: triterpenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of triterpenoid compounds, terpenoids with six isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of triterpenoid compounds, terpenoids with six isoprene units." [GOC:curators]
 synonym: "triterpene biosynthesis" NARROW []
 synonym: "triterpene biosynthetic process" NARROW []
 synonym: "triterpenoid anabolism" EXACT []
@@ -128648,7 +129030,7 @@
 id: GO:0016105
 name: triterpenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of triterpenoid compounds, terpenoids with six isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of triterpenoid compounds, terpenoids with six isoprene units." [GOC:curators]
 synonym: "triterpene catabolic process" NARROW []
 synonym: "triterpene catabolism" NARROW []
 synonym: "triterpenoid breakdown" EXACT []
@@ -128661,7 +129043,7 @@
 id: GO:0016106
 name: sesquiterpenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of sesquiterpenoid compounds, terpenoids with three isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of sesquiterpenoid compounds, terpenoids with three isoprene units." [GOC:curators]
 synonym: "sesquiterpenoid anabolism" EXACT []
 synonym: "sesquiterpenoid biosynthesis" EXACT []
 synonym: "sesquiterpenoid formation" EXACT []
@@ -128673,7 +129055,7 @@
 id: GO:0016107
 name: sesquiterpenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sesquiterpenoid compounds, terpenoids with three isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of sesquiterpenoid compounds, terpenoids with three isoprene units." [GOC:curators]
 synonym: "sesquiterpenoid breakdown" EXACT []
 synonym: "sesquiterpenoid catabolism" EXACT []
 synonym: "sesquiterpenoid degradation" EXACT []
@@ -128696,7 +129078,7 @@
 id: GO:0016109
 name: tetraterpenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of tetraterpenoid compounds, terpenoids with eight isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of tetraterpenoid compounds, terpenoids with eight isoprene units." [GOC:curators]
 synonym: "tetraterpene biosynthesis" NARROW []
 synonym: "tetraterpene biosynthetic process" NARROW []
 synonym: "tetraterpenoid anabolism" EXACT []
@@ -128709,7 +129091,7 @@
 id: GO:0016110
 name: tetraterpenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of tetraterpenoid compounds, terpenoids with eight isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of tetraterpenoid compounds, terpenoids with eight isoprene units." [GOC:curators]
 synonym: "tetraterpene catabolic process" NARROW []
 synonym: "tetraterpene catabolism" NARROW []
 synonym: "tetraterpenoid breakdown" EXACT []
@@ -128733,7 +129115,7 @@
 id: GO:0016112
 name: polyterpenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of polyterpenoid compounds, terpenoids with more than eight isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of polyterpenoid compounds, terpenoids with more than eight isoprene units." [GOC:curators]
 synonym: "polyterpene biosynthesis" NARROW []
 synonym: "polyterpene biosynthetic process" NARROW []
 synonym: "polyterpenoid anabolism" EXACT []
@@ -128746,7 +129128,7 @@
 id: GO:0016113
 name: polyterpenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of polyterpenoid compounds, terpenoids with more than eight isoprene units." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of polyterpenoid compounds, terpenoids with more than eight isoprene units." [GOC:curators]
 synonym: "polyterpene catabolic process" NARROW []
 synonym: "polyterpene catabolism" NARROW []
 synonym: "polyterpenoid breakdown" EXACT []
@@ -128790,7 +129172,7 @@
 id: GO:0016117
 name: carotenoid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of carotenoids, tetraterpenoid compounds in which two units of 4 isoprenoid residues joined head-to-tail are themselves joined tail-to-tail." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of carotenoids, tetraterpenoid compounds in which two units of 4 isoprenoid residues joined head-to-tail are themselves joined tail-to-tail." [GOC:curators]
 synonym: "carotenoid anabolism" EXACT []
 synonym: "carotenoid biosynthesis" EXACT []
 synonym: "carotenoid formation" EXACT []
@@ -128804,7 +129186,7 @@
 id: GO:0016118
 name: carotenoid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of carotenoids, tetraterpenoid compounds in which two units of 4 isoprenoid residues joined head-to-tail are themselves joined tail-to-tail." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of carotenoids, tetraterpenoid compounds in which two units of 4 isoprenoid residues joined head-to-tail are themselves joined tail-to-tail." [GOC:curators]
 synonym: "carotenoid breakdown" EXACT []
 synonym: "carotenoid catabolism" EXACT []
 synonym: "carotenoid degradation" EXACT []
@@ -128823,7 +129205,7 @@
 id: GO:0016120
 name: carotene biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of carotenes, hydrocarbon carotenoids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of carotenes, hydrocarbon carotenoids." [GOC:curators]
 synonym: "carotene anabolism" EXACT []
 synonym: "carotene biosynthesis" EXACT []
 synonym: "carotene formation" EXACT []
@@ -128835,7 +129217,7 @@
 id: GO:0016121
 name: carotene catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of carotenes, hydrocarbon carotenoids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of carotenes, hydrocarbon carotenoids." [GOC:curators]
 synonym: "carotene breakdown" EXACT []
 synonym: "carotene catabolism" EXACT []
 synonym: "carotene degradation" EXACT []
@@ -128854,7 +129236,7 @@
 id: GO:0016123
 name: xanthophyll biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of xanthophylls, oxygen-containing carotenoids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of xanthophylls, oxygen-containing carotenoids." [GOC:curators]
 synonym: "xanthophyll anabolism" EXACT []
 synonym: "xanthophyll biosynthesis" EXACT []
 synonym: "xanthophyll formation" EXACT []
@@ -128866,7 +129248,7 @@
 id: GO:0016124
 name: xanthophyll catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of xanthophylls, oxygen-containing carotenoids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of xanthophylls, oxygen-containing carotenoids." [GOC:curators]
 synonym: "xanthophyll breakdown" EXACT []
 synonym: "xanthophyll catabolism" EXACT []
 synonym: "xanthophyll degradation" EXACT []
@@ -128886,7 +129268,7 @@
 id: GO:0016126
 name: sterol biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of sterols, steroids with one or more hydroxyl groups and a hydrocarbon side-chain in the molecule." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of sterols, steroids with one or more hydroxyl groups and a hydrocarbon side-chain in the molecule." [GOC:curators]
 synonym: "sterol anabolism" EXACT []
 synonym: "sterol biosynthesis" EXACT []
 synonym: "sterol formation" EXACT []
@@ -128899,7 +129281,7 @@
 id: GO:0016127
 name: sterol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sterols, steroids with one or more hydroxyl groups and a hydrocarbon side-chain in the molecule." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of sterols, steroids with one or more hydroxyl groups and a hydrocarbon side-chain in the molecule." [GOC:curators]
 synonym: "sterol breakdown" EXACT []
 synonym: "sterol catabolism" EXACT []
 synonym: "sterol degradation" EXACT []
@@ -128920,7 +129302,7 @@
 id: GO:0016129
 name: phytosteroid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of phytosteroids, steroids that differ from animal steroids in having substitutions at C24 and/or a double bond at C22. Phytosteroids are so named because they occur in higher plants; some, notably ergosterol, are also found in fungi." [GOC:go_curators, GOC:mah, ISBN:0471331309]
+def: "The chemical reactions and pathways resulting in the formation of phytosteroids, steroids that differ from animal steroids in having substitutions at C24 and/or a double bond at C22. Phytosteroids are so named because they occur in higher plants; some, notably ergosterol, are also found in fungi." [GOC:curators, GOC:mah, ISBN:0471331309]
 synonym: "phytosteroid anabolism" EXACT []
 synonym: "phytosteroid biosynthesis" EXACT []
 synonym: "phytosteroid formation" EXACT []
@@ -128931,7 +129313,7 @@
 id: GO:0016130
 name: phytosteroid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of phytosteroids, steroids that differ from animal steroids in having substitutions at C24 and/or a double bond at C22. Phytosteroids are so named because they occur in higher plants; some, notably ergosterol, are also found in fungi." [GOC:go_curators, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of phytosteroids, steroids that differ from animal steroids in having substitutions at C24 and/or a double bond at C22. Phytosteroids are so named because they occur in higher plants; some, notably ergosterol, are also found in fungi." [GOC:curators, GOC:mah]
 synonym: "phytosteroid breakdown" EXACT []
 synonym: "phytosteroid catabolism" EXACT []
 synonym: "phytosteroid degradation" EXACT []
@@ -128987,7 +129369,7 @@
 id: GO:0016135
 name: saponin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of saponins, glycosides of plants in which the aglycan (sapogenin) group is a terpene or steroid and the sugar group is a glucose, a galactose, a pentose, a methylpentose or an oligosaccharide. Saponins are powerful surfactant agents and membrane active; they are, hence, toxic to animals on injection." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of saponins, glycosides of plants in which the aglycan (sapogenin) group is a terpene or steroid and the sugar group is a glucose, a galactose, a pentose, a methylpentose or an oligosaccharide. Saponins are powerful surfactant agents and membrane active; they are, hence, toxic to animals on injection." [GOC:curators]
 synonym: "saponin anabolism" EXACT []
 synonym: "saponin biosynthesis" EXACT []
 synonym: "saponin formation" EXACT []
@@ -128999,7 +129381,7 @@
 id: GO:0016136
 name: saponin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of saponins, glycosides of plants in which the aglycan (sapogenin) group is a terpene or steroid and the sugar group is a glucose, a galactose, a pentose, a methylpentose or an oligosaccharide. Saponins are powerful surfactant agents and membrane active; they are, hence, toxic to animals on injection." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of saponins, glycosides of plants in which the aglycan (sapogenin) group is a terpene or steroid and the sugar group is a glucose, a galactose, a pentose, a methylpentose or an oligosaccharide. Saponins are powerful surfactant agents and membrane active; they are, hence, toxic to animals on injection." [GOC:curators]
 synonym: "saponin breakdown" EXACT []
 synonym: "saponin catabolism" EXACT []
 synonym: "saponin degradation" EXACT []
@@ -129022,7 +129404,7 @@
 name: glycoside biosynthetic process
 namespace: biological_process
 alt_id: GO:0016141
-def: "The chemical reactions and pathways resulting in the formation of glycosides, compounds in which a glycosyl group is substituted into a hydroxyl, thiol or selenol group in another compound." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of glycosides, compounds in which a glycosyl group is substituted into a hydroxyl, thiol or selenol group in another compound." [GOC:curators]
 synonym: "glycoside anabolism" EXACT []
 synonym: "glycoside biosynthesis" EXACT []
 synonym: "glycoside formation" EXACT []
@@ -129040,7 +129422,7 @@
 name: glycoside catabolic process
 namespace: biological_process
 alt_id: GO:0016142
-def: "The chemical reactions and pathways resulting in the breakdown of glycosides, compounds in which a glycosyl group is substituted into a hydroxyl, thiol or selenol group in another compound." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glycosides, compounds in which a glycosyl group is substituted into a hydroxyl, thiol or selenol group in another compound." [GOC:curators]
 synonym: "glycoside breakdown" EXACT []
 synonym: "glycoside catabolism" EXACT []
 synonym: "glycoside degradation" EXACT []
@@ -129188,7 +129570,7 @@
 xref: EC:4.2.1.49
 xref: KEGG_REACTION:R02914
 xref: MetaCyc:UROCANATE-HYDRATASE-RXN
-xref: Reactome:R-HSA-70903 "urocanate + H2O => 4-imidazolone-5-propionate"
+xref: Reactome:R-HSA-70903 "urocanate + H2O => 4-imidazolone-5-propanoate"
 xref: RHEA:13101
 is_a: GO:0016836 ! hydro-lyase activity
 property_value: skos:exactMatch EC:4.2.1.49
@@ -129278,6 +129660,7 @@
 synonym: "phytate 3-phosphatase activity" EXACT []
 xref: EC:3.1.3.8
 xref: MetaCyc:RXN0-1001
+xref: Reactome:R-HSA-1855225 "IP6 is dephosphorylated to I(1,2,4,5,6)P5 by MINPP1 in the ER lumen"
 xref: RHEA:16989
 is_a: GO:0004446 ! inositol hexakisphosphate phosphatase activity
 property_value: skos:broadMatch EC:3.1.3.62
@@ -129487,7 +129870,7 @@
 id: GO:0016174
 name: NAD(P)H oxidase H2O2-forming activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + hydrogen peroxide." [EC:1.6.3.1, PMID:10401672, PMID:10601291, PMID:11822874]
+def: "Catalysis of the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + H2O2." [EC:1.6.3.1, PMID:10401672, PMID:10601291, PMID:11822874]
 synonym: "dual oxidase activity" RELATED [EC:1.6.3.1]
 synonym: "NAD(P)H oxidase activity" BROAD []
 synonym: "NAD(P)H:oxygen oxidoreductase activity" RELATED [EC:1.6.3.1]
@@ -129499,7 +129882,7 @@
 synonym: "thyroid oxidase 2 activity" NARROW [EC:1.6.3.1]
 synonym: "thyroid oxidase activity" NARROW [EC:1.6.3.1]
 xref: EC:1.6.3.1
-xref: Reactome:R-HSA-5693681 "DUOX1,2 reduce O2 to H2O2"
+xref: Reactome:R-HSA-5693681 "DUOX1 reduces O2 to H2O2"
 xref: Reactome:R-HSA-9698758 "FLT3 ITD- and NOX4-dependent H2O2 production"
 xref: RHEA:11260
 xref: RHEA:11264
@@ -129715,7 +130098,7 @@
 id: GO:0016202
 name: regulation of striated muscle tissue development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of striated muscle development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of striated muscle development." [GOC:curators]
 is_a: GO:0048634 ! regulation of muscle organ development
 is_a: GO:1901861 ! regulation of muscle tissue development
 intersection_of: GO:0065007 ! biological regulation
@@ -129804,7 +130187,6 @@
 xref: MetaCyc:4-COUMARATE--COA-LIGASE-RXN
 xref: RHEA:19641
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.12
 property_value: skos:exactMatch RHEA:19641
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -129813,8 +130195,10 @@
 id: GO:0016208
 name: AMP binding
 namespace: molecular_function
-def: "Binding to AMP, adenosine monophosphate." [GOC:go_curators]
+def: "Binding to AMP, adenosine monophosphate." [GOC:curators]
 xref: Reactome:R-HSA-380930 "Phosphorylated AMPK binds AMP"
+xref: Reactome:R-HSA-453342 "PYGM dimer, b form + 2 AMP => PYGM b dimer:AMP complex"
+xref: Reactome:R-HSA-453356 "PYGB dimer, b form + 2 AMP => PYGB b dimer:AMP complex"
 is_a: GO:0032559 ! adenyl ribonucleotide binding
 is_a: GO:0043168 ! anion binding
 is_a: GO:0043169 ! cation binding
@@ -129856,13 +130240,13 @@
 id: GO:0016211
 name: ammonia ligase activity
 namespace: molecular_function
-def: "Catalysis of the ligation of ammonia (NH3) to another substance via a carbon-nitrogen bond with concomitant breakage of a diphosphate linkage, usually in a nucleoside triphosphate." [GOC:jl]
+def: "Catalysis of the ligation of ammonia (NH4+) to another substance via a carbon-nitrogen bond with concomitant breakage of a diphosphate linkage, usually in a nucleoside triphosphate." [GOC:jl]
 is_a: GO:0016880 ! acid-ammonia (or amide) ligase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28243" xsd:anyURI
 
 [Term]
 id: GO:0016212
-name: kynurenine-oxoglutarate transaminase activity
+name: L-kynurenine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2-oxoglutarate + L-kynurenine = H2O + kynurenate + L-glutamate. The product 4-(2-aminophenyl)-2,4-dioxobutanoate is converted into kynurenate by a spontaneous reaction. Also acts on 3-hydroxykynurenine to form xanthurenate." [EC:2.6.1.7]
 synonym: "kynurenine 2-oxoglutarate transaminase activity" RELATED [EC:2.6.1.7]
@@ -129870,14 +130254,15 @@
 synonym: "kynurenine transaminase (cyclizing)" RELATED [EC:2.6.1.7]
 synonym: "kynurenine--oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.7]
 synonym: "kynurenine-oxoglutarate aminotransferase activity" EXACT []
+synonym: "kynurenine-oxoglutarate transaminase activity" EXACT []
 synonym: "L-kynurenine aminotransferase activity" RELATED [EC:2.6.1.7]
 synonym: "L-kynurenine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.7]
 xref: EC:2.6.1.7
 xref: MetaCyc:RXN-21858
 xref: MetaCyc:RXN-21859
 xref: Reactome:R-HSA-893583 "kynurenine + 2-oxoglutarate => 4-(2-aminophenyl)-2,4-dioxobutanoic acid + glutamate"
-xref: Reactome:R-HSA-893596 "PXLP-KYAT1 dimer transaminates L-KYN to AP-DOBu"
-xref: Reactome:R-HSA-901097 "kynurenine + pyruvate => 4-(2-aminophenyl)-2,4-dioxobutanoic acid + alanine [CCBL2]"
+xref: Reactome:R-HSA-893596 "PXLP-KYAT1 dimer transaminates L-KYN to KYNA"
+xref: Reactome:R-HSA-901097 "PXLP-KYAT3 (CCBL2) dimer transaminates L-KYN to KYNA"
 xref: RHEA:65560
 xref: RHEA:65884
 is_a: GO:0008483 ! transaminase activity
@@ -129887,6 +130272,7 @@
 property_value: skos:narrowMatch RHEA:65560
 property_value: skos:narrowMatch RHEA:65884
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25811" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0016213
@@ -129955,11 +130341,12 @@
 
 [Term]
 id: GO:0016217
-name: N-ethylammeline chlorohydrolase activity
+name: obsolete N-ethylammeline chlorohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: deethylsimazine + H2O = N-ethylammeline + chloride + H+." [MetaCyc:R465-RXN]
-xref: MetaCyc:R465-RXN
-is_a: GO:0019120 ! hydrolase activity, acting on halide bonds, in C-halide compounds
+def: "OBSOLETE. Catalysis of the reaction: deethylsimazine + H2O = N-ethylammeline + chloride + H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0016218
@@ -129987,7 +130374,7 @@
 id: GO:0016223
 name: beta-alanine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + 2-oxopropanoate = pyruvate + beta-alanine." [RHEA:14077]
+def: "Catalysis of the reaction: L-alanine + 3-oxopropanoate = beta-alanine + pyruvate." [RHEA:14077]
 synonym: "beta-alanine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.18]
 synonym: "beta-alanine-alpha-alanine transaminase activity" RELATED [EC:2.6.1.18]
 synonym: "beta-alanine-pyruvate aminotransferase activity" EXACT []
@@ -129997,7 +130384,7 @@
 xref: MetaCyc:2.6.1.18-RXN
 xref: Reactome:R-HSA-909776 "beta-alanine + pyruvate => 3-oxopropanoate + alanine"
 xref: RHEA:14077
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.18
 property_value: skos:exactMatch RHEA:14077
 
@@ -130064,6 +130451,7 @@
 def: "Catalysis of the synthesis of the HKK-1 carbohydrate epitope; adds a sulfate group to a precursor, GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R), forming sulfo-3GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R)." [PMID:9478973]
 synonym: "HNK-1 sulphotransferase activity" EXACT []
 xref: Reactome:R-HSA-6786048 "CHST10 transfers SO4(2-) from PAPS to GlcA-LacN on NCAM1"
+xref: Reactome:R-HSA-9940718 "CHST10 sulfates matriglycan chain"
 is_a: GO:0008146 ! sulfotransferase activity
 
 [Term]
@@ -130121,7 +130509,7 @@
 id: GO:0016239
 name: positive regulation of macroautophagy
 namespace: biological_process
-def: "Any process, such as recognition of nutrient depletion, that activates or increases the rate of macroautophagy to bring cytosolic macromolecules to the vacuole/lysosome for degradation." [GOC:go_curators, PMID:9412464]
+def: "Any process, such as recognition of nutrient depletion, that activates or increases the rate of macroautophagy to bring cytosolic macromolecules to the vacuole/lysosome for degradation." [GOC:curators, PMID:9412464]
 synonym: "activation of macroautophagy" NARROW []
 synonym: "positive regulation of starvation-induced autophagy" EXACT [GOC:ascb_2009, GOC:dph, GOC:tb]
 synonym: "stimulation of macroautophagy" NARROW []
@@ -130160,7 +130548,7 @@
 id: GO:0016242
 name: negative regulation of macroautophagy
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of macroautophagy." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of macroautophagy." [GOC:curators]
 synonym: "down regulation of macroautophagy" EXACT []
 synonym: "down-regulation of macroautophagy" EXACT []
 synonym: "downregulation of macroautophagy" EXACT []
@@ -130306,7 +130694,7 @@
 id: GO:0016259
 name: selenocysteine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:curators, ISBN:0198506732]
 synonym: "selenocysteine metabolism" EXACT []
 xref: Reactome:R-HSA-2408522 "Selenoamino acid metabolism"
 is_a: GO:1901605 ! alpha-amino acid metabolic process
@@ -130315,7 +130703,7 @@
 id: GO:0016260
 name: L-selenocysteine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of L-selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:curators, ISBN:0198506732]
 synonym: "selenocysteine anabolism" EXACT []
 synonym: "selenocysteine biosynthesis" EXACT []
 synonym: "selenocysteine formation" EXACT []
@@ -130334,7 +130722,7 @@
 id: GO:0016261
 name: selenocysteine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of selenocysteine, an essential component of glutathione peroxidase and some other proteins." [GOC:curators, ISBN:0198506732]
 synonym: "selenocysteine breakdown" EXACT []
 synonym: "selenocysteine catabolism" EXACT []
 synonym: "selenocysteine degradation" EXACT []
@@ -131092,7 +131480,7 @@
 id: GO:0016323
 name: basolateral plasma membrane
 namespace: cellular_component
-def: "The region of the plasma membrane that includes the basal end and sides of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:go_curators]
+def: "The region of the plasma membrane that includes the basal end and sides of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:curators]
 is_a: GO:0098590 ! plasma membrane region
 relationship: part_of GO:0009925 ! basal plasma membrane
 
@@ -131335,7 +131723,7 @@
 id: GO:0016352
 name: obsolete insecticide susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "insecticide susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -131345,7 +131733,7 @@
 id: GO:0016353
 name: obsolete carbamate susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "carbamate susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -131355,7 +131743,7 @@
 id: GO:0016354
 name: obsolete cyclodiene susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "cyclodiene susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -131375,7 +131763,7 @@
 id: GO:0016356
 name: obsolete organophosphorus susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "organophosphorus susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -131385,7 +131773,7 @@
 id: GO:0016357
 name: obsolete pyrethroid susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "pyrethroid susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -131403,7 +131791,7 @@
 id: GO:0016360
 name: sensory organ precursor cell fate determination
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into a sensory organ precursor cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into a sensory organ precursor cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 synonym: "sense organ precursor cell fate determination" EXACT [GOC:dph]
 is_a: GO:0060582 ! cell fate determination involved in pattern specification
 relationship: part_of GO:0008052 ! sensory organ boundary specification
@@ -131514,15 +131902,16 @@
 xref: Reactome:R-HSA-159443 "benzoate + Coenzyme A + ATP => benzoyl-CoA + AMP + pyrophosphate"
 xref: Reactome:R-HSA-159567 "ACSM2B-like proteins transform ST to ST-CoA"
 xref: Reactome:R-HSA-177157 "phenylacetate + Coenzyme A + ATP => phenylacetyl-CoA + AMP + pyrophosphate"
-xref: Reactome:R-HSA-6798345 "PXLP-GCAT dimer ligates CoASH to 2A-3OB to form Gly and Ac-CoA"
 xref: Reactome:R-HSA-9749971 "ACSM2B-like proteins transform 2,5-DHBA to 2,5-DHB-CoA"
-is_a: GO:0016877 ! ligase activity, forming carbon-sulfur bonds
+is_a: GO:0016878 ! acid-thiol ligase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31616" xsd:anyURI
 
 [Term]
 id: GO:0016406
 name: carnitine O-acyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an acyl group to an oxygen atom on the carnitine molecule." [GOC:ai]
+xref: Reactome:R-HSA-200410 "CPT2 converts acylcarnitine to acyl-CoA"
 is_a: GO:0008374 ! O-acyltransferase activity
 
 [Term]
@@ -131687,7 +132076,7 @@
 name: tRNA (guanine) methyltransferase activity
 namespace: molecular_function
 alt_id: GO:0016424
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine in tRNA = S-adenosyl-L-homocysteine + tRNA containing methylguanine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine in tRNA = S-adenosyl-L-homocysteine + tRNA containing methylguanine." [GOC:curators]
 synonym: "tRNA (guanosine) methyltransferase activity" EXACT []
 is_a: GO:0008175 ! tRNA methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
@@ -131697,7 +132086,7 @@
 id: GO:0016426
 name: tRNA (adenine) methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methyladenine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methyladenine." [GOC:curators]
 synonym: "tRNA (adenine-C2-)-methyltransferase activity" BROAD []
 is_a: GO:0008175 ! tRNA methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
@@ -131706,19 +132095,20 @@
 id: GO:0016427
 name: tRNA (cytidine) methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methylcytosine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methylcytosine." [GOC:curators]
 synonym: "tRNA (cytosine) methyltransferase activity" EXACT []
 is_a: GO:0008175 ! tRNA methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
 
 [Term]
 id: GO:0016428
-name: tRNA (cytidine-5-)-methyltransferase activity
+name: tRNA (cytidine-N5)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = a 5-methylcytidine in tRNA + S-adenosyl-L-homocysteine + H+. This modification can occur on seversal residues, including cytidine(34), cytidine(40), cytidine(48), and cytidine(49)." [RHEA:61468]
+def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = a 5-methylcytidine in tRNA + S-adenosyl-L-homocysteine + H+. This modification can occur on several residues, including cytidine(34), cytidine(40), cytidine(48), and cytidine(49)." [RHEA:61468]
 synonym: "S-adenosyl-L-methionine:tRNA (cytosine-5-)-methyltransferase activity" RELATED []
 synonym: "transfer ribonucleate cytosine 5-methyltransferase activity" RELATED []
 synonym: "transfer RNA cytosine 5-methyltransferase activity" RELATED []
+synonym: "tRNA (cytidine-5-)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine-5-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6782388 "NSUN2 methylates cytidine-34, cytidine-48 of unspliced tRNA(Leu)(CAA)"
 xref: Reactome:R-HSA-6782419 "TRDMT1 (DNMT2) methylates cytidine-38 of tRNA(Asp)"
@@ -131730,6 +132120,7 @@
 property_value: skos:exactMatch RHEA:61468
 property_value: skos:relatedMatch EC:2.1.1.202
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24771" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -131754,20 +132145,15 @@
 
 [Term]
 id: GO:0016430
-name: tRNA (adenine-N6)-methyltransferase activity
+name: obsolete tRNA (adenine-N6)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N6-methyladenine." [EC:2.1.1.55]
+def: "OBSOLETE. Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N6-methyladenine." [EC:2.1.1.55]
+comment: This term was obsoleted because there is no evidence that this reaction exists, see PMID:33658722.
 synonym: "S-adenosyl-L-methionine:tRNA (adenine-6-N-)-methyltransferase activity" RELATED [EC:2.1.1.55]
 synonym: "S-adenosyl-L-methionine:tRNA (adenine-N6-)-methyltransferase activity" RELATED [EC:2.1.1.55]
 synonym: "tRNA (adenine-N6-)-methyltransferase activity" EXACT []
-xref: EC:2.1.1.55
-xref: MetaCyc:TRNA-ADENINE-N6--METHYLTRANSFERASE-RXN
-xref: RHEA:16785
-is_a: GO:0008170 ! N-methyltransferase activity
-is_a: GO:0016426 ! tRNA (adenine) methyltransferase activity
-property_value: skos:exactMatch EC:2.1.1.55
-property_value: skos:exactMatch RHEA:16785
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0016432
@@ -131789,14 +132175,14 @@
 id: GO:0016433
 name: rRNA (adenine) methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyladenine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyladenine." [GOC:curators]
 is_a: GO:0008649 ! rRNA methyltransferase activity
 
 [Term]
 id: GO:0016434
 name: rRNA (cytosine) methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methylcytosine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methylcytosine." [GOC:curators]
 is_a: GO:0008649 ! rRNA methyltransferase activity
 
 [Term]
@@ -131811,7 +132197,7 @@
 id: GO:0016436
 name: rRNA (uridine) methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyluridine." [GOC:go-curators]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyluridine." [GOC:curators]
 is_a: GO:0008649 ! rRNA methyltransferase activity
 
 [Term]
@@ -132146,7 +132532,7 @@
 id: GO:0016479
 name: negative regulation of transcription by RNA polymerase I
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:curators]
 synonym: "down regulation of transcription from RNA polymerase I promoter" EXACT []
 synonym: "down-regulation of transcription from RNA polymerase I promoter" EXACT []
 synonym: "downregulation of transcription from RNA polymerase I promoter" EXACT []
@@ -132163,7 +132549,7 @@
 id: GO:0016480
 name: negative regulation of transcription by RNA polymerase III
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase III." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase III." [GOC:curators]
 synonym: "down regulation of transcription from RNA polymerase III promoter" EXACT []
 synonym: "down-regulation of transcription from RNA polymerase III promoter" EXACT []
 synonym: "downregulation of transcription from RNA polymerase III promoter" EXACT []
@@ -132216,6 +132602,7 @@
 synonym: "protein maturation by peptide bond cleavage" EXACT [GOC:bf]
 synonym: "protein maturation by peptide bond hydrolysis" EXACT [GOC:mah]
 synonym: "protein maturation by proteolysis" RELATED [GOC:mah]
+xref: Reactome:R-HSA-159782 "Removal of aminoterminal propeptides from gamma-carboxylated proteins"
 xref: Reactome:R-HSA-9033500 "TYSND1 cleaves peroxisomal proteins"
 is_a: GO:0006508 ! proteolysis
 is_a: GO:0051604 ! protein maturation
@@ -132235,7 +132622,7 @@
 id: GO:0016487
 name: farnesol metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving the sesquiterpenoid alcohol farnesol, 3,7,11-trimethyl-2,6,10,dodecatrien-1-ol." [GOC:go_curators]
+def: "The chemical reactions and pathways involving the sesquiterpenoid alcohol farnesol, 3,7,11-trimethyl-2,6,10,dodecatrien-1-ol." [GOC:curators]
 synonym: "farnesol metabolism" EXACT []
 is_a: GO:0006714 ! sesquiterpenoid metabolic process
 is_a: GO:0016093 ! polyprenol metabolic process
@@ -132245,7 +132632,7 @@
 id: GO:0016488
 name: farnesol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of the sesquiterpenoid alcohol farnesol, 3,7,11-trimethyl-2,6,10,dodecatrien-1-ol." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of the sesquiterpenoid alcohol farnesol, 3,7,11-trimethyl-2,6,10,dodecatrien-1-ol." [GOC:curators]
 synonym: "farnesol breakdown" EXACT []
 synonym: "farnesol catabolism" EXACT []
 synonym: "farnesol degradation" EXACT []
@@ -132280,15 +132667,11 @@
 synonym: "redox activity" EXACT []
 xref: EC:1.-.-.-
 xref: Reactome:R-HSA-1614362 "SUMF1 mediates the oxidation of cysteine to formylglycine, producing active arylsulfatases"
-xref: Reactome:R-HSA-209921 "Monoiodinated tyrosine can be deiodinated"
-xref: Reactome:R-HSA-209960 "Diiodinated tyrosine can be deiodinated"
 xref: Reactome:R-HSA-2161612 "PGH2 is reduced to PGF2a by FAM213B"
 xref: Reactome:R-HSA-265296 "Arachidonate is oxidised to 5S-HpETE by ALOX5"
 xref: Reactome:R-HSA-266051 "5S-HpETE is dehydrated to LTA4 by ALOX5"
 xref: Reactome:R-HSA-390425 "FAR1 reduces PalmCoA to HXOL"
 xref: Reactome:R-HSA-390438 "FAR2 reduces PalmCoA to HXOL"
-xref: Reactome:R-HSA-5662660 "Dopachrome is transformed to DHICA by DCT"
-xref: Reactome:R-HSA-8878581 "TYRP1 oxidises DHICA to IQCA"
 xref: Reactome:R-HSA-8936442 "MARC1,MARC2 reduce N-hydroxylated compounds"
 xref: Reactome:R-HSA-9020249 "Hydroperoxy reductase reduces 4(S)-Hp-17(S)-HDHA to RvD6"
 xref: Reactome:R-HSA-9020260 "Hydroperoxy reductase reduces 7(S)-Hp-17(S)-HDHA to RvD5"
@@ -132598,7 +132981,7 @@
 id: GO:0016525
 name: negative regulation of angiogenesis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of angiogenesis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of angiogenesis." [GOC:curators]
 synonym: "down regulation of angiogenesis" EXACT []
 synonym: "down-regulation of angiogenesis" EXACT []
 synonym: "downregulation of angiogenesis" EXACT []
@@ -132839,6 +133222,7 @@
 comment: The term 'RNA editing' (GO:0016547) was merged into 'RNA modification' (GO:0009451) on the basis of statements in the preface of Modification and Editing of RNA (ISBN:1555811337) that there is no clear distinction between modification and editing. Parallel changes were made for substrate (e.g. tRNA, rRNA, etc.) specific child terms of 'RNA editing'.
 synonym: "mRNA editing" NARROW [GOC:dph, GOC:hjd, GOC:tb]
 xref: Reactome:R-HSA-75072 "mRNA Editing"
+xref: Reactome:R-HSA-9937008 "Mitochondrial mRNA modification"
 is_a: GO:0009451 ! RNA modification
 is_a: GO:0016071 ! mRNA metabolic process
 
@@ -133122,7 +133506,7 @@
 id: GO:0016583
 name: obsolete nucleosome modeling
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it has been replaced by more specific terms.
 synonym: "nucleosome modeling" EXACT []
 is_obsolete: true
@@ -133572,7 +133956,6 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [EC:1.3.-.-]
 synonym: "oxidoreductase activity, acting on the CH-CH group of donors, other acceptors" NARROW []
 xref: EC:1.3.-.-
-xref: Reactome:R-HSA-2995334 "COX15 transforms heme O to heme A"
 xref: Reactome:R-HSA-9661710 "An unknown oxidase oxidises D-UBGN to UBN"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.3.-.-
@@ -133584,7 +133967,6 @@
 namespace: molecular_function
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces NAD or NADP." [EC:1.3.1.-]
 xref: EC:1.3.1.-
-xref: Reactome:R-HSA-4419979 "SRD5A3 reduces pPNOL to DCHOL"
 xref: Reactome:R-HSA-4755572 "Defective SRD5A3 does not reduce pPNOL to DCHOL"
 xref: Reactome:R-HSA-9661726 "An unknown reductase reduces D-UBGN to STBN"
 xref: Reactome:R-HSA-9661745 "An unknown BILR reduces BIL to D-UBGN"
@@ -133837,6 +134219,7 @@
 namespace: molecular_function
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces quinone or similar compound." [GOC:jl]
 xref: EC:1.5.5.-
+xref: Reactome:R-HSA-70670 "PRODH oxidises L-Pro to 1PYR-5COOH"
 is_a: GO:0016645 ! oxidoreductase activity, acting on the CH-NH group of donors
 property_value: skos:exactMatch EC:1.5.5.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134067,6 +134450,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces disulfide." [GOC:jl]
 synonym: "oxidoreductase activity, acting on sulphur group of donors, disulphide as acceptor" EXACT []
 xref: EC:1.8.4.-
+xref: Reactome:R-HSA-9035484 "MPST and TXN2 enable the synthesis of disulfanide from CysS248-MPST"
 is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
 property_value: skos:exactMatch EC:1.8.4.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134112,7 +134496,6 @@
 synonym: "oxidoreductase activity, acting on haem group of donors" EXACT []
 synonym: "oxidoreductase activity, acting on heme group of donors, other acceptors" NARROW []
 xref: EC:1.9.-.-
-xref: Reactome:R-HSA-5340226 "CYGB dioxygenates NO"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.9.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134213,7 +134596,7 @@
 id: GO:0016688
 name: L-ascorbate peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-ascorbate + hydrogen peroxide = dehydroascorbate + 2 H2O." [EC:1.11.1.11]
+def: "Catalysis of the reaction: L-ascorbate + H2O2 = dehydroascorbate + 2 H2O." [EC:1.11.1.11]
 synonym: "ascorbate peroxidase activity" RELATED [EC:1.11.1.11]
 synonym: "ascorbic acid peroxidase activity" RELATED [EC:1.11.1.11]
 synonym: "L-ascorbate:hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.11]
@@ -134231,7 +134614,7 @@
 id: GO:0016689
 name: manganese peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 Mn2+ + 2 H+ + hydrogen peroxide = 2 Mn3+ + 2 H2O." [EC:1.11.1.13]
+def: "Catalysis of the reaction: 2 Mn2+ + 2 H+ + H2O2 = 2 Mn3+ + 2 H2O." [EC:1.11.1.13]
 synonym: "Mn(II):hydrogen-peroxide oxidoreductase activity" RELATED [EC:1.11.1.13]
 synonym: "Mn-dependent (NADH-oxidizing) peroxidase activity" RELATED [EC:1.11.1.13]
 synonym: "Mn-dependent peroxidase activity" RELATED [EC:1.11.1.13]
@@ -134461,6 +134844,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from each of two donors, and molecular oxygen is reduced or incorporated into a donor." [EC:1.14.-.-]
 synonym: "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, miscellaneous" NARROW []
 xref: EC:1.14.-.-
+xref: Reactome:R-HSA-5333612 "MeSebGalNac is oxidatively cleaved to MeSeO2H and 2-acetamidoglucal"
 is_a: GO:0016491 ! oxidoreductase activity
 property_value: skos:exactMatch EC:1.14.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -134598,7 +134982,7 @@
 xref: EC:1.14.14.-
 xref: EC:1.14.14.1
 xref: MetaCyc:UNSPECIFIC-MONOOXYGENASE-RXN
-xref: Reactome:R-HSA-194678 "CYP51A1 demethylates LNSOL"
+xref: Reactome:R-HSA-194678 "CYP51A1 demethylates LAN"
 xref: Reactome:R-HSA-211966 "CYP2D6 4-hydroxylates debrisoquine"
 xref: RHEA:17149
 is_a: GO:0004497 ! monooxygenase activity
@@ -134929,6 +135313,7 @@
 xref: EC:2.-.-.-
 xref: Reactome:R-HSA-1483089 "PE is converted to PS by PTDSS2"
 xref: Reactome:R-HSA-1483186 "PC is converted to PS by PTDSS1"
+xref: Reactome:R-HSA-5333607 "GSSeH combines with bGalNAc derivative to form GSSebGalNac"
 xref: Reactome:R-HSA-5668414 "TRAF2 ubiquitinates cIAP1,2 in cIAP1,2:TRAF1:TRAF2:TRAF3:NIK"
 xref: Reactome:R-HSA-8868783 "TSR3 transfers aminocarboxypropyl group from S-adenosylmethionine to N(1)-methylpseudouridine-1248 of 18SE rRNA yielding N(1)-methyl-N(3)-aminocarboxypropylpseudouridine-1248"
 is_a: GO:0003824 ! catalytic activity
@@ -135168,6 +135553,7 @@
 synonym: "uridine diphosphoglucose-cellulose glucosyltransferase activity" RELATED [EC:2.4.1.12]
 xref: EC:2.4.1.12
 xref: MetaCyc:CELLULOSE-SYNTHASE-UDP-FORMING-RXN
+xref: Reactome:R-HSA-9932210 "Bcs complex synthesizes cellulose from UDP-Glc"
 xref: RHEA:19929
 is_a: GO:0016759 ! cellulose synthase activity
 is_a: GO:0035251 ! UDP-glucosyltransferase activity
@@ -135309,6 +135695,7 @@
 comment: Note that this term encompasses all kinase activities, as well as activities that transfer other phosphorus-containing groups such as diphosphate or nucleotides.
 subset: goslim_chembl
 xref: EC:2.7.-.-
+xref: Reactome:R-HSA-9932322 "BcsG transfers pEtN from PE onto cellulose"
 is_a: GO:0016740 ! transferase activity
 property_value: skos:exactMatch EC:2.7.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135321,6 +135708,7 @@
 subset: goslim_drosophila
 xref: EC:2.7.1.-
 xref: Reactome:R-HSA-2161193 "abacavir + AMP => abacavir monophosphate + adenosine"
+xref: Reactome:R-HSA-9941039 "FAM20B phosphorylates Xyl moiety"
 is_a: GO:0016772 ! transferase activity, transferring phosphorus-containing groups
 property_value: skos:exactMatch EC:2.7.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135389,6 +135777,8 @@
 xref: EC:2.7.8.-
 xref: Reactome:R-HSA-162742 "(ethanolamineP) mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI -> (ethanolamineP) mannose (a1-2) (ethanolamineP) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI"
 xref: Reactome:R-HSA-163069 "mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI + phosphatidylethanolamine -> (ethanolamineP) mannose (a1-2) mannose (a1-6) (ethanolamineP) mannose (a1-4) glucosaminyl-acyl-PI (acyl-GPI) + diacylglycerol"
+xref: Reactome:R-HSA-9940735 "FKTN transfers RboP to GalNAc-GlcNAc-ManP-DAG1"
+xref: Reactome:R-HSA-9940756 "FKRP dimer transfers RboP to matriglycan chain"
 is_a: GO:0016772 ! transferase activity, transferring phosphorus-containing groups
 property_value: skos:exactMatch EC:2.7.8.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135423,6 +135813,10 @@
 xref: EC:2.8.1.-
 xref: Reactome:R-HSA-1614618 "Persulfide sulfur is transferred onto sulfite"
 xref: Reactome:R-HSA-6782264 "CTU1:CTU2:URM1 thiolates uridine-34 in tRNAs"
+xref: Reactome:R-HSA-9012721 "MPST transfers sulfur atom from 3MPYR to HSO3- to form S2O3(2-) and PYR"
+xref: Reactome:R-HSA-9013471 "MPST transfers sulfur from 3MPYR to HCN to form HSCN"
+xref: Reactome:R-HSA-9013533 "MPST transfers sulfur from sulfanegen to HCN to form HSCN"
+xref: Reactome:R-HSA-9034756 "MPST transfers sulfur atom from 3MPYR to form CysS248-MPST"
 xref: Reactome:R-HSA-947538 "Transfer of sulfur from MOCS3-S-S onto MOCS2A"
 is_a: GO:0016782 ! transferase activity, transferring sulphur-containing groups
 property_value: skos:exactMatch EC:2.8.1.-
@@ -135439,10 +135833,6 @@
 synonym: "mercaptopyruvate sulfurtransferase activity" EXACT []
 xref: EC:2.8.1.2
 xref: MetaCyc:MERCAPYSTRANS-RXN
-xref: Reactome:R-HSA-9012721 "MPST transfers sulfur atom from 3MPYR to HSO3- to form S2O3(2-) and PYR"
-xref: Reactome:R-HSA-9013471 "MPST transfers sulfur from 3MPYR to HCN to form HSCN"
-xref: Reactome:R-HSA-9013533 "MPST transfers sulfur from sulfanegen to HCN to form HSCN"
-xref: Reactome:R-HSA-9034756 "MPST transfers sulfur atom from 3MPYR to form CysS248-MPST"
 xref: RHEA:21740
 is_a: GO:0016783 ! sulfurtransferase activity
 property_value: skos:exactMatch EC:2.8.1.2
@@ -135486,6 +135876,7 @@
 xref: Reactome:R-HSA-6788295 "HDHD1:Mg2+ dephosphorylates PURIDP"
 xref: Reactome:R-HSA-8938314 "ENPPs hydrolyse CoA-SH to PPANT, PAP"
 xref: Reactome:R-HSA-8952137 "Phospholipid phosphatase 6 hydrolyses Presqualene diphosphate to presqualene monophosphate"
+xref: Reactome:R-HSA-9942295 "QNG1 hydrolyzes queuosine 5'-monophosphate yielding queuine and ribose 5'-monophosphate"
 is_a: GO:0003824 ! catalytic activity
 property_value: skos:exactMatch EC:3.-.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135535,6 +135926,7 @@
 xref: Reactome:R-HSA-4419986 "Unknown pPPP phosphatase dephosphorylates pPPP to pPNOL"
 xref: Reactome:R-HSA-9636457 "SapM dephosphorylates PI3P"
 xref: Reactome:R-HSA-9837419 "PANK4 hydrolyzes PPANT to pantetheine"
+xref: Reactome:R-HSA-9940993 "PXYLP1 dephosphorylates Xyl moiety"
 is_a: GO:0042578 ! phosphoric ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.3.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25755" xsd:anyURI
@@ -135608,6 +136000,7 @@
 xref: EC:3.2.-.-
 xref: Reactome:R-HSA-1793176 "DS is cleaved from its proteoglycan"
 xref: Reactome:R-HSA-2065233 "CS is cleaved from its proteoglycan"
+xref: Reactome:R-HSA-5333609 "MeSebGalNac is hydrolysed to MeSeH and bGalNac"
 is_a: GO:0016787 ! hydrolase activity
 property_value: skos:exactMatch EC:3.2.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135763,7 +136156,7 @@
 namespace: molecular_function
 def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a cyclic amide." [EC:3.5.2.-]
 xref: EC:3.5.2.-
-xref: Reactome:R-HSA-70906 "4-imidazolone-5-propionate + H2O => N-formiminoglutamate + 2H+"
+xref: Reactome:R-HSA-70906 "4-imidazolone-5-propanoate + H2O => N-formiminoglutamate"
 is_a: GO:0016810 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
 property_value: skos:exactMatch EC:3.5.2.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -135976,6 +136369,7 @@
 xref: Reactome:R-HSA-6787757 "PXLP-K333-GADL1 decarboxylates acidic AAs"
 xref: Reactome:R-HSA-6814165 "PXLP-K333-GADL1 decarboxylates CSA to HTAU"
 xref: Reactome:R-HSA-71223 "2-amino-3-carboxymuconate semialdehyde => 2-aminomuconate semialdehyde + CO2"
+xref: Reactome:R-HSA-9856737 "Unknown enzyme(s) process 4-HMA to 4-Hbz"
 is_a: GO:0016830 ! carbon-carbon lyase activity
 property_value: skos:exactMatch EC:4.1.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136035,7 +136429,6 @@
 def: "Catalysis of the cleavage of a carbon-oxygen bond by elimination of water." [EC:4.2.1.-]
 xref: EC:4.2.1.-
 xref: Reactome:R-HSA-2161659 "PGE2 is dehydrated to PGA2"
-xref: Reactome:R-HSA-9014627 "SDS dimers:PXLP dehydrate L-Thr to 2AA"
 is_a: GO:0016835 ! carbon-oxygen lyase activity
 property_value: skos:exactMatch EC:4.2.1.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136075,7 +136468,7 @@
 id: GO:0016840
 name: carbon-nitrogen lyase activity
 namespace: molecular_function
-def: "Catalysis of the release of ammonia or one of its derivatives, with the formation of a double bond or ring. Enzymes with this activity may catalyze the actual elimination of the ammonia, amine or amide, e.g. CH-CH(-NH-R) = C=CH- + NH2-R. Others, however, catalyze elimination of another component, e.g. water, which is followed by spontaneous reactions that lead to breakage of the C-N bond, e.g. L-serine ammonia-lyase (EC:4.3.1.17), so that the overall reaction is C(-OH)-CH(-NH2) = CH2-CO- + NH3, i.e. an elimination with rearrangement. The sub-subclasses of EC:4.3 are the ammonia-lyases (EC:4.3.1), lyases acting on amides, amidines, etc. (EC:4.3.2), the amine-lyases (EC:4.3.3), and other carbon-nitrogen lyases (EC:4.3.99)." [EC:4.3.-.-]
+def: "Catalysis of the release of ammonia or one of its derivatives, with the formation of a double bond or ring. Enzymes with this activity may catalyze the actual elimination of the ammonia, amine or amide, e.g. CH-CH(-NH-R) = C=CH- + NH2-R. Others, however, catalyze elimination of another component, e.g. water, which is followed by spontaneous reactions that lead to breakage of the C-N bond, e.g. L-serine ammonia-lyase (EC:4.3.1.17), so that the overall reaction is C(-OH)-CH(-NH2) = CH2-CO- + NH4+, i.e. an elimination with rearrangement. The sub-subclasses of EC:4.3 are the ammonia-lyases (EC:4.3.1), lyases acting on amides, amidines, etc. (EC:4.3.2), the amine-lyases (EC:4.3.3), and other carbon-nitrogen lyases (EC:4.3.99)." [EC:4.3.-.-]
 synonym: "other carbon-nitrogen lyase activity" NARROW []
 xref: EC:4.3.-.-
 is_a: GO:0016829 ! lyase activity
@@ -136152,6 +136545,8 @@
 synonym: "carbon-sulphur lyase activity" EXACT []
 xref: EC:4.4.-.-
 xref: Reactome:R-HSA-1614567 "Excess homocysteine yields homolanthionine and H2S"
+xref: Reactome:R-HSA-1614591 "Excess cysteine yields lanthionine and H2S"
+xref: Reactome:R-HSA-1614614 "Cysteine is degraded to pyruvate and H2S"
 is_a: GO:0016829 ! lyase activity
 property_value: skos:exactMatch EC:4.4.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136272,7 +136667,7 @@
 xref: EC:5.-.-.-
 xref: Reactome:R-HSA-2161666 "PGA2 is isomerised to PGC2"
 xref: Reactome:R-HSA-2161735 "PGC2 is isomerised to PGB2"
-xref: Reactome:R-HSA-6787623 "TSTA3 dimer epimerises GDP-DHDMan to GDP-KDGal"
+xref: Reactome:R-HSA-6787623 "GFUS dimer epimerises GDP-DHDMan to GDP-KDGal"
 is_a: GO:0003824 ! catalytic activity
 property_value: skos:exactMatch EC:5.-.-.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136382,6 +136777,7 @@
 def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor are the same molecule, one or more carbon-carbon double bonds in the molecule are rearranged, and no oxidized product appears." [EC:5.3.3.-, GOC:mah]
 synonym: "intramolecular isomerase activity, transposing C=C bonds" EXACT []
 xref: EC:5.3.3.-
+xref: Reactome:R-HSA-2161563 "PGJ2 is isomerised to delta12-PGJ2"
 is_a: GO:0016860 ! intramolecular oxidoreductase activity
 property_value: skos:exactMatch EC:5.3.3.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -136576,7 +136972,7 @@
 id: GO:0016880
 name: acid-ammonia (or amide) ligase activity
 namespace: molecular_function
-def: "Catalysis of the ligation of an acid to ammonia (NH3) or an amide via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
+def: "Catalysis of the ligation of an acid to ammonia (NH4+) or an amide via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
 synonym: "amide synthase activity" EXACT []
 xref: EC:6.3.1.-
 is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
@@ -136681,6 +137077,9 @@
 xref: Reactome:R-HSA-9668415 "VPS4 mediates disassembly of ESCRTIII subunits to promote sealing of holes in the nuclear envelope"
 xref: Reactome:R-HSA-9706399 "RHOBTB3 hydrolyzes ATP"
 xref: Reactome:R-HSA-9749350 "ATP-dependent release of CDT1 from the OCCM complex"
+xref: Reactome:R-HSA-9948300 "RQT complex:K63polyUb-80S ribosome dissociates yielding K63polyUb-40S subunit and 60S subunit:peptidyl-tRNA with nascent peptide"
+xref: Reactome:R-HSA-9949632 "VCP hexamer extracts K48 polyubiquitinated alanine-tailed nascent peptide from VCP hexamer:UFD1:NPLOC4:LTN1:NEMF:TCF25:60S ribosome subunit:K48polyUb-nascent peptide"
+xref: Reactome:R-HSA-9955731 "ABCE1:PELO:80S Ribosome:non-stop mRNA:peptidyl-tRNA with elongating peptide dissociates yielding ABCE1:40S ribosomal subunit, PELO, and 60S ribosomal subunit:peptidyl-tRNA with nascent peptide"
 xref: RHEA:13065
 is_a: GO:0017111 ! ribonucleoside triphosphate phosphatase activity
 relationship: part_of GO:0140657 ! ATP-dependent activity
@@ -136808,6 +137207,7 @@
 synonym: "exoribonuclease activity, producing 5'-phosphomonoesters" EXACT []
 xref: EC:3.1.13.-
 xref: Reactome:R-HSA-429860 "DCP1-DCP2 complex decaps mRNA"
+xref: Reactome:R-HSA-9946784 "DENV mRNA is decapped"
 is_a: GO:0004532 ! RNA exonuclease activity
 property_value: skos:exactMatch EC:3.1.13.-
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
@@ -137484,8 +137884,7 @@
 xref: MetaCyc:RXN0-1141
 xref: MetaCyc:RXN0-5098
 xref: RHEA:49288
-is_a: GO:0016779 ! nucleotidyltransferase activity
-is_a: GO:0016879 ! ligase activity, forming carbon-nitrogen bonds
+is_a: GO:0016880 ! acid-ammonia (or amide) ligase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
 property_value: skos:exactMatch EC:6.3.1.20
 property_value: skos:exactMatch MetaCyc:RXN-17127
@@ -137493,6 +137892,7 @@
 property_value: skos:narrowMatch MetaCyc:RXN0-1141
 property_value: skos:narrowMatch MetaCyc:RXN0-5098
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31616" xsd:anyURI
 
 [Term]
 id: GO:0016980
@@ -137613,7 +138013,7 @@
 id: GO:0016990
 name: arginine deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-arginine + H2O = L-citrulline + NH3." [EC:3.5.3.6]
+def: "Catalysis of the reaction: L-arginine + H2O = L-citrulline + NH4+." [RHEA:19597]
 synonym: "arginine dihydrolase activity" RELATED [EC:3.5.3.6]
 synonym: "citrulline iminase activity" RELATED [EC:3.5.3.6]
 synonym: "L-arginine deiminase activity" RELATED [EC:3.5.3.6]
@@ -137740,7 +138140,7 @@
 name: cell wall macromolecule catabolic process
 namespace: biological_process
 alt_id: GO:0044039
-def: "The chemical reactions and pathways resulting in the breakdown of macromolecules that form part of a cell wall." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of macromolecules that form part of a cell wall." [GOC:curators]
 synonym: "cell wall breakdown" RELATED [GOC:mah]
 synonym: "cell wall catabolism" RELATED [GOC:mah]
 synonym: "cell wall degradation" RELATED [GOC:mah]
@@ -137763,7 +138163,7 @@
 id: GO:0017000
 name: antibiotic biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of an antibiotic, a substance produced by or derived from certain fungi, bacteria, and other organisms, that can destroy or inhibit the growth of other microorganisms." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of an antibiotic, a substance produced by or derived from certain fungi, bacteria, and other organisms, that can destroy or inhibit the growth of other microorganisms." [GOC:curators]
 subset: goslim_metagenomics
 synonym: "antibiotic anabolism" EXACT []
 synonym: "antibiotic biosynthesis" EXACT []
@@ -137776,7 +138176,7 @@
 id: GO:0017001
 name: antibiotic catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of antibiotic, a substance produced by or derived from certain fungi, bacteria, and other organisms, that can destroy or inhibit the growth of other microorganisms." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of antibiotic, a substance produced by or derived from certain fungi, bacteria, and other organisms, that can destroy or inhibit the growth of other microorganisms." [GOC:curators]
 synonym: "antibiotic breakdown" EXACT []
 synonym: "antibiotic catabolism" EXACT []
 synonym: "antibiotic degradation" EXACT []
@@ -138095,6 +138495,7 @@
 xref: Reactome:R-HSA-1606602 "ASAH1 hydrolyzes ceramide"
 xref: Reactome:R-HSA-428205 "ACER2 hydrolyzes ceramide (Golgi)"
 xref: Reactome:R-HSA-428231 "ACER1 hydrolyzes ceramide (endoplasmic reticulum)"
+xref: Reactome:R-HSA-428262 "ACER3 hydrolyzes phytoceramide"
 xref: RHEA:20856
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:exactMatch EC:3.5.1.23
@@ -138179,8 +138580,6 @@
 def: "OBSOLETE. Catalysis of the reaction: sphingosine + ATP = sphingosine 1-phosphate + ADP." [RHEA:35847]
 comment: This term was obsoleted because it represents a specific substrate of sphingosine kinase activity ; GO:0008481.
 synonym: "sphingosine kinase activity" BROAD []
-xref: Reactome:R-HSA-5218845 "p-SPHK1 phosphorylates sphingosine to sphingosine 1-phosphate"
-xref: Reactome:R-HSA-9625814 "SPHK1 phosphorylates sphingosine in response to E2 stimulation"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30243" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008481
@@ -138229,7 +138628,7 @@
 id: GO:0017055
 name: negative regulation of RNA polymerase II transcription preinitiation complex assembly
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:curators]
 synonym: "down regulation of RNA polymerase II transcriptional preinitiation complex assembly" EXACT []
 synonym: "down-regulation of RNA polymerase II transcriptional preinitiation complex assembly" EXACT []
 synonym: "downregulation of RNA polymerase II transcriptional preinitiation complex assembly" EXACT []
@@ -138255,11 +138654,10 @@
 synonym: "nuclear pore activity" RELATED []
 synonym: "nucleocytoplasmic transporter activity" EXACT []
 xref: Reactome:R-HSA-1176059 "Translocation of Influenza A virus nonstructural protein 1 (NS1A) into the nucleus"
-xref: Reactome:R-HSA-170796 "NPC transports GCK1:GKRP from cytosol to nucleoplasm"
+xref: Reactome:R-HSA-170796 "NPC transports GCK1:GCKR from cytosol to nucleoplasm"
 xref: Reactome:R-HSA-192627 "Viral mRNA Export"
 xref: Reactome:R-HSA-192925 "Export of Spliced Viral mRNA"
 xref: Reactome:R-HSA-5252041 "NPC transports Hikeshi:HSP70s:ATP from cytosol to nucleoplasm"
-xref: Reactome:R-HSA-5578744 "Importin-8 imports AGO2:miRNA into the nucleus"
 xref: Reactome:R-HSA-5661474 "Defective NPC does not transport GCK1:GKRP from cytosol to nucleoplasm"
 xref: Reactome:R-HSA-6783483 "tRNA:XPOT:RAN:GTP translocates from the nucleus to the cytosol"
 is_a: GO:0005198 ! structural molecule activity
@@ -138281,7 +138679,7 @@
 xref: EC:3.1.1.31
 xref: KEGG_REACTION:R02035
 xref: MetaCyc:6PGLUCONOLACT-RXN
-xref: Reactome:R-HSA-71296 "D-glucono-1,5-lactone 6-phosphate + H2O => 6-phospho-D-gluconate"
+xref: Reactome:R-HSA-71296 "PGLS hydrolyzes D-glucono-1,5-lactone 6-phosphate"
 xref: RHEA:12556
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 property_value: skos:exactMatch EC:3.1.1.31
@@ -138292,7 +138690,7 @@
 id: GO:0017058
 name: FH1 domain binding
 namespace: molecular_function
-def: "Binding to a FH1 domain of a protein, a proline-rich domain, usually located in front of a FH2 domain." [GOC:go_curators]
+def: "Binding to a FH1 domain of a protein, a proline-rich domain, usually located in front of a FH2 domain." [GOC:curators]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
@@ -138641,7 +139039,7 @@
 id: GO:0017088
 name: obsolete X-Pro dipeptidyl-peptidase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the hydrolysis of the terminal bond of Xaa-Pro-Xaa motifs to release unblocked, N-terminal dipeptides from substrates including Ala-Pro-para-nitroanilide and (sequentially, at the second, fourth and sixth bonds) of the motif Tyr-Pro-Phe-Pro-Gly-Pro-Ile." [GOC:curator]
+def: "OBSOLETE. Catalysis of the hydrolysis of the terminal bond of Xaa-Pro-Xaa motifs to release unblocked, N-terminal dipeptides from substrates including Ala-Pro-para-nitroanilide and (sequentially, at the second, fourth and sixth bonds) of the motif Tyr-Pro-Phe-Pro-Gly-Pro-Ile." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product.
 synonym: "X-Pro dipeptidyl-peptidase activity" EXACT []
 synonym: "Xaa-Pro dipeptidyl-peptidase activity" EXACT []
@@ -138873,6 +139271,7 @@
 xref: Reactome:R-HSA-5651782 "FEN1 bound to POLB cleaves displaced DNA strand (flap)"
 xref: Reactome:R-HSA-5687664 "FEN1 cleaves displaced ssDNA flaps during MMEJ"
 xref: Reactome:R-HSA-69152 "Removal of remaining Flap"
+xref: Reactome:R-HSA-9914870 "MGME1 cleaves 5' DNA flap from nascent mitochondrial DNA"
 is_a: GO:0016888 ! DNA endonuclease activity, producing 5'-phosphomonoesters
 is_a: GO:0048256 ! flap endonuclease activity
 
@@ -139018,14 +139417,15 @@
 
 [Term]
 id: GO:0017119
-name: Golgi transport complex
+name: COG complex
 namespace: cellular_component
 def: "A multisubunit tethering complex of the CATCHR family (complexes associated with tethering containing helical rods) that has a role in tethering vesicles to the Golgi prior to fusion. Composed of 8 subunits COG1-8." [GOC:krc, PMID:11980916, PMID:20972446, PMID:9792665]
-synonym: "COG complex" EXACT []
 synonym: "conserved oligomeric Golgi complex" EXACT []
+synonym: "Golgi transport complex" EXACT []
 synonym: "Sec34/35 complex" NARROW []
 is_a: GO:0099023 ! vesicle tethering complex
 relationship: part_of GO:0005794 ! Golgi apparatus
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31397" xsd:anyURI
 
 [Term]
 id: GO:0017120
@@ -139066,7 +139466,7 @@
 id: GO:0017124
 name: SH3 domain binding
 namespace: molecular_function
-def: "Binding to a SH3 domain (Src homology 3) of a protein, small protein modules containing approximately 50 amino acid residues found in a great variety of intracellular or membrane-associated proteins." [GOC:go_curators, Pfam:PF00018]
+def: "Binding to a SH3 domain (Src homology 3) of a protein, small protein modules containing approximately 50 amino acid residues found in a great variety of intracellular or membrane-associated proteins." [GOC:curators, Pfam:PF00018]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
@@ -139095,8 +139495,7 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phospholipids from one membrane bilayer leaflet to the other, by an ATP-independent mechanism." [GOC:cjm, PMID:20043909, PMID:20302864]
 comment: Nomenclature note. Scramblases are ATP-independent, non-selective, translocases inducing non-specific transbilayer movements across the membrane. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases).
-is_a: GO:0005548 ! phospholipid transporter activity
-is_a: GO:0140303 ! intramembrane lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 relationship: part_of GO:0017121 ! plasma membrane phospholipid scrambling
 
 [Term]
@@ -139192,7 +139591,7 @@
 id: GO:0017139
 name: obsolete arsenate sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "arsenate sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -139202,7 +139601,7 @@
 id: GO:0017141
 name: obsolete antibiotic susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "antibiotic susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -139212,7 +139611,7 @@
 id: GO:0017142
 name: obsolete toxin susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "toxin susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -139347,7 +139746,7 @@
 id: GO:0017156
 name: calcium-ion regulated exocytosis
 namespace: biological_process
-def: "The release of intracellular molecules (e.g. hormones, matrix proteins) contained within a membrane-bounded vesicle by fusion of the vesicle with the plasma membrane of a cell, induced by a rise in cytosolic calcium-ion levels." [GOC:go_curators]
+def: "The release of intracellular molecules (e.g. hormones, matrix proteins) contained within a membrane-bounded vesicle by fusion of the vesicle with the plasma membrane of a cell, induced by a rise in cytosolic calcium-ion levels." [GOC:curators]
 comment: Note that the calcium regulated exocytosis pathway can be leaky: some level of exocytosis via this pathway occurs spontaneously. This phenomenon underlies spontaneous neurotransmitter release from presynapses. This phenomenon is distinct from the unregulated, calcium independent exocytosis pathway.
 synonym: "calcium ion-dependent exocytosis" EXACT []
 is_a: GO:0045055 ! regulated exocytosis
@@ -139356,7 +139755,7 @@
 id: GO:0017157
 name: regulation of exocytosis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of exocytosis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of exocytosis." [GOC:curators]
 is_a: GO:0060627 ! regulation of vesicle-mediated transport
 is_a: GO:1903530 ! regulation of secretion by cell
 intersection_of: GO:0065007 ! biological regulation
@@ -139367,7 +139766,7 @@
 id: GO:0017158
 name: regulation of calcium ion-dependent exocytosis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:curators]
 is_a: GO:1903305 ! regulation of regulated secretory pathway
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0017156 ! calcium-ion regulated exocytosis
@@ -139603,7 +140002,7 @@
 id: GO:0017179
 name: obsolete peptidyl-diphthine metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving peptidyl-diphthine, a modified histidine residue." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving peptidyl-diphthine, a modified histidine residue." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "peptidyl-diphthine metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25694" xsd:anyURI
@@ -139613,7 +140012,7 @@
 id: GO:0017180
 name: obsolete protein histidyl modification to diphthine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of peptidyl-diphthine from other compounds, including peptidyl-histidine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of peptidyl-diphthine from other compounds, including peptidyl-histidine." [GOC:curators]
 comment: This term was obsoleted because there is no evidence that this process exists.
 synonym: "peptidyl-diphthine anabolism from peptidyl-histidine" EXACT []
 synonym: "peptidyl-diphthine biosynthetic process from peptidyl-histidine" EXACT []
@@ -139626,7 +140025,7 @@
 id: GO:0017181
 name: obsolete peptidyl-diphthine catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of peptidyl-diphthine, a modified histidine residue." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of peptidyl-diphthine, a modified histidine residue." [GOC:curators]
 comment: This term was obsoleted because there is no evidence that this process exists.
 synonym: "peptidyl-diphthine breakdown" EXACT []
 synonym: "peptidyl-diphthine catabolism" EXACT []
@@ -139638,7 +140037,7 @@
 id: GO:0017182
 name: obsolete peptidyl-diphthamide metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving peptidyl-diphthamide, a modified histidine residue." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving peptidyl-diphthamide, a modified histidine residue." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "peptidyl-diphthamide metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25694" xsd:anyURI
@@ -139661,7 +140060,7 @@
 id: GO:0017184
 name: obsolete peptidyl-diphthamide catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of peptidyl-diphthamide, a modified histidine residue." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of peptidyl-diphthamide, a modified histidine residue." [GOC:curators]
 comment: This term was obsoleted because there is no evidence that this process exists.
 synonym: "peptidyl-diphthamide breakdown" EXACT []
 synonym: "peptidyl-diphthamide catabolism" EXACT []
@@ -139715,7 +140114,7 @@
 xref: EC:2.3.1.17
 xref: KEGG_REACTION:R00487
 xref: MetaCyc:ASPARTATE-N-ACETYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-8954468 "NAT8L transfers acetyl group from Ac-CoA to L-Asp, forming NAA"
+xref: Reactome:R-HSA-8954468 "NAT8L transfers acetyl group from Ac-CoA to L-Asp, forming NAASP"
 xref: RHEA:14165
 is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.17
@@ -141677,14 +142076,13 @@
 namespace: biological_process
 def: "The alteration of an amino acid residue in a peptide." [GOC:mah]
 subset: gocheck_do_not_annotate
-subset: goslim_yeast
 is_a: GO:0036211 ! protein modification process
 
 [Term]
 id: GO:0018194
 name: peptidyl-alanine modification
 namespace: biological_process
-def: "The modification of peptidyl-alanine." [GOC:go_curators]
+def: "The modification of peptidyl-alanine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141692,7 +142090,7 @@
 id: GO:0018195
 name: peptidyl-arginine modification
 namespace: biological_process
-def: "The modification of peptidyl-arginine." [GOC:go_curators]
+def: "The modification of peptidyl-arginine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141700,7 +142098,7 @@
 id: GO:0018196
 name: obsolete peptidyl-asparagine modification
 namespace: biological_process
-def: "OBSOLETE. The modification of peptidyl-asparagine." [GOC:go_curators]
+def: "OBSOLETE. The modification of peptidyl-asparagine." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23829" xsd:anyURI
 is_obsolete: true
@@ -141717,7 +142115,7 @@
 id: GO:0018198
 name: peptidyl-cysteine modification
 namespace: biological_process
-def: "The modification of peptidyl-cysteine." [GOC:go_curators]
+def: "The modification of peptidyl-cysteine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141725,7 +142123,7 @@
 id: GO:0018199
 name: peptidyl-glutamine modification
 namespace: biological_process
-def: "The modification of peptidyl-glutamine." [GOC:go_curators]
+def: "The modification of peptidyl-glutamine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141733,7 +142131,7 @@
 id: GO:0018200
 name: peptidyl-glutamic acid modification
 namespace: biological_process
-def: "The modification of peptidyl-glutamic acid." [GOC:go_curators]
+def: "The modification of peptidyl-glutamic acid." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141741,7 +142139,7 @@
 id: GO:0018201
 name: peptidyl-glycine modification
 namespace: biological_process
-def: "The modification of peptidyl-glycine." [GOC:go_curators]
+def: "The modification of peptidyl-glycine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141757,7 +142155,7 @@
 id: GO:0018203
 name: peptidyl-isoleucine modification
 namespace: biological_process
-def: "The modification of peptidyl-isoleucine." [GOC:go_curators]
+def: "The modification of peptidyl-isoleucine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141765,7 +142163,7 @@
 id: GO:0018204
 name: peptidyl-leucine modification
 namespace: biological_process
-def: "The modification of peptidyl-leucine." [GOC:go_curators]
+def: "The modification of peptidyl-leucine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141773,7 +142171,7 @@
 id: GO:0018205
 name: peptidyl-lysine modification
 namespace: biological_process
-def: "The modification of peptidyl-lysine." [GOC:go_curators]
+def: "The modification of peptidyl-lysine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141781,7 +142179,7 @@
 id: GO:0018206
 name: peptidyl-methionine modification
 namespace: biological_process
-def: "The modification of peptidyl-methionine." [GOC:go_curators]
+def: "The modification of peptidyl-methionine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141789,7 +142187,7 @@
 id: GO:0018207
 name: peptidyl-phenylalanine modification
 namespace: biological_process
-def: "The modification of peptidyl-phenylalanine." [GOC:go_curators]
+def: "The modification of peptidyl-phenylalanine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141797,7 +142195,7 @@
 id: GO:0018208
 name: peptidyl-proline modification
 namespace: biological_process
-def: "The modification of peptidyl-proline." [GOC:go_curators]
+def: "The modification of peptidyl-proline." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141805,7 +142203,7 @@
 id: GO:0018209
 name: peptidyl-serine modification
 namespace: biological_process
-def: "The modification of peptidyl-serine." [GOC:go_curators]
+def: "The modification of peptidyl-serine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141813,7 +142211,7 @@
 id: GO:0018210
 name: peptidyl-threonine modification
 namespace: biological_process
-def: "The modification of peptidyl-threonine." [GOC:go_curators]
+def: "The modification of peptidyl-threonine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141829,7 +142227,7 @@
 id: GO:0018212
 name: peptidyl-tyrosine modification
 namespace: biological_process
-def: "The modification of peptidyl-tyrosine." [GOC:go_curators]
+def: "The modification of peptidyl-tyrosine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -141837,7 +142235,7 @@
 id: GO:0018213
 name: peptidyl-valine modification
 namespace: biological_process
-def: "The modification of peptidyl-valine." [GOC:go_curators]
+def: "The modification of peptidyl-valine." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0018193 ! peptidyl-amino acid modification
 
@@ -142392,15 +142790,6 @@
 def: "OBSOLETE. Catalysis of the reaction: ATP + biotin + protein = AMP + diphosphate + biotin-protein." [GOC:mah]
 comment: The reason for obsoletion is this term is an unnecessary grouping term.
 synonym: "biotin-apoprotein ligase activity" EXACT []
-xref: Reactome:R-HSA-2993447 "HLCS biotinylates 6x(PCCA:PCCB)"
-xref: Reactome:R-HSA-2993799 "HLCS biotinylates 6xMCCC1:6xMCCC2"
-xref: Reactome:R-HSA-2993802 "HLCS biotinylates PC:Mn2+"
-xref: Reactome:R-HSA-2993814 "HLCS biotinylates ACACA:Mn2+"
-xref: Reactome:R-HSA-3323184 "Defective HLCS does not biotinylate ACACA:Mn2+"
-xref: Reactome:R-HSA-4167511 "HLCS biotinylates ACACB"
-xref: Reactome:R-HSA-9035987 "Defective HLCS does not biotinylate 6xMCCC1:6xMCCC2"
-xref: Reactome:R-HSA-9035988 "Defective HLCS does not biotinylate PC:Mn2+"
-xref: Reactome:R-HSA-9035990 "Defective HLCS does not biotinylate 6x(PCCA:PCCB)"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28107" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004077
@@ -142479,9 +142868,6 @@
 def: "OBSOLETE. The glycosylation of protein via the N4 atom of peptidyl-asparagine forming N4-glycosyl-L-asparagine; the most common form is N-acetylglucosaminyl asparagine; N-acetylgalactosaminyl asparagine and N4 glucosyl asparagine also occur. This modification typically occurs in extracellular peptides with an N-X-(ST) motif. Partial modification has been observed to occur with cysteine, rather than serine or threonine, in the third position; secondary structure features are important, and proline in the second or fourth positions inhibits modification." [GOC:jsg]
 comment: The reason for obsoletion is that this term is equivalent to protein N-linked glycosylation.
 synonym: "protein amino acid N-linked glycosylation via asparagine" EXACT [GOC:bf]
-xref: Reactome:R-HSA-162699 "Synthesis of dolichyl-phosphate mannose"
-xref: Reactome:R-HSA-446203 "Asparagine N-linked glycosylation"
-xref: Reactome:R-HSA-480985 "Synthesis of dolichyl-phosphate-glucose"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30880" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0006487
@@ -144403,7 +144789,7 @@
 id: GO:0018488
 name: aryl-aldehyde oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an aromatic aldehyde + O2 + H2O = an aromatic acid + hydrogen peroxide." [EC:1.2.3.9]
+def: "Catalysis of the reaction: an aromatic aldehyde + O2 + H2O = an aromatic carboxylate + H2O2 + H+." [RHEA:18569]
 synonym: "aryl-aldehyde:oxygen oxidoreductase activity" RELATED [EC:1.2.3.9]
 xref: EC:1.2.3.9
 xref: MetaCyc:ARYL-ALDEHYDE-OXIDASE-RXN
@@ -144953,7 +145339,7 @@
 id: GO:0018527
 name: cyclohexylamine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclohexylamine + O2 + H2O = cyclohexanone + NH3 + hydrogen peroxide." [PMID:18451, RHEA:18433]
+def: "Catalysis of the reaction: cyclohexylamine + O2 + H2O = cyclohexanone + H2O2 + NH4+." [PMID:18451, RHEA:18433]
 synonym: "cyclohexylamine:oxygen oxidoreductase (deaminating)" RELATED [EC:1.4.3.12]
 xref: EC:1.4.3.12
 xref: MetaCyc:CYCLOHEXYLAMINE-OXIDASE-RXN
@@ -145001,7 +145387,7 @@
 xref: MetaCyc:R-6-HYDROXYNICOTINE-OXIDASE-RXN
 xref: RHEA:10012
 xref: UM-BBD_reactionID:r0477
-is_a: GO:0019116 ! hydroxy-nicotine oxidase activity
+is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
 property_value: skos:exactMatch EC:1.5.3.6
 property_value: skos:exactMatch RHEA:10012
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
@@ -145021,7 +145407,7 @@
 xref: MetaCyc:S-6-HYDROXYNICOTINE-OXIDASE-RXN
 xref: RHEA:11880
 xref: UM-BBD_reactionID:r0478
-is_a: GO:0019116 ! hydroxy-nicotine oxidase activity
+is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
 property_value: skos:exactMatch EC:1.5.3.5
 property_value: skos:exactMatch RHEA:11880
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -145190,7 +145576,6 @@
 def: "OBSOLETE. Catalysis of the removal of one or more nitrite (NO2-) groups from nitroglycerin or a derivative." [GOC:curators]
 comment: This term was obsoleted because there is no evidence that this specific reaction exists.
 synonym: "NG reductase activity" EXACT []
-xref: Reactome:R-HSA-9620103 "ALDH2 transforms GTN to NO"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28261" xsd:anyURI
 is_obsolete: true
 
@@ -145207,7 +145592,7 @@
 id: GO:0018549
 name: methanethiol oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: methanethiol + O2 + H2O = formaldehyde + hydrogen sulfide + hydrogen peroxide." [EC:1.8.3.4]
+def: "Catalysis of the reaction: methanethiol + O2 + H2O = hydrogen sulfide + formaldehyde + H2O2 + H+." [RHEA:11812]
 synonym: "(MM)-oxidase activity" RELATED [EC:1.8.3.4]
 synonym: "methanethiol:oxygen oxidoreductase activity" RELATED [EC:1.8.3.4]
 synonym: "methyl mercaptan oxidase activity" RELATED [EC:1.8.3.4]
@@ -145965,7 +146350,7 @@
 id: GO:0018618
 name: anthranilate 1,2-dioxygenase (deaminating, decarboxylating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: anthranilate + NADPH + H+ + O2 = catechol + CO2 + NADP+ + NH3." [EC:1.14.12.1]
+def: "Catalysis of the reaction: anthranilate + NAD(P)H + O2 + 3 H+ = catechol + NH4+ + CO2 + NAD(P)+." [EC:1.14.12.1]
 synonym: "AntA" RELATED []
 synonym: "AntB" RELATED []
 synonym: "AntC" RELATED []
@@ -147278,30 +147663,30 @@
 
 [Term]
 id: GO:0018711
-name: benzoyl acetate-CoA thiolase activity
+name: obsolete benzoyl acetate-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: benzoyl acetyl-CoA + CoA = acetyl-CoA + benzoyl-CoA." [UM-BBD_reactionID:r0243]
-xref: MetaCyc:RXN-2006
-xref: UM-BBD_reactionID:r0243
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: benzoyl acetyl-CoA + CoA = acetyl-CoA + benzoyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018712
-name: 3-hydroxybutyryl-CoA thiolase activity
+name: obsolete 3-hydroxybutyryl-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-hydroxy-5-oxohexanoyl-CoA + CoASH = 3-hydroxybutyryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0010]
-xref: MetaCyc:R7-RXN
-xref: UM-BBD_reactionID:r0010
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: 3-hydroxy-5-oxohexanoyl-CoA + CoASH = 3-hydroxybutyryl-CoA + acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018713
-name: 3-ketopimelyl-CoA thiolase activity
+name: obsolete 3-ketopimelyl-CoA thiolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-ketopimeloyl-CoA + CoA = glutaryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0197]
-xref: MetaCyc:RXN-8032
-xref: UM-BBD_reactionID:r0197
-is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
+def: "OBSOLETE. Catalysis of the reaction: 3-ketopimeloyl-CoA + CoA = glutaryl-CoA + acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018715
@@ -147341,12 +147726,18 @@
 
 [Term]
 id: GO:0018719
-name: 6-aminohexanoate transaminase activity
+name: 6-aminohexanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-aminohexanoate + alpha-ketoglutarate = glutamate + 6-oxohexanoate." [UM-BBD_reactionID:r0449]
+def: "Catalysis of the reaction: 6-aminohexanoate + 2-oxoglutarate = 6-oxohexanoate + L-glutamate." [RHEA:58200]
+synonym: "6-aminohexanoate aminotransferase activity" EXACT []
+xref: EC:2.6.1.116
 xref: MetaCyc:R562-RXN
+xref: RHEA:58200
 xref: UM-BBD_reactionID:r0449
 is_a: GO:0008483 ! transaminase activity
+property_value: skos:exactMatch EC:2.6.1.116
+property_value: skos:exactMatch RHEA:58200
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0018720
@@ -147681,7 +148072,7 @@
 id: GO:0018750
 name: biuret amidohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: biuret + H2O = urea + CO2 + NH3." [EC:3.5.1.84]
+def: "Catalysis of the reaction: biuret + H2O = urea-1-carboxylate + NH4+." [RHEA:17525]
 xref: EC:3.5.1.84
 xref: RHEA:17525
 xref: UM-BBD_reactionID:r0846
@@ -147701,12 +148092,12 @@
 
 [Term]
 id: GO:0018752
-name: epsilon-caprolactam lactamase activity
+name: obsolete epsilon-caprolactam lactamase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: epsilon-caprolactam + H2O = H+ + 6-aminohexanoate." [UM-BBD_reactionID:r0448]
-xref: MetaCyc:R561-RXN
-xref: UM-BBD_reactionID:r0448
-is_a: GO:0016812 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides
+def: "OBSOLETE. Catalysis of the reaction: epsilon-caprolactam + H2O = H+ + 6-aminohexanoate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018753
@@ -147725,40 +148116,48 @@
 
 [Term]
 id: GO:0018754
-name: ammelide aminohydrolase activity
+name: obsolete ammelide aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ammelide + H2O = cyanuric acid + NH3." [PMID:1991731]
+def: "OBSOLETE. Catalysis of the reaction: ammelide + H2O = cyanuric acid + NH4+." [PMID:1991731]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8017
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018755
-name: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine aminohydrolase activity
+name: obsolete 2-chloro-4-hydroxy-6-amino-1,3,5-triazine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + OH- = 2,4-dihydroxy-6-amino-1,3,5-triazine + Cl-." [UM-BBD_reactionID:r1414]
-xref: UM-BBD_reactionID:r1414
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+def: "OBSOLETE. Catalysis of the reaction: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + OH- = 2,4-dihydroxy-6-amino-1,3,5-triazine + Cl-." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018756
 name: ammeline aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ammeline + H2O = ammelide + NH3." [PMID:1991731]
+def: "Catalysis of the reaction: ammeline + H2O = ammelide + NH4+. Also converts melamine to ammeline." [EC:3.5.4.45, PMID:1991731]
+xref: EC:3.5.4.45
 xref: MetaCyc:RXN-8016
 xref: RHEA:26201
 is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: skos:exactMatch EC:3.5.4.45
 property_value: skos:exactMatch RHEA:26201
+property_value: skos:relatedMatch RHEA:26197
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0018757
-name: deisopropylhydroxyatrazine aminohydrolase activity
+name: obsolete deisopropylhydroxyatrazine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: deisopropylhydroxyatrazine + H2O = NH3 + 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine." [UM-BBD_reactionID:r0121]
+def: "OBSOLETE. Catalysis of the reaction: deisopropylhydroxyatrazine + H2O = NH4+ + 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine." [UM-BBD_reactionID:r0121]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: KEGG_REACTION:R05574
 xref: UM-BBD_reactionID:r0121
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018758
@@ -147826,7 +148225,7 @@
 id: GO:0018762
 name: aliphatic nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: R-CN + H2O = R-COOH + NH3." [EC:3.5.5.7]
+def: "Catalysis of the reaction: an aliphatic nitrile + 2 H2O = a carboxylate + NH4+." [RHEA:46188]
 synonym: "aliphatic nitrile aminohydrolase activity" RELATED [EC:3.5.5.7]
 xref: EC:3.5.5.7
 xref: MetaCyc:3.5.5.7-RXN
@@ -147886,13 +148285,12 @@
 
 [Term]
 id: GO:0018766
-name: dihydrophloroglucinol hydrolase activity
+name: obsolete dihydrophloroglucinol hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dihydrophloroglucinol + OH- = 3-hydroxy-5-oxohexanoate." [UM-BBD_reactionID:r0008]
-xref: KEGG_REACTION:R07831
-xref: MetaCyc:R6-RXN
-xref: UM-BBD_reactionID:r0008
-is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
+def: "OBSOLETE. Catalysis of the reaction: dihydrophloroglucinol + OH- = 3-hydroxy-5-oxohexanoate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018767
@@ -147954,12 +148352,12 @@
 
 [Term]
 id: GO:0018772
-name: trioxoheptanoate hydrolase activity
+name: obsolete trioxoheptanoate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4,6-trioxoheptanoate + H2O = acetylpyruvate + acetate." [MetaCyc:R306-RXN, UM-BBD_reactionID:r0094]
-xref: MetaCyc:R306-RXN
-xref: UM-BBD_reactionID:r0094
-is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
+def: "OBSOLETE. Catalysis of the reaction: 2,4,6-trioxoheptanoate + H2O = acetylpyruvate + acetate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018773
@@ -148423,12 +148821,12 @@
 
 [Term]
 id: GO:0018811
-name: cyclohex-1-ene-1-carboxyl-CoA hydratase activity
+name: obsolete cyclohex-1-ene-1-carboxyl-CoA hydratase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclohex-1-ene-1-carboxyl-CoA + H2O = 2-hydroxycyclohexane-1-carboxyl-CoA." [MetaCyc:R266-RXN, UM-BBD_reactionID:r0191]
-xref: MetaCyc:R266-RXN
-xref: UM-BBD_reactionID:r0191
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: cyclohex-1-ene-1-carboxyl-CoA + H2O = 2-hydroxycyclohexane-1-carboxyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018812
@@ -148496,12 +148894,12 @@
 
 [Term]
 id: GO:0018815
-name: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon hydratase-aldolase activity
+name: obsolete 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon hydratase-aldolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon + H2O = 2-oxobut-3-enanoate + 2,5,6-trihydroxy-3-methylpyridine." [MetaCyc:RXN-645, UM-BBD_reactionID:r0051]
-xref: MetaCyc:RXN-645
-xref: UM-BBD_reactionID:r0051
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon + H2O = 2-oxobut-3-enanoate + 2,5,6-trihydroxy-3-methylpyridine." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018816
@@ -148657,7 +149055,7 @@
 id: GO:0018826
 name: methionine gamma-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine = methanethiol + NH3 + 2-oxobutanoate." [EC:4.4.1.11, RHEA:23800]
+def: "Catalysis of the reaction: L-methionine + H2O = methanethiol + 2-oxobutanoate + NH4+." [RHEA:23800]
 synonym: "L-methioninase activity" RELATED [EC:4.4.1.11]
 synonym: "L-methionine gamma-lyase activity" RELATED [EC:4.4.1.11]
 synonym: "L-methionine methanethiol-lyase (deaminating)" RELATED [EC:4.4.1.11]
@@ -148981,21 +149379,18 @@
 xref: RHEA:33419
 xref: UM-BBD_reactionID:r0429
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.38
 property_value: skos:exactMatch RHEA:33419
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0018856
-name: benzoyl acetate-CoA ligase activity
+name: obsolete benzoyl acetate-CoA ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-oxo-3-phenylpropionate + CoA + ATP = AMP + diphosphate + benzoyl acetyl-CoA." [UM-BBD_reactionID:r0242]
-xref: KEGG_REACTION:R05452
-xref: SABIO-RK:5058
-xref: UM-BBD_reactionID:r0242
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+def: "OBSOLETE. Catalysis of the reaction: 3-oxo-3-phenylpropionate + CoA + ATP = AMP + diphosphate + benzoyl acetyl-CoA." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0018857
@@ -149021,7 +149416,6 @@
 xref: RHEA:10132
 xref: UM-BBD_reactionID:r0189
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.25
 property_value: skos:exactMatch RHEA:10132
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -149042,7 +149436,6 @@
 xref: RHEA:23116
 xref: UM-BBD_reactionID:r0156
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.27
 property_value: skos:exactMatch RHEA:23116
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -149064,7 +149457,6 @@
 xref: RHEA:10828
 xref: UM-BBD_reactionID:r0341
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.32
 property_value: skos:exactMatch RHEA:10828
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -149081,7 +149473,6 @@
 xref: RHEA:23220
 xref: UM-BBD_reactionID:r0139
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.33
 property_value: skos:exactMatch RHEA:23220
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -150501,7 +150892,7 @@
 id: GO:0018997
 name: obsolete electron transfer carrier
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because the parent terms cover its function.
 synonym: "electron transfer carrier" EXACT []
 is_obsolete: true
@@ -150952,10 +151343,6 @@
 synonym: "viral host defence evasion" NARROW []
 synonym: "viral host defense evasion" NARROW []
 synonym: "viral regulation of antiviral response" EXACT []
-xref: Reactome:R-HSA-164938 "Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters"
-xref: Reactome:R-HSA-164944 "Nef and signal transduction"
-xref: Reactome:R-HSA-168305 "Inhibition of Interferon Synthesis"
-xref: Reactome:R-HSA-169131 "Inhibition of PKR"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23043" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25016" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29236" xsd:anyURI
@@ -151016,8 +151403,6 @@
 synonym: "regulation of cellular process in host by virus" EXACT []
 synonym: "regulation of host cellular process by virus" EXACT []
 synonym: "viral host cell process manipulation" EXACT []
-xref: Reactome:R-HSA-168276 "NS1 Mediated Effects on Host Pathways"
-xref: Reactome:R-HSA-9833110 "RSV-host interactions"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26621" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0044068
@@ -151626,6 +152011,7 @@
 def: "Catalysis of the transfer of a myristoyl (CH3-[CH2]12-CO-) group to an acceptor molecule." [GOC:ai]
 xref: Reactome:R-HSA-141367 "Myristoylation of tBID by NMT1"
 xref: Reactome:R-HSA-162914 "Myristoylation of Nef"
+xref: Reactome:R-HSA-9919009 "NMT1 interacts with E"
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 
 [Term]
@@ -151657,10 +152043,12 @@
 
 [Term]
 id: GO:0019112
-name: phenanthrol glycosyltransferase activity
+name: obsolete phenanthrol glycosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: phenanthrol + glucose = phenanthryl-beta-D-glucopyranoside + H2O." [GOC:ai]
-is_a: GO:0016758 ! hexosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: phenanthrol + glucose = phenanthryl-beta-D-glucopyranoside + H2O." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019113
@@ -151696,11 +152084,14 @@
 
 [Term]
 id: GO:0019116
-name: hydroxy-nicotine oxidase activity
+name: obsolete hydroxy-nicotine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyrid-3-yl)-4-(methylamino)butan-1-one + hydrogen peroxide." [GOC:jl, PMID:16095622]
-is_a: GO:0016647 ! oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyrid-3-yl)-4-(methylamino)butan-1-one + hydrogen peroxide." [GOC:jl, PMID:16095622]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+is_obsolete: true
+consider: GO:0018530
+consider: GO:0018531
 
 [Term]
 id: GO:0019117
@@ -151954,7 +152345,6 @@
 namespace: molecular_function
 alt_id: GO:0046420
 def: "Catalysis of the reaction: N6-dimethylallyladenine + acceptor + H2O = adenine + 3-methylbut-2-enal + reduced electron acceptor." [EC:1.5.99.12]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "6-N-dimethylallyladenine:acceptor oxidoreductase activity" RELATED [EC:1.5.99.12]
 synonym: "cytokinin oxidase activity" RELATED []
 synonym: "N6-dimethylallyladenine:(acceptor) oxidoreductase activity" RELATED [EC:1.5.99.12]
@@ -151966,8 +152356,8 @@
 property_value: skos:exactMatch EC:1.5.99.12
 property_value: skos:exactMatch MetaCyc:1.5.99.12-RXN
 property_value: skos:exactMatch RHEA:13625
-property_value: skos:relatedMatch MetaCyc:PWY-2841
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019140
@@ -152118,7 +152508,7 @@
 id: GO:0019148
 name: D-cysteine desulfhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-cysteine + H2O = sulfide + NH3 + pyruvate." [EC:4.4.1.15]
+def: "Catalysis of the reaction: D-cysteine + H2O = hydrogen sulfide + pyruvate + NH4+ + H+." [RHEA:11268]
 synonym: "D-cysteine lyase activity" RELATED [EC:4.4.1.15]
 synonym: "D-cysteine sulfide-lyase (deaminating)" RELATED [EC:4.4.1.15]
 synonym: "D-cysteine sulfide-lyase (deaminating; pyruvate-forming)" RELATED [EC:4.4.1.15]
@@ -152134,7 +152524,7 @@
 id: GO:0019149
 name: 3-chloro-D-alanine dehydrochlorinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-chloro-D-alanine + H2O = pyruvate + chloride + NH3." [EC:4.5.1.2]
+def: "Catalysis of the reaction: 3-chloro-D-alanine + H2O = chloride + pyruvate + NH4+ + H+." [RHEA:18873]
 synonym: "3-chloro-D-alanine chloride-lyase (deaminating)" RELATED [EC:4.5.1.2]
 synonym: "3-chloro-D-alanine chloride-lyase (deaminating; pyruvate-forming)" RELATED [EC:4.5.1.2]
 synonym: "beta-chloro-D-alanine dehydrochlorinase activity" RELATED [EC:4.5.1.2]
@@ -152283,6 +152673,7 @@
 synonym: "mannokinase (phosphorylating)" RELATED [EC:2.7.1.7]
 xref: EC:2.7.1.7
 xref: MetaCyc:MANNKIN-RXN
+xref: Reactome:R-HSA-8931653 "POMK 6-phosphorylates mannose in GalNAc-GlcNAc-Man-DAG1"
 xref: Reactome:R-HSA-9909159 "HK1 phosphorylates Mannose to Man6P"
 xref: RHEA:11028
 is_a: GO:0004396 ! hexokinase activity
@@ -152294,7 +152685,7 @@
 id: GO:0019159
 name: nicotinamide-nucleotide amidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide + H2O = beta-nicotinate D-ribonucleotide + NH3." [EC:3.5.1.42]
+def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide + H2O = nicotinate beta-D-ribonucleotide + NH4+." [RHEA:12400]
 synonym: "nicotinamide mononucleotide amidohydrolase activity" RELATED [EC:3.5.1.42]
 synonym: "nicotinamide mononucleotide deamidase activity" RELATED [EC:3.5.1.42]
 synonym: "nicotinamide-D-ribonucleotide amidohydrolase activity" RELATED [EC:3.5.1.42]
@@ -152330,12 +152721,13 @@
 
 [Term]
 id: GO:0019161
-name: diamine transaminase activity
+name: diamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde + L-glutamate." [EC:2.6.1.29]
+def: "Catalysis of the reaction: an alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde + L-glutamate." [RHEA:18217]
 synonym: "amine transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "amine-ketoacid transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "diamine aminotransferase activity" EXACT []
+synonym: "diamine transaminase activity" BROAD []
 synonym: "diamine-ketoglutaric transaminase activity" RELATED [EC:2.6.1.29]
 synonym: "diamine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.29]
 xref: EC:2.6.1.29
@@ -152345,14 +152737,16 @@
 property_value: skos:exactMatch EC:2.6.1.29
 property_value: skos:exactMatch RHEA:18217
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019162
-name: pyridoxamine-oxaloacetate transaminase activity
+name: pyridoxamine:oxaloacetate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: oxaloacetate + pyridoxamine = L-aspartate + pyridoxal." [EC:2.6.1.31, RHEA:10844]
+def: "Catalysis of the reaction: oxaloacetate + pyridoxamine = L-aspartate + pyridoxal." [RHEA:10844]
 synonym: "pyridoxamine--oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.31]
 synonym: "pyridoxamine-oxaloacetate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-oxaloacetate transaminase activity" EXACT []
 synonym: "pyridoxamine:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.31]
 xref: EC:2.6.1.31
 xref: KEGG_REACTION:R01713
@@ -152362,17 +152756,19 @@
 property_value: skos:exactMatch EC:2.6.1.31
 property_value: skos:exactMatch RHEA:10844
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019163
-name: pyridoxamine-phosphate transaminase activity
+name: pyridoxamine-phosphate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + 2-oxoglutarate = pyridoxal 5'-phosphate + D-glutamate." [EC:2.6.1.54]
+def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + 2-oxoglutarate = pyridoxal 5'-phosphate + D-glutamate." [RHEA:15877]
 synonym: "pyridoxamine 5'-phosphate transaminase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine 5'-phosphate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine phosphate aminotransferase activity" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine-5'-phosphate:2-oxoglutarate aminotransferase (D-glutamate-forming)" RELATED [EC:2.6.1.54]
 synonym: "pyridoxamine-phosphate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-phosphate transaminase activity" BROAD []
 xref: EC:2.6.1.54
 xref: MetaCyc:PYRDAMPTRANS-RXN
 xref: RHEA:15877
@@ -152380,6 +152776,7 @@
 property_value: skos:exactMatch EC:2.6.1.54
 property_value: skos:exactMatch RHEA:15877
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019164
@@ -152560,10 +152957,11 @@
 
 [Term]
 id: GO:0019179
-name: dTDP-4-amino-4,6-dideoxy-D-glucose transaminase activity
+name: dTDP-4-amino-4,6-dideoxy-D-glucose:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-D-glucose + L-glutamate." [EC:2.6.1.33]
+def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-D-glucose + L-glutamate." [RHEA:19085]
 synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose aminotransferase activity" EXACT []
+synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose transaminase activity" EXACT []
 synonym: "dTDP-4-amino-4,6-dideoxy-D-glucose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.33]
 synonym: "TDP-4-keto-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.33]
 synonym: "TDP-4-oxo-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.33]
@@ -152578,14 +152976,16 @@
 property_value: skos:exactMatch EC:2.6.1.33
 property_value: skos:exactMatch RHEA:19085
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019180
-name: dTDP-4-amino-4,6-dideoxygalactose transaminase activity
+name: dTDP-4-amino-4,6-dideoxygalactose:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + dTDP-4-amino-4,6-dideoxy-D-galactose = L-glutamate + dTDP-4-dehydro-6-deoxy-D-galactose." [EC:2.6.1.59, RHEA:10368]
+def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-alpha-D-galactose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + L-glutamate." [RHEA:10368]
 synonym: "dTDP-4,6-dideoxy-D-galactose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.59]
 synonym: "dTDP-4-amino-4,6-dideoxygalactose aminotransferase activity" RELATED [EC:2.6.1.59]
+synonym: "dTDP-4-amino-4,6-dideoxygalactose transaminase activity" EXACT []
 synonym: "dTDP-fucosamine aminotransferase activity" EXACT []
 synonym: "thymidine diphosphate 4-keto-6-deoxy-D-glucose transaminase activity" RELATED [EC:2.6.1.59]
 synonym: "thymidine diphosphoaminodideoxygalactose aminotransferase activity" RELATED [EC:2.6.1.59]
@@ -152597,6 +152997,7 @@
 property_value: skos:exactMatch EC:2.6.1.59
 property_value: skos:exactMatch RHEA:10368
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0019181
@@ -152721,7 +153122,6 @@
 name: carbohydrate kinase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a carbohydrate substrate molecule." [GOC:jl]
-xref: Reactome:R-HSA-8931653 "POMK 6-phosphorylates Mannose in GalNAc-GlcNAc-Man-DAG1"
 is_a: GO:0016301 ! kinase activity
 
 [Term]
@@ -152807,6 +153207,7 @@
 name: kinase inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces the activity of a kinase." [GOC:mah]
+xref: Reactome:R-HSA-170824 "glucokinase (GCK1) + glucokinase regulatory protein (GKRP) <=> GCK1:GKRP complex"
 is_a: GO:0004857 ! enzyme inhibitor activity
 is_a: GO:0019207 ! kinase regulator activity
 relationship: negatively_regulates GO:0016301 ! kinase activity
@@ -152816,6 +153217,11 @@
 name: phosphatase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a phosphatase." [GOC:ai]
+xref: Reactome:R-HSA-6809238 "MTMR7 binds MTMR9"
+xref: Reactome:R-HSA-6809254 "MTMR8 binds MTMR9"
+xref: Reactome:R-HSA-6809309 "MTMR6 binds MTMR9"
+xref: Reactome:R-HSA-6809680 "MTMR12 binds MTM1"
+xref: Reactome:R-HSA-6809793 "MTMR2 binds SBF2"
 is_a: GO:0008047 ! enzyme activator activity
 is_a: GO:0019208 ! phosphatase regulator activity
 relationship: positively_regulates GO:0016791 ! phosphatase activity
@@ -152863,7 +153269,7 @@
 id: GO:0019216
 name: regulation of lipid metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:curators]
 synonym: "regulation of lipid metabolism" EXACT []
 xref: Reactome:R-HSA-2426168 "Activation of gene expression by SREBF (SREBP)"
 xref: Reactome:R-HSA-400206 "Regulation of lipid metabolism by PPARalpha"
@@ -152877,7 +153283,7 @@
 name: regulation of fatty acid metabolic process
 namespace: biological_process
 alt_id: GO:0006632
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:curators]
 synonym: "regulation of fatty acid metabolism" EXACT []
 is_a: GO:0010565 ! regulation of ketone metabolic process
 is_a: GO:0019216 ! regulation of lipid metabolic process
@@ -152890,7 +153296,7 @@
 id: GO:0019218
 name: regulation of steroid metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:curators]
 synonym: "regulation of steroid metabolism" EXACT []
 is_a: GO:0019216 ! regulation of lipid metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -152901,19 +153307,21 @@
 id: GO:0019219
 name: regulation of nucleobase-containing compound metabolic process
 namespace: biological_process
-def: "Any cellular process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:go_curators]
+def: "Any cellular process that modulates the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:curators]
+subset: gocheck_do_not_annotate
 synonym: "regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" RELATED [GOC:dph, GOC:tb]
 synonym: "regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolism" EXACT []
 is_a: GO:0080090 ! regulation of primary metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006139 ! nucleobase-containing compound metabolic process
 relationship: regulates GO:0006139 ! nucleobase-containing compound metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0019220
 name: obsolete regulation of phosphate metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "regulation of phosphate metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30570" xsd:anyURI
@@ -152939,7 +153347,7 @@
 name: regulation of metabolic process
 namespace: biological_process
 alt_id: GO:0044246
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_metagenomics
 synonym: "regulation of metabolism" EXACT []
@@ -153068,10 +153476,9 @@
 id: GO:0019239
 name: deaminase activity
 namespace: molecular_function
-def: "Catalysis of the removal of an amino group from a substrate, producing a substituted or nonsubstituted ammonia (NH3/NH2R)." [GOC:jl]
+def: "Catalysis of the removal of an amino group from a substrate, producing a substituted or nonsubstituted ammonia (NH4+/NH2R)." [GOC:jl]
 subset: goslim_pir
 xref: Reactome:R-HSA-74241 "ADA catalyzes the deamination of (deoxy)adenosine"
-xref: Reactome:R-HSA-9014641 "HRSP12 deaminates 2AA to 2OBUTA"
 xref: Reactome:R-HSA-9734745 "Defective ADA does not deaminate (deoxy)adenosine"
 xref: Reactome:R-HSA-9754964 "ADA deamidates RBV"
 is_a: GO:0016810 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
@@ -153094,7 +153501,7 @@
 id: GO:0019241
 name: L-citrulline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-citrulline, N5-carbamoyl-L-ornithine, an alpha amino acid not found in proteins." [ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of L-citrulline." [GOC:curators]
 synonym: "citrulline breakdown" EXACT []
 synonym: "citrulline catabolism" EXACT []
 synonym: "citrulline degradation" EXACT []
@@ -153103,6 +153510,7 @@
 is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
 is_a: GO:1901606 ! alpha-amino acid catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019242
@@ -153138,7 +153546,7 @@
 id: GO:0019244
 name: lactate biosynthetic process from pyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lactate from other compounds, including pyruvate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of lactate from other compounds, including pyruvate." [GOC:curators]
 synonym: "lactate anabolism from pyruvate" EXACT []
 synonym: "lactate formation from pyruvate" EXACT []
 synonym: "lactate synthesis from pyruvate" EXACT []
@@ -153151,7 +153559,7 @@
 id: GO:0019245
 name: D(-)-lactate biosynthetic process from pyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of D(-)-lactate from other compounds, including pyruvate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of D(-)-lactate from other compounds, including pyruvate." [GOC:curators]
 synonym: "D(-)-lactate anabolism from pyruvate" EXACT []
 synonym: "D(-)-lactate formation from pyruvate" EXACT []
 synonym: "D(-)-lactate synthesis from pyruvate" EXACT []
@@ -153162,7 +153570,7 @@
 id: GO:0019246
 name: L(+)-lactate biosynthetic process from pyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L(+)-lactate from other compounds, including pyruvate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L(+)-lactate from other compounds, including pyruvate." [GOC:curators]
 synonym: "L(+)-lactate anabolism from pyruvate" EXACT []
 synonym: "L(+)-lactate formation from pyruvate" EXACT []
 synonym: "L(+)-lactate synthesis from pyruvate" EXACT []
@@ -153195,7 +153603,7 @@
 id: GO:0019249
 name: lactate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lactate, the anion of lactic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of lactate, the anion of lactic acid." [GOC:curators]
 synonym: "lactate anabolism" EXACT []
 synonym: "lactate biosynthesis" EXACT []
 synonym: "lactate formation" EXACT []
@@ -153207,7 +153615,7 @@
 id: GO:0019250
 name: aerobic cobalamin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of cobalamin (vitamin B12) in the presence of oxygen." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of cobalamin (vitamin B12) in the presence of oxygen." [GOC:curators]
 synonym: "aerobic cobalamin anabolism" EXACT []
 synonym: "aerobic cobalamin biosynthesis" EXACT []
 synonym: "aerobic cobalamin formation" EXACT []
@@ -153225,7 +153633,7 @@
 id: GO:0019251
 name: anaerobic cobalamin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of cobalamin (vitamin B12) in the absence of oxygen." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of cobalamin (vitamin B12) in the absence of oxygen." [GOC:curators]
 synonym: "anaerobic cobalamin anabolism" EXACT []
 synonym: "anaerobic cobalamin biosynthesis" EXACT []
 synonym: "anaerobic cobalamin formation" EXACT []
@@ -153268,7 +153676,7 @@
 id: GO:0019254
 name: carnitine metabolic process, CoA-linked
 namespace: biological_process
-def: "The chemical reactions and pathways involving carnitine, where metabolism is linked to CoA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving carnitine, where metabolism is linked to CoA." [GOC:curators]
 synonym: "carnitine metabolism, CoA-linked" EXACT []
 xref: MetaCyc:CARNMET-PWY
 is_a: GO:0009437 ! carnitine metabolic process
@@ -153328,7 +153736,7 @@
 id: GO:0019260
 name: 1,2-dichloroethane catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of 1,2-dichloroethane, a major commodity chemical used, for example, in the manufacture of vinyl chloride." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of 1,2-dichloroethane, a major commodity chemical used, for example, in the manufacture of vinyl chloride." [GOC:curators]
 synonym: "1,2-dichloroethane breakdown" EXACT []
 synonym: "1,2-dichloroethane catabolism" EXACT []
 synonym: "1,2-dichloroethane degradation" EXACT []
@@ -153361,6 +153769,7 @@
 is_a: GO:0006054 ! N-acetylneuraminate metabolic process
 is_a: GO:0046348 ! amino sugar catabolic process
 is_a: GO:0046395 ! carboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
 
 [Term]
 id: GO:0019263
@@ -153382,33 +153791,28 @@
 synonym: "glycine anabolism from serine" EXACT []
 synonym: "glycine formation from serine" EXACT []
 synonym: "glycine synthesis from serine" EXACT []
-xref: MetaCyc:GLYSYN-PWY
 is_a: GO:0006545 ! glycine biosynthetic process
 is_a: GO:0006563 ! L-serine metabolic process
-property_value: skos:exactMatch MetaCyc:GLYSYN-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30758" xsd:anyURI
 
 [Term]
 id: GO:0019265
 name: glycine biosynthetic process, by transamination of glyoxylate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of glycine by the transamination of glyoxylate." [GOC:go_curators, MetaCyc:GLYSYN-ALA-PWY]
+def: "The chemical reactions and pathways resulting in the formation of glycine by the transamination of glyoxylate." [GOC:curators, MetaCyc:GLYSYN-ALA-PWY]
 synonym: "glycine anabolism, by transamination of glyoxylate" EXACT []
 synonym: "glycine formation, by transamination of glyoxylate" EXACT []
 synonym: "glycine synthesis, by transamination of glyoxylate" EXACT []
-xref: MetaCyc:GLYSYN-ALA-PWY
 is_a: GO:0006545 ! glycine biosynthetic process
-property_value: skos:exactMatch MetaCyc:GLYSYN-ALA-PWY
 
 [Term]
 id: GO:0019266
 name: L-asparagine biosynthetic process from oxaloacetate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of asparagine from other compounds, including oxaloacetate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of asparagine from other compounds, including oxaloacetate." [GOC:curators]
 synonym: "asparagine anabolism from oxaloacetate" EXACT []
 synonym: "asparagine formation from oxaloacetate" EXACT []
 synonym: "asparagine synthesis from oxaloacetate" EXACT []
-xref: MetaCyc:ASPARAGINE-BIOSYNTHESIS
 is_a: GO:0006107 ! oxaloacetate metabolic process
 is_a: GO:0043649 ! dicarboxylic acid catabolic process
 is_a: GO:0070981 ! L-asparagine biosynthetic process
@@ -153416,22 +153820,22 @@
 
 [Term]
 id: GO:0019267
-name: L-asparagine biosynthetic process from L-cysteine
+name: obsolete L-asparagine biosynthetic process from L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-asparagine from other compounds, including L-cysteine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-asparagine from other compounds, including L-cysteine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "asparagine anabolism from cysteine" EXACT []
 synonym: "asparagine formation from cysteine" EXACT []
 synonym: "asparagine synthesis from cysteine" EXACT []
-xref: MetaCyc:ASPSYNII-PWY
-is_a: GO:0046439 ! L-cysteine metabolic process
-is_a: GO:0070981 ! L-asparagine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0070981
 
 [Term]
 id: GO:0019268
 name: obsolete glutamate biosynthetic process, using glutamate dehydrogenase (NAD(P)+)
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, catalyzed by the enzyme glutamate dehydrogenase (NADP+)." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, catalyzed by the enzyme glutamate dehydrogenase (NADP+)." [GOC:curators]
 comment: This term was made obsolete because it represents a molecular function.
 synonym: "glutamate anabolism, using glutamate dehydrogenase (NAD(P)+)" EXACT []
 synonym: "glutamate biosynthetic process, using glutamate dehydrogenase (NAD(P)+)" EXACT []
@@ -153444,7 +153848,7 @@
 id: GO:0019269
 name: obsolete glutamate biosynthetic process, using glutamate synthase (NADPH)
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, catalyzed by the enzyme glutamate synthase (NADPH)." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of glutamate, catalyzed by the enzyme glutamate synthase (NADPH)." [GOC:curators]
 comment: This term was made obsolete because it refers to a molecular function.
 synonym: "glutamate anabolism, using glutamate synthase (NADPH)" EXACT []
 synonym: "glutamate biosynthetic process, using glutamate synthase (NADPH)" EXACT []
@@ -153477,21 +153881,19 @@
 id: GO:0019272
 name: L-alanine biosynthetic process from pyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of alanine from other compounds, including pyruvate." [GOC:go_curators, MetaCyc:ALANINE-SYN2-PWY]
+def: "The chemical reactions and pathways resulting in the formation of alanine from other compounds, including pyruvate." [GOC:curators]
 synonym: "L-alanine anabolism from pyruvate" EXACT []
 synonym: "L-alanine formation from pyruvate" EXACT []
 synonym: "L-alanine synthesis from pyruvate" EXACT []
-xref: MetaCyc:ALANINE-SYN2-PWY
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0042852 ! L-alanine biosynthetic process
-property_value: skos:exactMatch MetaCyc:ALANINE-SYN2-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 
 [Term]
 id: GO:0019273
 name: obsolete L-alanine biosynthetic process via ornithine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-alanine, via the intermediate ornithine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-alanine, via the intermediate ornithine." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this process exists in vivo.
 synonym: "L-alanine anabolism via ornithine" EXACT []
 synonym: "L-alanine formation via ornithine" EXACT []
@@ -153517,15 +153919,18 @@
 synonym: "UDP-N-acetylgalactosamine biosynthesis" EXACT []
 synonym: "UDP-N-acetylgalactosamine formation" EXACT []
 synonym: "UDP-N-acetylgalactosamine synthesis" EXACT []
+xref: MetaCyc:PWY-5512
 xref: MetaCyc:PWY-5514
 xref: MetaCyc:PWY-8013
 is_a: GO:0009226 ! nucleotide-sugar biosynthetic process
 is_a: GO:0019276 ! UDP-N-acetylgalactosamine metabolic process
 is_a: GO:0046349 ! amino sugar biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5512
 property_value: skos:narrowMatch MetaCyc:PWY-5514
 property_value: skos:narrowMatch MetaCyc:PWY-8013
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019278
@@ -153541,69 +153946,78 @@
 
 [Term]
 id: GO:0019279
-name: L-methionine biosynthetic process from L-homoserine via cystathionine
+name: obsolete L-methionine biosynthetic process from L-homoserine via cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediate cystathionine." [MetaCyc:HOMOSER-METSYN-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediate cystathionine." [MetaCyc:HOMOSER-METSYN-PWY]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine anabolism from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine formation from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from L-homoserine via cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from L-homoserine via cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:HOMOSER-METSYN-PWY
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0071265 ! L-methionine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019280
-name: L-methionine biosynthetic process from L-homoserine via O-acetyl-L-homoserine
+name: obsolete L-methionine biosynthetic process from L-homoserine via O-acetyl-L-homoserine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of methionine from L-homoserine, via the intermediate O-acetyl-L-homoserine using sulfur from hydrogen sulfide assimilation." [MetaCyc:HSERMETANA-PWY]
-xref: MetaCyc:HSERMETANA-PWY
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0071265 ! L-methionine biosynthetic process
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of methionine from L-homoserine, via the intermediate O-acetyl-L-homoserine using sulfur from hydrogen sulfide assimilation." [MetaCyc:HSERMETANA-PWY]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30025" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019281
-name: L-methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine
+name: obsolete L-methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine from other compounds, including homoserine, via the intermediates O-succinyl-L-homoserine and cystathionine." [MetaCyc:MET-SAM-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from other compounds, including homoserine, via the intermediates O-succinyl-L-homoserine and cystathionine." [GOC:curators]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine anabolism from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine formation from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:MET-SAM-PWY
-is_a: GO:0019279 ! L-methionine biosynthetic process from L-homoserine via cystathionine
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019283
-name: L-methionine biosynthetic process from L-homoserine via O-phospho-L-homoserine and cystathionine
+name: obsolete L-methionine biosynthetic process from L-homoserine via O-phospho-L-homoserine and cystathionine
 namespace: biological_process
-def: "he chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediates O-phospho-L-homoserine and cystathionine." [GOC:go_curators, MetaCyc:PWY-702]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-methionine from L-homoserine, via the intermediates O-phospho-L-homoserine and cystathionine." [GOC:curators, MetaCyc:PWY-702]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "L-methionine formation from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
-xref: MetaCyc:PWY-702
-is_a: GO:0006575 ! modified amino acid metabolic process
-is_a: GO:0006796 ! phosphate-containing compound metabolic process
-is_a: GO:0009092 ! L-homoserine metabolic process
-is_a: GO:0019637 ! organophosphate metabolic process
-is_a: GO:0071266 ! 'de novo' L-methionine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29576" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0019284
 name: L-methionine salvage from S-adenosylmethionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine from S-adenosylmethionine." [GOC:go_curators, GOC:vw, PMID:31950558]
+def: "The chemical reactions and pathways resulting in the formation of L-methionine from S-adenosylmethionine." [GOC:curators, GOC:vw, PMID:31950558]
 synonym: "L-methionine formation from S-adenosylmethionine" EXACT [GOC:mah]
 synonym: "L-methionine synthesis from S-adenosylmethionine" EXACT [GOC:mah]
+xref: MetaCyc:PWY-7270
+xref: MetaCyc:PWY-7527
+xref: MetaCyc:PWY-7528
 is_a: GO:0071267 ! L-methionine salvage
 relationship: part_of GO:0033353 ! S-adenosylmethionine cycle
+property_value: skos:narrowMatch MetaCyc:PWY-7270
+property_value: skos:narrowMatch MetaCyc:PWY-7527
+property_value: skos:narrowMatch MetaCyc:PWY-7528
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29900" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0019285
@@ -153630,7 +154044,7 @@
 id: GO:0019286
 name: glycine betaine biosynthetic process from glycine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of glycine betaine from other compounds, including glycine." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of glycine betaine from other compounds, including glycine." [GOC:curators]
 synonym: "glycine betaine anabolism from glycine" EXACT []
 synonym: "glycine betaine formation from glycine" EXACT []
 synonym: "glycine betaine synthesis from glycine" EXACT []
@@ -153644,7 +154058,7 @@
 id: GO:0019287
 name: isopentenyl diphosphate biosynthetic process, mevalonate pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of isopentenyl diphosphate, via the intermediate mevalonate. This pathway converts acetate, in the form of acetyl-CoA, to isopentenyl diphosphate (IPP), the fundamental unit in isoprenoid biosynthesis, through a series of mevalonate intermediates." [GOC:go_curators, MetaCyc:PWY-922]
+def: "The chemical reactions and pathways resulting in the formation of isopentenyl diphosphate, via the intermediate mevalonate. This pathway converts acetate, in the form of acetyl-CoA, to isopentenyl diphosphate (IPP), the fundamental unit in isoprenoid biosynthesis, through a series of mevalonate intermediates." [GOC:curators, MetaCyc:PWY-922]
 synonym: "Ac-MVA pathway" EXACT [PMID:14517367]
 synonym: "acetate-mevalonate pathway" EXACT [PMID:14517367]
 synonym: "isopentenyl diphosphate anabolism, mevalonate pathway" EXACT []
@@ -153660,7 +154074,7 @@
 id: GO:0019288
 name: isopentenyl diphosphate biosynthetic process, methylerythritol 4-phosphate pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of isopentenyl diphosphate by the mevalonate-independent pathway. Isopentenyl diphosphate (IPP) is the fundamental unit in isoprenoid biosynthesis and is biosynthesized from pyruvate and glyceraldehyde 3-phosphate via intermediates, including 1-deoxy-D-xylulose 5-phosphate." [GOC:go_curators, MetaCyc:NONMEVIPP-PWY, PMID:18948055]
+def: "The chemical reactions and pathways resulting in the formation of isopentenyl diphosphate by the mevalonate-independent pathway. Isopentenyl diphosphate (IPP) is the fundamental unit in isoprenoid biosynthesis and is biosynthesized from pyruvate and glyceraldehyde 3-phosphate via intermediates, including 1-deoxy-D-xylulose 5-phosphate." [GOC:curators, MetaCyc:NONMEVIPP-PWY, PMID:18948055]
 synonym: "isopentenyl diphosphate anabolism, mevalonate-independent pathway" EXACT []
 synonym: "isopentenyl diphosphate biosynthesis, mevalonate-independent" EXACT []
 synonym: "isopentenyl diphosphate biosynthesis, non-mevalonate pathway" EXACT []
@@ -153719,13 +154133,12 @@
 id: GO:0019292
 name: L-tyrosine biosynthetic process from chorismate via 4-hydroxyphenylpyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-tyrosine from other compounds, including chorismate, via the intermediate 4-hydroxyphenylpyruvate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-tyrosine from other compounds, including chorismate, via the intermediate 4-hydroxyphenylpyruvate." [GOC:curators]
 synonym: "L-tyrosine anabolism from chorismate via 4-hydroxyphenylpyruvate" EXACT []
 synonym: "L-tyrosine biosynthetic process from chorismate via p-hydroxyphenylpyruvate" EXACT []
 synonym: "L-tyrosine formation from chorismate via 4-hydroxyphenylpyruvate" EXACT []
 synonym: "L-tyrosine synthesis from chorismate via 4-hydroxyphenylpyruvate" EXACT []
 synonym: "tyrosine biosynthetic process from chorismate via 4-hydroxyphenylpyruvate" BROAD []
-xref: MetaCyc:TYRSYN
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 is_a: GO:0046417 ! chorismate metabolic process
 
@@ -153739,7 +154152,6 @@
 synonym: "tyrosine anabolism, by oxidation of phenylalanine" EXACT []
 synonym: "tyrosine formation, by oxidation of phenylalanine" EXACT []
 synonym: "tyrosine synthesis, by oxidation of phenylalanine" EXACT []
-xref: MetaCyc:PWY-6134
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20583" xsd:anyURI
 
@@ -153777,7 +154189,7 @@
 id: GO:0019296
 name: obsolete coenzyme M metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving coenzyme M (2-thioethansulfonate), a coenzyme involved in the utilization of methane by methanogenic prokaryotes." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving coenzyme M (2-thioethansulfonate), a coenzyme involved in the utilization of methane by methanogenic prokaryotes." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "coenzyme M metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -153787,7 +154199,7 @@
 id: GO:0019297
 name: obsolete coenzyme B metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving coenzyme B (7-mercaptoheptanoylthreonine phosphate), a coenzyme involved in the utilization of methane by methanogenic prokaryotes." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving coenzyme B (7-mercaptoheptanoylthreonine phosphate), a coenzyme involved in the utilization of methane by methanogenic prokaryotes." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "coenzyme B metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
@@ -153937,7 +154349,7 @@
 id: GO:0019310
 name: inositol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of inositol, 1,2,3,4,5,6-cyclohexanehexol, a growth factor for animals and microorganisms." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of inositol, 1,2,3,4,5,6-cyclohexanehexol, a growth factor for animals and microorganisms." [GOC:curators]
 synonym: "inositol breakdown" EXACT []
 synonym: "inositol catabolism" EXACT []
 synonym: "inositol degradation" EXACT []
@@ -154253,7 +154665,7 @@
 id: GO:0019338
 name: pentachlorophenol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pentachlorophenol, a chlorinated insecticide and fungicide used primarily to protect timber from fungal rot and wood boring insects. Pentachlorophenol is significantly toxic to mammals, plants, and many microorganisms." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of pentachlorophenol, a chlorinated insecticide and fungicide used primarily to protect timber from fungal rot and wood boring insects. Pentachlorophenol is significantly toxic to mammals, plants, and many microorganisms." [GOC:curators]
 synonym: "pentachlorophenol breakdown" EXACT []
 synonym: "pentachlorophenol catabolism" EXACT []
 synonym: "pentachlorophenol degradation" EXACT []
@@ -154317,47 +154729,64 @@
 
 [Term]
 id: GO:0019343
-name: L-cysteine biosynthetic process via L-cystathionine
+name: obsolete L-cysteine biosynthetic process via L-cystathionine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate L-cystathionine." [GOC:go_curators, MetaCyc:HOMOCYSDEGR-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate L-cystathionine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism via cystathionine" EXACT []
 synonym: "cysteine formation via cystathionine" EXACT []
 synonym: "cysteine synthesis via cystathionine" EXACT []
-xref: MetaCyc:HOMOCYSDEGR-PWY
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:HOMOCYSDEGR-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0019344
 name: L-cysteine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-cysteine, 2-amino-3-mercaptopropanoic acid." [GOC:curators]
 synonym: "cysteine anabolism" EXACT []
 synonym: "cysteine biosynthesis" EXACT []
 synonym: "cysteine formation" EXACT []
 synonym: "cysteine synthesis" EXACT []
+xref: MetaCyc:CYSTSYN-PWY
+xref: MetaCyc:HOMOCYSDEGR-PWY
+xref: MetaCyc:PWY-6308
+xref: MetaCyc:PWY-7289
+xref: MetaCyc:PWY-7870
+xref: MetaCyc:PWY-8009
+xref: MetaCyc:PWY-8010
+xref: MetaCyc:PWY-I9
 is_a: GO:0000097 ! sulfur amino acid biosynthetic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0046439 ! L-cysteine metabolic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:CYSTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:HOMOCYSDEGR-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-6308
+property_value: skos:narrowMatch MetaCyc:PWY-7289
+property_value: skos:narrowMatch MetaCyc:PWY-7870
+property_value: skos:narrowMatch MetaCyc:PWY-8009
+property_value: skos:narrowMatch MetaCyc:PWY-8010
+property_value: skos:narrowMatch MetaCyc:PWY-I9
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0019345
-name: L-cysteine biosynthetic process via S-sulfo-L-cysteine
+name: obsolete L-cysteine biosynthetic process via S-sulfo-L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate S-sulfo-L-cysteine." [MetaCyc:PWY-7289]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of L-cysteine, via the intermediate S-sulfo-L-cysteine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "cysteine anabolism via S-sulfo-L-cysteine" EXACT []
 synonym: "cysteine biosynthesis via S-sulpho-L-cysteine" EXACT []
 synonym: "cysteine biosynthetic process via S-sulpho-L-cysteine" EXACT []
 synonym: "cysteine formation via S-sulfo-L-cysteine" EXACT []
 synonym: "cysteine synthesis via S-sulfo-L-cysteine" EXACT []
-xref: MetaCyc:PWY-7289
-is_a: GO:0019344 ! L-cysteine biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY-7289
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019344
 
 [Term]
 id: GO:0019346
@@ -154444,7 +154873,7 @@
 id: GO:0019352
 name: protoporphyrinogen IX biosynthetic process from glycine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of protoporphyrinogen IX from other compounds, including glycine." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of protoporphyrinogen IX from other compounds, including glycine." [GOC:curators]
 synonym: "protoporphyrinogen IX anabolism from glycine" EXACT []
 synonym: "protoporphyrinogen IX formation from glycine" EXACT []
 synonym: "protoporphyrinogen IX synthesis from glycine" EXACT []
@@ -154482,7 +154911,7 @@
 id: GO:0019356
 name: obsolete nicotinate nucleotide biosynthetic process from tryptophan
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of nicotinate nucleotide from other compounds, including tryptophan." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of nicotinate nucleotide from other compounds, including tryptophan." [GOC:curators]
 comment: The reason for obsoletion is that this term is essentially equivalent to 'de novo' NAD+ biosynthetic process from L-tryptophan.
 synonym: "nicotinate nucleotide anabolism from tryptophan" EXACT []
 synonym: "nicotinate nucleotide formation from tryptophan" EXACT []
@@ -154495,7 +154924,7 @@
 id: GO:0019357
 name: nicotinate nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotides, any nucleotide that contains combined nicotinate (pyridine 3-carboxylic acid)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotides, any nucleotide that contains combined nicotinate (pyridine 3-carboxylic acid)." [GOC:curators]
 synonym: "nicotinate nucleotide anabolism" EXACT []
 synonym: "nicotinate nucleotide biosynthesis" EXACT []
 synonym: "nicotinate nucleotide formation" EXACT []
@@ -154506,7 +154935,7 @@
 id: GO:0019358
 name: nicotinate nucleotide salvage
 namespace: biological_process
-def: "The generation of nicotinate nucleotide without de novo synthesis." [GOC:go_curators]
+def: "The generation of nicotinate nucleotide without de novo synthesis." [GOC:curators]
 synonym: "nicotinate nucleotide biosynthesis, salvage pathway" EXACT []
 synonym: "nicotinate nucleotide biosynthetic process, salvage pathway" EXACT []
 xref: MetaCyc:PWY3O-224
@@ -154519,7 +154948,7 @@
 id: GO:0019359
 name: nicotinamide nucleotide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotides, any nucleotide that contains combined nicotinamide." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotides, any nucleotide that contains combined nicotinamide." [GOC:curators]
 synonym: "nicotinamide nucleotide anabolism" EXACT []
 synonym: "nicotinamide nucleotide biosynthesis" EXACT []
 synonym: "nicotinamide nucleotide formation" EXACT []
@@ -154531,7 +154960,7 @@
 id: GO:0019360
 name: nicotinamide nucleotide biosynthetic process from niacinamide
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotide from other compounds, including niacinamide." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of nicotinamide nucleotide from other compounds, including niacinamide." [GOC:curators]
 synonym: "nicotinamide nucleotide anabolism from niacinamide" EXACT []
 synonym: "nicotinamide nucleotide formation from niacinamide" EXACT []
 synonym: "nicotinamide nucleotide synthesis from niacinamide" EXACT []
@@ -154628,7 +155057,7 @@
 id: GO:0019370
 name: leukotriene biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of leukotriene, a pharmacologically active substance derived from a polyunsaturated fatty acid, such as arachidonic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of leukotriene, a pharmacologically active substance derived from a polyunsaturated fatty acid, such as arachidonic acid." [GOC:curators]
 synonym: "leukotriene anabolism" EXACT []
 synonym: "leukotriene biosynthesis" EXACT []
 synonym: "leukotriene formation" EXACT []
@@ -154700,7 +155129,7 @@
 id: GO:0019377
 name: glycolipid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glycolipid, a class of 1,2-di-O-acylglycerols joined at oxygen 3 by a glycosidic linkage to a carbohydrate part (usually a mono-, di- or tri-saccharide)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of glycolipid, a class of 1,2-di-O-acylglycerols joined at oxygen 3 by a glycosidic linkage to a carbohydrate part (usually a mono-, di- or tri-saccharide)." [GOC:curators]
 synonym: "glycolipid breakdown" EXACT []
 synonym: "glycolipid catabolism" EXACT []
 synonym: "glycolipid degradation" EXACT []
@@ -154755,7 +155184,7 @@
 id: GO:0019382
 name: carbon tetrachloride catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of carbon tetrachloride, a toxic, carcinogenic compound which is used as a general solvent in industrial degreasing operations. It is also used as grain fumigant and a chemical intermediate in the production of refrigerants." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of carbon tetrachloride, a toxic, carcinogenic compound which is used as a general solvent in industrial degreasing operations. It is also used as grain fumigant and a chemical intermediate in the production of refrigerants." [GOC:curators]
 synonym: "carbon tetrachloride breakdown" EXACT []
 synonym: "carbon tetrachloride catabolism" EXACT []
 synonym: "carbon tetrachloride degradation" EXACT []
@@ -155078,7 +155507,7 @@
 id: GO:0019409
 name: aerobic respiration, using ammonia as electron donor
 namespace: biological_process
-def: "The metabolic process in which ammonia (NH3) is oxidized to nitrite (NO2) in the presence of oxygen; enzymatic reactions convert ammonia to hydrazine, and hydrazine to nitrite." [MetaCyc:AMMOXID-PWY]
+def: "The metabolic process in which ammonia (NH4+) is oxidized to nitrite (NO2) in the presence of oxygen; enzymatic reactions convert ammonia to hydrazine, and hydrazine to nitrite." [MetaCyc:AMMOXID-PWY]
 synonym: "aerobic ammonia oxidation to nitrite via hydrazine" EXACT [GOC:mah]
 synonym: "nitrification" RELATED []
 xref: MetaCyc:AMMOXID-PWY
@@ -155117,7 +155546,7 @@
 id: GO:0019413
 name: acetate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of acetate, the anion of acetic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of acetate, the anion of acetic acid." [GOC:curators]
 synonym: "acetate anabolism" EXACT []
 synonym: "acetate biosynthesis" EXACT []
 synonym: "acetate formation" EXACT []
@@ -155138,7 +155567,7 @@
 id: GO:0019415
 name: acetate biosynthetic process from carbon monoxide
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of acetate from other compounds, including carbon monoxide." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of acetate from other compounds, including carbon monoxide." [GOC:curators]
 synonym: "acetate anabolism from carbon monoxide" EXACT []
 synonym: "acetate formation from carbon monoxide" EXACT []
 synonym: "acetate synthesis from carbon monoxide" EXACT []
@@ -155261,7 +155690,7 @@
 id: GO:0019428
 name: allantoin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of allantoin, (2,5-dioxo-4-imidazolidinyl)urea." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of allantoin, (2,5-dioxo-4-imidazolidinyl)urea." [GOC:curators]
 synonym: "allantoin anabolism" EXACT []
 synonym: "allantoin biosynthesis" EXACT []
 synonym: "allantoin formation" EXACT []
@@ -155303,7 +155732,7 @@
 id: GO:0019431
 name: acetyl-CoA biosynthetic process from ethanol
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of acetyl-CoA from ethanol via acetaldehyde." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of acetyl-CoA from ethanol via acetaldehyde." [GOC:curators]
 synonym: "acetyl-CoA anabolism from ethanol" EXACT []
 synonym: "acetyl-CoA formation from ethanol" EXACT []
 synonym: "acetyl-CoA synthesis from ethanol" EXACT []
@@ -155409,7 +155838,7 @@
 id: GO:0019440
 name: obsolete L-tryptophan catabolic process to indole-3-acetate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including indole-3-acetate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including indole-3-acetate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "tryptophan breakdown to indole-3-acetate" EXACT []
 synonym: "tryptophan catabolic process to IAA" EXACT []
@@ -155426,7 +155855,7 @@
 id: GO:0019441
 name: obsolete L-tryptophan catabolic process to L-kynurenine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including L-kynurenine." [GOC:go_curators, PMID:31704347]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including L-kynurenine." [GOC:curators, PMID:31704347]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "tryptophan breakdown to kynurenine" EXACT []
 synonym: "tryptophan catabolic process to kynurenine" EXACT []
@@ -155440,7 +155869,7 @@
 id: GO:0019442
 name: obsolete L-tryptophan catabolic process to acetyl-CoA
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including acetyl-CoA." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including acetyl-CoA." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "tryptophan breakdown to acetyl-CoA" EXACT []
 synonym: "tryptophan catabolic process to acetyl-CoA" EXACT []
@@ -155467,7 +155896,7 @@
 id: GO:0019444
 name: obsolete L-tryptophan catabolic process to catechol
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including catechol." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tryptophan into other compounds, including catechol." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "tryptophan breakdown to catechol" EXACT []
 synonym: "tryptophan catabolic process to catechol" EXACT []
@@ -155480,7 +155909,7 @@
 id: GO:0019445
 name: obsolete L-tyrosine catabolic process to fumarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tyrosine into other compounds, including fumarate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-tyrosine into other compounds, including fumarate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "tyrosine breakdown to fumarate" EXACT []
 synonym: "tyrosine degradation to fumarate" EXACT []
@@ -155492,7 +155921,7 @@
 id: GO:0019446
 name: obsolete tyrosine catabolic process to phosphoenolpyruvate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of tyrosine into other compounds, including phosphoenolpyruvate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of tyrosine into other compounds, including phosphoenolpyruvate." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this is a real process.
 synonym: "tyrosine breakdown to phosphoenolpyruvate" EXACT []
 synonym: "tyrosine degradation to phosphoenolpyruvate" EXACT []
@@ -155500,21 +155929,21 @@
 
 [Term]
 id: GO:0019447
-name: D-cysteine catabolic process
+name: obsolete D-cysteine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-cysteine, (S)-2-amino-3-mercaptopropanoic acid, which occurs naturally in firefly luciferin." [PMID:11527960]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-cysteine, (S)-2-amino-3-mercaptopropanoic acid, which occurs naturally in firefly luciferin." [PMID:11527960]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-cysteine.
 synonym: "D-cysteine breakdown" EXACT []
 synonym: "D-cysteine catabolism" EXACT []
 synonym: "D-cysteine degradation" EXACT []
-is_a: GO:0000098 ! sulfur amino acid catabolic process
-is_a: GO:0006534 ! cysteine metabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019448
 name: L-cysteine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-cysteine, the L-enantiomer of 2-amino-3-mercaptopropanoic acid, i.e. (2R)-2-amino-3-mercaptopropanoic acid." [GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-cysteine." [GOC:curators]
 synonym: "L-cysteine breakdown" EXACT []
 synonym: "L-cysteine catabolism" EXACT []
 synonym: "L-cysteine degradation" EXACT []
@@ -155529,12 +155958,13 @@
 property_value: skos:narrowMatch MetaCyc:LCYSDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-5329
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019449
 name: obsolete L-cysteine catabolic process to hypotaurine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine into other compounds, including hypotaurine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine into other compounds, including hypotaurine." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown to hypotaurine" EXACT []
 synonym: "L-cysteine degradation to hypotaurine" EXACT []
@@ -155546,7 +155976,7 @@
 id: GO:0019450
 name: obsolete L-cysteine catabolic process to pyruvate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine into other compounds, including pyruvate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine into other compounds, including pyruvate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown to pyruvate" EXACT []
 synonym: "L-cysteine degradation to pyruvate" EXACT []
@@ -155556,37 +155986,33 @@
 
 [Term]
 id: GO:0019451
-name: L-cysteine catabolic process to pyruvate, using cysteine dioxygenase
+name: obsolete L-cysteine catabolic process to pyruvate, using cysteine dioxygenase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown into pyruvate of L-cystine, catalyzed by the enzyme cysteine dioxygenase (EC:1.13.11.20)." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown into pyruvate of L-cystine, catalyzed by the enzyme cysteine dioxygenase (EC:1.13.11.20)." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown to pyruvate, using cysteine dioxygenase" EXACT []
 synonym: "L-cysteine degradation to pyruvate, using cysteine dioxygenase" EXACT []
-is_a: GO:0006090 ! pyruvate metabolic process
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0017172 ! cysteine dioxygenase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019452
-name: taurine biosynthetic process from L-cysteine
+name: obsolete taurine biosynthetic process from L-cysteine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of taurine (2-aminoethanesulfonic acid) from L-cysteine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of taurine (2-aminoethanesulfonic acid) from L-cysteine." [GOC:curators]
+comment: This term was obsoleted because it is too specific and is outside the scope of GO.
 synonym: "L-cysteine breakdown to taurine" EXACT []
 synonym: "L-cysteine degradation to taurine" EXACT []
-xref: MetaCyc:PWY-5331
-xref: MetaCyc:PWY-7850
-xref: MetaCyc:PWY-8359
-is_a: GO:0042412 ! taurine biosynthetic process
-is_a: GO:0046439 ! L-cysteine metabolic process
-property_value: skos:narrowMatch MetaCyc:PWY-5331
-property_value: skos:narrowMatch MetaCyc:PWY-7850
-property_value: skos:narrowMatch MetaCyc:PWY-8359
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0042412
 
 [Term]
 id: GO:0019453
 name: obsolete L-cysteine catabolic process via cystine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine, via the intermediate cystine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-cysteine, via the intermediate cystine." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine" EXACT []
 synonym: "L-cysteine degradation via cystine" EXACT []
@@ -155596,40 +156022,46 @@
 
 [Term]
 id: GO:0019454
-name: L-cysteine catabolic process via cystine, using glutathione-cystine transhydrogenase
+name: obsolete L-cysteine catabolic process via cystine, using glutathione-cystine transhydrogenase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme glutathione-cystine transhydrogenase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme glutathione-cystine transhydrogenase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using glutathione-cystine transhydrogenase" EXACT []
 synonym: "L-cysteine degradation via cystine, using glutathione-cystine transhydrogenase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0047141 ! glutathione-cystine transhydrogenase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019455
-name: L-cysteine catabolic process via cystine, using cystine reductase
+name: obsolete L-cysteine catabolic process via cystine, using cystine reductase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cystine reductase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cystine reductase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using cystine reductase" EXACT []
 synonym: "L-cysteine degradation via cystine, using cystine reductase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0050456 ! L-cystine reductase (NADH) activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019456
-name: L-cysteine catabolic process via cystine, using cysteine transaminase
+name: obsolete L-cysteine catabolic process via cystine, using cysteine transaminase
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cysteine transaminase." [GOC:jl]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown, via the compound cystine, of L-cysteine, catalyzed by the enzyme cysteine transaminase." [GOC:jl]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-cysteine breakdown via cystine, using cysteine transaminase" EXACT []
 synonym: "L-cysteine degradation via cystine, using cysteine transaminase" EXACT []
-is_a: GO:0019448 ! L-cysteine catabolic process
-relationship: has_part GO:0047801 ! L-cysteine transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22538" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019448
 
 [Term]
 id: GO:0019457
 name: obsolete L-methionine catabolic process to succinyl-CoA
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-methionine into other compounds, including succinyl-CoA." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-methionine into other compounds, including succinyl-CoA." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "methionine breakdown to succinyl-CoA" EXACT []
 synonym: "methionine degradation to succinyl-CoA" EXACT []
@@ -155641,7 +156073,7 @@
 id: GO:0019458
 name: obsolete L-methionine catabolic process via 2-oxobutanoate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-methionine, via the intermediate 2-oxobutanoate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-methionine, via the intermediate 2-oxobutanoate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "methionine breakdown via 2-oxobutanoate" EXACT []
 synonym: "methionine degradation via 2-oxobutanoate" EXACT []
@@ -155653,7 +156085,7 @@
 id: GO:0019460
 name: obsolete glutamine catabolic process to fumarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamine into other compounds, including fumarate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamine into other compounds, including fumarate." [GOC:curators]
 comment: The reason for obsoletion is that this term is the same as the parent term.
 synonym: "glutamate catabolic process to fumarate" NARROW [GOC:bf, GOC:jl]
 synonym: "glutamine breakdown to fumarate" EXACT []
@@ -155692,7 +156124,7 @@
 id: GO:0019463
 name: obsolete glycine catabolic process to creatine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glycine into other compounds, including creatine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glycine into other compounds, including creatine." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this process exists.
 synonym: "glycine breakdown to creatine" EXACT []
 synonym: "glycine degradation to creatine" EXACT []
@@ -155714,7 +156146,7 @@
 id: GO:0019465
 name: obsolete aspartate transamidation
 namespace: biological_process
-def: "OBSOLETE. The exchange of the amino group of aspartate, the anion derived from aspartic acid, for another amino group." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The exchange of the amino group of aspartate, the anion derived from aspartic acid, for another amino group." [GOC:curators, ISBN:0198506732]
 comment: The reason for obsoletion is that this term represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28786" xsd:anyURI
@@ -155722,26 +156154,30 @@
 
 [Term]
 id: GO:0019466
-name: L-ornithine catabolic process via proline
+name: obsolete L-ornithine catabolic process via proline
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine, via the intermediate proline." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-ornithine, via the intermediate proline." [GOC:curators]
+comment: This term was obsoleted because it represents a variant pathway of L-ornithine degradation and is out of scope for GO.
 synonym: "ornithine breakdown via proline" EXACT []
 synonym: "ornithine degradation via proline" EXACT []
-xref: MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
-is_a: GO:0006593 ! L-ornithine catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0006593
 
 [Term]
 id: GO:0019467
-name: L-ornithine catabolic process, by decarboxylation
+name: obsolete L-ornithine catabolic process, by decarboxylation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ornithine by decarboxylation." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-ornithine by decarboxylation." [GOC:curators]
+comment: This term was obsoleted because it represents a variant pathway of L-ornithine degradation and is out of scope for GO.
 synonym: "ornithine breakdown, by decarboxylation" EXACT []
 synonym: "ornithine degradation, by decarboxylation" EXACT []
 xref: MetaCyc:ORNDEG-PWY
-is_a: GO:0006593 ! L-ornithine catabolic process
-relationship: has_part GO:0016831 ! carboxy-lyase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0006593
 
 [Term]
 id: GO:0019468
@@ -155771,7 +156207,7 @@
 id: GO:0019470
 name: trans-4-hydroxy-L-proline catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of trans-4-hydroxy-L-proline, C5H9NO3, a derivative of the amino acid proline." [GOC:ai]
+def: "The chemical reactions and pathways resulting in the breakdown of trans-4-hydroxy-L-proline." [GOC:curators]
 synonym: "4-hydroxyproline breakdown" EXACT []
 synonym: "4-hydroxyproline catabolism" EXACT []
 synonym: "4-hydroxyproline degradation" EXACT []
@@ -155784,6 +156220,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5159
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019471
@@ -155812,7 +156249,7 @@
 id: GO:0019473
 name: obsolete L-lysine catabolic process to glutarate, by acetylation
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including glutarate, by acetylation." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including glutarate, by acetylation." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to glutarate, by acetylation" EXACT []
 synonym: "L-lysine degradation to glutarate, by acetylation" EXACT []
@@ -155824,7 +156261,7 @@
 id: GO:0019474
 name: obsolete L-lysine catabolic process to acetyl-CoA
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA" EXACT []
@@ -155852,20 +156289,21 @@
 
 [Term]
 id: GO:0019476
-name: D-lysine catabolic process
+name: obsolete D-lysine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-lysine, the D-enantiomer of lysine; i.e. (2R)-2,6-diaminohexanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-lysine, the D-enantiomer of lysine; i.e. (2R)-2,6-diaminohexanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-lysine.
 synonym: "D-lysine breakdown" EXACT []
 synonym: "D-lysine catabolism" EXACT []
 synonym: "D-lysine degradation" EXACT []
-is_a: GO:0006553 ! lysine metabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0019477
 name: L-lysine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine, the L-enantiomer of (S)-2,6-diaminohexanoic acid." [GOC:go_curators, GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-lysine." [GOC:curators]
 synonym: "L-lysine breakdown" EXACT []
 synonym: "L-lysine catabolism" EXACT []
 synonym: "L-lysine degradation" EXACT []
@@ -155874,6 +156312,9 @@
 xref: MetaCyc:PWY-5280
 xref: MetaCyc:PWY-5283
 xref: MetaCyc:PWY-5298
+xref: MetaCyc:PWY-5311
+xref: MetaCyc:PWY-5314
+xref: MetaCyc:PWY-5324
 xref: MetaCyc:PWY-6328
 xref: MetaCyc:PWY0-461
 xref: MetaCyc:PWY66-425
@@ -155885,10 +156326,15 @@
 property_value: skos:narrowMatch MetaCyc:PWY-5280
 property_value: skos:narrowMatch MetaCyc:PWY-5283
 property_value: skos:narrowMatch MetaCyc:PWY-5298
+property_value: skos:narrowMatch MetaCyc:PWY-5311
+property_value: skos:narrowMatch MetaCyc:PWY-5314
+property_value: skos:narrowMatch MetaCyc:PWY-5324
 property_value: skos:narrowMatch MetaCyc:PWY-6328
 property_value: skos:narrowMatch MetaCyc:PWY0-461
 property_value: skos:narrowMatch MetaCyc:PWY66-425
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019478
@@ -155926,7 +156372,7 @@
 id: GO:0019481
 name: obsolete L-alanine catabolic process, by transamination
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-alanine by transamination." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-alanine by transamination." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-alanine breakdown, by transamination" EXACT []
 synonym: "L-alanine degradation, by transamination" EXACT []
@@ -155952,15 +156398,26 @@
 synonym: "beta-alanine biosynthesis" EXACT []
 synonym: "beta-alanine formation" EXACT []
 synonym: "beta-alanine synthesis" EXACT []
+xref: MetaCyc:PWY-3941
+xref: MetaCyc:PWY-3981
+xref: MetaCyc:PWY-3982
+xref: MetaCyc:PWY-5155
+xref: MetaCyc:PWY-5760
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0019482 ! beta-alanine metabolic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-3941
+property_value: skos:narrowMatch MetaCyc:PWY-3981
+property_value: skos:narrowMatch MetaCyc:PWY-3982
+property_value: skos:narrowMatch MetaCyc:PWY-5155
+property_value: skos:narrowMatch MetaCyc:PWY-5760
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0019484
 name: beta-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine (3-aminopropanoic acid), an achiral amino acid and an isomer of alanine. It occurs free (e.g. in brain) and in combination (e.g. in pantothenate) but it is not a constituent of proteins." [GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine." [GOC:curators]
 synonym: "beta-alanine breakdown" EXACT []
 synonym: "beta-alanine catabolism" EXACT []
 synonym: "beta-alanine degradation" EXACT []
@@ -155974,25 +156431,31 @@
 property_value: skos:narrowMatch MetaCyc:PWY-1781
 property_value: skos:narrowMatch MetaCyc:PWY-8120
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019485
-name: beta-alanine catabolic process to L-alanine
+name: obsolete beta-alanine catabolic process to L-alanine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including L-alanine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including L-alanine." [GOC:curators]
+comment: This term was obsoleted because it represents a variant pathway of beta-alanine degradation and is out of scope for GO.
 synonym: "beta-alanine breakdown to L-alanine" EXACT []
 synonym: "beta-alanine degradation to L-alanine" EXACT []
-is_a: GO:0019484 ! beta-alanine catabolic process
-is_a: GO:0042851 ! L-alanine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019484
 
 [Term]
 id: GO:0019486
-name: beta-alanine catabolic process to mevalonate semialdehyde, by transamination
+name: obsolete beta-alanine catabolic process to mevalonate semialdehyde, by transamination
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including mevalonate semialdehyde, by transamination." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of beta-alanine into other compounds, including mevalonate semialdehyde, by transamination." [GOC:curators]
+comment: This term was obsoleted because it represents a variant pathway of beta-alanine degradation and is out of scope for GO.
 synonym: "beta-alanine breakdown to mevalonate semialdehyde, by transamination" EXACT []
 synonym: "beta-alanine degradation to mevalonate semialdehyde, by transamination" EXACT []
-is_a: GO:0019484 ! beta-alanine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019484
 
 [Term]
 id: GO:0019487
@@ -156068,7 +156531,7 @@
 id: GO:0019493
 name: obsolete L-arginine catabolic process to L-proline
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including L-proline." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including L-proline." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "arginine breakdown to proline" EXACT []
 synonym: "arginine degradation to proline" EXACT []
@@ -156080,7 +156543,7 @@
 id: GO:0019495
 name: obsolete proline catabolic process to 2-oxoglutarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of proline into other compounds, including 2-oxoglutarate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of proline into other compounds, including 2-oxoglutarate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "proline breakdown to 2-oxoglutarate" EXACT []
 synonym: "proline catabolic process to 2-ketoglutarate" EXACT []
@@ -156185,7 +156648,7 @@
 id: GO:0019504
 name: L-proline betaine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of stachydrine, N-methylproline methylbetaine, the betaine derivative of L-proline." [PMID:10689197, PMID:1804402, PMID:24520058]
+def: "The chemical reactions and pathways resulting in the breakdown of L-proline betaine (stachydrine)." [PMID:10689197, PMID:1804402, PMID:24520058]
 synonym: "stachydrine breakdown" EXACT []
 synonym: "stachydrine catabolic process" EXACT []
 synonym: "stachydrine catabolism" EXACT []
@@ -156200,6 +156663,7 @@
 property_value: skos:narrowMatch MetaCyc:PWY-8109
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23577" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0019505
@@ -156255,19 +156719,19 @@
 
 [Term]
 id: GO:0019509
-name: L-methionine salvage from methylthioadenosine
+name: obsolete L-methionine salvage from methylthioadenosine
 namespace: biological_process
-def: "The generation of L-methionine (2-amino-4-(methylthio)butanoic acid) from methylthioadenosine." [GOC:jl, MetaCyc:PWY-4361, PMID:19946895]
+def: "OBSOLETE. The generation of L-methionine (2-amino-4-(methylthio)butanoic acid) from methylthioadenosine." [GOC:jl, PMID:19946895]
+comment: This term was obsoleted because it represents a specific pathway beyond the specificity of GO.
 synonym: "methionine recycling" NARROW []
 synonym: "methionine regeneration" NARROW []
 synonym: "methionine salvage from methylthioadenosine" EXACT [GOC:mah]
 synonym: "methionine salvage pathway" EXACT []
-xref: MetaCyc:PWY-4361
-xref: MetaCyc:PWY-6753
-xref: MetaCyc:PWY-6754
-xref: MetaCyc:PWY-6756
 xref: Reactome:R-HSA-1237112 "Methionine salvage pathway"
-is_a: GO:0071267 ! L-methionine salvage
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033353
 
 [Term]
 id: GO:0019510
@@ -156298,7 +156762,7 @@
 id: GO:0019512
 name: lactose catabolic process via tagatose-6-phosphate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of lactose, via the intermediate tagatose-6-phosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of lactose, via the intermediate tagatose-6-phosphate." [GOC:curators]
 synonym: "lactose breakdown via tagatose-6-phosphate" EXACT []
 synonym: "lactose degradation via tagatose-6-phosphate" EXACT []
 xref: MetaCyc:LACTOSECAT-PWY
@@ -156329,7 +156793,7 @@
 id: GO:0019515
 name: lactose catabolic process via UDP-galactose
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of lactose, via the intermediate UDP-galactose." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of lactose, via the intermediate UDP-galactose." [GOC:curators]
 synonym: "lactose breakdown via UDP-galactose" EXACT []
 synonym: "lactose degradation via UDP-galactose" EXACT []
 is_a: GO:0005990 ! lactose catabolic process
@@ -156623,7 +157087,7 @@
 id: GO:0019541
 name: propionate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving propionate, the anion derived from propionic (propanoic) acid, a carboxylic acid important in the energy metabolism of ruminants." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving propionate, the anion derived from propionic (propanoic) acid, a carboxylic acid important in the energy metabolism of ruminants." [GOC:curators, ISBN:0198506732]
 synonym: "propanoate metabolic process" EXACT []
 synonym: "propanoate metabolism" EXACT []
 synonym: "propionate metabolism" EXACT []
@@ -156633,7 +157097,7 @@
 id: GO:0019542
 name: propionate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of propionate, the anion derived from propionic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of propionate, the anion derived from propionic acid." [GOC:curators]
 synonym: "propionate anabolism" EXACT []
 synonym: "propionate biosynthesis" EXACT []
 synonym: "propionate formation" EXACT []
@@ -156645,7 +157109,7 @@
 id: GO:0019543
 name: propionate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of propionate, the anion derived from propionic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of propionate, the anion derived from propionic acid." [GOC:curators]
 synonym: "propionate breakdown" EXACT []
 synonym: "propionate catabolism" EXACT []
 synonym: "propionate degradation" EXACT []
@@ -156656,7 +157120,7 @@
 id: GO:0019544
 name: obsolete L-arginine catabolic process to L-glutamate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including L-glutamate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including L-glutamate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "arginine breakdown to glutamate" EXACT []
 synonym: "arginine degradation to glutamate" EXACT []
@@ -156669,7 +157133,7 @@
 id: GO:0019545
 name: obsolete L-arginine catabolic process to succinate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including succinate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-arginine into other compounds, including succinate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "arginine breakdown to succinate" EXACT []
 synonym: "arginine degradation to succinate" EXACT []
@@ -156692,7 +157156,7 @@
 id: GO:0019547
 name: obsolete arginine catabolic process to ornithine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of arginine into other compounds, including ornithine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of arginine into other compounds, including ornithine." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "arginine breakdown to ornithine" EXACT []
 synonym: "arginine degradation to ornithine" EXACT []
@@ -156704,7 +157168,7 @@
 id: GO:0019548
 name: obsolete arginine catabolic process to spermine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of arginine into other compounds, including spermine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of arginine into other compounds, including spermine." [GOC:curators]
 comment: The reason for obsoletion is that this term was added in error.
 synonym: "arginine breakdown to spermine" EXACT []
 synonym: "arginine degradation to spermine" EXACT []
@@ -156716,7 +157180,7 @@
 id: GO:0019549
 name: obsolete glutamate catabolic process to succinate via succinate semialdehyde
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into succinate, that includes the conversion of 4-aminobutyrate to succinate semialdehyde by the pyruvate-dependent gamma aminobutyrate (GABA) transaminase." [GOC:bf, GOC:go_curators, MetaCyc:PWY-4321]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into succinate, that includes the conversion of 4-aminobutyrate to succinate semialdehyde by the pyruvate-dependent gamma aminobutyrate (GABA) transaminase." [GOC:bf, GOC:curators, MetaCyc:PWY-4321]
 comment: The reason for obsoletion is that this term was added in error (unnecessary specificity, these are steps in a single pathway)
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28355" xsd:anyURI
 is_obsolete: true
@@ -156726,7 +157190,7 @@
 id: GO:0019550
 name: obsolete L-glutamate catabolic process to aspartate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including aspartate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including aspartate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown to aspartate" EXACT []
 synonym: "glutamate degradation to aspartate" EXACT []
@@ -156738,7 +157202,7 @@
 id: GO:0019551
 name: obsolete glutamate catabolic process to 2-oxoglutarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into other compounds, including 2-oxoglutarate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into other compounds, including 2-oxoglutarate." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "glutamate breakdown to 2-oxoglutarate" EXACT []
 synonym: "glutamate catabolic process to 2-ketoglutarate" EXACT []
@@ -156754,35 +157218,34 @@
 
 [Term]
 id: GO:0019552
-name: L-glutamate fermentation via 2-hydroxyglutarate
+name: obsolete L-glutamate fermentation via 2-hydroxyglutarate
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate 2-hydroxyglutarate, yielding acetyl-CoA and butanoate. Acetyl-CoA can be further converted into acetate, releasing one ATP molecule." [PMID:17166837]
+def: "OBSOLETE. The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate 2-hydroxyglutarate, yielding acetyl-CoA and butanoate. Acetyl-CoA can be further converted into acetate, releasing one ATP molecule." [PMID:17166837]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-glutamate catabolic process via 2-hydroxyglutarate" EXACT []
-xref: MetaCyc:P162-PWY
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019665 ! amino acid fermentation
-property_value: skos:narrowMatch MetaCyc:P162-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29511" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0019553
-name: L-glutamate fermentation via L-citramalate
+name: obsolete L-glutamate fermentation via L-citramalate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate L-citramalate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate, via the intermediate L-citramalate." [GOC:curators]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown via L-citramalate" EXACT []
 synonym: "glutamate degradation via L-citramalate" EXACT []
 synonym: "L-glutamate catabolic process via L-citramalate" EXACT []
-xref: MetaCyc:GLUDEG-II-PWY
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019665 ! amino acid fermentation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0019554
 name: obsolete glutamate catabolic process to oxaloacetate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into other compounds, including oxaloacetate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of glutamate into other compounds, including oxaloacetate." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this function/process/component exists.
 synonym: "glutamate breakdown to oxaloacetate" EXACT []
 synonym: "glutamate degradation to oxaloacetate" EXACT []
@@ -156793,7 +157256,7 @@
 id: GO:0019555
 name: obsolete L-glutamate catabolic process to ornithine
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including ornithine." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including ornithine." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown to ornithine" EXACT []
 synonym: "glutamate degradation to ornithine" EXACT []
@@ -156805,7 +157268,7 @@
 id: GO:0019556
 name: obsolete L-histidine catabolic process to glutamate and formamide
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including glutamate and formamide." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including glutamate and formamide." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "histidine breakdown to glutamate and formamide" EXACT []
 synonym: "histidine catabolic process to glutamate and formamide" BROAD []
@@ -156818,7 +157281,7 @@
 id: GO:0019557
 name: obsolete L-histidine catabolic process to glutamate and formate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including glutamate and formate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including glutamate and formate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "histidine breakdown to glutamate and formate" EXACT []
 synonym: "histidine catabolic process to glutamate and formate" BROAD []
@@ -156831,7 +157294,7 @@
 id: GO:0019558
 name: obsolete L-histidine catabolic process to 2-oxoglutarate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including 2-oxoglutarate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including 2-oxoglutarate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "histidine breakdown to 2-oxoglutarate" EXACT []
 synonym: "histidine catabolic process to 2-ketoglutarate" EXACT []
@@ -156850,7 +157313,7 @@
 id: GO:0019559
 name: obsolete L-histidine catabolic process to imidazol-5-yl-lactate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including imidazol-5-yl-lactate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including imidazol-5-yl-lactate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "histidine breakdown to imidazol-5-yl-lactate" EXACT []
 synonym: "histidine degradation to imidazol-5-yl-lactate" EXACT []
@@ -156862,7 +157325,7 @@
 id: GO:0019560
 name: obsolete L-histidine catabolic process to hydantoin-5-propionate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including hydantoin-5-propionate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-histidine into other compounds, including hydantoin-5-propionate." [GOC:curators]
 comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "histidine breakdown to hydantoin-5-propionate" EXACT []
 synonym: "histidine degradation to hydantoin-5-propionate" EXACT []
@@ -156884,7 +157347,7 @@
 id: GO:0019562
 name: obsolete L-phenylalanine catabolic process to phosphoenolpyruvate
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-phenylalanine into other compounds, including phosphoenolpyruvate." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-phenylalanine into other compounds, including phosphoenolpyruvate." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this is a real process.
 synonym: "phenylalanine breakdown to phosphoenolpyruvate" EXACT []
 synonym: "phenylalanine degradation to phosphoenolpyruvate" EXACT []
@@ -156894,7 +157357,7 @@
 id: GO:0019563
 name: glycerol catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of glycerol, 1,2,3-propanetriol, a sweet, hygroscopic, viscous liquid, widely distributed in nature as a constituent of many lipids." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the breakdown of glycerol, 1,2,3-propanetriol, a sweet, hygroscopic, viscous liquid, widely distributed in nature as a constituent of many lipids." [GOC:curators, ISBN:0198506732]
 synonym: "glycerol breakdown" EXACT []
 synonym: "glycerol catabolism" EXACT []
 synonym: "glycerol degradation" EXACT []
@@ -156953,7 +157416,7 @@
 id: GO:0019569
 name: L-arabinose catabolic process to D-xylulose 5-phosphate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-arabinose into D-xylulose 5-phosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-arabinose into D-xylulose 5-phosphate." [GOC:curators]
 synonym: "L-arabinose breakdown to xylulose 5-phosphate" EXACT []
 synonym: "L-arabinose degradation to xylulose 5-phosphate" EXACT []
 xref: MetaCyc:ARABCAT-PWY
@@ -156965,7 +157428,7 @@
 id: GO:0019570
 name: L-arabinose catabolic process to 2-oxoglutarate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-arabinose into other compounds, including 2-oxoglutarate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-arabinose into other compounds, including 2-oxoglutarate." [GOC:curators]
 synonym: "L-arabinose breakdown to 2-oxoglutarate" EXACT []
 synonym: "L-arabinose catabolic process to 2-ketoglutarate" EXACT []
 synonym: "L-arabinose catabolic process to alpha-ketoglutarate" EXACT []
@@ -157002,7 +157465,7 @@
 id: GO:0019573
 name: D-arabinose catabolic process to D-xylulose 5-phosphate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-arabinose to form D-xylulose 5-phosphate. D-arabinose is converted into D-ribulose, which is phosphorylated to D-ribulose-5-phosphate, which is isomerized to give D-xylulose-5-phosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of D-arabinose to form D-xylulose 5-phosphate. D-arabinose is converted into D-ribulose, which is phosphorylated to D-ribulose-5-phosphate, which is isomerized to give D-xylulose-5-phosphate." [GOC:curators]
 synonym: "D-arabinose breakdown to xylulose 5-phosphate" EXACT []
 synonym: "D-arabinose degradation to xylulose 5-phosphate" EXACT []
 xref: MetaCyc:DARABCAT-PWY
@@ -157012,23 +157475,18 @@
 
 [Term]
 id: GO:0019574
-name: sucrose catabolic process via 3'-ketosucrose
+name: obsolete sucrose catabolic process via 3'-ketosucrose
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sucrose, which proceeds via the conversion of sucrose to 3'-ketosucrose. 3'-ketosucrose is hydrolyzed to 3-ketoglucose and fructose, and the 3-ketoglucose is then be converted to glucose." [GOC:bf, GOC:dgf, MetaCyc:SUCROSEUTIL2-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of sucrose, which proceeds via the conversion of sucrose to 3'-ketosucrose. 3'-ketosucrose is hydrolyzed to 3-ketoglucose and fructose, and the 3-ketoglucose is then be converted to glucose." [GOC:bf, GOC:dgf, MetaCyc:SUCROSEUTIL2-PWY]
+comment: This term was obsoleted because it represents a specific pathway out of scope for GO.
 synonym: "sucrose breakdown, using glucoside 3-dehydrogenase" EXACT []
 synonym: "sucrose catabolic process to D-glucose" EXACT [GOC:bf, MetaCyc:SUCROSEUTIL2-PWY]
 synonym: "sucrose catabolic process, using glucoside 3-dehydrogenase" EXACT [GOC:bf, MetaCyc:SUCROSEUTIL2-PWY]
 synonym: "sucrose catabolism, using glucoside 3-dehydrogenase" EXACT []
 synonym: "sucrose degradation, using glucoside 3-dehydrogenase" EXACT []
-xref: MetaCyc:SUCROSEUTIL2-PWY
-is_a: GO:0005987 ! sucrose catabolic process
-is_a: GO:0006006 ! glucose metabolic process
-is_a: GO:0006796 ! phosphate-containing compound metabolic process
-is_a: GO:0019637 ! organophosphate metabolic process
-is_a: GO:1901135 ! carbohydrate derivative metabolic process
-relationship: has_part GO:0008865 ! fructokinase activity
-relationship: has_part GO:0033757 ! glucoside 3-dehydrogenase activity
-relationship: has_part GO:0048258 ! 3-ketoglucose-reductase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0005987
 
 [Term]
 id: GO:0019575
@@ -157126,8 +157584,9 @@
 name: glucuronate metabolic process
 namespace: biological_process
 alt_id: GO:0019699
-def: "The chemical reactions and pathways involving glucuronate, any salt or ester of glucuronic acid, the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving glucuronate, any salt or ester of glucuronic acid, the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:curators, ISBN:0198506732]
 synonym: "glucuronate metabolism" EXACT []
+xref: Reactome:R-HSA-156588 "Glucuronidation"
 xref: Wikipedia:Glucuronic_acid
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 
@@ -157183,11 +157642,14 @@
 synonym: "mannitol breakdown" EXACT []
 synonym: "mannitol catabolism" EXACT []
 synonym: "mannitol degradation" EXACT []
+xref: MetaCyc:MANNIDEG-PWY
 xref: MetaCyc:PWY-3861
 is_a: GO:0019407 ! hexitol catabolic process
 is_a: GO:0019594 ! mannitol metabolic process
+property_value: skos:narrowMatch MetaCyc:MANNIDEG-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0019593
@@ -157225,7 +157687,7 @@
 id: GO:0019596
 name: mandelate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of mandelate, the anion of mandelic acid. Mandelic acid (alpha-hydroxybenzeneacetic acid) is an 8-carbon alpha-hydroxy acid (AHA) that is used in organic chemistry and as a urinary antiseptic." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of mandelate, the anion of mandelic acid. Mandelic acid (alpha-hydroxybenzeneacetic acid) is an 8-carbon alpha-hydroxy acid (AHA) that is used in organic chemistry and as a urinary antiseptic." [GOC:curators]
 synonym: "mandelate breakdown" EXACT []
 synonym: "mandelate catabolism" EXACT []
 synonym: "mandelate degradation" EXACT []
@@ -157236,7 +157698,7 @@
 id: GO:0019597
 name: (R)-mandelate catabolic process to benzoate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of (R)-mandelate into other compounds, including benzoate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of (R)-mandelate into other compounds, including benzoate." [GOC:curators]
 synonym: "(R)-mandelate breakdown to benzoate" EXACT []
 synonym: "(R)-mandelate degradation to benzoate" EXACT []
 is_a: GO:0018874 ! benzoate metabolic process
@@ -157247,7 +157709,7 @@
 id: GO:0019598
 name: (R)-mandelate catabolic process to catechol
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of (R)-mandelate into other compounds, including catechol." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of (R)-mandelate into other compounds, including catechol." [GOC:curators]
 synonym: "(R)-mandelate breakdown to catechol" EXACT []
 synonym: "(R)-mandelate degradation to catechol" EXACT []
 is_a: GO:0009713 ! catechol-containing compound biosynthetic process
@@ -157413,7 +157875,7 @@
 id: GO:0019614
 name: catechol-containing compound catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of catechol-containing compounds. Catechol is a compound containing a pyrocatechol nucleus or substituent." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of catechol-containing compounds. Catechol is a compound containing a pyrocatechol nucleus or substituent." [GOC:curators]
 synonym: "catechol breakdown" RELATED []
 synonym: "catechol catabolic process" RELATED []
 synonym: "catechol catabolism" RELATED []
@@ -157497,7 +157959,7 @@
 id: GO:0019621
 name: creatinine catabolic process to formate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of creatinine into other compounds, including formate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of creatinine into other compounds, including formate." [GOC:curators]
 synonym: "creatinine breakdown to formate" EXACT []
 synonym: "creatinine degradation to formate" EXACT []
 xref: MetaCyc:CRNFORCAT-PWY
@@ -157591,7 +158053,7 @@
 id: GO:0019629
 name: propionate catabolic process, 2-methylcitrate cycle
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of propionate that occurs in the 2-methylcitrate cycle." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of propionate that occurs in the 2-methylcitrate cycle." [GOC:curators]
 synonym: "propionate breakdown, 2-methylcitrate cycle" EXACT []
 synonym: "propionate degradation, 2-methylcitrate cycle" EXACT []
 is_a: GO:0019543 ! propionate catabolic process
@@ -157633,7 +158095,7 @@
 id: GO:0019633
 name: shikimate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of shikimate, (3R,4S,5R)--3,4,5-trihydroxycyclohex-1-ene-1-carboxylate, the anion of shikimic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of shikimate, (3R,4S,5R)--3,4,5-trihydroxycyclohex-1-ene-1-carboxylate, the anion of shikimic acid." [GOC:curators]
 synonym: "shikimate breakdown" EXACT []
 synonym: "shikimate catabolism" EXACT []
 synonym: "shikimate degradation" EXACT []
@@ -157723,7 +158185,7 @@
 id: GO:0019640
 name: D-glucuronate catabolic process to D-xylulose 5-phosphate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-glucuronate into D-xylulose 5-phosphate." [GOC:go_curators, PMID:27189775]
+def: "The chemical reactions and pathways resulting in the breakdown of D-glucuronate into D-xylulose 5-phosphate." [GOC:curators, PMID:27189775]
 synonym: "glucuronate breakdown to xylulose 5-phosphate" EXACT []
 synonym: "glucuronate catabolic process to xylulose 5-phosphate" EXACT []
 synonym: "glucuronate degradation to xylulose 5-phosphate" EXACT []
@@ -157854,7 +158316,7 @@
 name: purine fermentation
 namespace: biological_process
 def: "The anaerobic chemical reactions and pathways resulting in the breakdown of purine nucleobases, yielding acetate and CO2." [GOC:mah]
-synonym: "anaerobic purine base catabolic process" EXACT [GOC:go_curators]
+synonym: "anaerobic purine base catabolic process" EXACT [GOC:curators]
 synonym: "anaerobic purine base catabolism" EXACT []
 synonym: "anaerobic purine catabolic process" RELATED []
 synonym: "anaerobic purine nucleobase catabolic process" EXACT []
@@ -158022,7 +158484,7 @@
 id: GO:0019665
 name: amino acid fermentation
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of amino acids to ammonia, CO2, H2, acetate and short chain fatty acids, under anoxic conditions." [GOC:curators, GOC:jl]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of amino acids to ammonia, CO2, H2, acetate and short chain fatty acids, under anoxic conditions." [PMID:38821505]
 synonym: "anaerobic amino acid catabolic process" EXACT []
 is_a: GO:0006113 ! fermentation
 is_a: GO:0009063 ! amino acid catabolic process
@@ -158042,7 +158504,7 @@
 id: GO:0019667
 name: L-alanine fermentation
 namespace: biological_process
-def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-alanine, yielding acetylCoA. AcetylCoA can be further converted to ATP." [GOC:jl]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-alanine, yielding acetylCoA. AcetylCoA can be further converted to ATP." [GOC:jl, PMID:38821505]
 synonym: "anaerobic L-alanine catabolic process" EXACT []
 xref: MetaCyc:PROPFERM-PWY
 xref: MetaCyc:PWY-8188
@@ -158093,13 +158555,11 @@
 name: obsolete glutamate catabolic process via mesaconate and citramalate
 namespace: biological_process
 def: "OBSOLETE. The anaerobic chemical reactions and pathways resulting in the breakdown of glutamate via the intermediates mesaconate and S-citramalate, yielding energy in the form of ATP." [GOC:rynl]
-comment: The reason for obsoletion is that this term is an unnecessary grouping term.
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate fermentation via mesaconate and citramalate" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28390" xsd:anyURI
 is_obsolete: true
-consider: GO:0019553
 consider: GO:0033508
-consider: GO:0033509
 
 [Term]
 id: GO:0019672
@@ -158135,7 +158595,6 @@
 synonym: "oxidized NAD metabolism" EXACT []
 synonym: "oxidized nicotinamide adenine dinucleotide metabolic process" EXACT []
 synonym: "oxidized nicotinamide adenine dinucleotide metabolism" EXACT []
-xref: Reactome:R-HSA-196807 "Nicotinate metabolism"
 is_a: GO:0006163 ! purine nucleotide metabolic process
 is_a: GO:0046496 ! nicotinamide nucleotide metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21577" xsd:anyURI
@@ -158213,7 +158672,7 @@
 id: GO:0019678
 name: propionate metabolic process, methylmalonyl pathway
 namespace: biological_process
-def: "The chemical reactions and pathways involving propionate that occur in the methylmalonyl pathway." [GOC:go_curators]
+def: "The chemical reactions and pathways involving propionate that occur in the methylmalonyl pathway." [GOC:curators]
 synonym: "propionate metabolism, methylmalonyl pathway" EXACT []
 xref: MetaCyc:PROPIONMET-PWY
 is_a: GO:0019541 ! propionate metabolic process
@@ -158222,7 +158681,7 @@
 id: GO:0019679
 name: propionate metabolic process, methylcitrate cycle
 namespace: biological_process
-def: "The chemical reactions and pathways involving propionate that occur in the methylcitrate cycle." [GOC:go_curators]
+def: "The chemical reactions and pathways involving propionate that occur in the methylcitrate cycle." [GOC:curators]
 synonym: "propionate metabolism, methylcitrate cycle" EXACT []
 xref: MetaCyc:PWY0-42
 is_a: GO:0019541 ! propionate metabolic process
@@ -158315,7 +158774,7 @@
 id: GO:0019687
 name: pyruvate biosynthetic process from acetate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyruvate from other compounds, including acetate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of pyruvate from other compounds, including acetate." [GOC:curators]
 synonym: "pyruvate anabolism from acetate" EXACT []
 synonym: "pyruvate formation from acetate" EXACT []
 synonym: "pyruvate synthesis from acetate" EXACT []
@@ -158523,6 +158982,8 @@
 name: protein-cysteine S-acyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an acyl group to a sulfur atom on the cysteine of a protein molecule." [GOC:ai, RHEA:63372]
+xref: Reactome:R-HSA-158832 "The acetyl group from acetyl-CoA is transferred to NAT2"
+xref: Reactome:R-HSA-174959 "The acetyl group from acetyl-CoA is transferred to NAT1"
 xref: RHEA:63372
 is_a: GO:0016417 ! S-acyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -158566,7 +159027,7 @@
 id: GO:0019711
 name: obsolete peptidyl-beta-carboxyaspartic acid biosynthetic process from peptidyl-aspartic acid
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of peptidyl-beta-carboxyaspartic acid from other compounds, including peptidyl-aspartic acid." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of peptidyl-beta-carboxyaspartic acid from other compounds, including peptidyl-aspartic acid." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "peptidyl-beta-carboxyaspartic acid anabolism from peptidyl-aspartic acid" EXACT []
 synonym: "peptidyl-beta-carboxyaspartic acid formation from peptidyl-aspartic acid" EXACT []
@@ -158751,7 +159212,7 @@
 namespace: biological_process
 alt_id: GO:0006960
 alt_id: GO:0019735
-def: "An immune response against microbes mediated through a body fluid. Examples of this process are seen in the antimicrobial humoral response of Drosophila melanogaster and Mus musculus." [GOC:go_curators, GOC:mtg_sensu]
+def: "An immune response against microbes mediated through a body fluid. Examples of this process are seen in the antimicrobial humoral response of Drosophila melanogaster and Mus musculus." [GOC:curators, GOC:mtg_sensu]
 xref: Reactome:R-HSA-6803157 "Antimicrobial peptides"
 is_a: GO:0006959 ! humoral immune response
 is_a: GO:0140546 ! defense response to symbiont
@@ -158762,7 +159223,7 @@
 namespace: biological_process
 alt_id: GO:0006961
 alt_id: GO:0019733
-def: "An immune response against bacteria mediated through a body fluid. Examples of this process are the antibacterial humoral responses in Mus musculus and Drosophila melanogaster." [GOC:go_curators, GOC:mtg_sensu]
+def: "An immune response against bacteria mediated through a body fluid. Examples of this process are the antibacterial humoral responses in Mus musculus and Drosophila melanogaster." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0019730 ! antimicrobial humoral response
 is_a: GO:0042742 ! defense response to bacterium
 
@@ -158772,7 +159233,7 @@
 namespace: biological_process
 alt_id: GO:0006966
 alt_id: GO:0019734
-def: "An immune response against a fungus mediated through a body fluid. An example of this process is the antifungal humoral response in Drosophila melanogaster." [GOC:go_curators, GOC:mtg_sensu]
+def: "An immune response against a fungus mediated through a body fluid. An example of this process is the antifungal humoral response in Drosophila melanogaster." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0019730 ! antimicrobial humoral response
 is_a: GO:0050832 ! defense response to fungus
 
@@ -158861,7 +159322,7 @@
 id: GO:0019747
 name: regulation of isoprenoid metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving isoprenoids." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving isoprenoids." [GOC:curators]
 synonym: "regulation of isoprenoid metabolism" EXACT []
 is_a: GO:0019216 ! regulation of lipid metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -158872,7 +159333,7 @@
 id: GO:0019748
 name: secondary metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in many of the chemical changes of compounds that are not necessarily required for growth and maintenance of cells, and are often unique to a taxon. In multicellular organisms secondary metabolism is generally carried out in specific cell types, and may be useful for the organism as a whole. In unicellular organisms, secondary metabolism is often used for the production of antibiotics or for the utilization and acquisition of unusual nutrients." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in many of the chemical changes of compounds that are not necessarily required for growth and maintenance of cells, and are often unique to a taxon. In multicellular organisms secondary metabolism is generally carried out in specific cell types, and may be useful for the organism as a whole. In unicellular organisms, secondary metabolism is often used for the production of antibiotics or for the utilization and acquisition of unusual nutrients." [GOC:curators]
 subset: goslim_chembl
 subset: goslim_euk_cellular_processes_ribbon
 subset: goslim_pir
@@ -159197,6 +159658,7 @@
 namespace: molecular_function
 def: "Catalysis of the activation of the small ubiquitin-related modifier ISG15, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
 synonym: "UBE1L" NARROW []
+xref: Reactome:R-HSA-1169397 "Activation of ISG15 by UBA7 E1 enzyme"
 is_a: GO:0008641 ! ubiquitin-like modifier activating enzyme activity
 
 [Term]
@@ -159450,6 +159912,8 @@
 xref: Reactome:R-HSA-6804468 "PIAS1 SUMOylates SP3 with SUMO2"
 xref: Reactome:R-HSA-6804485 "PIAS1 SUMOylates L3MBTL2 with SUMO2"
 xref: Reactome:R-HSA-8956365 "ZBED1 (DREF) SUMOylates CHD3 with SUMO1"
+xref: Reactome:R-HSA-9920126 "UBC9 SUMOylates NS5"
+xref: Reactome:R-HSA-9926476 "UBE2I sumoylates MITF-M"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -159668,7 +160132,7 @@
 synonym: "N-acyl-L-aspartate amidohydrolase activity" RELATED [EC:3.5.1.15]
 xref: EC:3.5.1.15
 xref: MetaCyc:ASPARTOACYLASE-RXN
-xref: Reactome:R-HSA-5691507 "ASPA deacetylates NAA to acetate and L-aspartate"
+xref: Reactome:R-HSA-5691507 "ASPA deacetylates NAASP to acetate and L-aspartate"
 xref: RHEA:10872
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
 property_value: skos:exactMatch EC:3.5.1.15
@@ -159882,7 +160346,7 @@
 id: GO:0019830
 name: obsolete cadmium sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "cadmium sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -159892,7 +160356,7 @@
 id: GO:0019831
 name: obsolete chromate sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "chromate sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -159902,7 +160366,7 @@
 id: GO:0019832
 name: obsolete mercuric sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "mercuric sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -159961,7 +160425,7 @@
 id: GO:0019837
 name: obsolete herbicide susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "herbicide susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -160117,7 +160581,7 @@
 id: GO:0019854
 name: L-ascorbic acid catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-ascorbic acid; L-ascorbic acid ionizes to give L-ascorbate, (2R)-2-[(1S)-1,2-dihydroxyethyl]-4-hydroxy-5-oxo-2,5-dihydrofuran-3-olate, which is required as a cofactor in the oxidation of prolyl residues to hydroxyprolyl, and other reactions." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of L-ascorbic acid; L-ascorbic acid ionizes to give L-ascorbate, (2R)-2-[(1S)-1,2-dihydroxyethyl]-4-hydroxy-5-oxo-2,5-dihydrofuran-3-olate, which is required as a cofactor in the oxidation of prolyl residues to hydroxyprolyl, and other reactions." [GOC:curators]
 synonym: "ascorbate catabolic process" EXACT []
 synonym: "ascorbate catabolism" EXACT []
 synonym: "L-ascorbic acid breakdown" EXACT []
@@ -160137,6 +160601,7 @@
 name: calcium channel inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents, or reduces the activity of a calcium channel." [GOC:mah]
+xref: Reactome:R-HSA-400046 "L-type Calcium Channels close in pancreatic beta cells"
 is_a: GO:0005246 ! calcium channel regulator activity
 is_a: GO:0008200 ! ion channel inhibitor activity
 relationship: negatively_regulates GO:0005262 ! calcium channel activity
@@ -160145,10 +160610,10 @@
 id: GO:0019856
 name: pyrimidine nucleobase biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of pyrimidine nucleobases, 1,3-diazine, organic nitrogenous bases." [GOC:curators]
 synonym: "pyrimidine base anabolism" EXACT []
 synonym: "pyrimidine base biosynthesis" EXACT []
-synonym: "pyrimidine base biosynthetic process" EXACT [GOC:go_curators]
+synonym: "pyrimidine base biosynthetic process" EXACT [GOC:curators]
 synonym: "pyrimidine base formation" EXACT []
 synonym: "pyrimidine base synthesis" EXACT []
 is_a: GO:0006206 ! pyrimidine nucleobase metabolic process
@@ -160179,7 +160644,7 @@
 id: GO:0019859
 name: thymine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:go_curators]
+def: "The chemical reactions and pathways involving thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:curators]
 synonym: "thymine metabolism" EXACT []
 is_a: GO:0006206 ! pyrimidine nucleobase metabolic process
 
@@ -160187,7 +160652,7 @@
 id: GO:0019860
 name: uracil metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:curators]
 synonym: "uracil metabolism" EXACT []
 is_a: GO:0006206 ! pyrimidine nucleobase metabolic process
 
@@ -160246,7 +160711,7 @@
 id: GO:0019867
 name: outer membrane
 namespace: cellular_component
-def: "The external membrane of Gram-negative bacteria or certain organelles such as mitochondria and chloroplasts; freely permeable to most ions and metabolites." [GOC:go_curators]
+def: "The external membrane of Gram-negative bacteria or certain organelles such as mitochondria and chloroplasts; freely permeable to most ions and metabolites." [GOC:curators]
 subset: goslim_metagenomics
 is_a: GO:0016020 ! membrane
 
@@ -160264,6 +160729,8 @@
 name: potassium channel inhibitor activity
 namespace: molecular_function
 def: "Binds to and stops, prevents, or reduces the activity of a potassium channel." [GOC:mah]
+xref: Reactome:R-HSA-381644 "ER calcium channels open in response to activated PKA"
+xref: Reactome:R-HSA-381713 "Potassium voltage-gated channels close in reponse to PKA"
 is_a: GO:0008200 ! ion channel inhibitor activity
 is_a: GO:0015459 ! potassium channel regulator activity
 relationship: negatively_regulates GO:0005267 ! potassium channel activity
@@ -160294,7 +160761,7 @@
 id: GO:0019873
 name: obsolete tellurium sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "tellurium sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -160359,7 +160826,7 @@
 id: GO:0019878
 name: L-lysine biosynthetic process via aminoadipic acid
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine by the aminoadipic pathway." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine by the aminoadipic pathway." [GOC:curators]
 synonym: "lysine anabolism via aminoadipic acid" EXACT []
 synonym: "lysine biosynthesis, aminoadipic acid pathway" EXACT []
 synonym: "lysine biosynthesis, aminoadipic pathway" EXACT []
@@ -160386,7 +160853,7 @@
 id: GO:0019880
 name: obsolete bacteriocin susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "bacteriocin susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -160396,7 +160863,7 @@
 id: GO:0019881
 name: obsolete streptomycin susceptibility/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'susceptibility/resistance' implies a phenotype rather than a biological process.
 synonym: "streptomycin susceptibility/resistance" EXACT []
 is_obsolete: true
@@ -160498,7 +160965,7 @@
 id: GO:0019889
 name: pteridine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving pteridine, pyrazino(2,3-dipyrimidine), the parent structure of pterins and the pteroyl group." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving pteridine, pyrazino(2,3-dipyrimidine), the parent structure of pterins and the pteroyl group." [GOC:curators, ISBN:0198506732]
 synonym: "pteridine metabolism" EXACT []
 is_a: GO:0042558 ! pteridine-containing compound metabolic process
 
@@ -160608,7 +161075,7 @@
 id: GO:0019904
 name: protein domain specific binding
 namespace: molecular_function
-def: "Binding to a specific domain of a protein." [GOC:go_curators]
+def: "Binding to a specific domain of a protein." [GOC:curators]
 subset: goslim_chembl
 synonym: "protein domain-specific binding" EXACT []
 is_a: GO:0005515 ! protein binding
@@ -160621,12 +161088,9 @@
 alt_id: GO:0030349
 alt_id: GO:0050430
 alt_id: GO:0051535
-def: "Binding to a syntaxin, a SNAP receptor involved in the docking of synaptic vesicles at the presynaptic zone of a synapse." [ISBN:0198506732]
-synonym: "syntaxin-13 binding" NARROW []
-synonym: "syntaxin-2 binding" NARROW []
-synonym: "syntaxin-5 binding" NARROW []
-synonym: "syntaxin-6 binding" NARROW []
+def: "Binding to a syntaxin, a SNAP receptor involved in the docking of vesicles." [ISBN:0198506732]
 is_a: GO:0000149 ! SNARE binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31404" xsd:anyURI
 
 [Term]
 id: GO:0019907
@@ -160686,7 +161150,7 @@
 name: obsolete cyclin-dependent protein kinase activating kinase activity
 namespace: molecular_function
 alt_id: GO:0019913
-def: "OBSOLETE. Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein; increases the activity of a cyclin-dependent protein kinase (CDK)." [GOC:go_curators]
+def: "OBSOLETE. Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein; increases the activity of a cyclin-dependent protein kinase (CDK)." [GOC:curators]
 comment: The reason for obsoletion is that the term is s kinase+ substrate, but covers a number of unrelated kinases.
 synonym: "CAK" EXACT []
 synonym: "cdk-activating kinase activity" EXACT []
@@ -160765,7 +161229,7 @@
 id: GO:0019920
 name: obsolete peptidyl-1-thioglycine biosynthetic process, internal
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of internal peptidyl-1-thioglycine, which has an internal C=S bond, instead of an internal C=O bond, in the peptide." [GOC:go_curators, PMID:10660523, RESID:AA0265]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of internal peptidyl-1-thioglycine, which has an internal C=S bond, instead of an internal C=O bond, in the peptide." [GOC:curators, PMID:10660523, RESID:AA0265]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "peptidyl-1-thioglycine anabolism, internal" EXACT []
 synonym: "peptidyl-1-thioglycine formation, internal" EXACT []
@@ -160777,7 +161241,7 @@
 id: GO:0019921
 name: obsolete peptidyl-1-thioglycine biosynthetic process, carboxy-terminal
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of carboxy-terminal peptidyl-1-thioglycine, which has a carboxy-terminal thiocarboxy-C(=O)-SH bond." [GOC:go_curators, PMID:10660523, RESID:AA0265]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of carboxy-terminal peptidyl-1-thioglycine, which has a carboxy-terminal thiocarboxy-C(=O)-SH bond." [GOC:curators, PMID:10660523, RESID:AA0265]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "peptidyl-1-thioglycine anabolism, carboxy-terminal" EXACT []
 synonym: "peptidyl-1-thioglycine formation, carboxy-terminal" EXACT []
@@ -160982,7 +161446,7 @@
 id: GO:0019941
 name: modification-dependent protein catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent modification of the target protein." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent modification of the target protein." [GOC:curators]
 synonym: "modification-dependent protein breakdown" EXACT []
 synonym: "modification-dependent protein catabolism" EXACT []
 synonym: "modification-dependent protein degradation" EXACT []
@@ -161287,7 +161751,7 @@
 id: GO:0019987
 name: obsolete negative regulation of anti-apoptosis
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of anti-apoptosis." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of anti-apoptosis." [GOC:curators, GOC:mtg_apoptosis]
 comment: This term was made obsolete because it was ill-defined.
 synonym: "down regulation of anti-apoptosis" EXACT []
 synonym: "down-regulation of anti-apoptosis" EXACT []
@@ -161542,7 +162006,7 @@
 id: GO:0020027
 name: hemoglobin metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving hemoglobin, including its uptake and utilization." [GOC:go_curators, GOC:jl]
+def: "The chemical reactions and pathways involving hemoglobin, including its uptake and utilization." [GOC:curators, GOC:jl]
 synonym: "haemoglobin metabolic process" EXACT []
 synonym: "haemoglobin metabolism" EXACT []
 synonym: "hemoglobin metabolism" EXACT []
@@ -161629,6 +162093,7 @@
 namespace: molecular_function
 def: "Binding to a heme, a compound composed of iron complexed in a porphyrin (tetrapyrrole) ring." [GOC:ai]
 synonym: "haem binding" EXACT []
+xref: Reactome:R-HSA-9661419 "ALB binds extracellular heme"
 is_a: GO:0046906 ! tetrapyrrole binding
 
 [Term]
@@ -167050,7 +167515,7 @@
 id: GO:0022825
 name: obsolete copper-exporting ATPase activity
 namespace: molecular_function
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it was a placeholder during work on transporter terms, and was not defined.
 synonym: "copper-exporting ATPase activity" EXACT []
 is_obsolete: true
@@ -167248,8 +167713,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + glucose(out) = protein histidine + glucose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "glucose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022856
@@ -167257,8 +167723,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sorbitol(out) = protein histidine + sorbitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "sorbitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015576 ! sorbitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022857
@@ -167278,12 +167745,26 @@
 synonym: "uptake permease activity" RELATED []
 synonym: "uptake transmembrane transporter activity" RELATED []
 xref: Reactome:R-HSA-1236947 "Egress of internalized antigen to the cytosol via sec61"
+xref: Reactome:R-HSA-2142859 "Growing HA is extruded from the cell by ABCC5"
+xref: Reactome:R-HSA-2161500 "abacavir [extracellular] => abacavir [cytosol]"
 xref: Reactome:R-HSA-429036 "SLC2A9 transports Fru, Glc, urate"
 xref: Reactome:R-HSA-434650 "MATEs mediate extrusion of xenobiotics"
+xref: Reactome:R-HSA-549129 "OCT1 transports organic cations into hepatic cells"
+xref: Reactome:R-HSA-549279 "OCT2 mediates tubular uptake of organic cations in the kidney"
+xref: Reactome:R-HSA-549304 "OCT3 mediates renal clearance of organic cations"
+xref: Reactome:R-HSA-549322 "OCT1 transports organic cations out of hepatic cells"
+xref: Reactome:R-HSA-561041 "OAT1,2,3 transport organic anions with antiport of dicarboxylic acids"
+xref: Reactome:R-HSA-561054 "OCT2 mediates tubular secretion of organic cations in the kidney"
+xref: Reactome:R-HSA-561072 "OCT3 mediates renal uptake of organic cations"
 xref: Reactome:R-HSA-5638209 "Defective SLC2A9 does not transport Fru, Glc, urate"
 xref: Reactome:R-HSA-5671707 "Fe3+ dissociates from SLC22A17:LCN2:2,5DHBA"
 xref: Reactome:R-HSA-6784434 "An unknown carrier transports cytosolic glyoxylate to the peroxisome"
 xref: Reactome:R-HSA-6784436 "An unknown carrier transports mitochondrial glyoxylate to the cytosol"
+xref: Reactome:R-HSA-8954513 "NAASP translocates from mitochondrial matrix to cytosol"
+xref: Reactome:R-HSA-9794270 "SLCO1A2 transports Cipro(1+) into the cytosol"
+xref: Reactome:R-HSA-9794523 "SLC22A1 transports Cipro into renal cell"
+xref: Reactome:R-HSA-9794830 "SLC22A8 transports Cipro into renal cell"
+xref: Reactome:R-HSA-9795207 "SLC22A1 transports Cipro into hepatic cell"
 is_a: GO:0005215 ! transporter activity
 relationship: part_of GO:0055085 ! transmembrane transport
 
@@ -167337,17 +167818,23 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + lactose(out) = protein histidine + lactose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "lactose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015155 ! lactose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022870
 name: protein-N(PI)-phosphohistidine-mannose phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannose(out) = protein histidine + mannose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane according to the reaction: D-mannose(out) + N(pros)-phospho-L-histidyl-[protein] = D-mannose 6-phosphate(in) + L-histidyl-[protein]. This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49232]
 synonym: "mannose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.191
+xref: RHEA:49232
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015578 ! mannose transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.191
+property_value: skos:exactMatch RHEA:49232
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022871
@@ -167358,13 +167845,14 @@
 xref: EC:2.7.1.206
 xref: MetaCyc:RXN-15094
 xref: RHEA:49296
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
 property_value: skos:exactMatch EC:2.7.1.206
-property_value: skos:narrowMatch RHEA:49296
+property_value: skos:exactMatch RHEA:49296
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17289" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26941" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022872
@@ -167372,8 +167860,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannitol(out) = protein histidine + mannitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "mannitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015575 ! mannitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022873
@@ -167382,16 +167871,22 @@
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + maltose(out) = protein histidine + maltose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "maltose PTS transporter activity" EXACT []
 is_a: GO:0005363 ! maltose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022874
 name: protein-N(PI)-phosphohistidine-cellobiose phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + cellobiose(out) = protein histidine + cellobiose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane according to the reaction: D-cellobiose(out) + N(pros)-phospho-L-histidyl-[protein] = 6-phospho-beta-D-glucosyl-(1->4)-D-glucose(in) + L-histidyl-[protein]. This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49292]
 synonym: "cellobiose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.205
+xref: RHEA:49292
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0019191 ! cellobiose transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.205
+property_value: skos:exactMatch RHEA:49292
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022875
@@ -167399,8 +167894,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactitol(out) = protein histidine + galactitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "galactitol PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015577 ! galactitol transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022876
@@ -167408,8 +167904,9 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactosamine(out) = protein histidine + galactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "galactosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0019196 ! galactosamine transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022877
@@ -167418,7 +167915,8 @@
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + fructose(out) = protein histidine + fructose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "fructose PTS transporter activity" EXACT []
 is_a: GO:0005353 ! fructose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022878
@@ -167426,12 +167924,15 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sucrose(out) = protein histidine + sucrose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "sucrose PTS transporter activity" EXACT []
+xref: EC:2.7.1.211
 xref: MetaCyc:SUCROSEPHOSPHO-RXN
 xref: RHEA:49236
 is_a: GO:0008515 ! sucrose transmembrane transporter activity
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: skos:exactMatch EC:2.7.1.211
 property_value: skos:exactMatch RHEA:49236
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022879
@@ -167439,18 +167940,24 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + trehalose(out) = protein histidine + trehalose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "trehalose PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015574 ! trehalose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022880
 name: protein-N(PI)-phosphohistidine-N-acetylglucosamine phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylglucosamine(out) = protein histidine + N-acetylglucosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane by the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, RHEA:49240]
 synonym: "N-acetylglucosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+xref: EC:2.7.1.193
+xref: RHEA:49240
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015572 ! N-acetylglucosamine transmembrane transporter activity
+property_value: skos:exactMatch EC:2.7.1.193
+property_value: skos:exactMatch RHEA:49240
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27517" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022881
@@ -167458,18 +167965,19 @@
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylgalactosamine(out) = protein histidine + N-acetylgalactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
 synonym: "N-acetylgalactosamine PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
 is_a: GO:0015571 ! N-acetylgalactosamine transmembrane transporter activity
 is_a: GO:0055056 ! D-glucose transmembrane transporter activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022882
 name: protein-N(PI)-phosphohistidine-beta-glucoside phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
+def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, PMID:19251853]
 synonym: "beta-glucoside PTS transporter activity" EXACT []
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
-is_a: GO:0015573 ! beta-glucoside transmembrane transporter activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 
 [Term]
 id: GO:0022883
@@ -168279,6 +168787,7 @@
 synonym: "divalent metal ion transport" NARROW []
 synonym: "heavy metal ion transport" NARROW []
 synonym: "metal ion export" NARROW []
+xref: Reactome:R-HSA-425410 "Metal ion SLC transporters"
 is_a: GO:0006812 ! monoatomic cation transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23033" xsd:anyURI
 created_by: mah
@@ -168904,7 +169413,7 @@
 id: GO:0030068
 name: obsolete lytic viral life cycle
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because more specific terms were created.
 synonym: "lytic viral life cycle" EXACT []
 is_obsolete: true
@@ -169105,7 +169614,7 @@
 id: GO:0030092
 name: obsolete regulation of flagellum assembly
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the formation of a flagellum." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the formation of a flagellum." [GOC:curators]
 comment: This term was made obsolete because microtubule-based flagellum (GO:0009434) and motile cilium (GO:0031514), as referenced by this term, were determined to be equivalent and merged. When a merge for this term was considered, it was noticed that both eukaryotic and prokaryotic proteins are annotated to it because its meaning had been ambiguous so obsoletion was considered safest.
 synonym: "regulation of flagella assembly" RELATED [GOC:mah]
 synonym: "regulation of flagella biogenesis" RELATED []
@@ -169179,7 +169688,7 @@
 name: lymphocyte differentiation
 namespace: biological_process
 alt_id: GO:0046650
-def: "The process in which a relatively unspecialized precursor cell acquires specialized features of a lymphocyte. A lymphocyte is a leukocyte commonly found in the blood and lymph that has the characteristics of a large nucleus, a neutral staining cytoplasm, and prominent heterochromatin." [CL:0000542, GOC:go_curators]
+def: "The process in which a relatively unspecialized precursor cell acquires specialized features of a lymphocyte. A lymphocyte is a leukocyte commonly found in the blood and lymph that has the characteristics of a large nucleus, a neutral staining cytoplasm, and prominent heterochromatin." [CL:0000542, GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "lymphocyte cell differentiation" EXACT []
 synonym: "lymphocyte development" RELATED [GOC:add]
@@ -169199,7 +169708,7 @@
 id: GO:0030100
 name: regulation of endocytosis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of endocytosis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of endocytosis." [GOC:curators]
 is_a: GO:0051128 ! regulation of cellular component organization
 is_a: GO:0060627 ! regulation of vesicle-mediated transport
 intersection_of: GO:0065007 ! biological regulation
@@ -169559,7 +170068,7 @@
 id: GO:0030139
 name: endocytic vesicle
 namespace: cellular_component
-def: "A membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance. Endocytic vesicles fuse with early endosomes to deliver the cargo for further sorting." [GOC:go_curators, PMID:19696797]
+def: "A membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance. Endocytic vesicles fuse with early endosomes to deliver the cargo for further sorting." [GOC:curators, PMID:19696797]
 synonym: "endocytotic transport vesicle" EXACT []
 synonym: "endocytotic vesicle" EXACT []
 xref: NIF_Subcellular:sao1362520468
@@ -169627,6 +170136,7 @@
 synonym: "uridine diphosphoacetylglucosamine-alpha-mannoside beta1->6-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.155]
 xref: EC:2.4.1.155
 xref: MetaCyc:2.4.1.155-RXN
+xref: Reactome:R-HSA-8932427 "MGAT5B transfers GlcNAc from UDP-GlcNAc to GlcNAc-Man-DAG1"
 xref: Reactome:R-HSA-9696980 "Spike trimer glycoside chains get additional branches"
 xref: Reactome:R-HSA-975916 "Addition of GlcNAc to position 5 by MGAT5"
 xref: RHEA:16921
@@ -169870,6 +170380,7 @@
 subset: goslim_generic
 subset: goslim_pombe
 subset: goslim_prokaryote
+subset: goslim_yeast
 synonym: "cellular protein breakdown" EXACT []
 synonym: "cellular protein catabolic process" EXACT []
 synonym: "cellular protein catabolism" EXACT []
@@ -169897,7 +170408,7 @@
 id: GO:0030165
 name: PDZ domain binding
 namespace: molecular_function
-def: "Binding to a PDZ domain of a protein, a domain found in diverse signaling proteins." [GOC:go_curators, Pfam:PF00595]
+def: "Binding to a PDZ domain of a protein, a domain found in diverse signaling proteins." [GOC:curators, Pfam:PF00595]
 synonym: "DHR-domain binding" EXACT []
 synonym: "GLGF-domain binding" EXACT []
 is_a: GO:0019904 ! protein domain specific binding
@@ -169923,6 +170434,7 @@
 synonym: "proteoglycan breakdown" EXACT []
 synonym: "proteoglycan catabolism" EXACT []
 synonym: "proteoglycan degradation" EXACT []
+xref: Reactome:R-HSA-2024101 "CS/DS degradation"
 is_a: GO:0006029 ! proteoglycan metabolic process
 is_a: GO:0006516 ! glycoprotein catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -169979,7 +170491,7 @@
 id: GO:0030173
 name: obsolete integral component of Golgi membrane
 namespace: cellular_component
-def: "OBSOLETE. The component of the Golgi membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:go_curators]
+def: "OBSOLETE. The component of the Golgi membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:curators]
 comment: This term was obsoleted because it represents protein topology, not a cellular component.
 synonym: "Golgi integral membrane protein" RELATED []
 synonym: "integral to Golgi membrane" NARROW []
@@ -170031,7 +170543,7 @@
 name: positive regulation of Wnt signaling pathway
 namespace: biological_process
 alt_id: GO:0045811
-def: "Any process that activates or increases the frequency, rate or extent of Wnt signal transduction." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of Wnt signal transduction." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of frizzled signaling pathway" NARROW []
 synonym: "activation of Wnt receptor signaling pathway" NARROW []
 synonym: "positive regulation of frizzled signaling pathway" EXACT []
@@ -170058,7 +170570,7 @@
 name: negative regulation of Wnt signaling pathway
 namespace: biological_process
 alt_id: GO:0045810
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Wnt signaling pathway." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Wnt signaling pathway." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of frizzled signaling pathway" EXACT []
 synonym: "down regulation of Wnt receptor signaling pathway" EXACT []
 synonym: "down-regulation of frizzled signaling pathway" EXACT []
@@ -170346,6 +170858,7 @@
 synonym: "heparan sulphate proteoglycan metabolic process" EXACT []
 synonym: "heparan sulphate proteoglycan metabolism" EXACT []
 synonym: "heparin proteoglycan metabolic process" RELATED []
+xref: Reactome:R-HSA-1638091 "Heparan sulfate/heparin (HS-GAG) metabolism"
 is_a: GO:0006029 ! proteoglycan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29294" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -170380,7 +170893,6 @@
 synonym: "chondroitin sulfate metabolism" EXACT []
 synonym: "chondroitin sulphate metabolic process" EXACT []
 synonym: "chondroitin sulphate metabolism" EXACT []
-xref: Reactome:R-HSA-1793185 "Chondroitin sulfate/dermatan sulfate metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0050654
@@ -170412,7 +170924,6 @@
 synonym: "chondroitin sulfate synthesis" EXACT []
 synonym: "chondroitin sulphate biosynthesis" EXACT []
 synonym: "chondroitin sulphate biosynthetic process" EXACT []
-xref: Reactome:R-HSA-2022870 "Chondroitin sulfate biosynthesis"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
@@ -170453,7 +170964,6 @@
 synonym: "dermatan sulfate synthesis" EXACT []
 synonym: "dermatan sulphate biosynthesis" EXACT []
 synonym: "dermatan sulphate biosynthetic process" EXACT []
-xref: Reactome:R-HSA-2022923 "Dermatan sulfate biosynthesis"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28977" xsd:anyURI
 is_obsolete: true
@@ -170531,7 +171041,6 @@
 synonym: "hyaluronan biosynthesis" EXACT []
 synonym: "hyaluronan formation" EXACT []
 synonym: "hyaluronan synthesis" EXACT []
-xref: Reactome:R-HSA-2142850 "Hyaluronan biosynthesis and export"
 is_a: GO:0006024 ! glycosaminoglycan biosynthetic process
 is_a: GO:0030212 ! hyaluronan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29745" xsd:anyURI
@@ -170545,7 +171054,7 @@
 synonym: "hyaluronan catabolism" EXACT []
 synonym: "hyaluronan degradation" EXACT []
 xref: MetaCyc:PWY-7645
-xref: Reactome:R-HSA-2160916 "Hyaluronan uptake and degradation"
+xref: Reactome:R-HSA-2160916 "Hyaluronan degradation"
 is_a: GO:0006027 ! glycosaminoglycan catabolic process
 is_a: GO:0030212 ! hyaluronan metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27107" xsd:anyURI
@@ -170581,6 +171090,7 @@
 synonym: "T lymphocyte differentiation" EXACT []
 synonym: "T-cell differentiation" EXACT []
 synonym: "T-lymphocyte differentiation" EXACT []
+xref: Reactome:R-HSA-9945266 "Differentiation of T cells"
 is_a: GO:0030098 ! lymphocyte differentiation
 is_a: GO:0042110 ! T cell activation
 
@@ -171207,14 +171717,14 @@
 id: GO:0030274
 name: LIM domain binding
 namespace: molecular_function
-def: "Binding to a LIM domain (for Lin-11 Isl-1 Mec-3) of a protein, a domain with seven conserved cysteine residues and a histidine, that binds two zinc ions and acts as an interface for protein-protein interactions." [GOC:go_curators, Pfam:PF00412]
+def: "Binding to a LIM domain (for Lin-11 Isl-1 Mec-3) of a protein, a domain with seven conserved cysteine residues and a histidine, that binds two zinc ions and acts as an interface for protein-protein interactions." [GOC:curators, Pfam:PF00412]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
 id: GO:0030275
 name: LRR domain binding
 namespace: molecular_function
-def: "Binding to a LRR domain (leucine rich repeats) of a protein." [GOC:go_curators, Pfam:PF00560]
+def: "Binding to a LRR domain (leucine rich repeats) of a protein." [GOC:curators, Pfam:PF00560]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
@@ -171236,7 +171746,7 @@
 id: GO:0030278
 name: regulation of ossification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:curators]
 synonym: "regulation of bone biosynthesis" EXACT []
 synonym: "regulation of bone formation" EXACT []
 is_a: GO:0051239 ! regulation of multicellular organismal process
@@ -171248,7 +171758,7 @@
 id: GO:0030279
 name: negative regulation of ossification
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:curators]
 synonym: "down regulation of ossification" EXACT []
 synonym: "down-regulation of ossification" EXACT []
 synonym: "downregulation of ossification" EXACT []
@@ -171291,7 +171801,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: testosterone + NAD(P)+ = androst-4-ene-3,17-dione + NAD(P)H + H+." [EC:1.1.1.51]
 comment: This term was obsoleted because it is an unnecessary grouing class.
-xref: Reactome:R-HSA-193064 "HSD17B3-like proteins reducde ANDST to TEST"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28298" xsd:anyURI
 is_obsolete: true
 consider: GO:0047035
@@ -171311,7 +171820,7 @@
 id: GO:0030285
 name: obsolete integral component of synaptic vesicle membrane
 namespace: cellular_component
-def: "OBSOLETE. The component of the synaptic vesicle membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:dos, GOC:go_curators]
+def: "OBSOLETE. The component of the synaptic vesicle membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:curators, GOC:dos]
 comment: This term was obsoleted because it represents protein topology, not a cellular component.
 synonym: "integral to synaptic vesicle membrane" NARROW []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23424" xsd:anyURI
@@ -171504,7 +172013,7 @@
 id: GO:0030307
 name: positive regulation of cell growth
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate, extent or direction of cell growth." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate, extent or direction of cell growth." [GOC:curators]
 synonym: "activation of cell growth" NARROW []
 synonym: "stimulation of cell growth" NARROW []
 synonym: "up regulation of cell growth" EXACT []
@@ -171521,7 +172030,7 @@
 id: GO:0030308
 name: negative regulation of cell growth
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate, extent or direction of cell growth." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate, extent or direction of cell growth." [GOC:curators]
 synonym: "down regulation of cell growth" EXACT []
 synonym: "down-regulation of cell growth" EXACT []
 synonym: "downregulation of cell growth" EXACT []
@@ -171568,7 +172077,7 @@
 id: GO:0030312
 name: external encapsulating structure
 namespace: cellular_component
-def: "A structure that lies outside the plasma membrane and surrounds the entire cell or cells. This does not include the periplasmic space." [GOC:go_curators]
+def: "A structure that lies outside the plasma membrane and surrounds the entire cell or cells. This does not include the periplasmic space." [GOC:curators]
 comment: The outer membrane (of gram negative bacteria) or cell wall (of yeast or Gram positive bacteria) are defined as parts of this structure, see 'external encapsulating structure part'.
 subset: goslim_chembl
 subset: goslim_drosophila
@@ -171746,11 +172255,13 @@
 name: DNA damage response, signal transduction by p53 class mediator
 namespace: biological_process
 alt_id: GO:0006976
-def: "A cascade of processes induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage." [GOC:go_curators]
+def: "A cascade of processes induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage." [GOC:curators]
 synonym: "DNA damage response, activation of p53" RELATED []
 synonym: "p53 signaling pathway" RELATED [GOC:mah, GOC:vk]
 synonym: "p53-mediated DNA damage response" BROAD []
 synonym: "TP53 signaling pathway" RELATED [GOC:mah, GOC:vk]
+xref: Reactome:R-HSA-6791312 "TP53 Regulates Transcription of Cell Cycle Genes"
+xref: Reactome:R-HSA-69563 "p53-Dependent G1 DNA Damage Response"
 is_a: GO:0042770 ! signal transduction in response to DNA damage
 is_a: GO:0072331 ! signal transduction by p53 class mediator
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26003" xsd:anyURI
@@ -171775,7 +172286,7 @@
 id: GO:0030334
 name: regulation of cell migration
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cell migration." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of cell migration." [GOC:curators]
 is_a: GO:2000145 ! regulation of cell motility
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0016477 ! cell migration
@@ -171785,7 +172296,7 @@
 id: GO:0030335
 name: positive regulation of cell migration
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cell migration." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cell migration." [GOC:curators]
 synonym: "activation of cell migration" NARROW []
 synonym: "stimulation of cell migration" NARROW []
 synonym: "up regulation of cell migration" EXACT []
@@ -171801,7 +172312,7 @@
 id: GO:0030336
 name: negative regulation of cell migration
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell migration." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell migration." [GOC:curators]
 synonym: "down regulation of cell migration" EXACT []
 synonym: "down-regulation of cell migration" EXACT []
 synonym: "downregulation of cell migration" EXACT []
@@ -172576,26 +173087,28 @@
 id: GO:0030418
 name: nicotianamine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+def: "The chemical reactions and pathways resulting in the formation of nicotianamine." [GOC:curators, PMID:10069850]
 synonym: "nicotianamine anabolism" EXACT []
 synonym: "nicotianamine biosynthesis" EXACT []
 synonym: "nicotianamine formation" EXACT []
 synonym: "nicotianamine synthesis" EXACT []
+xref: MetaCyc:PWY-5957
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0042401 ! biogenic amine biosynthetic process
 is_a: GO:0072351 ! tricarboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 
 [Term]
 id: GO:0030419
-name: nicotianamine catabolic process
+name: obsolete nicotianamine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of nicotianamine, 2(S),3'2(S),3''(S)-N-(N-(3-amino-3-carboxypropyl)-3-amino-3-carboxypropyl)-azetidine-2-carboxylic acid." [GOC:mah, PMID:10069850]
+comment: This term was obsoleted because there is no evidence that a specific pathway for nicotianamine degradation exists.
 synonym: "nicotianamine breakdown" EXACT []
 synonym: "nicotianamine catabolism" EXACT []
 synonym: "nicotianamine degradation" EXACT []
-is_a: GO:0009063 ! amino acid catabolic process
-is_a: GO:0042402 ! biogenic amine catabolic process
-is_a: GO:0072352 ! tricarboxylic acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0030420
@@ -172698,7 +173211,7 @@
 synonym: "L-kynurenine hydrolase activity" RELATED [EC:3.7.1.3]
 xref: EC:3.7.1.3
 xref: MetaCyc:KYNURENINASE-RXN
-xref: Reactome:R-HSA-71217 "3-hydroxykynurenine + H2O => 3-hydroxyanthranilate + alanine"
+xref: Reactome:R-HSA-71217 "3-hydroxykynurenine + H2O => 3-hydroxyanthranilate + alanine + H+"
 xref: RHEA:16813
 xref: RHEA:25143
 is_a: GO:0016823 ! hydrolase activity, acting on carbon-carbon bonds, in ketonic substances
@@ -172899,7 +173412,7 @@
 id: GO:0030449
 name: regulation of complement activation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of complement activation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of complement activation." [GOC:curators]
 synonym: "regulation of complement cascade" EXACT [GOC:add]
 xref: Reactome:R-HSA-977606 "Regulation of Complement cascade"
 is_a: GO:0002697 ! regulation of immune effector process
@@ -172912,7 +173425,7 @@
 id: GO:0030450
 name: regulation of complement activation, classical pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the classical pathway of complement activation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the classical pathway of complement activation." [GOC:curators]
 synonym: "regulation of complement cascade, classical pathway" EXACT [GOC:add]
 is_a: GO:0002923 ! regulation of humoral immune response mediated by circulating immunoglobulin
 is_a: GO:0030449 ! regulation of complement activation
@@ -172924,7 +173437,7 @@
 id: GO:0030451
 name: regulation of complement activation, alternative pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the alternative pathway of complement activation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the alternative pathway of complement activation." [GOC:curators]
 synonym: "regulation of complement cascade, alternative pathway" EXACT [GOC:add]
 is_a: GO:0030449 ! regulation of complement activation
 is_a: GO:0045088 ! regulation of innate immune response
@@ -172996,7 +173509,7 @@
 id: GO:0030463
 name: obsolete cell aging (sensu Fungi)
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because it does not describe a biological process that is distinct from 'cell aging'.
 synonym: "cell aging (sensu Fungi)" EXACT []
 is_obsolete: true
@@ -173015,7 +173528,7 @@
 id: GO:0030465
 name: obsolete autophagic death (sensu Fungi)
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because a more correct term has been created.
 synonym: "autophagic death (sensu Fungi)" EXACT []
 is_obsolete: true
@@ -173234,7 +173747,7 @@
 id: GO:0030495
 name: bacteriochlorophyll catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of bacteriochlorophyll, any of the chlorophylls of photosynthetic bacteria. They differ structurally from the chlorophylls of higher plants." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of bacteriochlorophyll, any of the chlorophylls of photosynthetic bacteria. They differ structurally from the chlorophylls of higher plants." [GOC:curators]
 synonym: "bacteriochlorophyll breakdown" EXACT []
 synonym: "bacteriochlorophyll catabolism" EXACT []
 synonym: "bacteriochlorophyll degradation" EXACT []
@@ -173260,7 +173773,7 @@
 id: GO:0030500
 name: regulation of bone mineralization
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of bone mineralization." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of bone mineralization." [GOC:curators]
 is_a: GO:0030278 ! regulation of ossification
 is_a: GO:0070167 ! regulation of biomineral tissue development
 intersection_of: GO:0065007 ! biological regulation
@@ -173271,7 +173784,7 @@
 id: GO:0030501
 name: positive regulation of bone mineralization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of bone mineralization." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of bone mineralization." [GOC:curators]
 synonym: "activation of bone mineralization" NARROW []
 synonym: "stimulation of bone mineralization" NARROW []
 synonym: "up regulation of bone mineralization" EXACT []
@@ -173288,7 +173801,7 @@
 id: GO:0030502
 name: negative regulation of bone mineralization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of bone mineralization." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of bone mineralization." [GOC:curators]
 synonym: "down regulation of bone mineralization" EXACT []
 synonym: "down-regulation of bone mineralization" EXACT []
 synonym: "downregulation of bone mineralization" EXACT []
@@ -173307,7 +173820,6 @@
 def: "OBSOLETE. Enables the transfer of inorganic diphosphate across a membrane." [PMID:11326272]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
 synonym: "inorganic pyrophosphate transporter activity" EXACT []
-xref: Reactome:R-HSA-5226964 "ANKH transports PPi from cytosol to extracellular region"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0022857
@@ -173394,7 +173906,7 @@
 id: GO:0030511
 name: positive regulation of transforming growth factor beta receptor signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of TGF-beta receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of TGF-beta receptor signaling pathway activity." [GOC:curators]
 synonym: "activation of transforming growth factor beta receptor signaling pathway" NARROW []
 synonym: "positive regulation of TGF-beta receptor signaling pathway" EXACT []
 synonym: "positive regulation of TGFbeta receptor signaling pathway" EXACT []
@@ -173434,7 +173946,7 @@
 name: positive regulation of BMP signaling pathway
 namespace: biological_process
 alt_id: GO:0090098
-def: "Any process that activates or increases the frequency, rate or extent of BMP signaling pathway activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of BMP signaling pathway activity." [GOC:curators]
 synonym: "activation of BMP signaling pathway" NARROW []
 synonym: "positive regulation of BMP receptor signaling pathway" RELATED []
 synonym: "positive regulation of BMP signalling pathway" EXACT []
@@ -173460,7 +173972,7 @@
 name: negative regulation of BMP signaling pathway
 namespace: biological_process
 alt_id: GO:0090099
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the BMP signaling pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the BMP signaling pathway." [GOC:curators]
 synonym: "down regulation of BMP signaling pathway" EXACT []
 synonym: "down-regulation of BMP signaling pathway" EXACT []
 synonym: "downregulation of BMP signaling pathway" EXACT []
@@ -173492,7 +174004,7 @@
 id: GO:0030516
 name: regulation of axon extension
 namespace: biological_process
-def: "Any process that modulates the rate, direction or extent of axon extension." [GOC:go_curators]
+def: "Any process that modulates the rate, direction or extent of axon extension." [GOC:curators]
 is_a: GO:0048638 ! regulation of developmental growth
 is_a: GO:0061387 ! regulation of extent of cell growth
 intersection_of: GO:0065007 ! biological regulation
@@ -173663,7 +174175,7 @@
 id: GO:0030535
 name: obsolete adult feeding behavior (sensu Insecta)
 namespace: biological_process
-def: "OBSOLETE. Feeding behavior in a fully developed and mature organism, as described in insects." [GOC:go_curators, GOC:jid]
+def: "OBSOLETE. Feeding behavior in a fully developed and mature organism, as described in insects." [GOC:curators, GOC:jid]
 comment: This term was made obsolete because there is no clear difference between the sensu Insecta term and the generic term.
 synonym: "adult feeding behavior (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -174017,7 +174529,7 @@
 name: bile acid catabolic process
 namespace: biological_process
 alt_id: GO:0019613
-def: "The chemical reactions and pathways resulting in the breakdown of bile acids, any of a group of steroid carboxylic acids occurring in bile." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of bile acids, any of a group of steroid carboxylic acids occurring in bile." [GOC:curators]
 synonym: "bile acid 7alpha-dehydroxylation pathway" NARROW []
 synonym: "bile acid breakdown" EXACT []
 synonym: "bile acid catabolism" EXACT []
@@ -174082,7 +174594,7 @@
 id: GO:0030579
 name: obsolete ubiquitin-dependent SMAD protein catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of SMAD signaling proteins by ubiquitination and targeting to the proteasome." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of SMAD signaling proteins by ubiquitination and targeting to the proteasome." [GOC:curators]
 comment: The reason for obsoletion is that the substrate should be captured as the input.
 synonym: "ubiquitin-dependent SMAD protein breakdown" EXACT []
 synonym: "ubiquitin-dependent SMAD protein catabolism" EXACT []
@@ -174632,7 +175144,6 @@
 synonym: "D-alanine formation" EXACT []
 synonym: "D-alanine synthesis" EXACT []
 is_a: GO:0006523 ! alanine biosynthetic process
-is_a: GO:0046436 ! D-alanine metabolic process
 is_a: GO:0046437 ! D-amino acid biosynthetic process
 
 [Term]
@@ -175056,6 +175567,12 @@
 synonym: "protein recruiting activity" RELATED []
 synonym: "protein-containing complex scaffold activity" BROAD []
 synonym: "protein-protein adaptor" NARROW []
+xref: Reactome:R-HSA-3780997 "PPP1R3C binds to glycogen:GYG2:GYS2"
+xref: Reactome:R-HSA-3781001 "EPM2A dimer binds PPP1R3C:phosphoglycogen-GYG1 complex"
+xref: Reactome:R-HSA-3781021 "EPM2A dimer binds PPP1R3C:phosphoglycogen-GYG2 complex"
+xref: Reactome:R-HSA-3781023 "PPP1R3C binds to glycogen:GYG1:GYS1"
+xref: Reactome:R-HSA-8871366 "p-Y389,400-RUFY1 binds RAB4A:GTP, RAB5:GTP, RAB14:GTP"
+xref: Reactome:R-HSA-8871376 "p-Y389,400-RUFY1 binds PI3P"
 is_a: GO:0060090 ! molecular adaptor activity
 relationship: has_part GO:0005515 ! protein binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18655" xsd:anyURI
@@ -175504,7 +176021,7 @@
 id: GO:0030716
 name: oocyte fate determination
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into an oocyte cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into an oocyte cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 synonym: "oocyte cell fate determination" EXACT []
 is_a: GO:0001709 ! cell fate determination
 
@@ -175630,7 +176147,6 @@
 xref: Reactome:R-HSA-5694494 "AACS ligates CoA-SH to ACA, forming ACA-CoA"
 xref: RHEA:16117
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.16
 property_value: skos:exactMatch RHEA:16117
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -175777,7 +176293,7 @@
 id: GO:0030738
 name: tyramine N-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tyramine = N-methyltyramine + S-adenosyl-L-homocysteine + H+." [EC:2.1.1.27, RHEA:14865]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + tyramine = N-methyltyramine + S-adenosyl-L-homocysteine + H+." [RHEA:14865]
 synonym: "DIB O-methyltransferase (3,5-diiodo-4-hydroxy-benzoic acid)" RELATED [EC:2.1.1.27]
 synonym: "S-adenosyl-L-methionine:tyramine N-methyltransferase activity" RELATED [EC:2.1.1.27]
 synonym: "S-adenosyl-methionine:tyramine N-methyltransferase activity" RELATED [EC:2.1.1.27]
@@ -175852,15 +176368,16 @@
 id: GO:0030742
 name: GTP-dependent protein binding
 namespace: molecular_function
-def: "Binding to a protein or protein complex when at least one of the interacting partners is in the GTP-bound state." [GOC:go_curators, GOC:krc]
+def: "Binding to a protein or protein complex when at least one of the interacting partners is in the GTP-bound state." [GOC:curators, GOC:krc]
 comment: This term may be used to annotate both partners in a GTP-dependent binding interaction, both the GTP-bound protein and the protein(s) which interact with it.
 is_a: GO:0005515 ! protein binding
 
 [Term]
 id: GO:0030743
-name: 23S rRNA (adenosine(1067)-2'-O)-methyltransferase activity
+name: 23S rRNA (adenosine(1067)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: adenosine(1067) in 23S rRNA + S-adenosyl-L-methionine = 2'-O-methyladenosine(1067) in 23S rRNA + H+ + S-adenosyl-L-homocysteine." [EC:2.1.1.230]
+synonym: "23S rRNA (adenosine(1067)-2'-O)-methyltransferase activity" EXACT []
 synonym: "ribosomal ribonucleate adenosine 2'-methyltransferase activity" RELATED [EC:2.1.1.230]
 synonym: "RNA-pentose methylase activity" RELATED []
 synonym: "rRNA adenosine 2'-methylase activity" RELATED [EC:2.1.1.230]
@@ -176329,7 +176846,7 @@
 id: GO:0030772
 name: tryptophan 2-C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + L-tryptophan = S-adenosyl-L-homocysteine + L-2-methyltryptophan + H+." [EC:2.1.1.106, RHEA:17321]
+def: "Catalysis of the reaction: L-tryptophan + S-adenosyl-L-methionine = 2-methyl-L-tryptophan + S-adenosyl-L-homocysteine + H+." [RHEA:17321]
 synonym: "S-adenosyl-L-methionine:L-tryptophan 2-C-methyltransferase activity" RELATED [EC:2.1.1.106]
 synonym: "S-adenosylmethionine:tryptophan 2-methyltransferase activity" RELATED [EC:2.1.1.106]
 synonym: "tryptophan 2-methyltransferase activity" RELATED [EC:2.1.1.106]
@@ -179376,12 +179893,12 @@
 synonym: "negative regulation of septation initiation network" EXACT []
 synonym: "negative regulation of septation initiation signaling cascade" RELATED [GOC:signaling]
 synonym: "negative regulation of septation initiation signalling" EXACT []
-is_a: GO:0010974 ! negative regulation of division septum assembly
 is_a: GO:0031029 ! regulation of septation initiation signaling
 is_a: GO:0051058 ! negative regulation of small GTPase mediated signal transduction
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0031028 ! septation initiation signaling
 relationship: negatively_regulates GO:0031028 ! septation initiation signaling
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31584" xsd:anyURI
 
 [Term]
 id: GO:0031031
@@ -179396,7 +179913,6 @@
 synonym: "up regulation of septation initiation signaling" EXACT []
 synonym: "up-regulation of septation initiation signaling" EXACT []
 synonym: "upregulation of septation initiation signaling" EXACT []
-is_a: GO:0010973 ! positive regulation of division septum assembly
 is_a: GO:0031029 ! regulation of septation initiation signaling
 is_a: GO:0051057 ! positive regulation of small GTPase mediated signal transduction
 intersection_of: GO:0065007 ! biological regulation
@@ -180003,12 +180519,14 @@
 name: organelle membrane
 namespace: cellular_component
 def: "A membrane that is one of the two lipid bilayers of an organelle envelope or the outermost membrane of single membrane bound organelle." [GOC:dos, GOC:mah]
+subset: gocheck_do_not_annotate
 synonym: "intracellular membrane" RELATED [NIF_Subcellular:sao830981606]
 xref: NIF_Subcellular:sao830981606
 is_a: GO:0016020 ! membrane
 intersection_of: GO:0016020 ! membrane
 intersection_of: part_of GO:0043227 ! membrane-bounded organelle
 relationship: part_of GO:0043227 ! membrane-bounded organelle
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31446" xsd:anyURI
 
 [Term]
 id: GO:0031091
@@ -181066,6 +181584,8 @@
 name: phosphatidylcholine binding
 namespace: molecular_function
 def: "Binding to a phosphatidylcholine, a glycophospholipid in which a phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:mah, ISBN:0198506732]
+xref: Reactome:R-HSA-8873794 "PCTP binds PC"
+xref: Reactome:R-HSA-8873923 "STARD10 binds LPCAT1 and PC"
 is_a: GO:0005543 ! phospholipid binding
 is_a: GO:0043169 ! cation binding
 is_a: GO:0050997 ! quaternary ammonium group binding
@@ -181376,24 +181896,26 @@
 id: GO:0031240
 name: external side of cell outer membrane
 namespace: cellular_component
-def: "The side of the outer membrane that is opposite to the side that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
+def: "The leaflet of the outer membrane that is opposite to the side that faces the periplasm of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mlg, GOC:mtg_sensu]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of cell outer membrane" EXACT [GOC:ab]
 synonym: "external side of outer membrane" RELATED []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009279 ! cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0031241
 name: periplasmic side of cell outer membrane
 namespace: cellular_component
-def: "The side (leaflet) of the outer membrane that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
+def: "The leaflet of a outer cell membrane that faces the periplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mlg, GOC:mtg_sensu]
 comment: In GO, 'internal side' still refers to part of the membrane and does not refer to components beyond (inside of) the membrane.
 synonym: "internal leaflet of cell outer membrane" EXACT [GOC:ab]
 synonym: "internal side of cell outer membrane" EXACT []
 synonym: "internal side of outer membrane" EXACT []
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009279 ! cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0031242
@@ -182004,7 +182526,7 @@
 synonym: "T-cell co-stimulation" EXACT []
 synonym: "T-cell costimulation" EXACT []
 synonym: "T-lymphocyte costimulation" EXACT []
-xref: Reactome:R-HSA-388841 "Costimulation by the CD28 family"
+xref: Reactome:R-HSA-388841 "Regulation of T cell activation by CD28 family"
 is_a: GO:0031294 ! lymphocyte costimulation
 is_a: GO:0050870 ! positive regulation of T cell activation
 
@@ -182055,9 +182577,10 @@
 
 [Term]
 id: GO:0031299
-name: taurine-pyruvate aminotransferase activity
+name: taurine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyruvate + taurine = L-alanine + sulfoacetaldehyde." [PMID:11082195, RHEA:10420]
+def: "Catalysis of the reaction: taurine + pyruvate = sulfoacetaldehyde + L-alanine." [PMID:11082195, RHEA:10420]
+synonym: "taurine-pyruvate aminotransferase activity" EXACT []
 synonym: "taurine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.77]
 synonym: "Tpa" RELATED [EC:2.6.1.77]
 xref: EC:2.6.1.77
@@ -182068,6 +182591,7 @@
 property_value: skos:exactMatch EC:2.6.1.77
 property_value: skos:exactMatch RHEA:10420
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0031300
@@ -182370,7 +182894,6 @@
 synonym: "up regulation of cellular metabolic process" EXACT []
 synonym: "up-regulation of cellular metabolic process" EXACT []
 synonym: "upregulation of cellular metabolic process" EXACT []
-xref: Reactome:R-HSA-163765 "ChREBP activates metabolic gene expression"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27189" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
@@ -183251,7 +183774,6 @@
 name: regulation of protein modification process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the covalent alteration of one or more amino acid residues within a protein." [GOC:mah, GOC:tb]
-subset: goslim_yeast
 is_a: GO:0051246 ! regulation of protein metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0036211 ! protein modification process
@@ -184421,7 +184943,6 @@
 synonym: "chromatin silencing at pericentric region" RELATED [GOC:mah]
 synonym: "heterochromatic silencing at centromere" EXACT []
 synonym: "pericentric heterochromatin assembly" EXACT []
-is_a: GO:0140462 ! pericentric heterochromatin organization
 is_a: GO:0140719 ! constitutive heterochromatin formation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18954" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19112" xsd:anyURI
@@ -184842,6 +185363,7 @@
 xref: Reactome:R-HSA-1234177 "Cytosolic PHD2,3 hydroxylates proline residues on HIF1A"
 xref: Reactome:R-HSA-1234179 "Cytosolic PHD2,3 hydroxylates proline residues on EPAS1 (HIF2A)"
 xref: Reactome:R-HSA-1234181 "Nuclear PHD1,3 hydroxylates proline residues on HIF1A"
+xref: Reactome:R-HSA-9918779 "Proline hydroxylases hydroxylate Polyprotein"
 xref: RHEA:63484
 is_a: GO:0031543 ! peptidyl-proline dioxygenase activity
 property_value: skos:exactMatch RHEA:63484
@@ -189029,7 +189551,7 @@
 id: GO:0031992
 name: energy transducer activity
 namespace: molecular_function
-def: "The biological transducer activity that accepts energy and converts it to another form, often by transfer to another molecule within the cell." [GOC:go_curators]
+def: "The biological transducer activity that accepts energy and converts it to another form, often by transfer to another molecule within the cell." [GOC:curators]
 subset: goslim_pir
 synonym: "light harvesting activity" RELATED []
 synonym: "photon capture" RELATED []
@@ -189384,7 +189906,9 @@
 synonym: "up regulation of insulin secretion" EXACT []
 synonym: "up-regulation of insulin secretion" EXACT []
 synonym: "upregulation of insulin secretion" EXACT []
+xref: Reactome:R-HSA-381676 "Glucagon-like Peptide-1 (GLP1) regulates insulin secretion"
 xref: Reactome:R-HSA-399997 "Acetylcholine regulates insulin secretion"
+xref: Reactome:R-HSA-400451 "Free fatty acids regulate insulin secretion"
 is_a: GO:0050714 ! positive regulation of protein secretion
 is_a: GO:0050796 ! regulation of insulin secretion
 is_a: GO:0090277 ! positive regulation of peptide hormone secretion
@@ -189644,6 +190168,7 @@
 name: bile acid binding
 namespace: molecular_function
 def: "Binding to a bile acid, a steroid carboxylic acids occurring in bile." [GOC:rph]
+xref: Reactome:R-HSA-8873850 "STARD5 binds DCA, LCA"
 is_a: GO:0033293 ! monocarboxylic acid binding
 
 [Term]
@@ -191488,17 +192013,17 @@
 
 [Term]
 id: GO:0032220
-name: plasma membrane fusion involved in cytogamy
+name: obsolete plasma membrane fusion involved in cytogamy
 namespace: biological_process
 alt_id: GO:0070872
-def: "The joining of two or more lipid bilayer membranes that surround cells, that contributes to cytogamy." [GOC:mah, PMID:29134248]
+def: "OBSOLETE. The joining of two or more lipid bilayer membranes that surround cells, that contributes to cytogamy." [GOC:mah, PMID:29134248]
+comment: precomposed term, represents a GO-CAM model.
 synonym: "plasma membrane fusion during cytogamy" RELATED [GOC:dph, GOC:tb]
 synonym: "plasma membrane organization involved in conjugation with cellular fusion" EXACT []
-is_a: GO:0045026 ! plasma membrane fusion
-intersection_of: GO:0045026 ! plasma membrane fusion
-intersection_of: part_of GO:0000755 ! cytogamy
-relationship: part_of GO:0000755 ! cytogamy
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16258" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
+is_obsolete: true
+consider: GO:0140253
 created_by: mah
 creation_date: 2009-08-20T02:24:56Z
 
@@ -191989,10 +192514,12 @@
 name: methylation
 namespace: biological_process
 def: "The process in which a methyl group is covalently attached to a molecule." [GOC:mah]
+subset: gocheck_obsoletion_candidate
 subset: goslim_chembl
 xref: Reactome:R-HSA-156581 "Methylation"
 xref: Wikipedia:Methylation
 is_a: GO:0008152 ! metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27423" xsd:anyURI
 
 [Term]
 id: GO:0032260
@@ -192054,6 +192581,7 @@
 def: "Binding to phosphatidylinositol-3-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' position." [GOC:bf, PMID:10209156, PMID:11395417, PMID:11557775]
 synonym: "phosphatidylinositol 3-phosphate binding" EXACT []
 synonym: "PtdIns-3-P binding" EXACT []
+xref: Reactome:R-HSA-8870489 "PLEKHA4,(5,6) bind PI3P"
 is_a: GO:1901981 ! phosphatidylinositol phosphate binding
 
 [Term]
@@ -194344,7 +194872,7 @@
 id: GO:0032473
 name: cytoplasmic side of mitochondrial outer membrane
 namespace: cellular_component
-def: "The external (cytoplasmic) face of the mitochondrial outer membrane." [GOC:mah]
+def: "The leaflet of the mitochondrial outer membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:mah]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "cytosolic side of mitochondrial outer membrane" EXACT []
 synonym: "external leaflet of mitochondrial outer membrane" EXACT [GOC:ab]
@@ -194352,6 +194880,7 @@
 synonym: "external side of mitochondrial outer membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005741 ! mitochondrial outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0032474
@@ -195038,7 +195567,7 @@
 id: GO:0032543
 name: mitochondrial translation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a protein in a mitochondrion. This is a ribosome-mediated process in which the information in messenger RNA (mRNA) is used to specify the sequence of amino acids in the protein; the mitochondrion has its own ribosomes and transfer RNAs, and uses a genetic code that differs from the nuclear code." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of a protein in a mitochondrion. This is a ribosome-mediated process in which the information in messenger RNA (mRNA) is used to specify the sequence of amino acids in the protein; the mitochondrion has its own ribosomes and transfer RNAs, and uses a genetic code that differs from the nuclear code." [GOC:curators]
 subset: goslim_yeast
 synonym: "mitochondrial protein anabolism" EXACT []
 synonym: "mitochondrial protein biosynthesis" EXACT []
@@ -195057,7 +195586,7 @@
 id: GO:0032544
 name: plastid translation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a protein in a plastid. This is a ribosome-mediated process in which the information in messenger RNA (mRNA) is used to specify the sequence of amino acids in the protein; the plastid has its own ribosomes and transfer RNAs, and uses a genetic code that differs from the nuclear code." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of a protein in a plastid. This is a ribosome-mediated process in which the information in messenger RNA (mRNA) is used to specify the sequence of amino acids in the protein; the plastid has its own ribosomes and transfer RNAs, and uses a genetic code that differs from the nuclear code." [GOC:curators]
 synonym: "plastid protein anabolism" EXACT []
 synonym: "plastid protein biosynthesis" EXACT []
 synonym: "plastid protein formation" EXACT []
@@ -198701,6 +199230,7 @@
 name: lead ion binding
 namespace: molecular_function
 def: "Binding to lead (Pb) ions." [GOC:mah]
+xref: Reactome:R-HSA-190141 "ALAD binds to Pb2+"
 is_a: GO:0046872 ! metal ion binding
 
 [Term]
@@ -199397,64 +199927,34 @@
 
 [Term]
 id: GO:0032847
-name: regulation of cellular pH reduction
+name: obsolete regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "regulation of cell pH reduction" EXACT []
-synonym: "regulation of cellular acidification" EXACT []
-synonym: "regulation of intracellular acidification" RELATED []
-synonym: "regulation of intracellular pH reduction" EXACT []
-synonym: "regulation of reduction of cellular pH" EXACT []
-synonym: "regulation of reduction of pH in cell" EXACT []
-is_a: GO:0050789 ! regulation of biological process
-is_a: GO:0051453 ! regulation of intracellular pH
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0051452 ! intracellular pH reduction
-relationship: regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that modulates the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0051452
+consider: GO:0051454
 
 [Term]
 id: GO:0032848
-name: negative regulation of cellular pH reduction
+name: obsolete negative regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "down regulation of cellular pH reduction" EXACT []
-synonym: "down-regulation of cellular pH reduction" EXACT []
-synonym: "downregulation of cellular pH reduction" EXACT []
-synonym: "inhibition of cellular pH reduction" NARROW []
-synonym: "negative regulation of cell pH reduction" EXACT []
-synonym: "negative regulation of cellular acidification" EXACT []
-synonym: "negative regulation of intracellular acidification" RELATED []
-synonym: "negative regulation of intracellular pH reduction" EXACT []
-synonym: "negative regulation of reduction of cellular pH" EXACT []
-synonym: "negative regulation of reduction of pH in cell" EXACT []
-is_a: GO:0032847 ! regulation of cellular pH reduction
-is_a: GO:0048519 ! negative regulation of biological process
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: negatively_regulates GO:0051452 ! intracellular pH reduction
-relationship: negatively_regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0051454
 
 [Term]
 id: GO:0032849
-name: positive regulation of cellular pH reduction
+name: obsolete positive regulation of cellular pH reduction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
-synonym: "activation of cellular pH reduction" NARROW []
-synonym: "positive regulation of cell pH reduction" EXACT []
-synonym: "positive regulation of cellular acidification" EXACT []
-synonym: "positive regulation of intracellular acidification" RELATED []
-synonym: "positive regulation of intracellular pH reduction" EXACT []
-synonym: "positive regulation of reduction of cellular pH" EXACT []
-synonym: "positive regulation of reduction of pH in cell" EXACT []
-synonym: "stimulation of cellular pH reduction" NARROW []
-synonym: "up regulation of cellular pH reduction" EXACT []
-synonym: "up-regulation of cellular pH reduction" EXACT []
-synonym: "upregulation of cellular pH reduction" EXACT []
-is_a: GO:0032847 ! regulation of cellular pH reduction
-is_a: GO:0048518 ! positive regulation of biological process
-is_a: GO:0051452 ! intracellular pH reduction
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0051452 ! intracellular pH reduction
-relationship: positively_regulates GO:0051452 ! intracellular pH reduction
+def: "OBSOLETE. Any process that activates or increases the frequency, rate, or extent of a process that reduces the internal pH of a cell." [GOC:mah]
+comment: This term was obsoleted because it represents 'regulation of regulation', which is redundant with direct pH regulation terms.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0051452
 
 [Term]
 id: GO:0032865
@@ -200892,7 +201392,8 @@
 xref: Reactome:R-HSA-1307803 "TIMM22 inserts proteins into inner membrane"
 xref: Reactome:R-HSA-9902096 "COX18 inserts nascent MT-CO2 in COX20:TMEM77"
 xref: Reactome:R-HSA-9906955 "MT-ND4 is translated"
-is_a: GO:0140597 ! protein carrier chaperone
+xref: Reactome:R-HSA-9926981 "Bam complex inserts Hbp into outer membrane"
+is_a: GO:0140597 ! protein carrier activity
 relationship: part_of GO:0090150 ! establishment of protein localization to membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/16976" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20414" xsd:anyURI
@@ -202130,13 +202631,14 @@
 
 [Term]
 id: GO:0033094
-name: putrescine--2-oxoglutarate transaminase activity
+name: putrescine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H2O. The enzymatic part of the reaction produces 4-aminobutanal that spontaneously cyclizes to form 1-pyrroline." [EC:2.6.1.82, GOC:mlg, RHEA:12268]
+def: "Catalysis of the reaction: putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H2O. The enzymatic part of the reaction produces 4-aminobutanal that spontaneously cyclizes to form 1-pyrroline." [EC:2.6.1.82, RHEA:12268]
 synonym: "butane-1,4-diamine:2-oxoglutarate aminotransferase activity" RELATED []
 synonym: "PAT activity" RELATED []
 synonym: "putrescine aminotransferase activity" BROAD [EC:2.6.1.82]
 synonym: "putrescine transaminase activity" BROAD []
+synonym: "putrescine--2-oxoglutarate transaminase activity" EXACT []
 synonym: "putrescine-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.82]
 synonym: "putrescine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.82]
 synonym: "putrescine:alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.82]
@@ -202149,6 +202651,7 @@
 property_value: skos:exactMatch RHEA:12268
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25992" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033095
@@ -203164,7 +203667,7 @@
 id: GO:0033196
 name: tryparedoxin peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: tryparedoxin + hydrogen peroxide = tryparedoxin disulfide + H2O." [GOC:mah, PMID:32388269]
+def: "Catalysis of the reaction: tryparedoxin + H2O2 = tryparedoxin disulfide + H2O." [GOC:mah, PMID:32388269]
 synonym: "TXNPx activity" EXACT []
 xref: MetaCyc:1.11.1.15-RXN
 is_a: GO:0004601 ! peroxidase activity
@@ -204258,7 +204761,7 @@
 id: GO:0033300
 name: dehydroascorbic acid transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of dehydroascorbate, 5-(1,2-dihydroxyethyl)furan-2,3,4(5H)-trione, from one side of a membrane to the other." [GOC:go_curators]
+def: "Enables the transfer of dehydroascorbate, 5-(1,2-dihydroxyethyl)furan-2,3,4(5H)-trione, from one side of a membrane to the other." [GOC:curators]
 synonym: "dehydroascorbate transporter activity" EXACT []
 synonym: "dehydroascorbic acid transporter activity" RELATED []
 xref: Reactome:R-HSA-198818 "SLC2A1,3 transports DeHA from extracellular region to cytosol"
@@ -204810,12 +205313,13 @@
 id: GO:0033353
 name: S-adenosylmethionine cycle
 namespace: biological_process
-def: "A cyclic series of interconversions involving S-adenosylmethionine, S-adenosyl-L-homocysteine, L-cysteine, and L-methionine. Couples utilization of the methyl group of SAM with recycling of the homocysteinyl group and regeneration of methionine." [GOC:mah, MetaCyc:PWY-5041]
+def: "A cyclic series of interconversions involving S-adenosylmethionine (SAM), S-adenosyl-L-homocysteine, L-homocysteine, and L-methionine. Couples utilization of the methyl group of SAM with recycling of the homocysteinyl group and regeneration of L-methionine." [MetaCyc:PWY-5041, PMID:31950558, PMID:39394448]
 synonym: "activated methyl cycle" EXACT []
 synonym: "SAM cycle" EXACT []
 xref: MetaCyc:PWY-5041
 is_a: GO:0046500 ! S-adenosylmethionine metabolic process
 relationship: has_part GO:0008898 ! S-adenosylmethionine-homocysteine S-methyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
 
 [Term]
 id: GO:0033354
@@ -204829,7 +205333,7 @@
 id: GO:0033355
 name: ascorbate glutathione cycle
 namespace: biological_process
-def: "A cyclic series of interconversions involving L-ascorbate and glutathione that scavenges hydrogen peroxide and reduces it to water, with concomitant oxidation of NADPH." [GOC:mah, MetaCyc:PWY-2261]
+def: "A cyclic series of interconversions involving L-ascorbate and glutathione that scavenges H2O2 and reduces it to water, with concomitant oxidation of NADPH." [GOC:mah, MetaCyc:PWY-2261]
 synonym: "hydrogen peroxide detoxification" RELATED []
 xref: MetaCyc:PWY-2261
 is_a: GO:0006749 ! glutathione metabolic process
@@ -204877,13 +205381,12 @@
 id: GO:0033359
 name: L-lysine biosynthetic process via diaminopimelate and N-succinyl-2-amino-6-ketopimelate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and N-succinyl-2-amino-6-ketopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah, MetaCyc:DAPLYSINESYN-PWY]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and N-succinyl-2-amino-6-ketopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah]
 synonym: "lysine anabolism via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine biosynthetic process via diaminopimelic acid and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine formation via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
 synonym: "lysine synthesis via diaminopimelate and N-succinyl-2-amino-6-ketopimelate" EXACT []
-xref: MetaCyc:DAPLYSINESYN-PWY
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28523" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
@@ -204892,13 +205395,12 @@
 id: GO:0033360
 name: L-lysine biosynthetic process via diaminopimelate and L-2-acetamido-6-oxoheptanedioate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and L-2-acetamido-6-oxoheptanedioate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah, MetaCyc:PWY-2941]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediates diaminopimelate and L-2-acetamido-6-oxoheptanedioate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in four enzymatic steps." [GOC:mah]
 synonym: "lysine anabolism via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine biosynthetic process via diaminopimelic acid and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine formation via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
 synonym: "lysine synthesis via diaminopimelate and L-2-acetamido-6-oxoheptanedioate" EXACT []
-xref: MetaCyc:PWY-2941
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -204906,12 +205408,11 @@
 id: GO:0033361
 name: L-lysine biosynthetic process via diaminopimelate, dehydrogenase pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in a single enzymatic step." [GOC:mah, GOC:pr, MetaCyc:PWY-2942]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in a single enzymatic step." [GOC:mah, GOC:pr]
 synonym: "lysine anabolism via diaminopimelate, dehydrogenase pathway" EXACT []
 synonym: "lysine biosynthesis via diaminopimelic acid, dehydrogenase pathway" EXACT []
 synonym: "lysine formation via diaminopimelate, dehydrogenase pathway" EXACT []
 synonym: "lysine synthesis via diaminopimelate, dehydrogenase pathway" EXACT []
-xref: MetaCyc:PWY-2942
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 relationship: has_part GO:0047850 ! diaminopimelate dehydrogenase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
@@ -204920,15 +205421,14 @@
 id: GO:0033362
 name: L-lysine biosynthetic process via diaminopimelate, diaminopimelate-aminotransferase pathway
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in two enzymatic steps." [GOC:mah, GOC:pr, MetaCyc:PWY-5097]
+def: "The chemical reactions and pathways resulting in the formation of lysine, via the intermediate diaminopimelate; in this pathway tetrahydrodipicolinate is converted to meso-diaminopimelate in two enzymatic steps." [GOC:mah, GOC:pr]
 synonym: "lysine anabolism via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine biosynthesis via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine formation via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
 synonym: "lysine synthesis via diaminopimelate, diaminopimelate-aminotransferase pathway" EXACT []
-xref: MetaCyc:PWY-5097
 is_a: GO:0009089 ! L-lysine biosynthetic process via diaminopimelate
 relationship: has_part GO:0008837 ! diaminopimelate epimerase activity
-relationship: has_part GO:0010285 ! L,L-diaminopimelate aminotransferase activity
+relationship: has_part GO:0010285 ! L,L-diaminopimelate:2-oxoglutarate transaminase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
 [Term]
@@ -205199,14 +205699,17 @@
 synonym: "geranylgeranyl diphosphate biosynthesis" EXACT []
 synonym: "geranylgeranyl diphosphate formation" EXACT []
 synonym: "geranylgeranyl diphosphate synthesis" EXACT []
+xref: MetaCyc:PWY-5120
 xref: MetaCyc:PWY-5121
 xref: MetaCyc:PWY-5910
 is_a: GO:0008654 ! phospholipid biosynthetic process
 is_a: GO:0016114 ! terpenoid biosynthetic process
 is_a: GO:0033385 ! geranylgeranyl diphosphate metabolic process
+property_value: skos:narrowMatch MetaCyc:PWY-5120
 property_value: skos:narrowMatch MetaCyc:PWY-5121
 property_value: skos:narrowMatch MetaCyc:PWY-5910
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0033387
@@ -205303,7 +205806,6 @@
 synonym: "beta-alanine biosynthesis via 1,3 diaminopropane" EXACT []
 synonym: "beta-alanine formation via 1,3 diaminopropane" EXACT []
 synonym: "beta-alanine synthesis via 1,3 diaminopropane" EXACT []
-xref: MetaCyc:PWY-3981
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -205315,7 +205817,6 @@
 synonym: "beta-alanine biosynthesis via 3-hydroxypropionate" EXACT []
 synonym: "beta-alanine formation via 3-hydroxypropionate" EXACT []
 synonym: "beta-alanine synthesis via 3-hydroxypropionate" EXACT []
-xref: MetaCyc:PWY-3941
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -205326,7 +205827,6 @@
 synonym: "beta-alanine anabolism via 3-ureidopropionate" EXACT []
 synonym: "beta-alanine formation via 3-ureidopropionate" EXACT []
 synonym: "beta-alanine synthesis via 3-ureidopropionate" EXACT []
-xref: MetaCyc:PWY-3982
 is_a: GO:0019483 ! beta-alanine biosynthetic process
 
 [Term]
@@ -206180,7 +206680,11 @@
 synonym: "UDP-D-galacturonate formation" EXACT []
 synonym: "UDP-D-galacturonate synthesis" EXACT []
 xref: MetaCyc:PWY-4
+xref: MetaCyc:PWY-4861
 is_a: GO:0009226 ! nucleotide-sugar biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-4
+property_value: skos:narrowMatch MetaCyc:PWY-4861
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0033481
@@ -206290,7 +206794,6 @@
 synonym: "cholesterol formation via desmosterol" EXACT []
 synonym: "cholesterol synthesis via desmosterol" EXACT []
 xref: MetaCyc:PWY66-4
-xref: Reactome:R-HSA-6807047 "Cholesterol biosynthesis via desmosterol"
 is_a: GO:0006695 ! cholesterol biosynthetic process
 
 [Term]
@@ -206303,7 +206806,6 @@
 synonym: "cholesterol formation via lathosterol" EXACT []
 synonym: "cholesterol synthesis via lathosterol" EXACT []
 xref: MetaCyc:PWY66-341
-xref: Reactome:R-HSA-6807062 "Cholesterol biosynthesis via lathosterol"
 is_a: GO:0006695 ! cholesterol biosynthetic process
 
 [Term]
@@ -206505,31 +207007,44 @@
 
 [Term]
 id: GO:0033508
-name: L-glutamate catabolic process to pyruvate
+name: L-glutamate fermentation
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate into pyruvate." [MetaCyc:PWY-5087]
+def: "The anaerobic chemical reactions and pathways resulting in the breakdown of L-glutamate, yielding energy in the form of ATP." [PMID:38821505]
 synonym: "glutamate breakdown to butyrate" EXACT []
 synonym: "glutamate degradation to butyrate" EXACT []
+synonym: "L-glutamate catabolic process to pyruvate" NARROW []
+synonym: "L-glutamate fermentation to propionate" NARROW []
+synonym: "L-glutamate fermentation to pyruvate" NARROW []
+synonym: "L-glutamate fermentation via 2-hydroxyglutarate" NARROW []
+synonym: "L-glutamate fermentation via L-citramalate" NARROW []
+xref: MetaCyc:GLUDEG-II-PWY
+xref: MetaCyc:P162-PWY
 xref: MetaCyc:PWY-5087
+xref: MetaCyc:PWY-5088
+xref: MetaCyc:PWY-8190
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0006538 ! L-glutamate catabolic process
 is_a: GO:0019665 ! amino acid fermentation
+property_value: skos:narrowMatch MetaCyc:GLUDEG-II-PWY
+property_value: skos:narrowMatch MetaCyc:P162-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5087
+property_value: skos:narrowMatch MetaCyc:PWY-5088
+property_value: skos:narrowMatch MetaCyc:PWY-8190
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31218" xsd:anyURI
 
 [Term]
 id: GO:0033509
-name: L-glutamate fermentation to propionate
+name: obsolete L-glutamate fermentation to propionate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including propionate." [GOC:mah, MetaCyc:PWY-5088]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-glutamate into other compounds, including propionate." [GOC:mah]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "glutamate breakdown to propionate" EXACT []
 synonym: "glutamate degradation to propionate" EXACT []
 synonym: "L-glutamate catabolic process to propionate" EXACT []
-xref: MetaCyc:PWY-5088
-is_a: GO:0006538 ! L-glutamate catabolic process
-is_a: GO:0019541 ! propionate metabolic process
-is_a: GO:0019665 ! amino acid fermentation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0033508
 
 [Term]
 id: GO:0033510
@@ -206555,34 +207070,40 @@
 
 [Term]
 id: GO:0033512
-name: L-lysine catabolic process to acetyl-CoA via L-saccharopine
+name: obsolete L-lysine catabolic process to acetyl-CoA via L-saccharopine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-saccharopine." [GOC:mah, MetaCyc:LYSINE-DEG1-PWY]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-saccharopine." [GOC:mah, MetaCyc:LYSINE-DEG1-PWY]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via saccharopine" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via saccharopine" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033513
-name: L-lysine catabolic process to acetyl-CoA via 5-aminopentanamide
+name: obsolete L-lysine catabolic process to acetyl-CoA via 5-aminopentanamide
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate 5-aminopentanamide." [GOC:mah, MetaCyc:PWY-5280]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate 5-aminopentanamide." [GOC:mah, MetaCyc:PWY-5280]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via 5-aminopentanamide" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via 5-aminopentanamide" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033514
-name: L-lysine catabolic process to acetyl-CoA via L-pipecolate
+name: obsolete L-lysine catabolic process to acetyl-CoA via L-pipecolate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-pipecolate." [GOC:mah, MetaCyc:PWY66-425]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of L-lysine into other compounds, including acetyl-CoA, via the intermediate L-pipecolate." [GOC:mah, MetaCyc:PWY66-425]
+comment: This term was obsoleted because it represents a specific instance of a pathway that should be captured as a GO-CAM model.
 synonym: "L-lysine breakdown to acetyl-CoA via L-pipecolate" EXACT []
 synonym: "L-lysine degradation to acetyl-CoA via L-pipecolate" EXACT []
-is_a: GO:0006084 ! acetyl-CoA metabolic process
-is_a: GO:0019477 ! L-lysine catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0019477
 
 [Term]
 id: GO:0033515
@@ -206607,7 +207128,7 @@
 synonym: "L-methionine synthesis from homoserine via O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 synonym: "methionine biosynthetic process from homoserine via O-phospho-L-homoserine and cystathionine" EXACT [GOC:mah]
 is_obsolete: true
-replaced_by: GO:0019283
+replaced_by: GO:0071266
 
 [Term]
 id: GO:0033517
@@ -207169,7 +207690,7 @@
 name: lactoferrin transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of lactoferrin from one side of a membrane to the other." [GOC:mlg]
-is_a: GO:0008320 ! protein transmembrane transporter activity
+is_a: GO:0008320 ! transmembrane protein transporter activity
 relationship: part_of GO:0033571 ! lactoferrin transport
 
 [Term]
@@ -207323,7 +207844,6 @@
 synonym: "L-tyrosine formation from chorismate via L-arogenate" EXACT []
 synonym: "L-tyrosine synthesis from chorismate via L-arogenate" EXACT []
 synonym: "tyrosine biosynthetic process from chorismate via L-arogenate" BROAD []
-xref: MetaCyc:PWY-3461
 is_a: GO:0006571 ! L-tyrosine biosynthetic process
 is_a: GO:0009095 ! aromatic amino acid family biosynthetic process, prephenate pathway
 is_a: GO:0046417 ! chorismate metabolic process
@@ -207333,12 +207853,11 @@
 id: GO:0033585
 name: L-phenylalanine biosynthetic process from chorismate via phenylpyruvate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-phenylalanine from other compounds, including chorismate, via the intermediate phenylpyruvate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-phenylalanine from other compounds, including chorismate, via the intermediate phenylpyruvate." [GOC:curators]
 synonym: "L-phenylalanine anabolism from chorismate via phenylpyruvate" EXACT []
 synonym: "L-phenylalanine biosynthesis from chorismate via phenylpyruvate" EXACT []
 synonym: "L-phenylalanine formation from chorismate via phenylpyruvate" EXACT []
 synonym: "L-phenylalanine synthesis from chorismate via phenylpyruvate" EXACT []
-xref: MetaCyc:PHESYN
 is_a: GO:0009094 ! L-phenylalanine biosynthetic process
 is_a: GO:0046417 ! chorismate metabolic process
 
@@ -207346,12 +207865,11 @@
 id: GO:0033586
 name: L-phenylalanine biosynthetic process from chorismate via L-arogenate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-phenylalanine from other compounds, including chorismate, via the intermediate L-arogenate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of L-phenylalanine from other compounds, including chorismate, via the intermediate L-arogenate." [GOC:curators]
 synonym: "L-phenylalanine anabolism from chorismate via L-arogenate" EXACT []
 synonym: "L-phenylalanine biosynthesis from chorismate via L-arogenate" EXACT []
 synonym: "L-phenylalanine formation from chorismate via L-arogenate" EXACT []
 synonym: "L-phenylalanine synthesis from chorismate via L-arogenate" EXACT []
-xref: MetaCyc:PWY-3462
 is_a: GO:0009094 ! L-phenylalanine biosynthetic process
 is_a: GO:0009095 ! aromatic amino acid family biosynthetic process, prephenate pathway
 is_a: GO:0046417 ! chorismate metabolic process
@@ -208450,7 +208968,7 @@
 id: GO:0033692
 name: obsolete cellular polysaccharide biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of polysaccharides, polymers of many (typically more than 10) monosaccharide residues linked glycosidically, occurring at the level of an individual cell." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of polysaccharides, polymers of many (typically more than 10) monosaccharide residues linked glycosidically, occurring at the level of an individual cell." [GOC:curators]
 comment: This term was obsoleted because it is not different from its parent, polysaccharide biosynthetic process ; GO:0000271.
 synonym: "cellular glycan biosynthesis" EXACT []
 synonym: "cellular glycan biosynthetic process" EXACT []
@@ -208758,7 +209276,7 @@
 id: GO:0033713
 name: choline:oxygen 1-oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: choline + O2 = betaine aldehyde + hydrogen peroxide." [EC:1.1.3.17, RHEA:13505]
+def: "Catalysis of the reaction: choline + O2 = betaine aldehyde + H2O2." [EC:1.1.3.17, RHEA:13505]
 synonym: "choline oxidase activity" BROAD [EC:1.1.3.17]
 xref: EC:1.1.3.17
 xref: MetaCyc:CHOLINE-OXIDASE-RXN
@@ -208803,7 +209321,7 @@
 id: GO:0033716
 name: nucleoside oxidase (hydrogen peroxide-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reactions: adenosine + 2 O2 = 9-riburonosyladenine + 2 hydrogen peroxide." [EC:1.1.3.39, RHEA:15489]
+def: "Catalysis of the reaction: adenosine + 2 O2 + H2O = 9-riburonosyladenine + 2 H2O2 + H+." [RHEA:15489]
 comment: Two other reactions might be associated with this activity: (1a) adenosine + O2 = 5'-dehydroadenosine + hydrogen peroxide, and (1b) 5'-dehydroadenosine + O2 = 9-riburonosyladenine + hydrogen peroxide.
 synonym: "nucleoside:oxygen 5'-oxidoreductase (hydrogen peroxide-forming) activity" RELATED [EC:1.1.3.39]
 xref: EC:1.1.3.39
@@ -209782,9 +210300,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 7alpha-hydroxycholest-4-en-3-one + H+ + NADPH + O2 = 7alpha,12alpha-dihydroxycholest-4-en-3-one + H2O + NADP+." [PMID:10051404, PMID:1400444]
 comment: This term was obsoleted because it represents a specific susbstrate of sterol 12-alpha-hydroxylase activity ; GO:0008397.
-xref: Reactome:R-HSA-192157 "CYP8B1 12-hydroxylates 4CHOL7aOLONE"
-xref: Reactome:R-HSA-193709 "CYP8B1 12-hydroxylates 4CHOL7a,24(S)DIOL"
-xref: Reactome:R-HSA-193845 "CYP8B1 12-hydroxylates 4CHOL7a,27DONE"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24843" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29331" xsd:anyURI
 is_obsolete: true
@@ -209794,7 +210309,7 @@
 id: GO:0033779
 name: obsolete 5-beta-cholestane-3-alpha,7-alpha-diol 12-alpha-hydroxylase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: 5beta-cholestane-3alpha,7alpha-diol + H+ + NADPH + O2 = 5beta-cholestane-3alpha,7alpha,12alpha-triol + H2O + NADP+." [GOC:curator]
+def: "OBSOLETE. Catalysis of the reaction: 5beta-cholestane-3alpha,7alpha-diol + H+ + NADPH + O2 = 5beta-cholestane-3alpha,7alpha,12alpha-triol + H2O + NADP+." [GOC:curators]
 comment: This term was obsoleted because it represents a specific susbstrate of sterol 12-alpha-hydroxylase activity ; GO:0008397.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24843" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29331" xsd:anyURI
@@ -210125,6 +210640,7 @@
 synonym: "type III iodothyronine deiodinase activity" NARROW [EC:1.21.99.3]
 xref: EC:1.21.99.3
 xref: MetaCyc:1.97.1.11-RXN
+xref: Reactome:R-HSA-350869 "Thyroxine is deiodinated to reverse triiodothyronine (RT3)"
 xref: RHEA:18897
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.99.3
@@ -210171,7 +210687,7 @@
 id: GO:0033801
 name: vitexin 2''-O-rhamnoside 7-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + vitexin 2''-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2''-O-beta-L-rhamnoside." [EC:2.1.1.153]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + vitexin 2''-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2''-O-beta-L-rhamnoside." [RHEA:11432]
 synonym: "S-adenosyl-L-methionine:vitexin-2''-O-beta-L-rhamnoside 7-O-methyltransferase activity" RELATED [EC:2.1.1.153]
 xref: EC:2.1.1.153
 xref: MetaCyc:RXN-4981
@@ -211009,14 +211525,15 @@
 
 [Term]
 id: GO:0033852
-name: thyroid-hormone transaminase activity
+name: thyroid-hormone:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3,5,3'-triiodo-L-thyronine = 3,5,3'-triiodothyropyruvate + L-glutamate." [EC:2.6.1.26, RHEA:19133]
+def: "Catalysis of the reaction: 3,3',5-triiodo-L-thyronine + 2-oxoglutarate = 3,5,3'-triiodothyropyruvate + L-glutamate." [RHEA:19133]
 synonym: "3,5-dinitrotyrosine aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "3,5-dinitrotyrosine transaminase activity" RELATED [EC:2.6.1.26]
 synonym: "L-3,5,3'-triiodothyronine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "thyroid hormone aminotransferase activity" RELATED [EC:2.6.1.26]
 synonym: "thyroid-hormone aminotransferase activity" RELATED [EC:2.6.1.26]
+synonym: "thyroid-hormone transaminase activity" BROAD []
 xref: EC:2.6.1.26
 xref: KEGG_REACTION:R03952
 xref: MetaCyc:THYROID-HORMONE-AMINOTRANSFERASE-RXN
@@ -211025,12 +211542,14 @@
 property_value: skos:exactMatch EC:2.6.1.26
 property_value: skos:exactMatch RHEA:19133
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033853
-name: aspartate-prephenate aminotransferase activity
+name: L-aspartate:prephenate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-arogenate + oxaloacetate = prephenate + L-aspartate." [RHEA:20445]
+synonym: "aspartate-prephenate aminotransferase activity" EXACT []
 synonym: "L-arogenate:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.78]
 synonym: "L-aspartate:prephenate aminotransferase activity" RELATED [EC:2.6.1.78]
 synonym: "PAT" RELATED [EC:2.6.1.78]
@@ -211039,16 +211558,18 @@
 xref: EC:2.6.1.78
 xref: MetaCyc:PREPHENATE-ASP-TRANSAMINE-RXN
 xref: RHEA:20445
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.78
 property_value: skos:exactMatch RHEA:20445
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033854
-name: glutamate-prephenate aminotransferase activity
+name: L-glutamate:prephenate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-arogenate = L-glutamate + prephenate." [EC:2.6.1.79, RHEA:22880]
+def: "Catalysis of the reaction: 2-oxoglutarate + L-arogenate = L-glutamate + prephenate." [RHEA:22880]
+synonym: "glutamate-prephenate aminotransferase activity" EXACT []
 synonym: "L-arogenate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.79]
 synonym: "L-glutamate:prephenate aminotransferase activity" RELATED [EC:2.6.1.79]
 synonym: "PAT" RELATED [EC:2.6.1.79]
@@ -211057,20 +211578,22 @@
 xref: KEGG_REACTION:R07276
 xref: MetaCyc:PREPHENATE-TRANSAMINE-RXN
 xref: RHEA:22880
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.79
 property_value: skos:exactMatch RHEA:22880
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033855
-name: nicotianamine aminotransferase activity
+name: nicotianamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + nicotianamine = 3''-deamino-3''-oxonicotianamine + L-glutamate." [EC:2.6.1.80, RHEA:22104]
+def: "Catalysis of the reaction: nicotianamine + 2-oxoglutarate = 3''-deamino-3''-oxonicotianamine + L-glutamate." [RHEA:22104]
 synonym: "NAAT" RELATED [EC:2.6.1.80]
 synonym: "NAAT-I" RELATED [EC:2.6.1.80]
 synonym: "NAAT-II" RELATED [EC:2.6.1.80]
 synonym: "NAAT-III" RELATED [EC:2.6.1.80]
+synonym: "nicotianamine aminotransferase activity" EXACT []
 synonym: "nicotianamine transaminase activity" RELATED [EC:2.6.1.80]
 synonym: "nicotianamine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.80]
 xref: EC:2.6.1.80
@@ -211081,6 +211604,7 @@
 property_value: skos:exactMatch EC:2.6.1.80
 property_value: skos:exactMatch RHEA:22104
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0033856
@@ -212034,6 +212558,8 @@
 xref: Reactome:R-HSA-548890 "Removal of the third glucose by glucosidase II and release from the chaperone"
 xref: Reactome:R-HSA-9683663 "N-glycan trimming of Spike"
 xref: Reactome:R-HSA-9694364 "N-glycan glucose trimming of Spike"
+xref: Reactome:R-HSA-9932352 "Glucosidase II removes glucose residue from Glu2Man9GlucNAc2-CDH1"
+xref: Reactome:R-HSA-9932913 "Glucosidase II removes glucose residue from Glu1Man9GlucNAc2-CDH1"
 is_a: GO:0090599 ! alpha-glucosidase activity
 property_value: skos:exactMatch EC:3.2.1.84
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -214092,7 +214618,6 @@
 synonym: "negative regulation by symbiont of plant hypersensitive response" NARROW [GOC:pamgo_curators]
 synonym: "suppression by symbiont of host defense-related programmed cell death" EXACT []
 synonym: "suppression of HR" NARROW [GOC:pamgo_curators]
-xref: Reactome:R-HSA-9636569 "Suppression of autophagy"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29278" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0052041
@@ -215792,7 +216317,7 @@
 id: GO:0034203
 name: glycolipid translocation
 namespace: biological_process
-def: "The translocation, or flipping, of glycolipid molecules from one monolayer of a membrane bilayer to the opposite monolayer." [GOC:go_curators, PMID:11807558]
+def: "The translocation, or flipping, of glycolipid molecules from one monolayer of a membrane bilayer to the opposite monolayer." [GOC:curators, PMID:11807558]
 is_a: GO:0034204 ! lipid translocation
 is_a: GO:0046836 ! glycolipid transport
 
@@ -216169,7 +216694,7 @@
 name: macrophage fusion
 namespace: biological_process
 def: "The binding and fusion of a macrophage to one or more other cells to form a multinucleated cell." [GOC:sl]
-is_a: GO:0000768 ! syncytium formation by plasma membrane fusion
+is_a: GO:0000768 ! syncytium formation by cell-cell fusion
 
 [Term]
 id: GO:0034239
@@ -216208,12 +216733,11 @@
 name: negative regulation of syncytium formation by plasma membrane fusion
 namespace: biological_process
 def: "Any process that decreases the frequency, rate or extent of the formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual cells." [GOC:mah]
-is_a: GO:0051093 ! negative regulation of developmental process
 is_a: GO:0051129 ! negative regulation of cellular component organization
 is_a: GO:0060142 ! regulation of syncytium formation by plasma membrane fusion
 intersection_of: GO:0065007 ! biological regulation
-intersection_of: negatively_regulates GO:0000768 ! syncytium formation by plasma membrane fusion
-relationship: negatively_regulates GO:0000768 ! syncytium formation by plasma membrane fusion
+intersection_of: negatively_regulates GO:0000768 ! syncytium formation by cell-cell fusion
+relationship: negatively_regulates GO:0000768 ! syncytium formation by cell-cell fusion
 
 [Term]
 id: GO:0034243
@@ -216533,16 +217057,15 @@
 
 [Term]
 id: GO:0034269
-name: discadenine catabolic process
+name: obsolete discadenine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of discadenine, (2S)-2-amino-4-{6-[(3-methylbut-2-en-1-yl)amino]-3H-purin-3-yl}butanoic acid." [GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of discadenine, (2S)-2-amino-4-{6-[(3-methylbut-2-en-1-yl)amino]-3H-purin-3-yl}butanoic acid." [GOC:mah]
+comment: This term was obsoleted because there is no evidence that a specific pathway for discadenine degradation exists.
 synonym: "discadenine breakdown" EXACT []
 synonym: "discadenine catabolism" EXACT []
 synonym: "discadenine degradation" EXACT []
-is_a: GO:0009823 ! cytokinin catabolic process
-is_a: GO:0072523 ! purine-containing compound catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034270
@@ -217473,7 +217996,6 @@
 synonym: "nicotinamide riboside salvage pathway" EXACT []
 synonym: "NR salvage pathway" EXACT []
 xref: MetaCyc:PWY3O-4106
-xref: Reactome:R-HSA-197264 "Nicotinamide salvaging"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29050" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0034355
@@ -217786,6 +218308,7 @@
 synonym: "gamma-aminobutyric acid-alpha-ketoglutaric acid aminotransferase activity" RELATED [EC:2.6.1.19]
 xref: EC:2.6.1.19
 xref: MetaCyc:GABATRANSAM-RXN
+xref: Reactome:R-HSA-916855 "PXLP-K357-ABAT dimer:2Fe-2S transforms GABA to SUCCSA"
 xref: RHEA:23352
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch EC:2.6.1.19
@@ -218806,6 +219329,7 @@
 synonym: "chondroitin 2-O-sulfotransferase activity" EXACT []
 synonym: "chondroitin 2-O-sulphotransferase activity" EXACT []
 xref: MetaCyc:RXN-11560
+xref: Reactome:R-HSA-9941305 "UST transfers sulfate to (2-)GlcA-GalNAc-6-sulfate"
 is_a: GO:0034481 ! chondroitin sulfotransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
 
@@ -219379,31 +219903,37 @@
 
 [Term]
 id: GO:0034547
-name: N-cyclopropylmelamine deaminase activity
+name: obsolete N-cyclopropylmelamine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyromazine + H2O = N-cyclopropylammeline + NH3." [UM-BBD_reactionID:r0825]
+def: "OBSOLETE. Catalysis of the reaction: cyromazine + H2O = N-cyclopropylammeline + NH3." [UM-BBD_reactionID:r0825]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8018
 xref: UM-BBD_reactionID:r0825
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034548
-name: N-cyclopropylammeline deaminase activity
+name: obsolete N-cyclopropylammeline deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-cyclopropylammeline + H2O = N-cyclopropylammelide + NH3." [UM-BBD_reactionID:r0826]
+def: "OBSOLETE. Catalysis of the reaction: N-cyclopropylammeline + H2O = N-cyclopropylammelide + NH3." [UM-BBD_reactionID:r0826]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:RXN-8019
 xref: UM-BBD_reactionID:r0826
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034549
-name: N-cyclopropylammelide alkylamino hydrolase activity
+name: obsolete N-cyclopropylammelide alkylamino hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-cyclopropylammelide + H2O = cyclopropylamine + cyanuric acid." [UM-BBD_reactionID:r0827]
+def: "OBSOLETE. Catalysis of the reaction: N-cyclopropylammelide + H2O = cyclopropylamine + cyanuric acid." [UM-BBD_reactionID:r0827]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: KEGG_REACTION:R06972
 xref: MetaCyc:RXN-8020
 xref: UM-BBD_reactionID:r0827
-is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034550
@@ -219472,13 +220002,13 @@
 
 [Term]
 id: GO:0034557
-name: 2-hydroxylaminobenzoate reductase activity
+name: obsolete 2-hydroxylaminobenzoate reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-hydroxylaminobenzoate + NAD(P)H = anthranilate + NAD(P)+ + H2O." [MetaCyc:RXN-8848]
+def: "OBSOLETE. Catalysis of the reaction: 2-hydroxylaminobenzoate + NAD(P)H = anthranilate + NAD(P)+ + H2O." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "o-hydroxylaminobenzoate nitroreductase activity" RELATED [UM-BBD_reactionID:r0850]
-xref: MetaCyc:RXN-8848
-xref: UM-BBD_reactionID:r0850
-is_a: GO:0016638 ! oxidoreductase activity, acting on the CH-NH2 group of donors
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034558
@@ -219985,8 +220515,6 @@
 def: "OBSOLETE. Catalysis of the exonucleolytic cleavage of oligoribonucleotides to yield ribonucleoside 5'-phosphates." [GOC:mah]
 comment: This term was obsoleted because it corresponds to an assay.
 synonym: "RNA oligonucleotidase activity" EXACT []
-xref: Reactome:R-HSA-9009950 "PDE12 cleaves 2'-5' oligoadenylates"
-xref: Reactome:R-HSA-9615042 "Viral 2',5'-PDE cleaves 2'-5' oligoadenylates"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24408" xsd:anyURI
 is_obsolete: true
 
@@ -220038,6 +220566,7 @@
 synonym: "H4biopterin binding" EXACT []
 synonym: "sapropterin binding" EXACT []
 xref: Reactome:R-HSA-1497784 "The cofactor BH4 is required for electron transfer in the eNOS catalytic cycle"
+xref: Reactome:R-HSA-1497796 "BH2 binding can lead to eNOS uncoupling"
 is_a: GO:0005488 ! binding
 
 [Term]
@@ -220411,18 +220940,21 @@
 name: nucleobase-containing compound biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:mah]
+subset: gocheck_do_not_annotate
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid anabolism" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid biosynthesis" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid formation" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid synthesis" NARROW []
 is_a: GO:0006139 ! nucleobase-containing compound metabolic process
 is_a: GO:0009058 ! biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0034655
 name: nucleobase-containing compound catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:mah]
+subset: gocheck_do_not_annotate
 subset: goslim_chembl
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid breakdown" EXACT []
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid catabolic process" RELATED [GOC:dph, GOC:tb]
@@ -220430,6 +220962,7 @@
 synonym: "nucleobase, nucleoside, nucleotide and nucleic acid degradation" EXACT []
 is_a: GO:0006139 ! nucleobase-containing compound metabolic process
 is_a: GO:0009056 ! catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0034656
@@ -220570,7 +221103,7 @@
 id: GO:0034670
 name: chemotaxis to arachidonate
 namespace: biological_process
-def: "The directed movement of a motile cell or organism in response to the presence of arachidonic acid." [GOC:go_curators, PMID:18202452]
+def: "The directed movement of a motile cell or organism in response to the presence of arachidonic acid." [GOC:curators, PMID:18202452]
 synonym: "chemotaxis to arachidonic acid" EXACT []
 is_a: GO:0006935 ! chemotaxis
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28648" xsd:anyURI
@@ -222091,13 +222624,12 @@
 
 [Term]
 id: GO:0034823
-name: citronellyl-CoA ligase activity
+name: obsolete citronellyl-CoA ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: citronellate + CoASH + ATP = citronellyl-CoA + AMP + PPi." [UM-BBD_reactionID:r1157]
-xref: KEGG_REACTION:R08088
-xref: UM-BBD_reactionID:r1157
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+def: "OBSOLETE. Catalysis of the reaction: citronellate + CoASH + ATP = citronellyl-CoA + AMP + PPi." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034824
@@ -222916,12 +223448,12 @@
 
 [Term]
 id: GO:0034912
-name: phthalate 3,4-cis-dihydrodiol dehydrogenase activity
+name: obsolete phthalate 3,4-cis-dihydrodiol dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: phthalate 3,4-cis-dihydrodiol + NAD+ = 3,4-dihydroxyphthalate + NADH + H+." [UM-BBD_reactionID:r1445]
-xref: MetaCyc:RXN5F9-56
-xref: UM-BBD_reactionID:r1445
-is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: phthalate 3,4-cis-dihydrodiol + NAD+ = 3,4-dihydroxyphthalate + NADH + H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0034914
@@ -224199,7 +224731,7 @@
 id: GO:0035050
 name: embryonic heart tube development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the embryonic heart tube over time, from its formation to the mature structure. The heart tube forms as the heart rudiment from the heart field." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the embryonic heart tube over time, from its formation to the mature structure. The heart tube forms as the heart rudiment from the heart field." [GOC:curators]
 is_a: GO:0035295 ! tube development
 is_a: GO:0060429 ! epithelium development
 relationship: part_of GO:0007507 ! heart development
@@ -224891,7 +225423,7 @@
 name: forelimb morphogenesis
 namespace: biological_process
 alt_id: GO:0035140
-def: "The process in which the anatomical structures of the forelimb are generated and organized. The forelimbs are the front limbs of an animal, e.g. the arms of a human." [GOC:go_curators]
+def: "The process in which the anatomical structures of the forelimb are generated and organized. The forelimbs are the front limbs of an animal, e.g. the arms of a human." [GOC:curators]
 synonym: "arm morphogenesis" NARROW [GOC:bf, GOC:cjm]
 is_a: GO:0035108 ! limb morphogenesis
 
@@ -224899,7 +225431,7 @@
 id: GO:0035137
 name: hindlimb morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the hindlimb are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the hindlimb are generated and organized." [GOC:curators]
 is_a: GO:0035108 ! limb morphogenesis
 
 [Term]
@@ -226015,9 +226547,11 @@
 id: GO:0035248
 name: alpha-1,4-N-acetylgalactosaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an N-acetyl-beta-D-galactosaminyl derivative + UDP-N-acetyl-alpha-D-galactosamine = an N-acetyl-alpha-D-galactosaminyl-(1->4)-N-acetyl-beta-D-galactosaminyl derivative + UDP + H+." [PMID:15130086]
+def: "Catalysis of the reaction: an N-acetyl-beta-D-galactosaminyl derivative + UDP-N-acetyl-alpha-D-galactosamine = an N-acetyl-alpha-D-galactosaminyl-(1->4)-N-acetyl-beta-D-galactosaminyl derivative + UDP + H+." [PMID:15130086, RHEA:86251]
 synonym: "alpha-1,4-GalNAc transferase activity" EXACT []
+xref: RHEA:86251
 is_a: GO:0008376 ! acetylgalactosaminyltransferase activity
+property_value: skos:exactMatch RHEA:86251
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31147" xsd:anyURI
 
 [Term]
@@ -226051,9 +226585,8 @@
 name: UDP-xylosyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a xylosyl group from UDP-xylose to an acceptor molecule." [PMID:30127001]
-xref: Reactome:R-HSA-5617138 "B4GAT1:GYLTL1B transfers Xyl from UDP-Xyl to GlcA-Xyl-GlcA"
 xref: Reactome:R-HSA-6785668 "Defective LARGE does not transfer Xyl from UDP-Xyl to GlcA"
-xref: Reactome:R-HSA-9638090 "B4GAT1:LARGE transfers Xyl from UDP-Xyl to GlcA-Xyl-GlcA"
+xref: Reactome:R-HSA-9638090 "LARGE1,2 elongates matriglycan chain with Xyl"
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
 is_a: GO:0042285 ! xylosyltransferase activity
 
@@ -227284,6 +227817,7 @@
 namespace: molecular_function
 def: "Binding to a selenocysteine insertion sequence (SECIS), a regulatory sequence within mRNA which directs incorporation of a selenocysteine at a stop codon (UGA) during translation." [GOC:imk, PMID:10760958]
 synonym: "SECIS binding" EXACT []
+xref: Reactome:R-HSA-2408529 "Sec-tRNA(Sec):EEFSEC:GTP binds to 80S Ribosome"
 is_a: GO:0003729 ! mRNA binding
 created_by: bf
 creation_date: 2010-03-11T10:42:28Z
@@ -227410,12 +227944,14 @@
 
 [Term]
 id: GO:0035380
-name: very long-chain-3-hydroxyacyl-CoA dehydrogenase activity
+name: very long-chain (3S)-3-hydroxyacyl-CoA dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a very long-chain fatty acid residue. A very long-chain fatty acid has an aliphatic tail containing more than 22 carbons." [GOC:pde]
+def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a very long-chain fatty acid residue. A very long-chain fatty acid has an aliphatic tail containing more than 22 carbons." [RHEA:78927]
 comment: While there is not universal consensus on the lengths of short-, medium-, long- and very-long-chain fatty acids, the GO uses the definitions in ChEBI (see CHEBI:26666, CHEBI:59554, CHEBI:15904 and CHEBI:27283).
+synonym: "very long-chain-3-hydroxyacyl-CoA dehydrogenase activity" BROAD []
 synonym: "very-long-chain-3-hydroxyacyl-CoA dehydrogenase activity" EXACT []
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+xref: RHEA:78927
+is_a: GO:0003857 ! (3S)-3-hydroxyacyl-CoA dehydrogenase (NAD+) activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26445" xsd:anyURI
 created_by: bf
 creation_date: 2010-03-18T11:23:05Z
@@ -228027,6 +228563,7 @@
 def: "The process in which a phosphate is transported across a membrane." [GOC:vw]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "phosphate ion membrane transport" EXACT []
+xref: Reactome:R-HSA-427652 "Sodium-coupled phosphate cotransporters"
 is_a: GO:0006817 ! phosphate ion transport
 is_a: GO:0055085 ! transmembrane transport
 created_by: bf
@@ -228806,9 +229343,11 @@
 synonym: "carnosine biosynthesis" EXACT []
 synonym: "carnosine formation" EXACT []
 synonym: "carnosine synthesis" EXACT []
+xref: MetaCyc:PWY66-420
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0035498 ! carnosine metabolic process
 is_a: GO:0046394 ! carboxylic acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: bf
 creation_date: 2010-04-29T01:20:56Z
 
@@ -228858,7 +229397,7 @@
 id: GO:0035504
 name: obsolete regulation of myosin light chain kinase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22455" xsd:anyURI
 is_obsolete: true
@@ -228869,7 +229408,7 @@
 id: GO:0035505
 name: obsolete positive regulation of myosin light chain kinase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22455" xsd:anyURI
 is_obsolete: true
@@ -228880,7 +229419,7 @@
 id: GO:0035506
 name: obsolete negative regulation of myosin light chain kinase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of myosin light chain kinase activity." [GOC:bf, GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28482" xsd:anyURI
 is_obsolete: true
@@ -228891,7 +229430,7 @@
 id: GO:0035507
 name: obsolete regulation of myosin-light-chain-phosphatase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:curators]
 comment: The reason for obsoletion is that this term represents a single-step process (regulation of MF).
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28499" xsd:anyURI
 is_obsolete: true
@@ -228902,7 +229441,7 @@
 id: GO:0035508
 name: obsolete positive regulation of myosin-light-chain-phosphatase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:curators]
 comment: The reason for obsoletion is that this term represents a single-step process (regulation of MF).
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28499" xsd:anyURI
 is_obsolete: true
@@ -228913,7 +229452,7 @@
 id: GO:0035509
 name: obsolete negative regulation of myosin-light-chain-phosphatase activity
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of myosin-light-chain-phosphatase activity." [GOC:bf, GOC:curators]
 comment: The reason for obsoletion is that this term represents a single-step process (regulation of MF).
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28499" xsd:anyURI
 is_obsolete: true
@@ -229690,9 +230229,13 @@
 synonym: "histone H4K20me demethylase activity" EXACT []
 xref: Reactome:R-HSA-2172678 "PHF8 demethylates histone H4K20me1"
 xref: RHEA:67804
+xref: RHEA:85907
+xref: RHEA:85911
 is_a: GO:0016706 ! 2-oxoglutarate-dependent dioxygenase activity
 is_a: GO:0141058 ! histone H4 demethylase activity
 property_value: skos:narrowMatch RHEA:67804
+property_value: skos:narrowMatch RHEA:85907
+property_value: skos:narrowMatch RHEA:85911
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21089" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29636" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30578" xsd:anyURI
@@ -230448,7 +230991,7 @@
 id: GO:0035636
 name: obsolete multi-organism signaling
 namespace: biological_process
-def: "OBSOLETE. The transfer of information between living organisms." [GOC:go_curators]
+def: "OBSOLETE. The transfer of information between living organisms." [GOC:curators]
 comment: This term was obsoleted because it was a grouping term but had no subclasses.
 synonym: "multi-organism signalling" EXACT [GOC:bf]
 synonym: "pheromone signaling" NARROW [GOC:bf]
@@ -230462,7 +231005,7 @@
 id: GO:0035637
 name: obsolete multicellular organismal signaling
 namespace: biological_process
-def: "OBSOLETE. The transfer of information occurring at the level of a multicellular organism." [GOC:go_curators]
+def: "OBSOLETE. The transfer of information occurring at the level of a multicellular organism." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "multicellular organismal signalling" EXACT [GOC:bf]
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24029" xsd:anyURI
@@ -230859,6 +231402,7 @@
 def: "The process in which an oligopeptide is transported across a membrane. Oligopeptides are molecules that contain a small number (2 to 20) of amino-acid residues connected by peptide linkages." [GOC:vw, ISBN:0198506732]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "oligopeptide membrane transport" EXACT []
+xref: Reactome:R-HSA-9959399 "SLC-mediated transport of oligopeptides"
 is_a: GO:0006857 ! oligopeptide transport
 is_a: GO:0055085 ! transmembrane transport
 created_by: bf
@@ -231421,6 +231965,7 @@
 def: "A process in which a sodium ion is transported from one side of a membrane to the other by means of some agent such as a transporter or pore." [GOC:vw]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "sodium ion membrane transport" EXACT []
+xref: Reactome:R-HSA-425986 "Sodium/Proton exchangers"
 is_a: GO:0006814 ! sodium ion transport
 is_a: GO:0098655 ! monoatomic cation transmembrane transport
 created_by: bf
@@ -231696,11 +232241,14 @@
 
 [Term]
 id: GO:0035751
-name: regulation of lysosomal lumen pH
+name: obsolete regulation of lysosomal lumen pH
 namespace: biological_process
-def: "Any process that modulates the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:rph]
-is_a: GO:0051453 ! regulation of intracellular pH
-relationship: part_of GO:0007040 ! lysosome organization
+def: "OBSOLETE. Any process that modulates the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:rph]
+comment: The reason for obsoletion is that this term was an unnecessary grouping term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0007042
+consider: GO:0035752
 created_by: bf
 creation_date: 2011-03-23T10:23:41Z
 
@@ -231710,7 +232258,6 @@
 namespace: biological_process
 def: "Any process that increases the pH of the lysosomal lumen, measured by the concentration of the hydrogen ion." [GOC:bf, GOC:rph]
 synonym: "lysosome pH elevation" EXACT [GOC:bf, GOC:rph]
-is_a: GO:0035751 ! regulation of lysosomal lumen pH
 is_a: GO:0051454 ! intracellular pH elevation
 created_by: bf
 creation_date: 2011-03-23T10:24:24Z
@@ -231902,7 +232449,7 @@
 id: GO:0035770
 name: ribonucleoprotein granule
 namespace: cellular_component
-def: "A non-membranous macromolecular complex containing proteins and translationally silenced mRNAs. RNA granules contain proteins that control the localization, stability, and translation of their RNA cargo. Different types of RNA granules (RGs) exist, depending on the cell type and cellular conditions." [GOC:go_curators, GOC:sp, PMID:16520386, PMID:20368989, PMID:21436445]
+def: "A non-membranous macromolecular complex containing proteins and translationally silenced mRNAs. RNA granules contain proteins that control the localization, stability, and translation of their RNA cargo. Different types of RNA granules (RGs) exist, depending on the cell type and cellular conditions." [GOC:curators, GOC:sp, PMID:16520386, PMID:20368989, PMID:21436445]
 synonym: "mRNP granule" EXACT []
 synonym: "RNA granule" NARROW []
 synonym: "RNP granule" EXACT []
@@ -233509,18 +234056,21 @@
 namespace: biological_process
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
 synonym: "response to social isolation" EXACT [PMID:20203532]
-is_a: GO:0006950 ! response to stress
+is_a: GO:0033555 ! multicellular organismal response to stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 created_by: bf
 creation_date: 2011-06-22T09:39:57Z
 
 [Term]
 id: GO:0035901
-name: cellular response to isolation stress
+name: obsolete cellular response to isolation stress
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
+def: "OBSOLETE. Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a lack of contact with other members of the same species." [GOC:bf, PMID:20203532]
+comment: This term was obsoleted because it represents the molecular activities involved in response to isolation stress ; GO:0035900.
 synonym: "cellular response to social isolation" EXACT [PMID:20203532]
-is_a: GO:0033554 ! cellular response to stress
-is_a: GO:0035900 ! response to isolation stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0035900
 created_by: bf
 creation_date: 2011-06-22T09:43:34Z
 
@@ -235421,6 +235971,7 @@
 synonym: "peptidyl-succinyllysine desuccinylase activity" EXACT []
 synonym: "succinyl lysine desuccinylase activity" BROAD [GOC:bf]
 synonym: "succinyllysine desuccinylase activity" BROAD []
+xref: Reactome:R-HSA-9955518 "SIRT5 desuccinylates CPS1"
 xref: RHEA:47668
 is_a: GO:0141218 ! NAD-dependent protein lysine deacylase activity
 relationship: part_of GO:0036049 ! peptidyl-lysine desuccinylation
@@ -235711,7 +236262,7 @@
 id: GO:0036081
 name: extracellular ammonia-gated monoatomic ion channel activity
 namespace: molecular_function
-def: "Enables the transmembrane transfer of an ion by a channel that opens when extracellular ammonia (NH3) has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
+def: "Enables the transmembrane transfer of an ion by a channel that opens when extracellular ammonia (NH4+) has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
 synonym: "extracellular ammonia-gated ion channel activity" BROAD []
 synonym: "ionotropic ammonia receptor activity" RELATED [PMID:19135896]
 is_a: GO:0005230 ! extracellular ligand-gated monoatomic ion channel activity
@@ -235793,12 +236344,14 @@
 id: GO:0036088
 name: D-serine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-serine, the D-enantiomer of serine, i.e. (2S)-2-amino-3-hydroxypropanoic acid." [GOC:imk]
+def: "The chemical reactions and pathways resulting in the breakdown of D-serine." [GOC:curators]
 synonym: "D-serine breakdown" EXACT [GOC:bf]
 synonym: "D-serine catabolism" EXACT [GOC:bf]
 synonym: "D-serine degradation" EXACT [GOC:bf]
+xref: MetaCyc:PWY0-1535
 is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:0070178 ! D-serine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 created_by: bf
 creation_date: 2012-01-06T10:58:51Z
 
@@ -236782,7 +237335,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid -> 2-methoxy-6-decaprenylphenol + CO2." [GOC:mw, PMID:620805, PMID:7028108]
 comment: The reason for obsoletion is that this term is equivalent to 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity.
-xref: Reactome:R-HSA-2162195 "COQ4 decarboxylates MHDB"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0120539
@@ -239302,7 +239854,7 @@
 id: GO:0036381
 name: pyridoxal 5'-phosphate synthase (glutamine hydrolysing) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine = pyridoxal 5'-phosphate + L-glutamate + 3 H2O + phosphate. The reaction occurs in two steps: L-glutamine + H2O = L-glutamate + NH3, and subsequently D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + NH3 = pyridoxal 5'-phosphate + 4 H2O + phosphate." [EC:4.3.3.6, GOC:rs]
+def: "Catalysis of the reaction: D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine = pyridoxal 5'-phosphate + L-glutamate + 3 H2O + phosphate. The reaction occurs in two steps: L-glutamine + H2O = L-glutamate + NH4+, and subsequently D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + NH4+ = pyridoxal 5'-phosphate + 4 H2O + phosphate." [EC:4.3.3.6, GOC:rs]
 synonym: "PdxST activity" RELATED [EC:4.3.3.6]
 synonym: "pyridoxal 5'-phosphate synthase (glutamine hydrolyzing) activity" RELATED [EC:4.3.3.6]
 xref: EC:4.3.3.6
@@ -239471,8 +240023,9 @@
 id: GO:0036393
 name: thiocyanate peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: thiocyanate (SCN-) + hydrogen peroxide (H2O2) = hypothiocyanite (OSCN-) + 2 H2O. Catalyzes the hydrogen peroxide oxidation of thiocyanate." [GOC:pm, PMID:12626341]
+def: "Catalysis of the reaction: thiocyanate + H2O2 + H+ = hypothiocyanous acid + H2O." [GOC:pm, PMID:12626341, RHEA:69416]
 synonym: "lactoperoxidase activity" BROAD [PMID:12626341, Wikipedia:Lactoperoxidase]
+xref: RHEA:69416
 is_a: GO:0004601 ! peroxidase activity
 relationship: part_of GO:0018969 ! thiocyanate metabolic process
 created_by: bf
@@ -239911,12 +240464,12 @@
 
 [Term]
 id: GO:0036426
-name: ditrans, polycis-undecaprenyl-phosphate mannosyltransferase activity
+name: obsolete ditrans, polycis-undecaprenyl-phosphate mannosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ditrans,octacis-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate+ GDP." [GOC:curators]
-xref: KEGG_REACTION:R07257
-xref: MetaCyc:2.4.1.54-RXN
-is_a: GO:0047267 ! undecaprenyl-phosphate mannosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: ditrans,octacis-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate+ GDP." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: bf
 creation_date: 2013-09-16T13:21:01Z
 
@@ -240262,6 +240815,7 @@
 def: "Catalysis of the transfer of a iron-sulfur cluster from one compound (donor) to another (acceptor)." [GOC:bhm]
 synonym: "Fe-S transferase activity" EXACT [GOC:bf]
 xref: Reactome:R-HSA-2564828 "CIA Targeting Complex transfers 4Fe-4S cluster to apoproteins"
+xref: Reactome:R-HSA-8878815 "Formation of 4Fe-4S cluster on ISCA1:ISCA2"
 is_a: GO:0016782 ! transferase activity, transferring sulphur-containing groups
 created_by: bf
 creation_date: 2013-11-14T11:17:37Z
@@ -241585,11 +242139,13 @@
 synonym: "receptor activity involved in receptor-mediated endocytosis" NARROW [GOC:bf]
 synonym: "transport receptor activity" BROAD [GOC:signaling]
 xref: Reactome:R-HSA-203716 "eNOS:Caveolin-1 complex binds to Nostrin"
+xref: Reactome:R-HSA-2160915 "Receptor-mediated uptake of HA"
 xref: Reactome:R-HSA-3000103 "CUBN:AMN binds CBLIF:RCbl"
 xref: Reactome:R-HSA-3000112 "CD320-mediated TCN2:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-3000122 "CD320 binds extracellular TCN2:RCbl"
 xref: Reactome:R-HSA-3000137 "CUBN:AMN-mediated CBLIF:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-350168 "LRP2-mediated uptake of extracellular CUBN:GC:25(OH)D"
+xref: Reactome:R-HSA-350186 "CUBN binds GC:25(OH)D"
 xref: Reactome:R-HSA-9759202 "LRP2-mediated TCN2:RCbl uptake and delivery to lysosome"
 xref: Reactome:R-HSA-9759209 "LRP2 binds extracellular TCN2:RCbl"
 is_a: GO:0003674 ! molecular_function
@@ -246888,7 +247444,7 @@
 id: GO:0040013
 name: negative regulation of locomotion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of locomotion of a cell or organism." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of locomotion of a cell or organism." [GOC:curators]
 synonym: "down regulation of locomotion" EXACT []
 synonym: "down-regulation of locomotion" EXACT []
 synonym: "downregulation of locomotion" EXACT []
@@ -246938,7 +247494,7 @@
 id: GO:0040017
 name: positive regulation of locomotion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of locomotion of a cell or organism." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of locomotion of a cell or organism." [GOC:curators]
 synonym: "activation of locomotion" NARROW []
 synonym: "stimulation of locomotion" NARROW []
 synonym: "up regulation of locomotion" EXACT []
@@ -246954,7 +247510,7 @@
 id: GO:0040018
 name: positive regulation of multicellular organism growth
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of growth of an organism to reach its usual body size." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of growth of an organism to reach its usual body size." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "positive regulation of body growth" EXACT [GOC:dph, GOC:tb]
 synonym: "positive regulation of body size" EXACT [GOC:dph, GOC:tb]
 is_a: GO:0040014 ! regulation of multicellular organism growth
@@ -246968,7 +247524,7 @@
 id: GO:0040019
 name: positive regulation of embryonic development
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of embryonic development." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of embryonic development." [GOC:curators]
 synonym: "activation of embryonic development" NARROW []
 synonym: "stimulation of embryonic development" NARROW []
 synonym: "up regulation of embryonic development" EXACT []
@@ -247146,7 +247702,7 @@
 id: GO:0040036
 name: regulation of fibroblast growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "regulation of FGF receptor signaling pathway" EXACT []
 synonym: "regulation of FGF receptor signalling pathway" EXACT []
 synonym: "regulation of FGFR signaling pathway" EXACT []
@@ -247160,7 +247716,7 @@
 id: GO:0040037
 name: negative regulation of fibroblast growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "down regulation of fibroblast growth factor receptor signaling pathway" EXACT []
 synonym: "down-regulation of fibroblast growth factor receptor signaling pathway" EXACT []
 synonym: "downregulation of fibroblast growth factor receptor signaling pathway" EXACT []
@@ -247211,6 +247767,7 @@
 synonym: "translocation of peptides or proteins into other organism involved in symbiotic interaction" BROAD []
 synonym: "transport of peptides or proteins into host" EXACT []
 synonym: "transport of peptides or proteins into other organism during symbiotic interaction" BROAD []
+xref: Reactome:R-HSA-9760173 "Secretion of toxins"
 is_a: GO:0044417 ! translocation of molecules into host
 
 [Term]
@@ -247449,7 +248006,6 @@
 namespace: molecular_function
 def: "Binds to and stops, prevents or reduces an ATP hydrolysis activity." [GOC:jl]
 synonym: "adenosinetriphosphatase inhibitor" EXACT []
-is_a: GO:0060590 ! ATPase regulator activity
 is_a: GO:0140678 ! molecular function inhibitor activity
 relationship: negatively_regulates GO:0140657 ! ATP-dependent activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
@@ -247619,7 +248175,7 @@
 id: GO:0042053
 name: regulation of dopamine metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:curators]
 synonym: "regulation of dopamine metabolism" EXACT []
 is_a: GO:0042069 ! regulation of catecholamine metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -247648,7 +248204,7 @@
 id: GO:0042056
 name: chemoattractant activity
 namespace: molecular_function
-def: "Providing the environmental signal that initiates the directed movement of a motile cell or organism towards a higher concentration of that signal." [GOC:go_curators, ISBN:0198506732]
+def: "Providing the environmental signal that initiates the directed movement of a motile cell or organism towards a higher concentration of that signal." [GOC:curators, ISBN:0198506732]
 subset: goslim_chembl
 subset: goslim_pir
 synonym: "attractant" BROAD []
@@ -247673,7 +248229,7 @@
 id: GO:0042058
 name: regulation of epidermal growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "regulation of EGF receptor signaling pathway" EXACT []
 synonym: "regulation of EGF receptor signalling pathway" EXACT []
 synonym: "regulation of EGFR signaling pathway" EXACT []
@@ -247686,7 +248242,7 @@
 id: GO:0042059
 name: negative regulation of epidermal growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "down regulation of epidermal growth factor receptor signaling pathway" EXACT []
 synonym: "down-regulation of epidermal growth factor receptor signaling pathway" EXACT []
 synonym: "downregulation of epidermal growth factor receptor signaling pathway" EXACT []
@@ -247767,7 +248323,7 @@
 id: GO:0042068
 name: obsolete regulation of pteridine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "regulation of pteridine metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30570" xsd:anyURI
@@ -247777,7 +248333,7 @@
 id: GO:0042069
 name: regulation of catecholamine metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving catecholamines." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving catecholamines." [GOC:curators]
 synonym: "regulation of catecholamine metabolism" EXACT []
 is_a: GO:0033238 ! regulation of amine metabolic process
 is_a: GO:0080090 ! regulation of primary metabolic process
@@ -247919,7 +248475,7 @@
 id: GO:0042081
 name: obsolete GSI anchor metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving glycosylsphingolipidinositol (GSI) anchors, which attach membrane proteins to the lipid bilayer of the cell membrane." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving glycosylsphingolipidinositol (GSI) anchors, which attach membrane proteins to the lipid bilayer of the cell membrane." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "GPI/GSI anchor metabolic process" BROAD []
 synonym: "GPI/GSI anchor metabolism" BROAD []
@@ -247931,7 +248487,7 @@
 id: GO:0042082
 name: obsolete GSI anchor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of a glycosylsphingolipidinositol (GSI) anchor that attaches some membrane proteins to the lipid bilayer of the cell membrane. The sphingolipid group is linked, via the C-6 hydroxyl residue of inositol to a carbohydrate chain which is itself linked to the protein via a ethanolamine phosphate group, its amino group forming an amide linkage with the C-terminal carboxyl of the protein. Some GSI anchors have variants on this canonical linkage." [GOC:go_curators, GOC:jsg]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of a glycosylsphingolipidinositol (GSI) anchor that attaches some membrane proteins to the lipid bilayer of the cell membrane. The sphingolipid group is linked, via the C-6 hydroxyl residue of inositol to a carbohydrate chain which is itself linked to the protein via a ethanolamine phosphate group, its amino group forming an amide linkage with the C-terminal carboxyl of the protein. Some GSI anchors have variants on this canonical linkage." [GOC:curators, GOC:jsg]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "GPI/GSI anchor biosynthesis" BROAD []
 synonym: "GPI/GSI anchor biosynthetic process" BROAD []
@@ -248607,12 +249163,13 @@
 
 [Term]
 id: GO:0042162
-name: telomeric DNA binding
+name: telomeric repeat DNA binding
 namespace: molecular_function
-def: "Binding to a telomere, a specific structure at the end of a linear chromosome required for the integrity and maintenance of the end." [GOC:jl, SO:0000624]
-synonym: "telomere binding" EXACT []
+def: "Any molecular function by which a gene product interacts selectively and non-covalently with a telomeric DNA repeat sequence." [GOC:vw, SO:0000624]
+synonym: "telomere binding" BROAD []
 synonym: "telomeric repeat binding" EXACT []
 is_a: GO:0043565 ! sequence-specific DNA binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31610" xsd:anyURI
 
 [Term]
 id: GO:0042163
@@ -248683,14 +249240,14 @@
 id: GO:0042169
 name: SH2 domain binding
 namespace: molecular_function
-def: "Binding to a SH2 domain (Src homology 2) of a protein, a protein domain of about 100 amino-acid residues and belonging to the alpha + beta domain class." [GOC:go_curators, Pfam:PF00017]
+def: "Binding to a SH2 domain (Src homology 2) of a protein, a protein domain of about 100 amino-acid residues and belonging to the alpha + beta domain class." [GOC:curators, Pfam:PF00017]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
 id: GO:0042170
 name: plastid membrane
 namespace: cellular_component
-def: "Either of the lipid bilayers that surround a plastid and form the plastid envelope." [GOC:mah]
+def: "Any lipid bilayer that surrounds a plastid or is part of the plastid envelope. Primary plastids (in plants, green algae, red algae, and glaucophytes) are surrounded by two membranes; complex plastids derived from secondary endosymbiosis are surrounded by three or four membranes." [GOC:mah, PMID:20124341]
 is_a: GO:0031090 ! organelle membrane
 intersection_of: GO:0016020 ! membrane
 intersection_of: part_of GO:0009536 ! plastid
@@ -248718,7 +249275,7 @@
 id: GO:0042174
 name: negative regulation of sporulation resulting in formation of a cellular spore
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sporulation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sporulation." [GOC:curators]
 synonym: "down regulation of sporulation" EXACT []
 synonym: "down-regulation of sporulation" EXACT []
 synonym: "downregulation of sporulation" EXACT []
@@ -248750,7 +249307,7 @@
 namespace: biological_process
 alt_id: GO:1903362
 alt_id: GO:2000598
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of a protein by the destruction of the native, active configuration, with or without the hydrolysis of peptide bonds." [GOC:go_curators, GOC:jl]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of a protein by the destruction of the native, active configuration, with or without the hydrolysis of peptide bonds." [GOC:curators, GOC:jl]
 synonym: "regulation of cellular protein breakdown" EXACT [GOC:TermGenie]
 synonym: "regulation of cellular protein catabolic process" EXACT []
 synonym: "regulation of cellular protein catabolism" EXACT [GOC:TermGenie]
@@ -248864,7 +249421,7 @@
 id: GO:0042181
 name: ketone biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of ketones, a class of organic compounds that contain the carbonyl group, CO, and in which the carbonyl group is bonded only to carbon atoms. The general formula for a ketone is RCOR, where R and R are alkyl or aryl groups." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of ketones, a class of organic compounds that contain the carbonyl group, CO, and in which the carbonyl group is bonded only to carbon atoms. The general formula for a ketone is RCOR, where R and R are alkyl or aryl groups." [GOC:curators]
 synonym: "ketone anabolism" EXACT []
 synonym: "ketone biosynthesis" EXACT []
 synonym: "ketone formation" EXACT []
@@ -248876,7 +249433,7 @@
 id: GO:0042182
 name: ketone catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of ketones, a class of organic compounds that contain the carbonyl group, CO, and in which the carbonyl group is bonded only to carbon atoms. The general formula for a ketone is RCOR, where R and R are alkyl or aryl groups." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of ketones, a class of organic compounds that contain the carbonyl group, CO, and in which the carbonyl group is bonded only to carbon atoms. The general formula for a ketone is RCOR, where R and R are alkyl or aryl groups." [GOC:curators]
 synonym: "ketone breakdown" EXACT []
 synonym: "ketone catabolism" EXACT []
 synonym: "ketone degradation" EXACT []
@@ -248900,7 +249457,7 @@
 id: GO:0042184
 name: xylene catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of xylene, a mixture of three colorless, aromatic hydrocarbon liquids, ortho-, meta- and para-xylene." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of xylene, a mixture of three colorless, aromatic hydrocarbon liquids, ortho-, meta- and para-xylene." [GOC:curators]
 synonym: "xylene breakdown" EXACT []
 synonym: "xylene catabolism" EXACT []
 synonym: "xylene degradation" EXACT []
@@ -248912,7 +249469,7 @@
 id: GO:0042185
 name: m-xylene catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of m-xylene, 1,3-dimethylbenzene, a colorless, liquid aromatic hydrocarbon." [GOC:go_curators, GOC:jl]
+def: "The chemical reactions and pathways resulting in the breakdown of m-xylene, 1,3-dimethylbenzene, a colorless, liquid aromatic hydrocarbon." [GOC:curators, GOC:jl]
 synonym: "m-xylene breakdown" EXACT []
 synonym: "m-xylene catabolism" EXACT []
 synonym: "m-xylene degradation" EXACT []
@@ -249029,7 +249586,7 @@
 id: GO:0042194
 name: quinate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of quinate, the anion of quinic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of quinate, the anion of quinic acid." [GOC:curators]
 synonym: "quinate anabolism" EXACT []
 synonym: "quinate biosynthesis" EXACT []
 synonym: "quinate formation" EXACT []
@@ -249305,21 +249862,21 @@
 
 [Term]
 id: GO:0042217
-name: 1-aminocyclopropane-1-carboxylate catabolic process
+name: obsolete 1-aminocyclopropane-1-carboxylate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:curators]
+comment: This term was obsoleted because there is no evidence that a specific pathway for 1-aminocyclopropane-1-carboxylate degradation exists.
 synonym: "1-aminocyclopropane-1-carboxylate breakdown" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate catabolism" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate degradation" EXACT []
-is_a: GO:0018871 ! 1-aminocyclopropane-1-carboxylate metabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
-is_a: GO:1901606 ! alpha-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0042218
 name: 1-aminocyclopropane-1-carboxylate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of 1-aminocyclopropane-1-carboxylate, a natural product found in plant tissues. It is a key intermediate in the biosynthesis of ethylene (ethene), a fruit-ripening hormone in plants." [GOC:curators]
 synonym: "1-aminocyclopropane-1-carboxylate anabolism" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate biosynthesis" EXACT []
 synonym: "1-aminocyclopropane-1-carboxylate formation" EXACT []
@@ -249895,6 +250452,7 @@
 def: "Catalysis of the transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y = Y-ISG15 + X, where both X-ISG15 and Y-ISG15 are covalent linkages." [GOC:mah, PMID:12826404]
 synonym: "ISG15 conjugating enzyme activity" NARROW []
 xref: Reactome:R-HSA-5653754 "UBE2L6:TRIM25 ISGylates monoUb:K164-PCNA"
+xref: Reactome:R-HSA-9927247 "ISGylation of DDX58 (RIG-I)"
 is_a: GO:0019787 ! ubiquitin-like protein transferase activity
 
 [Term]
@@ -249966,7 +250524,7 @@
 id: GO:0042304
 name: regulation of fatty acid biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids, any of the aliphatic monocarboxylic acids that can be liberated by hydrolysis from naturally occurring fats and oils." [GOC:go_curators, GOC:jl]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids, any of the aliphatic monocarboxylic acids that can be liberated by hydrolysis from naturally occurring fats and oils." [GOC:curators, GOC:jl]
 synonym: "regulation of fatty acid anabolism" EXACT []
 synonym: "regulation of fatty acid biosynthesis" EXACT []
 synonym: "regulation of fatty acid formation" EXACT []
@@ -250155,7 +250713,7 @@
 id: GO:0042321
 name: negative regulation of circadian sleep/wake cycle, sleep
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the duration or quality of sleep, a readily reversible state of reduced awareness and metabolic activity that occurs periodically in many animals." [GOC:go_curators, GOC:jl, ISBN:0192800981]
+def: "Any process that stops, prevents or reduces the duration or quality of sleep, a readily reversible state of reduced awareness and metabolic activity that occurs periodically in many animals." [GOC:curators, GOC:jl, ISBN:0192800981]
 synonym: "down regulation of circadian sleep/wake cycle, sleep" EXACT []
 synonym: "down-regulation of circadian sleep/wake cycle, sleep" EXACT []
 synonym: "downregulation of circadian sleep/wake cycle, sleep" EXACT []
@@ -250172,7 +250730,7 @@
 id: GO:0042322
 name: negative regulation of circadian sleep/wake cycle, REM sleep
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the duration or quality of rapid eye movement (REM) sleep." [GOC:go_curators, GOC:jl]
+def: "Any process that stops, prevents or reduces the duration or quality of rapid eye movement (REM) sleep." [GOC:curators, GOC:jl]
 synonym: "down regulation of circadian sleep/wake cycle, REM sleep" EXACT []
 synonym: "down-regulation of circadian sleep/wake cycle, REM sleep" EXACT []
 synonym: "downregulation of circadian sleep/wake cycle, REM sleep" EXACT []
@@ -250618,7 +251176,7 @@
 synonym: "tocopherol metabolic process" EXACT []
 synonym: "tocopherol metabolism" EXACT []
 synonym: "vitamin E metabolism" EXACT []
-xref: Reactome:R-HSA-8877627 "Vitamin E"
+xref: Reactome:R-HSA-8877627 "Vitamin E transport"
 is_a: GO:0008152 ! metabolic process
 
 [Term]
@@ -250647,6 +251205,7 @@
 synonym: "fat-soluble vitamin biosynthesis" EXACT []
 synonym: "fat-soluble vitamin formation" EXACT []
 synonym: "fat-soluble vitamin synthesis" EXACT []
+xref: Reactome:R-HSA-6806667 "Metabolism of fat-soluble vitamins"
 is_a: GO:0009110 ! vitamin biosynthetic process
 
 [Term]
@@ -250668,6 +251227,7 @@
 synonym: "water-soluble vitamin biosynthesis" EXACT []
 synonym: "water-soluble vitamin formation" EXACT []
 synonym: "water-soluble vitamin synthesis" EXACT []
+xref: Reactome:R-HSA-196849 "Metabolism of water-soluble vitamins and cofactors"
 is_a: GO:0009110 ! vitamin biosynthetic process
 
 [Term]
@@ -250684,7 +251244,7 @@
 id: GO:0042366
 name: cobalamin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom." [GOC:curators]
 synonym: "cobalamin breakdown" EXACT []
 synonym: "cobalamin catabolism" EXACT []
 synonym: "cobalamin degradation" EXACT []
@@ -251086,7 +251646,7 @@
 id: GO:0042402
 name: biogenic amine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways occurring at the level of individual cells resulting in the breakdown of biogenic amines, any of a group of naturally occurring, biologically active amines, such as norepinephrine, histamine, and serotonin, many of which act as neurotransmitters." [GOC:go_curators, GOC:jl, ISBN:0198506732]
+def: "The chemical reactions and pathways occurring at the level of individual cells resulting in the breakdown of biogenic amines, any of a group of naturally occurring, biologically active amines, such as norepinephrine, histamine, and serotonin, many of which act as neurotransmitters." [GOC:curators, GOC:jl, ISBN:0198506732]
 synonym: "biogenic amine breakdown" EXACT []
 synonym: "biogenic amine catabolism" EXACT []
 synonym: "biogenic amine degradation" EXACT []
@@ -251190,7 +251750,6 @@
 xref: MetaCyc:6-CARBOXYHEXANOATE--COA-LIGASE-RXN
 xref: RHEA:14781
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.14
 property_value: skos:exactMatch RHEA:14781
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -251204,8 +251763,15 @@
 synonym: "taurine biosynthesis" EXACT []
 synonym: "taurine formation" EXACT []
 synonym: "taurine synthesis" EXACT []
+xref: MetaCyc:PWY-5331
+xref: MetaCyc:PWY-7850
+xref: MetaCyc:PWY-8359
 is_a: GO:0019530 ! taurine metabolic process
 is_a: GO:0046305 ! alkanesulfonate biosynthetic process
+property_value: skos:narrowMatch MetaCyc:PWY-5331
+property_value: skos:narrowMatch MetaCyc:PWY-7850
+property_value: skos:narrowMatch MetaCyc:PWY-8359
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
 
 [Term]
 id: GO:0042413
@@ -251385,6 +251951,7 @@
 xref: MetaCyc:CHOLINE-BETAINE-ANA-PWY
 xref: MetaCyc:P542-PWY
 xref: MetaCyc:PWY-3721
+xref: MetaCyc:PWY-7167
 xref: MetaCyc:PWY-7494
 xref: Reactome:R-HSA-6798163 "Choline catabolism"
 is_a: GO:0019695 ! choline metabolic process
@@ -251392,9 +251959,11 @@
 property_value: skos:narrowMatch MetaCyc:CHOLINE-BETAINE-ANA-PWY
 property_value: skos:narrowMatch MetaCyc:P542-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3721
+property_value: skos:narrowMatch MetaCyc:PWY-7167
 property_value: skos:narrowMatch MetaCyc:PWY-7494
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30537" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0042427
@@ -251666,7 +252235,7 @@
 id: GO:0042451
 name: purine nucleoside biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of any purine nucleoside, one of a family of organic molecules consisting of a purine base covalently bonded to a sugar ribose (a ribonucleoside) or deoxyribose (a deoxyribonucleoside)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of any purine nucleoside, one of a family of organic molecules consisting of a purine base covalently bonded to a sugar ribose (a ribonucleoside) or deoxyribose (a deoxyribonucleoside)." [GOC:curators]
 synonym: "purine nucleoside anabolism" EXACT []
 synonym: "purine nucleoside biosynthesis" EXACT []
 synonym: "purine nucleoside formation" EXACT []
@@ -251736,7 +252305,7 @@
 id: GO:0042458
 name: obsolete nopaline catabolic process to proline
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of nopaline into other compounds, including proline." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of nopaline into other compounds, including proline." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "nopaline breakdown to proline" EXACT []
 synonym: "nopaline degradation to proline" EXACT []
@@ -251747,7 +252316,7 @@
 id: GO:0042459
 name: obsolete octopine catabolic process to proline
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of octopine into other compounds, including proline." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of octopine into other compounds, including proline." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "octopine breakdown to proline" EXACT []
 synonym: "octopine degradation to proline" EXACT []
@@ -251759,7 +252328,7 @@
 name: photoreceptor cell development
 namespace: biological_process
 alt_id: GO:0046531
-def: "Development of a photoreceptor, a cell that responds to incident electromagnetic radiation, particularly visible light." [GOC:go_curators]
+def: "Development of a photoreceptor, a cell that responds to incident electromagnetic radiation, particularly visible light." [GOC:curators]
 synonym: "photoreceptor morphogenesis" EXACT []
 is_a: GO:0048666 ! neuron development
 relationship: part_of GO:0046530 ! photoreceptor cell differentiation
@@ -252448,14 +253017,14 @@
 id: GO:0042549
 name: photosystem II stabilization
 namespace: biological_process
-def: "The stabilization of the photosystem II protein complex, resulting from the phosphorylation of its structural protein subunits, in a cell actively involved in photosynthesis." [GOC:go_curators]
+def: "The stabilization of the photosystem II protein complex, resulting from the phosphorylation of its structural protein subunits, in a cell actively involved in photosynthesis." [GOC:curators]
 is_a: GO:0042548 ! regulation of photosynthesis, light reaction
 
 [Term]
 id: GO:0042550
 name: photosystem I stabilization
 namespace: biological_process
-def: "The stabilization of the photosystem I protein complex, resulting from the phosphorylation of its structural protein subunits, in a cell actively involved in photosynthesis." [GOC:go_curators]
+def: "The stabilization of the photosystem I protein complex, resulting from the phosphorylation of its structural protein subunits, in a cell actively involved in photosynthesis." [GOC:curators]
 is_a: GO:0042548 ! regulation of photosynthesis, light reaction
 
 [Term]
@@ -252692,7 +253261,7 @@
 id: GO:0042576
 name: obsolete aspartyl aminopeptidase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the release of an N-terminal aspartate or glutamate from a peptide, with a preference for aspartate." [GOC:curator]
+def: "OBSOLETE. Catalysis of the release of an N-terminal aspartate or glutamate from a peptide, with a preference for aspartate." [GOC:curators]
 comment: This term was made obsolete because it represents a gene product.
 synonym: "aspartyl aminopeptidase activity" EXACT []
 is_obsolete: true
@@ -252852,14 +253421,14 @@
 id: GO:0042593
 name: glucose homeostasis
 namespace: biological_process
-def: "Any process involved in the maintenance of an internal steady state of glucose within an organism or cell." [GOC:go_curators]
+def: "Any process involved in the maintenance of an internal steady state of glucose within an organism or cell." [GOC:curators]
 is_a: GO:0033500 ! carbohydrate homeostasis
 
 [Term]
 id: GO:0042594
 name: response to starvation
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a starvation stimulus, deprivation of nourishment." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a starvation stimulus, deprivation of nourishment." [GOC:curators]
 subset: goslim_yeast
 is_a: GO:0006950 ! response to stress
 is_a: GO:0031667 ! response to nutrient levels
@@ -252868,7 +253437,7 @@
 id: GO:0042595
 name: behavioral response to starvation
 namespace: biological_process
-def: "Any process that results in a change in the behavior of an organism as a result of deprivation of nourishment." [GOC:go_curators]
+def: "Any process that results in a change in the behavior of an organism as a result of deprivation of nourishment." [GOC:curators]
 synonym: "behavioural response to starvation" EXACT []
 is_a: GO:0042594 ! response to starvation
 
@@ -252877,7 +253446,7 @@
 name: fear response
 namespace: biological_process
 alt_id: GO:0001663
-def: "The response of an organism to a perceived external threat." [GOC:go_curators]
+def: "The response of an organism to a perceived external threat." [GOC:curators]
 synonym: "physiological fear response" EXACT []
 is_a: GO:0033555 ! multicellular organismal response to stress
 
@@ -252886,7 +253455,7 @@
 name: periplasmic space
 namespace: cellular_component
 alt_id: GO:0005620
-def: "The region between the inner (cytoplasmic) and outer membrane (Gram-negative Bacteria) or cytoplasmic membrane and cell wall (Fungi and Gram-positive Bacteria)." [GOC:go_curators, GOC:md]
+def: "The region between the inner (cytoplasmic) and outer membrane (Gram-negative Bacteria) or cytoplasmic membrane and cell wall (Fungi and Gram-positive Bacteria)." [GOC:curators, GOC:md]
 subset: goslim_metagenomics
 subset: goslim_pir
 subset: goslim_prokaryote
@@ -253160,10 +253729,12 @@
 synonym: "P-P-bond-hydrolysis-driven transmembrane transporter activity" BROAD []
 synonym: "P-P-bond-hydrolysis-driven transporter" BROAD []
 xref: EC:7.2.2.-
+xref: Reactome:R-HSA-1454916 "The ABCC family mediates organic anion transport"
 xref: Reactome:R-HSA-1467457 "ABCA8,B1,B5 transport xenobiotics from cytosol to extracellular region"
 xref: Reactome:R-HSA-2161506 "abacavir [cytosol] + ATP + H2O => abacavir[extracellular] + ADP + phosphate"
 xref: Reactome:R-HSA-2161538 "abacavir [cytosol] + ATP + H2O => abacavir[extracellular] + ADP + phosphate"
 xref: Reactome:R-HSA-390393 "Peroxisomal uptake of very long-chain fatty acyl CoA"
+xref: Reactome:R-HSA-5690340 "Defective ABCC6 does not transport organic anion from cytosol to extracellular region"
 xref: Reactome:R-HSA-9033499 "PEX1:PEX6:PEX26:ZFAND6 dissociates Ub:PEX5L and PEX7 from PEX14:PEX13:PEX2:PEX10:PEX12 and translocates PEX5L and PEX7 from the peroxisomal membrane to the cytosol"
 xref: Reactome:R-HSA-9033505 "PEX1:PEX6:PEX26:ZFAND6:Ub:PEX5S,L:PEX14:PEX13:PEX2:PEX10:PEX12 dissociates yielding cytosolic Ub:PEX5S,L and membrane PEX14:PEX13:PEX2:PEX10:PEX12"
 xref: Reactome:R-HSA-9659680 "ABCB1 transports xenobiotics out of the cell"
@@ -253228,7 +253799,7 @@
 id: GO:0042631
 name: cellular response to water deprivation
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of deprivation of water." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of deprivation of water." [GOC:curators]
 synonym: "cellular response to drought" EXACT []
 is_a: GO:0009414 ! response to water deprivation
 is_a: GO:0033554 ! cellular response to stress
@@ -253240,7 +253811,7 @@
 namespace: biological_process
 alt_id: GO:2000188
 alt_id: GO:2000189
-def: "Any process involved in the maintenance of an internal steady state of cholesterol within an organism or cell." [GOC:go_curators]
+def: "Any process involved in the maintenance of an internal steady state of cholesterol within an organism or cell." [GOC:curators]
 synonym: "positive regulation of cholesterol homeostasis" RELATED []
 synonym: "regulation of cholesterol homeostasis" RELATED []
 is_a: GO:0055092 ! sterol homeostasis
@@ -253249,14 +253820,14 @@
 id: GO:0042633
 name: hair cycle
 namespace: biological_process
-def: "The cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair; one of the collection or mass of filaments growing from the skin of an animal, and forming a covering for a part of the head or for any part or the whole of the body." [GOC:go_curators, PMID:12230507]
+def: "The cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair; one of the collection or mass of filaments growing from the skin of an animal, and forming a covering for a part of the head or for any part or the whole of the body." [GOC:curators, PMID:12230507]
 is_a: GO:0042303 ! molting cycle
 
 [Term]
 id: GO:0042634
 name: regulation of hair cycle
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:go_curators, PMID:12230507]
+def: "Any process that modulates the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:curators, PMID:12230507]
 is_a: GO:0051239 ! regulation of multicellular organismal process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0042633 ! hair cycle
@@ -253266,7 +253837,7 @@
 id: GO:0042635
 name: positive regulation of hair cycle
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:go_curators, PMID:12230507]
+def: "Any process that activates or increases the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:curators, PMID:12230507]
 synonym: "activation of hair cycle" NARROW []
 synonym: "stimulation of hair cycle" NARROW []
 synonym: "up regulation of hair cycle" EXACT []
@@ -253282,7 +253853,7 @@
 id: GO:0042636
 name: negative regulation of hair cycle
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:go_curators, PMID:12230507]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the cyclical phases of growth (anagen), regression (catagen), quiescence (telogen), and shedding (exogen) in the life of a hair." [GOC:curators, PMID:12230507]
 synonym: "down regulation of hair cycle" EXACT []
 synonym: "down-regulation of hair cycle" EXACT []
 synonym: "downregulation of hair cycle" EXACT []
@@ -253340,7 +253911,7 @@
 id: GO:0042641
 name: actomyosin
 namespace: cellular_component
-def: "Any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
+def: "Any complex of actin, myosin, and accessory proteins." [GOC:curators]
 synonym: "actomyosin complex" EXACT []
 synonym: "actomyosin structure" EXACT []
 is_a: GO:0110165 ! cellular anatomical structure
@@ -253350,7 +253921,7 @@
 id: GO:0042642
 name: actomyosin, myosin complex part
 namespace: cellular_component
-def: "The myosin part of any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
+def: "The myosin part of any complex of actin, myosin, and accessory proteins." [GOC:curators]
 is_a: GO:0016459 ! myosin complex
 intersection_of: GO:0016459 ! myosin complex
 intersection_of: part_of GO:0042641 ! actomyosin
@@ -253360,7 +253931,7 @@
 id: GO:0042643
 name: obsolete actomyosin, actin portion
 namespace: cellular_component
-def: "OBSOLETE. The actin part of any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
+def: "OBSOLETE. The actin part of any complex of actin, myosin, and accessory proteins." [GOC:curators]
 comment: This term was obsoleted because it corresponds to a gene product, actin.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19570" xsd:anyURI
 is_obsolete: true
@@ -253516,7 +254087,7 @@
 id: GO:0042659
 name: regulation of cell fate specification
 namespace: biological_process
-def: "Any process that mediates the adoption of a specific fate by a cell." [GOC:go_curators]
+def: "Any process that mediates the adoption of a specific fate by a cell." [GOC:curators]
 is_a: GO:0010453 ! regulation of cell fate commitment
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0001708 ! cell fate specification
@@ -253526,7 +254097,7 @@
 id: GO:0042660
 name: positive regulation of cell fate specification
 namespace: biological_process
-def: "Any process that activates or enables a cell to adopt a specific fate." [GOC:go_curators]
+def: "Any process that activates or enables a cell to adopt a specific fate." [GOC:curators]
 synonym: "activation of cell fate specification" NARROW []
 synonym: "stimulation of cell fate specification" NARROW []
 synonym: "up regulation of cell fate specification" EXACT []
@@ -253542,7 +254113,7 @@
 id: GO:0042661
 name: regulation of mesodermal cell fate specification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of mesoderm cell fate specification." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of mesoderm cell fate specification." [GOC:curators]
 is_a: GO:0042659 ! regulation of cell fate specification
 is_a: GO:1905770 ! regulation of mesodermal cell differentiation
 is_a: GO:2000027 ! regulation of animal organ morphogenesis
@@ -253554,7 +254125,7 @@
 id: GO:0042662
 name: negative regulation of mesodermal cell fate specification
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of mesoderm cell fate specification." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of mesoderm cell fate specification." [GOC:curators]
 synonym: "down regulation of mesodermal cell fate specification" EXACT []
 synonym: "down-regulation of mesodermal cell fate specification" EXACT []
 synonym: "downregulation of mesodermal cell fate specification" EXACT []
@@ -253571,7 +254142,7 @@
 id: GO:0042663
 name: regulation of endodermal cell fate specification
 namespace: biological_process
-def: "Any process that mediates the specification of a cell into an endoderm cell." [GOC:go_curators]
+def: "Any process that mediates the specification of a cell into an endoderm cell." [GOC:curators]
 synonym: "regulation of endoderm cell fate specification" EXACT []
 is_a: GO:0042659 ! regulation of cell fate specification
 is_a: GO:1903224 ! regulation of endodermal cell differentiation
@@ -253583,7 +254154,7 @@
 id: GO:0042664
 name: negative regulation of endodermal cell fate specification
 namespace: biological_process
-def: "Any process that restricts, stops or prevents a cell from specifying into an endoderm cell." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from specifying into an endoderm cell." [GOC:curators]
 synonym: "down regulation of endodermal cell fate specification" EXACT []
 synonym: "down-regulation of endodermal cell fate specification" EXACT []
 synonym: "downregulation of endodermal cell fate specification" EXACT []
@@ -253602,7 +254173,7 @@
 id: GO:0042665
 name: regulation of ectodermal cell fate specification
 namespace: biological_process
-def: "Any process that mediates the specification of a cell into an ectoderm cell." [GOC:go_curators]
+def: "Any process that mediates the specification of a cell into an ectoderm cell." [GOC:curators]
 synonym: "regulation of ectoderm cell fate specification" EXACT []
 is_a: GO:0042659 ! regulation of cell fate specification
 intersection_of: GO:0065007 ! biological regulation
@@ -253613,7 +254184,7 @@
 id: GO:0042666
 name: negative regulation of ectodermal cell fate specification
 namespace: biological_process
-def: "Any process that restricts, stops or prevents a cell from specifying into an ectoderm cell." [GOC:go_curators]
+def: "Any process that restricts, stops or prevents a cell from specifying into an ectoderm cell." [GOC:curators]
 synonym: "down regulation of ectodermal cell fate specification" EXACT []
 synonym: "down-regulation of ectodermal cell fate specification" EXACT []
 synonym: "downregulation of ectodermal cell fate specification" EXACT []
@@ -253631,7 +254202,7 @@
 id: GO:0042667
 name: auditory receptor cell fate specification
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into an auditory hair cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into an auditory hair cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:curators]
 synonym: "auditory hair cell fate specification" EXACT []
 is_a: GO:0048665 ! neuron fate specification
 relationship: part_of GO:0009912 ! auditory receptor cell fate commitment
@@ -253640,7 +254211,7 @@
 id: GO:0042668
 name: auditory receptor cell fate determination
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into an auditory hair cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into an auditory hair cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 synonym: "auditory hair cell fate determination" EXACT []
 is_a: GO:0048664 ! neuron fate determination
 relationship: part_of GO:0009912 ! auditory receptor cell fate commitment
@@ -253649,7 +254220,7 @@
 id: GO:0042669
 name: regulation of inner ear auditory receptor cell fate specification
 namespace: biological_process
-def: "Any process that mediates the specification of a cell into an auditory hair cell." [GOC:go_curators]
+def: "Any process that mediates the specification of a cell into an auditory hair cell." [GOC:curators]
 synonym: "regulation of auditory hair cell fate specification" EXACT []
 is_a: GO:0042659 ! regulation of cell fate specification
 is_a: GO:0045607 ! regulation of inner ear auditory receptor cell differentiation
@@ -253661,14 +254232,14 @@
 id: GO:0042670
 name: retinal cone cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires the specialized features of a retinal cone cell." [GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires the specialized features of a retinal cone cell." [GOC:curators]
 is_a: GO:0060219 ! camera-type eye photoreceptor cell differentiation
 
 [Term]
 id: GO:0042671
 name: retinal cone cell fate determination
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into a retinal cone cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into a retinal cone cell regardless of its environment; upon determination, the cell fate cannot be reversed." [GOC:curators]
 is_a: GO:0043703 ! photoreceptor cell fate determination
 relationship: part_of GO:0046551 ! retinal cone cell fate commitment
 
@@ -253676,7 +254247,7 @@
 id: GO:0042672
 name: retinal cone cell fate specification
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into a retinal cone cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into a retinal cone cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [GOC:curators]
 is_a: GO:0043704 ! photoreceptor cell fate specification
 relationship: part_of GO:0046551 ! retinal cone cell fate commitment
 
@@ -253684,7 +254255,7 @@
 id: GO:0042673
 name: regulation of retinal cone cell fate specification
 namespace: biological_process
-def: "Any process that mediates the specification of a cell into a retinal cone cell." [GOC:go_curators]
+def: "Any process that mediates the specification of a cell into a retinal cone cell." [GOC:curators]
 is_a: GO:0042659 ! regulation of cell fate specification
 is_a: GO:0060222 ! regulation of retinal cone cell fate commitment
 intersection_of: GO:0065007 ! biological regulation
@@ -253759,7 +254330,7 @@
 id: GO:0042684
 name: cardioblast cell fate commitment
 namespace: biological_process
-def: "The process in which a cell becomes committed to becoming a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "The process in which a cell becomes committed to becoming a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 is_a: GO:0060911 ! cardiac cell fate commitment
 relationship: part_of GO:0010002 ! cardioblast differentiation
 
@@ -253767,7 +254338,7 @@
 id: GO:0042685
 name: cardioblast cell fate specification
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into a cardioblast cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into a cardioblast cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 is_a: GO:0060912 ! cardiac cell fate specification
 relationship: part_of GO:0042684 ! cardioblast cell fate commitment
 
@@ -253775,7 +254346,7 @@
 id: GO:0042686
 name: regulation of cardioblast cell fate specification
 namespace: biological_process
-def: "Any process that mediates the specification of a cell into a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:go_curators]
+def: "Any process that mediates the specification of a cell into a cardioblast. A cardioblast is a cardiac precursor cell. It is a cell that has been committed to a cardiac fate, but will undergo more cell division rather than terminally differentiating." [GOC:curators]
 is_a: GO:0051890 ! regulation of cardioblast differentiation
 is_a: GO:2000043 ! regulation of cardiac cell fate specification
 intersection_of: GO:0065007 ! biological regulation
@@ -253793,7 +254364,7 @@
 id: GO:0042689
 name: regulation of crystal cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of crystal cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of crystal cell differentiation." [GOC:curators]
 is_a: GO:0045610 ! regulation of hemocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0042688 ! crystal cell differentiation
@@ -253803,7 +254374,7 @@
 id: GO:0042690
 name: negative regulation of crystal cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of crystal cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of crystal cell differentiation." [GOC:curators]
 synonym: "down regulation of crystal cell differentiation" EXACT []
 synonym: "down-regulation of crystal cell differentiation" EXACT []
 synonym: "downregulation of crystal cell differentiation" EXACT []
@@ -253818,7 +254389,7 @@
 id: GO:0042691
 name: positive regulation of crystal cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of crystal cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of crystal cell differentiation." [GOC:curators]
 synonym: "activation of crystal cell differentiation" NARROW []
 synonym: "stimulation of crystal cell differentiation" NARROW []
 synonym: "up regulation of crystal cell differentiation" EXACT []
@@ -253834,7 +254405,7 @@
 id: GO:0042692
 name: muscle cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires specialized features of a muscle cell." [CL:0000187, GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires specialized features of a muscle cell." [CL:0000187, GOC:curators]
 synonym: "myogenesis" RELATED []
 is_a: GO:0030154 ! cell differentiation
 relationship: part_of GO:0061061 ! muscle structure development
@@ -253843,7 +254414,7 @@
 id: GO:0042693
 name: muscle cell fate commitment
 namespace: biological_process
-def: "The process in which the cellular identity of muscle cells is acquired and determined." [CL:0000187, GOC:go_curators]
+def: "The process in which the cellular identity of muscle cells is acquired and determined." [CL:0000187, GOC:curators]
 is_a: GO:0045165 ! cell fate commitment
 relationship: part_of GO:0042692 ! muscle cell differentiation
 
@@ -253851,7 +254422,7 @@
 id: GO:0042694
 name: muscle cell fate specification
 namespace: biological_process
-def: "The process in which a cell becomes capable of differentiating autonomously into a muscle cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [CL:0000187, GOC:go_curators]
+def: "The process in which a cell becomes capable of differentiating autonomously into a muscle cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed." [CL:0000187, GOC:curators]
 is_a: GO:0001708 ! cell fate specification
 relationship: part_of GO:0042693 ! muscle cell fate commitment
 
@@ -253954,7 +254525,7 @@
 id: GO:0042705
 name: ocellus photoreceptor cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires specialized features of a photoreceptor cell found in the ocellus." [GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires specialized features of a photoreceptor cell found in the ocellus." [GOC:curators]
 is_a: GO:0046530 ! photoreceptor cell differentiation
 
 [Term]
@@ -253998,6 +254569,7 @@
 subset: gocheck_do_not_annotate
 subset: goslim_candida
 subset: goslim_pir
+xref: Reactome:R-HSA-9931953 "Biofilm formation"
 is_a: GO:0098630 ! aggregation of unicellular organisms
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17727" xsd:anyURI
 
@@ -254013,7 +254585,7 @@
 id: GO:0042712
 name: paternal behavior
 namespace: biological_process
-def: "Male behaviors associated with the care and rearing offspring." [GOC:go_curators]
+def: "Male behaviors associated with the care and rearing offspring." [GOC:curators]
 synonym: "paternal behaviour" EXACT []
 is_a: GO:0060746 ! parental behavior
 
@@ -254450,7 +255022,7 @@
 id: GO:0042753
 name: positive regulation of circadian rhythm
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of a circadian rhythm behavior." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of a circadian rhythm behavior." [GOC:curators]
 synonym: "activation of circadian rhythm" NARROW []
 synonym: "stimulation of circadian rhythm" NARROW []
 synonym: "up regulation of circadian rhythm" EXACT []
@@ -254466,7 +255038,7 @@
 id: GO:0042754
 name: negative regulation of circadian rhythm
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of a circadian rhythm behavior." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of a circadian rhythm behavior." [GOC:curators]
 synonym: "down regulation of circadian rhythm" EXACT []
 synonym: "down-regulation of circadian rhythm" EXACT []
 synonym: "downregulation of circadian rhythm" EXACT []
@@ -254566,7 +255138,7 @@
 id: GO:0042762
 name: regulation of sulfur metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving sulfur, the nonmetallic element sulfur or compounds that contain sulfur." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving sulfur, the nonmetallic element sulfur or compounds that contain sulfur." [GOC:curators]
 synonym: "regulation of sulfur metabolism" EXACT []
 synonym: "regulation of sulphur metabolic process" EXACT []
 synonym: "regulation of sulphur metabolism" EXACT []
@@ -254587,7 +255159,7 @@
 id: GO:0042764
 name: ascospore-type prospore
 namespace: cellular_component
-def: "An immature spore undergoing development. The spore usually consists of nucleic acid, prospore membrane(s) that encase the nucleic acid, and ultimately a cell wall that covers the membrane(s). This type of spore is observed in ascospore-forming fungi." [GOC:go_curators]
+def: "An immature spore undergoing development. The spore usually consists of nucleic acid, prospore membrane(s) that encase the nucleic acid, and ultimately a cell wall that covers the membrane(s). This type of spore is observed in ascospore-forming fungi." [GOC:curators]
 is_a: GO:0042763 ! intracellular immature spore
 
 [Term]
@@ -254664,7 +255236,7 @@
 id: GO:0042769
 name: obsolete DNA damage response, detection of DNA damage
 namespace: biological_process
-def: "OBSOLETE. The series of events required to receive a stimulus indicating DNA damage has occurred and convert it to a molecular signal." [GOC:go_curators]
+def: "OBSOLETE. The series of events required to receive a stimulus indicating DNA damage has occurred and convert it to a molecular signal." [GOC:curators]
 comment: This term has been obsoleted because it represents a molecular function.
 synonym: "detection of DNA damage during DNA damage response" EXACT []
 synonym: "DNA damage response, perception of DNA damage" RELATED []
@@ -254679,7 +255251,7 @@
 name: signal transduction in response to DNA damage
 namespace: biological_process
 alt_id: GO:0006975
-def: "A cascade of processes induced by the detection of DNA damage within a cell." [GOC:go_curators]
+def: "A cascade of processes induced by the detection of DNA damage within a cell." [GOC:curators]
 synonym: "DNA damage induced protein phosphorylation" NARROW []
 synonym: "DNA damage response, signal transduction" EXACT []
 synonym: "response to DNA damage stimulus by intracellular signaling cascade" EXACT [GOC:dph, GOC:tb]
@@ -254691,7 +255263,7 @@
 id: GO:0042771
 name: intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator
 namespace: biological_process
-def: "The series of molecular signals in which an intracellular signal is conveyed to trigger the apoptotic death of a cell. The pathway is induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage, and ends when the execution phase of apoptosis is triggered." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "The series of molecular signals in which an intracellular signal is conveyed to trigger the apoptotic death of a cell. The pathway is induced by the cell cycle regulator phosphoprotein p53, or an equivalent protein, in response to the detection of DNA damage, and ends when the execution phase of apoptosis is triggered." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "DNA damage response, signal transduction by p53 class mediator resulting in induction of apoptosis" RELATED []
 is_a: GO:0008630 ! intrinsic apoptotic signaling pathway in response to DNA damage
 is_a: GO:0072332 ! intrinsic apoptotic signaling pathway by p53 class mediator
@@ -254700,7 +255272,7 @@
 id: GO:0042772
 name: DNA damage response, signal transduction resulting in transcription
 namespace: biological_process
-def: "A cascade of processes initiated in response to the detection of DNA damage, and resulting in the induction of transcription." [GOC:go_curators]
+def: "A cascade of processes initiated in response to the detection of DNA damage, and resulting in the induction of transcription." [GOC:curators]
 is_a: GO:0042770 ! signal transduction in response to DNA damage
 
 [Term]
@@ -254762,14 +255334,14 @@
 id: GO:0042778
 name: tRNA end turnover
 namespace: biological_process
-def: "The process in which the 3'-terminal CCA of a tRNA is removed and restored. This often happens to uncharged tRNA." [GOC:go_curators]
+def: "The process in which the 3'-terminal CCA of a tRNA is removed and restored. This often happens to uncharged tRNA." [GOC:curators]
 is_a: GO:0006399 ! tRNA metabolic process
 
 [Term]
 id: GO:0042779
 name: obsolete tRNA 3'-trailer cleavage
 namespace: biological_process
-def: "OBSOLETE. Cleavage of the 3'-end of the pre-tRNA as part of the process of generating the mature 3'-end of the tRNA; may involve endonucleolytic or exonucleolytic cleavage, or both." [GOC:go_curators]
+def: "OBSOLETE. Cleavage of the 3'-end of the pre-tRNA as part of the process of generating the mature 3'-end of the tRNA; may involve endonucleolytic or exonucleolytic cleavage, or both." [GOC:curators]
 comment: The reason for obsoletion is that the term represents a molecular function.
 synonym: "removal of tRNA 3'-trailer sequence" EXACT []
 synonym: "tRNA 3'-end cleavage" EXACT []
@@ -254781,7 +255353,7 @@
 id: GO:0042780
 name: tRNA 3'-end processing
 namespace: biological_process
-def: "The process in which the 3' end of a pre-tRNA molecule is converted to that of a mature tRNA." [GOC:go_curators]
+def: "The process in which the 3' end of a pre-tRNA molecule is converted to that of a mature tRNA." [GOC:curators]
 synonym: "tRNA 3' processing" EXACT []
 is_a: GO:0008033 ! tRNA processing
 is_a: GO:0031123 ! RNA 3'-end processing
@@ -255044,13 +255616,14 @@
 synonym: "histone methyltransferase activity (H3-K4 specific)" EXACT []
 synonym: "histone-H3K4 methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-1214188 "PRDM9 trimethylates histone H3"
-xref: Reactome:R-HSA-3364026 "SET1 complex trimethylates H3K4 at the MYC gene"
+xref: Reactome:R-HSA-3364026 "MLL2 complex trimethylates H3K4 at the MYC gene"
 xref: Reactome:R-HSA-8865498 "KMT2A trimethylates nucleosomes at the SPI1 gene locus producing H3K4Me3 mark"
 xref: Reactome:R-HSA-8936481 "Core MLL complex methylates H3K4Me2-Nucleosome at the ITGA2B gene promoter"
 xref: Reactome:R-HSA-8936621 "Core MLL complex methylates H3K4Me2-Nucleosome at the GP1BA gene promoter"
 xref: Reactome:R-HSA-8937016 "Core MLL complex methylates H3K4Me2-Nucleosome at the THBS1 gene promoter"
 xref: Reactome:R-HSA-8937050 "Core MLL complex methylates H3K4Me2-Nucleosome at the MIR27A gene promoter"
-xref: Reactome:R-HSA-9843121 "KMT2D,(KMT2C) complex monomethylates nucleosomes at PPARG:RXRA-bound enhancers"
+xref: Reactome:R-HSA-9843121 "MLL4-ASCOM, (MLL3-ASCOM) complex monomethylates nucleosomes at PPARG:RXRA-bound enhancers"
+xref: Reactome:R-HSA-9909628 "Trimethylation of CD274 gene chromatin by MLL1/3 complex"
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24866" xsd:anyURI
@@ -255180,7 +255753,7 @@
 id: GO:0042813
 name: Wnt receptor activity
 namespace: molecular_function
-def: "Combining with a Wnt protein and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:go_curators]
+def: "Combining with a Wnt protein and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:curators]
 subset: goslim_chembl
 synonym: "frizzled receptor activity" RELATED []
 synonym: "frizzled-2 receptor activity" RELATED []
@@ -255367,7 +255940,7 @@
 id: GO:0042834
 name: peptidoglycan binding
 namespace: molecular_function
-def: "Interacting selectively and non-covalently, in a non-covalent manner, with peptidoglycan, any of a class of glycoconjugates found in bacterial cell walls." [GOC:go_curators, PMID:14698226]
+def: "Interacting selectively and non-covalently, in a non-covalent manner, with peptidoglycan, any of a class of glycoconjugates found in bacterial cell walls." [GOC:curators, PMID:14698226]
 is_a: GO:0005539 ! glycosaminoglycan binding
 
 [Term]
@@ -255570,6 +256143,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
 synonym: "L-alanine metabolism" EXACT []
+xref: Reactome:R-HSA-8964540 "Alanine metabolism"
 is_a: GO:0006522 ! alanine metabolic process
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
@@ -255578,23 +256152,28 @@
 id: GO:0042852
 name: L-alanine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah, MetaCyc:PWY0-1061]
+def: "The chemical reactions and pathways resulting in the formation of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
 synonym: "L-alanine anabolism" EXACT []
 synonym: "L-alanine biosynthesis" EXACT []
 synonym: "L-alanine formation" EXACT []
 synonym: "L-alanine synthesis" EXACT []
-xref: MetaCyc:PWY0-1061
+xref: MetaCyc:ALANINE-SYN2-PWY
+xref: MetaCyc:ALANINE-VALINESYN-PWY
+xref: MetaCyc:PWY0-1021
 is_a: GO:0006523 ! alanine biosynthetic process
 is_a: GO:0009079 ! pyruvate family amino acid biosynthetic process
 is_a: GO:0042851 ! L-alanine metabolic process
-property_value: skos:exactMatch MetaCyc:PWY0-1061
+property_value: skos:narrowMatch MetaCyc:ALANINE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:ALANINE-VALINESYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY0-1021
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29444" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 
 [Term]
 id: GO:0042853
 name: L-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of L-alanine, the L-enantiomer of 2-aminopropanoic acid, i.e. (2S)-2-aminopropanoic acid." [GOC:jl, GOC:jsg, GOC:mah]
+def: "The chemical reactions and pathways resulting in the breakdown of L-alanine." [GOC:curators]
 synonym: "L-alanine breakdown" EXACT []
 synonym: "L-alanine catabolism" EXACT []
 synonym: "L-alanine degradation" EXACT []
@@ -255608,6 +256187,7 @@
 property_value: skos:narrowMatch MetaCyc:ALADEG-PWY
 property_value: skos:narrowMatch MetaCyc:ALANINE-DEG3-PWY
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30202" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 
 [Term]
 id: GO:0042854
@@ -255784,11 +256364,12 @@
 id: GO:0042866
 name: pyruvate biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyruvate, 2-oxopropanoate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of pyruvate, 2-oxopropanoate." [GOC:curators]
 synonym: "pyruvate anabolism" EXACT []
 synonym: "pyruvate biosynthesis" EXACT []
 synonym: "pyruvate formation" EXACT []
 synonym: "pyruvate synthesis" EXACT []
+xref: Reactome:R-HSA-70268 "Pyruvate metabolism"
 is_a: GO:0006090 ! pyruvate metabolic process
 is_a: GO:0072330 ! monocarboxylic acid biosynthetic process
 
@@ -255796,7 +256377,7 @@
 id: GO:0042867
 name: pyruvate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pyruvate, 2-oxopropanoate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of pyruvate, 2-oxopropanoate." [GOC:curators]
 synonym: "pyruvate breakdown" EXACT []
 synonym: "pyruvate catabolism" EXACT []
 synonym: "pyruvate degradation" EXACT []
@@ -255815,7 +256396,7 @@
 id: GO:0042869
 name: aldarate transmembrane transport
 namespace: biological_process
-def: "The process in which aldarate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:go_curators]
+def: "The process in which aldarate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:curators]
 synonym: "aldarate transport" RELATED []
 is_a: GO:0006835 ! dicarboxylic acid transport
 is_a: GO:1905039 ! carboxylic acid transmembrane transport
@@ -255859,7 +256440,7 @@
 id: GO:0042876
 name: aldarate transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of aldarate from one side of a membrane to the other." [GOC:go_curators, PMID:15034926]
+def: "Enables the transfer of aldarate from one side of a membrane to the other." [GOC:curators, PMID:15034926]
 is_a: GO:0046943 ! carboxylic acid transmembrane transporter activity
 relationship: part_of GO:0042869 ! aldarate transmembrane transport
 
@@ -256135,7 +256716,7 @@
 name: xenobiotic transport
 namespace: biological_process
 alt_id: GO:0015893
-def: "The directed movement of a xenobiotic into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:go_curators, GOC:krc]
+def: "The directed movement of a xenobiotic into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:curators, GOC:krc]
 subset: goslim_pir
 synonym: "drug transport" RELATED []
 is_a: GO:0006810 ! transport
@@ -256159,7 +256740,7 @@
 alt_id: GO:0015559
 alt_id: GO:0015564
 alt_id: GO:0090484
-def: "Enables the directed movement of a xenobiotic from one side of a membrane to the other. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:go_curators, GOC:krc]
+def: "Enables the directed movement of a xenobiotic from one side of a membrane to the other. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:curators, GOC:krc]
 subset: goslim_metagenomics
 synonym: "drug transmembrane transporter activity" RELATED []
 synonym: "drug transporter activity" RELATED []
@@ -256169,6 +256750,15 @@
 synonym: "xenobiotic transporter activity" RELATED []
 xref: Reactome:R-HSA-5625574 "Defective SLC22A18 does not exchange extracellular organic cations for cytosolic H+"
 xref: Reactome:R-HSA-597628 "SLC22A18 exchanges extracellular organic cations for cytosolic H+"
+xref: Reactome:R-HSA-9911512 "AcrA:AcrB:TolC transports antibiotics from the cytosol to the periplasm"
+xref: Reactome:R-HSA-9911517 "MdfA transports tetracyclines from the cytosol to the periplasmic space"
+xref: Reactome:R-HSA-9913106 "TetB exports tetracyclines"
+xref: Reactome:R-HSA-9913128 "TetA exports tetracyclines"
+xref: Reactome:R-HSA-9913437 "Pef exports fluoroquinolones"
+xref: Reactome:R-HSA-9913672 "OqxA exports fluoroquinolones"
+xref: Reactome:R-HSA-9914089 "MdtE:MdtF:TolC exports beta-lactams to the periplasm"
+xref: Reactome:R-HSA-9931163 "Gdx dimer exports cationic aromatics to periplasm"
+xref: Reactome:R-HSA-9931565 "EmrE dimer exports xenobiotics to periplasm"
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0042908 ! xenobiotic transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19460" xsd:anyURI
@@ -256234,7 +256824,7 @@
 id: GO:0042920
 name: 3-hydroxyphenylpropionic acid transmembrane transport
 namespace: biological_process
-def: "The directed movement of 3-hydroxyphenylpropionic acid across a lipid bilayer, from one side of a membrane to the other." [GOC:go_curators, PMID:9098055]
+def: "The directed movement of 3-hydroxyphenylpropionic acid across a lipid bilayer, from one side of a membrane to the other." [GOC:curators, PMID:9098055]
 synonym: "3-(3-hydroxyphenyl)propionic acid transport" EXACT []
 synonym: "3-hydroxyphenylpropionic acid transport" BROAD []
 synonym: "m-hydroxyphenylpropionic acid transport" EXACT []
@@ -256331,6 +256921,8 @@
 def: "Enables the directed movement of the siderochrome enterochelin, a cyclic trimer of 2, 3 dihydroxybenzoylserine from one side of a membrane to the other." [GOC:jl]
 synonym: "enterobactin transporter activity" RELATED []
 synonym: "enterochelin transporter activity" EXACT []
+xref: Reactome:R-HSA-9911595 "EntS transports enterobactin from cytosol to periplasmic space"
+xref: Reactome:R-HSA-9911600 "Efflux pump complexes transport enterobactin from periplasmic space to extracellular region"
 is_a: GO:0015343 ! siderophore-iron transmembrane transporter activity
 relationship: part_of GO:0042930 ! enterobactin transport
 
@@ -256609,7 +257201,7 @@
 id: GO:0042964
 name: obsolete thioredoxin reduction
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the reduction of thioredoxin, a small disulfide-containing redox protein that serves as a general protein disulfide oxidoreductase." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the reduction of thioredoxin, a small disulfide-containing redox protein that serves as a general protein disulfide oxidoreductase." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "thioredoxin anabolism" EXACT []
 synonym: "thioredoxin biosynthesis" EXACT []
@@ -256622,7 +257214,7 @@
 id: GO:0042965
 name: obsolete glutaredoxin biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of a small disulfide-containing redox protein that serves as a glutathione-disulfide oxidoreductase." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of a small disulfide-containing redox protein that serves as a glutathione-disulfide oxidoreductase." [GOC:curators]
 comment: This term was made obsolete because it refers to the biosynthesis of a protein.
 synonym: "glutaredoxin anabolism" EXACT []
 synonym: "glutaredoxin biosynthesis" EXACT []
@@ -256634,7 +257226,7 @@
 id: GO:0042966
 name: biotin carboxyl carrier protein biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of the biotin carboxyl carrier protein, a subunit of acetyl-coenzyme A carboxylase." [GOC:go_curators, PMID:8102363]
+def: "The chemical reactions and pathways resulting in the formation of the biotin carboxyl carrier protein, a subunit of acetyl-coenzyme A carboxylase." [GOC:curators, PMID:8102363]
 synonym: "BCCP biosynthesis" EXACT []
 synonym: "BCCP biosynthetic process" EXACT []
 synonym: "biotin carboxyl carrier protein anabolism" EXACT []
@@ -256647,7 +257239,7 @@
 id: GO:0042967
 name: obsolete acyl-carrier-protein biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of acyl-carrier protein." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of acyl-carrier protein." [GOC:curators]
 comment: The reason for obsoletion is that acyl-carrier-protein is synthesized by the ribosome like any other protein.
 synonym: "ACP biosynthesis" EXACT []
 synonym: "ACP biosynthetic process" EXACT []
@@ -256674,7 +257266,7 @@
 id: GO:0042969
 name: obsolete lactone transport
 namespace: biological_process
-def: "OBSOLETE. The directed movement of lactone from one side of a membrane to the other. A lactone is a cyclic ester of a hydroxy carboxylic acid, containing a 1-oxacycloalkan-2-one structure, or an analogue having unsaturation or heteroatoms replacing one or more carbon atoms of the ring." [GOC:go_curators]
+def: "OBSOLETE. The directed movement of lactone from one side of a membrane to the other. A lactone is a cyclic ester of a hydroxy carboxylic acid, containing a 1-oxacycloalkan-2-one structure, or an analogue having unsaturation or heteroatoms replacing one or more carbon atoms of the ring." [GOC:curators]
 comment: This term aws obsoleted because there is no evidence for a specific pathway to transport lactone.
 is_obsolete: true
 
@@ -256682,7 +257274,7 @@
 id: GO:0042970
 name: L-homoserine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of homoserine from one side of a membrane to the other. Homoserine is alpha-amino-gamma-hydroxybutyric acid, an intermediate in the biosynthesis of cystathionine, threonine and methionine." [GOC:go_curators, ISBN:0198506732]
+def: "Enables the transfer of homoserine from one side of a membrane to the other. Homoserine is alpha-amino-gamma-hydroxybutyric acid, an intermediate in the biosynthesis of cystathionine, threonine and methionine." [GOC:curators, ISBN:0198506732]
 synonym: "homoserine transmembrane transporter activity" BROAD []
 synonym: "homoserine transporter activity" BROAD []
 is_a: GO:0015175 ! neutral L-amino acid transmembrane transporter activity
@@ -256694,7 +257286,7 @@
 id: GO:0042971
 name: obsolete lactone transmembrane transporter activity
 namespace: molecular_function
-def: "OBSOLETE. Enables the transfer of lactone from one side of a membrane to the other. A lactone is a cyclic ester of a hydroxy carboxylic acid, containing a 1-oxacycloalkan-2-one structure, or an analogue having unsaturation or heteroatoms replacing one or more carbon atoms of the ring." [GOC:go_curators]
+def: "OBSOLETE. Enables the transfer of lactone from one side of a membrane to the other. A lactone is a cyclic ester of a hydroxy carboxylic acid, containing a 1-oxacycloalkan-2-one structure, or an analogue having unsaturation or heteroatoms replacing one or more carbon atoms of the ring." [GOC:curators]
 comment: This term was obsoleted because there is no evidence for a specific lactone transporter.
 is_obsolete: true
 
@@ -256791,6 +257383,7 @@
 namespace: molecular_function
 def: "Binds to and increases ornithine decarboxylase activity." [GOC:jl]
 synonym: "L-ornithine carboxy-lyase activator activity" NARROW []
+xref: Reactome:R-HSA-9955574 "ODC dimerizes"
 is_a: GO:0008047 ! enzyme activator activity
 is_a: GO:0042979 ! ornithine decarboxylase regulator activity
 relationship: positively_regulates GO:0004586 ! ornithine decarboxylase activity
@@ -256833,7 +257426,7 @@
 id: GO:0042982
 name: amyloid precursor protein metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:go_curators]
+def: "The chemical reactions and pathways involving amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:curators]
 synonym: "amyloid precursor protein metabolism" EXACT []
 synonym: "APP metabolic process" EXACT []
 synonym: "APP metabolism" EXACT []
@@ -256843,7 +257436,7 @@
 id: GO:0042983
 name: amyloid precursor protein biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:curators]
 synonym: "amyloid precursor protein anabolism" EXACT []
 synonym: "amyloid precursor protein biosynthesis" EXACT []
 synonym: "amyloid precursor protein formation" EXACT []
@@ -256857,7 +257450,7 @@
 id: GO:0042984
 name: regulation of amyloid precursor protein biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:curators]
 synonym: "regulation of amyloid precursor protein anabolism" EXACT []
 synonym: "regulation of amyloid precursor protein biosynthesis" EXACT []
 synonym: "regulation of amyloid precursor protein formation" EXACT []
@@ -256873,7 +257466,7 @@
 id: GO:0042985
 name: negative regulation of amyloid precursor protein biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:curators]
 synonym: "down regulation of amyloid precursor protein biosynthetic process" EXACT []
 synonym: "down-regulation of amyloid precursor protein biosynthetic process" EXACT []
 synonym: "downregulation of amyloid precursor protein biosynthetic process" EXACT []
@@ -256894,7 +257487,7 @@
 id: GO:0042986
 name: positive regulation of amyloid precursor protein biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of amyloid precursor protein (APP), the precursor of amyloid-beta." [GOC:curators]
 synonym: "activation of amyloid precursor protein biosynthetic process" NARROW []
 synonym: "positive regulation of amyloid precursor protein anabolism" EXACT []
 synonym: "positive regulation of amyloid precursor protein biosynthesis" EXACT []
@@ -256916,7 +257509,7 @@
 id: GO:0042987
 name: amyloid precursor protein catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of amyloid precursor protein (APP), the precursor of amyloid-beta, a glycoprotein associated with Alzheimer's disease." [GOC:curators]
 synonym: "amyloid precursor protein breakdown" EXACT []
 synonym: "amyloid precursor protein catabolism" EXACT []
 synonym: "amyloid precursor protein degradation" EXACT []
@@ -256936,7 +257529,7 @@
 id: GO:0042989
 name: obsolete sequestering of actin monomers
 namespace: biological_process
-def: "OBSOLETE. The selective interaction of actin monomers with specific molecules that inhibit their polymerization by preventing their access to other monomers." [GOC:go_curators]
+def: "OBSOLETE. The selective interaction of actin monomers with specific molecules that inhibit their polymerization by preventing their access to other monomers." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function, not biological process
 synonym: "actin monomer retention" EXACT []
 synonym: "actin monomer sequestering" EXACT []
@@ -257215,7 +257808,7 @@
 namespace: biological_process
 alt_id: GO:0001747
 alt_id: GO:0031075
-def: "The process whose specific outcome is the progression of the camera-type eye over time, from its formation to the mature structure. The camera-type eye is an organ of sight that receives light through an aperture and focuses it through a lens, projecting it on a photoreceptor field." [GOC:go_curators, GOC:mtg_sensu]
+def: "The process whose specific outcome is the progression of the camera-type eye over time, from its formation to the mature structure. The camera-type eye is an organ of sight that receives light through an aperture and focuses it through a lens, projecting it on a photoreceptor field." [GOC:curators, GOC:mtg_sensu]
 is_a: GO:0001654 ! eye development
 
 [Term]
@@ -257234,8 +257827,8 @@
 namespace: biological_process
 def: "Any process that modulates the binding and fusion of a sperm to the oocyte plasma membrane." [GOC:jl, PMID:11483596]
 synonym: "regulation of sperm-oocyte fusion" NARROW []
+is_a: GO:0050794 ! regulation of cellular process
 is_a: GO:0051239 ! regulation of multicellular organismal process
-is_a: GO:1903729 ! regulation of plasma membrane organization
 is_a: GO:2000241 ! regulation of reproductive process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0007342 ! fusion of sperm to egg plasma membrane involved in single fertilization
@@ -257253,7 +257846,7 @@
 synonym: "inhibition of sperm-oocyte fusion" NARROW []
 synonym: "negative regulation of sperm-oocyte fusion" NARROW []
 is_a: GO:0043012 ! regulation of fusion of sperm to egg plasma membrane
-is_a: GO:0051129 ! negative regulation of cellular component organization
+is_a: GO:0048523 ! negative regulation of cellular process
 is_a: GO:0051241 ! negative regulation of multicellular organismal process
 is_a: GO:2000242 ! negative regulation of reproductive process
 intersection_of: GO:0065007 ! biological regulation
@@ -257291,7 +257884,7 @@
 id: GO:0043021
 name: ribonucleoprotein complex binding
 namespace: molecular_function
-def: "Binding to a complex of RNA and protein." [GOC:bf, GOC:go_curators, GOC:vk]
+def: "Binding to a complex of RNA and protein." [GOC:bf, GOC:curators, GOC:vk]
 subset: goslim_pir
 synonym: "protein-RNA complex binding" EXACT [GOC:bf, GOC:vk]
 synonym: "ribonucleoprotein binding" EXACT [GOC:bf, GOC:vk]
@@ -257303,7 +257896,7 @@
 name: ribosome binding
 namespace: molecular_function
 alt_id: GO:0030376
-def: "Binding to a ribosome." [GOC:go_curators]
+def: "Binding to a ribosome." [GOC:curators]
 synonym: "ribosome receptor activity" NARROW []
 is_a: GO:0043021 ! ribonucleoprotein complex binding
 
@@ -257311,21 +257904,21 @@
 id: GO:0043023
 name: ribosomal large subunit binding
 namespace: molecular_function
-def: "Binding to a large ribosomal subunit." [GOC:go_curators]
+def: "Binding to a large ribosomal subunit." [GOC:curators]
 is_a: GO:0043022 ! ribosome binding
 
 [Term]
 id: GO:0043024
 name: ribosomal small subunit binding
 namespace: molecular_function
-def: "Binding to a small ribosomal subunit." [GOC:go_curators]
+def: "Binding to a small ribosomal subunit." [GOC:curators]
 is_a: GO:0043022 ! ribosome binding
 
 [Term]
 id: GO:0043025
 name: neuronal cell body
 namespace: cellular_component
-def: "The portion of a neuron that includes the nucleus, but excludes cell projections such as axons and dendrites." [GOC:go_curators]
+def: "The portion of a neuron that includes the nucleus, but excludes cell projections such as axons and dendrites." [GOC:curators]
 comment: Note that 'cell body' and 'cell soma' are not used in the literature for cells that lack projections, nor for some cells (e.g. yeast with mating projections) that do have projections.
 subset: goslim_pir
 synonym: "neuron cell body" EXACT []
@@ -257507,7 +258100,7 @@
 id: GO:0043045
 name: epigenetic programming of gene expression
 namespace: biological_process
-def: "A epigenetic process that happens during embryonic development that modulates gene expression potential at later stages of development of the organism, including the adult. Epigenetic regulation takes place via chromatin remodeling either by modifying higher order chromatin fiber structure, nucleosomal histones, or cytosine DNA methylation." [GOC:go_curators, PMID:12138111, PMID:22868271]
+def: "A epigenetic process that happens during embryonic development that modulates gene expression potential at later stages of development of the organism, including the adult. Epigenetic regulation takes place via chromatin remodeling either by modifying higher order chromatin fiber structure, nucleosomal histones, or cytosine DNA methylation." [GOC:curators, PMID:12138111, PMID:22868271]
 synonym: "de novo DNA methylation" RELATED []
 synonym: "DNA methylation involved in embryo development" RELATED []
 synonym: "epigenetic regulation of embryonic gene expression" RELATED []
@@ -257519,7 +258112,7 @@
 id: GO:0043046
 name: obsolete DNA methylation involved in gamete generation
 namespace: biological_process
-def: "OBSOLETE. The covalent transfer of a methyl group to C-5 of cytosine that contributes to the establishment of DNA methylation patterns in the gamete." [GOC:go_curators, PMID:12138111]
+def: "OBSOLETE. The covalent transfer of a methyl group to C-5 of cytosine that contributes to the establishment of DNA methylation patterns in the gamete." [GOC:curators, PMID:12138111]
 comment: This term was obsoleted because it represent the same process as genomic imprinting ; GO:0071514 .
 synonym: "de novo DNA methylation" RELATED []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25313" xsd:anyURI
@@ -257532,7 +258125,7 @@
 namespace: molecular_function
 def: "Binding to single-stranded telomere-associated DNA." [GOC:jl, ISBN:0321000382]
 synonym: "telomeric ssDNA binding" EXACT [GOC:mah]
-is_a: GO:0042162 ! telomeric DNA binding
+is_a: GO:0042162 ! telomeric repeat DNA binding
 is_a: GO:0098847 ! sequence-specific single stranded DNA binding
 
 [Term]
@@ -257545,6 +258138,7 @@
 synonym: "dolichyl monophosphate formation" EXACT []
 synonym: "dolichyl monophosphate synthesis" EXACT []
 xref: MetaCyc:PWY-6129
+xref: Reactome:R-HSA-446199 "Synthesis of dolichyl-phosphate"
 is_a: GO:0008654 ! phospholipid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30675" xsd:anyURI
 
@@ -257552,7 +258146,7 @@
 id: GO:0043049
 name: otic placode formation
 namespace: biological_process
-def: "The initial developmental process that will lead to the formation of the vertebrate inner ear. The otic placode forms as a thickening of the head ectoderm adjacent to the developing hindbrain." [GOC:go_curators, PMID:12668634]
+def: "The initial developmental process that will lead to the formation of the vertebrate inner ear. The otic placode forms as a thickening of the head ectoderm adjacent to the developing hindbrain." [GOC:curators, PMID:12668634]
 is_a: GO:0060788 ! ectodermal placode formation
 relationship: part_of GO:0030916 ! otic vesicle formation
 
@@ -257617,21 +258211,21 @@
 id: GO:0043056
 name: forward locomotion
 namespace: biological_process
-def: "Anterior movement of an organism, following the direction of the head of the animal." [GOC:go_curators]
+def: "Anterior movement of an organism, following the direction of the head of the animal." [GOC:curators]
 is_a: GO:0033058 ! directional locomotion
 
 [Term]
 id: GO:0043057
 name: backward locomotion
 namespace: biological_process
-def: "Posterior movement of an organism, e.g. following the direction of the tail of an animal." [GOC:go_curators]
+def: "Posterior movement of an organism, e.g. following the direction of the tail of an animal." [GOC:curators]
 is_a: GO:0033058 ! directional locomotion
 
 [Term]
 id: GO:0043058
 name: regulation of backward locomotion
 namespace: biological_process
-def: "Any process that modulates the speed, mechanical force, or rhythm of the posterior movement of an organism." [GOC:go_curators]
+def: "Any process that modulates the speed, mechanical force, or rhythm of the posterior movement of an organism." [GOC:curators]
 is_a: GO:0040012 ! regulation of locomotion
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0043057 ! backward locomotion
@@ -257641,7 +258235,7 @@
 id: GO:0043059
 name: regulation of forward locomotion
 namespace: biological_process
-def: "Any process that modulates the speed, mechanical force, or rhythm of the anterior movement of an organism." [GOC:go_curators]
+def: "Any process that modulates the speed, mechanical force, or rhythm of the anterior movement of an organism." [GOC:curators]
 is_a: GO:0040012 ! regulation of locomotion
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0043056 ! forward locomotion
@@ -257797,7 +258391,7 @@
 id: GO:0043073
 name: germ cell nucleus
 namespace: cellular_component
-def: "The nucleus of a germ cell, a reproductive cell in multicellular organisms." [CL:0000586, GOC:go_curators]
+def: "The nucleus of a germ cell, a reproductive cell in multicellular organisms." [CL:0000586, GOC:curators]
 synonym: "germ-cell nucleus" EXACT []
 is_a: GO:0005634 ! nucleus
 
@@ -257994,7 +258588,7 @@
 name: purine nucleobase salvage
 namespace: biological_process
 def: "Any process that generates purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, from derivatives of them without de novo synthesis." [GOC:jl]
-synonym: "purine base salvage" EXACT [GOC:go_curators]
+synonym: "purine base salvage" EXACT [GOC:curators]
 is_a: GO:0009113 ! purine nucleobase biosynthetic process
 is_a: GO:0043101 ! purine-containing compound salvage
 
@@ -258030,7 +258624,7 @@
 name: pyrimidine nucleobase salvage
 namespace: biological_process
 def: "Any process that generates pyrimidine nucleobases, 1,3-diazine organic nitrogenous bases, from derivatives of them without de novo synthesis." [GOC:jl]
-synonym: "pyrimidine base salvage" EXACT [GOC:go_curators]
+synonym: "pyrimidine base salvage" EXACT [GOC:curators]
 is_a: GO:0008655 ! pyrimidine-containing compound salvage
 is_a: GO:0019856 ! pyrimidine nucleobase biosynthetic process
 
@@ -258094,7 +258688,7 @@
 id: GO:0043107
 name: type IV pilus-dependent motility
 namespace: biological_process
-def: "Any process involved in the controlled movement of a bacterial cell which is dependent on the presence of type IV pili. Includes social gliding motility and twitching motility." [GOC:go_curators, PMID:12704238]
+def: "Any process involved in the controlled movement of a bacterial cell which is dependent on the presence of type IV pili. Includes social gliding motility and twitching motility." [GOC:curators, PMID:12704238]
 synonym: "social gliding motility" NARROW []
 synonym: "TFP-dependent motility" EXACT []
 synonym: "TFP-dependent movement" EXACT []
@@ -258107,7 +258701,7 @@
 id: GO:0043108
 name: pilus retraction
 namespace: biological_process
-def: "The process of withdrawing a pilus back into a cell." [GOC:go_curators, PMID:17355871]
+def: "The process of withdrawing a pilus back into a cell." [GOC:curators, PMID:17355871]
 is_a: GO:0043711 ! pilus organization
 
 [Term]
@@ -258348,7 +258942,6 @@
 name: ubiquitin binding
 namespace: molecular_function
 def: "Binding to ubiquitin, a protein that when covalently bound to other cellular proteins marks them for proteolytic degradation." [GOC:ecd]
-xref: Reactome:R-HSA-1169404 "Transfer of ISG15 from E1 to E2 (UBCH8)"
 xref: Reactome:R-HSA-205008 "Polyubiquitinated NRIF binds to p62 (Sequestosome)"
 xref: Reactome:R-HSA-983152 "Transfer of ubiquitin from E1 to E2"
 is_a: GO:0032182 ! ubiquitin-like protein binding
@@ -258413,7 +259006,7 @@
 synonym: "glycerol-1-phosphatase activity" BROAD [EC:3.1.3.21]
 synonym: "glycerol-3-phosphatase activity" BROAD []
 xref: MetaCyc:RXN-14965
-xref: Reactome:R-HSA-8955794 "PGP:Mg2+ dimer hydrolyses 3PG to glycerol"
+xref: Reactome:R-HSA-8955794 "PGP:Mg2+ dimer hydrolyses G3P to glycerol"
 xref: RHEA:66372
 is_a: GO:0016791 ! phosphatase activity
 property_value: skos:broadMatch EC:3.1.3.21
@@ -258468,6 +259061,7 @@
 synonym: "DNA helicase IV activity" NARROW []
 xref: EC:5.6.2.3
 xref: Reactome:R-HSA-6782131 "ERCC2-facilitated RNA Pol II backtracking in TC-NER"
+xref: Reactome:R-HSA-9914438 "TWNK (PEO1) unwinds double-stranded mitochondrial DNA at the replication fork"
 is_a: GO:0003678 ! DNA helicase activity
 property_value: skos:exactMatch EC:5.6.2.3
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23533" xsd:anyURI
@@ -258493,7 +259087,7 @@
 id: GO:0043144
 name: sno(s)RNA processing
 namespace: biological_process
-def: "Any process involved in the conversion of a primary snoRNA family RNA transcript into a mature snoRNA (eukaryota) or sRNA (archaea)." [GOC:go_curators, GOC:krc, PMID:12773397]
+def: "Any process involved in the conversion of a primary snoRNA family RNA transcript into a mature snoRNA (eukaryota) or sRNA (archaea)." [GOC:curators, GOC:krc, PMID:12773397]
 subset: goslim_yeast
 is_a: GO:0006396 ! RNA processing
 is_a: GO:0016074 ! sno(s)RNA metabolic process
@@ -258503,7 +259097,7 @@
 id: GO:0043145
 name: sno(s)RNA 3'-end cleavage
 namespace: biological_process
-def: "The endonucleolytic cleavage of snoRNA 3' ends, which is required for mature snoRNAs to be functional." [GOC:go_curators, PMID:12773397]
+def: "The endonucleolytic cleavage of snoRNA 3' ends, which is required for mature snoRNAs to be functional." [GOC:curators, PMID:12773397]
 synonym: "sno(s)RNA 3' end cleavage" EXACT []
 synonym: "snoRNA 3'-end cleavage" NARROW []
 synonym: "sRNA 3'-end cleavage" NARROW []
@@ -258514,7 +259108,7 @@
 id: GO:0043149
 name: stress fiber assembly
 namespace: biological_process
-def: "The aggregation, arrangement and bonding together of a set of components to form a stress fiber. A stress fiber is a contractile actin filament bundle that consists of short actin filaments with alternating polarity." [GOC:go_curators, GOC:mah, PMID:16651381]
+def: "The aggregation, arrangement and bonding together of a set of components to form a stress fiber. A stress fiber is a contractile actin filament bundle that consists of short actin filaments with alternating polarity." [GOC:curators, GOC:mah, PMID:16651381]
 synonym: "actin cable assembly" RELATED [GOC:mah]
 synonym: "actin cable formation" RELATED [GOC:mah]
 synonym: "stress fibre biosynthesis" RELATED []
@@ -258526,7 +259120,7 @@
 id: GO:0043150
 name: DNA synthesis involved in double-strand break repair via homologous recombination
 namespace: biological_process
-def: "The synthesis of DNA that contributes to the process of double-strand break repair via homologous recombination." [GOC:go_curators]
+def: "The synthesis of DNA that contributes to the process of double-strand break repair via homologous recombination." [GOC:curators]
 synonym: "DNA synthesis during double-strand break repair via homologous recombination" RELATED [GOC:dph, GOC:tb]
 is_a: GO:0000731 ! DNA synthesis involved in DNA repair
 intersection_of: GO:0071897 ! DNA biosynthetic process
@@ -258537,7 +259131,7 @@
 id: GO:0043151
 name: obsolete DNA synthesis involved in double-strand break repair via single-strand annealing
 namespace: biological_process
-def: "OBSOLETE. The synthesis of DNA that contributes to the process of double-strand break repair via single-strand annealing." [GOC:go_curators]
+def: "OBSOLETE. The synthesis of DNA that contributes to the process of double-strand break repair via single-strand annealing." [GOC:curators]
 comment: This term was obsoleted because it is a pre-composed term that should be captured as a GO-CAM model.
 synonym: "DNA synthesis during double-strand break repair via single-strand annealing" RELATED [GOC:dph, GOC:tb]
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31306" xsd:anyURI
@@ -258573,7 +259167,6 @@
 synonym: "inhibition of caspase activity" NARROW []
 synonym: "negative regulation of caspase activation" EXACT []
 synonym: "negative regulation of caspase activity" BROAD []
-xref: Reactome:R-HSA-5218900 "CASP8 activity is inhibited"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -258631,7 +259224,7 @@
 id: GO:0043160
 name: acrosomal lumen
 namespace: cellular_component
-def: "The volume enclosed within the acrosome membrane." [GOC:go_curators]
+def: "The volume enclosed within the acrosome membrane." [GOC:curators]
 is_a: GO:0034774 ! secretory granule lumen
 is_a: GO:0043202 ! lysosomal lumen
 relationship: part_of GO:0001669 ! acrosomal vesicle
@@ -258640,7 +259233,7 @@
 id: GO:0043161
 name: proteasome-mediated ubiquitin-dependent protein catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of ubiquitin, and mediated by the proteasome." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of ubiquitin, and mediated by the proteasome." [GOC:curators]
 synonym: "proteasomal pathway" EXACT []
 synonym: "proteasomal processing" RELATED []
 synonym: "proteasomal ubiquitin-dependent protein breakdown" EXACT []
@@ -258817,14 +259410,14 @@
 id: GO:0043179
 name: rhythmic excitation
 namespace: biological_process
-def: "Any process involved in the generation of rhythmic, synchronous excitatory synaptic inputs in a neural circuit." [GOC:go_curators, ISBN:0195088433]
+def: "Any process involved in the generation of rhythmic, synchronous excitatory synaptic inputs in a neural circuit." [GOC:curators, ISBN:0195088433]
 is_a: GO:0060024 ! rhythmic synaptic transmission
 
 [Term]
 id: GO:0043180
 name: rhythmic inhibition
 namespace: biological_process
-def: "Any process involved in the generation of rhythmic, synchronous inhibitory synaptic inputs in a neural circuit." [GOC:go_curators]
+def: "Any process involved in the generation of rhythmic, synchronous inhibitory synaptic inputs in a neural circuit." [GOC:curators]
 is_a: GO:0060024 ! rhythmic synaptic transmission
 
 [Term]
@@ -259056,7 +259649,7 @@
 id: GO:0043207
 name: response to external biotic stimulus
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an external biotic stimulus, an external stimulus caused by, or produced by living things." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an external biotic stimulus, an external stimulus caused by, or produced by living things." [GOC:curators]
 is_a: GO:0009605 ! response to external stimulus
 is_a: GO:0009607 ! response to biotic stimulus
 
@@ -259247,8 +259840,6 @@
 synonym: "anion-transporting ATPase activity" EXACT []
 synonym: "ATP-dependent anion transmembrane transporter activity" EXACT []
 synonym: "ATPase-coupled anion transmembrane transporter activity" EXACT []
-xref: Reactome:R-HSA-1454916 "The ABCC family mediates organic anion transport"
-xref: Reactome:R-HSA-5690340 "Defective ABCC6 does not transport organic anion from cytosol to extracellular region"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0042626
@@ -259257,7 +259848,7 @@
 id: GO:0043226
 name: organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton, and prokaryotic structures such as anammoxosomes and pirellulosomes. Excludes the plasma membrane." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton, and prokaryotic structures such as anammoxosomes and pirellulosomes. Excludes the plasma membrane." [GOC:curators]
 subset: goslim_chembl
 subset: goslim_generic
 subset: goslim_pir
@@ -259271,7 +259862,7 @@
 id: GO:0043227
 name: membrane-bounded organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "membrane-enclosed organelle" EXACT []
 xref: NIF_Subcellular:sao414196390
@@ -259285,7 +259876,7 @@
 id: GO:0043228
 name: membraneless organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_mouse
 synonym: "biological condensate" RELATED []
@@ -259302,7 +259893,7 @@
 id: GO:0043229
 name: intracellular organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function, occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton. Excludes the plasma membrane." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function, occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton. Excludes the plasma membrane." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_pir
 is_a: GO:0043226 ! organelle
@@ -259326,7 +259917,7 @@
 id: GO:0043231
 name: intracellular membrane-bounded organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_pir
 synonym: "intracellular membrane-enclosed organelle" EXACT []
@@ -259339,7 +259930,7 @@
 id: GO:0043232
 name: intracellular membraneless organelle
 namespace: cellular_component
-def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane and occurring within the cell. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:go_curators]
+def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane and occurring within the cell. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_pir
 synonym: "intracellular non-membrane-bounded organelle" EXACT []
@@ -259365,7 +259956,7 @@
 id: GO:0043235
 name: receptor complex
 namespace: cellular_component
-def: "Any protein complex that undergoes combination with a hormone, neurotransmitter, drug or intracellular messenger to initiate a change in cell function." [GOC:go_curators]
+def: "Any protein complex that undergoes combination with a hormone, neurotransmitter, drug or intracellular messenger to initiate a change in cell function." [GOC:curators]
 subset: goslim_pir
 is_a: GO:0032991 ! protein-containing complex
 
@@ -259385,7 +259976,7 @@
 id: GO:0043237
 name: laminin-1 binding
 namespace: molecular_function
-def: "Binding to laminin-1, a glycoprotein trimer with the subunit composition alpha1, beta1, gamma1." [GOC:go_curators]
+def: "Binding to laminin-1, a glycoprotein trimer with the subunit composition alpha1, beta1, gamma1." [GOC:curators]
 synonym: "laminin-111 binding" EXACT [GOC:dph, PMID:15979864]
 is_a: GO:0043236 ! laminin binding
 
@@ -259472,7 +260063,7 @@
 id: GO:0043248
 name: proteasome assembly
 namespace: biological_process
-def: "The aggregation, arrangement and bonding together of a mature, active proteasome complex." [GOC:go_curators, PMID:10872471]
+def: "The aggregation, arrangement and bonding together of a mature, active proteasome complex." [GOC:curators, PMID:10872471]
 synonym: "26S proteasome assembly" NARROW []
 synonym: "proteasome complex assembly" EXACT []
 synonym: "proteasome maturation" EXACT []
@@ -259493,7 +260084,7 @@
 id: GO:0043250
 name: obsolete sodium-dependent organic anion transmembrane transporter activity
 namespace: molecular_function
-def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other, in a sodium dependent manner." [GOC:go_curators]
+def: "OBSOLETE. Enables the transfer of organic anions from one side of a membrane to the other, in a sodium dependent manner." [GOC:curators]
 comment: The reason for obsoletion is that this term was an unnecessary and confusing grouping term.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30715" xsd:anyURI
 is_obsolete: true
@@ -259503,15 +260094,15 @@
 id: GO:0043251
 name: sodium-dependent organic anion transport
 namespace: biological_process
-def: "The directed, sodium-dependent, movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
+def: "The directed, sodium-dependent, movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
 is_a: GO:0015711 ! organic anion transport
 
 [Term]
 id: GO:0043252
 name: sodium-independent organic anion transport
 namespace: biological_process
-def: "The directed, sodium-independent, movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:go_curators]
-xref: Reactome:R-HSA-879518 "Transport of organic anions"
+def: "The directed, sodium-independent, movement of organic anions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [GOC:curators]
+xref: Reactome:R-HSA-879518 "Organic anion transport by SLCO transporters"
 is_a: GO:0015711 ! organic anion transport
 
 [Term]
@@ -259559,9 +260150,9 @@
 subset: goslim_pir
 synonym: "laminin complex" BROAD []
 is_a: GO:0032991 ! protein-containing complex
-relationship: part_of GO:0140144 ! non-collagenous component of basement membrane
 relationship: part_of GO:0140349 ! laminin network
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30807" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31490" xsd:anyURI
 
 [Term]
 id: GO:0043257
@@ -259895,7 +260486,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "regulation of caspase activation" NARROW []
 synonym: "regulation of caspase activity" BROAD []
-xref: Reactome:R-HSA-205025 "NADE modulates death signalling"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 
@@ -259903,7 +260493,7 @@
 id: GO:0043282
 name: chordate pharyngeal muscle development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the pharyngeal muscle over time, from its formation to the mature structure. A pharyngeal muscle is any muscle that forms part of the pharynx." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the pharyngeal muscle over time, from its formation to the mature structure. A pharyngeal muscle is any muscle that forms part of the pharynx." [GOC:curators]
 is_a: GO:0007517 ! muscle organ development
 relationship: part_of GO:0160093 ! chordate pharynx development
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18285" xsd:anyURI
@@ -259979,7 +260569,7 @@
 id: GO:0043292
 name: contractile muscle fiber
 namespace: cellular_component
-def: "Fibers, composed of actin, myosin, and associated proteins, found in cells of smooth or striated muscle." [GOC:go_curators, ISBN:0815316194]
+def: "Fibers, composed of actin, myosin, and associated proteins, found in cells of smooth or striated muscle." [GOC:curators, ISBN:0815316194]
 synonym: "contractile fibre" EXACT []
 is_a: GO:0043232 ! intracellular membraneless organelle
 is_a: GO:0099512 ! supramolecular fiber
@@ -260020,7 +260610,7 @@
 id: GO:0043296
 name: apical junction complex
 namespace: cellular_component
-def: "A functional unit located near the cell apex at the points of contact between epithelial cells, which in vertebrates is composed of the tight junction, the zonula adherens, and desmosomes and in some invertebrates, such as Drosophila, is composed of the subapical complex (SAC), the zonula adherens and the septate junction. Functions in the regulation of cell polarity, tissue integrity and intercellular adhesion and permeability." [GOC:go_curators, GOC:kmv, PMID:12525486, PMID:15196556]
+def: "A functional unit located near the cell apex at the points of contact between epithelial cells, which in vertebrates is composed of the tight junction, the zonula adherens, and desmosomes and in some invertebrates, such as Drosophila, is composed of the subapical complex (SAC), the zonula adherens and the septate junction. Functions in the regulation of cell polarity, tissue integrity and intercellular adhesion and permeability." [GOC:curators, GOC:kmv, PMID:12525486, PMID:15196556]
 synonym: "apical cell junction complex" EXACT [GOC:mah]
 synonym: "apical junction" EXACT []
 is_a: GO:0005911 ! cell-cell junction
@@ -260029,7 +260619,7 @@
 id: GO:0043297
 name: apical junction assembly
 namespace: biological_process
-def: "The formation of an apical junction, a functional unit located near the cell apex at the points of contact between epithelial cells composed of the tight junction, the zonula adherens junction and the desmosomes, by the aggregation, arrangement and bonding together of its constituents." [GOC:go_curators, PMID:10854689, PMID:14729475, PMID:15196556]
+def: "The formation of an apical junction, a functional unit located near the cell apex at the points of contact between epithelial cells composed of the tight junction, the zonula adherens junction and the desmosomes, by the aggregation, arrangement and bonding together of its constituents." [GOC:curators, PMID:10854689, PMID:14729475, PMID:15196556]
 synonym: "apical junction complex assembly" EXACT [GOC:mah]
 is_a: GO:0007043 ! cell-cell junction assembly
 
@@ -260448,14 +261038,14 @@
 id: GO:0043325
 name: phosphatidylinositol-3,4-bisphosphate binding
 namespace: molecular_function
-def: "Binding to phosphatidylinositol-3,4-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' and 4' positions." [GOC:bf, GOC:go_curators]
+def: "Binding to phosphatidylinositol-3,4-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' and 4' positions." [GOC:bf, GOC:curators]
 is_a: GO:1902936 ! phosphatidylinositol bisphosphate binding
 
 [Term]
 id: GO:0043326
 name: chemotaxis to folate
 namespace: biological_process
-def: "The directed movement of a motile cell or organism in response to the presence of folate." [GOC:go_curators]
+def: "The directed movement of a motile cell or organism in response to the presence of folate." [GOC:curators]
 is_a: GO:0006935 ! chemotaxis
 is_a: GO:0051593 ! response to folic acid
 
@@ -260463,7 +261053,7 @@
 id: GO:0043327
 name: chemotaxis to cAMP
 namespace: biological_process
-def: "The directed movement of a motile cell or organism in response to the presence of 3',5'-cAMP." [GOC:go_curators]
+def: "The directed movement of a motile cell or organism in response to the presence of 3',5'-cAMP." [GOC:curators]
 synonym: "chemotaxis to 3',5' cAMP" EXACT []
 synonym: "chemotaxis to 3',5'-cAMP" EXACT []
 synonym: "chemotaxis to adenosine 3',5'-cyclophosphate" EXACT []
@@ -260513,7 +261103,7 @@
 id: GO:0043330
 name: response to exogenous dsRNA
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an exogenous double-stranded RNA stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an exogenous double-stranded RNA stimulus." [GOC:curators]
 comment: Note that the presence of exogenous double-stranded RNA is usually indicative of a viral infection. Consider also annotating to 'response to virus ; GO:0009615'.
 synonym: "response to exogenous double-stranded RNA" EXACT []
 synonym: "response to viral dsRNA" NARROW []
@@ -260618,7 +261208,7 @@
 id: GO:0043353
 name: enucleate erythrocyte differentiation
 namespace: biological_process
-def: "The process in which a myeloid precursor cell acquires specialized features of an erythrocyte without a nucleus. An example of this process is found in Mus musculus." [GOC:go_curators]
+def: "The process in which a myeloid precursor cell acquires specialized features of an erythrocyte without a nucleus. An example of this process is found in Mus musculus." [GOC:curators]
 synonym: "enucleate RBC differentiation" EXACT [CL:0000232]
 synonym: "enucleate red blood cell differentiation" EXACT [CL:0000232]
 is_a: GO:0030218 ! erythrocyte differentiation
@@ -260627,7 +261217,7 @@
 id: GO:0043354
 name: enucleate erythrocyte maturation
 namespace: biological_process
-def: "A developmental process, independent of morphogenetic (shape) change, that is required for an enucleate erythrocyte to attain its fully functional state. An enucleate erythrocyte is an erythrocyte without a nucleus." [GOC:go_curators]
+def: "A developmental process, independent of morphogenetic (shape) change, that is required for an enucleate erythrocyte to attain its fully functional state. An enucleate erythrocyte is an erythrocyte without a nucleus." [GOC:curators]
 synonym: "enucleate RBC maturation" EXACT [CL:0000232]
 synonym: "enucleate red blood cell maturation" EXACT [CL:0000232]
 is_a: GO:0043249 ! erythrocyte maturation
@@ -261070,7 +261660,7 @@
 id: GO:0043393
 name: regulation of protein binding
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of protein binding." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of protein binding." [GOC:curators]
 subset: gocheck_do_not_annotate
 is_a: GO:0051098 ! regulation of binding
 intersection_of: GO:0065007 ! biological regulation
@@ -261097,7 +261687,7 @@
 id: GO:0043396
 name: corticotropin-releasing hormone secretion
 namespace: biological_process
-def: "The regulated release of corticotropin-releasing hormone (CRH), a polypeptide hormone involved in the stress response. CRH is produced by the hypothalamus and stimulates corticotropic cells of the anterior lobe of the pituitary to produce corticotropic hormone (CTH) and other biologically active substances e.g. 2-endorphin, release of CRH is affected by serum levels of cortisol, by stress and by the sleep/wake cycle." [GOC:go_curators, PMID:11027914]
+def: "The regulated release of corticotropin-releasing hormone (CRH), a polypeptide hormone involved in the stress response. CRH is produced by the hypothalamus and stimulates corticotropic cells of the anterior lobe of the pituitary to produce corticotropic hormone (CTH) and other biologically active substances e.g. 2-endorphin, release of CRH is affected by serum levels of cortisol, by stress and by the sleep/wake cycle." [GOC:curators, PMID:11027914]
 synonym: "corticotropin-releasing factor secretion" EXACT []
 synonym: "CRF secretion" EXACT []
 synonym: "CRH secretion" EXACT []
@@ -261107,7 +261697,7 @@
 id: GO:0043397
 name: regulation of corticotropin-releasing hormone secretion
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of corticotropin-releasing hormone secretion." [GOC:go_curators, PMID:11027914]
+def: "Any process that modulates the frequency, rate or extent of corticotropin-releasing hormone secretion." [GOC:curators, PMID:11027914]
 synonym: "regulation of corticotropin-releasing factor secretion" EXACT []
 synonym: "regulation of CRF secretion" EXACT []
 synonym: "regulation of CRH secretion" EXACT []
@@ -261120,7 +261710,7 @@
 id: GO:0043398
 name: HLH domain binding
 namespace: molecular_function
-def: "Binding to a Helix Loop Helix domain, a domain of 40-50 residues that occurs in specific DNA-binding proteins that act as transcription factors. The domain is formed of two amphipathic helices joined by a variable length linker region that can form a loop and it mediates protein dimerization." [GOC:go_curators, Prosite:PDOC0038]
+def: "Binding to a Helix Loop Helix domain, a domain of 40-50 residues that occurs in specific DNA-binding proteins that act as transcription factors. The domain is formed of two amphipathic helices joined by a variable length linker region that can form a loop and it mediates protein dimerization." [GOC:curators, Prosite:PDOC0038]
 is_a: GO:0019904 ! protein domain specific binding
 
 [Term]
@@ -261189,7 +261779,7 @@
 id: GO:0043405
 name: regulation of MAP kinase activity
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of MAP kinase activity." [GOC:dph, GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of MAP kinase activity." [GOC:curators, GOC:dph]
 subset: gocheck_obsoletion_candidate
 synonym: "Regulation of MAPK activity" EXACT [GOC:dph]
 synonym: "regulation of mitogen activated protein kinase activity" EXACT []
@@ -261203,7 +261793,7 @@
 id: GO:0043406
 name: positive regulation of MAP kinase activity
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of MAP kinase activity." [GOC:dph, GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of MAP kinase activity." [GOC:curators, GOC:dph]
 subset: gocheck_obsoletion_candidate
 synonym: "positive regulation of mitogen activated protein kinase activity" EXACT []
 synonym: "positive regulation of mitogen-activated protein kinase activity" EXACT []
@@ -261222,7 +261812,7 @@
 id: GO:0043407
 name: negative regulation of MAP kinase activity
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of MAP kinase activity." [GOC:dph, GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of MAP kinase activity." [GOC:curators, GOC:dph]
 subset: gocheck_obsoletion_candidate
 synonym: "down regulation of MAPK activity" EXACT []
 synonym: "down-regulation of MAPK activity" EXACT []
@@ -261241,7 +261831,7 @@
 id: GO:0043408
 name: regulation of MAPK cascade
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of signal transduction mediated by the MAP kinase (MAPK) cascade." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of signal transduction mediated by the MAP kinase (MAPK) cascade." [GOC:curators]
 synonym: "regulation of MAP kinase cascade" EXACT [GOC:signaling]
 synonym: "regulation of MAP kinase kinase kinase cascade" EXACT []
 synonym: "regulation of MAPKKK cascade" EXACT [GOC:signaling]
@@ -261258,7 +261848,7 @@
 id: GO:0043409
 name: negative regulation of MAPK cascade
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of signal transduction mediated by the MAPKKK cascade." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of signal transduction mediated by the MAPKKK cascade." [GOC:curators]
 synonym: "down regulation of MAPK cascade" EXACT [GOC:signaling]
 synonym: "down regulation of MAPKKK cascade" EXACT []
 synonym: "down-regulation of MAPK cascade" EXACT [GOC:signaling]
@@ -261284,7 +261874,7 @@
 id: GO:0043410
 name: positive regulation of MAPK cascade
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of signal transduction mediated by the MAPK cascade." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of signal transduction mediated by the MAPK cascade." [GOC:curators]
 synonym: "activation of MAPK cascade" NARROW [GOC:signaling]
 synonym: "activation of MAPKKK cascade" NARROW []
 synonym: "positive regulation of MAP kinase cascade" EXACT [GOC:signaling]
@@ -261310,7 +261900,7 @@
 id: GO:0043411
 name: obsolete myopalladin binding
 namespace: molecular_function
-def: "OBSOLETE. Binding to myopalladin, a myofibrillar protein with titin-like Ig domains." [GOC:go_curators]
+def: "OBSOLETE. Binding to myopalladin, a myofibrillar protein with titin-like Ig domains." [GOC:curators]
 comment: This term was made obsolete because it represents binding to an individual protein.
 synonym: "myopalladin binding" EXACT []
 is_obsolete: true
@@ -261320,7 +261910,7 @@
 id: GO:0043412
 name: macromolecule modification
 namespace: biological_process
-def: "The covalent alteration of one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological macromolecule, resulting in a change in its properties." [GOC:go_curators]
+def: "The covalent alteration of one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological macromolecule, resulting in a change in its properties." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 subset: goslim_pir
 is_a: GO:0043170 ! macromolecule metabolic process
@@ -261340,7 +261930,7 @@
 id: GO:0043414
 name: macromolecule methylation
 namespace: biological_process
-def: "The covalent attachment of a methyl residue to one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological macromolecule." [GOC:go_curators]
+def: "The covalent attachment of a methyl residue to one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological macromolecule." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0032259 ! methylation
 is_a: GO:0043412 ! macromolecule modification
@@ -261940,7 +262530,7 @@
 namespace: biological_process
 def: "OBSOLETE. The anaerobic conversion of pyrimidine nucleobases, yielding energy in the form of ATP." [GOC:jl]
 comment: The reason for obsoletion is that this term is not clearly defined.
-synonym: "pyrimidine base fermentation" EXACT [GOC:go_curators]
+synonym: "pyrimidine base fermentation" EXACT [GOC:curators]
 synonym: "pyrimidine fermentation" RELATED []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29511" xsd:anyURI
 is_obsolete: true
@@ -262170,7 +262760,7 @@
 id: GO:0043489
 name: RNA stabilization
 namespace: biological_process
-def: "Prevention of degradation of RNA molecules." [GOC:go_curators]
+def: "Prevention of degradation of RNA molecules." [GOC:curators]
 is_a: GO:0043487 ! regulation of RNA stability
 is_a: GO:1902369 ! negative regulation of RNA catabolic process
 
@@ -262186,7 +262776,7 @@
 xref: Reactome:R-HSA-9856872 "Malate-aspartate shuttle"
 xref: Wikipedia:Malate-aspartate_shuttle
 is_a: GO:0019674 ! NAD+ metabolic process
-relationship: has_part GO:0004069 ! L-aspartate:2-oxoglutarate aminotransferase activity
+relationship: has_part GO:0004069 ! L-aspartate:2-oxoglutarate transaminase activity
 relationship: has_part GO:0030060 ! L-malate dehydrogenase (NAD+) activity
 relationship: has_part GO:1990542 ! mitochondrial transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28982" xsd:anyURI
@@ -262244,11 +262834,13 @@
 id: GO:0043495
 name: protein-membrane adaptor activity
 namespace: molecular_function
-def: "The binding activity of a molecule that brings together a protein or a protein complex with a membrane, either via membrane lipid binding or by interacting with a membrane protein, to establish or maintain the localization of the protein, protein complex or organelle." [GOC:go_curators]
+def: "The binding activity of a molecule that brings together a protein or a protein complex with a membrane, either via membrane lipid binding or by interacting with a membrane protein, to establish or maintain the localization of the protein, protein complex or organelle." [GOC:curators]
 synonym: "anchoring" RELATED []
 synonym: "protein membrane adaptor" EXACT []
 synonym: "protein membrane adaptor activity" EXACT []
 synonym: "protein membrane anchor" RELATED []
+xref: Reactome:R-HSA-5333678 "CPNEs bind PL"
+xref: Reactome:R-HSA-6809764 "SBF1 binds MTMR2"
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19090" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24964" xsd:anyURI
@@ -262321,7 +262913,7 @@
 id: GO:0043502
 name: regulation of muscle adaptation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of muscle adaptation." [GOC:go_curators, GOC:mtg_muscle]
+def: "Any process that modulates the frequency, rate or extent of muscle adaptation." [GOC:curators, GOC:mtg_muscle]
 synonym: "regulation of muscle plasticity" RELATED []
 is_a: GO:0048583 ! regulation of response to stimulus
 is_a: GO:0090257 ! regulation of muscle system process
@@ -262355,7 +262947,7 @@
 id: GO:0043505
 name: CENP-A containing nucleosome
 namespace: cellular_component
-def: "A form of nucleosome located only at the centromere, in which the histone H3 is replaced by the variant form CENP-A (sometimes known as CenH3)." [GOC:go_curators, PMID:15175412, PMID:16183641]
+def: "A form of nucleosome located only at the centromere, in which the histone H3 is replaced by the variant form CENP-A (sometimes known as CenH3)." [GOC:curators, PMID:15175412, PMID:16183641]
 synonym: "CenH3 containing nucleosome" EXACT []
 synonym: "CENP-S-T-W-X" EXACT [GOC:vw, PMID:22304917]
 synonym: "centromere specific nucleosome" RELATED []
@@ -262417,7 +263009,7 @@
 name: activin A complex
 namespace: cellular_component
 alt_id: GO:0048181
-def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-A subunits (sometimes known as activin beta-A or activin/inhibin beta-A)." [GOC:go_curators]
+def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-A subunits (sometimes known as activin beta-A or activin/inhibin beta-A)." [GOC:curators]
 comment: Note that the actions of the activin complex are the opposite of those of the inhibin complex, which is a dimer of an inhibin beta-A or inhibin beta-B subunit and an inhibin alpha subunit. See also the cellular component term 'inhibin complex ; GO:0043511'.
 is_a: GO:0048180 ! activin complex
 
@@ -262426,7 +263018,7 @@
 name: activin B complex
 namespace: cellular_component
 alt_id: GO:0048182
-def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-B subunits (sometimes known as activin beta-B or activin/inhibin beta-B)." [GOC:go_curators]
+def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-B subunits (sometimes known as activin beta-B or activin/inhibin beta-B)." [GOC:curators]
 comment: Note that the actions of the activin complex are the opposite of those of the inhibin complex, which is a dimer of an inhibin beta-A or inhibin beta-B subunit and a inhibin alpha subunit. See 'inhibin complex ; GO:0043511'.
 is_a: GO:0048180 ! activin complex
 
@@ -262573,7 +263165,7 @@
 id: GO:0043523
 name: regulation of neuron apoptotic process
 namespace: biological_process
-def: "Any process that modulates the occurrence or rate of cell death by apoptotic process in neurons." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "Any process that modulates the occurrence or rate of cell death by apoptotic process in neurons." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "regulation of apoptosis of neuronal cells" EXACT []
 synonym: "regulation of apoptosis of neurons" EXACT []
 synonym: "regulation of neuron apoptosis" NARROW []
@@ -262590,7 +263182,7 @@
 id: GO:0043524
 name: negative regulation of neuron apoptotic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell death by apoptotic process in neurons." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell death by apoptotic process in neurons." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "down regulation of neuron apoptosis" EXACT []
 synonym: "down-regulation of neuron apoptosis" EXACT []
 synonym: "downregulation of neuron apoptosis" EXACT []
@@ -262608,7 +263200,7 @@
 id: GO:0043525
 name: positive regulation of neuron apoptotic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cell death of neurons by apoptotic process." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "Any process that activates or increases the frequency, rate or extent of cell death of neurons by apoptotic process." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "activation of neuron apoptosis" NARROW []
 synonym: "positive regulation of neuron apoptosis" NARROW []
 synonym: "positive regulation of programmed cell death, neurons" EXACT []
@@ -262690,7 +263282,7 @@
 id: GO:0043533
 name: inositol 1,3,4,5 tetrakisphosphate binding
 namespace: molecular_function
-def: "Binding to inositol 1,3,4,5 tetrakisphosphate." [GOC:go_curators]
+def: "Binding to inositol 1,3,4,5 tetrakisphosphate." [GOC:curators]
 synonym: "InsP4 binding" EXACT []
 synonym: "IP4 binding" EXACT []
 is_a: GO:0043168 ! anion binding
@@ -262707,7 +263299,7 @@
 id: GO:0043535
 name: regulation of blood vessel endothelial cell migration
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:curators]
 is_a: GO:0010594 ! regulation of endothelial cell migration
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0043534 ! blood vessel endothelial cell migration
@@ -262717,7 +263309,7 @@
 id: GO:0043536
 name: positive regulation of blood vessel endothelial cell migration
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:curators]
 synonym: "activation of blood vessel endothelial cell migration" NARROW []
 synonym: "stimulation of blood vessel endothelial cell migration" NARROW []
 synonym: "up regulation of blood vessel endothelial cell migration" EXACT []
@@ -262733,7 +263325,7 @@
 id: GO:0043537
 name: negative regulation of blood vessel endothelial cell migration
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the migration of the endothelial cells of blood vessels." [GOC:curators]
 synonym: "down regulation of blood vessel endothelial cell migration" EXACT []
 synonym: "down-regulation of blood vessel endothelial cell migration" EXACT []
 synonym: "downregulation of blood vessel endothelial cell migration" EXACT []
@@ -262748,7 +263340,7 @@
 id: GO:0043538
 name: obsolete regulation of actin phosphorylation
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the transfer of one or more phosphate groups to an actin molecule." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the transfer of one or more phosphate groups to an actin molecule." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28278" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28482" xsd:anyURI
@@ -262758,7 +263350,7 @@
 id: GO:0043539
 name: protein serine/threonine kinase activator activity
 namespace: molecular_function
-def: "Binds to and increases the activity of a protein serine/threonine kinase." [GOC:go_curators]
+def: "Binds to and increases the activity of a protein serine/threonine kinase." [GOC:curators]
 synonym: "protein ser/thr kinase activator activity" EXACT []
 is_a: GO:0030295 ! protein kinase activator activity
 relationship: positively_regulates GO:0004674 ! protein serine/threonine kinase activity
@@ -262791,7 +263383,7 @@
 id: GO:0043542
 name: endothelial cell migration
 namespace: biological_process
-def: "The orderly movement of an endothelial cell into the extracellular matrix to form an endothelium." [GOC:go_curators]
+def: "The orderly movement of an endothelial cell into the extracellular matrix to form an endothelium." [GOC:curators]
 is_a: GO:0016477 ! cell migration
 
 [Term]
@@ -262800,7 +263392,6 @@
 namespace: biological_process
 def: "The addition of an acyl group, any group or radical of the form RCO- where R is an organic group, to a protein amino acid." [GOC:jl]
 subset: gocheck_obsoletion_candidate
-subset: goslim_yeast
 synonym: "protein amino acid acylation" EXACT [GOC:bf]
 is_a: GO:0036211 ! protein modification process
 
@@ -262808,7 +263399,7 @@
 id: GO:0043544
 name: lipoamide binding
 namespace: molecular_function
-def: "Binding to lipoamide, the functional form of lipoic acid in which the carboxyl group is attached to protein by an amide linkage to a lysine amino group." [GOC:go_curators]
+def: "Binding to lipoamide, the functional form of lipoic acid in which the carboxyl group is attached to protein by an amide linkage to a lysine amino group." [GOC:curators]
 is_a: GO:1901681 ! sulfur compound binding
 
 [Term]
@@ -263331,7 +263922,7 @@
 name: endospore external encapsulating structure
 namespace: cellular_component
 alt_id: GO:0055030
-def: "The structures that lie outside the inner membrane and surround the entire endospore; consists of a peptidoglycan-containing inner layer (the endospore cortex) surrounded by a multilayered proteinaceous coat. An exosporium may be present as an extreme outer layer." [GOC:go_curators, PMID:15035041]
+def: "The structures that lie outside the inner membrane and surround the entire endospore; consists of a peptidoglycan-containing inner layer (the endospore cortex) surrounded by a multilayered proteinaceous coat. An exosporium may be present as an extreme outer layer." [GOC:curators, PMID:15035041]
 subset: goslim_pir
 synonym: "endospore wall" EXACT [GOC:mah]
 synonym: "peptidoglycan-based spore wall" EXACT [GOC:mtg_sensu]
@@ -263585,7 +264176,7 @@
 id: GO:0043615
 name: astrocyte cell migration
 namespace: biological_process
-def: "The orderly movement of an astrocyte, a class of large neuroglial (macroglial) cells in the central nervous system, the largest and most numerous neuroglial cells in the brain and spinal cord." [CL:0000127, GOC:go_curators]
+def: "The orderly movement of an astrocyte, a class of large neuroglial (macroglial) cells in the central nervous system, the largest and most numerous neuroglial cells in the brain and spinal cord." [CL:0000127, GOC:curators]
 synonym: "astrocyte migration" EXACT []
 synonym: "astrocytic glial cell migration" EXACT []
 is_a: GO:0008347 ! glial cell migration
@@ -264229,10 +264820,6 @@
 synonym: "Cu2+-exporting ATPase activity" RELATED [EC:7.2.2.9]
 xref: EC:7.2.2.9
 xref: MetaCyc:TRANS-RXN-178
-xref: Reactome:R-HSA-3697838 "ATP7A transfers Cu from ATOX1 to SOD3"
-xref: Reactome:R-HSA-6803545 "ATP7A transports cytosolic Cu2+ to phagosomal lumen"
-xref: Reactome:R-HSA-936802 "ATP7A transports cytosolic Cu2+ to extracellular region"
-xref: Reactome:R-HSA-936895 "ATP7B transports cytosolic Cu2+ to Golgi lumen"
 xref: RHEA:10376
 is_a: GO:0005375 ! copper ion transmembrane transporter activity
 is_a: GO:0015662 ! P-type ion transporter activity
@@ -264717,10 +265304,12 @@
 
 [Term]
 id: GO:0043730
-name: 5-ureido-4-imidazole carboxylate hydrolase activity
+name: obsolete 5-ureido-4-imidazole carboxylate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-ureido-4-imidazole carboxylate + H2O = 5-amino-4-imidazole carboxylate + NH3 + CO2." [GOC:jl]
-is_a: GO:0016813 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
+def: "OBSOLETE. Catalysis of the reaction: 5-ureido-4-imidazole carboxylate + H2O = 5-amino-4-imidazole carboxylate + NH3 + CO2." [GOC:jl]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0043731
@@ -265011,12 +265600,16 @@
 xref: EC:2.3.1.168
 xref: MetaCyc:2.3.1.168-RXN
 xref: RHEA:18865
+xref: RHEA:84647
+xref: RHEA:84651
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 property_value: skos:broadMatch EC:1.2.1.25
 property_value: skos:broadMatch MetaCyc:1.2.1.25-RXN
 property_value: skos:exactMatch EC:2.3.1.168
 property_value: skos:exactMatch MetaCyc:2.3.1.168-RXN
 property_value: skos:narrowMatch RHEA:18865
+property_value: skos:narrowMatch RHEA:84647
+property_value: skos:narrowMatch RHEA:84651
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17835" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29951" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30031" xsd:anyURI
@@ -265088,7 +265681,7 @@
 xref: EC:6.2.1.13
 xref: MetaCyc:ACETATE--COA-LIGASE-ADP-FORMING-RXN
 xref: RHEA:15081
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.13
 property_value: skos:exactMatch RHEA:15081
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -265111,12 +265704,16 @@
 
 [Term]
 id: GO:0043760
-name: acetyldiaminopimelate aminotransferase activity
+name: obsolete acetyldiaminopimelate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-acetyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-acetyl-2-L-amino-6-oxoheptanedioate + L-glutamate." [PMID:1906065]
+def: "OBSOLETE. Catalysis of the reaction: N-acetyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-acetyl-2-L-amino-6-oxoheptanedioate + L-glutamate." [PMID:1906065]
+comment: This term was obsoleted because there is no evidence that this reaction exists. This was an alternative succinyldiaminopimelate aminotransferase substrate tested in PMID:1906065.
+synonym: "acetyldiaminopimelate aminotransferase activity" BROAD []
 synonym: "N-acetyl-diaminopimelate aminotransferase activity" EXACT []
 synonym: "N-acetyl-L,L-diaminopimelate aminotransferase activity" EXACT []
-is_a: GO:0008483 ! transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+consider: GO:0009016
 
 [Term]
 id: GO:0043761
@@ -265558,7 +266155,7 @@
 id: GO:0043799
 name: glycine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reactions: (1) glycine + H2O + O2 = glyoxylate + NH3 + hydrogen peroxide; (2) D-alanine + H2O + O2 = pyruvate + NH3 + hydrogen peroxide; (3) sarcosine + H2O + O2 = glyoxylate + methylamine + hydrogen peroxide; (4) N-ethylglycine + H2O + O2 = glyoxylate + ethylamine + hydrogen peroxide." [EC:1.4.3.19]
+def: "Catalysis of the reactions: (1) glycine + H2O + O2 = glyoxylate + NH4+ + H2O2; (2) D-alanine + H2O + O2 = pyruvate + NH4+ + H2O2; (3) sarcosine + H2O + O2 = glyoxylate + methylamine + H2O2; (4) N-ethylglycine + H2O + O2 = glyoxylate + ethylamine + H2O2." [EC:1.4.3.19]
 synonym: "glycine:oxygen oxidoreductase (deaminating)" EXACT []
 xref: EC:1.4.3.19
 xref: MetaCyc:1.4.3.19-RXN
@@ -265948,9 +266545,9 @@
 
 [Term]
 id: GO:0043825
-name: succinylornithine transaminase activity
+name: succinylornithine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-succinyl-L-ornithine = N-succinyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.81, RHEA:16953]
+def: "Catalysis of the reaction: N(2)-succinyl-L-ornithine + 2-oxoglutarate = N-succinyl-L-glutamate 5-semialdehyde + L-glutamate." [RHEA:16953]
 synonym: "2-N-succinyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity" RELATED [EC:2.6.1.81]
 synonym: "AstC" RELATED []
 synonym: "N(2)-succinylornithine 5-aminotransferase activity" RELATED [EC:2.6.1.81]
@@ -265960,6 +266557,7 @@
 synonym: "succinyl ornithine transaminase activity" EXACT []
 synonym: "succinyl-ornithine transaminase activity" RELATED [EC:2.6.1.81]
 synonym: "succinylornithine aminotransferase activity" BROAD [EC:2.6.1.81]
+synonym: "succinylornithine transaminase activity" EXACT []
 xref: EC:2.6.1.81
 xref: KEGG_REACTION:R04217
 xref: MetaCyc:SUCCORNTRANSAM-RXN
@@ -265968,6 +266566,7 @@
 property_value: skos:exactMatch EC:2.6.1.81
 property_value: skos:exactMatch RHEA:16953
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0043826
@@ -266018,7 +266617,7 @@
 id: GO:0043829
 name: tRNA-specific adenosine-37 deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine-37 + H2O = inosine-37 + NH3, in a tRNA-Ala molecule." [PMID:8915538, PMID:9707437, RHEA:50968]
+def: "Catalysis of the reaction: adenosine-37 + H2O = inosine-37 + NH4+, in a tRNA-Ala molecule." [PMID:8915538, PMID:9707437, RHEA:50968]
 synonym: "TAD1" RELATED []
 synonym: "tRNA(Ala)-A37 deaminase activity" EXACT []
 synonym: "tRNA-specific adenosine deaminase 1" EXACT []
@@ -266314,7 +266913,7 @@
 id: GO:0043852
 name: monomethylamine methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: monomethylamine + a monomethylamine corrinoid protein = a methylated monomethylamine corrinoid protein + NH3." [PMID:9195968]
+def: "Catalysis of the reaction: Co(I)-[methylamine-specific corrinoid protein] + methylamine + H+ = methyl-Co(III)-[methylamine-specific corrinoid protein] + NH4+." [PMID:9195968, RHEA:26059]
 comment: This function is the first step in the pathway of methanogenesis from monomethylamine.
 synonym: "MMAMT" RELATED []
 synonym: "monomethylamine:corrinoid methyltransferase activity" EXACT []
@@ -266644,7 +267243,7 @@
 id: GO:0043877
 name: galactosamine-6-phosphate isomerase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-galactosamine 6-phosphate + H2O = D-tagatose 6-phosphate + NH3." [PMID:10931310]
+def: "Catalysis of the reaction: D-galactosamine 6-phosphate + H2O = D-tagatose 6-phosphate + NH4+." [PMID:10931310]
 comment: This function is part of the pathway of N-acetyl-galactosamine and galactosamine utilization.
 synonym: "AgaI" RELATED []
 synonym: "galactosamine-6-phosphate deaminase activity" EXACT []
@@ -267139,11 +267738,15 @@
 
 [Term]
 id: GO:0043911
-name: D-lysine transaminase activity
+name: D-lysine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-lysine + 2-oxoglutarate = L-glutamate + 6-amino-2-oxohexanoate." [GOC:jl, PMID:17259313]
+def: "Catalysis of the reaction: D-lysine + 2-oxoglutarate = 6-amino-2-oxohexanoate + L-glutamate." [PMID:17259313, RHEA:37587]
 synonym: "D-lysine aminotransferase activity" EXACT []
-is_a: GO:0008483 ! transaminase activity
+synonym: "D-lysine transaminase activity" BROAD []
+xref: RHEA:37587
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: skos:exactMatch RHEA:37587
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0043912
@@ -267168,21 +267771,27 @@
 
 [Term]
 id: GO:0043914
-name: NADPH:sulfur oxidoreductase activity
+name: NAD(P)H sulfur oxidoreductase (CoA-dependent) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NADPH + H+ + sulfur = hydrogen sulfide + NADP+." [GOC:jl, PMID:17449625]
+def: "Catalysis of the reaction: hydrogen sulfide + NAD(P)+ = sulfur + NAD(P)H." [EC:1.8.1.18, PMID:17449625]
 synonym: "CoA-dependent NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "coenzyme A-dependent NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H elemental sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H sulfur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H sulphur oxidoreductase activity" EXACT []
 synonym: "NAD(P)H:sulfur oxidoreductase activity" EXACT []
+synonym: "NADPH:sulfur oxidoreductase activity" NARROW []
 synonym: "NADPH:sulphur oxidoreductase activity" EXACT []
 synonym: "NSR" EXACT []
 xref: EC:1.8.1.18
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+xref: RHEA:36595
+xref: RHEA:36599
+is_a: GO:0016668 ! oxidoreductase activity, acting on a sulfur group of donors, NAD(P) as acceptor
 property_value: skos:exactMatch EC:1.8.1.18
+property_value: skos:narrowMatch RHEA:36595
+property_value: skos:narrowMatch RHEA:36599
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0043915
@@ -267648,7 +268257,6 @@
 xref: EC:6.2.1.36
 xref: RHEA:26534
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.36
 property_value: skos:exactMatch RHEA:26534
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -267728,10 +268336,12 @@
 
 [Term]
 id: GO:0043960
-name: L-erythro-3-methylmalyl-CoA dehydratase activity
+name: obsolete L-erythro-3-methylmalyl-CoA dehydratase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-erythro-3-methylmalyl-CoA = mesaconyl-CoA + H2O." [GOC:jl]
-is_a: GO:0016836 ! hydro-lyase activity
+def: "OBSOLETE. Catalysis of the reaction: L-erythro-3-methylmalyl-CoA = mesaconyl-CoA + H2O." [GOC:jl]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0043961
@@ -268249,6 +268859,8 @@
 synonym: "regulation by symbiont of host system process" NARROW []
 xref: Reactome:R-HSA-9635644 "Inhibition of membrane repair"
 xref: Reactome:R-HSA-9636667 "Manipulation of host energy metabolism"
+xref: Reactome:R-HSA-9833110 "RSV-host interactions"
+xref: Reactome:R-HSA-9918481 "Dengue Virus-Host Interactions"
 is_a: GO:0035821 ! modulation of process of another organism
 is_a: GO:0051701 ! biological process involved in interaction with host
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18719" xsd:anyURI
@@ -268654,7 +269266,7 @@
 id: GO:0044038
 name: cell wall macromolecule biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of a macromolecule destined to form part of a cell wall." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of a macromolecule destined to form part of a cell wall." [GOC:curators]
 synonym: "cell wall macromolecule anabolism" EXACT [GOC:mah]
 synonym: "cell wall macromolecule biosynthesis" EXACT [GOC:mah]
 synonym: "cell wall macromolecule biosynthetic process at cellular level" EXACT [GOC:mah]
@@ -270092,6 +270704,7 @@
 name: protein folding chaperone
 namespace: molecular_function
 def: "Binding to a protein or a protein-containing complex to assist the protein folding process." [GOC:mtg_cambridge_2009]
+comment: Note that a protein folding chaperone binds an unfolded protein to fold it. Do not confuse with unfolded protein holdase activity, which binds an unfolded protein and keeps it unfolded to deliver it to its final destination.
 subset: goslim_chembl
 subset: goslim_drosophila
 subset: goslim_generic
@@ -270105,6 +270718,7 @@
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0051082 ! unfolded protein binding
 relationship: part_of GO:0006457 ! protein folding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30552" xsd:anyURI
 created_by: jl
 creation_date: 2009-09-25T11:33:48Z
 
@@ -270628,11 +271242,10 @@
 id: GO:0044237
 name: obsolete cellular metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways by which individual cells transform chemical substances." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways by which individual cells transform chemical substances." [GOC:curators]
 comment: The reason for obsoletion is that this term is now redundant with GO:0008152, metabolic process
 synonym: "cellular metabolism" EXACT []
 synonym: "intermediary metabolism" RELATED [GOC:mah]
-xref: Reactome:R-HSA-1428517 "Aerobic respiration and respiratory electron transport"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008152
@@ -270641,13 +271254,14 @@
 id: GO:0044238
 name: primary metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving those compounds which are formed as a part of the normal anabolic and catabolic processes. These processes take place in most, if not all, cells of the organism." [GOC:go_curators]
+def: "The chemical reactions and pathways involving those compounds which are formed as a part of the normal anabolic and catabolic processes. These processes take place in most, if not all, cells of the organism." [GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_euk_cellular_processes_ribbon
 subset: goslim_pir
 subset: goslim_plant_ribbon
 subset: goslim_prokaryote_ribbon
 synonym: "primary metabolism" EXACT []
+xref: Reactome:R-HSA-71291 "Metabolism of amino acids and derivatives"
 xref: Wikipedia:Primary_metabolite
 is_a: GO:0008152 ! metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26424" xsd:anyURI
@@ -270668,7 +271282,7 @@
 id: GO:0044241
 name: lipid digestion
 namespace: biological_process
-def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested lipids into components that may be easily absorbed and directed into metabolism." [GOC:go_curators]
+def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested lipids into components that may be easily absorbed and directed into metabolism." [GOC:curators]
 xref: Reactome:R-HSA-192456 "Digestion of dietary lipid"
 is_a: GO:0007586 ! digestion
 
@@ -270690,7 +271304,7 @@
 id: GO:0044245
 name: polysaccharide digestion
 namespace: biological_process
-def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested polysaccharides into components that may be easily absorbed and directed into metabolism." [GOC:go_curators]
+def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested polysaccharides into components that may be easily absorbed and directed into metabolism." [GOC:curators]
 xref: Reactome:R-HSA-189085 "Digestion of dietary carbohydrate"
 is_a: GO:0007586 ! digestion
 
@@ -270775,7 +271389,7 @@
 id: GO:0044256
 name: protein digestion
 namespace: biological_process
-def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested proteins into components that may be easily absorbed and directed into metabolism." [GOC:go_curators]
+def: "The whole of the physical, chemical, and biochemical processes carried out by living organisms to break down ingested proteins into components that may be easily absorbed and directed into metabolism." [GOC:curators]
 is_a: GO:0007586 ! digestion
 
 [Term]
@@ -270953,11 +271567,13 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:pde, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 subset: goslim_chembl
 subset: goslim_flybase_ribbon
 subset: goslim_metagenomics
 synonym: "small molecule metabolism" EXACT []
 is_a: GO:0008152 ! metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:05:20Z
 
@@ -270967,9 +271583,11 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 synonym: "small molecule catabolism" EXACT []
 is_a: GO:0009056 ! catabolic process
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:06:10Z
 
@@ -270979,11 +271597,13 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of small molecules, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:pde, GOC:vw]
 comment: Small molecules in GO include monosaccharides but exclude disaccharides and polysaccharides.
+subset: gocheck_do_not_annotate
 subset: goslim_prokaryote
 subset: goslim_prokaryote_ribbon
 synonym: "small molecule biosynthesis" EXACT []
 is_a: GO:0009058 ! biosynthetic process
 is_a: GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: jl
 creation_date: 2010-01-26T12:06:49Z
 
@@ -271122,7 +271742,7 @@
 id: GO:0044297
 name: cell body
 namespace: cellular_component
-def: "The portion of a cell bearing surface projections such as axons, dendrites, cilia, or flagella that includes the nucleus, but excludes all cell projections." [GOC:go_curators]
+def: "The portion of a cell bearing surface projections such as axons, dendrites, cilia, or flagella that includes the nucleus, but excludes all cell projections." [GOC:curators]
 comment: Note that 'cell body' and 'cell soma' are not used in the literature for cells that lack projections, nor for some cells (e.g. yeast with mating projections) that do have projections.
 synonym: "cell soma" EXACT []
 xref: FBbt:00005107
@@ -272284,6 +272904,7 @@
 synonym: "symbiotic interaction between organisms" RELATED []
 synonym: "symbiotic interaction between species" RELATED []
 synonym: "symbiotic process" RELATED []
+xref: Reactome:R-HSA-9839923 "Dengue Virus Infection"
 is_a: GO:0044419 ! biological process involved in interspecies interaction between organisms
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14807" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20191" xsd:anyURI
@@ -272309,6 +272930,7 @@
 synonym: "adhesion to other organism during symbiotic interaction" RELATED [GOC:dph]
 synonym: "adhesion to other organism involved in symbiotic interaction" EXACT []
 synonym: "host adhesion" NARROW []
+xref: Reactome:R-HSA-9638630 "Attachment of bacteria to epithelial cells"
 is_a: GO:0051701 ! biological process involved in interaction with host
 
 [Term]
@@ -273876,7 +274498,7 @@
 id: GO:0044540
 name: L-cystine L-cysteine-lyase (deaminating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-cystine + H2O = pyruvate + NH3 + thiocysteine. Thiocysteine is also known as cysteine persulfide." [RHEA:24927]
+def: "Catalysis of the reaction: L-cystine + H2O = S-sulfanyl-L-cysteine + pyruvate + NH4+." [RHEA:24927]
 xref: EC:4.4.1.35
 xref: KEGG_REACTION:R02408
 xref: RHEA:24927
@@ -277099,6 +277721,7 @@
 synonym: "iron acquisition" BROAD []
 synonym: "iron acquisition by symbiont from host" EXACT []
 synonym: "iron acquisition by symbiont from host heme" NARROW []
+xref: Reactome:R-HSA-9638482 "Metal ion assimilation from the host"
 is_a: GO:0044002 ! acquisition of nutrients from host
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17896" xsd:anyURI
 created_by: jl
@@ -277652,7 +278275,7 @@
 id: GO:0045019
 name: negative regulation of nitric oxide biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:curators]
 synonym: "down regulation of nitric oxide biosynthetic process" EXACT []
 synonym: "down-regulation of nitric oxide biosynthetic process" EXACT []
 synonym: "downregulation of nitric oxide biosynthetic process" EXACT []
@@ -277728,14 +278351,16 @@
 
 [Term]
 id: GO:0045026
-name: plasma membrane fusion
+name: obsolete plasma membrane fusion
 namespace: biological_process
 alt_id: GO:0006947
-def: "The joining of the lipid bilayer membrane that surround a cell with that of another cell, producing a single cell." [GOC:elh, GOC:mtg_muscle]
+def: "OBSOLETE. The joining of the lipid bilayer membrane that surround a cell with that of another cell, producing a single cell." [GOC:elh, GOC:mtg_muscle]
+comment: This term is represented by another term.
 synonym: "cell fusion" BROAD []
 synonym: "cell-cell fusion" BROAD []
-is_a: GO:0007009 ! plasma membrane organization
-is_a: GO:0061025 ! membrane fusion
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31270" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0140253
 
 [Term]
 id: GO:0045027
@@ -278125,6 +278750,7 @@
 def: "The process in which a relatively unspecialized T cell acquires the specialized features of a T-helper 1 (Th1) cell. A Th1 cell is a CD4-positive, alpha-beta T cell that has the phenotype T-bet-positive and produces interferon-gamma." [CL:0000545, GOC:ebc]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "T-helper 1 cell development" RELATED [GOC:add]
+xref: Reactome:R-HSA-9942503 "Differentiation of naive CD+ T cells to T helper 1 cells (Th1 cells)"
 is_a: GO:0002287 ! alpha-beta T cell activation involved in immune response
 is_a: GO:0002292 ! T cell differentiation involved in immune response
 is_a: GO:0042093 ! T-helper cell differentiation
@@ -278222,7 +278848,7 @@
 id: GO:0045071
 name: negative regulation of viral genome replication
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of viral genome replication." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of viral genome replication." [GOC:curators]
 synonym: "down regulation of viral genome replication" EXACT []
 synonym: "down-regulation of viral genome replication" EXACT []
 synonym: "downregulation of viral genome replication" EXACT []
@@ -278281,7 +278907,7 @@
 id: GO:0045091
 name: regulation of single stranded viral RNA replication via double stranded DNA intermediate
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of single stranded viral RNA replication via double stranded DNA intermediate." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of single stranded viral RNA replication via double stranded DNA intermediate." [GOC:curators]
 synonym: "regulation of retroviral genome replication" EXACT [GOC:bf, GOC:jl]
 is_a: GO:0045069 ! regulation of viral genome replication
 is_a: GO:2001141 ! regulation of RNA biosynthetic process
@@ -278479,7 +279105,7 @@
 id: GO:0045112
 name: integrin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of integrins, a large family of transmembrane proteins that act as receptors for cell-adhesion molecules." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways resulting in the formation of integrins, a large family of transmembrane proteins that act as receptors for cell-adhesion molecules." [GOC:curators, ISBN:0198506732]
 synonym: "integrin anabolism" EXACT []
 synonym: "integrin biosynthesis" EXACT []
 synonym: "integrin formation" EXACT []
@@ -278491,7 +279117,7 @@
 id: GO:0045113
 name: regulation of integrin biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:curators]
 synonym: "regulation of integrin anabolism" EXACT []
 synonym: "regulation of integrin biosynthesis" EXACT []
 synonym: "regulation of integrin formation" EXACT []
@@ -278505,7 +279131,7 @@
 id: GO:0045114
 name: beta 2 integrin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of beta 2 integrins, a subfamily of integrins which contain the beta 2 subunit." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of beta 2 integrins, a subfamily of integrins which contain the beta 2 subunit." [GOC:curators]
 synonym: "beta 2 integrin anabolism" EXACT []
 synonym: "beta 2 integrin biosynthesis" EXACT []
 synonym: "beta 2 integrin formation" EXACT []
@@ -278516,7 +279142,7 @@
 id: GO:0045115
 name: regulation of beta 2 integrin biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:curators]
 synonym: "regulation of beta 2 integrin anabolism" EXACT []
 synonym: "regulation of beta 2 integrin biosynthesis" EXACT []
 synonym: "regulation of beta 2 integrin formation" EXACT []
@@ -278540,7 +279166,7 @@
 id: GO:0045117
 name: azole transmembrane transport
 namespace: biological_process
-def: "The directed movement of azoles, heterocyclic compounds found in many biologically important substances, across a lipid bilayer, across a membrane." [GOC:go_curators, ISBN:3527307206, Wikipedia:Azole]
+def: "The directed movement of azoles, heterocyclic compounds found in many biologically important substances, across a lipid bilayer, across a membrane." [GOC:curators, ISBN:3527307206, Wikipedia:Azole]
 synonym: "azole transport" RELATED []
 is_a: GO:0055085 ! transmembrane transport
 is_a: GO:0071705 ! nitrogen compound transport
@@ -278627,6 +279253,7 @@
 name: bioactive lipid receptor activity
 namespace: molecular_function
 def: "Combining with a bioactive lipid and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex. A bioactive lipid is a lipid for which changes in lipid levels result in functional consequences in a variety of cellular processes." [GOC:bf, GOC:mah, PMID:12215548, PMID:18216770]
+xref: Reactome:R-HSA-400434 "FFAR1 binds fatty acids"
 is_a: GO:0004930 ! G protein-coupled receptor activity
 
 [Term]
@@ -278692,7 +279319,7 @@
 synonym: "keratan sulphotransferase activity" EXACT []
 xref: EC:2.8.2.21
 xref: MetaCyc:KERATAN-SULFOTRANSFERASE-RXN
-xref: Reactome:R-HSA-2046175 "Further sulfation on galactose residues produces KSPG"
+xref: Reactome:R-HSA-2046175 "CHST1 transfers sulfate to Gal on keratan chain"
 is_a: GO:0050698 ! proteoglycan sulfotransferase activity
 property_value: skos:exactMatch EC:2.8.2.21
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28380" xsd:anyURI
@@ -278802,7 +279429,7 @@
 id: GO:0045139
 name: obsolete copper sensitivity/resistance
 namespace: biological_process
-def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:go_curators]
+def: "OBSOLETE. (Was not defined before being made obsolete)." [GOC:curators]
 comment: This term was made obsolete because 'sensitivity/resistance' implies a phenotype rather than a biological process.
 synonym: "copper sensitivity/resistance" EXACT []
 is_obsolete: true
@@ -278892,7 +279519,7 @@
 id: GO:0045147
 name: obsolete regulation of initiation of acetate catabolic process by acetate
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "regulation of initiation of acetate breakdown by acetate" EXACT []
 synonym: "regulation of initiation of acetate degradation by acetate" EXACT []
@@ -279085,7 +279712,7 @@
 id: GO:0045168
 name: cell-cell signaling involved in cell fate commitment
 namespace: biological_process
-def: "Signaling at long or short range between cells that results in the commitment of a cell to a certain fate." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Signaling at long or short range between cells that results in the commitment of a cell to a certain fate." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "cell fate commitment, cell-cell signaling" EXACT []
 synonym: "cell fate commitment, cell-cell signalling" EXACT []
 synonym: "cell-cell signaling during in cell fate commitment" EXACT []
@@ -279132,7 +279759,7 @@
 id: GO:0045173
 name: O-sialoglycoprotein catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of O-sialoglycoproteins, glycoproteins which contain sialic acid as one of their carbohydrates. They are often found on or in the cell or tissue membranes and participate in a variety of biological activities." [GOC:go_curators, PMID:8824323]
+def: "The chemical reactions and pathways resulting in the breakdown of O-sialoglycoproteins, glycoproteins which contain sialic acid as one of their carbohydrates. They are often found on or in the cell or tissue membranes and participate in a variety of biological activities." [GOC:curators, PMID:8824323]
 synonym: "O-sialoglycoprotein breakdown" EXACT []
 synonym: "O-sialoglycoprotein catabolism" EXACT []
 synonym: "O-sialoglycoprotein degradation" EXACT []
@@ -279301,7 +279928,7 @@
 id: GO:0045188
 name: regulation of circadian sleep/wake cycle, non-REM sleep
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of non-rapid eye movement sleep." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of non-rapid eye movement sleep." [GOC:curators]
 synonym: "regulation of non-REM sleep" EXACT []
 is_a: GO:0045187 ! regulation of circadian sleep/wake cycle, sleep
 intersection_of: GO:0065007 ! biological regulation
@@ -279354,7 +279981,7 @@
 id: GO:0045192
 name: obsolete low-density lipoprotein catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of low-density lipoproteins, also known as beta lipoproteins, one of the classes of lipoproteins found in the bloodstream of animals, acting as a carrier for cholesterol and fats." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of low-density lipoproteins, also known as beta lipoproteins, one of the classes of lipoproteins found in the bloodstream of animals, acting as a carrier for cholesterol and fats." [GOC:curators, ISBN:0198506732]
 comment: This term was made obsolete because low-density lipoprotein is a macromolecular complex, not a single class of molecule; its degradation involves the dissociation of non-covalently attached constituents as well as some breaking and reforming of covalent bonds.
 synonym: "LDL catabolic process" EXACT []
 synonym: "LDL catabolism" EXACT []
@@ -279368,7 +279995,7 @@
 id: GO:0045193
 name: obsolete acetylated low-density lipoprotein catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of acetylated low-density lipoproteins." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of acetylated low-density lipoproteins." [GOC:curators]
 comment: This term was made obsolete because low-density lipoprotein is a macromolecular complex, not a single class of molecule; its degradation involves the dissociation of non-covalently attached constituents as well as some breaking and reforming of covalent bonds.
 synonym: "Ac-LDL catabolic process" EXACT []
 synonym: "Ac-LDL catabolism" EXACT []
@@ -279382,7 +280009,7 @@
 id: GO:0045194
 name: obsolete oxidized low-density lipoprotein catabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of oxidized low-density lipoproteins." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of oxidized low-density lipoproteins." [GOC:curators]
 comment: This term was made obsolete because low-density lipoprotein is a macromolecular complex, not a single class of molecule; its degradation involves the dissociation of non-covalently attached constituents as well as some breaking and reforming of covalent bonds.
 synonym: "Ox-LDL catabolic process" EXACT []
 synonym: "Ox-LDL catabolism" EXACT []
@@ -279482,7 +280109,7 @@
 id: GO:0045203
 name: obsolete integral component of cell outer membrane
 namespace: cellular_component
-def: "OBSOLETE. The component of the cell outer membrane consisting of the gene products having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:dos, GOC:go_curators, GOC:mtg_sensu]
+def: "OBSOLETE. The component of the cell outer membrane consisting of the gene products having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane." [GOC:curators, GOC:dos, GOC:mtg_sensu]
 comment: This term was obsoleted because it represents protein topology, not a cellular component.
 synonym: "integral to cell outer membrane" NARROW []
 synonym: "integral to external membrane" RELATED []
@@ -279605,7 +280232,7 @@
 id: GO:0045213
 name: obsolete neurotransmitter receptor metabolic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways involving neurotransmitter receptors." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways involving neurotransmitter receptors." [GOC:curators]
 comment: The reason for obsoletion is that 'neurotransmitter' is unnecessary grouping in the metabolism branch of the ontology, and it creates true path violations.
 synonym: "neurotransmitter receptor metabolism" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25736" xsd:anyURI
@@ -279654,7 +280281,7 @@
 id: GO:0045219
 name: obsolete regulation of FasL production
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "regulation of FasL anabolism" EXACT []
 synonym: "regulation of FasL biosynthesis" EXACT []
@@ -279668,7 +280295,7 @@
 id: GO:0045220
 name: obsolete positive regulation of FasL production
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "activation of FasL biosynthetic process" NARROW []
 synonym: "positive regulation of FasL anabolism" EXACT []
@@ -279688,7 +280315,7 @@
 id: GO:0045221
 name: obsolete negative regulation of FasL production
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of FasL." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "down regulation of FasL biosynthetic process" EXACT []
 synonym: "down-regulation of FasL biosynthetic process" EXACT []
@@ -279707,7 +280334,7 @@
 id: GO:0045222
 name: obsolete CD4 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of CD4, a CD marker that occurs on T-helper cells and is involved in MHC class II restricted interactions." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of CD4, a CD marker that occurs on T-helper cells and is involved in MHC class II restricted interactions." [GOC:curators, ISBN:0198506732]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "CD4 anabolism" EXACT []
 synonym: "CD4 biosynthesis" EXACT []
@@ -279721,7 +280348,7 @@
 id: GO:0045223
 name: obsolete regulation of CD4 production
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "regulation of CD4 anabolism" EXACT []
 synonym: "regulation of CD4 biosynthesis" EXACT []
@@ -279736,7 +280363,7 @@
 id: GO:0045224
 name: obsolete positive regulation of CD4 production
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "activation of CD4 biosynthetic process" NARROW []
 synonym: "positive regulation of CD4 anabolism" EXACT []
@@ -279756,7 +280383,7 @@
 id: GO:0045225
 name: obsolete negative regulation of CD4 production
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of CD4." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a gene product.
 synonym: "down regulation of CD4 biosynthetic process" EXACT []
 synonym: "down-regulation of CD4 biosynthetic process" EXACT []
@@ -279775,7 +280402,7 @@
 id: GO:0045226
 name: obsolete extracellular polysaccharide biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of polysaccharides used in extracellular structures." [GOC:ai, GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of polysaccharides used in extracellular structures." [GOC:ai, GOC:curators]
 comment: The reason for obsoletion is that this term is now redundant with GO:0000271, polysaccharide biosynthetic process
 synonym: "extracellular polysaccharide anabolism" EXACT []
 synonym: "extracellular polysaccharide biosynthesis" EXACT []
@@ -279789,7 +280416,7 @@
 id: GO:0045227
 name: capsule polysaccharide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of polysaccharides that make up the capsule, a protective structure surrounding some species of bacteria and fungi." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of polysaccharides that make up the capsule, a protective structure surrounding some species of bacteria and fungi." [GOC:curators]
 synonym: "capsular polysaccharide biosynthesis" EXACT []
 synonym: "capsular polysaccharide biosynthetic process" EXACT []
 synonym: "capsule polysaccharide anabolism" EXACT []
@@ -279803,7 +280430,7 @@
 id: GO:0045228
 name: slime layer polysaccharide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of polysaccharides in the slime layer, a diffused layer of polysaccharide exterior to the bacterial cell wall." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of polysaccharides in the slime layer, a diffused layer of polysaccharide exterior to the bacterial cell wall." [GOC:curators]
 synonym: "slime layer polysaccharide anabolism" EXACT []
 synonym: "slime layer polysaccharide biosynthesis" EXACT []
 synonym: "slime layer polysaccharide formation" EXACT []
@@ -280551,15 +281178,16 @@
 
 [Term]
 id: GO:0045303
-name: diaminobutyrate-2-oxoglutarate transaminase activity
+name: L-2,4-diaminobutyrate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-2,4-diaminobutyrate = L-aspartate 4-semialdehyde + L-glutamate." [EC:2.6.1.76, RHEA:11160]
+def: "Catalysis of the reaction: L-2,4-diaminobutanoate + 2-oxoglutarate = L-aspartate 4-semialdehyde + L-glutamate." [RHEA:11160]
 synonym: "2,4-diaminobutyrate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "DAB aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "DABA aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminibutyric acid aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminobutyrate transaminase activity" RELATED [EC:2.6.1.76]
 synonym: "diaminobutyrate--2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.76]
+synonym: "diaminobutyrate-2-oxoglutarate transaminase activity" EXACT []
 synonym: "EctB" RELATED [EC:2.6.1.76]
 synonym: "L-2,4-diaminobutanoate:2-oxoglutarate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
 synonym: "L-2,4-diaminobutyrate:2-ketoglutarate 4-aminotransferase activity" RELATED [EC:2.6.1.76]
@@ -280572,6 +281200,7 @@
 property_value: skos:exactMatch EC:2.6.1.76
 property_value: skos:exactMatch RHEA:11160
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0045304
@@ -280619,7 +281248,7 @@
 id: GO:0045309
 name: protein phosphorylated amino acid binding
 namespace: molecular_function
-def: "Binding to a phosphorylated amino acid residue within a protein." [GOC:go_curators]
+def: "Binding to a phosphorylated amino acid residue within a protein." [GOC:curators]
 synonym: "phosphoprotein amino acid binding" RELATED []
 is_a: GO:0051219 ! phosphoprotein binding
 
@@ -280627,7 +281256,7 @@
 id: GO:0045310
 name: obsolete phosphoserine/phosphothreonine binding
 namespace: molecular_function
-def: "OBSOLETE. Interacting selectively with a phosphorylated serine or threonine residue within a protein." [GOC:go_curators]
+def: "OBSOLETE. Interacting selectively with a phosphorylated serine or threonine residue within a protein." [GOC:curators]
 comment: This term was made obsolete because it represents two functions.
 synonym: "phosphoserine/phosphothreonine binding" EXACT []
 is_obsolete: true
@@ -280646,7 +281275,7 @@
 id: GO:0045312
 name: nor-spermidine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of nor-spermidine, a compound related to spermidine, N-(3-aminopropyl)-1,4-diaminobutane." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of nor-spermidine, a compound related to spermidine, N-(3-aminopropyl)-1,4-diaminobutane." [GOC:curators]
 synonym: "nor-spermidine anabolism" EXACT []
 synonym: "nor-spermidine biosynthesis" EXACT []
 synonym: "nor-spermidine formation" EXACT []
@@ -280884,7 +281513,7 @@
 id: GO:0045334
 name: clathrin-coated endocytic vesicle
 namespace: cellular_component
-def: "A clathrin-coated, membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance." [GOC:go_curators]
+def: "A clathrin-coated, membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance." [GOC:curators]
 xref: NIF_Subcellular:sao1243595998
 is_a: GO:0030136 ! clathrin-coated vesicle
 is_a: GO:0030139 ! endocytic vesicle
@@ -280893,7 +281522,7 @@
 id: GO:0045335
 name: phagocytic vesicle
 namespace: cellular_component
-def: "A membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:go_curators, ISBN:0198506732]
+def: "A membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:curators, ISBN:0198506732]
 synonym: "phagosome" EXACT []
 xref: Wikipedia:Phagosome
 is_a: GO:0030139 ! endocytic vesicle
@@ -280902,7 +281531,7 @@
 id: GO:0045336
 name: clathrin-coated phagocytic vesicle
 namespace: cellular_component
-def: "A clathrin-coated, membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:go_curators, ISBN:0198506732]
+def: "A clathrin-coated, membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:curators, ISBN:0198506732]
 synonym: "clathrin-coated phagosome" EXACT []
 is_a: GO:0045334 ! clathrin-coated endocytic vesicle
 is_a: GO:0045335 ! phagocytic vesicle
@@ -280925,7 +281554,7 @@
 id: GO:0045338
 name: farnesyl diphosphate metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving farnesyl diphosphate, an intermediate in carotenoid, sesquiterpene, squalene and sterol biosynthesis, as well as a substrate in protein farnesylation." [GOC:go_curators]
+def: "The chemical reactions and pathways involving farnesyl diphosphate, an intermediate in carotenoid, sesquiterpene, squalene and sterol biosynthesis, as well as a substrate in protein farnesylation." [GOC:curators]
 synonym: "farnesyl diphosphate metabolism" EXACT []
 is_a: GO:0006644 ! phospholipid metabolic process
 is_a: GO:0006721 ! terpenoid metabolic process
@@ -280934,7 +281563,7 @@
 id: GO:0045339
 name: farnesyl diphosphate catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of farnesyl diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of farnesyl diphosphate." [GOC:curators]
 synonym: "farnesyl diphosphate breakdown" EXACT []
 synonym: "farnesyl diphosphate catabolism" EXACT []
 synonym: "farnesyl diphosphate degradation" EXACT []
@@ -280946,7 +281575,7 @@
 id: GO:0045340
 name: mercury ion binding
 namespace: molecular_function
-def: "Binding to a mercury ion (Hg2+)." [GOC:go_curators]
+def: "Binding to a mercury ion (Hg2+)." [GOC:curators]
 synonym: "Hg ion binding" EXACT []
 synonym: "mercury binding" EXACT []
 is_a: GO:0046914 ! transition metal ion binding
@@ -280955,7 +281584,7 @@
 id: GO:0045341
 name: MHC class I biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of major histocompatibility protein class I." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of major histocompatibility protein class I." [GOC:curators]
 synonym: "major histocompatibility complex class I biosynthesis" EXACT []
 synonym: "major histocompatibility complex class I biosynthetic process" EXACT []
 synonym: "MHC class I anabolism" EXACT []
@@ -280968,7 +281597,7 @@
 id: GO:0045342
 name: MHC class II biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of major histocompatibility protein class II." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of major histocompatibility protein class II." [GOC:curators]
 synonym: "major histocompatibility complex class II biosynthesis" EXACT []
 synonym: "major histocompatibility complex class II biosynthetic process" EXACT []
 synonym: "MHC class II anabolism" EXACT []
@@ -280981,7 +281610,7 @@
 id: GO:0045343
 name: regulation of MHC class I biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:curators]
 synonym: "regulation of major histocompatibility complex class I biosynthesis" EXACT []
 synonym: "regulation of major histocompatibility complex class I biosynthetic process" EXACT []
 synonym: "regulation of MHC class I anabolism" EXACT []
@@ -280997,7 +281626,7 @@
 id: GO:0045344
 name: negative regulation of MHC class I biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:curators]
 synonym: "down regulation of MHC class I biosynthetic process" EXACT []
 synonym: "down-regulation of MHC class I biosynthetic process" EXACT []
 synonym: "downregulation of MHC class I biosynthetic process" EXACT []
@@ -281018,7 +281647,7 @@
 id: GO:0045345
 name: positive regulation of MHC class I biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class I." [GOC:curators]
 synonym: "activation of MHC class I biosynthetic process" NARROW []
 synonym: "positive regulation of major histocompatibility complex class I biosynthesis" EXACT []
 synonym: "positive regulation of major histocompatibility complex class I biosynthetic process" EXACT []
@@ -281040,7 +281669,7 @@
 id: GO:0045346
 name: regulation of MHC class II biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:curators]
 synonym: "regulation of major histocompatibility complex class II biosynthesis" EXACT []
 synonym: "regulation of major histocompatibility complex class II biosynthetic process" EXACT []
 synonym: "regulation of MHC class II anabolism" EXACT []
@@ -281056,7 +281685,7 @@
 id: GO:0045347
 name: negative regulation of MHC class II biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:curators]
 synonym: "down regulation of MHC class II biosynthetic process" EXACT []
 synonym: "down-regulation of MHC class II biosynthetic process" EXACT []
 synonym: "downregulation of MHC class II biosynthetic process" EXACT []
@@ -281077,7 +281706,7 @@
 id: GO:0045348
 name: positive regulation of MHC class II biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of MHC class II." [GOC:curators]
 synonym: "activation of MHC class II biosynthetic process" NARROW []
 synonym: "positive regulation of major histocompatibility complex class II biosynthesis" EXACT []
 synonym: "positive regulation of major histocompatibility complex class II biosynthetic process" EXACT []
@@ -281124,7 +281753,7 @@
 id: GO:0045428
 name: regulation of nitric oxide biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:curators]
 synonym: "regulation of nitric oxide anabolism" EXACT []
 synonym: "regulation of nitric oxide biosynthesis" EXACT []
 synonym: "regulation of nitric oxide formation" EXACT []
@@ -281139,7 +281768,7 @@
 id: GO:0045429
 name: positive regulation of nitric oxide biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide." [GOC:curators]
 synonym: "activation of nitric oxide biosynthetic process" NARROW []
 synonym: "positive regulation of nitric oxide anabolism" EXACT []
 synonym: "positive regulation of nitric oxide biosynthesis" EXACT []
@@ -281299,7 +281928,7 @@
 id: GO:0045443
 name: juvenile hormone secretion
 namespace: biological_process
-def: "The regulated release of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:go_curators, ISBN:0198547684]
+def: "The regulated release of juvenile hormones, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:curators, ISBN:0198547684]
 is_a: GO:0046865 ! terpenoid transport
 is_a: GO:0060986 ! endocrine hormone secretion
 is_a: GO:0140353 ! lipid export from cell
@@ -281308,7 +281937,7 @@
 id: GO:0045444
 name: fat cell differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires specialized features of an adipocyte, an animal connective tissue cell specialized for the synthesis and storage of fat." [CL:0000136, GOC:go_curators]
+def: "The process in which a relatively unspecialized cell acquires specialized features of an adipocyte, an animal connective tissue cell specialized for the synthesis and storage of fat." [CL:0000136, GOC:curators]
 synonym: "adipocyte cell differentiation" EXACT []
 synonym: "adipocyte differentiation" EXACT []
 synonym: "adipogenesis" RELATED []
@@ -281319,7 +281948,7 @@
 id: GO:0045445
 name: myoblast differentiation
 namespace: biological_process
-def: "The process in which a relatively unspecialized cell acquires specialized features of a myoblast. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into striated muscle fibers." [CL:0000056, GOC:go_curators, GOC:mtg_muscle]
+def: "The process in which a relatively unspecialized cell acquires specialized features of a myoblast. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into striated muscle fibers." [CL:0000056, GOC:curators, GOC:mtg_muscle]
 synonym: "myoblast cell differentiation" EXACT []
 is_a: GO:0030154 ! cell differentiation
 relationship: part_of GO:0061061 ! muscle structure development
@@ -281328,7 +281957,7 @@
 id: GO:0045446
 name: endothelial cell differentiation
 namespace: biological_process
-def: "The process in which a mesodermal, bone marrow or neural crest cell acquires specialized features of an endothelial cell, a thin flattened cell. A layer of such cells lines the inside surfaces of body cavities, blood vessels, and lymph vessels, making up the endothelium." [CL:0000115, GOC:go_curators]
+def: "The process in which a mesodermal, bone marrow or neural crest cell acquires specialized features of an endothelial cell, a thin flattened cell. A layer of such cells lines the inside surfaces of body cavities, blood vessels, and lymph vessels, making up the endothelium." [CL:0000115, GOC:curators]
 is_a: GO:0030855 ! epithelial cell differentiation
 relationship: part_of GO:0003158 ! endothelium development
 
@@ -281336,7 +281965,7 @@
 id: GO:0045448
 name: mitotic cell cycle, embryonic
 namespace: biological_process
-def: "The eukaryotic cell cycle in which a cell is duplicated without changing ploidy, occurring in the embryo." [GOC:go_curators]
+def: "The eukaryotic cell cycle in which a cell is duplicated without changing ploidy, occurring in the embryo." [GOC:curators]
 is_a: GO:0000278 ! mitotic cell cycle
 relationship: part_of GO:0009790 ! embryo development
 
@@ -281344,7 +281973,7 @@
 id: GO:0045450
 name: bicoid mRNA localization
 namespace: biological_process
-def: "Any process in which bicoid mRNA is transported to and maintained within the oocyte as part of the specification of the anterior/posterior axis." [GOC:go_curators]
+def: "Any process in which bicoid mRNA is transported to and maintained within the oocyte as part of the specification of the anterior/posterior axis." [GOC:curators]
 synonym: "bicoid mRNA localisation" EXACT [GOC:mah]
 synonym: "establishment and maintenance of bicoid mRNA localization" EXACT []
 is_a: GO:0060811 ! intracellular mRNA localization involved in anterior/posterior axis specification
@@ -281354,7 +281983,7 @@
 id: GO:0045451
 name: pole plasm oskar mRNA localization
 namespace: biological_process
-def: "Any process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:go_curators]
+def: "Any process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:curators]
 synonym: "establishment and maintenance of oskar mRNA localization in pole plasm" EXACT []
 synonym: "establishment and maintenance of pole plasm oskar mRNA localization" EXACT []
 synonym: "oocyte pole plasm oskar mRNA localization" EXACT []
@@ -281404,7 +282033,7 @@
 id: GO:0045456
 name: ecdysteroid biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of ecdysteroids, a group of polyhydroxylated ketosteroids which initiate post-embryonic development." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of ecdysteroids, a group of polyhydroxylated ketosteroids which initiate post-embryonic development." [GOC:curators]
 synonym: "ecdysteroid anabolism" EXACT []
 synonym: "ecdysteroid biosynthesis" EXACT []
 synonym: "ecdysteroid formation" EXACT []
@@ -281418,14 +282047,14 @@
 id: GO:0045457
 name: ecdysteroid secretion
 namespace: biological_process
-def: "The regulated release of ecdysteroids, a group of polyhydroxylated ketosteroids which initiate post-embryonic development." [GOC:go_curators]
+def: "The regulated release of ecdysteroids, a group of polyhydroxylated ketosteroids which initiate post-embryonic development." [GOC:curators]
 is_a: GO:0035929 ! steroid hormone secretion
 
 [Term]
 id: GO:0045458
 name: recombination within rDNA repeats
 namespace: biological_process
-def: "Genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:go_curators, ISBN:0198506732]
+def: "Genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:curators, ISBN:0198506732]
 comment: Note that this term was reinstated from obsolete.
 synonym: "recombination within ribosomal DNA repeats" EXACT []
 is_a: GO:0006310 ! DNA recombination
@@ -281542,21 +282171,21 @@
 name: response to ethanol
 namespace: biological_process
 alt_id: GO:0017036
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethanol stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethanol stimulus." [GOC:curators]
 is_a: GO:0097305 ! response to alcohol
 
 [Term]
 id: GO:0045472
 name: response to ether
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a ether stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a ether stimulus." [GOC:curators]
 is_a: GO:1901700 ! response to oxygen-containing compound
 
 [Term]
 id: GO:0045473
 name: obsolete response to ethanol (sensu Insecta)
 namespace: biological_process
-def: "OBSOLETE. A change in state or activity of an insect (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethanol stimulus." [GOC:go_curators, GOC:jid]
+def: "OBSOLETE. A change in state or activity of an insect (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethanol stimulus." [GOC:curators, GOC:jid]
 comment: This term was made obsolete because there is no clear difference between the sensu Insecta term and the generic term.
 synonym: "response to ethanol (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -281566,7 +282195,7 @@
 id: GO:0045474
 name: obsolete response to ether (sensu Insecta)
 namespace: biological_process
-def: "OBSOLETE. A change in state or activity of an insect (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ether stimulus." [GOC:go_curators, GOC:jid]
+def: "OBSOLETE. A change in state or activity of an insect (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ether stimulus." [GOC:curators, GOC:jid]
 comment: This term was made obsolete because there is no clear difference between the sensu Insecta term and the generic term.
 synonym: "response to ether (sensu Insecta)" EXACT []
 is_obsolete: true
@@ -281576,7 +282205,7 @@
 id: GO:0045475
 name: locomotor rhythm
 namespace: biological_process
-def: "The rhythm of the locomotor activity of an organism during its 24 hour activity cycle." [GOC:go_curators]
+def: "The rhythm of the locomotor activity of an organism during its 24 hour activity cycle." [GOC:curators]
 synonym: "circadian locomotor activity rhythm" NARROW []
 is_a: GO:0007626 ! locomotory behavior
 is_a: GO:0048512 ! circadian behavior
@@ -281612,7 +282241,7 @@
 id: GO:0045478
 name: fusome organization
 namespace: biological_process
-def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the fusome, a large intracellular spectrin-rich structure found in insect germline cells and mammalian hematopoietic cells." [GOC:dph, GOC:go_curators, GOC:jl, GOC:mah]
+def: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the fusome, a large intracellular spectrin-rich structure found in insect germline cells and mammalian hematopoietic cells." [GOC:curators, GOC:dph, GOC:jl, GOC:mah]
 subset: goslim_pir
 synonym: "fusome organisation" EXACT []
 synonym: "fusome organization and biogenesis" RELATED [GOC:mah]
@@ -281632,7 +282261,7 @@
 id: GO:0045480
 name: galactose oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-galactose + O2 = D-galacto-hexodialdose + hydrogen peroxide." [EC:1.1.3.9]
+def: "Catalysis of the reaction: D-galactose + O2 = D-galacto-hexodialdose + H2O2." [RHEA:24160]
 synonym: "beta-galactose oxidase activity" RELATED [EC:1.1.3.9]
 synonym: "D-galactose oxidase activity" RELATED [EC:1.1.3.9]
 synonym: "D-galactose:oxygen 6-oxidoreductase activity" RELATED [EC:1.1.3.9]
@@ -281696,9 +282325,10 @@
 
 [Term]
 id: GO:0045484
-name: L-lysine 6-transaminase activity
+name: L-lysine:2-oxoglutarate 6-transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-lysine = L-glutamate + allysine." [EC:2.6.1.36, RHEA:21200]
+def: "Catalysis of the reaction: 2-oxoglutarate + L-lysine = L-glutamate + allysine." [RHEA:21200]
+synonym: "L-lysine 6-transaminase activity" EXACT []
 synonym: "L-lysine aminotransferase activity" RELATED [EC:2.6.1.36]
 synonym: "L-lysine transaminase activity" RELATED [EC:2.6.1.36]
 synonym: "L-lysine-alpha-ketoglutarate 6-aminotransferase activity" RELATED [EC:2.6.1.36]
@@ -281712,10 +282342,11 @@
 xref: KEGG_REACTION:R00457
 xref: MetaCyc:L-LYSINE-AMINOTRANSFERASE-RXN
 xref: RHEA:21200
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.36
 property_value: skos:exactMatch RHEA:21200
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0045485
@@ -281753,7 +282384,7 @@
 id: GO:0045487
 name: gibberellin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of gibberellin. Gibberellins are a class of highly modified terpenes that function as plant growth regulators." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of gibberellin. Gibberellins are a class of highly modified terpenes that function as plant growth regulators." [GOC:curators]
 synonym: "gibberellic acid breakdown" NARROW []
 synonym: "gibberellic acid catabolic process" NARROW []
 synonym: "gibberellic acid catabolism" NARROW []
@@ -281775,7 +282406,7 @@
 id: GO:0045489
 name: pectin biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pectin, a polymer containing a backbone of alpha-1,4-linked D-galacturonic acid residues." [GOC:go_curators, PMID:11931668]
+def: "The chemical reactions and pathways resulting in the formation of pectin, a polymer containing a backbone of alpha-1,4-linked D-galacturonic acid residues." [GOC:curators, PMID:11931668]
 synonym: "pectin anabolism" EXACT []
 synonym: "pectin biosynthesis" EXACT []
 synonym: "pectin formation" EXACT []
@@ -281787,7 +282418,7 @@
 id: GO:0045490
 name: pectin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of pectin, a polymer containing a backbone of alpha-1,4-linked D-galacturonic acid residues." [GOC:go_curators, PMID:11931668]
+def: "The chemical reactions and pathways resulting in the breakdown of pectin, a polymer containing a backbone of alpha-1,4-linked D-galacturonic acid residues." [GOC:curators, PMID:11931668]
 synonym: "pectin breakdown" EXACT []
 synonym: "pectin catabolism" EXACT []
 synonym: "pectin degradation" EXACT []
@@ -281798,7 +282429,7 @@
 id: GO:0045491
 name: xylan metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:go_curators, PMID:11931668]
+def: "The chemical reactions and pathways involving xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:curators, PMID:11931668]
 synonym: "xylan metabolism" EXACT []
 is_a: GO:0005976 ! polysaccharide metabolic process
 
@@ -281806,7 +282437,7 @@
 id: GO:0045492
 name: xylan biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:go_curators, PMID:11931668]
+def: "The chemical reactions and pathways resulting in the formation of xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:curators, PMID:11931668]
 synonym: "xylan anabolism" EXACT []
 synonym: "xylan biosynthesis" EXACT []
 synonym: "xylan formation" EXACT []
@@ -281818,7 +282449,7 @@
 id: GO:0045493
 name: xylan catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:go_curators, PMID:11931668]
+def: "The chemical reactions and pathways resulting in the breakdown of xylan, a polymer containing a beta-1,4-linked D-xylose backbone." [GOC:curators, PMID:11931668]
 synonym: "xylan breakdown" EXACT []
 synonym: "xylan catabolism" EXACT []
 synonym: "xylan degradation" EXACT []
@@ -281888,7 +282519,7 @@
 id: GO:0045501
 name: regulation of sevenless signaling pathway
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the sevenless signaling pathway." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the sevenless signaling pathway." [GOC:curators]
 synonym: "regulation of sev signaling pathway" EXACT []
 synonym: "regulation of sevenless signalling pathway" EXACT []
 is_a: GO:0009966 ! regulation of signal transduction
@@ -281963,7 +282594,7 @@
 id: GO:0045510
 name: interleukin-24 binding
 namespace: molecular_function
-def: "Binding to interleukin-24." [GOC:go_curators]
+def: "Binding to interleukin-24." [GOC:curators]
 synonym: "IL-24 binding" EXACT []
 is_a: GO:0019955 ! cytokine binding
 
@@ -281971,7 +282602,7 @@
 id: GO:0045511
 name: interleukin-25 binding
 namespace: molecular_function
-def: "Binding to interleukin-25." [GOC:go_curators]
+def: "Binding to interleukin-25." [GOC:curators]
 synonym: "IL-25 binding" EXACT []
 is_a: GO:0019975 ! interleukin-17 binding
 
@@ -281979,7 +282610,7 @@
 id: GO:0045512
 name: interleukin-26 binding
 namespace: molecular_function
-def: "Binding to interleukin-26." [GOC:go_curators]
+def: "Binding to interleukin-26." [GOC:curators]
 synonym: "IL-26 binding" EXACT []
 is_a: GO:0019955 ! cytokine binding
 
@@ -281987,7 +282618,7 @@
 id: GO:0045513
 name: interleukin-27 binding
 namespace: molecular_function
-def: "Binding to interleukin-27." [GOC:go_curators]
+def: "Binding to interleukin-27." [GOC:curators]
 synonym: "IL-27 binding" EXACT []
 is_a: GO:0019975 ! interleukin-17 binding
 
@@ -281995,7 +282626,7 @@
 id: GO:0045514
 name: interleukin-16 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-16 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-16 receptor." [GOC:curators]
 synonym: "IL-16" NARROW []
 synonym: "interleukin-16 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282004,7 +282635,7 @@
 id: GO:0045515
 name: interleukin-18 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-18 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-18 receptor." [GOC:curators]
 synonym: "IL-18" NARROW []
 synonym: "interleukin-18 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282013,7 +282644,7 @@
 id: GO:0045516
 name: interleukin-19 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-19 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-19 receptor." [GOC:curators]
 synonym: "IL-19" NARROW []
 synonym: "interleukin-19 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282022,7 +282653,7 @@
 id: GO:0045517
 name: interleukin-20 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-20 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-20 receptor." [GOC:curators]
 synonym: "IL-20" NARROW []
 synonym: "interleukin-20 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282031,7 +282662,7 @@
 id: GO:0045518
 name: interleukin-22 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-22 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-22 receptor." [GOC:curators]
 synonym: "IL-22" NARROW []
 synonym: "interleukin-22 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282040,7 +282671,7 @@
 id: GO:0045519
 name: interleukin-23 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-23 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-23 receptor." [GOC:curators]
 synonym: "IL-23" NARROW []
 synonym: "interleukin-23 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282049,7 +282680,7 @@
 id: GO:0045520
 name: interleukin-24 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-24 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-24 receptor." [GOC:curators]
 synonym: "IL-24" NARROW []
 synonym: "interleukin-24 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282058,7 +282689,7 @@
 id: GO:0045521
 name: interleukin-25 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-25 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-25 receptor." [GOC:curators]
 synonym: "IL-25" NARROW []
 synonym: "interleukin-25 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282067,7 +282698,7 @@
 id: GO:0045522
 name: interleukin-26 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-26 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-26 receptor." [GOC:curators]
 synonym: "IL-26" NARROW []
 synonym: "interleukin-26 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282076,7 +282707,7 @@
 id: GO:0045523
 name: interleukin-27 receptor binding
 namespace: molecular_function
-def: "Binding to an interleukin-27 receptor." [GOC:go_curators]
+def: "Binding to an interleukin-27 receptor." [GOC:curators]
 synonym: "IL-27" NARROW []
 synonym: "interleukin-27 receptor ligand" NARROW []
 is_a: GO:0005126 ! cytokine receptor binding
@@ -282085,7 +282716,7 @@
 id: GO:0045540
 name: regulation of cholesterol biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:curators]
 synonym: "regulation of cholesterol anabolism" EXACT []
 synonym: "regulation of cholesterol biosynthesis" EXACT []
 synonym: "regulation of cholesterol formation" EXACT []
@@ -282102,7 +282733,7 @@
 id: GO:0045541
 name: negative regulation of cholesterol biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:curators]
 synonym: "down regulation of cholesterol biosynthetic process" EXACT []
 synonym: "down-regulation of cholesterol biosynthetic process" EXACT []
 synonym: "downregulation of cholesterol biosynthetic process" EXACT []
@@ -282123,7 +282754,7 @@
 id: GO:0045542
 name: positive regulation of cholesterol biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of cholesterol." [GOC:curators]
 synonym: "activation of cholesterol biosynthetic process" NARROW []
 synonym: "positive regulation of cholesterol anabolism" EXACT []
 synonym: "positive regulation of cholesterol biosynthesis" EXACT []
@@ -282181,7 +282812,7 @@
 id: GO:0045545
 name: syndecan binding
 namespace: molecular_function
-def: "Binding to syndecan, an integral membrane proteoglycan (250-300 kDa) associated largely with epithelial cells." [GOC:go_curators, PMID:9355727]
+def: "Binding to syndecan, an integral membrane proteoglycan (250-300 kDa) associated largely with epithelial cells." [GOC:curators, PMID:9355727]
 is_a: GO:0043394 ! proteoglycan binding
 
 [Term]
@@ -282292,7 +282923,7 @@
 id: GO:0045557
 name: obsolete TRAIL receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "TRAIL receptor anabolism" EXACT []
 synonym: "TRAIL receptor biosynthesis" EXACT []
@@ -282305,7 +282936,7 @@
 id: GO:0045558
 name: obsolete TRAIL receptor 1 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of TRAIL-R1 (TNF-related apoptosis inducing ligand receptor 1), which engages a caspase-dependent apoptotic pathway." [GOC:go_curators, PMID:9311998]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of TRAIL-R1 (TNF-related apoptosis inducing ligand receptor 1), which engages a caspase-dependent apoptotic pathway." [GOC:curators, PMID:9311998]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "TRAIL receptor 1 anabolism" EXACT []
 synonym: "TRAIL receptor 1 biosynthesis" EXACT []
@@ -282319,7 +282950,7 @@
 id: GO:0045559
 name: obsolete TRAIL receptor 2 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of TRAIL-R2 (TNF-related apoptosis inducing ligand receptor 2), which engages a caspase-dependent apoptotic pathway and mediates apoptosis via the intracellular adaptor molecule FADD/MORT1." [GOC:go_curators, PMID:9311998]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of TRAIL-R2 (TNF-related apoptosis inducing ligand receptor 2), which engages a caspase-dependent apoptotic pathway and mediates apoptosis via the intracellular adaptor molecule FADD/MORT1." [GOC:curators, PMID:9311998]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "TRAIL receptor 2 anabolism" EXACT []
 synonym: "TRAIL receptor 2 biosynthesis" EXACT []
@@ -282333,7 +282964,7 @@
 id: GO:0045560
 name: obsolete regulation of TRAIL receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "regulation of TRAIL receptor anabolism" EXACT []
 synonym: "regulation of TRAIL receptor biosynthesis" EXACT []
@@ -282346,7 +282977,7 @@
 id: GO:0045561
 name: obsolete regulation of TRAIL receptor 1 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "regulation of TRAIL receptor 1 anabolism" EXACT []
 synonym: "regulation of TRAIL receptor 1 biosynthesis" EXACT []
@@ -282359,7 +282990,7 @@
 id: GO:0045562
 name: obsolete regulation of TRAIL receptor 2 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "regulation of TRAIL receptor 2 anabolism" EXACT []
 synonym: "regulation of TRAIL receptor 2 biosynthesis" EXACT []
@@ -282372,7 +283003,7 @@
 id: GO:0045563
 name: obsolete negative regulation of TRAIL receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "down regulation of TRAIL receptor biosynthetic process" EXACT []
 synonym: "down-regulation of TRAIL receptor biosynthetic process" EXACT []
@@ -282389,7 +283020,7 @@
 id: GO:0045564
 name: obsolete positive regulation of TRAIL receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of the TRAIL (TNF-related apoptosis inducing ligand) receptor." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "activation of TRAIL receptor biosynthetic process" NARROW []
 synonym: "positive regulation of TRAIL receptor anabolism" EXACT []
@@ -282407,7 +283038,7 @@
 id: GO:0045565
 name: obsolete negative regulation of TRAIL receptor 1 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "down regulation of TRAIL receptor 1 biosynthetic process" EXACT []
 synonym: "down-regulation of TRAIL receptor 1 biosynthetic process" EXACT []
@@ -282424,7 +283055,7 @@
 id: GO:0045566
 name: obsolete positive regulation of TRAIL receptor 1 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 1." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "activation of TRAIL receptor 1 biosynthetic process" NARROW []
 synonym: "positive regulation of TRAIL receptor 1 anabolism" EXACT []
@@ -282442,7 +283073,7 @@
 id: GO:0045567
 name: obsolete negative regulation of TRAIL receptor 2 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "down regulation of TRAIL receptor 2 biosynthetic process" EXACT []
 synonym: "down-regulation of TRAIL receptor 2 biosynthetic process" EXACT []
@@ -282459,7 +283090,7 @@
 id: GO:0045568
 name: obsolete positive regulation of TRAIL receptor 2 biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of TRAIL (TNF-related apoptosis inducing ligand) receptor 2." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "activation of TRAIL receptor 2 biosynthetic process" NARROW []
 synonym: "positive regulation of TRAIL receptor 2 anabolism" EXACT []
@@ -282477,7 +283108,7 @@
 id: GO:0045569
 name: TRAIL binding
 namespace: molecular_function
-def: "Binding to TRAIL (TNF-related apoptosis inducing ligand), a member of the tumor necrosis factor ligand family that rapidly induces apoptosis in a variety of transformed cell lines." [GOC:go_curators, PMID:9082980]
+def: "Binding to TRAIL (TNF-related apoptosis inducing ligand), a member of the tumor necrosis factor ligand family that rapidly induces apoptosis in a variety of transformed cell lines." [GOC:curators, PMID:9082980]
 subset: goslim_chembl
 synonym: "Apo-2L binding" EXACT []
 is_a: GO:0005515 ! protein binding
@@ -282486,7 +283117,7 @@
 id: GO:0045570
 name: regulation of imaginal disc growth
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the growth of the imaginal disc." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the growth of the imaginal disc." [GOC:curators]
 is_a: GO:0048638 ! regulation of developmental growth
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0007446 ! imaginal disc growth
@@ -282496,7 +283127,7 @@
 id: GO:0045571
 name: negative regulation of imaginal disc growth
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of imaginal disc growth." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of imaginal disc growth." [GOC:curators]
 synonym: "down regulation of imaginal disc growth" EXACT []
 synonym: "down-regulation of imaginal disc growth" EXACT []
 synonym: "downregulation of imaginal disc growth" EXACT []
@@ -282511,7 +283142,7 @@
 id: GO:0045572
 name: positive regulation of imaginal disc growth
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of imaginal disc growth." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of imaginal disc growth." [GOC:curators]
 synonym: "activation of imaginal disc growth" NARROW []
 synonym: "stimulation of imaginal disc growth" NARROW []
 synonym: "up regulation of imaginal disc growth" EXACT []
@@ -282527,7 +283158,7 @@
 id: GO:0045574
 name: sterigmatocystin catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sterigmatocystin, a carcinogenic mycotoxin produced in high yields by strains of the common molds." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of sterigmatocystin, a carcinogenic mycotoxin produced in high yields by strains of the common molds." [GOC:curators]
 synonym: "sterigmatocystin breakdown" EXACT []
 synonym: "sterigmatocystin catabolism" EXACT []
 synonym: "sterigmatocystin degradation" EXACT []
@@ -282553,7 +283184,7 @@
 id: GO:0045577
 name: regulation of B cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of B cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of B cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of B cell development" RELATED [GOC:add]
 synonym: "regulation of B lymphocyte differentiation" EXACT []
@@ -282569,7 +283200,7 @@
 id: GO:0045578
 name: negative regulation of B cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of B cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of B cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of B cell differentiation" EXACT []
 synonym: "down-regulation of B cell differentiation" EXACT []
@@ -282590,7 +283221,7 @@
 id: GO:0045579
 name: positive regulation of B cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of B cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of B cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of B cell differentiation" NARROW []
 synonym: "positive regulation of B cell development" RELATED [GOC:add]
@@ -282612,7 +283243,7 @@
 id: GO:0045580
 name: regulation of T cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of T cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of T cell development" RELATED [GOC:add]
 synonym: "regulation of T lymphocyte differentiation" EXACT []
@@ -282628,7 +283259,7 @@
 id: GO:0045581
 name: negative regulation of T cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of T cell differentiation" EXACT []
 synonym: "down-regulation of T cell differentiation" EXACT []
@@ -282649,7 +283280,7 @@
 id: GO:0045582
 name: positive regulation of T cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of T cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of T cell differentiation" NARROW []
 synonym: "positive regulation of T cell development" RELATED [GOC:add]
@@ -282671,7 +283302,7 @@
 id: GO:0045583
 name: regulation of cytotoxic T cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of cytotoxic T cell development" RELATED [GOC:add]
 synonym: "regulation of cytotoxic T lymphocyte differentiation" EXACT []
@@ -282686,7 +283317,7 @@
 id: GO:0045584
 name: negative regulation of cytotoxic T cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of cytotoxic T cell differentiation" EXACT []
 synonym: "down-regulation of cytotoxic T cell differentiation" EXACT []
@@ -282706,7 +283337,7 @@
 id: GO:0045585
 name: positive regulation of cytotoxic T cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cytotoxic T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of cytotoxic T cell differentiation" NARROW []
 synonym: "positive regulation of cytotoxic T cell development" RELATED [GOC:add]
@@ -282727,7 +283358,7 @@
 id: GO:0045586
 name: regulation of gamma-delta T cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of gamma-delta T cell development" RELATED [GOC:add]
 synonym: "regulation of gamma-delta T lymphocyte differentiation" EXACT []
@@ -282743,7 +283374,7 @@
 id: GO:0045587
 name: negative regulation of gamma-delta T cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of gamma-delta T cell differentiation" EXACT []
 synonym: "down-regulation of gamma-delta T cell differentiation" EXACT []
@@ -282764,7 +283395,7 @@
 id: GO:0045588
 name: positive regulation of gamma-delta T cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of gamma-delta T cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of gamma-delta T cell differentiation" NARROW []
 synonym: "positive regulation of gamma-delta T cell development" RELATED [GOC:add]
@@ -282851,7 +283482,7 @@
 id: GO:0045592
 name: regulation of cumulus cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:curators]
 synonym: "regulation of ovarian cumulus cell differentiation" EXACT []
 is_a: GO:0030856 ! regulation of epithelial cell differentiation
 is_a: GO:2000387 ! regulation of antral ovarian follicle growth
@@ -282863,7 +283494,7 @@
 id: GO:0045593
 name: negative regulation of cumulus cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:curators]
 synonym: "down regulation of cumulus cell differentiation" EXACT []
 synonym: "down-regulation of cumulus cell differentiation" EXACT []
 synonym: "downregulation of cumulus cell differentiation" EXACT []
@@ -282880,7 +283511,7 @@
 id: GO:0045594
 name: positive regulation of cumulus cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of ovarian cumulus cell differentiation." [GOC:curators]
 synonym: "activation of cumulus cell differentiation" NARROW []
 synonym: "positive regulation of ovarian cumulus cell differentiation" EXACT []
 synonym: "stimulation of cumulus cell differentiation" NARROW []
@@ -282898,7 +283529,7 @@
 id: GO:0045595
 name: regulation of cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cell differentiation, the process in which relatively unspecialized cells acquire specialized structural and functional features." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of cell differentiation, the process in which relatively unspecialized cells acquire specialized structural and functional features." [GOC:curators]
 is_a: GO:0050793 ! regulation of developmental process
 is_a: GO:0050794 ! regulation of cellular process
 intersection_of: GO:0065007 ! biological regulation
@@ -282909,7 +283540,7 @@
 id: GO:0045596
 name: negative regulation of cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell differentiation." [GOC:curators]
 synonym: "down regulation of cell differentiation" EXACT []
 synonym: "down-regulation of cell differentiation" EXACT []
 synonym: "downregulation of cell differentiation" EXACT []
@@ -282925,7 +283556,7 @@
 id: GO:0045597
 name: positive regulation of cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cell differentiation." [GOC:curators]
 synonym: "activation of cell differentiation" NARROW []
 synonym: "stimulation of cell differentiation" NARROW []
 synonym: "up regulation of cell differentiation" EXACT []
@@ -282942,7 +283573,7 @@
 id: GO:0045598
 name: regulation of fat cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of adipocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of adipocyte differentiation." [GOC:curators]
 synonym: "regulation of adipocyte cell differentiation" EXACT []
 synonym: "regulation of adipocyte differentiation" EXACT []
 is_a: GO:0045595 ! regulation of cell differentiation
@@ -282954,7 +283585,7 @@
 id: GO:0045599
 name: negative regulation of fat cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of adipocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of adipocyte differentiation." [GOC:curators]
 synonym: "down regulation of fat cell differentiation" EXACT []
 synonym: "down-regulation of fat cell differentiation" EXACT []
 synonym: "downregulation of fat cell differentiation" EXACT []
@@ -282971,7 +283602,7 @@
 id: GO:0045600
 name: positive regulation of fat cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of adipocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of adipocyte differentiation." [GOC:curators]
 synonym: "activation of fat cell differentiation" NARROW []
 synonym: "positive regulation of adipocyte cell differentiation" EXACT []
 synonym: "positive regulation of adipocyte differentiation" EXACT []
@@ -282989,7 +283620,7 @@
 id: GO:0045601
 name: regulation of endothelial cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of endothelial cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of endothelial cell differentiation." [GOC:curators]
 is_a: GO:0030856 ! regulation of epithelial cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0045446 ! endothelial cell differentiation
@@ -282999,7 +283630,7 @@
 id: GO:0045602
 name: negative regulation of endothelial cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of endothelial cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of endothelial cell differentiation." [GOC:curators]
 synonym: "down regulation of endothelial cell differentiation" EXACT []
 synonym: "down-regulation of endothelial cell differentiation" EXACT []
 synonym: "downregulation of endothelial cell differentiation" EXACT []
@@ -283014,7 +283645,7 @@
 id: GO:0045603
 name: positive regulation of endothelial cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of endothelial cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of endothelial cell differentiation." [GOC:curators]
 synonym: "activation of endothelial cell differentiation" NARROW []
 synonym: "stimulation of endothelial cell differentiation" NARROW []
 synonym: "up regulation of endothelial cell differentiation" EXACT []
@@ -283030,7 +283661,7 @@
 id: GO:0045604
 name: regulation of epidermal cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of epidermal cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of epidermal cell differentiation." [GOC:curators]
 synonym: "regulation of hypodermal cell differentiation" RELATED [GOC:kmv, GOC:rk]
 is_a: GO:0030856 ! regulation of epithelial cell differentiation
 is_a: GO:0045682 ! regulation of epidermis development
@@ -283042,7 +283673,7 @@
 id: GO:0045605
 name: negative regulation of epidermal cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermal cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermal cell differentiation." [GOC:curators]
 synonym: "down regulation of epidermal cell differentiation" EXACT []
 synonym: "down-regulation of epidermal cell differentiation" EXACT []
 synonym: "downregulation of epidermal cell differentiation" EXACT []
@@ -283059,7 +283690,7 @@
 id: GO:0045606
 name: positive regulation of epidermal cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of epidermal cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of epidermal cell differentiation." [GOC:curators]
 synonym: "activation of epidermal cell differentiation" NARROW []
 synonym: "positive regulation of hypodermal cell differentiation" RELATED [GOC:kmv, GOC:rk]
 synonym: "stimulation of epidermal cell differentiation" NARROW []
@@ -283077,7 +283708,7 @@
 id: GO:0045607
 name: regulation of inner ear auditory receptor cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of auditory hair cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of auditory hair cell differentiation." [GOC:curators]
 synonym: "regulation of auditory hair cell differentiation" EXACT []
 synonym: "regulation of auditory receptor cell differentiation" RELATED []
 is_a: GO:0045604 ! regulation of epidermal cell differentiation
@@ -283090,7 +283721,7 @@
 id: GO:0045608
 name: negative regulation of inner ear auditory receptor cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of auditory hair cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of auditory hair cell differentiation." [GOC:curators]
 synonym: "down regulation of auditory receptor cell differentiation" EXACT []
 synonym: "down-regulation of auditory receptor cell differentiation" EXACT []
 synonym: "downregulation of auditory receptor cell differentiation" EXACT []
@@ -283108,7 +283739,7 @@
 id: GO:0045609
 name: positive regulation of inner ear auditory receptor cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of auditory hair cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of auditory hair cell differentiation." [GOC:curators]
 synonym: "activation of auditory receptor cell differentiation" NARROW []
 synonym: "positive regulation of auditory hair cell differentiation" EXACT []
 synonym: "positive regulation of auditory receptor cell differentiation" RELATED []
@@ -283127,7 +283758,7 @@
 id: GO:0045610
 name: regulation of hemocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of hemocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of hemocyte differentiation." [GOC:curators]
 synonym: "regulation of arthropod blood cell differentiation" EXACT []
 is_a: GO:0002682 ! regulation of immune system process
 is_a: GO:0045595 ! regulation of cell differentiation
@@ -283140,7 +283771,7 @@
 id: GO:0045611
 name: negative regulation of hemocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of hemocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of hemocyte differentiation." [GOC:curators]
 synonym: "down regulation of hemocyte differentiation" EXACT []
 synonym: "down-regulation of hemocyte differentiation" EXACT []
 synonym: "downregulation of hemocyte differentiation" EXACT []
@@ -283157,7 +283788,7 @@
 id: GO:0045612
 name: positive regulation of hemocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of hemocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of hemocyte differentiation." [GOC:curators]
 synonym: "activation of hemocyte differentiation" NARROW []
 synonym: "positive regulation of arthropod blood cell differentiation" EXACT []
 synonym: "stimulation of hemocyte differentiation" NARROW []
@@ -283175,7 +283806,7 @@
 id: GO:0045613
 name: regulation of plasmatocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of plasmatocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of plasmatocyte differentiation." [GOC:curators]
 is_a: GO:0045610 ! regulation of hemocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0042387 ! plasmatocyte differentiation
@@ -283185,7 +283816,7 @@
 id: GO:0045614
 name: negative regulation of plasmatocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of plasmatocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of plasmatocyte differentiation." [GOC:curators]
 synonym: "down regulation of plasmatocyte differentiation" EXACT []
 synonym: "down-regulation of plasmatocyte differentiation" EXACT []
 synonym: "downregulation of plasmatocyte differentiation" EXACT []
@@ -283200,7 +283831,7 @@
 id: GO:0045615
 name: positive regulation of plasmatocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of plasmatocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of plasmatocyte differentiation." [GOC:curators]
 synonym: "activation of plasmatocyte differentiation" NARROW []
 synonym: "stimulation of plasmatocyte differentiation" NARROW []
 synonym: "up regulation of plasmatocyte differentiation" EXACT []
@@ -283216,7 +283847,7 @@
 id: GO:0045616
 name: regulation of keratinocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of keratinocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of keratinocyte differentiation." [GOC:curators]
 xref: Reactome:R-HSA-8939242 "RUNX1 regulates transcription of genes involved in differentiation of keratinocytes"
 is_a: GO:0045604 ! regulation of epidermal cell differentiation
 intersection_of: GO:0065007 ! biological regulation
@@ -283227,7 +283858,7 @@
 id: GO:0045617
 name: negative regulation of keratinocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of keratinocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of keratinocyte differentiation." [GOC:curators]
 synonym: "down regulation of keratinocyte differentiation" EXACT []
 synonym: "down-regulation of keratinocyte differentiation" EXACT []
 synonym: "downregulation of keratinocyte differentiation" EXACT []
@@ -283243,7 +283874,7 @@
 id: GO:0045618
 name: positive regulation of keratinocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of keratinocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of keratinocyte differentiation." [GOC:curators]
 synonym: "activation of keratinocyte differentiation" NARROW []
 synonym: "stimulation of keratinocyte differentiation" NARROW []
 synonym: "up regulation of keratinocyte differentiation" EXACT []
@@ -283260,7 +283891,7 @@
 id: GO:0045619
 name: regulation of lymphocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of lymphocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of lymphocyte differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of lymphocyte development" RELATED [GOC:add]
 is_a: GO:0051249 ! regulation of lymphocyte activation
@@ -283273,7 +283904,7 @@
 id: GO:0045620
 name: negative regulation of lymphocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of lymphocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of lymphocyte differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of lymphocyte differentiation" EXACT []
 synonym: "down-regulation of lymphocyte differentiation" EXACT []
@@ -283291,7 +283922,7 @@
 id: GO:0045621
 name: positive regulation of lymphocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of lymphocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of lymphocyte differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of lymphocyte differentiation" NARROW []
 synonym: "positive regulation of lymphocyte development" RELATED [GOC:add]
@@ -283310,7 +283941,7 @@
 id: GO:0045622
 name: regulation of T-helper cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of T-helper cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of T-helper cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of T-helper cell development" RELATED [GOC:add]
 is_a: GO:0043370 ! regulation of CD4-positive, alpha-beta T cell differentiation
@@ -283323,7 +283954,7 @@
 id: GO:0045623
 name: negative regulation of T-helper cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of T-helper cell differentiation" EXACT []
 synonym: "down-regulation of T-helper cell differentiation" EXACT []
@@ -283341,7 +283972,7 @@
 id: GO:0045624
 name: positive regulation of T-helper cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of T-helper cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of T-helper cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of T-helper cell differentiation" NARROW []
 synonym: "positive regulation of T-helper cell development" RELATED [GOC:add]
@@ -283360,7 +283991,7 @@
 id: GO:0045625
 name: regulation of T-helper 1 cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of T-helper 1 cell development" RELATED [GOC:add]
 is_a: GO:0002697 ! regulation of immune effector process
@@ -283374,7 +284005,7 @@
 id: GO:0045626
 name: negative regulation of T-helper 1 cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of T-helper 1 cell differentiation" EXACT []
 synonym: "down-regulation of T-helper 1 cell differentiation" EXACT []
@@ -283393,7 +284024,7 @@
 id: GO:0045627
 name: positive regulation of T-helper 1 cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of T-helper 1 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of T-helper 1 cell differentiation" NARROW []
 synonym: "positive regulation of T-helper 1 cell development" RELATED [GOC:add]
@@ -283412,7 +284043,7 @@
 id: GO:0045628
 name: regulation of T-helper 2 cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "regulation of T-helper 2 cell development" RELATED [GOC:add]
 is_a: GO:0002697 ! regulation of immune effector process
@@ -283426,7 +284057,7 @@
 id: GO:0045629
 name: negative regulation of T-helper 2 cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "down regulation of T-helper 2 cell differentiation" EXACT []
 synonym: "down-regulation of T-helper 2 cell differentiation" EXACT []
@@ -283445,7 +284076,7 @@
 id: GO:0045630
 name: positive regulation of T-helper 2 cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of T-helper 2 cell differentiation." [GOC:curators]
 comment: Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.
 synonym: "activation of T-helper 2 cell differentiation" NARROW []
 synonym: "positive regulation of T-helper 2 cell development" RELATED [GOC:add]
@@ -283465,7 +284096,7 @@
 id: GO:0045631
 name: regulation of mechanoreceptor differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of mechanoreceptor differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of mechanoreceptor differentiation." [GOC:curators]
 is_a: GO:0045664 ! regulation of neuron differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0042490 ! mechanoreceptor differentiation
@@ -283475,7 +284106,7 @@
 id: GO:0045632
 name: negative regulation of mechanoreceptor differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of mechanoreceptor differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of mechanoreceptor differentiation." [GOC:curators]
 synonym: "down regulation of mechanoreceptor differentiation" EXACT []
 synonym: "down-regulation of mechanoreceptor differentiation" EXACT []
 synonym: "downregulation of mechanoreceptor differentiation" EXACT []
@@ -283490,7 +284121,7 @@
 id: GO:0045633
 name: positive regulation of mechanoreceptor differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of mechanoreceptor differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of mechanoreceptor differentiation." [GOC:curators]
 synonym: "activation of mechanoreceptor differentiation" NARROW []
 synonym: "stimulation of mechanoreceptor differentiation" NARROW []
 synonym: "up regulation of mechanoreceptor differentiation" EXACT []
@@ -283506,7 +284137,7 @@
 id: GO:0045634
 name: regulation of melanocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of melanocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of melanocyte differentiation." [GOC:curators]
 synonym: "regulation of melanophore differentiation" EXACT []
 is_a: GO:0050932 ! regulation of pigment cell differentiation
 intersection_of: GO:0065007 ! biological regulation
@@ -283517,7 +284148,7 @@
 id: GO:0045635
 name: negative regulation of melanocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of melanocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of melanocyte differentiation." [GOC:curators]
 synonym: "down regulation of melanocyte differentiation" EXACT []
 synonym: "down-regulation of melanocyte differentiation" EXACT []
 synonym: "downregulation of melanocyte differentiation" EXACT []
@@ -283533,7 +284164,7 @@
 id: GO:0045636
 name: positive regulation of melanocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of melanocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of melanocyte differentiation." [GOC:curators]
 synonym: "activation of melanocyte differentiation" NARROW []
 synonym: "positive regulation of melanophore differentiation" EXACT []
 synonym: "stimulation of melanocyte differentiation" NARROW []
@@ -283550,7 +284181,7 @@
 id: GO:0045637
 name: regulation of myeloid cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of myeloid cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of myeloid cell differentiation." [GOC:curators]
 xref: Reactome:R-HSA-8939246 "RUNX1 regulates transcription of genes involved in differentiation of myeloid cells"
 is_a: GO:1903706 ! regulation of hemopoiesis
 intersection_of: GO:0065007 ! biological regulation
@@ -283561,7 +284192,7 @@
 id: GO:0045638
 name: negative regulation of myeloid cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of myeloid cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of myeloid cell differentiation." [GOC:curators]
 synonym: "down regulation of myeloid cell differentiation" EXACT []
 synonym: "down-regulation of myeloid cell differentiation" EXACT []
 synonym: "downregulation of myeloid cell differentiation" EXACT []
@@ -283576,7 +284207,7 @@
 id: GO:0045639
 name: positive regulation of myeloid cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of myeloid cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of myeloid cell differentiation." [GOC:curators]
 synonym: "activation of myeloid cell differentiation" NARROW []
 synonym: "stimulation of myeloid cell differentiation" NARROW []
 synonym: "up regulation of myeloid cell differentiation" EXACT []
@@ -283592,7 +284223,7 @@
 id: GO:0045640
 name: regulation of basophil differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of basophil differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of basophil differentiation." [GOC:curators]
 is_a: GO:0030852 ! regulation of granulocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030221 ! basophil differentiation
@@ -283602,7 +284233,7 @@
 id: GO:0045641
 name: negative regulation of basophil differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of basophil differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of basophil differentiation." [GOC:curators]
 synonym: "down regulation of basophil differentiation" EXACT []
 synonym: "down-regulation of basophil differentiation" EXACT []
 synonym: "downregulation of basophil differentiation" EXACT []
@@ -283617,7 +284248,7 @@
 id: GO:0045642
 name: positive regulation of basophil differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of basophil differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of basophil differentiation." [GOC:curators]
 synonym: "activation of basophil differentiation" NARROW []
 synonym: "stimulation of basophil differentiation" NARROW []
 synonym: "up regulation of basophil differentiation" EXACT []
@@ -283633,7 +284264,7 @@
 id: GO:0045643
 name: regulation of eosinophil differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of eosinophil differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of eosinophil differentiation." [GOC:curators]
 is_a: GO:0030852 ! regulation of granulocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030222 ! eosinophil differentiation
@@ -283643,7 +284274,7 @@
 id: GO:0045644
 name: negative regulation of eosinophil differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of eosinophil differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of eosinophil differentiation." [GOC:curators]
 synonym: "down regulation of eosinophil differentiation" EXACT []
 synonym: "down-regulation of eosinophil differentiation" EXACT []
 synonym: "downregulation of eosinophil differentiation" EXACT []
@@ -283658,7 +284289,7 @@
 id: GO:0045645
 name: positive regulation of eosinophil differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of eosinophil differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of eosinophil differentiation." [GOC:curators]
 synonym: "activation of eosinophil differentiation" NARROW []
 synonym: "stimulation of eosinophil differentiation" NARROW []
 synonym: "up regulation of eosinophil differentiation" EXACT []
@@ -283674,7 +284305,7 @@
 id: GO:0045646
 name: regulation of erythrocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of erythrocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of erythrocyte differentiation." [GOC:curators]
 synonym: "regulation of RBC differentiation" EXACT [CL:0000232]
 synonym: "regulation of red blood cell differentiation" EXACT [CL:0000232]
 is_a: GO:0045637 ! regulation of myeloid cell differentiation
@@ -283686,7 +284317,7 @@
 id: GO:0045647
 name: negative regulation of erythrocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of erythrocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of erythrocyte differentiation." [GOC:curators]
 synonym: "down regulation of erythrocyte differentiation" EXACT []
 synonym: "down-regulation of erythrocyte differentiation" EXACT []
 synonym: "downregulation of erythrocyte differentiation" EXACT []
@@ -283703,7 +284334,7 @@
 id: GO:0045648
 name: positive regulation of erythrocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of erythrocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of erythrocyte differentiation." [GOC:curators]
 synonym: "activation of erythrocyte differentiation" NARROW []
 synonym: "positive regulation of RBC differentiation" EXACT [CL:0000232]
 synonym: "positive regulation of red blood cell differentiation" EXACT [CL:0000232]
@@ -283721,7 +284352,7 @@
 id: GO:0045649
 name: regulation of macrophage differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of macrophage differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of macrophage differentiation." [GOC:curators]
 is_a: GO:0002761 ! regulation of myeloid leukocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030225 ! macrophage differentiation
@@ -283731,7 +284362,7 @@
 id: GO:0045650
 name: negative regulation of macrophage differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of macrophage differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of macrophage differentiation." [GOC:curators]
 synonym: "down regulation of macrophage differentiation" EXACT []
 synonym: "down-regulation of macrophage differentiation" EXACT []
 synonym: "downregulation of macrophage differentiation" EXACT []
@@ -283746,7 +284377,7 @@
 id: GO:0045651
 name: positive regulation of macrophage differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of macrophage differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of macrophage differentiation." [GOC:curators]
 synonym: "activation of macrophage differentiation" NARROW []
 synonym: "stimulation of macrophage differentiation" NARROW []
 synonym: "up regulation of macrophage differentiation" EXACT []
@@ -283762,7 +284393,7 @@
 id: GO:0045652
 name: regulation of megakaryocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of megakaryocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of megakaryocyte differentiation." [GOC:curators]
 xref: Reactome:R-HSA-8936459 "RUNX1 regulates genes involved in megakaryocyte differentiation and platelet function"
 is_a: GO:0045637 ! regulation of myeloid cell differentiation
 intersection_of: GO:0065007 ! biological regulation
@@ -283773,7 +284404,7 @@
 id: GO:0045653
 name: negative regulation of megakaryocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of megakaryocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of megakaryocyte differentiation." [GOC:curators]
 synonym: "down regulation of megakaryocyte differentiation" EXACT []
 synonym: "down-regulation of megakaryocyte differentiation" EXACT []
 synonym: "downregulation of megakaryocyte differentiation" EXACT []
@@ -283788,7 +284419,7 @@
 id: GO:0045654
 name: positive regulation of megakaryocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of megakaryocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of megakaryocyte differentiation." [GOC:curators]
 synonym: "activation of megakaryocyte differentiation" NARROW []
 synonym: "stimulation of megakaryocyte differentiation" NARROW []
 synonym: "up regulation of megakaryocyte differentiation" EXACT []
@@ -283804,7 +284435,7 @@
 id: GO:0045655
 name: regulation of monocyte differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of monocyte differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of monocyte differentiation." [GOC:curators]
 is_a: GO:0002761 ! regulation of myeloid leukocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030224 ! monocyte differentiation
@@ -283814,7 +284445,7 @@
 id: GO:0045656
 name: negative regulation of monocyte differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of monocyte differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of monocyte differentiation." [GOC:curators]
 synonym: "down regulation of monocyte differentiation" EXACT []
 synonym: "down-regulation of monocyte differentiation" EXACT []
 synonym: "downregulation of monocyte differentiation" EXACT []
@@ -283829,7 +284460,7 @@
 id: GO:0045657
 name: positive regulation of monocyte differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of monocyte differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of monocyte differentiation." [GOC:curators]
 synonym: "activation of monocyte differentiation" NARROW []
 synonym: "stimulation of monocyte differentiation" NARROW []
 synonym: "up regulation of monocyte differentiation" EXACT []
@@ -283845,7 +284476,7 @@
 id: GO:0045658
 name: regulation of neutrophil differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of neutrophil differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of neutrophil differentiation." [GOC:curators]
 is_a: GO:0030852 ! regulation of granulocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030223 ! neutrophil differentiation
@@ -283855,7 +284486,7 @@
 id: GO:0045659
 name: negative regulation of neutrophil differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of neutrophil differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of neutrophil differentiation." [GOC:curators]
 synonym: "down regulation of neutrophil differentiation" EXACT []
 synonym: "down-regulation of neutrophil differentiation" EXACT []
 synonym: "downregulation of neutrophil differentiation" EXACT []
@@ -283870,7 +284501,7 @@
 id: GO:0045660
 name: positive regulation of neutrophil differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of neutrophil differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of neutrophil differentiation." [GOC:curators]
 synonym: "activation of neutrophil differentiation" NARROW []
 synonym: "stimulation of neutrophil differentiation" NARROW []
 synonym: "up regulation of neutrophil differentiation" EXACT []
@@ -283886,7 +284517,7 @@
 id: GO:0045661
 name: regulation of myoblast differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:go_curators, GOC:mtg_muscle]
+def: "Any process that modulates the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:curators, GOC:mtg_muscle]
 is_a: GO:0045595 ! regulation of cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0045445 ! myoblast differentiation
@@ -283896,7 +284527,7 @@
 id: GO:0045662
 name: negative regulation of myoblast differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:go_curators, GOC:mtg_muscle]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:curators, GOC:mtg_muscle]
 synonym: "down regulation of myoblast differentiation" EXACT []
 synonym: "down-regulation of myoblast differentiation" EXACT []
 synonym: "downregulation of myoblast differentiation" EXACT []
@@ -283911,7 +284542,7 @@
 id: GO:0045663
 name: positive regulation of myoblast differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:go_curators, GOC:mtg_muscle]
+def: "Any process that activates or increases the frequency, rate or extent of myoblast differentiation. A myoblast is a mononucleate cell type that, by fusion with other myoblasts, gives rise to the myotubes that eventually develop into skeletal muscle fibers." [CL:0000056, GOC:curators, GOC:mtg_muscle]
 synonym: "activation of myoblast differentiation" NARROW []
 synonym: "stimulation of myoblast differentiation" NARROW []
 synonym: "up regulation of myoblast differentiation" EXACT []
@@ -283927,7 +284558,7 @@
 id: GO:0045664
 name: regulation of neuron differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of neuron differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of neuron differentiation." [GOC:curators]
 is_a: GO:0045595 ! regulation of cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030182 ! neuron differentiation
@@ -283937,7 +284568,7 @@
 id: GO:0045665
 name: negative regulation of neuron differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of neuron differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of neuron differentiation." [GOC:curators]
 synonym: "down regulation of neuron differentiation" EXACT []
 synonym: "down-regulation of neuron differentiation" EXACT []
 synonym: "downregulation of neuron differentiation" EXACT []
@@ -283952,7 +284583,7 @@
 id: GO:0045666
 name: positive regulation of neuron differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of neuron differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of neuron differentiation." [GOC:curators]
 synonym: "activation of neuron differentiation" NARROW []
 synonym: "stimulation of neuron differentiation" NARROW []
 synonym: "up regulation of neuron differentiation" EXACT []
@@ -283968,7 +284599,7 @@
 id: GO:0045667
 name: regulation of osteoblast differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of osteoblast differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of osteoblast differentiation." [GOC:curators]
 is_a: GO:0045595 ! regulation of cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0001649 ! osteoblast differentiation
@@ -283978,7 +284609,7 @@
 id: GO:0045668
 name: negative regulation of osteoblast differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of osteoblast differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of osteoblast differentiation." [GOC:curators]
 synonym: "down regulation of osteoblast differentiation" EXACT []
 synonym: "down-regulation of osteoblast differentiation" EXACT []
 synonym: "downregulation of osteoblast differentiation" EXACT []
@@ -283993,7 +284624,7 @@
 id: GO:0045669
 name: positive regulation of osteoblast differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of osteoblast differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of osteoblast differentiation." [GOC:curators]
 synonym: "activation of osteoblast differentiation" NARROW []
 synonym: "stimulation of osteoblast differentiation" NARROW []
 synonym: "up regulation of osteoblast differentiation" EXACT []
@@ -284009,7 +284640,7 @@
 id: GO:0045670
 name: regulation of osteoclast differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of osteoclast differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of osteoclast differentiation." [GOC:curators]
 is_a: GO:0002761 ! regulation of myeloid leukocyte differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0030316 ! osteoclast differentiation
@@ -284019,7 +284650,7 @@
 id: GO:0045671
 name: negative regulation of osteoclast differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of osteoclast differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of osteoclast differentiation." [GOC:curators]
 synonym: "down regulation of osteoclast differentiation" EXACT []
 synonym: "down-regulation of osteoclast differentiation" EXACT []
 synonym: "downregulation of osteoclast differentiation" EXACT []
@@ -284034,7 +284665,7 @@
 id: GO:0045672
 name: positive regulation of osteoclast differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of osteoclast differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of osteoclast differentiation." [GOC:curators]
 synonym: "activation of osteoclast differentiation" NARROW []
 synonym: "stimulation of osteoclast differentiation" NARROW []
 synonym: "up regulation of osteoclast differentiation" EXACT []
@@ -284050,7 +284681,7 @@
 id: GO:0045676
 name: regulation of R7 cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of R7 differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of R7 differentiation." [GOC:curators]
 is_a: GO:0110116 ! regulation of compound eye photoreceptor cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0045466 ! R7 cell differentiation
@@ -284060,7 +284691,7 @@
 id: GO:0045677
 name: negative regulation of R7 cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of R7cell differentiation." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of R7cell differentiation." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of R7 differentiation" EXACT []
 synonym: "down-regulation of R7 differentiation" EXACT []
 synonym: "downregulation of R7 differentiation" EXACT []
@@ -284076,7 +284707,7 @@
 id: GO:0045678
 name: positive regulation of R7 cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of R7 cell differentiation." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of R7 cell differentiation." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of R7 differentiation" NARROW []
 synonym: "positive regulation of R7 differentiation" EXACT [GOC:dph, GOC:tb]
 synonym: "stimulation of R7 differentiation" NARROW []
@@ -284093,7 +284724,7 @@
 id: GO:0045679
 name: regulation of R8 cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of R8 differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of R8 differentiation." [GOC:curators]
 is_a: GO:0110116 ! regulation of compound eye photoreceptor cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0045465 ! R8 cell differentiation
@@ -284103,7 +284734,7 @@
 id: GO:0045680
 name: negative regulation of R8 cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of R8 cell differentiation." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of R8 cell differentiation." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of R8 differentiation" EXACT []
 synonym: "down-regulation of R8 differentiation" EXACT []
 synonym: "downregulation of R8 differentiation" EXACT []
@@ -284119,7 +284750,7 @@
 id: GO:0045681
 name: positive regulation of R8 cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of R8 cell differentiation." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of R8 cell differentiation." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of R8 differentiation" NARROW []
 synonym: "positive regulation of R8 differentiation" EXACT [GOC:dph, GOC:tb]
 synonym: "stimulation of R8 differentiation" NARROW []
@@ -284136,7 +284767,7 @@
 id: GO:0045682
 name: regulation of epidermis development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of epidermis development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of epidermis development." [GOC:curators]
 synonym: "regulation of epidermal development" EXACT []
 synonym: "regulation of hypodermis development" RELATED [GOC:kmv, GOC:rk]
 is_a: GO:0050793 ! regulation of developmental process
@@ -284148,7 +284779,7 @@
 id: GO:0045683
 name: negative regulation of epidermis development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermis development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of epidermis development." [GOC:curators]
 synonym: "down regulation of epidermis development" EXACT []
 synonym: "down-regulation of epidermis development" EXACT []
 synonym: "downregulation of epidermis development" EXACT []
@@ -284165,7 +284796,7 @@
 id: GO:0045684
 name: positive regulation of epidermis development
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of epidermis development." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of epidermis development." [GOC:curators]
 synonym: "activation of epidermis development" NARROW []
 synonym: "positive regulation of epidermal development" EXACT []
 synonym: "positive regulation of hypodermis development" RELATED [GOC:kmv, GOC:rk]
@@ -284183,7 +284814,7 @@
 id: GO:0045685
 name: regulation of glial cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of glia cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of glia cell differentiation." [GOC:curators]
 synonym: "regulation of glia cell differentiation" EXACT []
 synonym: "regulation of neuroglia differentiation" EXACT []
 is_a: GO:0045595 ! regulation of cell differentiation
@@ -284195,7 +284826,7 @@
 id: GO:0045686
 name: negative regulation of glial cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of glia cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of glia cell differentiation." [GOC:curators]
 synonym: "down regulation of glial cell differentiation" EXACT []
 synonym: "down-regulation of glial cell differentiation" EXACT []
 synonym: "downregulation of glial cell differentiation" EXACT []
@@ -284212,7 +284843,7 @@
 id: GO:0045687
 name: positive regulation of glial cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of glia cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of glia cell differentiation." [GOC:curators]
 synonym: "activation of glial cell differentiation" NARROW []
 synonym: "positive regulation of glia cell differentiation" EXACT []
 synonym: "positive regulation of neuroglia differentiation" EXACT []
@@ -284230,7 +284861,7 @@
 id: GO:0045688
 name: regulation of antipodal cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of antipodal cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that modulates the frequency, rate or extent of antipodal cell differentiation." [GOC:curators, GOC:mtg_plant]
 is_a: GO:0045595 ! regulation of cell differentiation
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0009557 ! antipodal cell differentiation
@@ -284240,7 +284871,7 @@
 id: GO:0045689
 name: negative regulation of antipodal cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of antipodal cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of antipodal cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "down regulation of antipodal cell differentiation" EXACT []
 synonym: "down-regulation of antipodal cell differentiation" EXACT []
 synonym: "downregulation of antipodal cell differentiation" EXACT []
@@ -284255,7 +284886,7 @@
 id: GO:0045690
 name: positive regulation of antipodal cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of antipodal cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that activates or increases the frequency, rate or extent of antipodal cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "activation of antipodal cell differentiation" NARROW []
 synonym: "stimulation of antipodal cell differentiation" NARROW []
 synonym: "up regulation of antipodal cell differentiation" EXACT []
@@ -284272,7 +284903,7 @@
 id: GO:0045691
 name: regulation of embryo sac central cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of female gametophyte central cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that modulates the frequency, rate or extent of female gametophyte central cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "regulation of female gametophyte central cell differentiation" EXACT []
 is_a: GO:0045595 ! regulation of cell differentiation
 is_a: GO:2000026 ! regulation of multicellular organismal development
@@ -284284,7 +284915,7 @@
 id: GO:0045692
 name: negative regulation of embryo sac central cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryo sac central cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryo sac central cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "down regulation of female gametophyte central cell differentiation" EXACT []
 synonym: "down-regulation of female gametophyte central cell differentiation" EXACT []
 synonym: "downregulation of female gametophyte central cell differentiation" EXACT []
@@ -284301,7 +284932,7 @@
 id: GO:0045693
 name: positive regulation of embryo sac central cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of embryo sac central cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that activates or increases the frequency, rate or extent of embryo sac central cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "activation of female gametophyte central cell differentiation" NARROW []
 synonym: "positive regulation of female gametophyte central cell differentiation" EXACT []
 synonym: "stimulation of female gametophyte central cell differentiation" NARROW []
@@ -284319,7 +284950,7 @@
 id: GO:0045694
 name: regulation of embryo sac egg cell differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that modulates the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "regulation of female gametophyte egg cell differentiation" EXACT []
 is_a: GO:0045595 ! regulation of cell differentiation
 is_a: GO:2000241 ! regulation of reproductive process
@@ -284331,7 +284962,7 @@
 id: GO:0045695
 name: negative regulation of embryo sac egg cell differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "down regulation of female gametophyte egg cell differentiation" EXACT []
 synonym: "down-regulation of female gametophyte egg cell differentiation" EXACT []
 synonym: "downregulation of female gametophyte egg cell differentiation" EXACT []
@@ -284349,7 +284980,7 @@
 id: GO:0045696
 name: positive regulation of embryo sac egg cell differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:go_curators, GOC:mtg_plant]
+def: "Any process that activates or increases the frequency, rate or extent of embryo sac egg cell differentiation." [GOC:curators, GOC:mtg_plant]
 synonym: "activation of female gametophyte egg cell differentiation" NARROW []
 synonym: "positive regulation of female gametophyte egg cell differentiation" EXACT []
 synonym: "stimulation of female gametophyte egg cell differentiation" NARROW []
@@ -284368,7 +284999,7 @@
 id: GO:0045697
 name: regulation of synergid differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of synergid cell differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of synergid cell differentiation." [GOC:curators]
 synonym: "regulation of synergid cell differentiation" EXACT []
 is_a: GO:0045595 ! regulation of cell differentiation
 intersection_of: GO:0065007 ! biological regulation
@@ -284379,7 +285010,7 @@
 id: GO:0045698
 name: negative regulation of synergid differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of synergid cell differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of synergid cell differentiation." [GOC:curators]
 synonym: "down regulation of synergid differentiation" EXACT []
 synonym: "down-regulation of synergid differentiation" EXACT []
 synonym: "downregulation of synergid differentiation" EXACT []
@@ -284396,7 +285027,7 @@
 id: GO:0045699
 name: positive regulation of synergid differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of synergid cell differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of synergid cell differentiation." [GOC:curators]
 synonym: "activation of synergid differentiation" NARROW []
 synonym: "positive regulation of synergid cell differentiation" EXACT []
 synonym: "stimulation of synergid differentiation" NARROW []
@@ -284414,7 +285045,7 @@
 id: GO:0045700
 name: regulation of spermatid nuclear differentiation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of spermatid nuclear differentiation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of spermatid nuclear differentiation." [GOC:curators]
 is_a: GO:0051239 ! regulation of multicellular organismal process
 is_a: GO:1903353 ! regulation of nucleus organization
 intersection_of: GO:0065007 ! biological regulation
@@ -284425,7 +285056,7 @@
 id: GO:0045701
 name: negative regulation of spermatid nuclear differentiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of spermatid nuclear differentiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of spermatid nuclear differentiation." [GOC:curators]
 synonym: "down regulation of spermatid nuclear differentiation" EXACT []
 synonym: "down-regulation of spermatid nuclear differentiation" EXACT []
 synonym: "downregulation of spermatid nuclear differentiation" EXACT []
@@ -284443,7 +285074,7 @@
 id: GO:0045702
 name: positive regulation of spermatid nuclear differentiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of spermatid nuclear differentiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of spermatid nuclear differentiation." [GOC:curators]
 synonym: "activation of spermatid nuclear differentiation" NARROW []
 synonym: "stimulation of spermatid nuclear differentiation" NARROW []
 synonym: "up regulation of spermatid nuclear differentiation" EXACT []
@@ -284468,7 +285099,7 @@
 id: GO:0045704
 name: regulation of salivary gland boundary specification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of salivary gland determination." [GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of salivary gland determination." [GOC:curators, GOC:tb]
 synonym: "regulation of salivary gland determination" EXACT [GOC:tb]
 is_a: GO:0050793 ! regulation of developmental process
 is_a: GO:0051239 ! regulation of multicellular organismal process
@@ -284480,7 +285111,7 @@
 id: GO:0045705
 name: negative regulation of salivary gland boundary specification
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination." [GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination." [GOC:curators, GOC:tb]
 synonym: "down regulation of salivary gland determination" EXACT []
 synonym: "down-regulation of salivary gland determination" EXACT []
 synonym: "downregulation of salivary gland determination" EXACT []
@@ -284497,7 +285128,7 @@
 id: GO:0045706
 name: positive regulation of salivary gland boundary specification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination." [GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination." [GOC:curators, GOC:tb]
 synonym: "activation of salivary gland determination" NARROW []
 synonym: "positive regulation of salivary gland determination" EXACT [GOC:tb]
 synonym: "stimulation of salivary gland determination" NARROW []
@@ -284515,7 +285146,7 @@
 id: GO:0045707
 name: regulation of adult salivary gland boundary specification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:curators, GOC:tb]
 synonym: "regulation of adult salivary gland determination" EXACT [GOC:tb]
 is_a: GO:0045704 ! regulation of salivary gland boundary specification
 intersection_of: GO:0065007 ! biological regulation
@@ -284526,7 +285157,7 @@
 id: GO:0045708
 name: regulation of larval salivary gland boundary specification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:curators, GOC:tb]
 synonym: "regulation of larval salivary gland determination" EXACT [GOC:tb]
 is_a: GO:0045704 ! regulation of salivary gland boundary specification
 intersection_of: GO:0065007 ! biological regulation
@@ -284537,7 +285168,7 @@
 id: GO:0045709
 name: negative regulation of adult salivary gland boundary specification
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:curators, GOC:tb]
 synonym: "down regulation of adult salivary gland determination" EXACT []
 synonym: "down-regulation of adult salivary gland determination" EXACT []
 synonym: "downregulation of adult salivary gland determination" EXACT []
@@ -284553,7 +285184,7 @@
 id: GO:0045710
 name: negative regulation of larval salivary gland boundary specification
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:curators, GOC:tb]
 synonym: "down regulation of larval salivary gland determination" EXACT []
 synonym: "down-regulation of larval salivary gland determination" EXACT []
 synonym: "downregulation of larval salivary gland determination" EXACT []
@@ -284569,7 +285200,7 @@
 id: GO:0045711
 name: positive regulation of adult salivary gland boundary specification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination in an adult organism." [GOC:curators, GOC:tb]
 synonym: "activation of adult salivary gland determination" NARROW []
 synonym: "positive regulation of adult salivary gland determination" EXACT [GOC:tb]
 synonym: "stimulation of adult salivary gland determination" NARROW []
@@ -284586,7 +285217,7 @@
 id: GO:0045712
 name: positive regulation of larval salivary gland boundary specification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of salivary gland determination in a larval organism." [GOC:curators, GOC:tb]
 synonym: "activation of larval salivary gland determination" NARROW []
 synonym: "positive regulation of larval salivary gland determination" EXACT [GOC:tb]
 synonym: "stimulation of larval salivary gland determination" NARROW []
@@ -284603,7 +285234,7 @@
 id: GO:0045713
 name: obsolete low-density lipoprotein particle receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of low-density lipoprotein receptors, cell surface proteins that mediate the endocytosis of low-density lipoprotein particles by cells." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of low-density lipoprotein receptors, cell surface proteins that mediate the endocytosis of low-density lipoprotein particles by cells." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "LDLr biosynthesis" EXACT []
 synonym: "LDLr biosynthetic process" EXACT []
@@ -284619,7 +285250,7 @@
 id: GO:0045714
 name: obsolete regulation of low-density lipoprotein particle receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein particle receptors." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein particle receptors." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "regulation of LDLr biosynthesis" EXACT []
 synonym: "regulation of LDLr biosynthetic process" EXACT []
@@ -284635,7 +285266,7 @@
 id: GO:0045715
 name: obsolete negative regulation of low-density lipoprotein particle receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein particle receptors." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein particle receptors." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "down regulation of low-density lipoprotein receptor biosynthetic process" EXACT []
 synonym: "down-regulation of low-density lipoprotein receptor biosynthetic process" EXACT []
@@ -284653,7 +285284,7 @@
 id: GO:0045716
 name: obsolete positive regulation of low-density lipoprotein particle receptor biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein receptors." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of low-density lipoprotein receptors." [GOC:curators]
 comment: The reason for obsoletion is that those represent specific gene products, and genes annotated to those terms should be annotated to 'regulation of gene expression' or 'regulation of transcription' or some signaling term.
 synonym: "activation of low-density lipoprotein receptor biosynthetic process" NARROW []
 synonym: "positive regulation of low-density lipoprotein receptor anabolism" EXACT []
@@ -284672,7 +285303,7 @@
 id: GO:0045717
 name: negative regulation of fatty acid biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids." [GOC:curators]
 synonym: "down regulation of fatty acid biosynthetic process" EXACT []
 synonym: "down-regulation of fatty acid biosynthetic process" EXACT []
 synonym: "downregulation of fatty acid biosynthetic process" EXACT []
@@ -284692,7 +285323,7 @@
 id: GO:0045718
 name: obsolete negative regulation of flagellum assembly
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the formation of a flagellum." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the formation of a flagellum." [GOC:curators]
 comment: This term was made obsolete because microtubule-based flagellum (GO:0009434) and motile cilium (GO:0031514), as referenced by this term, were determined to be equivalent and merged. When a merge for this term was considered, it was noticed that both eukaryotic and prokaryotic proteins are annotated to it because its meaning had been ambiguous so obsoletion was considered safest.
 synonym: "down regulation of flagellum assembly" EXACT []
 synonym: "down-regulation of flagellum assembly" EXACT []
@@ -284708,7 +285339,7 @@
 id: GO:0045719
 name: negative regulation of glycogen biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:curators]
 synonym: "down regulation of glycogen biosynthetic process" EXACT []
 synonym: "down-regulation of glycogen biosynthetic process" EXACT []
 synonym: "downregulation of glycogen biosynthetic process" EXACT []
@@ -284728,7 +285359,7 @@
 id: GO:0045720
 name: negative regulation of integrin biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:curators]
 synonym: "down regulation of integrin biosynthetic process" EXACT []
 synonym: "down-regulation of integrin biosynthetic process" EXACT []
 synonym: "downregulation of integrin biosynthetic process" EXACT []
@@ -284748,7 +285379,7 @@
 id: GO:0045721
 name: negative regulation of gluconeogenesis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of gluconeogenesis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of gluconeogenesis." [GOC:curators]
 synonym: "down regulation of gluconeogenesis" EXACT []
 synonym: "down-regulation of gluconeogenesis" EXACT []
 synonym: "downregulation of gluconeogenesis" EXACT []
@@ -284765,7 +285396,7 @@
 id: GO:0045722
 name: positive regulation of gluconeogenesis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of gluconeogenesis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of gluconeogenesis." [GOC:curators]
 synonym: "activation of gluconeogenesis" NARROW []
 synonym: "stimulation of gluconeogenesis" NARROW []
 synonym: "up regulation of gluconeogenesis" EXACT []
@@ -284782,7 +285413,7 @@
 id: GO:0045723
 name: positive regulation of fatty acid biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of fatty acids." [GOC:curators]
 synonym: "activation of fatty acid biosynthetic process" NARROW []
 synonym: "positive regulation of fatty acid anabolism" EXACT []
 synonym: "positive regulation of fatty acid biosynthesis" EXACT []
@@ -284803,7 +285434,7 @@
 id: GO:0045724
 name: positive regulation of cilium assembly
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the formation of a cilium." [GOC:cilia, GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the formation of a cilium." [GOC:cilia, GOC:curators]
 synonym: "activation of cilium assembly" NARROW []
 synonym: "positive regulation of flagellum assembly" BROAD []
 synonym: "positive regulation of flagellum biogenesis" RELATED [GOC:mah]
@@ -284822,7 +285453,7 @@
 id: GO:0045725
 name: positive regulation of glycogen biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of glycogen." [GOC:curators]
 synonym: "activation of glycogen biosynthetic process" NARROW []
 synonym: "positive regulation of glycogen anabolism" EXACT []
 synonym: "positive regulation of glycogen biosynthesis" EXACT []
@@ -284843,7 +285474,7 @@
 id: GO:0045726
 name: positive regulation of integrin biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of integrins." [GOC:curators]
 synonym: "activation of integrin biosynthetic process" NARROW []
 synonym: "positive regulation of integrin anabolism" EXACT []
 synonym: "positive regulation of integrin biosynthesis" EXACT []
@@ -284865,7 +285496,7 @@
 name: positive regulation of translation
 namespace: biological_process
 alt_id: GO:0045946
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of proteins by the translation of mRNA or circRNA." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of proteins by the translation of mRNA or circRNA." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of protein biosynthetic process" NARROW []
 synonym: "positive regulation of protein anabolism" EXACT []
 synonym: "positive regulation of protein biosynthesis" EXACT []
@@ -284920,7 +285551,7 @@
 namespace: biological_process
 alt_id: GO:1903364
 alt_id: GO:2000600
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of a protein by the destruction of the native, active configuration, with or without the hydrolysis of peptide bonds." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of a protein by the destruction of the native, active configuration, with or without the hydrolysis of peptide bonds." [GOC:curators]
 synonym: "activation of cellular protein breakdown" NARROW [GOC:TermGenie]
 synonym: "activation of cellular protein catabolic process" NARROW [GOC:TermGenie]
 synonym: "activation of cellular protein catabolism" NARROW [GOC:TermGenie]
@@ -284967,7 +285598,7 @@
 name: acetate catabolic process
 namespace: biological_process
 alt_id: GO:0019663
-def: "The chemical reactions and pathways resulting in the breakdown of acetate, the anion of acetic acid." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of acetate, the anion of acetic acid." [GOC:curators]
 synonym: "acetate breakdown" EXACT []
 synonym: "acetate catabolism" EXACT []
 synonym: "acetate degradation" EXACT []
@@ -284979,7 +285610,7 @@
 id: GO:0045734
 name: regulation of acetate catabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate, the anion of acetic acid." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate, the anion of acetic acid." [GOC:curators]
 synonym: "regulation of acetate breakdown" EXACT []
 synonym: "regulation of acetate catabolism" EXACT []
 synonym: "regulation of acetate degradation" EXACT []
@@ -285008,7 +285639,7 @@
 id: GO:0045736
 name: negative regulation of cyclin-dependent protein serine/threonine kinase activity
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cyclin-dependent protein serine/threonine kinase activity." [GOC:go_curators, GOC:pr]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cyclin-dependent protein serine/threonine kinase activity." [GOC:curators, GOC:pr]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0000079 ! regulation of cyclin-dependent protein serine/threonine kinase activity
 is_a: GO:0045786 ! negative regulation of cell cycle
@@ -285022,7 +285653,7 @@
 id: GO:0045737
 name: positive regulation of cyclin-dependent protein serine/threonine kinase activity
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of CDK activity." [GOC:go_curators, GOC:pr]
+def: "Any process that activates or increases the frequency, rate or extent of CDK activity." [GOC:curators, GOC:pr]
 subset: gocheck_obsoletion_candidate
 synonym: "activation of cyclin-dependent protein kinase activity" BROAD []
 synonym: "positive regulation of CDK activity" BROAD []
@@ -285043,7 +285674,7 @@
 id: GO:0045738
 name: negative regulation of DNA repair
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA repair." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA repair." [GOC:curators]
 synonym: "down regulation of DNA repair" EXACT []
 synonym: "down-regulation of DNA repair" EXACT []
 synonym: "downregulation of DNA repair" EXACT []
@@ -285059,7 +285690,7 @@
 id: GO:0045739
 name: positive regulation of DNA repair
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of DNA repair." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of DNA repair." [GOC:curators]
 synonym: "activation of DNA repair" NARROW []
 synonym: "stimulation of DNA repair" NARROW []
 synonym: "up regulation of DNA repair" EXACT []
@@ -285076,7 +285707,7 @@
 id: GO:0045740
 name: positive regulation of DNA replication
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of DNA replication." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of DNA replication." [GOC:curators]
 synonym: "activation of DNA replication" NARROW []
 synonym: "stimulation of DNA replication" NARROW []
 synonym: "up regulation of DNA replication" EXACT []
@@ -285093,7 +285724,7 @@
 id: GO:0045741
 name: obsolete positive regulation of epidermal growth factor-activated receptor activity
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of EGF-activated receptor activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of EGF-activated receptor activity." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function. Consider molecular function regulator activity ; GO:0098772 or a child.
 synonym: "activation of epidermal growth factor receptor activity" NARROW []
 synonym: "positive regulation of EGF receptor activity" EXACT []
@@ -285112,7 +285743,7 @@
 id: GO:0045742
 name: positive regulation of epidermal growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of epidermal growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "activation of epidermal growth factor receptor signaling pathway" NARROW []
 synonym: "positive regulation of EGF receptor signaling pathway" EXACT []
 synonym: "positive regulation of EGF receptor signalling pathway" EXACT []
@@ -285132,7 +285763,7 @@
 id: GO:0045743
 name: positive regulation of fibroblast growth factor receptor signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of fibroblast growth factor receptor signaling pathway activity." [GOC:curators]
 synonym: "activation of fibroblast growth factor receptor signaling pathway" NARROW []
 synonym: "positive regulation of FGF receptor signaling pathway" EXACT []
 synonym: "positive regulation of FGF receptor signalling pathway" EXACT []
@@ -285151,7 +285782,7 @@
 id: GO:0045744
 name: negative regulation of G protein-coupled receptor signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of G protein-coupled receptor signaling pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of G protein-coupled receptor signaling pathway." [GOC:curators]
 synonym: "down regulation of G-protein coupled receptor protein signaling pathway" EXACT []
 synonym: "down-regulation of G-protein coupled receptor protein signaling pathway" EXACT []
 synonym: "downregulation of G-protein coupled receptor protein signaling pathway" EXACT []
@@ -285173,7 +285804,7 @@
 id: GO:0045745
 name: positive regulation of G protein-coupled receptor signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of G protein-coupled receptor signaling pathway activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of G protein-coupled receptor signaling pathway activity." [GOC:curators]
 synonym: "activation of G-protein coupled receptor protein signaling pathway" NARROW []
 synonym: "positive regulation of G protein coupled receptor protein signaling pathway" EXACT []
 synonym: "positive regulation of G protein coupled receptor protein signalling pathway" EXACT []
@@ -285197,7 +285828,7 @@
 id: GO:0045746
 name: negative regulation of Notch signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Notch signaling pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Notch signaling pathway." [GOC:curators]
 synonym: "down regulation of Notch signaling pathway" EXACT []
 synonym: "down-regulation of Notch signaling pathway" EXACT []
 synonym: "downregulation of Notch signaling pathway" EXACT []
@@ -285216,7 +285847,7 @@
 id: GO:0045747
 name: positive regulation of Notch signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the Notch signaling pathway." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the Notch signaling pathway." [GOC:curators]
 synonym: "activation of Notch signaling pathway" NARROW []
 synonym: "positive regulation of N signaling pathway" EXACT []
 synonym: "positive regulation of N signalling pathway" EXACT []
@@ -285235,7 +285866,7 @@
 id: GO:0045748
 name: positive regulation of R8 cell spacing in compound eye
 namespace: biological_process
-def: "Any process that activates or enforces the correct R8 cell spacing in a compound eye." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or enforces the correct R8 cell spacing in a compound eye." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of R8 spacing" NARROW []
 synonym: "positive regulation of R8 spacing" EXACT [GOC:dph, GOC:tb]
 synonym: "stimulation of R8 spacing" NARROW []
@@ -285248,7 +285879,7 @@
 id: GO:0045749
 name: obsolete negative regulation of S phase of mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of S phase of mitotic cell cycle activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of S phase of mitotic cell cycle activity." [GOC:curators]
 comment: This term was made obsolete as part of the cell cycle overhaul. This is because it was decided that cell cycle phases are intervals and so cannot be regulated, rather it is the transitions between the phases that are regulated.
 synonym: "down regulation of S phase of mitotic cell cycle" EXACT []
 synonym: "down-regulation of S phase of mitotic cell cycle" EXACT []
@@ -285262,7 +285893,7 @@
 id: GO:0045750
 name: obsolete positive regulation of S phase of mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of S phase of mitotic cell cycle activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of S phase of mitotic cell cycle activity." [GOC:curators]
 comment: This term was made obsolete as part of the cell cycle overhaul. This is because it was decided that cell cycle phases are intervals and so cannot be regulated, rather it is the transitions between the phases that are regulated.
 synonym: "activation of S phase of mitotic cell cycle" NARROW []
 synonym: "positive regulation of S phase of mitotic cell cycle" EXACT []
@@ -285277,7 +285908,7 @@
 id: GO:0045751
 name: negative regulation of Toll signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Tl signaling pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the Tl signaling pathway." [GOC:curators]
 synonym: "down regulation of Toll signaling pathway" EXACT []
 synonym: "down-regulation of Toll signaling pathway" EXACT []
 synonym: "downregulation of Toll signaling pathway" EXACT []
@@ -285294,7 +285925,7 @@
 id: GO:0045752
 name: positive regulation of Toll signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the Tl signaling pathway." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the Tl signaling pathway." [GOC:curators]
 synonym: "activation of Toll signaling pathway" NARROW []
 synonym: "positive regulation of Tl signaling pathway" EXACT []
 synonym: "positive regulation of Tl signalling pathway" EXACT []
@@ -285312,7 +285943,7 @@
 id: GO:0045753
 name: negative regulation of acetate catabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:curators]
 synonym: "down regulation of acetate catabolic process" EXACT []
 synonym: "down-regulation of acetate catabolic process" EXACT []
 synonym: "downregulation of acetate catabolic process" EXACT []
@@ -285331,7 +285962,7 @@
 id: GO:0045754
 name: positive regulation of acetate catabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:curators]
 synonym: "activation of acetate catabolic process" NARROW []
 synonym: "positive regulation of acetate breakdown" EXACT []
 synonym: "positive regulation of acetate catabolism" EXACT []
@@ -285351,7 +285982,7 @@
 id: GO:0045755
 name: obsolete negative regulation of initiation of acetate catabolic process by acetate
 namespace: biological_process
-def: "OBSOLETE. Any process that stops or prevents the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops or prevents the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "down regulation of initiation of acetate catabolic process by acetate" EXACT []
 synonym: "down-regulation of initiation of acetate catabolic process by acetate" EXACT []
@@ -285366,7 +285997,7 @@
 id: GO:0045756
 name: obsolete positive regulation of initiation of acetate catabolic process by acetate
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the activation, by acetate, of the chemical reactions and pathways resulting in the breakdown of acetate." [GOC:curators]
 comment: The reason for obsoletion is that this term represents a molecular function.
 synonym: "activation of initiation of acetate catabolic process by acetate" NARROW []
 synonym: "positive regulation of initiation of acetate breakdown by acetate" EXACT []
@@ -285382,7 +286013,7 @@
 id: GO:0045757
 name: obsolete negative regulation of actin polymerization and/or depolymerization
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of actin polymerization and/or depolymerization." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of actin polymerization and/or depolymerization." [GOC:curators]
 comment: This term was made obsolete because it has been split.
 synonym: "negative regulation of actin polymerization and/or depolymerization" EXACT []
 is_obsolete: true
@@ -285393,7 +286024,7 @@
 id: GO:0045758
 name: obsolete positive regulation of actin polymerization and/or depolymerization
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of actin polymerization and/or depolymerization." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of actin polymerization and/or depolymerization." [GOC:curators]
 comment: This term was made obsolete because it has been split.
 synonym: "positive regulation of actin polymerization and/or depolymerization" EXACT []
 is_obsolete: true
@@ -285404,7 +286035,7 @@
 id: GO:0045759
 name: negative regulation of action potential
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:curators]
 synonym: "down regulation of action potential" EXACT []
 synonym: "down-regulation of action potential" EXACT []
 synonym: "downregulation of action potential" EXACT []
@@ -285419,7 +286050,7 @@
 id: GO:0045760
 name: positive regulation of action potential
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:curators]
 synonym: "activation of action potential" NARROW []
 synonym: "stimulation of action potential" NARROW []
 synonym: "up regulation of action potential" EXACT []
@@ -285435,7 +286066,7 @@
 id: GO:0045761
 name: regulation of adenylate cyclase activity
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of adenylate cyclase activity." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of adenylate cyclase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "adenylate cyclase regulator" RELATED []
 synonym: "regulation of adenylyl cyclase activity" EXACT []
@@ -285448,7 +286079,7 @@
 id: GO:0045762
 name: positive regulation of adenylate cyclase activity
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of adenylate cyclase activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of adenylate cyclase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "adenylate cyclase activator" RELATED []
 synonym: "positive regulation of adenylyl cyclase activity" EXACT []
@@ -285467,7 +286098,7 @@
 id: GO:0045763
 name: negative regulation of amino acid metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving amino acid." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving amino acid." [GOC:curators]
 synonym: "down regulation of amino acid metabolic process" EXACT []
 synonym: "down-regulation of amino acid metabolic process" EXACT []
 synonym: "downregulation of amino acid metabolic process" EXACT []
@@ -285484,7 +286115,7 @@
 id: GO:0045764
 name: positive regulation of amino acid metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving amino acid." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving amino acid." [GOC:curators]
 synonym: "activation of amino acid metabolic process" NARROW []
 synonym: "positive regulation of amino acid metabolism" EXACT []
 synonym: "positive regulation of cellular amino acid metabolic process" EXACT []
@@ -285502,7 +286133,7 @@
 id: GO:0045765
 name: regulation of angiogenesis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of angiogenesis." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of angiogenesis." [GOC:curators]
 is_a: GO:0022603 ! regulation of anatomical structure morphogenesis
 is_a: GO:1901342 ! regulation of vasculature development
 intersection_of: GO:0065007 ! biological regulation
@@ -285513,7 +286144,7 @@
 id: GO:0045766
 name: positive regulation of angiogenesis
 namespace: biological_process
-def: "Any process that activates or increases angiogenesis." [GOC:go_curators]
+def: "Any process that activates or increases angiogenesis." [GOC:curators]
 synonym: "activation of angiogenesis" NARROW []
 synonym: "stimulation of angiogenesis" NARROW []
 synonym: "up regulation of angiogenesis" EXACT []
@@ -285529,7 +286160,7 @@
 id: GO:0045767
 name: obsolete regulation of anti-apoptosis
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of anti-apoptosis." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of anti-apoptosis." [GOC:curators, GOC:mtg_apoptosis]
 comment: This term was made obsolete because it was ill-defined.
 synonym: "regulation of anti-apoptosis" EXACT []
 is_obsolete: true
@@ -285538,7 +286169,7 @@
 id: GO:0045768
 name: obsolete positive regulation of anti-apoptosis
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of anti-apoptosis." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of anti-apoptosis." [GOC:curators, GOC:mtg_apoptosis]
 comment: This term was made obsolete because it was ill-defined.
 synonym: "activation of anti-apoptosis" NARROW []
 synonym: "positive regulation of anti-apoptosis" EXACT []
@@ -285552,7 +286183,7 @@
 id: GO:0045769
 name: negative regulation of asymmetric cell division
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of asymmetric cell division." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of asymmetric cell division." [GOC:curators]
 synonym: "down regulation of asymmetric cell division" EXACT []
 synonym: "down-regulation of asymmetric cell division" EXACT []
 synonym: "downregulation of asymmetric cell division" EXACT []
@@ -285567,7 +286198,7 @@
 id: GO:0045770
 name: positive regulation of asymmetric cell division
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of asymmetric cell division." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of asymmetric cell division." [GOC:curators]
 synonym: "activation of asymmetric cell division" NARROW []
 synonym: "stimulation of asymmetric cell division" NARROW []
 synonym: "up regulation of asymmetric cell division" EXACT []
@@ -285583,7 +286214,7 @@
 id: GO:0045771
 name: negative regulation of autophagosome size
 namespace: biological_process
-def: "Any process that reduces autophagosome size." [GOC:autophagy, GOC:go_curators]
+def: "Any process that reduces autophagosome size." [GOC:autophagy, GOC:curators]
 synonym: "down regulation of autophagic vacuole size" EXACT []
 synonym: "down-regulation of autophagic vacuole size" EXACT []
 synonym: "downregulation of autophagic vacuole size" EXACT []
@@ -285595,7 +286226,7 @@
 id: GO:0045772
 name: positive regulation of autophagosome size
 namespace: biological_process
-def: "Any process that increases autophagosome size." [GOC:autophagy, GOC:go_curators]
+def: "Any process that increases autophagosome size." [GOC:autophagy, GOC:curators]
 synonym: "activation of autophagic vacuole size" NARROW []
 synonym: "positive regulation of autophagic vacuole size" EXACT [GOC:autophagy]
 synonym: "stimulation of autophagic vacuole size" NARROW []
@@ -285608,7 +286239,7 @@
 id: GO:0045773
 name: positive regulation of axon extension
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of axon extension." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of axon extension." [GOC:curators]
 synonym: "activation of axon extension" NARROW []
 synonym: "stimulation of axon extension" NARROW []
 synonym: "up regulation of axon extension" EXACT []
@@ -285626,7 +286257,7 @@
 id: GO:0045774
 name: negative regulation of beta 2 integrin biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:curators]
 synonym: "down regulation of beta 2 integrin biosynthetic process" EXACT []
 synonym: "down-regulation of beta 2 integrin biosynthetic process" EXACT []
 synonym: "downregulation of beta 2 integrin biosynthetic process" EXACT []
@@ -285645,7 +286276,7 @@
 id: GO:0045775
 name: positive regulation of beta 2 integrin biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of beta 2 integrins." [GOC:curators]
 synonym: "activation of beta 2 integrin biosynthetic process" NARROW []
 synonym: "positive regulation of beta 2 integrin anabolism" EXACT []
 synonym: "positive regulation of beta 2 integrin biosynthesis" EXACT []
@@ -285665,7 +286296,7 @@
 id: GO:0045776
 name: negative regulation of blood pressure
 namespace: biological_process
-def: "Any process in which the force of blood traveling through the circulatory system is decreased." [GOC:go_curators, GOC:mtg_cardio]
+def: "Any process in which the force of blood traveling through the circulatory system is decreased." [GOC:curators, GOC:mtg_cardio]
 synonym: "down regulation of blood pressure" EXACT []
 synonym: "down-regulation of blood pressure" EXACT []
 synonym: "downregulation of blood pressure" EXACT []
@@ -285676,7 +286307,7 @@
 id: GO:0045777
 name: positive regulation of blood pressure
 namespace: biological_process
-def: "Any process in which the force of blood traveling through the circulatory system is increased." [GOC:go_curators, GOC:mtg_cardio]
+def: "Any process in which the force of blood traveling through the circulatory system is increased." [GOC:curators, GOC:mtg_cardio]
 synonym: "activation of blood pressure" NARROW []
 synonym: "stimulation of blood pressure" NARROW []
 synonym: "up regulation of blood pressure" EXACT []
@@ -285688,7 +286319,7 @@
 id: GO:0045778
 name: positive regulation of ossification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of ossification, the formation of bone or of a bony substance or the conversion of fibrous tissue or of cartilage into bone or a bony substance." [GOC:curators]
 synonym: "activation of ossification" NARROW []
 synonym: "positive regulation of bone biosynthesis" EXACT []
 synonym: "positive regulation of bone formation" EXACT []
@@ -285706,7 +286337,7 @@
 id: GO:0045779
 name: negative regulation of bone resorption
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of bone resorption." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of bone resorption." [GOC:curators]
 synonym: "down regulation of bone resorption" EXACT []
 synonym: "down-regulation of bone resorption" EXACT []
 synonym: "downregulation of bone resorption" EXACT []
@@ -285721,7 +286352,7 @@
 id: GO:0045780
 name: positive regulation of bone resorption
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of bone resorption." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of bone resorption." [GOC:curators]
 synonym: "activation of bone resorption" NARROW []
 synonym: "stimulation of bone resorption" NARROW []
 synonym: "up regulation of bone resorption" EXACT []
@@ -285737,7 +286368,7 @@
 id: GO:0045781
 name: negative regulation of cell budding
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell budding." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cell budding." [GOC:curators]
 synonym: "down regulation of cell budding" EXACT []
 synonym: "down-regulation of cell budding" EXACT []
 synonym: "downregulation of cell budding" EXACT []
@@ -285754,7 +286385,7 @@
 id: GO:0045782
 name: positive regulation of cell budding
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cell budding." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cell budding." [GOC:curators]
 synonym: "activation of cell budding" NARROW []
 synonym: "positive regulation of budding" BROAD []
 synonym: "stimulation of cell budding" NARROW []
@@ -285772,7 +286403,7 @@
 id: GO:0045783
 name: obsolete negative regulation of calcium in ER
 namespace: biological_process
-def: "OBSOLETE. Any process that reduces the concentration of calcium in the ER." [GOC:go_curators]
+def: "OBSOLETE. Any process that reduces the concentration of calcium in the ER." [GOC:curators]
 comment: This term was made obsolete because the term name is ambiguous and as a result, the term was incorrectly placed in the ontology (it was a descendant of 'protein transport').
 synonym: "negative regulation of calcium in ER" EXACT []
 is_obsolete: true
@@ -285783,7 +286414,7 @@
 id: GO:0045784
 name: obsolete positive regulation of calcium in ER
 namespace: biological_process
-def: "OBSOLETE. Any process that increases the concentration of calcium in the ER." [GOC:go_curators]
+def: "OBSOLETE. Any process that increases the concentration of calcium in the ER." [GOC:curators]
 comment: This term was made obsolete because the term name is ambiguous and as a result, the term was incorrectly placed in the ontology (it was a descendant of 'protein transport').
 synonym: "positive regulation of calcium in ER" EXACT []
 is_obsolete: true
@@ -285794,7 +286425,7 @@
 id: GO:0045785
 name: positive regulation of cell adhesion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cell adhesion." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cell adhesion." [GOC:curators]
 synonym: "activation of cell adhesion" NARROW []
 synonym: "stimulation of cell adhesion" NARROW []
 synonym: "up regulation of cell adhesion" EXACT []
@@ -285810,7 +286441,7 @@
 id: GO:0045786
 name: negative regulation of cell cycle
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of progression through the cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of progression through the cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of progression through cell cycle" EXACT []
 synonym: "down-regulation of progression through cell cycle" EXACT []
 synonym: "downregulation of progression through cell cycle" EXACT []
@@ -285828,7 +286459,7 @@
 id: GO:0045787
 name: positive regulation of cell cycle
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of progression through the cell cycle." [GOC:go_curators]
+def: "Any process that activates or increases the rate or extent of progression through the cell cycle." [GOC:curators]
 synonym: "activation of progression through cell cycle" NARROW []
 synonym: "positive regulation of cell cycle progression" EXACT []
 synonym: "positive regulation of progression through cell cycle" EXACT [GOC:dph, GOC:tb]
@@ -285847,7 +286478,7 @@
 id: GO:0045792
 name: negative regulation of cell size
 namespace: biological_process
-def: "Any process that reduces cell size." [GOC:go_curators]
+def: "Any process that reduces cell size." [GOC:curators]
 synonym: "down regulation of cell size" EXACT []
 synonym: "down-regulation of cell size" EXACT []
 synonym: "downregulation of cell size" EXACT []
@@ -285858,7 +286489,7 @@
 id: GO:0045793
 name: positive regulation of cell size
 namespace: biological_process
-def: "Any process that increases cell size." [GOC:go_curators]
+def: "Any process that increases cell size." [GOC:curators]
 synonym: "activation of cell size" NARROW []
 synonym: "stimulation of cell size" NARROW []
 synonym: "up regulation of cell size" EXACT []
@@ -285870,7 +286501,7 @@
 id: GO:0045794
 name: negative regulation of cell volume
 namespace: biological_process
-def: "Any process that decreases cell volume." [GOC:go_curators]
+def: "Any process that decreases cell volume." [GOC:curators]
 synonym: "cell regulatory volume decrease" RELATED [PMID:12388065]
 synonym: "RVD" RELATED [PMID:12388065]
 is_a: GO:0006884 ! cell volume homeostasis
@@ -285879,14 +286510,14 @@
 id: GO:0045795
 name: positive regulation of cell volume
 namespace: biological_process
-def: "Any process that increases cell volume." [GOC:go_curators]
+def: "Any process that increases cell volume." [GOC:curators]
 is_a: GO:0006884 ! cell volume homeostasis
 
 [Term]
 id: GO:0045796
 name: negative regulation of intestinal cholesterol absorption
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of uptake of cholesterol into the blood by absorption from the intestine." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of uptake of cholesterol into the blood by absorption from the intestine." [GOC:curators]
 synonym: "down regulation of cholesterol absorption" EXACT []
 synonym: "down-regulation of cholesterol absorption" EXACT []
 synonym: "downregulation of cholesterol absorption" EXACT []
@@ -285901,7 +286532,7 @@
 id: GO:0045797
 name: positive regulation of intestinal cholesterol absorption
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of uptake of cholesterol into the blood by absorption from the intestine." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of uptake of cholesterol into the blood by absorption from the intestine." [GOC:curators]
 synonym: "activation of cholesterol absorption" NARROW []
 synonym: "stimulation of cholesterol absorption" NARROW []
 synonym: "up regulation of cholesterol absorption" EXACT []
@@ -285917,7 +286548,7 @@
 id: GO:0045800
 name: negative regulation of chitin-based cuticle tanning
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of chitin-based cuticular tanning." [GOC:go_curators, GOC:jid, GOC:mtg_sensu]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of chitin-based cuticular tanning." [GOC:curators, GOC:jid, GOC:mtg_sensu]
 synonym: "down regulation of cuticle tanning" EXACT []
 synonym: "down-regulation of cuticle tanning" EXACT []
 synonym: "downregulation of cuticle tanning" EXACT []
@@ -285935,7 +286566,7 @@
 id: GO:0045801
 name: positive regulation of chitin-based cuticle tanning
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of chitin-based cuticular tanning." [GOC:go_curators, GOC:jid, GOC:mtg_sensu]
+def: "Any process that activates or increases the frequency, rate or extent of chitin-based cuticular tanning." [GOC:curators, GOC:jid, GOC:mtg_sensu]
 synonym: "activation of cuticle tanning" NARROW []
 synonym: "positive regulation of cuticle hardening" NARROW []
 synonym: "positive regulation of cuticle tanning" EXACT []
@@ -285954,7 +286585,7 @@
 id: GO:0045802
 name: obsolete negative regulation of cytoskeleton
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:curators]
 comment: This term was made obsolete because cytoskeleton is not a process, so the term made no sense.
 synonym: "negative regulation of cytoskeleton" EXACT []
 is_obsolete: true
@@ -285964,7 +286595,7 @@
 id: GO:0045803
 name: obsolete positive regulation of cytoskeleton
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the organization, biogenesis or maintenance of the cytoskeleton." [GOC:curators]
 comment: This term was made obsolete because cytoskeleton is not a process, so the term made no sense.
 synonym: "positive regulation of cytoskeleton" EXACT []
 is_obsolete: true
@@ -285974,7 +286605,7 @@
 id: GO:0045804
 name: negative regulation of eclosion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of eclosion." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of eclosion." [GOC:curators]
 synonym: "down regulation of eclosion" EXACT []
 synonym: "down-regulation of eclosion" EXACT []
 synonym: "downregulation of eclosion" EXACT []
@@ -285990,7 +286621,7 @@
 id: GO:0045805
 name: positive regulation of eclosion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of eclosion." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of eclosion." [GOC:curators]
 synonym: "activation of eclosion" NARROW []
 synonym: "stimulation of eclosion" NARROW []
 synonym: "up regulation of eclosion" EXACT []
@@ -286006,7 +286637,7 @@
 id: GO:0045806
 name: negative regulation of endocytosis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of endocytosis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of endocytosis." [GOC:curators]
 synonym: "down regulation of endocytosis" EXACT []
 synonym: "down-regulation of endocytosis" EXACT []
 synonym: "downregulation of endocytosis" EXACT []
@@ -286022,7 +286653,7 @@
 id: GO:0045807
 name: positive regulation of endocytosis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of endocytosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of endocytosis." [GOC:curators]
 synonym: "activation of endocytosis" NARROW []
 synonym: "stimulation of endocytosis" NARROW []
 synonym: "up regulation of endocytosis" EXACT []
@@ -286039,7 +286670,7 @@
 id: GO:0045808
 name: negative regulation of establishment of competence for transformation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of establishment of competence for transformation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of establishment of competence for transformation." [GOC:curators]
 synonym: "down regulation of establishment of competence for transformation" EXACT []
 synonym: "down-regulation of establishment of competence for transformation" EXACT []
 synonym: "downregulation of establishment of competence for transformation" EXACT []
@@ -286056,7 +286687,7 @@
 id: GO:0045809
 name: positive regulation of establishment of competence for transformation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of establishment of competence for transformation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of establishment of competence for transformation." [GOC:curators]
 synonym: "activation of establishment of competence for transformation" NARROW []
 synonym: "activator of the establishment of competence for transformation activity" RELATED []
 synonym: "stimulation of establishment of competence for transformation" NARROW []
@@ -286074,7 +286705,7 @@
 id: GO:0045812
 name: negative regulation of Wnt signaling pathway, calcium modulating pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of frizzled-2 signaling pathway" EXACT []
 synonym: "down-regulation of frizzled-2 signaling pathway" EXACT []
 synonym: "downregulation of frizzled-2 signaling pathway" EXACT []
@@ -286093,7 +286724,7 @@
 id: GO:0045813
 name: positive regulation of Wnt signaling pathway, calcium modulating pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors leads to an increase in intracellular calcium and activation of protein kinase C (PKC)." [GOC:curators]
 synonym: "activation of frizzled-2 signaling pathway" NARROW []
 synonym: "positive regulation of frizzled-2 signaling pathway" EXACT []
 synonym: "positive regulation of frizzled-2 signalling pathway" EXACT []
@@ -286157,7 +286788,7 @@
 id: GO:0045818
 name: negative regulation of glycogen catabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:curators]
 synonym: "down regulation of glycogen catabolic process" EXACT []
 synonym: "down-regulation of glycogen catabolic process" EXACT []
 synonym: "downregulation of glycogen catabolic process" EXACT []
@@ -286178,7 +286809,7 @@
 id: GO:0045819
 name: positive regulation of glycogen catabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of glycogen." [GOC:curators]
 synonym: "activation of glycogen catabolic process" NARROW []
 synonym: "positive regulation of glycogen breakdown" EXACT []
 synonym: "positive regulation of glycogen catabolism" EXACT []
@@ -286200,7 +286831,7 @@
 id: GO:0045820
 name: negative regulation of glycolytic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of glycolysis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of glycolysis." [GOC:curators]
 synonym: "down regulation of glycolysis" EXACT []
 synonym: "down-regulation of glycolysis" EXACT []
 synonym: "downregulation of glycolysis" EXACT []
@@ -286217,7 +286848,7 @@
 id: GO:0045821
 name: positive regulation of glycolytic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of glycolysis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of glycolysis." [GOC:curators]
 synonym: "activation of glycolysis" NARROW []
 synonym: "stimulation of glycolysis" NARROW []
 synonym: "up regulation of glycolysis" EXACT []
@@ -286236,7 +286867,7 @@
 id: GO:0045822
 name: negative regulation of heart contraction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of heart contraction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of heart contraction." [GOC:curators]
 synonym: "down regulation of heart contraction" EXACT []
 synonym: "down-regulation of heart contraction" EXACT []
 synonym: "downregulation of heart contraction" EXACT []
@@ -286252,7 +286883,7 @@
 id: GO:0045823
 name: positive regulation of heart contraction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of heart contraction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of heart contraction." [GOC:curators]
 synonym: "activation of heart contraction" NARROW []
 synonym: "positive regulation of cardiac contraction" EXACT []
 synonym: "stimulation of heart contraction" NARROW []
@@ -286269,7 +286900,7 @@
 id: GO:0045824
 name: negative regulation of innate immune response
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the innate immune response." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the innate immune response." [GOC:curators]
 synonym: "down regulation of innate immune response" EXACT []
 synonym: "down-regulation of innate immune response" EXACT []
 synonym: "downregulation of innate immune response" EXACT []
@@ -286287,7 +286918,7 @@
 id: GO:0045825
 name: obsolete negative regulation of intermediate filament polymerization and/or depolymerization
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of intermediate filament polymerization and/or depolymerization." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of intermediate filament polymerization and/or depolymerization." [GOC:curators]
 comment: This term was made obsolete because it has been split.
 synonym: "negative regulation of intermediate filament polymerization and/or depolymerization" EXACT []
 is_obsolete: true
@@ -286298,7 +286929,7 @@
 id: GO:0045826
 name: obsolete positive regulation of intermediate filament polymerization and/or depolymerization
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of intermediate filament polymerization and/or depolymerization." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of intermediate filament polymerization and/or depolymerization." [GOC:curators]
 comment: This term was made obsolete because it has been split.
 synonym: "positive regulation of intermediate filament polymerization and/or depolymerization" EXACT []
 is_obsolete: true
@@ -286309,7 +286940,7 @@
 id: GO:0045827
 name: negative regulation of isoprenoid metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving isoprenoid." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving isoprenoid." [GOC:curators]
 synonym: "down regulation of isoprenoid metabolic process" EXACT []
 synonym: "down-regulation of isoprenoid metabolic process" EXACT []
 synonym: "downregulation of isoprenoid metabolic process" EXACT []
@@ -286325,7 +286956,7 @@
 id: GO:0045828
 name: obsolete positive regulation of isoprenoid metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving isoprenoid." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving isoprenoid." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of isoprenoid metabolic process" NARROW []
 synonym: "positive regulation of isoprenoid metabolism" EXACT []
@@ -286340,7 +286971,7 @@
 id: GO:0045829
 name: negative regulation of isotype switching
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of isotype switching." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of isotype switching." [GOC:curators]
 synonym: "down regulation of isotype switching" EXACT []
 synonym: "down-regulation of isotype switching" EXACT []
 synonym: "downregulation of isotype switching" EXACT []
@@ -286362,7 +286993,7 @@
 id: GO:0045830
 name: positive regulation of isotype switching
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of isotype switching." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of isotype switching." [GOC:curators]
 synonym: "activation of isotype switching" NARROW []
 synonym: "positive regulation of class switch recombination" EXACT []
 synonym: "positive regulation of class switching" EXACT []
@@ -286385,7 +287016,7 @@
 id: GO:0045831
 name: obsolete negative regulation of light-activated channel activity
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of light-activated channel activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of light-activated channel activity." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "down regulation of light-activated channel activity" EXACT []
 synonym: "down-regulation of light-activated channel activity" EXACT []
@@ -286398,7 +287029,7 @@
 id: GO:0045832
 name: obsolete positive regulation of light-activated channel activity
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of light-activated channel activity." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of light-activated channel activity." [GOC:curators]
 comment: This term was obsoleted because it represents a molecular function.
 synonym: "activation of light-activated channel activity" NARROW []
 synonym: "stimulation of light-activated channel activity" NARROW []
@@ -286412,7 +287043,7 @@
 id: GO:0045833
 name: negative regulation of lipid metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:curators]
 synonym: "down regulation of lipid metabolic process" EXACT []
 synonym: "down-regulation of lipid metabolic process" EXACT []
 synonym: "downregulation of lipid metabolic process" EXACT []
@@ -286428,7 +287059,7 @@
 id: GO:0045834
 name: positive regulation of lipid metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving lipids." [GOC:curators]
 synonym: "activation of lipid metabolic process" NARROW []
 synonym: "positive regulation of lipid metabolism" EXACT []
 synonym: "stimulation of lipid metabolic process" NARROW []
@@ -286445,7 +287076,7 @@
 id: GO:0045835
 name: negative regulation of meiotic nuclear division
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of meiosis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of meiosis." [GOC:curators]
 synonym: "down regulation of meiosis" EXACT []
 synonym: "down-regulation of meiosis" EXACT []
 synonym: "downregulation of meiosis" EXACT []
@@ -286463,7 +287094,7 @@
 id: GO:0045836
 name: positive regulation of meiotic nuclear division
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of meiosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of meiosis." [GOC:curators]
 synonym: "activation of meiosis" NARROW []
 synonym: "positive regulation of meiosis" EXACT []
 synonym: "stimulation of meiosis" NARROW []
@@ -286482,7 +287113,7 @@
 id: GO:0045837
 name: negative regulation of membrane potential
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of establishment or extent of a membrane potential, the electric potential existing across any membrane arising from charges in the membrane itself and from the charges present in the media on either side of the membrane." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of establishment or extent of a membrane potential, the electric potential existing across any membrane arising from charges in the membrane itself and from the charges present in the media on either side of the membrane." [GOC:curators]
 synonym: "down regulation of membrane potential" EXACT []
 synonym: "down-regulation of membrane potential" EXACT []
 synonym: "downregulation of membrane potential" EXACT []
@@ -286494,7 +287125,7 @@
 id: GO:0045838
 name: positive regulation of membrane potential
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of establishment or extent of a membrane potential, the electric potential existing across any membrane arising from charges in the membrane itself and from the charges present in the media on either side of the membrane." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of establishment or extent of a membrane potential, the electric potential existing across any membrane arising from charges in the membrane itself and from the charges present in the media on either side of the membrane." [GOC:curators]
 synonym: "activation of membrane potential" NARROW []
 synonym: "elevation of membrane potential" EXACT [GOC:rph]
 synonym: "stimulation of membrane potential" NARROW []
@@ -286507,7 +287138,7 @@
 id: GO:0045839
 name: negative regulation of mitotic nuclear division
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of mitosis. Mitosis is the division of the eukaryotic cell nucleus to produce two daughter nuclei that, usually, contain the identical chromosome complement to their mother." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of mitosis. Mitosis is the division of the eukaryotic cell nucleus to produce two daughter nuclei that, usually, contain the identical chromosome complement to their mother." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of mitosis" EXACT []
 synonym: "down-regulation of mitosis" EXACT []
 synonym: "downregulation of mitosis" EXACT []
@@ -286525,7 +287156,7 @@
 id: GO:0045840
 name: positive regulation of mitotic nuclear division
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of mitosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of mitosis." [GOC:curators]
 synonym: "activation of mitosis" NARROW []
 synonym: "mitogenic activity" NARROW []
 synonym: "positive regulation of mitosis" EXACT []
@@ -286544,7 +287175,7 @@
 id: GO:0045841
 name: negative regulation of mitotic metaphase/anaphase transition
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the cell cycle process in which a cell progresses from metaphase to anaphase during mitosis, triggered by the activation of the anaphase promoting complex by Cdc20/Sleepy homolog which results in the degradation of Securin." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the cell cycle process in which a cell progresses from metaphase to anaphase during mitosis, triggered by the activation of the anaphase promoting complex by Cdc20/Sleepy homolog which results in the degradation of Securin." [GOC:curators]
 synonym: "down regulation of mitotic metaphase/anaphase transition" EXACT []
 synonym: "down-regulation of mitotic metaphase/anaphase transition" EXACT []
 synonym: "downregulation of mitotic metaphase/anaphase transition" EXACT []
@@ -286562,7 +287193,7 @@
 id: GO:0045842
 name: positive regulation of mitotic metaphase/anaphase transition
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the cell cycle process in which a cell progresses from metaphase to anaphase during mitosis, triggered by the activation of the anaphase promoting complex by Cdc20/Sleepy homolog which results in the degradation of Securin." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the cell cycle process in which a cell progresses from metaphase to anaphase during mitosis, triggered by the activation of the anaphase promoting complex by Cdc20/Sleepy homolog which results in the degradation of Securin." [GOC:curators]
 synonym: "activation of mitotic metaphase/anaphase transition" NARROW []
 synonym: "stimulation of mitotic metaphase/anaphase transition" NARROW []
 synonym: "up regulation of mitotic metaphase/anaphase transition" EXACT []
@@ -286581,7 +287212,7 @@
 id: GO:0045843
 name: negative regulation of striated muscle tissue development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of striated muscle development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of striated muscle development." [GOC:curators]
 synonym: "down regulation of striated muscle development" EXACT []
 synonym: "down-regulation of striated muscle development" EXACT []
 synonym: "downregulation of striated muscle development" EXACT []
@@ -286597,7 +287228,7 @@
 id: GO:0045844
 name: positive regulation of striated muscle tissue development
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of striated muscle development." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of striated muscle development." [GOC:curators]
 synonym: "activation of striated muscle development" NARROW []
 synonym: "stimulation of striated muscle development" NARROW []
 synonym: "up regulation of striated muscle development" EXACT []
@@ -286614,7 +287245,7 @@
 id: GO:0045847
 name: negative regulation of nitrogen utilization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of nitrogen utilization." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of nitrogen utilization." [GOC:curators]
 synonym: "down regulation of nitrogen utilization" EXACT []
 synonym: "down-regulation of nitrogen utilization" EXACT []
 synonym: "downregulation of nitrogen utilization" EXACT []
@@ -286629,7 +287260,7 @@
 id: GO:0045848
 name: positive regulation of nitrogen utilization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of nitrogen utilization." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of nitrogen utilization." [GOC:curators]
 synonym: "activation of nitrogen utilization" NARROW []
 synonym: "stimulation of nitrogen utilization" NARROW []
 synonym: "up regulation of nitrogen utilization" EXACT []
@@ -286645,7 +287276,7 @@
 id: GO:0045849
 name: negative regulation of nurse cell apoptotic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of nurse cell apoptotic process." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of nurse cell apoptotic process." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "down regulation of nurse cell apoptosis" EXACT []
 synonym: "down-regulation of nurse cell apoptosis" EXACT []
 synonym: "downregulation of nurse cell apoptosis" EXACT []
@@ -286662,7 +287293,7 @@
 id: GO:0045850
 name: positive regulation of nurse cell apoptotic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of nurse cell apoptotic process." [GOC:go_curators, GOC:mtg_apoptosis]
+def: "Any process that activates or increases the frequency, rate or extent of nurse cell apoptotic process." [GOC:curators, GOC:mtg_apoptosis]
 synonym: "activation of nurse cell apoptosis" NARROW []
 synonym: "positive regulation of nurse cell apoptosis" NARROW []
 synonym: "stimulation of nurse cell apoptosis" NARROW []
@@ -286680,7 +287311,7 @@
 id: GO:0045851
 name: pH reduction
 namespace: biological_process
-def: "Any process that reduces the internal pH of an organism, part of an organism or a cell, measured by the concentration of the hydrogen ion." [GOC:go_curators]
+def: "Any process that reduces the internal pH of an organism, part of an organism or a cell, measured by the concentration of the hydrogen ion." [GOC:curators]
 synonym: "acidification" EXACT []
 is_a: GO:0006885 ! regulation of pH
 
@@ -286688,14 +287319,14 @@
 id: GO:0045852
 name: pH elevation
 namespace: biological_process
-def: "Any process that increases the internal pH of an organism, part of an organism or a cell, measured by the concentration of the hydrogen ion." [GOC:go_curators]
+def: "Any process that increases the internal pH of an organism, part of an organism or a cell, measured by the concentration of the hydrogen ion." [GOC:curators]
 is_a: GO:0006885 ! regulation of pH
 
 [Term]
 id: GO:0045853
 name: negative regulation of bicoid mRNA localization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the process in which bicoid mRNA is transported to, or maintained in, a specific location." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the process in which bicoid mRNA is transported to, or maintained in, a specific location." [GOC:curators]
 synonym: "down regulation of bicoid mRNA localization" EXACT []
 synonym: "down-regulation of bicoid mRNA localization" EXACT []
 synonym: "downregulation of bicoid mRNA localization" EXACT []
@@ -286711,7 +287342,7 @@
 id: GO:0045854
 name: positive regulation of bicoid mRNA localization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the process in which bicoid mRNA is transported to, or maintained in, a specific location." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the process in which bicoid mRNA is transported to, or maintained in, a specific location." [GOC:curators]
 synonym: "activation of bicoid mRNA localization" NARROW []
 synonym: "positive regulation of bicoid mRNA localisation" EXACT [GOC:mah]
 synonym: "stimulation of bicoid mRNA localization" NARROW []
@@ -286729,7 +287360,7 @@
 id: GO:0045855
 name: negative regulation of pole plasm oskar mRNA localization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of a process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of a process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:curators]
 synonym: "down regulation of pole plasm oskar mRNA localization" EXACT []
 synonym: "down-regulation of pole plasm oskar mRNA localization" EXACT []
 synonym: "downregulation of pole plasm oskar mRNA localization" EXACT []
@@ -286746,7 +287377,7 @@
 id: GO:0045856
 name: positive regulation of pole plasm oskar mRNA localization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the process in which oskar mRNA is transported to, or maintained in, the oocyte pole plasm." [GOC:curators]
 synonym: "activation of pole plasm oskar mRNA localization" NARROW []
 synonym: "positive regulation of oocyte pole plasm oskar mRNA localization" EXACT []
 synonym: "positive regulation of pole plasm oskar mRNA localisation" EXACT [GOC:mah]
@@ -286764,7 +287395,7 @@
 id: GO:0045857
 name: obsolete negative regulation of molecular function, epigenetic
 namespace: biological_process
-def: "OBSOLETE. Any heritable epigenetic process that stops, prevents, or reduces the frequency, rate or extent of protein function by self-perpetuating conformational conversions of normal proteins in healthy cells. This is distinct from, though mechanistically analogous to, disease states associated with prion propagation and amyloidogenesis. A single protein, if it carries a glutamine/asparagine-rich ('prion') domain, can sometimes stably exist in at least two distinct physical states, each associated with a different phenotype; propagation of one of these traits is achieved by a self-perpetuating change in the protein from one form to the other, mediated by conformational changes in the glutamine/asparagine-rich domain. Prion domains are both modular and transferable to other proteins, on which they can confer a heritable epigenetic alteration of function; existing bioinformatics data indicate that they are rare in non-eukarya, but common in eukarya." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "OBSOLETE. Any heritable epigenetic process that stops, prevents, or reduces the frequency, rate or extent of protein function by self-perpetuating conformational conversions of normal proteins in healthy cells. This is distinct from, though mechanistically analogous to, disease states associated with prion propagation and amyloidogenesis. A single protein, if it carries a glutamine/asparagine-rich ('prion') domain, can sometimes stably exist in at least two distinct physical states, each associated with a different phenotype; propagation of one of these traits is achieved by a self-perpetuating change in the protein from one form to the other, mediated by conformational changes in the glutamine/asparagine-rich domain. Prion domains are both modular and transferable to other proteins, on which they can confer a heritable epigenetic alteration of function; existing bioinformatics data indicate that they are rare in non-eukarya, but common in eukarya." [GOC:curators, GOC:dph, GOC:tb]
 comment: This term was obsoleted because it is not an active process.
 synonym: "down regulation of protein activity, epigenetic" EXACT []
 synonym: "down-regulation of protein activity, epigenetic" EXACT []
@@ -286778,7 +287409,7 @@
 id: GO:0045858
 name: obsolete positive regulation of molecular function, epigenetic
 namespace: biological_process
-def: "OBSOLETE. Any heritable epigenetic process that increases the frequency, rate or extent of protein function by self-perpetuating conformational conversions of normal proteins in healthy cells. This is distinct from, though mechanistically analogous to, disease states associated with prion propagation and amyloidogenesis. A single protein, if it carries a glutamine/asparagine-rich ('prion') domain, can sometimes stably exist in at least two distinct physical states, each associated with a different phenotype; propagation of one of these traits is achieved by a self-perpetuating change in the protein from one form to the other, mediated by conformational changes in the glutamine/asparagine-rich domain. Prion domains are both modular and transferable to other proteins, on which they can confer a heritable epigenetic alteration of function; existing bioinformatics data indicate that they are rare in non-eukarya, but common in eukarya." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "OBSOLETE. Any heritable epigenetic process that increases the frequency, rate or extent of protein function by self-perpetuating conformational conversions of normal proteins in healthy cells. This is distinct from, though mechanistically analogous to, disease states associated with prion propagation and amyloidogenesis. A single protein, if it carries a glutamine/asparagine-rich ('prion') domain, can sometimes stably exist in at least two distinct physical states, each associated with a different phenotype; propagation of one of these traits is achieved by a self-perpetuating change in the protein from one form to the other, mediated by conformational changes in the glutamine/asparagine-rich domain. Prion domains are both modular and transferable to other proteins, on which they can confer a heritable epigenetic alteration of function; existing bioinformatics data indicate that they are rare in non-eukarya, but common in eukarya." [GOC:curators, GOC:dph, GOC:tb]
 comment: This term was obsoleted because it is not an active process.
 synonym: "activation of protein activity, epigenetic" NARROW []
 synonym: "positive regulation of protein activity, epigenetic" EXACT [GOC:dph, GOC:tb]
@@ -286793,7 +287424,7 @@
 id: GO:0045859
 name: regulation of protein kinase activity
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of protein kinase activity." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of protein kinase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 is_a: GO:0001932 ! regulation of protein phosphorylation
 is_a: GO:0043549 ! regulation of kinase activity
@@ -286805,7 +287436,7 @@
 id: GO:0045860
 name: positive regulation of protein kinase activity
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of protein kinase activity." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of protein kinase activity." [GOC:curators]
 subset: gocheck_obsoletion_candidate
 synonym: "stimulation of protein kinase activity" NARROW []
 synonym: "up regulation of protein kinase activity" EXACT []
@@ -286822,7 +287453,7 @@
 id: GO:0045861
 name: negative regulation of proteolysis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein." [GOC:curators]
 synonym: "down regulation of proteolysis" EXACT []
 synonym: "down-regulation of proteolysis" EXACT []
 synonym: "downregulation of proteolysis" EXACT []
@@ -286838,7 +287469,7 @@
 id: GO:0045862
 name: positive regulation of proteolysis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein." [GOC:curators]
 synonym: "activation of proteolysis" NARROW []
 synonym: "positive regulation of peptidolysis" EXACT []
 synonym: "stimulation of proteolysis" NARROW []
@@ -286855,7 +287486,7 @@
 id: GO:0045863
 name: obsolete negative regulation of pteridine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "down regulation of pteridine metabolic process" EXACT []
 synonym: "down-regulation of pteridine metabolic process" EXACT []
@@ -286869,7 +287500,7 @@
 id: GO:0045864
 name: obsolete positive regulation of pteridine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving pteridine." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of pteridine metabolic process" NARROW []
 synonym: "positive regulation of pteridine metabolism" EXACT []
@@ -286884,7 +287515,7 @@
 id: GO:0045865
 name: obsolete regulation of recombination within rDNA repeats
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:curators, ISBN:0198506732]
 comment: This term was made obsolete because it describes a substrate-specific process.
 synonym: "regulation of recombination within rDNA repeats" EXACT []
 is_obsolete: true
@@ -286894,7 +287525,7 @@
 id: GO:0045866
 name: obsolete positive regulation of recombination within rDNA repeats
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:go_curators, ISBN:0198506732]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of genetic recombination within the DNA of the genes coding for ribosomal RNA." [GOC:curators, ISBN:0198506732]
 comment: This term was made obsolete because it describes a substrate-specific process.
 synonym: "positive regulation of recombination within rDNA repeats" EXACT []
 is_obsolete: true
@@ -286904,7 +287535,7 @@
 id: GO:0045869
 name: negative regulation of single stranded viral RNA replication via double stranded DNA intermediate
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of single stranded viral RNA replication via double stranded DNA intermediate." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of single stranded viral RNA replication via double stranded DNA intermediate." [GOC:curators]
 synonym: "down regulation of retroviral genome replication" EXACT []
 synonym: "down-regulation of retroviral genome replication" EXACT []
 synonym: "downregulation of retroviral genome replication" EXACT []
@@ -286922,7 +287553,7 @@
 id: GO:0045870
 name: positive regulation of single stranded viral RNA replication via double stranded DNA intermediate
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of retroviral genome replication." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of retroviral genome replication." [GOC:curators]
 synonym: "activation of retroviral genome replication" NARROW []
 synonym: "positive regulation of retroviral genome replication" EXACT [GOC:bf, GOC:jl]
 synonym: "stimulation of retroviral genome replication" NARROW []
@@ -286940,7 +287571,7 @@
 id: GO:0045871
 name: obsolete negative regulation of rhodopsin gene expression
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of rhodopsin gene expression." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of rhodopsin gene expression." [GOC:curators, GOC:dph, GOC:tb]
 comment: This term was obsoleted because it refers to a specific gene product. The gene product should be captured as an annotation extension using 'has input'.
 synonym: "down regulation of rhodopsin gene activity" EXACT []
 synonym: "down-regulation of rhodopsin gene activity" EXACT []
@@ -286954,7 +287585,7 @@
 id: GO:0045872
 name: obsolete positive regulation of rhodopsin gene expression
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of rhodopsin gene expression." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of rhodopsin gene expression." [GOC:curators, GOC:dph, GOC:tb]
 comment: This term was obsoleted because it refers to a specific gene product. The gene product should be captured as an annotation extension using 'has input'.
 synonym: "activation of rhodopsin gene activity" NARROW []
 synonym: "positive regulation of rhodopsin gene activity" EXACT [GOC:dph, GOC:tb]
@@ -286969,7 +287600,7 @@
 id: GO:0045873
 name: negative regulation of sevenless signaling pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the sevenless signaling pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the sevenless signaling pathway." [GOC:curators]
 synonym: "down regulation of sevenless signaling pathway" EXACT []
 synonym: "down-regulation of sevenless signaling pathway" EXACT []
 synonym: "downregulation of sevenless signaling pathway" EXACT []
@@ -286988,7 +287619,7 @@
 id: GO:0045874
 name: positive regulation of sevenless signaling pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the sevenless signaling pathway." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the sevenless signaling pathway." [GOC:curators]
 synonym: "activation of sevenless signaling pathway" NARROW []
 synonym: "positive regulation of sev signaling pathway" EXACT []
 synonym: "positive regulation of sevenless signalling pathway" EXACT []
@@ -287006,7 +287637,7 @@
 id: GO:0045875
 name: negative regulation of sister chromatid cohesion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sister chromatid cohesion." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sister chromatid cohesion." [GOC:curators]
 synonym: "down regulation of sister chromatid cohesion" EXACT []
 synonym: "down-regulation of sister chromatid cohesion" EXACT []
 synonym: "downregulation of sister chromatid cohesion" EXACT []
@@ -287022,7 +287653,7 @@
 id: GO:0045876
 name: positive regulation of sister chromatid cohesion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of sister chromatid cohesion." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of sister chromatid cohesion." [GOC:curators]
 synonym: "activation of sister chromatid cohesion" NARROW []
 synonym: "stimulation of sister chromatid cohesion" NARROW []
 synonym: "up regulation of sister chromatid cohesion" EXACT []
@@ -287040,7 +287671,7 @@
 name: negative regulation of smoothened signaling pathway
 namespace: biological_process
 alt_id: GO:0045877
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of smoothened signaling." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of smoothened signaling." [GOC:curators]
 synonym: "down regulation of smoothened signaling pathway" EXACT []
 synonym: "down-regulation of smoothened signaling pathway" EXACT []
 synonym: "downregulation of smoothened signaling pathway" EXACT []
@@ -287062,7 +287693,7 @@
 name: positive regulation of smoothened signaling pathway
 namespace: biological_process
 alt_id: GO:0045878
-def: "Any process that activates or increases the frequency, rate or extent of smoothened signaling." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of smoothened signaling." [GOC:curators]
 synonym: "activation of smoothened signaling pathway" NARROW []
 synonym: "positive regulation of hedgehog signaling pathway" EXACT [GOC:bf, GOC:ecd]
 synonym: "positive regulation of hh signaling pathway" EXACT [GOC:bf, GOC:ecd]
@@ -287083,7 +287714,7 @@
 id: GO:0045881
 name: positive regulation of sporulation resulting in formation of a cellular spore
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of sporulation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of sporulation." [GOC:curators]
 synonym: "activation of sporulation" NARROW []
 synonym: "stimulation of sporulation" NARROW []
 synonym: "up regulation of sporulation" EXACT []
@@ -287099,7 +287730,7 @@
 id: GO:0045882
 name: negative regulation of sulfur utilization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sulfur utilization." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of sulfur utilization." [GOC:curators]
 synonym: "down regulation of sulfur utilization" EXACT []
 synonym: "down-regulation of sulfur utilization" EXACT []
 synonym: "downregulation of sulfur utilization" EXACT []
@@ -287115,7 +287746,7 @@
 id: GO:0045883
 name: positive regulation of sulfur utilization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of sulfur utilization." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of sulfur utilization." [GOC:curators]
 synonym: "activation of sulfur utilization" NARROW []
 synonym: "positive regulation of sulphur utilization" EXACT []
 synonym: "stimulation of sulfur utilization" NARROW []
@@ -287132,7 +287763,7 @@
 id: GO:0045884
 name: obsolete regulation of survival gene product expression
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:dph, GOC:go_curators, GOC:mtg_apoptosis, GOC:tb]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:curators, GOC:dph, GOC:mtg_apoptosis, GOC:tb]
 comment: This term was made obsolete because the meaning of the term is ambiguous: the same gene product may or may not have a role in apoptosis depending on the cell type, tissue type, condition, etc.
 synonym: "regulation of survival gene product activity" BROAD [GOC:dph, GOC:tb]
 synonym: "regulation of survival gene product expression" EXACT []
@@ -287143,7 +287774,7 @@
 id: GO:0045885
 name: obsolete positive regulation of survival gene product expression
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:dph, GOC:go_curators, GOC:mtg_apoptosis, GOC:tb]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of survival gene product expression; survival gene products are those that antagonize the apoptotic program. Regulation can be at the transcriptional, translational, or posttranslational level." [GOC:curators, GOC:dph, GOC:mtg_apoptosis, GOC:tb]
 comment: This term was made obsolete because the meaning of the term is ambiguous: the same gene product may or may not have a role in apoptosis depending on the cell type, tissue type, condition, etc.
 synonym: "activation of survival gene product activity" BROAD []
 synonym: "positive regulation of survival gene product activity" BROAD [GOC:dph, GOC:tb]
@@ -287159,7 +287790,7 @@
 id: GO:0045886
 name: negative regulation of synaptic assembly at neuromuscular junction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of synaptic assembly at neuromuscular junction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of synaptic assembly at neuromuscular junction." [GOC:curators]
 synonym: "down regulation of synaptic growth at neuromuscular junction" EXACT []
 synonym: "down-regulation of synaptic growth at neuromuscular junction" EXACT []
 synonym: "downregulation of synaptic growth at neuromuscular junction" EXACT []
@@ -287177,7 +287808,7 @@
 id: GO:0045887
 name: positive regulation of synaptic assembly at neuromuscular junction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of synaptic assembly at neuromuscular junction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of synaptic assembly at neuromuscular junction." [GOC:curators]
 synonym: "activation of synaptic growth at neuromuscular junction" NARROW []
 synonym: "positive regulation of synaptic growth at neuromuscular junction" RELATED []
 synonym: "stimulation of synaptic growth at neuromuscular junction" NARROW []
@@ -287196,7 +287827,7 @@
 id: GO:0045888
 name: obsolete regulation of transcription of homeotic gene (Polycomb group)
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "regulation of transcription of homeotic gene (Polycomb group)" EXACT []
 is_obsolete: true
@@ -287206,7 +287837,7 @@
 id: GO:0045889
 name: obsolete positive regulation of transcription of homeotic gene (Polycomb group)
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription of homeotic genes of the Polycomb group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "positive regulation of transcription of homeotic gene (Polycomb group)" EXACT []
 is_obsolete: true
@@ -287216,7 +287847,7 @@
 id: GO:0045890
 name: obsolete regulation of transcription of homeotic gene (trithorax group)
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "regulation of transcription of homeotic gene (trithorax group)" EXACT []
 is_obsolete: true
@@ -287226,7 +287857,7 @@
 id: GO:0045891
 name: obsolete negative regulation of transcription of homeotic gene (trithorax group)
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of transcription of homeotic genes of the trithorax group." [GOC:curators]
 comment: This term was made obsolete because it relates to a specific gene family rather than a process.
 synonym: "negative regulation of transcription of homeotic gene (trithorax group)" EXACT []
 is_obsolete: true
@@ -287239,7 +287870,7 @@
 alt_id: GO:0016481
 alt_id: GO:0032582
 alt_id: GO:0061021
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 synonym: "down regulation of gene-specific transcription" RELATED []
 synonym: "down regulation of transcription, DNA-dependent" EXACT []
 synonym: "down-regulation of gene-specific transcription" RELATED []
@@ -287266,7 +287897,7 @@
 alt_id: GO:0043193
 alt_id: GO:0045941
 alt_id: GO:0061020
-def: "Any process that activates or increases the frequency, rate or extent of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any process that activates or increases the frequency, rate or extent of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 synonym: "activation of gene-specific transcription" RELATED []
 synonym: "activation of transcription, DNA-dependent" NARROW []
 synonym: "positive regulation of cellular transcription, DNA-dependent" EXACT []
@@ -287292,7 +287923,7 @@
 id: GO:0045894
 name: negative regulation of mating-type specific transcription, DNA-templated
 namespace: biological_process
-def: "Any mating-type specific process that stops, prevents or reduces the rate of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any mating-type specific process that stops, prevents or reduces the rate of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 synonym: "down regulation of transcription, mating-type specific" EXACT []
 synonym: "down-regulation of transcription, mating-type specific" EXACT []
 synonym: "downregulation of transcription, mating-type specific" EXACT []
@@ -287305,7 +287936,7 @@
 id: GO:0045895
 name: positive regulation of mating-type specific transcription, DNA-templated
 namespace: biological_process
-def: "Any mating-type specific process that activates or increases the rate of cellular DNA-templated transcription." [GOC:go_curators, GOC:txnOH]
+def: "Any mating-type specific process that activates or increases the rate of cellular DNA-templated transcription." [GOC:curators, GOC:txnOH]
 synonym: "activation of transcription, mating-type specific" NARROW []
 synonym: "positive regulation of mating-type specific transcription, DNA-dependent" EXACT [GOC:txnOH]
 synonym: "stimulation of transcription, mating-type specific" NARROW []
@@ -287319,7 +287950,7 @@
 id: GO:0045896
 name: obsolete regulation of transcription during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "mitotic regulation of transcription" EXACT []
 synonym: "regulation of transcription, mitotic" EXACT [GOC:mah]
@@ -287332,7 +287963,7 @@
 id: GO:0045897
 name: obsolete positive regulation of transcription during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "activation of transcription during mitosis" NARROW [GOC:mah]
 synonym: "mitotic activation of transcription" EXACT []
@@ -287350,7 +287981,7 @@
 id: GO:0045898
 name: regulation of RNA polymerase II transcription preinitiation complex assembly
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:curators]
 synonym: "regulation of RNA polymerase II transcriptional pre-initiation complex assembly" EXACT []
 synonym: "regulation of RNA polymerase II transcriptional pre-initiation complex biosynthesis" EXACT []
 synonym: "regulation of RNA polymerase II transcriptional preinitiation complex assembly" EXACT []
@@ -287365,7 +287996,7 @@
 id: GO:0045899
 name: positive regulation of RNA polymerase II transcription preinitiation complex assembly
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of RNA polymerase II transcriptional preinitiation complex assembly." [GOC:curators]
 synonym: "activation of RNA polymerase II transcriptional preinitiation complex assembly" NARROW []
 synonym: "positive regulation of RNA polymerase II transcriptional pre-initiation complex assembly" EXACT []
 synonym: "positive regulation of RNA polymerase II transcriptional pre-initiation complex biosynthesis" EXACT []
@@ -287386,7 +288017,7 @@
 id: GO:0045900
 name: negative regulation of translational elongation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational elongation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational elongation." [GOC:curators]
 synonym: "down regulation of translational elongation" EXACT []
 synonym: "down-regulation of translational elongation" EXACT []
 synonym: "downregulation of translational elongation" EXACT []
@@ -287401,7 +288032,7 @@
 id: GO:0045901
 name: positive regulation of translational elongation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of translational elongation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of translational elongation." [GOC:curators]
 synonym: "activation of translational elongation" NARROW []
 synonym: "stimulation of translational elongation" NARROW []
 synonym: "up regulation of translational elongation" EXACT []
@@ -287442,7 +288073,7 @@
 id: GO:0045904
 name: negative regulation of translational termination
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational termination." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational termination." [GOC:curators]
 synonym: "down regulation of translational termination" EXACT []
 synonym: "down-regulation of translational termination" EXACT []
 synonym: "downregulation of translational termination" EXACT []
@@ -287458,7 +288089,7 @@
 id: GO:0045905
 name: positive regulation of translational termination
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of translational termination." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of translational termination." [GOC:curators]
 synonym: "activation of translational termination" NARROW []
 synonym: "stimulation of translational termination" NARROW []
 synonym: "up regulation of translational termination" EXACT []
@@ -287475,7 +288106,7 @@
 id: GO:0045906
 name: negative regulation of vasoconstriction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of vasoconstriction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of vasoconstriction." [GOC:curators]
 synonym: "down regulation of vasoconstriction" EXACT []
 synonym: "down-regulation of vasoconstriction" EXACT []
 synonym: "downregulation of vasoconstriction" EXACT []
@@ -287490,7 +288121,7 @@
 id: GO:0045907
 name: positive regulation of vasoconstriction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of vasoconstriction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of vasoconstriction." [GOC:curators]
 synonym: "activation of vasoconstriction" NARROW []
 synonym: "stimulation of vasoconstriction" NARROW []
 synonym: "up regulation of vasoconstriction" EXACT []
@@ -287506,7 +288137,7 @@
 id: GO:0045910
 name: negative regulation of DNA recombination
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA recombination." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of DNA recombination." [GOC:curators]
 synonym: "down regulation of DNA recombination" EXACT []
 synonym: "down-regulation of DNA recombination" EXACT []
 synonym: "downregulation of DNA recombination" EXACT []
@@ -287521,7 +288152,7 @@
 id: GO:0045911
 name: positive regulation of DNA recombination
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of DNA recombination." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of DNA recombination." [GOC:curators]
 synonym: "activation of DNA recombination" NARROW []
 synonym: "stimulation of DNA recombination" NARROW []
 synonym: "up regulation of DNA recombination" EXACT []
@@ -287537,7 +288168,7 @@
 id: GO:0045912
 name: negative regulation of carbohydrate metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving carbohydrate." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving carbohydrate." [GOC:curators]
 synonym: "down regulation of carbohydrate metabolic process" EXACT []
 synonym: "down-regulation of carbohydrate metabolic process" EXACT []
 synonym: "downregulation of carbohydrate metabolic process" EXACT []
@@ -287553,7 +288184,7 @@
 id: GO:0045913
 name: positive regulation of carbohydrate metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving carbohydrate." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving carbohydrate." [GOC:curators]
 synonym: "activation of carbohydrate metabolic process" NARROW []
 synonym: "positive regulation of carbohydrate metabolism" EXACT []
 synonym: "stimulation of carbohydrate metabolic process" NARROW []
@@ -287570,7 +288201,7 @@
 id: GO:0045914
 name: obsolete negative regulation of catecholamine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving catecholamine." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving catecholamine." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "down regulation of catecholamine metabolic process" EXACT []
 synonym: "down-regulation of catecholamine metabolic process" EXACT []
@@ -287584,7 +288215,7 @@
 id: GO:0045915
 name: obsolete positive regulation of catecholamine metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving catecholamine." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving catecholamine." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of catecholamine metabolic process" NARROW []
 synonym: "positive regulation of catecholamine metabolism" EXACT []
@@ -287599,7 +288230,7 @@
 id: GO:0045916
 name: negative regulation of complement activation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation." [GOC:curators]
 synonym: "down regulation of complement activation" EXACT []
 synonym: "down-regulation of complement activation" EXACT []
 synonym: "downregulation of complement activation" EXACT []
@@ -287616,7 +288247,7 @@
 id: GO:0045917
 name: positive regulation of complement activation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of complement activation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of complement activation." [GOC:curators]
 synonym: "activation of complement activation" NARROW []
 synonym: "positive regulation of complement cascade" EXACT [GOC:add]
 synonym: "stimulation of complement activation" NARROW []
@@ -287634,7 +288265,7 @@
 id: GO:0045918
 name: negative regulation of cytolysis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cytolysis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of cytolysis." [GOC:curators]
 synonym: "down regulation of cytolysis" EXACT []
 synonym: "down-regulation of cytolysis" EXACT []
 synonym: "downregulation of cytolysis" EXACT []
@@ -287649,7 +288280,7 @@
 id: GO:0045919
 name: positive regulation of cytolysis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of cytolysis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of cytolysis." [GOC:curators]
 synonym: "activation of cytolysis" NARROW []
 synonym: "stimulation of cytolysis" NARROW []
 synonym: "up regulation of cytolysis" EXACT []
@@ -287665,7 +288296,7 @@
 id: GO:0045920
 name: negative regulation of exocytosis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of exocytosis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of exocytosis." [GOC:curators]
 synonym: "down regulation of exocytosis" EXACT []
 synonym: "down-regulation of exocytosis" EXACT []
 synonym: "downregulation of exocytosis" EXACT []
@@ -287680,7 +288311,7 @@
 id: GO:0045921
 name: positive regulation of exocytosis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of exocytosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of exocytosis." [GOC:curators]
 synonym: "activation of exocytosis" NARROW []
 synonym: "stimulation of exocytosis" NARROW []
 synonym: "up regulation of exocytosis" EXACT []
@@ -287696,7 +288327,7 @@
 id: GO:0045922
 name: negative regulation of fatty acid metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:curators]
 synonym: "down regulation of fatty acid metabolic process" EXACT []
 synonym: "down-regulation of fatty acid metabolic process" EXACT []
 synonym: "downregulation of fatty acid metabolic process" EXACT []
@@ -287713,7 +288344,7 @@
 id: GO:0045923
 name: positive regulation of fatty acid metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving fatty acids." [GOC:curators]
 synonym: "activation of fatty acid metabolic process" NARROW []
 synonym: "positive regulation of fatty acid metabolism" EXACT []
 synonym: "stimulation of fatty acid metabolic process" NARROW []
@@ -287732,7 +288363,7 @@
 name: regulation of female receptivity
 namespace: biological_process
 alt_id: GO:0060181
-def: "Any process that modulates the frequency, rate or extent of the willingness or readiness of a female to receive male advances." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of the willingness or readiness of a female to receive male advances." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "female receptivity" RELATED [GOC:dph, GOC:tb]
 is_a: GO:0060180 ! female mating behavior
 is_a: GO:0065008 ! regulation of biological quality
@@ -287741,7 +288372,7 @@
 id: GO:0045925
 name: positive regulation of female receptivity
 namespace: biological_process
-def: "Any process that activates or increases the receptiveness of a female to male advances." [GOC:go_curators]
+def: "Any process that activates or increases the receptiveness of a female to male advances." [GOC:curators]
 synonym: "activation of female receptivity" NARROW []
 synonym: "stimulation of female receptivity" NARROW []
 synonym: "up regulation of female receptivity" EXACT []
@@ -287753,7 +288384,7 @@
 id: GO:0045926
 name: negative regulation of growth
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of growth, the increase in size or mass of all or part of an organism." [GOC:go_curators]
+def: "Any process that stops, prevents or reduces the rate or extent of growth, the increase in size or mass of all or part of an organism." [GOC:curators]
 synonym: "down regulation of growth" EXACT []
 synonym: "down-regulation of growth" EXACT []
 synonym: "downregulation of growth" EXACT []
@@ -287768,7 +288399,7 @@
 id: GO:0045927
 name: positive regulation of growth
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of growth, the increase in size or mass of all or part of an organism." [GOC:go_curators]
+def: "Any process that activates or increases the rate or extent of growth, the increase in size or mass of all or part of an organism." [GOC:curators]
 synonym: "activation of growth" NARROW []
 synonym: "stimulation of growth" NARROW []
 synonym: "up regulation of growth" EXACT []
@@ -287784,7 +288415,7 @@
 id: GO:0045928
 name: negative regulation of juvenile hormone metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:curators]
 synonym: "down regulation of juvenile hormone metabolic process" EXACT []
 synonym: "down-regulation of juvenile hormone metabolic process" EXACT []
 synonym: "downregulation of juvenile hormone metabolic process" EXACT []
@@ -287800,7 +288431,7 @@
 id: GO:0045929
 name: obsolete positive regulation of juvenile hormone metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving juvenile hormone." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of juvenile hormone metabolic process" NARROW []
 synonym: "positive regulation of juvenile hormone metabolism" EXACT []
@@ -287815,7 +288446,7 @@
 id: GO:0045930
 name: negative regulation of mitotic cell cycle
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of progression through the mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of progression through the mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of progression through mitotic cell cycle" EXACT []
 synonym: "down-regulation of progression through mitotic cell cycle" EXACT []
 synonym: "downregulation of progression through mitotic cell cycle" EXACT []
@@ -287833,7 +288464,7 @@
 id: GO:0045931
 name: positive regulation of mitotic cell cycle
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of progression through the mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the rate or extent of progression through the mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of progression through mitotic cell cycle" NARROW []
 synonym: "positive regulation of mitotic cell cycle progression" EXACT []
 synonym: "positive regulation of progression through mitotic cell cycle" EXACT [GOC:dph, GOC:tb]
@@ -287851,7 +288482,7 @@
 id: GO:0045932
 name: negative regulation of muscle contraction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of muscle contraction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of muscle contraction." [GOC:curators]
 synonym: "down regulation of muscle contraction" EXACT []
 synonym: "down-regulation of muscle contraction" EXACT []
 synonym: "downregulation of muscle contraction" EXACT []
@@ -287866,7 +288497,7 @@
 id: GO:0045933
 name: positive regulation of muscle contraction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of muscle contraction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of muscle contraction." [GOC:curators]
 synonym: "activation of muscle contraction" NARROW []
 synonym: "stimulation of muscle contraction" NARROW []
 synonym: "up regulation of muscle contraction" EXACT []
@@ -287882,7 +288513,8 @@
 id: GO:0045934
 name: negative regulation of nucleobase-containing compound metabolic process
 namespace: biological_process
-def: "Any cellular process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:go_curators]
+def: "Any cellular process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:curators]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
 synonym: "down-regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
 synonym: "downregulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" EXACT []
@@ -287894,12 +288526,13 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0006139 ! nucleobase-containing compound metabolic process
 relationship: negatively_regulates GO:0006139 ! nucleobase-containing compound metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 
 [Term]
 id: GO:0045935
 name: obsolete positive regulation of nucleobase-containing compound metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any cellular process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:go_curators]
+def: "OBSOLETE. Any cellular process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleobases, nucleosides, nucleotides and nucleic acids." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" NARROW []
 synonym: "positive regulation of nucleobase, nucleoside, nucleotide and nucleic acid metabolic process" RELATED [GOC:dph, GOC:tb]
@@ -287915,7 +288548,7 @@
 id: GO:0045936
 name: negative regulation of phosphate metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:curators]
 synonym: "down regulation of phosphate metabolic process" EXACT []
 synonym: "down-regulation of phosphate metabolic process" EXACT []
 synonym: "downregulation of phosphate metabolic process" EXACT []
@@ -287930,7 +288563,7 @@
 id: GO:0045937
 name: positive regulation of phosphate metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving phosphates." [GOC:curators]
 synonym: "activation of phosphate metabolic process" NARROW []
 synonym: "positive regulation of phosphate metabolism" EXACT []
 synonym: "stimulation of phosphate metabolic process" NARROW []
@@ -287946,7 +288579,7 @@
 id: GO:0045938
 name: positive regulation of circadian sleep/wake cycle, sleep
 namespace: biological_process
-def: "Any process that activates or increases the duration or quality of sleep, a readily reversible state of reduced awareness and metabolic activity that occurs periodically in many animals." [GOC:go_curators]
+def: "Any process that activates or increases the duration or quality of sleep, a readily reversible state of reduced awareness and metabolic activity that occurs periodically in many animals." [GOC:curators]
 synonym: "activation of circadian sleep/wake cycle, sleep" NARROW []
 synonym: "positive regulation of sleep" EXACT []
 synonym: "stimulation of circadian sleep/wake cycle, sleep" NARROW []
@@ -287964,7 +288597,7 @@
 id: GO:0045939
 name: negative regulation of steroid metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:curators]
 synonym: "down regulation of steroid metabolic process" EXACT []
 synonym: "down-regulation of steroid metabolic process" EXACT []
 synonym: "downregulation of steroid metabolic process" EXACT []
@@ -287980,7 +288613,7 @@
 id: GO:0045940
 name: positive regulation of steroid metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving steroids." [GOC:curators]
 synonym: "activation of steroid metabolic process" NARROW []
 synonym: "positive regulation of steroid metabolism" EXACT []
 synonym: "stimulation of steroid metabolic process" NARROW []
@@ -287997,7 +288630,7 @@
 id: GO:0045942
 name: negative regulation of phosphorus utilization
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of phosphorus utilization." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of phosphorus utilization." [GOC:curators]
 synonym: "down regulation of phosphorus utilization" EXACT []
 synonym: "down-regulation of phosphorus utilization" EXACT []
 synonym: "downregulation of phosphorus utilization" EXACT []
@@ -288012,7 +288645,7 @@
 id: GO:0045943
 name: positive regulation of transcription by RNA polymerase I
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:go_curators, GOC:txnOH]
+def: "Any process that activates or increases the frequency, rate or extent of transcription mediated by RNA polymerase I." [GOC:curators, GOC:txnOH]
 synonym: "activation of transcription from RNA polymerase I promoter" NARROW []
 synonym: "positive regulation of transcription from Pol I promoter" EXACT []
 synonym: "positive regulation of transcription from RNA polymerase I promoter" EXACT []
@@ -288032,7 +288665,7 @@
 namespace: biological_process
 alt_id: GO:0010552
 alt_id: GO:0045817
-def: "Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:go_curators, GOC:txnOH]
+def: "Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:curators, GOC:txnOH]
 synonym: "activation of global transcription from RNA polymerase II promoter" RELATED []
 synonym: "activation of transcription from RNA polymerase II promoter" EXACT []
 synonym: "positive regulation of gene-specific transcription from RNA polymerase II promoter" RELATED []
@@ -288061,7 +288694,7 @@
 id: GO:0045945
 name: positive regulation of transcription by RNA polymerase III
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of transcription mediated by RNA polymerase III." [GOC:go_curators, GOC:txnOH]
+def: "Any process that activates or increases the frequency, rate or extent of transcription mediated by RNA polymerase III." [GOC:curators, GOC:txnOH]
 synonym: "activation of transcription from RNA polymerase III promoter" NARROW []
 synonym: "positive regulation of transcription from Pol III promoter" EXACT []
 synonym: "positive regulation of transcription from RNA polymerase III promoter" EXACT []
@@ -288079,7 +288712,7 @@
 id: GO:0045947
 name: negative regulation of translational initiation
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational initiation." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of translational initiation." [GOC:curators]
 synonym: "down regulation of translational initiation" EXACT []
 synonym: "down-regulation of translational initiation" EXACT []
 synonym: "downregulation of translational initiation" EXACT []
@@ -288094,7 +288727,7 @@
 id: GO:0045948
 name: positive regulation of translational initiation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of translational initiation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of translational initiation." [GOC:curators]
 synonym: "activation of translational initiation" NARROW []
 synonym: "stimulation of translational initiation" NARROW []
 synonym: "up regulation of translational initiation" EXACT []
@@ -288110,7 +288743,7 @@
 id: GO:0045949
 name: positive regulation of phosphorus utilization
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of phosphorus utilization." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of phosphorus utilization." [GOC:curators]
 synonym: "activation of phosphorus utilization" NARROW []
 synonym: "stimulation of phosphorus utilization" NARROW []
 synonym: "up regulation of phosphorus utilization" EXACT []
@@ -288126,7 +288759,7 @@
 id: GO:0045950
 name: negative regulation of mitotic recombination
 namespace: biological_process
-def: "Any process that inhibits or decreases the rate of DNA recombination during mitosis." [GOC:go_curators, GOC:hjd]
+def: "Any process that inhibits or decreases the rate of DNA recombination during mitosis." [GOC:curators, GOC:hjd]
 synonym: "down regulation of mitotic recombination" EXACT []
 synonym: "down-regulation of mitotic recombination" EXACT []
 synonym: "downregulation of mitotic recombination" EXACT []
@@ -288142,7 +288775,7 @@
 id: GO:0045951
 name: positive regulation of mitotic recombination
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of DNA recombination during mitosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of DNA recombination during mitosis." [GOC:curators]
 synonym: "activation of mitotic recombination" NARROW []
 synonym: "positive regulation of recombination within rDNA repeats" NARROW []
 synonym: "stimulation of mitotic recombination" NARROW []
@@ -288159,7 +288792,7 @@
 id: GO:0045952
 name: regulation of juvenile hormone catabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:curators]
 synonym: "regulation of juvenile hormone breakdown" EXACT []
 synonym: "regulation of juvenile hormone catabolism" EXACT []
 synonym: "regulation of juvenile hormone degradation" EXACT []
@@ -288221,7 +288854,7 @@
 id: GO:0045955
 name: negative regulation of calcium ion-dependent exocytosis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:curators]
 synonym: "down regulation of calcium ion-dependent exocytosis" EXACT []
 synonym: "down-regulation of calcium ion-dependent exocytosis" EXACT []
 synonym: "downregulation of calcium ion-dependent exocytosis" EXACT []
@@ -288236,7 +288869,7 @@
 id: GO:0045956
 name: positive regulation of calcium ion-dependent exocytosis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of calcium ion-dependent exocytosis." [GOC:curators]
 synonym: "activation of calcium ion-dependent exocytosis" NARROW []
 synonym: "stimulation of calcium ion-dependent exocytosis" NARROW []
 synonym: "up regulation of calcium ion-dependent exocytosis" EXACT []
@@ -288252,7 +288885,7 @@
 id: GO:0045957
 name: negative regulation of complement activation, alternative pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation by the alternative pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation by the alternative pathway." [GOC:curators]
 synonym: "down regulation of complement activation, alternative pathway" EXACT []
 synonym: "down-regulation of complement activation, alternative pathway" EXACT []
 synonym: "downregulation of complement activation, alternative pathway" EXACT []
@@ -288269,7 +288902,7 @@
 id: GO:0045958
 name: positive regulation of complement activation, alternative pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of complement activation by the alternative pathway." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of complement activation by the alternative pathway." [GOC:curators]
 synonym: "activation of complement activation, alternative pathway" NARROW []
 synonym: "positive regulation of complement cascade, alternative pathway" EXACT [GOC:add]
 synonym: "stimulation of complement activation, alternative pathway" NARROW []
@@ -288287,7 +288920,7 @@
 id: GO:0045959
 name: negative regulation of complement activation, classical pathway
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation by the classical pathway." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of complement activation by the classical pathway." [GOC:curators]
 synonym: "down regulation of complement activation, classical pathway" EXACT []
 synonym: "down-regulation of complement activation, classical pathway" EXACT []
 synonym: "downregulation of complement activation, classical pathway" EXACT []
@@ -288304,7 +288937,7 @@
 id: GO:0045960
 name: positive regulation of complement activation, classical pathway
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of complement activation by the classical pathway." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of complement activation by the classical pathway." [GOC:curators]
 synonym: "activation of complement activation, classical pathway" NARROW []
 synonym: "positive regulation of complement cascade, classical pathway" EXACT [GOC:add]
 synonym: "stimulation of complement activation, classical pathway" NARROW []
@@ -288322,7 +288955,7 @@
 id: GO:0045961
 name: negative regulation of development, heterochronic
 namespace: biological_process
-def: "Any process that modulates the consistent predetermined time point at which an integrated living unit or organism progresses from an initial condition to a later condition and decreases the rate at which this time point is reached." [GOC:go_curators]
+def: "Any process that modulates the consistent predetermined time point at which an integrated living unit or organism progresses from an initial condition to a later condition and decreases the rate at which this time point is reached." [GOC:curators]
 synonym: "down regulation of development, heterochronic" EXACT []
 synonym: "down-regulation of development, heterochronic" EXACT []
 synonym: "downregulation of development, heterochronic" EXACT []
@@ -288333,7 +288966,7 @@
 id: GO:0045962
 name: positive regulation of development, heterochronic
 namespace: biological_process
-def: "Any process that modulates the consistent predetermined time point at which an integrated living unit or organism progresses from an initial condition to a later condition and increases the rate at which this time point is reached." [GOC:go_curators]
+def: "Any process that modulates the consistent predetermined time point at which an integrated living unit or organism progresses from an initial condition to a later condition and increases the rate at which this time point is reached." [GOC:curators]
 synonym: "activation of development, heterochronic" NARROW []
 synonym: "stimulation of development, heterochronic" NARROW []
 synonym: "up regulation of development, heterochronic" EXACT []
@@ -288345,7 +288978,7 @@
 id: GO:0045963
 name: negative regulation of dopamine metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:curators]
 synonym: "down regulation of dopamine metabolic process" EXACT []
 synonym: "down-regulation of dopamine metabolic process" EXACT []
 synonym: "downregulation of dopamine metabolic process" EXACT []
@@ -288361,7 +288994,7 @@
 id: GO:0045964
 name: positive regulation of dopamine metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving dopamine." [GOC:curators]
 synonym: "activation of dopamine metabolic process" NARROW []
 synonym: "positive regulation of dopamine metabolism" EXACT []
 synonym: "stimulation of dopamine metabolic process" NARROW []
@@ -288378,7 +289011,7 @@
 id: GO:0045965
 name: obsolete negative regulation of ecdysteroid metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving ecdysteroids." [GOC:go_curators]
+def: "OBSOLETE. Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving ecdysteroids." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "down regulation of ecdysteroid metabolic process" EXACT []
 synonym: "down-regulation of ecdysteroid metabolic process" EXACT []
@@ -288392,7 +289025,7 @@
 id: GO:0045966
 name: obsolete positive regulation of ecdysteroid metabolic process
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving ecdysteroids." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving ecdysteroids." [GOC:curators]
 comment: This term was obsoleted because it is an unnecessary grouping class.
 synonym: "activation of ecdysteroid metabolic process" NARROW []
 synonym: "positive regulation of ecdysteroid metabolism" EXACT []
@@ -288420,7 +289053,7 @@
 id: GO:0045968
 name: negative regulation of juvenile hormone biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:curators]
 synonym: "down regulation of juvenile hormone biosynthetic process" EXACT []
 synonym: "down-regulation of juvenile hormone biosynthetic process" EXACT []
 synonym: "downregulation of juvenile hormone biosynthetic process" EXACT []
@@ -288441,7 +289074,7 @@
 id: GO:0045969
 name: positive regulation of juvenile hormone biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of juvenile hormone." [GOC:curators]
 synonym: "activation of juvenile hormone biosynthetic process" NARROW []
 synonym: "positive regulation of juvenile hormone anabolism" EXACT []
 synonym: "positive regulation of juvenile hormone biosynthesis" EXACT []
@@ -288462,7 +289095,7 @@
 id: GO:0045970
 name: negative regulation of juvenile hormone catabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:curators]
 synonym: "down regulation of juvenile hormone catabolic process" EXACT []
 synonym: "down-regulation of juvenile hormone catabolic process" EXACT []
 synonym: "downregulation of juvenile hormone catabolic process" EXACT []
@@ -288481,7 +289114,7 @@
 id: GO:0045971
 name: positive regulation of juvenile hormone catabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of juvenile hormone." [GOC:curators]
 synonym: "activation of juvenile hormone catabolic process" NARROW []
 synonym: "positive regulation of juvenile hormone breakdown" EXACT []
 synonym: "positive regulation of juvenile hormone catabolism" EXACT []
@@ -288501,7 +289134,7 @@
 id: GO:0045972
 name: negative regulation of juvenile hormone secretion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the regulated release of juvenile hormone." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the regulated release of juvenile hormone." [GOC:curators]
 synonym: "down regulation of juvenile hormone secretion" EXACT []
 synonym: "down-regulation of juvenile hormone secretion" EXACT []
 synonym: "downregulation of juvenile hormone secretion" EXACT []
@@ -288518,7 +289151,7 @@
 id: GO:0045973
 name: positive regulation of juvenile hormone secretion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the regulated release of juvenile hormone." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the regulated release of juvenile hormone." [GOC:curators]
 synonym: "activation of juvenile hormone secretion" NARROW []
 synonym: "stimulation of juvenile hormone secretion" NARROW []
 synonym: "up regulation of juvenile hormone secretion" EXACT []
@@ -288535,14 +289168,14 @@
 id: GO:0045974
 name: regulation of translation, ncRNA-mediated
 namespace: biological_process
-def: "Any process, mediated by small non-coding RNAs, that modulates the frequency, rate or extent that mRNAs are effectively translated into protein." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process, mediated by small non-coding RNAs, that modulates the frequency, rate or extent that mRNAs are effectively translated into protein." [GOC:curators, GOC:dph, GOC:tb]
 is_a: GO:0006417 ! regulation of translation
 
 [Term]
 id: GO:0045975
 name: positive regulation of translation, ncRNA-mediated
 namespace: biological_process
-def: "Any process, mediated by small non-coding RNAs, that activates or increases the rate that mRNAs are effectively translated into protein." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process, mediated by small non-coding RNAs, that activates or increases the rate that mRNAs are effectively translated into protein." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of mRNA translation, ncRNA-mediated" NARROW []
 synonym: "stimulation of mRNA translation, ncRNA-mediated" NARROW []
 synonym: "up regulation of mRNA translation, ncRNA-mediated" EXACT []
@@ -288555,7 +289188,7 @@
 id: GO:0045976
 name: negative regulation of mitotic cell cycle, embryonic
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of progression through the embryonic mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of progression through the embryonic mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of progression through embryonic mitotic cell cycle" EXACT []
 synonym: "down-regulation of progression through embryonic mitotic cell cycle" EXACT []
 synonym: "downregulation of progression through embryonic mitotic cell cycle" EXACT []
@@ -288573,7 +289206,7 @@
 id: GO:0045977
 name: positive regulation of mitotic cell cycle, embryonic
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of progression through the embryonic mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of progression through the embryonic mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of mitotic cell cycle, embryonic" NARROW []
 synonym: "positive regulation of embryonic mitotic cell cycle" EXACT [GOC:dph, GOC:tb]
 synonym: "positive regulation of embryonic mitotic cell cycle progression" EXACT []
@@ -288592,7 +289225,7 @@
 id: GO:0045978
 name: negative regulation of nucleoside metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:curators]
 synonym: "down regulation of nucleoside metabolic process" EXACT []
 synonym: "down-regulation of nucleoside metabolic process" EXACT []
 synonym: "downregulation of nucleoside metabolic process" EXACT []
@@ -288609,7 +289242,7 @@
 id: GO:0045979
 name: positive regulation of nucleoside metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleosides." [GOC:curators]
 synonym: "activation of nucleoside metabolic process" NARROW []
 synonym: "positive regulation of nucleoside metabolism" EXACT []
 synonym: "stimulation of nucleoside metabolic process" NARROW []
@@ -288626,7 +289259,7 @@
 id: GO:0045980
 name: negative regulation of nucleotide metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:curators]
 synonym: "down regulation of nucleotide metabolic process" EXACT []
 synonym: "down-regulation of nucleotide metabolic process" EXACT []
 synonym: "downregulation of nucleotide metabolic process" EXACT []
@@ -288644,7 +289277,7 @@
 id: GO:0045981
 name: positive regulation of nucleotide metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving nucleotides." [GOC:curators]
 synonym: "activation of nucleotide metabolic process" NARROW []
 synonym: "positive regulation of nucleotide metabolism" EXACT []
 synonym: "stimulation of nucleotide metabolic process" NARROW []
@@ -288662,12 +289295,12 @@
 id: GO:0045982
 name: negative regulation of purine nucleobase metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving purine nucleobases." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving purine nucleobases." [GOC:curators]
 synonym: "down regulation of purine base metabolic process" EXACT []
 synonym: "down-regulation of purine base metabolic process" EXACT []
 synonym: "downregulation of purine base metabolic process" EXACT []
 synonym: "inhibition of purine base metabolic process" NARROW []
-synonym: "negative regulation of purine base metabolic process" EXACT [GOC:go_curators]
+synonym: "negative regulation of purine base metabolic process" EXACT [GOC:curators]
 synonym: "negative regulation of purine base metabolism" EXACT []
 is_a: GO:0006141 ! regulation of purine nucleobase metabolic process
 is_a: GO:0045934 ! negative regulation of nucleobase-containing compound metabolic process
@@ -288680,7 +289313,7 @@
 id: GO:0045983
 name: positive regulation of purine nucleobase metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving purine bases." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving purine bases." [GOC:curators]
 synonym: "activation of purine base metabolic process" NARROW []
 synonym: "positive regulation of purine base metabolic process" EXACT []
 synonym: "positive regulation of purine base metabolism" EXACT []
@@ -288698,12 +289331,12 @@
 id: GO:0045984
 name: negative regulation of pyrimidine nucleobase metabolic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:curators]
 synonym: "down regulation of pyrimidine base metabolic process" EXACT []
 synonym: "down-regulation of pyrimidine base metabolic process" EXACT []
 synonym: "downregulation of pyrimidine base metabolic process" EXACT []
 synonym: "inhibition of pyrimidine base metabolic process" NARROW []
-synonym: "negative regulation of pyrimidine base metabolic process" EXACT [GOC:go_curators]
+synonym: "negative regulation of pyrimidine base metabolic process" EXACT [GOC:curators]
 synonym: "negative regulation of pyrimidine base metabolism" EXACT []
 is_a: GO:0045934 ! negative regulation of nucleobase-containing compound metabolic process
 is_a: GO:0062014 ! negative regulation of small molecule metabolic process
@@ -288715,9 +289348,9 @@
 id: GO:0045985
 name: positive regulation of pyrimidine nucleobase metabolic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving pyrimidine nucleobases." [GOC:curators]
 synonym: "activation of pyrimidine base metabolic process" NARROW []
-synonym: "positive regulation of pyrimidine base metabolic process" EXACT [GOC:go_curators]
+synonym: "positive regulation of pyrimidine base metabolic process" EXACT [GOC:curators]
 synonym: "positive regulation of pyrimidine base metabolism" EXACT []
 synonym: "stimulation of pyrimidine base metabolic process" NARROW []
 synonym: "up regulation of pyrimidine base metabolic process" EXACT []
@@ -288733,7 +289366,7 @@
 id: GO:0045986
 name: negative regulation of smooth muscle contraction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of smooth muscle contraction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of smooth muscle contraction." [GOC:curators]
 synonym: "down regulation of smooth muscle contraction" EXACT []
 synonym: "down-regulation of smooth muscle contraction" EXACT []
 synonym: "downregulation of smooth muscle contraction" EXACT []
@@ -288749,7 +289382,7 @@
 id: GO:0045987
 name: positive regulation of smooth muscle contraction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of smooth muscle contraction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of smooth muscle contraction." [GOC:curators]
 synonym: "activation of smooth muscle contraction" NARROW []
 synonym: "stimulation of smooth muscle contraction" NARROW []
 synonym: "up regulation of smooth muscle contraction" EXACT []
@@ -288765,7 +289398,7 @@
 id: GO:0045988
 name: negative regulation of striated muscle contraction
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of striated muscle contraction." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of striated muscle contraction." [GOC:curators]
 synonym: "down regulation of striated muscle contraction" EXACT []
 synonym: "down-regulation of striated muscle contraction" EXACT []
 synonym: "downregulation of striated muscle contraction" EXACT []
@@ -288780,7 +289413,7 @@
 id: GO:0045989
 name: positive regulation of striated muscle contraction
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of striated muscle contraction." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of striated muscle contraction." [GOC:curators]
 synonym: "activation of striated muscle contraction" NARROW []
 synonym: "stimulation of striated muscle contraction" NARROW []
 synonym: "up regulation of striated muscle contraction" EXACT []
@@ -288796,7 +289429,7 @@
 id: GO:0045990
 name: carbon catabolite regulation of transcription
 namespace: biological_process
-def: "A transcription regulation process in which the presence of one carbon source leads to the modulation of the frequency, rate, or extent of transcription of specific genes involved in the metabolism of other carbon sources." [GOC:go_curators, GOC:mah, PMID:18359269, PMID:9618445]
+def: "A transcription regulation process in which the presence of one carbon source leads to the modulation of the frequency, rate, or extent of transcription of specific genes involved in the metabolism of other carbon sources." [GOC:curators, GOC:mah, PMID:18359269, PMID:9618445]
 synonym: "regulation of transcription by carbon catabolites" EXACT [GOC:mah]
 is_a: GO:0006355 ! regulation of DNA-templated transcription
 is_a: GO:0031670 ! cellular response to nutrient
@@ -288814,7 +289447,7 @@
 id: GO:0045992
 name: negative regulation of embryonic development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryonic development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of embryonic development." [GOC:curators]
 synonym: "down regulation of embryonic development" EXACT []
 synonym: "down-regulation of embryonic development" EXACT []
 synonym: "downregulation of embryonic development" EXACT []
@@ -288830,7 +289463,7 @@
 id: GO:0045993
 name: negative regulation of translational initiation by iron
 namespace: biological_process
-def: "Any process involving iron that stops, prevents or reduces the rate of translational initiation." [GOC:go_curators]
+def: "Any process involving iron that stops, prevents or reduces the rate of translational initiation." [GOC:curators]
 synonym: "down regulation of translational initiation by iron" EXACT []
 synonym: "down-regulation of translational initiation by iron" EXACT []
 synonym: "downregulation of translational initiation by iron" EXACT []
@@ -288842,7 +289475,7 @@
 id: GO:0045994
 name: positive regulation of translational initiation by iron
 namespace: biological_process
-def: "Any process involving iron that activates or increases the rate of translational initiation." [GOC:go_curators]
+def: "Any process involving iron that activates or increases the rate of translational initiation." [GOC:curators]
 synonym: "activation of translational initiation by iron" NARROW []
 synonym: "stimulation of translational initiation by iron" NARROW []
 synonym: "up regulation of translational initiation by iron" EXACT []
@@ -288855,7 +289488,7 @@
 id: GO:0045995
 name: regulation of embryonic development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of embryonic development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of embryonic development." [GOC:curators]
 is_a: GO:2000026 ! regulation of multicellular organismal development
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0009790 ! embryo development
@@ -288865,7 +289498,7 @@
 id: GO:0045996
 name: obsolete negative regulation of transcription by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that stops, prevents or reduces the rate of transcription." [GOC:go_curators]
+def: "OBSOLETE. Any process involving pheromones that stops, prevents or reduces the rate of transcription." [GOC:curators]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 synonym: "down regulation of transcription by pheromones" EXACT []
 synonym: "down-regulation of transcription by pheromones" EXACT []
@@ -288878,7 +289511,7 @@
 id: GO:0045997
 name: negative regulation of ecdysteroid biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:curators]
 synonym: "down regulation of ecdysteroid biosynthetic process" EXACT []
 synonym: "down-regulation of ecdysteroid biosynthetic process" EXACT []
 synonym: "downregulation of ecdysteroid biosynthetic process" EXACT []
@@ -288898,7 +289531,7 @@
 id: GO:0045998
 name: positive regulation of ecdysteroid biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of ecdysteroids." [GOC:curators]
 synonym: "activation of ecdysteroid biosynthetic process" NARROW []
 synonym: "positive regulation of ecdysteroid anabolism" EXACT []
 synonym: "positive regulation of ecdysteroid biosynthesis" EXACT []
@@ -288919,7 +289552,7 @@
 id: GO:0045999
 name: negative regulation of ecdysteroid secretion
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the regulated release of ecdysteroid." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of the regulated release of ecdysteroid." [GOC:curators]
 synonym: "down regulation of ecdysteroid secretion" EXACT []
 synonym: "down-regulation of ecdysteroid secretion" EXACT []
 synonym: "downregulation of ecdysteroid secretion" EXACT []
@@ -288934,7 +289567,7 @@
 id: GO:0046000
 name: positive regulation of ecdysteroid secretion
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of the regulated release of ecdysteroid." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of the regulated release of ecdysteroid." [GOC:curators]
 synonym: "activation of ecdysteroid secretion" NARROW []
 synonym: "stimulation of ecdysteroid secretion" NARROW []
 synonym: "up regulation of ecdysteroid secretion" EXACT []
@@ -288950,7 +289583,7 @@
 id: GO:0046001
 name: negative regulation of preblastoderm mitotic cell cycle
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of progression through the preblastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of progression through the preblastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of progression through preblastoderm mitotic cell cycle" EXACT []
 synonym: "down-regulation of progression through preblastoderm mitotic cell cycle" EXACT []
 synonym: "downregulation of progression through preblastoderm mitotic cell cycle" EXACT []
@@ -288967,7 +289600,7 @@
 id: GO:0046002
 name: positive regulation of preblastoderm mitotic cell cycle
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of progression through the preblastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the rate or extent of progression through the preblastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of progression through preblastoderm mitotic cell cycle" NARROW []
 synonym: "positive regulation of preblastoderm mitotic cell cycle progression" EXACT []
 synonym: "positive regulation of progression through preblastoderm mitotic cell cycle" EXACT [GOC:dph, GOC:tb]
@@ -288985,7 +289618,7 @@
 id: GO:0046003
 name: negative regulation of syncytial blastoderm mitotic cell cycle
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of progression through the syncytial blastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that stops, prevents or reduces the rate or extent of progression through the syncytial blastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "down regulation of progression through syncytial blastoderm mitotic cell cycle" EXACT []
 synonym: "down-regulation of progression through syncytial blastoderm mitotic cell cycle" EXACT []
 synonym: "downregulation of progression through syncytial blastoderm mitotic cell cycle" EXACT []
@@ -289002,7 +289635,7 @@
 id: GO:0046004
 name: positive regulation of syncytial blastoderm mitotic cell cycle
 namespace: biological_process
-def: "Any process that activates or increases the rate or extent of progression through the syncytial blastoderm mitotic cell cycle." [GOC:dph, GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the rate or extent of progression through the syncytial blastoderm mitotic cell cycle." [GOC:curators, GOC:dph, GOC:tb]
 synonym: "activation of progression through syncytial blastoderm mitotic cell cycle" NARROW []
 synonym: "positive regulation of progression through syncytial blastoderm mitotic cell cycle" EXACT [GOC:dph, GOC:tb]
 synonym: "positive regulation of syncytial blastoderm cell cycle progression" EXACT []
@@ -289020,7 +289653,7 @@
 id: GO:0046005
 name: positive regulation of circadian sleep/wake cycle, REM sleep
 namespace: biological_process
-def: "Any process that activates or increases the duration or quality of rapid eye movement (REM) sleep." [GOC:go_curators]
+def: "Any process that activates or increases the duration or quality of rapid eye movement (REM) sleep." [GOC:curators]
 synonym: "activation of circadian sleep/wake cycle, REM sleep" NARROW []
 synonym: "positive regulation of REM sleep" EXACT []
 synonym: "stimulation of circadian sleep/wake cycle, REM sleep" NARROW []
@@ -289037,7 +289670,7 @@
 id: GO:0046006
 name: regulation of activated T cell proliferation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of activated T cell proliferation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of activated T cell proliferation." [GOC:curators]
 synonym: "regulation of activated T lymphocyte proliferation" EXACT []
 synonym: "regulation of activated T-cell proliferation" EXACT []
 synonym: "regulation of activated T-lymphocyte proliferation" EXACT []
@@ -289050,7 +289683,7 @@
 id: GO:0046007
 name: negative regulation of activated T cell proliferation
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of activated T cell proliferation." [GOC:go_curators]
+def: "Any process that stops, prevents or reduces the rate or extent of activated T cell proliferation." [GOC:curators]
 synonym: "down regulation of activated T cell proliferation" EXACT []
 synonym: "down-regulation of activated T cell proliferation" EXACT []
 synonym: "downregulation of activated T cell proliferation" EXACT []
@@ -289068,14 +289701,14 @@
 id: GO:0046008
 name: regulation of female receptivity, post-mating
 namespace: biological_process
-def: "Any process that modulates the receptiveness of a female to male advances subsequent to mating." [GOC:go_curators]
+def: "Any process that modulates the receptiveness of a female to male advances subsequent to mating." [GOC:curators]
 is_a: GO:0045924 ! regulation of female receptivity
 
 [Term]
 id: GO:0046009
 name: positive regulation of female receptivity, post-mating
 namespace: biological_process
-def: "Any process that increases the receptiveness of a female to male advances subsequent to mating." [GOC:go_curators]
+def: "Any process that increases the receptiveness of a female to male advances subsequent to mating." [GOC:curators]
 synonym: "activation of female receptivity, post-mating" NARROW []
 synonym: "stimulation of female receptivity, post-mating" NARROW []
 synonym: "up regulation of female receptivity, post-mating" EXACT []
@@ -289088,7 +289721,7 @@
 id: GO:0046010
 name: positive regulation of circadian sleep/wake cycle, non-REM sleep
 namespace: biological_process
-def: "Any process that activates or increases the duration or quality of non-rapid eye movement (NREM) sleep." [GOC:go_curators]
+def: "Any process that activates or increases the duration or quality of non-rapid eye movement (NREM) sleep." [GOC:curators]
 synonym: "activation of circadian sleep/wake cycle, non-REM sleep" NARROW []
 synonym: "positive regulation of non-REM sleep" EXACT []
 synonym: "stimulation of circadian sleep/wake cycle, non-REM sleep" NARROW []
@@ -289105,14 +289738,14 @@
 id: GO:0046011
 name: regulation of oskar mRNA translation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of oskar mRNA translation. To ensure the localization of Oskar protein at the posterior pole of the oocyte, translation of oskar mRNA is repressed during its transport to the posterior pole and activated upon localization of the mRNA at the posterior cortex." [GOC:go_curators, PMID:12538512]
+def: "Any process that modulates the frequency, rate or extent of oskar mRNA translation. To ensure the localization of Oskar protein at the posterior pole of the oocyte, translation of oskar mRNA is repressed during its transport to the posterior pole and activated upon localization of the mRNA at the posterior cortex." [GOC:curators, PMID:12538512]
 is_a: GO:0006417 ! regulation of translation
 
 [Term]
 id: GO:0046012
 name: positive regulation of oskar mRNA translation
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of oskar mRNA translation." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of oskar mRNA translation." [GOC:curators]
 synonym: "activation of oskar mRNA translation" NARROW []
 synonym: "stimulation of oskar mRNA translation" NARROW []
 synonym: "up regulation of oskar mRNA translation" EXACT []
@@ -289125,7 +289758,7 @@
 id: GO:0046013
 name: regulation of T cell homeostatic proliferation
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of resting T cell proliferation." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of resting T cell proliferation." [GOC:curators]
 synonym: "regulation of resting T cell proliferation" EXACT []
 synonym: "regulation of T lymphocyte homeostatic proliferation" EXACT []
 synonym: "regulation of T-cell homeostatic proliferation" EXACT []
@@ -289139,7 +289772,7 @@
 id: GO:0046014
 name: negative regulation of T cell homeostatic proliferation
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the rate or extent of resting T cell proliferation." [GOC:go_curators]
+def: "Any process that stops, prevents or reduces the rate or extent of resting T cell proliferation." [GOC:curators]
 synonym: "down regulation of T cell homeostatic proliferation" EXACT []
 synonym: "down-regulation of T cell homeostatic proliferation" EXACT []
 synonym: "downregulation of T cell homeostatic proliferation" EXACT []
@@ -289158,14 +289791,14 @@
 id: GO:0046015
 name: regulation of transcription by glucose
 namespace: biological_process
-def: "Any process involving glucose that modulates the frequency, rate or extent or transcription." [GOC:go_curators]
+def: "Any process involving glucose that modulates the frequency, rate or extent or transcription." [GOC:curators]
 is_a: GO:0006355 ! regulation of DNA-templated transcription
 
 [Term]
 id: GO:0046016
 name: positive regulation of transcription by glucose
 namespace: biological_process
-def: "Any process involving glucose that activates or increases the rate of transcription." [GOC:go_curators]
+def: "Any process involving glucose that activates or increases the rate of transcription." [GOC:curators]
 synonym: "activation of transcription by glucose" NARROW []
 synonym: "stimulation of transcription by glucose" NARROW []
 synonym: "up regulation of transcription by glucose" EXACT []
@@ -289178,7 +289811,7 @@
 id: GO:0046017
 name: obsolete regulation of transcription from RNA polymerase I promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "mitotic regulation of transcription from Pol I promoter" EXACT []
 synonym: "regulation of transcription from Pol I promoter during mitosis" EXACT [GOC:dph, GOC:tb]
@@ -289192,7 +289825,7 @@
 id: GO:0046018
 name: obsolete positive regulation of transcription from RNA polymerase I promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase I promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "activation of transcription from RNA polymerase I promoter during mitosis" NARROW [GOC:mah]
 synonym: "mitotic activation of transcription from Pol I promoter" EXACT []
@@ -289211,7 +289844,7 @@
 id: GO:0046019
 name: obsolete regulation of transcription from RNA polymerase II promoter by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that modulates the frequency, rate or extent or transcription from an RNA polymerase II promoter." [GOC:go_curators, GOC:txnOH]
+def: "OBSOLETE. Any process involving pheromones that modulates the frequency, rate or extent or transcription from an RNA polymerase II promoter." [GOC:curators, GOC:txnOH]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 synonym: "regulation of transcription from Pol II promoter by pheromones" EXACT []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12739" xsd:anyURI
@@ -289221,7 +289854,7 @@
 id: GO:0046020
 name: obsolete negative regulation of transcription from RNA polymerase II promoter by pheromones
 namespace: biological_process
-def: "OBSOLETE. Any process involving pheromones that stops, prevents or reduces the rate of transcription from an RNA polymerase II promoter." [GOC:go_curators]
+def: "OBSOLETE. Any process involving pheromones that stops, prevents or reduces the rate of transcription from an RNA polymerase II promoter." [GOC:curators]
 comment: This term has been obsoleted because it represents a GO-CAM model.
 synonym: "down regulation of transcription from RNA polymerase II promoter by pheromones" EXACT []
 synonym: "down-regulation of transcription from RNA polymerase II promoter by pheromones" EXACT []
@@ -289235,7 +289868,7 @@
 id: GO:0046021
 name: obsolete regulation of transcription from RNA polymerase II promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "mitotic regulation of transcription from Pol II promoter" EXACT []
 synonym: "regulation of transcription from Pol II promoter, mitotic" EXACT []
@@ -289249,7 +289882,7 @@
 id: GO:0046022
 name: obsolete positive regulation of transcription from RNA polymerase II promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase II promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "activation of transcription from RNA polymerase II promoter during mitosis" NARROW [GOC:mah]
 synonym: "mitotic activation of transcription from Pol II promoter" EXACT []
@@ -289268,7 +289901,7 @@
 id: GO:0046023
 name: obsolete regulation of transcription from RNA polymerase III promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase III promoter that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. A cell cycle process that modulates the frequency, rate or extent of transcription from an RNA polymerase III promoter that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "mitotic regulation of transcription from Pol III promoter" EXACT []
 synonym: "regulation of transcription from Pol III promoter, mitotic" EXACT []
@@ -289282,7 +289915,7 @@
 id: GO:0046024
 name: obsolete positive regulation of transcription from RNA polymerase III promoter during mitotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase III that occurs during the mitotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of transcription from an RNA polymerase III that occurs during the mitotic cell cycle." [GOC:curators]
 comment: The reason for obsoletion is that 'during' should be captured using an annotation extension.
 synonym: "activation of transcription from RNA polymerase III promoter during mitosis" NARROW [GOC:mah]
 synonym: "mitotic activation of transcription from Pol III promoter" EXACT []
@@ -289385,7 +290018,7 @@
 id: GO:0046031
 name: ADP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving ADP, adenosine 5'-diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving ADP, adenosine 5'-diphosphate." [GOC:curators]
 synonym: "ADP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009179 ! purine ribonucleoside diphosphate metabolic process
@@ -289394,7 +290027,7 @@
 id: GO:0046032
 name: ADP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of ADP, adenosine 5'-diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of ADP, adenosine 5'-diphosphate." [GOC:curators]
 synonym: "ADP breakdown" EXACT []
 synonym: "ADP catabolism" EXACT []
 synonym: "ADP degradation" EXACT []
@@ -289406,7 +290039,7 @@
 id: GO:0046033
 name: AMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving AMP, adenosine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving AMP, adenosine monophosphate." [GOC:curators]
 synonym: "adenylate forming enzyme activity" RELATED []
 synonym: "AMP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
@@ -289416,7 +290049,7 @@
 id: GO:0046034
 name: ATP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving ATP, adenosine triphosphate, a universally important coenzyme and enzyme regulator." [GOC:go_curators]
+def: "The chemical reactions and pathways involving ATP, adenosine triphosphate, a universally important coenzyme and enzyme regulator." [GOC:curators]
 synonym: "ATP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009205 ! purine ribonucleoside triphosphate metabolic process
@@ -289425,7 +290058,7 @@
 id: GO:0046035
 name: CMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving CMP, cytidine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving CMP, cytidine monophosphate." [GOC:curators]
 synonym: "CMP metabolism" EXACT []
 is_a: GO:0009173 ! pyrimidine ribonucleoside monophosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289435,7 +290068,7 @@
 name: CTP metabolic process
 namespace: biological_process
 alt_id: GO:0006243
-def: "The chemical reactions and pathways involving CTP, cytidine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving CTP, cytidine triphosphate." [GOC:curators]
 synonym: "CTP deamination" NARROW []
 synonym: "CTP metabolism" EXACT []
 is_a: GO:0009208 ! pyrimidine ribonucleoside triphosphate metabolic process
@@ -289445,7 +290078,7 @@
 id: GO:0046037
 name: GMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving GMP, guanosine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving GMP, guanosine monophosphate." [GOC:curators]
 synonym: "GMP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009167 ! purine ribonucleoside monophosphate metabolic process
@@ -289454,7 +290087,7 @@
 id: GO:0046038
 name: GMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of GMP, guanosine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of GMP, guanosine monophosphate." [GOC:curators]
 synonym: "GMP breakdown" EXACT []
 synonym: "GMP catabolism" EXACT []
 synonym: "GMP degradation" EXACT []
@@ -289466,7 +290099,7 @@
 id: GO:0046039
 name: GTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving GTP, guanosine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving GTP, guanosine triphosphate." [GOC:curators]
 synonym: "GTP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009205 ! purine ribonucleoside triphosphate metabolic process
@@ -289475,7 +290108,7 @@
 id: GO:0046040
 name: IMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving IMP, inosine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving IMP, inosine monophosphate." [GOC:curators]
 synonym: "IMP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009167 ! purine ribonucleoside monophosphate metabolic process
@@ -289484,7 +290117,7 @@
 id: GO:0046041
 name: ITP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving ITP, inosine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving ITP, inosine triphosphate." [GOC:curators]
 synonym: "ITP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0009205 ! purine ribonucleoside triphosphate metabolic process
@@ -289493,7 +290126,7 @@
 id: GO:0046042
 name: ITP biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of ITP, inosine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of ITP, inosine triphosphate." [GOC:curators]
 synonym: "ITP anabolism" EXACT []
 synonym: "ITP biosynthesis" EXACT []
 synonym: "ITP formation" EXACT []
@@ -289506,7 +290139,7 @@
 id: GO:0046043
 name: TDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving TDP, ribosylthymine diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving TDP, ribosylthymine diphosphate." [GOC:curators]
 synonym: "TDP metabolism" EXACT []
 is_a: GO:0009193 ! pyrimidine ribonucleoside diphosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289515,7 +290148,7 @@
 id: GO:0046044
 name: TMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving TMP, ribosylthymine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving TMP, ribosylthymine monophosphate." [GOC:curators]
 synonym: "TMP metabolism" EXACT []
 is_a: GO:0009173 ! pyrimidine ribonucleoside monophosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289524,7 +290157,7 @@
 id: GO:0046045
 name: TMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of TMP, ribosylthymine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of TMP, ribosylthymine monophosphate." [GOC:curators]
 synonym: "TMP breakdown" EXACT []
 synonym: "TMP catabolism" EXACT []
 synonym: "TMP degradation" EXACT []
@@ -289536,7 +290169,7 @@
 id: GO:0046046
 name: TTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving TTP, ribosylthymine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving TTP, ribosylthymine triphosphate." [GOC:curators]
 synonym: "TTP metabolism" EXACT []
 is_a: GO:0009208 ! pyrimidine ribonucleoside triphosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289545,7 +290178,7 @@
 id: GO:0046047
 name: TTP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of TTP, ribosylthymine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of TTP, ribosylthymine triphosphate." [GOC:curators]
 synonym: "TTP breakdown" EXACT []
 synonym: "TTP catabolism" EXACT []
 synonym: "TTP degradation" EXACT []
@@ -289558,7 +290191,7 @@
 id: GO:0046048
 name: UDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving UDP, uridine (5'-)diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving UDP, uridine (5'-)diphosphate." [GOC:curators]
 synonym: "UDP metabolism" EXACT []
 is_a: GO:0009193 ! pyrimidine ribonucleoside diphosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289567,7 +290200,7 @@
 id: GO:0046049
 name: UMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving UMP, uridine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving UMP, uridine monophosphate." [GOC:curators]
 synonym: "UMP metabolism" EXACT []
 is_a: GO:0009173 ! pyrimidine ribonucleoside monophosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289576,7 +290209,7 @@
 id: GO:0046050
 name: UMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of UMP, uridine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of UMP, uridine monophosphate." [GOC:curators]
 synonym: "UMP breakdown" EXACT []
 synonym: "UMP catabolism" EXACT []
 synonym: "UMP degradation" EXACT []
@@ -289588,7 +290221,7 @@
 id: GO:0046051
 name: UTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving UTP, uridine (5'-)triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving UTP, uridine (5'-)triphosphate." [GOC:curators]
 synonym: "UTP metabolism" EXACT []
 is_a: GO:0009208 ! pyrimidine ribonucleoside triphosphate metabolic process
 is_a: GO:0009218 ! pyrimidine ribonucleotide metabolic process
@@ -289597,7 +290230,7 @@
 id: GO:0046052
 name: UTP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of UTP, uridine (5'-)triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of UTP, uridine (5'-)triphosphate." [GOC:curators]
 synonym: "UTP breakdown" EXACT []
 synonym: "UTP catabolism" EXACT []
 synonym: "UTP degradation" EXACT []
@@ -289610,7 +290243,7 @@
 id: GO:0046053
 name: dAMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dAMP, deoxyadenosine monophosphate (2'-deoxyadenosine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dAMP, deoxyadenosine monophosphate (2'-deoxyadenosine 5'-phosphate)." [GOC:curators]
 synonym: "dAMP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009170 ! purine deoxyribonucleoside monophosphate metabolic process
@@ -289619,7 +290252,7 @@
 id: GO:0046054
 name: dGMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dGMP, deoxyguanosine monophosphate (2'-deoxyguanosine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dGMP, deoxyguanosine monophosphate (2'-deoxyguanosine 5'-phosphate)." [GOC:curators]
 synonym: "dGMP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009170 ! purine deoxyribonucleoside monophosphate metabolic process
@@ -289628,7 +290261,7 @@
 id: GO:0046055
 name: dGMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dGMP, deoxyguanosine monophosphate (2'-deoxyguanosine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dGMP, deoxyguanosine monophosphate (2'-deoxyguanosine 5'-phosphate)." [GOC:curators]
 synonym: "dGMP breakdown" EXACT []
 synonym: "dGMP catabolism" EXACT []
 synonym: "dGMP degradation" EXACT []
@@ -289640,7 +290273,7 @@
 id: GO:0046056
 name: dADP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dADP, deoxyadenosine diphosphate (2'-deoxyadenosine 5'-diphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dADP, deoxyadenosine diphosphate (2'-deoxyadenosine 5'-diphosphate)." [GOC:curators]
 synonym: "dADP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009182 ! purine deoxyribonucleoside diphosphate metabolic process
@@ -289649,7 +290282,7 @@
 id: GO:0046057
 name: dADP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dADP, deoxyadenosine diphosphate (2'-deoxyadenosine 5'-diphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dADP, deoxyadenosine diphosphate (2'-deoxyadenosine 5'-diphosphate)." [GOC:curators]
 synonym: "dADP breakdown" EXACT []
 synonym: "dADP catabolism" EXACT []
 synonym: "dADP degradation" EXACT []
@@ -289661,7 +290294,7 @@
 id: GO:0046058
 name: cAMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving the nucleotide cAMP (cyclic AMP, adenosine 3',5'-cyclophosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving the nucleotide cAMP (cyclic AMP, adenosine 3',5'-cyclophosphate)." [GOC:curators]
 synonym: "3',5' cAMP metabolic process" EXACT []
 synonym: "3',5' cAMP metabolism" EXACT []
 synonym: "3',5'-cAMP metabolic process" EXACT []
@@ -289679,7 +290312,7 @@
 id: GO:0046059
 name: dAMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dAMP, deoxyadenosine monophosphate (2'-deoxyadenosine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dAMP, deoxyadenosine monophosphate (2'-deoxyadenosine 5'-phosphate)." [GOC:curators]
 synonym: "dAMP breakdown" EXACT []
 synonym: "dAMP catabolism" EXACT []
 synonym: "dAMP degradation" EXACT []
@@ -289691,7 +290324,7 @@
 id: GO:0046060
 name: dATP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dATP, deoxyadenosine triphosphate (2'-deoxyadenosine 5'-triphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dATP, deoxyadenosine triphosphate (2'-deoxyadenosine 5'-triphosphate)." [GOC:curators]
 synonym: "dATP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009215 ! purine deoxyribonucleoside triphosphate metabolic process
@@ -289700,7 +290333,7 @@
 id: GO:0046061
 name: dATP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dATP, deoxyadenosine triphosphate (2'-deoxyadenosine 5'-triphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dATP, deoxyadenosine triphosphate (2'-deoxyadenosine 5'-triphosphate)." [GOC:curators]
 synonym: "dATP breakdown" EXACT []
 synonym: "dATP catabolism" EXACT []
 synonym: "dATP degradation" EXACT []
@@ -289712,7 +290345,7 @@
 id: GO:0046062
 name: dCDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dCDP, deoxycytidine 5'-diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dCDP, deoxycytidine 5'-diphosphate." [GOC:curators]
 synonym: "dCDP metabolism" EXACT []
 is_a: GO:0009196 ! pyrimidine deoxyribonucleoside diphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289721,7 +290354,7 @@
 id: GO:0046063
 name: dCMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dCMP, deoxycytidine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dCMP, deoxycytidine monophosphate." [GOC:curators]
 synonym: "dCMP metabolism" EXACT []
 is_a: GO:0009176 ! pyrimidine deoxyribonucleoside monophosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289730,7 +290363,7 @@
 id: GO:0046064
 name: dCMP biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of dCMP, deoxycytidine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of dCMP, deoxycytidine monophosphate." [GOC:curators]
 synonym: "dCMP anabolism" EXACT []
 synonym: "dCMP biosynthesis" EXACT []
 synonym: "dCMP formation" EXACT []
@@ -289743,7 +290376,7 @@
 id: GO:0046065
 name: dCTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dCTP, deoxycytidine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dCTP, deoxycytidine triphosphate." [GOC:curators]
 synonym: "dCTP metabolism" EXACT []
 is_a: GO:0009211 ! pyrimidine deoxyribonucleoside triphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289752,7 +290385,7 @@
 id: GO:0046066
 name: dGDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dGDP, deoxyguanosine diphosphate, (2'-deoxyguanosine 5'-diphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dGDP, deoxyguanosine diphosphate, (2'-deoxyguanosine 5'-diphosphate)." [GOC:curators]
 synonym: "dGDP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009182 ! purine deoxyribonucleoside diphosphate metabolic process
@@ -289761,7 +290394,7 @@
 id: GO:0046067
 name: dGDP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dGDP, deoxyguanosine diphosphate, (2'-deoxyguanosine 5'-diphosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dGDP, deoxyguanosine diphosphate, (2'-deoxyguanosine 5'-diphosphate)." [GOC:curators]
 synonym: "dGDP breakdown" EXACT []
 synonym: "dGDP catabolism" EXACT []
 synonym: "dGDP degradation" EXACT []
@@ -289773,7 +290406,7 @@
 id: GO:0046068
 name: cGMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving cyclic GMP, guanosine 3',5'-phosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving cyclic GMP, guanosine 3',5'-phosphate." [GOC:curators]
 synonym: "cGMP metabolism" EXACT []
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0052652 ! cyclic purine nucleotide metabolic process
@@ -289782,7 +290415,7 @@
 id: GO:0046069
 name: cGMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of cyclic GMP, guanosine 3',5'-phosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of cyclic GMP, guanosine 3',5'-phosphate." [GOC:curators]
 synonym: "cGMP breakdown" EXACT []
 synonym: "cGMP catabolism" EXACT []
 synonym: "cGMP degradation" EXACT []
@@ -289794,7 +290427,7 @@
 id: GO:0046070
 name: dGTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dGTP, guanosine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dGTP, guanosine triphosphate." [GOC:curators]
 synonym: "dGTP metabolism" EXACT []
 is_a: GO:0009151 ! purine deoxyribonucleotide metabolic process
 is_a: GO:0009215 ! purine deoxyribonucleoside triphosphate metabolic process
@@ -289803,7 +290436,7 @@
 id: GO:0046071
 name: dGTP biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of dGTP, guanosine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of dGTP, guanosine triphosphate." [GOC:curators]
 synonym: "dGTP anabolism" EXACT []
 synonym: "dGTP biosynthesis" EXACT []
 synonym: "dGTP formation" EXACT []
@@ -289816,7 +290449,7 @@
 id: GO:0046072
 name: dTDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dTDP, deoxyribosylthymine diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dTDP, deoxyribosylthymine diphosphate." [GOC:curators]
 synonym: "dTDP metabolism" EXACT []
 is_a: GO:0009196 ! pyrimidine deoxyribonucleoside diphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289825,7 +290458,7 @@
 id: GO:0046073
 name: dTMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dTMP, deoxyribosylthymine monophosphate (2'-deoxyribosylthymine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dTMP, deoxyribosylthymine monophosphate (2'-deoxyribosylthymine 5'-phosphate)." [GOC:curators]
 synonym: "dTMP metabolism" EXACT []
 is_a: GO:0009176 ! pyrimidine deoxyribonucleoside monophosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289834,7 +290467,7 @@
 id: GO:0046074
 name: dTMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dTMP, deoxyribosylthymine monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dTMP, deoxyribosylthymine monophosphate." [GOC:curators]
 synonym: "dTMP breakdown" EXACT []
 synonym: "dTMP catabolism" EXACT []
 synonym: "dTMP degradation" EXACT []
@@ -289846,7 +290479,7 @@
 id: GO:0046075
 name: dTTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dTTP, deoxyribosylthymine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dTTP, deoxyribosylthymine triphosphate." [GOC:curators]
 synonym: "dTTP metabolism" EXACT []
 is_a: GO:0009211 ! pyrimidine deoxyribonucleoside triphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289855,7 +290488,7 @@
 id: GO:0046076
 name: dTTP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dTTP, deoxyribosylthymine triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dTTP, deoxyribosylthymine triphosphate." [GOC:curators]
 synonym: "dTTP breakdown" EXACT []
 synonym: "dTTP catabolism" EXACT []
 synonym: "dTTP degradation" EXACT []
@@ -289867,7 +290500,7 @@
 id: GO:0046077
 name: dUDP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dUDP, deoxyuridine (5'-)diphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dUDP, deoxyuridine (5'-)diphosphate." [GOC:curators]
 synonym: "dUDP metabolism" EXACT []
 is_a: GO:0009196 ! pyrimidine deoxyribonucleoside diphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289876,7 +290509,7 @@
 id: GO:0046078
 name: dUMP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dUMP, deoxyuridine (5'-)monophosphate (2'-deoxyuridine 5'-phosphate)." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dUMP, deoxyuridine (5'-)monophosphate (2'-deoxyuridine 5'-phosphate)." [GOC:curators]
 synonym: "dUMP metabolism" EXACT []
 is_a: GO:0009176 ! pyrimidine deoxyribonucleoside monophosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289885,7 +290518,7 @@
 id: GO:0046079
 name: dUMP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dUMP, deoxyuridine (5'-)monophosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dUMP, deoxyuridine (5'-)monophosphate." [GOC:curators]
 synonym: "dUMP breakdown" EXACT []
 synonym: "dUMP catabolism" EXACT []
 synonym: "dUMP degradation" EXACT []
@@ -289897,7 +290530,7 @@
 id: GO:0046080
 name: dUTP metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving dUTP, deoxyuridine (5'-)triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving dUTP, deoxyuridine (5'-)triphosphate." [GOC:curators]
 synonym: "dUTP metabolism" EXACT []
 is_a: GO:0009211 ! pyrimidine deoxyribonucleoside triphosphate metabolic process
 is_a: GO:0009219 ! pyrimidine deoxyribonucleotide metabolic process
@@ -289906,7 +290539,7 @@
 id: GO:0046081
 name: dUTP catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of dUTP, deoxyuridine (5'-)triphosphate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of dUTP, deoxyuridine (5'-)triphosphate." [GOC:curators]
 synonym: "dUTP breakdown" EXACT []
 synonym: "dUTP catabolism" EXACT []
 synonym: "dUTP degradation" EXACT []
@@ -289918,7 +290551,7 @@
 id: GO:0046082
 name: obsolete 5-methylcytosine biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of 5-methylcytosine, a methylated base of DNA." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of 5-methylcytosine, a methylated base of DNA." [GOC:curators]
 comment: This term was obsoleted because itis redundant with DNA methylation-dependent heterochromatin formation ; GO:0006346 or its children.
 synonym: "5-methylcytosine anabolism" EXACT []
 synonym: "5-methylcytosine biosynthesis" EXACT []
@@ -289932,7 +290565,7 @@
 id: GO:0046083
 name: adenine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving adenine, 6-aminopurine, one of the five main bases found in nucleic acids and a component of numerous important derivatives of its corresponding ribonucleoside, adenosine." [GOC:go_curators]
+def: "The chemical reactions and pathways involving adenine, 6-aminopurine, one of the five main bases found in nucleic acids and a component of numerous important derivatives of its corresponding ribonucleoside, adenosine." [GOC:curators]
 synonym: "adenine metabolism" EXACT []
 is_a: GO:0006144 ! purine nucleobase metabolic process
 
@@ -289940,7 +290573,7 @@
 id: GO:0046084
 name: adenine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of adenine, 6-aminopurine, one of the five main bases found in nucleic acids and a component of numerous important derivatives of its corresponding ribonucleoside, adenosine." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of adenine, 6-aminopurine, one of the five main bases found in nucleic acids and a component of numerous important derivatives of its corresponding ribonucleoside, adenosine." [GOC:curators]
 synonym: "adenine anabolism" EXACT []
 synonym: "adenine biosynthesis" EXACT []
 synonym: "adenine formation" EXACT []
@@ -289952,7 +290585,7 @@
 id: GO:0046085
 name: adenosine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:go_curators]
+def: "The chemical reactions and pathways involving adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:curators]
 synonym: "adenosine metabolism" EXACT []
 is_a: GO:0046128 ! purine ribonucleoside metabolic process
 
@@ -289960,7 +290593,7 @@
 id: GO:0046086
 name: adenosine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of adenosine, adenine riboside, a ribonucleoside found widely distributed in cells of every type as the free nucleoside and in combination in nucleic acids and various nucleoside coenzymes." [GOC:curators]
 synonym: "adenosine anabolism" EXACT []
 synonym: "adenosine biosynthesis" EXACT []
 synonym: "adenosine formation" EXACT []
@@ -289972,7 +290605,7 @@
 id: GO:0046087
 name: cytidine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving cytidine, cytosine riboside, a widely distributed nucleoside." [GOC:go_curators]
+def: "The chemical reactions and pathways involving cytidine, cytosine riboside, a widely distributed nucleoside." [GOC:curators]
 synonym: "cytidine metabolism" EXACT []
 is_a: GO:0046131 ! pyrimidine ribonucleoside metabolic process
 
@@ -289980,7 +290613,7 @@
 id: GO:0046088
 name: obsolete cytidine biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cytidine, cytosine riboside, a widely distributed nucleoside." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cytidine, cytosine riboside, a widely distributed nucleoside." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this process exists.
 synonym: "cytidine anabolism" EXACT []
 synonym: "cytidine biosynthesis" EXACT []
@@ -289993,7 +290626,7 @@
 id: GO:0046089
 name: obsolete cytosine biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cytosine, 4-amino-2-hydroxypyrimidine, a pyrimidine derivative that is one of the five main bases found in nucleic acids; it occurs widely in cytidine derivatives." [GOC:go_curators]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of cytosine, 4-amino-2-hydroxypyrimidine, a pyrimidine derivative that is one of the five main bases found in nucleic acids; it occurs widely in cytidine derivatives." [GOC:curators]
 comment: The reason for obsoletion is that there is no evidence that this process exists.
 synonym: "cytosine anabolism" EXACT []
 synonym: "cytosine biosynthesis" EXACT []
@@ -290006,7 +290639,7 @@
 id: GO:0046090
 name: deoxyadenosine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyadenosine metabolism" EXACT []
 is_a: GO:0046122 ! purine deoxyribonucleoside metabolic process
 
@@ -290014,7 +290647,7 @@
 id: GO:0046091
 name: deoxyadenosine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of deoxyadenosine, 2-deoxyribosyladenine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyadenosine anabolism" EXACT []
 synonym: "deoxyadenosine biosynthesis" EXACT []
 synonym: "deoxyadenosine formation" EXACT []
@@ -290026,7 +290659,7 @@
 id: GO:0046092
 name: deoxycytidine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxycytidine metabolism" EXACT []
 is_a: GO:0046125 ! pyrimidine deoxyribonucleoside metabolic process
 
@@ -290034,7 +290667,7 @@
 id: GO:0046093
 name: deoxycytidine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of deoxycytidine, 2-deoxyribosylcytosine, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxycytidine anabolism" EXACT []
 synonym: "deoxycytidine biosynthesis" EXACT []
 synonym: "deoxycytidine formation" EXACT []
@@ -290046,7 +290679,7 @@
 id: GO:0046094
 name: deoxyinosine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving deoxyinosine, hypoxanthine deoxyriboside." [GOC:go_curators]
+def: "The chemical reactions and pathways involving deoxyinosine, hypoxanthine deoxyriboside." [GOC:curators]
 synonym: "deoxyinosine metabolism" EXACT []
 is_a: GO:0046122 ! purine deoxyribonucleoside metabolic process
 
@@ -290054,7 +290687,7 @@
 id: GO:0046095
 name: deoxyinosine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of deoxyinosine, hypoxanthine deoxyriboside." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of deoxyinosine, hypoxanthine deoxyriboside." [GOC:curators]
 synonym: "deoxyinosine anabolism" EXACT []
 synonym: "deoxyinosine biosynthesis" EXACT []
 synonym: "deoxyinosine formation" EXACT []
@@ -290066,7 +290699,7 @@
 id: GO:0046096
 name: deoxyuridine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyuridine metabolism" EXACT []
 is_a: GO:0046125 ! pyrimidine deoxyribonucleoside metabolic process
 
@@ -290074,7 +290707,7 @@
 id: GO:0046097
 name: deoxyuridine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of deoxyuridine, 2-deoxyribosyluracil, one of the four major nucleosides of DNA." [GOC:curators]
 synonym: "deoxyuridine anabolism" EXACT []
 synonym: "deoxyuridine biosynthesis" EXACT []
 synonym: "deoxyuridine formation" EXACT []
@@ -290086,7 +290719,7 @@
 id: GO:0046098
 name: guanine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:go_curators]
+def: "The chemical reactions and pathways involving guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:curators]
 synonym: "guanine metabolism" EXACT []
 is_a: GO:0006144 ! purine nucleobase metabolic process
 
@@ -290094,7 +290727,7 @@
 id: GO:0046099
 name: guanine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of guanine, 2-amino-6-hydroxypurine, a purine that is one of the five main bases found in nucleic acids and a component of a number of phosphorylated guanosine derivatives whose metabolic or regulatory functions are important." [GOC:curators]
 synonym: "guanine anabolism" EXACT []
 synonym: "guanine biosynthesis" EXACT []
 synonym: "guanine formation" EXACT []
@@ -290106,7 +290739,7 @@
 id: GO:0046100
 name: hypoxanthine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving hypoxanthine, 6-hydroxy purine, an intermediate in the degradation of adenylate. Its ribonucleoside is known as inosine and its ribonucleotide as inosinate." [GOC:go_curators]
+def: "The chemical reactions and pathways involving hypoxanthine, 6-hydroxy purine, an intermediate in the degradation of adenylate. Its ribonucleoside is known as inosine and its ribonucleotide as inosinate." [GOC:curators]
 synonym: "hypoxanthine metabolism" EXACT []
 is_a: GO:0006144 ! purine nucleobase metabolic process
 
@@ -290114,7 +290747,7 @@
 id: GO:0046101
 name: hypoxanthine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of hypoxanthine, 6-hydroxy purine, an intermediate in the degradation of adenylate. Its ribonucleoside is known as inosine and its ribonucleotide as inosinate." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of hypoxanthine, 6-hydroxy purine, an intermediate in the degradation of adenylate. Its ribonucleoside is known as inosine and its ribonucleotide as inosinate." [GOC:curators]
 synonym: "hypoxanthine anabolism" EXACT []
 synonym: "hypoxanthine biosynthesis" EXACT []
 synonym: "hypoxanthine formation" EXACT []
@@ -290126,7 +290759,7 @@
 id: GO:0046102
 name: inosine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:go_curators]
+def: "The chemical reactions and pathways involving inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:curators]
 synonym: "inosine metabolism" EXACT []
 is_a: GO:0046128 ! purine ribonucleoside metabolic process
 
@@ -290134,7 +290767,7 @@
 id: GO:0046103
 name: inosine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of inosine, hypoxanthine riboside, a nucleoside found free but not in combination in nucleic acids except in the anticodons of some tRNAs." [GOC:curators]
 synonym: "inosine anabolism" EXACT []
 synonym: "inosine biosynthesis" EXACT []
 synonym: "inosine formation" EXACT []
@@ -290146,7 +290779,7 @@
 id: GO:0046104
 name: thymidine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways involving thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:curators]
 synonym: "deoxyribosylthymine metabolic process" EXACT []
 synonym: "deoxyribosylthymine metabolism" EXACT []
 synonym: "thymidine metabolism" EXACT []
@@ -290156,7 +290789,7 @@
 id: GO:0046105
 name: thymidine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of thymidine, deoxyribosylthymine thymine 2-deoxyriboside, a deoxynucleoside very widely distributed but occurring almost entirely as phosphoric esters in deoxynucleotides and deoxyribonucleic acid, DNA." [GOC:curators]
 synonym: "deoxyribosylthymine biosynthesis" EXACT []
 synonym: "deoxyribosylthymine biosynthetic process" EXACT []
 synonym: "thymidine anabolism" EXACT []
@@ -290170,7 +290803,7 @@
 id: GO:0046106
 name: thymine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of thymine, 5-methyluracil, one of the two major pyrimidine bases present (as thymidine) in DNA but not found in RNA other than (as ribothymidine) in transfer RNA, where it is a minor base." [GOC:curators]
 synonym: "thymine anabolism" EXACT []
 synonym: "thymine biosynthesis" EXACT []
 synonym: "thymine formation" EXACT []
@@ -290182,7 +290815,7 @@
 id: GO:0046107
 name: uracil biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of uracil, 2,4-dioxopyrimidine, one of the pyrimidine bases occurring in RNA, but not in DNA." [GOC:curators]
 synonym: "uracil anabolism" EXACT []
 synonym: "uracil biosynthesis" EXACT []
 synonym: "uracil formation" EXACT []
@@ -290194,7 +290827,7 @@
 id: GO:0046108
 name: uridine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways involving uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:curators]
 synonym: "uridine metabolism" EXACT []
 is_a: GO:0046131 ! pyrimidine ribonucleoside metabolic process
 
@@ -290202,7 +290835,7 @@
 id: GO:0046109
 name: uridine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of uridine, uracil riboside, a ribonucleoside very widely distributed but occurring almost entirely as phosphoric esters in ribonucleotides and ribonucleic acids." [GOC:curators]
 synonym: "uridine anabolism" EXACT []
 synonym: "uridine biosynthesis" EXACT []
 synonym: "uridine formation" EXACT []
@@ -290214,7 +290847,7 @@
 id: GO:0046110
 name: xanthine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways involving xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:curators]
 synonym: "xanthine metabolism" EXACT []
 is_a: GO:0006144 ! purine nucleobase metabolic process
 
@@ -290222,7 +290855,7 @@
 id: GO:0046111
 name: xanthine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of xanthine, 2,6-dihydroxypurine, a purine formed in the metabolic breakdown of guanine but not present in nucleic acids." [GOC:curators]
 synonym: "xanthine anabolism" EXACT []
 synonym: "xanthine biosynthesis" EXACT []
 synonym: "xanthine formation" EXACT []
@@ -290257,7 +290890,7 @@
 id: GO:0046114
 name: guanosine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of guanine, guanine riboside, a nucleoside with a wide species distribution." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of guanine, guanine riboside, a nucleoside with a wide species distribution." [GOC:curators]
 synonym: "guanosine anabolism" EXACT []
 synonym: "guanosine biosynthesis" EXACT []
 synonym: "guanosine formation" EXACT []
@@ -290270,7 +290903,7 @@
 name: guanosine catabolic process
 namespace: biological_process
 alt_id: GO:0006160
-def: "The chemical reactions and pathways resulting in the breakdown of guanine, guanine riboside, a nucleoside with a wide species distribution." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of guanine, guanine riboside, a nucleoside with a wide species distribution." [GOC:curators]
 synonym: "guanosine breakdown" EXACT []
 synonym: "guanosine catabolism" EXACT []
 synonym: "guanosine degradation" EXACT []
@@ -290842,23 +291475,27 @@
 name: alcohol catabolic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the breakdown of alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [GOC:ai]
+subset: gocheck_do_not_annotate
 synonym: "alcohol breakdown" EXACT []
 synonym: "alcohol catabolism" EXACT []
 synonym: "alcohol degradation" EXACT []
 is_a: GO:0006066 ! alcohol metabolic process
 is_a: GO:0044282 ! small molecule catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0046165
 name: alcohol biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of alcohols, any of a class of compounds containing one or more hydroxyl groups attached to a saturated carbon atom." [GOC:ai]
+subset: gocheck_do_not_annotate
 synonym: "alcohol anabolism" EXACT []
 synonym: "alcohol biosynthesis" EXACT []
 synonym: "alcohol formation" EXACT []
 synonym: "alcohol synthesis" EXACT []
 is_a: GO:0006066 ! alcohol metabolic process
 is_a: GO:0044283 ! small molecule biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 
 [Term]
 id: GO:0046166
@@ -291659,7 +292296,7 @@
 id: GO:0046233
 name: obsolete 3-hydroxyphenylacetate biosynthetic process
 namespace: biological_process
-def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of 3-hydroxyphenylacetate, 1,3-benzenediol monoacetate, also known as resorcinol monoacetate." [GOC:curator]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the formation of 3-hydroxyphenylacetate, 1,3-benzenediol monoacetate, also known as resorcinol monoacetate." [GOC:curators]
 comment: This term was made obsolete because 3-hydroxyphenylacetate is not synthesized by living organisms and GO does not cover non-biological processes.
 synonym: "3-hydroxyphenylacetate anabolism" EXACT []
 synonym: "3-hydroxyphenylacetate biosynthesis" EXACT []
@@ -293854,7 +294491,7 @@
 id: GO:0046411
 name: 2-keto-3-deoxygluconate transmembrane transport
 namespace: biological_process
-def: "The process in which 2-keto-3-deoxygluconate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:go_curators]
+def: "The process in which 2-keto-3-deoxygluconate is transported across a lipid bilayer, from one side of a membrane to the other." [GOC:curators]
 synonym: "2-keto-3-deoxygluconate transport" RELATED []
 is_a: GO:0015718 ! monocarboxylic acid transport
 is_a: GO:0015749 ! monosaccharide transmembrane transport
@@ -294152,12 +294789,15 @@
 
 [Term]
 id: GO:0046436
-name: D-alanine metabolic process
+name: obsolete D-alanine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-alanine, the D-enantiomer of the amino acid alanine, i.e. (2R)-2-aminopropanoic acid." [GOC:ai, GOC:jsg]
+def: "OBSOLETE. The chemical reactions and pathways involving D-alanine, the D-enantiomer of the amino acid alanine, i.e. (2R)-2-aminopropanoic acid." [GOC:ai, GOC:jsg]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-alanine metabolism" EXACT []
-is_a: GO:0006522 ! alanine metabolic process
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
+consider: GO:0030632
+consider: GO:0055130
 
 [Term]
 id: GO:0046437
@@ -294188,7 +294828,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving L-cysteine, the L-enantiomer of 2-amino-3-mercaptopropanoic acid, i.e. (2R)-2-amino-3-mercaptopropanoic acid." [GOC:ai, GOC:jsg, GOC:mah]
 synonym: "L-cysteine metabolism" EXACT []
-is_a: GO:0006534 ! cysteine metabolic process
+is_a: GO:0000096 ! sulfur amino acid metabolic process
 is_a: GO:0170033 ! L-amino acid metabolic process
 is_a: GO:0170039 ! proteinogenic amino acid metabolic process
 
@@ -294858,7 +295498,7 @@
 id: GO:0046500
 name: S-adenosylmethionine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving S-adenosylmethionine, S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism." [GOC:go_curators, ISBN:0198506732]
+def: "The chemical reactions and pathways involving S-adenosylmethionine, S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism." [GOC:curators, ISBN:0198506732]
 synonym: "S-adenosyl methionine metabolic process" EXACT []
 synonym: "S-adenosyl methionine metabolism" EXACT []
 synonym: "S-adenosylmethionine metabolism" EXACT []
@@ -295153,8 +295793,8 @@
 xref: EC:5.3.1.23
 xref: KEGG_REACTION:R04420
 xref: MetaCyc:5.3.1.23-RXN
-xref: Reactome:R-HSA-1237096 "Methylthio-ribose-P = Methylthio-ribulose-P"
-xref: Reactome:R-HSA-1299507 "Methylthio-ribulose-P = Methylthio-ribose-P"
+xref: Reactome:R-HSA-1237096 "Methylthio-ribose-P => Methylthio-ribulose-P"
+xref: Reactome:R-HSA-1299507 "Methylthio-ribulose-P => Methylthio-ribose-P"
 xref: RHEA:19989
 is_a: GO:0016861 ! intramolecular oxidoreductase activity, interconverting aldoses and ketoses
 property_value: skos:exactMatch EC:5.3.1.23
@@ -295262,7 +295902,7 @@
 name: regulation of photoreceptor cell differentiation
 namespace: biological_process
 alt_id: GO:0045673
-def: "Any process that modulates the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:curators]
 synonym: "regulation of photoreceptor differentiation" EXACT []
 is_a: GO:0045664 ! regulation of neuron differentiation
 intersection_of: GO:0065007 ! biological regulation
@@ -295274,7 +295914,7 @@
 name: negative regulation of photoreceptor cell differentiation
 namespace: biological_process
 alt_id: GO:0045674
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:curators]
 synonym: "down regulation of photoreceptor cell differentiation" EXACT []
 synonym: "down regulation of photoreceptor differentiation" EXACT []
 synonym: "down-regulation of photoreceptor cell differentiation" EXACT []
@@ -295296,7 +295936,7 @@
 name: positive regulation of photoreceptor cell differentiation
 namespace: biological_process
 alt_id: GO:0045675
-def: "Any process that activates or increases the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:go_curators]
+def: "Any process that activates or increases the frequency, rate or extent of photoreceptor cell differentiation. An example of this process is found in Drosophila melanogaster." [GOC:curators]
 synonym: "activation of photoreceptor cell differentiation" NARROW []
 synonym: "activation of photoreceptor differentiation" NARROW []
 synonym: "positive regulation of photoreceptor differentiation" EXACT []
@@ -295826,14 +296466,15 @@
 id: GO:0046572
 name: versicolorin B synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: versiconal = versicolorin B + H2O." [MetaCyc:RXN-9494, PMID:8784203]
+def: "Catalysis of the reaction: (2S-3S)-versiconal hemiacetal = versicolorin B + H2O." [PMID:8784203, RHEA:33859]
 xref: EC:4.2.1.143
 xref: MetaCyc:RXN-9494
 xref: RHEA:33859
-is_a: GO:0016835 ! carbon-oxygen lyase activity
+is_a: GO:0016836 ! hydro-lyase activity
 property_value: skos:exactMatch EC:4.2.1.143
 property_value: skos:exactMatch RHEA:33859
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31620" xsd:anyURI
 
 [Term]
 id: GO:0046573
@@ -296374,7 +297015,7 @@
 id: GO:0046618
 name: xenobiotic export from cell
 namespace: biological_process
-def: "The directed movement of a xenobiotic from a cell, into the extracellular region. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:go_curators, GOC:krc]
+def: "The directed movement of a xenobiotic from a cell, into the extracellular region. A xenobiotic is a compound foreign to the organism exposed to it. It may be synthesized by another organism (like ampicilin) or it can be a synthetic chemical." [GOC:curators, GOC:krc]
 synonym: "drug export" RELATED []
 synonym: "xenobiotic export" RELATED []
 is_a: GO:0042908 ! xenobiotic transport
@@ -296437,21 +297078,23 @@
 synonym: "sphingolipid flippase activity" EXACT []
 synonym: "sphingolipid floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "sphingolipid-translocating ATPase activity" RELATED []
-is_a: GO:0046624 ! sphingolipid transporter activity
+is_a: GO:0046624 ! sphingolipid intramembrane carrier activity
 is_a: GO:0140328 ! floppase activity
 relationship: part_of GO:0099039 ! sphingolipid translocation
 property_value: skos:broadMatch RHEA:66132
 
 [Term]
 id: GO:0046624
-name: sphingolipid transporter activity
+name: sphingolipid intramembrane carrier activity
 namespace: molecular_function
 def: "Enables the directed movement of sphingolipids into, out of or within a cell, or between cells. Sphingolipids are a class of lipids containing the long-chain amine diol sphingosine or a closely related base (a sphingoid)." [GOC:ai, ISBN:0198506732]
 subset: gocheck_do_not_annotate
+synonym: "sphingolipid intramembrane transporter activity" BROAD []
 xref: Reactome:R-HSA-9695890 "SPNS2,MFSD2B transport S1P from cytosol to extracellular region"
 xref: Reactome:R-HSA-9843721 "ABCC1,ABCG2 transport C18-S1P to extracellular region"
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31121" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 
 [Term]
 id: GO:0046625
@@ -297133,7 +297776,9 @@
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an antibiotic stimulus. An antibiotic is a chemical substance produced by a microorganism which has the capacity to inhibit the growth of or to kill other microorganisms." [GOC:ai, GOC:ef]
 subset: goslim_chembl
 synonym: "antibiotic susceptibility/resistance" RELATED []
+xref: Reactome:R-HSA-9638771 "Action of antimicrobials"
 xref: Reactome:R-HSA-9639775 "Antimicrobial action and antimicrobial resistance in Mtb"
+xref: Reactome:R-HSA-9913143 "Antimicrobial resistance"
 is_a: GO:0042221 ! response to chemical
 
 [Term]
@@ -298098,7 +298743,7 @@
 id: GO:0046785
 name: microtubule polymerization
 namespace: biological_process
-def: "The addition of tubulin heterodimers to one or both ends of a microtubule." [GOC:ai, GOC:go_curators]
+def: "The addition of tubulin heterodimers to one or both ends of a microtubule." [GOC:ai, GOC:curators]
 synonym: "microtubule assembly" EXACT []
 synonym: "microtubule formation" RELATED []
 is_a: GO:0031109 ! microtubule polymerization or depolymerization
@@ -298390,12 +299035,12 @@
 
 [Term]
 id: GO:0046820
-name: 4-amino-4-deoxychorismate synthase activity
+name: aminodeoxychorismate synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-glutamine + chorismate = 4-amino-4-deoxychorismate + L-glutamate. It is composed of two enzymatic activities (which may be present on one or two polypeptides); the first is a glutaminase which yields ammonia from glutamine, releasing glutamate. The ammonia is used by the second activity which catalyzes the amination of chorismate to form 4-amino-4-deoxychorismate." [EC:2.6.1.85, RHEA:11672]
 comment: Note that the name 'para-aminobenzoic acid synthase' was initially given to the 'aminodeoxychorismate synthase' activity before the additional 4-amino-4-deoxychorismate lyase activity was discovered. It is the lyase activity that actually produces para-aminobenzoic acid from 4-amino-4-deoxychorismate.
+synonym: "4-amino-4-deoxychorismate synthase activity" EXACT []
 synonym: "ADC synthase activity" RELATED [EC:2.6.1.85]
-synonym: "aminodeoxychorismate synthase activity" RELATED []
 synonym: "chorismate:L-glutamine amido-ligase activity" RELATED [EC:2.6.1.85]
 synonym: "p-aminobenzoate synthetase" RELATED []
 synonym: "PabB activity" RELATED [EC:2.6.1.85]
@@ -298409,6 +299054,7 @@
 property_value: skos:exactMatch EC:2.6.1.85
 property_value: skos:exactMatch RHEA:11672
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0046821
@@ -299488,7 +300134,7 @@
 id: GO:0046912
 name: acyltransferase activity, acyl groups converted into alkyl on transfer
 namespace: molecular_function
-def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor), with the acyl group being converted into alkyl on transfer." [GOC:jl]
+def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor), with the acyl group being converted into alkyl on transfer." [EC:2.3.3.-, GOC:jl]
 synonym: "acyltransferase, acyl groups converted into alkyl on transfer" EXACT []
 synonym: "transferase activity, transferring acyl groups, acyl groups converted into alkyl on transfer" EXACT []
 xref: EC:2.3.3.-
@@ -299564,7 +300210,7 @@
 synonym: "alpha(1,3)-fucosyltransferase activity" EXACT []
 synonym: "alpha-(1,3)-fucosyltransferase activity" EXACT []
 synonym: "alpha-1,3-fucosyltransferase activity" EXACT []
-xref: Reactome:R-HSA-9603984 "FUT4,5,9,(10,11) transfer Fuc to Type 2 chains to form LeX"
+xref: Reactome:R-HSA-9603984 "FUT4,5,9 transfer Fuc to Type 2 chains to form LeX"
 xref: Reactome:R-HSA-9605609 "FUT3 transfers Fuc to Type 1 MSGG to form sLeA"
 xref: Reactome:R-HSA-9605644 "FUT3 transfers Fuc to Type 1 DSGG to form dsLeA"
 xref: Reactome:R-HSA-9605682 "FUT3,5,6,7 transfers Fuc to Type 2 MSGG to form sLeX"
@@ -299595,7 +300241,6 @@
 xref: Reactome:R-HSA-5173192 "POFUT2 transfers fucose to TSR domain-containing proteins"
 is_a: GO:0008417 ! fucosyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
-relationship: part_of GO:0036066 ! protein O-linked glycosylation via fucose
 property_value: skos:exactMatch EC:2.4.1.221
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -299969,7 +300614,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving ketone bodies, any one of the three substances: acetoacetate, D-3-hydroxybutyrate (beta-hydroxybutyrate) or acetone, as carried out by individual cells. Although 3-hydroxybutyrate is not a ketone, it is classed as a ketone body because it exists in an equilibrium with acetoacetate. Ketone bodies may accumulate in excessive amounts in the body in starvation, diabetes mellitus or in other defects of carbohydrate metabolism." [ISBN:0198506732]
 comment: The reason for obsoletion is that this term is redundant to GO:1902224, ketone body metabolic process
 synonym: "cellular ketone body metabolism" EXACT []
-xref: Reactome:R-HSA-74182 "Ketone body metabolism"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29021" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:1902224
@@ -300138,7 +300782,7 @@
 synonym: "3'-phosphoadenosine 5'-phosphosulphate transporter activity" EXACT []
 synonym: "adenosine 3'-phosphate 5'-phosphosulfate transmembrane transporter activity" EXACT []
 synonym: "PAPS transporter activity" EXACT []
-xref: Reactome:R-HSA-741449 "SLC35B2,3 transport cytosolic PAPS to Golgi lumen"
+xref: Reactome:R-HSA-741449 "SLC35B2,3 exchange cytosolic PAPS with Golgi PAP"
 is_a: GO:0000295 ! adenine nucleotide transmembrane transporter activity
 is_a: GO:0005346 ! purine ribonucleotide transmembrane transporter activity
 is_a: GO:1901682 ! sulfur compound transmembrane transporter activity
@@ -300287,6 +300931,8 @@
 synonym: "histone methyltransferase activity (H3-K27 specific)" EXACT []
 synonym: "histone-H3K27 methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-3240295 "PRC2-EZH2 trimethylates nucleosomes associated with CDKN2A promoter"
+xref: Reactome:R-HSA-9764932 "PRC2 (EZH2) core complex methylates nucleosomes at CDH1 gene promoter"
+xref: Reactome:R-HSA-9909631 "Trimethylation of CD274 gene promoter by EZH2"
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24337" xsd:anyURI
@@ -301348,6 +301994,7 @@
 xref: EC:1.1.1.64 {status="skos:exactMatch"}
 xref: KEGG_REACTION:R01838
 xref: MetaCyc:1.1.1.64-RXN
+xref: Reactome:R-HSA-193064 "HSD17B3-like proteins reducde ANDST to TEST"
 xref: RHEA:14981
 xref: Wikipedia:Testosterone_17beta-dehydrogenase_(NADP+)
 is_a: GO:0072582 ! 17-beta-hydroxysteroid dehydrogenase (NADP+) activity
@@ -302698,6 +303345,7 @@
 synonym: "P2C reductase activity" RELATED [EC:1.5.1.21]
 xref: EC:1.5.1.21
 xref: MetaCyc:RXN-8166
+xref: Reactome:R-HSA-5693347 "CRYM reduces P2C to PPCA"
 xref: RHEA:12524
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.21
@@ -302730,7 +303378,6 @@
 synonym: "thiomorpholine-3-carboxylate:NAD(P)+ 5,6-oxidoreductase activity" RELATED [EC:1.5.1.25]
 xref: EC:1.5.1.25
 xref: MetaCyc:1.5.1.25-RXN
-xref: Reactome:R-HSA-5693347 "CRYM reduces P2C to PPCA"
 is_a: GO:0016646 ! oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.5.1.25
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -303156,7 +303803,7 @@
 id: GO:0047151
 name: tRNA (uracil(54)-C5)-methyltransferase activity, 5,10-methylenetetrahydrofolate-dependent
 namespace: molecular_function
-def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + tRNA containing uridine at position 54 + FADH + H+ = tetrahydrofolate + tRNA containing ribothymidine at position 54 + FAD+." [EC:2.1.1.74, MetaCyc:2.1.1.74-RXN]
+def: "Catalysis of the reaction: uridine54 in tRNA + (6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + NAD(P)H + H+ = 5-methyluridine54 in tRNA + (6S)-5,6,7,8-tetrahydrofolate + NAD(P)+." [EC:2.1.1.74]
 synonym: "5,10-methylenetetrahydrofolate:tRNA (uracil-5-)-methyltransferase activity" RELATED [EC:2.1.1.74]
 synonym: "5,10-methylenetetrahydrofolate:tRNA-UPsiC (uracil-5-)-methyl-transferase activity" RELATED [EC:2.1.1.74]
 synonym: "folate-dependent ribothymidyl synthase activity" RELATED [EC:2.1.1.74]
@@ -303167,9 +303814,11 @@
 xref: KEGG_REACTION:R03704
 xref: MetaCyc:2.1.1.74-RXN
 xref: RHEA:16873
+xref: RHEA:62372
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
 property_value: skos:exactMatch EC:2.1.1.74
-property_value: skos:exactMatch RHEA:16873
+property_value: skos:narrowMatch RHEA:16873
+property_value: skos:narrowMatch RHEA:62372
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26118" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -303976,7 +304625,7 @@
 id: GO:0047198
 name: L-cysteine-S-conjugate N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-substituted L-cysteine + acetyl-CoA = S-substituted N-acetyl-L-cysteine + CoA + H+." [EC:2.3.1.80, RHEA:19213]
+def: "Catalysis of the reaction: S-substituted L-cysteine + acetyl-CoA = S-substituted N-acetyl-L-cysteine + CoA + H+." [RHEA:19213]
 synonym: "acetyl-CoA:S-substituted L-cysteine N-acetyltransferase activity" RELATED [EC:2.3.1.80]
 synonym: "cysteine-S-conjugate N-acetyltransferase activity" EXACT []
 xref: EC:2.3.1.80
@@ -304008,7 +304657,7 @@
 id: GO:0047200
 name: tetrahydrodipicolinate N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + acetyl-CoA + H2O = L-2-acetamido-6-oxopimelate + CoA." [EC:2.3.1.89, RHEA:13085]
+def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + acetyl-CoA + H2O = L-2-acetamido-6-oxopimelate + CoA." [RHEA:13085]
 synonym: "acetyl-CoA:(S)-2,3,4,5-tetrahydrodipicolinate-2,6-dicarboxylate N2-acetyltransferase activity" RELATED [EC:2.3.1.89]
 synonym: "acetyl-CoA:(S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate 2-N-acetyltransferase activity" RELATED [EC:2.3.1.89]
 synonym: "acetyl-CoA:L-2,3,4,5-tetrahydrodipicolinate N2-acetyltransferase activity" RELATED [EC:2.3.1.89]
@@ -304411,7 +305060,6 @@
 synonym: "uridine diphosphoacetylglucosamine-mucin beta(1->3)-acetylglucosaminyltransferase (elongating)" RELATED [EC:2.4.1.146]
 xref: EC:2.4.1.146
 xref: MetaCyc:2.4.1.146-RXN
-xref: Reactome:R-HSA-5617037 "POMGNT1 transfers GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 xref: Reactome:R-HSA-5617096 "Defective POMGNT1 does not transfer GlcNAc from UDP-GlcNAc to Man-O-Ser-DAG1"
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 is_a: GO:0140103 ! catalytic activity, acting on a glycoprotein
@@ -304431,6 +305079,7 @@
 synonym: "uridine diphosphoacetylglucosamine-mucin beta(1->3)-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.147]
 xref: EC:2.4.1.147
 xref: MetaCyc:2.4.1.147-RXN
+xref: Reactome:R-HSA-914010 "B3GNT6 transfers GlcNAc to Tn antigen"
 xref: RHEA:46880
 xref: RHEA:46884
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
@@ -304671,10 +305320,10 @@
 synonym: "uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II activity" RELATED [EC:2.4.1.175]
 xref: EC:2.4.1.175
 xref: MetaCyc:RXN-16325
-xref: Reactome:R-HSA-1971482 "The addition of GalNAc to the terminal glucuronate residue forms chondroitin"
-xref: Reactome:R-HSA-1971487 "CHPF,CHSY3 transfer GalNAc to chondroitin"
+xref: Reactome:R-HSA-1971482 "CSGALNACTs transfer GalNAc to CS linker glycan"
+xref: Reactome:R-HSA-1971487 "Chondroitin polymerase complex transfers GalNAc to CS chain"
 xref: Reactome:R-HSA-3595176 "Defective CHSY1 does not transfer GalNAc to chondroitin"
-xref: Reactome:R-HSA-9632033 "CHSY1 transfers GalNAc to chondroitin"
+xref: Reactome:R-HSA-9632033 "CSGALNACTs transfer GalNAc to CS chain"
 is_a: GO:0008376 ! acetylgalactosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.175
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -305117,7 +305766,7 @@
 id: GO:0047263
 name: obsolete N-acylsphingosine galactosyltransferase activity
 namespace: molecular_function
-def: "OBSOLETE. Catalysis of the reaction: ceramide + UDP-galactose = D-galactosylceramide + UDP." [GOC:curator]
+def: "OBSOLETE. Catalysis of the reaction: ceramide + UDP-galactose = D-galactosylceramide + UDP." [GOC:curators]
 comment: The reason for obsoletion is that this term is equivalent to GO:0003851.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27410" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28173" xsd:anyURI
@@ -305550,6 +306199,7 @@
 synonym: "monosialoganglioside sialyltransferase" EXACT [EC:2.4.3.2]
 xref: EC:2.4.3.2
 xref: MetaCyc:RXN-18256
+xref: Reactome:R-HSA-981497 "ST3GAL1-4 transfers sialic acid to the T antigen at the alpha 3 position"
 xref: RHEA:52380
 is_a: GO:0008373 ! sialyltransferase activity
 property_value: skos:exactMatch EC:2.4.3.2
@@ -305689,39 +306339,41 @@
 id: GO:0047296
 name: homospermidine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 putrescine = NH3 + sym-homospermidine." [EC:2.5.1.44]
+def: "Catalysis of the reaction: 2 putrescine = sym-homospermidine + NH4+." [RHEA:18645]
 synonym: "putrescine:putrescine 4-aminobutyltransferase (ammonia-forming)" RELATED [EC:2.5.1.44]
-xref: EC:2.5.1.44
 xref: MetaCyc:2.5.1.44-RXN
 xref: RHEA:18645
 is_a: GO:0016765 ! transferase activity, transferring alkyl or aryl (other than methyl) groups
-property_value: skos:exactMatch EC:2.5.1.44
+property_value: skos:broadMatch EC:2.5.1.44
 property_value: skos:exactMatch RHEA:18645
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0047297
-name: asparagine-oxo-acid transaminase activity
+name: L-asparagine:oxo-acid transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2-oxo acid + L-asparagine = an amino acid + 2-oxosuccinamate." [RHEA:19813]
+def: "Catalysis of the reaction: L-asparagine + a 2-oxocarboxylate = 2-oxosuccinamate + an L-alpha-amino acid." [RHEA:19813]
 synonym: "asparagine--oxo-acid aminotransferase activity" RELATED [EC:2.6.1.14]
 synonym: "asparagine-keto acid aminotransferase activity" RELATED [EC:2.6.1.14]
 synonym: "asparagine-oxo-acid aminotransferase activity" EXACT []
+synonym: "asparagine-oxo-acid transaminase activity" EXACT []
 synonym: "L-asparagine:2-oxo-acid aminotransferase activity" RELATED [EC:2.6.1.14]
 xref: EC:2.6.1.14
 xref: MetaCyc:2.6.1.14-RXN
 xref: RHEA:19813
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.14
 property_value: skos:exactMatch RHEA:19813
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047298
-name: (S)-3-amino-2-methylpropionate transaminase activity
+name: (S)-3-amino-2-methylpropionate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate." [EC:2.6.1.22, RHEA:13993]
+def: "Catalysis of the reaction: (S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate." [RHEA:13993]
 synonym: "(S)-3-amino-2-methylpropanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.22]
 synonym: "(S)-3-amino-2-methylpropionate aminotransferase activity" EXACT []
+synonym: "(S)-3-amino-2-methylpropionate transaminase activity" EXACT []
 synonym: "beta-aminobutyric transaminase activity" RELATED [EC:2.6.1.22]
 synonym: "beta-aminoisobutyrate-alpha-ketoglutarate transaminase activity" RELATED [EC:2.6.1.22]
 synonym: "L-3-aminoisobutyrate aminotransferase activity" RELATED [EC:2.6.1.22]
@@ -305736,32 +306388,36 @@
 property_value: skos:exactMatch EC:2.6.1.22
 property_value: skos:exactMatch RHEA:13993
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047299
-name: tryptophan-phenylpyruvate transaminase activity
+name: L-tryptophan:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-tryptophan = 3-(indol-3-yl)pyruvate + L-phenylalanine." [EC:2.6.1.28, RHEA:13741]
+def: "Catalysis of the reaction: 3-phenylpyruvate + L-tryptophan = indole-3-pyruvate + L-phenylalanine." [RHEA:13741]
 synonym: "L-tryptophan-alpha-ketoisocaproate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "L-tryptophan:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "tryptophan--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.28]
 synonym: "tryptophan-phenylpyruvate aminotransferase activity" EXACT []
+synonym: "tryptophan-phenylpyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.28
 xref: KEGG_REACTION:R01376
 xref: MetaCyc:2.6.1.28-RXN
 xref: RHEA:13741
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.28
 property_value: skos:exactMatch RHEA:13741
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047300
-name: pyridoxamine-pyruvate transaminase activity
+name: pyridoxamine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyridoxamine + pyruvate = L-alanine + pyridoxal." [EC:2.6.1.30, RHEA:12841]
+def: "Catalysis of the reaction: pyridoxamine + pyruvate = L-alanine + pyridoxal." [RHEA:12841]
 synonym: "pyridoxamine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.30]
 synonym: "pyridoxamine-pyruvate aminotransferase activity" EXACT []
+synonym: "pyridoxamine-pyruvate transaminase activity" EXACT []
 synonym: "pyridoxamine-pyruvic transaminase" BROAD [EC:2.6.1.30]
 synonym: "pyridoxamine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.30]
 synonym: "pyridoxamineu-pyruvic transaminase activity" RELATED [EC:2.6.1.30]
@@ -305773,32 +306429,36 @@
 property_value: skos:exactMatch EC:2.6.1.30
 property_value: skos:exactMatch RHEA:12841
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047301
-name: valine-3-methyl-2-oxovalerate transaminase activity
+name: L-valine:3-methyl-2-oxovalerate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + L-valine = 3-methyl-2-oxobutanoate + L-isoleucine." [EC:2.6.1.32, RHEA:11468]
+def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + L-valine = 3-methyl-2-oxobutanoate + L-isoleucine." [RHEA:11468]
 synonym: "L-valine:(S)-3-methyl-2-oxopentanoate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--3-methyl-2-oxovalerate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--isoleucine aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine--isoleucine transaminase activity" RELATED [EC:2.6.1.32]
 synonym: "valine-2-keto-methylvalerate aminotransferase activity" RELATED [EC:2.6.1.32]
 synonym: "valine-3-methyl-2-oxovalerate aminotransferase activity" EXACT []
+synonym: "valine-3-methyl-2-oxovalerate transaminase activity" EXACT []
 xref: EC:2.6.1.32
 xref: KEGG_REACTION:R02200
 xref: MetaCyc:2.6.1.32-RXN
 xref: RHEA:11468
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.32
 property_value: skos:exactMatch RHEA:11468
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047302
-name: UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose transaminase activity
+name: UDP-N-acetylbacillosamine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + UDP-2-acetamido-4-amino-2,4,6-trideoxy-D-glucose = L-glutamate + UDP-2-acetamido-4-dehydro-2,6-dideoxy-beta-D-glucose." [PMID:16286454, RHEA:31663]
+def: "Catalysis of the reaction: UDP-N-acetylbacillosamine + 2-oxoglutarate = UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-hex-4-ulose + L-glutamate." [PMID:16286454, RHEA:31663]
+synonym: "UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose transaminase activity" BROAD []
 synonym: "UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.34]
 synonym: "UDP-4-amino-2-acetamido-2,4,6-trideoxyglucose aminotransferase activity" EXACT []
 synonym: "UDP-4-amino-2-acetamido-2,4,6-trideoxyglucose transaminase activity" EXACT []
@@ -305811,30 +306471,33 @@
 property_value: skos:exactMatch EC:2.6.1.34
 property_value: skos:exactMatch RHEA:31663
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047303
-name: glycine-oxaloacetate transaminase activity
+name: glycine:oxaloacetate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + oxaloacetate = L-aspartate + glyoxylate." [EC:2.6.1.35, RHEA:17141]
+def: "Catalysis of the reaction: glycine + oxaloacetate = L-aspartate + glyoxylate." [RHEA:17141]
 synonym: "glycine--oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 synonym: "glycine-oxalacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 synonym: "glycine-oxaloacetate aminotransferase activity" EXACT []
+synonym: "glycine-oxaloacetate transaminase activity" EXACT []
 synonym: "glycine:oxaloacetate aminotransferase activity" RELATED [EC:2.6.1.35]
 xref: EC:2.6.1.35
 xref: KEGG_REACTION:R00373
 xref: MetaCyc:2.6.1.35-RXN
 xref: RHEA:17141
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.35
 property_value: skos:exactMatch RHEA:17141
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047304
-name: 2-aminoethylphosphonate-pyruvate transaminase activity
+name: 2-aminoethylphosphonate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (2-aminoethyl)phosphonate + pyruvate = L-alanine + phosphonoacetaldehyde." [EC:2.6.1.37, RHEA:17021]
+def: "Catalysis of the reaction: (2-aminoethyl)phosphonate + pyruvate = L-alanine + phosphonoacetaldehyde." [RHEA:17021]
 synonym: "(2-aminoethyl)phosphonate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "(2-aminoethyl)phosphonate transaminase activity" RELATED [EC:2.6.1.37]
 synonym: "(2-aminoethyl)phosphonate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.37]
@@ -305843,6 +306506,7 @@
 synonym: "2-aminoethylphosphonate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "2-aminoethylphosphonate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.37]
 synonym: "2-aminoethylphosphonate-pyruvate aminotransferase activity" EXACT []
+synonym: "2-aminoethylphosphonate-pyruvate transaminase activity" EXACT []
 xref: EC:2.6.1.37
 xref: KEGG_REACTION:R04152
 xref: MetaCyc:2.6.1.37-RXN
@@ -305851,20 +306515,21 @@
 property_value: skos:exactMatch EC:2.6.1.37
 property_value: skos:exactMatch RHEA:17021
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047305
-name: (R)-3-amino-2-methylpropionate-pyruvate transaminase activity
+name: (R)-3-amino-2-methylpropionate:pyruvate transaminase activity
 namespace: molecular_function
 alt_id: GO:0047314
-def: "Catalysis of the reaction: (2R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate + L-alanine." [EC:2.6.1.40, RHEA:18393]
-comment: Note that this function was EC:2.6.1.61.
+def: "Catalysis of the reaction: (R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate + L-alanine." [RHEA:18393]
 synonym: "(R)-3-amino-2-methylpropanoate aminotransferase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropanoate transaminase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropanoate:pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "(R)-3-amino-2-methylpropionate transaminase activity" EXACT []
 synonym: "(R)-3-amino-2-methylpropionate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "(R)-3-amino-2-methylpropionate-pyruvate aminotransferase activity" EXACT []
+synonym: "(R)-3-amino-2-methylpropionate-pyruvate transaminase activity" EXACT []
 synonym: "beta-aminoisobutyrate--pyruvate transaminase activity" RELATED [EC:2.6.1.40]
 synonym: "beta-aminoisobutyrate-pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
 synonym: "D-3-aminoisobutyrate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.40]
@@ -305881,33 +306546,37 @@
 property_value: skos:exactMatch EC:2.6.1.40
 property_value: skos:exactMatch RHEA:18393
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047306
-name: D-methionine-pyruvate transaminase activity
+name: D-methionine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine." [EC:2.6.1.41, RHEA:23836]
+def: "Catalysis of the reaction: D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine." [RHEA:23836]
 synonym: "D-methionine aminotransferase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine transaminase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine--pyruvate aminotransferase activity" RELATED [EC:2.6.1.41]
 synonym: "D-methionine-pyruvate aminotransferase activity" EXACT []
+synonym: "D-methionine-pyruvate transaminase activity" EXACT []
 synonym: "D-methionine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.41]
 xref: EC:2.6.1.41
 xref: KEGG_REACTION:R03001
 xref: MetaCyc:2.6.1.41-RXN
 xref: RHEA:23836
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.41
 property_value: skos:exactMatch RHEA:23836
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047307
-name: diaminobutyrate-pyruvate transaminase activity
+name: L-2,4-diaminobutyrate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-2,4-diaminobutyrate + pyruvate = L-alanine + L-aspartate 4-semialdehyde." [EC:2.6.1.46, RHEA:12380]
+def: "Catalysis of the reaction: L-2,4-diaminobutyrate + pyruvate = L-alanine + L-aspartate 4-semialdehyde." [RHEA:12380]
 synonym: "diaminobutyrate--pyruvate aminotransferase activity" RELATED [EC:2.6.1.46]
 synonym: "diaminobutyrate-pyruvate aminotransferase activity" EXACT []
+synonym: "diaminobutyrate-pyruvate transaminase activity" EXACT []
 synonym: "L-2,4-diaminobutanoate:pyruvate aminotransferase activity" RELATED [EC:2.6.1.46]
 synonym: "L-diaminobutyric acid transaminase activity" RELATED [EC:2.6.1.46]
 xref: EC:2.6.1.46
@@ -305918,12 +306587,13 @@
 property_value: skos:exactMatch EC:2.6.1.46
 property_value: skos:exactMatch RHEA:12380
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047308
-name: alanine-oxomalonate transaminase activity
+name: L-alanine:oxomalonate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + oxomalonate = aminomalonate + pyruvate." [EC:2.6.1.47, RHEA:18809]
+def: "Catalysis of the reaction: L-alanine + oxomalonate = aminomalonate + pyruvate." [RHEA:18809]
 synonym: "alanine--oxomalonate aminotransferase activity" RELATED [EC:2.6.1.47]
 synonym: "alanine-ketomalonate (mesoxalate) transaminase activity" RELATED [EC:2.6.1.47]
 synonym: "alanine-oxomalonate aminotransferase activity" EXACT []
@@ -305933,19 +306603,21 @@
 xref: KEGG_REACTION:R02970
 xref: MetaCyc:2.6.1.47-RXN
 xref: RHEA:18809
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.47
 property_value: skos:exactMatch RHEA:18809
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047309
-name: dihydroxyphenylalanine transaminase activity
+name: dihydroxyphenylalanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-dopa = 3,4-dihydroxyphenylpyruvate + L-glutamate." [EC:2.6.1.49, RHEA:15273]
+def: "Catalysis of the reaction: L-dopa + 2-oxoglutarate = 3-(3,4-dihydroxyphenyl)pyruvate + L-glutamate." [RHEA:15273]
 synonym: "3,4-dihydroxy-L-phenylalanine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.49]
 synonym: "aspartate-DOPP transaminase (ADT)" RELATED [EC:2.6.1.49]
 synonym: "dihydroxyphenylalanine aminotransferase activity" EXACT []
+synonym: "dihydroxyphenylalanine transaminase activity" EXACT []
 synonym: "DOPA aminotransferase activity" RELATED [EC:2.6.1.49]
 synonym: "dopa transaminase activity" RELATED [EC:2.6.1.49]
 synonym: "glutamate-DOPP transaminase (GDT)" RELATED [EC:2.6.1.49]
@@ -305955,19 +306627,20 @@
 xref: KEGG_REACTION:R02077
 xref: MetaCyc:2.6.1.49-RXN
 xref: RHEA:15273
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.49
 property_value: skos:exactMatch RHEA:15273
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
 id: GO:0047310
-name: glutamine-scyllo-inositol transaminase activity
+name: L-glutamine:scyllo-inositol transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4,6/3,5-pentahydroxycyclohexanone + L-glutamine = 1-amino-1-deoxy-scyllo-inositol + 2-oxoglutaramate." [EC:2.6.1.50, RHEA:22920]
+def: "Catalysis of the reaction: scyllo-inosose + L-glutamine = 1-amino-1-deoxy-scyllo-inositol + 2-oxoglutaramate." [RHEA:22920]
 synonym: "glutamine scyllo-inosose aminotransferase activity" RELATED [EC:2.6.1.50]
 synonym: "glutamine--scyllo-inosose aminotransferase activity" RELATED [EC:2.6.1.50]
 synonym: "glutamine--scyllo-inosose transaminase activity" RELATED [EC:2.6.1.50]
+synonym: "glutamine-scyllo-inositol transaminase activity" EXACT []
 synonym: "glutamine-scyllo-inosose aminotransferase activity" EXACT []
 synonym: "glutamine-scyllo-inosose transaminase activity" EXACT []
 synonym: "L-glutamine-keto-scyllo-inositol aminotransferase activity" RELATED [EC:2.6.1.50]
@@ -305977,17 +306650,19 @@
 xref: KEGG_REACTION:R02781
 xref: MetaCyc:2.6.1.50-RXN
 xref: RHEA:22920
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.50
 property_value: skos:exactMatch RHEA:22920
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047311
-name: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase activity
+name: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine." [EC:2.6.1.56, RHEA:15497]
+def: "Catalysis of the reaction: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine." [RHEA:15497]
 synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol aminotransferase activity" EXACT []
+synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase activity" EXACT []
 synonym: "1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol:pyruvate aminotransferase activity" RELATED [EC:2.6.1.56]
 synonym: "guanidinoaminodideoxy-scyllo-inositol-pyruvate aminotransferase activity" RELATED [EC:2.6.1.56]
 synonym: "L-alanine-N-amidino-3-(or 5-)keto-scyllo-inosamine transaminase activity" RELATED [EC:2.6.1.56]
@@ -305999,53 +306674,58 @@
 property_value: skos:exactMatch EC:2.6.1.56
 property_value: skos:exactMatch RHEA:15497
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047312
-name: L-phenylalanine-pyruvate transaminase activity
+name: L-phenylalanine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: pyruvate + L-phenylalanine = phenylpyruvate + L-alanine." [RHEA:13053]
+def: "Catalysis of the reaction: L-phenylalanine + pyruvate = 3-phenylpyruvate + L-alanine." [RHEA:13053]
 synonym: "histidine aminotransferase activity" RELATED [GOC:kad]
 synonym: "histidine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.58, GOC:kad]
 synonym: "L-histidine:pyruvate aminotransferase activity" RELATED [GOC:kad]
 synonym: "L-phenylalanine(L-histidine):pyruvate aminotransferase activity" BROAD [EC:2.6.1.58]
+synonym: "L-phenylalanine-pyruvate transaminase activity" EXACT []
 synonym: "L-phenylalanine:pyruvate aminotransferase activity" EXACT []
-synonym: "L-phenylalanine:pyruvate transaminase activity" EXACT [GOC:kad]
 synonym: "phenylalanine (histidine) aminotransferase activity" BROAD [EC:2.6.1.58]
 synonym: "phenylalanine(histidine) aminotransferase activity" BROAD []
 synonym: "phenylalanine(histidine) transaminase activity" BROAD [EC:2.6.1.58]
 synonym: "phenylalanine(histidine):pyruvate aminotransferase activity" BROAD [EC:2.6.1.58]
 xref: KEGG_REACTION:R00692
 xref: MetaCyc:2.6.1.58-RXN
-xref: Reactome:R-HSA-893593 "phenylalanine + pyruvate => 3-(indol-3-yl)pyruvate + alanine"
+xref: Reactome:R-HSA-893593 "phenylalanine + pyruvate => keto-phenylpyruvate + alanine"
 xref: RHEA:13053
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:broadMatch EC:2.6.1.58
 property_value: skos:exactMatch RHEA:13053
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047313
-name: aromatic-amino-acid-glyoxylate transaminase activity
+name: aromatic-amino-acid:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glyoxylate + an aromatic amino acid = L-glycine + an aromatic oxo acid." [EC:2.6.1.60, MetaCyc:2.6.1.60-RXN]
+def: "Catalysis of the reaction: an aromatic L-alpha-amino acid + glyoxylate = an aromatic oxo-acid + glycine." [RHEA:10900]
 synonym: "aromatic-amino-acid--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.60]
 synonym: "aromatic-amino-acid-glyoxylate aminotransferase activity" EXACT []
+synonym: "aromatic-amino-acid-glyoxylate transaminase activity" EXACT []
 synonym: "aromatic-amino-acid:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.60]
 xref: EC:2.6.1.60
 xref: MetaCyc:2.6.1.60-RXN
 xref: RHEA:10900
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.60
 property_value: skos:exactMatch RHEA:10900
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047315
-name: kynurenine-glyoxylate transaminase activity
+name: L-kynurenine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate + glycine." [EC:2.6.1.63, RHEA:19249]
+def: "Catalysis of the reaction: L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate + glycine." [RHEA:19249]
 synonym: "kynurenine--glyoxylate aminotransferase activity" RELATED [EC:2.6.1.63]
 synonym: "kynurenine-glyoxylate aminotransferase activity" EXACT []
+synonym: "kynurenine-glyoxylate transaminase activity" EXACT []
 synonym: "L-kynurenine:glyoxylate aminotransferase (cyclizing)" RELATED [EC:2.6.1.63]
 xref: EC:2.6.1.63
 xref: KEGG_REACTION:R01957
@@ -306055,34 +306735,38 @@
 property_value: skos:exactMatch EC:2.6.1.63
 property_value: skos:exactMatch RHEA:19249
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047316
-name: glutamine-phenylpyruvate transaminase activity
+name: L-glutamine:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-glutamine = 2-oxoglutaramate + L-phenylalanine." [EC:2.6.1.64, RHEA:17593]
+def: "Catalysis of the reaction: 3-phenylpyruvate + L-glutamine = 2-oxoglutaramate + L-phenylalanine." [RHEA:17593]
 synonym: "glutamine transaminase K activity" RELATED [EC:2.6.1.64]
 synonym: "glutamine--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.64]
 synonym: "glutamine-phenylpyruvate aminotransferase activity" EXACT []
+synonym: "glutamine-phenylpyruvate transaminase activity" EXACT []
 synonym: "L-glutamine:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.64]
 xref: EC:2.6.1.64
 xref: KEGG_REACTION:R01375
 xref: MetaCyc:2.6.1.64-RXN
 xref: RHEA:17593
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.64
 property_value: skos:exactMatch RHEA:17593
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047317
-name: N6-acetyl-beta-lysine transaminase activity
+name: N6-acetyl-beta-lysine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3-amino-6-acetamidohexanoate = L-glutamate + 3-oxo-6-acetamidohexanoate." [EC:2.6.1.65, MetaCyc:2.6.1.65-RXN]
+def: "Catalysis of the reaction: 2-oxoglutarate + 3-amino-6-acetamidohexanoate = L-glutamate + 3-oxo-6-acetamidohexanoate." [RHEA:16889]
 synonym: "6-acetamido-3-aminohexanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "epsilon-acetyl-beta-lysine aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "N(6)-acetyl-beta-lysine aminotransferase activity" RELATED [EC:2.6.1.65]
 synonym: "N6-acetyl-beta-lysine aminotransferase activity" EXACT []
+synonym: "N6-acetyl-beta-lysine transaminase activity" EXACT []
 xref: EC:2.6.1.65
 xref: MetaCyc:2.6.1.65-RXN
 xref: RHEA:16889
@@ -306090,29 +306774,33 @@
 property_value: skos:exactMatch EC:2.6.1.65
 property_value: skos:exactMatch RHEA:16889
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047319
-name: aspartate-phenylpyruvate transaminase activity
+name: L-aspartate:phenylpyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: keto-phenylpyruvate + L-aspartate = L-phenylalanine + oxaloacetate." [RHEA:14097]
+def: "Catalysis of the reaction: L-aspartate + 3-phenylpyruvate = oxaloacetate + L-phenylalanine." [RHEA:14097]
 synonym: "aspartate--phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.70]
+synonym: "aspartate-phenylpyruvate transaminase activity" EXACT []
 synonym: "aspartate:phenylpyruvate aminotransferase activity" EXACT []
 synonym: "L-aspartate:phenylpyruvate aminotransferase activity" RELATED [EC:2.6.1.70]
 xref: EC:2.6.1.70
 xref: KEGG_REACTION:R00695
 xref: MetaCyc:2.6.1.70-RXN
 xref: RHEA:14097
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.70
 property_value: skos:exactMatch RHEA:14097
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047320
-name: D-4-hydroxyphenylglycine transaminase activity
+name: D-4-hydroxyphenylglycine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + D-4-hydroxyphenylglycine = 4-hydroxyphenylglyoxylate + L-glutamate." [EC:2.6.1.72, RHEA:15589]
+def: "Catalysis of the reaction: 2-oxoglutarate + D-4-hydroxyphenylglycine = 4-hydroxyphenylglyoxylate + L-glutamate." [RHEA:15589]
 synonym: "D-4-hydroxyphenylglycine aminotransferase activity" EXACT []
+synonym: "D-4-hydroxyphenylglycine transaminase activity" EXACT []
 synonym: "D-4-hydroxyphenylglycine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.72]
 synonym: "D-hydroxyphenylglycine aminotransferase activity" RELATED [EC:2.6.1.72]
 xref: EC:2.6.1.72
@@ -306123,6 +306811,7 @@
 property_value: skos:exactMatch EC:2.6.1.72
 property_value: skos:exactMatch RHEA:15589
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047321
@@ -306190,7 +306879,6 @@
 synonym: "STK2" RELATED [EC:2.7.11.4]
 xref: EC:2.7.11.4
 xref: MetaCyc:2.7.11.4-RXN
-xref: Reactome:R-HSA-5693148 "BCKDK phosphorylates BCKDH"
 xref: Reactome:R-HSA-9912480 "BCKDK loss-of-function mutations do not phosphorylate BCKDH"
 xref: RHEA:17301
 is_a: GO:0004674 ! protein serine/threonine kinase activity
@@ -306299,7 +306987,7 @@
 id: GO:0047329
 name: phosphoramidate-hexose phosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: hexose + phosphoramidate = hexose 1-phosphate + NH3." [EC:2.7.1.62, MetaCyc:2.7.1.62-RXN]
+def: "Catalysis of the reaction: phosphoramidate + a D-hexose + H+ = alpha-D-hexose 1-phosphate + NH4+." [RHEA:10972]
 synonym: "phosphoramidate-hexose transphosphorylase activity" RELATED [EC:2.7.1.62]
 synonym: "phosphoramidate:hexose 1-phosphotransferase activity" RELATED [EC:2.7.1.62]
 synonym: "phosphoramidic-hexose transphosphorylase activity" RELATED [EC:2.7.1.62]
@@ -306687,6 +307375,7 @@
 xref: EC:2.7.7.40
 xref: KEGG_REACTION:R02921
 xref: MetaCyc:2.7.7.40-RXN
+xref: Reactome:R-HSA-9940844 "CRPPA dimer transfers cytidylyl to Rbt5P"
 xref: RHEA:12456
 is_a: GO:0070567 ! cytidylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.40
@@ -307335,6 +308024,7 @@
 synonym: "branched-chain oxo-acid dehydrogenase phosphatase activity" EXACT []
 xref: EC:3.1.3.52
 xref: MetaCyc:3.1.3.52-RXN
+xref: Reactome:R-HSA-5693153 "PPM1K dephosphorylates p-BCKDH"
 xref: RHEA:77247
 is_a: GO:0004721 ! phosphoprotein phosphatase activity
 property_value: skos:exactMatch EC:3.1.3.52
@@ -307767,7 +308457,7 @@
 id: GO:0047411
 name: 2-(acetamidomethylene)succinate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + 2-(acetamidomethylene)succinate = CO2 + NH3 + succinate semialdehyde + acetate." [EC:3.5.1.29, MetaCyc:3.5.1.29-RXN]
+def: "Catalysis of the reaction: 2-(acetamidomethylene)succinate + 2 H2O + H+ = succinate semialdehyde + acetate + NH4+ + CO2." [RHEA:10432]
 synonym: "2-(acetamidomethylene)succinate amidohydrolase (deaminating, decarboxylating)" RELATED [EC:3.5.1.29]
 synonym: "alpha-(N-acetylaminomethylene)succinic acid hydrolase activity" RELATED [EC:3.5.1.29]
 xref: EC:3.5.1.29
@@ -307867,7 +308557,7 @@
 id: GO:0047417
 name: N-carbamoyl-D-amino acid hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + N-carbamoyl-D-amino acid = CO2 + NH3 + D-amino acid." [EC:3.5.1.77, MetaCyc:3.5.1.77-RXN]
+def: "Catalysis of the reaction: an N-carbamoyl-D-amino acid + H2O + 2 H+ = a D-alpha-amino acid + NH4+ + CO2." [RHEA:11000]
 synonym: "N-carbamoyl-D-amino acid amidohydrolase activity" RELATED [EC:3.5.1.77]
 xref: EC:3.5.1.77
 xref: MetaCyc:3.5.1.77-RXN
@@ -307974,7 +308664,7 @@
 id: GO:0047424
 name: methylenediurea deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + methylenediurea = CO2 + 2 NH3 + N-hydroxymethylurea." [EC:3.5.3.21, MetaCyc:3.5.3.21-RXN, RHEA:15929]
+def: "Catalysis of the reaction: methylenediurea + 2 H2O + 2 H+ = N-(hydroxymethyl)urea + 2 NH4+ + CO2." [RHEA:15929]
 synonym: "methylenediurea aminohydrolase activity" RELATED [EC:3.5.3.21]
 synonym: "methylenediurease activity" RELATED [EC:3.5.3.21]
 xref: EC:3.5.3.21
@@ -308007,7 +308697,7 @@
 id: GO:0047426
 name: ricinine nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + ricinine = NH3 + 3-carboxy-4-methoxy-N-methyl-2-pyridone." [EC:3.5.5.2]
+def: "Catalysis of the reaction: 	ricinine + 2 H2O = 4-methoxy-1-methyl-2-oxo-1,2-dihydropyridine-3-carboxylate + NH4+." [RHEA:22704]
 synonym: "ricinine aminohydrolase activity" RELATED [EC:3.5.5.2]
 xref: EC:3.5.5.2
 xref: MetaCyc:3.5.5.2-RXN
@@ -308037,7 +308727,7 @@
 id: GO:0047428
 name: arylacetonitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 H2O + 4-chlorophenylacetonitrile = 4-chlorophenylacetate + NH3." [EC:3.5.5.5]
+def: "Catalysis of the reaction: 4-chlorophenylacetonitrile + 2 H2O = 4-chlorophenylacetate + NH4+." [RHEA:20657]
 synonym: "arylacetonitrile aminohydrolase activity" RELATED [EC:3.5.5.5]
 xref: EC:3.5.5.5
 xref: MetaCyc:3.5.5.5-RXN
@@ -308379,7 +309069,7 @@
 id: GO:0047447
 name: erythro-3-hydroxyaspartate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: erythro-3-hydroxy-L-aspartate = NH3 + oxaloacetic acid." [EC:4.3.1.20, MetaCyc:4.3.1.20-RXN]
+def: "Catalysis of the reaction: 3-hydroxy-L-aspartate = oxaloacetate + NH4+." [RHEA:19757]
 synonym: "3-hydroxyaspartate dehydratase activity" BROAD [EC:4.3.1.20]
 synonym: "erythro-3-hydroxy-L(s)-aspartate hydro-lyase (deaminating) activity" RELATED [EC:4.3.1.20]
 synonym: "erythro-3-hydroxy-Ls-aspartate ammonia-lyase (oxaloacetate-forming)" RELATED [EC:4.3.1.20]
@@ -308844,11 +309534,13 @@
 id: GO:0047473
 name: D-alanine [D-alanyl carrier protein] ligase activity
 namespace: molecular_function
-def: "Catalysis of the ATP-dependent activation of D-alanine and its transfer as a thiol ester to the phosphopantheinyl prosthetic group of a D-alanyl carrier protein, according to the reaction: ATP + D-alanine + a [D-alaninyl carrier protein] = a D-alanyl-[D-alanyl carrier protein] + AMP + diphosphate." [GOC:pg]
+def: "Catalysis of the ATP-dependent activation of D-alanine and its transfer as a thiol ester to the phosphopantheinyl prosthetic group of a D-alanyl carrier protein, according to the reaction: holo-[D-alanyl-carrier protein] + D-alanine + ATP = D-alanyl-[D-alanyl-carrier protein] + AMP + diphosphate." [RHEA:55132]
 synonym: "D-alanine-activating enzyme activity" RELATED []
 synonym: "D-alanine-D-alanyl carrier protein ligase activity" RELATED []
 synonym: "D-alanine-poly(phosphoribitol) ligase activity" BROAD []
 synonym: "D-alanine:poly(phosphoribitol) ligase (AMP-forming)" BROAD []
+xref: EC:6.2.1.54
+xref: RHEA:55132
 is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14237" xsd:anyURI
 
@@ -308887,7 +309579,6 @@
 xref: MetaCyc:RXN-10819
 xref: RHEA:20956
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.30
 property_value: skos:exactMatch RHEA:20956
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -308905,7 +309596,6 @@
 xref: MetaCyc:6.2.1.28-RXN
 xref: RHEA:21776
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.28
 property_value: skos:exactMatch RHEA:21776
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -309774,13 +310464,17 @@
 id: GO:0047530
 name: 2,4-diaminopentanoate dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2,4-diaminopentanoate + H2O + NAD(P)+ = 2-amino-4-oxopentanoate + NH3 + NAD(P)H + H+." [EC:1.4.1.12, MetaCyc:24-DIAMINOPENTANOATE-DEHYDROGENASE-RXN]
+def: "Catalysis of the reaction: (2R,4S)-2,4-diaminopentanoate + NADP+ + H2O = (2R)-2-amino-4-oxopentanoate + NH4+ + NADPH + H+." [EC:1.4.1.12]
 synonym: "2,4-diaminopentanoate:NAD(P)+ oxidoreductase (deaminating)" RELATED [EC:1.4.1.12]
 synonym: "2,4-diaminopentanoic acid C4 dehydrogenase activity" RELATED [EC:1.4.1.12]
 xref: EC:1.4.1.12
 xref: MetaCyc:24-DIAMINOPENTANOATE-DEHYDROGENASE-RXN
+xref: RHEA:24192
+xref: RHEA:24196
 is_a: GO:0016639 ! oxidoreductase activity, acting on the CH-NH2 group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.4.1.12
+property_value: skos:narrowMatch RHEA:24192
+property_value: skos:narrowMatch RHEA:24196
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -309859,10 +310553,11 @@
 
 [Term]
 id: GO:0047536
-name: 2-aminoadipate transaminase activity
+name: L-2-aminoadipate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-2-aminoadipate = 2-oxoadipate + L-glutamate." [EC:2.6.1.39, RHEA:12601]
+def: "Catalysis of the reaction: L-2-aminoadipate + 2-oxoglutarate = 2-oxoadipate + L-glutamate." [RHEA:12601]
 synonym: "2-aminoadipate aminotransferase activity" EXACT []
+synonym: "2-aminoadipate transaminase activity" EXACT []
 synonym: "2-aminoadipic aminotransferase activity" RELATED [EC:2.6.1.39]
 synonym: "alpha-aminoadipate aminotransferase activity" RELATED [EC:2.6.1.39]
 synonym: "glutamate-alpha-ketoadipate transaminase activity" RELATED [EC:2.6.1.39]
@@ -309878,13 +310573,15 @@
 property_value: skos:exactMatch EC:2.6.1.39
 property_value: skos:exactMatch RHEA:12601
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047537
-name: 2-aminohexanoate transaminase activity
+name: L-2-aminohexanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate." [EC:2.6.1.67, MetaCyc:2-AMINOHEXANOATE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate." [RHEA:23600]
 synonym: "2-aminohexanoate aminotransferase activity" EXACT []
+synonym: "2-aminohexanoate transaminase activity" EXACT []
 synonym: "L-2-aminohexanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.67]
 synonym: "leucine L-norleucine: 2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.67]
 synonym: "norleucine (leucine) aminotransferase activity" RELATED [EC:2.6.1.67]
@@ -309897,6 +310594,7 @@
 property_value: skos:exactMatch EC:2.6.1.67
 property_value: skos:exactMatch RHEA:23600
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047538
@@ -309958,7 +310656,6 @@
 xref: MetaCyc:2-FUROATE--COA-LIGASE-RXN
 xref: RHEA:19269
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.31
 property_value: skos:exactMatch RHEA:19269
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -310625,19 +311322,21 @@
 
 [Term]
 id: GO:0047578
-name: 4-hydroxyglutamate transaminase activity
+name: (4S)-4-hydroxy-L-glutamate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-hydroxy-L-glutamate + 2-oxoglutarate = 4-hydroxy-2-oxoglutarate + L-glutamate." [EC:2.6.1.23, MetaCyc:4-HYDROXYGLUTAMATE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: (4S)-4-hydroxy-L-glutamate + 2-oxoglutarate = (4R)-4-hydroxy-2-oxoglutarate + L-glutamate." [RHEA:31235]
 synonym: "4-hydroxy-L-glutamate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.23]
 synonym: "4-hydroxyglutamate aminotransferase activity" EXACT []
+synonym: "4-hydroxyglutamate transaminase activity" EXACT []
 xref: EC:2.6.1.23
 xref: MetaCyc:4-HYDROXYGLUTAMATE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-6784393 "PXLP-K279-GOT2 dimer transaminates 4-OH-L-glutamate to 4-hydroxy-2-oxoglutarate (HOG)"
-xref: RHEA:10480
+xref: RHEA:31235
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch EC:2.6.1.23
-property_value: skos:exactMatch RHEA:10480
+property_value: skos:exactMatch RHEA:31235
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047579
@@ -310792,7 +311491,7 @@
 id: GO:0047588
 name: 5-aminopentanamidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-aminopentanamide + H2O = 5-aminopentanoate + NH3." [EC:3.5.1.30, MetaCyc:5-AMINOPENTANAMIDASE-RXN]
+def: "Catalysis of the reaction: 5-aminopentanamide + H2O = 5-aminopentanoate + NH4+." [RHEA:15677]
 synonym: "5-aminonorvaleramidase activity" RELATED [EC:3.5.1.30]
 synonym: "5-aminopentanamide amidohydrolase activity" RELATED [EC:3.5.1.30]
 synonym: "5-aminovaleramidase activity" RELATED [EC:3.5.1.30]
@@ -310806,11 +311505,12 @@
 
 [Term]
 id: GO:0047589
-name: 5-aminovalerate transaminase activity
+name: 5-aminovalerate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 5-aminopentanoate = 5-oxopentanoate + L-glutamate." [EC:2.6.1.48, RHEA:10212]
+def: "Catalysis of the reaction: 5-aminopentanoate + 2-oxoglutarate = 5-oxopentanoate + L-glutamate." [RHEA:10212]
 synonym: "5-aminopentanoate:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.48]
 synonym: "5-aminovalerate aminotransferase activity" EXACT []
+synonym: "5-aminovalerate transaminase activity" EXACT []
 synonym: "delta-aminovalerate aminotransferase activity" RELATED [EC:2.6.1.48]
 synonym: "delta-aminovalerate transaminase activity" RELATED [EC:2.6.1.48]
 xref: EC:2.6.1.48
@@ -310821,6 +311521,7 @@
 property_value: skos:exactMatch EC:2.6.1.48
 property_value: skos:exactMatch RHEA:10212
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047590
@@ -311196,7 +311897,7 @@
 xref: KEGG_REACTION:R00394
 xref: MetaCyc:ACID--COA-LIGASE-GDP-FORMING-RXN
 xref: RHEA:10968
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.10
 property_value: skos:exactMatch RHEA:10968
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -311393,7 +312094,7 @@
 id: GO:0047623
 name: adenosine-phosphate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an adenosine-phosphate + H20 = an inosine phosphate + NH3. Catalyzes the deamination of AMP, ADP or ATP." [EC:3.5.4.17]
+def: "Catalysis of the reaction: an adenosine-phosphate + H20 = an inosine phosphate + NH4+. Catalyzes the deamination of AMP, ADP or ATP." [EC:3.5.4.17]
 comment: Consider instead annotating to one of the more specific terms: AMP deaminase activity ; GO:0003876, ADP deaminase activity ; GO:0047629, or ATP deaminase activity ; GO:0047692.
 synonym: "adenine nucleotide deaminase activity" RELATED [EC:3.5.4.17]
 synonym: "adenosine (phosphate) deaminase activity" RELATED [EC:3.5.4.17]
@@ -311485,7 +312186,7 @@
 id: GO:0047629
 name: ADP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ADP + H2O = IDP + NH3." [EC:3.5.4.7, MetaCyc:ADP-DEAMINASE-RXN]
+def: "Catalysis of the reaction: ADP + H2O + H+ = IDP + NH4+." [RHEA:12741]
 synonym: "adenosine diphosphate deaminase activity" RELATED [EC:3.5.4.7]
 synonym: "adenosinepyrophosphate deaminase activity" RELATED [EC:3.5.4.7]
 synonym: "ADP aminohydrolase activity" RELATED [EC:3.5.4.7]
@@ -311539,7 +312240,7 @@
 id: GO:0047632
 name: agmatine deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: agmatine + H2O = N-carbamoylputrescine + NH3." [EC:3.5.3.12, MetaCyc:AGMATINE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: agmatine + H2O = N-carbamoylputrescine + NH4+." [RHEA:18037]
 synonym: "agmatine amidinohydrolase" BROAD [EC:3.5.3.12]
 synonym: "agmatine iminohydrolase activity" RELATED [EC:3.5.3.12]
 xref: EC:3.5.3.12
@@ -311588,23 +312289,25 @@
 
 [Term]
 id: GO:0047635
-name: alanine-oxo-acid transaminase activity
+name: L-alanine:oxo-acid transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-alanine + a 2-oxo acid = pyruvate + an L-amino acid." [EC:2.6.1.12, MetaCyc:ALANINE--OXO-ACID-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-alanine + a 2-oxocarboxylate = an L-alpha-amino acid + pyruvate." [RHEA:19953]
 synonym: "alanine--oxo-acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-keto acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-oxo acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "alanine-oxo-acid aminotransferase activity" EXACT []
+synonym: "GO Molecular Function help_outline	\nGO:0047635" EXACT []
 synonym: "L-alanine-alpha-keto acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "L-alanine:2-oxo-acid aminotransferase activity" RELATED [EC:2.6.1.12]
 synonym: "leucine-alanine transaminase activity" RELATED [EC:2.6.1.12]
 xref: EC:2.6.1.12
 xref: MetaCyc:ALANINE--OXO-ACID-AMINOTRANSFERASE-RXN
 xref: RHEA:19953
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.12
 property_value: skos:exactMatch RHEA:19953
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047636
@@ -311883,7 +312586,7 @@
 id: GO:0047652
 name: allantoate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: allantoate + H2O + H+ = CO2 + NH3 + ureidoglycine." [EC:3.5.3.9, MetaCyc:ALLANTOATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: allantoate + H2O + 2 H+ = (S)-2-ureidoglycine + NH4+ + CO2." [RHEA:27485]
 synonym: "allantoate amidinohydrolase (decarboxylating)" RELATED [EC:3.5.3.9]
 synonym: "allantoate amidohydrolase activity" RELATED [EC:3.5.3.9]
 xref: EC:3.5.3.9
@@ -312081,7 +312784,7 @@
 id: GO:0047664
 name: aminoimidazolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction:4-aminoimidazole + H20 = imidazol-4-one + NH3." [RHEA:22348]
+def: "Catalysis of the reaction: 4-aminoimidazole + H2O + H+ = imidazol-4-one + NH4+." [RHEA:22348]
 synonym: "4-aminoimidazole aminohydrolase activity" RELATED [EC:3.5.4.8]
 synonym: "4-aminoimidazole hydrolase activity" RELATED [EC:3.5.4.8]
 xref: EC:3.5.4.8
@@ -312094,9 +312797,9 @@
 
 [Term]
 id: GO:0047665
-name: aminolevulinate transaminase activity
+name: aminolevulinate:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine." [EC:2.6.1.43, RHEA:12480]
+def: "Catalysis of the reaction: 5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine." [RHEA:12480]
 synonym: "4,5-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "4,5-dioxovaleric acid aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "4,5-dioxovaleric acid transaminase activity" RELATED [EC:2.6.1.43]
@@ -312106,7 +312809,7 @@
 synonym: "alanine-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "alanine-gamma,delta-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
 synonym: "alanine:4,5-dioxovalerate aminotransferase activity" RELATED [EC:2.6.1.43]
-synonym: "aminolevulinate aminotransferase activity" EXACT []
+synonym: "aminolevulinate aminotransferase activity" BROAD []
 synonym: "aminolevulinic acid transaminase activity" RELATED [EC:2.6.1.43]
 synonym: "dioxovalerate transaminase activity" RELATED [EC:2.6.1.43]
 synonym: "DOVA transaminase activity" RELATED [EC:2.6.1.43]
@@ -312123,6 +312826,7 @@
 property_value: skos:exactMatch EC:2.6.1.43
 property_value: skos:exactMatch RHEA:12480
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047666
@@ -312562,15 +313266,14 @@
 id: GO:0047692
 name: ATP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ATP + H2O = ITP + NH3." [EC:3.5.4.18, MetaCyc:ATP-DEAMINASE-RXN]
+def: "Catalysis of the reaction: ATP + H2O + H+ = ITP + NH4+." [RHEA:13037]
 synonym: "adenosine triphosphate deaminase activity" RELATED [EC:3.5.4.18]
 synonym: "ATP aminohydrolase activity" RELATED [EC:3.5.4.18]
 xref: EC:3.5.4.18
 xref: MetaCyc:ATP-DEAMINASE-RXN
-xref: RHEA:13037
 is_a: GO:0047623 ! adenosine-phosphate deaminase activity
+property_value: skos:broadMatch RHEA:13037
 property_value: skos:exactMatch EC:3.5.4.18
-property_value: skos:exactMatch RHEA:13037
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -312679,7 +313382,7 @@
 id: GO:0047698
 name: beta-alanyl-CoA ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: beta-alanyl-CoA = acryloyl-CoA + NH3." [EC:4.3.1.6, MetaCyc:BETA-ALANYL-COA-AMMONIA-LYASE-RXN]
+def: "Catalysis of the reaction: beta-alanyl-CoA = acryloyl-CoA + NH4+." [RHEA:12416]
 synonym: "b-alanyl-CoA ammonia-lyase activity" EXACT []
 synonym: "beta-alanyl coenzyme A ammonia-lyase activity" RELATED [EC:4.3.1.6]
 synonym: "beta-alanyl-CoA ammonia-lyase (acryloyl-CoA-forming)" RELATED [EC:4.3.1.6]
@@ -312831,7 +313534,6 @@
 xref: MetaCyc:BIOTIN--COA-LIGASE-RXN
 xref: RHEA:19681
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.11
 property_value: skos:exactMatch RHEA:19681
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -312900,7 +313602,7 @@
 id: GO:0047711
 name: blasticidin-S deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: blasticidin S + H2O = deaminohydroxyblasticidin S + NH3." [EC:3.5.4.23, MetaCyc:BLASTICIDIN-S-DEAMINASE-RXN]
+def: "Catalysis of the reaction: blasticidin S + H2O + H+ = deaminohydroxyblasticidin S + NH4+." [RHEA:10148]
 synonym: "blasticidin-S aminohydrolase activity" RELATED [EC:3.5.4.23]
 xref: EC:3.5.4.23
 xref: MetaCyc:BLASTICIDIN-S-DEAMINASE-RXN
@@ -313153,16 +313855,17 @@
 id: GO:0047726
 name: iron-cytochrome-c reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ferrocytochrome c + Fe3+ = ferricytochrome c + Fe2+." [EC:1.9.98.1, MetaCyc:IRON--CYTOCHROME-C-REDUCTASE-RXN]
-synonym: "ferrocytochrome-c:Fe3+ oxidoreductase activity" RELATED [EC:1.9.98.1]
-synonym: "iron-cytochrome c reductase activity" RELATED [EC:1.9.98.1]
-xref: EC:1.9.98.1
+def: "Catalysis of the reaction: Fe3+ + Fe(II)-[cytochrome c] = Fe2+ + Fe(III)-[cytochrome c]." [EC:1.16.2.1, RHEA:15617]
+synonym: "ferrocytochrome-c:Fe3+ oxidoreductase activity" RELATED [EC:1.16.2.1]
+synonym: "iron-cytochrome c reductase activity" RELATED [EC:1.16.2.1]
+xref: EC:1.16.2.1
 xref: MetaCyc:IRON--CYTOCHROME-C-REDUCTASE-RXN
 xref: RHEA:15617
-is_a: GO:0016675 ! oxidoreductase activity, acting on a heme group of donors
-property_value: skos:exactMatch EC:1.9.98.1
+is_a: GO:0016722 ! oxidoreductase activity, acting on metal ions
+property_value: skos:exactMatch EC:1.16.2.1
 property_value: skos:exactMatch RHEA:15617
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31488" xsd:anyURI
 
 [Term]
 id: GO:0047727
@@ -313393,10 +314096,11 @@
 
 [Term]
 id: GO:0047740
-name: cephalosporin-C transaminase activity
+name: cephalosporin-C:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (7R)-7-(5-carboxylato-5-oxopentanamido)deacetylcephalosporanate + D-glutamate = 2-oxoglutarate + cephalosporin C." [EC:2.6.1.74, RHEA:14553]
+def: "Catalysis of the reaction: (7R)-7-(5-carboxylato-5-oxopentanamido)deacetylcephalosporanate + D-glutamate = 2-oxoglutarate + cephalosporin C." [RHEA:14553]
 synonym: "cephalosporin C aminotransferase activity" RELATED [EC:2.6.1.74]
+synonym: "cephalosporin-C transaminase activity" EXACT []
 synonym: "cephalosporin-C:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.74]
 synonym: "L-alanine:cephalosporin-C aminotransferase activity" RELATED [EC:2.6.1.74]
 xref: EC:2.6.1.74
@@ -313407,6 +314111,7 @@
 property_value: skos:exactMatch EC:2.6.1.74
 property_value: skos:exactMatch RHEA:14553
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047741
@@ -313535,7 +314240,6 @@
 xref: RHEA:47128
 xref: RHEA:47136
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.7
 property_value: skos:narrowMatch RHEA:22976
 property_value: skos:narrowMatch RHEA:23532
@@ -313681,7 +314385,7 @@
 synonym: "chondroitin 4-sulphotransferase activity" EXACT []
 xref: EC:2.8.2.5
 xref: MetaCyc:CHONDROITIN-4-SULFOTRANSFERASE-RXN
-xref: Reactome:R-HSA-1971483 "Chondroitin can be sulfated on position 4 of GalNAc by CHST9, 11, 12 and 13"
+xref: Reactome:R-HSA-1971483 "CHST11,12,13,9 transfer sulfate to (4-)GalNAc"
 xref: RHEA:16101
 is_a: GO:0034481 ! chondroitin sulfotransferase activity
 property_value: skos:exactMatch EC:2.8.2.5
@@ -313700,7 +314404,7 @@
 synonym: "urunosyl C-5 epimerase activity" RELATED [EC:5.1.3.19]
 xref: EC:5.1.3.19
 xref: MetaCyc:CHONDROITIN-GLUCURONATE-5-EPIMERASE-RXN
-xref: Reactome:R-HSA-2022052 "Dermatan-sulfate epimerase (DSE) converts chondroitin sulfate (CS) to dermatan sulfate (DS)"
+xref: Reactome:R-HSA-2022052 "DSE converts GlcA to IdoA"
 xref: RHEA:21084
 is_a: GO:0016857 ! racemase and epimerase activity, acting on carbohydrates and derivatives
 property_value: skos:exactMatch EC:5.1.3.19
@@ -314039,7 +314743,7 @@
 xref: KEGG_REACTION:R01322
 xref: MetaCyc:CITRATE--COA-LIGASE-RXN
 xref: RHEA:21472
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.18
 property_value: skos:exactMatch RHEA:21472
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -314062,7 +314766,7 @@
 id: GO:0047781
 name: citrullinase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + citrulline = NH3 + CO2 + L-ornithine." [EC:3.5.1.20, MetaCyc:CITRULLINASE-RXN]
+def: "Catalysis of the reaction: L-citrulline + H2O + 2 H+ = L-ornithine + NH4+ + CO2." [RHEA:11940]
 synonym: "citrulline hydrolase activity" RELATED [EC:3.5.1.20]
 synonym: "citrulline ureidase activity" RELATED [EC:3.5.1.20]
 synonym: "L-citrulline 5-N-carbamoyldihydrolase activity" RELATED [EC:3.5.1.20]
@@ -314248,7 +314952,7 @@
 id: GO:0047790
 name: creatinine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: creatinine + H2O = N-methylhydantoin + NH3." [EC:3.5.4.21, MetaCyc:CREATININE-DEAMINASE-RXN]
+def: "Catalysis of the reaction: creatinine + H2O + H+ = N-methylhydantoin + NH4+." [RHEA:12681]
 synonym: "creatinine desiminase activity" RELATED [EC:3.5.4.21]
 synonym: "creatinine hydrolase" BROAD [EC:3.5.4.21]
 synonym: "creatinine iminohydrolase activity" RELATED [EC:3.5.4.21]
@@ -314434,28 +315138,31 @@
 
 [Term]
 id: GO:0047801
-name: L-cysteine transaminase activity
+name: L-cysteine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-cysteine + 2-oxoglutarate = mercaptopyruvate + L-glutamate." [RHEA:17441]
+def: "Catalysis of the reaction: L-cysteine + 2-oxoglutarate = 2-oxo-3-sulfanylpropanoate + L-glutamate." [RHEA:17441]
 synonym: "cysteine aminotransferase activity" BROAD []
 synonym: "cysteine transaminase activity" EXACT [EC:2.6.1.3]
+synonym: "L-cysteine transaminase activity" BROAD []
 synonym: "L-cysteine:2-oxoglutarate aminotransferase activity" EXACT []
 xref: EC:2.6.1.3
 xref: MetaCyc:CYSTEINE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-9012597 "GOT2 dimer transfers amino group from L-Cys to 2OG to form 3MPYR and Glu"
 xref: RHEA:17441
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.3
 property_value: skos:exactMatch RHEA:17441
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047802
-name: cysteine-conjugate transaminase activity
+name: cysteine-conjugate:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + S-(4-bromophenyl)-L-cysteine = (4-bromophenylsulfanyl)pyruvate + L-glutamate." [EC:2.6.1.75, RHEA:13485]
+def: "Catalysis of the reaction: 2-oxoglutarate + S-(4-bromophenyl)-L-cysteine = (4-bromophenylsulfanyl)pyruvate + L-glutamate." [RHEA:13485]
 synonym: "cysteine conjugate aminotransferase activity" RELATED [EC:2.6.1.75]
 synonym: "cysteine-conjugate alpha-ketoglutarate transaminase (CAT-1)" RELATED [EC:2.6.1.75]
+synonym: "cysteine-conjugate transaminase activity" EXACT []
 synonym: "S-(4-bromophenyl)-L-cysteine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.75]
 xref: EC:2.6.1.75
 xref: KEGG_REACTION:R04338
@@ -314465,6 +315172,7 @@
 property_value: skos:exactMatch EC:2.6.1.75
 property_value: skos:exactMatch RHEA:13485
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047803
@@ -314607,11 +315315,12 @@
 
 [Term]
 id: GO:0047810
-name: D-alanine-2-oxoglutarate aminotransferase activity
+name: D-alanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: D-alanine + 2-oxoglutarate = pyruvate + D-glutamate." [EC:2.6.1.21]
+def: "Catalysis of the reaction: D-alanine + 2-oxoglutarate = pyruvate + D-glutamate." [RHEA:15869]
 synonym: "D-alanine aminotransferase activity" EXACT []
 synonym: "D-alanine transaminase activity" BROAD [EC:2.6.1.21]
+synonym: "D-alanine-2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "D-alanine-D-glutamate transaminase activity" RELATED [EC:2.6.1.21]
 synonym: "D-amino acid aminotransferase activity" BROAD [EC:2.6.1.21]
 synonym: "D-amino acid transaminase activity" BROAD [EC:2.6.1.21]
@@ -314622,10 +315331,11 @@
 xref: EC:2.6.1.21
 xref: MetaCyc:D-ALANINE-AMINOTRANSFERASE-RXN
 xref: RHEA:15869
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.21
 property_value: skos:exactMatch RHEA:15869
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047811
@@ -314653,10 +315363,11 @@
 xref: EC:2.3.1.36
 xref: MetaCyc:D-AMINO-ACID-N-ACETYLTRANSFERASE-RXN
 xref: RHEA:20704
-is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.36
 property_value: skos:exactMatch RHEA:20704
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047813
@@ -315057,9 +315768,11 @@
 xref: MetaCyc:D-TRYPTOPHAN-N-MALONYLTRANSFERASE-RXN
 xref: RHEA:23320
 is_a: GO:0050735 ! N-malonyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.112
 property_value: skos:exactMatch RHEA:23320
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047837
@@ -315320,7 +316033,7 @@
 xref: EC:6.2.1.23
 xref: MetaCyc:DICARBOXYLATE--COA-LIGASE-RXN
 xref: RHEA:14289
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.23
 property_value: skos:exactMatch RHEA:14289
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -315453,25 +316166,20 @@
 name: diiodophenylpyruvate reductase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 3-(3,5-diiodo-4-hydroxyphenyl)lactate + NAD+ = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + H+ + NADH." [RHEA:20293]
-synonym: "2-oxo acid reductase activity" RELATED [EC:1.1.1.96]
-synonym: "3-(3,5-diiodo-4-hydroxyphenyl)lactate:NAD+ oxidoreductase activity" RELATED [EC:1.1.1.96]
-synonym: "aromatic alpha-keto acid" RELATED [EC:1.1.1.96]
-synonym: "KAR" RELATED [EC:1.1.1.96]
-xref: EC:1.1.1.96
 xref: KEGG_REACTION:R03431
 xref: MetaCyc:DIIODOPHENYLPYRUVATE-REDUCTASE-RXN
 xref: RHEA:20293
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
-property_value: skos:exactMatch EC:1.1.1.96
 property_value: skos:exactMatch RHEA:20293
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30138" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31488" xsd:anyURI
 
 [Term]
 id: GO:0047861
-name: diiodotyrosine transaminase activity
+name: 3,5-diiodo-L-tyrosine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + 3,5-diiodo-L-tyrosine = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + L-glutamate." [EC:2.6.1.24, RHEA:19781]
+def: "Catalysis of the reaction: 2-oxoglutarate + 3,5-diiodo-L-tyrosine = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + L-glutamate." [RHEA:19781]
 synonym: "3,5-diiodo-L-tyrosine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.24]
 synonym: "diiodotyrosine aminotransferase activity" EXACT []
 synonym: "halogenated tyrosine aminotransferase activity" RELATED [EC:2.6.1.24]
@@ -315484,6 +316192,7 @@
 property_value: skos:exactMatch EC:2.6.1.24
 property_value: skos:exactMatch RHEA:19781
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047862
@@ -315871,7 +316580,7 @@
 id: GO:0047883
 name: ethanolamine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ethanolamine + H2O + O2 = glycolaldehyde + NH3 + H2O2." [RHEA:18581]
+def: "Catalysis of the reaction: ethanolamine + O2 + H2O = glycolaldehyde + H2O2 + NH4+." [RHEA:18581]
 synonym: "ethanolamine:oxygen oxidoreductase (deaminating)" RELATED [EC:1.4.3.8]
 xref: EC:1.4.3.8
 xref: MetaCyc:ETHANOLAMINE-OXIDASE-RXN
@@ -316672,7 +317381,7 @@
 name: glucosaminate ammonia-lyase activity
 namespace: molecular_function
 alt_id: GO:0050443
-def: "Catalysis of the reaction: D-glucosaminate = 2-dehydro-3-deoxy-D-gluconate + NH3." [EC:4.3.1.9]
+def: "Catalysis of the reaction: 2-amino-2-deoxy-D-gluconate = 2-dehydro-3-deoxy-D-gluconate + NH4+." [RHEA:12488]
 comment: Note that this function was EC:4.3.1.21.
 synonym: "2-amino-2-deoxy-D-gluconate ammonia-lyase activity" RELATED [EC:4.3.1.9]
 synonym: "2-amino-2-deoxy-D-gluconate hydro-lyase (deaminating) activity" RELATED [EC:4.3.1.9]
@@ -316933,9 +317642,9 @@
 
 [Term]
 id: GO:0047945
-name: L-glutamine:pyruvate aminotransferase activity
+name: L-glutamine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine." [EC:2.6.1.15, RHEA:10400]
+def: "Catalysis of the reaction: L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine." [RHEA:10400]
 synonym: "gamma-glutaminyltransferase activity" RELATED [EC:2.6.1.15]
 synonym: "glutaminase II activity" RELATED [EC:2.6.1.15]
 synonym: "glutamine transaminase activity" BROAD [EC:2.6.1.15]
@@ -316948,26 +317657,29 @@
 synonym: "glutamine-oxo acid aminotransferase activity" BROAD [EC:2.6.1.15]
 synonym: "glutamine-pyruvate transaminase activity" BROAD [EC:2.6.1.15]
 synonym: "L-glutamine transaminase L" RELATED [EC:2.6.1.15]
+synonym: "L-glutamine:pyruvate aminotransferase activity" EXACT []
 xref: EC:2.6.1.15
 xref: KEGG_REACTION:R00576
 xref: MetaCyc:GLUTAMINE--PYRUVATE-AMINOTRANSFERASE-RXN
 xref: Reactome:R-HSA-893616 "glutamine + pyruvate => 2-oxoglutaramate + alanine"
 xref: RHEA:10400
-is_a: GO:0070548 ! L-glutamine aminotransferase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.15
 property_value: skos:exactMatch RHEA:10400
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047946
-name: glutamine N-acyltransferase activity
+name: L-glutamine N-acyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine." [EC:2.3.1.68, MetaCyc:GLUTAMINE-N-ACYLTRANSFERASE-RXN]
+def: "Catalysis of the reaction: acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine." [RHEA:18469]
 synonym: "acyl-CoA:L-glutamine N-acyltransferase activity" RELATED [EC:2.3.1.68]
+synonym: "glutamine N-acyltransferase activity" BROAD []
 xref: EC:2.3.1.68
 xref: MetaCyc:GLUTAMINE-N-ACYLTRANSFERASE-RXN
 xref: RHEA:18469
-is_a: GO:0016410 ! N-acyltransferase activity
+is_a: GO:0140085 ! L-amino-acid N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.68
 property_value: skos:exactMatch RHEA:18469
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -317000,7 +317712,7 @@
 xref: KEGG_REACTION:R02402
 xref: MetaCyc:GLUTARATE--COA-LIGASE-RXN
 xref: RHEA:14169
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.6
 property_value: skos:exactMatch RHEA:14169
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -317020,16 +317732,17 @@
 id: GO:0047950
 name: glutathione oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2 glutathione + O2 = glutathione disulfide + H2O2." [EC:1.8.3.3, RHEA:24112]
+def: "Catalysis of the reaction: 2 glutathione + O2 = glutathione disulfide + H2O2." [RHEA:24112]
 synonym: "glutathione:oxygen oxidoreductase activity" RELATED [EC:1.8.3.3]
 xref: EC:1.8.3.3
 xref: KEGG_REACTION:R00120
 xref: MetaCyc:GLUTATHIONE-OXIDASE-RXN
 xref: RHEA:24112
-is_a: GO:0016670 ! oxidoreductase activity, acting on a sulfur group of donors, oxygen as acceptor
+is_a: GO:0016972 ! thiol oxidase activity
 property_value: skos:exactMatch EC:1.8.3.3
 property_value: skos:exactMatch RHEA:24112
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
 
 [Term]
 id: GO:0047951
@@ -317160,29 +317873,31 @@
 
 [Term]
 id: GO:0047958
-name: glycine:2-oxoglutarate aminotransferase activity
+name: glycine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + 2-oxoglutarate = glyoxylate + L-glutamate." [EC:2.6.1.4, MetaCyc:GLYCINE-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: glycine + 2-oxoglutarate = glyoxylate + L-glutamate." [RHEA:14089]
 synonym: "glutamate-glyoxylate transaminase activity" EXACT []
 synonym: "glutamic-glyoxylic transaminase activity" RELATED [EC:2.6.1.4]
 synonym: "glycine aminotransferase activity" BROAD []
 synonym: "glycine transaminase activity" BROAD [EC:2.6.1.4]
+synonym: "glycine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "glyoxylate-glutamate aminotransferase activity" RELATED [EC:2.6.1.4]
 synonym: "glyoxylate-glutamic transaminase activity" RELATED [EC:2.6.1.4]
 synonym: "L-glutamate:glyoxylate aminotransferase activity" EXACT []
 xref: EC:2.6.1.4
 xref: MetaCyc:GLYCINE-AMINOTRANSFERASE-RXN
 xref: RHEA:14089
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.4
 property_value: skos:exactMatch RHEA:14089
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0047959
 name: glycine dehydrogenase (cytochrome) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glycine + H2O + 2 ferricytochrome c = glyoxylate + NH3 + 2 ferrocytochrome c." [EC:1.4.2.1, MetaCyc:GLYCINE-DEHYDROGENASE-CYTOCHROME-RXN]
+def: "Catalysis of the reaction: 2 Fe(III)-[cytochrome c] + glycine + H2O = 2 Fe(II)-[cytochrome c] + glyoxylate + NH4+ + 2 H+." [RHEA:16909]
 synonym: "glycine-cytochrome c reductase activity" RELATED [EC:1.4.2.1]
 synonym: "glycine:ferricytochrome-c oxidoreductase (deaminating)" RELATED [EC:1.4.2.1]
 synonym: "reductase, glycine-cytochrome c" RELATED [EC:1.4.2.1]
@@ -317214,7 +317929,7 @@
 id: GO:0047961
 name: glycine N-acyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acyl-CoA + glycine = CoA + N-acylglycine." [EC:2.3.1.13, MetaCyc:GLYCINE-N-ACYLTRANSFERASE-RXN]
+def: "Catalysis of the reaction: acyl-CoA + glycine = CoA + N-acylglycine." [RHEA:19869]
 synonym: "acyl-CoA:glycine N-acyltransferase activity" RELATED [EC:2.3.1.13]
 synonym: "glycine acyltransferase activity" RELATED [EC:2.3.1.13]
 synonym: "glycine-N-acylase activity" RELATED [EC:2.3.1.13]
@@ -317222,10 +317937,11 @@
 xref: MetaCyc:GLYCINE-N-ACYLTRANSFERASE-RXN
 xref: Reactome:R-HSA-2534040 "Unknown NAT N-acylates Gly in GNAT1"
 xref: RHEA:19869
-is_a: GO:0016410 ! N-acyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.13
 property_value: skos:exactMatch RHEA:19869
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0047962
@@ -317443,7 +318159,7 @@
 id: GO:0047974
 name: guanosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanosine + H2O = xanthosine + NH3." [EC:3.5.4.15, MetaCyc:GUANOSINE-DEAMINASE-RXN]
+def: "Catalysis of the reaction: guanosine + H2O + H+ = xanthosine + NH4+." [RHEA:12861]
 synonym: "guanosine aminase activity" RELATED [EC:3.5.4.15]
 synonym: "guanosine aminohydrolase activity" RELATED [EC:3.5.4.15]
 xref: EC:3.5.4.15
@@ -317564,7 +318280,7 @@
 id: GO:0047981
 name: L-histidine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-histidine + acetyl-CoA = N(alpha)-acetyl-L-histidine + CoA + H+." [EC:2.3.1.33, RHEA:24596]
+def: "Catalysis of the reaction: L-histidine + acetyl-CoA = N(alpha)-acetyl-L-histidine + CoA + H+." [RHEA:24596]
 synonym: "acetyl-CoA:L-histidine N-acetyltransferase activity" RELATED [EC:2.3.1.33]
 synonym: "acetylhistidine synthetase activity" RELATED [EC:2.3.1.33]
 synonym: "histidine acetyltransferase activity" RELATED [EC:2.3.1.33]
@@ -317582,7 +318298,7 @@
 id: GO:0047982
 name: homocysteine desulfhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-homocysteine + H2O = sulfide + NH3 + 2-oxobutanoate." [EC:4.4.1.2, MetaCyc:HOMOCYSTEINE-DESULFHYDRASE-RXN]
+def: "Catalysis of the reaction: L-homocysteine + H2O = 2-oxobutanoate + hydrogen sulfide + NH4+ + H+." [RHEA:14501]
 synonym: "homocysteine desulfurase activity" RELATED [EC:4.4.1.2]
 synonym: "L-homocysteine hydrogen-sulfide-lyase (deaminating)" RELATED [EC:4.4.1.2]
 synonym: "L-homocysteine hydrogen-sulfide-lyase (deaminating; 2-oxobutanoate-forming)" RELATED [EC:4.4.1.2]
@@ -318058,8 +318774,8 @@
 synonym: "receptor agonist activity" RELATED [GOC:molecular_function_refactoring]
 synonym: "signaling molecule" BROAD []
 synonym: "signaling receptor ligand activity" EXACT []
-xref: Reactome:R-HSA-163625 "Glucagon binds to Glucagon receptor"
 xref: Reactome:R-HSA-381612 "GLP1R binds GLP1"
+xref: Reactome:R-HSA-8848663 "ADIPOQ trimer binds ADIPOR dimers"
 is_a: GO:0030546 ! signaling receptor activator activity
 relationship: has_part GO:0005102 ! signaling receptor binding
 relationship: part_of GO:0007165 ! signal transduction
@@ -319382,7 +320098,7 @@
 id: GO:0048180
 name: activin complex
 namespace: cellular_component
-def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits, inhibin beta-A and inhibin beta-B (sometimes known as activin beta or activin/inhibin beta). There are three forms of activin complex, activin A, which is composed of 2 inhibin beta-A subunits, activin B, which is composed of 2 inhibin beta-B subunits, and activin AB, which is composed of an inhibin beta-A and an inhibin beta-B subunit." [GOC:go_curators]
+def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits, inhibin beta-A and inhibin beta-B (sometimes known as activin beta or activin/inhibin beta). There are three forms of activin complex, activin A, which is composed of 2 inhibin beta-A subunits, activin B, which is composed of 2 inhibin beta-B subunits, and activin AB, which is composed of an inhibin beta-A and an inhibin beta-B subunit." [GOC:curators]
 comment: Note that the actions of the activin complex are the opposite of those of the inhibin complex, which is a dimer of an inhibin beta-A or inhibin beta-B subunit and a inhibin alpha subunit. See 'inhibin complex ; GO:0043511'.
 subset: goslim_pir
 is_a: GO:0032991 ! protein-containing complex
@@ -319392,7 +320108,7 @@
 id: GO:0048183
 name: activin AB complex
 namespace: cellular_component
-def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits (sometimes known as activin beta or activin/inhibin beta), inhibin beta-A and inhibin beta-B." [GOC:go_curators]
+def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits (sometimes known as activin beta or activin/inhibin beta), inhibin beta-A and inhibin beta-B." [GOC:curators]
 comment: Note that the actions of the activin complex are the opposite of those of the inhibin complex, which is a dimer of an inhibin beta-A or inhibin beta-B subunit and a inhibin alpha subunit. See 'inhibin complex ; GO:0043511'.
 synonym: "inhibin beta-A" NARROW []
 synonym: "inhibin beta-B" NARROW []
@@ -320059,17 +320775,18 @@
 
 [Term]
 id: GO:0048258
-name: 3-ketoglucose-reductase activity
+name: obsolete 3-ketoglucose-reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NADP+ + 3-dehydro-alpha-D-glucose = NADPH + alpha-D-glucose." [MetaCyc:KETOGLUCOSE-REDUCTASE-RXN]
-xref: MetaCyc:KETOGLUCOSE-REDUCTASE-RXN
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: NADP+ + 3-dehydro-alpha-D-glucose = NADPH + alpha-D-glucose." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0048259
 name: regulation of receptor-mediated endocytosis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:go_curators, GOC:tb]
+def: "Any process that modulates the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:curators, GOC:tb]
 synonym: "regulation of receptor mediated endocytosis" EXACT [GOC:tb]
 is_a: GO:0030100 ! regulation of endocytosis
 intersection_of: GO:0065007 ! biological regulation
@@ -320080,7 +320797,7 @@
 id: GO:0048260
 name: positive regulation of receptor-mediated endocytosis
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:go_curators, GOC:tb]
+def: "Any process that activates or increases the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:curators, GOC:tb]
 synonym: "activation of receptor mediated endocytosis" NARROW []
 synonym: "positive regulation of receptor mediated endocytosis" EXACT [GOC:tb]
 synonym: "stimulation of receptor mediated endocytosis" NARROW []
@@ -320097,7 +320814,7 @@
 id: GO:0048261
 name: negative regulation of receptor-mediated endocytosis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport." [GOC:curators]
 synonym: "down regulation of receptor mediated endocytosis" EXACT []
 synonym: "down-regulation of receptor mediated endocytosis" EXACT []
 synonym: "downregulation of receptor mediated endocytosis" EXACT []
@@ -320154,7 +320871,8 @@
 def: "Any process that results in a change in the behavior of an organism as a result of a pain stimulus. Pain stimuli cause activation of nociceptors, peripheral receptors for pain, include receptors which are sensitive to painful mechanical stimuli, extreme heat or cold, and chemical stimuli." [GOC:jid]
 synonym: "behavioural response to pain" EXACT []
 is_a: GO:0007610 ! behavior
-relationship: part_of GO:0048265 ! response to pain
+is_a: GO:0048265 ! response to pain
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 
 [Term]
 id: GO:0048268
@@ -320656,7 +321374,7 @@
 id: GO:0048317
 name: seed morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the seed are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the seed are generated and organized." [GOC:curators]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0090698 ! post-embryonic plant morphogenesis
 relationship: part_of GO:0048316 ! seed development
@@ -320672,7 +321390,7 @@
 id: GO:0048319
 name: axial mesoderm morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the axial mesoderm are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the axial mesoderm are generated and organized." [GOC:curators]
 is_a: GO:0048332 ! mesoderm morphogenesis
 relationship: part_of GO:0048318 ! axial mesoderm development
 
@@ -320816,7 +321534,7 @@
 id: GO:0048332
 name: mesoderm morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the mesoderm are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the mesoderm are generated and organized." [GOC:curators]
 is_a: GO:0048729 ! tissue morphogenesis
 relationship: part_of GO:0007498 ! mesoderm development
 
@@ -320911,7 +321629,7 @@
 id: GO:0048340
 name: paraxial mesoderm morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the paraxial mesoderm are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the paraxial mesoderm are generated and organized." [GOC:curators]
 is_a: GO:0048332 ! mesoderm morphogenesis
 is_a: GO:0072132 ! mesenchyme morphogenesis
 relationship: part_of GO:0048339 ! paraxial mesoderm development
@@ -321176,7 +321894,7 @@
 id: GO:0048366
 name: leaf development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the leaf over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the leaf over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048827 ! phyllome development
 
 [Term]
@@ -321184,7 +321902,7 @@
 name: shoot system development
 namespace: biological_process
 alt_id: GO:0022621
-def: "The process whose specific outcome is the progression of the shoot system over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the shoot system over time, from its formation to the mature structure." [GOC:curators]
 synonym: "shoot development" EXACT []
 is_a: GO:0048731 ! system development
 
@@ -321192,7 +321910,7 @@
 id: GO:0048368
 name: lateral mesoderm development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the lateral mesoderm over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the lateral mesoderm over time, from its formation to the mature structure." [GOC:curators]
 synonym: "lateral plate mesoderm development" EXACT []
 is_a: GO:0007498 ! mesoderm development
 is_a: GO:0060485 ! mesenchyme development
@@ -321201,7 +321919,7 @@
 id: GO:0048369
 name: lateral mesoderm morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the lateral mesoderm are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the lateral mesoderm are generated and organized." [GOC:curators]
 synonym: "lateral plate mesoderm morphogenesis" EXACT []
 is_a: GO:0048332 ! mesoderm morphogenesis
 is_a: GO:0072132 ! mesenchyme morphogenesis
@@ -321458,7 +322176,7 @@
 id: GO:0048390
 name: intermediate mesoderm morphogenesis
 namespace: biological_process
-def: "The process in which the anatomical structures of the intermediate mesoderm are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the intermediate mesoderm are generated and organized." [GOC:curators]
 is_a: GO:0048332 ! mesoderm morphogenesis
 relationship: part_of GO:0048389 ! intermediate mesoderm development
 
@@ -321639,7 +322357,7 @@
 name: floral organ development
 namespace: biological_process
 alt_id: GO:0048433
-def: "The process whose specific outcome is the progression of the floral organ over time, from its formation to the mature structure." [GOC:go_curators, GOC:PO_curators, PO:0025395]
+def: "The process whose specific outcome is the progression of the floral organ over time, from its formation to the mature structure." [GOC:curators, GOC:PO_curators, PO:0025395]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0099402 ! plant organ development
 relationship: part_of GO:0009908 ! flower development
@@ -321649,7 +322367,7 @@
 name: floral whorl development
 namespace: biological_process
 alt_id: GO:0048413
-def: "The process whose specific outcome is the progression of a floral whorl over time, from its formation to the mature structure. A floral whorl is a circular arrangement of parts of a flower arising from a stem of a plant." [GOC:dph, GOC:go_curators, GOC:PO_curators, GOC:tb, PO:0025023]
+def: "The process whose specific outcome is the progression of a floral whorl over time, from its formation to the mature structure. A floral whorl is a circular arrangement of parts of a flower arising from a stem of a plant." [GOC:curators, GOC:dph, GOC:PO_curators, GOC:tb, PO:0025023]
 comment: Consider instead annotating to one of the more specific child terms, or to 'floral organ development ; GO:0048438' or one of its child terms.
 synonym: "collective phyllome structure development" BROAD [GOC:PO_curators, PO:0025023]
 is_a: GO:0003006 ! developmental process involved in reproduction
@@ -321661,7 +322379,7 @@
 name: flower morphogenesis
 namespace: biological_process
 alt_id: GO:0048411
-def: "The process in which the anatomical structures of the flower are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the flower are generated and organized." [GOC:curators]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0010016 ! shoot system morphogenesis
 is_a: GO:0090698 ! post-embryonic plant morphogenesis
@@ -321672,7 +322390,7 @@
 name: carpel development
 namespace: biological_process
 alt_id: GO:0048429
-def: "The process whose specific outcome is the progression of the carpel over time, from its formation to the mature structure. A carpel is an organ (generally believed to be a modified foliar unit) at the centre of a flower, bearing one or more ovules and having its margins fused together or with other carpels to enclose the ovule in an ovary, and consisting also of a stigma and usually a style." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the carpel over time, from its formation to the mature structure. A carpel is an organ (generally believed to be a modified foliar unit) at the centre of a flower, bearing one or more ovules and having its margins fused together or with other carpels to enclose the ovule in an ovary, and consisting also of a stigma and usually a style." [GOC:curators]
 is_a: GO:0048437 ! floral organ development
 is_a: GO:0048827 ! phyllome development
 relationship: part_of GO:0048467 ! gynoecium development
@@ -321682,7 +322400,7 @@
 name: petal development
 namespace: biological_process
 alt_id: GO:0048417
-def: "The process whose specific outcome is the progression of the petal over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the petal over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048437 ! floral organ development
 is_a: GO:0048827 ! phyllome development
 relationship: part_of GO:0048465 ! corolla development
@@ -321692,7 +322410,7 @@
 name: sepal development
 namespace: biological_process
 alt_id: GO:0048421
-def: "The process whose specific outcome is the progression of the sepal over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the sepal over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048437 ! floral organ development
 is_a: GO:0048827 ! phyllome development
 relationship: part_of GO:0048464 ! flower calyx development
@@ -321702,7 +322420,7 @@
 name: stamen development
 namespace: biological_process
 alt_id: GO:0048425
-def: "The process whose specific outcome is the progression of the stamen over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the stamen over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048437 ! floral organ development
 is_a: GO:0048827 ! phyllome development
 relationship: part_of GO:0048466 ! androecium development
@@ -321712,7 +322430,7 @@
 name: floral organ morphogenesis
 namespace: biological_process
 alt_id: GO:0048434
-def: "The process in which the anatomical structures of the floral organ are generated and organized." [GOC:go_curators, GOC:PO_curators, PO:0025395]
+def: "The process in which the anatomical structures of the floral organ are generated and organized." [GOC:curators, GOC:PO_curators, PO:0025395]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0090697 ! post-embryonic plant organ morphogenesis
 relationship: part_of GO:0048437 ! floral organ development
@@ -321722,7 +322440,7 @@
 name: carpel morphogenesis
 namespace: biological_process
 alt_id: GO:0048430
-def: "The process in which the anatomical structures of the carpel are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the carpel are generated and organized." [GOC:curators]
 is_a: GO:0048444 ! floral organ morphogenesis
 relationship: part_of GO:0048440 ! carpel development
 
@@ -321731,7 +322449,7 @@
 name: petal morphogenesis
 namespace: biological_process
 alt_id: GO:0048418
-def: "The process in which the anatomical structures of the petal are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the petal are generated and organized." [GOC:curators]
 is_a: GO:0048444 ! floral organ morphogenesis
 relationship: part_of GO:0048441 ! petal development
 
@@ -321740,7 +322458,7 @@
 name: sepal morphogenesis
 namespace: biological_process
 alt_id: GO:0048422
-def: "The process in which the anatomical structures of the sepal are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the sepal are generated and organized." [GOC:curators]
 is_a: GO:0048444 ! floral organ morphogenesis
 relationship: part_of GO:0048442 ! sepal development
 
@@ -321749,7 +322467,7 @@
 name: stamen morphogenesis
 namespace: biological_process
 alt_id: GO:0048426
-def: "The process in which the anatomical structures of the stamen are generated and organized." [GOC:go_curators]
+def: "The process in which the anatomical structures of the stamen are generated and organized." [GOC:curators]
 is_a: GO:0048444 ! floral organ morphogenesis
 relationship: part_of GO:0048443 ! stamen development
 
@@ -321836,7 +322554,7 @@
 name: floral whorl morphogenesis
 namespace: biological_process
 alt_id: GO:0048414
-def: "The process in which the anatomical structures of the floral whorl are generated and organized." [GOC:go_curators, GOC:PO_curators, PO:0025023]
+def: "The process in which the anatomical structures of the floral whorl are generated and organized." [GOC:curators, GOC:PO_curators, PO:0025023]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0090698 ! post-embryonic plant morphogenesis
 relationship: part_of GO:0048438 ! floral whorl development
@@ -321908,28 +322626,28 @@
 id: GO:0048464
 name: flower calyx development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the flower calyx over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the flower calyx over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048438 ! floral whorl development
 
 [Term]
 id: GO:0048465
 name: corolla development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the corolla over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the corolla over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048438 ! floral whorl development
 
 [Term]
 id: GO:0048466
 name: androecium development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the androecium over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the androecium over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0048438 ! floral whorl development
 
 [Term]
 id: GO:0048467
 name: gynoecium development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the gynoecium over time, from its formation to the mature structure. The gynoecium is the collective name for the carpels of a flower." [GOC:go_curators, PO:0008062]
+def: "The process whose specific outcome is the progression of the gynoecium over time, from its formation to the mature structure. The gynoecium is the collective name for the carpels of a flower." [GOC:curators, PO:0008062]
 synonym: "pistil development" EXACT []
 is_a: GO:0048438 ! floral whorl development
 
@@ -321937,7 +322655,7 @@
 id: GO:0048468
 name: cell development
 namespace: biological_process
-def: "The cellular developmental process in which a specific cell progresses from an immature to a mature state. Cell development start once cell commitment has taken place." [GOC:go_curators]
+def: "The cellular developmental process in which a specific cell progresses from an immature to a mature state. Cell development start once cell commitment has taken place." [GOC:curators]
 subset: goslim_candida
 synonym: "terminal differentiation" RELATED [GOC:dph, GOC:tb]
 is_a: GO:0048856 ! anatomical structure development
@@ -321949,7 +322667,7 @@
 id: GO:0048469
 name: cell maturation
 namespace: biological_process
-def: "The cellular developmental process, independent of morphogenetic (shape) change, that is required for a specific cell to attain its fully functional state." [GOC:go_curators]
+def: "The cellular developmental process, independent of morphogenetic (shape) change, that is required for a specific cell to attain its fully functional state." [GOC:curators]
 subset: goslim_pir
 synonym: "functional differentiation" RELATED [GOC:dph]
 is_a: GO:0048869 ! cellular developmental process
@@ -322370,7 +323088,7 @@
 id: GO:0048512
 name: circadian behavior
 namespace: biological_process
-def: "The specific behavior of an organism that recurs with a regularity of approximately 24 hours." [GOC:bf, GOC:go_curators, GOC:pr]
+def: "The specific behavior of an organism that recurs with a regularity of approximately 24 hours." [GOC:bf, GOC:curators, GOC:pr]
 synonym: "circadian rhythm behavior" EXACT []
 is_a: GO:0007622 ! rhythmic behavior
 is_a: GO:0007623 ! circadian rhythm
@@ -322623,7 +323341,7 @@
 id: GO:0048532
 name: anatomical structure arrangement
 namespace: biological_process
-def: "The process that gives rise to the configuration of the constituent parts of an anatomical structure. This process pertains to the physical shaping of a rudimentary structure. Anatomical structures can be macroscopic such as a carpel, or microscopic such as an acrosome." [GOC:go_curators]
+def: "The process that gives rise to the configuration of the constituent parts of an anatomical structure. This process pertains to the physical shaping of a rudimentary structure. Anatomical structures can be macroscopic such as a carpel, or microscopic such as an acrosome." [GOC:curators]
 synonym: "anatomical structure organization" BROAD []
 synonym: "anatomical structure structural organization" EXACT []
 synonym: "organization of an anatomical structure" EXACT []
@@ -322756,7 +323474,7 @@
 id: GO:0048545
 name: response to steroid hormone
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a steroid hormone stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a steroid hormone stimulus." [GOC:curators]
 synonym: "response to steroid hormone stimulus" EXACT [GOC:dos]
 is_a: GO:0009725 ! response to hormone
 is_a: GO:0033993 ! response to lipid
@@ -322766,7 +323484,7 @@
 name: digestive tract morphogenesis
 namespace: biological_process
 alt_id: GO:0048547
-def: "The process in which the anatomical structures of the digestive tract are generated and organized. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:dph, GOC:go_curators, PMID:12618131]
+def: "The process in which the anatomical structures of the digestive tract are generated and organized. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:curators, GOC:dph, PMID:12618131]
 synonym: "alimentary canal morphogenesis" EXACT []
 synonym: "digestive tube morphogenesis" EXACT []
 synonym: "gastrointestinal tract morphogenesis" EXACT []
@@ -322779,7 +323497,7 @@
 id: GO:0048548
 name: regulation of pinocytosis
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:curators]
 is_a: GO:0030100 ! regulation of endocytosis
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0006907 ! pinocytosis
@@ -322789,7 +323507,7 @@
 id: GO:0048549
 name: positive regulation of pinocytosis
 namespace: biological_process
-def: "Any process that activates, maintains or increases the rate of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:go_curators]
+def: "Any process that activates, maintains or increases the rate of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:curators]
 synonym: "activation of pinocytosis" NARROW []
 synonym: "stimulation of pinocytosis" NARROW []
 synonym: "up regulation of pinocytosis" EXACT []
@@ -322805,7 +323523,7 @@
 id: GO:0048550
 name: negative regulation of pinocytosis
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of pinocytosis. Pinocytosis is the process in which cells take in liquid material from their external environment; literally 'cell drinking'. Liquid is enclosed in vesicles, formed by invagination of the plasma membrane. These vesicles then move into the cell and pass their contents to endosomes." [GOC:curators]
 synonym: "down regulation of pinocytosis" EXACT []
 synonym: "down-regulation of pinocytosis" EXACT []
 synonym: "downregulation of pinocytosis" EXACT []
@@ -322840,7 +323558,7 @@
 name: embryonic digestive tract morphogenesis
 namespace: biological_process
 alt_id: GO:0048558
-def: "The process in which the anatomical structures of the digestive tract are generated and organized during embryonic development. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:go_curators]
+def: "The process in which the anatomical structures of the digestive tract are generated and organized during embryonic development. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:curators]
 synonym: "embryonic gut morphogenesis" BROAD [GOC:dph]
 is_a: GO:0048562 ! embryonic organ morphogenesis
 relationship: part_of GO:0048546 ! digestive tract morphogenesis
@@ -322895,7 +323613,7 @@
 name: photosystem I assembly
 namespace: biological_process
 alt_id: GO:0010251
-def: "The aggregation, arrangement and bonding together of a set of components to form a photosystem I complex on the thylakoid membrane." [GOC:go_curators]
+def: "The aggregation, arrangement and bonding together of a set of components to form a photosystem I complex on the thylakoid membrane." [GOC:curators]
 is_a: GO:0065003 ! protein-containing complex assembly
 relationship: part_of GO:0019684 ! photosynthesis, light reaction
 
@@ -322903,7 +323621,7 @@
 id: GO:0048565
 name: digestive tract development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the digestive tract over time, from its formation to the mature structure. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the digestive tract over time, from its formation to the mature structure. The digestive tract is the anatomical structure through which food passes and is processed." [GOC:curators]
 synonym: "gut development" BROAD [GOC:dph]
 synonym: "intestinal development" NARROW []
 synonym: "intestine development" NARROW []
@@ -322914,7 +323632,7 @@
 id: GO:0048566
 name: embryonic digestive tract development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the gut over time, from its formation to the mature structure during embryonic development. The gut is the region of the digestive tract extending from the beginning of the intestines to the anus." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the gut over time, from its formation to the mature structure during embryonic development. The gut is the region of the digestive tract extending from the beginning of the intestines to the anus." [GOC:curators]
 is_a: GO:0048565 ! digestive tract development
 is_a: GO:0048568 ! embryonic organ development
 
@@ -323198,7 +323916,7 @@
 id: GO:0048588
 name: developmental cell growth
 namespace: biological_process
-def: "The growth of a cell, where growth contributes to the progression of the cell over time from one condition to another." [GOC:go_curators, GOC:isa_complete]
+def: "The growth of a cell, where growth contributes to the progression of the cell over time from one condition to another." [GOC:curators, GOC:isa_complete]
 subset: gocheck_do_not_annotate
 synonym: "developmental growth of a unicellular organism" EXACT []
 is_a: GO:0016049 ! cell growth
@@ -323210,7 +323928,7 @@
 id: GO:0048589
 name: developmental growth
 namespace: biological_process
-def: "The increase in size or mass of an entire organism, a part of an organism or a cell, where the increase in size or mass has the specific outcome of the progression of the organism over time from one condition to another." [GOC:go_curators]
+def: "The increase in size or mass of an entire organism, a part of an organism or a cell, where the increase in size or mass has the specific outcome of the progression of the organism over time from one condition to another." [GOC:curators]
 is_a: GO:0040007 ! growth
 intersection_of: GO:0040007 ! growth
 intersection_of: part_of GO:0032502 ! developmental process
@@ -323271,7 +323989,7 @@
 id: GO:0048599
 name: oocyte development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of an oocyte over time, from initial commitment of the cell to its specific fate, to the fully functional differentiated cell." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of an oocyte over time, from initial commitment of the cell to its specific fate, to the fully functional differentiated cell." [GOC:curators]
 is_a: GO:0007281 ! germ cell development
 relationship: part_of GO:0009994 ! oocyte differentiation
 
@@ -323279,7 +323997,7 @@
 id: GO:0048600
 name: oocyte fate commitment
 namespace: biological_process
-def: "The process in which the developmental fate of a cell becomes restricted such that it will develop into an oocyte." [GOC:go_curators]
+def: "The process in which the developmental fate of a cell becomes restricted such that it will develop into an oocyte." [GOC:curators]
 is_a: GO:0003006 ! developmental process involved in reproduction
 is_a: GO:0045165 ! cell fate commitment
 relationship: part_of GO:0009994 ! oocyte differentiation
@@ -323288,7 +324006,7 @@
 id: GO:0048601
 name: oocyte morphogenesis
 namespace: biological_process
-def: "The process in which the structures of an oocyte are generated and organized. This process occurs while the initially relatively unspecialized cell is acquiring the specialized features of an oocyte." [GOC:go_curators]
+def: "The process in which the structures of an oocyte are generated and organized. This process occurs while the initially relatively unspecialized cell is acquiring the specialized features of an oocyte." [GOC:curators]
 synonym: "oocyte morphogenesis during differentiation" EXACT []
 is_a: GO:0000902 ! cell morphogenesis
 is_a: GO:0003006 ! developmental process involved in reproduction
@@ -323438,7 +324156,7 @@
 id: GO:0048623
 name: seed germination on parent plant
 namespace: biological_process
-def: "The process in which a seed germinates before being shed from the parent plant." [GOC:go_curators]
+def: "The process in which a seed germinates before being shed from the parent plant." [GOC:curators]
 synonym: "non-vegetative vivipary" BROAD []
 synonym: "pre-harvest sprouting" EXACT []
 synonym: "vivipary" BROAD []
@@ -323448,7 +324166,7 @@
 id: GO:0048624
 name: plantlet formation on parent plant
 namespace: biological_process
-def: "The process in which a new plantlet develops from a meristem on the plant body. As part of this process, when the plantlet is large enough to live independently, the physical connection between the new plantlet and the main plant is severed." [GOC:go_curators]
+def: "The process in which a new plantlet develops from a meristem on the plant body. As part of this process, when the plantlet is large enough to live independently, the physical connection between the new plantlet and the main plant is severed." [GOC:curators]
 synonym: "vegetative vivipary" BROAD []
 synonym: "vivipary" BROAD []
 is_a: GO:0003006 ! developmental process involved in reproduction
@@ -323557,7 +324275,7 @@
 id: GO:0048634
 name: regulation of muscle organ development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of muscle development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of muscle development." [GOC:curators]
 is_a: GO:0050793 ! regulation of developmental process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0007517 ! muscle organ development
@@ -323567,7 +324285,7 @@
 id: GO:0048635
 name: negative regulation of muscle organ development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of muscle development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of muscle development." [GOC:curators]
 synonym: "down regulation of muscle development" EXACT []
 synonym: "down-regulation of muscle development" EXACT []
 synonym: "downregulation of muscle development" EXACT []
@@ -323583,7 +324301,7 @@
 id: GO:0048636
 name: positive regulation of muscle organ development
 namespace: biological_process
-def: "Any process that activates, maintains or increases the rate of muscle development." [GOC:go_curators]
+def: "Any process that activates, maintains or increases the rate of muscle development." [GOC:curators]
 synonym: "activation of muscle development" NARROW []
 synonym: "stimulation of muscle development" NARROW []
 synonym: "up regulation of muscle development" EXACT []
@@ -323600,7 +324318,7 @@
 id: GO:0048638
 name: regulation of developmental growth
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of developmental growth." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of developmental growth." [GOC:curators]
 is_a: GO:0040008 ! regulation of growth
 is_a: GO:0050793 ! regulation of developmental process
 intersection_of: GO:0065007 ! biological regulation
@@ -323611,7 +324329,7 @@
 id: GO:0048639
 name: positive regulation of developmental growth
 namespace: biological_process
-def: "Any process that activates, maintains or increases the rate of developmental growth." [GOC:go_curators]
+def: "Any process that activates, maintains or increases the rate of developmental growth." [GOC:curators]
 synonym: "activation of developmental growth" NARROW []
 synonym: "stimulation of developmental growth" NARROW []
 synonym: "up regulation of developmental growth" EXACT []
@@ -323628,7 +324346,7 @@
 id: GO:0048640
 name: negative regulation of developmental growth
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of developmental growth." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of developmental growth." [GOC:curators]
 synonym: "down regulation of developmental growth" EXACT []
 synonym: "down-regulation of developmental growth" EXACT []
 synonym: "downregulation of developmental growth" EXACT []
@@ -323644,7 +324362,7 @@
 id: GO:0048641
 name: regulation of skeletal muscle tissue development
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of skeletal muscle tissue development." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of skeletal muscle tissue development." [GOC:curators]
 is_a: GO:0016202 ! regulation of striated muscle tissue development
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0007519 ! skeletal muscle tissue development
@@ -323654,7 +324372,7 @@
 id: GO:0048642
 name: negative regulation of skeletal muscle tissue development
 namespace: biological_process
-def: "Any process that stops, prevents, or reduces the frequency, rate or extent of skeletal muscle tissue development." [GOC:go_curators]
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of skeletal muscle tissue development." [GOC:curators]
 synonym: "down regulation of skeletal muscle development" EXACT []
 synonym: "down-regulation of skeletal muscle development" EXACT []
 synonym: "downregulation of skeletal muscle development" EXACT []
@@ -323669,7 +324387,7 @@
 id: GO:0048643
 name: positive regulation of skeletal muscle tissue development
 namespace: biological_process
-def: "Any process that activates, maintains or increases the rate of skeletal muscle tissue development." [GOC:go_curators]
+def: "Any process that activates, maintains or increases the rate of skeletal muscle tissue development." [GOC:curators]
 synonym: "activation of skeletal muscle development" NARROW []
 synonym: "stimulation of skeletal muscle development" NARROW []
 synonym: "up regulation of skeletal muscle development" EXACT []
@@ -326576,7 +327294,7 @@
 id: GO:0050001
 name: D-glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + D-glutamate." [EC:3.5.1.35, MetaCyc:D-GLUTAMINASE-RXN]
+def: "Catalysis of the reaction: D-glutamine + H2O = D-glutamate + NH4+." [RHEA:22840]
 synonym: "D-glutamine amidohydrolase activity" RELATED [EC:3.5.1.35]
 xref: EC:3.5.1.35
 xref: MetaCyc:D-GLUTAMINASE-RXN
@@ -326979,7 +327697,7 @@
 id: GO:0050025
 name: L-glutamate oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutamate + O2 + H2O = 2-oxoglutarate + NH3 + H2O2." [RHEA:20728]
+def: "Catalysis of the reaction: L-glutamate + O2 + H2O = 2-oxoglutarate + H2O2 + NH4+." [RHEA:20728]
 synonym: "glutamate (acceptor) dehydrogenase activity" RELATED [EC:1.4.3.11]
 synonym: "glutamate oxidase activity" RELATED [EC:1.4.3.11]
 synonym: "glutamic acid oxidase activity" RELATED [EC:1.4.3.11]
@@ -327049,7 +327767,6 @@
 name: L-lysine oxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: H2O + L-lysine + O2 = 6-amino-2-oxohexanoate + H2O2 + NH4+." [RHEA:14437]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-lysine alpha-oxidase activity" RELATED [EC:1.4.3.14]
 synonym: "L-lysine:oxygen 2-oxidoreductase (deaminating)" RELATED [EC:1.4.3.14]
 synonym: "L-lysyl-alpha-oxidase activity" RELATED [EC:1.4.3.14]
@@ -327062,8 +327779,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R00447
 property_value: skos:exactMatch MetaCyc:L-LYSINE-OXIDASE-RXN
 property_value: skos:exactMatch RHEA:14437
-property_value: skos:relatedMatch MetaCyc:PWY-5311
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050030
@@ -327409,7 +328126,7 @@
 id: GO:0050050
 name: L-leucine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-leucine + acetyl-CoA = N-acetyl-L-leucine + CoA + H+." [EC:2.3.1.66, RHEA:20089]
+def: "Catalysis of the reaction: L-leucine + acetyl-CoA = N-acetyl-L-leucine + CoA + H+." [RHEA:20089]
 synonym: "acetyl-CoA:L-leucine N-acetyltransferase activity" RELATED [EC:2.3.1.66]
 synonym: "leucine acetyltransferase activity" RELATED [EC:2.3.1.66]
 xref: EC:2.3.1.66
@@ -327664,25 +328381,26 @@
 
 [Term]
 id: GO:0050065
-name: lysine-pyruvate 6-transaminase activity
+name: L-lysine:pyruvate 6-transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-lysine + pyruvate = L-alanine + L-allysine." [EC:2.6.1.71, RHEA:19393]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: L-lysine + pyruvate = (S)-2-amino-6-oxohexanoate + L-alanine." [RHEA:19393]
 synonym: "L-lysine:pyruvate aminotransferase activity" RELATED [EC:2.6.1.71]
 synonym: "Lys-AT" RELATED [EC:2.6.1.71]
 synonym: "lysine--pyruvate 6-aminotransferase activity" EXACT []
+synonym: "lysine-pyruvate 6-transaminase activity" EXACT []
 synonym: "lysine-pyruvate aminotransferase activity" RELATED [EC:2.6.1.71]
 xref: EC:2.6.1.71
 xref: KEGG_REACTION:R00453
 xref: MetaCyc:LYSINE--PYRUVATE-6-AMINOTRANSFERASE-RXN
 xref: RHEA:19393
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.71
 property_value: skos:exactMatch KEGG_REACTION:R00453
 property_value: skos:exactMatch MetaCyc:LYSINE--PYRUVATE-6-AMINOTRANSFERASE-RXN
 property_value: skos:exactMatch RHEA:19393
-property_value: skos:relatedMatch MetaCyc:PWY-5324
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050066
@@ -327820,7 +328538,7 @@
 xref: EC:6.2.1.9
 xref: MetaCyc:MALATE--COA-LIGASE-RXN
 xref: RHEA:26193
-is_a: GO:0016878 ! acid-thiol ligase activity
+is_a: GO:0016405 ! CoA-ligase activity
 property_value: skos:exactMatch EC:6.2.1.9
 property_value: skos:exactMatch RHEA:26193
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -328143,20 +328861,22 @@
 
 [Term]
 id: GO:0050094
-name: methionine-glyoxylate transaminase activity
+name: L-methionine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine + glyoxylate = 4-methylthio-2-oxobutanoate + glycine." [EC:2.6.1.73, RHEA:22884]
+def: "Catalysis of the reaction: glyoxylate + L-methionine = 4-methylsulfanyl-2-oxobutanoate + glycine." [RHEA:22884]
 synonym: "L-methionine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.73]
 synonym: "methionine-glyoxylate aminotransferase activity" RELATED [EC:2.6.1.73]
+synonym: "methionine-glyoxylate transaminase activity" EXACT []
 synonym: "MGAT activity" RELATED [EC:2.6.1.73]
 xref: EC:2.6.1.73
 xref: KEGG_REACTION:R00652
 xref: MetaCyc:METHIONINE--GLYOXYLATE-TRANSAMINASE-RXN
 xref: RHEA:22884
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0010326 ! L-methionine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.73
 property_value: skos:exactMatch RHEA:22884
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050095
@@ -328825,9 +329545,11 @@
 xref: MetaCyc:N6-HYDROXYLYSINE-O-ACETYLTRANSFERASE-RXN
 xref: RHEA:22388
 is_a: GO:0016413 ! O-acetyltransferase activity
+is_a: GO:0140379 ! amino acid acyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.102
 property_value: skos:exactMatch RHEA:22388
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 
 [Term]
 id: GO:0050134
@@ -328970,7 +329692,7 @@
 id: GO:0050142
 name: nitrogenase (flavodoxin) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6 reduced flavodoxin + 6 H+ + N2 + n ATP = 6 oxidized flavodoxin + 2 NH3 + n ADP + n phosphate." [EC:1.19.6.1, MetaCyc:NITROGENASE-FLAVODOXIN-RXN]
+def: "Catalysis of the reaction: N2 + 4 reduced [flavodoxin] + 16 ATP + 16 H2O = 4 oxidized [flavodoxin] + H2 + 2 NH4+ + 16 ADP + 16 phosphate + 18 H+." [RHEA:15645]
 synonym: "reduced flavodoxin:dinitrogen oxidoreductase (ATP-hydrolysing)" RELATED [EC:1.19.6.1]
 xref: EC:1.19.6.1
 xref: MetaCyc:NITROGENASE-FLAVODOXIN-RXN
@@ -329439,12 +330161,13 @@
 
 [Term]
 id: GO:0050170
-name: peptidyl-glutaminase activity
+name: protein-glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: alpha-N-peptidyl-L-glutamine + H2O = alpha-N-peptidyl-L-glutamate + NH3." [EC:3.5.1.43, MetaCyc:PEPTIDYL-GLUTAMINASE-RXN]
+def: "Catalysis of the reaction: [protein]-C-terminal-L-glutamine + H2O = [protein]-C-terminal-L-glutamate + NH4+." [RHEA:10032]
 synonym: "peptideglutaminase activity" RELATED [EC:3.5.1.43]
 synonym: "peptidoglutaminase activity" RELATED [EC:3.5.1.43]
 synonym: "peptidoglutaminase I activity" NARROW [EC:3.5.1.43]
+synonym: "peptidyl-glutaminase activity" EXACT []
 synonym: "peptidyl-L-glutamine amidohydrolase activity" RELATED [EC:3.5.1.43]
 xref: EC:3.5.1.43
 xref: MetaCyc:PEPTIDYL-GLUTAMINASE-RXN
@@ -329456,9 +330179,10 @@
 
 [Term]
 id: GO:0050171
-name: phenol beta-glucosyltransferase activity
+name: obsolete phenol beta-glucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside." [EC:2.4.1.35, MetaCyc:PHENOL-BETA-GLUCOSYLTRANSFERASE-RXN]
+def: "OBSOLETE. Catalysis of the reaction: UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside." [EC:2.4.1.35]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 synonym: "phenol b-glucosyltransferase activity" EXACT []
 synonym: "phenol-beta-D-glucosyltransferase activity" RELATED [EC:2.4.1.35]
 synonym: "UDP glucosyltransferase activity" RELATED [EC:2.4.1.35]
@@ -329469,9 +330193,9 @@
 synonym: "uridine diphosphoglucosyltransferase activity" RELATED [EC:2.4.1.35]
 xref: EC:2.4.1.35
 xref: MetaCyc:PHENOL-BETA-GLUCOSYLTRANSFERASE-RXN
-is_a: GO:0035251 ! UDP-glucosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.35
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0050172
@@ -329544,7 +330268,7 @@
 id: GO:0050176
 name: L-phenylalanine N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-phenylalanine + acetyl-CoA = N-acetyl-L-phenylalanine + CoA + H+." [EC:2.3.1.53, RHEA:17801]
+def: "Catalysis of the reaction: L-phenylalanine + acetyl-CoA = N-acetyl-L-phenylalanine + CoA + H+." [RHEA:17801]
 synonym: "acetyl-CoA-L-phenylalanine alpha-N-acetyltransferase activity" RELATED [EC:2.3.1.53]
 synonym: "acetyl-CoA:L-phenylalanine N-acetyltransferase activity" RELATED [EC:2.3.1.53]
 synonym: "phenylalanine N-acetyltransferase activity" EXACT []
@@ -329942,7 +330666,6 @@
 xref: Reactome:R-HSA-389622 "phytanate + CoA-SH + ATP => phytanoyl-CoA + AMP + pyrophosphate"
 xref: RHEA:21380
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.24
 property_value: skos:exactMatch RHEA:21380
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -330009,7 +330732,7 @@
 xref: EC:2.7.1.52
 xref: KEGG_REACTION:R03161
 xref: MetaCyc:FUCOKINASE-RXN
-xref: Reactome:R-HSA-6787540 "FUK phosphorylates beta-Fuc to Fuc1P"
+xref: Reactome:R-HSA-6787540 "FCSK phosphorylates beta-Fuc to Fuc1P"
 xref: RHEA:13241
 is_a: GO:0016301 ! kinase activity
 is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
@@ -330047,7 +330770,6 @@
 xref: MetaCyc:OXALATE--COA-LIGASE-RXN
 xref: RHEA:18293
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch EC:6.2.1.8
 property_value: skos:exactMatch RHEA:18293
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -330197,22 +330919,20 @@
 id: GO:0050212
 name: progesterone 11-alpha-monooxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: AH2 + O2 + progesterone = 11alpha-hydroxyprogesterone + A + H2O." [RHEA:18205]
-synonym: "progesterone 11-alpha-hydroxylase activity" RELATED [EC:1.14.99.14]
+def: "Catalysis of the reaction: AH2 + O2 + progesterone = 11alpha-hydroxyprogesterone + A + H2O." [PMID:39766209, RHEA:86055]
+synonym: "progesterone 11-alpha-hydroxylase activity" RELATED [EC:1.14.14.197]
 synonym: "progesterone 11a-monooxygenase activity" EXACT []
-synonym: "progesterone 11alpha-hydroxylase activity" RELATED [EC:1.14.99.14]
-synonym: "progesterone 11alpha-monooxygenase activity" RELATED [EC:1.14.99.14]
-synonym: "progesterone,hydrogen-donor:oxygen oxidoreductase (11alpha-hydroxylating)" RELATED [EC:1.14.99.14]
-xref: EC:1.14.99.14
+synonym: "progesterone 11alpha-hydroxylase activity" RELATED [EC:1.14.14.197]
+synonym: "progesterone 11alpha-monooxygenase activity" RELATED [EC:1.14.14.197]
+synonym: "progesterone,hydrogen-donor:oxygen oxidoreductase (11alpha-hydroxylating)" RELATED [EC:1.14.14.197]
 xref: KEGG_REACTION:R02214
 xref: MetaCyc:PROGESTERONE-11-ALPHA-MONOOXYGENASE-RXN
-xref: PMID:39766209
-xref: RHEA:18205
+xref: RHEA:86055
 is_a: GO:0050214 ! progesterone monooxygenase activity
-property_value: skos:exactMatch EC:1.14.99.14
-property_value: skos:exactMatch RHEA:18205
+property_value: skos:exactMatch RHEA:86055
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29311" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31423" xsd:anyURI
 
 [Term]
 id: GO:0050213
@@ -330661,7 +331381,7 @@
 id: GO:0050239
 name: pyrithiamine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + H2O = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + NH3." [EC:3.5.4.20, MetaCyc:PYRITHIAMIN-DEAMINASE-RXN]
+def: "Catalysis of the reaction: 1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(2-hydroxyethyl)-2-methylpyridinium + H2O + H+ = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(2-hydroxyethyl)-2-methylpyridinium + NH4+." [RHEA:14537]
 synonym: "1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium-bromide aminohydrolase activity" RELATED [EC:3.5.4.20]
 xref: EC:3.5.4.20
 xref: MetaCyc:PYRITHIAMIN-DEAMINASE-RXN
@@ -331347,7 +332067,7 @@
 id: GO:0050279
 name: sepiapterin deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: sepiapterin + H2O = xanthopterin-B2 + NH3." [EC:3.5.4.24, MetaCyc:SEPIAPTERIN-DEAMINASE-RXN]
+def: "Catalysis of the reaction: L-sepiapterin + H2O + H+ = (S)-xanthopterin-B2 + NH4+." [RHEA:14025]
 synonym: "sepiapterin aminohydrolase activity" RELATED [EC:3.5.4.24]
 xref: EC:3.5.4.24
 xref: MetaCyc:SEPIAPTERIN-DEAMINASE-RXN
@@ -331375,19 +332095,21 @@
 
 [Term]
 id: GO:0050281
-name: L-serine-glyoxylate transaminase activity
+name: L-serine:glyoxylate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine + glyoxylate = 3-hydroxypyruvate + glycine." [EC:2.6.1.45]
+def: "Catalysis of the reaction: L-serine + glyoxylate = 3-hydroxypyruvate + glycine." [RHEA:19125]
+synonym: "L-serine-glyoxylate transaminase activity" EXACT []
 synonym: "L-serine:glyoxylate aminotransferase activity" RELATED [EC:2.6.1.45]
 synonym: "serine--glyoxylate aminotransferase activity" EXACT []
 synonym: "SGAT activity" RELATED [EC:2.6.1.45]
 xref: EC:2.6.1.45
 xref: MetaCyc:SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN
 xref: RHEA:19125
-is_a: GO:0008483 ! transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.45
 property_value: skos:exactMatch RHEA:19125
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050282
@@ -331410,7 +332132,7 @@
 id: GO:0050283
 name: serine-sulfate ammonia-lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-serine O-sulfate + H2O = pyruvate + NH3 + sulfate." [EC:4.3.1.10, MetaCyc:SERINE-SULFATE-AMMONIA-LYASE-RXN]
+def: "Catalysis of the reaction: -serine O-sulfate + H2O = sulfate + pyruvate + NH4+ + H+." [RHEA:15605]
 synonym: "(L-SOS)lyase activity" RELATED [EC:4.3.1.10]
 synonym: "L-serine-O-sulfate ammonia-lyase (pyruvate-forming)" RELATED [EC:4.3.1.10]
 synonym: "serine-sulphate ammonia-lyase activity" EXACT []
@@ -331758,7 +332480,6 @@
 name: L-lysine 6-dehydrogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-lysine + NAD+ = L-1-piperideine-6-carboxylate + NH4+ + NADH + 2 H+." [RHEA:12408]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 synonym: "L-lysine epsilon-dehydrogenase activity" RELATED [EC:1.4.1.18]
 synonym: "L-lysine:NAD+ 6-oxidoreductase (deaminating)" RELATED [EC:1.4.1.18]
 synonym: "LysDH activity" RELATED [EC:1.4.1.18]
@@ -331769,11 +332490,11 @@
 property_value: skos:exactMatch EC:1.4.1.18
 property_value: skos:exactMatch MetaCyc:LYSINE-6-DEHYDROGENASE-RXN
 property_value: skos:exactMatch RHEA:12408
-property_value: skos:relatedMatch MetaCyc:PWY-5314
 property_value: skos:relatedMatch RHEA:64792
 property_value: skos:relatedMatch RHEA:64800
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30646" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050304
@@ -332084,13 +332805,14 @@
 
 [Term]
 id: GO:0050322
-name: taurine-2-oxoglutarate transaminase activity
+name: taurine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: taurine + 2-oxoglutarate = sulfoacetaldehyde + L-glutamate." [PMID:16535664, RHEA:16353]
 synonym: "taurine aminotransferase activity" EXACT []
 synonym: "taurine transaminase activity" EXACT []
 synonym: "taurine--alpha-ketoglutarate aminotransferase activity" RELATED [EC:2.6.1.55]
 synonym: "taurine--glutamate transaminase activity" RELATED [EC:2.6.1.55]
+synonym: "taurine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "taurine:2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.55]
 xref: EC:2.6.1.55
 xref: MetaCyc:RXN-2301
@@ -332099,6 +332821,7 @@
 property_value: skos:exactMatch EC:2.6.1.55
 property_value: skos:exactMatch RHEA:16353
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050323
@@ -332723,7 +333446,7 @@
 id: GO:0050360
 name: tryptophan 2'-dioxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + O2 = 3-indoleglycolaldehyde + CO2 + NH3." [EC:1.13.99.3, MetaCyc:TRYPTOPHAN-2-DIOXYGENASE-RXN]
+def: "Catalysis of the reaction: L-tryptophan + O2 + H+ = 3-indoleglycolaldehyde + NH4+ + CO2." [RHEA:22620]
 synonym: "indole-3-alkane alpha-hydroxylase activity" RELATED [EC:1.13.99.3]
 synonym: "indolyl-3-alkan alpha-hydroxylase activity" RELATED [EC:1.13.99.3]
 synonym: "L-tryptophan:oxygen 2'-oxidoreductase (side-chain-cleaving)" RELATED [EC:1.13.99.3]
@@ -332760,23 +333483,25 @@
 
 [Term]
 id: GO:0050362
-name: L-tryptophan:2-oxoglutarate aminotransferase activity
+name: L-tryptophan:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + 2-oxoglutarate = indolepyruvate + L-glutamate." [EC:2.6.1.27, MetaCyc:TRYPTOPHAN-AMINOTRANSFERASE-RXN]
+def: "Catalysis of the reaction: L-tryptophan + 2-oxoglutarate = indole-3-pyruvate + L-glutamate." [RHEA:14093]
 synonym: "5-hydroxytryptophan-ketoglutaric transaminase activity" RELATED [EC:2.6.1.27]
 synonym: "hydroxytryptophan aminotransferase activity" RELATED [EC:2.6.1.27]
 synonym: "L-phenylalanine-2-oxoglutarate aminotransferase activity" RELATED [EC:2.6.1.27]
 synonym: "L-tryptophan aminotransferase activity" BROAD [EC:2.6.1.27]
 synonym: "L-tryptophan transaminase activity" BROAD [EC:2.6.1.27]
+synonym: "L-tryptophan:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "tryptophan aminotransferase activity" EXACT []
 synonym: "tryptophan transaminase activity" BROAD [EC:2.6.1.27]
 xref: EC:2.6.1.27
 xref: MetaCyc:TRYPTOPHAN-AMINOTRANSFERASE-RXN
 xref: RHEA:14093
-is_a: GO:0070529 ! L-tryptophan aminotransferase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch EC:2.6.1.27
 property_value: skos:exactMatch RHEA:14093
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0050363
@@ -333036,8 +333761,7 @@
 id: GO:0050378
 name: UDP-glucuronate 4-epimerase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-alpha-D-galacturonate." [EC:5.1.3.6, RHEA:11404]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
+def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-alpha-D-galacturonate." [RHEA:11404]
 synonym: "UDP glucuronic epimerase activity" RELATED [EC:5.1.3.6]
 synonym: "UDP-D-galacturonic acid 4-epimerase activity" RELATED [EC:5.1.3.6]
 synonym: "UDP-galacturonate 4-epimerase activity" RELATED [EC:5.1.3.6]
@@ -333054,8 +333778,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R01385
 property_value: skos:exactMatch MetaCyc:UDP-GLUCURONATE-4-EPIMERASE-RXN
 property_value: skos:exactMatch RHEA:11404
-property_value: skos:relatedMatch MetaCyc:PWY-4861
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0050379
@@ -333646,7 +334370,7 @@
 id: GO:0050414
 name: formimidoylaspartate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-formimidoyl-L-aspartate + H2O = N-formyl-L-aspartate + NH3." [EC:3.5.3.5, MetaCyc:FORMIMINOASPARTATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: N-formimidoyl-L-aspartate + H2O = N-formyl-L-aspartate + NH4+." [RHEA:13661]
 synonym: "formiminoaspartate deiminase activity" EXACT []
 synonym: "N-formimidoyl-L-aspartate iminohydrolase activity" RELATED [EC:3.5.3.5]
 xref: EC:3.5.3.5
@@ -333680,7 +334404,7 @@
 id: GO:0050416
 name: formimidoylglutamate deiminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H2O = N-formyl-L-glutamate + NH3." [EC:3.5.3.13, MetaCyc:FORMIMINOGLUTAMATE-DEIMINASE-RXN]
+def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H2O = N-formyl-L-glutamate + NH4+." [RHEA:22832]
 synonym: "formiminoglutamate deiminase activity" EXACT []
 synonym: "formiminoglutamic iminohydrolase activity" RELATED [EC:3.5.3.13]
 synonym: "N-formimidoyl-L-glutamate iminohydrolase activity" RELATED [EC:3.5.3.13]
@@ -333696,7 +334420,7 @@
 id: GO:0050417
 name: glutamin-(asparagin-)ase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + L-glutamate; and H2O + L-asparagine = NH3 + L-aspartate." [EC:3.5.1.38]
+def: "Catalysis of the reactions: H2O + L-glutamine = NH4 + L-glutamate; and H2O + L-asparagine = NH4 + L-aspartate." [EC:3.5.1.38]
 synonym: "glutaminase-(asparagin-)ase activity" EXACT []
 synonym: "glutaminase-asparaginase activity" RELATED [EC:3.5.1.38]
 synonym: "L-ASNase/L-GLNase activity" RELATED [EC:3.5.1.38]
@@ -333714,7 +334438,7 @@
 id: GO:0050418
 name: hydroxylamine reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: NH3 + H2O + acceptor = hydroxylamine + reduced acceptor." [EC:1.7.99.1]
+def: "Catalysis of the reaction: an acceptor + NH4+ + H2O = hydroxylamine + a reduced acceptor + H+." [RHEA:22052]
 synonym: "ammonia:(acceptor) oxidoreductase activity" RELATED [EC:1.7.99.1]
 synonym: "ammonia:acceptor oxidoreductase activity" RELATED [EC:1.7.99.1]
 synonym: "hydroxylamine (acceptor) reductase activity" RELATED [EC:1.7.99.1]
@@ -333869,6 +334593,7 @@
 synonym: "phosphoadenosine phosphosulfate metabolic process" EXACT []
 synonym: "phosphoadenosine phosphosulfate metabolism" EXACT []
 xref: Reactome:R-HSA-156584 "Cytosolic sulfonation of small molecules"
+xref: Reactome:R-HSA-174362 "Transport and metabolism of PAPS"
 is_a: GO:0006790 ! sulfur compound metabolic process
 is_a: GO:0009150 ! purine ribonucleotide metabolic process
 is_a: GO:0034035 ! purine ribonucleoside bisphosphate metabolic process
@@ -335002,7 +335727,7 @@
 synonym: "spHAS" RELATED [EC:2.4.1.212]
 xref: EC:2.4.1.212
 xref: MetaCyc:2.4.1.212-RXN
-xref: Reactome:R-HSA-2160851 "HAS1,2,3 mediate the polymerization of HA"
+xref: Reactome:R-HSA-2160851 "HAS1,2,3 mediate the polymerization and export of HA"
 xref: RHEA:12528
 is_a: GO:0008194 ! UDP-glycosyltransferase activity
 is_a: GO:0016758 ! hexosyltransferase activity
@@ -335120,12 +335845,9 @@
 synonym: "UDP-N-acetyl-D-glucosamine:beta-D-glucuronosyl-(1->4)-N-acetyl-alpha-D-glucosaminyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase activity" RELATED [EC:2.4.1.224]
 xref: EC:2.4.1.224
 xref: MetaCyc:2.4.1.224-RXN
-xref: Reactome:R-HSA-2022851 "EXT1:EXT2 transfer GlcNAc to the heparan chain"
-xref: Reactome:R-HSA-2022919 "EXT1:EXT2 transfers GlcNAc to the terminal GlcA residue"
+xref: Reactome:R-HSA-2022851 "EXT1:EXT2 transfers GlcNAc to the heparan chain"
 xref: Reactome:R-HSA-3656254 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcNAc to the heparan chain"
 xref: Reactome:R-HSA-3656261 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcNAc to the heparan chain"
-xref: Reactome:R-HSA-9036283 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcNAc to the terminal GlcA residue"
-xref: Reactome:R-HSA-9036290 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcNAc to heparan"
 xref: RHEA:16213
 is_a: GO:0008375 ! acetylglucosaminyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.224
@@ -335144,7 +335866,7 @@
 synonym: "UDP-alpha-D-glucuronate:N-acetyl-alpha-D-glucosaminyl-(1->4)-beta-D-glucuronosyl-proteoglycan 4-beta-glucuronosyltransferase activity" RELATED [EC:2.4.1.225]
 xref: EC:2.4.1.225
 xref: MetaCyc:2.4.1.225-RXN
-xref: Reactome:R-HSA-2022856 "EXT1:EXT2 transfers GlcNAc to heparan"
+xref: Reactome:R-HSA-2022856 "EXT1:EXT2 transfers GlcA to heparan"
 xref: Reactome:R-HSA-2076392 "EXT1:EXT2 transfers GlcA to heparan"
 xref: Reactome:R-HSA-3656257 "Defective EXT1 (in EXT1:EXT2) does not transfer GlcA to heparan"
 xref: Reactome:R-HSA-3656267 "Defective EXT2 (in EXT1:EXT2) does not transfer GlcA to heparan"
@@ -335167,9 +335889,8 @@
 synonym: "N-acetylgalactosaminyl-proteoglycan 3-b-glucuronosyltransferase activity" EXACT []
 xref: EC:2.4.1.226
 xref: MetaCyc:RXN-16324
-xref: Reactome:R-HSA-1971491 "CHPF,CHPF2,CHSY3 transfer GlcA to chondroitin"
+xref: Reactome:R-HSA-1971491 "Chondroitin polymerase complex transfers GlcA to CS chain"
 xref: Reactome:R-HSA-3595178 "Defective CHSY1 does not transfer GlcA to chondroitin"
-xref: Reactome:R-HSA-9632034 "CHSY1 transfers GlcA to chondroitin"
 is_a: GO:0015020 ! glucuronosyltransferase activity
 property_value: skos:exactMatch EC:2.4.1.226
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -335233,7 +335954,7 @@
 id: GO:0050514
 name: homospermidine synthase (spermidine-specific) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: spermidine + putrescine = sym-homospermidine + propane-1,3-diamine." [EC:2.5.1.45]
+def: "Catalysis of the reaction: spermidine + putrescine = sym-homospermidine + propane-1,3-diamine." [RHEA:11236]
 synonym: "spermidine:putrescine 4-aminobutyltransferase (propane-1,3-diamine-forming)" RELATED [EC:2.5.1.45]
 xref: EC:2.5.1.45
 xref: MetaCyc:2.5.1.45-RXN
@@ -336129,7 +336850,7 @@
 id: GO:0050568
 name: protein-glutamine glutaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: protein L-glutamine + H2O = protein L-glutamate + NH3." [EC:3.5.1.44, MetaCyc:CHEBDEAMID-RXN]
+def: "Catalysis of the reaction: L-glutaminyl-[protein] + H2O = L-glutamyl-[protein] + NH4+." [RHEA:16441]
 synonym: "destabilase activity" RELATED [EC:3.5.1.44]
 synonym: "glutaminyl-peptide glutaminase activity" RELATED [EC:3.5.1.44]
 synonym: "glutaminylpeptide glutaminase activity" RELATED [EC:3.5.1.44]
@@ -336188,6 +336909,7 @@
 xref: EC:1.1.1.263
 xref: KEGG_REACTION:R05682
 xref: MetaCyc:1.1.1.263-RXN
+xref: Reactome:R-HSA-9773976 "AKR1E2 reduces 1,5-aF to 1,5-aG"
 xref: RHEA:20665
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.1.1.263
@@ -336269,7 +336991,7 @@
 synonym: "GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)" RELATED [EC:1.1.1.271]
 xref: EC:1.1.1.271
 xref: MetaCyc:1.1.1.271-RXN
-xref: Reactome:R-HSA-6787642 "TSTA3 dimer reduces GDP-KDGal to GDP-Fuc"
+xref: Reactome:R-HSA-6787642 "GFUS dimer reduces GDP-KDGal to GDP-Fuc"
 xref: RHEA:18885
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
 relationship: part_of GO:0042350 ! GDP-L-fucose biosynthetic process
@@ -336860,6 +337582,7 @@
 xref: MetaCyc:RXN66-14
 xref: Reactome:R-HSA-194674 "4,4-dimethylcholesta-8(9),14,24-trien-3beta-ol is reduced to 4,4-dimethylcholesta-8(9),24-dien-3beta-ol [LBR]"
 xref: Reactome:R-HSA-194698 "4,4-dimethylcholesta-8(9),14,24-trien-3beta-ol is reduced to 4,4-dimethylcholesta-8(9),24-dien-3beta-ol [TM7SF2]"
+xref: Reactome:R-HSA-9946140 "TM7SF2 reduces 4,4-diMeCholesta-8,14-dien-3-ol to 4,4-diMe5-cholest-8-en-3-ol"
 xref: RHEA:18561
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.3.1.70
@@ -337459,6 +338182,7 @@
 synonym: "chondroitin sulphate proteoglycan biosynthesis" EXACT []
 synonym: "chondroitin sulphate proteoglycan biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6567
+xref: Reactome:R-HSA-2022870 "CS-GAG biosynthesis"
 is_a: GO:0030166 ! proteoglycan biosynthetic process
 is_a: GO:0050654 ! chondroitin sulfate proteoglycan metabolic process
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
@@ -337480,6 +338204,7 @@
 synonym: "dermatan sulphate proteoglycan biosynthesis" EXACT []
 synonym: "dermatan sulphate proteoglycan biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6568
+xref: Reactome:R-HSA-2022923 "DS-GAG biosynthesis"
 is_a: GO:0030166 ! proteoglycan biosynthetic process
 is_a: GO:0050655 ! dermatan sulfate proteoglycan metabolic process
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
@@ -337592,7 +338317,8 @@
 xref: KEGG_REACTION:R10869
 xref: MetaCyc:RXN-7953
 xref: MetaCyc:RXN-7954
-xref: Reactome:R-HSA-2018659 "Chondroitin 4-sulfate (C4S) can be further sulfated on position 6 by CHST15"
+xref: Reactome:R-HSA-2018659 "CHST15 transfers sulfate to (6-)GalNAc-4-sulfate"
+xref: Reactome:R-HSA-9941312 "CHST15 transfers sulfate to (6-)GalNAc-4-sulfate"
 xref: RHEA:54300
 xref: RHEA:54304
 is_a: GO:0008146 ! sulfotransferase activity
@@ -337931,6 +338657,9 @@
 synonym: "downregulation of antiviral response" EXACT []
 synonym: "inhibition of antiviral response" NARROW []
 synonym: "negative regulation of antiviral response" EXACT [GOC:dph]
+xref: Reactome:R-HSA-164938 "Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters"
+xref: Reactome:R-HSA-164944 "Nef and signal transduction"
+xref: Reactome:R-HSA-168276 "NS1 Mediated Effects on Host Pathways"
 is_a: GO:0002832 ! negative regulation of response to biotic stimulus
 is_a: GO:0031348 ! negative regulation of defense response
 is_a: GO:0032102 ! negative regulation of response to external stimulus
@@ -338383,6 +339112,7 @@
 def: "Binding to a low-density lipoprotein receptor." [GOC:ai]
 synonym: "LDL receptor binding" EXACT []
 synonym: "low-density lipoprotein receptor binding" EXACT [GOC:dph]
+xref: Reactome:R-HSA-2423785 "CR:atREs binds apoE and HSPG"
 is_a: GO:0070325 ! lipoprotein particle receptor binding
 
 [Term]
@@ -338450,7 +339180,7 @@
 id: GO:0050761
 name: depsipeptide metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:go_curators]
+def: "The chemical reactions and pathways involving depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:curators]
 subset: goslim_pir
 synonym: "depsipeptide metabolism" EXACT []
 is_a: GO:0008152 ! metabolic process
@@ -338459,7 +339189,7 @@
 id: GO:0050762
 name: depsipeptide catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the breakdown of depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:curators]
 synonym: "depsipeptide breakdown" EXACT []
 synonym: "depsipeptide catabolism" EXACT []
 synonym: "depsipeptide degradation" EXACT []
@@ -338470,7 +339200,7 @@
 id: GO:0050763
 name: depsipeptide biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:go_curators]
+def: "The chemical reactions and pathways resulting in the formation of depsipeptides, a linear or cyclic compound composed of both amino acids and hydroxy acids in peptide and ester bonds respectively." [GOC:curators]
 synonym: "depsipeptide anabolism" EXACT []
 synonym: "depsipeptide biosynthesis" EXACT []
 synonym: "depsipeptide formation" EXACT []
@@ -338808,7 +339538,7 @@
 name: regulation of biological process
 namespace: biological_process
 alt_id: GO:0050791
-def: "Any process that modulates the frequency, rate or extent of a biological process. Biological processes are regulated by many means; examples include the control of gene expression, protein modification or interaction with a protein or substrate molecule." [GOC:ai, GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of a biological process. Biological processes are regulated by many means; examples include the control of gene expression, protein modification or interaction with a protein or substrate molecule." [GOC:ai, GOC:curators]
 subset: gocheck_do_not_annotate
 subset: goslim_candida
 subset: goslim_pir
@@ -338837,7 +339567,7 @@
 id: GO:0050792
 name: regulation of viral process
 namespace: biological_process
-def: "Any process that modulates the rate or extent of the viral life cycle, the set of processes by which a virus reproduces and spreads among hosts." [GOC:go_curators, GOC:tb]
+def: "Any process that modulates the rate or extent of the viral life cycle, the set of processes by which a virus reproduces and spreads among hosts." [GOC:curators, GOC:tb]
 synonym: "regulation of viral reproduction" NARROW [GOC:bf, GOC:jl]
 is_a: GO:0050789 ! regulation of biological process
 intersection_of: GO:0065007 ! biological regulation
@@ -338848,7 +339578,7 @@
 id: GO:0050793
 name: regulation of developmental process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of development, the biological process whose specific outcome is the progression of a multicellular organism over time from an initial condition (e.g. a zygote, or a young adult) to a later condition (e.g. a multicellular animal or an aged adult)." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of development, the biological process whose specific outcome is the progression of a multicellular organism over time from an initial condition (e.g. a zygote, or a young adult) to a later condition (e.g. a multicellular animal or an aged adult)." [GOC:curators]
 is_a: GO:0050789 ! regulation of biological process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0032502 ! developmental process
@@ -338859,7 +339589,7 @@
 name: regulation of cellular process
 namespace: biological_process
 alt_id: GO:0051244
-def: "Any process that modulates the frequency, rate or extent of a cellular process, any of those that are carried out at the cellular level, but are not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of a cellular process, any of those that are carried out at the cellular level, but are not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level." [GOC:curators]
 subset: gocheck_do_not_annotate
 synonym: "regulation of cellular physiological process" EXACT []
 is_a: GO:0050789 ! regulation of biological process
@@ -338871,7 +339601,7 @@
 id: GO:0050795
 name: regulation of behavior
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of behavior, the internally coordinated responses (actions or inactions) of whole living organisms (individuals or groups) to internal or external stimuli." [GOC:go_curators, GOC:pr]
+def: "Any process that modulates the frequency, rate or extent of behavior, the internally coordinated responses (actions or inactions) of whole living organisms (individuals or groups) to internal or external stimuli." [GOC:curators, GOC:pr]
 synonym: "regulation of behaviour" EXACT []
 is_a: GO:0051239 ! regulation of multicellular organismal process
 intersection_of: GO:0065007 ! biological regulation
@@ -341682,7 +342412,7 @@
 id: GO:0051037
 name: obsolete regulation of transcription involved in meiotic cell cycle
 namespace: biological_process
-def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription as part of a meiotic cell cycle." [GOC:go_curators]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of transcription as part of a meiotic cell cycle." [GOC:curators]
 comment: This term was obsoleted because it represents a GO-CAM model.
 synonym: "meiotic regulation of transcription" EXACT []
 synonym: "regulation of transcription, meiotic" EXACT [GOC:mah]
@@ -341970,7 +342700,7 @@
 id: GO:0051056
 name: regulation of small GTPase mediated signal transduction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of small GTPase mediated signal transduction." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of small GTPase mediated signal transduction." [GOC:curators]
 synonym: "regulation of small GTPase-mediated signal transduction" EXACT []
 xref: Reactome:R-HSA-9012999 "RHO GTPase cycle"
 is_a: GO:1902531 ! regulation of intracellular signal transduction
@@ -342337,7 +343067,6 @@
 def: "OBSOLETE. The process of assisting in the correct noncovalent assembly of posttranslational proteins and does not depend on additional protein cofactors. This function occurs over one or more cycles of nucleotide-dependent binding and release." [GOC:rb]
 comment: This term represent a GO-CAM model
 synonym: "chaperone cofactor-independent protein folding" EXACT [GOC:rb]
-xref: Reactome:R-HSA-390450 "Folding of actin by CCT/TriC"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21869" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0006457
@@ -342383,7 +343112,6 @@
 synonym: "regulation of sequence-specific DNA binding transcription factor activity" EXACT []
 synonym: "regulation of thyroid hormone receptor activity" NARROW []
 synonym: "regulation of transcription factor activity" BROAD [GOC:dph, GOC:tb]
-xref: Reactome:R-HSA-9708530 "Regulation of BACH1 activity"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30856" xsd:anyURI
 is_obsolete: true
 
@@ -342625,7 +343353,6 @@
 xref: MetaCyc:DCARNCOALIG-RXN
 xref: RHEA:30543
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016881 ! acid-amino acid ligase activity
 property_value: skos:exactMatch RHEA:30543
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -342633,14 +343360,21 @@
 id: GO:0051109
 name: crotonobetaine-CoA ligase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: CoA + crotono-betaine + ATP = AMP + diphosphate + crotonobetainyl-CoA." [MetaCyc:CROTCOALIG-RXN]
+def: "Catalysis of the reaction: (R)-carnitine + ATP + CoA = (R)-carnitinyl-CoA + AMP + diphosphate. Also catalyzes the transfer of CoA to crotonobetaine and gamma-butyrobetaine." [EC:6.2.1.48]
 synonym: "crotonobetaine-CoA synthase activity" EXACT []
 synonym: "crotonobetaine/carnitine-CoA ligase activity" BROAD []
+xref: EC:6.2.1.48
 xref: MetaCyc:CROTCOALIG-RXN
+xref: RHEA:28514
 xref: RHEA:30079
+xref: RHEA:55960
 is_a: GO:0016405 ! CoA-ligase activity
-property_value: skos:exactMatch RHEA:30079
+property_value: skos:exactMatch EC:6.2.1.48
+property_value: skos:narrowMatch RHEA:28514
+property_value: skos:narrowMatch RHEA:30079
+property_value: skos:narrowMatch RHEA:55960
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31616" xsd:anyURI
 
 [Term]
 id: GO:0051110
@@ -342762,8 +343496,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid = (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate. 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid is also known as 12S-HpETE, and (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate as hepoxilin A3." [PMID:15123652]
 comment: The reason for obsoletion is that this term is equivalent to hydroperoxy icosatetraenoate isomerase activity.
-xref: Reactome:R-HSA-2161794 "Arachidonate is converted to HXA3/B3 by ALOX12"
-xref: Reactome:R-HSA-8942208 "ALOXE3 isomerises 12R-HpETE to HXA3"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29695" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0106255
@@ -342775,7 +343507,6 @@
 def: "OBSOLETE. The chemical reactions and pathways involving hepoxilins, a class of bioactive icosanoids with roles in the regulation of cell physiology." [PMID:15123652]
 comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "hepoxilin metabolism" EXACT []
-xref: Reactome:R-HSA-2142696 "Synthesis of Hepoxilins (HX) and Trioxilins (TrX)"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30524" xsd:anyURI
 is_obsolete: true
 
@@ -344641,10 +345372,11 @@
 xref: KEGG_REACTION:R02864
 xref: MetaCyc:SIROHEME-FERROCHELAT-RXN
 xref: RHEA:24360
-is_a: GO:0004325 ! ferrochelatase activity
+is_a: GO:0016829 ! lyase activity
 property_value: skos:exactMatch EC:4.99.1.4
 property_value: skos:exactMatch RHEA:24360
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31620" xsd:anyURI
 
 [Term]
 id: GO:0051267
@@ -344938,7 +345670,7 @@
 id: GO:0051289
 name: protein homotetramerization
 namespace: biological_process
-def: "The formation of a protein homotetramer, a macromolecular structure consisting of four noncovalently associated identical subunits." [GOC:go_curators]
+def: "The formation of a protein homotetramer, a macromolecular structure consisting of four noncovalently associated identical subunits." [GOC:curators]
 synonym: "protein homotetramer assembly" EXACT []
 synonym: "protein homotetramer biosynthesis" EXACT []
 synonym: "protein homotetramer biosynthetic process" EXACT []
@@ -344950,7 +345682,7 @@
 id: GO:0051290
 name: protein heterotetramerization
 namespace: biological_process
-def: "The formation of a protein heterotetramer, a macromolecular structure consisting of four noncovalently associated subunits, of which not all are identical." [GOC:go_curators]
+def: "The formation of a protein heterotetramer, a macromolecular structure consisting of four noncovalently associated subunits, of which not all are identical." [GOC:curators]
 synonym: "protein heterotetramer assembly" EXACT []
 synonym: "protein heterotetramer biosynthesis" EXACT []
 synonym: "protein heterotetramer biosynthetic process" EXACT []
@@ -345070,7 +345802,7 @@
 id: GO:0051301
 name: cell division
 namespace: biological_process
-def: "The process resulting in division and partitioning of components of a cell to form more cells; may or may not be accompanied by the physical separation of a cell into distinct, individually membrane-bounded daughter cells." [GOC:di, GOC:go_curators, GOC:pr]
+def: "The process resulting in division and partitioning of components of a cell to form more cells; may or may not be accompanied by the physical separation of a cell into distinct, individually membrane-bounded daughter cells." [GOC:curators, GOC:di, GOC:pr]
 comment: Note that this term differs from 'cytokinesis ; GO:0000910' in that cytokinesis does not include nuclear division.
 subset: goslim_chembl
 subset: goslim_drosophila
@@ -345082,7 +345814,7 @@
 id: GO:0051302
 name: regulation of cell division
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of the physical partitioning and separation of a cell into daughter cells." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of the physical partitioning and separation of a cell into daughter cells." [GOC:curators]
 is_a: GO:0050794 ! regulation of cellular process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0051301 ! cell division
@@ -348608,17 +349340,16 @@
 
 [Term]
 id: GO:0051600
-name: regulation of endocytosis by exocyst localization
+name: obsolete regulation of endocytosis by exocyst localization
 namespace: biological_process
-def: "Any process in which an exocyst is transported to, or maintained in, a specific location that results in the modulation of endocytosis. An exocyst is a protein complex peripherally associated with the plasma membrane that determines where vesicles dock and fuse." [GOC:ai, GOC:dph, GOC:tb]
+def: "OBSOLETE. Any process in which an exocyst is transported to, or maintained in, a specific location that results in the modulation of endocytosis. An exocyst is a protein complex peripherally associated with the plasma membrane that determines where vesicles dock and fuse." [GOC:ai, GOC:dph, GOC:tb]
+comment: The reason for obsoletion is that this represents a phenotype.
 synonym: "regulation of endocytosis by exocyst localisation" EXACT [GOC:mah]
 synonym: "regulation of site selection of endocytosis" RELATED []
 synonym: "relocation of endocytosis" RELATED []
 synonym: "spatial regulation of endocytosis" RELATED []
-is_a: GO:0030100 ! regulation of endocytosis
-is_a: GO:0051601 ! exocyst localization
-intersection_of: GO:0051601 ! exocyst localization
-intersection_of: regulates GO:0006897 ! endocytosis
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31201" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0051601
@@ -348667,6 +349398,7 @@
 subset: goslim_pombe
 subset: goslim_prokaryote
 subset: goslim_yeast
+xref: Reactome:R-HSA-9918432 "Maturation of DENV proteins"
 is_a: GO:0019538 ! protein metabolic process
 relationship: part_of GO:0010467 ! gene expression
 
@@ -349797,6 +350529,7 @@
 synonym: "growth of symbiont in host vacuole" NARROW []
 synonym: "interaction with host" EXACT []
 xref: Reactome:R-HSA-9635486 "Infection with Mycobacterium tuberculosis"
+xref: Reactome:R-HSA-9640148 "Infection with Enterobacteria"
 xref: Reactome:R-HSA-9658195 "Leishmania infection"
 is_a: GO:0044403 ! biological process involved in symbiotic interaction
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20191" xsd:anyURI
@@ -350069,7 +350802,6 @@
 def: "OBSOLETE. Catalysis of the reaction: C-terminal protein amino acid methyl ester + H2O = protein amino acid + methanol." [PMID:10318862, PMID:8650216]
 comment: This seems to be outside the scope of GO: it represent a specific protein and a 'domain.
 synonym: "protein phosphatase methylesterase activity" NARROW []
-xref: Reactome:R-HSA-8856951 "PP2A demethylation by PPME1"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27558" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0051723
@@ -350086,6 +350818,7 @@
 xref: EC:3.1.1.89
 xref: MetaCyc:RXN-12322
 xref: Reactome:R-HSA-5367022 "DPH7 hydrolyzes a methyl group on Me-diphthine EEF2"
+xref: Reactome:R-HSA-8856951 "PP2A demethylation by PPME1"
 xref: RHEA:48548
 is_a: GO:0052689 ! carboxylic ester hydrolase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
@@ -350111,6 +350844,7 @@
 synonym: "oxidized nicotinamide adenine dinucleotide transmembrane transporter activity" EXACT []
 synonym: "reduced NAD transporter activity" EXACT []
 synonym: "reduced nicotinamide adenine dinucleotide transmembrane transporter activity" EXACT []
+xref: Reactome:R-HSA-9938195 "SLC25A51 mediates mitochondrial uptake of NAD+"
 is_a: GO:0000295 ! adenine nucleotide transmembrane transporter activity
 relationship: part_of GO:0035352 ! NAD transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21577" xsd:anyURI
@@ -350897,7 +351631,7 @@
 id: GO:0051788
 name: response to misfolded protein
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a misfolded protein stimulus." [GOC:go_curators]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a misfolded protein stimulus." [GOC:curators]
 is_a: GO:0035966 ! response to topologically incorrect protein
 
 [Term]
@@ -353031,10 +353765,9 @@
 id: GO:0051975
 name: L-lysine biosynthetic process via alpha-aminoadipate and saccharopine
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and saccharopine. This pathway is used by yeast and fungi to synthesize the essential amino acid L-lysine, and pathway intermediates are often incorporated into secondary metabolic processes. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This is then reduced to form alpha-aminoadipate 6-semialdehyde, which is metabolized to saccharopine and finally L-lysine." [MetaCyc:LYSINE-AMINOAD-PWY]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and saccharopine. This pathway is used by yeast and fungi to synthesize the essential amino acid L-lysine, and pathway intermediates are often incorporated into secondary metabolic processes. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This is then reduced to form alpha-aminoadipate 6-semialdehyde, which is metabolized to saccharopine and finally L-lysine." [GOC:curators]
 synonym: "lysine biosynthesis via aminoadipic acid and saccharopine" EXACT []
 synonym: "lysine biosynthetic process via aminoadipic acid and saccharopine" EXACT []
-xref: MetaCyc:LYSINE-AMINOAD-PWY
 is_a: GO:0019878 ! L-lysine biosynthetic process via aminoadipic acid
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -353042,10 +353775,9 @@
 id: GO:0051976
 name: L-lysine biosynthetic process via alpha-aminoadipate and N2-acetyl-alpha-aminoadipate
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and N2-acetyl-alpha-aminoadipate. This pathway of prokaryotic lysine biosynthesis via alpha-aminoadipate was discovered in the hyper-thermophilic Gram-negative eubacterium Thermus thermophilus. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This undergoes acetylation, to form N2-acetyl-alpha-aminoadipate, and is then phosphorylated to give N2-acetyl-alpha-aminoadipyl-delta-phosphate. This is converted to N2-acetyl-alpha-aminoadipate semialdehyde, which is then converted to N2-acetyl-L-lysine. A final deacetylation reaction produces L-lysine." [MetaCyc:PWY-3081]
+def: "The chemical reactions and pathways resulting in the formation of L-lysine via the intermediates alpha-aminoadipic acid and N2-acetyl-alpha-aminoadipate. This pathway of prokaryotic lysine biosynthesis via alpha-aminoadipate was discovered in the hyper-thermophilic Gram-negative eubacterium Thermus thermophilus. The pathway proceeds as follows: alpha-ketoglutarate is converted to homocitrate, which is metabolized to 3-carboxyhex-2-enedioate and then homoisocitrate. This is then decarboxylated to form alpha-ketoadipate, which is then converted to alpha-aminoadipate. This undergoes acetylation, to form N2-acetyl-alpha-aminoadipate, and is then phosphorylated to give N2-acetyl-alpha-aminoadipyl-delta-phosphate. This is converted to N2-acetyl-alpha-aminoadipate semialdehyde, which is then converted to N2-acetyl-L-lysine. A final deacetylation reaction produces L-lysine." [GOC:curators]
 synonym: "lysine biosynthesis via aminoadipic acid and N2-acetyl-alpha-aminoadipate" EXACT []
 synonym: "lysine biosynthetic process via aminoadipic acid and N2-acetyl-alpha-aminoadipate" EXACT []
-xref: MetaCyc:PWY-3081
 is_a: GO:0019878 ! L-lysine biosynthetic process via aminoadipic acid
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30820" xsd:anyURI
 
@@ -353063,7 +353795,6 @@
 namespace: molecular_function
 def: "Enables the directed movement of lysophospholipids from one side of a membrane to the other. A lysophospholipid is a phospholipid that lacks one of its fatty acyl chains; it is an intermediate formed during digestion of dietary and biliary phospholipids." [GOC:ai]
 synonym: "lysophospholipid transporter activity" BROAD []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0015370 ! solute:sodium symporter activity
 is_a: GO:0015605 ! organophosphate ester transmembrane transporter activity
 is_a: GO:0170055 ! lipid transmembrane transporter activity
@@ -353268,11 +353999,13 @@
 
 [Term]
 id: GO:0051994
-name: P-methyltransferase activity
+name: obsolete P-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of a methyl group to the phosphorus atom of an acceptor molecule." [GOC:ai]
-is_a: GO:0008168 ! methyltransferase activity
+def: "OBSOLETE. Catalysis of the transfer of a methyl group to the phosphorus atom of an acceptor molecule." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25591" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2010-07-16T04:39:34Z
 
@@ -353298,6 +354031,8 @@
 synonym: "squalene synthetase activity" RELATED [EC:2.5.1.21]
 xref: EC:2.5.1.21
 xref: MetaCyc:RXN66-281
+xref: Reactome:R-HSA-191402 "Reduction of presqualene diphosphate to form squalene"
+xref: Reactome:R-HSA-191405 "Two FAPP molecules dimerize to form presqualene diphosphate"
 xref: RHEA:32295
 xref: RHEA:32299
 is_a: GO:0004659 ! prenyltransferase activity
@@ -353863,6 +354598,7 @@
 synonym: "negative regulation by symbiont of host programmed cell death" EXACT []
 synonym: "suppression by symbiont of host PCD" EXACT []
 synonym: "suppression by symbiont of host programmed cell death" EXACT []
+xref: Reactome:R-HSA-9636569 "Suppression of autophagy"
 is_a: GO:0052040 ! symbiont-mediated perturbation of host programmed cell death
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23652" xsd:anyURI
 
@@ -355005,8 +355741,8 @@
 synonym: "modulation by symbiont of host apoptotic process" EXACT []
 synonym: "modulation by virus of host apoptosis" NARROW []
 synonym: "regulation by virus of host apoptosis" NARROW []
+xref: Reactome:R-HSA-9920951 "Dengue virus modulates apoptosis"
 xref: VZ:1518 "Apoptosis modulation"
-is_a: GO:0051709 ! regulation of killing of cells of another organism
 is_a: GO:0052040 ! symbiont-mediated perturbation of host programmed cell death
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25986" xsd:anyURI
 
@@ -358266,6 +359002,7 @@
 synonym: "modulation by organism of immune response of other organism involved in symbiotic interaction" BROAD []
 synonym: "modulation by symbiont of host immune response" EXACT []
 synonym: "perturbation of host immune response" EXACT []
+xref: Reactome:R-HSA-9920588 "Dengue virus activates/modulates innate and adaptive immune responses"
 is_a: GO:0052031 ! symbiont-mediated perturbation of host defense response
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25569" xsd:anyURI
 
@@ -358685,7 +359422,7 @@
 id: GO:0052595
 name: aliphatic amine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: an aliphatic amine + H2O + O2 = an aldehyde + NH3 + hydrogen peroxide + H+." [MetaCyc:AMINEOXID-RXN]
+def: "Catalysis of the reaction: an aliphatic amine + H2O + O2 = an aldehyde + NH4+ + H2O2 + H+." [MetaCyc:AMINEOXID-RXN]
 synonym: "aliphatic-amine oxidase activity" EXACT []
 synonym: "aliphatic-amine:oxygen oxidoreductase(deaminating) activity" EXACT []
 xref: MetaCyc:AMINEOXID-RXN
@@ -359549,44 +360286,51 @@
 
 [Term]
 id: GO:0052654
-name: L-leucine-2-oxoglutarate transaminase activity
+name: L-leucine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-leucine = 4-methyl-2-oxopentanoate + L-glutamatic acid." [RHEA:18321]
+def: "Catalysis of the reaction:L-leucine + 2-oxoglutarate = 4-methyl-2-oxopentanoate + L-glutamate." [RHEA:18321]
 subset: goslim_chembl
 synonym: "L-leucine aminotransferase activity" EXACT []
+synonym: "L-leucine-2-oxoglutarate transaminase activity" EXACT []
+xref: EC:2.6.1.6
 xref: KEGG_REACTION:R01090
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERLEU-RXN
 xref: RHEA:18321
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
+property_value: skos:exactMatch EC:2.6.1.6
 property_value: skos:exactMatch RHEA:18321
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28176" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0052655
-name: L-valine-2-oxoglutarate transaminase activity
+name: L-valine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-valine = 3-methyl-2-oxobutanoic acid + L-glutamatic acid." [RHEA:24813]
+def: "Catalysis of the reaction: L-valine + 2-oxoglutarate = 3-methyl-2-oxobutanoate + L-glutamate." [RHEA:24813]
 subset: goslim_chembl
 synonym: "L-valine aminotransferase activity" EXACT []
+synonym: "L-valine-2-oxoglutarate transaminase activity" EXACT []
 xref: KEGG_REACTION:R01214
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERVAL-RXN
 xref: RHEA:24813
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch RHEA:24813
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0052656
-name: L-isoleucine-2-oxoglutarate transaminase activity
+name: L-isoleucine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-oxoglutarate + L-isoleucine = (S)-3-methyl-2-oxopentanoic acid + L-glutamic acid." [RHEA:24801]
+def: "Catalysis of the reaction: L-isoleucine + 2-oxoglutarate = (S)-3-methyl-2-oxopentanoate + L-glutamate." [RHEA:24801]
 subset: goslim_chembl
 synonym: "L-isoleucine aminotransferase activity" EXACT []
+synonym: "L-isoleucine-2-oxoglutarate transaminase activity" EXACT []
 xref: KEGG_REACTION:R02199
 xref: MetaCyc:BRANCHED-CHAINAMINOTRANSFERILEU-RXN
 xref: RHEA:24801
-is_a: GO:0004084 ! branched-chain-amino-acid transaminase activity
+is_a: GO:0004084 ! branched-chain-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:exactMatch RHEA:24801
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 
@@ -360064,27 +360808,26 @@
 synonym: "perillyl-CoA synthetase activity" BROAD [UM-BBD_reactionID:r0731]
 xref: KEGG_REACTION:R06368
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 
 [Term]
 id: GO:0052687
-name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (ADP-forming) activity
+name: obsolete (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (ADP-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP + phosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06396]
+def: "OBSOLETE. Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP + phosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06396]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "3-isopropenyl-6-oxoheptanoyl-CoA synthetase activity" BROAD [UM-BBD_reactionID:r0737]
-xref: KEGG_REACTION:R06396
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052688
-name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (AMP-forming) activity
+name: obsolete (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (AMP-forming) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + AMP + diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06515]
+def: "OBSOLETE. Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + AMP + diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG_REACTION:R06515]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "3-isopropenyl-6-oxoheptanoyl-CoA synthetase activity" BROAD [UM-BBD_reactionID:r0737]
-xref: KEGG_REACTION:R06515
-is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052689
@@ -360193,7 +360936,6 @@
 synonym: "cellular glucuronide biosynthetic process" EXACT []
 synonym: "cellular glucuronoside biosynthesis" EXACT []
 synonym: "cellular glucuronoside biosynthetic process" EXACT []
-xref: Reactome:R-HSA-156588 "Glucuronidation"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28506" xsd:anyURI
 is_obsolete: true
 creation_date: 2011-07-27T05:28:18Z
@@ -360530,10 +361272,12 @@
 
 [Term]
 id: GO:0052715
-name: mannosyl-diinositol phosphorylceramide phospholipase activity
+name: obsolete mannosyl-diinositol phosphorylceramide phospholipase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: mannosyl-diinositol phosphorylceramide + H2O = C26-phytoceramide + mannosyldiphosphorylinositol." [GOC:ai]
-is_a: GO:0052713 ! inositol phosphorylceramide phospholipase activity
+def: "OBSOLETE. Catalysis of the reaction: mannosyl-diinositol phosphorylceramide + H2O = C26-phytoceramide + mannosyldiphosphorylinositol." [GOC:ai]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2011-08-05T08:02:57Z
 
@@ -360560,7 +361304,7 @@
 id: GO:0052717
 name: tRNA-specific adenosine-34 deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: adenosine-34 + H2O = inosine-34 + NH3, in a tRNA-Ala molecule." [PMID:17875641, RHEA:43168]
+def: "Catalysis of the reaction: adenosine34 in tRNA + H2O + H+ = inosine34 in tRNA + NH4+." [PMID:17875641, RHEA:43168]
 synonym: "tRNA(Ala)-A34 deaminase activity" EXACT []
 xref: EC:3.5.4.33
 xref: RHEA:43168
@@ -360873,12 +361617,13 @@
 
 [Term]
 id: GO:0052735
-name: tRNA (cytidine-3-)-methyltransferase activity
+name: tRNA (cytidine-N3)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a cytidine in tRNA + S-adenosyl-L-methionine = an N3-methylcytidine in tRNA + H+ + S-adenosyl-L-homocysteine." [RHEA:60908]
 synonym: "S-adenosyl-L-methionine:tRNA (cytosine-3-)-methyltransferase activity" EXACT []
 synonym: "S-adenosyl-L-methionine:tRNA cytosine-3-methyltransferase activity" EXACT []
 synonym: "transfer ribonucleate cytosine 3-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine-3-)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine 3)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytosine-3-)-methyltransferase activity" EXACT []
 synonym: "tRNA cytosine 3-methyltransferase activity" EXACT []
@@ -360887,6 +361632,7 @@
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
 property_value: skos:exactMatch RHEA:60908
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25722" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26109" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
 [Term]
@@ -360933,7 +361679,6 @@
 def: "OBSOLETE. Catalysis of the reaction: phosphatidylserine + H2O = 2-acyl-sn-glycero-3-phosphoserine + fatty acid." [KEGG_REACTION:R04034]
 comment: The reason for obsoletion is that this molecular function is equivalent to phospholipase A1 activity.
 xref: KEGG_REACTION:R04034
-xref: Reactome:R-HSA-8869425 "PLA1A hydrolyses PS to 2-acyl LPS"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28816" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0008970
@@ -361075,7 +361820,7 @@
 id: GO:0052750
 name: versatile peroxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: reactive black 5 + hydrogen peroxide = oxidized reactive black 5 + 2 H2O." [RHEA:22396]
+def: "Catalysis of the reaction: reactive black 5 + H2O2 = oxidized reactive black 5 + 2 H2O." [RHEA:22396]
 synonym: "reactive-black-5:hydrogen-peroxide oxidoreductase activity" BROAD [EC:1.11.1.16]
 xref: EC:1.11.1.16
 xref: KEGG_REACTION:R07612
@@ -361100,15 +361845,12 @@
 
 [Term]
 id: GO:0052752
-name: reduced coenzyme F420:heterodisulfide oxidoreductase activity
+name: obsolete reduced coenzyme F420:heterodisulfide oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: reduced coenzyme F420 + CoB-S-S-CoM = coenzyme F420 + CoM-SH + CoB-SH." [GOC:mengo_curators, PMID:9914308]
-synonym: "coenzyme F420-dependent heterodisulfide oxidoreductase activity" EXACT [PMID:9914308]
-synonym: "F420-dependent heterodisulfide oxidoreductase activity" EXACT [PMID:9914308]
-synonym: "F420H2:heterodisulfide oxidoreductase activity" EXACT []
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
-created_by: ai
-creation_date: 2011-09-21T04:21:16Z
+def: "OBSOLETE. Catalysis of the reaction: reduced coenzyme F420 + CoB-S-S-CoM = coenzyme F420 + CoM-SH + CoB-SH." [GOC:mengo_curators, PMID:9914308]
+comment: The reason for obsoletion is that this reaction MF has no EC or RHEA xref and no annotations.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0052753
@@ -361123,17 +361865,17 @@
 
 [Term]
 id: GO:0052754
-name: GTP:coenzyme F420 guanyltransferase activity
+name: obsolete GTP:coenzyme F420 guanyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: GTP + factor gamma-F420-2 + H+ = coenzyme F390-G + diphosphate." [GOC:mengo_curators, MetaCyc:RXN-18273]
+def: "OBSOLETE. Catalysis of the reaction: GTP + factor gamma-F420-2 + H+ = coenzyme F390-G + diphosphate." [GOC:mengo_curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "coenzyme F390-G synthetase activity" EXACT []
 synonym: "GTP:coenzyme F420 guanylyltransferase activity" EXACT []
-xref: MetaCyc:RXN-18273
-is_a: GO:0070568 ! guanylyltransferase activity
-property_value: skos:exactMatch MetaCyc:RXN-18273
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28324" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28503" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28776" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 created_by: ai
 creation_date: 2011-09-22T03:25:30Z
 
@@ -362011,7 +362753,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = myo-inositol 1,3,4,5,6-pentakisphosphate + phosphate." [GOC:curators]
 comment: The reason for obsoletion is that this term is equivalent to inositol hexakisphosphate 3-phosphatase activity.
-xref: Reactome:R-HSA-1855225 "IP6 is dephosphorylated to I(1,2,4,5,6)P5 by MINPP1 in the ER lumen"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30385" xsd:anyURI
 is_obsolete: true
@@ -362354,6 +363095,7 @@
 xref: KEGG_REACTION:R09541
 xref: MetaCyc:RXN0-6555
 xref: Reactome:R-HSA-917811 "STEAP3,STEAP4 reduce Fe3+ to Fe2+"
+xref: Reactome:R-HSA-9927945 "YqjH reduces FeEnt"
 xref: RHEA:28795
 is_a: GO:0000293 ! ferric-chelate reductase activity
 is_a: GO:0016723 ! oxidoreductase activity, acting on metal ions, NAD or NADP as acceptor
@@ -362391,6 +363133,7 @@
 synonym: "NADHX epimerase activity" NARROW []
 synonym: "NADPHX epimerase activity" NARROW []
 xref: EC:5.1.99.6
+xref: Reactome:R-HSA-6806966 "APOA1BP dimer epimerises R-NAD(P)HX to S-NAD(P)HX"
 xref: RHEA:32215
 xref: RHEA:32227
 is_a: GO:0016854 ! racemase and epimerase activity
@@ -362405,7 +363148,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: (R)-NADPHX = (S)-NADPHX." [PMID:21994945]
 comment: This term was obsoleted because it represents the same activity as NADHX epimerase activity ; GO:0052856.
-xref: Reactome:R-HSA-6806966 "APOA1BP dimer epimerises R-NAD(P)HX to S-NAD(P)HX"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0052856
@@ -362565,9 +363307,11 @@
 id: GO:0052868
 name: protein-lysine lysyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: protein-lysine + protein-lysine = protein N6-(lysyl)-L-lysine + protein. This reaction is the addition of lysine group from one protein to a lysine residue in a second protein, producing N6-(lysyl)-L-lysine." [PMID:20729861]
+def: "Catalysis of the reaction: D-beta-lysine + L-lysyl-[protein] + ATP = N6-((3R)-3,6-diaminohexanoyl)-L-lysyl-[protein] + AMP + diphosphate + H+n. This reaction is the addition of lysine group from one protein to a lysine residue in a second protein, producing N6-(lysyl)-L-lysine." [PMID:20729861, RHEA:83435]
+xref: RHEA:83435
 is_a: GO:0016755 ! aminoacyltransferase activity
 is_a: GO:0140096 ! catalytic activity, acting on a protein
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31478" xsd:anyURI
 created_by: ai
 creation_date: 2012-01-11T03:05:27Z
 
@@ -363079,23 +363823,29 @@
 synonym: "N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity" NARROW [EC:1.5.3.13]
 xref: EC:1.5.3.13
 xref: KEGG_REACTION:R03899
-xref: MetaCyc:RXN-12090
+xref: MetaCyc:RXN-10460
+xref: MetaCyc:RXN-12091
+xref: MetaCyc:RXN-9940
+xref: Reactome:R-HSA-141348 "PAO:FAD oxidises NASPM to PTCN"
+xref: Reactome:R-HSA-141351 "PAOX:FAD oxidises NASPN to SPM"
 xref: RHEA:25800
 xref: RHEA:25812
 xref: RHEA:25868
 is_a: GO:0046592 ! polyamine oxidase activity
 property_value: skos:exactMatch EC:1.5.3.13
 property_value: skos:exactMatch KEGG_REACTION:R03899
-property_value: skos:exactMatch MetaCyc:RXN-12090
+property_value: skos:narrowMatch MetaCyc:RXN-10460
+property_value: skos:narrowMatch MetaCyc:RXN-12091
+property_value: skos:narrowMatch MetaCyc:RXN-9940
 property_value: skos:narrowMatch RHEA:25800
 property_value: skos:narrowMatch RHEA:25812
 property_value: skos:narrowMatch RHEA:25868
-property_value: skos:relatedMatch MetaCyc:RXN-9940
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24701" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25837" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28526" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31136" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0052904
@@ -363103,8 +363853,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: H2O + N(1)-acetylspermidine + O2 = 3-acetamidopropanal + H2O2 + putrescine." [GOC:curators]
 comment: This term was obsoleted because it represents a specific substrate of N(1)-acetylpolyamine oxidase (3-acetamidopropanal-forming) activity ; GO:0052903.
-xref: Reactome:R-HSA-141348 "PAO:FAD oxidises NASPM to PTCN"
-xref: Reactome:R-HSA-141351 "PAOX:FAD oxidises NASPN to SPM"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24701" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 is_obsolete: true
@@ -363993,17 +364741,21 @@
 
 [Term]
 id: GO:0055042
-name: 5-valerolactone hydrolase activity
+name: obsolete 5-valerolactone hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-valerolactone + H2O = 5-hydroxyvalerate." [GOC:jid, GOC:mlg]
-is_a: GO:0046573 ! lactonohydrolase activity
+def: "OBSOLETE. Catalysis of the reaction: 5-valerolactone + H2O = 5-hydroxyvalerate." [GOC:jid, GOC:mlg]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0055043
-name: 5-oxovalerate dehydrogenase activity
+name: obsolete 5-oxovalerate dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-oxovalerate + NADP+ + H2O = glutarate + NADPH + H+." [GOC:jid, GOC:mlg]
-is_a: GO:0016620 ! oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 5-oxovalerate + NADP+ + H2O = glutarate + NADPH + H+." [GOC:jid, GOC:mlg]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0055044
@@ -364118,9 +364870,11 @@
 def: "Enables the transfer of the D-enantiomer of the hexose monosaccharide glucose from one side of a membrane to the other." [GOC:jid, GOC:jsg, GOC:mah]
 xref: Reactome:R-HSA-429094 "SLC2A6,8,10,12 transport Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-450095 "GLUT2 (SLC2A2) transports Glc from cytosol to extracellular region"
+xref: Reactome:R-HSA-499981 "Transport of Extracellular Glucose to the Cytosol by GLUT1 and GLUT2"
 xref: Reactome:R-HSA-5339524 "GLUT1 (SLC2A1) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-5632804 "SLC2A1 tetramer does not transport Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-5632871 "Defective SLC2A10 does not transport Glc from extracellular region to cytosol"
+xref: Reactome:R-HSA-5653873 "SLC2A1 tetramer transports Glc from cytosol to Golgi lumen"
 xref: Reactome:R-HSA-8981553 "GLUT14 (SLC2A14) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-8981564 "GLUT3 (SLC2A3) tetramer transports Glc from extracellular region to cytosol"
 xref: Reactome:R-HSA-8981570 "GLUT4 (SLC2A4) tetramer transports Glc from extracellular region to cytosol"
@@ -364781,15 +365535,18 @@
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of L-proline, an L-enantiomer of a chiral, cyclic, nonessential alpha-amino acid found in peptide linkage in proteins." [GOC:ecd]
 synonym: "pyrrolidine-2-carboxylic acid biosynthetic process" BROAD []
+xref: MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
 xref: MetaCyc:PROSYN-PWY
 xref: MetaCyc:PWY-3341
 xref: MetaCyc:PWY-4281 {status="skos:narrowMatch"}
 xref: MetaCyc:PWY-4981
 is_a: GO:0006560 ! L-proline metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ORN-AMINOPENTANOATE-CAT-PWY
 property_value: skos:narrowMatch MetaCyc:PROSYN-PWY
 property_value: skos:narrowMatch MetaCyc:PWY-3341
 property_value: skos:narrowMatch MetaCyc:PWY-4981
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: jid
 creation_date: 2009-10-22T01:59:56Z
 
@@ -364797,10 +365554,13 @@
 id: GO:0055130
 name: D-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-alanine, the D-enantiomer of the amino acid alanine." [GOC:ecd]
+def: "The chemical reactions and pathways resulting in the breakdown of D-alanine." [GOC:curators]
 synonym: "(2R)-2-aminopropanoic acid catabolic process" EXACT []
+xref: MetaCyc:PWY-8448
+is_a: GO:0006522 ! alanine metabolic process
 is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:0046436 ! D-alanine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31405" xsd:anyURI
 created_by: jid
 creation_date: 2009-10-22T02:02:42Z
 
@@ -365679,10 +366439,10 @@
 subset: goslim_prokaryote
 subset: goslim_prokaryote_ribbon
 synonym: "binding, bridging" EXACT []
-synonym: "protein complex scaffold activity" BROAD []
-synonym: "protein-containing complex scaffold activity" BROAD []
+xref: Reactome:R-HSA-2564826 "4Fe-4S cluster assembles on NUBP2:NUBP1 scaffold"
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0005488 ! binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31509" xsd:anyURI
 
 [Term]
 id: GO:0060091
@@ -366191,10 +366951,11 @@
 id: GO:0060138
 name: fetal process involved in parturition
 namespace: biological_process
-def: "A reproductive process occurring in the embryo that results in birth." [GOC:dph]
+def: "A reproductive process occurring in the fetus that results in birth." [GOC:dph]
 is_a: GO:0048609 ! multicellular organismal reproductive process
 relationship: part_of GO:0007567 ! parturition
 relationship: part_of GO:0009792 ! embryo development ending in birth or egg hatching
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31526" xsd:anyURI
 
 [Term]
 id: GO:0060140
@@ -366213,6 +366974,7 @@
 def: "A process in which a symbiont initiates, promotes, or enhances the formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual host cells. Syncytia are produced by viruses that are able to fuse directly at the cell surface without requiring endocytosis." [GOC:dph]
 synonym: "positive regulation of syncytium formation by virus" EXACT []
 synonym: "syncytium formation induced by viral infection" EXACT [VZ:5957]
+xref: Reactome:R-HSA-9733458 "Induction of Cell-Cell Fusion"
 xref: VZ:5957
 is_a: GO:0044068 ! symbiont-mediated perturbation of host cellular process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28791" xsd:anyURI
@@ -366222,23 +366984,21 @@
 name: regulation of syncytium formation by plasma membrane fusion
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual cells." [GOC:dph]
-is_a: GO:0050793 ! regulation of developmental process
 is_a: GO:0051128 ! regulation of cellular component organization
 intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0000768 ! syncytium formation by plasma membrane fusion
-relationship: regulates GO:0000768 ! syncytium formation by plasma membrane fusion
+intersection_of: regulates GO:0000768 ! syncytium formation by cell-cell fusion
+relationship: regulates GO:0000768 ! syncytium formation by cell-cell fusion
 
 [Term]
 id: GO:0060143
 name: positive regulation of syncytium formation by plasma membrane fusion
 namespace: biological_process
 def: "Any process that increases the frequency, rate or extent of the formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual cells." [GOC:dph]
-is_a: GO:0051094 ! positive regulation of developmental process
 is_a: GO:0051130 ! positive regulation of cellular component organization
 is_a: GO:0060142 ! regulation of syncytium formation by plasma membrane fusion
 intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0000768 ! syncytium formation by plasma membrane fusion
-relationship: positively_regulates GO:0000768 ! syncytium formation by plasma membrane fusion
+intersection_of: positively_regulates GO:0000768 ! syncytium formation by cell-cell fusion
+relationship: positively_regulates GO:0000768 ! syncytium formation by cell-cell fusion
 
 [Term]
 id: GO:0060144
@@ -367080,6 +367840,7 @@
 name: lipase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a lipase, an enzyme that catalyzes of the hydrolysis of a lipid." [GOC:BHF, GOC:dph, GOC:tb]
+xref: Reactome:R-HSA-163549 "phosphorylated HSL dimer + FABP4 -> phosphorylated HSL dimer:FABP4 complex"
 is_a: GO:0008047 ! enzyme activator activity
 relationship: positively_regulates GO:0016298 ! lipase activity
 
@@ -370825,9 +371586,10 @@
 namespace: molecular_function
 def: "Binds to and modulates the activity of an ATP hydrolysis activity." [GOC:dph, GOC:tb]
 synonym: "ATP hydrolysis regulator activity" EXACT []
-is_a: GO:0098772 ! molecular function regulator activity
-relationship: regulates GO:0140657 ! ATP-dependent activity
+is_a: GO:0060589 ! nucleoside-triphosphatase regulator activity
+relationship: regulates GO:0016887 ! ATP hydrolysis activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20955" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31398" xsd:anyURI
 created_by: dph
 creation_date: 2009-05-08T02:49:24Z
 
@@ -380099,7 +380861,7 @@
 id: GO:0061402
 name: obsolete positive regulation of transcription from RNA polymerase II promoter in response to acidic pH
 namespace: biological_process
-def: "OBSOLETE. Any process that increases the frequency, rate or extent of transcription from an RNA polymerase II promoter as a result of a pH stimulus with pH < 7." [GOC:dph, GOC:go_curators]
+def: "OBSOLETE. Any process that increases the frequency, rate or extent of transcription from an RNA polymerase II promoter as a result of a pH stimulus with pH < 7." [GOC:curators, GOC:dph]
 comment: This term was obsoleted because it represents a GO-CAM model.
 synonym: "positive regulation of transcription from RNA polymerase II promoter in response to acidity" BROAD []
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26004" xsd:anyURI
@@ -381106,12 +381868,14 @@
 
 [Term]
 id: GO:0061502
-name: early endosome to recycling endosome transport
+name: obsolete early endosome to recycling endosome transport
 namespace: biological_process
-def: "The directed movement of substances, in membrane-bounded vesicles, from the early sorting endosomes to the recycling endosomes." [GOC:dph, GOC:kmv, PMID:21474295]
-is_a: GO:0098927 ! vesicle-mediated transport between endosomal compartments
-relationship: occurs_in GO:0005737 ! cytoplasm
+def: "OBSOLETE. The directed movement of substances, in membrane-bounded vesicles, from the early sorting endosomes to the recycling endosomes." [GOC:dph, GOC:kmv, PMID:21474295]
+comment: This term is described by another GO term
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26386" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0032456
 created_by: dph
 creation_date: 2013-01-04T07:24:56Z
 
@@ -382277,7 +383041,6 @@
 synonym: "N-terminal propionyltransferase activity" EXACT [PMID:23043182]
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 is_a: GO:0140096 ! catalytic activity, acting on a protein
-relationship: part_of GO:0061606 ! N-terminal protein amino acid propionylation
 created_by: dph
 creation_date: 2014-02-11T14:24:35Z
 
@@ -382287,6 +383050,7 @@
 namespace: molecular_function
 def: "Combining with a nuclear import signal (NIS) on a cargo to be transported, to mediate transport of the cargo through the nuclear pore, from the cytoplasm to the nuclear lumen. The cargo can be either a RNA or a protein." [GOC:dph, GOC:pg, GOC:vw, PMID:28713609, Wikipedia:Nuclear_transport]
 synonym: "importin activity" RELATED []
+xref: Reactome:R-HSA-2065550 "SREBP1A,1C,2 binds Importin beta-1"
 is_a: GO:0140142 ! nucleocytoplasmic carrier activity
 intersection_of: GO:0140142 ! nucleocytoplasmic carrier activity
 intersection_of: part_of GO:0051170 ! import into nucleus
@@ -382714,6 +383478,15 @@
 xref: Reactome:R-HSA-9762091 "NFE2L2 is ubiquitinated by BTRC:SKP1:CUL1:RBX1"
 xref: Reactome:R-HSA-9766645 "CUL3:RBX1 ubiquitinates KEAP1"
 xref: Reactome:R-HSA-990526 "Recruitment of ITCH and K48 ubiquitination of MAVS"
+xref: Reactome:R-HSA-9929490 "CD274 ubiquitination by SPOP:CUL3:RBX1 complex"
+xref: Reactome:R-HSA-9931298 "Ubiquitination of CD274 by ERAD complex"
+xref: Reactome:R-HSA-9931430 "MIB2-mediated K63-ubiquitination of CD274"
+xref: Reactome:R-HSA-9932510 "BTRC:CUL1:SKP1:RBX1 (SCF-beta-TrCP1) ubiquitinylates phosphorylated PER1,2,3"
+xref: Reactome:R-HSA-9932512 "FBXL3:CUL1:SKP1:RBX1 ubiquitinylates phosphorylated CRY1,2"
+xref: Reactome:R-HSA-9945766 "NS3 is ubiquitinated"
+xref: Reactome:R-HSA-9946390 "NS1 dimer is ubiquitinated"
+xref: Reactome:R-HSA-9948362 "LTN1 K48 polyubiquitinates the nascent peptide in NEMF:60S ribosomal subunit:peptidyl-tRNA with nascent peptide"
+xref: Reactome:R-HSA-9954737 "RCHY1 K48 polyubiquitinates alanine-tailed nascent peptide"
 is_a: GO:0004842 ! ubiquitin-protein transferase activity
 is_a: GO:0061659 ! ubiquitin-like protein ligase activity
 property_value: skos:narrowMatch EC:2.3.2.26
@@ -382949,6 +383722,7 @@
 namespace: molecular_function
 def: "Isoenergetic transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y = Y-ISG15 + X, where both the X-ISG15 and Y-ISG15 linkages are thioester bonds between the C-terminal amino acid of ISG15 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
 synonym: "E2" BROAD [GOC:dph]
+xref: Reactome:R-HSA-1169404 "Transfer of ISG15 from E1 to E2 (UBCH8)"
 is_a: GO:0042296 ! ISG15 transferase activity
 is_a: GO:0061650 ! ubiquitin-like protein conjugating enzyme activity
 created_by: dph
@@ -383052,6 +383826,13 @@
 namespace: molecular_function
 def: "Catalysis of the transfer of a ISG15 to a substrate protein via the reaction X-ISG15 + S = X + S-ISG15, where X is either an E2 or E3 enzyme, the X-ISG15 linkage is a thioester bond, and the S-ISG15 linkage is an isopeptide bond between the C-terminal amino acid of ISG15 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
 synonym: "E3" BROAD [GOC:dph]
+xref: Reactome:R-HSA-1169394 "ISGylation of IRF3"
+xref: Reactome:R-HSA-1169395 "ISGylation of viral protein NS1"
+xref: Reactome:R-HSA-1169398 "ISGylation of host protein filamin B"
+xref: Reactome:R-HSA-1169402 "ISGylation of E2 conjugating enzymes"
+xref: Reactome:R-HSA-1169405 "ISGylation of protein phosphatase 1 beta (PP2CB)"
+xref: Reactome:R-HSA-1169406 "ISGylation of host proteins"
+xref: Reactome:R-HSA-9833973 "ISGylation of PKR"
 is_a: GO:0042296 ! ISG15 transferase activity
 is_a: GO:0061659 ! ubiquitin-like protein ligase activity
 created_by: dph
@@ -383204,6 +383985,7 @@
 name: importin-alpha family protein binding
 namespace: molecular_function
 def: "Binding to a member of the importin-alpha family." [PMID:15350979, PMID:17170104, PMID:23734157]
+xref: Reactome:R-HSA-2065539 "SREBP1A,1C,2:Importin beta-1 dissociates"
 is_a: GO:0005515 ! protein binding
 created_by: dph
 creation_date: 2015-01-10T18:25:22Z
@@ -383315,7 +384097,7 @@
 is_a: GO:0030163 ! protein catabolic process
 relationship: has_part GO:0005102 ! signaling receptor binding
 relationship: has_part GO:0051087 ! protein-folding chaperone binding
-relationship: has_part GO:0140597 ! protein carrier chaperone
+relationship: has_part GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20850" xsd:anyURI
 created_by: dph
 creation_date: 2015-03-04T14:35:26Z
@@ -383472,6 +384254,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: N(6)-glutaryl-L-lysyl-[protein] + NAD+ + H2O = 2''-O-glutaryl-ADP-D-ribose + nicotinamide + L-lysyl-[protein]." [PMID:24703693, RHEA:47664]
 comment: This reaction is the removal of a glutaryl group from a glutarylated lysine residue of a protein or peptide.
+xref: Reactome:R-HSA-9955504 "SIRT5 deglutarylates CPS1"
 xref: RHEA:47664
 is_a: GO:0141218 ! NAD-dependent protein lysine deacylase activity
 property_value: skos:exactMatch RHEA:47664
@@ -383632,6 +384415,7 @@
 def: "Catalysis of the reaction: L-threonine + hydrogencarbonate + ATP = L-threonylcarbamoyladenylate + diphosphate + H2O." [RHEA:36407]
 xref: EC:2.7.7.87
 xref: MetaCyc:RXN-14569
+xref: Reactome:R-HSA-9611218 "YRDC synthesizes threonylcarbamoyladenylate"
 xref: RHEA:36407
 is_a: GO:0016779 ! nucleotidyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.87
@@ -383649,7 +384433,8 @@
 synonym: "t6A synthase activity" EXACT []
 xref: EC:2.3.1.234
 xref: MetaCyc:RXN-14570
-xref: Reactome:R-HSA-6784494 "EKC complex threonylcarbamoylates A37 of tRNAs"
+xref: Reactome:R-HSA-6784494 "KEOPS (EKC) complex threonylcarbamoylates A37 of tRNAs"
+xref: Reactome:R-HSA-9611197 "OSGEPL1 threonylcarbamoylates adenosine-37 in tRNA"
 xref: RHEA:37059
 is_a: GO:0016747 ! acyltransferase activity, transferring groups other than amino-acyl groups
 is_a: GO:0140101 ! catalytic activity, acting on a tRNA
@@ -383687,6 +384472,7 @@
 namespace: molecular_function
 def: "Combining selectively with extracellular folic acid and delivering it into the cell via endocytosis." [GOC:BHF, GOC:hal]
 synonym: "folate receptor activity" EXACT []
+xref: Reactome:R-HSA-8940134 "FOLR2 binds FOLA"
 is_a: GO:0038024 ! cargo receptor activity
 relationship: has_part GO:0005542 ! folic acid binding
 relationship: part_of GO:0015884 ! folic acid transport
@@ -383947,6 +384733,7 @@
 xref: Reactome:R-HSA-9701531 "p-Y705,S727 STAT3 is acetylated"
 xref: Reactome:R-HSA-9756494 "CREBBP acetylates SARS-CoV-2 N at K375"
 xref: Reactome:R-HSA-9825747 "CREBBP acetylates HINT1 dimer"
+xref: Reactome:R-HSA-9920716 "CLOCK acetylates BMAL1 on lysine-538"
 xref: RHEA:45948
 is_a: GO:0034212 ! protein N-acetyltransferase activity
 property_value: skos:exactMatch EC:2.3.1.48
@@ -385187,7 +385974,7 @@
 name: telomeric G-quadruplex DNA binding
 namespace: molecular_function
 def: "Binding to telomeric G-quadruplex DNA structures, in which groups of four guanines adopt a flat, cyclic Hoogsteen hydrogen-bonding arrangement known as a guanine tetrad. The stacking of guanine tetrads results in G-quadruplex DNA structures in telomeres." [GOC:BHF, GOC:BHF_telomere, GOC:nc, PMID:16142245, PMID:9512530]
-is_a: GO:0042162 ! telomeric DNA binding
+is_a: GO:0042162 ! telomeric repeat DNA binding
 is_a: GO:0051880 ! G-quadruplex DNA binding
 
 [Term]
@@ -385957,6 +386744,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: glutathione = 5-oxoproline + L-cysteinylglycine." [PMID:23070364, PMID:27913623]
 xref: EC:4.3.2.7
+xref: Reactome:R-HSA-6785928 "CHAC1,2 cleaves GSH to OPRO and CysGly"
 xref: RHEA:47724
 is_a: GO:0016842 ! amidine-lyase activity
 property_value: skos:exactMatch EC:4.3.2.7
@@ -386049,7 +386837,6 @@
 namespace: biological_process
 def: "The binding and fusion of a sperm, with the plasma membrane of the oocyte as part of the process of double fertilization forming a zygote and endosperm." [PMID:21123745]
 is_a: GO:0022412 ! cellular process involved in reproduction in multicellular organism
-is_a: GO:0045026 ! plasma membrane fusion
 relationship: part_of GO:0009567 ! double fertilization forming a zygote and endosperm
 created_by: dph
 creation_date: 2017-09-05T18:43:24Z
@@ -386206,6 +386993,12 @@
 name: mRNA (adenine-N1-)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + adenine in mRNA = S-adenosyl-L-homocysteine + N(1)-methyladenine in mRNA." [PMID:29072297]
+xref: Reactome:R-HSA-9937508 "TRMT61B methylates adenosine-1472 of MT-CO1 mRNA"
+xref: Reactome:R-HSA-9937512 "TRMT61B methylates adenosine-371 of MT-ND1 mRNA"
+xref: Reactome:R-HSA-9937518 "TRMT61B methylates adenosine-333, adenosine-458, and adenosine-707 of MT-CO3 mRNA"
+xref: Reactome:R-HSA-9937521 "TRMT61B methylates adenosine-185 of MT-CO2 mRNA"
+xref: Reactome:R-HSA-9937529 "TRMT61B methylates adenosine-616 and adenosine-617 of MT-CYB mRNA"
+xref: Reactome:R-HSA-9937535 "TRMT61B methylates adenosine-252 of MT-ND4L mRNA"
 is_a: GO:0008174 ! mRNA methyltransferase activity
 created_by: dph
 creation_date: 2017-11-08T13:26:35Z
@@ -386713,12 +387506,14 @@
 name: regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "regulation of small molecule metabolism" EXACT []
 xref: Reactome:R-HSA-9861718 "Regulation of pyruvate metabolism"
 is_a: GO:0019222 ! regulation of metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044281 ! small molecule metabolic process
 relationship: regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:11:54Z
 
@@ -386727,12 +387522,14 @@
 name: positive regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "positive regulation of small molecule metabolism" EXACT []
 is_a: GO:0009893 ! positive regulation of metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0044281 ! small molecule metabolic process
 relationship: positively_regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:25:10Z
 
@@ -386741,12 +387538,14 @@
 name: negative regulation of small molecule metabolic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of a small molecule metabolic process." [GOC:vw]
+subset: gocheck_do_not_annotate
 synonym: "negative regulation of small molecule metabolism" EXACT []
 is_a: GO:0009892 ! negative regulation of metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0044281 ! small molecule metabolic process
 relationship: negatively_regulates GO:0044281 ! small molecule metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: dph
 creation_date: 2018-03-19T14:27:23Z
 
@@ -387024,10 +387823,12 @@
 
 [Term]
 id: GO:0062045
-name: L-lysine alpha-aminotransferase activity
+name: L-lysine:pyruvate alpha-transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-lysine + pyruvate= epsilon-amino-alpha-ketocaproic acid (KAC) + alanine." [PMID:27758894, PMID:28330936]
-is_a: GO:0008483 ! transaminase activity
+synonym: "L-lysine alpha-aminotransferase activity" BROAD []
+is_a: GO:0140385 ! amino acid transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dph
 creation_date: 2018-05-11T12:13:57Z
 
@@ -388044,7 +388845,7 @@
 id: GO:0062146
 name: 4-chloro-allylglycine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-chloro-L-lysine + AH2 + O2 = A + formaldehyde + H2O + L-2-amino-4-chloropent-4-enoate + NH4(+)." [PMID:30867596, RHEA:59888]
+def: "Catalysis of the reaction: 4-chloro-L-lysine + AH2 + O2 = A + formaldehyde + H2O + L-2-amino-4-chloropent-4-enoate + NH4+." [PMID:30867596, RHEA:59888]
 xref: RHEA:59888
 is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
 property_value: skos:exactMatch RHEA:59888
@@ -389313,11 +390114,12 @@
 
 [Term]
 id: GO:0070007
-name: glutamic-type endopeptidase activity
+name: obsolete glutamic-type endopeptidase activity
 namespace: molecular_function
-def: "Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE, https://www.ebi.ac.uk/merops/about/glossary.shtml#ENDOPEPTIDASE]
-is_a: GO:0004175 ! endopeptidase activity
-is_a: GO:0070002 ! glutamic-type peptidase activity
+def: "OBSOLETE. Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE, https://www.ebi.ac.uk/merops/about/glossary.shtml#ENDOPEPTIDASE]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0070008
@@ -389591,9 +390393,10 @@
 
 [Term]
 id: GO:0070039
-name: rRNA (guanosine-2'-O-)-methyltransferase activity
+name: rRNA (guanosine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylguanosine." [GOC:imk, GOC:mah]
+synonym: "rRNA (guanosine-2'-O-)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-6793096 "MRM3 (RNMTL1) methylates guanosine-1370 of 16S rRNA yielding 2'-O-methylguanosine-1370"
 xref: Reactome:R-HSA-6793122 "MRM1 methylates guanosine-1145 of 16S rRNA yielding 2'-O-methylguanosine-1145"
 is_a: GO:0016435 ! rRNA (guanine) methyltransferase activity
@@ -390463,6 +391266,7 @@
 def: "The process resulting in the release of a polypeptide chain from the ribosome in a mitochondrion, usually in response to a termination codon (note that mitochondria use variants of the universal genetic code that differ between different taxa)." [GOC:mah, http://mitogenome.org/index.php/Genetic_Code_of_mitochondria]
 synonym: "mitochondrial translation termination" EXACT [GOC:mah]
 xref: Reactome:R-HSA-5419276 "Mitochondrial translation termination"
+xref: Reactome:R-HSA-9937383 "Mitochondrial ribosome-associated quality control"
 is_a: GO:0006415 ! translational termination
 intersection_of: GO:0006415 ! translational termination
 intersection_of: occurs_in GO:0005739 ! mitochondrion
@@ -390984,11 +391788,15 @@
 
 [Term]
 id: GO:0070178
-name: D-serine metabolic process
+name: obsolete D-serine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-serine, the D-enantiomer of serine, i.e. (2R)-2-amino-3-hydroxypropanoic acid." [CHEBI:16523, GOC:jsg, GOC:mah]
+def: "OBSOLETE. The chemical reactions and pathways involving D-serine, the D-enantiomer of serine, i.e. (2R)-2-amino-3-hydroxypropanoic acid." [CHEBI:16523, GOC:jsg, GOC:mah]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-serine metabolism" EXACT []
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
+consider: GO:0036088
+consider: GO:0070179
 
 [Term]
 id: GO:0070179
@@ -391000,7 +391808,6 @@
 synonym: "D-serine formation" EXACT []
 synonym: "D-serine synthesis" EXACT []
 is_a: GO:0046437 ! D-amino acid biosynthetic process
-is_a: GO:0070178 ! D-serine metabolic process
 
 [Term]
 id: GO:0070180
@@ -391494,12 +392301,14 @@
 
 [Term]
 id: GO:0070226
-name: sulfur:ferric ion oxidoreductase activity
+name: obsolete sulfur:ferric ion oxidoreductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a perthiol + 4 Fe3+ + 3 H2O = sulfite + a thiol + 4 Fe2+ + 8 H+." [MetaCyc:SULFFEOXIDO-RXN]
+def: "OBSOLETE. Catalysis of the reaction: S-sulfanylglutathione + 4 Fe3+ + 3 H2O = sulfite + 4 Fe2+ + glutathione + 6 H+." [GOC:curators]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "hydrogen sulfide:ferric ion oxidoreductase" EXACT [MetaCyc:SULFFEOXIDO-RXN]
-xref: MetaCyc:SULFFEOXIDO-RXN
-is_a: GO:0016667 ! oxidoreductase activity, acting on a sulfur group of donors
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31518" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0070227
@@ -391898,7 +392707,6 @@
 xref: Reactome:R-HSA-389632 "pristanate + CoA-SH + ATP => pristanoyl-CoA + AMP + pyrophosphate"
 xref: RHEA:47264
 is_a: GO:0016405 ! CoA-ligase activity
-is_a: GO:0016878 ! acid-thiol ligase activity
 property_value: skos:exactMatch RHEA:47264
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 
@@ -392079,6 +392887,8 @@
 def: "A gasdermin-dependent inflammatory response that is associated with the generation of pyrogenic mediators such as IL-1beta and IL-18. Gasdermins are activated by caspase-1 or caspase-4/11, or by certain granzymes. In some, but not all cells, it can lead to pyroptotic programmed cell death." [PMID:18846107, PMID:21760595, PMID:33883744]
 synonym: "pyroptosis" BROAD []
 xref: Reactome:R-HSA-5620971 "Pyroptosis"
+xref: Reactome:R-HSA-9960519 "CASP4-mediated substrate cleavage"
+xref: Reactome:R-HSA-9960525 "CASP5-mediated substrate cleavage"
 is_a: GO:0006954 ! inflammatory response
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27753" xsd:anyURI
 
@@ -392132,7 +392942,7 @@
 id: GO:0070275
 name: aerobic ammonia oxidation to nitrite via pyruvic oxime
 namespace: biological_process
-def: "The metabolic process in which ammonia (NH3) is oxidized to nitrite (NO2) in the presence of oxygen. Hydroxylamine is produced enzymatically, and, in the presence of pyruvate, forms pyruvic oxime in a spontaneous, non-enzymatic reaction; pyruvic oxime is then converted to nitrite." [MetaCyc:PWY-2242]
+def: "The metabolic process in which ammonia (NH4+) is oxidized to nitrite (NO2) in the presence of oxygen. Hydroxylamine is produced enzymatically, and, in the presence of pyruvate, forms pyruvic oxime in a spontaneous, non-enzymatic reaction; pyruvic oxime is then converted to nitrite." [MetaCyc:PWY-2242]
 xref: MetaCyc:PWY-2242
 is_a: GO:0019329 ! ammonia oxidation
 
@@ -394225,7 +395035,7 @@
 id: GO:0070472
 name: regulation of uterine smooth muscle contraction
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of uterine smooth muscle contraction." [GOC:go_curators]
+def: "Any process that modulates the frequency, rate or extent of uterine smooth muscle contraction." [GOC:curators]
 synonym: "regulation of myometrial contraction" EXACT [GOC:sl]
 synonym: "regulation of myometrial smooth muscle contraction" EXACT [GOC:sl]
 synonym: "regulation of myometrium contraction" EXACT [GOC:sl]
@@ -394238,7 +395048,7 @@
 id: GO:0070473
 name: negative regulation of uterine smooth muscle contraction
 namespace: biological_process
-def: "Any process that decreases the frequency, rate or extent of uterine smooth muscle contraction." [GOC:go_curators]
+def: "Any process that decreases the frequency, rate or extent of uterine smooth muscle contraction." [GOC:curators]
 synonym: "down regulation of uterine smooth muscle contraction" EXACT [GOC:mah]
 synonym: "down-regulation of uterine smooth muscle contraction" EXACT [GOC:mah]
 synonym: "downregulation of uterine smooth muscle contraction" EXACT [GOC:mah]
@@ -394257,7 +395067,7 @@
 id: GO:0070474
 name: positive regulation of uterine smooth muscle contraction
 namespace: biological_process
-def: "Any process that increases the frequency, rate or extent of uterine smooth muscle contraction." [GOC:go_curators]
+def: "Any process that increases the frequency, rate or extent of uterine smooth muscle contraction." [GOC:curators]
 synonym: "activation of uterine smooth muscle contraction" NARROW [GOC:mah]
 synonym: "positive regulation of myometrial contraction" EXACT [GOC:sl]
 synonym: "positive regulation of myometrial smooth muscle contraction" EXACT [GOC:sl]
@@ -394756,6 +395566,7 @@
 name: 11-beta-hydroxysteroid dehydrogenase (NAD+) activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an 11-beta-hydroxysteroid + NAD+ = an 11-oxosteroid + NADH + H+." [PMID:15761036, RHEA:53116]
+xref: Reactome:R-HSA-194023 "HSD11B2,HSD11B1 dimer oxidise CORT to COR"
 xref: RHEA:53116
 xref: Wikipedia:11beta-hydroxysteroid_dehydrogenase
 is_a: GO:0033764 ! steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
@@ -394817,10 +395628,14 @@
 
 [Term]
 id: GO:0070529
-name: L-tryptophan aminotransferase activity
+name: obsolete L-tryptophan transaminase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of an amino group from L-tryptophan to an acceptor, usually a 2-oxo acid." [GOC:mah]
-is_a: GO:0008483 ! transaminase activity
+def: "OBSOLETE. Catalysis of the transfer of an amino group from L-tryptophan to an acceptor, usually a 2-oxo acid." [GOC:mah]
+comment: This term was obsoleted because it is an unnecessary grouping term.
+synonym: "L-tryptophan aminotransferase activity" EXACT []
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0047299
 
 [Term]
 id: GO:0070530
@@ -394972,10 +395787,14 @@
 
 [Term]
 id: GO:0070548
-name: L-glutamine aminotransferase activity
+name: obsolete L-glutamine transaminase activity
 namespace: molecular_function
-def: "Catalysis of the transfer of an amino group from L-glutamine to an acceptor, usually a 2-oxo acid." [GOC:mah]
-is_a: GO:0008483 ! transaminase activity
+def: "OBSOLETE. Catalysis of the transfer of an amino group from L-glutamine to an acceptor, usually a 2-oxo acid." [GOC:mah]
+comment: This term was obsoleted because it is too general. Consider other glutamine transaminase activity terms.
+synonym: "L-glutamine aminotransferase activity" EXACT []
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+is_obsolete: true
+consider: GO:0047945
 
 [Term]
 id: GO:0070549
@@ -395173,6 +395992,7 @@
 name: adenylyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of an adenylyl group to an acceptor." [GOC:mah]
+xref: Reactome:R-HSA-9935837 "The TRAMP:RNA exosome oligoadenylates pre-rRNA 5' external transcribed spacer (5'ETS) fragment"
 is_a: GO:0016779 ! nucleotidyltransferase activity
 created_by: mah
 creation_date: 2009-04-15T11:04:22Z
@@ -395258,9 +396078,9 @@
 def: "Catalysis of the hydrolysis of a dipeptide by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, https://www.ebi.ac.uk/merops/about/glossary.shtml#CATTYPE]
 synonym: "metallo-exo-dipeptidase activity" EXACT [GOC:rb]
 synonym: "metalloexodipeptidase activity" EXACT [GOC:mah]
-xref: Reactome:R-HSA-2022398 "ACE:Zn2+ hydrolyzes Angiotensin-(1-9) to Angiotensin-(1-7)"
-xref: Reactome:R-HSA-2022405 "ACE:Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
-xref: Reactome:R-HSA-2065355 "Secreted ACE:Zn2+ hydrolyzes Angiotensin-(1-10) to Angiotensin-(1-8)"
+xref: Reactome:R-HSA-2022398 "ACE:Zn2+ hydrolyzes AGT(25-33) (Angiotensin-(1-9)) to AGT(25-31) (Angiotensin-(1-7))"
+xref: Reactome:R-HSA-2022405 "ACE:Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
+xref: Reactome:R-HSA-2065355 "Secreted ACE:Zn2+ hydrolyzes AGT(25-34) (Angiotensin-(1-10)) to AGT(25-32) (Angiotensin-(1-8))"
 is_a: GO:0008235 ! metalloexopeptidase activity
 is_a: GO:0016805 ! dipeptidase activity
 created_by: mah
@@ -395465,6 +396285,7 @@
 synonym: "calcium ion membrane transport" EXACT []
 synonym: "transmembrane calcium transport" EXACT [GOC:mah]
 xref: Reactome:R-HSA-3295583 "TRP channels"
+xref: Reactome:R-HSA-425561 "Sodium/Calcium exchangers"
 is_a: GO:0006816 ! calcium ion transport
 is_a: GO:0098655 ! monoatomic cation transmembrane transport
 created_by: mah
@@ -396609,9 +397430,10 @@
 
 [Term]
 id: GO:0070677
-name: rRNA (cytosine-2'-O-)-methyltransferase activity
+name: rRNA (cytosine-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylcytosine." [GOC:mah, PMID:19400805]
+synonym: "rRNA (cytosine-2'-O-)-methyltransferase activity" EXACT []
 is_a: GO:0016434 ! rRNA (cytosine) methyltransferase activity
 is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
 property_value: skos:broadMatch RHEA:58956
@@ -397678,7 +398500,7 @@
 id: GO:0070773
 name: protein-N-terminal glutamine amidohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N-terminal L-glutaminyl-[protein] + H2O = N-terminal L-glutamyl-[protein] + NH3. This reaction is the deamidation of an N-terminal glutamine residue of a protein." [PMID:19560421, RHEA:50680]
+def: "Catalysis of the reaction: N-terminal L-glutaminyl-[protein] + H2O = N-terminal L-glutamyl-[protein] + NH4+. This reaction is the deamidation of an N-terminal glutamine residue of a protein." [PMID:19560421, RHEA:50680]
 synonym: "NtQ-amidase activity" EXACT [PMID:19560421]
 xref: EC:3.5.1.122
 xref: RHEA:50680
@@ -397697,7 +398519,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: a phytoceramide + H2O = a fatty acid + phytosphingosine." [GOC:pde, PMID:11356846]
 comment: The reason for obsoletion is that this term represents a specific substrate of GO:0017040 N-acylsphingosine amidohydrolase activity.
-xref: Reactome:R-HSA-428262 "ACER3 hydrolyzes phytoceramide"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28294" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0017040
@@ -400224,8 +401045,13 @@
 synonym: "L-asparagine biosynthesis" EXACT [GOC:mah]
 synonym: "L-asparagine formation" EXACT [GOC:mah]
 synonym: "L-asparagine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:ASPARAGINE-BIOSYNTHESIS
+xref: MetaCyc:ASPARAGINESYN-PWY
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 is_a: GO:0070982 ! L-asparagine metabolic process
+property_value: skos:narrowMatch MetaCyc:ASPARAGINE-BIOSYNTHESIS
+property_value: skos:narrowMatch MetaCyc:ASPARAGINESYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-10-27T12:59:42Z
 
@@ -402931,7 +403757,7 @@
 id: GO:0071229
 name: cellular response to acid chemical
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus by the chemical structure of the anion portion of the dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:go_curators, GOC:mah, Wikipedia:Acid]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus by the chemical structure of the anion portion of the dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:curators, GOC:mah, Wikipedia:Acid]
 comment: This term should be used to describe a response to a specific acid as a chemical. E.g., if a cell were responding to glutamate, then the response would be glutamate-specific; the cell is actually responding to the chemical structure of the anion portion of the dissociated acid. Note that this term is in the subset of terms that should not be used for direct gene product annotation. Instead, select a child term or, if no appropriate child term exists, please request a new term. Direct annotations to this term may be amended during annotation QC. If annotating experiments where an acid is playing a role as a proton donor, please annotate to GO:0071468 'cellular response to acidic pH' instead.
 subset: gocheck_do_not_annotate
 synonym: "cellular response to acid" BROAD []
@@ -403372,6 +404198,7 @@
 is_a: GO:0006555 ! methionine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31068" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:28:15Z
 
@@ -403379,12 +404206,26 @@
 id: GO:0071266
 name: 'de novo' L-methionine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-methionine, the L-enantiomer of (2S)-2-amino-4-(methylsulfanyl)butanoic acid, from simpler components." [GOC:ecd]
+def: "The chemical reactions and pathways resulting in the formation of L-methionine, the L-enantiomer of (2S)-2-amino-4-(methylsulfanyl)butanoic acid, from simpler components such as L-aspartate or L-homoserine. This process occurs in plants, bacteria, archae and fungi." [PMID:12818659, PMID:24939187]
 synonym: "'de novo' L-methionine anabolism" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine biosynthesis" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine formation" EXACT [GOC:mah]
 synonym: "'de novo' L-methionine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:HOMOSER-METSYN-PWY
+xref: MetaCyc:HSERMETANA-PWY
+xref: MetaCyc:PWY-5345
+xref: MetaCyc:PWY-5347
+xref: MetaCyc:PWY-702
+xref: MetaCyc:PWY-7977
 is_a: GO:0071265 ! L-methionine biosynthetic process
+property_value: skos:narrowMatch MetaCyc:HOMOSER-METSYN-PWY
+property_value: skos:narrowMatch MetaCyc:HSERMETANA-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5345
+property_value: skos:narrowMatch MetaCyc:PWY-5347
+property_value: skos:narrowMatch MetaCyc:PWY-702
+property_value: skos:narrowMatch MetaCyc:PWY-7977
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31318" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:34:41Z
 
@@ -403393,9 +404234,18 @@
 name: L-methionine salvage
 namespace: biological_process
 def: "Any process that generates L-methionine from derivatives of it, without de novo synthesis." [GOC:ecd]
+xref: MetaCyc:ADENOSYLHOMOCYSCAT-PWY
+xref: MetaCyc:PWY-4361
+xref: MetaCyc:PWY-7174
+xref: MetaCyc:PWY-8132
 is_a: GO:0043102 ! amino acid salvage
 is_a: GO:0071265 ! L-methionine biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ADENOSYLHOMOCYSCAT-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-4361
+property_value: skos:narrowMatch MetaCyc:PWY-7174
+property_value: skos:narrowMatch MetaCyc:PWY-8132
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29900" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T04:36:49Z
 
@@ -403419,16 +404269,18 @@
 id: GO:0071269
 name: L-homocysteine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of L-homocysteine, the L-enantiomer of 2-amino-4-sulfanylbutanoic acid." [GOC:ecd, GOC:mah]
+def: "The chemical reactions and pathways resulting in the formation of L-homocysteine." [GOC:curators]
 synonym: "L-homocysteine anabolism" EXACT [GOC:mah]
 synonym: "L-homocysteine biosynthesis" EXACT [GOC:mah]
 synonym: "L-homocysteine formation" EXACT [GOC:mah]
 synonym: "L-homocysteine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:PWY-5344
 is_a: GO:0000097 ! sulfur amino acid biosynthetic process
 is_a: GO:0009070 ! serine family amino acid biosynthetic process
 is_a: GO:0050667 ! homocysteine metabolic process
 is_a: GO:0170034 ! L-amino acid biosynthetic process
 is_a: GO:0170043 ! non-proteinogenic amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: mah
 creation_date: 2009-12-09T05:36:37Z
 
@@ -405726,7 +406578,7 @@
 id: GO:0071468
 name: cellular response to acidic pH
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH < 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:go_curators, GOC:mah, Wikipedia:PH]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH < 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:curators, GOC:mah, Wikipedia:PH]
 comment: This term should be used to annotate instances where a cell is responding to a chemical that is playing the role of an acid (e.g. proton donor) and therefore lowering the pH. If instead you wish to describe a response to a specific acid as a chemical, such as the anion portion of glutamate, please annotate to the appropriate child of GO:0071229 'cellular response to acid chemical'.
 synonym: "cellular response to acidity" BROAD []
 is_a: GO:0010447 ! response to acidic pH
@@ -405738,7 +406590,7 @@
 id: GO:0071469
 name: cellular response to alkaline pH
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH > 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:go_curators, GOC:mah, Wikipedia:PH]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pH stimulus with pH > 7. pH is a measure of the acidity or basicity of an aqueous solution." [GOC:curators, GOC:mah, Wikipedia:PH]
 synonym: "cellular response to alkalinity" BROAD []
 synonym: "cellular response to basic pH" EXACT [GOC:mah]
 is_a: GO:0010446 ! response to alkaline pH
@@ -406389,7 +407241,7 @@
 id: GO:0071522
 name: ureidoglycine aminohydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ureidoglycine + H2O = S-ureidoglycolate + NH3." [MetaCyc:URUR-RXN, PMID:19935661, PMID:20038185]
+def: "Catalysis of the reaction: (S)-2-ureidoglycine + H2O = (S)-ureidoglycolate + NH4+." [PMID:19935661, PMID:20038185, RHEA:25241]
 xref: EC:3.5.3.26
 xref: MetaCyc:URUR-RXN
 xref: RHEA:25241
@@ -406414,15 +407266,17 @@
 id: GO:0071524
 name: pyrrolysine biosynthetic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the formation of pyrrolysine, N6-{[(2R,3R)-3-methyl-3,4-dihydro-2H-pyrrol-2-yl]carbonyl}-L-lysine." [GOC:dh, PMID:17204561]
+def: "The chemical reactions and pathways resulting in the formation of pyrrolysine." [GOC:curators, PMID:17204561]
 synonym: "monomethylamine methyltransferase cofactor lysine adduct biosynthetic process" EXACT [CHEBI:21860]
 synonym: "pyrrolysine anabolism" EXACT [GOC:mah]
 synonym: "pyrrolysine biosynthesis" EXACT [GOC:mah]
 synonym: "pyrrolysine formation" EXACT [GOC:mah]
 synonym: "pyrrolysine synthesis" EXACT [GOC:mah]
+xref: MetaCyc:PWY-6994
 is_a: GO:0008652 ! amino acid biosynthetic process
 is_a: GO:0042398 ! modified amino acid biosynthetic process
 is_a: GO:0170038 ! proteinogenic amino acid biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: mah
 creation_date: 2010-01-07T02:40:24Z
 
@@ -407321,7 +408175,7 @@
 id: GO:0071598
 name: neuronal ribonucleoprotein granule
 namespace: cellular_component
-def: "A ribonucleoprotein complex that is found in the cytoplasm of axons and dendrites, and transports translationally silenced mRNAs to dendritic synapses, where they are released and translated in response to specific exogenous stimuli." [GOC:BHF, GOC:go_curators, GOC:mah, PMID:19015237, PMID:20368989]
+def: "A ribonucleoprotein complex that is found in the cytoplasm of axons and dendrites, and transports translationally silenced mRNAs to dendritic synapses, where they are released and translated in response to specific exogenous stimuli." [GOC:BHF, GOC:curators, GOC:mah, PMID:19015237, PMID:20368989]
 synonym: "neuronal RNA granule" NARROW []
 synonym: "neuronal RNP granule" EXACT []
 is_a: GO:0036464 ! cytoplasmic ribonucleoprotein granule
@@ -411418,6 +412272,7 @@
 def: "Binding to the oxidized form, FAD, of flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [GOC:mah]
 synonym: "oxidized flavin adenine dinucleotide binding" EXACT [GOC:mah]
 synonym: "oxidized flavine-adenine dinucleotide binding" EXACT [GOC:mah]
+xref: Reactome:R-HSA-9854672 "FAD attaches to SDHA"
 is_a: GO:0050660 ! flavin adenine dinucleotide binding
 created_by: mah
 creation_date: 2010-10-07T12:30:28Z
@@ -415491,7 +416346,7 @@
 def: "The directed movement of proteins from the plasma membrane across the periplasmic space to the outer membrane or cell wall." [GOC:mah, PMID:20378773]
 synonym: "chaperone-mediated protein transport across periplasmic space" NARROW []
 is_a: GO:0015031 ! protein transport
-relationship: has_part GO:0140597 ! protein carrier chaperone
+relationship: has_part GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 created_by: mah
 creation_date: 2010-10-28T04:24:59Z
@@ -415735,6 +416590,7 @@
 name: rescue of stalled cytosolic ribosome
 namespace: biological_process
 def: "A process of cytosolic translational elongation that takes place when a cytosolic ribosome has stalled during translation, and results in freeing the ribosome from the stalled translation complex." [GOC:jh2, GOC:mah, PMID:18557701, PMID:19170872, PMID:20117091, PMID:20185543]
+xref: Reactome:R-HSA-9948299 "Ribosome-associated quality control"
 is_a: GO:0002182 ! cytoplasmic translational elongation
 is_a: GO:0032790 ! ribosome disassembly
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22227" xsd:anyURI
@@ -417044,6 +417900,7 @@
 synonym: "ammonia transport" BROAD []
 synonym: "ammonium membrane transport" EXACT []
 synonym: "ammonium transport" BROAD []
+xref: Reactome:R-HSA-444411 "Rhesus glycoproteins mediate ammonium transport"
 is_a: GO:0055085 ! transmembrane transport
 is_a: GO:0071705 ! nitrogen compound transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22228" xsd:anyURI
@@ -418059,7 +418916,6 @@
 namespace: molecular_function
 def: "Binds to and modulates the activity of ATP-hydrolyzing DNA topoisomerase. DNA topoisomerase (ATP-hydrolyzing) regulator activity catalyzes a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:mah]
 is_a: GO:0030234 ! enzyme regulator activity
-is_a: GO:0060590 ! ATPase regulator activity
 relationship: regulates GO:0003918 ! DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity
 created_by: mah
 creation_date: 2011-02-09T05:31:50Z
@@ -418610,7 +419466,7 @@
 name: osteoclast fusion
 namespace: biological_process
 def: "The plasma membrane fusion process that results in fusion of mononuclear osteoclasts to form a multinuclear osteoclast." [CL:0000092, CL:0000779, GOC:BHF, GOC:mah, PMID:12713016]
-is_a: GO:0000768 ! syncytium formation by plasma membrane fusion
+is_a: GO:0000768 ! syncytium formation by cell-cell fusion
 relationship: part_of GO:0072674 ! multinuclear osteoclast differentiation
 created_by: mah
 creation_date: 2011-02-22T02:14:34Z
@@ -423306,12 +424162,14 @@
 id: GO:0080002
 name: UDP-glucose:4-aminobenzoate acylglucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-aminobenzoate + UDP-alpha-D-glucose = 1-O-(4-aminobenzoyl)-beta-D-glucose + UDP." [PMID:18385129]
+def: "Catalysis of the reaction: 4-aminobenzoate + UDP-alpha-D-glucose = 1-O-(4-aminobenzoyl)-beta-D-glucose + UDP." [PMID:18385129, RHEA:85851]
 synonym: "UDP-glucose:p-aminobenzoate acylglucosyltransferase activity" RELATED []
 synonym: "UDP-glucose:p-aminobenzoate glucosyltransferase activity" RELATED []
 synonym: "UDP-glucose:pABA acylglucosyltransferase activity" RELATED []
 xref: MetaCyc:RXN-6142
+xref: RHEA:85851
 is_a: GO:0035251 ! UDP-glucosyltransferase activity
+property_value: skos:exactMatch RHEA:85851
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28273" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28775" xsd:anyURI
 
@@ -423923,10 +424781,11 @@
 id: GO:0080061
 name: indole-3-acetonitrile nitrilase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: indole-3-acetonitrile + 2 H2O = indole-3-acetic acid + NH3." [RHEA:45776]
+def: "Catalysis of the reaction: (indol-3-yl)acetonitrile + 2 H2O = (indol-3-yl)acetate + NH4+." [RHEA:45776]
 xref: MetaCyc:RXN-1404
 xref: RHEA:45776
 is_a: GO:0000257 ! nitrilase activity
+property_value: skos:broadMatch EC:3.5.5.1
 property_value: skos:exactMatch RHEA:45776
 created_by: dhl
 creation_date: 2009-04-13T03:30:42Z
@@ -424195,10 +425054,12 @@
 name: regulation of primary metabolic process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism involving those compounds formed as a part of the normal anabolic and catabolic processes. These processes take place in most, if not all, cells of the organism." [PMID:19211694]
+subset: gocheck_do_not_annotate
 is_a: GO:0019222 ! regulation of metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044238 ! primary metabolic process
 relationship: regulates GO:0044238 ! primary metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31498" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-22T04:30:52Z
 
@@ -424271,40 +425132,48 @@
 
 [Term]
 id: GO:0080097
-name: L-tryptophan:pyruvate aminotransferase activity
+name: L-tryptophan:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tryptophan + pyruvate = 3-(indol-3-yl)pyruvate + L-alanine." [RHEA:27586]
+def: "Catalysis of the reaction: L-tryptophan + pyruvate = indole-3-pyruvate + L-alanine." [RHEA:27586]
+synonym: "L-tryptophan:pyruvate aminotransferase activity" EXACT []
 xref: EC:2.6.1.99
 xref: MetaCyc:RXN-10139
 xref: RHEA:27586
-is_a: GO:0070529 ! L-tryptophan aminotransferase activity
+is_a: GO:0047635 ! L-alanine:oxo-acid transaminase activity
 property_value: skos:exactMatch EC:2.6.1.99
 property_value: skos:exactMatch RHEA:27586
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:45:23Z
 
 [Term]
 id: GO:0080098
-name: L-tyrosine-pyruvate transaminase activity
+name: L-tyrosine:pyruvate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-tyrosine + pyruvate = (4-hydroxyphenyl)pyruvate + L-alanine." [MetaCyc:RXN3O-4157]
+def: "Catalysis of the reaction: L-tyrosine + pyruvate = 3-(4-hydroxyphenyl)pyruvate + L-alanine." [MetaCyc:RXN3O-4157]
 synonym: "L-tyrosine aminotransferase activity" BROAD []
+synonym: "L-tyrosine-pyruvate transaminase activity" EXACT []
 synonym: "L-tyrosine:pyruvate aminotransferase activity" EXACT []
 xref: MetaCyc:RXN3O-4157
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0140385 ! amino acid transaminase activity
 property_value: skos:broadMatch EC:2.6.1.58
+property_value: skos:exactMatch MetaCyc:RXN3O-4157
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:47:53Z
 
 [Term]
 id: GO:0080099
-name: L-methionine:2-oxoglutarate aminotransferase activity
+name: L-methionine:2-oxoglutarate transaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-methionine + 2-oxoglutarate = 4-methylthio-2-oxobutyrate + L-glutamate." [PMID:18394996]
-comment: This reaction falls within the larger set of reactions associated with EC:2.6.1.5 (See BRENDA: http://www.brenda-enzymes.org/php/result_flat.php4?ecno=2.6.1.5).
+def: "Catalysis of the reaction: L-methionine + 2-oxoglutarate = 4-methylsulfanyl-2-oxobutanoate + L-glutamate." [PMID:18394996, RHEA:25796]
+synonym: "L-methionine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-methionine:alpha-ketoglutarate aminotransferase activity" EXACT []
-is_a: GO:0010326 ! methionine-oxo-acid transaminase activity
+xref: MetaCyc:RXN-14147
+xref: RHEA:25796
+is_a: GO:0010326 ! L-methionine:oxo-acid transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-23T03:51:20Z
 
@@ -424654,20 +425523,21 @@
 
 [Term]
 id: GO:0080130
-name: L-phenylalanine-2-oxoglutarate transaminase activity
+name: L-phenylalanine:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: L-phenylalanine + 2-oxoglutarate = 3-phenylpyruvate + L-glutamate." [PMID:18394996, RHEA:25152]
+synonym: "L-phenylalanine-2-oxoglutarate transaminase activity" EXACT []
 synonym: "L-phenylalanine:2-oxoglutarate aminotransferase activity" EXACT []
 synonym: "L-phenylalanine:alpha-ketoglutarate aminotransferase activity" EXACT []
 xref: KEGG_REACTION:R00694
 xref: MetaCyc:RXN-10814
 xref: RHEA:25152
-is_a: GO:0008793 ! aromatic-amino-acid transaminase activity
+is_a: GO:0008793 ! aromatic-amino-acid:2-oxoglutarate transaminase activity
 property_value: skos:broadMatch EC:2.6.1.5
-property_value: skos:broadMatch EC:2.6.1.57
 property_value: skos:exactMatch RHEA:25152
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28070" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31139" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: dhl
 creation_date: 2009-04-29T03:42:47Z
 
@@ -424847,8 +425717,6 @@
 def: "Catalysis of the reaction: L-cysteine + H2O = hydrogen sulfide + pyruvate + NH4+ + H+." [PMID:19955263, RHEA:24931]
 xref: EC:4.4.1.28
 xref: MetaCyc:LCYSDESULF-RXN
-xref: Reactome:R-HSA-1614591 "Excess cysteine yields lanthionine and H2S"
-xref: Reactome:R-HSA-1614614 "Cysteine is degraded to serine and H2S"
 xref: RHEA:24931
 is_a: GO:0016846 ! carbon-sulfur lyase activity
 property_value: skos:broadMatch EC:4.4.1.1
@@ -425125,7 +425993,7 @@
 id: GO:0080170
 name: hydrogen peroxide transmembrane transport
 namespace: biological_process
-def: "The process in which hydrogen peroxide is transported across a membrane." [GOC:tb]
+def: "The process in which H2O2 is transported across a membrane." [GOC:tb]
 comment: Note that this term is not intended for use in annotating lateral movement within membranes.
 synonym: "hydrogen peroxide membrane transport" EXACT []
 is_a: GO:0055085 ! transmembrane transport
@@ -431720,6 +432588,7 @@
 name: regulation of ornithine metabolic process
 namespace: biological_process
 def: "Any process that modulates the rate, frequency, or extent of the chemical reactions and pathways involving ornithine, an amino acid only rarely found in proteins, but which is important in living organisms as an intermediate in the reactions of the urea cycle and in arginine biosynthesis." [GOC:dph, GOC:jp, GOC:tb]
+xref: Reactome:R-HSA-350562 "Regulation of ornithine decarboxylase (ODC)"
 is_a: GO:0006521 ! regulation of amino acid metabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
@@ -432280,7 +433149,7 @@
 name: nicotinate transmembrane transporter activity
 namespace: molecular_function
 def: "Enables the transfer of nicotinate from one side of a membrane to the other." [GOC:tb]
-xref: Reactome:R-HSA-8869603 "SLC22A13 transports NCA from extracellular region to cytosol"
+xref: Reactome:R-HSA-8869603 "SLC22A13 exchanges extracellular NCA for cytosolic urate"
 is_a: GO:0008028 ! monocarboxylic acid transmembrane transporter activity
 relationship: part_of GO:2001142 ! nicotinate transport
 created_by: tb
@@ -432474,7 +433343,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: ATP + palmitic acid + CoA = AMP + diphosphate + palmitoyl-CoA." [GOC:al, PMID:18071249]
 synonym: "palmitoyl-CoA synthetase activity" EXACT []
-xref: Reactome:R-HSA-434382 "ACSL3,4 ligates coenzyme A (CoA-SH) to palmitate yielding palmitoyl-coenzyme A in the pancreatic beta cell"
+xref: Reactome:R-HSA-434382 "ACSL3,4 ligates coenzyme A (CoA-SH) to a long chain fatty acid yielding fatty acyl-coenzyme A"
 is_a: GO:0004467 ! long-chain fatty acid-CoA ligase activity
 created_by: tb
 creation_date: 2012-07-25T03:31:31Z
@@ -432941,10 +433810,11 @@
 
 [Term]
 id: GO:0090486
-name: small RNA 2'-O-methyltransferase activity
+name: small RNA 2'-O-ribose methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a nucleoside residue in a small RNA molecule. Reaction: S-adenosyl-L-methionine + small RNA = S-adenosyl-L-homocysteine + small RNA containing a 3'-terminal 2'-O-methylnucleotide." [GOC:tb, GOC:vw, RHEA:37887]
 synonym: "small RNA 2'-O-methyltransferase" EXACT []
+synonym: "small RNA 2'-O-methyltransferase activity" EXACT []
 xref: EC:2.1.1.386
 xref: RHEA:37887
 is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
@@ -433644,7 +434514,7 @@
 synonym: "phosphatidylcholine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "phosphatidylcholine-translocating ATPase activity" BROAD []
 xref: RHEA:38583
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+is_a: GO:0008525 ! phosphatidylcholine intramembrane carrier activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch RHEA:38583
@@ -433678,7 +434548,6 @@
 synonym: "phosphatidylserine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
 synonym: "phosphatidylserine-translocating ATPase activity" BROAD []
 xref: RHEA:38567
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch RHEA:38567
@@ -433744,7 +434613,8 @@
 name: protein-N(PI)-phosphohistidine-N,N'-diacetylchitobiose phosphotransferase system transporter activity
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N,N'-diacetylchitobiose(out) = protein histidine + N,N'-diacetylchitobiose phosphate(in)." [GOC:am, PMID:10913119]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 created_by: tb
 creation_date: 2014-04-08T14:55:02Z
 
@@ -433995,10 +434865,13 @@
 
 [Term]
 id: GO:0090590
-name: protein-N(PI)-phosphohistidine-D-glucosamine phosphotransferase system transporter activity
+name: obsolete protein-N(PI)-phosphohistidine-D-glucosamine phosphotransferase system transporter activity
 namespace: molecular_function
-def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + D-glucosamine(out) = protein histidine + glucosamine-6-phosphate(in)." [PMID:8246840]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+def: "OBSOLETE. Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + D-glucosamine(out) = protein histidine + glucosamine-6-phosphate(in)." [PMID:8246840]
+comment: This term was obsoleted because it represents a specific substrate of protein-N(PI)-phosphohistidine-mannose phosphotransferase system transporter activity ; GO:0022870 (see PMID:8246840).
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0022870
 created_by: tb
 creation_date: 2014-06-04T16:33:01Z
 
@@ -434007,7 +434880,8 @@
 name: protein-N(PI)-phosphohistidine-N-acetyl-mannosamine phosphotransferase system transporter activity
 namespace: molecular_function
 def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetyl-mannosamine(out) = protein histidine +N-acetyl- mannosamine-6-phosphate(in)." [PMID:9864311]
-is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
+is_a: GO:0008982 ! protein-N(PI)-phosphohistidine-carbohydrate phosphotransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
 created_by: tb
 creation_date: 2014-06-04T16:33:01Z
 
@@ -434048,7 +434922,7 @@
 id: GO:0090595
 name: L-lysine N6-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acetyl-CoA + L-lysine = CoA + H+ + N(6)-acetyl-L-lysine." [RHEA:69476]
+def: "Catalysis of the reaction: acetyl-CoA + L-lysine = CoA + H+ + N6-acetyl-L-lysine." [RHEA:69476]
 synonym: "acetyl-CoA:L-lysine N6-acetyltransferase" EXACT []
 xref: EC:2.3.1.306
 xref: MetaCyc:LYSACET-RXN
@@ -434208,7 +435082,7 @@
 id: GO:0090612
 name: cAMP deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cyclic adenosine monophosphate + H2O = cyclic inosine monophosphate + NH3." [PMID:24074367, RHEA:22908]
+def: "Catalysis of the reaction: 3',5'-cyclic AMP + H2O + H+ = 3',5'-cyclic IMP + NH4+." [PMID:24074367, RHEA:22908]
 synonym: "cyclic adenosine monophosphate deaminase activity" EXACT []
 xref: EC:3.5.4.46
 xref: RHEA:22908
@@ -434225,7 +435099,7 @@
 id: GO:0090613
 name: 5'-deoxyadenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5'deoxyadenosine + H2O = 5'deoxyinosine + NH3." [PMID:23968233, RHEA:42892]
+def: "Catalysis of the reaction: 5'-deoxyadenosine + H2O + H+ = 5'-deoxyinosine + NH4+." [PMID:23968233, RHEA:42892]
 xref: EC:3.5.4.41
 xref: RHEA:42892
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
@@ -434240,7 +435114,7 @@
 id: GO:0090614
 name: 5'-methylthioadenosine deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5'methyl thioadenosine + H2O = 5'methyl thioinosine + NH3." [PMID:23968233, RHEA:25025]
+def: "Catalysis of the reaction: S-methyl-5'-thioadenosine + H2O + H+ = S-methyl-5'-thioinosine + NH4+." [PMID:23968233, RHEA:25025]
 xref: EC:3.5.4.31
 xref: RHEA:25025
 is_a: GO:0016814 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
@@ -434680,7 +435554,7 @@
 namespace: molecular_function
 def: "Binding to a junction formed at the point where double-stranded telomeric DNA becomes a single-stranded G-rich telomeric DNA 3' overhang." [GOC:BHF, GOC:BHF_telomere, GOC:bhm, GOC:nc, PMID:21852327]
 is_a: GO:0000406 ! double-strand/single-strand DNA junction binding
-is_a: GO:0042162 ! telomeric DNA binding
+is_a: GO:0042162 ! telomeric repeat DNA binding
 created_by: tb
 creation_date: 2015-07-02T11:33:39Z
 
@@ -436163,10 +437037,21 @@
 synonym: "L-glutamate biosynthesis" EXACT []
 synonym: "L-glutamate formation" EXACT []
 synonym: "L-glutamate synthesis" EXACT []
+xref: MetaCyc:GLUGLNSYN-PWY
+xref: MetaCyc:GLUTAMATE-SYN2-PWY
+xref: MetaCyc:GLUTSYN-PWY
+xref: MetaCyc:GLUTSYNIII-PWY
+xref: MetaCyc:PWY-4341
 is_a: GO:0006536 ! glutamate metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
 is_a: GO:0043650 ! dicarboxylic acid biosynthetic process
 property_value: skos:broadMatch MetaCyc:PWY-5505
+property_value: skos:narrowMatch MetaCyc:GLUGLNSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTAMATE-SYN2-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTSYN-PWY
+property_value: skos:narrowMatch MetaCyc:GLUTSYNIII-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-4341
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pr
 creation_date: 2011-05-24T04:30:38Z
 
@@ -437217,7 +438102,6 @@
 comment: The reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "caspase activity" BROAD []
 synonym: "metacaspase activity" NARROW [GOC:mtg_apoptosis]
-xref: Reactome:R-HSA-6800797 "The PIDDosome activates CASP2"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004197
@@ -437279,7 +438163,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Binding to an organic cyclic compound, any molecular entity that contains carbon arranged in a cyclic molecular structure." [GOC:sjw, PMID:7583672]
 comment: The reason for obsoletion is that this term was an unnecessary grouping term.
-xref: Reactome:R-HSA-1497796 "BH2 binding can lead to eNOS uncoupling"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29143" xsd:anyURI
 is_obsolete: true
 created_by: pr
@@ -437647,6 +438530,7 @@
 comment: This term can be used to annotate gene products involved in apoptotic events happening downstream of the cross-talk point between the extrinsic and intrinsic apoptotic pathways. The cross-talk starts when caspase-8 cleaves Bid and truncated Bid interacts with mitochondria. From this point on it is not possible to distinguish between extrinsic and intrinsic pathways.
 synonym: "apoptotic signalling pathway" EXACT [GOC:mah]
 synonym: "induction of apoptosis by extracellular signals" EXACT []
+xref: Reactome:R-HSA-111459 "Activation of caspases through apoptosome-mediated cleavage"
 xref: Reactome:R-HSA-193681 "Ceramide signalling"
 xref: Reactome:R-HSA-204998 "Cell death signalling via NRAGE, NRIF and NADE"
 xref: Reactome:R-HSA-75157 "FasL/ CD95L signaling"
@@ -438450,7 +439334,6 @@
 namespace: molecular_function
 def: "OBSOLETE. Catalysis of the reaction: leukotriene A4 = eoxin A4." [GOC:mw, PMID:18184802, PMID:18647347]
 comment: The reason for obsoletion is that this term is equivalent to leukotriene-C4 synthase activity.
-xref: Reactome:R-HSA-2162019 "LTA4 is converted to EXA4 by ALOX15"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29695" xsd:anyURI
 is_obsolete: true
 replaced_by: GO:0004464
@@ -438877,7 +439760,6 @@
 comment: Components of the DISC (death-inducing signaling complex) may be annotated to this term. These include ligand-bound receptors such as FAS/CD95 (though care should be taken because FAS can also act as a non-apoptotic signal transducer), and signaling molecules such as FADD (FAS-associated protein with a death domain), cIAPs (cellular inhibitor of apoptosis proteins), c-FLIPs and caspases 8 and 10. Note that GO:0097296 should only be used to annotate gene products directly involved in the initiation or start up of an inactive cysteine-type endopeptidase activity involved in the apoptotic signaling pathway (also known as an initiator caspase). If the evidence provided is not sufficient to account for direct activation of this enzymatic activity, consider using the more generic terms GO:2001269 positive regulation of cysteine-type endopeptidase activity involved in apoptotic signaling pathway or GO:0043280 positive regulation of cysteine-type endopeptidase activity involved in apoptotic process.\nThe reason for obsoletion is that this term represents a GO-CAM model.
 synonym: "activation of cysteine-type endopeptidase activity involved in apoptotic signalling pathway" EXACT []
 synonym: "activation of initiator caspase activity" NARROW []
-xref: Reactome:R-HSA-140534 "Caspase activation via Death Receptors in the presence of ligand"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28468" xsd:anyURI
 is_obsolete: true
 created_by: pr
@@ -439561,6 +440443,7 @@
 synonym: "protein O-GlcNAc transferase activity" EXACT []
 synonym: "UDP-N-acetyl-D-glucosamine:protein-O-beta-N-acetyl-D-glucosaminyl transferase" EXACT []
 xref: EC:2.4.1.255
+xref: Reactome:R-HSA-8879117 "POMGNT2 transfers GlcNAc to Man-DAG1"
 xref: Reactome:R-HSA-9687828 "O-GlcNAcylation of RIPK3 (TLR4 signaling)"
 xref: RHEA:48904
 xref: RHEA:48908
@@ -441321,7 +442204,7 @@
 id: GO:0097532
 name: stress response to acid chemical
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in organismal or cellular homeostasis caused by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:aa, GOC:BHF, GOC:go_curators, GOC:rl, PMID:10615049, PMID:19170886, Wikipedia:Acid]
+def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in organismal or cellular homeostasis caused by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:aa, GOC:BHF, GOC:curators, GOC:rl, PMID:10615049, PMID:19170886, Wikipedia:Acid]
 synonym: "response to acid stress" BROAD []
 synonym: "stress response to acid" BROAD []
 is_a: GO:0001101 ! response to acid chemical
@@ -441333,7 +442216,7 @@
 id: GO:0097533
 name: cellular stress response to acid chemical
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in cellular homeostasis caused by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:aa, GOC:BHF, GOC:go_curators, GOC:rl, PMID:10615049, PMID:19170886, Wikipedia:Acid]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in cellular homeostasis caused by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form." [GOC:aa, GOC:BHF, GOC:curators, GOC:rl, PMID:10615049, PMID:19170886, Wikipedia:Acid]
 synonym: "cellular response to acid stress" BROAD []
 synonym: "cellular stress response to acid" BROAD []
 is_a: GO:0097532 ! stress response to acid chemical
@@ -443245,7 +444128,7 @@
 id: GO:0097697
 name: tRNA (5-carboxymethoxyuridine(34)-5-O)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-carboxymethoxyuridine34 in tRNA + S-adenosyl-L-methionine = 5-methoxycarbonylmethoxyuridine34 in tRNA + S-adenosyl-L-homocysteine." [PMID:26681692, RHEA:54080]
+def: "Catalysis of the reaction: 5-carboxymethoxyuridine34 in tRNA + S-adenosyl-L-methionine = 5-methoxycarbonylmethoxyuridine34 in tRNA + S-adenosyl-L-homocysteine. The methylation occurs on the modified base." [PMID:26681692, RHEA:54080]
 synonym: "tRNA 5-carboxymethoxyuridine methyltransferase activity" RELATED []
 xref: RHEA:54080
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -444697,10 +445580,11 @@
 id: GO:0098538
 name: lumenal side of transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the transport vesicle membrane that faces the lumen." [GOC:ab]
+def: "The leaflet of a transport vesicle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "internal side of transport vesicle membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0030658 ! transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:43:55Z
 
@@ -444708,10 +445592,11 @@
 id: GO:0098539
 name: cytoplasmic side of transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the transport vesicle membrane that faces the cytoplasm." [GOC:ab]
+def: "The leaflet of the transport vesicle membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "external side of transport vesicle membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0030658 ! transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:48:07Z
 
@@ -444719,10 +445604,11 @@
 id: GO:0098540
 name: lumenal side of trans-Golgi network transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the lumen." [GOC:ab]
+def: "The leaflet of a trans-Golgi network transport vesicle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "internal side of trans-Golgi network transport vesicle membrane" RELATED []
 is_a: GO:0098538 ! lumenal side of transport vesicle membrane
 relationship: part_of GO:0012510 ! trans-Golgi network transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:55:02Z
 
@@ -444730,10 +445616,11 @@
 id: GO:0098541
 name: cytoplasmic side of trans-Golgi network transport vesicle membrane
 namespace: cellular_component
-def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the cytoplasm." [GOC:ab]
+def: "The leaflet of the trans-Golgi network transport vesicle membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab]
 synonym: "external side of trans-Golgi network transport vesicle membrane" EXACT []
 is_a: GO:0098539 ! cytoplasmic side of transport vesicle membrane
 relationship: part_of GO:0012510 ! trans-Golgi network transport vesicle membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-10-24T11:56:30Z
 
@@ -444793,9 +445680,10 @@
 id: GO:0098547
 name: lumenal side of Golgi membrane
 namespace: cellular_component
-def: "The side of the Golgi membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the Golgi membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0000139 ! Golgi membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T10:44:23Z
 
@@ -444803,9 +445691,10 @@
 id: GO:0098548
 name: cytoplasmic side of Golgi membrane
 namespace: cellular_component
-def: "The side (leaflet) of the Golgi membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the Golgi membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0000139 ! Golgi membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T10:49:33Z
 
@@ -444820,27 +445709,29 @@
 id: GO:0098550
 name: lumenal side of early endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the early endosome membrane that faces the lumen." [GOC:lr]
+def: "The leaflet of the early endosome membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "internal leaflet of early endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of early endosome membrane" EXACT []
 is_a: GO:0098565 ! lumenal side of endosome membrane
 relationship: part_of GO:0031901 ! early endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098551
 name: lumenal side of late endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the late endosome membrane that faces the lumen." [GOC:lr]
+def: "The leaflet of the late endosomal membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 synonym: "internal leaflet of late endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of late endosome membrane" EXACT []
 is_a: GO:0098565 ! lumenal side of endosome membrane
 relationship: part_of GO:0031902 ! late endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098552
 name: side of membrane
 namespace: cellular_component
-def: "A cellular component consisting of one leaflet of a membrane bilayer and any proteins embedded or anchored in it or attached to its surface." [GOC:dos]
+def: "A cellular component consisting of one leaflet of a membrane bilayer and any protein embedded or anchored in it or attached to its surface." [GOC:dos]
 is_a: GO:0110165 ! cellular anatomical structure
 relationship: has_part GO:0097478 ! leaflet of membrane bilayer
 relationship: part_of GO:0016020 ! membrane
@@ -444851,9 +445742,10 @@
 id: GO:0098553
 name: lumenal side of endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plasma membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005789 ! endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:08:37Z
 
@@ -444861,9 +445753,10 @@
 id: GO:0098554
 name: cytoplasmic side of endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005789 ! endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:09:47Z
 
@@ -444871,9 +445764,10 @@
 id: GO:0098555
 name: lumenal side of rough endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the rough endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098553 ! lumenal side of endoplasmic reticulum membrane
 relationship: part_of GO:0030867 ! rough endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:13:24Z
 
@@ -444881,9 +445775,10 @@
 id: GO:0098556
 name: cytoplasmic side of rough endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the rough endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098554 ! cytoplasmic side of endoplasmic reticulum membrane
 relationship: part_of GO:0030867 ! rough endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:15:44Z
 
@@ -444891,9 +445786,10 @@
 id: GO:0098557
 name: cytoplasmic side of smooth endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the smooth endoplasmic reticulum membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098554 ! cytoplasmic side of endoplasmic reticulum membrane
 relationship: part_of GO:0030868 ! smooth endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:17:58Z
 
@@ -444901,9 +445797,10 @@
 id: GO:0098558
 name: lumenal side of smooth endoplasmic reticulum membrane
 namespace: cellular_component
-def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
+def: "The leaflet of the smooth endoplasmic reticulum membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0098553 ! lumenal side of endoplasmic reticulum membrane
 relationship: part_of GO:0030868 ! smooth endoplasmic reticulum membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-03T12:19:28Z
 
@@ -444911,23 +445808,25 @@
 id: GO:0098559
 name: cytoplasmic side of early endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the early endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the early endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of early endosome membrane" EXACT [GOC:ab]
 synonym: "external side of early endosome membrane" EXACT []
 is_a: GO:0010009 ! cytoplasmic side of endosome membrane
 relationship: part_of GO:0031901 ! early endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098560
 name: cytoplasmic side of late endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the late endosome membrane that faces the cytoplasm." [GOC:lr]
+def: "The leaflet of the late endosome membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:lr]
 comment: In GO, 'external side' still refers to part of the membrane and does not refer to components beyond (outside of) the membrane.
 synonym: "external leaflet of late endosome membrane" EXACT [GOC:ab]
 synonym: "external side of late endosome membrane" EXACT []
 is_a: GO:0010009 ! cytoplasmic side of endosome membrane
 relationship: part_of GO:0031902 ! late endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098561
@@ -444941,8 +445840,9 @@
 id: GO:0098562
 name: cytoplasmic side of membrane
 namespace: cellular_component
-def: "The side of a membrane that faces the cytoplasm." [GOC:dos]
+def: "The leaflet of a membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-12T11:21:30Z
 
@@ -444969,11 +445869,12 @@
 id: GO:0098565
 name: lumenal side of endosome membrane
 namespace: cellular_component
-def: "The side (leaflet) of the endosome membrane that faces the lumen." [GOC:dos]
+def: "The leaflet of the endosome membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 synonym: "internal leaflet of endosome membrane" EXACT [GOC:ab]
 synonym: "internal side of endosome membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0010008 ! endosome membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098566
@@ -444987,8 +445888,9 @@
 id: GO:0098567
 name: periplasmic side of plasma membrane
 namespace: cellular_component
-def: "The side (leaflet) of a plasma membrane that faces the periplasm, and all proteins embedded in it or attached to its surface." [GOC:dos]
+def: "The leaflet of the plasma membrane that faces the periplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0009897 ! external side of plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2013-12-19T09:31:30Z
 
@@ -444996,39 +445898,44 @@
 id: GO:0098568
 name: external side of mycolate outer membrane
 namespace: cellular_component
-def: "The side (leaflet) of the mycolate outer membrane that faces the environment and any proteins embedded in it or loosely bound to its surface." [GOC:dos, PMID:18316738, PMID:18567661]
+def: "The leaflet of the mycolate outer membrane that faces the periplasm of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos, PMID:18316738, PMID:18567661]
 is_a: GO:0031240 ! external side of cell outer membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098569
 name: internal side of mycolate outer membrane
 namespace: cellular_component
-def: "The side of the mycolate outer membrane that faces the cell wall peptidoglycan. It is rich in long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network." [GOC:dos, PMID:18316738, PMID:18567661]
+def: "The leaflet of the mycolate outer membrane that faces the cell wall peptidoglycan, including any protein embedded in, attached to, or peripherally associated with it. It is rich in long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network." [GOC:dos, PMID:18316738, PMID:18567661]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098570
 name: stromal side of plastid inner membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid inner membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of the plastid inner membrane that faces the stroma, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0009528 ! plastid inner membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098571
 name: lumenal side of plastid thylakoid membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid thylakoid membrane that faces the lumen, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of a plastid thylakoid membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0055035 ! plastid thylakoid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098572
 name: stromal side of plastid thylakoid membrane
 namespace: cellular_component
-def: "The side (leaflet) of the plastid thylakoid membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
+def: "The leaflet of the plastid thylakoid membrane that faces the stroma, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
 relationship: part_of GO:0055035 ! plastid thylakoid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098573
@@ -445045,28 +445952,31 @@
 id: GO:0098574
 name: cytoplasmic side of lysosomal membrane
 namespace: cellular_component
-def: "The side (leaflet) of the lysosomal membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the lysosomal membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 synonym: "external leaflet of lysosomal membrane" EXACT [GOC:dos]
 synonym: "external side of lysosomal membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0005765 ! lysosomal membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098575
 name: lumenal side of lysosomal membrane
 namespace: cellular_component
-def: "The side (leaflet) of the lysosomal membrane that faces the lumen." [GOC:dos]
+def: "The leaflet of the lysosomal membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 synonym: "internal leaflet of lysosomal membrane" EXACT [GOC:ab]
 synonym: "internal side of lysosomal membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005765 ! lysosomal membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098576
 name: lumenal side of membrane
 namespace: cellular_component
-def: "Any side (leaflet) of a membrane that faces the lumen of an organelle." [GOC:dos]
+def: "The leaflet of an organelle membrane that faces the lumen, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098552 ! side of membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0098577
@@ -445191,9 +446101,10 @@
 id: GO:0098591
 name: external side of apical plasma membrane
 namespace: cellular_component
-def: "The leaflet the apical region of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:ab, GOC:dos]
+def: "The leaflet the apical region of the plasma membrane that faces the extracellular side of the cell, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0009897 ! external side of plasma membrane
 relationship: part_of GO:0016324 ! apical plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2014-03-06T12:04:33Z
 
@@ -445201,9 +446112,10 @@
 id: GO:0098592
 name: cytoplasmic side of apical plasma membrane
 namespace: cellular_component
-def: "The side (leaflet) of the apical region of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
+def: "The leaflet of the apical region of the plasma membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:ab, GOC:dos]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0016324 ! apical plasma membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: dos
 creation_date: 2014-03-06T12:07:32Z
 
@@ -445352,12 +446264,13 @@
 
 [Term]
 id: GO:0098607
-name: methylselenocysteine deselenhydrase activity
+name: obsolete methylselenocysteine deselenhydrase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: Se-Methyl-L-selenocysteine + H2O = pyruvic acid + NH3 + Methaneselenol." [PMID:17451884, PMID:20383543]
+def: "OBSOLETE. Catalysis of the reaction: Se-Methyl-L-selenocysteine + H2O = pyruvic acid + NH3 + Methaneselenol." [PMID:17451884, PMID:20383543]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: Reactome:R-HSA-2408539 "MeSec is hydrolysed to MeSeH by PXLP-K212-CTH"
-is_a: GO:0016846 ! carbon-sulfur lyase activity
-property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30815" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 created_by: dos
 creation_date: 2014-04-16T11:24:12Z
 
@@ -445694,6 +446607,7 @@
 is_a: GO:0005581 ! collagen trimer
 intersection_of: GO:0005581 ! collagen trimer
 intersection_of: part_of GO:0098645 ! complex of network-forming collagens
+relationship: part_of GO:0031012 ! extracellular matrix
 relationship: part_of GO:0098645 ! complex of network-forming collagens
 
 [Term]
@@ -445717,7 +446631,7 @@
 synonym: "Supramolecular collagen assembly" EXACT [PMID:21421911]
 is_a: GO:0099080 ! supramolecular complex
 relationship: has_part GO:0005581 ! collagen trimer
-relationship: part_of GO:0031012 ! extracellular matrix
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31491" xsd:anyURI
 
 [Term]
 id: GO:0098645
@@ -445730,14 +446644,18 @@
 
 [Term]
 id: GO:0098646
-name: interstitial hexagonal collagen network
+name: hexagonal collagen network
 namespace: cellular_component
-def: "A hexagonal shaped collagen network formed by collagen type X trimer, mostly found in the cartilage hyaline matrix, which located in the interstitial area." [PMID:21421911, PMID:34948124]
+def: "A hexagonal-shaped collagen network formed by collagen type X trimer or collagen type VIII trimers. Collagen type X trimers are mostly found in the cartilage hyaline matrix, which is located in the interstitial area. Note that collagen type VIII has been found in the basement membrane, but the localization is not definitive." [PMID:21421911, PMID:3047147, PMID:31387942, PMID:34948124]
+synonym: "basement membrane hexagonal collagen network" NARROW []
 synonym: "collagen hexagonal network" EXACT [PMID:21421911]
+synonym: "hexagonal collagen network of basement membrane" NARROW []
+synonym: "interstitial hexagonal collagen network" NARROW []
 is_a: GO:0098645 ! complex of network-forming collagens
 relationship: part_of GO:0140152 ! collagenous component of interstitial matrix
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29415" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30146" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31494" xsd:anyURI
 
 [Term]
 id: GO:0098647
@@ -446759,6 +447677,7 @@
 name: 1-phosphatidylinositol 4-kinase activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of 1-phosphatidylinositol 4-kinase." [PMID:21288895]
+xref: Reactome:R-HSA-1676152 "PI4KB binds to ARF1/3:GTP at the Golgi membrane"
 is_a: GO:0019209 ! kinase activator activity
 relationship: positively_regulates GO:0004430 ! 1-phosphatidylinositol 4-kinase activity
 
@@ -448266,7 +449185,7 @@
 id: GO:0098900
 name: regulation of action potential
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:dos, GOC:dph, GOC:go_curators, GOC:tb, ISBN:978-0-07-139011-8]
+def: "Any process that modulates the frequency, rate or extent of action potential creation, propagation or termination. This typically occurs via modulation of the activity or expression of voltage-gated ion channels." [GOC:curators, GOC:dos, GOC:dph, GOC:tb, ISBN:978-0-07-139011-8]
 comment: The ion channels through which current flows during an action potential should be annotated to the process 'action potential'. Gene products involved in modulating the characteristics of an action potential via changing the expression levels or the activity of these channels (e.g. modulating their kinetics or voltage sensitivity) should be annotated to this regulation term.
 is_a: GO:0042391 ! regulation of membrane potential
 is_a: GO:0050789 ! regulation of biological process
@@ -450289,6 +451208,7 @@
 name: potassium channel activator activity
 namespace: molecular_function
 def: "Binds to and increases the activity of a potassium channel, resulting in its opening." [GOC:dos]
+xref: Reactome:R-HSA-400063 "Low conductance potassium channels in pancreatic beta cells open in response to epinephrine"
 is_a: GO:0015459 ! potassium channel regulator activity
 is_a: GO:0099103 ! channel activator activity
 relationship: positively_regulates GO:0005267 ! potassium channel activity
@@ -452296,6 +453216,10 @@
 name: ligand-gated calcium channel activity
 namespace: molecular_function
 def: "Enables the transmembrane transfer of a calcium ions by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:dos]
+xref: Reactome:R-HSA-1168376 "STIM1 oligomerizes"
+xref: Reactome:R-HSA-169680 "IP3 binds to the IP3 receptor, opening the endoplasmic reticulum Ca2+ channel"
+xref: Reactome:R-HSA-169683 "IP3R:I(1,4,5)P3 tetramer transports Ca2+ from ER lumen to cytosol"
+xref: Reactome:R-HSA-4420052 "Decreasing cGMP concentration promotes intracellular Ca2+ release in response to WNT"
 is_a: GO:0005262 ! calcium channel activity
 is_a: GO:0099094 ! ligand-gated monoatomic cation channel activity
 relationship: part_of GO:0019722 ! calcium-mediated signaling
@@ -452412,9 +453336,10 @@
 id: GO:0099617
 name: matrix side of mitochondrial inner membrane
 namespace: cellular_component
-def: "The side (leaflet) of the mitochondrial inner membrane that faces the matrix." [GOC:dos]
+def: "The leaflet of a mitochondrial inner membrane that faces the matrix, including any protein embedded in, attached to, or peripherally associated with it." [GOC:dos]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0005743 ! mitochondrial inner membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 
 [Term]
 id: GO:0099618
@@ -452449,11 +453374,12 @@
 
 [Term]
 id: GO:0099620
-name: UDP-4-amino-4-deoxy-L-arabinose aminotransferase activity
+name: UDP-4-amino-4-deoxy-L-arabinose:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-4-amino-4-deoxy-beta-L-arabinose + 2-oxoglutarate = UDP-beta-L-threo-pentopyranos-4-ulose + L-glutamate." [PMID:12429098, PMID:12704196, RHEA:24710]
 synonym: "UDP-(beta-L-threo-pentapyranosyl-4''-ulose diphosphate) aminotransferase" EXACT []
 synonym: "UDP-4-amino-4-deoxy-beta-L-arabinose:2-oxoglutarate aminotransferase" EXACT []
+synonym: "UDP-4-amino-4-deoxy-L-arabinose aminotransferase" BROAD []
 synonym: "UDP-4-amino-4-deoxy-L-arabinose---oxoglutarate aminotransferase" EXACT []
 synonym: "UDP-Ara4O aminotransferase" EXACT []
 synonym: "UDP-L-Ara4N transaminase" EXACT []
@@ -452463,6 +453389,7 @@
 property_value: skos:exactMatch EC:2.6.1.87
 property_value: skos:exactMatch RHEA:24710
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0099621
@@ -453523,6 +454450,7 @@
 comment: There are two main classes of deubiquitinating enzymes: cysteine proteases (i.e., thiol dependent) and metalloproteases.
 synonym: "ubiquitinyl hydrolase activity" EXACT []
 xref: Reactome:R-HSA-9674127 "USP30 deubiquitinates ATM dimer:Ub-p-PEX5"
+xref: Reactome:R-HSA-9929353 "Deubiquitination of CD274"
 is_a: GO:0019783 ! ubiquitin-like protein peptidase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17398" xsd:anyURI
 created_by: mec
@@ -453989,19 +454917,22 @@
 
 [Term]
 id: GO:0102037
-name: 4-nitrotoluene monooxygenase activity
+name: obsolete 4-nitrotoluene monooxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + 4-nitrotoluene + NADH + O2 = 4-nitrobenzyl alcohol + NAD+ + H2O." [GOC:pz]
-xref: MetaCyc:R361-RXN
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + 4-nitrotoluene + NADH + O2 = 4-nitrobenzyl alcohol + NAD+ + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102038
-name: 4-nitrobenzyl alcohol oxidase activity
+name: obsolete 4-nitrobenzyl alcohol oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-nitrobenzyl alcohol + O2 = 4-nitrobenzaldehyde + hydrogen peroxide." [GOC:pz]
+def: "OBSOLETE. Catalysis of the reaction: 4-nitrobenzyl alcohol + O2 = 4-nitrobenzaldehyde + hydrogen peroxide." [GOC:pz]
+comment: This term was obsoleted because there is no evidence that this reaction exists.
 xref: MetaCyc:R362-RXN
-is_a: GO:0016899 ! oxidoreductase activity, acting on the CH-OH group of donors, oxygen as acceptor
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31530" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102039
@@ -454070,11 +455001,12 @@
 
 [Term]
 id: GO:0102044
-name: 3-chlorobenzoate-4,5-oxygenase activity
+name: obsolete 3-chlorobenzoate-4,5-oxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 3-chlorobenzoate + O2 + a reduced electron acceptor = 3-chlorobenzoate-cis-4,5-diol + an oxidized electron acceptor." [GOC:pz]
-xref: MetaCyc:RXN-10421
-is_a: GO:0016708 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of two atoms of oxygen into one donor
+def: "OBSOLETE. Catalysis of the reaction: 3-chlorobenzoate + O2 + a reduced electron acceptor = 3-chlorobenzoate-cis-4,5-diol + an oxidized electron acceptor." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102045
@@ -454725,11 +455657,12 @@
 
 [Term]
 id: GO:0102115
-name: peptidoglycan asparagine synthase activity
+name: obsolete peptidoglycan asparagine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparatyl)-L-lysyl-D-alanyl-D-alanine + ammonium + ATP = H+ + ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparaginyl)-L-lysyl-D-alanyl-D-alanine + AMP + diphosphoric acid." [EC:6.3.1.-, GOC:pz]
-xref: MetaCyc:RXN-11338
-is_a: GO:0016880 ! acid-ammonia (or amide) ligase activity
+def: "OBSOLETE. Catalysis of the reaction: ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparatyl)-L-lysyl-D-alanyl-D-alanine + ammonium + ATP = H+ + ditrans,octacis-undecaprenyldiphospho-N-acetyl-(N-acetylglucosaminyl)muramoyl-L-alanyl-gamma-D-isoglutaminyl-N-(beta-D-asparaginyl)-L-lysyl-D-alanyl-D-alanine + AMP + diphosphoric acid." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102116
@@ -454827,11 +455760,12 @@
 
 [Term]
 id: GO:0102126
-name: coniferyl aldehyde 5-hydroxylase activity
+name: obsolete coniferyl aldehyde 5-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + coniferyl aldehyde + NADPH + O2 = 5-hydroxy-coniferaldehyde + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-1142
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + coniferyl aldehyde + NADPH + O2 = 5-hydroxy-coniferaldehyde + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102127
@@ -454900,19 +455834,21 @@
 
 [Term]
 id: GO:0102134
-name: (22S)-22-hydroxy-campesterol C-23 hydroxylase activity
+name: obsolete (22S)-22-hydroxy-campesterol C-23 hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (22S)-22-hydroxycampesterol + NADPH + O2 = (22R,23R)-22,23-dihydroxycampesterol + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-11529
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (22S)-22-hydroxycampesterol + NADPH + O2 = (22R,23R)-22,23-dihydroxycampesterol + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102135
-name: (22S)-22-hydroxy-campest-4-en-3-one C-23 hydroxylase activity
+name: obsolete (22S)-22-hydroxy-campest-4-en-3-one C-23 hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (22S)-22-hydroxycampest-4-en-3-one + NADPH + O2 = (22R,23R)-22,23-dihydroxy-campest-4-en-3-one + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-11530
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (22S)-22-hydroxycampest-4-en-3-one + NADPH + O2 = (22R,23R)-22,23-dihydroxy-campest-4-en-3-one + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102136
@@ -455065,7 +456001,7 @@
 id: GO:0102149
 name: farnesylcysteine lyase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-[(2E,6E)-farnesyl]-L-cysteine + O2 + H2O = (2-trans,6-trans)-farnesal + L-cysteine + hydrogen peroxide." [EC:1.8.3.6, GOC:pz]
+def: "Catalysis of the reaction: S-[(2E,6E)-farnesyl]-L-cysteine + O2 + H2O = (2-trans,6-trans)-farnesal + L-cysteine + H2O2." [EC:1.8.3.6, GOC:pz]
 xref: EC:1.8.3.6
 xref: MetaCyc:RXN-11623
 xref: RHEA:30231
@@ -455923,11 +456859,12 @@
 
 [Term]
 id: GO:0102235
-name: 1-penten-3-one reductase activity
+name: obsolete 1-penten-3-one reductase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 1-penten-3-one + NADPH + H+ = 1-pentan-3-one + NADP." [GOC:pz]
-xref: MetaCyc:RXN-12295
-is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: 1-penten-3-one + NADPH + H+ = 1-pentan-3-one + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102236
@@ -456060,11 +456997,12 @@
 
 [Term]
 id: GO:0102249
-name: phosphatidylcholine:diacylglycerol cholinephosphotransferase activity
+name: obsolete phosphatidylcholine:diacylglycerol cholinephosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a phosphatidylcholine + a 1,2-diacyl-sn-glycerol = a 1,2-diacyl-sn-glycerol + a phosphatidylcholine." [GOC:pz, MetaCyc:RXN-12386]
-xref: MetaCyc:RXN-12386
-is_a: GO:0016780 ! phosphotransferase activity, for other substituted phosphate groups
+def: "OBSOLETE. Catalysis of the reaction: a phosphatidylcholine + a 1,2-diacyl-sn-glycerol = a 1,2-diacyl-sn-glycerol + a phosphatidylcholine." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102250
@@ -456723,7 +457661,7 @@
 id: GO:0102317
 name: 4-methylaminobutyrate oxidase (demethylating) activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-(methylamino)butyric acid + O2 + H2O = gamma-aminobutyric acid + formaldehyde + hydrogen peroxide." [EC:1.5.3.19, GOC:pz]
+def: "Catalysis of the reaction: 4-(methylamino)butanoate + O2 + H2O = 4-aminobutanoate + formaldehyde + H2O2." [RHEA:33907]
 xref: EC:1.5.3.19
 xref: MetaCyc:RXN-13067
 xref: RHEA:33907
@@ -457134,7 +458072,9 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: mithramycin DK + NADPH + H+ = iso-mithramycin + NADP+. Note that  iso-mithramycin (iso-MTM) is a C2-epimer of mithramycin.  This intermediate appears to be biologically important because it is non-toxic, then exported out of the cell and finally spontaneously epimerized into the toxic mithramycin. This could represent a self-resistance mechanism against MTM toxicity." [PMID:31702856]
 xref: MetaCyc:RXN-13385
+xref: RHEA:85451
 is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+property_value: skos:exactMatch RHEA:85451
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30329" xsd:anyURI
 
@@ -457249,11 +458189,12 @@
 
 [Term]
 id: GO:0102369
-name: 11alpha-30-dihydroxy beta-amyrin dehydrogenase activity
+name: obsolete 11alpha-30-dihydroxy beta-amyrin dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 11alpha,30-dihydroxy-beta-amyrin + NADPH + O2 + H+ = 30-hydroxy-11-oxo-beta-amyrin + NADP + 2 H2O." [GOC:pz]
-xref: MetaCyc:RXN-13491
-is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
+def: "OBSOLETE. Catalysis of the reaction: 11alpha,30-dihydroxy-beta-amyrin + NADPH + O2 + H+ = 30-hydroxy-11-oxo-beta-amyrin + NADP + 2 H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102370
@@ -458174,13 +459115,14 @@
 
 [Term]
 id: GO:0102459
-name: 8-oxo-dADP diphosphate phosphatase activity
+name: obsolete 8-oxo-dADP diphosphate phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 8-oxo-dADP + H2O = 8-oxo-dAMP + hydrogenphosphate + H+." [GOC:pz]
+def: "OBSOLETE. Catalysis of the reaction: 8-oxo-dADP + H2O = 8-oxo-dAMP + hydrogenphosphate + H+." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
 synonym: "8-oxo-deoxyadenine diphosphate phosphatase activity" EXACT []
-xref: MetaCyc:RXN-14005
-is_a: GO:0017110 ! nucleoside diphosphate phosphatase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23401" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102460
@@ -458228,11 +459170,12 @@
 
 [Term]
 id: GO:0102465
-name: zeaxanthin 2,2'-beta-hydroxylase activity
+name: obsolete zeaxanthin 2,2'-beta-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: zeaxanthin + 2 NADH + 2 H+ + 2 O2 = nostoxanthin + 2 NAD + 2 H2O." [GOC:pz]
-xref: MetaCyc:RXN-14018
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: zeaxanthin + 2 NADH + 2 H+ + 2 O2 = nostoxanthin + 2 NAD + 2 H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102466
@@ -459982,11 +460925,12 @@
 
 [Term]
 id: GO:0102641
-name: (R)-lactaldehyde dehydrogenase activity
+name: obsolete (R)-lactaldehyde dehydrogenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (R)-propane-1,2-diol + NADP = (R)-lactaldehyde + NADPH + H+." [GOC:pz]
-xref: MetaCyc:RXN-15743
-is_a: GO:0016616 ! oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
+def: "OBSOLETE. Catalysis of the reaction: (R)-propane-1,2-diol + NADP = (R)-lactaldehyde + NADPH + H+." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102643
@@ -461203,11 +462147,12 @@
 
 [Term]
 id: GO:0102762
-name: eriodictyol 4'-O-methyltransferase activity
+name: obsolete eriodictyol 4'-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: eriodictyol + S-adenosyl-L-methionine = H+ + hesperetin(1-) + S-adenosyl-L-homocysteine." [GOC:pz, MetaCyc:RXN-7754]
-xref: MetaCyc:RXN-7754
-is_a: GO:0008168 ! methyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: eriodictyol + S-adenosyl-L-methionine = H+ + hesperetin(1-) + S-adenosyl-L-homocysteine." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102763
@@ -461324,6 +462269,7 @@
 synonym: "sphingolipid long-chain base 4-hydroxylase activity" RELATED []
 xref: EC:1.14.18.5
 xref: MetaCyc:RXN-14250
+xref: Reactome:R-HSA-428260 "DEGS2 oxygenates dihydroceramide"
 xref: RHEA:55808
 is_a: GO:0016716 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, another compound as one donor, and incorporation of one atom of oxygen
 property_value: skos:exactMatch EC:1.14.18.5
@@ -461382,7 +462328,7 @@
 id: GO:0102778
 name: delta9-tetrahydrocannabinolate synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cannabigerolate + O2 = delta(9)-tetrahydrocannabinolic acid + hydrogen peroxide." [GOC:pz, RHEA:34135]
+def: "Catalysis of the reaction: cannabigerolate + O2 = delta(9)-tetrahydrocannabinolic acid +  H2O2." [RHEA:34135]
 xref: EC:1.21.3.7
 xref: MetaCyc:RXN-7854
 xref: RHEA:34135
@@ -461395,7 +462341,7 @@
 id: GO:0102779
 name: cannabidiolate synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cannabigerolate + O2 = cannabidiolate + hydrogen peroxide." [GOC:pz, RHEA:34411]
+def: "Catalysis of the reaction: cannabigerolate + O2 = cannabidiolate + H2O2." [RHEA:34411]
 xref: EC:1.21.3.8
 xref: MetaCyc:RXN-7855
 xref: RHEA:34411
@@ -461792,12 +462738,13 @@
 
 [Term]
 id: GO:0102818
-name: lycopene cleavage oxygenase activity
+name: obsolete lycopene cleavage oxygenase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: lycopene + 2 O2 = 2 sulcatone + bixin aldehyde." [GOC:pz, MetaCyc:RXN-8236]
-xref: MetaCyc:RXN-8236
-is_a: GO:0016703 ! oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of one atom of oxygen (internal monooxygenases or internal mixed function oxidases)
+def: "OBSOLETE. Catalysis of the reaction: lycopene + 2 O2 = 2 sulcatone + bixin aldehyde." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28942" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102819
@@ -462992,9 +463939,10 @@
 
 [Term]
 id: GO:0102933
-name: GDP-4-dehydro-6-deoxy-D-mannose-4-aminotransferase activity
+name: GDP-perosamine synthase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: GDP-4-amino-4,6-dideoxy-alpha-D-mannose + 2-oxoglutarate = GDP-4-dehydro-6-deoxy-alpha-D-mannose + L-glutamate." [GOC:pz, RHEA:36779]
+def: "Catalysis of the reaction: GDP-alpha-D-perosamine + 2-oxoglutarate = GDP-4-dehydro-alpha-D-rhamnose + L-glutamate." [RHEA:36779]
+synonym: "GDP-4-dehydro-6-deoxy-D-mannose-4-aminotransferase activity" EXACT []
 xref: EC:2.6.1.102
 xref: MetaCyc:RXN-8953
 xref: RHEA:36779
@@ -463002,6 +463950,7 @@
 property_value: skos:exactMatch EC:2.6.1.102
 property_value: skos:exactMatch RHEA:36779
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 
 [Term]
 id: GO:0102934
@@ -463036,11 +463985,12 @@
 
 [Term]
 id: GO:0102937
-name: 16-alpha-hydroxygypsogenate-UDP-glucosyltransferase activity
+name: obsolete 16-alpha-hydroxygypsogenate-UDP-glucosyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: UDP-alpha-D-glucose + 16-alpha-hydroxygypsogenate = 16-alpha-hydroxygypsogenate-28-beta-D-glucoside + UDP." [GOC:pz]
-xref: MetaCyc:RXN-9014
-is_a: GO:0016758 ! hexosyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: UDP-alpha-D-glucose + 16-alpha-hydroxygypsogenate = 16-alpha-hydroxygypsogenate-28-beta-D-glucoside + UDP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102938
@@ -463129,11 +464079,12 @@
 
 [Term]
 id: GO:0102947
-name: (+)-delta-cadinene-8-hydroxylase activity
+name: obsolete (+)-delta-cadinene-8-hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (+)-delta-cadinene + NADPH + O2 = 8-hydroxy-(+)-delta-cadinene + NADP + H2O." [GOC:pz]
-xref: MetaCyc:RXN-9045
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (+)-delta-cadinene + NADPH + O2 = 8-hydroxy-(+)-delta-cadinene + NADP + H2O." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102948
@@ -463181,11 +464132,12 @@
 
 [Term]
 id: GO:0102953
-name: hypoglycin A gamma-glutamyl transpeptidase activity
+name: obsolete hypoglycin A gamma-glutamyl transpeptidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: glutathionate + hypoglycin A = L-cysteinylglycine + hypoglycin B." [GOC:pz]
-xref: MetaCyc:RXN-9157
-is_a: GO:0016755 ! aminoacyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: glutathionate + hypoglycin A = L-cysteinylglycine + hypoglycin B." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0102954
@@ -463526,7 +464478,6 @@
 name: trehalose synthase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: an NDP-alpha-D-glucose + D-glucose = a ribonucleoside 5'-diphosphate + alpha,alpha-trehalose + H+." [RHEA:47416]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:2.4.1.245
 xref: MetaCyc:RXN-9603
 xref: RHEA:47416
@@ -463534,10 +464485,8 @@
 property_value: skos:exactMatch EC:2.4.1.245
 property_value: skos:exactMatch MetaCyc:RXN-9603
 property_value: skos:exactMatch RHEA:47416
-property_value: skos:relatedMatch MetaCyc:PWY-2622
-property_value: skos:relatedMatch MetaCyc:PWY-5983
-property_value: skos:relatedMatch MetaCyc:PWY-5985
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28527" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 
 [Term]
 id: GO:0102987
@@ -463753,11 +464702,12 @@
 
 [Term]
 id: GO:0103004
-name: 9,10-epoxystearate hydroxylase activity
+name: obsolete 9,10-epoxystearate hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + 9,10-epoxystearate + O2 + NADPH = 9,10-epoxy-18-hydroxystearate + H2O + NADP." [GOC:pz]
-xref: MetaCyc:RXN-9805
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + 9,10-epoxystearate + O2 + NADPH = 9,10-epoxy-18-hydroxystearate + H2O + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103005
@@ -463770,11 +464720,12 @@
 
 [Term]
 id: GO:0103006
-name: 9,10-dihydroxystearate hydroxylase activity
+name: obsolete 9,10-dihydroxystearate hydroxylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H+ + (9R,10S)-dihydroxystearate + O2 + NADPH = 9,10,18-trihydroxystearate + H2O + NADP." [GOC:pz]
-xref: MetaCyc:RXN-9807
-is_a: GO:0016709 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
+def: "OBSOLETE. Catalysis of the reaction: H+ + (9R,10S)-dihydroxystearate + O2 + NADPH = 9,10,18-trihydroxystearate + H2O + NADP." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103007
@@ -463978,11 +464929,12 @@
 
 [Term]
 id: GO:0103033
-name: beta-galactosidase activity (lactose isomerization)
+name: obsolete beta-galactosidase activity (lactose isomerization)
 namespace: molecular_function
-def: "Catalysis of the reaction: alpha-lactose = beta-(1->6)-galactobiose." [EC:5.4.1.-, GOC:pz]
-xref: MetaCyc:RXN0-5363
-is_a: GO:0016867 ! intramolecular acyltransferase activity
+def: "OBSOLETE. Catalysis of the reaction: alpha-lactose = beta-(1->6)-galactobiose." [GOC:pz]
+comment: This term was obsoleted because it represents a specific substrate.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31581" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103035
@@ -464268,14 +465220,14 @@
 
 [Term]
 id: GO:0103068
-name: leukotriene C4 gamma-glutamyl transferase activity
+name: obsolete leukotriene C4 gamma-glutamyl transferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: leukotriene C4 + a standard alpha amino acid = leukotriene D4 + an (gamma-L-glutamyl)-L-amino acid." [GOC:pz, PMID:1676842, PMID:9139674]
+def: "OBSOLETE. Catalysis of the reaction: leukotriene C4 + a standard alpha amino acid = leukotriene D4 + an (gamma-L-glutamyl)-L-amino acid." [GOC:pz, PMID:1676842, PMID:9139674]
 xref: EC:2.3.2.2
-xref: MetaCyc:RXN66-336
-is_a: GO:0016755 ! aminoacyltransferase activity
 property_value: skos:exactMatch EC:2.3.2.2
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31373" xsd:anyURI
+is_obsolete: true
 
 [Term]
 id: GO:0103069
@@ -464621,19 +465573,18 @@
 
 [Term]
 id: GO:0103111
-name: protein-N(pi)-phosphohistidine--N-acetyl-D-glucosamine phosphotransferase activity
+name: obsolete protein-N(pi)-phosphohistidine--N-acetyl-D-glucosamine phosphotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in)." [RHEA:49240]
+def: "OBSOLETE. Catalysis of the reaction: N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in)." [RHEA:49240]
+comment: This term was obsoleted because it represents the same reaction as [protein-N(PI)-phosphohistidine-N-acetylglucosamine phosphotransferase system transporter activity ; GO:0022880.
 synonym: "D-glucosamine PTS permease activity" EXACT []
 synonym: "N-acetyl-D-glucosamine PTS permease activity" EXACT [EC:2.7.1.193]
-xref: EC:2.7.1.193
-xref: RHEA:49240
-is_a: GO:0016773 ! phosphotransferase activity, alcohol group as acceptor
-property_value: skos:exactMatch EC:2.7.1.193
-property_value: skos:exactMatch RHEA:49240
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27410" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27694" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31586" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0022880
 
 [Term]
 id: GO:0103113
@@ -465234,15 +466185,13 @@
 
 [Term]
 id: GO:0106045
-name: guanine deglycation, methylglyoxal removal
+name: obsolete guanine deglycation, methylglyoxal removal
 namespace: biological_process
-def: "The removal of methylglyoxal from a glycated guanine, to form lactate and a deglycated guanine." [PMID:28596309]
-is_a: GO:0009438 ! methylglyoxal metabolic process
-is_a: GO:0051595 ! response to methylglyoxal
-is_a: GO:0106044 ! guanine deglycation
-is_a: GO:0140041 ! cellular detoxification of methylglyoxal
-created_by: hjd
-creation_date: 2017-08-01T18:17:28Z
+def: "OBSOLETE. The removal of methylglyoxal from a glycated guanine, to form lactate and a deglycated guanine." [PMID:28596309]
+comment: The reason for obsoletion is that this term represents a molecular function rather than a biological process.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31367" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0036524
 
 [Term]
 id: GO:0106046
@@ -465289,14 +466238,17 @@
 
 [Term]
 id: GO:0106050
-name: tRNA 2'-O-methyltransferase activity
+name: obsolete tRNA 2'-O-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing a 2'-O-nucleotide." [PMID:17242307]
+def: "OBSOLETE. Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing a 2'-O-nucleotide." [PMID:17242307]
+comment: This term was obsoleted because it is an unnecessray grouping term.
 xref: Reactome:R-HSA-6788668 "TRMT13 2'-O-methylates adenosine-4 in tRNA"
 xref: Reactome:R-HSA-6788684 "TRMT13 2'-O-methylates cytidine-4 in tRNA"
 xref: Reactome:R-HSA-9024159 "FTSJ1 2'-O-methylates cytidine-32 in tRNA(Phe)"
-is_a: GO:0008175 ! tRNA methyltransferase activity
-is_a: GO:0062105 ! RNA 2'-O-methyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31519" xsd:anyURI
+is_obsolete: true
+consider: GO:0106339
+consider: GO:0106340
 created_by: hjd
 creation_date: 2017-08-11T20:30:31Z
 
@@ -465368,10 +466320,11 @@
 
 [Term]
 id: GO:0106059
-name: tRNA (cytidine(56)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(56)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + cytidine56 in tRNA= S-adenosyl-L-homocysteine + 2'-O-methylcytidine56 in tRNA." [RHEA:42968]
 synonym: "tRNA (cytidine 56-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(56)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.206
 xref: MetaCyc:RXN-11869
 xref: RHEA:42968
@@ -466459,13 +467412,16 @@
 
 [Term]
 id: GO:0106162
-name: mRNA N-acetyltransferase activity
+name: mRNA cytidine N-acetyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a cytidine in mRNA + acetyl-CoA + ATP + H2O = ADP + an N(4)-acetylcytidine in mRNA + CoA + H+ + phosphate." [GOC:sp, PMID:30449621, RHEA:58480]
+synonym: "mRNA N-acetyltransferase activity" BROAD []
 xref: RHEA:58480
 is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140640 ! catalytic activity, acting on a nucleic acid
 property_value: skos:exactMatch RHEA:58480
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
+property_value: terms:contributor "https://github.com/geneontology/go-ontology/issues/16755" xsd:anyURI
 created_by: hjd
 creation_date: 2018-12-18T19:19:06Z
 
@@ -466681,9 +467637,10 @@
 id: GO:0106186
 name: cytoplasmic side of plasma membrane, cell tip
 namespace: cellular_component
-def: "The leaflet the plasma membrane at the cell tip that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:vw, PMID:28292899]
+def: "The leaflet the plasma membrane at the cell tip that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [GOC:vw, PMID:28292899]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0031520 ! plasma membrane of cell tip
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: hjd
 creation_date: 2019-05-01T17:48:02Z
 
@@ -467053,6 +468010,7 @@
 name: peptide glutaryltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: glutaryl-CoA + L-lysyl-[protein] = CoA + H+ + N6-glutaryl-L-lysyl-[protein]." [GOC:sp, PMID:31542297]
+xref: Reactome:R-HSA-9858590 "DLST transfers glutaryl to CoA"
 xref: RHEA:18009
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:18009
@@ -467357,6 +468315,8 @@
 namespace: molecular_function
 def: "A hydroperoxy icosatetraenoate = a hydroxy epoxy icosatrienoate." [PMID:12881489, RHEA:55560]
 xref: EC:5.4.4.7
+xref: Reactome:R-HSA-2161794 "Arachidonate is converted to HXA3/B3 by ALOX12"
+xref: Reactome:R-HSA-8942208 "ALOXE3 isomerises 12R-HpETE to HXA3"
 xref: RHEA:55560
 is_a: GO:0050486 ! intramolecular hydroxytransferase activity
 property_value: skos:exactMatch EC:5.4.4.7
@@ -467964,6 +468924,7 @@
 xref: Reactome:R-HSA-9841265 "p-S473-AKT1 phosphorylates ESR1 on serine-167"
 xref: Reactome:R-HSA-9860792 "Phospho-AKT1 phosphorylates NOS3 (eNOS) on serine-1177"
 xref: Reactome:R-HSA-9860808 "p-T816-PKN2 phosphorylates NOS3 (eNOS) on serine-1179"
+xref: Reactome:R-HSA-9948087 "CAMK2 complex phosphorylates VIM:NS4A"
 xref: RHEA:17989
 is_a: GO:0004672 ! protein kinase activity
 property_value: skos:exactMatch RHEA:17989
@@ -468182,7 +469143,7 @@
 id: GO:0106329
 name: L-phenylalaine oxidase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + L-phenylalanine + O2 = 3-phenylpyruvate + H2O2 + NH4(+)." [RHEA:61240]
+def: "Catalysis of the reaction: H2O + L-phenylalanine + O2 = 3-phenylpyruvate + H2O2 + NH4+." [RHEA:61240]
 xref: RHEA:61240
 is_a: GO:0001716 ! L-amino-acid oxidase activity
 property_value: skos:exactMatch RHEA:61240
@@ -468256,7 +469217,7 @@
 id: GO:0106335
 name: tRNA (5-carboxymethyluridine(34)-5-O)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 5-(carboxymethyl)uridine34 in tRNA + S-adenosyl-L-methionine = 5-(2-methoxy-2-oxoethyl)uridine34 in tRNA + S-adenosyl-L-homocysteine." [PMID:20123966, RHEA:43208]
+def: "Catalysis of the reaction: 5-(carboxymethyl)uridine34 in tRNA + S-adenosyl-L-methionine = 5-(2-methoxy-2-oxoethyl)uridine34 in tRNA + S-adenosyl-L-homocysteine. The methylation occurs on the modified base." [PMID:20123966, RHEA:43208]
 xref: EC:2.1.1.229
 xref: RHEA:43208
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -468280,10 +469241,11 @@
 
 [Term]
 id: GO:0106339
-name: tRNA (cytidine(32)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(32)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(32) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(32) in tRNA + S-adenosyl-L-homocysteine + H+." [PMID:25404562, RHEA:42932]
 synonym: "tRNA (cytidine 32-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(32)-2'-O)-methyltransferase activity" EXACT []
 xref: MetaCyc:RXN-11866
 xref: RHEA:42932
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
@@ -468302,12 +469264,15 @@
 
 [Term]
 id: GO:0106340
-name: tRNA (guanosine(34)-2'-O)-methyltransferase activity
+name: tRNA (guanosine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanosine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylguanosine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105]
+def: "Catalysis of the reaction: guanosine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylguanosine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105, RHEA:85171]
+synonym: "tRNA (guanosine(34)-2'-O)-methyltransferase activity" EXACT []
 xref: Reactome:R-HSA-9024161 "FTSJ1 2'-O-methylates guanosine-34 in tRNA(Phe)"
+xref: RHEA:85171
 is_a: GO:0016423 ! tRNA (guanine) methyltransferase activity
 property_value: skos:broadMatch EC:2.1.1.205
+property_value: skos:exactMatch RHEA:85171
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25257" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26085" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26086" xsd:anyURI
@@ -468389,9 +469354,10 @@
 
 [Term]
 id: GO:0106347
-name: U2 snRNA 2'-O-methyladenosine m6 methyltransferase activity
+name: U2 snRNA (2'-O-methyladenosine-N6)-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+." [PMID:31913360, RHEA:62672]
+def: "Catalysis of the reaction: a 2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-2'-O-methyladenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+; methylates the 6th position of adenine residues with a pre-deposited 2'-O-methylation." [PMID:31913360, RHEA:62672]
+synonym: "U2 snRNA (2'-O-methyladenosine) m6 methyltransferase activity" EXACT []
 xref: RHEA:62672
 is_a: GO:0106346 ! snRNA methyltransferase activity
 property_value: skos:exactMatch RHEA:62672
@@ -468401,9 +469367,11 @@
 
 [Term]
 id: GO:0106348
-name: U2 snRNA adenosine m6 methyltransferase activity
+name: U2 snRNA (adenine-N6)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: a adenosine in U2 snRNA + S-adenosyl-L-methionine = an N6-methyl-adenosine in U2 snRNA + S-adenosyl-L-homocysteine + H+." [PMID:32637152]
+synonym: "U2 snRNA adenosine m6 methyltransferase activity" EXACT []
+synonym: "U2 snRNA adenosine N6 methyltransferase activity" EXACT []
 is_a: GO:0106346 ! snRNA methyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27392" xsd:anyURI
 created_by: hjd
@@ -469851,11 +470819,14 @@
 
 [Term]
 id: GO:0110035
-name: rDNA spacer replication fork barrier binding, bending
+name: obsolete rDNA spacer replication fork barrier binding, bending
 namespace: molecular_function
-def: "The activity of binding selectively, and in a sequence-specific manner, a replication fork barrier found in rDNA spacers, and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:al, GOC:vw, PMID:27035982]
-is_a: GO:0043110 ! rDNA spacer replication fork barrier binding
-is_a: GO:0044374 ! sequence-specific DNA binding, bending
+def: "OBSOLETE. The activity of binding selectively, and in a sequence-specific manner, a replication fork barrier found in rDNA spacers, and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:al, GOC:vw, PMID:27035982]
+comment: This term was obsoleted because it is precomposed.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+consider: GO:0008301
+consider: GO:0043110
 created_by: kmv
 creation_date: 2017-07-17T21:09:50Z
 
@@ -471502,13 +472473,19 @@
 namespace: molecular_function
 def: "Removes a lipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle. This results in intermembrane transfer of lipids." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane lipid transfer activity" EXACT []
-synonym: "lipid carrier activity" EXACT []
+synonym: "lipid carrier activity" BROAD []
+xref: Reactome:R-HSA-8856630 "CIDEA:CIDEC binds lipid droplets"
+xref: Reactome:R-HSA-8858270 "HILPDA binds lipid droplets"
+xref: Reactome:R-HSA-8862380 "HSD17B13 binds lipid droplets"
+xref: Reactome:R-HSA-8874705 "TTPA transports alpha-TOH from lysosomal membrane to plasma membrane"
 xref: Reactome:R-HSA-9856502 "mito-STARD7 transports Q10 to outer mitochondrial membrane"
 xref: Reactome:R-HSA-9856510 "STARD7 transports Q10 to plasma membrane"
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0005215 ! transporter activity
+is_a: GO:0005319 ! lipid carrier activity
 relationship: part_of GO:0120009 ! intermembrane lipid transfer
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17648" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 created_by: krc
 creation_date: 2017-03-17T03:17:29Z
 
@@ -471519,9 +472496,9 @@
 def: "Removes a phospholipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane phospholipid transfer activity" NARROW []
 synonym: "phospholipid carrier activity" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0120013 ! lipid transfer activity
 relationship: has_part GO:0005543 ! phospholipid binding
+relationship: part_of GO:0015914 ! phospholipid transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
 created_by: krc
 creation_date: 2017-03-17T03:34:17Z
@@ -471550,7 +472527,6 @@
 def: "Removes a sphingolipid from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle." [GOC:krc, PMID:20823909, PMID:24220498, PMID:25797198]
 synonym: "intermembrane sphingolipid transfer activity" NARROW []
 synonym: "sphingolipid carrier activity" NARROW []
-is_a: GO:0046624 ! sphingolipid transporter activity
 is_a: GO:0120013 ! lipid transfer activity
 relationship: has_part GO:0046625 ! sphingolipid binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471566,6 +472542,7 @@
 synonym: "ceramide carrier activity" EXACT []
 synonym: "ceramide transporter activity" BROAD []
 synonym: "intermembrane ceramide transfer activity" NARROW []
+xref: Reactome:R-HSA-9956909 "CERT1-2 transfers CERA from the ER membrane to the Golgi membrane"
 is_a: GO:0120016 ! sphingolipid transfer activity
 relationship: has_part GO:0097001 ! ceramide binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471582,7 +472559,7 @@
 xref: Reactome:R-HSA-1483087 "PI is exchanged with PC by PITPNB"
 xref: Reactome:R-HSA-1483211 "PC:PITPNB is transported from the Golgi membrane to the ER membrane"
 xref: Reactome:R-HSA-1483219 "PC is exchanged with PI by PITPNB"
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+xref: Reactome:R-HSA-9956308 "STARD7 transports PC from ER membrane to mitochondrial outer membrane"
 is_a: GO:0120014 ! phospholipid transfer activity
 relationship: has_part GO:0031210 ! phosphatidylcholine binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/12329" xsd:anyURI
@@ -471923,9 +472900,10 @@
 
 [Term]
 id: GO:0120048
-name: U6 snRNA (adenine-(43)-N(6))-methyltransferase activity
+name: U6 snRNA (adenine(43)-N6)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: adenosine in U6 snRNA + S-adenosyl-L-methionine = H+ + N(6)-methyladenosine in U6 snRNA + S-adenosyl-L-homocysteine." [PMID:28525753, PMID:32266935, RHEA:52808]
+synonym: "U6 snRNA m(6)A methyltransferase activity" EXACT []
 xref: EC:2.1.1.346
 xref: MetaCyc:RXN-18779
 xref: RHEA:52808
@@ -471953,6 +472931,7 @@
 name: ribitol beta-1,4-xylosyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: UDP-D-xylose + D-ribitol 5-phosphate-R = UDP + beta1,4-xylosyl-D-ribitol 5-phosphate-R." [PMID:27733679]
+xref: Reactome:R-HSA-9940805 "RXYLT1 transfers initial Xyl to matriglycan chain"
 is_a: GO:0035252 ! UDP-xylosyltransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/13641" xsd:anyURI
 created_by: krc
@@ -474020,7 +474999,7 @@
 id: GO:0120216
 name: matrilin complex
 namespace: cellular_component
-def: "A cartilage extracellular matrix complex that mediates interactions between major components of the extracellular matrix such as collagens and proteoglycans and contributes to their fibrillar network. Exists as an obligate homotrimer." [PMID:10367731, PMID:15075323, PMID:15094109, PMID:9699631]
+def: "A cartilage extracellular matrix complex that mediates interactions between major components of the extracellular matrix such as collagens and proteoglycans and contributes to their fibrillar network. Exists as an obligate homotrimer." [PMID:10367731, PMID:15075323, PMID:29310791, PMID:9699631]
 synonym: "matrilin family complex" RELATED [GOC:bhm]
 synonym: "matrilin-1 complex" NARROW [GOC:bhm]
 synonym: "matrilin-2 complex" NARROW [GOC:bhm]
@@ -474366,10 +475345,11 @@
 id: GO:0120242
 name: 2-iminobutanoate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-iminobutanoate + H2O = 2-oxobutanoate + NH4(+)." [RHEA:39975]
+def: "Catalysis of the reaction: 2-iminobutanoate + H2O = 2-oxobutanoate + NH4+." [RHEA:39975]
 synonym: "2-iminobutanoate/2-iminopropanoate deaminase" BROAD [EC:3.5.99.10]
 xref: KEGG_REACTION:R11098
 xref: MetaCyc:RXN-15123
+xref: Reactome:R-HSA-9014641 "HRSP12 deaminates 2-iminobutanoate to 2OBUTA"
 xref: RHEA:39975
 is_a: GO:0120241 ! 2-iminobutanoate/2-iminopropanoate deaminase activity
 property_value: skos:exactMatch RHEA:39975
@@ -474382,7 +475362,7 @@
 id: GO:0120243
 name: 2-iminopropanoate deaminase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 2-iminopropanoate + H2O = NH4(+) + pyruvate." [RHEA:40671]
+def: "Catalysis of the reaction: 2-iminopropanoate + H2O = NH4+ + pyruvate." [RHEA:40671]
 synonym: "2-iminobutanoate/2-iminopropanoate deaminase" BROAD [EC:3.5.99.10]
 xref: KEGG_REACTION:R11099
 xref: MetaCyc:RXN-15127
@@ -475114,7 +476094,7 @@
 id: GO:0120294
 name: peptide serotonyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutaminyl-[protein] + serotonin = 5-serotonyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:14697203]
+def: "Catalysis of the reaction: L-glutaminyl-[protein] + serotonin = 5-serotonyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:14697203]
 xref: RHEA:66552
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66552
@@ -475127,7 +476107,7 @@
 id: GO:0120295
 name: histone serotonyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-glutaminyl-[histone] + serotonin = 5-serotonyl-L-glutamyl-[histone] + NH4(+)." [GOC:sp, PMID:30867594]
+def: "Catalysis of the reaction: L-glutaminyl-[histone] + serotonin = 5-serotonyl-L-glutamyl-[histone] + NH4+." [GOC:sp, PMID:30867594]
 is_a: GO:0120294 ! peptide serotonyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
 property_value: skos:broadMatch RHEA:66552
@@ -475140,7 +476120,7 @@
 id: GO:0120296
 name: peptide dopaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dopamine + L-glutaminyl-[protein] = 5-dopaminyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:22858378, PMID:32273471]
+def: "Catalysis of the reaction: dopamine + L-glutaminyl-[protein] = 5-dopaminyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:22858378, PMID:32273471]
 xref: RHEA:66556
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66556
@@ -475153,7 +476133,7 @@
 id: GO:0120297
 name: histone dopaminyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: dopamine + L-glutaminyl-[histone] = 5-dopaminyl-L-glutamyl-[histone] + NH4(+)." [GOC:sp, PMID:32273471]
+def: "Catalysis of the reaction: dopamine + L-glutaminyl-[histone] = 5-dopaminyl-L-glutamyl-[histone] + NH4+." [GOC:sp, PMID:32273471]
 is_a: GO:0120296 ! peptide dopaminyltransferase activity
 is_a: GO:0140993 ! histone modifying activity
 property_value: skos:broadMatch RHEA:66556
@@ -475166,7 +476146,7 @@
 id: GO:0120298
 name: peptide noradrenalinyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: (R)-noradrenaline + L-glutaminyl-[protein] = 5-(R)-noradrenalinyl-L-glutamyl-[protein] + NH4(+)." [GOC:sp, PMID:22858378]
+def: "Catalysis of the reaction: (R)-noradrenaline + L-glutaminyl-[protein] = 5-(R)-noradrenalinyl-L-glutamyl-[protein] + NH4+." [GOC:sp, PMID:22858378]
 xref: RHEA:66560
 is_a: GO:0016410 ! N-acyltransferase activity
 property_value: skos:exactMatch RHEA:66560
@@ -476107,7 +477087,6 @@
 name: choline trimethylamine lyase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: choline = acetaldehyde + trimethylamine." [RHEA:35095]
-comment: Note that this term has a MetaCyc pathway reference as the pathway only has a single step.
 xref: EC:4.3.99.4
 xref: KEGG_REACTION:R10285
 xref: MetaCyc:RXN-13946
@@ -476117,8 +477096,8 @@
 property_value: skos:exactMatch KEGG_REACTION:R10285
 property_value: skos:exactMatch MetaCyc:RXN-13946
 property_value: skos:exactMatch RHEA:35095
-property_value: skos:relatedMatch MetaCyc:PWY-7167
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28808" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31587" xsd:anyURI
 created_by: sjm
 creation_date: 2024-08-30T13:13:55Z
 
@@ -476228,6 +477207,7 @@
 synonym: "Glycosaminoglycan biosynthesis, linkage tetrasaccharide" EXACT []
 synonym: "glycosaminoglycan-protein linkage region biosynthesis" EXACT [MetaCyc:PWY-6557]
 xref: MetaCyc:PWY-6557
+xref: Reactome:R-HSA-1971475 "Glycosaminoglycan-protein linkage region biosynthesis"
 is_a: GO:0180064 ! protein O-linked glycosylation via xylose
 relationship: part_of GO:0015012 ! heparan sulfate proteoglycan biosynthetic process
 relationship: part_of GO:0030210 ! heparin proteoglycan biosynthetic process
@@ -476316,6 +477296,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: a 4-hydroxy-3-methoxy-5-(all-trans-polyprenyl)benzoate + H+ = a 2-methoxy-6-(all-trans-polyprenyl)phenol + CO2." [PMID:38295803, PMID:38425362, RHEA:81179]
 xref: EC:4.1.1.130
+xref: Reactome:R-HSA-2162195 "COQ4 decarboxylates MHDB"
 xref: RHEA:81179
 is_a: GO:0016831 ! carboxy-lyase activity
 relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
@@ -476423,6 +477404,7 @@
 def: "Catalysis of the reaction: Fe(II)-heme o + 2 acceptor + H2O = Fe(II)-heme a + 2 acceptor-H2. The conversion of heme o to heme a occcurs by two successive hydroxylations of the methyl group at C8 using water as the oxygen source. The first hydroxylation forms heme i, the second hydroxylation results in an unstable dihydroxymethyl group, which spontaneously dehydrates, resulting in the formyl group of heme A." [PMID:30397130, RHEA:63388]
 xref: EC:1.17.99.9
 xref: MetaCyc:RXN-18399
+xref: Reactome:R-HSA-2995334 "COX15 transforms heme O to heme A"
 xref: RHEA:63388
 is_a: GO:0016725 ! oxidoreductase activity, acting on CH or CH2 groups
 property_value: skos:exactMatch EC:1.17.99.9
@@ -476527,15 +477509,17 @@
 
 [Term]
 id: GO:0120554
-name: 2-aminobutanoate transaminase activity
+name: (2S)-2-aminobutanoate:2-oxoglutarate transaminase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: (2S)-2-aminobutanoate + 2-oxoglutarate = 2-oxobutanoate + L-glutamate." [PMID:27827456, RHEA:70223]
+synonym: "2-aminobutanoate transaminase activity" EXACT []
 xref: MetaCyc:RXN-20408
 xref: RHEA:70223
 is_a: GO:0008483 ! transaminase activity
 property_value: skos:exactMatch MetaCyc:RXN-20408
 property_value: skos:exactMatch RHEA:70223
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30260" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
 created_by: sjm
 creation_date: 2025-05-15T14:38:07Z
 
@@ -477576,7 +478560,7 @@
 id: GO:0140070
 name: hydrogen peroxide channel activity
 namespace: molecular_function
-def: "Enables the energy-independent facilitated diffusion of hydrogen peroxide through a transmembrane aqueous pore or channel." [PMID:23541115, PMID:27256569]
+def: "Enables the energy-independent facilitated diffusion of H2O2 through a transmembrane aqueous pore or channel." [PMID:23541115, PMID:27256569]
 synonym: "hydrogen peroxide transmembrane transporter activity" RELATED []
 is_a: GO:0015267 ! channel activity
 relationship: part_of GO:0080170 ! hydrogen peroxide transmembrane transport
@@ -477757,11 +478741,12 @@
 id: GO:0140085
 name: L-amino-acid N-acetyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: acetyl-CoA + a L-amino acid = CoA + an N-acetyl-L-amino-acid. In some cases acetyl phosphate can be used as a donor." [GOC:pg, RHEA:83863]
+def: "Catalysis of the reaction: acetyl-CoA + a L-amino acid = CoA + an N-acetyl-L-amino-acid. In some cases other acetyl containing molecules can be used as donor." [GOC:pg, RHEA:83863]
 xref: RHEA:83863
-is_a: GO:0008080 ! N-acetyltransferase activity
-property_value: skos:exactMatch RHEA:83863
+is_a: GO:0140379 ! amino acid acyltransferase activity
+property_value: skos:narrowMatch RHEA:83863
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29604" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
 created_by: pg
 creation_date: 2025-01-24T10:40:36Z
 
@@ -477985,10 +478970,23 @@
 subset: goslim_prokaryote
 subset: goslim_prokaryote_ribbon
 xref: Reactome:R-HSA-1449687 "Corticosteroids bind to CBG in blood"
+xref: Reactome:R-HSA-196126 "STAR mediates CHOL translocation from cytosol to mitochondrial intermembrane space"
+xref: Reactome:R-HSA-209738 "VD3 binds GC"
+xref: Reactome:R-HSA-209944 "25(OH)D binds GC"
+xref: Reactome:R-HSA-2395764 "atREs binds to nascent CM"
+xref: Reactome:R-HSA-2399913 "atRAL binds to RBP2"
+xref: Reactome:R-HSA-2404142 "atROL binds RBP1"
+xref: Reactome:R-HSA-2855259 "atROL binds RBP1 (parenchymal cell)"
 xref: Reactome:R-HSA-3149494 "MMACHC:cob(II)alamin binds MMADHC"
 xref: Reactome:R-HSA-3149563 "MMADHC targets transport of cytosolic cob(II)alamin to mitochondria"
 xref: Reactome:R-HSA-3159259 "MMAA:MUT binds AdoCbl"
 xref: Reactome:R-HSA-3204318 "cob(II)alamin is transferred from MMACHC:MMADHC:cob(II)alamin to MTRR:MTR"
+xref: Reactome:R-HSA-5246478 "APOM binds retinoids"
+xref: Reactome:R-HSA-8870499 "PLEKHA3,8 bind PI4P, ARF1"
+xref: Reactome:R-HSA-8932980 "SERINC3,5,(1,2,4) transport L-Ser from cytosol to plasma membrane"
+xref: Reactome:R-HSA-8963851 "VD3 dissociates from GC"
+xref: Reactome:R-HSA-9661432 "BIL binds ALB"
+xref: Reactome:R-HSA-9663511 "BIL binds GSTA1, FABP1"
 is_a: GO:0003674 ! molecular_function
 relationship: has_part GO:0005488 ! binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/14221" xsd:anyURI
@@ -478599,12 +479597,14 @@
 
 [Term]
 id: GO:0140155
-name: hexagonal collagen network of basement membrane
+name: obsolete hexagonal collagen network of basement membrane
 namespace: cellular_component
-def: "Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane." [PMID:21421911, PMID:2376131, PMID:3047147, PMID:31387942]
-is_a: GO:0098645 ! complex of network-forming collagens
-relationship: part_of GO:0140143 ! collagenous component of basement membrane
+def: "OBSOLETE. Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane." [PMID:21421911, PMID:2376131, PMID:3047147, PMID:31387942]
+comment: This term was obsoleted because it is an uncessary grouping term. There is not sufficient evidence to show that collagen VIII is only located in the basement membrane in all organs and species.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29402" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31492" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0098646
 created_by: pg
 creation_date: 2025-03-26T06:55:36Z
 
@@ -478687,11 +479687,12 @@
 
 [Term]
 id: GO:0140164
-name: Golgi transport complex binding
+name: COG complex binding
 namespace: molecular_function
 def: "Binding to a Golgi transport complex, a multisubunit tethering complex of the CATCHR family." [GOC:ha, PMID:28100664]
-synonym: "COG complex binding" EXACT []
+synonym: "Golgi transport complex binding" EXACT []
 is_a: GO:0044877 ! protein-containing complex binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31397" xsd:anyURI
 created_by: pg
 creation_date: 2018-02-08T11:49:38Z
 
@@ -478865,7 +479866,7 @@
 id: GO:0140177
 name: membrane-membrane adaptor activity
 namespace: molecular_function
-def: "The binding activity of a molecule that brings together two membranes, either via membrane lipid binding or by interacting with a membrane protein, to establish or maintain the localization of the protein, protein complex or organelle." [GOC:curator]
+def: "The binding activity of a molecule that brings together two membranes, either via membrane lipid binding or by interacting with a membrane protein, to establish or maintain the localization of the protein, protein complex or organelle." [GOC:curators]
 is_a: GO:0060090 ! molecular adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24964" xsd:anyURI
 created_by: pg
@@ -478875,10 +479876,11 @@
 id: GO:0140178
 name: lumenal side of cis-Golgi network membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi network directly contacts the Golgi lumen and may be involved in glycosylation, cargo sorting, or interactions with resident Golgi proteins." [PMID:1747103, PMID:29802621, PMID:34080016]
+def: "The leaflet of the cis-Golgi network membrane directly contacts the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where glycosylation, cargo sorting, or interactions with resident Golgi proteins occur." [PMID:1747103, PMID:29802621, PMID:34080016]
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0033106 ! cis-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30389" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-05T08:47:14Z
 
@@ -478886,10 +479888,11 @@
 id: GO:0140179
 name: cytoplasmic side of cis-Golgi network membrane
 namespace: cellular_component
-def: "The leaflet of the membrane bilayer of the cis-Golgi network faces the cytoplasm and is where interactions with cytosolic proteins occur, including those for vesicle tethering, fusion, and Golgi-associated trafficking activities." [PMID:1747103, PMID:23913272, PMID:34080016, PMID:34597626]
+def: "The leaflet of the cis-Golgi network membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins, including those for vesicle tethering, fusion, and Golgi-associated trafficking activities." [PMID:1747103, PMID:23913272, PMID:34080016, PMID:34597626]
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0033106 ! cis-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30388" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-05T08:50:47Z
 
@@ -478898,8 +479901,9 @@
 name: centriole scaffold activity
 namespace: molecular_function
 def: "The binding activity of a protein that contributes to the stable formation of a centriole by forming persistent structures, or templates, upon which other centriolar proteins assemble." [PMID:29784964, PMID:32956907, PMID:34702818]
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30443" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31499" xsd:anyURI
 created_by: pg
 creation_date: 2025-06-11T13:59:35Z
 
@@ -479250,6 +480254,8 @@
 name: folic acid:proton symporter activity
 namespace: molecular_function
 def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: folic acid(out) + H+(out) = folic acid(in) + H+(in). The main folic acid symporter is the Proton-Coupled Folate Transporter (PCFT/SLC46A1), which has similar affinity for transport of reduced folates (5-methyl THF, 5-formyl THF) and folic acid." [PMID:24745983]
+xref: Reactome:R-HSA-200646 "Cytosolic folate export across the plasma membrane"
+xref: Reactome:R-HSA-200729 "SLC46A1 transports FOLA from extracellular region to cytosol"
 xref: TC:2.A.1.50.1
 is_a: GO:0008517 ! folic acid transmembrane transporter activity
 is_a: GO:0015295 ! solute:proton symporter activity
@@ -479816,8 +480822,8 @@
 def: "A cellular process in which two or more cells combine together, their plasma membrane fusing, producing a single cell. In some cases, nuclei fuse, producing a polyploid cell, while in other cases, nuclei remain separate, producing a syncytium." [Wikipedia:Cell_fusion]
 synonym: "cell cell fusion" EXACT []
 synonym: "cell fusion" BROAD []
+synonym: "plasma membrane fusion" EXACT []
 is_a: GO:0009987 ! cellular process
-relationship: has_part GO:0045026 ! plasma membrane fusion
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15939" xsd:anyURI
 created_by: pg
 creation_date: 2018-08-13T18:38:18Z
@@ -480414,6 +481420,7 @@
 synonym: "small molecular sensor activity" EXACT []
 synonym: "small molecule sensing activity" EXACT []
 synonym: "small molecule sensor activity" NARROW []
+xref: Reactome:R-HSA-9854368 "ROS,RNS oxidize ACO2:4Fe-4S"
 is_a: GO:0098772 ! molecular function regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28772" xsd:anyURI
 created_by: pg
@@ -480451,15 +481458,18 @@
 
 [Term]
 id: GO:0140303
-name: intramembrane lipid transporter activity
+name: intramembrane lipid carrier activity
 namespace: molecular_function
 def: "Enables the transport of a lipid from a region of a membrane to a different region on the same membrane." [PMID:16828084]
 synonym: "flippase activity" RELATED []
+synonym: "intramembrane lipid transporter activity" BROAD []
 synonym: "translocase activity" BROAD []
-is_a: GO:0005319 ! lipid transporter activity
+is_a: GO:0005215 ! transporter activity
+is_a: GO:0005319 ! lipid carrier activity
 relationship: part_of GO:0034204 ! lipid translocation
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/17648" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19477" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31454" xsd:anyURI
 created_by: pg
 creation_date: 2019-03-01T20:14:09Z
 
@@ -480527,13 +481537,16 @@
 
 [Term]
 id: GO:0140309
-name: unfolded protein carrier activity
+name: unfolded protein holdase activity
 namespace: molecular_function
-def: "A protein carrier activity that binds to a protein in an unfolded state and escorts it between two different cellular components. The unfolded protein carrier prevents aggregation of the target protein." [PMID:39488384]
-synonym: "holdase" BROAD []
+def: "A protein carrier activity that binds to a protein in an unfolded state and escorts it to an acceptor molecule or to a specific location. The unfolded protein carrier prevents aggregation of the target protein while it's being delivers to its final destination." [PMID:39488384]
+comment: Note that an holdase binds an unfolded protein and keeps it unfolded, unlike a protein folding chaperone, which binds an unfolded protein to fold it.
+synonym: "holdase" EXACT []
 synonym: "holdase-carrier chaperone" EXACT []
-is_a: GO:0140597 ! protein carrier chaperone
+synonym: "unfolded protein carrier activity" EXACT []
+is_a: GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30552" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31601" xsd:anyURI
 created_by: pg
 creation_date: 2025-11-03T13:14:32Z
 
@@ -480556,6 +481569,7 @@
 name: protein sequestering activity
 namespace: molecular_function
 def: "Binding to a protein to prevent it from interacting with other partners or to inhibit its localization to the area of the cell or complex where it is active." [PMID:1493333]
+xref: Reactome:R-HSA-1678694 "Heparanase 2 (HPSE2) binds heparan sulfate proteoglycans"
 is_a: GO:0140313 ! molecular sequestering activity
 intersection_of: GO:0140313 ! molecular sequestering activity
 intersection_of: has_part GO:0005515 ! protein binding
@@ -480593,6 +481607,7 @@
 subset: goslim_prokaryote_ribbon
 xref: Reactome:R-HSA-203553 "eNOS binds NOSIP"
 xref: Reactome:R-HSA-203680 "eNOS:NOSIP translocation from caveolae to intracellular compartments"
+xref: Reactome:R-HSA-2404134 "RBP4:atROL binds TTR"
 xref: Reactome:R-HSA-3245898 "TCN1 binds correnoids in the circulation"
 is_a: GO:0003674 ! molecular_function
 created_by: pg
@@ -480726,7 +481741,11 @@
 name: lysophospholipase C activity
 namespace: molecular_function
 def: "Hydrolysis of a lysoglycerophospholipid at the first phosphodiester bond between the phosphate and glycerol." [PMID:32917725]
+xref: RHEA:86375
+xref: RHEA:86379
 is_a: GO:0004629 ! C-type glycerophospholipase activity
+property_value: skos:narrowMatch RHEA:86375
+property_value: skos:narrowMatch RHEA:86379
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30949" xsd:anyURI
 created_by: pg
 creation_date: 2025-11-10T12:35:51Z
@@ -480747,12 +481766,13 @@
 
 [Term]
 id: GO:0140326
-name: ATPase-coupled intramembrane lipid transporter activity
+name: ATPase-coupled intramembrane lipid carrier activity
 namespace: molecular_function
 alt_id: GO:0004012
 alt_id: GO:0008557
 def: "Catalysis of the movement of lipids from one membrane leaflet to the other, driven by ATP hydrolysis. This includes flippases and floppases." [PMID:16828084]
 synonym: "aminophospholipid-transporting ATPase" RELATED []
+synonym: "ATPase-coupled intramembrane lipid transporter activity" BROAD []
 synonym: "ATPase-coupled phospholipid transporter activity" RELATED []
 synonym: "ATPase-dependent phospholipid transporter activity" RELATED []
 synonym: "phospholipid flippase activity" NARROW []
@@ -480763,7 +481783,7 @@
 xref: MetaCyc:3.6.3.1-RXN
 xref: Reactome:R-HSA-939763 "P-type ATPases type IV transport external-facing APLs to internal side of the plasma membrane"
 xref: Reactome:R-HSA-947591 "P-type ATPases type IV transport internal-facing APLs to external side of the plasma membrane"
-is_a: GO:0140303 ! intramembrane lipid transporter activity
+is_a: GO:0140303 ! intramembrane lipid carrier activity
 is_a: GO:0140657 ! ATP-dependent activity
 property_value: skos:broadMatch RHEA:66132
 property_value: skos:exactMatch EC:7.6.2.1
@@ -480781,7 +481801,7 @@
 def: "Catalysis of the movement of lipids from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:20043909, PMID:25284293, Wikipedia:Flippase]
 comment: Nomenclature note. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases). Scramblases are ATP-independent, non-selective, inducing non-specific transbilayer movements across the membrane. The direction of the translocation should be taken into account for annotation (from the exoplasmic to the cytosolic leaflet of a membrane).
 synonym: "flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0140326 ! ATPase-coupled intramembrane lipid transporter activity
+is_a: GO:0140326 ! ATPase-coupled intramembrane lipid carrier activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
 creation_date: 2019-04-29T12:51:21Z
@@ -480793,7 +481813,7 @@
 def: "Catalysis of the movement of a lipid from the cytosolic to the exoplasmic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:20043909, PMID:25284293, Wikipedia:Flippase]
 comment: Nomenclature note. Flippases and floppases are ATP-dependent transbilayer lipid translocators. According to an extensively used, though not universal, nomenclature, they catalyze lipid transfer towards the inward monolayer (flippases) or towards the outward monolayer (floppases). Scramblases are ATP-independent, non-selective, inducing non-specific transbilayer movements across the membrane. The direction of the translocation should be taken into account for annotation (from the cytosolic to the exoplasmic leaflet of a membrane).
 synonym: "floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
-is_a: GO:0140326 ! ATPase-coupled intramembrane lipid transporter activity
+is_a: GO:0140326 ! ATPase-coupled intramembrane lipid carrier activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
 creation_date: 2019-04-29T12:58:53Z
@@ -480849,7 +481869,6 @@
 namespace: molecular_function
 def: "Catalysis of the movement of a glycerophospholipid from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:26212235]
 synonym: "glycerophospholipid flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140327 ! flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -480874,7 +481893,9 @@
 synonym: "glycoprotein phospholipase C activity" EXACT []
 synonym: "glycosylphosphatidyl inositol-anchor protein phospholipase C activity" EXACT []
 synonym: "GPI-PLC activity" EXACT []
+xref: RHEA:86259
 is_a: GO:0004629 ! C-type glycerophospholipase activity
+property_value: skos:narrowMatch RHEA:86259
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28868" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31078" xsd:anyURI
 created_by: pg
@@ -480945,7 +481966,6 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phosphatidylethanolamine from the cytosolic to the exoplasmic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:10029989]
 synonym: "phosphatidylethanolamine floppase activity (cytosolic to exoplasmic leaflet)" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0140328 ! floppase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -480987,7 +482007,7 @@
 namespace: molecular_function
 def: "Catalysis of the movement of phosphatidylcholine from the exoplasmic to the cytosolic leaflet of a membrane, using energy from the hydrolysis of ATP." [PMID:11870854]
 synonym: "phosphatidylcholine flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0008525 ! phosphatidylcholine transporter activity
+is_a: GO:0008525 ! phosphatidylcholine intramembrane carrier activity
 is_a: GO:0140333 ! glycerophospholipid flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -481039,7 +482059,9 @@
 def: "A supramolecular network that forms mainly through linkages of the short laminin arms between the N-terminal regions of one alpha, one beta, and one gamma chain. The laminin network associates with the collagen network to create the stable, sheet-like structure of the basement membrane ECM." [PMID:1577869, PMID:21421915, PMID:23263632, PMID:28187500]
 synonym: "laminin complex" BROAD []
 is_a: GO:0099080 ! supramolecular complex
+relationship: part_of GO:0140144 ! non-collagenous component of basement membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31053" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31490" xsd:anyURI
 created_by: pg
 creation_date: 2025-12-01T12:38:57Z
 
@@ -481063,7 +482085,7 @@
 synonym: "glucosylceramide flippase activity" NARROW []
 synonym: "glycoceramide flippase activity" EXACT []
 synonym: "glycosylceramide flippase activity (exoplasmic to cytosolic leaflet)" EXACT []
-is_a: GO:0046624 ! sphingolipid transporter activity
+is_a: GO:0046624 ! sphingolipid intramembrane carrier activity
 is_a: GO:0140327 ! flippase activity
 property_value: skos:broadMatch RHEA:66132
 created_by: pg
@@ -481224,7 +482246,7 @@
 namespace: cellular_component
 def: "A ribonucleoprotein granule located in the cytoplasm and the nucleus. GW-bodies minimally contain the Argonaute2 (Ago2) and TNRC6B proteins, together with specific target RNAs." [PMID:16418578, PMID:26930655, PMID:29576456]
 synonym: "GW-body" EXACT []
-is_a: GO:0036464 ! cytoplasmic ribonucleoprotein granule
+is_a: GO:0035770 ! ribonucleoprotein granule
 created_by: pg
 creation_date: 2019-07-25T19:26:48Z
 
@@ -481297,8 +482319,10 @@
 name: ornithine lipid biosynthetic process
 namespace: biological_process
 def: "The chemical reactions and pathways resulting in the formation of ornithine lipids, any of a class of phosphorus-free bacterial membrane lipids with an L-ornithine-derived head group." [PMID:22724388]
+xref: MetaCyc:PWY-6818
 is_a: GO:0008652 ! amino acid biosynthetic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31271" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31510" xsd:anyURI
 created_by: pg
 creation_date: 2026-01-20T12:30:41Z
 
@@ -481368,6 +482392,29 @@
 creation_date: 2026-01-21T10:01:05Z
 
 [Term]
+id: GO:0140378
+name: protein complex scaffold activity
+namespace: molecular_function
+def: "A structural molecule activity of a protein-containing complex component that serves to hold the complex together. Protein complex scaffolds are integral members of complexes." [PMID:41298522]
+synonym: "protein-complex scaffolding activity" EXACT []
+synonym: "protein-containing complex scaffold activity" EXACT []
+is_a: GO:0005198 ! structural molecule activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31444" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T08:05:02Z
+
+[Term]
+id: GO:0140379
+name: amino acid acyltransferase activity
+namespace: molecular_function
+def: "Catalysis of the transfer of an acyl group to an acceptor group on a free amino acid, usually using acetyl-CoA as donor." [GOC:curators]
+synonym: "amino acid N-acetyltransferase activity" NARROW []
+is_a: GO:0016746 ! acyltransferase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31474" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T11:14:33Z
+
+[Term]
 id: GO:0140380
 name: psilocybin biosynthetic process
 namespace: biological_process
@@ -481443,6 +482490,64 @@
 creation_date: 2019-11-20T13:25:55Z
 
 [Term]
+id: GO:0140385
+name: amino acid transaminase activity
+namespace: molecular_function
+def: "Transfer of an amino group from an amino acid to an acceptor, usually a 2-oxo acid." [GOC:curators]
+is_a: GO:0008483 ! transaminase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31140" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-02T13:30:09Z
+
+[Term]
+id: GO:0140386
+name: tRNA (cytidine(4)/adenine(4)-2'-O-ribose)-methyltransferase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: cytidine(4) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(4) in tRNA + S-adenosyl-L-homocysteine + H+. Acts on  cytidine(4) on tRNAPro and tRNAGly(GCC), and on adenosine(4) in tRNAHis." [EC:2.1.1.225, PMID:17242307]
+xref: EC:2.1.1.225
+xref: RHEA:32767
+xref: RHEA:43192
+xref: RHEA:43196
+is_a: GO:0016426 ! tRNA (adenine) methyltransferase activity
+is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
+property_value: skos:exactMatch EC:2.1.1.225
+property_value: skos:narrowMatch RHEA:32767
+property_value: skos:narrowMatch RHEA:43192
+property_value: skos:narrowMatch RHEA:43196
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31521" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-06T12:44:45Z
+
+[Term]
+id: GO:0140387
+name: negative regulation of L-glutamate biosynthetic process
+namespace: biological_process
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of L-glutamate biosynthetic process." [PMID:16899818, PMID:29649217]
+synonym: "negative regulation of glutamate biosynthetic process" BROAD []
+synonym: "negative regulation of L-glutamate biosynthesis" BROAD []
+is_a: GO:2000212 ! negative regulation of glutamate metabolic process
+is_a: GO:2000283 ! negative regulation of amino acid biosynthetic process
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: negatively_regulates GO:0097054 ! L-glutamate biosynthetic process
+relationship: negatively_regulates GO:0097054 ! L-glutamate biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31582" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-17T08:55:47Z
+
+[Term]
+id: GO:0140388
+name: protein translocation chaperone activity
+namespace: molecular_function
+def: "Binding to the exposed unfolded portion of a polypeptide as it emerges from a membrane translocation channel/pore to drive its unidirectional transport across a membrane. Iterative cycles of ATP-dependent binding/rebinding by the chaperone prevents back sliding and drive vectorial transport." [PMID:11733493]
+synonym: "import motor chaperone activity" RELATED []
+synonym: "translocation ratchet activity" RELATED []
+is_a: GO:0140597 ! protein carrier activity
+is_a: GO:0140657 ! ATP-dependent activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31580" xsd:anyURI
+created_by: pg
+creation_date: 2026-02-18T12:58:22Z
+
+[Term]
 id: GO:0140393
 name: norsolorinic acid ketoreductase activity
 namespace: molecular_function
@@ -482196,11 +483301,13 @@
 
 [Term]
 id: GO:0140462
-name: pericentric heterochromatin organization
+name: obsolete pericentric heterochromatin organization
 namespace: biological_process
-def: "The organization of chromatin into heterochromatin at the pericentric region of a chromosome." [PMID:26744419]
-is_a: GO:0031507 ! heterochromatin formation
+def: "OBSOLETE. The organization of chromatin into heterochromatin at the pericentric region of a chromosome." [PMID:26744419]
+comment: The reason for obsoletion is that the term is not clearly defined and usage has been inconsistent.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/19396" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31331" xsd:anyURI
+is_obsolete: true
 created_by: pg
 creation_date: 2020-05-06T07:47:09Z
 
@@ -483074,6 +484181,7 @@
 name: defense response to symbiont
 namespace: biological_process
 def: "Reactions triggered in response to the presence of a symbiont that act to protect or prevent damage to the host." [GOC:pg]
+xref: Reactome:R-HSA-9953170 "GBP-mediated host defense"
 is_a: GO:0098542 ! defense response to other organism
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20261" xsd:anyURI
 created_by: pg
@@ -483228,13 +484336,15 @@
 name: membrane protein dislocase activity
 namespace: molecular_function
 def: "The activity of removing a protein from a membrane, by binding to a transmembrane helical fragment of a tail-anchored protein and releasing it from the the hydrophobic region of one or both lipid bilayers. The reaction is driven by ATP hydrolysis." [PMID:24821790, PMID:28712723, PMID:32973005]
-synonym: "transmembrane helix dislocase\ntransmembrane protein dislocase activity" RELATED []
+synonym: "transmembrane helix dislocase" RELATED []
+synonym: "transmembrane protein dislocase activity" RELATED []
 xref: RHEA:66168
-is_a: GO:0140318 ! protein transporter activity
+is_a: GO:0140597 ! protein carrier activity
 is_a: GO:0140657 ! ATP-dependent activity
 property_value: skos:narrowMatch RHEA:66168
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20237" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22209" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30369" xsd:anyURI
 created_by: pg
 creation_date: 2020-12-15T07:13:21Z
 
@@ -483366,6 +484476,10 @@
 comment: Note that this enzyme transports Cu(+) or Ag(+), and cannot transport the divalent ions, contrary to EC:7.2.2.9, which mainly transports the divalent copper ion (source: EC:7.2.2.8).
 xref: EC:7.2.2.8
 xref: MetaCyc:RXN-14455
+xref: Reactome:R-HSA-3697838 "ATP7A transfers Cu from ATOX1 to SOD3"
+xref: Reactome:R-HSA-6803545 "ATP7A transports cytosolic Cu2+ to phagosomal lumen"
+xref: Reactome:R-HSA-936802 "ATP7A transports cytosolic Cu1+ to extracellular region"
+xref: Reactome:R-HSA-936895 "ATP7B transports cytosolic Cu1+ to Golgi lumen"
 xref: RHEA:25792
 is_a: GO:0005375 ! copper ion transmembrane transporter activity
 is_a: GO:0015662 ! P-type ion transporter activity
@@ -483549,16 +484663,16 @@
 
 [Term]
 id: GO:0140597
-name: protein carrier chaperone
+name: protein carrier activity
 namespace: molecular_function
-def: "Binding to and carrying a protein between two different cellular components by moving along with the target protein." [PMID:7628437]
-synonym: "protein carrier activity" EXACT []
+def: "Binding to and carrying a protein to an acceptor molecule or to a specific location by moving along with the target protein." [PMID:7628437]
+synonym: "protein carrier chaperone" EXACT []
 synonym: "protein chaperone" BROAD []
 is_a: GO:0140104 ! molecular carrier activity
-relationship: has_part GO:0005515 ! protein binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22127" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22356" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31601" xsd:anyURI
 created_by: pg
 creation_date: 2021-02-10T06:16:16Z
 
@@ -483567,7 +484681,7 @@
 name: lipoprotein carrier activity
 namespace: molecular_function
 def: "Binding to and carrying a lipoprotein between two different cellular locations by moving along with the target lipoprotein." [PMID:7628437]
-is_a: GO:0140597 ! protein carrier chaperone
+is_a: GO:0140597 ! protein carrier activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15917" xsd:anyURI
 created_by: pg
 creation_date: 2021-02-10T06:16:26Z
@@ -483732,6 +484846,8 @@
 synonym: "monoiodotyrosine deiodinase activity" RELATED []
 synonym: "tyrosine iodinase activity" RELATED []
 xref: EC:1.21.1.1
+xref: Reactome:R-HSA-209921 "Monoiodinated tyrosine can be deiodinated"
+xref: Reactome:R-HSA-209960 "Diiodinated tyrosine can be deiodinated"
 xref: RHEA:32479
 is_a: GO:0046992 ! oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
 property_value: skos:exactMatch EC:1.21.1.1
@@ -484551,6 +485667,7 @@
 namespace: molecular_function
 def: "A molecular function regulator that activates or increases the activity of its target via non-covalent binding that does not result in covalent modification to the target." [GOC:curators]
 comment: This term should only be used in cases when the regulator directly interacts with the enzyme, but does not result in a covalent modification.
+xref: Reactome:R-HSA-8866941 "MID1IP1 binds THRSP"
 is_a: GO:0098772 ! molecular function regulator activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/21782" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25165" xsd:anyURI
@@ -484996,7 +486113,7 @@
 synonym: "histone carrier activity" EXACT []
 synonym: "nucleosome remodeling activity" RELATED []
 xref: Reactome:R-HSA-9822185 "HIRA and NPM2 assemble H3.3-containing nucleosomes on paternal DNA"
-is_a: GO:0140597 ! protein carrier chaperone
+is_a: GO:0140597 ! protein carrier activity
 relationship: has_part GO:0042393 ! histone binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22126" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22129" xsd:anyURI
@@ -485352,8 +486469,9 @@
 id: GO:0140740
 name: ADP-riboxanase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: L-arginyl-[protein] + NAD+ = ADP-riboxanated L-argininyl-[protein] + H+ + NH4(+) + nicotinamide." [PMID:34671164, RHEA:69500]
+def: "Catalysis of the reaction: L-arginyl-[protein] + NAD+ = ADP-riboxanated L-argininyl-[protein] + H+ + NH4+ + nicotinamide." [PMID:34671164, RHEA:69500]
 comment: Note that this activity has two steps: a transfer of an ADP-ribose group, followed by the elimination of an ammonia group.
+xref: Reactome:R-HSA-9956624 "OspC3 ADP-riboxanates CASP4"
 xref: RHEA:69500
 xref: RHEA:75883
 xref: RHEA:75887
@@ -485588,14 +486706,16 @@
 id: GO:0140759
 name: histone H3K56 methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 56) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 56). This reaction is the addition of a methyl group to the lysine residue at position 56 of the histone H3 protein." [PMID:23451023, PMID:28743002]
+def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 56) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 56). This reaction is the addition of a methyl group to the lysine residue at position 56 of the histone H3 protein." [PMID:23451023, PMID:28743002, RHEA:85519]
 comment: Comment: Note that the residue position corresponds to the canonical human H3 histone (UniProtKB:P84243); this residue is conserved across all eukaryotes. Residue 1 is the first residue following removal of the initiating Methionine (Met). Note that each histone is encoded by multiple genes, and sequences may vary across different genes within an organism.
 synonym: "histone H3K56 methylase activity" EXACT []
 synonym: "histone lysine N-methyltransferase activity (H3-K56 specific)" EXACT []
 synonym: "histone methyltransferase activity (H3-K56 specific)" EXACT []
 synonym: "histone-H3K56 methyltransferase activity" EXACT []
+xref: RHEA:85519
 is_a: GO:0016279 ! protein-lysine N-methyltransferase activity
 is_a: GO:0140938 ! histone H3 methyltransferase activity
+property_value: skos:exactMatch RHEA:85519
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22757" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24337" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28520" xsd:anyURI
@@ -485615,8 +486735,12 @@
 synonym: "histone H3-tri/dimethyl-lysine-56 demethylase activity" EXACT []
 synonym: "histone H3K56me2 demethylase activity" RELATED []
 synonym: "histone H3K56me3 demethylase activity" RELATED []
+xref: RHEA:85927
+xref: RHEA:85931
 is_a: GO:0016706 ! 2-oxoglutarate-dependent dioxygenase activity
 is_a: GO:0141052 ! histone H3 demethylase activity
+property_value: skos:narrowMatch RHEA:85927
+property_value: skos:narrowMatch RHEA:85931
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22757" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29636" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30578" xsd:anyURI
@@ -485724,6 +486848,7 @@
 synonym: "protein substrate chaperone activity" RELATED []
 synonym: "protein-substrate adaptor activity" BROAD []
 xref: Reactome:R-HSA-202137 "AKT1 binds eNOS complex via HSP90"
+xref: Reactome:R-HSA-5653886 "B4GALT1 binds LALBA"
 is_a: GO:0030674 ! protein-macromolecule adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22972" xsd:anyURI
 created_by: pg
@@ -486533,6 +487658,7 @@
 name: lactoperoxidase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2 a phenolic donor + H2O2 = 2 a phenolic radical donor + 2 H2O." [PMID:19339248, RHEA:56136]
+synonym: "heme peroxidase activity" RELATED []
 synonym: "peroxidase activity" BROAD []
 xref: EC:1.11.1.7
 xref: MetaCyc:PEROXID-RXN
@@ -487690,6 +488816,11 @@
 name: halogenase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: R-CH + a halogen + oxygen donor = R-C-halogen +H2O." [PMID:28466644, PMID:34368824]
+xref: Reactome:R-HSA-209815 "Tyrosine is monoiodinated"
+xref: Reactome:R-HSA-209840 "Two DITs combine to form thyroxine"
+xref: Reactome:R-HSA-209925 "DIT and MIT combine to form triiodothyronine"
+xref: Reactome:R-HSA-209973 "Tyrosine is diiodinated"
+xref: Reactome:R-HSA-350901 "Iodide is organified"
 is_a: GO:0016705 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23587" xsd:anyURI
 created_by: pg
@@ -490001,6 +491132,7 @@
 namespace: molecular_function
 def: "Binding to and responding, e.g. by conformational change, to changes in the cellular level of glucose." [PMID:17470517]
 synonym: "glucose sensing activity" EXACT []
+xref: Reactome:R-HSA-170810 "nucleoplasmic GCK1:GCKR complex => glucokinase (GCK1) + glucokinase regulatory protein (GCKR)"
 is_a: GO:0140299 ! molecular sensor activity
 relationship: has_part GO:0005536 ! D-glucose binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25267" xsd:anyURI
@@ -490117,10 +491249,11 @@
 
 [Term]
 id: GO:0141098
-name: tRNA (cytidine(34)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(34) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(34) in tRNA + H+ + S-adenosyl-L-homocysteine." [PMID:31943105, RHEA:43084]
 synonym: "tRNA (cytidine 34-2'-O)-methyltransferase activity" EXACT []
+synonym: "tRNA (cytidine(34)-2'-O)-methyltransferase activity" EXACT []
 xref: MetaCyc:RXN-11860
 xref: RHEA:43084
 is_a: GO:0016427 ! tRNA (cytidine) methyltransferase activity
@@ -490134,9 +491267,10 @@
 
 [Term]
 id: GO:0141100
-name: tRNA (guanine(18)-2'-O)-methyltransferase activity
+name: tRNA (guanine(18)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine 18 in tRNA= S-adenosyl-L-homocysteine + 2'-O-methylguanosine 18 in tRNA + H+." [PMID:25404562]
+synonym: "tRNA (guanine(18)-2'-O-)-methyltransferase activity" EXACT []
 synonym: "tRNA (guanosine(18)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.34
 xref: MetaCyc:2.1.1.34-RXN
@@ -490151,10 +491285,11 @@
 
 [Term]
 id: GO:0141101
-name: tRNA(Ser) (uridine(44)-2'-O-)-methyltransferase activity
+name: tRNA(Ser) (uridine(44)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: Catalysis of the reaction: S-adenosyl-L-methionine + uridine44 in tRNASer = 2'-O-methyluridine44 in tRNASer + H+ + S-adenosyl-L-homocysteine." [PMID:18025252, RHEA:43100]
 synonym: "tRNA(Ser) (uracil(44)-2'-O-)-methyltransferase activity" EXACT []
+synonym: "tRNA(Ser) (uridine(44)-2'-O-)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.211
 xref: MetaCyc:RXN-12368
 xref: Reactome:R-HSA-6788707 "TRMT44 2'-O-methylates uridine-44 in tRNA(Ser)"
@@ -490169,9 +491304,10 @@
 
 [Term]
 id: GO:0141102
-name: tRNA (5-carboxymethylaminomethyluridine(34)-2'-O)-methyltransferase activity
+name: tRNA (5-carboxymethylaminomethyluridine(34)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 5-carboxymethylaminomethyluridine34 in tRNA(Leu) + S-adenosyl-L-methionine = 5-carboxymethylaminomethyl-2'-O-methyluridine34 in tRNA(Leu) + H+ + S-adenosyl-L-homocysteine." [PMID:20855540, RHEA:43088]
+synonym: "tRNA (5-carboxymethylaminomethyluridine(34)-2'-O)-methyltransferase activity" EXACT []
 synonym: "tRNA (cytidine(34)/5-carboxymethylaminomethyluridine(34)-2'-O)- methyltransferase activity" EXACT []
 xref: RHEA:43088
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -490360,6 +491496,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: Fe(II)-heme b-[protein] + nitric oxide + O2 = Fe(III)-heme b-[protein] + nitrate." [PMID:20511233]
 comment: Note that this activity is similar to nitric oxide dioxygenase activity ; GO:0008941, but GO:000894 uses NADPH as the electron donor.
+xref: Reactome:R-HSA-5340226 "CYGB dioxygenates NO"
 xref: RHEA:78091
 is_a: GO:0016675 ! oxidoreductase activity, acting on a heme group of donors
 property_value: skos:exactMatch RHEA:78091
@@ -490836,6 +491973,7 @@
 synonym: "cAMP/PKA signaling cassette" EXACT []
 synonym: "PKA signaling" BROAD []
 synonym: "protein kinase A signaling" BROAD []
+xref: Reactome:R-HSA-163358 "PKA-mediated phosphorylation of key metabolic factors"
 is_a: GO:0141124 ! intracellular signaling cassette
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26899" xsd:anyURI
 created_by: pg
@@ -492485,9 +493623,10 @@
 id: GO:0150103
 name: reactive gliosis
 namespace: biological_process
-def: "A neuroinflammatory response, occurring over several days, during which glial cells undergo nonspecific reactive changes in response to damage to the central nervous system (CNS); typically involves the proliferation or hypertrophy of different types of glial cells." [GOC:aruk, GOC:bc, PMID:24462092]
+def: "A coordinated response of central nervous system (CNS) glial cells-including astrocytes, microglia, and NG2 glia-to injury, infection, or disease, characterized by changes in gene expression, morphology, proliferation, and extracellular matrix production. Reactive gliosis can range from transient glial activation to the formation of a persistent glial scar border." [GOC:aruk, GOC:bc, PMID:20012068, PMID:24267886, PMID:24462092, PMID:27049947]
 synonym: "gliosis" BROAD []
 is_a: GO:0150076 ! neuroinflammatory response
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30886" xsd:anyURI
 created_by: bc
 creation_date: 2019-05-21T12:37:31Z
 
@@ -493982,8 +495121,9 @@
 def: "The binding activity of a molecule that brings together an ubiquitin ligase and an ubiquitin ligase-substrate adaptor, permitting those molecules to function in a coordinated way." [PMID:27664236]
 synonym: "core protein activity for the CUL-RING ubiquitin ligase complex" NARROW []
 synonym: "core protein activity for the Cullin-RING ubiquitin ligase complex" NARROW []
-is_a: GO:0030674 ! protein-macromolecule adaptor activity
+is_a: GO:0140378 ! protein complex scaffold activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24591" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31499" xsd:anyURI
 created_by: rynl
 creation_date: 2022-12-21T22:41:32Z
 
@@ -494013,6 +495153,8 @@
 name: non-canonical inflammasome complex assembly
 namespace: biological_process
 def: "The aggregation, arrangement and bonding together of a set of components to form a non-canonical inflammasome complex." [PMID:27291964]
+xref: Reactome:R-HSA-9948001 "CASP4 inflammasome assembly"
+xref: Reactome:R-HSA-9948011 "CASP5 inflammasome assembly"
 is_a: GO:0065003 ! protein-containing complex assembly
 created_by: rynl
 creation_date: 2023-01-25T22:38:01Z
@@ -494216,7 +495358,7 @@
 id: GO:0160096
 name: nematode pharyngeal muscle development
 namespace: biological_process
-def: "The process whose specific outcome is the progression of the nematode pharyngeal muscle over time, from its formation to the mature structure." [GOC:go_curators]
+def: "The process whose specific outcome is the progression of the nematode pharyngeal muscle over time, from its formation to the mature structure." [GOC:curators]
 is_a: GO:0007517 ! muscle organ development
 relationship: part_of GO:0160094 ! nematode pharynx development
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/18285" xsd:anyURI
@@ -495446,6 +496588,7 @@
 def: "Catalysis of the reaction: a di-trans,poly-cis-polyprenol + NAD+ = a di-trans,poly-cis-polyprenal + NADH + H+." [PMID:38821050, RHEA:80719]
 synonym: "polyprenol dehydrogenase activity" BROAD []
 xref: MetaCyc:RXN-24567
+xref: Reactome:R-HSA-9937298 "DHRSX oxidizes polyprenol to polyprenal"
 xref: RHEA:80719
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:broadMatch EC:1.1.1.441
@@ -495478,6 +496621,7 @@
 def: "Catalysis of the reaction: ditrans,polycis-polyprenal+ H+ + NADPH = ditrans,polycis-dolichal+ NADP." [PMID:38821050, RHEA:80727]
 xref: EC:1.3.1.94
 xref: MetaCyc:RXN-9971
+xref: Reactome:R-HSA-9937295 "SRD5A3 reduces polyprenal to dolichal"
 xref: RHEA:80727
 is_a: GO:0016628 ! oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
 property_value: skos:exactMatch EC:1.3.1.94
@@ -495564,10 +496708,11 @@
 
 [Term]
 id: GO:0160206
-name: tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase activity
+name: tRNA (cytidine(32)/uridine(32)-2'-O-ribose)-methyltransferase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: cytidine(32)/uridine(32) in tRNA + S-adenosyl-L-methionine = 2'-O-methylcytidine(32)/2'-O-methyluridine(32) in tRNA + H+ + S-adenosyl-L-homocysteine." [EC:2.1.1.200]
 synonym: "TrMet(Xm32) activity" EXACT []
+synonym: "tRNA (cytidine(32)/uridine(32)-2'-O)-methyltransferase activity" EXACT []
 xref: EC:2.1.1.200
 xref: RHEA:42936
 is_a: GO:0016300 ! tRNA (uridine) methyltransferase activity
@@ -496050,6 +497195,7 @@
 name: NADPH-iron-sulfur [2Fe-2S] protein oxidoreductase activity
 namespace: molecular_function
 def: "Catalysis of the reaction: 2 oxidized [2Fe-2S]-[protein] + NADPH = 2 reduced [2Fe-2S]-[protein] + NADP+ + H+." [RHEA:67716]
+xref: Reactome:R-HSA-2564824 "NADPH reduces NDOR1:CIAPIN1"
 xref: RHEA:67716
 is_a: GO:0016731 ! oxidoreductase activity, acting on iron-sulfur proteins as donors, NAD or NADP as acceptor
 property_value: skos:exactMatch RHEA:67716
@@ -496358,9 +497504,11 @@
 id: GO:0160272
 name: RNA 2',3'-cyclic phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 3'-end 2',3'-cyclophospho-ribonucleotide-RNA + 2 H2O = a 3'-end ribonucleotide-RNA + phosphate + H+." [PMID:32732418]
+def: "Catalysis of the reaction: a 3'-end 2',3'-cyclophospho-ribonucleotide-RNA + 2 H2O = a 3'-end ribonucleotide-RNA + phosphate + H+." [PMID:32732418, RHEA:85011]
+xref: RHEA:85011
 is_a: GO:0016791 ! phosphatase activity
 is_a: GO:0140098 ! catalytic activity, acting on RNA
+property_value: skos:exactMatch RHEA:85011
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28001" xsd:anyURI
 created_by: rynl
 creation_date: 2025-05-28T05:44:27Z
@@ -496369,11 +497517,13 @@
 id: GO:0160273
 name: RNA 2'-phosphatase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: H2O + a 3'-end 2'-phospho-ribonucleotide-RNA = a 3'-end ribonucleotide-RNA + phosphate." [PMID:36180430]
+def: "Catalysis of the reaction: H2O + a 3'-end 2'-phospho-ribonucleotide-RNA = a 3'-end ribonucleotide-RNA + phosphate." [PMID:36180430, RHEA:85191]
 synonym: "Ribonucleic acid 2'-phosphatase activity" EXACT []
 synonym: "RNA 2'-Pase activity" EXACT []
+xref: RHEA:85191
 is_a: GO:0016791 ! phosphatase activity
 is_a: GO:0140098 ! catalytic activity, acting on RNA
+property_value: skos:exactMatch RHEA:85191
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28002" xsd:anyURI
 created_by: rynl
 creation_date: 2025-05-28T16:38:49Z
@@ -496420,12 +497570,13 @@
 id: GO:0160277
 name: lumenal side of cis-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi cisternae membrane that faces the Golgi lumen and is involved in glycosylation and lumen-facing cargo recognition." [PMID:23913272, PMID:34597626]
+def: "The leaflet of the cis-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where glycosylation and lumen-facing cargo recognition occur." [PMID:23913272, PMID:34597626]
 synonym: "lumenal face of cis-Golgi cisternae membrane" EXACT []
 synonym: "lumenal leaflet of cis-Golgi cisternae membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990674 ! Golgi cis cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30405" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-21T00:29:16Z
 
@@ -496433,12 +497584,13 @@
 id: GO:0160278
 name: cytoplasmic side of cis-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the cis-Golgi cisternae membrane that faces the cytoplasm and is the site of interactions with cytosolic proteins, including those involved in vesicle budding, membrane tethering, and lipid or protein trafficking." [PMID:10922460, PMID:34597626, PMID:39331042]
+def: "The leaflet of the cis-Golgi cisternae membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins, including those involved in vesicle budding, membrane tethering, and lipid or protein trafficking." [PMID:10922460, PMID:34597626, PMID:39331042]
 synonym: "cytoplasmic face of cis-Golgi cisternae membrane" EXACT []
 synonym: "cytoplasmic leaflet of cis-Golgi cisternae membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990674 ! Golgi cis cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30406" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T17:19:30Z
 
@@ -496446,12 +497598,13 @@
 id: GO:0160279
 name: cytoplasmic side of medial-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the medial-Golgi cisternae that faces the cytoplasm participates in interactions with cytosolic proteins involved in membrane trafficking, vesicle docking, and cargo sorting." [PMID:28777890, PMID:34597626, PMID:39658747]
+def: "The leaflet of the medial-Golgi cisternae membrane that faces the cytoplasm,including any protein embedded in, attached to, or peripherally associated with it. This is the site of interactions with cytosolic proteins involved in membrane trafficking, vesicle docking, and cargo sorting." [PMID:28777890, PMID:34597626, PMID:39658747]
 synonym: "cytoplasmic face of Golgi medial cisterna membrane" EXACT []
 synonym: "cytoplasmic leaflet of Golgi medial cisterna membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990675 ! Golgi medial cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30407" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T21:12:40Z
 
@@ -496459,12 +497612,13 @@
 id: GO:0160280
 name: lumenal side of medial-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the medial-Golgi cisternae that directly contacts the Golgi lumen, hosts glycosyltransferases and enzymes to modify N-linked oligosaccharides by trimming mannose and adding N-acetylglucosamine." [PMID:28777890, PMID:34597626, PMID:36658747, PMID:39331042]
+def: "The leaflet of the medial-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where protein glycosylation occur." [PMID:28777890, PMID:34597626, PMID:36658747, PMID:39331042]
 synonym: "lumenal face of medial-Golgi cisterna membrane" EXACT []
 synonym: "lumenal leaflet of Golgi medial cisterna membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990675 ! Golgi medial cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30418" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T21:36:28Z
 
@@ -496472,13 +497626,14 @@
 id: GO:0160281
 name: cytoplasmic side of trans-Golgi network membrane
 namespace: cellular_component
-def: "The leaflet of the membrane bilayer of the trans-Golgi network faces the cytoplasm and is crucial for lipid and protein sorting and trafficking, as it interacts with cytoplasmic proteins involved in vesicle formation and targeting." [PMID:10922460, PMID:23913272, PMID:34597626]
+def: "The leaflet of the trans-Golgi network membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site of interaction where cytoplasmic proteins interact with proteins involved in vesicle formation and targeting." [PMID:10922460, PMID:23913272, PMID:34597626]
 synonym: "cytoplasmic face of trans-Golgi network membrane" EXACT []
 synonym: "cytoplasmic leaflet of trans-Golgi network membrane" EXACT []
 synonym: "cytoplasmic side of trans-Golgi network" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:0032588 ! trans-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30427" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-22T23:41:33Z
 
@@ -496486,13 +497641,14 @@
 id: GO:0160282
 name: lumenal side of trans-Golgi network membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi network membrane that faces the Golgi lumen." [PMID:23913272, PMID:34597626, PMID:38307322]
+def: "The leaflet of the trans-Golgi network membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it." [PMID:23913272, PMID:34597626, PMID:38307322]
 synonym: "lumenal face of trans-Golgi network membrane" EXACT []
 synonym: "lumenal leaflet of trans-Golgi network membrane" EXACT []
 synonym: "lumenal side of trans-Golgi network" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:0032588 ! trans-Golgi network membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30428" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-23T00:11:57Z
 
@@ -496500,9 +497656,11 @@
 id: GO:0160283
 name: queuine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of queuine from one side of a membrane to the other according to the reaction: queuine(out) = queuine(in)." [PMID:40526720]
+def: "Enables the transfer of queuine from one side of a membrane to the other according to the reaction: queuine(out) = queuine(in)." [PMID:40526720, RHEA:85679]
+xref: RHEA:85679
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0160284 ! queuine import across plasma membrane
+property_value: skos:exactMatch RHEA:85679
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30470" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-24T17:27:38Z
@@ -496522,9 +497680,11 @@
 id: GO:0160286
 name: queuosine transmembrane transporter activity
 namespace: molecular_function
-def: "Enables the transfer of queuosine from one side of a membrane to the other according to the reaction: queuosine(out) = queuosine(in)." [PMID:40526720]
+def: "Enables the transfer of queuosine from one side of a membrane to the other according to the reaction: queuosine(out) = queuosine(in)." [PMID:40526720, RHEA:85683]
+xref: RHEA:85683
 is_a: GO:0005337 ! nucleoside transmembrane transporter activity
 relationship: part_of GO:0160287 ! queuosine import across plasma membrane
+property_value: skos:exactMatch RHEA:85683
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30470" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T00:30:53Z
@@ -496544,13 +497704,14 @@
 id: GO:0160288
 name: cytoplasmic side of trans-Golgi cisterna membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi cisternae that faces the cytoplasm. It is the site of protein and lipid interaction, vesicle formation, and cargo sorting towards post-Golgi destinations such as endosomes, lysosomes, and the plasma membrane." [PMID:23913272, PMID:34597626, PMID:37566051]
+def: "The leaflet of the trans-Golgi cisternae membrane that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it. This is the site where vesicle formation, and cargo sorting towards post-Golgi destinations such as endosomes, lysosomes and the plasma membrane occur." [PMID:23913272, PMID:34597626, PMID:37566051]
 synonym: "cytoplasmic face of Golgi trans cisterna membrane" EXACT []
 synonym: "cytoplasmic leaflet of Golgi trans cisterna membrane" EXACT []
 synonym: "cytoplasmic side of Golgi trans cisternae membrane" EXACT []
 is_a: GO:0098562 ! cytoplasmic side of membrane
 relationship: part_of GO:1990676 ! Golgi trans cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30509" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T05:51:59Z
 
@@ -496558,13 +497719,14 @@
 id: GO:0160289
 name: lumenal side of trans-Golgi cisternae membrane
 namespace: cellular_component
-def: "The membrane leaflet of the trans-Golgi cisternae membrane that faces the Golgi lumen. It is involved in the final steps of glycan processing, proteolytic modification, and cargo maturation prior to sorting at the trans-Golgi network." [PMID:34597626, PMID:38307322, PMID:6121819]
+def: "The leaflet of the trans-Golgi cisternae membrane that faces the Golgi lumen, including any protein embedded in, attached to, or peripherally associated with it. This is the site where the final steps of glycan processing, proteolytic modification, and cargo maturation prior to sorting at the trans-Golgi network occur." [PMID:34597626, PMID:38307322, PMID:6121819]
 synonym: "lumenal face of trans-Golgi cisternae membrane" EXACT []
 synonym: "lumenal leaflet of trans-Golgi cisternae membrane" EXACT []
 synonym: "lumenal side of Golgi trans cisternae membrane" EXACT []
 is_a: GO:0098576 ! lumenal side of membrane
 relationship: part_of GO:1990676 ! Golgi trans cisterna membrane
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30510" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: rynl
 creation_date: 2025-06-27T06:11:48Z
 
@@ -496584,10 +497746,12 @@
 id: GO:0160291
 name: phosphatidylinositol-4-phosphate-cholesterol exchange activity
 namespace: molecular_function
-def: "Catalysis of the reaction: a 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(out) + cholesterol(in) = 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(in) + cholesterol(out). This reaction results in the exchange of cholesterol for phosphatidylinositol-4-phosphate (PI(4)P) between membranes." [PMID:24209621, PMID:29596003]
+def: "Catalysis of the reaction: a 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(out) + cholesterol(in) = 1,2-diacyl-sn-glycero-3-phospho-(1D-myo-inositol 4-phosphate)(in) + cholesterol(out). This reaction results in the exchange of cholesterol for phosphatidylinositol-4-phosphate (PI(4)P) between membranes." [PMID:24209621, PMID:29596003, RHEA:84179]
+xref: RHEA:84179
 is_a: GO:0008526 ! phosphatidylinositol transfer activity
 is_a: GO:0120020 ! cholesterol transfer activity
 is_a: GO:7770011 ! lipid exchange activity
+property_value: skos:exactMatch RHEA:84179
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30432" xsd:anyURI
 created_by: rynl
@@ -496663,9 +497827,13 @@
 id: GO:0160298
 name: pyridinol guanylyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + GTP + H+ = guanylylpyridinol + diphosphate." [PMID:24249552]
+def: "Catalysis of the reaction: 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + GTP + H+ = guanylylpyridinol + diphosphate." [PMID:24249552, RHEA:85479]
+xref: EC:2.7.7.109
 xref: KEGG_REACTION:R13518
+xref: RHEA:85479
 is_a: GO:0070568 ! guanylyltransferase activity
+property_value: skos:exactMatch EC:2.7.7.109
+property_value: skos:exactMatch RHEA:85479
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30718" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T00:46:39Z
@@ -496685,9 +497853,13 @@
 id: GO:0160301
 name: guanylylpyridinol adenylase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: guanylylpyridinol + ATP + H+ = guanylylpyridinol-AMP + diphosphate." [PMID:25882909]
+def: "Catalysis of the reaction: guanylylpyridinol + ATP + H+ = guanylylpyridinol-AMP + diphosphate." [PMID:25882909, RHEA:85487]
+xref: EC:2.7.7.110
 xref: KEGG_REACTION:R13520
+xref: RHEA:85487
 is_a: GO:0016779 ! nucleotidyltransferase activity
+property_value: skos:exactMatch EC:2.7.7.110
+property_value: skos:exactMatch RHEA:85487
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30720" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T22:49:37Z
@@ -496696,9 +497868,11 @@
 id: GO:0160302
 name: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol 3-C-methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol + S-adenosyl-L-methionine = 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + S-adenosyl-L-homocysteine + H+." [PMID:27391308]
+def: "Catalysis of the reaction: 6-carboxymethyl-5-methyl-4-hydroxypyridin-2-ol + S-adenosyl-L-methionine = 6-carboxymethyl-3,5-dimethyl-4-hydroxypyridin-2-ol + S-adenosyl-L-homocysteine + H+." [PMID:27391308, RHEA:85483]
 xref: KEGG_REACTION:R13499
+xref: RHEA:85483
 is_a: GO:0008168 ! methyltransferase activity
+property_value: skos:exactMatch RHEA:85483
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30721" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-21T23:15:13Z
@@ -496721,9 +497895,11 @@
 id: GO:0160304
 name: 4-vinylphenol methyltransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: 4-vinylphenol + S-adenosyl-L-methionine = 4-vinylanisole + S-adenosyl-L-homocysteine + H+." [PMID:40562929]
+def: "Catalysis of the reaction: 4-vinylphenol + S-adenosyl-L-methionine = 4-vinylanisole + S-adenosyl-L-homocysteine + H+." [PMID:40562929, RHEA:86075]
+xref: RHEA:86075
 is_a: GO:0008171 ! O-methyltransferase activity
 is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
+property_value: skos:exactMatch RHEA:86075
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30708" xsd:anyURI
 created_by: rynl
 creation_date: 2025-08-23T00:16:21Z
@@ -496890,17 +498066,13 @@
 
 [Term]
 id: GO:0160320
-name: protein-tyrosine iodotransferase activity
+name: obsolete protein-tyrosine iodotransferase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O and [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O." [PMID:36537574, PMID:8651700]
-xref: RHEA:48956
-xref: RHEA:48960
-is_a: GO:0140905 ! haloperoxidase activity
-property_value: skos:narrowMatch RHEA:48956
-property_value: skos:narrowMatch RHEA:48960
+def: "OBSOLETE. Catalysis of the reaction: [thyroglobulin]-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3-iodo-L-tyrosine + 2 H2O and [thyroglobulin]-3-iodo-L-tyrosine + iodide + H2O2 + H+ = [thyroglobulin]-3,5-diiodo-L-tyrosine + 2 H2O." [PMID:36537574, PMID:8651700]
+comment: The reason for obsoletion is that the reactions covered by this term (RHEA:48956 and RHEA:48960) are now subsumed by the updated GO:0004447 (iodide peroxidase activity), which covers all 5 thyroid peroxidase reactions.
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30596" xsd:anyURI
-created_by: rynl
-creation_date: 2026-01-21T23:56:40Z
+is_obsolete: true
+replaced_by: GO:0004447
 
 [Term]
 id: GO:0170001
@@ -496970,6 +498142,8 @@
 name: endoplasmic reticulum-lipid droplet tether activity
 namespace: molecular_function
 def: "The binding activity of a molecule that brings together a lipid droplet with an endoplasmic reticulum membrane, via membrane lipid binding, to establish membrane contact sites and mediate exchange and communication." [PMID:35389430]
+xref: Reactome:R-HSA-8857686 "FITM2 binds TAGs"
+xref: Reactome:R-HSA-9942190 "FITM1 binds TAGs"
 is_a: GO:0043495 ! protein-membrane adaptor activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25428" xsd:anyURI
 created_by: ew
@@ -497135,6 +498309,8 @@
 def: "The chemical reactions and pathways involving an L-amino acid." [GOC:edw]
 synonym: "L-alpha-amino acid metabolism" EXACT []
 synonym: "L-amino acid metabolism" EXACT []
+xref: Reactome:R-HSA-8963693 "Aspartate and asparagine metabolism"
+xref: Reactome:R-HSA-8964539 "Glutamate and glutamine metabolism"
 is_a: GO:1901605 ! alpha-amino acid metabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/23268" xsd:anyURI
 created_by: ew
@@ -497278,7 +498454,6 @@
 def: "Enables the transfer of isopentenyl pyrophosphate from one side of a membrane to the other." [GOC:ew, PMID:37813972]
 synonym: "IPP transmembrane transport activity" EXACT []
 synonym: "isopentenyl diphosphate transmembrane transport activity" EXACT []
-is_a: GO:0005548 ! phospholipid transporter activity
 is_a: GO:0015605 ! organophosphate ester transmembrane transporter activity
 is_a: GO:0170055 ! lipid transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26285" xsd:anyURI
@@ -497393,7 +498568,6 @@
 namespace: molecular_function
 def: "Enables the transfer of a lipid from one side of a membrane to the other." [GOC:curators]
 subset: gocheck_do_not_annotate
-is_a: GO:0005319 ! lipid transporter activity
 is_a: GO:0022857 ! transmembrane transporter activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27252" xsd:anyURI
 created_by: ew
@@ -497479,6 +498653,7 @@
 name: transmembrane transport from lysosomal lumen to cytosol
 namespace: biological_process
 def: "The directed movement of a solute from the lysosomal lumen across the lysosomal membrane and into the cytosol." [PMID:26631267]
+synonym: "export from lysosome lumen to cytosol" EXACT []
 is_a: GO:0007041 ! lysosomal transport
 is_a: GO:0034486 ! vacuolar transmembrane transport
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/20070" xsd:anyURI
@@ -497517,9 +498692,7 @@
 def: "The chemical reactions and pathways resulting in the formation of arginine (2-amino-5-guanidinopentanoic acid) via the intermediate compound N-acetyl-L-citrulline." [PMID:16585758]
 synonym: "arginine biosynthesis via N-acetyl intermediates" EXACT []
 synonym: "arginine biosynthesis, N-acetylated pathway" EXACT []
-xref: MetaCyc:PWY-5154
 is_a: GO:0006526 ! L-arginine biosynthetic process
-property_value: skos:exactMatch MetaCyc:PWY-5154
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29602" xsd:anyURI
 created_by: ew
 creation_date: 2025-03-14T17:04:09Z
@@ -497583,6 +498756,17 @@
 creation_date: 2025-08-12T21:56:57Z
 
 [Term]
+id: GO:0170074
+name: RADAR pathway
+namespace: biological_process
+def: "A quality control pathway that degrades peroxisomal matrix protein receptors when the recycling machinery is blocked. When recycling to the cytosol fails, cargo-free receptors accumulate at the peroxisomal membrane, where they are polyubiquitinated and subsequently degraded by the ubiquitin-proteasome system (UPS)." [PMID:16390998, PMID:17011644, PMID:41076631]
+synonym: "Receptor Accumulation and Degradation in the Absence of Recycling" EXACT []
+is_a: GO:0043161 ! proteasome-mediated ubiquitin-dependent protein catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31570" xsd:anyURI
+created_by: ew
+creation_date: 2026-02-17T22:02:27Z
+
+[Term]
 id: GO:0180000
 name: histone methyltransferase inhibitor activity
 namespace: molecular_function
@@ -497729,6 +498913,7 @@
 def: "Any process involved in transcription termination-coupled 3' processing of RNA polymerase II mRNA transcripts by the 3' end cleavage and addition of a poly(A) tail." [PMID:31499460]
 synonym: "cotranscriptional 3' processing of RNA polymerase II mRNA transcripts" EXACT []
 synonym: "mRNA polyadenylation" RELATED []
+xref: Reactome:R-HSA-9770562 "mRNA Polyadenylation"
 is_a: GO:0031124 ! mRNA 3'-end processing
 is_a: GO:0180012 ! co-transcriptional RNA 3'-end processing, cleavage and polyadenylation pathway
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/24921" xsd:anyURI
@@ -497853,7 +499038,7 @@
 namespace: molecular_function
 def: "Catalysis of the reaction: S-adenosyl-L-methionine + U6 snRNA = S-adenosyl-L-homocysteine + U6 containing a 3'-terminal 2'-O-methylnucleotide + H+." [PMID:37403782, RHEA:63088]
 xref: RHEA:63088
-is_a: GO:0090486 ! small RNA 2'-O-methyltransferase activity
+is_a: GO:0090486 ! small RNA 2'-O-ribose methyltransferase activity
 property_value: skos:exactMatch RHEA:63088
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25706" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30193" xsd:anyURI
@@ -498402,6 +499587,21 @@
 creation_date: 2025-06-27T07:22:01Z
 
 [Term]
+id: GO:0180067
+name: terreate biosynthetic process
+namespace: biological_process
+def: "The chemical reactions and pathways resulting in the formation of terreate, a fungal metabolite with ecological, antimicrobial, antiproliferative, and antioxidative activities." [PMID:25265334]
+synonym: "terreic acid biosynthetic process" RELATED []
+is_a: GO:0097176 ! epoxide metabolic process
+is_a: GO:0120255 ! olefinic compound biosynthetic process
+is_a: GO:1901503 ! ether biosynthetic process
+is_a: GO:1901663 ! quinone biosynthetic process
+is_a: GO:1902645 ! tertiary alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31114" xsd:anyURI
+created_by: PomBase:vw
+creation_date: 2026-02-23T10:27:33Z
+
+[Term]
 id: GO:0180068
 name: negative regulation of carbohydrate utilization
 namespace: biological_process
@@ -498412,6 +499612,20 @@
 creation_date: 2026-01-19T11:52:03Z
 
 [Term]
+id: GO:0180069
+name: positive regulation of terreate biosynthetic process
+namespace: biological_process
+def: "Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of terreate." [PMID:25265334]
+is_a: GO:0010566 ! regulation of ketone biosynthetic process
+is_a: GO:1902932 ! positive regulation of alcohol biosynthetic process
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: positively_regulates GO:0180067 ! terreate biosynthetic process
+relationship: positively_regulates GO:0180067 ! terreate biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31114" xsd:anyURI
+created_by: PomBase:vw
+creation_date: 2026-02-23T15:24:17Z
+
+[Term]
 id: GO:0198738
 name: obsolete cell-cell signaling by wnt
 namespace: biological_process
@@ -506319,7 +507533,7 @@
 id: GO:1900371
 name: regulation of purine nucleotide biosynthetic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:go_curators, GOC:TermGenie]
+def: "Any process that modulates the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:curators, GOC:TermGenie]
 synonym: "regulation of purine nucleotide anabolism" EXACT [GOC:TermGenie]
 synonym: "regulation of purine nucleotide biosynthesis" EXACT [GOC:TermGenie]
 synonym: "regulation of purine nucleotide formation" EXACT [GOC:TermGenie]
@@ -506336,7 +507550,7 @@
 id: GO:1900372
 name: negative regulation of purine nucleotide biosynthetic process
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:go_curators, GOC:TermGenie]
+def: "Any process that stops, prevents or reduces the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:curators, GOC:TermGenie]
 synonym: "down regulation of purine nucleotide anabolism" EXACT [GOC:TermGenie]
 synonym: "down regulation of purine nucleotide biosynthesis" EXACT [GOC:TermGenie]
 synonym: "down regulation of purine nucleotide biosynthetic process" EXACT [GOC:TermGenie]
@@ -506374,7 +507588,7 @@
 id: GO:1900373
 name: positive regulation of purine nucleotide biosynthetic process
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:go_curators, GOC:TermGenie]
+def: "Any process that activates or increases the frequency, rate or extent of purine nucleotide biosynthetic processes." [GOC:curators, GOC:TermGenie]
 synonym: "activation of purine nucleotide anabolism" EXACT [GOC:TermGenie]
 synonym: "activation of purine nucleotide biosynthesis" EXACT [GOC:TermGenie]
 synonym: "activation of purine nucleotide biosynthetic process" NARROW [GOC:TermGenie]
@@ -506412,7 +507626,7 @@
 id: GO:1900374
 name: obsolete positive regulation of mating type switching by regulation of transcription from RNA polymerase II promoter
 namespace: biological_process
-def: "OBSOLETE. A regulation of transcription from RNA polymerase II promoter that results in positive regulation of mating type switching." [GOC:go_curators, GOC:TermGenie, PMID:8804308]
+def: "OBSOLETE. A regulation of transcription from RNA polymerase II promoter that results in positive regulation of mating type switching." [GOC:curators, GOC:TermGenie, PMID:8804308]
 comment: This term was obsoleted because it represents a GO-CAM model.
 synonym: "activation of mating type switching by regulation of transcription from RNA polymerase II promoter" NARROW [GOC:TermGenie]
 synonym: "stimulation of mating type switching by regulation of transcription from RNA polymerase II promoter" NARROW [GOC:TermGenie]
@@ -506427,7 +507641,7 @@
 id: GO:1900375
 name: obsolete positive regulation of inositol biosynthetic process by positive regulation of transcription from RNA polymerase II promoter
 namespace: biological_process
-def: "OBSOLETE. A positive regulation of transcription from RNA polymerase II promoter that results in positive regulation of inositol biosynthetic process." [GOC:go_curators, GOC:TermGenie, PMID:2004420]
+def: "OBSOLETE. A positive regulation of transcription from RNA polymerase II promoter that results in positive regulation of inositol biosynthetic process." [GOC:curators, GOC:TermGenie, PMID:2004420]
 comment: This term was obsoleted because it represents a GO-CAM model.
 synonym: "activation of inositol anabolism by positive regulation of transcription from RNA polymerase II promoter" EXACT [GOC:TermGenie]
 synonym: "activation of inositol biosynthesis by positive regulation of transcription from RNA polymerase II promoter" EXACT [GOC:TermGenie]
@@ -507294,12 +508508,14 @@
 name: regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 is_a: GO:0009894 ! regulation of catabolic process
 is_a: GO:0062012 ! regulation of small molecule metabolic process
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0046164 ! alcohol catabolic process
 relationship: regulates GO:0046164 ! alcohol catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28908" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:18Z
 
@@ -507308,6 +508524,7 @@
 name: negative regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of  alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "down-regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "downregulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
@@ -507318,6 +508535,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0046164 ! alcohol catabolic process
 relationship: negatively_regulates GO:0046164 ! alcohol catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:28Z
 
@@ -507326,6 +508544,7 @@
 name: positive regulation of alcohol catabolic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of alcohol catabolic process within a cell." [GOC:TermGenie]
+subset: gocheck_do_not_annotate
 synonym: "activation of cellular alcohol catabolic process" NARROW [GOC:TermGenie]
 synonym: "up regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
 synonym: "up-regulation of cellular alcohol catabolic process" EXACT [GOC:TermGenie]
@@ -507337,6 +508556,7 @@
 intersection_of: positively_regulates GO:0046164 ! alcohol catabolic process
 relationship: positively_regulates GO:0046164 ! alcohol catabolic process
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28908" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: krc
 creation_date: 2012-04-20T10:21:46Z
 
@@ -507432,6 +508652,7 @@
 synonym: "negative regulation of defence response to bacterium" EXACT [GOC:TermGenie]
 synonym: "negative regulation of defense response to bacteria" EXACT [GOC:TermGenie]
 synonym: "negative regulation of defense response to bacterium, incompatible interaction" NARROW []
+xref: Reactome:R-HSA-9968551 "Regulation of GBP-mediated host defense"
 is_a: GO:0002832 ! negative regulation of response to biotic stimulus
 is_a: GO:0031348 ! negative regulation of defense response
 is_a: GO:0032102 ! negative regulation of response to external stimulus
@@ -515476,14 +516697,15 @@
 
 [Term]
 id: GO:1900829
-name: D-tyrosine catabolic process
+name: obsolete D-tyrosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-tyrosine." [GOC:TermGenie, PMID:10766779]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-tyrosine." [GOC:TermGenie, PMID:10766779]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-tyrosine.
 synonym: "D-tyrosine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-tyrosine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-tyrosine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009074 ! aromatic amino acid family catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: se
 creation_date: 2012-06-06T09:27:24Z
 
@@ -515516,14 +516738,15 @@
 
 [Term]
 id: GO:1900832
-name: D-leucine catabolic process
+name: obsolete D-leucine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-leucine." [GOC:TermGenie, PMID:10918062]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-leucine." [GOC:TermGenie, PMID:10918062]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-leucine.
 synonym: "D-leucine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-leucine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-leucine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009083 ! branched-chain amino acid catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: se
 creation_date: 2012-06-06T09:37:40Z
 
@@ -519793,16 +521016,15 @@
 
 [Term]
 id: GO:1901053
-name: sarcosine catabolic process
+name: obsolete sarcosine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of sarcosine." [GOC:TermGenie, GOC:yaf, PMID:17951379]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of sarcosine." [GOC:TermGenie, GOC:yaf, PMID:17951379]
+comment: This term was obsoleted because there is no evidence that a specific pathway for sarcosine degradation exists.
 synonym: "sarcosine breakdown" EXACT [GOC:TermGenie]
 synonym: "sarcosine catabolism" EXACT [GOC:TermGenie]
 synonym: "sarcosine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
-is_a: GO:1901052 ! sarcosine metabolic process
-is_a: GO:1901606 ! alpha-amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: yaf
 creation_date: 2012-06-28T04:52:29Z
 
@@ -527525,7 +528747,7 @@
 name: azole transmembrane transporter activity
 namespace: molecular_function
 alt_id: GO:0045118
-def: "Enables the directed movement of azoles, heterocyclic compound found in many biologically important substances, from one side of a membrane to the other." [GOC:go_curators, ISBN:3527307206, Wikipedia:Azole]
+def: "Enables the directed movement of azoles, heterocyclic compound found in many biologically important substances, from one side of a membrane to the other." [GOC:curators, ISBN:3527307206, Wikipedia:Azole]
 synonym: "azole transporter activity" RELATED []
 is_a: GO:0022857 ! transmembrane transporter activity
 relationship: part_of GO:0045117 ! azole transmembrane transport
@@ -527843,16 +529065,13 @@
 
 [Term]
 id: GO:1901494
-name: regulation of cysteine metabolic process
+name: obsolete regulation of cysteine metabolic process
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of cysteine metabolic process." [GOC:TermGenie]
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of cysteine metabolic process." [GOC:TermGenie]
+comment: This term was obsoleted because it was an unnecessray grouping term.
 synonym: "regulation of cysteine metabolism" EXACT [GOC:TermGenie]
-is_a: GO:0006521 ! regulation of amino acid metabolic process
-is_a: GO:0042762 ! regulation of sulfur metabolic process
-is_a: GO:0062012 ! regulation of small molecule metabolic process
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0006534 ! cysteine metabolic process
-relationship: regulates GO:0006534 ! cysteine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31281" xsd:anyURI
+is_obsolete: true
 created_by: vw
 creation_date: 2012-10-16T14:28:42Z
 
@@ -528870,7 +530089,6 @@
 namespace: biological_process
 def: "OBSOLETE. The chemical reactions and pathways involving organonitrogen compound." [GOC:pr, GOC:TermGenie]
 comment: The reason for obsoletion is that this term was an unnecessary grouping term.
-xref: Reactome:R-HSA-71291 "Metabolism of amino acids and derivatives"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/26424" xsd:anyURI
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27194" xsd:anyURI
 is_obsolete: true
@@ -530336,7 +531554,7 @@
 name: quinone metabolic process
 namespace: biological_process
 alt_id: GO:0042375
-def: "The chemical reactions and pathways involving quinone." [GOC:go_curators, GOC:pr, GOC:TermGenie]
+def: "The chemical reactions and pathways involving quinone." [GOC:curators, GOC:pr, GOC:TermGenie]
 synonym: "quinone cofactor metabolic process" RELATED []
 synonym: "quinone cofactor metabolism" RELATED []
 synonym: "quinone metabolism" EXACT [GOC:TermGenie]
@@ -530349,7 +531567,7 @@
 name: quinone catabolic process
 namespace: biological_process
 alt_id: GO:0042378
-def: "The chemical reactions and pathways resulting in the breakdown of quinone." [GOC:go_curators, GOC:pr, GOC:TermGenie]
+def: "The chemical reactions and pathways resulting in the breakdown of quinone." [GOC:curators, GOC:pr, GOC:TermGenie]
 synonym: "quinone breakdown" EXACT [GOC:TermGenie]
 synonym: "quinone catabolism" EXACT [GOC:TermGenie]
 synonym: "quinone cofactor breakdown" RELATED []
@@ -531282,8 +532500,12 @@
 synonym: "L-glutamine biosynthesis" EXACT [GOC:TermGenie]
 synonym: "L-glutamine formation" EXACT [GOC:TermGenie]
 synonym: "L-glutamine synthesis" EXACT [GOC:TermGenie]
+xref: MetaCyc:GLNSYN-PWY
 is_a: GO:0006541 ! glutamine metabolic process
 is_a: GO:0009084 ! L-glutamine family amino acid biosynthetic process
+property_value: skos:broadMatch MetaCyc:PWY-5505
+property_value: skos:narrowMatch MetaCyc:GLNSYN-PWY
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pde
 creation_date: 2012-12-13T23:32:52Z
 
@@ -531296,8 +532518,21 @@
 synonym: "L-isoleucine biosynthesis" EXACT [GOC:TermGenie]
 synonym: "L-isoleucine formation" EXACT [GOC:TermGenie]
 synonym: "L-isoleucine synthesis" EXACT [GOC:TermGenie]
+xref: MetaCyc:ILEUSYN-PWY
+xref: MetaCyc:PWY-5101
+xref: MetaCyc:PWY-5103
+xref: MetaCyc:PWY-5104
+xref: MetaCyc:PWY-5108
+is_a: GO:0006549 ! isoleucine metabolic process
 is_a: GO:0009067 ! aspartate family amino acid biosynthetic process
-is_a: GO:0009097 ! isoleucine biosynthetic process
+is_a: GO:0009082 ! branched-chain amino acid biosynthetic process
+property_value: skos:narrowMatch MetaCyc:ILEUSYN-PWY
+property_value: skos:narrowMatch MetaCyc:PWY-5101
+property_value: skos:narrowMatch MetaCyc:PWY-5103
+property_value: skos:narrowMatch MetaCyc:PWY-5104
+property_value: skos:narrowMatch MetaCyc:PWY-5108
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30848" xsd:anyURI
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31525" xsd:anyURI
 created_by: pde
 creation_date: 2012-12-13T23:32:56Z
 
@@ -533560,6 +534795,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving nicotinate." [GOC:TermGenie, GOC:yaf, UniPathway:UPA00830]
 synonym: "nicotinate metabolism" EXACT [GOC:TermGenie]
+xref: Reactome:R-HSA-196807 "Nicotinate metabolism"
 is_a: GO:0009820 ! alkaloid metabolic process
 is_a: GO:0032787 ! monocarboxylic acid metabolic process
 is_a: GO:0072524 ! pyridine-containing compound metabolic process
@@ -537728,15 +538964,15 @@
 
 [Term]
 id: GO:1902079
-name: D-valine catabolic process
+name: obsolete D-valine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of D-valine." [GOC:TermGenie, PMID:23085840]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of D-valine." [GOC:TermGenie, PMID:23085840]
+comment: This term was obsoleted because there is no evidence that there is a specific pathway to degrade D-valine.
 synonym: "D-valine breakdown" EXACT [GOC:TermGenie]
 synonym: "D-valine catabolism" EXACT [GOC:TermGenie]
 synonym: "D-valine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0009083 ! branched-chain amino acid catabolic process
-is_a: GO:0019478 ! D-amino acid catabolic process
-is_a: GO:1902114 ! D-valine metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: al
 creation_date: 2013-04-23T21:34:29Z
 
@@ -538446,12 +539682,13 @@
 
 [Term]
 id: GO:1902114
-name: D-valine metabolic process
+name: obsolete D-valine metabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways involving D-valine." [GOC:TermGenie, PMID:23085840]
+def: "OBSOLETE. The chemical reactions and pathways involving D-valine." [GOC:TermGenie, PMID:23085840]
+comment: This term was obsoleted because it is an unnecessary grouping term.
 synonym: "D-valine metabolism" EXACT [GOC:TermGenie]
-is_a: GO:0006573 ! valine metabolic process
-is_a: GO:0046416 ! D-amino acid metabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31357" xsd:anyURI
+is_obsolete: true
 created_by: al
 creation_date: 2013-05-13T23:27:23Z
 
@@ -540295,6 +541532,7 @@
 namespace: biological_process
 def: "The chemical reactions and pathways involving ketone body." [GOC:pr, GOC:TermGenie]
 synonym: "ketone body metabolism" EXACT [GOC:TermGenie]
+xref: Reactome:R-HSA-74182 "Ketone body metabolism"
 is_a: GO:0006091 ! generation of precursor metabolites and energy
 is_a: GO:0044281 ! small molecule metabolic process
 is_a: GO:1901568 ! fatty acid derivative metabolic process
@@ -544318,6 +545556,7 @@
 name: chloride transmembrane transport
 namespace: biological_process
 def: "The process in which chloride is transported across a membrane." [GOC:TermGenie, GOC:vw]
+xref: Reactome:R-HSA-426117 "Cation-coupled Chloride cotransporters"
 is_a: GO:0006821 ! chloride transport
 is_a: GO:0098656 ! monoatomic anion transmembrane transport
 created_by: bf
@@ -548947,8 +550186,7 @@
 synonym: "sulfurated eukaryotic molybdenum cofactor(2-) formation" EXACT [GOC:TermGenie]
 synonym: "sulfurated eukaryotic molybdenum cofactor(2-) synthesis" EXACT [GOC:TermGenie]
 xref: MetaCyc:PWY-5963
-is_a: GO:0006793 ! phosphorus metabolic process
-is_a: GO:0009058 ! biosynthetic process
+is_a: GO:0006777 ! Mo-molybdopterin cofactor biosynthetic process
 relationship: has_part GO:0008265 ! molybdenum cofactor sulfurtransferase activity
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28388" xsd:anyURI
 created_by: dph
@@ -549009,6 +550247,7 @@
 synonym: "Mo-molybdopterin-cytosine-dinucleotide cofactor biosynthetic process" EXACT []
 xref: MetaCyc:PWY-6476
 is_a: GO:0006777 ! Mo-molybdopterin cofactor biosynthetic process
+is_a: GO:0009165 ! nucleotide biosynthetic process
 relationship: has_part GO:0061602 ! molybdenum cofactor cytidylyltransferase activity
 created_by: dph
 creation_date: 2014-03-06T14:00:43Z
@@ -552035,6 +553274,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0046165 ! alcohol biosynthetic process
 relationship: regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:03Z
 
@@ -552043,6 +553283,7 @@
 name: negative regulation of alcohol biosynthetic process
 namespace: biological_process
 def: "Any process that stops, prevents or reduces the frequency, rate or extent of alcohol biosynthetic process." [GO_REF:0000058, GOC:mengo_curators, GOC:TermGenie, PMID:23332010]
+subset: gocheck_do_not_annotate
 synonym: "down regulation of alcohol anabolism" EXACT [GOC:TermGenie]
 synonym: "down regulation of alcohol biosynthesis" EXACT [GOC:TermGenie]
 synonym: "down regulation of alcohol biosynthetic process" EXACT [GOC:TermGenie]
@@ -552074,6 +553315,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: negatively_regulates GO:0046165 ! alcohol biosynthetic process
 relationship: negatively_regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:13Z
 
@@ -552082,6 +553324,7 @@
 name: positive regulation of alcohol biosynthetic process
 namespace: biological_process
 def: "Any process that activates or increases the frequency, rate or extent of alcohol biosynthetic process." [GO_REF:0000058, GOC:mengo_curators, GOC:TermGenie, PMID:23332010]
+subset: gocheck_do_not_annotate
 synonym: "activation of alcohol anabolism" NARROW [GOC:TermGenie]
 synonym: "activation of alcohol biosynthesis" NARROW [GOC:TermGenie]
 synonym: "activation of alcohol biosynthetic process" NARROW [GOC:TermGenie]
@@ -552113,6 +553356,7 @@
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: positively_regulates GO:0046165 ! alcohol biosynthetic process
 relationship: positively_regulates GO:0046165 ! alcohol biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31485" xsd:anyURI
 created_by: tt
 creation_date: 2014-04-22T21:55:19Z
 
@@ -552160,6 +553404,7 @@
 namespace: molecular_function
 def: "Binding to phosphatidylinositol bisphosphate." [GO_REF:0000067, GOC:bhm, GOC:TermGenie, PMID:18690034]
 comment: An example of this is KCNJ2 in human (P63252) in PMID:18690034 (inferred from direct assay)
+xref: Reactome:R-HSA-8870332 "PTPN13:PLEKHA1,2 bind PI(3,4)P2"
 is_a: GO:0043168 ! anion binding
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27059" xsd:anyURI
 created_by: bhm
@@ -552349,7 +553594,6 @@
 synonym: "metalloendoproteinase activity involved in amyloid precursor protein degradation" NARROW [GOC:TermGenie]
 synonym: "metalloendoproteinase activity involved in APP catabolic process" NARROW [GOC:TermGenie]
 synonym: "metalloendoproteinase activity involved in APP catabolism" NARROW [GOC:TermGenie]
-xref: Reactome:R-HSA-9010034 "ADAM10:Zn2+:TSPANs cleaves APP(18-770)"
 is_a: GO:0004222 ! metalloendopeptidase activity
 intersection_of: GO:0004222 ! metalloendopeptidase activity
 intersection_of: part_of GO:0042987 ! amyloid precursor protein catabolic process
@@ -552593,7 +553837,6 @@
 synonym: "upregulation of TPK I" RELATED [GOC:TermGenie]
 synonym: "upregulation of TPK II" RELATED [GOC:TermGenie]
 synonym: "upregulation of TTK" RELATED [GOC:TermGenie]
-xref: Reactome:R-HSA-9032845 "Activated NTRK2 signals through CDK5"
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/22455" xsd:anyURI
 is_obsolete: true
 created_by: sjp
@@ -552700,31 +553943,25 @@
 
 [Term]
 id: GO:1902954
-name: regulation of early endosome to recycling endosome transport
+name: obsolete regulation of early endosome to recycling endosome transport
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
-is_a: GO:0032386 ! regulation of intracellular transport
-is_a: GO:0060627 ! regulation of vesicle-mediated transport
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0061502 ! early endosome to recycling endosome transport
-relationship: regulates GO:0061502 ! early endosome to recycling endosome transport
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
+comment: This term is described by another GO term
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:2001135
 created_by: sjp
 creation_date: 2014-05-02T09:06:23Z
 
 [Term]
 id: GO:1902955
-name: positive regulation of early endosome to recycling endosome transport
+name: obsolete positive regulation of early endosome to recycling endosome transport
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
-synonym: "activation of early endosome to recycling endosome transport" NARROW [GOC:TermGenie]
-synonym: "up regulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-synonym: "up-regulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-synonym: "upregulation of early endosome to recycling endosome transport" EXACT [GOC:TermGenie]
-is_a: GO:0032388 ! positive regulation of intracellular transport
-is_a: GO:1902954 ! regulation of early endosome to recycling endosome transport
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0061502 ! early endosome to recycling endosome transport
-relationship: positively_regulates GO:0061502 ! early endosome to recycling endosome transport
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of early endosome to recycling endosome transport." [GO_REF:0000058, GOC:sjp, GOC:TermGenie, PMID:22621900]
+comment: This term is described by another GO term.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31240" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:2001137
 created_by: sjp
 creation_date: 2014-05-02T09:06:29Z
 
@@ -558015,13 +559252,16 @@
 
 [Term]
 id: GO:1903212
-name: protein localization to mating-type region heterochromatin
+name: obsolete protein localization to mating-type region heterochromatin
 namespace: biological_process
-def: "A process in which a protein is transported to, or maintained in, a location within a mating-type region heterochromatin." [GO_REF:0000087, GOC:TermGenie, PMID:18761674]
+def: "OBSOLETE. A process in which a protein is transported to, or maintained in, a location within a mating-type region heterochromatin." [GO_REF:0000087, GOC:TermGenie, PMID:18761674]
+comment: This term was obsoleted because it represents a precomposed term.
 synonym: "protein localisation in mating-type region heterochromatin" EXACT [GOC:TermGenie]
 synonym: "protein localisation to mating-type region heterochromatin" EXACT [GOC:TermGenie]
 synonym: "protein localization in mating-type region heterochromatin" EXACT [GOC:TermGenie]
-is_a: GO:0097355 ! protein localization to heterochromatin
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0071168
 created_by: mah
 creation_date: 2014-07-24T09:38:12Z
 
@@ -564778,15 +566018,14 @@
 
 [Term]
 id: GO:1903559
-name: 3-cyano-L-alanine catabolic process
+name: obsolete 3-cyano-L-alanine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of 3-cyano-L-alanine." [GO_REF:0000068, GOC:kmv, GOC:TermGenie, PMID:24100226, PMID:24843024]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of 3-cyano-L-alanine." [GO_REF:0000068, GOC:kmv, GOC:TermGenie, PMID:24100226, PMID:24843024]
+comment: This term was obsoleted because there is no evidence that a specific pathway for 3-cyano-L-alanine degradation exists.
 synonym: "3-cyano-L-alanine breakdown" EXACT [GOC:TermGenie]
 synonym: "3-cyano-L-alanine catabolism" EXACT [GOC:TermGenie]
 synonym: "3-cyano-L-alanine degradation" EXACT [GOC:TermGenie]
-is_a: GO:0050899 ! nitrile catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+is_obsolete: true
 created_by: kmv
 creation_date: 2014-10-21T17:13:10Z
 
@@ -589048,6 +590287,8 @@
 name: all-trans-retinol binding
 namespace: molecular_function
 def: "Binding to all-trans-retinol." [GO_REF:0000067, GOC:kmv, GOC:TermGenie, PMID:19828452]
+xref: Reactome:R-HSA-2404135 "RBP4 binds atROL"
+xref: Reactome:R-HSA-975633 "atROL binds to RBP2"
 is_a: GO:0019841 ! retinol binding
 created_by: kmv
 creation_date: 2015-10-27T21:14:47Z
@@ -604651,47 +605892,45 @@
 
 [Term]
 id: GO:1905526
-name: regulation of Golgi lumen acidification
+name: obsolete regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that modulates the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
-is_a: GO:0032847 ! regulation of cellular pH reduction
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: regulates GO:0061795 ! Golgi lumen acidification
-relationship: regulates GO:0061795 ! Golgi lumen acidification
+def: "OBSOLETE. Any process that modulates the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant. Direction is unspecified.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+consider: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:25Z
 
 [Term]
 id: GO:1905527
-name: negative regulation of Golgi lumen acidification
+name: obsolete negative regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that stops, prevents or reduces the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+def: "OBSOLETE. Any process that stops, prevents or reduces the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant.
 synonym: "down regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "down-regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "downregulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "inhibition of Golgi lumen acidification" NARROW [GOC:TermGenie]
-is_a: GO:0032848 ! negative regulation of cellular pH reduction
-is_a: GO:1905526 ! regulation of Golgi lumen acidification
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: negatively_regulates GO:0061795 ! Golgi lumen acidification
-relationship: negatively_regulates GO:0061795 ! Golgi lumen acidification
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:33Z
 
 [Term]
 id: GO:1905528
-name: positive regulation of Golgi lumen acidification
+name: obsolete positive regulation of Golgi lumen acidification
 namespace: biological_process
-def: "Any process that activates or increases the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+def: "OBSOLETE. Any process that activates or increases the frequency, rate or extent of Golgi lumen acidification." [GO_REF:0000058, GOC:dph, GOC:TermGenie, PMID:23447592]
+comment: This term was added in error. Regulating regulation is redundant.
 synonym: "activation of Golgi lumen acidification" NARROW [GOC:TermGenie]
 synonym: "up regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "up-regulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
 synonym: "upregulation of Golgi lumen acidification" EXACT [GOC:TermGenie]
-is_a: GO:0032849 ! positive regulation of cellular pH reduction
-is_a: GO:1905526 ! regulation of Golgi lumen acidification
-intersection_of: GO:0065007 ! biological regulation
-intersection_of: positively_regulates GO:0061795 ! Golgi lumen acidification
-relationship: positively_regulates GO:0061795 ! Golgi lumen acidification
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29164" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0061795
 created_by: dph
 creation_date: 2016-10-05T13:16:41Z
 
@@ -614860,6 +616099,8 @@
 namespace: molecular_function
 def: "Catalysis of the removal of the cap from an unmethylated 5'-end capped RNA resulting in the release of the entire cap structure (GpppN) and a 5' monophosphorylated RNA." [GOC:dgf, PMID:20802481]
 synonym: "G(5')pppN pyrophosphatase activity" RELATED []
+xref: Reactome:R-HSA-9935300 "DXO decaps NAD-capped RNA (NAD-RNA)"
+xref: Reactome:R-HSA-9935319 "DXO decaps incompletely capped RNA (GpppN-RNA)"
 is_a: GO:0016891 ! RNA endonuclease activity producing 5'-phosphomonoesters, hydrolytic mechanism
 created_by: tb
 creation_date: 2013-08-26T17:51:00Z
@@ -617749,7 +618990,7 @@
 id: GO:1990451
 name: cellular stress response to acidic pH
 namespace: biological_process
-def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in the homeostasis of organismal or cellular pH (with pH < 7). pH is a measure of the acidity or basicity of an aqueous solution." [GOC:BHF, GOC:go_curators, GOC:rl, PMID:10615049]
+def: "Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a disturbance in the homeostasis of organismal or cellular pH (with pH < 7). pH is a measure of the acidity or basicity of an aqueous solution." [GOC:BHF, GOC:curators, GOC:rl, PMID:10615049]
 comment: An example of this is NOX1 in human (Q9Y5S8) in PMID:10615049.
 synonym: "cellular stress response to acidity" BROAD []
 is_a: GO:0062197 ! cellular response to chemical stress
@@ -621301,9 +622542,10 @@
 id: GO:1990780
 name: cytoplasmic side of dendritic spine plasma membrane
 namespace: cellular_component
-def: "The leaflet of the plasma membrane that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface surrounding a dendritic spine." [PMID:9275233]
+def: "The leaflet of the plasma membrane in the dendritic spine region that faces the cytoplasm, including any protein embedded in, attached to, or peripherally associated with it." [PMID:9275233]
 is_a: GO:0009898 ! cytoplasmic side of plasma membrane
 relationship: part_of GO:0032591 ! dendritic spine membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31515" xsd:anyURI
 created_by: sl
 creation_date: 2015-06-23T18:35:39Z
 
@@ -621719,8 +622961,11 @@
 synonym: "RNA adenylyltransferase activity" EXACT []
 xref: EC:2.7.7.19
 xref: MetaCyc:POLYNUCLEOTIDE-ADENYLYLTRANSFERASE-RXN
-xref: Reactome:R-HSA-72185 "mRNA polyadenylation"
 xref: Reactome:R-HSA-9833031 "Polyadenylation of respiratory syncytial virus subgenomic positive-sense mRNAs"
+xref: Reactome:R-HSA-9936998 "MTPAP polyadenylates mitochondrial mRNA"
+xref: Reactome:R-HSA-9970189 "PABPN1-stimulated PAPOL synthesizes polyA tail at distal 3'UTR PAS"
+xref: Reactome:R-HSA-9970318 "PABPN1-stimulated PAPOL synthesizes polyA tail at proximal 3'UTR PAS"
+xref: Reactome:R-HSA-9970320 "PABPN1-stimulated PAPOL synthesizes polyA tail at intronic PAS"
 xref: RHEA:11332
 is_a: GO:0070566 ! adenylyltransferase activity
 property_value: skos:exactMatch EC:2.7.7.19
@@ -622461,6 +623706,7 @@
 def: "Catalysis of the reaction: a cytidine in 18S rRNA + acetyl-CoA + ATP + H2O = an N(4)-acetylcytidine in 18S rRNA + ADP + phosphate + CoA + H+." [PMID:25402480, RHEA:51424]
 xref: RHEA:51424
 is_a: GO:0008080 ! N-acetyltransferase activity
+is_a: GO:0140102 ! catalytic activity, acting on a rRNA
 property_value: skos:exactMatch RHEA:51424
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29506" xsd:anyURI
 created_by: al
@@ -622746,7 +623992,8 @@
 name: response to psychosocial stress
 namespace: biological_process
 def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of exposure to aversive or demanding psychological and social conditions that tax or exceed the behavioral resources of the organism." [PMID:22922217, PMID:26458179]
-is_a: GO:0006950 ! response to stress
+is_a: GO:0033555 ! multicellular organismal response to stress
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31407" xsd:anyURI
 created_by: sl
 creation_date: 2015-12-07T20:38:35Z
 
@@ -623091,12 +624338,15 @@
 
 [Term]
 id: GO:1990943
-name: mating type region replication fork barrier binding
+name: obsolete mating type region replication fork barrier binding
 namespace: molecular_function
-def: "Binding to the replication fork barrier found in the mating type region of fission yeast." [PMID:18723894]
+def: "OBSOLETE. Binding to the replication fork barrier found in the mating type region of fission yeast." [PMID:18723894]
+comment: This term was obsoleted because it represents a precomposed term.
 synonym: "RTS1 barrier binding" EXACT []
 synonym: "RTS1 element binding" EXACT []
-is_a: GO:0031634 ! replication fork barrier binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31529" xsd:anyURI
+is_obsolete: true
+replaced_by: GO:0031634
 created_by: mah
 creation_date: 2016-04-06T16:08:43Z
 
@@ -624154,6 +625404,7 @@
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of cell-cell adhesion mediated by cadherin." [GOC:obol]
 xref: Reactome:R-HSA-9764260 "Regulation of Expression and Function of Type II Classical Cadherins"
+xref: Reactome:R-HSA-9764274 "Regulation of Expression and Function of Type I Classical Cadherins"
 is_a: GO:0022407 ! regulation of cell-cell adhesion
 intersection_of: GO:0065007 ! biological regulation
 intersection_of: regulates GO:0044331 ! cell-cell adhesion mediated by cadherin
@@ -640593,6 +641844,7 @@
 namespace: biological_process
 def: "Any process that modulates the frequency, rate or extent of apoptotic signaling pathway." [GOC:mtg_apoptosis]
 synonym: "regulation of apoptotic signalling pathway" EXACT [GOC:mah]
+xref: Reactome:R-HSA-205025 "NADE modulates death signalling"
 is_a: GO:0009966 ! regulation of signal transduction
 is_a: GO:0042981 ! regulation of apoptotic process
 intersection_of: GO:0065007 ! biological regulation
@@ -641552,15 +642804,15 @@
 
 [Term]
 id: GO:2001297
-name: N(omega)-methyl-L-arginine catabolic process
+name: obsolete N(omega)-methyl-L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of N(omega)-methyl-L-arginine." [GOC:rs, PMID:10510241]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of N(omega)-methyl-L-arginine." [GOC:rs, PMID:10510241]
+comment: The reason for obsoletion is that there is no evidence that this function/process/component exists.
 synonym: "N(omega)-methyl-L-arginine breakdown" EXACT [GOC:obol]
 synonym: "N(omega)-methyl-L-arginine catabolism" EXACT [GOC:obol]
 synonym: "N(omega)-methyl-L-arginine degradation" EXACT [GOC:obol]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: pr
 creation_date: 2012-02-14T03:38:21Z
 
@@ -641578,15 +642830,15 @@
 
 [Term]
 id: GO:2001299
-name: N(omega),N(omega)-dimethyl-L-arginine catabolic process
+name: obsolete N(omega),N(omega)-dimethyl-L-arginine catabolic process
 namespace: biological_process
-def: "The chemical reactions and pathways resulting in the breakdown of N(omega),N(omega)-dimethyl-L-arginine, a methyl-L-arginine having two methyl groups both attached to the primary amino moiety of the guanidino group." [GOC:rs, PMID:10510241]
+def: "OBSOLETE. The chemical reactions and pathways resulting in the breakdown of N(omega),N(omega)-dimethyl-L-arginine, a methyl-L-arginine having two methyl groups both attached to the primary amino moiety of the guanidino group." [GOC:rs, PMID:10510241]
+comment: The reason for obsoletion is that there is no evidence that this function/process/component exists.
 synonym: "N(omega),N(omega)-dimethyl-L-arginine breakdown" EXACT [GOC:obol]
 synonym: "N(omega),N(omega)-dimethyl-L-arginine catabolism" EXACT [GOC:obol]
 synonym: "N(omega),N(omega)-dimethyl-L-arginine degradation" EXACT [GOC:obol]
-is_a: GO:0042219 ! modified amino acid catabolic process
-is_a: GO:0170035 ! L-amino acid catabolic process
-is_a: GO:0170044 ! non-proteinogenic amino acid catabolic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31361" xsd:anyURI
+is_obsolete: true
 created_by: pr
 creation_date: 2012-02-14T03:40:03Z
 
@@ -641911,8 +643163,10 @@
 id: GO:7770002
 name: sodium:ammonium:chloride symporter activity
 namespace: molecular_function
-def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + NH4+(out) + Cl-(out) = Na+(in) + NH4+(in) + Cl-(in)." [PMID:12657561]
+def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + NH4+(out) + Cl-(out) = Na+(in) + NH4+(in) + Cl-(in)." [PMID:12657561, RHEA:85511]
+xref: RHEA:85511
 is_a: GO:0015378 ! sodium:chloride symporter activity
+property_value: skos:exactMatch RHEA:85511
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30468" xsd:anyURI
 created_by: dragon-ai-agent
 creation_date: 2025-06-19T23:39:51Z
@@ -641993,9 +643247,11 @@
 id: GO:7770009
 name: L-tryptophan conjugated cholate hydrolase activity
 namespace: molecular_function
-def: "Catalysis of the reaction: cholate + L-tryptophan = L-tryptophocholate + H2O." [PMID:40446798]
+def: "Catalysis of the reaction: cholate + L-tryptophan = L-tryptophocholate + H2O." [PMID:40446798, RHEA:85423]
 xref: CHEBI:234456
+xref: RHEA:85423
 is_a: GO:7770003 ! amino acid conjugated cholate hydrolase activity
+property_value: skos:exactMatch RHEA:85423
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30487" xsd:anyURI
 created_by: dragon-ai-agent
 creation_date: 2025-06-25T00:18:58Z
@@ -642313,6 +643569,196 @@
 property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30887" xsd:anyURI
 created_by: dragon-ai-agent
 
+[Term]
+id: GO:7770038
+name: melanosomal lumen acidification
+namespace: biological_process
+def: "Any process that reduces the pH of the melanosomal lumen, measured by the concentration of the hydrogen ion." [PMID:23205854, PMID:35469906]
+synonym: "melanosome pH reduction" EXACT []
+is_a: GO:0051452 ! intracellular pH reduction
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28838" xsd:anyURI
+created_by: dragon-ai-agent
+
+[Term]
+id: GO:7770039
+name: melanosomal lumen pH elevation
+namespace: biological_process
+def: "Any process that increases the pH of the melanosomal lumen, measured by the concentration of the hydrogen ion." [PMID:11310796, PMID:27140606, PMID:32966160, PMID:35469906]
+synonym: "melanosome pH elevation" EXACT []
+is_a: GO:0051454 ! intracellular pH elevation
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28838" xsd:anyURI
+created_by: dragon-ai-agent
+
+[Term]
+id: GO:7770040
+name: epiplastidial membrane
+namespace: cellular_component
+def: "The outermost lipid bilayer surrounding complex plastids in organisms that acquired their plastids through secondary endosymbiosis. This membrane is derived from the host cell's endomembrane system and is continuous with the rough endoplasmic reticulum and the outer nuclear envelope. In some species, the cytoplasmic face may be studded with ribosomes." [PMID:15937721, PMID:27179349, PMID:39383259]
+synonym: "cERM" EXACT []
+synonym: "chloroplast ER membrane" EXACT []
+synonym: "epiplastid rough ER" EXACT []
+synonym: "EpM" EXACT []
+synonym: "EPrER" EXACT []
+is_a: GO:0042170 ! plastid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31421" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:11:34Z
+
+[Term]
+id: GO:7770041
+name: periplastidial membrane
+namespace: cellular_component
+def: "The lipid bilayer that forms the second outermost membrane of complex plastids in organisms that acquired their plastids through secondary endosymbiosis. The periplastidial membrane is evolutionarily derived from the plasma membrane of the engulfed algal endosymbiont and together with the outer envelope membrane of the plastid defines the periplastidial compartment." [PMID:15937721, PMID:27179349, PMID:39383259]
+synonym: "PPM" EXACT []
+is_a: GO:0042170 ! plastid membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31421" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:11:34Z
+
+[Term]
+id: GO:7770042
+name: bindosome complex
+namespace: cellular_component
+def: "A membrane-associated protein complex found in archaea that consists of sugar binding proteins assembled via a type IV pilin-like assembly system. The complex is located at the cell surface and functions in sugar uptake by facilitating the interaction between solute binding proteins and ABC transporters. The complex forms high molecular mass structures (400-600 kDa) that are integrated within or closely associated with the S-layer." [PMID:21234771, PMID:38877064]
+is_a: GO:0098796 ! membrane protein complex
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31422" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-29T01:57:42Z
+
+[Term]
+id: GO:7770043
+name: lipid chaperone activity
+namespace: molecular_function
+def: "Binding to a lipid and delivering it to an acceptor protein, including enzymes, nuclear receptors, and other lipid-binding proteins. The lipid may be presented while bound to the chaperone for enzymatic modification or signaling, or fully transferred to the acceptor protein." [PMID:16216074, PMID:18511927, PMID:25847235, PMID:26658141]
+is_a: GO:0005319 ! lipid carrier activity
+relationship: has_part GO:0008289 ! lipid binding
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31403" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-30T01:54:24Z
+
+[Term]
+id: GO:7770044
+name: phycoerythrobilin synthase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: biliverdin IXalpha + 2 reduced ferredoxin + 4 H+ = (3Z)-phycoerythrobilin + 2 oxidized ferredoxin." [PMID:18662988, RHEA:25359]
+xref: EC:1.3.7.6
+xref: RHEA:25359
+is_a: GO:0016636 ! oxidoreductase activity, acting on the CH-CH group of donors, iron-sulfur protein as acceptor
+property_value: skos:exactMatch EC:1.3.7.6
+property_value: skos:exactMatch RHEA:25359
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31443" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-01-30T04:39:58Z
+
+[Term]
+id: GO:7770045
+name: negative regulation of GCN2-mediated signaling
+namespace: biological_process
+def: "Any process that stops, prevents, or reduces the frequency, rate or extent of GCN2-mediated signaling." [PMID:23899355]
+is_a: GO:0080135 ! regulation of cellular response to stress
+is_a: GO:1902532 ! negative regulation of intracellular signal transduction
+intersection_of: GO:0065007 ! biological regulation
+intersection_of: negatively_regulates GO:0140469 ! GCN2-mediated signaling
+relationship: negatively_regulates GO:0140469 ! GCN2-mediated signaling
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31453" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-04T18:20:09Z
+
+[Term]
+id: GO:7770046
+name: obsolete N6-methyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6-methyladenosine = ADP + H+ + N6-methyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770047
+name: obsolete N6,N6-dimethyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6,N6-dimethyladenosine = ADP + H+ + N6,N6-dimethyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770048
+name: obsolete N6-isopentenyladenosine kinase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: ATP + N6-isopentenyladenosine = ADP + H+ + N6-isopentenyl-AMP." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770049
+name: obsolete N6,N6-dimethyl-AMP deaminase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: H2O + H+ + N6,N6-dimethyl-AMP = IMP + dimethylamine." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770050
+name: obsolete N6-isopentenyl-AMP deaminase activity
+namespace: molecular_function
+def: "OBSOLETE. Catalysis of the reaction: H2O + H+ + N6-isopentenyl-AMP = IMP + dimethylallylamine." [PMID:40840445]
+comment: The reason for obsoletion is that this term was added in error.
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/30800" xsd:anyURI
+is_obsolete: true
+created_by: dragon-ai-agent
+creation_date: 2026-02-09T05:59:37Z
+
+[Term]
+id: GO:7770051
+name: mitochondrial-lysosomal membrane contact site
+namespace: cellular_component
+def: "An organelle membrane contact site between the mitochondrial outer membrane and the lysosomal membrane." [PMID:29364868, PMID:30898429, PMID:32703809, PMID:33372681, PMID:33753743, PMID:35249745]
+synonym: "lysosome-mitochondrion membrane contact site" EXACT []
+synonym: "mitochondria-lysosome contact site" EXACT []
+synonym: "mitochondria-lysosome MCS" EXACT []
+synonym: "mitochondria-lysosome membrane contact site" EXACT []
+is_a: GO:1990816 ! vacuole-mitochondrion membrane contact site
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31553" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-11T18:32:29Z
+
+[Term]
+id: GO:7770052
+name: endoplasmic reticulum-lysosome membrane contact site
+namespace: cellular_component
+def: "An organelle membrane contact site between the endoplasmic reticulum membrane and the lysosomal membrane." [PMID:31537798, PMID:39370902]
+synonym: "ER-lysosome contact site" EXACT []
+synonym: "ER-lysosome MCS" EXACT []
+synonym: "lysosome-ER membrane contact site" EXACT []
+is_a: GO:0098853 ! endoplasmic reticulum-vacuole membrane contact site
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31541" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-12T00:09:12Z
+
+[Term]
+id: GO:7770053
+name: nucleotide-binding leucine-rich repeat receptor complex
+namespace: cellular_component
+def: "A protein complex that contains one or more nucleotide-binding leucine-rich repeat (NLR) proteins and functions as an intracellular innate immune receptor. Upon activation by pathogen effectors or danger signals, the complex oligomerizes with adaptor and effector proteins to trigger downstream immune responses including programmed cell death." [PMID:26355215, PMID:30948526, PMID:30948527]
+synonym: "NLR complex" EXACT []
+synonym: "NLR receptor complex" EXACT []
+synonym: "NOD-like receptor complex" RELATED []
+is_a: GO:0043235 ! receptor complex
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/31604" xsd:anyURI
+created_by: dragon-ai-agent
+creation_date: 2026-02-19T17:46:49Z
+
 [Typedef]
 id: ends_during
 name: ends during
```
